<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">PLoS Clin Trials</journal-id><journal-id journal-id-type="iso-abbrev">PLoS Clin Trials</journal-id><journal-id journal-id-type="publisher-id">pctr</journal-id><journal-id journal-id-type="publisher-id">plct</journal-id><journal-id journal-id-type="pmc">plosclin</journal-id><journal-title-group><journal-title>PLoS Clinical Trials</journal-title></journal-title-group><issn pub-type="epub">1555-5887</issn><publisher><publisher-name>Public Library of Science</publisher-name><publisher-loc>San Francisco, USA</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">16878178</article-id><article-id pub-id-type="pmc">1523224</article-id><article-id pub-id-type="doi">10.1371/journal.pctr.0010018</article-id><article-id pub-id-type="publisher-id">05-PLCT-CT-00005R4</article-id><article-id pub-id-type="sici">plct-01-03-05</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group><subj-group subj-group-type="Discipline"><subject>HIV/AIDS</subject><subject>Infectious Diseases</subject><subject>Pathology</subject></subj-group></article-categories><title-group><article-title>A Randomised Trial Comparing Genotypic and Virtual Phenotypic Interpretation of HIV Drug Resistance: The CREST Study</article-title><alt-title alt-title-type="running-head">The CREST study</alt-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Hales</surname><given-names>Gillian</given-names></name><xref ref-type="aff" rid="aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Birch</surname><given-names>Chris</given-names></name><xref ref-type="aff" rid="aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Crowe</surname><given-names>Suzanne</given-names></name><xref ref-type="aff" rid="aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Workman</surname><given-names>Cassy</given-names></name><xref ref-type="aff" rid="aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Hoy</surname><given-names>Jennifer F</given-names></name><xref ref-type="aff" rid="aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Law</surname><given-names>Matthew G</given-names></name><xref ref-type="aff" rid="aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Kelleher</surname><given-names>Anthony D</given-names></name><xref ref-type="aff" rid="aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Lincoln</surname><given-names>Douglas</given-names></name><xref ref-type="aff" rid="aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Emery</surname><given-names>Sean</given-names></name><xref ref-type="aff" rid="aff1">1</xref><xref ref-type="corresp" rid="cor1">*</xref></contrib><on-behalf-of>on behalf of the CREST investigators</on-behalf-of></contrib-group><aff id="aff1"><label>1</label>National Centre in HIV Epidemiology and Clinical Research, University of New South Wales, Sydney, Australia
</aff><aff id="aff2"><label>2</label>Victorian Infectious Diseases Reference Laboratory, North Melbourne, Australia
</aff><aff id="aff3"><label>3</label>Burnet Centre, Melbourne, Australia
</aff><aff id="aff4"><label>4</label>AIDS Research Initiative, Sydney, Australia
</aff><aff id="aff5"><label>5</label>Department of Medicine Alfred Hospital, Monash University, Melbourne, Australia
</aff><author-notes><corresp id="cor1">* To whom correspondence should be addressed. E-mail: <email>semery@nchecr.unsw.edu.au</email></corresp></author-notes><pub-date pub-type="collection"><month>7</month><year>2006</year></pub-date><pub-date pub-type="epub"><day>28</day><month>7</month><year>2006</year></pub-date><volume>1</volume><issue>3</issue><elocation-id>e18</elocation-id><history><date date-type="received"><day>9</day><month>12</month><year>2005</year></date><date date-type="accepted"><day>21</day><month>6</month><year>2006</year></date></history><permissions><copyright-statement> © 2006 Hales et al.</copyright-statement><copyright-year>2006</copyright-year><license xlink:href="http://creativecommons.org/licenses/by/4.0/"><license-p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</license-p></license></permissions><abstract><sec id="st1"><title><offsets xml_i="4670" xml_f="4681" txt_i="11" txt_f="22">Objectives:</offsets></title><p><offsets xml_i="4692" xml_f="4886" txt_i="23" txt_f="217">The aim of this study was to compare the efficacy of different HIV drug resistance test reports (genotype and virtual phenotype) in patients who were changing their antiretroviral therapy (ART).</offsets></p></sec><sec id="st2"><title><offsets xml_i="4917" xml_f="4924" txt_i="219" txt_f="226">Design:</offsets></title><p><offsets xml_i="4935" xml_f="4986" txt_i="227" txt_f="278">Randomised, open-label trial with 48-week followup.</offsets></p></sec><sec id="st3"><title><offsets xml_i="5017" xml_f="5025" txt_i="280" txt_f="288">Setting:</offsets></title><p><offsets xml_i="5036" xml_f="5130" txt_i="289" txt_f="383">The study was conducted in a network of primary healthcare sites in Australia and New Zealand.</offsets></p></sec><sec id="st4"><title><offsets xml_i="5161" xml_f="5174" txt_i="385" txt_f="398">Participants:</offsets></title><p><offsets xml_i="5185" xml_f="5491" txt_i="399" txt_f="699">Patients failing current ART with plasma HIV RNA &gt; 2000 copies/mL who wished to change their current ART were eligible. Subjects were required to be &gt; 18 years of age, previously treated with ART, have no intercurrent illnesses requiring active therapy, and to have provided written informed consent.</offsets></p></sec><sec id="st5"><title><offsets xml_i="5522" xml_f="5536" txt_i="701" txt_f="715">Interventions:</offsets></title><p><offsets xml_i="5547" xml_f="5720" txt_i="716" txt_f="889">Eligible subjects were randomly assigned to receive a genotype (group A) or genotype plus virtual phenotype (group B) prior to selection of their new antiretroviral regimen.</offsets></p></sec><sec id="st6"><title><offsets xml_i="5751" xml_f="5768" txt_i="891" txt_f="908">Outcome Measures:</offsets></title><p><offsets xml_i="5779" xml_f="5949" txt_i="909" txt_f="1079">Patient groups were compared for patterns of ART selection and surrogate outcomes (plasma viral load and CD4 counts) on an intention-to-treat basis over a 48-week period.</offsets></p></sec><sec id="st7"><title><offsets xml_i="5980" xml_f="5988" txt_i="1081" txt_f="1089">Results:</offsets></title><p><offsets xml_i="5999" xml_f="6050" txt_i="1090" txt_f="1141">Three hundred and twenty seven patients completing </offsets><underline><offsets xml_i="6061" xml_f="6065" txt_i="1141" txt_f="1142">&gt;</offsets></underline><offsets xml_i="6077" xml_f="6238" txt_i="1142" txt_f="1303"> one month of followup were included in these analyses. Resistance tests were the primary means by which ART regimens were selected (group A: 64%, group B: 62%; </offsets><italic><offsets xml_i="6246" xml_f="6247" txt_i="1303" txt_f="1304">p</offsets></italic><offsets xml_i="6256" xml_f="6440" txt_i="1304" txt_f="1488"> = 0.32). At 48 weeks, there were no significant differences between the groups for mean change from baseline plasma HIV RNA (group A: 0.68 log copies/mL, group B: 0.58 log copies/mL; </offsets><italic><offsets xml_i="6448" xml_f="6449" txt_i="1488" txt_f="1489">p</offsets></italic><offsets xml_i="6458" xml_f="6534" txt_i="1489" txt_f="1565"> = 0.23) and mean change from baseline CD4+ cell count (group A: 37 cells/mm</offsets><sup><offsets xml_i="6539" xml_f="6540" txt_i="1565" txt_f="1566">3</offsets></sup><offsets xml_i="6546" xml_f="6568" txt_i="1566" txt_f="1588">, group B: 50 cells/mm</offsets><sup><offsets xml_i="6573" xml_f="6574" txt_i="1588" txt_f="1589">3</offsets></sup><offsets xml_i="6580" xml_f="6582" txt_i="1589" txt_f="1591">; </offsets><italic><offsets xml_i="6590" xml_f="6591" txt_i="1591" txt_f="1592">p</offsets></italic><offsets xml_i="6600" xml_f="6609" txt_i="1592" txt_f="1601"> = 0.28).</offsets></p></sec><sec id="st8"><title><offsets xml_i="6640" xml_f="6652" txt_i="1603" txt_f="1615">Conclusions:</offsets></title><p><offsets xml_i="6663" xml_f="6919" txt_i="1616" txt_f="1872">In the absence of clear demonstrated benefits arising from the use of the virtual phenotype interpretation, this study suggests resistance testing using genotyping linked to a reliable interpretive algorithm is adequate for the management of HIV infection.</offsets></p></sec></abstract><abstract abstract-type="editor"><title><offsets xml_i="6980" xml_f="7000" txt_i="1885" txt_f="1905">Editorial Commentary</offsets></title><sec id="st9"><title></title><p><offsets xml_i="7040" xml_f="7041" txt_i="1907" txt_f="1908">
</offsets><bold><offsets xml_i="7047" xml_f="7058" txt_i="1908" txt_f="1919">Background:</offsets></bold><offsets xml_i="7065" xml_f="8398" txt_i="1919" txt_f="3252"> Antiretroviral drugs are used to treat patients with HIV infection, with good evidence that they improve prognosis. However, mutations develop in the HIV genome that allow it to evade successful treatment—known as drug resistance—and such mutations are known against every class of antiretroviral drug. Resistance can cause treatment failure and limit the treatment options available. Different types of tests are often used to detect resistance and to work out whether patients should switch to a different drug regimen. Currently, the different types of tests include genotype testing (direct sequencing of genes from virus samples infecting a patient); phenotype testing (a test that assesses the sensitivity of a patient's HIV sample to different drugs), and virtual phenotype testing (a way of interpreting genotype data that estimates the likely viral response to different drugs). The researchers of this study did a trial to find out whether providing an additional virtual phenotype report would be beneficial to patients, as compared with a genotype report alone. The main outcome was HIV viral load after 12 months of treatment, but the researchers also looked at differences in drug regimens prescribed, number of treatment changes in the study, and changes in CD4+ (the type of white blood cell infected by HIV) counts.</offsets></p><p><offsets xml_i="8405" xml_f="8406" txt_i="3253" txt_f="3254">
</offsets><bold><offsets xml_i="8412" xml_f="8434" txt_i="3254" txt_f="3276">What this trial shows:</offsets></bold><offsets xml_i="8441" xml_f="9164" txt_i="3276" txt_f="3999"> The researchers found that the main endpoint of the trial (HIV viral load after 12 months) was no different in patients whose clinicians had received a virtual phenotype report as well as a genotype report, compared with those who had received a genotype report alone. In addition, the average number of drugs prescribed was no different between patients in the two different arms of the trial, and there was no difference in number of drug regimen changes, and no change in immune response (measured using CD4+ cell levels). However, more drugs predicted to be sensitive were prescribed by clinicians who got both a genotype and virtual phenotype report, as compared with clinicians who received only the genotype report.</offsets></p><p><offsets xml_i="9171" xml_f="9172" txt_i="4000" txt_f="4001">
</offsets><bold><offsets xml_i="9178" xml_f="9204" txt_i="4001" txt_f="4027">Strengths and limitations:</offsets></bold><offsets xml_i="9211" xml_f="10338" txt_i="4027" txt_f="5154"> The size of the trial (338 patients recruited) was large enough to properly test the hypothesis that providing a virtual phenotype report as well as a genotype report would result in lower HIV viral loads. Randomization of patients to either intervention ensured that the comparison groups were well-balanced, and the researchers also tested whether selection bias had affected the results (i.e., testing for the possibility that clinicians could predict which intervention participants would receive, and change recruitment into the trial as a result). They found no evidence for selection bias occurring within the trial. However, interpreting the results is difficult because the trial did not directly compare the two different testing platforms, but rather looked at whether providing a virtual phenotype report as well as a genotype report was better than providing a genotype report alone. The investigators also acknowledge that since the trial was conducted, the cutoffs for interpreting genotype information as resistant have been lowered. The findings may therefore not translate precisely to the current situation.</offsets></p><p><offsets xml_i="10345" xml_f="10346" txt_i="5155" txt_f="5156">
</offsets><bold><offsets xml_i="10352" xml_f="10381" txt_i="5156" txt_f="5185">Contribution to the evidence:</offsets></bold><offsets xml_i="10388" xml_f="10911" txt_i="5185" txt_f="5708"> Other cohort studies and clinical trials have shown that patients offered resistance testing respond better to antiretroviral therapy compared with those who were not, but the clinical effectiveness of different resistance testing methods is not known. This study provides additional data on the respective benefits of genotype testing versus genotype plus provision of virtual phenotype. Another trial comparing genotype versus virtual phenotype has also found that the different interpretation methods perform similarly.</offsets></p></sec></abstract><counts><page-count count="9"></page-count></counts><custom-meta-group><custom-meta><meta-name>citation</meta-name><meta-value>Hales G, Birch C, Crowe S, Workman C, Hoy JF, et al. (2006) A randomised trial comparing genotypic and virtual phenotypic interpretation of HIV drug resistance: The CREST study. PLoS Clin Trials 1(3): e18. DOI: <ext-link ext-link-type="doi" xlink:href="10.1371/journal.pctr.0010018">10.1371/journal.pctr.0010018</ext-link></meta-value></custom-meta><custom-meta><meta-name>trial-registry</meta-name><meta-value><named-content content-type="trial-name">Registry</named-content><named-content content-type="trial-number">ACTRN012605000781640</named-content><named-content content-type="trial-url">http://www.actr.org.au/trial_view.aspx?ID=928</named-content></meta-value></custom-meta><custom-meta><meta-name>trial-phase</meta-name><meta-value>IV</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="s1"><title><offsets xml_i="11899" xml_f="11911" txt_i="5717" txt_f="5729">INTRODUCTION</offsets></title><p><offsets xml_i="11922" xml_f="12043" txt_i="5730" txt_f="5851">HIV drug resistance was first described three years after the introduction of zidovudine for treatment of HIV infection [</offsets><xref rid="pctr-0010018-b001" ref-type="bibr"><offsets xml_i="12089" xml_f="12090" txt_i="5851" txt_f="5852">1</offsets></xref><offsets xml_i="12097" xml_f="12307" txt_i="5852" txt_f="6062">]. Resistance has subsequently been described for each licensed antiretroviral therapy (ART). Some mutations causing resistance to one drug are known to confer cross-resistance to others within the same class [</offsets><xref rid="pctr-0010018-b002" ref-type="bibr"><offsets xml_i="12353" xml_f="12354" txt_i="6062" txt_f="6063">2</offsets></xref><offsets xml_i="12361" xml_f="12362" txt_i="6063" txt_f="6064">–</offsets><xref rid="pctr-0010018-b004" ref-type="bibr"><offsets xml_i="12408" xml_f="12409" txt_i="6064" txt_f="6065">4</offsets></xref><offsets xml_i="12416" xml_f="12539" txt_i="6065" txt_f="6188">]. The development of resistance to ART significantly reduces drug efficacy and is an important cause of treatment failure.</offsets></p><p><offsets xml_i="12546" xml_f="12652" txt_i="6189" txt_f="6295">HIV drug resistance testing is a recommended component of several international ART treatment guidelines [</offsets><xref rid="pctr-0010018-b005" ref-type="bibr"><offsets xml_i="12698" xml_f="12699" txt_i="6295" txt_f="6296">5</offsets></xref><offsets xml_i="12706" xml_f="12707" txt_i="6296" txt_f="6297">,</offsets><xref rid="pctr-0010018-b006" ref-type="bibr"><offsets xml_i="12753" xml_f="12754" txt_i="6297" txt_f="6298">6</offsets></xref><offsets xml_i="12761" xml_f="13494" txt_i="6298" txt_f="7031">]. Three different methodologies are currently used to evaluate ART susceptibility in clinical HIV isolates; genotype, phenotype, and virtual phenotype. In addition to the different analytical techniques, myriad platforms for the interpretation of laboratory data are available. Genotype testing involves the sequencing of viral genes and the interpretation of mutations detected in the gene sequence. Phenotypic testing directly measures the susceptibility of a clinical HIV isolate to antiretroviral drugs. The virtual phenotype predicts ART susceptibility on the basis of mapping defined point mutations against an extensive database comprising virus gene sequences with defined phenotypic susceptibility to antiretroviral drugs [</offsets><xref rid="pctr-0010018-b007" ref-type="bibr"><offsets xml_i="13540" xml_f="13541" txt_i="7031" txt_f="7032">7</offsets></xref><offsets xml_i="13548" xml_f="13550" txt_i="7032" txt_f="7034">].</offsets></p><p><offsets xml_i="13557" xml_f="14599" txt_i="7035" txt_f="8077">None of the current methods for resistance testing is ideal. Genotypic testing may not correlate completely with phenotype, requires expert interpretation, and relies heavily on the availability of unvalidated, rules-based algorithms. The technique requires a minimum plasma HIV-1 viral load of approximately 1,000 RNA copies/mL. Phenotypic testing is slow and only available in highly specialised laboratories. Moreover, the relationship between phenotypic inhibitory concentrations (IC50, IC90) and virologic response under continuing drug pressure (the so-called cutoff) requires constant adjustment as more laboratory and clinical information becomes available. Both types of resistance testing may fail to detect minor quasi-species that may have clinical significance. Quality control of genotypic testing for sequence quality, the ability to detect nucleotide mixtures, the interpretation of sequences obtained, and clinician familiarity with the report generated, so that informed treatment choices are made, remain issues of concern.</offsets></p><p><offsets xml_i="14606" xml_f="14852" txt_i="8078" txt_f="8324">In general, both genotypic and phenotypic resistance testing has been shown to provide virologic benefit (reduction in viral load and increased proportion of patients with HIV viral load below the limits of detection) compared with control arms [</offsets><xref rid="pctr-0010018-b008" ref-type="bibr"><offsets xml_i="14898" xml_f="14899" txt_i="8324" txt_f="8325">8</offsets></xref><offsets xml_i="14906" xml_f="14907" txt_i="8325" txt_f="8326">–</offsets><xref rid="pctr-0010018-b011" ref-type="bibr"><offsets xml_i="14953" xml_f="14955" txt_i="8326" txt_f="8328">11</offsets></xref><offsets xml_i="14962" xml_f="15133" txt_i="8328" txt_f="8499">]. Clinicians have been encouraged to incorporate resistance testing as part of their routine clinical assessment of patients prior to initiating and/or changing therapy [</offsets><xref rid="pctr-0010018-b012" ref-type="bibr"><offsets xml_i="15179" xml_f="15181" txt_i="8499" txt_f="8501">12</offsets></xref><offsets xml_i="15188" xml_f="15295" txt_i="8501" txt_f="8608">]. There is preliminary evidence that there are costs/benefits arising from the use of resistance testing [</offsets><xref rid="pctr-0010018-b013" ref-type="bibr"><offsets xml_i="15341" xml_f="15343" txt_i="8608" txt_f="8610">13</offsets></xref><offsets xml_i="15350" xml_f="15351" txt_i="8610" txt_f="8611">,</offsets><xref rid="pctr-0010018-b014" ref-type="bibr"><offsets xml_i="15397" xml_f="15399" txt_i="8611" txt_f="8613">14</offsets></xref><offsets xml_i="15406" xml_f="15533" txt_i="8613" txt_f="8740">]. More recently, in a meta-analysis of published trials, the clinical effectiveness of HIV resistance testing was questioned [</offsets><xref rid="pctr-0010018-b015" ref-type="bibr"><offsets xml_i="15579" xml_f="15581" txt_i="8740" txt_f="8742">15</offsets></xref><offsets xml_i="15588" xml_f="15590" txt_i="8742" txt_f="8744">].</offsets></p><p><offsets xml_i="15597" xml_f="15665" txt_i="8745" txt_f="8813">While phenotypic and virtual phenotypic testing have been compared [</offsets><xref rid="pctr-0010018-b016" ref-type="bibr"><offsets xml_i="15711" xml_f="15713" txt_i="8813" txt_f="8815">16</offsets></xref><offsets xml_i="15720" xml_f="15721" txt_i="8815" txt_f="8816">,</offsets><xref rid="pctr-0010018-b017" ref-type="bibr"><offsets xml_i="15767" xml_f="15769" txt_i="8816" txt_f="8818">17</offsets></xref><offsets xml_i="15776" xml_f="16451" txt_i="8818" txt_f="9493">], the relative utilities of virtual phenotypic and genotypic testing have not been assessed. The CREST (can resistance testing enhance selection of therapy) Study was designed as a randomized, prospective evaluation of antiretroviral prescribing and surrogate marker outcome following provision of resistance tests using a genotype alone or a genotype plus virtual phenotype. The study also supported a national quality assurance program for genotype testing (that incorporated a unifying interpretative algorithm) and the development of clinician familiarity with HIV-1 resistance testing at a time when such testing was not routinely available in Australia or New Zealand.</offsets></p></sec><sec id="s2"><title><offsets xml_i="16481" xml_f="16488" txt_i="9495" txt_f="9502">METHODS</offsets></title><sec id="s2a"><title><offsets xml_i="16517" xml_f="16529" txt_i="9503" txt_f="9515">Participants</offsets></title><p><offsets xml_i="16540" xml_f="17279" txt_i="9516" txt_f="10252">HIV-infected patients taking combination ART, with plasma HIV RNA viral load &gt; 2000 copies/mL, who were willing to change therapy and who were more than 18 years of age were eligible to enter the study. Patients were ineligible if they were ART naive, experiencing an acute illness warranting therapeutic management, or judged by the investigator as being unable to understand or comply with the protocol. All patients were required to give written, informed consent prior to entering the study. Institutional ethics approval was granted for all sites involved in the study. Subjects were recruited from a network comprising 41 clinical sites (hospitals, sexual health clinics, and primary care facilities) in Australia and New Zealand.</offsets></p></sec><sec id="s2b"><title><offsets xml_i="17310" xml_f="17323" txt_i="10254" txt_f="10267">Interventions</offsets></title><p><offsets xml_i="17334" xml_f="17481" txt_i="10268" txt_f="10415">Patients were randomly assigned to receive a genotypic resistance test (group A) or a genotypic resistance test plus a virtual phenotype (group B).</offsets></p></sec><sec id="s2c"><title><offsets xml_i="17512" xml_f="17530" txt_i="10417" txt_f="10435">Resistance Testing</offsets></title><p><offsets xml_i="17541" xml_f="17643" txt_i="10436" txt_f="10538">Each clinical site selected one laboratory to undertake genotype testing for their study participants.</offsets></p></sec><sec id="s2d"><title><offsets xml_i="17674" xml_f="17690" txt_i="10540" txt_f="10556">Genotype Testing</offsets></title><p><offsets xml_i="17701" xml_f="18063" txt_i="10557" txt_f="10919">Blood was collected from eligible patients into tubes containing ACD or EDTA anticoagulant at the randomisation visit and sent to the designated laboratory for genotyping of HIV reverse transcriptase (RT) and protease (PR) genes. The laboratories used a variety of platforms to generate the HIV genotype; TRUGENE HIV-1 assay (Visible Genetics, Ontario, Canada), </offsets><italic><offsets xml_i="18071" xml_f="18072" txt_i="10919" txt_f="10920">n</offsets></italic><offsets xml_i="18081" xml_f="18191" txt_i="10920" txt_f="11030"> = 1; ViroSeq HIV-1 Genotyping System Version 2 (Applied Biosystems, Foster City, California, United States), </offsets><italic><offsets xml_i="18199" xml_f="18200" txt_i="11030" txt_f="11031">n</offsets></italic><offsets xml_i="18209" xml_f="18472" txt_i="11031" txt_f="11294"> = 2; six laboratories used in-house assays with primers that enabled amplification and sequencing of the PR and RT genes. All laboratories participated in an ongoing quality assurance program to ensure that the differing technologies yielded consistent results [</offsets><xref rid="pctr-0010018-b018" ref-type="bibr"><offsets xml_i="18518" xml_f="18520" txt_i="11294" txt_f="11296">18</offsets></xref><offsets xml_i="18527" xml_f="18528" txt_i="11296" txt_f="11297">,</offsets><xref rid="pctr-0010018-b019" ref-type="bibr"><offsets xml_i="18574" xml_f="18576" txt_i="11297" txt_f="11299">19</offsets></xref><offsets xml_i="18583" xml_f="18664" txt_i="11299" txt_f="11380">]. Genotype results were reported using a trial specific, rules-based algorithm (</offsets><xref ref-type="table" rid="pctr-0010018-t001"><offsets xml_i="18711" xml_f="18718" txt_i="11380" txt_f="11387">Table 1</offsets></xref><offsets xml_i="18725" xml_f="18855" txt_i="11387" txt_f="11517">) and a common report format. The report identified a given isolate as being sensitive, intermediate, or resistant to a given ART.</offsets></p><table-wrap id="pctr-0010018-t001" content-type="2col" position="float"><label><offsets xml_i="18938" xml_f="18945" txt_i="11518" txt_f="11525">Table 1</offsets></label><caption><p><offsets xml_i="18965" xml_f="19072" txt_i="11525" txt_f="11632">Rules-Based Algorithm for Determining Genotypic Susceptibility to PI, NNRTI, and NRTI in the CREST Substudy</offsets></p></caption><graphic xlink:href="pctr.0010018.t001"></graphic></table-wrap></sec><sec id="s2e"><title><offsets xml_i="19176" xml_f="19193" txt_i="11634" txt_f="11651">Virtual Phenotype</offsets></title><p><offsets xml_i="19204" xml_f="19781" txt_i="11652" txt_f="12223">The viral RT and PR sequences generated for patients in group B were sent electronically to Virco (Mechelen, Belgium) for generation of a virtual phenotype report. When there were insufficient matches in the database, a Virco rules-based interpretation of the sequence was made. At the time the study was performed, the VircoGEN II platform employed arbitrary clinical cutoffs for each class of ART (&lt;4-fold, 4–10-fold and &gt;10-fold changes for NNRTIs, PIs, and NRTIs, respectively). VircoGEN II was the only format of virtual phenotype used for the duration of the study.</offsets></p></sec><sec id="s2f"><title><offsets xml_i="19812" xml_f="19822" txt_i="12225" txt_f="12235">Objectives</offsets></title><p><offsets xml_i="19833" xml_f="20058" txt_i="12236" txt_f="12461">Our primary hypothesis was that provision of the virtual phenotype report would result in significant clinical benefit relative to provision of an HIV genotype report alone in terms of changes in plasma HIV RNA over 48 weeks.</offsets></p><p><offsets xml_i="20065" xml_f="20299" txt_i="12462" txt_f="12696">We were also interested in patterns of antiretroviral selection and use following receipt of the study resistance test results, changes in CD4+ cell count, and any associations between clinical outcome and a range of baseline factors.</offsets></p></sec><sec id="s2g"><title><offsets xml_i="20330" xml_f="20338" txt_i="12698" txt_f="12706">Outcomes</offsets></title><p><offsets xml_i="20349" xml_f="21026" txt_i="12707" txt_f="13381">The primary endpoint was mean change in log plasma HIV–RNA load between the baseline measurement and that obtained 48 weeks later. Secondary endpoints were: the differences between the planned medication and the regimen selected at baseline following provision of the resistance test result; the proportion of patients with undetectable (&lt;400 copies/mL) plasma HIV-RNA load at 48 weeks; the time weighted average change in log plasma HIV–RNA load from baseline; the number of ART treatment changes over duration of study; the change in CD4+ cell count from baseline to 48 weeks; the time-weighted average change in CD4+ cell count from baseline and time to first ART change.</offsets></p></sec><sec id="s2h"><title><offsets xml_i="21057" xml_f="21068" txt_i="13383" txt_f="13394">Sample Size</offsets></title><p><offsets xml_i="21079" xml_f="21236" txt_i="13395" txt_f="13552">Based on an estimate of variability in mean change in log plasma HIV-RNA load between baseline and 48 weeks, corresponding to a standard deviation of 1.0 log</offsets><sub><offsets xml_i="21241" xml_f="21243" txt_i="13552" txt_f="13554">10</offsets></sub><offsets xml_i="21249" xml_f="21327" txt_i="13554" txt_f="13632">, 300 patients were required to detect a difference between groups of 0.35 log</offsets><sub><offsets xml_i="21332" xml_f="21334" txt_i="13632" txt_f="13634">10</offsets></sub><offsets xml_i="21340" xml_f="21399" txt_i="13634" txt_f="13693"> with 80% power using a two-sided significance level of 5%.</offsets></p></sec><sec id="s2i"><title><offsets xml_i="21430" xml_f="21443" txt_i="13695" txt_f="13708">Randomization</offsets></title><p><offsets xml_i="21454" xml_f="22113" txt_i="13709" txt_f="14362">Randomization was performed using a central telephone randomization office located at NCHECR. The randomization was stratified by site, first or subsequent combination antiretroviral treatment change, and baseline plasma HIV-RNA load (&lt;/&gt;10,000 copies/mL). Computer-generated stratified randomization lists were generated using a blocking factor of two. The block size was known only to key staff at the coordinating centre. It was not known by any of the sites. Eligible patients were randomised in equal proportions to receive either an HIV genotype report (group A) or both an HIV genotype report and a virtual phenotype report (VircoGen II—group B).</offsets></p><p><offsets xml_i="22120" xml_f="22394" txt_i="14363" txt_f="14637">Investigators recorded the ARV regimen they would next prescribe based on their clinical judgment prior to receiving the resistance report. Demographic, CD4+ cell count, HIV RNA load, CDC classification, ART history, and ART intolerance data were also collected at baseline.</offsets></p></sec><sec id="s2j"><title><offsets xml_i="22425" xml_f="22451" txt_i="14639" txt_f="14665">Treatment Failure on Study</offsets></title><p><offsets xml_i="22462" xml_f="22754" txt_i="14666" txt_f="14952">Patients who failed treatment on the study (defined as plasma HIV RNA load &gt; 5,000 copies/mL on two occasions more than two weeks apart after having achieved &lt;400 copies/mL while on the study) could receive an additional HIV resistance test consistent with their original randomisation.</offsets></p></sec><sec id="s2k"><title><offsets xml_i="22785" xml_f="22804" txt_i="14954" txt_f="14973">Statistical Methods</offsets></title><p><offsets xml_i="22815" xml_f="23396" txt_i="14974" txt_f="15555">Primary treatment comparisons were performed using a modified intention to treat (MITT) approach, using all randomised patients with baseline and at least one followup visit. Continuous endpoints were analysed using nonparametric rank-sum tests; binary variables using Fisher's Exact tests. Time to event endpoints were analysed using survival analysis methods. Secondary treatment comparisons were also performed using a strict intention to treat (SITT) approach, including all patients without data as failures, joint worst outcomes, or events at time zero, as appropriate. SITT </offsets><italic><offsets xml_i="23404" xml_f="23405" txt_i="15555" txt_f="15556">p</offsets></italic><offsets xml_i="23414" xml_f="23464" txt_i="15556" txt_f="15606">-values are presented in addition to MITT results.</offsets></p><p><offsets xml_i="23471" xml_f="24040" txt_i="15607" txt_f="16176">Two prespecified subgroup analyses were performed—according to number of previous antiretroviral combinations received (≤7 versus 8+ combinations), and by number of drugs resistant according to genotype (≤6 versus 7+ resistant drugs). Evidence for subgroup effects were formally assessed by testing for statistically significant interactions between subgroup and randomized treatment group. At the request of the journal, we also performed one further subgroup analysis according to whether patients were the first or second randomized within each randomization block [</offsets><xref rid="pctr-0010018-b020" ref-type="bibr"><offsets xml_i="24086" xml_f="24088" txt_i="16176" txt_f="16178">20</offsets></xref><offsets xml_i="24095" xml_f="24382" txt_i="16178" txt_f="16465">]. The aim of this analysis was to try to assess whether the small block size of two adopted in this trial may have led to biased patient allocation, which might be detectable as differing treatment effects according to whether a patient was the first or second randomized in each block.</offsets></p></sec><sec id="s2l"><title><offsets xml_i="24413" xml_f="24431" txt_i="16467" txt_f="16485">Resistance Testing</offsets></title><p><offsets xml_i="24442" xml_f="24646" txt_i="16486" txt_f="16690">Baseline resistance profiles were analysed according to randomisation group using the genotype results only. Additionally, a formal comparison of genotype interpretation according to the Crest Algorithm (</offsets><xref ref-type="table" rid="pctr-0010018-t001"><offsets xml_i="24693" xml_f="24700" txt_i="16690" txt_f="16697">Table 1</offsets></xref><offsets xml_i="24707" xml_f="25078" txt_i="16697" txt_f="17068">) and virtual phenotype (using locally derived sequence data analysed by VircoGen II) was undertaken on group B patients. Resistance profiles were summarised according to the total number of ART to which a patient's virus was classified as resistant, of intermediate resistance, or sensitive, and the proportion of patients whose virus was resistant to all available ART.</offsets></p><p><offsets xml_i="25085" xml_f="25573" txt_i="17069" txt_f="17557">The results of resistance testing for each ART were cross-tabulated against the genotype or virtual phenotype. The proportion of genotype or virtual phenotype results that were derived from a rules-based assessment was calculated. The number of drugs against which the virus was classified as sensitive or intermediate by genotype but resistant by virtual phenotype and vice versa was assessed. A score of the overall difference between genotype and virtual phenotype was then calculated.</offsets></p></sec></sec><sec id="s3"><title><offsets xml_i="25609" xml_f="25616" txt_i="17560" txt_f="17567">RESULTS</offsets></title><sec id="s3a"><title><offsets xml_i="25645" xml_f="25707" txt_i="17568" txt_f="17630">Patient Disposition, Recruitment, and Baseline Characteristics</offsets></title><p><offsets xml_i="25718" xml_f="26178" txt_i="17631" txt_f="18091">A total of 338 patients were randomised into the study between October 2001 and April 2002. Eleven (3.3%) did not have any followup data and were excluded from all analyses. The remaining 327 patients composed the intent-to treat population. Twenty-five patients (7.6%) were lost to followup before 48 weeks, 12 from group A and 13 from group B. These patients were included in analyses using their available data. For participant disposition, please refer to </offsets><xref ref-type="fig" rid="pctr-0010018-g001"><offsets xml_i="26223" xml_f="26231" txt_i="18091" txt_f="18099">Figure 1</offsets></xref><offsets xml_i="26238" xml_f="26325" txt_i="18099" txt_f="18186">. We did not record the numbers of patients screened for the trial who were ineligible.</offsets></p><fig id="pctr-0010018-g001" position="float"><label><offsets xml_i="26381" xml_f="26389" txt_i="18187" txt_f="18195">Figure 1</offsets></label><caption><title><offsets xml_i="26413" xml_f="26436" txt_i="18195" txt_f="18218">Participant Disposition</offsets></title></caption><graphic xlink:href="pctr.0010018.g001"></graphic></fig><p><offsets xml_i="26513" xml_f="26612" txt_i="18219" txt_f="18318">Baseline patient characteristics were summarised according to randomisation group and are shown in </offsets><xref ref-type="table" rid="pctr-0010018-t002"><offsets xml_i="26659" xml_f="26666" txt_i="18318" txt_f="18325">Table 2</offsets></xref><offsets xml_i="26673" xml_f="27084" txt_i="18325" txt_f="18736">. The groups were well matched for all AIDS/demographic variables, including gender, prior AIDS, and risk factor. Patients in group A were statistically significantly older than in group B, but the mean difference of two years is not considered clinically relevant. Median plasma HIV RNA at baseline was 16,750 copies/mL in group A and 16,300 copies/mL in group B. Baseline mean CD4+ cell count was 290 cells/mm</offsets><sup><offsets xml_i="27089" xml_f="27090" txt_i="18736" txt_f="18737">3</offsets></sup><offsets xml_i="27096" xml_f="27124" txt_i="18737" txt_f="18765"> in group A and 318 cells/mm</offsets><sup><offsets xml_i="27129" xml_f="27130" txt_i="18765" txt_f="18766">3</offsets></sup><offsets xml_i="27136" xml_f="27854" txt_i="18766" txt_f="19484"> in group B. All patients had previously received nucleoside RT inhibitors (NRTIs), 80% of group A and 79% of group B, had previously received a non-nucleoside RT inhibitor (NNRTI), and 90% in both groups had previously received a protease inhibitor (PI). The mean total number of ARV drugs ever received was eight and the mean duration of ARV treatment was six years in both groups. The mean number of combinations of ARVs used was eight for group A and seven for group B. On average, patients in both treatment groups were known to be intolerant to one ARV drug. The median turnaround times for generation of resistance reports were 33 days (range 16–118 days) for group A and 33 days (range 7–127 days) for group B.</offsets></p><table-wrap id="pctr-0010018-t002" content-type="2col" position="float"><label><offsets xml_i="27937" xml_f="27944" txt_i="19485" txt_f="19492">Table 2</offsets></label><caption><p><offsets xml_i="27964" xml_f="28065" txt_i="19492" txt_f="19593">Summary of Patient Baseline Characteristics according to Genotype and Genotype Plus Virtual Phenotype</offsets></p></caption><graphic xlink:href="pctr.0010018.t002"></graphic></table-wrap></sec><sec id="s3b"><title><offsets xml_i="28169" xml_f="28192" txt_i="19595" txt_f="19618">Outcomes and Estimation</offsets></title><p><offsets xml_i="28203" xml_f="28388" txt_i="19619" txt_f="19804">After 48 weeks there were no significant differences between the groups for mean change from baseline plasma HIV RNA (group A = −0.68 log copies/mL, group B = −0.58 log copies/mL: MITT </offsets><italic><offsets xml_i="28396" xml_f="28397" txt_i="19804" txt_f="19805">p</offsets></italic><offsets xml_i="28406" xml_f="28421" txt_i="19805" txt_f="19820"> = 0.230, SITT </offsets><italic><offsets xml_i="28429" xml_f="28430" txt_i="19820" txt_f="19821">p</offsets></italic><offsets xml_i="28439" xml_f="28450" txt_i="19821" txt_f="19832"> = 0.685) (</offsets><xref ref-type="fig" rid="pctr-0010018-g002"><offsets xml_i="28495" xml_f="28503" txt_i="19832" txt_f="19840">Figure 2</offsets></xref><offsets xml_i="28510" xml_f="28625" txt_i="19840" txt_f="19955">). The time-weighted mean viral log change from baseline was −0.61 in group A and −0.58 for those in group B (MITT </offsets><italic><offsets xml_i="28633" xml_f="28634" txt_i="19955" txt_f="19956">p</offsets></italic><offsets xml_i="28643" xml_f="28658" txt_i="19956" txt_f="19971"> = 0.876, SITT </offsets><italic><offsets xml_i="28666" xml_f="28667" txt_i="19971" txt_f="19972">p</offsets></italic><offsets xml_i="28676" xml_f="28686" txt_i="19972" txt_f="19982"> = 0.952).</offsets></p><fig id="pctr-0010018-g002" position="float"><label><offsets xml_i="28742" xml_f="28750" txt_i="19983" txt_f="19991">Figure 2</offsets></label><caption><title><offsets xml_i="28774" xml_f="28821" txt_i="19991" txt_f="20038">Comparison of Virological Response by Study Arm</offsets></title></caption><graphic xlink:href="pctr.0010018.g002"></graphic></fig><p><offsets xml_i="28898" xml_f="29008" txt_i="20039" txt_f="20149">A total of 67 patients (46%) in group A and 60 (42%) in group B had undetectable viral loads at week 48 (MITT </offsets><italic><offsets xml_i="29016" xml_f="29017" txt_i="20149" txt_f="20150">p</offsets></italic><offsets xml_i="29026" xml_f="29041" txt_i="20150" txt_f="20165"> = 0.553, SITT </offsets><italic><offsets xml_i="29049" xml_f="29050" txt_i="20165" txt_f="20166">p</offsets></italic><offsets xml_i="29059" xml_f="29198" txt_i="20166" txt_f="20305"> = 0.502). Mean change from baseline CD4+ cell count at 48 weeks was not significantly different between study groups (group A +37 cells/mm</offsets><sup><offsets xml_i="29203" xml_f="29204" txt_i="20305" txt_f="20306">3</offsets></sup><offsets xml_i="29210" xml_f="29232" txt_i="20306" txt_f="20328">, group B +50 cells/mm</offsets><sup><offsets xml_i="29237" xml_f="29238" txt_i="20328" txt_f="20329">3</offsets></sup><offsets xml_i="29244" xml_f="29251" txt_i="20329" txt_f="20336">; MITT </offsets><italic><offsets xml_i="29259" xml_f="29260" txt_i="20336" txt_f="20337">p</offsets></italic><offsets xml_i="29269" xml_f="29284" txt_i="20337" txt_f="20352"> = 0.275, SITT </offsets><italic><offsets xml_i="29292" xml_f="29293" txt_i="20352" txt_f="20353">p</offsets></italic><offsets xml_i="29302" xml_f="29384" txt_i="20353" txt_f="20435"> = 0.296). The mean CD4+ weighted change from baseline at week 48 was +24 cells/mm</offsets><sup><offsets xml_i="29389" xml_f="29390" txt_i="20435" txt_f="20436">3</offsets></sup><offsets xml_i="29396" xml_f="29433" txt_i="20436" txt_f="20473"> in group A patients and +39 cells/mm</offsets><sup><offsets xml_i="29438" xml_f="29439" txt_i="20473" txt_f="20474">3</offsets></sup><offsets xml_i="29445" xml_f="29463" txt_i="20474" txt_f="20492"> in group B (MITT </offsets><italic><offsets xml_i="29471" xml_f="29472" txt_i="20492" txt_f="20493">p</offsets></italic><offsets xml_i="29481" xml_f="29496" txt_i="20493" txt_f="20508"> = 0.401, SITT </offsets><italic><offsets xml_i="29504" xml_f="29505" txt_i="20508" txt_f="20509">p</offsets></italic><offsets xml_i="29514" xml_f="29743" txt_i="20509" txt_f="20738"> = 0.221). Nine (5.5%) patients in group A and seven in group B (4.3%) experienced an AIDS-defining illness during the study. The time to developing an AIDS defining illness was not significantly different between the arms (MITT </offsets><italic><offsets xml_i="29751" xml_f="29752" txt_i="20738" txt_f="20739">p</offsets></italic><offsets xml_i="29761" xml_f="29776" txt_i="20739" txt_f="20754"> = 0.625, SITT </offsets><italic><offsets xml_i="29784" xml_f="29785" txt_i="20754" txt_f="20755">p</offsets></italic><offsets xml_i="29794" xml_f="29881" txt_i="20755" txt_f="20842"> = 0.579). Eight patients died during the study, six from group A and two from group B.</offsets></p><p><offsets xml_i="29888" xml_f="30524" txt_i="20843" txt_f="21479">We performed two prespecified subgroup analyses, in which we examined our study endpoints in patients with or without extensive prior treatment histories, and with multiple drug resistance point mutations or not. We did not observe any evidence of benefit for either resistance test platform in any of these analyses (unpublished data). The analysis according to randomization order also did not reveal any evidence of treatment effects. In patients randomized first within each block, mean log viral load decreases at 48 weeks were −0.70 logs (SD = 1.14) and −0.62 (SD = 0.97) in genotype and virtual phenotype patients, respectively (</offsets><italic><offsets xml_i="30532" xml_f="30533" txt_i="21479" txt_f="21480">p</offsets></italic><offsets xml_i="30542" xml_f="30668" txt_i="21480" txt_f="21606"> = 0.574). In patients randomized second, the mean log decreases were −0.65 (SD = 0.94) and −0.51 (SD = 1.09), respectively, (</offsets><italic><offsets xml_i="30676" xml_f="30677" txt_i="21606" txt_f="21607">p</offsets></italic><offsets xml_i="30686" xml_f="30798" txt_i="21607" txt_f="21719"> = 0.144). There was no statistical evidence of an interaction between treatment group and randomization order (</offsets><italic><offsets xml_i="30806" xml_f="30807" txt_i="21719" txt_f="21720">p</offsets></italic><offsets xml_i="30816" xml_f="30826" txt_i="21720" txt_f="21730"> = 0.535).</offsets></p></sec><sec id="s3c"><title><offsets xml_i="30857" xml_f="30876" txt_i="21732" txt_f="21751">Ancillary Analyses </offsets></title><sec id="s3c1"><title><offsets xml_i="30906" xml_f="30934" txt_i="21752" txt_f="21780">ARV prescribing at baseline.</offsets></title><p><offsets xml_i="30945" xml_f="31214" txt_i="21781" txt_f="22050">The average number of ARVs in regimens prior to randomisation was three in both groups. The mean duration of the current ARV regimen was 18 months in group A and 20 months in group B. The ARV drug class prescribed at study entry prior to resistance testing is shown in </offsets><xref ref-type="table" rid="pctr-0010018-t003"><offsets xml_i="31261" xml_f="31268" txt_i="22050" txt_f="22057">Table 3</offsets></xref><offsets xml_i="31275" xml_f="31393" txt_i="22057" txt_f="22175">. Three patients (1.9%) in group A, and none in group B, had evidence of genotypic resistance across all drug classes.</offsets></p><table-wrap id="pctr-0010018-t003" content-type="1col" position="float"><label><offsets xml_i="31476" xml_f="31483" txt_i="22176" txt_f="22183">Table 3</offsets></label><caption><p><offsets xml_i="31503" xml_f="31553" txt_i="22183" txt_f="22233">Summary of the ARV Drug Classes Used at Enrollment</offsets></p></caption><graphic xlink:href="pctr.0010018.t003"></graphic></table-wrap><p><offsets xml_i="31633" xml_f="31652" txt_i="22234" txt_f="22253">Data summarised in </offsets><xref ref-type="table" rid="pctr-0010018-t004"><offsets xml_i="31699" xml_f="31706" txt_i="22253" txt_f="22260">Table 4</offsets></xref><offsets xml_i="31713" xml_f="31835" txt_i="22260" txt_f="22382"> indicate that the virtual phenotype reported significantly more sensitive ARVs than the genotype platform (MITT and SITT </offsets><italic><offsets xml_i="31843" xml_f="31844" txt_i="22382" txt_f="22383">p</offsets></italic><offsets xml_i="31853" xml_f="32002" txt_i="22383" txt_f="22529"> &lt; 0.001). Furthermore, on average clinicians prescribed significantly more sensitive drugs to patients in arm B than in arm A (MITT and SITT and </offsets><italic><offsets xml_i="32010" xml_f="32011" txt_i="22529" txt_f="22530">p</offsets></italic><offsets xml_i="32020" xml_f="32375" txt_i="22530" txt_f="22882"> &lt;0.001). There were no apparent differences between study groups for discrepancies between the planned and actual regimen employed. This is despite clear evidence that the overwhelming majority of patients in both groups started a regimen of therapy different from the one planned following receipt and interpretation of either resistance test result.</offsets></p><table-wrap id="pctr-0010018-t004" content-type="1col" position="float"><label><offsets xml_i="32458" xml_f="32465" txt_i="22883" txt_f="22890">Table 4</offsets></label><caption><p><offsets xml_i="32485" xml_f="32551" txt_i="22890" txt_f="22956">Selected Resistance Test Data and Prescribing Practice Information</offsets></p></caption><graphic xlink:href="pctr.0010018.t004"></graphic></table-wrap><p><offsets xml_i="32631" xml_f="32819" txt_i="22957" txt_f="23145">The planned ARV regimen was compared with the prescribed ARV regimen following resistance testing. The mean number of ARVs planned and then prescribed in both study groups was three (MITT </offsets><italic><offsets xml_i="32827" xml_f="32828" txt_i="23145" txt_f="23146">p</offsets></italic><offsets xml_i="32837" xml_f="32852" txt_i="23146" txt_f="23161"> = 0.311, SITT </offsets><italic><offsets xml_i="32860" xml_f="32861" txt_i="23161" txt_f="23162">p</offsets></italic><offsets xml_i="32870" xml_f="33043" txt_i="23162" txt_f="23335"> = 0.342). Thirty two patients (19%) in group A and 22 (14%) in group B were actually prescribed the ARV regimen that had been planned based on best clinical judgment (MITT </offsets><italic><offsets xml_i="33051" xml_f="33052" txt_i="23335" txt_f="23336">p</offsets></italic><offsets xml_i="33061" xml_f="33076" txt_i="23336" txt_f="23351"> = 0.180, SITT </offsets><italic><offsets xml_i="33084" xml_f="33085" txt_i="23351" txt_f="23352">p</offsets></italic><offsets xml_i="33094" xml_f="33254" txt_i="23352" txt_f="23512"> = 0.182). The reasons cited as being most important for selection of the prescribed ARV regimen were: resistance test result (group A, 64%; group B, 62%; MITT </offsets><italic><offsets xml_i="33262" xml_f="33263" txt_i="23512" txt_f="23513">p</offsets></italic><offsets xml_i="33272" xml_f="33287" txt_i="23513" txt_f="23528"> = 0.732, SITT </offsets><italic><offsets xml_i="33295" xml_f="33296" txt_i="23528" txt_f="23529">p</offsets></italic><offsets xml_i="33305" xml_f="33366" txt_i="23529" txt_f="23590"> = 0.824), and ARV history (group A: 32%; group B: 28%; MITT </offsets><italic><offsets xml_i="33374" xml_f="33375" txt_i="23590" txt_f="23591">p</offsets></italic><offsets xml_i="33384" xml_f="33399" txt_i="23591" txt_f="23606"> = 0.631, SITT </offsets><italic><offsets xml_i="33407" xml_f="33408" txt_i="23606" txt_f="23607">p</offsets></italic><offsets xml_i="33417" xml_f="33678" txt_i="23607" txt_f="23868"> = 0.634). The median number of ARV drugs against which plasma virus was sensitive was six in group A and eight in group B. The median number of drugs prescribed against which the virus was sensitive in the resistance test was one in group A and two in group B.</offsets></p></sec><sec id="s3c2"><title><offsets xml_i="33710" xml_f="33739" txt_i="23870" txt_f="23899">ARV changes during the study.</offsets></title><p><offsets xml_i="33750" xml_f="33871" txt_i="23900" txt_f="24021">The mean number of changes to prescribed ARV drugs for any reason was three in group A patients and 2.7 in group B (MITT </offsets><italic><offsets xml_i="33879" xml_f="33880" txt_i="24021" txt_f="24022">p</offsets></italic><offsets xml_i="33889" xml_f="33904" txt_i="24022" txt_f="24037"> = 0.702, SITT </offsets><italic><offsets xml_i="33912" xml_f="33913" txt_i="24037" txt_f="24038">p</offsets></italic><offsets xml_i="33922" xml_f="34054" txt_i="24038" txt_f="24170"> = 0.649). Neither the time to first ARV change nor time to treatment failure were significantly different between the groups (MITT </offsets><italic><offsets xml_i="34062" xml_f="34063" txt_i="24170" txt_f="24171">p</offsets></italic><offsets xml_i="34072" xml_f="34087" txt_i="24171" txt_f="24186"> = 0.882, SITT </offsets><italic><offsets xml_i="34095" xml_f="34096" txt_i="24186" txt_f="24187">p</offsets></italic><offsets xml_i="34105" xml_f="34124" txt_i="24187" txt_f="24206"> = 0.829, and MITT </offsets><italic><offsets xml_i="34132" xml_f="34133" txt_i="24206" txt_f="24207">p</offsets></italic><offsets xml_i="34142" xml_f="34157" txt_i="24207" txt_f="24222"> = 0.382, SITT </offsets><italic><offsets xml_i="34165" xml_f="34166" txt_i="24222" txt_f="24223">p</offsets></italic><offsets xml_i="34175" xml_f="34300" txt_i="24223" txt_f="24348"> = 0.557, respectively). The number of protocol-defined treatment failures was 11 (6.7%) in group A and 16 (9.8%) in group B.</offsets></p></sec><sec id="s3c3"><title><offsets xml_i="34332" xml_f="34351" txt_i="24350" txt_f="24369">Resistance results.</offsets></title><p><offsets xml_i="34362" xml_f="34471" txt_i="24370" txt_f="24479">Concordance between the results from genotyping and virtual phenotyping for each drug class were determined (</offsets><xref ref-type="fig" rid="pctr-0010018-g003"><offsets xml_i="34516" xml_f="34525" txt_i="24479" txt_f="24488">Figures 3</offsets></xref><offsets xml_i="34532" xml_f="34533" txt_i="24488" txt_f="24489">–</offsets><xref ref-type="fig" rid="pctr-0010018-g005"><offsets xml_i="34578" xml_f="34579" txt_i="24489" txt_f="24490">5</offsets></xref><offsets xml_i="34586" xml_f="34777" txt_i="24490" txt_f="24681">). There was a significant difference between the susceptibility reported for all NRTIs, with genotype results classifying more patients as harbouring resistant virus than virtual phenotype (</offsets><italic><offsets xml_i="34785" xml_f="34786" txt_i="24681" txt_f="24682">p</offsets></italic><offsets xml_i="34795" xml_f="34986" txt_i="24682" txt_f="24873">-values ≤ 0.002 for all drug classes). NNRTI susceptibility was reported with greater concordance, and only delavirdine resistance was reported more often by genotype than virtual phenotype (</offsets><italic><offsets xml_i="34994" xml_f="34995" txt_i="24873" txt_f="24874">p</offsets></italic><offsets xml_i="35004" xml_f="35192" txt_i="24874" txt_f="25062"> = 0.02). There was no significant difference in test concordance for indinavir and nelfinavir, but genotype results reported significantly more clinical isolates resistant to saquinavir (</offsets><italic><offsets xml_i="35200" xml_f="35201" txt_i="25062" txt_f="25063">p</offsets></italic><offsets xml_i="35210" xml_f="35240" txt_i="25063" txt_f="25090"> = &lt;0.001) and amprenavir (</offsets><italic><offsets xml_i="35248" xml_f="35249" txt_i="25090" txt_f="25091">p</offsets></italic><offsets xml_i="35258" xml_f="35347" txt_i="25091" txt_f="25177"> = &lt;0.001) and virtual phenotype reported significantly more resistance to ritonavir (</offsets><italic><offsets xml_i="35355" xml_f="35356" txt_i="25177" txt_f="25178">p</offsets></italic><offsets xml_i="35365" xml_f="35375" txt_i="25178" txt_f="25188"> = 0.001).</offsets></p><fig id="pctr-0010018-g003" position="float"><label><offsets xml_i="35431" xml_f="35439" txt_i="25189" txt_f="25197">Figure 3</offsets></label><caption><title><offsets xml_i="35463" xml_f="35525" txt_i="25197" txt_f="25259">Comparison of Genotype and Virtual Phenotype Reports for NRTIs</offsets></title></caption><graphic xlink:href="pctr.0010018.g003"></graphic></fig><fig id="pctr-0010018-g004" position="float"><label><offsets xml_i="35651" xml_f="35659" txt_i="25260" txt_f="25268">Figure 4</offsets></label><caption><title><offsets xml_i="35683" xml_f="35743" txt_i="25268" txt_f="25328">Comparison of Genotype and Virtual Phenotype Reports for PIs</offsets></title></caption><graphic xlink:href="pctr.0010018.g004"></graphic></fig><fig id="pctr-0010018-g005" position="float"><label><offsets xml_i="35869" xml_f="35877" txt_i="25329" txt_f="25337">Figure 5</offsets></label><caption><title><offsets xml_i="35901" xml_f="35964" txt_i="25337" txt_f="25400">Comparison of Genotype and Virtual Phenotype Reports for NNRTIs</offsets></title></caption><graphic xlink:href="pctr.0010018.g005"></graphic></fig></sec></sec></sec><sec id="s4"><title><offsets xml_i="36076" xml_f="36086" txt_i="25404" txt_f="25414">DISCUSSION</offsets></title><sec id="s4a"><title><offsets xml_i="36115" xml_f="36129" txt_i="25415" txt_f="25429">Interpretation</offsets></title><p><offsets xml_i="36140" xml_f="36729" txt_i="25430" txt_f="26019">The CREST study was designed to identify whether there was any benefit in providing a virtual phenotype report in addition to a rules-based genotype to assist in the selection of ARV therapy. A no-resistance test control arm was not used because at the time of conducting the trial neither investigators nor patient advocate groups felt there was sufficient clinical equipoise for a control group. Approximately 40% of study patients had viral loads below the level of detection at week 48. Previous studies have also shown the benefit of resistance data when selecting a new ARV regimen [</offsets><xref rid="pctr-0010018-b008" ref-type="bibr"><offsets xml_i="36775" xml_f="36776" txt_i="26019" txt_f="26020">8</offsets></xref><offsets xml_i="36783" xml_f="36784" txt_i="26020" txt_f="26021">–</offsets><xref rid="pctr-0010018-b011" ref-type="bibr"><offsets xml_i="36830" xml_f="36832" txt_i="26021" txt_f="26023">11</offsets></xref><offsets xml_i="36839" xml_f="37147" txt_i="26023" txt_f="26331">]. The choice of which test to use remains a decision for individual clinicians, who may be influenced by factors such as the test method offered, confidence in the quality of performance of complex laboratory testing procedures, cost, turnaround time, and ease of interpretation of the test report provided.</offsets></p><p><offsets xml_i="37154" xml_f="37308" txt_i="26332" txt_f="26486">In this heavily pretreated group of patients, we found no virological or immunological differences at week 48 between these two resistance test platforms.</offsets></p><p><offsets xml_i="37315" xml_f="37952" txt_i="26487" txt_f="27121">Subsequent to this study, the clinical cutoffs for resistance employed by virtual phenotyping were lowered from &gt;10-fold to 2.0, 2.0, and 1.8 for didanosine, zalcitabine, and stavudine, respectively. As such it is tempting to dismiss the relevance of the findings. We would argue that the impact of the new cutoffs would be to reduce the number of drugs that are reported back to clinicians as being sensitive and in so doing further reduce the apparent differences between the two virtual phenotype and the genotypic test platforms. This view cannot be informed by data from our trial because the necessary analyses are not possible.</offsets></p></sec><sec id="s4b"><title><offsets xml_i="37983" xml_f="37999" txt_i="27123" txt_f="27139">Generalizability</offsets></title><p><offsets xml_i="38010" xml_f="38248" txt_i="27140" txt_f="27378">It is reasonably well-established that extensive prior use of ART results in multidrug resistant virus and that the utility of drug resistance testing in heavily pretreated patients is questionable because their options are very limited [</offsets><xref rid="pctr-0010018-b006" ref-type="bibr"><offsets xml_i="38294" xml_f="38295" txt_i="27378" txt_f="27379">6</offsets></xref><offsets xml_i="38302" xml_f="38303" txt_i="27379" txt_f="27380">,</offsets><xref rid="pctr-0010018-b010" ref-type="bibr"><offsets xml_i="38349" xml_f="38351" txt_i="27380" txt_f="27382">10</offsets></xref><offsets xml_i="38358" xml_f="38676" txt_i="27382" txt_f="27700">]. However, we found no evidence to suggest that the two test platforms provided any clear benefit in the predefined patient strata with more or less than eight prior regimens and those with virus with resistance to more or less than six available drugs. This observation is supported by findings from the MuSa Study [</offsets><xref rid="pctr-0010018-b021" ref-type="bibr"><offsets xml_i="38722" xml_f="38724" txt_i="27700" txt_f="27702">21</offsets></xref><offsets xml_i="38731" xml_f="38733" txt_i="27702" txt_f="27704">].</offsets></p><p><offsets xml_i="38740" xml_f="39946" txt_i="27705" txt_f="28911">Our randomised comparison provided clinicians with either one or two forms of report and interpretation describing the scale and scope of antiretroviral drug resistance in clinical isolates taken from patients in the trial. It is not possible to ascertain with confidence which report carried more weight when clinicians reviewed the results provided for patients in group B. Consequently there may be concern that many decisions regarding future treatment in that group were made on the basis of interpretations provided by the genotype result alone. However, our data clearly show that virtual phenotype results provided clinicians and their patients with significantly more treatment options and this resulted in significantly more apparently active drugs being prescribed for patients enrolled in group B. As such, we contend that for the most part our randomised comparison maintained integrity and that the results should be viewed as valid and robust comparisons. Of some importance in this regard is the fact that it is not possible to exclude an interpretation of genotype from the virtual phenotype report since the observed point mutations for a given sample are an integral part of that report.</offsets></p><p><offsets xml_i="39953" xml_f="40542" txt_i="28912" txt_f="29501">A majority of clinical HIV isolates in each study arm had at least one ARV with either intermediate or full sensitivity. However, clinician prescribing was limited as not all of these ARVs could be used. For example, a viable combination regimen could not be constructed from three NNRTI drugs. A further limitation of this study is that the impact of low dose ritonavir, used to inhibit cytochrome P450 isoenzymes and thereby increase the plasma levels of co-administered PIs, was not examined. Further work is needed to interpret resistance results when a boosted PI regimen is planned [</offsets><xref rid="pctr-0010018-b022" ref-type="bibr"><offsets xml_i="40588" xml_f="40590" txt_i="29501" txt_f="29503">22</offsets></xref><offsets xml_i="40597" xml_f="40598" txt_i="29503" txt_f="29504">,</offsets><xref rid="pctr-0010018-b023" ref-type="bibr"><offsets xml_i="40644" xml_f="40646" txt_i="29504" txt_f="29506">23</offsets></xref><offsets xml_i="40653" xml_f="40769" txt_i="29506" txt_f="29622">]. Ongoing refinement and modification of the test platforms does impact upon the generalizability of these results.</offsets></p><p><offsets xml_i="40776" xml_f="41589" txt_i="29623" txt_f="30436">CREST resulted in all Australian and New Zealand HIV reference laboratories being able to offer HIV genotype testing and participation in a local quality assurance program. Other outcomes were sharing of expertise, collaboration between different laboratories, and the availability of a standardised genotypic interpretation and reporting format developed and maintained by Australian experts. The study was undertaken in a wide variety of settings and provided investigators with access to HIV resistance test reports. The genotype report was designed to be user friendly for both those experienced and inexperienced in interpreting the results. The provision of a resistance test result significantly impacted the choice of ARVs prescribed across all sites, both in general practice and in specialist hospitals.</offsets></p></sec><sec id="s4c"><title><offsets xml_i="41620" xml_f="41636" txt_i="30438" txt_f="30454">Overall Evidence</offsets></title><p><offsets xml_i="41647" xml_f="42070" txt_i="30455" txt_f="30878">Overall, CREST found that HIV resistance test results affected ARV prescribing but there was no additional benefit derived from receiving a virtual phenotype report in addition to a genotype report. This may be due to factors including the quality of the genotype interpretation developed for CREST and the high biological cutoff used in the virtual phenotype analysis. Similar data have been generated in one other trial [</offsets><xref rid="pctr-0010018-b021" ref-type="bibr"><offsets xml_i="42116" xml_f="42118" txt_i="30878" txt_f="30880">21</offsets></xref><offsets xml_i="42125" xml_f="42430" txt_i="30880" txt_f="31185">]. Across a range of studies conducted in a diverse patient population there is little data to indicate that any of the myriad interpretation methods offer reliable and predictable benefits. The choice of test platform should probably therefore reflect factors such as availability, cost, and familiarity.</offsets></p></sec></sec><sec sec-type="supplementary-material" id="s5"><title><offsets xml_i="42500" xml_f="42522" txt_i="31188" txt_f="31210">SUPPORTING INFORMATION</offsets></title><supplementary-material content-type="local-data" id="pctr-0010018-sd001"><label><offsets xml_i="42611" xml_f="42628" txt_i="31211" txt_f="31228">CONSORT Checklist</offsets></label><media xlink:href="pctr.0010018.sd001.doc"><caption><p><offsets xml_i="42691" xml_f="42727" txt_i="31228" txt_f="31264">Click here for additional data file.</offsets></p></caption></media><p><offsets xml_i="42752" xml_f="42763" txt_i="31265" txt_f="31276">(56 KB DOC)</offsets></p></supplementary-material><supplementary-material content-type="local-data" id="pctr-0010018-sd002"><label><offsets xml_i="42873" xml_f="42887" txt_i="31277" txt_f="31291">Trial Protocol</offsets></label><media xlink:href="pctr.0010018.sd002.doc"><caption><p><offsets xml_i="42950" xml_f="42986" txt_i="31291" txt_f="31327">Click here for additional data file.</offsets></p></caption></media><p><offsets xml_i="43011" xml_f="43023" txt_i="31328" txt_f="31340">(1.0 MB DOC)</offsets></p></supplementary-material></sec></body><back><ack><sec id="ack"><title></title><p>We are extremely grateful to the patients for their participation. The NCHECR is funded by the Australian Government Department of Health and Ageing and is affiliated with the Faculty of Medicine at the University of New South Wales. We would also like to acknowledge the involvement of the following medical scientists: A. Dunne (Burnet Centre), B. Schroeder (Auckland Hospital), D. Sayer (Royal Perth Hospital), G. Bryson (Royal Brisbane Hospital), H. Salem (Royal Prince Alfred Hospital), L. Leas (St. Vincent's Hospital), L. Rawlings (Institute of Medicine and Veterinary Science), N. Saksena (Westmead Hospital), S. Land (National Reference Laboratory), T. Middleton (Victoria Infectious Diseases Reference Laboratory); and these participating investigators and coordinators in Australia, Northern Territory: S. Huffam/P. Knibbs (Royal Darwin Hospital), in Queensland: D. Orth/G. Lister (Brunswick Street Practice), K. Clare/H. Ree (AIDS Medical Unit), A. Allworth/N. Gerns (Royal Brisbane Hospital), D. Bradford/C. Wilson (Dolls House), D. Sowden/A. Walker (Nambour Hospital), M. O'Sullivan/F. Clark (Gold Coast Sexual Health Clinic), in New South Wales: A. Carr/G. Dolan (St. Vincent's Hospital), J. Quin/G. Keogh (Bigge Park Clinic), R. Garsia (Royal Prince Alfred), B. Donovan/C. Bourne (Sydney Hospital), P. Konecny/T. White (St. George Hospital), M. McMurchie/C. Rogers (229 Oxford Street), R. Finlayson/D. Wheatley (Taylor Square Private Clinic), M. Bloch/T. Frater (HHGP), R. McFarlane/R. Vale (407 Doctors), C. Workman (AIDS Initiative), D. Dwyer/E. Keating (Westmead Hospital), N. Doong/J. Hudson (Burwood Practice), M. Boyle/P. Dobson (John Hunter Hospital), K. Clezy/S. Ryan (Prince of Wales Hospital), M. Kelly/J Sarangapany (Albion Street Clinic), D. Allen/H. Blacklaws (Gosford Sexual Health Clinic), C. Carmody/S. Brennan (Wagga Wagga Sexual Health Clinic); in Australian Capital Territory: T. Meng Soo/P. Haberl (Canberra); in Victoria: J. Hoy/C. McCormack (The Alfred Hospital), N. Roth/H. Wood (Prahran Market Clinic), J. Anderson/J. Patching (Carlton Clinic), N. Medland (Centre Clinic), I. Chenoweth (Middle Park Clinic), T. Schmidt/F. Macfarlane (Melbourne Sexual Health Clinic), A. Street/J. Roney (Royal Melbourne Hospital); in South Australia: D. Shaw/W. Ferguson (Royal Adelaide Hospital), G. Rogers/M. Curry (Care and Prevention Programme); in Western Australia: M. French/J. Leung (Royal Perth Hospital), M. Beaman/A. Chian (Freemantle Hospital); and these participating investigators and coordinators in New Zealand: M. Thomas/F. Porteous (Auckland Hospital), R. Franklin (Auckland Sexual Health Clinic), G. Mills/J. Morgan (Waikato Hospital), T. Blackmore (Wellington Hospital), A. Pithie (Christchurch Hospital).</p><fn-group><fn id="ack1" fn-type="con"><p>
<bold>Author Contributions.</bold>
GH and SE designed the study. CW, MGL, DL, and SE analyzed the data. SC, CW, and JFH enrolled patients. SC, JFH, and ADK collected data or did experiments for the study. GH wrote the first draft of the paper. SC, CW, JFH, MGL, ADK, and SE contributed to the writing of the paper. GH wrote the protocol and coordinated the study. SC assisted in study design and presentation of results. ADK supervised the conduct of the resistance assays for one of the major sites that contributed data to this collaborative project and ensured the internal quality control for these data.</p><p>
<bold>Competing Interests.</bold> SE received travel support from BMS, Chiron, GlaxoSmithKline, and Merck Sharpe and Dohme; received research funding from Abbott, Australia Technology, Bristol Myers Squibb, Boehringer-Ingleheim, Chiron, GlaxoSmithKline, Gilead, Merck Sharpe and Dohme, and Virax Immunotherapeutics; and received honoraria from Bristol Myers Squibb, Chiron, and Roche. GH has been employed by Roche Products this year, but was not their employee during the conduct of this study. CW receives research funds from Tibotec, Johnson and Johnson, Roche, Merck Sharpe and Dohme, Abbott, GlaxoSmithKline, Boehringer-Ingleheim, Pfizer, and Novartis. CW is on the advisory board of Merck Sharpe and Dohme, Tibotec, Roche, and Abbott, and received travel grants from Roche and Abbott. CW has no stocks in any pharmaceutical companies that she is aware of, but does have an investment account with a management fund and does not know what companies they invest in. MGL has received travel grants from GlaxoSmithKline and consultancy from Johnson and Johnson Research. SC, JFH, ADK, and DL declare that they have no competing interests.</p></fn><fn id="n102" fn-type="financial-disclosure"><p>
<bold>Funding:</bold> We thank the following companies who provided support: Diagnostic Technology, Perkin Elmer Biosystems, Abbott Australasia, Boehringer Ingelheim, Bristol Myers Squibb, GlaxoSmithKline, Merck Sharpe and Dohme, and Roche Products. These organisations were not involved in study design, the collection, analysis, or interpretation of data, or writing the manuscript, or decisions to submit for publication. Representatives of Tibotec-Virco were involved in the design of the trial and provision of resistance test results. They reviewed this manuscript but declined authorship on the basis of irreconcilable differences relating to trial methodology and conclusions. They were therefore not involved in the interpretation of data, writing the manuscript, nor the decision to publish.</p></fn></fn-group></sec></ack><glossary><title>Abbreviations</title><def-list><def-item><term>ART</term><def><p>antiretroviral therapy</p></def></def-item><def-item><term>CREST</term><def><p>can resistance testing enhance selection of therapy</p></def></def-item><def-item><term>MITT</term><def><p>modified intention to treat</p></def></def-item><def-item><term>NNRTI</term><def><p>non-nucleoside RT inhibitors</p></def></def-item><def-item><term>NRTI</term><def><p>nucleoside RT inhibitors</p></def></def-item><def-item><term>PR</term><def><p>protease</p></def></def-item><def-item><term>RT</term><def><p>reverse transcriptase</p></def></def-item><def-item><term>SITT</term><def><p>strict intention to treat</p></def></def-item></def-list></glossary><ref-list><title>REFERENCES</title><ref id="pctr-0010018-b001"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Larder</surname><given-names>BA</given-names></name><name><surname>Darby</surname><given-names>G</given-names></name><name><surname>Richman</surname><given-names>D</given-names></name></person-group>
<year>1989</year>
<article-title>HIV with reduced sensitivity to zidovudine (AZT) isolated during prolonged therapy</article-title>
<source>Science</source>
<volume>243</volume>
<fpage>1731</fpage>
<lpage>1734</lpage>
<pub-id pub-id-type="pmid">2467383</pub-id></element-citation></ref><ref id="pctr-0010018-b002"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Smith</surname><given-names>M</given-names></name><name><surname>Salomon</surname><given-names>H</given-names></name><name><surname>Wainberg</surname><given-names>MA</given-names></name></person-group>
<year>1994</year>
<article-title>Development and significance of nucleoside drug resistance in infection caused by the human immunodeficiency virus type 1</article-title>
<source>Clin Invest Med</source>
<volume>17</volume>
<issue>(3)</issue>
<fpage>226</fpage>
<lpage>243</lpage>
<pub-id pub-id-type="pmid">7523016</pub-id></element-citation></ref><ref id="pctr-0010018-b003"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Mayers</surname><given-names>DL</given-names></name></person-group>
<year>1997</year>
<article-title>Prevalence and incidence of resistance to zidovudine and other antiretroviral drugs</article-title>
<source>Am J Med</source>
<volume>102</volume>
<fpage>70</fpage>
<lpage>75</lpage>
<pub-id pub-id-type="pmid">9845501</pub-id></element-citation></ref><ref id="pctr-0010018-b004"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Condra</surname><given-names>JH</given-names></name><name><surname>Schleif</surname><given-names>WA</given-names></name><name><surname>Blahy</surname><given-names>OM</given-names></name><etal></etal></person-group>
<year>1995</year>
<article-title>In vivo emergence of HIV-1 variants resistant to multiple protease inhibitors</article-title>
<source>Nature</source>
<volume>374</volume>
<fpage>569</fpage>
<lpage>571</lpage>
<pub-id pub-id-type="pmid">7700387</pub-id></element-citation></ref><ref id="pctr-0010018-b005"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Hirsch</surname><given-names>MS</given-names></name><name><surname>Conway</surname><given-names>B</given-names></name><name><surname>D'Aquila</surname><given-names>RT</given-names></name><etal></etal></person-group>
<year>1998</year>
<article-title>Antiretroviral drug resistance testing in adults with HIV infection: Implications for clinical management. International AIDS Society—USA Panel</article-title>
<source>JAMA</source>
<volume>24</volume>
<fpage>1984</fpage>
<lpage>1991</lpage>
</element-citation></ref><ref id="pctr-0010018-b006"><element-citation publication-type="journal">
<collab>The EuroGuidelines Group for HIV Resistance</collab>
<year>2001</year>
<article-title>Clinical and laboratory guidelines for the use of HIV-1 drug resistance testing as part of treatment management: Recommendations for the European setting</article-title>
<source>AIDS</source>
<volume>15</volume>
<fpage>309</fpage>
<lpage>320</lpage>
<pub-id pub-id-type="pmid">11273210</pub-id></element-citation></ref><ref id="pctr-0010018-b007"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Bacheler</surname><given-names>L</given-names></name><name><surname>van Marck</surname><given-names>H</given-names></name><name><surname>Vermeiren</surname><given-names>H</given-names></name><etal></etal></person-group>
<year>2003</year>
<article-title>The Virtual Phenotype: A quantitative prediction of HIV drug susceptibility from viral genotype</article-title>
<source>HIV Med</source>
<volume>4</volume>
<fpage>178</fpage>
</element-citation></ref><ref id="pctr-0010018-b008"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Durant</surname><given-names>J</given-names></name><name><surname>Clevenbergh</surname><given-names>P</given-names></name><name><surname>Halfon</surname><given-names>P</given-names></name><etal></etal></person-group>
<year>1999</year>
<article-title>Drug-resistance genotyping in HIV-1 therapy: The VIRADAPT randomised controlled trial</article-title>
<source>Lancet</source>
<volume>353</volume>
<fpage>2195</fpage>
<lpage>2199</lpage>
<pub-id pub-id-type="pmid">10392984</pub-id></element-citation></ref><ref id="pctr-0010018-b009"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Baxter</surname><given-names>JD</given-names></name><name><surname>Mayers</surname><given-names>DL</given-names></name><name><surname>Wentworth</surname><given-names>DN</given-names></name><etal></etal></person-group>
<year>2000</year>
<article-title>A randomized study of antiretroviral management based on plasma genotypic antiretroviral resistance testing in patients failing therapy: CPCRA 046 Study Team for the Terry Beirn Community Programs for Clinical Research on AIDS</article-title>
<source>AIDS</source>
<volume>14</volume>
<fpage>F83</fpage>
<lpage>F93</lpage>
<pub-id pub-id-type="pmid">10894268</pub-id></element-citation></ref><ref id="pctr-0010018-b010"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Lorenzi</surname><given-names>P</given-names></name><name><surname>Opravil</surname><given-names>M</given-names></name><name><surname>Hirschel</surname><given-names>B</given-names></name><etal></etal></person-group>
<year>1999</year>
<article-title>Impact of drug resistance mutations on virologic response to salvage therapy</article-title>
<source>AIDS</source>
<volume>13</volume>
<fpage>F17</fpage>
<lpage>F21</lpage>
<pub-id pub-id-type="pmid">10202819</pub-id></element-citation></ref><ref id="pctr-0010018-b011"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Middleton</surname><given-names>T</given-names></name><name><surname>Smith</surname><given-names>D</given-names></name><name><surname>Larder</surname><given-names>B</given-names></name><etal></etal></person-group>
<year>2001</year>
<article-title>Baseline antiretroviral drug susceptibility influences treatment response in patients receiving saquinavir-enhancing therapy</article-title>
<source>HIV Clin Trials</source>
<volume>2</volume>
<fpage>445</fpage>
<lpage>452</lpage>
<pub-id pub-id-type="pmid">11742431</pub-id></element-citation></ref><ref id="pctr-0010018-b012"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Chan</surname><given-names>S</given-names></name><name><surname>Hulgan</surname><given-names>T</given-names></name><name><surname>D'Aquila</surname><given-names>R</given-names></name></person-group>
<year>2004</year>
<article-title>The role of baseline HIV-1 resistance testing in patients with established infection</article-title>
<source>Curr Infect Dis Rep</source>
<volume>6</volume>
<fpage>243</fpage>
<lpage>249</lpage>
<pub-id pub-id-type="pmid">15142489</pub-id></element-citation></ref><ref id="pctr-0010018-b013"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Chaix</surname><given-names>C</given-names></name><name><surname>Grenier-Sennelier</surname><given-names>C</given-names></name><name><surname>Durant</surname><given-names>J</given-names></name><etal></etal></person-group>
<year>2000</year>
<article-title>Economic evaluation of drug resistance genotyping for the adaptation of treatment in HIV-infected patients in the VIRADAPT study. Proceedings of the 39th Interscience Conference on Antimicrobial Agents and Chemotherapy; 26–29 September 1999; San Francisco, United States</article-title>
<source>JAIDS</source>
<volume>24</volume>
<fpage>227</fpage>
<lpage>231</lpage>
<pub-id pub-id-type="pmid">10969346</pub-id></element-citation></ref><ref id="pctr-0010018-b014"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Corzillius</surname><given-names>M</given-names></name><name><surname>Muhlberger</surname><given-names>N</given-names></name><name><surname>Sroczynski</surname><given-names>G</given-names></name><etal></etal></person-group>
<year>2004</year>
<article-title>Cost effectiveness analysis of routine use of genotypic antiretroviral resistance testing after failure of antiretroviral treatment for HIV</article-title>
<source>Antivir Ther</source>
<volume>9</volume>
<fpage>27</fpage>
<lpage>36</lpage>
<pub-id pub-id-type="pmid">15040534</pub-id></element-citation></ref><ref id="pctr-0010018-b015"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Dunn</surname><given-names>DT</given-names></name><name><surname>Gibb</surname><given-names>DM</given-names></name><name><surname>Babiker</surname><given-names>AG</given-names></name><etal></etal></person-group>
<year>2004</year>
<article-title>HIV drug resistance testing: Is the evidence really there?</article-title>
<source>Antiviral Ther</source>
<volume>9</volume>
<fpage>641</fpage>
<lpage>648</lpage>
</element-citation></ref><ref id="pctr-0010018-b016"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Mazzotta</surname><given-names>F</given-names></name><name><surname>Lo Caputo</surname><given-names>S</given-names></name><name><surname>Torti</surname><given-names>C</given-names></name><etal></etal></person-group>
<collab>for the Genotipo-Fenotipo di Resistenza (GenPheRex) Group of the Italian Management Standardizzato di Terapia Antiretrovirale (MASTER) Cohort</collab>
<year>2003</year>
<article-title>Real versus virtual phenotype to guide treatment in heavily pretreated patients: 48 week follow-up of the Genotipo-Fenotipo di Resistenza (GenPheRex) Trial</article-title>
<source>JAIDS</source>
<volume>31</volume>
<fpage>268</fpage>
<lpage>280</lpage>
</element-citation></ref><ref id="pctr-0010018-b017"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Perez-Elias</surname><given-names>MJ</given-names></name><name><surname>Garcia-Arata</surname><given-names>I</given-names></name><name><surname>Muñoz</surname><given-names>V</given-names></name><collab>Realvirfen Study Group</collab></person-group>
<year>2003</year>
<article-title>Phenotype or virtual phenotype for choosing antiretroviral therapy after failure: A prospective randomized study</article-title>
<source>Antivir Ther</source>
<volume>8</volume>
<fpage>577</fpage>
<lpage>584</lpage>
<pub-id pub-id-type="pmid">14760891</pub-id></element-citation></ref><ref id="pctr-0010018-b018"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Sayer</surname><given-names>DC</given-names></name><name><surname>Land</surname><given-names>S</given-names></name><name><surname>Gizzarelli</surname><given-names>L</given-names></name><etal></etal></person-group>
<year>2003</year>
<article-title>Quality assessment program for genotypic antiretroviral testing improves detection of drug resistance mutations</article-title>
<source>J Clin Microbiol</source>
<volume>41</volume>
<fpage>227</fpage>
<lpage>236</lpage>
<pub-id pub-id-type="pmid">12517853</pub-id></element-citation></ref><ref id="pctr-0010018-b019"><element-citation publication-type="confproc">
<person-group person-group-type="author"><name><surname>Land</surname><given-names>S</given-names></name><name><surname>Sayer</surname><given-names>D</given-names></name><name><surname>Gizzarelli</surname><given-names>L</given-names></name><etal></etal></person-group>
<year>2004</year>
<article-title>An international external quality assessment scheme to standardise the interpretation on HIV-1 genotypic antiretroviral resistance testing</article-title>
<conf-name>XV International AIDS Conference; </conf-name>
<conf-date>11–16 July 2004,</conf-date>
<conf-loc>Bangkok, Thailand.</conf-loc>
<comment>B3086.</comment>
</element-citation></ref><ref id="pctr-0010018-b020"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Berger</surname><given-names>VW</given-names></name><name><surname>Exner</surname><given-names>DV</given-names></name></person-group>
<year>1999</year>
<article-title>Detecting selection bias in randomized clinical trials</article-title>
<source>Cont Clin Trials</source>
<volume>20</volume>
<fpage>319</fpage>
<lpage>327</lpage>
</element-citation></ref><ref id="pctr-0010018-b021"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Gianotti</surname><given-names>V</given-names></name><name><surname>Mondino</surname><given-names>V</given-names></name><name><surname>Rossi</surname><given-names>MC</given-names></name><etal></etal></person-group>
<year>2006</year>
<article-title>Comparison of a rules based algorithm with a phenotype based algorithm for the interpretation of HIV genotypes in guiding salvage regimens in HIV infected patients by a randomized clinical trial: The MuSa Study</article-title>
<source>Clin Infect Dis</source>
<volume>42</volume>
<fpage>1470</fpage>
<lpage>1480</lpage>
<pub-id pub-id-type="pmid">16619162</pub-id></element-citation></ref><ref id="pctr-0010018-b022"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Winston</surname><given-names>A</given-names></name><name><surname>Hales</surname><given-names>G</given-names></name><name><surname>Amin</surname><given-names>J</given-names></name><etal></etal></person-group>
<year>2005</year>
<article-title>The normalised inhibitory quotient of boosted protease inhibitors is predictive of viral load response in treatment-experienced HIV-1-infected individuals</article-title>
<source>AIDS</source>
<volume>19</volume>
<fpage>1393</fpage>
<lpage>1399</lpage>
<pub-id pub-id-type="pmid">16103770</pub-id></element-citation></ref><ref id="pctr-0010018-b023"><element-citation publication-type="journal">
<person-group person-group-type="author"><name><surname>Hoefnagel</surname><given-names>JGM</given-names></name><name><surname>Koopmans</surname><given-names>PP</given-names></name><name><surname>Burger</surname><given-names>DM</given-names></name><name><surname>Schuurman</surname><given-names>R</given-names></name><name><surname>Galanna</surname><given-names>JMD</given-names></name></person-group>
<year>2005</year>
<article-title>The role of the inhibitory quotient in HIV therapy</article-title>
<source>Antivir Ther</source>
<volume>10</volume>
<fpage>879</fpage>
<lpage>892</lpage>
<pub-id pub-id-type="pmid">16430193</pub-id></element-citation></ref></ref-list></back></article>