<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">J Transl Med</journal-id><journal-title>Journal of Translational Medicine</journal-title><issn pub-type="epub">1479-5876</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">17257431</article-id><article-id pub-id-type="pmc">1796543</article-id><article-id pub-id-type="publisher-id">1479-5876-5-7</article-id><article-id pub-id-type="doi">10.1186/1479-5876-5-7</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Immune-mediated changes in actinic keratosis following topical treatment with imiquimod 5% cream</article-title></title-group><contrib-group><contrib id="A1" contrib-type="author"><name><surname>Torres</surname><given-names>Abel</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>abelt@aol.com</email></contrib><contrib id="A2" contrib-type="author"><name><surname>Storey</surname><given-names>Leslie</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>lesliestorey13@yahoo.com</email></contrib><contrib id="A3" contrib-type="author"><name><surname>Anders</surname><given-names>Makala</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>makalaanders@yahoo.com</email></contrib><contrib id="A4" contrib-type="author"><name><surname>Miller</surname><given-names>Richard L</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>rlmiller1@MMM.com</email></contrib><contrib id="A5" contrib-type="author"><name><surname>Bulbulian</surname><given-names>Barbara J</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>bjbulbulian@MMM.com</email></contrib><contrib id="A6" contrib-type="author"><name><surname>Jin</surname><given-names>Jizhong</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>jjin1@MMM.com</email></contrib><contrib id="A7" contrib-type="author"><name><surname>Raghavan</surname><given-names>Shalini</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>sraghavan1@MMM.com</email></contrib><contrib id="A8" contrib-type="author"><name><surname>Lee</surname><given-names>James</given-names></name><xref ref-type="aff" rid="I3">3</xref><email>jlee107@cntus.jnj.com</email></contrib><contrib id="A9" contrib-type="author"><name><surname>Slade</surname><given-names>Herbert B</given-names></name><xref ref-type="aff" rid="I3">3</xref><email>hbslade@comcast.net</email></contrib><contrib id="A10" corresp="yes" contrib-type="author"><name><surname>Birmachu</surname><given-names>Woubalem</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>wbirmachu@comcast.net</email></contrib></contrib-group><aff id="I1"><label>1</label>Dermatology Office, Loma Linda University Medical Center, Loma Linda, California, USA</aff><aff id="I2"><label>2</label>Pharmacology, 3M Pharmaceuticals, St Paul, Minnesota, USA</aff><aff id="I3"><label>3</label>Medical &amp; Scientific Affairs, 3M Pharmaceuticals, St Paul, Minnesota, USA</aff><pub-date pub-type="collection"><year>2007</year></pub-date><pub-date pub-type="epub"><day>26</day><month>1</month><year>2007</year></pub-date><volume>5</volume><fpage>7</fpage><lpage>7</lpage><ext-link ext-link-type="uri" xlink:href="http://www.translational-medicine.com/content/5/1/7"></ext-link><history><date date-type="received"><day>11</day><month>6</month><year>2006</year></date><date date-type="accepted"><day>26</day><month>1</month><year>2007</year></date></history><permissions><copyright-statement>Copyright © 2007 Torres et al; licensee BioMed Central Ltd.</copyright-statement><copyright-year>2007</copyright-year><copyright-holder>Torres et al; licensee BioMed Central Ltd.</copyright-holder><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0"></ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></license></permissions><abstract><sec><title><offsets xml_i="4387" xml_f="4397" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="4408" xml_f="4606" txt_i="22" txt_f="220">The objective of this study was to identify the molecular processes responsible for the anti-lesional activity of imiquimod in subjects with actinic keratosis using global gene expression profiling.</offsets></p></sec><sec sec-type="methods"><title><offsets xml_i="4647" xml_f="4654" txt_i="222" txt_f="229">Methods</offsets></title><p><offsets xml_i="4665" xml_f="5263" txt_i="230" txt_f="828">A double-blind, placebo-controlled, randomized study was conducted to evaluate gene expression changes in actinic keratosis treated with imiquimod 5% cream. Male subjects (N = 17) with ≥ 5 actinic keratosis on the scalp applied placebo cream or imiquimod 3 times a week on nonconsecutive days for 4 weeks. To elucidate the molecular processes involved in actinic keratosis lesion regression by imiquimod, gene expression analysis using oligonucleotide arrays and real time reverse transcriptase polymerase chain reaction were performed on shave biopsies of lesions taken before and after treatment.</offsets></p></sec><sec><title><offsets xml_i="5285" xml_f="5292" txt_i="830" txt_f="837">Results</offsets></title><p><offsets xml_i="5303" xml_f="5804" txt_i="838" txt_f="1339">Imiquimod modulated the expression of a large number of genes important in both the innate and adaptive immune response, including increased expression of interferon-inducible genes with known antiviral, anti-proliferative and immune modulatory activity, as well as various Toll-like receptors. In addition, imiquimod increased the expression of genes associated with activation of macrophages, dendritic cells, cytotoxic T cells, and natural killer cells, as well as activation of apoptotic pathways.</offsets></p></sec><sec><title><offsets xml_i="5826" xml_f="5836" txt_i="1341" txt_f="1351">Conclusion</offsets></title><p><offsets xml_i="5847" xml_f="6091" txt_i="1352" txt_f="1596">Data suggest that topical application of imiquimod stimulates cells in the skin to secrete cytokines and chemokines that lead to inflammatory cell influx into the lesions and subsequent apoptotic and immune cell-mediated destruction of lesions.</offsets></p></sec></abstract></article-meta></front><body><sec><title><offsets xml_i="6153" xml_f="6163" txt_i="1605" txt_f="1615">Background</offsets></title><p><offsets xml_i="6174" xml_f="6362" txt_i="1616" txt_f="1804">Actinic keratosis (AK) are common, cutaneous, precancerous neoplasms appearing as rough, dry, scaly lesions that occur primarily on the sun-exposed skin of middle-aged and elderly people [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="6393" xml_f="6394" txt_i="1804" txt_f="1805">1</offsets></xref><offsets xml_i="6401" xml_f="6402" txt_i="1805" txt_f="1806">-</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="6433" xml_f="6434" txt_i="1806" txt_f="1807">3</offsets></xref><offsets xml_i="6441" xml_f="6617" txt_i="1807" txt_f="1983">]. Although the exact mechanism of pathogenesis of AK development is unknown, part of the pathogenesis may involve suppression of the immune response against dysplastic cells [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="6648" xml_f="6649" txt_i="1983" txt_f="1984">4</offsets></xref><offsets xml_i="6656" xml_f="6811" txt_i="1984" txt_f="2139">]. It is believed that prolonged ultraviolet exposure changes the immune surveillance mechanism of the skin, contributing to the tolerance of tumor cells [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="6842" xml_f="6843" txt_i="2139" txt_f="2140">5</offsets></xref><offsets xml_i="6850" xml_f="6977" txt_i="2140" txt_f="2267">]. If left untreated, AK can progress to squamous cell carcinoma, a locally aggressive and occasionally metastatic tumor type [</offsets><xref ref-type="bibr" rid="B6"><offsets xml_i="7008" xml_f="7009" txt_i="2267" txt_f="2268">6</offsets></xref><offsets xml_i="7016" xml_f="7104" txt_i="2268" txt_f="2356">]. Standard treatment of AK includes various types of surgical and chemical treatments [</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="7135" xml_f="7136" txt_i="2356" txt_f="2357">7</offsets></xref><offsets xml_i="7143" xml_f="7144" txt_i="2357" txt_f="2358">,</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="7175" xml_f="7176" txt_i="2358" txt_f="2359">8</offsets></xref><offsets xml_i="7183" xml_f="7284" txt_i="2359" txt_f="2460">], which are often associated with scarring and infection, and may not address sub clinical lesions [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="7315" xml_f="7316" txt_i="2460" txt_f="2461">8</offsets></xref><offsets xml_i="7323" xml_f="7325" txt_i="2461" txt_f="2463">].</offsets></p><p><offsets xml_i="7332" xml_f="7529" txt_i="2464" txt_f="2661">Toll-like receptors (TLR) are pattern recognition receptors that detect pathogen-associated molecular patterns (PAMPs) and play key roles in the activation of innate and adaptive immune responses [</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="7560" xml_f="7561" txt_i="2661" txt_f="2662">9</offsets></xref><offsets xml_i="7568" xml_f="7569" txt_i="2662" txt_f="2663">,</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="7601" xml_f="7603" txt_i="2663" txt_f="2665">10</offsets></xref><offsets xml_i="7610" xml_f="7725" txt_i="2665" txt_f="2780">]. Currently, 10 human TLRs have been identified. The natural ligands for all but TLR10 have also been identified [</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="7756" xml_f="7757" txt_i="2780" txt_f="2781">9</offsets></xref><offsets xml_i="7764" xml_f="7886" txt_i="2781" txt_f="2903">]. Toll-like receptors are primarily expressed on immune cells such as monocytes, dendritic cells (DCs), and lymphocytes [</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="7918" xml_f="7920" txt_i="2903" txt_f="2905">11</offsets></xref><offsets xml_i="7927" xml_f="8049" txt_i="2905" txt_f="3027">], but some TLRs are also expressed on nonimmune cells, including endothelial cells, epithelial cells, and keratinocytes [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="8081" xml_f="8083" txt_i="3027" txt_f="3029">12</offsets></xref><offsets xml_i="8090" xml_f="8092" txt_i="3029" txt_f="3031">].</offsets></p><p><offsets xml_i="8099" xml_f="8205" txt_i="3032" txt_f="3138">The role of TLRs in the pathogenesis and treatment of dermatological diseases is increasingly recognized [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="8237" xml_f="8239" txt_i="3138" txt_f="3140">13</offsets></xref><offsets xml_i="8246" xml_f="8370" txt_i="3140" txt_f="3264">]. Imiquimod, a member of a class of drugs termed immune response modifiers has been shown to be a selective TLR7 agonist [[</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="8402" xml_f="8404" txt_i="3264" txt_f="3266">14</offsets></xref><offsets xml_i="8411" xml_f="8412" txt_i="3266" txt_f="3267">,</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="8444" xml_f="8446" txt_i="3267" txt_f="3269">15</offsets></xref><offsets xml_i="8453" xml_f="8685" txt_i="3269" txt_f="3501">], and unpublished internal data]. Imiquimod is the first TLR-agonist pharmaceutical product approved for human use, and is indicated for the topical treatment of external genital and perianal warts caused by human papilloma virus [</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="8717" xml_f="8719" txt_i="3501" txt_f="3503">16</offsets></xref><offsets xml_i="8726" xml_f="8811" txt_i="3503" txt_f="3588">]. Recently, the approved indications have been expanded to include treatment of AK [</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="8843" xml_f="8845" txt_i="3588" txt_f="3590">17</offsets></xref><offsets xml_i="8852" xml_f="8892" txt_i="3590" txt_f="3630">] and superficial basal cell carcinoma [</offsets><xref ref-type="bibr" rid="B18"><offsets xml_i="8924" xml_f="8926" txt_i="3630" txt_f="3632">18</offsets></xref><offsets xml_i="8933" xml_f="8934" txt_i="3632" txt_f="3633">-</offsets><xref ref-type="bibr" rid="B20"><offsets xml_i="8966" xml_f="8968" txt_i="3633" txt_f="3635">20</offsets></xref><offsets xml_i="8975" xml_f="8977" txt_i="3635" txt_f="3637">].</offsets></p><p><offsets xml_i="8984" xml_f="9292" txt_i="3638" txt_f="3946">The antiviral and anti-tumor activity of imiquimod is believed to be due to the activation of the innate immune response, specifically activation of antigen-presenting cells such as monocytes, macrophages and plasmacytoid and myeloid DCs to induce interferon alpha (IFNα) and other cytokines and chemokines [</offsets><xref ref-type="bibr" rid="B21"><offsets xml_i="9324" xml_f="9326" txt_i="3946" txt_f="3948">21</offsets></xref><offsets xml_i="9333" xml_f="9334" txt_i="3948" txt_f="3949">,</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="9366" xml_f="9368" txt_i="3949" txt_f="3951">15</offsets></xref><offsets xml_i="9375" xml_f="9491" txt_i="3951" txt_f="4067">]. Imiquimod also enhances co stimulatory molecule expression important for triggering an adaptive immune response [</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="9523" xml_f="9525" txt_i="4067" txt_f="4069">15</offsets></xref><offsets xml_i="9532" xml_f="9663" txt_i="4069" txt_f="4200">]. Topical application of the drug has been shown to induce IFNα and interleukin 6 (IL6) in AK lesions and external genital warts [</offsets><xref ref-type="bibr" rid="B22"><offsets xml_i="9695" xml_f="9697" txt_i="4200" txt_f="4202">22</offsets></xref><offsets xml_i="9704" xml_f="9705" txt_i="4202" txt_f="4203">,</offsets><xref ref-type="bibr" rid="B23"><offsets xml_i="9737" xml_f="9739" txt_i="4203" txt_f="4205">23</offsets></xref><offsets xml_i="9746" xml_f="9887" txt_i="4205" txt_f="4346">]. Imiquimod and the chemically related immune response modifier resiquimod have also shown potent vaccine adjuvant effects in mice and man [</offsets><xref ref-type="bibr" rid="B23"><offsets xml_i="9919" xml_f="9921" txt_i="4346" txt_f="4348">23</offsets></xref><offsets xml_i="9928" xml_f="9929" txt_i="4348" txt_f="4349">-</offsets><xref ref-type="bibr" rid="B27"><offsets xml_i="9961" xml_f="9963" txt_i="4349" txt_f="4351">27</offsets></xref><offsets xml_i="9970" xml_f="10183" txt_i="4351" txt_f="4564">]. Even though the immune-modulatory activity of imiquimod is well established, the precise molecular changes responsible for the antilesional activity of topically applied imiquimod in AK is not fully understood.</offsets></p><p><offsets xml_i="10190" xml_f="10386" txt_i="4565" txt_f="4761">The objective of this study was to explore the molecular processes responsible for the antilesional activity of imiquimod in subjects with actinic keratosis using global gene expression profiling.</offsets></p></sec><sec sec-type="materials|methods"><title><offsets xml_i="10437" xml_f="10458" txt_i="4763" txt_f="4784">Methods and Materials</offsets></title><sec><title><offsets xml_i="10478" xml_f="10521" txt_i="4785" txt_f="4828">Institutional review board/informed consent</offsets></title><p><offsets xml_i="10532" xml_f="11186" txt_i="4829" txt_f="5483">This study was conducted at Loma Linda University School of Medicine/Medical Center, Department of Internal Medicine, Division of Dermatology, Loma Linda, California. The study protocol, subject informed consent documents, and subject information documents were submitted to and received approval from the study center's Institution Review Board. This study was conducted according to the Code of Federal Regulations of the United States Food and Drug Administration (21 CFR Part 56, Institutional Review Boards, and Part 50, Protection of Human Subjects) and the International Conference on Harmonization Edition 6, Guideline for Good Clinical Practice.</offsets></p></sec><sec><title><offsets xml_i="11208" xml_f="11221" txt_i="5485" txt_f="5498">Study Conduct</offsets></title><p><offsets xml_i="11232" xml_f="11399" txt_i="5499" txt_f="5666">This was a phase II, double-blind, placebo-controlled, randomized parallel group study. Randomized subjects had at least 5 clinically visible AK lesions within a 25-cm</offsets><sup><offsets xml_i="11404" xml_f="11406" txt_i="5666" txt_f="5668">2 </offsets></sup><offsets xml_i="11412" xml_f="11941" txt_i="5668" txt_f="6197">area on the balding scalp. Subjects were randomized to imiquimod or placebo cream in a 3:1 ratio and applied study cream to the treatment area 3 times per week for 4 weeks. Study cream was applied prior to normal sleeping hours and remained on the skin for approximately 8 hours before it was removed. Safety evaluations were made at all treatment and post treatment visits, and included monitoring of adverse events and local skin reactions, as well as photographing the treatment area and reviewing any concomitant medications.</offsets></p><p><offsets xml_i="11948" xml_f="13290" txt_i="6198" txt_f="7540">At the screening visit, AK lesions were assessed clinically and by confocal microscopy and a representative lesion confirmed by histology. Because AK lesions are in general small in size, histology and gene expression analysis could not be performed on the same biopsy. Therefore, confocal microscopy was performed to establish a correlation of the confocal images and their respective non-sun exposed non-lesional skin, sun exposed non-lesional skin, actinic keratoses lesions, and squamous cell carcinoma. All subjects with lesions histologically identified as having a degree of dysplesia suggestive of squamous cell carcinoma were disqualified from the study. All sites identified as AK lesions were marked and a plastic template of their locations made for exact identification at a later time. Thereafter, changes in AK lesions due to treatment with imiquimod were assessed clinically and by confocal microscopy. Lesions were scored as cleared if the skin exhibited normal epidermis as assessed clinically and by confocal microscopy. Assessment of lesion regression and the results of the confocal microscopy as they relate to aberrant gene expression in AK are discussed in a manuscript submitted for publication (Torres et al, 2006. Micro Array Analysis of Aberrant Gene Expression in Actinic Keratosis: Effect of Imiquimod 5% Cream).</offsets></p><p><offsets xml_i="13297" xml_f="14548" txt_i="7541" txt_f="8792">At the treatment initiation visit, a shave biopsy was taken for gene expression analysis from an untreated AK lesion, from a sun-exposed non-lesional site on the head and from a non-lesional sun-unexposed site from under the arm area. An additional biopsy was taken at each subsequent study visit, (treatment period weeks 1, 2, and 4) and at 4 weeks post treatment of either a remaining AK, or if no AKs were present, of nonlesional skin at a previously identified lesional site. Thus each biopsy was of a different AK lesion. In an effort to standardize the amount of tissue that was removed at each biopsy, the same size punch was used to score the skin surrounding all the lesions to be biopsied for gene expression studies with an attempt to shave-biopsy the lesion at the papillary dermis level. Biopsies were taken 8 hr to 16 hr after the third treatment of the treatment period. The nonlesional, sun-unexposed site biopsy was used to establish a baseline for comparison of gene expression changes in AK before and after treatment with imiquimod. Shave biopsies were immediately immersed in RNALater (Ambion, Austin, Texas), equilibrated at room temperature for 1 hour, kept at 4°C for 24 hours, and then stored at -20°C prior to RNA extraction.</offsets></p></sec><sec><title><offsets xml_i="14570" xml_f="14601" txt_i="8794" txt_f="8825">RNA extraction and purification</offsets></title><p><offsets xml_i="14612" xml_f="15253" txt_i="8826" txt_f="9467">Total RNA from the biopsy samples was extracted and purified using Qiagen RNeasy Mini Kit Protocol for the Isolation of Total RNA from Heart, Muscle and Skin Tissue (Qiagen, Valencia, California) according to manufacturer's instructions. RNA yield varied from 1.5 to 14 υg. The purity of the RNA was determined by the 260 nm/280 nm absorbance ratio. The median 260 nm/280 nm ratio value for the 119 samples was 2.0 (range 1.9 to 2.2). RNA integrity was determined using the Agilent 2100 Bioanalyzer and the RNA 6000 Nano Assay (Agilant Technologies, Palo Alto, CA). All samples gave 28S/18S ratio between 2 and 3 indicating good quality RNA.</offsets></p></sec><sec><title><offsets xml_i="15275" xml_f="15295" txt_i="9469" txt_f="9489">Micro array analysis</offsets></title><p><offsets xml_i="15306" xml_f="15478" txt_i="9490" txt_f="9662">Samples for micro array analysis were prepared by 2 rounds of linear target amplification according to the Affymetrix instructions for eukaryotic small sample preparation [</offsets><xref ref-type="bibr" rid="B28"><offsets xml_i="15510" xml_f="15512" txt_i="9662" txt_f="9664">28</offsets></xref><offsets xml_i="15519" xml_f="15520" txt_i="9664" txt_f="9665">,</offsets><xref ref-type="bibr" rid="B29"><offsets xml_i="15552" xml_f="15554" txt_i="9665" txt_f="9667">29</offsets></xref><offsets xml_i="15561" xml_f="15724" txt_i="9667" txt_f="9830">]. Briefly, double-stranded cDNA was synthesized from 100 ng of total RNA with oligo(dT)24T7 primer (Affymetrix, Santa Clara, California), followed by 2 cycles of </offsets><italic><offsets xml_i="15732" xml_f="15741" txt_i="9830" txt_f="9839">in vitro </offsets></italic><offsets xml_i="15750" xml_f="15792" txt_i="9839" txt_f="9881">transcription of cRNA. The first cycle of </offsets><italic><offsets xml_i="15800" xml_f="15808" txt_i="9881" txt_f="9889">invitro </offsets></italic><offsets xml_i="15817" xml_f="16882" txt_i="9889" txt_f="10954">transcription was performed using a T7 polymerase (MEGAscript T7 Kit, Ambion, Austin, Texas) and the second cycle using Enzo BioArray High Yield RNA Transcript Labeling Kit (Affymetrix). The median value for the biotinylated cRNA yield over the 119 samples was 90 (range 42 to 121). The biotinylated cRNA was hybridized to Affymetrix U133A and U133B GeneChip arrays containing 22,253 probes sets each. Each array was hybridized for 16 hours, washed, then stained with streptavidin-phycoerythrin conjugate and scanned according to manufacturer's instructions. Images were analyzed using Micro Array Suite Version 5 (MAS5). Chips were normalized to a global average intensity of 150 to allow chip-to-chip comparison. The quality of the images was ascertained by monitoring the noise, background, percent transcript present, and the 3'/5' ratio for the housekeeping gene glyceraldehyde-3-phosphate dehydrogenase (GAPDH) which ranged from 1.5 to 5 and beta actin which ranged from 7 to 24. These values are similar to those reported for double amplification protocols [</offsets><xref ref-type="bibr" rid="B29"><offsets xml_i="16914" xml_f="16916" txt_i="10954" txt_f="10956">29</offsets></xref><offsets xml_i="16923" xml_f="16924" txt_i="10956" txt_f="10957">,</offsets><xref ref-type="bibr" rid="B30"><offsets xml_i="16956" xml_f="16958" txt_i="10957" txt_f="10959">30</offsets></xref><offsets xml_i="16965" xml_f="16967" txt_i="10959" txt_f="10961">].</offsets></p></sec><sec><title><offsets xml_i="16989" xml_f="17054" txt_i="10963" txt_f="11028">TaqMan™ real time reverse transcriptase polymerase chain reaction</offsets></title><p><offsets xml_i="17065" xml_f="17586" txt_i="11029" txt_f="11550">TaqMan real time RT-PCR was performed for a number of the genes to confirm the micro array results. cDNA was reverse transcribed from total RNA using Invitrogen Superscript First-Strand Synthesis System for RT-PCR (Invitrogen, Carlsbad, California). Real time RT-PCR was performed using the Applied Biosystems 7900HT™ sequence detection instrument (Applied Biosystems, Foster City, California), and TaqMan low density custom array micro fluidic cards (Applied Biosystems, Foster City, California) as described previously[</offsets><xref ref-type="bibr" rid="B31"><offsets xml_i="17618" xml_f="17620" txt_i="11550" txt_f="11552">31</offsets></xref><offsets xml_i="17627" xml_f="18632" txt_i="11552" txt_f="12557">]. The micro fluidic cards consisted of 8 ports with 24 different TaqMan primer pair/probe sets arrayed in duplicate in a 384-well micro plate footprint. Each well contained a gene-specific forward and reverse primer, as well as a gene-specific probe, which is labeled at the 5' position with 6FAM (areporter dye) and at the 3' position with minor groove binder/non-fluorescent quencher. Samples were mixed with TaqMan Universal PCR Master Mix (Applied Biosystems, Foster City, California), applied to each port of the card, and analyzed by PCR on the 7900HT instrument using Applied Biosystems Sequence Detection System 2.0 software according to the manufacturer's instructions. A total of 46 selected genes associated with the Toll-like receptor pathway, apoptosis, cell cycle, and immune cell infiltration were analyzed using the TaqMan arrays. The housekeeping gene glyceraldehyde-3-phosphate dehydrogenase (GAPDH) was used to normalize each sample. The TLR copy number was calculated for 2 ng of RNA.</offsets></p></sec><sec><title><offsets xml_i="18654" xml_f="18691" txt_i="12559" txt_f="12596">Analysis of affymetrix gene chip data</offsets></title><p><offsets xml_i="18702" xml_f="19846" txt_i="12597" txt_f="13741">Signals from the gene chip images of biopsy samples for untreated normal skin obtained from sun-unexposed sites were used as a control for the calculation of changes in expression in samples of pretreatment AK lesions and AK lesions during and after treatment (treatment weeks 1, 2, and 4 and 4 weeks post treatment). The statistical algorithm in MAS5 evaluates the image for the expression signal, the absent/present call, and the p-value associated with the signal. It also evaluates the fold change of the sample relative to the designated control sample expressed as the signal log2 ratio, the p-value associated with the fold change, and the direction of change (increased, I; decreased, D; or no call, NC). Using the Affymetrix data mining tool software, the expression data from MAS5 were filtered on the basis of specific criteria to identify differentially expressed genes. A given gene was retained if 1sample from the series passed the following criteria: signal detection p-value ≤ 0.01, signal log2 ratio ≤ -2 or ≥ 2, and a change call designation of 'increased' (I) or 'decreased' (D). A total of 1682 genes passed these criteria.</offsets></p><p><offsets xml_i="19853" xml_f="20122" txt_i="13742" txt_f="14011">There was a large subject-to-subject variation in the magnitude of changes in expression, as well as the temporal pattern of expression (subject-specific variation in magnitude of response with treatment time) of the 1682 genes. This variation is exemplified in Figure </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="20152" xml_f="20153" txt_i="14011" txt_f="14012">1</offsets></xref><offsets xml_i="20160" xml_f="21076" txt_i="14012" txt_f="14928">, which shows the variation in the temporal expression of IRF7 for selected subjects. Peak expression varied from week 1-treatment to week 4-treatment depending on the subject. This variation may be due to the inherent differences in the responses of subjects and (or) due to variation in the time from treatment to biopsy (range 8 to 16 hr). In order to identify differentially expressed genes in the treatment group without regard to variation in temporal response, the data were reduced in the following manner. The fold change values for samples in the week 1, week 2 and week 4 treatment groups for the 1682 genes for each subject were compared to identify the maximum change in expression (increased or decreased) due to imiquimod or placebo treatment. This value was designated as the treatment-response fold change for imiquimod or placebo and used in the ANOVA described in the Statistical Analysis section.</offsets></p><fig position="float" id="F1"><label><offsets xml_i="21117" xml_f="21125" txt_i="14929" txt_f="14937">Figure 1</offsets></label><caption><p><offsets xml_i="21145" xml_f="21604" txt_i="14937" txt_f="15396">Variation in the temporal expression of IRF7 during treatment with imiquimod as determined by real time RT-PCR. 'AK,' designates pretreatment AK lesions. 'WK1 IMIQ', 'WK2 IMIQ' and 'WK4 IMIQ' designate treatment times week 1, 2 and 4. '4 WK Post designates' 4 weeks post end of treatment (WK4 treatment). Fold change was calculated with respect to sun-unexposed, non-lesional skin. S-01, S-02, S-06, S-15 and S-16 are samples from subjects 1, 2, 6, 15 and 16.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-1"></graphic></fig><p><offsets xml_i="21675" xml_f="22111" txt_i="15397" txt_f="15833">Cluster analysis was performed with Spotfire DecisionSite-8.1 for Functional Genomics (Spotfire Inc, Somerville, Massachusetts), using the Unweighted Pair-Group Method with Arithmetic mean (UPGMA) and the Euclidean similarity measure. Functional categorization of genes was based on gene ontology analysis using the Ontology Browser in Spotfire and gene descriptions at the National Center for Biotechnology Information (NCBI) website [</offsets><xref ref-type="bibr" rid="B32"><offsets xml_i="22143" xml_f="22145" txt_i="15833" txt_f="15835">32</offsets></xref><offsets xml_i="22152" xml_f="22309" txt_i="15835" txt_f="15992">]. The ontology browser calculates a Fisher's Exact Test p-value, which reflects the chance that the gene ontology category is represented by random chance [</offsets><xref ref-type="bibr" rid="B33"><offsets xml_i="22341" xml_f="22343" txt_i="15992" txt_f="15994">33</offsets></xref><offsets xml_i="22350" xml_f="22397" txt_i="15994" txt_f="16038">]. P-value &lt; 0.05 is considered significant.</offsets></p></sec><sec><title><offsets xml_i="22419" xml_f="22439" txt_i="16040" txt_f="16060">Statistical analysis</offsets></title><p><offsets xml_i="22450" xml_f="23638" txt_i="16061" txt_f="17246">The natural log of the fold change (with respect to sun-unexposed normal skin) from the Affymetrix gene expression and the real time RT-PCR experiments were used in an ANOVA to determine statistically significant changes in expression between sample groups. Due to the small sample size of the vehicle group, 4 subjects, compared to 13 subjects in the imiquimod-treated group, a one way ANOVA comparing vehicle-treated subjects to imiquimod-treated subjects is not expected to yield a reliable determination of imiquimod-response genes. Therefore, a 2-way ANOVA using a blocking factor to account for repeat observations on the same subject was used to compare: (1) the fold change for pretreatment AK (n = 13) and the fold change during treatment with imiquimod (n = 13), and (2) the fold change for pretreatment AK (n = 13) and the fold change for samples taken 4 weeks after the last imiquimod treatment (n = 13). The 'during treatment' fold change is the maximum response fold change value (decreased or increased) selected from week 1, week 2 and week 4 treatment fold changes. Differences between sample groups were considered significant if the p-value for the ANOVA was &lt; 0.05.</offsets></p></sec></sec><sec><title><offsets xml_i="23666" xml_f="23688" txt_i="17249" txt_f="17271">Results and Discussion</offsets></title><sec><title><offsets xml_i="23708" xml_f="23746" txt_i="17272" txt_f="17310">Demographics and response to treatment</offsets></title><p><offsets xml_i="23757" xml_f="24685" txt_i="17311" txt_f="18239">Seventeen white males were randomized to receive treatment (13 to imiquimod and 4 to placebo). The mean age was 75 years (range, 62 to 89 years). All 17 subjects in the study completed the treatment and post treatment portions of the study. The median number of AK lesions at baseline was 10 per subject (range, 6 to 13 lesions). Because of the short duration of the follow-up period (4 weeks for the post treatment period), efficacy was not measured in this study. However, clinical clearance was observed in 25% of the imiquimod-treated subjects 4 weeks after the end of treatment. Imiquimod-treated subjects 1, 2, 4, and 6 and placebo-treated subject 7 were assessed as having clinical clearance of lesions 4 weeks post treatment as determined by return of the lesional site to normal skin. The complete clearance rate in a study where AK subjects were treated for 16 weeks, with a post treatment period of 8 weeks, was 57% [</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="24717" xml_f="24719" txt_i="18239" txt_f="18241">17</offsets></xref><offsets xml_i="24726" xml_f="24728" txt_i="18241" txt_f="18243">].</offsets></p></sec><sec><title><offsets xml_i="24750" xml_f="24840" txt_i="18245" txt_f="18335">Analysis of global gene expression using Affymetix GeneChips: Gene Ontology classification</offsets></title><p><offsets xml_i="24851" xml_f="25401" txt_i="18336" txt_f="18877">A 2-way analysis of variance (ANOVA) was performed comparing the treatment response gene expression fold change (the maximum response value from week 1, week 2 and week 4 treatments) of samples from the imiquimod-treated subjects to the gene expression fold change values of their respective pretreatment AK samples. This comparison resulted in 530 unique genes that had p-values &lt; 0.05 and a median fold change from pretreatment AK &lt;-2 for suppressed genes and &gt;2 for induced genes. Data for the 530 genes is documented in [Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="25450" xml_f="25451" txt_i="18877" txt_f="18878">1</offsets></xref><offsets xml_i="25458" xml_f="25678" txt_i="18878" txt_f="19095">]. Two-way ANOVA comparing pretreatment AK samples and samples taken 4 weeks after the last imiquimod treatment resulted in 111 unique genes that differentiated the 2 groups with a p-value &lt; 0.05 [see Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="25727" xml_f="25728" txt_i="19095" txt_f="19096">1</offsets></xref><offsets xml_i="25735" xml_f="25985" txt_i="19096" txt_f="19346">]. The expression of the rest of the genes returned to basal levels 4 weeks post treatment. Of the total number of differentially regulated genes during imiquimod treatment or 4 weeks post treatment, 87% were up-regulated and 13% were down-regulated.</offsets></p><p><offsets xml_i="25992" xml_f="25998" txt_i="19347" txt_f="19353">Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="26030" xml_f="26031" txt_i="19353" txt_f="19354">1</offsets></xref><offsets xml_i="26038" xml_f="27048" txt_i="19354" txt_f="20358"> summarizes the gene ontology classification of imiquimod-modulated genes from the Affymetrix analysis. P-values for representation of ontology categories are given for categories with p-values &lt; 0.05. Of the 530 unique genes whose expression was modulated by treatment, 436 genes had some annotation in the ontology database. Of these, 106 were annotated to the immune response category, 123 to signal transduction, 72 to receptor activity, and 25 to inflammatory response. Specific genes in several molecular function categories are also listed. Other ontology categories that were represented in the data but had p-values &gt;0.05 for representation in the gene ontology category included metabolism (148 genes), development (37 genes) and regulation of transcription (24 genes). The ontology analysis shows that imiquimod treatment of AK results in global gene expression changes impacting various cellular processes, with immune response and signal transduction being the 2 major processes represented.</offsets></p><table-wrap position="float" id="T1"><label><offsets xml_i="27096" xml_f="27103" txt_i="20359" txt_f="20366">Table 1</offsets></label><caption><p><offsets xml_i="27123" xml_f="27192" txt_i="20366" txt_f="20435">Gene Ontology Classification of Imiquimod-Induced Genes in AK Lesions</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><td align="left"><bold><offsets xml_i="27277" xml_f="27285" txt_i="20436" txt_f="20444">Ontology</offsets></bold></td><td align="center"><bold><offsets xml_i="27322" xml_f="27339" txt_i="20445" txt_f="20462">Genes in the Data</offsets></bold><sup><offsets xml_i="27351" xml_f="27352" txt_i="20462" txt_f="20463">1</offsets></sup></td><td align="center"><bold><offsets xml_i="27388" xml_f="27403" txt_i="20464" txt_f="20479">Annotated Genes</offsets></bold><sup><offsets xml_i="27415" xml_f="27416" txt_i="20479" txt_f="20480">2</offsets></sup></td><td align="center"><bold><offsets xml_i="27452" xml_f="27459" txt_i="20481" txt_f="20488">P-value</offsets></bold><sup><offsets xml_i="27471" xml_f="27472" txt_i="20488" txt_f="20489">3</offsets></sup></td><td align="center"><bold><offsets xml_i="27508" xml_f="27528" txt_i="20490" txt_f="20510">Increased Expression</offsets></bold></td><td align="center"><bold><offsets xml_i="27565" xml_f="27585" txt_i="20511" txt_f="20531">Decreased Expression</offsets></bold></td><td align="left"><bold><offsets xml_i="27620" xml_f="27663" txt_i="20532" txt_f="20575">Description of Processes and Specific Genes</offsets></bold></td></tr></thead><tbody><tr><td align="left"><offsets xml_i="27716" xml_f="27721" txt_i="20576" txt_f="20581">Total</offsets></td><td align="center"><offsets xml_i="27745" xml_f="27748" txt_i="20582" txt_f="20585">436</offsets></td><td align="center"><offsets xml_i="27772" xml_f="27777" txt_i="20586" txt_f="20591">17479</offsets></td><td></td><td align="center"><offsets xml_i="27810" xml_f="27813" txt_i="20593" txt_f="20596">369</offsets></td><td align="center"><offsets xml_i="27837" xml_f="27839" txt_i="20597" txt_f="20599">67</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="27879" xml_f="27898" txt_i="20601" txt_f="20620">Response to stimuli</offsets></td><td align="center"><offsets xml_i="27922" xml_f="27925" txt_i="20621" txt_f="20624">137</offsets></td><td align="center"><offsets xml_i="27949" xml_f="27953" txt_i="20625" txt_f="20629">2475</offsets></td><td align="center"><offsets xml_i="27977" xml_f="27985" txt_i="20630" txt_f="20638">2.13E-19</offsets></td><td align="center"><offsets xml_i="28009" xml_f="28012" txt_i="20639" txt_f="20642">125</offsets></td><td align="center"><offsets xml_i="28036" xml_f="28038" txt_i="20643" txt_f="20645">12</offsets></td><td align="left"><offsets xml_i="28060" xml_f="28192" txt_i="20646" txt_f="20778">defense response, response to external biotic stimuli including, pest and pathogens, antimicrobial, anti-fungal, anti-viral response</offsets></td></tr><tr><td align="left"><offsets xml_i="28223" xml_f="28239" txt_i="20779" txt_f="20795">Defense response</offsets></td><td align="center"><offsets xml_i="28263" xml_f="28266" txt_i="20796" txt_f="20799">105</offsets></td><td align="center"><offsets xml_i="28290" xml_f="28294" txt_i="20800" txt_f="20804">1174</offsets></td><td align="center"><offsets xml_i="28318" xml_f="28326" txt_i="20805" txt_f="20813">1.54E-40</offsets></td><td align="center"><offsets xml_i="28350" xml_f="28353" txt_i="20814" txt_f="20817">102</offsets></td><td align="center"><offsets xml_i="28377" xml_f="28378" txt_i="20818" txt_f="20819">3</offsets></td><td align="left"><offsets xml_i="28400" xml_f="28438" txt_i="20820" txt_f="20858">immune response, inflammatory response</offsets></td></tr><tr><td align="left"><offsets xml_i="28469" xml_f="28484" txt_i="20859" txt_f="20874">Immune response</offsets></td><td align="center"><offsets xml_i="28508" xml_f="28511" txt_i="20875" txt_f="20878">106</offsets></td><td align="center"><offsets xml_i="28535" xml_f="28539" txt_i="20879" txt_f="20883">1020</offsets></td><td align="center"><offsets xml_i="28563" xml_f="28571" txt_i="20884" txt_f="20892">3.05E-40</offsets></td><td align="center"><offsets xml_i="28595" xml_f="28598" txt_i="20893" txt_f="20896">104</offsets></td><td align="center"><offsets xml_i="28622" xml_f="28623" txt_i="20897" txt_f="20898">2</offsets></td><td align="left"><offsets xml_i="28645" xml_f="28827" txt_i="20899" txt_f="21081">cellular defense response, humoral defense response, antigen binding, pattern binding, cytokine synthesis, chemokine synthesis, antigen presentation processing, inflammatory response</offsets></td></tr><tr><td align="left"><offsets xml_i="28858" xml_f="28876" txt_i="21082" txt_f="21100">Response to stress</offsets></td><td align="center"><offsets xml_i="28900" xml_f="28902" txt_i="21101" txt_f="21103">68</offsets></td><td align="center"><offsets xml_i="28926" xml_f="28930" txt_i="21104" txt_f="21108">1337</offsets></td><td align="center"><offsets xml_i="28954" xml_f="28962" txt_i="21109" txt_f="21117">1.77E-12</offsets></td><td align="center"><offsets xml_i="28986" xml_f="28988" txt_i="21118" txt_f="21120">63</offsets></td><td align="center"><offsets xml_i="29012" xml_f="29013" txt_i="21121" txt_f="21122">5</offsets></td><td align="left"><offsets xml_i="29035" xml_f="29128" txt_i="21123" txt_f="21216">response to pest and pathogen, inflammatory response, response to virus, response to wounding</offsets></td></tr><tr><td align="left"><offsets xml_i="29159" xml_f="29180" txt_i="21217" txt_f="21238">Inflammatory response</offsets></td><td align="center"><offsets xml_i="29204" xml_f="29206" txt_i="21239" txt_f="21241">25</offsets></td><td align="center"><offsets xml_i="29230" xml_f="29233" txt_i="21242" txt_f="21245">244</offsets></td><td align="center"><offsets xml_i="29257" xml_f="29265" txt_i="21246" txt_f="21254">4.35E-11</offsets></td><td align="center"><offsets xml_i="29289" xml_f="29291" txt_i="21255" txt_f="21257">25</offsets></td><td align="center"><offsets xml_i="29315" xml_f="29316" txt_i="21258" txt_f="21259">0</offsets></td><td align="left"><offsets xml_i="29338" xml_f="29402" txt_i="21260" txt_f="21324">defensive reaction (by vertebrate tissue) to infection or injury</offsets></td></tr><tr><td align="left"><offsets xml_i="29433" xml_f="29453" txt_i="21325" txt_f="21345">Response to wounding</offsets></td><td align="center"><offsets xml_i="29477" xml_f="29479" txt_i="21346" txt_f="21348">39</offsets></td><td align="center"><offsets xml_i="29503" xml_f="29506" txt_i="21349" txt_f="21352">395</offsets></td><td align="center"><offsets xml_i="29530" xml_f="29538" txt_i="21353" txt_f="21361">3.72E-16</offsets></td><td align="center"><offsets xml_i="29562" xml_f="29564" txt_i="21362" txt_f="21364">39</offsets></td><td align="center"><offsets xml_i="29588" xml_f="29589" txt_i="21365" txt_f="21366">0</offsets></td><td align="left"><offsets xml_i="29611" xml_f="29671" txt_i="21367" txt_f="21427">defensive reaction (by vertebrate tissue) to physical injury</offsets></td></tr><tr><td align="left"><offsets xml_i="29702" xml_f="29721" txt_i="21428" txt_f="21447">Signal transduction</offsets></td><td align="center"><offsets xml_i="29745" xml_f="29748" txt_i="21448" txt_f="21451">123</offsets></td><td align="center"><offsets xml_i="29772" xml_f="29776" txt_i="21452" txt_f="21456">3547</offsets></td><td align="center"><offsets xml_i="29800" xml_f="29808" txt_i="21457" txt_f="21465">4.70E-06</offsets></td><td align="center"><offsets xml_i="29832" xml_f="29834" txt_i="21466" txt_f="21468">95</offsets></td><td align="center"><offsets xml_i="29858" xml_f="29860" txt_i="21469" txt_f="21471">17</offsets></td><td align="left"><offsets xml_i="29882" xml_f="30003" txt_i="21472" txt_f="21593">cytokine signaling, death receptor signaling, G-protein coupled receptor signaling, intigrin binding, MHC protein binding</offsets></td></tr><tr><td align="left"><offsets xml_i="30034" xml_f="30051" txt_i="21594" txt_f="21611">Receptor activity</offsets></td><td align="center"><offsets xml_i="30075" xml_f="30077" txt_i="21612" txt_f="21614">72</offsets></td><td align="center"><offsets xml_i="30101" xml_f="30105" txt_i="21615" txt_f="21619">1914</offsets></td><td align="center"><offsets xml_i="30129" xml_f="30137" txt_i="21620" txt_f="21628">1.80E-07</offsets></td><td align="center"><offsets xml_i="30161" xml_f="30163" txt_i="21629" txt_f="21631">62</offsets></td><td align="center"><offsets xml_i="30187" xml_f="30189" txt_i="21632" txt_f="21634">10</offsets></td><td align="left"><offsets xml_i="30211" xml_f="30402" txt_i="21635" txt_f="21826">chemokine receptors, pattern recognition receptors, immunoglobulin receptors, complement receptors, MHC I\MHCII receptors, scavenger receptors, Hematopoietin/interferon class (D200) receptors</offsets></td></tr><tr><td align="left"><offsets xml_i="30433" xml_f="30448" txt_i="21827" txt_f="21842">Antigen binding</offsets></td><td align="center"><offsets xml_i="30472" xml_f="30473" txt_i="21843" txt_f="21844">8</offsets></td><td align="center"><offsets xml_i="30497" xml_f="30499" txt_i="21845" txt_f="21847">91</offsets></td><td align="center"><offsets xml_i="30523" xml_f="30531" txt_i="21848" txt_f="21856">2.66E-03</offsets></td><td align="center"><offsets xml_i="30555" xml_f="30556" txt_i="21857" txt_f="21858">8</offsets></td><td align="center"><offsets xml_i="30580" xml_f="30581" txt_i="21859" txt_f="21860">0</offsets></td><td align="left"><italic><offsets xml_i="30611" xml_f="30666" txt_i="21861" txt_f="21916">SCF1, IGLV2-14, LAG3, LILRA1, SLAMF1, TRA@, TRGC2, TAP2</offsets></italic></td></tr><tr><td align="left"><offsets xml_i="30706" xml_f="30726" txt_i="21917" txt_f="21937">Carbohydrate binding</offsets></td><td align="center"><offsets xml_i="30750" xml_f="30752" txt_i="21938" txt_f="21940">16</offsets></td><td align="center"><offsets xml_i="30776" xml_f="30779" txt_i="21941" txt_f="21944">326</offsets></td><td align="center"><offsets xml_i="30803" xml_f="30811" txt_i="21945" txt_f="21953">1.18E-03</offsets></td><td align="center"><offsets xml_i="30835" xml_f="30837" txt_i="21954" txt_f="21956">14</offsets></td><td align="center"><offsets xml_i="30861" xml_f="30862" txt_i="21957" txt_f="21958">2</offsets></td><td align="left"><italic><offsets xml_i="30892" xml_f="31003" txt_i="21959" txt_f="22070">CCL8, CD69, CLEC4A, FCN1, FGFR2, GALNT7, HMMR, KLRC1/KLRC2, KLRF1, LGALS2, LGALS9, POSTN, PTN, SELL, SELPLG, SN</offsets></italic></td></tr><tr><td align="left"><offsets xml_i="31043" xml_f="31058" txt_i="22071" txt_f="22086">Pattern binding</offsets></td><td align="center"><offsets xml_i="31082" xml_f="31084" txt_i="22087" txt_f="22089">10</offsets></td><td align="center"><offsets xml_i="31108" xml_f="31111" txt_i="22090" txt_f="22093">159</offsets></td><td align="center"><offsets xml_i="31135" xml_f="31143" txt_i="22094" txt_f="22102">4.58E-02</offsets></td><td align="center"><offsets xml_i="31167" xml_f="31168" txt_i="22103" txt_f="22104">8</offsets></td><td align="center"><offsets xml_i="31192" xml_f="31193" txt_i="22105" txt_f="22106">2</offsets></td><td align="left"><italic><offsets xml_i="31223" xml_f="31282" txt_i="22107" txt_f="22166">CCL8, CD14, FGFR2, HMMR, POSTN, PTN, TLR2, TLR7, TLR8, TLR4</offsets></italic></td></tr><tr><td align="left"><offsets xml_i="31322" xml_f="31340" txt_i="22167" txt_f="22185">Cytokine receptors</offsets></td><td align="center"><offsets xml_i="31364" xml_f="31365" txt_i="22186" txt_f="22187">9</offsets></td><td align="center"><offsets xml_i="31389" xml_f="31391" txt_i="22188" txt_f="22190">99</offsets></td><td align="center"><offsets xml_i="31415" xml_f="31423" txt_i="22191" txt_f="22199">1.97E-04</offsets></td><td align="center"><offsets xml_i="31447" xml_f="31448" txt_i="22200" txt_f="22201">9</offsets></td><td align="center"><offsets xml_i="31472" xml_f="31473" txt_i="22202" txt_f="22203">0</offsets></td><td align="left"><italic><offsets xml_i="31503" xml_f="31567" txt_i="22204" txt_f="22268">CCR5, CCR1, CD74, CSF2RB, CXCR4, IL10RA, IL1RL1, IL21R, TNFRSF1B</offsets></italic></td></tr><tr><td align="left"><offsets xml_i="31607" xml_f="31616" txt_i="22269" txt_f="22278">Cytokines</offsets></td><td align="center"><offsets xml_i="31640" xml_f="31642" txt_i="22279" txt_f="22281">12</offsets></td><td align="center"><offsets xml_i="31666" xml_f="31669" txt_i="22282" txt_f="22285">257</offsets></td><td align="center"><offsets xml_i="31693" xml_f="31701" txt_i="22286" txt_f="22294">1.11E-04</offsets></td><td align="center"><offsets xml_i="31725" xml_f="31727" txt_i="22295" txt_f="22297">11</offsets></td><td align="center"><offsets xml_i="31751" xml_f="31752" txt_i="22298" txt_f="22299">1</offsets></td><td align="left"><italic><offsets xml_i="31782" xml_f="31866" txt_i="22300" txt_f="22384">CCL3, CCL5, CCL8, CXCL5, CXCL10, CXCL11, CXCL12, CXCL16, TNFSF10, ECGF1, PTN, SLURP1</offsets></italic></td></tr><tr><td align="left"><offsets xml_i="31906" xml_f="31928" txt_i="22385" txt_f="22407">Immunoglobulin binding</offsets></td><td align="center"><offsets xml_i="31952" xml_f="31953" txt_i="22408" txt_f="22409">4</offsets></td><td align="center"><offsets xml_i="31977" xml_f="31979" txt_i="22410" txt_f="22412">25</offsets></td><td align="center"><offsets xml_i="32003" xml_f="32011" txt_i="22413" txt_f="22421">1.56E-03</offsets></td><td align="center"><offsets xml_i="32035" xml_f="32036" txt_i="22422" txt_f="22423">4</offsets></td><td align="center"><offsets xml_i="32060" xml_f="32061" txt_i="22424" txt_f="22425">0</offsets></td><td align="left"><italic><offsets xml_i="32091" xml_f="32113" txt_i="22426" txt_f="22448">FCER1A, FCER1G, FCGR3B</offsets></italic><offsets xml_i="32122" xml_f="32124" txt_i="22448" txt_f="22450">, </offsets><italic><offsets xml_i="32132" xml_f="32143" txt_i="22450" txt_f="22461">214511_x_at</offsets></italic></td></tr><tr><td align="left"><offsets xml_i="32183" xml_f="32209" txt_i="22462" txt_f="22488">Cystein type endopeptidase</offsets></td><td align="center"><offsets xml_i="32233" xml_f="32234" txt_i="22489" txt_f="22490">9</offsets></td><td align="center"><offsets xml_i="32258" xml_f="32261" txt_i="22491" txt_f="22494">169</offsets></td><td align="center"><offsets xml_i="32285" xml_f="32293" txt_i="22495" txt_f="22503">8.21E-03</offsets></td><td align="center"><offsets xml_i="32317" xml_f="32318" txt_i="22504" txt_f="22505">9</offsets></td><td align="center"><offsets xml_i="32342" xml_f="32343" txt_i="22506" txt_f="22507">0</offsets></td><td align="left"><italic><offsets xml_i="32373" xml_f="32431" txt_i="22508" txt_f="22566">CASP1, CASP8, CTSB, CTSL, CTSC, STSS, LGMN, TNFAIP3, USP18</offsets></italic></td></tr></tbody></table><table-wrap-foot><p><sup><offsets xml_i="32491" xml_f="32492" txt_i="22567" txt_f="22568">1</offsets></sup><offsets xml_i="32498" xml_f="32587" txt_i="22568" txt_f="22657">Number of genes in the data for which annotation was found in the Gene Ontology database.</offsets></p><p><sup><offsets xml_i="32599" xml_f="32600" txt_i="22658" txt_f="22659">2</offsets></sup><offsets xml_i="32606" xml_f="32673" txt_i="22659" txt_f="22726">Total number of gene with annotation in the Gene Ontology database.</offsets></p><p><sup><offsets xml_i="32685" xml_f="32686" txt_i="22727" txt_f="22728">3</offsets></sup><offsets xml_i="32692" xml_f="32757" txt_i="22728" txt_f="22793">P-value for representation of gene ontology category in the data.</offsets></p></table-wrap-foot></table-wrap></sec><sec><title><offsets xml_i="32810" xml_f="32940" txt_i="22795" txt_f="22925">Validation of selected genes observed in the Affymetrix experiment using real time reverse transcriptase polymerase chain reaction</offsets></title><p><offsets xml_i="32951" xml_f="33209" txt_i="22926" txt_f="23184">Micro array analysis may not be sensitive enough to capture all changes in gene expression. In comparison, real time reverse transcriptase polymerase chain reaction (RT-PCR) has been shown to be more sensitive for the detection of low abundance transcripts [</offsets><xref ref-type="bibr" rid="B31"><offsets xml_i="33241" xml_f="33243" txt_i="23184" txt_f="23186">31</offsets></xref><offsets xml_i="33250" xml_f="33415" txt_i="23186" txt_f="23351">]. Therefore, we analyzed AK samples and imiquimod treated AK samples for a selected set of genes using real time RT-PCR. The data is summarized in [Additional File </offsets><xref ref-type="supplementary-material" rid="S2"><offsets xml_i="33464" xml_f="33465" txt_i="23351" txt_f="23352">2</offsets></xref><offsets xml_i="33472" xml_f="33792" txt_i="23352" txt_f="23672">] which compares the median fold change values (13 imiquimod-treated subjects) measured using both Affymetrix GeneChip analysis and RT-PCR for a subset of the genes. In general, good agreement was obtained between the 2 methods, with the RT-PCR data showing similar or higher expression than the Affymetrix data. Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="33822" xml_f="33823" txt_i="23672" txt_f="23673">2</offsets></xref><offsets xml_i="33830" xml_f="34847" txt_i="23673" txt_f="24687"> shows a comparison of the 2 methods for the expression of interferon regulatory factor 7 (IRF7). The individual fold change values for the 13 imiquimod-treated subjects during imiquimod treatment were used for the regression analysis (R Square = 0.83). In general, the direction of change in expression for each gene was the same in both assays whereas the magnitude of the fold change values was higher for the RT-PCR analysis than the Affymetrix analysis. Of the 46 genes evaluated on both gene expression platforms, only 3 genes (CD80, MyD88 and TLR6) were identified as having changed in expression in the RT-PCR experiments only (p value ANOVA analysis &lt; 0.05). We did not identify any genes that were differentially expressed by the Affymetrix method that were not also differentially expressed by real time RT-PCR. Thus, overall good agreement was obtained between the 2 methods, with differences between the 2 methods likely due to the increased sensitivity and quantitative nature of the RT-PCR platform.</offsets></p><fig position="float" id="F2"><label><offsets xml_i="34888" xml_f="34896" txt_i="24688" txt_f="24696">Figure 2</offsets></label><caption><p><offsets xml_i="34916" xml_f="35045" txt_i="24696" txt_f="24825">Comparison of gene expression data obtained by Affymetrix GeneChip analysis and real time RT-PCR. Linear regression analysis for </offsets><italic><offsets xml_i="35053" xml_f="35058" txt_i="24825" txt_f="24830">IRF7 </offsets></italic><offsets xml_i="35067" xml_f="35475" txt_i="24830" txt_f="25238">comparing fold change values for individual subjects as measured by Affymetrix analysis and real time RT-PCR analysis. The imiquimod response fold change, which is the maximum response from week 1, week 2 and week 4 treatment times was used. Fold change was calculated relative to sun-unexposed, nonlesional skin samples for 13 subjects treated with imiquimod. The R square value for the comparison was 0.83.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-2"></graphic></fig></sec><sec><title><offsets xml_i="35561" xml_f="35652" txt_i="25240" txt_f="25331">Imiquimod increases expression of pattern-recognition receptors of the innate immune system</offsets></title><p><offsets xml_i="35663" xml_f="35724" txt_i="25332" txt_f="25393">Since imiquimod is a TLR7 agonist, we sought to determine if </offsets><italic><offsets xml_i="35732" xml_f="35737" txt_i="25393" txt_f="25398">TLR7 </offsets></italic><offsets xml_i="35746" xml_f="36022" txt_i="25398" txt_f="25674">and other TLRs were expressed in AK lesions and if treatment with imiquimod altered their expression. We analyzed expression levels of various TLRs in biopsy samples of pretreatment AK and after treatment with imiquimod using RT-PCR. We also measured the expression levels of </offsets><italic><offsets xml_i="36030" xml_f="36035" txt_i="25674" txt_f="25679">MyD88</offsets></italic><offsets xml_i="36044" xml_f="36093" txt_i="25679" txt_f="25728">, an adaptor molecule for various TLRs including </offsets><italic><offsets xml_i="36101" xml_f="36105" txt_i="25728" txt_f="25732">TLR7</offsets></italic><offsets xml_i="36114" xml_f="36120" txt_i="25732" txt_f="25738">; and </offsets><italic><offsets xml_i="36128" xml_f="36132" txt_i="25738" txt_f="25742">IRF7</offsets></italic><offsets xml_i="36141" xml_f="36271" txt_i="25742" txt_f="25872">, a transcription factor recently shown to be important in the regulation of interferon gene expression through the TLR7 pathway [</offsets><xref ref-type="bibr" rid="B34"><offsets xml_i="36303" xml_f="36305" txt_i="25872" txt_f="25874">34</offsets></xref><offsets xml_i="36312" xml_f="36322" txt_i="25874" txt_f="25884">]. Figure </offsets><xref ref-type="fig" rid="F3"><offsets xml_i="36352" xml_f="36353" txt_i="25884" txt_f="25885">3</offsets></xref><offsets xml_i="36360" xml_f="36417" txt_i="25885" txt_f="25942"> shows the median expression levels of the various TLRs, </offsets><italic><offsets xml_i="36425" xml_f="36431" txt_i="25942" txt_f="25948">MyD88 </offsets></italic><offsets xml_i="36440" xml_f="36444" txt_i="25948" txt_f="25952">and </offsets><italic><offsets xml_i="36452" xml_f="36457" txt_i="25952" txt_f="25957">IRF7 </offsets></italic><offsets xml_i="36466" xml_f="36574" txt_i="25957" txt_f="26065">for subjects treated with imiquimod (n = 13) during treatment and 4 weeks post treatment. The expression of </offsets><italic><offsets xml_i="36582" xml_f="36586" txt_i="26065" txt_f="26069">TLR1</offsets></italic><offsets xml_i="36595" xml_f="36597" txt_i="26069" txt_f="26071">, </offsets><italic><offsets xml_i="36605" xml_f="36609" txt_i="26071" txt_f="26075">TLR3</offsets></italic><offsets xml_i="36618" xml_f="36620" txt_i="26075" txt_f="26077">, </offsets><italic><offsets xml_i="36628" xml_f="36632" txt_i="26077" txt_f="26081">TLR6</offsets></italic><offsets xml_i="36641" xml_f="36643" txt_i="26081" txt_f="26083">, </offsets><italic><offsets xml_i="36651" xml_f="36655" txt_i="26083" txt_f="26087">TLR7</offsets></italic><offsets xml_i="36664" xml_f="36666" txt_i="26087" txt_f="26089">, </offsets><italic><offsets xml_i="36674" xml_f="36678" txt_i="26089" txt_f="26093">TLR8</offsets></italic><offsets xml_i="36687" xml_f="36689" txt_i="26093" txt_f="26095">, </offsets><italic><offsets xml_i="36697" xml_f="36701" txt_i="26095" txt_f="26099">TLR9</offsets></italic><offsets xml_i="36710" xml_f="36712" txt_i="26099" txt_f="26101">, </offsets><italic><offsets xml_i="36720" xml_f="36726" txt_i="26101" txt_f="26107">MyD88 </offsets></italic><offsets xml_i="36735" xml_f="36739" txt_i="26107" txt_f="26111">and </offsets><italic><offsets xml_i="36747" xml_f="36752" txt_i="26111" txt_f="26116">IRF7 </offsets></italic><offsets xml_i="36761" xml_f="36879" txt_i="26116" txt_f="26231">were all increased at statistically significant levels (p-value &lt; 0.05, during treatment [see also Additional File </offsets><xref ref-type="supplementary-material" rid="S2"><offsets xml_i="36928" xml_f="36929" txt_i="26231" txt_f="26232">2</offsets></xref><offsets xml_i="36936" xml_f="36955" txt_i="26232" txt_f="26251">]. The increase in </offsets><italic><offsets xml_i="36963" xml_f="36968" txt_i="26251" txt_f="26256">TLR4 </offsets></italic><offsets xml_i="36977" xml_f="37270" txt_i="26256" txt_f="26549">expression did not reach a statistically significant level (p-value = 0.066) in the imiquimod-treated samples, whereas the Affymetrix data shows a statistically significant increase (p-value of 0.028). The most statistically significant changes observed upon treatment with imiquimod were for </offsets><italic><offsets xml_i="37278" xml_f="37282" txt_i="26549" txt_f="26553">TLR3</offsets></italic><offsets xml_i="37291" xml_f="37293" txt_i="26553" txt_f="26555">, </offsets><italic><offsets xml_i="37301" xml_f="37305" txt_i="26555" txt_f="26559">TLR7</offsets></italic><offsets xml_i="37314" xml_f="37316" txt_i="26559" txt_f="26561">, </offsets><italic><offsets xml_i="37324" xml_f="37328" txt_i="26561" txt_f="26565">TLR8</offsets></italic><offsets xml_i="37337" xml_f="37343" txt_i="26565" txt_f="26571">, and </offsets><italic><offsets xml_i="37351" xml_f="37355" txt_i="26571" txt_f="26575">IRF7</offsets></italic><offsets xml_i="37364" xml_f="37532" txt_i="26575" txt_f="26743">, with good agreement between the Affymetrix and RT-PCR analysis both for the magnitude of change and for the p-values. There was no change in the expression levels of </offsets><italic><offsets xml_i="37540" xml_f="37545" txt_i="26743" txt_f="26748">TLR5 </offsets></italic><offsets xml_i="37554" xml_f="37558" txt_i="26748" txt_f="26752">and </offsets><italic><offsets xml_i="37566" xml_f="37572" txt_i="26752" txt_f="26758">TLR10 </offsets></italic><offsets xml_i="37581" xml_f="37713" txt_i="26758" txt_f="26890">upon treatment with imiquimod. The data are consistent with previous reports of induction of TLRs by various TLR agonists and IFNα [</offsets><xref ref-type="bibr" rid="B36"><offsets xml_i="37745" xml_f="37747" txt_i="26890" txt_f="26892">35</offsets></xref><offsets xml_i="37754" xml_f="37755" txt_i="26892" txt_f="26893">-</offsets><xref ref-type="bibr" rid="B37"><offsets xml_i="37787" xml_f="37789" txt_i="26893" txt_f="26895">37</offsets></xref><offsets xml_i="37796" xml_f="37798" txt_i="26895" txt_f="26897">][</offsets><xref ref-type="bibr" rid="B38"><offsets xml_i="37830" xml_f="37832" txt_i="26897" txt_f="26899">38</offsets></xref><offsets xml_i="37839" xml_f="37840" txt_i="26899" txt_f="26900">,</offsets><xref ref-type="bibr" rid="B39"><offsets xml_i="37872" xml_f="37874" txt_i="26900" txt_f="26902">39</offsets></xref><offsets xml_i="37881" xml_f="38167" txt_i="26902" txt_f="27188">]. The increased expression of the TLRs may be a result of increased expression of the genes in cells resident in the skin (e.g., DCs, macrophages) or due to the influx of cells with high expression of these genes (e.g., DCs, macrophages, plasmacytoid DCs). The increased expression of </offsets><italic><offsets xml_i="38175" xml_f="38179" txt_i="27188" txt_f="27192">TLR3</offsets></italic><offsets xml_i="38188" xml_f="38190" txt_i="27192" txt_f="27194">, </offsets><italic><offsets xml_i="38198" xml_f="38202" txt_i="27194" txt_f="27198">TLR7</offsets></italic><offsets xml_i="38211" xml_f="38217" txt_i="27198" txt_f="27204">, and </offsets><italic><offsets xml_i="38225" xml_f="38230" txt_i="27204" txt_f="27209">TLR8 </offsets></italic><offsets xml_i="38239" xml_f="38421" txt_i="27209" txt_f="27391">is consistent with increased expression observed in human peripheral blood mononuclear cells upon treatment with imiquimod. The data are also consistent with increased expression of </offsets><italic><offsets xml_i="38429" xml_f="38434" txt_i="27391" txt_f="27396">TLR7 </offsets></italic><offsets xml_i="38443" xml_f="38477" txt_i="27396" txt_f="27430">observed in imiquimod-treated AK [</offsets><xref ref-type="bibr" rid="B22"><offsets xml_i="38509" xml_f="38511" txt_i="27430" txt_f="27432">22</offsets></xref><offsets xml_i="38518" xml_f="38762" txt_i="27432" txt_f="27676">]. In summary, treatment of AK lesions with imiquimod results in increased expression of several TLRs and TLR pathway components, thus potentially priming for further amplification of the innate immune system. The increase in the expression of </offsets><italic><offsets xml_i="38770" xml_f="38775" txt_i="27676" txt_f="27681">IRF7 </offsets></italic><offsets xml_i="38784" xml_f="38917" txt_i="27681" txt_f="27814">is also predicted to amplify the innate immune response by increased expression of type 1 interferons and interferon-inducible genes.</offsets></p><fig position="float" id="F3"><label><offsets xml_i="38958" xml_f="38966" txt_i="27815" txt_f="27823">Figure 3</offsets></label><caption><p><offsets xml_i="38986" xml_f="38992" txt_i="27823" txt_f="27829">Basal </offsets><italic><offsets xml_i="39000" xml_f="39003" txt_i="27829" txt_f="27832">TLR</offsets></italic><offsets xml_i="39012" xml_f="39014" txt_i="27832" txt_f="27834">, </offsets><italic><offsets xml_i="39022" xml_f="39026" txt_i="27834" txt_f="27838">IRF7</offsets></italic><offsets xml_i="39035" xml_f="39041" txt_i="27838" txt_f="27844">, and </offsets><italic><offsets xml_i="39049" xml_f="39055" txt_i="27844" txt_f="27850">MyD88 </offsets></italic><offsets xml_i="39064" xml_f="39663" txt_i="27850" txt_f="28446">gene expression in skin biopsies as determined by real time RT-PCR. White bars represent pretreatment AK, black bars represent during imiquimod treatment (maximum response value from week 1, week 2 and week 4 treatment times), and hatched bars represent 4-weeks post treatment. Relative copy number was determined as outlined in Methods and Materials section. Asterisks indicate those genes that had p-values &lt; 0.05 in the ANOVA, comparing expression in pretreatment AK samples to the maximum response expression in samples from subjects (n = 13) during imiquimod treatment. [See Additional file </offsets><xref ref-type="supplementary-material" rid="S2"><offsets xml_i="39712" xml_f="39713" txt_i="28446" txt_f="28447">2</offsets></xref><offsets xml_i="39720" xml_f="39722" txt_i="28447" txt_f="28449">].</offsets></p></caption><graphic xlink:href="1479-5876-5-7-3"></graphic></fig><p><offsets xml_i="39793" xml_f="40013" txt_i="28450" txt_f="28670">In addition to the various TLR(s) which recognize viral and bacterial components, an intracellular antiviral pathway which detects viral RNA and results in the induction of type1 interferons has recently been described [</offsets><xref ref-type="bibr" rid="B40"><offsets xml_i="40045" xml_f="40047" txt_i="28670" txt_f="28672">40</offsets></xref><offsets xml_i="40054" xml_f="40055" txt_i="28672" txt_f="28673">-</offsets><xref ref-type="bibr" rid="B42"><offsets xml_i="40087" xml_f="40089" txt_i="28673" txt_f="28675">42</offsets></xref><offsets xml_i="40096" xml_f="40186" txt_i="28675" txt_f="28765">]. The central components of this pathway, which are also inducible by type1 interferons [</offsets><xref ref-type="bibr" rid="B43"><offsets xml_i="40218" xml_f="40220" txt_i="28765" txt_f="28767">43</offsets></xref><offsets xml_i="40227" xml_f="40228" txt_i="28767" txt_f="28768">,</offsets><xref ref-type="bibr" rid="B44"><offsets xml_i="40260" xml_f="40262" txt_i="28768" txt_f="28770">44</offsets></xref><offsets xml_i="40269" xml_f="40275" txt_i="28770" txt_f="28776">] are </offsets><italic><offsets xml_i="40283" xml_f="40289" txt_i="28776" txt_f="28782">DDX58 </offsets></italic><offsets xml_i="40298" xml_f="40299" txt_i="28782" txt_f="28783">(</offsets><italic><offsets xml_i="40307" xml_f="40312" txt_i="28783" txt_f="28788">RIG-I</offsets></italic><offsets xml_i="40321" xml_f="40357" txt_i="28788" txt_f="28824">, retinoid acid inducible gene) and </offsets><italic><offsets xml_i="40365" xml_f="40371" txt_i="28824" txt_f="28830">IFIH1 </offsets></italic><offsets xml_i="40380" xml_f="40381" txt_i="28830" txt_f="28831">(</offsets><italic><offsets xml_i="40389" xml_f="40393" txt_i="28831" txt_f="28835">MDA5</offsets></italic><offsets xml_i="40402" xml_f="40632" txt_i="28835" txt_f="29065">, melanoma differentiation antigen 5). Both genes contain helicase domains responsible for detection of double-stranded RNA. They also contain caspase recruitment domains (CARDdomains), which are responsible for signaling through </offsets><italic><offsets xml_i="40640" xml_f="40645" txt_i="29065" txt_f="29070">TBK1 </offsets></italic><offsets xml_i="40654" xml_f="40682" txt_i="29070" txt_f="29098">(resulting in activation of </offsets><italic><offsets xml_i="40690" xml_f="40695" txt_i="29098" txt_f="29103">NFKB </offsets></italic><offsets xml_i="40704" xml_f="40708" txt_i="29103" txt_f="29107">and </offsets><italic><offsets xml_i="40716" xml_f="40720" txt_i="29107" txt_f="29111">IRF3</offsets></italic><offsets xml_i="40729" xml_f="40865" txt_i="29111" txt_f="29247">), as well as induction of type-1 interferons. The activation of this pathway leads to growth inhibition as well as antiviral activity [</offsets><xref ref-type="bibr" rid="B45"><offsets xml_i="40897" xml_f="40899" txt_i="29247" txt_f="29249">45</offsets></xref><offsets xml_i="40906" xml_f="40916" txt_i="29249" txt_f="29259">]. Figure </offsets><xref ref-type="fig" rid="F4"><offsets xml_i="40946" xml_f="40948" txt_i="29259" txt_f="29261">4a</offsets></xref><offsets xml_i="40955" xml_f="40960" txt_i="29261" txt_f="29266"> and </offsets><xref ref-type="fig" rid="F4"><offsets xml_i="40990" xml_f="40992" txt_i="29266" txt_f="29268">4b</offsets></xref><offsets xml_i="40999" xml_f="41030" txt_i="29268" txt_f="29299"> show changes in expression of </offsets><italic><offsets xml_i="41038" xml_f="41044" txt_i="29299" txt_f="29305">DDX58 </offsets></italic><offsets xml_i="41053" xml_f="41057" txt_i="29305" txt_f="29309">and </offsets><italic><offsets xml_i="41065" xml_f="41071" txt_i="29309" txt_f="29315">IFIH1 </offsets></italic><offsets xml_i="41080" xml_f="41274" txt_i="29315" txt_f="29509">upon treatment with imiquimod as determined from Affymetrix GeneChip analysis. The expression of both genes was increased to statistically significant levels during treatment. The expression of </offsets><italic><offsets xml_i="41282" xml_f="41288" txt_i="29509" txt_f="29515">DDX58 </offsets></italic><offsets xml_i="41297" xml_f="41404" txt_i="29515" txt_f="29622">remained higher than its pretreatment level (p-value = 0.004), four weeks, post treatment, whereas that of </offsets><italic><offsets xml_i="41412" xml_f="41418" txt_i="29622" txt_f="29628">IFIH1 </offsets></italic><offsets xml_i="41427" xml_f="41827" txt_i="29628" txt_f="30028">returned to basal level. The induction of several members of the cytoplasmic helicase innate immune pathway, as well as several TLRs, indicates that in addition to activation of the TLR7 pathway, treatment with imiquimod also results in priming of other innate pathways. These pathways may augment other aspects of the innate immune response and may be important for eliminating pre-neoplastic cells.</offsets></p><fig position="float" id="F4"><label><offsets xml_i="41868" xml_f="41876" txt_i="30029" txt_f="30037">Figure 4</offsets></label><caption><p><offsets xml_i="41896" xml_f="42045" txt_i="30037" txt_f="30186">Increase in expression of the helicase family of virus-sensing genes upon treatment with imiquimod as determined by Affymetrix GeneChip analysis. (A)</offsets><italic><offsets xml_i="42053" xml_f="42059" txt_i="30186" txt_f="30192">DDX58 </offsets></italic><offsets xml_i="42068" xml_f="42069" txt_i="30192" txt_f="30193">(</offsets><italic><offsets xml_i="42077" xml_f="42082" txt_i="30193" txt_f="30198">RIG-I</offsets></italic><offsets xml_i="42091" xml_f="42100" txt_i="30198" txt_f="30207">) and (B)</offsets><italic><offsets xml_i="42108" xml_f="42114" txt_i="30207" txt_f="30213">IFIH1 </offsets></italic><offsets xml_i="42123" xml_f="42124" txt_i="30213" txt_f="30214">(</offsets><italic><offsets xml_i="42132" xml_f="42136" txt_i="30214" txt_f="30218">MDA5</offsets></italic><offsets xml_i="42145" xml_f="42804" txt_i="30218" txt_f="30877">). Experimental conditions are as described in the Methods and Materials section. Box plots were generated using MINITAB version 14. 'AK,' 'IMIQ,' and 'Post' designate pretreatment AK, imiquimod-treated skin during treatment (maximum response from week 1, week 2 and week 4), and imiquimod-treated skin 4 weeks post treatment, respectively. Fold change was calculated with respect to sun-unexposed skin. Boxes indicate the median 95% confidence intervals, asterisks designate outliers, and the lines connect median values. P values are given for 2-way ANOVA, comparing the fold change values for pretreatment AK samples and imiquimod-treated samples (n = 13).</offsets></p></caption><graphic xlink:href="1479-5876-5-7-4"></graphic></fig></sec><sec><title><offsets xml_i="42890" xml_f="43014" txt_i="30879" txt_f="31003">Imiquimod induces a large number of type I interferon-inducible genes with growth inhibitory and immune-stimulatory activity</offsets></title><p><offsets xml_i="43025" xml_f="43224" txt_i="31004" txt_f="31203">Imidazoquinoline TLR7 agonists such as imiquimod and resiquimod are know to induce various cytokines, including interferon-α, IL6, MCP-1 and IL12 as well as the co-stimulatory markers CD80 and CD86 [</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="43256" xml_f="43258" txt_i="31203" txt_f="31205">15</offsets></xref><offsets xml_i="43265" xml_f="43266" txt_i="31205" txt_f="31206">,</offsets><xref ref-type="bibr" rid="B46"><offsets xml_i="43298" xml_f="43300" txt_i="31206" txt_f="31208">46</offsets></xref><offsets xml_i="43307" xml_f="43505" txt_i="31208" txt_f="31406">]. Type I interferons are known to be powerful regulators of the innate and adaptive immune system through the induction of various genes with antiviral, anti-tumor and immune regulatory functions [</offsets><xref ref-type="bibr" rid="B43"><offsets xml_i="43537" xml_f="43539" txt_i="31406" txt_f="31408">43</offsets></xref><offsets xml_i="43546" xml_f="43547" txt_i="31408" txt_f="31409">,</offsets><xref ref-type="bibr" rid="B44"><offsets xml_i="43579" xml_f="43581" txt_i="31409" txt_f="31411">44</offsets></xref><offsets xml_i="43588" xml_f="43589" txt_i="31411" txt_f="31412">,</offsets><xref ref-type="bibr" rid="B47"><offsets xml_i="43621" xml_f="43623" txt_i="31412" txt_f="31414">47</offsets></xref><offsets xml_i="43630" xml_f="43631" txt_i="31414" txt_f="31415">-</offsets><xref ref-type="bibr" rid="B51"><offsets xml_i="43663" xml_f="43665" txt_i="31415" txt_f="31417">51</offsets></xref><offsets xml_i="43672" xml_f="43887" txt_i="31417" txt_f="31632">]. In this study, we did not detect increased expression of interferon upon treatment with imiquimod, but observed the increased expression of a large number of IFN-inducible genes (114 genes), [see Additional file </offsets><xref ref-type="supplementary-material" rid="S3"><offsets xml_i="43936" xml_f="43937" txt_i="31632" txt_f="31633">3</offsets></xref><offsets xml_i="43944" xml_f="44396" txt_i="31633" txt_f="32085">]. The lack of detection of type 1 interferons after imiquimod treatment in this study may be due to the early induction and degradation of their respective mRNA. Biopsies were taken approximately 8 to 16 hr after application of the drug. We have observed that mRNA for type 1 interferons in human blood mononuclear cells treated with imiquimod peaks in 1 to 2 hours and declines to basal levels 6 to 8 hours post treatment (unpublished internal data).</offsets></p><p><offsets xml_i="44403" xml_f="44580" txt_i="32086" txt_f="32263">Analysis of gene ontology classification of the interferon-inducible genes increased by treatment with imiquimod identified 46 genes with immune response classification. Figure </offsets><xref ref-type="fig" rid="F5"><offsets xml_i="44610" xml_f="44611" txt_i="32263" txt_f="32264">5</offsets></xref><offsets xml_i="44618" xml_f="44757" txt_i="32264" txt_f="32403"> shows a 2-way hierarchical clustering of the log2 transformed fold changes of all of the 530 imiquimod-induced genes [see Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="44806" xml_f="44807" txt_i="32403" txt_f="32404">1</offsets></xref><offsets xml_i="44814" xml_f="44824" txt_i="32404" txt_f="32414">]. Figure </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="44854" xml_f="44855" txt_i="32414" txt_f="32415">6</offsets></xref><offsets xml_i="44862" xml_f="45001" txt_i="32415" txt_f="32554"> exhibits clustering of the 46 interferon-inducible genes which classify as immune response genes Two main clusters are apparent in Figure </offsets><xref ref-type="fig" rid="F5"><offsets xml_i="45031" xml_f="45032" txt_i="32554" txt_f="32555">5</offsets></xref><offsets xml_i="45039" xml_f="45537" txt_i="32555" txt_f="33053">. One cluster consists of 8 imiquimod-treated samples. The second and larger cluster consists of all of the pretreatment AK samples, the vehicle-treated samples (designated as placebo in the figure) and five of the imiquimod-treated samples for subjects 02, 05, 09, 12 and 17 (IMIQ-02, IMIQ-05, IMIQ-09, IMIQ-12 and IMIQ-17). The fact that the vehicle-treated samples cluster with pretreatment AK lesions indicates the lack of a significant vehicle-effect in the gene expression profile. In Figure </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="45567" xml_f="45568" txt_i="33053" txt_f="33054">6</offsets></xref><offsets xml_i="45575" xml_f="45791" txt_i="33054" txt_f="33270">, 10 of the imiquimod-treated samples appear in 1 cluster, with IMIQ-02 and IMIQ-05 now as part of the IMIQ-treated cluster. This cluster is characterized by high expression of the interferon-inducible genes such as </offsets><italic><offsets xml_i="45799" xml_f="45802" txt_i="33270" txt_f="33273">MX1</offsets></italic><offsets xml_i="45811" xml_f="45813" txt_i="33273" txt_f="33275">, </offsets><italic><offsets xml_i="45821" xml_f="45827" txt_i="33275" txt_f="33281">IFIT1 </offsets></italic><offsets xml_i="45836" xml_f="45840" txt_i="33281" txt_f="33285">and </offsets><italic><offsets xml_i="45848" xml_f="45853" txt_i="33285" txt_f="33290">IFIT3</offsets></italic><offsets xml_i="45862" xml_f="46623" txt_i="33290" txt_f="34051">. The cluster also contains the samples placebo-03 and AK-11, indicating that these AK lesions were already manifesting spontaneous immune response. The imiquimod-treated samples 09, 12, and 17 are clustered with the pretreatment AK samples in both figures, indicating that Subjects 09, 12, and 17 had little gene expression response to imiquimod. Thus, the 46 interferon-inducible immune response genes segregate the imiquimod-treated samples from pretreatment AK and placebo-treated samples better than the whole set of 530 genes. This group of genes may therefore be good predictors of gene expression response to imiquimod. Collectively, these genes have been reported as 'interferon-signature' genes induced by several IFNα subtypes and IFNβ in monocytes [</offsets><xref ref-type="bibr" rid="B51"><offsets xml_i="46655" xml_f="46657" txt_i="34051" txt_f="34053">51</offsets></xref><offsets xml_i="46664" xml_f="46666" txt_i="34053" txt_f="34055">].</offsets></p><fig position="float" id="F5"><label><offsets xml_i="46707" xml_f="46715" txt_i="34056" txt_f="34064">Figure 5</offsets></label><caption><p><offsets xml_i="46735" xml_f="47784" txt_i="34064" txt_f="35113">Cluster analysis of genes regulated by imiquimod treatment. Two-way hierarchical clustering was performed as described in the Methods and Materials section. 'AK,' 'IMIQ,' and 'Placebo' designate the fold change with respect to sun-unexposed, nonlesional skin for pretreatment AK samples; for samples during imiquimod treatment (maximum response from week 1, week 2 and week 4), and samples for vehicle-treatment (maximum response from week 1, week 2 and week 4) respectively. Numbers designate subjects. Hierarchical clustering was performed using the Unweighted Pair-Group Method with Arithmetic mean (UPGMA) and the Euclidean similarity measure. Red, white, and green indicate up-regulated, unchanged, and down-regulated genes, respectively. The color bar insert shows the corresponding expression levels. The cluster consists of 530 imiquimod-responsive genes whose expression was statistically different when comparing the AK group of samples to the IMIQ group of samples. Expression changes for the 530 genes are documented in [Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="47833" xml_f="47834" txt_i="35113" txt_f="35114">1</offsets></xref><offsets xml_i="47841" xml_f="47843" txt_i="35114" txt_f="35116">].</offsets></p></caption><graphic xlink:href="1479-5876-5-7-5"></graphic></fig><fig position="float" id="F6"><label><offsets xml_i="47948" xml_f="47956" txt_i="35117" txt_f="35125">Figure 6</offsets></label><caption><p><offsets xml_i="47976" xml_f="48206" txt_i="35125" txt_f="35355">Cluster of 46 immune response genes that are also known to be inducible upon treatment with type 1 interferons as reported in [18, 43, 45, 48, 51, 102] Cluster analysis was performed as described in Figure 5. [See Additional file </offsets><xref ref-type="supplementary-material" rid="S3"><offsets xml_i="48255" xml_f="48256" txt_i="35355" txt_f="35356">3</offsets></xref><offsets xml_i="48263" xml_f="48265" txt_i="35356" txt_f="35358">].</offsets></p></caption><graphic xlink:href="1479-5876-5-7-6"></graphic></fig><p><offsets xml_i="48336" xml_f="48488" txt_i="35359" txt_f="35511">Heterogeneity of pretreatment AK lesions was not documented by clinical assessment in this study. However, the gene expression profiles shown in Figure </offsets><xref ref-type="fig" rid="F5"><offsets xml_i="48518" xml_f="48519" txt_i="35511" txt_f="35512">5</offsets></xref><offsets xml_i="48526" xml_f="48538" txt_i="35512" txt_f="35524"> and Figure </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="48568" xml_f="48569" txt_i="35524" txt_f="35525">6</offsets></xref><offsets xml_i="48576" xml_f="48658" txt_i="35525" txt_f="35607"> indicate heterogeneity in lesions. In addition to AK 11 and Placebo 03 in Figure </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="48688" xml_f="48689" txt_i="35607" txt_f="35608">6</offsets></xref><offsets xml_i="48696" xml_f="49080" txt_i="35608" txt_f="35992"> which cluster with the imiquimod treatment group indicating some level of immune response in these samples, other AK lesions also show low levels of expression of interferon inducible genes. For example, AK 06, AK 03, and AK 07 show low levels of expression of several interferon-inducible genes whereas AK 16, AK 17, AK 12 and AK 15 show normal to slightly depressed levels (Figure </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="49110" xml_f="49111" txt_i="35992" txt_f="35993">6</offsets></xref><offsets xml_i="49118" xml_f="49120" txt_i="35993" txt_f="35995">).</offsets></p><p><offsets xml_i="49127" xml_f="49351" txt_i="35996" txt_f="36220">Interferon-inducible genes regulate diverse cellular processes, such as cell growth and differentiation, cell death, and T-cell co stimulation, activation, and migration. These genes have been reported to possess antiviral [</offsets><xref ref-type="bibr" rid="B52"><offsets xml_i="49383" xml_f="49385" txt_i="36220" txt_f="36222">52</offsets></xref><offsets xml_i="49392" xml_f="49393" txt_i="36222" txt_f="36223">-</offsets><xref ref-type="bibr" rid="B54"><offsets xml_i="49425" xml_f="49427" txt_i="36223" txt_f="36225">54</offsets></xref><offsets xml_i="49434" xml_f="49452" txt_i="36225" txt_f="36243">], pro-apoptotic [</offsets><xref ref-type="bibr" rid="B55"><offsets xml_i="49484" xml_f="49486" txt_i="36243" txt_f="36245">55</offsets></xref><offsets xml_i="49493" xml_f="49494" txt_i="36245" txt_f="36246">-</offsets><xref ref-type="bibr" rid="B57"><offsets xml_i="49526" xml_f="49528" txt_i="36246" txt_f="36248">57</offsets></xref><offsets xml_i="49535" xml_f="49573" txt_i="36248" txt_f="36286">], and anti-proliferative activities [</offsets><xref ref-type="bibr" rid="B58"><offsets xml_i="49605" xml_f="49607" txt_i="36286" txt_f="36288">58</offsets></xref><offsets xml_i="49614" xml_f="49615" txt_i="36288" txt_f="36289">,</offsets><xref ref-type="bibr" rid="B59"><offsets xml_i="49647" xml_f="49649" txt_i="36289" txt_f="36291">59</offsets></xref><offsets xml_i="49656" xml_f="49880" txt_i="36291" txt_f="36515">]. The interferon-inducible genes which increased following imiquimod treatment include those known to be induced by viruses as well as those with known anti-viral activity. These include the 2'5'-oligoadenylate synthetases </offsets><italic><offsets xml_i="49888" xml_f="49892" txt_i="36515" txt_f="36519">OAS1</offsets></italic><offsets xml_i="49901" xml_f="49903" txt_i="36519" txt_f="36521">, </offsets><italic><offsets xml_i="49911" xml_f="49915" txt_i="36521" txt_f="36525">OAS2</offsets></italic><offsets xml_i="49924" xml_f="49926" txt_i="36525" txt_f="36527">, </offsets><italic><offsets xml_i="49934" xml_f="49939" txt_i="36527" txt_f="36532">OAS3 </offsets></italic><offsets xml_i="49948" xml_f="49952" txt_i="36532" txt_f="36536">and </offsets><italic><offsets xml_i="49960" xml_f="49964" txt_i="36536" txt_f="36540">OASL</offsets></italic><offsets xml_i="49973" xml_f="50061" txt_i="36540" txt_f="36628">, the genes encoding the interferon-inducible proteins with tetratricopepetide repeats, </offsets><italic><offsets xml_i="50069" xml_f="50074" txt_i="36628" txt_f="36633">IFIT1</offsets></italic><offsets xml_i="50083" xml_f="50085" txt_i="36633" txt_f="36635">, </offsets><italic><offsets xml_i="50093" xml_f="50098" txt_i="36635" txt_f="36640">IFIT2</offsets></italic><offsets xml_i="50107" xml_f="50113" txt_i="36640" txt_f="36646">, and </offsets><italic><offsets xml_i="50121" xml_f="50126" txt_i="36646" txt_f="36651">IFIT3</offsets></italic><offsets xml_i="50135" xml_f="50137" txt_i="36651" txt_f="36653">, </offsets><italic><offsets xml_i="50145" xml_f="50150" txt_i="36653" txt_f="36658">IFTM1</offsets></italic><offsets xml_i="50159" xml_f="50206" txt_i="36658" txt_f="36705">, and other interferon inducible genes such as </offsets><italic><offsets xml_i="50214" xml_f="50219" txt_i="36705" txt_f="36710">IFI35</offsets></italic><offsets xml_i="50228" xml_f="50230" txt_i="36710" txt_f="36712">, </offsets><italic><offsets xml_i="50238" xml_f="50243" txt_i="36712" txt_f="36717">IFI16</offsets></italic><offsets xml_i="50252" xml_f="50254" txt_i="36717" txt_f="36719">, </offsets><italic><offsets xml_i="50262" xml_f="50265" txt_i="36719" txt_f="36722">MX1</offsets></italic><offsets xml_i="50274" xml_f="50276" txt_i="36722" txt_f="36724">, </offsets><italic><offsets xml_i="50284" xml_f="50287" txt_i="36724" txt_f="36727">MX2</offsets></italic><offsets xml_i="50296" xml_f="50298" txt_i="36727" txt_f="36729">, </offsets><italic><offsets xml_i="50306" xml_f="50314" txt_i="36729" txt_f="36737">EIF2AK2 </offsets></italic><offsets xml_i="50323" xml_f="50324" txt_i="36737" txt_f="36738">(</offsets><italic><offsets xml_i="50332" xml_f="50336" txt_i="36738" txt_f="36742">PRKR</offsets></italic><offsets xml_i="50345" xml_f="50348" txt_i="36742" txt_f="36745">), </offsets><italic><offsets xml_i="50356" xml_f="50361" txt_i="36745" txt_f="36750">G1P2 </offsets></italic><offsets xml_i="50370" xml_f="50371" txt_i="36750" txt_f="36751">(</offsets><italic><offsets xml_i="50379" xml_f="50384" txt_i="36751" txt_f="36756">ISG15</offsets></italic><offsets xml_i="50393" xml_f="50396" txt_i="36756" txt_f="36759">), </offsets><italic><offsets xml_i="50404" xml_f="50408" txt_i="36759" txt_f="36763">G1P3</offsets></italic><offsets xml_i="50417" xml_f="50419" txt_i="36763" txt_f="36765">, </offsets><italic><offsets xml_i="50427" xml_f="50432" txt_i="36765" txt_f="36770">ISG20</offsets></italic><offsets xml_i="50441" xml_f="50443" txt_i="36770" txt_f="36772">, </offsets><italic><offsets xml_i="50451" xml_f="50457" txt_i="36772" txt_f="36778">RSAD2 </offsets></italic><offsets xml_i="50466" xml_f="50467" txt_i="36778" txt_f="36779">(</offsets><italic><offsets xml_i="50475" xml_f="50479" txt_i="36779" txt_f="36783">Cig5</offsets></italic><offsets xml_i="50488" xml_f="50491" txt_i="36783" txt_f="36786">), </offsets><italic><offsets xml_i="50499" xml_f="50509" txt_i="36786" txt_f="36796">CCL8 (MCP2</offsets></italic><offsets xml_i="50518" xml_f="50521" txt_i="36796" txt_f="36799">), </offsets><italic><offsets xml_i="50529" xml_f="50543" txt_i="36799" txt_f="36813">CXCL10 (IP10) </offsets></italic><offsets xml_i="50552" xml_f="50556" txt_i="36813" txt_f="36817">and </offsets><italic><offsets xml_i="50564" xml_f="50571" txt_i="36817" txt_f="36824">CXCL11 </offsets></italic><offsets xml_i="50580" xml_f="50581" txt_i="36824" txt_f="36825">(</offsets><italic><offsets xml_i="50589" xml_f="50593" txt_i="36825" txt_f="36829">ITAC</offsets></italic><offsets xml_i="50602" xml_f="50605" txt_i="36829" txt_f="36832">) [</offsets><xref ref-type="bibr" rid="B43"><offsets xml_i="50637" xml_f="50639" txt_i="36832" txt_f="36834">43</offsets></xref><offsets xml_i="50646" xml_f="50647" txt_i="36834" txt_f="36835">,</offsets><xref ref-type="bibr" rid="B44"><offsets xml_i="50679" xml_f="50681" txt_i="36835" txt_f="36837">44</offsets></xref><offsets xml_i="50688" xml_f="50689" txt_i="36837" txt_f="36838">,</offsets><xref ref-type="bibr" rid="B47"><offsets xml_i="50721" xml_f="50723" txt_i="36838" txt_f="36840">47</offsets></xref><offsets xml_i="50730" xml_f="50731" txt_i="36840" txt_f="36841">,</offsets><xref ref-type="bibr" rid="B49"><offsets xml_i="50763" xml_f="50765" txt_i="36841" txt_f="36843">49</offsets></xref><offsets xml_i="50772" xml_f="50773" txt_i="36843" txt_f="36844">-</offsets><xref ref-type="bibr" rid="B52"><offsets xml_i="50805" xml_f="50807" txt_i="36844" txt_f="36846">52</offsets></xref><offsets xml_i="50814" xml_f="50815" txt_i="36846" txt_f="36847">,</offsets><xref ref-type="bibr" rid="B54"><offsets xml_i="50847" xml_f="50849" txt_i="36847" txt_f="36849">54</offsets></xref><offsets xml_i="50856" xml_f="50857" txt_i="36849" txt_f="36850">,</offsets><xref ref-type="bibr" rid="B60"><offsets xml_i="50889" xml_f="50891" txt_i="36850" txt_f="36852">60</offsets></xref><offsets xml_i="50898" xml_f="50899" txt_i="36852" txt_f="36853">,</offsets><xref ref-type="bibr" rid="B61"><offsets xml_i="50931" xml_f="50933" txt_i="36853" txt_f="36855">61</offsets></xref><offsets xml_i="50940" xml_f="51081" txt_i="36855" txt_f="36996">]. Several of the interferon-inducible genes were also increased at statistically significant levels 4 weeks post treatment. These included: </offsets><italic><offsets xml_i="51089" xml_f="51093" txt_i="36996" txt_f="37000">IRF7</offsets></italic><offsets xml_i="51102" xml_f="51104" txt_i="37000" txt_f="37002">, </offsets><italic><offsets xml_i="51112" xml_f="51117" txt_i="37002" txt_f="37007">IFI44</offsets></italic><offsets xml_i="51126" xml_f="51128" txt_i="37007" txt_f="37009">, </offsets><italic><offsets xml_i="51136" xml_f="51194" txt_i="37009" txt_f="37067">IFIT2, IFIT3, IFITM1, IFI35, RSAD2 (Cig5), G1P2, MX1, OAS1</offsets></italic><offsets xml_i="51203" xml_f="51209" txt_i="37067" txt_f="37073">, and </offsets><italic><offsets xml_i="51217" xml_f="51222" txt_i="37073" txt_f="37078">OAS2 </offsets></italic><offsets xml_i="51231" xml_f="51252" txt_i="37078" txt_f="37099">[see Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="51301" xml_f="51302" txt_i="37099" txt_f="37100">1</offsets></xref><offsets xml_i="51309" xml_f="51311" txt_i="37100" txt_f="37102">].</offsets></p><p><offsets xml_i="51318" xml_f="51487" txt_i="37103" txt_f="37272">In addition to interferon-inducible genes with antiviral activity, several genes known or predicted to possess growth-inhibition and/or cell-differentiating activities [</offsets><xref ref-type="bibr" rid="B49"><offsets xml_i="51519" xml_f="51521" txt_i="37272" txt_f="37274">49</offsets></xref><offsets xml_i="51528" xml_f="51529" txt_i="37274" txt_f="37275">,</offsets><xref ref-type="bibr" rid="B50"><offsets xml_i="51561" xml_f="51563" txt_i="37275" txt_f="37277">50</offsets></xref><offsets xml_i="51570" xml_f="51609" txt_i="37277" txt_f="37316">] were induced by imiquimod, including </offsets><italic><offsets xml_i="51617" xml_f="51622" txt_i="37316" txt_f="37321">IFI16</offsets></italic><offsets xml_i="51631" xml_f="51633" txt_i="37321" txt_f="37323">, </offsets><italic><offsets xml_i="51641" xml_f="51646" txt_i="37323" txt_f="37328">AIM2 </offsets></italic><offsets xml_i="51655" xml_f="51656" txt_i="37328" txt_f="37329">[</offsets><xref ref-type="bibr" rid="B62"><offsets xml_i="51688" xml_f="51690" txt_i="37329" txt_f="37331">62</offsets></xref><offsets xml_i="51697" xml_f="51698" txt_i="37331" txt_f="37332">,</offsets><xref ref-type="bibr" rid="B63"><offsets xml_i="51730" xml_f="51732" txt_i="37332" txt_f="37334">63</offsets></xref><offsets xml_i="51739" xml_f="51742" txt_i="37334" txt_f="37337">], </offsets><italic><offsets xml_i="51750" xml_f="51756" txt_i="37337" txt_f="37343">IFIH1 </offsets></italic><offsets xml_i="51765" xml_f="51800" txt_i="37343" txt_f="37378">(melanoma differentiation antigen, </offsets><italic><offsets xml_i="51808" xml_f="51812" txt_i="37378" txt_f="37382">MDA5</offsets></italic><offsets xml_i="51821" xml_f="51824" txt_i="37382" txt_f="37385">) [</offsets><xref ref-type="bibr" rid="B45"><offsets xml_i="51856" xml_f="51858" txt_i="37385" txt_f="37387">45</offsets></xref><offsets xml_i="51865" xml_f="51868" txt_i="37387" txt_f="37390">], </offsets><italic><offsets xml_i="51876" xml_f="51883" txt_i="37390" txt_f="37397">CXCL10 </offsets></italic><offsets xml_i="51892" xml_f="51893" txt_i="37397" txt_f="37398">(</offsets><italic><offsets xml_i="51901" xml_f="51905" txt_i="37398" txt_f="37402">IP10</offsets></italic><offsets xml_i="51914" xml_f="51917" txt_i="37402" txt_f="37405">) [</offsets><xref ref-type="bibr" rid="B58"><offsets xml_i="51949" xml_f="51951" txt_i="37405" txt_f="37407">58</offsets></xref><offsets xml_i="51958" xml_f="51964" txt_i="37407" txt_f="37413">] and </offsets><italic><offsets xml_i="51972" xml_f="51985" txt_i="37413" txt_f="37426">EIF2AK2 (PRKR</offsets></italic><offsets xml_i="51994" xml_f="51997" txt_i="37426" txt_f="37429">) [</offsets><xref ref-type="bibr" rid="B64"><offsets xml_i="52029" xml_f="52031" txt_i="37429" txt_f="37431">64</offsets></xref><offsets xml_i="52038" xml_f="52127" txt_i="37431" txt_f="37520">]. Some of the interferon-inducible genes also possess pro-apoptotic activity, including </offsets><italic><offsets xml_i="52135" xml_f="52138" txt_i="37520" txt_f="37523">MX1</offsets></italic><offsets xml_i="52147" xml_f="52149" txt_i="37523" txt_f="37525">, </offsets><italic><offsets xml_i="52157" xml_f="52165" txt_i="37525" txt_f="37533">TNFSF10 </offsets></italic><offsets xml_i="52174" xml_f="52175" txt_i="37533" txt_f="37534">(</offsets><italic><offsets xml_i="52183" xml_f="52188" txt_i="37534" txt_f="37539">TRAIL</offsets></italic><offsets xml_i="52197" xml_f="52200" txt_i="37539" txt_f="37542">), </offsets><italic><offsets xml_i="52208" xml_f="52212" txt_i="37542" txt_f="37546">OAS1</offsets></italic><offsets xml_i="52221" xml_f="52227" txt_i="37546" txt_f="37552">, and </offsets><italic><offsets xml_i="52235" xml_f="52240" txt_i="37552" txt_f="37557">PRF1 </offsets></italic><offsets xml_i="52249" xml_f="52250" txt_i="37557" txt_f="37558">[</offsets><xref ref-type="bibr" rid="B55"><offsets xml_i="52282" xml_f="52284" txt_i="37558" txt_f="37560">55</offsets></xref><offsets xml_i="52291" xml_f="52292" txt_i="37560" txt_f="37561">,</offsets><xref ref-type="bibr" rid="B57"><offsets xml_i="52324" xml_f="52326" txt_i="37561" txt_f="37563">57</offsets></xref><offsets xml_i="52333" xml_f="52334" txt_i="37563" txt_f="37564">,</offsets><xref ref-type="bibr" rid="B65"><offsets xml_i="52366" xml_f="52368" txt_i="37564" txt_f="37566">65</offsets></xref><offsets xml_i="52375" xml_f="52385" txt_i="37566" txt_f="37576">]. Figure </offsets><xref ref-type="fig" rid="F7"><offsets xml_i="52415" xml_f="52417" txt_i="37576" txt_f="37578">7a</offsets></xref><offsets xml_i="52424" xml_f="52436" txt_i="37578" txt_f="37590"> and Figure </offsets><xref ref-type="fig" rid="F7"><offsets xml_i="52466" xml_f="52468" txt_i="37590" txt_f="37592">7b</offsets></xref><offsets xml_i="52475" xml_f="52534" txt_i="37592" txt_f="37651"> show changes in the expression of the pro-apoptotic genes </offsets><italic><offsets xml_i="52542" xml_f="52550" txt_i="37651" txt_f="37659">TNFSF10 </offsets></italic><offsets xml_i="52559" xml_f="52560" txt_i="37659" txt_f="37660">(</offsets><italic><offsets xml_i="52568" xml_f="52573" txt_i="37660" txt_f="37665">TRAIL</offsets></italic><offsets xml_i="52582" xml_f="52588" txt_i="37665" txt_f="37671">) and </offsets><italic><offsets xml_i="52596" xml_f="52600" txt_i="37671" txt_f="37675">MX1 </offsets></italic><offsets xml_i="52609" xml_f="52696" txt_i="37675" txt_f="37762">with imiquimod treatment as determined from the Affymetrix analysis. The expression of </offsets><italic><offsets xml_i="52704" xml_f="52708" txt_i="37762" txt_f="37766">MX1 </offsets></italic><offsets xml_i="52717" xml_f="52820" txt_i="37766" txt_f="37869">remained elevated at statistically significant level 4 weeks post treatment, whereas the expression of </offsets><italic><offsets xml_i="52828" xml_f="52836" txt_i="37869" txt_f="37877">TNFSF10 </offsets></italic><offsets xml_i="52845" xml_f="53027" txt_i="37877" txt_f="38059">returned to basal levels. The data are consistent with the observation of increased expression of IFNα-inducible genes in imiquimod-treated BCC and cutaneous T-cell lymphoma (CTCL) [</offsets><xref ref-type="bibr" rid="B66"><offsets xml_i="53059" xml_f="53061" txt_i="38059" txt_f="38061">66</offsets></xref><offsets xml_i="53068" xml_f="53069" txt_i="38061" txt_f="38062">,</offsets><xref ref-type="bibr" rid="B67"><offsets xml_i="53101" xml_f="53103" txt_i="38062" txt_f="38064">67</offsets></xref><offsets xml_i="53110" xml_f="53181" txt_i="38064" txt_f="38135">]. Thus, interferon-inducible genes with pro-apoptotic activity (e.g., </offsets><italic><offsets xml_i="53189" xml_f="53201" txt_i="38135" txt_f="38147">MX1, TNFSF10</offsets></italic><offsets xml_i="53210" xml_f="53250" txt_i="38147" txt_f="38187">) and growth inhibitory activity (e.g., </offsets><italic><offsets xml_i="53258" xml_f="53276" txt_i="38187" txt_f="38205">IFIH1, AIM2, IFI16</offsets></italic><offsets xml_i="53285" xml_f="53394" txt_i="38205" txt_f="38314">) may result in growth inhibition of neoplastic cells whereas those with immune-stimulatory activity such as </offsets><italic><offsets xml_i="53402" xml_f="53424" txt_i="38314" txt_f="38336">CXCL10 (IP10), CXCL11 </offsets></italic><offsets xml_i="53433" xml_f="53434" txt_i="38336" txt_f="38337">(</offsets><italic><offsets xml_i="53442" xml_f="53446" txt_i="38337" txt_f="38341">ITAC</offsets></italic><offsets xml_i="53455" xml_f="53462" txt_i="38341" txt_f="38348">), and </offsets><italic><offsets xml_i="53470" xml_f="53475" txt_i="38348" txt_f="38353">CCL8 </offsets></italic><offsets xml_i="53484" xml_f="53485" txt_i="38353" txt_f="38354">(</offsets><italic><offsets xml_i="53493" xml_f="53497" txt_i="38354" txt_f="38358">MCP2</offsets></italic><offsets xml_i="53506" xml_f="53742" txt_i="38358" txt_f="38594">) may facilitate cell-mediated lesion destruction by recruiting immune cells into the lesions. Indeed, further evidence for the recruitment of immune cells into AK lesions with imiquimod treatment is presented in the following sections.</offsets></p><fig position="float" id="F7"><label><offsets xml_i="53783" xml_f="53791" txt_i="38595" txt_f="38603">Figure 7</offsets></label><caption><p><offsets xml_i="53811" xml_f="53950" txt_i="38603" txt_f="38742">Imiquimod treatment is associated with an increase in expression of pro-apoptotic genes as determined by Affymetrix GeneChip analysis. (A) </offsets><italic><offsets xml_i="53958" xml_f="53966" txt_i="38742" txt_f="38750">TNFSF10 </offsets></italic><offsets xml_i="53975" xml_f="53978" txt_i="38750" txt_f="38753">(B)</offsets><italic><offsets xml_i="53986" xml_f="53989" txt_i="38753" txt_f="38756">MX1</offsets></italic><offsets xml_i="53998" xml_f="54067" txt_i="38756" txt_f="38825">. Experimental conditions and analysis are as described for Figure 4.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-7"></graphic></fig></sec><sec><title><offsets xml_i="54153" xml_f="54257" txt_i="38827" txt_f="38931">Imiquimod induces the expression of chemokines responsible for recruitment of immune cells to AK lesions</offsets></title><p><offsets xml_i="54268" xml_f="54304" txt_i="38932" txt_f="38968">Gene ontology classification (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="54336" xml_f="54337" txt_i="38968" txt_f="38969">1</offsets></xref><offsets xml_i="54344" xml_f="54507" txt_i="38969" txt_f="39132">) shows that 106 genes classified as immune responsive were induced by imiquimod, some of which are also shown to be interferon α/β-inducible [see Additional File </offsets><xref ref-type="supplementary-material" rid="S3"><offsets xml_i="54556" xml_f="54557" txt_i="39132" txt_f="39133">3</offsets></xref><offsets xml_i="54564" xml_f="54693" txt_i="39133" txt_f="39262">]. Out of the 106 immune response genes, several were classified as cytokines/chemokines and chemokine receptors. In addition to </offsets><italic><offsets xml_i="54701" xml_f="54707" txt_i="39262" txt_f="39268">CXCL10</offsets></italic><offsets xml_i="54716" xml_f="54718" txt_i="39268" txt_f="39270">, </offsets><italic><offsets xml_i="54726" xml_f="54733" txt_i="39270" txt_f="39277">CXCL11 </offsets></italic><offsets xml_i="54742" xml_f="54746" txt_i="39277" txt_f="39281">and </offsets><italic><offsets xml_i="54754" xml_f="54759" txt_i="39281" txt_f="39286">CCL8 </offsets></italic><offsets xml_i="54768" xml_f="54849" txt_i="39286" txt_f="39367">mentioned above, other chemokines were induced by imiquimod treatment, including </offsets><italic><offsets xml_i="54857" xml_f="54862" txt_i="39367" txt_f="39372">CCL3 </offsets></italic><offsets xml_i="54871" xml_f="54872" txt_i="39372" txt_f="39373">(</offsets><italic><offsets xml_i="54880" xml_f="54885" txt_i="39373" txt_f="39378">MIP1a</offsets></italic><offsets xml_i="54894" xml_f="54897" txt_i="39378" txt_f="39381">), </offsets><italic><offsets xml_i="54905" xml_f="54910" txt_i="39381" txt_f="39386">CCL4 </offsets></italic><offsets xml_i="54919" xml_f="54920" txt_i="39386" txt_f="39387">(</offsets><italic><offsets xml_i="54928" xml_f="54933" txt_i="39387" txt_f="39392">MIP1b</offsets></italic><offsets xml_i="54942" xml_f="54945" txt_i="39392" txt_f="39395">), </offsets><italic><offsets xml_i="54953" xml_f="54958" txt_i="39395" txt_f="39400">CCL5 </offsets></italic><offsets xml_i="54967" xml_f="54977" txt_i="39400" txt_f="39410">(Rantes), </offsets><italic><offsets xml_i="54985" xml_f="54992" txt_i="39410" txt_f="39417">CXCL12 </offsets></italic><offsets xml_i="55001" xml_f="55002" txt_i="39417" txt_f="39418">(</offsets><italic><offsets xml_i="55010" xml_f="55014" txt_i="39418" txt_f="39422">SDF1</offsets></italic><offsets xml_i="55023" xml_f="55030" txt_i="39422" txt_f="39429">), and </offsets><italic><offsets xml_i="55038" xml_f="55044" txt_i="39429" txt_f="39435">CXCL16</offsets></italic><offsets xml_i="55053" xml_f="55083" txt_i="39435" txt_f="39465">, and the chemokine receptors </offsets><italic><offsets xml_i="55091" xml_f="55095" txt_i="39465" txt_f="39469">CCR1</offsets></italic><offsets xml_i="55104" xml_f="55106" txt_i="39469" txt_f="39471">, </offsets><italic><offsets xml_i="55114" xml_f="55118" txt_i="39471" txt_f="39475">CCR5</offsets></italic><offsets xml_i="55127" xml_f="55133" txt_i="39475" txt_f="39481">, and </offsets><italic><offsets xml_i="55141" xml_f="55146" txt_i="39481" txt_f="39486">CXCR4</offsets></italic><offsets xml_i="55155" xml_f="55164" txt_i="39486" txt_f="39495">. Figure </offsets><xref ref-type="fig" rid="F8"><offsets xml_i="55194" xml_f="55196" txt_i="39495" txt_f="39497">8a</offsets></xref><offsets xml_i="55203" xml_f="55208" txt_i="39497" txt_f="39502"> and </offsets><xref ref-type="fig" rid="F8"><offsets xml_i="55238" xml_f="55240" txt_i="39502" txt_f="39504">8b</offsets></xref><offsets xml_i="55247" xml_f="55284" txt_i="39504" txt_f="39541"> illustrate changes in expression of </offsets><italic><offsets xml_i="55292" xml_f="55299" txt_i="39541" txt_f="39548">CXCL10 </offsets></italic><offsets xml_i="55308" xml_f="55312" txt_i="39548" txt_f="39552">and </offsets><italic><offsets xml_i="55320" xml_f="55327" txt_i="39552" txt_f="39559">CXCL11 </offsets></italic><offsets xml_i="55336" xml_f="55532" txt_i="39559" txt_f="39755">observed upon treatment of AK lesions with imiquimod. The magnitude of the changes observed in the expression of chemokine genes during treatment ranged from a median fold change value of 1.8 for </offsets><italic><offsets xml_i="55540" xml_f="55547" txt_i="39755" txt_f="39762">CXCL12 </offsets></italic><offsets xml_i="55556" xml_f="55557" txt_i="39762" txt_f="39763">(</offsets><italic><offsets xml_i="55565" xml_f="55569" txt_i="39763" txt_f="39767">SDF1</offsets></italic><offsets xml_i="55578" xml_f="55592" txt_i="39767" txt_f="39781">) to 40.8 for </offsets><italic><offsets xml_i="55600" xml_f="55606" txt_i="39781" txt_f="39787">CXCL11</offsets></italic><offsets xml_i="55615" xml_f="55869" txt_i="39787" txt_f="40041">. The expression of both genes returned to pretreatment levels 4 weeks post treatment. These data are consistent with previous reports of increased expression of chemokines genes and their receptors upon topical treatment of BCC and CTCL with imiquimod [</offsets><xref ref-type="bibr" rid="B67"><offsets xml_i="55901" xml_f="55903" txt_i="40041" txt_f="40043">67</offsets></xref><offsets xml_i="55910" xml_f="55924" txt_i="40043" txt_f="40057">], as well as </offsets><italic><offsets xml_i="55932" xml_f="55941" txt_i="40057" txt_f="40066">in vitro </offsets></italic><offsets xml_i="55950" xml_f="56097" txt_i="40066" txt_f="40213">studies of human blood mononuclear cells stimulated with other imidazoquinoline TLR7 agonists showing the induction of CXCL10 and CXCL11 proteins [</offsets><xref ref-type="bibr" rid="B46"><offsets xml_i="56129" xml_f="56131" txt_i="40213" txt_f="40215">46</offsets></xref><offsets xml_i="56138" xml_f="56676" txt_i="40215" txt_f="40753">]. The cocktail of chemokines up-regulated by imiquimod is consistent with recruitment and/or activation of macrophages, DCs, plasmacytoid DCs, gamma/delta T cells, cytotoxic T cells, and natural killer (NK) cells, and is also consistent with the cell surface markers indicating the presence of these cells upon imiquimod treatment. Indeed, topical treatment of various neoplastic skin conditions with imiquimod cream have shown inflammatory conditions at the site of treatment, indicating the infiltration of immune cells into the site [</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="56708" xml_f="56710" txt_i="40753" txt_f="40755">17</offsets></xref><offsets xml_i="56717" xml_f="56718" txt_i="40755" txt_f="40756">,</offsets><xref ref-type="bibr" rid="B19"><offsets xml_i="56750" xml_f="56752" txt_i="40756" txt_f="40758">19</offsets></xref><offsets xml_i="56759" xml_f="56760" txt_i="40758" txt_f="40759">,</offsets><xref ref-type="bibr" rid="B20"><offsets xml_i="56792" xml_f="56794" txt_i="40759" txt_f="40761">20</offsets></xref><offsets xml_i="56801" xml_f="56802" txt_i="40761" txt_f="40762">,</offsets><xref ref-type="bibr" rid="B68"><offsets xml_i="56834" xml_f="56836" txt_i="40762" txt_f="40764">68</offsets></xref><offsets xml_i="56843" xml_f="56844" txt_i="40764" txt_f="40765">,</offsets><xref ref-type="bibr" rid="B69"><offsets xml_i="56876" xml_f="56878" txt_i="40765" txt_f="40767">69</offsets></xref><offsets xml_i="56885" xml_f="57014" txt_i="40767" txt_f="40896">]. In this study, the gene expression fingerprints that indicate recruitment of various immune cells are further discussed below.</offsets></p><fig position="float" id="F8"><label><offsets xml_i="57055" xml_f="57063" txt_i="40897" txt_f="40905">Figure 8</offsets></label><caption><p><offsets xml_i="57083" xml_f="57201" txt_i="40905" txt_f="41023">Increase in expression of chemokines after treatment with imiquimod as determined by Affymetrix GeneChip analysis. (A)</offsets><italic><offsets xml_i="57209" xml_f="57216" txt_i="41023" txt_f="41030">CXCL10 </offsets></italic><offsets xml_i="57225" xml_f="57226" txt_i="41030" txt_f="41031">(</offsets><italic><offsets xml_i="57234" xml_f="57238" txt_i="41031" txt_f="41035">IP10</offsets></italic><offsets xml_i="57247" xml_f="57253" txt_i="41035" txt_f="41041">), (B)</offsets><italic><offsets xml_i="57261" xml_f="57268" txt_i="41041" txt_f="41048">CXCL11 </offsets></italic><offsets xml_i="57277" xml_f="57278" txt_i="41048" txt_f="41049">(</offsets><italic><offsets xml_i="57286" xml_f="57290" txt_i="41049" txt_f="41053">ITAC</offsets></italic><offsets xml_i="57299" xml_f="57369" txt_i="41053" txt_f="41123">). Experimental conditions and analysis are as described for Figure 4.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-8"></graphic></fig></sec><sec><title><offsets xml_i="57455" xml_f="57556" txt_i="41125" txt_f="41226">Imiquimod increases the expression of genes predictive of infiltrating macrophage and dendritic cells</offsets></title><p><offsets xml_i="57567" xml_f="57732" txt_i="41227" txt_f="41392">Of the imiquimod-induced genes classified as immune response in gene ontology, several have receptor activity and (or) are hematopoietic cell surface markers (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="57764" xml_f="57765" txt_i="41392" txt_f="41393">1</offsets></xref><offsets xml_i="57772" xml_f="58007" txt_i="41393" txt_f="41628">). The increased expression of these genes indicates the recruitment of various immune cell types to the lesion sites. Macrophage and/or monocyte infiltration of AK lesions upon treatment with imiquimod was indicated by an increase in </offsets><italic><offsets xml_i="58015" xml_f="58019" txt_i="41628" txt_f="41632">CD14</offsets></italic><offsets xml_i="58028" xml_f="58030" txt_i="41632" txt_f="41634">, </offsets><italic><offsets xml_i="58038" xml_f="58043" txt_i="41634" txt_f="41639">CD163</offsets></italic><offsets xml_i="58052" xml_f="58058" txt_i="41639" txt_f="41645">, and </offsets><italic><offsets xml_i="58066" xml_f="58074" txt_i="41645" txt_f="41653">CLECSF9 </offsets></italic><offsets xml_i="58083" xml_f="58084" txt_i="41653" txt_f="41654">(</offsets><italic><offsets xml_i="58092" xml_f="58099" txt_i="41654" txt_f="41661">CLECSF4</offsets></italic><offsets xml_i="58108" xml_f="58110" txt_i="41661" txt_f="41663">, </offsets><italic><offsets xml_i="58118" xml_f="58124" txt_i="41663" txt_f="41669">MINCLE</offsets></italic><offsets xml_i="58133" xml_f="58184" txt_i="41669" txt_f="41720">), a C-type lectin found on activated macrophages [</offsets><xref ref-type="bibr" rid="B70"><offsets xml_i="58216" xml_f="58218" txt_i="41720" txt_f="41722">70</offsets></xref><offsets xml_i="58225" xml_f="58301" txt_i="41722" txt_f="41798">]. The increase in expression of genes of the classical complement pathway, </offsets><italic><offsets xml_i="58309" xml_f="58313" txt_i="41798" txt_f="41802">C1QA</offsets></italic><offsets xml_i="58322" xml_f="58324" txt_i="41802" txt_f="41804">, </offsets><italic><offsets xml_i="58332" xml_f="58336" txt_i="41804" txt_f="41808">C1QB</offsets></italic><offsets xml_i="58345" xml_f="58347" txt_i="41808" txt_f="41810">, </offsets><italic><offsets xml_i="58355" xml_f="58360" txt_i="41810" txt_f="41815">C3AR1</offsets></italic><offsets xml_i="58369" xml_f="58375" txt_i="41815" txt_f="41821">, and </offsets><italic><offsets xml_i="58383" xml_f="58387" txt_i="41821" txt_f="41825">C5R1</offsets></italic><offsets xml_i="58396" xml_f="58466" txt_i="41825" txt_f="41895">, also indicates an increase in and/or the activation of macrophages [</offsets><xref ref-type="bibr" rid="B71"><offsets xml_i="58498" xml_f="58500" txt_i="41895" txt_f="41897">71</offsets></xref><offsets xml_i="58507" xml_f="58659" txt_i="41897" txt_f="42049">]. The data are consistent with histologic observation of macrophage and/or monocyte infiltration after application ofimiquimod in the treatment of AK [</offsets><xref ref-type="bibr" rid="B72"><offsets xml_i="58691" xml_f="58693" txt_i="42049" txt_f="42051">72</offsets></xref><offsets xml_i="58700" xml_f="58726" txt_i="42051" txt_f="42077">] and in lentigo maligna [</offsets><xref ref-type="bibr" rid="B73"><offsets xml_i="58758" xml_f="58760" txt_i="42077" txt_f="42079">73</offsets></xref><offsets xml_i="58767" xml_f="58768" txt_i="42079" txt_f="42080">,</offsets><xref ref-type="bibr" rid="B74"><offsets xml_i="58800" xml_f="58802" txt_i="42080" txt_f="42082">74</offsets></xref><offsets xml_i="58809" xml_f="58811" txt_i="42082" txt_f="42084">].</offsets></p><p><offsets xml_i="58818" xml_f="58892" txt_i="42085" txt_f="42159">The presence of DCs is shown by increases in the co stimulatory molecules </offsets><italic><offsets xml_i="58900" xml_f="58905" txt_i="42159" txt_f="42164">CD86 </offsets></italic><offsets xml_i="58914" xml_f="58918" txt_i="42164" txt_f="42168">and </offsets><italic><offsets xml_i="58926" xml_f="58933" txt_i="42168" txt_f="42175">CLEC4A </offsets></italic><offsets xml_i="58942" xml_f="58943" txt_i="42175" txt_f="42176">(</offsets><italic><offsets xml_i="58951" xml_f="58955" txt_i="42176" txt_f="42180">DCIR</offsets></italic><offsets xml_i="58964" xml_f="58966" txt_i="42180" txt_f="42182">, </offsets><italic><offsets xml_i="58974" xml_f="58981" txt_i="42182" txt_f="42189">CLECSF6</offsets></italic><offsets xml_i="58990" xml_f="59045" txt_i="42189" txt_f="42244">), as well as by 3 leukocyte immunoglobulin receptors: </offsets><italic><offsets xml_i="59053" xml_f="59060" txt_i="42244" txt_f="42251">LILRB3 </offsets></italic><offsets xml_i="59069" xml_f="59070" txt_i="42251" txt_f="42252">(</offsets><italic><offsets xml_i="59078" xml_f="59082" txt_i="42252" txt_f="42256">ILT3</offsets></italic><offsets xml_i="59091" xml_f="59097" txt_i="42256" txt_f="42262">) and </offsets><italic><offsets xml_i="59105" xml_f="59112" txt_i="42262" txt_f="42269">LILRB1 </offsets></italic><offsets xml_i="59121" xml_f="59122" txt_i="42269" txt_f="42270">(</offsets><italic><offsets xml_i="59130" xml_f="59134" txt_i="42270" txt_f="42274">ILT2</offsets></italic><offsets xml_i="59143" xml_f="59203" txt_i="42274" txt_f="42334">) which are expressed in both myeloid and plasmacytoid DCs [</offsets><xref ref-type="bibr" rid="B75"><offsets xml_i="59235" xml_f="59237" txt_i="42334" txt_f="42336">75</offsets></xref><offsets xml_i="59244" xml_f="59253" txt_i="42336" txt_f="42345">], while </offsets><italic><offsets xml_i="59261" xml_f="59266" txt_i="42345" txt_f="42350">ILT7 </offsets></italic><offsets xml_i="59275" xml_f="59276" txt_i="42350" txt_f="42351">(</offsets><italic><offsets xml_i="59284" xml_f="59290" txt_i="42351" txt_f="42357">LILRA4</offsets></italic><offsets xml_i="59299" xml_f="59301" txt_i="42357" txt_f="42359">, </offsets><italic><offsets xml_i="59309" xml_f="59314" txt_i="42359" txt_f="42364">CD85g</offsets></italic><offsets xml_i="59323" xml_f="59360" txt_i="42364" txt_f="42401">) is restricted to plasmacytoid DCs [</offsets><xref ref-type="bibr" rid="B76"><offsets xml_i="59392" xml_f="59394" txt_i="42401" txt_f="42403">76</offsets></xref><offsets xml_i="59401" xml_f="59402" txt_i="42403" txt_f="42404">,</offsets><xref ref-type="bibr" rid="B77"><offsets xml_i="59434" xml_f="59436" txt_i="42404" txt_f="42406">77</offsets></xref><offsets xml_i="59443" xml_f="59453" txt_i="42406" txt_f="42416">]. Figure </offsets><xref ref-type="fig" rid="F9"><offsets xml_i="59483" xml_f="59485" txt_i="42416" txt_f="42418">9a</offsets></xref><offsets xml_i="59492" xml_f="59504" txt_i="42418" txt_f="42430"> and Figure </offsets><xref ref-type="fig" rid="F9"><offsets xml_i="59534" xml_f="59536" txt_i="42430" txt_f="42432">9b</offsets></xref><offsets xml_i="59543" xml_f="59585" txt_i="42432" txt_f="42474"> illustrate the increase in expression of </offsets><italic><offsets xml_i="59593" xml_f="59598" txt_i="42474" txt_f="42479">CD86 </offsets></italic><offsets xml_i="59607" xml_f="59660" txt_i="42479" txt_f="42532">and ILT7 with imiquimod treatment. The expression of </offsets><italic><offsets xml_i="59668" xml_f="59673" txt_i="42532" txt_f="42537">CD86 </offsets></italic><offsets xml_i="59682" xml_f="59945" txt_i="42537" txt_f="42800">remained elevated 4 weeks post treatment. Increased expression is taken as indication of recruitment of these cells to the site of treatment. These observations are consistent with previous studies using topically-applied imiquimod in the treatment of human BCC [</offsets><xref ref-type="bibr" rid="B69"><offsets xml_i="59977" xml_f="59979" txt_i="42800" txt_f="42802">69</offsets></xref><offsets xml_i="59986" xml_f="60010" txt_i="42802" txt_f="42826">] and melanoma in mice [</offsets><xref ref-type="bibr" rid="B78"><offsets xml_i="60042" xml_f="60044" txt_i="42826" txt_f="42828">78</offsets></xref><offsets xml_i="60051" xml_f="60121" txt_i="42828" txt_f="42898">], showing recruitment of plasmacytoid DCs into the site of treatment.</offsets></p><fig position="float" id="F9"><label><offsets xml_i="60162" xml_f="60170" txt_i="42899" txt_f="42907">Figure 9</offsets></label><caption><p><offsets xml_i="60190" xml_f="60337" txt_i="42907" txt_f="43054">Increase in expression of genes indicating the infiltration of DCs upon treatment with imiquimod as determined by Affymetrix GeneChip analysis. (A)</offsets><italic><offsets xml_i="60345" xml_f="60350" txt_i="43054" txt_f="43059">CD86 </offsets></italic><offsets xml_i="60359" xml_f="60363" txt_i="43059" txt_f="43063">(B) </offsets><italic><offsets xml_i="60371" xml_f="60375" txt_i="43063" txt_f="43067">ILT7</offsets></italic><offsets xml_i="60384" xml_f="60453" txt_i="43067" txt_f="43136">. Experimental conditions and analysis are as described for Figure 4.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-9"></graphic></fig><p><offsets xml_i="60524" xml_f="60555" txt_i="43137" txt_f="43168">It is interesting to note that </offsets><italic><offsets xml_i="60563" xml_f="60568" txt_i="43168" txt_f="43173">CD1C </offsets></italic><offsets xml_i="60577" xml_f="60650" txt_i="43173" txt_f="43246">is markedly decreased upon treatment with imiquimod [see Additional file </offsets><xref ref-type="supplementary-material" rid="S3"><offsets xml_i="60699" xml_f="60700" txt_i="43246" txt_f="43247">3</offsets></xref><offsets xml_i="60707" xml_f="60763" txt_i="43247" txt_f="43303">]. CD1c is found on Langerhans cells as well as on DCs [</offsets><xref ref-type="bibr" rid="B79"><offsets xml_i="60795" xml_f="60797" txt_i="43303" txt_f="43305">79</offsets></xref><offsets xml_i="60804" xml_f="60805" txt_i="43305" txt_f="43306">,</offsets><xref ref-type="bibr" rid="B80"><offsets xml_i="60837" xml_f="60839" txt_i="43306" txt_f="43308">80</offsets></xref><offsets xml_i="60846" xml_f="60865" txt_i="43308" txt_f="43327">]. The decrease in </offsets><italic><offsets xml_i="60873" xml_f="60878" txt_i="43327" txt_f="43332">CD1C </offsets></italic><offsets xml_i="60887" xml_f="61099" txt_i="43332" txt_f="43544">expression may reflect (although not exclusively) the migration of CD1C+ Langerhans cells out of the dermis. Migration of Langerhans cells out of mouse dermis was observed after topical treatment with imiquimod [</offsets><xref ref-type="bibr" rid="B78"><offsets xml_i="61131" xml_f="61133" txt_i="43544" txt_f="43546">78</offsets></xref><offsets xml_i="61140" xml_f="61141" txt_i="43546" txt_f="43547">,</offsets><xref ref-type="bibr" rid="B81"><offsets xml_i="61173" xml_f="61175" txt_i="43547" txt_f="43549">81</offsets></xref><offsets xml_i="61182" xml_f="61222" txt_i="43549" txt_f="43589">]. In the case of the Palamara studies [</offsets><xref ref-type="bibr" rid="B78"><offsets xml_i="61254" xml_f="61256" txt_i="43589" txt_f="43591">78</offsets></xref><offsets xml_i="61263" xml_f="61383" txt_i="43591" txt_f="43711">] in mouse melanoma, Langerhans cells were observed to return to normal levels in the dermis by day 20. The decrease in </offsets><italic><offsets xml_i="61391" xml_f="61396" txt_i="43711" txt_f="43716">CD1C </offsets></italic><offsets xml_i="61405" xml_f="61590" txt_i="43716" txt_f="43901">observed upon treatment of AK lesions with imiquimod therefore suggests the activation and migration of Langerhans cells to the lymph nodes and is consistent with previous observations.</offsets></p></sec><sec><title><offsets xml_i="61612" xml_f="61725" txt_i="43903" txt_f="44016">Imiquimod increases the expression of genes predictive of infiltrating cytotoxic T cells and natural killer cells</offsets></title><p><offsets xml_i="61736" xml_f="62101" txt_i="44017" txt_f="44382">Natural killer cells mediate lysis of tumor cells as well as virally-infected cells. Natural killer cells preferentially express several calcium-dependent (C-type) lectins, known as the NKG2 family, which have been implicated in the regulation of NK cell function, and are believed to be important for NK cell-mediated tumor rejection and T-cell mediated immunity [</offsets><xref ref-type="bibr" rid="B82"><offsets xml_i="62133" xml_f="62135" txt_i="44382" txt_f="44384">82</offsets></xref><offsets xml_i="62142" xml_f="62188" txt_i="44384" txt_f="44430">]. Transcripts for 3 of these C-type lectins, </offsets><italic><offsets xml_i="62196" xml_f="62201" txt_i="44430" txt_f="44435">KLRC1</offsets></italic><offsets xml_i="62210" xml_f="62211" txt_i="44435" txt_f="44436">/</offsets><italic><offsets xml_i="62219" xml_f="62221" txt_i="44436" txt_f="44438">C2</offsets></italic><offsets xml_i="62230" xml_f="62232" txt_i="44438" txt_f="44440">, </offsets><italic><offsets xml_i="62240" xml_f="62246" txt_i="44440" txt_f="44446">KLRK1 </offsets></italic><offsets xml_i="62255" xml_f="62256" txt_i="44446" txt_f="44447">(</offsets><italic><offsets xml_i="62264" xml_f="62269" txt_i="44447" txt_f="44452">NKG2D</offsets></italic><offsets xml_i="62278" xml_f="62284" txt_i="44452" txt_f="44458">) and </offsets><italic><offsets xml_i="62292" xml_f="62298" txt_i="44458" txt_f="44464">KLRF1 </offsets></italic><offsets xml_i="62307" xml_f="62474" txt_i="44464" txt_f="44631">were increased in expression in imiquimod-treated samples. In addition, several genes important to the cytolytic function of NK cells and cytotoxic T cells, including </offsets><italic><offsets xml_i="62482" xml_f="62489" txt_i="44631" txt_f="44638">TYROBP </offsets></italic><offsets xml_i="62498" xml_f="62499" txt_i="44638" txt_f="44639">(</offsets><italic><offsets xml_i="62507" xml_f="62512" txt_i="44639" txt_f="44644">DAP12</offsets></italic><offsets xml_i="62521" xml_f="62527" txt_i="44644" txt_f="44650">) and </offsets><italic><offsets xml_i="62535" xml_f="62540" txt_i="44650" txt_f="44655">CD69 </offsets></italic><offsets xml_i="62549" xml_f="62584" txt_i="44655" txt_f="44690">(early T-cell activation antigen), </offsets><italic><offsets xml_i="62592" xml_f="62598" txt_i="44690" txt_f="44696">ITGAL </offsets></italic><offsets xml_i="62607" xml_f="62608" txt_i="44696" txt_f="44697">(</offsets><italic><offsets xml_i="62616" xml_f="62621" txt_i="44697" txt_f="44702">CD11a</offsets></italic><offsets xml_i="62630" xml_f="62633" txt_i="44702" txt_f="44705">), </offsets><italic><offsets xml_i="62641" xml_f="62647" txt_i="44705" txt_f="44711">ITGB2 </offsets></italic><offsets xml_i="62656" xml_f="62657" txt_i="44711" txt_f="44712">(</offsets><italic><offsets xml_i="62665" xml_f="62669" txt_i="44712" txt_f="44716">CD18</offsets></italic><offsets xml_i="62678" xml_f="62681" txt_i="44716" txt_f="44719">), </offsets><italic><offsets xml_i="62689" xml_f="62694" txt_i="44719" txt_f="44724">ICAM1</offsets></italic><offsets xml_i="62703" xml_f="62729" txt_i="44724" txt_f="44750">, and the ligands for the </offsets><italic><offsets xml_i="62737" xml_f="62742" txt_i="44750" txt_f="44755">NKG2 </offsets></italic><offsets xml_i="62751" xml_f="62762" txt_i="44755" txt_f="44766">receptors, </offsets><italic><offsets xml_i="62770" xml_f="62775" txt_i="44766" txt_f="44771">MICA </offsets></italic><offsets xml_i="62784" xml_f="62788" txt_i="44771" txt_f="44775">and </offsets><italic><offsets xml_i="62796" xml_f="62801" txt_i="44775" txt_f="44780">MICB </offsets></italic><offsets xml_i="62810" xml_f="62827" txt_i="44780" txt_f="44797">[Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="62876" xml_f="62877" txt_i="44797" txt_f="44798">1</offsets></xref><offsets xml_i="62884" xml_f="62887" txt_i="44798" txt_f="44801">] [</offsets><xref ref-type="bibr" rid="B83"><offsets xml_i="62919" xml_f="62921" txt_i="44801" txt_f="44803">83</offsets></xref><offsets xml_i="62928" xml_f="62929" txt_i="44803" txt_f="44804">-</offsets><xref ref-type="bibr" rid="B88"><offsets xml_i="62961" xml_f="62963" txt_i="44804" txt_f="44806">88</offsets></xref><offsets xml_i="62970" xml_f="63116" txt_i="44806" txt_f="44952">] were increased in expression with imiquimod treatment. Also, genes of the granule products of NK cells and cytotoxic T cells, such as granzymes </offsets><italic><offsets xml_i="63124" xml_f="63128" txt_i="44952" txt_f="44956">GZMA</offsets></italic><offsets xml_i="63137" xml_f="63139" txt_i="44956" txt_f="44958">, </offsets><italic><offsets xml_i="63147" xml_f="63151" txt_i="44958" txt_f="44962">GZMB</offsets></italic><offsets xml_i="63160" xml_f="63162" txt_i="44962" txt_f="44964">, </offsets><italic><offsets xml_i="63170" xml_f="63174" txt_i="44964" txt_f="44968">GZMK</offsets></italic><offsets xml_i="63183" xml_f="63189" txt_i="44968" txt_f="44974">, and </offsets><italic><offsets xml_i="63197" xml_f="63203" txt_i="44974" txt_f="44980">GYNYL </offsets></italic><offsets xml_i="63212" xml_f="63225" txt_i="44980" txt_f="44993">(granulysin, </offsets><italic><offsets xml_i="63233" xml_f="63237" txt_i="44993" txt_f="44997">NKG5</offsets></italic><offsets xml_i="63246" xml_f="63249" txt_i="44997" txt_f="45000">), </offsets><italic><offsets xml_i="63257" xml_f="63261" txt_i="45000" txt_f="45004">PRF </offsets></italic><offsets xml_i="63270" xml_f="63285" txt_i="45004" txt_f="45019">(perforin) and </offsets><italic><offsets xml_i="63293" xml_f="63298" txt_i="45019" txt_f="45024">NKG7 </offsets></italic><offsets xml_i="63307" xml_f="63308" txt_i="45024" txt_f="45025">(</offsets><italic><offsets xml_i="63316" xml_f="63320" txt_i="45025" txt_f="45029">GIG1</offsets></italic><offsets xml_i="63329" xml_f="63331" txt_i="45029" txt_f="45031">, </offsets><italic><offsets xml_i="63339" xml_f="63345" txt_i="45031" txt_f="45037">GMP-17</offsets></italic><offsets xml_i="63354" xml_f="63440" txt_i="45037" txt_f="45123">) were increased in expression, indicating that these cells are cytolytically active [</offsets><xref ref-type="bibr" rid="B89"><offsets xml_i="63472" xml_f="63474" txt_i="45123" txt_f="45125">89</offsets></xref><offsets xml_i="63481" xml_f="63482" txt_i="45125" txt_f="45126">-</offsets><xref ref-type="bibr" rid="B93"><offsets xml_i="63514" xml_f="63516" txt_i="45126" txt_f="45128">93</offsets></xref><offsets xml_i="63523" xml_f="63533" txt_i="45128" txt_f="45138">]. Figure </offsets><xref ref-type="fig" rid="F10"><offsets xml_i="63564" xml_f="63567" txt_i="45138" txt_f="45141">10a</offsets></xref><offsets xml_i="63574" xml_f="63579" txt_i="45141" txt_f="45146"> and </offsets><xref ref-type="fig" rid="F10"><offsets xml_i="63610" xml_f="63613" txt_i="45146" txt_f="45149">10b</offsets></xref><offsets xml_i="63620" xml_f="63655" txt_i="45149" txt_f="45184"> show changes in the expression of </offsets><italic><offsets xml_i="63663" xml_f="63668" txt_i="45184" txt_f="45189">GZMA </offsets></italic><offsets xml_i="63677" xml_f="63681" txt_i="45189" txt_f="45193">and </offsets><italic><offsets xml_i="63689" xml_f="63694" txt_i="45193" txt_f="45198">NKG7 </offsets></italic></p><fig position="float" id="F10"><label><offsets xml_i="63745" xml_f="63754" txt_i="45199" txt_f="45208">Figure 10</offsets></label><caption><p><offsets xml_i="63774" xml_f="63933" txt_i="45208" txt_f="45367">Increase in expression of genes associated with NK cells, and cytotoxic T cells after treatment with imiquimod as determined by Affymetrix gene expression. (A)</offsets><italic><offsets xml_i="63941" xml_f="63946" txt_i="45367" txt_f="45372">GZMA </offsets></italic><offsets xml_i="63955" xml_f="63959" txt_i="45372" txt_f="45376">(B) </offsets><italic><offsets xml_i="63967" xml_f="63971" txt_i="45376" txt_f="45380">NKG7</offsets></italic><offsets xml_i="63980" xml_f="64049" txt_i="45380" txt_f="45449">. Experimental conditions and analysis are as described for Figure 4.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-10"></graphic></fig></sec><sec><title><offsets xml_i="64136" xml_f="64238" txt_i="45451" txt_f="45553">Imiquimod increases the expression of genes predictive of the activation of the adaptive immune system</offsets></title><p><offsets xml_i="64249" xml_f="64415" txt_i="45554" txt_f="45720">The presence of activated T cells in imiquimod-treated AK lesions is indicated by the expression of several members of the T-cell activation pathway [Additional File </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="64464" xml_f="64465" txt_i="45720" txt_f="45721">1</offsets></xref><offsets xml_i="64472" xml_f="64574" txt_i="45721" txt_f="45823">]). These include the T-cell receptor (TCR) subunits TRD and TRG, TCR-signaling pathway genes such as </offsets><italic><offsets xml_i="64582" xml_f="64585" txt_i="45823" txt_f="45826">Fyn</offsets></italic><offsets xml_i="64594" xml_f="64596" txt_i="45826" txt_f="45828">, </offsets><italic><offsets xml_i="64604" xml_f="64608" txt_i="45828" txt_f="45832">Fyb </offsets></italic><offsets xml_i="64617" xml_f="64621" txt_i="45832" txt_f="45836">and </offsets><italic><offsets xml_i="64629" xml_f="64634" txt_i="45836" txt_f="45841">LCP2 </offsets></italic><offsets xml_i="64643" xml_f="64644" txt_i="45841" txt_f="45842">[</offsets><xref ref-type="bibr" rid="B94"><offsets xml_i="64676" xml_f="64678" txt_i="45842" txt_f="45844">94</offsets></xref><offsets xml_i="64685" xml_f="64686" txt_i="45844" txt_f="45845">,</offsets><xref ref-type="bibr" rid="B95"><offsets xml_i="64718" xml_f="64720" txt_i="45845" txt_f="45847">95</offsets></xref><offsets xml_i="64727" xml_f="64788" txt_i="45847" txt_f="45908">], and other genes associated with T-cell activation such as </offsets><italic><offsets xml_i="64796" xml_f="64799" txt_i="45908" txt_f="45911">HCK</offsets></italic><offsets xml_i="64808" xml_f="64810" txt_i="45911" txt_f="45913">, </offsets><italic><offsets xml_i="64818" xml_f="64822" txt_i="45913" txt_f="45917">CD69</offsets></italic><offsets xml_i="64831" xml_f="64833" txt_i="45917" txt_f="45919">, </offsets><italic><offsets xml_i="64841" xml_f="64847" txt_i="45919" txt_f="45925">PTPRC </offsets></italic><offsets xml_i="64856" xml_f="64857" txt_i="45925" txt_f="45926">(</offsets><italic><offsets xml_i="64865" xml_f="64869" txt_i="45926" txt_f="45930">CD45</offsets></italic><offsets xml_i="64878" xml_f="64880" txt_i="45930" txt_f="45932">) </offsets><italic><offsets xml_i="64888" xml_f="64893" txt_i="45932" txt_f="45937">SELL </offsets></italic><offsets xml_i="64902" xml_f="64903" txt_i="45937" txt_f="45938">(</offsets><italic><offsets xml_i="64911" xml_f="64916" txt_i="45938" txt_f="45943">CD62L</offsets></italic><offsets xml_i="64925" xml_f="64940" txt_i="45943" txt_f="45958">, L-Selectin), </offsets><italic><offsets xml_i="64948" xml_f="64954" txt_i="45958" txt_f="45964">ITGA4 </offsets></italic><offsets xml_i="64963" xml_f="65019" txt_i="45964" txt_f="46020">(antigen CD49D, alpha 4 subunit of VLA-4 receptor), and </offsets><italic><offsets xml_i="65027" xml_f="65031" txt_i="46020" txt_f="46024">LAG3</offsets></italic><offsets xml_i="65040" xml_f="65219" txt_i="46024" txt_f="46203">. In addition, treatment of AK lesions with imiquimod resulted in a small but significant increase in the expression of CD8β, suggesting that there is an increase in CD8 T cells [</offsets><xref ref-type="bibr" rid="B96"><offsets xml_i="65251" xml_f="65253" txt_i="46203" txt_f="46205">96</offsets></xref><offsets xml_i="65260" xml_f="65261" txt_i="46205" txt_f="46206">,</offsets><xref ref-type="bibr" rid="B97"><offsets xml_i="65293" xml_f="65295" txt_i="46206" txt_f="46208">97</offsets></xref><offsets xml_i="65302" xml_f="65418" txt_i="46208" txt_f="46324">]. The presence of CD8 memory T cells in imiquimod-treated subjects is also suggested by increases in expression of </offsets><italic><offsets xml_i="65426" xml_f="65430" txt_i="46324" txt_f="46328">SELL</offsets></italic><offsets xml_i="65439" xml_f="65441" txt_i="46328" txt_f="46330">, </offsets><italic><offsets xml_i="65449" xml_f="65454" txt_i="46330" txt_f="46335">NT5E </offsets></italic><offsets xml_i="65463" xml_f="65464" txt_i="46335" txt_f="46336">(</offsets><italic><offsets xml_i="65472" xml_f="65476" txt_i="46336" txt_f="46340">CD73</offsets></italic><offsets xml_i="65485" xml_f="65488" txt_i="46340" txt_f="46343">), </offsets><italic><offsets xml_i="65496" xml_f="65503" txt_i="46343" txt_f="46350">LGALS2 </offsets></italic><offsets xml_i="65512" xml_f="65530" txt_i="46350" txt_f="46368">(galectin 2), and </offsets><italic><offsets xml_i="65538" xml_f="65544" txt_i="46368" txt_f="46374">LAIR1 </offsets></italic><offsets xml_i="65553" xml_f="65693" txt_i="46374" txt_f="46514">(leukocyte-associated immunoglobulin-like receptor 1), which was recently identified to be differentially expressed in mouse memory Tcells [</offsets><xref ref-type="bibr" rid="B98"><offsets xml_i="65725" xml_f="65727" txt_i="46514" txt_f="46516">98</offsets></xref><offsets xml_i="65734" xml_f="65744" txt_i="46516" txt_f="46526">]. Figure </offsets><xref ref-type="fig" rid="F11"><offsets xml_i="65775" xml_f="65778" txt_i="46526" txt_f="46529">11a</offsets></xref><offsets xml_i="65785" xml_f="65797" txt_i="46529" txt_f="46541"> and Figure </offsets><xref ref-type="fig" rid="F11"><offsets xml_i="65828" xml_f="65831" txt_i="46541" txt_f="46544">11b</offsets></xref><offsets xml_i="65838" xml_f="65880" txt_i="46544" txt_f="46586"> illustrate the increase in expression of </offsets><italic><offsets xml_i="65888" xml_f="65893" txt_i="46586" txt_f="46591">SELL </offsets></italic><offsets xml_i="65902" xml_f="66343" txt_i="46591" txt_f="47032">and NT5E respectively. The expression levels of these genes returned to pretreatment levels 4 weeks post treatment. Taken together, the increased expression of Tcell receptor genes, T cell activation marker genes and genes important for T cell co stimulation suggests that treatment with imiquimod results in the infiltration of T cells into AK lesions. The increased expression of genes important for the development of T-cell memory (e.g. </offsets><italic><offsets xml_i="66351" xml_f="66355" txt_i="47032" txt_f="47036">SELL</offsets></italic><offsets xml_i="66364" xml_f="66366" txt_i="47036" txt_f="47038">, </offsets><italic><offsets xml_i="66374" xml_f="66378" txt_i="47038" txt_f="47042">NT5E</offsets></italic><offsets xml_i="66387" xml_f="66697" txt_i="47042" txt_f="47352">), as well as genes associated with cytotoxic T cells (e.g. granzymes and perforin) suggests that imiquimod treatment recruits both memory and effector T cells. These results are consistent with the observation of increased CD3, CD4 and CD8 positive cells after topical application of imiquimod to AK lesions [</offsets><xref ref-type="bibr" rid="B72"><offsets xml_i="66729" xml_f="66731" txt_i="47352" txt_f="47354">72</offsets></xref><offsets xml_i="66738" xml_f="66788" txt_i="47354" txt_f="47404">], as well as infiltration of CD8 T cells in BCC [</offsets><xref ref-type="bibr" rid="B19"><offsets xml_i="66820" xml_f="66822" txt_i="47404" txt_f="47406">19</offsets></xref><offsets xml_i="66829" xml_f="66830" txt_i="47406" txt_f="47407">,</offsets><xref ref-type="bibr" rid="B66"><offsets xml_i="66862" xml_f="66864" txt_i="47407" txt_f="47409">66</offsets></xref><offsets xml_i="66871" xml_f="66946" txt_i="47409" txt_f="47484">] and in cutaneous squamous cell carcinoma treated with topical imiquimod [</offsets><xref ref-type="bibr" rid="B99"><offsets xml_i="66978" xml_f="66980" txt_i="47484" txt_f="47486">99</offsets></xref><offsets xml_i="66987" xml_f="67330" txt_i="47486" txt_f="47829">]. The increased expression of genes important in the activation and co stimulation of cells of the adaptive immune system is consistent with the infiltration of cells important in the development of the adaptive immune system. These observations are also consistent with the reported adjuvant activity of imiquimod and its analog resiquimod [</offsets><xref ref-type="bibr" rid="B26"><offsets xml_i="67362" xml_f="67364" txt_i="47829" txt_f="47831">26</offsets></xref><offsets xml_i="67371" xml_f="67431" txt_i="47831" txt_f="47891">], and the low recurrence rates observed in animal studies [</offsets><xref ref-type="bibr" rid="B100"><offsets xml_i="67464" xml_f="67467" txt_i="47891" txt_f="47894">100</offsets></xref><offsets xml_i="67474" xml_f="67513" txt_i="47894" txt_f="47933">] and clinical studies with imiquimod [</offsets><xref ref-type="bibr" rid="B101"><offsets xml_i="67546" xml_f="67549" txt_i="47933" txt_f="47936">101</offsets></xref><offsets xml_i="67556" xml_f="67656" txt_i="47936" txt_f="48036">]. These same observations suggest that imiquimod may also prevent recurrence of AK lesions as well.</offsets></p><fig position="float" id="F11"><label><offsets xml_i="67698" xml_f="67707" txt_i="48037" txt_f="48046">Figure 11</offsets></label><caption><p><offsets xml_i="67727" xml_f="67877" txt_i="48046" txt_f="48196">Increase in expression of genes associated with lymphocyte function after treatment with imiquimod as determined by Affymetrix GeneChip analysis. (A) </offsets><italic><offsets xml_i="67885" xml_f="67890" txt_i="48196" txt_f="48201">SELL </offsets></italic><offsets xml_i="67899" xml_f="67900" txt_i="48201" txt_f="48202">(</offsets><italic><offsets xml_i="67908" xml_f="67913" txt_i="48202" txt_f="48207">CD62L</offsets></italic><offsets xml_i="67922" xml_f="67928" txt_i="48207" txt_f="48213">) (B) </offsets><italic><offsets xml_i="67936" xml_f="67941" txt_i="48213" txt_f="48218">NT5E </offsets></italic><offsets xml_i="67950" xml_f="67951" txt_i="48218" txt_f="48219">(</offsets><italic><offsets xml_i="67959" xml_f="67963" txt_i="48219" txt_f="48223">DC73</offsets></italic><offsets xml_i="67972" xml_f="68042" txt_i="48223" txt_f="48293">). Experimental conditions and analysis are as described for Figure 4.</offsets></p></caption><graphic xlink:href="1479-5876-5-7-11"></graphic></fig><p><offsets xml_i="68114" xml_f="68921" txt_i="48294" txt_f="49101">In summary, the data suggest that the therapeutic effect of imiquimod in the treatment of AK involves the stimulation of both the innate and adaptive immune responses. The role of type 1 interferons in imiquimod's mechanism of action is underscored by the induction of large numbers of IFNα/β-inducible genes with tumor growth-inhibitory and immune stimulatory activity. Data indicate that the antilesional activity of imiquimod is a result of the induction of a strong immune cell-mediated cytolytic and apoptotic gene expression program that leads to destruction of AK lesions and sun-damaged cells. The development of immune memory indicated by this study, as well as the observation of low recurrence in clinical studies, makes imiquimod a unique therapy for AK as well as for other cutaneous neoplasms.</offsets></p></sec></sec><sec><title><offsets xml_i="68949" xml_f="68962" txt_i="49104" txt_f="49117">Abbreviations</offsets></title><p><offsets xml_i="68973" xml_f="69246" txt_i="49118" txt_f="49391">AK–actinic keratosis(ses), ANOVA–analysis of variance, BCC–basal cell carcinoma, DC–dendritic cells, IFN–interferon, IL–interleukin, IRF–interferon regulatory factor, NK–natural killer [cells], RT-PCR–Reverse Transcriptase Polymerase Chain Reaction, TLR–Toll like receptors</offsets></p></sec><sec><title><offsets xml_i="69268" xml_f="69287" txt_i="49393" txt_f="49412">Competing interests</offsets></title><p><offsets xml_i="69298" xml_f="69471" txt_i="49413" txt_f="49586">Abel Torres, MD, JD, has acted as a lecturer for 3M Pharmaceutical and has participated in 3M Pharmaceutical funded research. He also does consulting for 3M Pharmaceuticals.</offsets></p><p><offsets xml_i="69478" xml_f="69634" txt_i="49587" txt_f="49743">RM, BJB, JJ, SR, and WB are employees of 3M Pharmaceuticals. JL and HB were employees of 3M Pharmaceuticals at the time of the study and manuscript writing.</offsets></p></sec><sec><title><offsets xml_i="69656" xml_f="69678" txt_i="49745" txt_f="49767">Authors' contributions</offsets></title><p><offsets xml_i="69689" xml_f="69881" txt_i="49768" txt_f="49960">AT, LS and MA conducted the clinical study. JL was the clinical coordinator. Gene expression analysis was performed at 3M Pharmaceuticals by WB, BB and JJ. All authors approved the manuscript.</offsets></p></sec><sec sec-type="supplementary-material"><title><offsets xml_i="69937" xml_f="69959" txt_i="49962" txt_f="49984">Supplementary Material</offsets></title><supplementary-material content-type="local-data" id="S1"><caption><title><offsets xml_i="70041" xml_f="70058" txt_i="49985" txt_f="50002">Additional file 1</offsets></title><p><offsets xml_i="70069" xml_f="70433" txt_i="50003" txt_f="50364">Imiquimod regulated genes. This file summarizes fold change in expression for 530 imiquimod regulated genes before and after treatment with imiquimod. Genes were selected on the basis of an ANOVA analysis comparing pre-treatment AK expression values to expression values during imiquimod treatment or 4-weeks after treatment with imiquimod, with P-values &lt; 0.05</offsets></p></caption><media xlink:href="1479-5876-5-7-S1.doc" mimetype="application" mime-subtype="msword"><caption><p><offsets xml_i="70545" xml_f="70564" txt_i="50365" txt_f="50384">Click here for file</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="S2"><caption><title><offsets xml_i="70685" xml_f="70702" txt_i="50385" txt_f="50402">Additional file 2</offsets></title><p><offsets xml_i="70713" xml_f="71106" txt_i="50403" txt_f="50796">Comparison of differential expression as measured by Affymetrix gene array and real time RT-PCR. This file summarizes a comparison of differential gene expression as measured by Affymetrix gene chip analysis and real time RT-PCR using low density TaqMan arrays. Expression changes for several toll-like receptors and other selected genes before and after treatment with imiquimod are reported.</offsets></p></caption><media xlink:href="1479-5876-5-7-S2.doc" mimetype="application" mime-subtype="msword"><caption><p><offsets xml_i="71218" xml_f="71237" txt_i="50797" txt_f="50816">Click here for file</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="S3"><caption><title><offsets xml_i="71358" xml_f="71375" txt_i="50817" txt_f="50834">Additional file 3</offsets></title><p><offsets xml_i="71386" xml_f="71696" txt_i="50835" txt_f="51145">Imiquimod-induced genes with immune response gene ontology category and/or known to be IFNα/β-inducible. This file reports on expression changes before and after treatment with imiquimod, for selected genes with gene ontology category of immune response and (or) are know to be inducible by type 1 interferons.</offsets></p></caption><media xlink:href="1479-5876-5-7-S3.doc" mimetype="application" mime-subtype="msword"><caption><p><offsets xml_i="71808" xml_f="71827" txt_i="51146" txt_f="51165">Click here for file</offsets></p></caption></media></supplementary-material></sec></body><back><ack><sec><title>Acknowledgements</title><p>We thank Rhonda Dick for helping with manuscript preparation, and Dr. Mark Tomai for valuable criticism of the manuscript. Both are employees of 3M Pharmaceuticals.</p></sec></ack><ref-list><ref id="B1"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Callen</surname><given-names>JP</given-names></name><name><surname>Bickers</surname><given-names>DR</given-names></name><name><surname>Moy</surname><given-names>RL</given-names></name></person-group><article-title>Actinic keratosis</article-title><source>J Am Acad Dermatol</source><year>1997</year><volume>36</volume><fpage>650</fpage><lpage>653</lpage><pub-id pub-id-type="pmid">9092763</pub-id><pub-id pub-id-type="doi">10.1016/S0190-9622(97)70265-2</pub-id></citation></ref><ref id="B2"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Schwartz</surname><given-names>RA</given-names></name></person-group><article-title>The actinic keratosis: a perspective and update</article-title><source>Dermatol Surg</source><year>1997</year><volume>23</volume><fpage>1009</fpage><lpage>1019</lpage><pub-id pub-id-type="pmid">9391557</pub-id><pub-id pub-id-type="doi">10.1016/S1076-0512(97)00353-1</pub-id></citation></ref><ref id="B3"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Marks</surname><given-names>R</given-names></name><name><surname>Rennie</surname><given-names>G</given-names></name><name><surname>Selwood</surname><given-names>T</given-names></name></person-group><article-title>The relationship of basal cell carcinomas and squamous cell carcinomas to solar keratosis</article-title><source>Arch Dermatol</source><year>1988</year><volume>124</volume><fpage>1039</fpage><lpage>1042</lpage><pub-id pub-id-type="pmid">3389848</pub-id><pub-id pub-id-type="doi">10.1001/archderm.124.7.1039</pub-id></citation></ref><ref id="B4"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Nghiem</surname><given-names>DX</given-names></name><name><surname>Kazimi</surname><given-names>N</given-names></name><name><surname>Mitchell</surname><given-names>DL</given-names></name><name><surname>Vink</surname><given-names>AA</given-names></name><name><surname>Ananthaswamy</surname><given-names>HN</given-names></name><name><surname>Kripke</surname><given-names>ML</given-names></name><name><surname>Ullric</surname><given-names></given-names></name></person-group><article-title>Mechanisms underlying the suppression of established immune responses by ultraviolet radiation</article-title><source>J Invest Dermatol</source><year>2002</year><volume>119</volume><fpage>600</fpage><lpage>608</lpage><pub-id pub-id-type="pmid">12230501</pub-id><pub-id pub-id-type="doi">10.1046/j.1523-1747.2002.01845.x</pub-id></citation></ref><ref id="B5"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Yantsos</surname><given-names>VA</given-names></name><name><surname>Conrad</surname><given-names>N</given-names></name><name><surname>Zabawski</surname><given-names>E</given-names></name><name><surname>Cockerell</surname><given-names>CJ</given-names></name></person-group><article-title>Incipient intraepidermal cutaneous squamous cell carcinoma: A proposal for reclassifying and grading solar (actinic) keratosis</article-title><source>Semin Cutan Med Surg</source><year>1999</year><volume>18</volume><fpage>3</fpage><lpage>14</lpage><pub-id pub-id-type="pmid">10188837</pub-id><pub-id pub-id-type="doi">10.1016/S1085-5629(99)80003-0</pub-id></citation></ref><ref id="B6"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Glogau</surname><given-names>RG</given-names></name></person-group><article-title>The risk of progression to invasive disease</article-title><source>J Am Acad Dermatol</source><year>2000</year><volume>42</volume><fpage>23</fpage><lpage>24</lpage><pub-id pub-id-type="pmid">10607353</pub-id><pub-id pub-id-type="doi">10.1067/mjd.2000.103339</pub-id></citation></ref><ref id="B7"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Silapunt</surname><given-names>S</given-names></name><name><surname>Goldberg</surname><given-names>LH</given-names></name><name><surname>Alam</surname><given-names>M</given-names></name></person-group><article-title>Topical and light-based treatments for actinic keratosis</article-title><source>Semin Cutan Med Surg</source><year>2003</year><volume>22</volume><fpage>162</fpage><lpage>170</lpage><pub-id pub-id-type="pmid">14649583</pub-id><pub-id pub-id-type="doi">10.1016/S1085-5629(03)00040-3</pub-id></citation></ref><ref id="B8"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Fu</surname><given-names>W</given-names></name><name><surname>Cockerell</surname><given-names>CJ</given-names></name></person-group><article-title>The actinic (solar) keratosis: a 21st-century perspective</article-title><source>Arch Dermatol</source><year>2003</year><volume>139</volume><fpage>66</fpage><lpage>70</lpage><pub-id pub-id-type="pmid">12533168</pub-id><pub-id pub-id-type="doi">10.1001/archderm.139.1.66</pub-id></citation></ref><ref id="B9"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Takeda</surname><given-names>K</given-names></name><name><surname>Akira</surname><given-names>S</given-names></name></person-group><article-title>Microbial recognition by Toll-like receptors</article-title><source>J Dermatol Sci</source><year>2004</year><volume>34</volume><fpage>73</fpage><lpage>82</lpage><pub-id pub-id-type="pmid">15033189</pub-id><pub-id pub-id-type="doi">10.1016/j.jdermsci.2003.10.002</pub-id></citation></ref><ref id="B10"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Pasare</surname><given-names>C</given-names></name><name><surname>Medzhitov</surname><given-names>R</given-names></name></person-group><article-title>t Toll-like receptors linking innate and adaptive immunity</article-title><source>Microbes Infect</source><year>2004</year><volume>6</volume><fpage>1382</fpage><lpage>1387</lpage><pub-id pub-id-type="pmid">15596124</pub-id><pub-id pub-id-type="doi">10.1016/j.micinf.2004.08.018</pub-id></citation></ref><ref id="B11"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Hornung</surname><given-names>V</given-names></name><name><surname>Rothenfusser</surname><given-names>S</given-names></name><name><surname>Britsch</surname><given-names>S</given-names></name><name><surname>Krug</surname><given-names>A</given-names></name><name><surname>Jahrsdorfer</surname><given-names>B</given-names></name><name><surname>Giese</surname><given-names>T</given-names></name><name><surname>Endres</surname><given-names>S</given-names></name><name><surname>Hartmann</surname><given-names>G</given-names></name></person-group><article-title>Quantitative expression of toll-like receptor 1–10 mRNA in cellular subsets of human peripheral blood mononuclear cells and sensitivity to CpG oligodeoxynucleotides</article-title><source>J Immunol</source><year>2002</year><volume>168</volume><fpage>4531</fpage><lpage>4537</lpage><pub-id pub-id-type="pmid">11970999</pub-id></citation></ref><ref id="B12"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kollisch</surname><given-names>G</given-names></name><name><surname>Kalali</surname><given-names>BN</given-names></name><name><surname>Voelcker</surname><given-names>V</given-names></name><name><surname>Wallich</surname><given-names>R</given-names></name><name><surname>Behrendt</surname><given-names>H</given-names></name><name><surname>Ring</surname><given-names>J</given-names></name><name><surname>Bauer</surname><given-names>S</given-names></name><name><surname>Jakob</surname><given-names>T</given-names></name><name><surname>Mempel</surname><given-names>M</given-names></name><name><surname>Ollert</surname><given-names>M</given-names></name></person-group><article-title>Various members of the Toll-like receptor family contribute to the innate immune response of human epidermal keratinocytes</article-title><source>Immunology</source><year>2005</year><volume>114</volume><fpage>531</fpage><lpage>541</lpage><pub-id pub-id-type="pmid">15804290</pub-id><pub-id pub-id-type="doi">10.1111/j.1365-2567.2005.02122.x</pub-id></citation></ref><ref id="B13"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>McInturff</surname><given-names>JE</given-names></name><name><surname>Modlin</surname><given-names>RL</given-names></name><name><surname>Kim</surname><given-names>J</given-names></name></person-group><article-title>The role of toll-like receptors in the pathogenesis and treatment of dermatological disease</article-title><source>J Invest Dermatol</source><year>2005</year><volume>125</volume><fpage>1</fpage><lpage>8</lpage><pub-id pub-id-type="pmid">15982296</pub-id><pub-id pub-id-type="doi">10.1111/j.0022-202X.2004.23459.x</pub-id></citation></ref><ref id="B14"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Hemmi</surname><given-names>H</given-names></name><name><surname>Kaisho</surname><given-names>T</given-names></name><name><surname>Takeuchi</surname><given-names>O</given-names></name><name><surname>Sato</surname><given-names>S</given-names></name><name><surname>Sanjo</surname><given-names>H</given-names></name><name><surname>Hoshino</surname><given-names>K</given-names></name><name><surname>Horiuchi</surname><given-names>T</given-names></name><name><surname>Tomizawa</surname><given-names>H</given-names></name><name><surname>Takeda</surname><given-names>K</given-names></name><name><surname>Akira</surname><given-names>S</given-names></name></person-group><article-title>Small anti-viral compounds activate immune cells via the TLR7 MyD88-dependent signaling pathway</article-title><source>Nat Immunol</source><year>2002</year><volume>3</volume><fpage>196</fpage><lpage>200</lpage><pub-id pub-id-type="pmid">11812998</pub-id><pub-id pub-id-type="doi">10.1038/ni758</pub-id></citation></ref><ref id="B15"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Gibson</surname><given-names>SJ</given-names></name><name><surname>Lindh</surname><given-names>JM</given-names></name><name><surname>Riter</surname><given-names>TR</given-names></name><name><surname>Gleason</surname><given-names>RM</given-names></name><name><surname>Rogers</surname><given-names>LM</given-names></name><name><surname>Fuller</surname><given-names>AE</given-names></name><name><surname>Oesterich</surname><given-names>JL</given-names></name><name><surname>Gorden</surname><given-names>KB</given-names></name><name><surname>Qiu</surname><given-names>X</given-names></name><name><surname>McKane</surname><given-names>SW</given-names></name><name><surname>Noelle</surname><given-names>RJ</given-names></name><name><surname>Miller</surname><given-names>RL</given-names></name><name><surname>Kedl</surname><given-names>RM</given-names></name><name><surname>Fitzgerald-Bocarsly</surname><given-names>P</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name><name><surname>Vasilakos</surname><given-names>JP</given-names></name></person-group><article-title>Plasmacytoid dendritic cells produce cytokines and mature in response to the TLR7 agonists, imiquimod and resiquimod</article-title><source>Cell Immunol</source><year>2002</year><volume>218</volume><fpage>74</fpage><lpage>86</lpage><pub-id pub-id-type="pmid">12470615</pub-id><pub-id pub-id-type="doi">10.1016/S0008-8749(02)00517-8</pub-id></citation></ref><ref id="B16"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Beutner</surname><given-names>KR</given-names></name><name><surname>Tyring</surname><given-names>SK</given-names></name><name><surname>Trofatter</surname><given-names>KF</given-names><suffix>Jr</suffix></name><name><surname>Douglas</surname><given-names>JM</given-names><suffix>Jr</suffix></name><name><surname>Spruance</surname><given-names>S</given-names></name><name><surname>Owens</surname><given-names>ML</given-names></name><name><surname>Fox</surname><given-names>TL</given-names></name><name><surname>Hougham</surname><given-names>AJ</given-names></name><name><surname>Schmitt</surname><given-names>KA</given-names></name></person-group><article-title>Imiquimod, a patient-applied immune-response modifier for treatment of external genital warts</article-title><source>Antimicrob Agents Chemother</source><year>1998</year><volume>42</volume><fpage>789</fpage><lpage>794</lpage><pub-id pub-id-type="pmid">9559784</pub-id></citation></ref><ref id="B17"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Szeimies</surname><given-names>RM</given-names></name><name><surname>Gerritsen</surname><given-names>MJ</given-names></name><name><surname>Gupta</surname><given-names>G</given-names></name><name><surname>Ortonne</surname><given-names>JP</given-names></name><name><surname>Serresi</surname><given-names>S</given-names></name><name><surname>Bichel</surname><given-names>J</given-names></name><name><surname>Lee</surname><given-names>JH</given-names></name><name><surname>Fox</surname><given-names>TL</given-names></name><name><surname>Alomar</surname><given-names>A</given-names></name></person-group><article-title>Imiquimod 5% cream for the treatment of actinic keratosis: results from a phase III, randomized, double-blind, vehicle-controlled, clinical trial with histology</article-title><source>J Am Acad Dermatol</source><year>2004</year><volume>51</volume><fpage>547</fpage><lpage>555</lpage><pub-id pub-id-type="pmid">15389189</pub-id><pub-id pub-id-type="doi">10.1016/j.jaad.2004.02.022</pub-id></citation></ref><ref id="B18"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Geisse</surname><given-names>JK</given-names></name><name><surname>Rich</surname><given-names>P</given-names></name><name><surname>Pandya</surname><given-names>A</given-names></name><name><surname>Gross</surname><given-names>K</given-names></name><name><surname>Andres</surname><given-names>K</given-names></name><name><surname>Ginkel</surname><given-names>A</given-names></name><name><surname>Owens</surname><given-names>M</given-names></name></person-group><article-title>Imiquimod 5% cream for the treatment of superficial basal cell carcinoma: a double-blind, randomized, vehicle-controlled study</article-title><source>J Am Acad Dermatol</source><year>2002</year><volume>47</volume><fpage>390</fpage><lpage>8</lpage><pub-id pub-id-type="pmid">12196749</pub-id><pub-id pub-id-type="doi">10.1067/mjd.2002.126215</pub-id></citation></ref><ref id="B19"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Urosevic</surname><given-names>M</given-names></name><name><surname>Maier</surname><given-names>T</given-names></name><name><surname>Benninghoff</surname><given-names>B</given-names></name><name><surname>Slade</surname><given-names>H</given-names></name><name><surname>Burg</surname><given-names>G</given-names></name><name><surname>Dummer</surname><given-names>R</given-names></name></person-group><article-title>Mechanisms underlying imiquimod-induced regression of basal cell carcinoma in vivo</article-title><source>Arch Dermatol</source><year>2003</year><volume>139</volume><fpage>1325</fpage><lpage>1332</lpage><pub-id pub-id-type="pmid">14568837</pub-id><pub-id pub-id-type="doi">10.1001/archderm.139.10.1325</pub-id></citation></ref><ref id="B20"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Barnetson</surname><given-names>RStC</given-names></name><name><surname>Stachell</surname><given-names>A</given-names></name><name><surname>Zhuang</surname><given-names>L</given-names></name><name><surname>Slade</surname><given-names>HB</given-names></name><name><surname>Halliday</surname><given-names>GM</given-names></name></person-group><article-title>Imiquimod induced regression of clinically diagnosed superficial basal cell carcinoma is associated with early infiltration by CD4 T cells and dendritic cells</article-title><source>Clin Exp Dermatol</source><year>2004</year><volume>29</volume><fpage>639</fpage><lpage>643</lpage><pub-id pub-id-type="pmid">15550144</pub-id><pub-id pub-id-type="doi">10.1111/j.1365-2230.2004.01614.x</pub-id></citation></ref><ref id="B21"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Miller</surname><given-names>RL</given-names></name><name><surname>Gerster</surname><given-names>JF</given-names></name><name><surname>Owens</surname><given-names>ML</given-names></name><name><surname>Slade</surname><given-names>HB</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name></person-group><article-title>Imiquimod applied topically: A novel immune response modifier and a new class of drug</article-title><source>Int J Immunopharmacol</source><year>1999</year><volume>21</volume><fpage>1</fpage><lpage>14</lpage><pub-id pub-id-type="pmid">10411278</pub-id><pub-id pub-id-type="doi">10.1016/S0192-0561(98)00068-X</pub-id></citation></ref><ref id="B22"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Lysa</surname><given-names>B</given-names></name><name><surname>Tartler</surname><given-names>U</given-names></name><name><surname>Wolf</surname><given-names>R</given-names></name><name><surname>Arenberger</surname><given-names>P</given-names></name><name><surname>Benninghoff</surname><given-names>B</given-names></name><name><surname>Ruzicka</surname><given-names>T</given-names></name><name><surname>Hengge</surname><given-names>UR</given-names></name><name><surname>Walz</surname><given-names>M</given-names></name></person-group><article-title>Gene expression in actinic keratosis: Pharmacological modulation by imiquimod</article-title><source>Br J Dermatol</source><year>2004</year><volume>151</volume><fpage>1150</fpage><lpage>1159</lpage><pub-id pub-id-type="pmid">15606509</pub-id><pub-id pub-id-type="doi">10.1111/j.1365-2133.2004.06236.x</pub-id></citation></ref><ref id="B23"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Arany</surname><given-names>I</given-names></name><name><surname>Tyring</surname><given-names>SK</given-names></name><name><surname>Stanley</surname><given-names>MA</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name><name><surname>Miller</surname><given-names>RL</given-names></name><name><surname>Smith</surname><given-names>MH</given-names></name><name><surname>McDermott</surname><given-names>DJ</given-names></name><name><surname>Slade</surname><given-names>HB</given-names></name></person-group><article-title>Enhancement of the innate and cellular immune response in patients with genital warts treated with topical imiquimod cream 5%</article-title><source>Antiviral Res</source><year>1999</year><volume>43</volume><fpage>55</fpage><lpage>63</lpage><pub-id pub-id-type="pmid">10480263</pub-id><pub-id pub-id-type="doi">10.1016/S0166-3542(99)00033-9</pub-id></citation></ref><ref id="B24"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Bernstein</surname><given-names>DI</given-names></name><name><surname>Miller</surname><given-names>RL</given-names></name><name><surname>Harrison</surname><given-names>CJ</given-names></name></person-group><article-title>Adjuvant effects of imiquimod on a herpes simplex virus type 2 glycoprotein vaccine in guinea pigs</article-title><source>J Infect Dis</source><year>1993</year><volume>167</volume><fpage>731</fpage><lpage>735</lpage><pub-id pub-id-type="pmid">8382722</pub-id></citation></ref><ref id="B25"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Bernstein</surname><given-names>DI</given-names></name><name><surname>Harrison</surname><given-names>CJ</given-names></name><name><surname>Tepe</surname><given-names>ER</given-names></name><name><surname>Shahwan</surname><given-names>A</given-names></name><name><surname>Miller</surname><given-names>R</given-names></name></person-group><article-title>Effect of imiquimod as an adjuvant for immunotherapy of genital HSV in guinea-pigs</article-title><source>Vaccine</source><year>1995</year><volume>13</volume><fpage>72</fpage><lpage>76</lpage><pub-id pub-id-type="pmid">7762282</pub-id><pub-id pub-id-type="doi">10.1016/0264-410X(95)80014-5</pub-id></citation></ref><ref id="B26"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Vasilakos</surname><given-names>JP</given-names></name><name><surname>Smith</surname><given-names>RM</given-names></name><name><surname>Gibson</surname><given-names>SJ</given-names></name><name><surname>Lindh</surname><given-names>JM</given-names></name><name><surname>Pederson</surname><given-names>LK</given-names></name><name><surname>Reiter</surname><given-names>MJ</given-names></name><name><surname>Smith</surname><given-names>MH</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name></person-group><article-title>Adjuvant activities of immune response modifier R-848: Comparison with CpG ODN</article-title><source>Cell Immunol</source><year>2000</year><volume>204</volume><fpage>64</fpage><lpage>74</lpage><pub-id pub-id-type="pmid">11006019</pub-id><pub-id pub-id-type="doi">10.1006/cimm.2000.1689</pub-id></citation></ref><ref id="B27"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Nair</surname><given-names>S</given-names></name><name><surname>McLaughlin</surname><given-names>C</given-names></name><name><surname>Weizer</surname><given-names>A</given-names></name><name><surname>Su</surname><given-names>Z</given-names></name><name><surname>Boczkowski</surname><given-names>D</given-names></name><name><surname>Dannull</surname><given-names>J</given-names></name><name><surname>Vieweg</surname><given-names>J</given-names></name><name><surname>Gilboa</surname><given-names>E</given-names></name></person-group><article-title>Injection of immature dendritic cells into adjuvant-treated skin obviates the need for ex vivo maturation</article-title><source>J Immunol</source><year>2003</year><volume>171</volume><fpage>6275</fpage><lpage>6282</lpage><pub-id pub-id-type="pmid">14634145</pub-id></citation></ref><ref id="B28"><citation citation-type="other"><person-group person-group-type="author"><collab>Affymetrix Inc</collab></person-group><article-title>Gene chip expression analysis technical manual</article-title><year>2003</year></citation></ref><ref id="B29"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Schindler</surname><given-names>H</given-names></name><name><surname>Wiese</surname><given-names>A</given-names></name><name><surname>Auer</surname><given-names>J</given-names></name><name><surname>Burtscher</surname><given-names>H</given-names></name></person-group><article-title>cRNA target preparation for microarrays: Comparison of gene expression profiles generated with different amplification procedures</article-title><source>Anal Biochem</source><year>2005</year><volume>344</volume><fpage>92</fpage><lpage>101</lpage><pub-id pub-id-type="pmid">16038869</pub-id><pub-id pub-id-type="doi">10.1016/j.ab.2005.06.006</pub-id></citation></ref><ref id="B30"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Klur</surname><given-names>S</given-names></name><name><surname>Toy</surname><given-names>K</given-names></name><name><surname>Williams</surname><given-names>MP</given-names></name><name><surname>Certa</surname><given-names>U</given-names></name></person-group><article-title>Evaluation of procedures for amplification of small-size samples for hybridization on microarrays</article-title><source>Genomics</source><year>2004</year><volume>83</volume><fpage>508</fpage><lpage>517</lpage><pub-id pub-id-type="pmid">14962677</pub-id><pub-id pub-id-type="doi">10.1016/j.ygeno.2003.09.005</pub-id></citation></ref><ref id="B31"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Abruzzo</surname><given-names>LV</given-names></name><name><surname>Lee</surname><given-names>KY</given-names></name><name><surname>Fuller</surname><given-names>A</given-names></name><name><surname>Silverman</surname><given-names>A</given-names></name><name><surname>Keating</surname><given-names>MJ</given-names></name><name><surname>Medeiros</surname><given-names>LJ</given-names></name><name><surname>Coombes</surname><given-names>KR</given-names></name></person-group><article-title>Validation of oligonucleotide microarray data using microfluidic low-density arrays: A new statistical method to normalize real-time RT-PCR data</article-title><source>Biotechniques</source><year>2005</year><volume>38</volume><fpage>785</fpage><lpage>792</lpage><pub-id pub-id-type="pmid">15945375</pub-id></citation></ref><ref id="B32"><citation citation-type="other"><ext-link ext-link-type="uri" xlink:href="http://www.geneontology.org/"></ext-link></citation></ref><ref id="B33"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Zeeberg</surname><given-names>BR</given-names></name><name><surname>Feng</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>G</given-names></name><name><surname>Wang</surname><given-names>MD</given-names></name><name><surname>Fojo</surname><given-names>AT</given-names></name><name><surname>Sunshine</surname><given-names>M</given-names></name><name><surname>Narasimhan</surname><given-names>S</given-names></name><name><surname>Kane</surname><given-names>DW</given-names></name><name><surname>Reinhold</surname><given-names>WC</given-names></name><name><surname>Lababidi</surname><given-names>S</given-names></name><name><surname>Bussey</surname><given-names>KJ</given-names></name><name><surname>Riss</surname><given-names>J</given-names></name><name><surname>Barrett</surname><given-names>JC</given-names></name><name><surname>Weinstein</surname><given-names>JN</given-names></name></person-group><article-title>GoMiner: a resource for biological interpretation of genomic and proteomic data</article-title><source>Genome Biol</source><year>2003</year><volume>4</volume><fpage>R28</fpage><pub-id pub-id-type="pmid">12702209</pub-id><pub-id pub-id-type="doi">10.1186/gb-2003-4-4-r28</pub-id></citation></ref><ref id="B34"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kawai</surname><given-names>T</given-names></name><name><surname>Sato</surname><given-names>S</given-names></name><name><surname>Ishii</surname><given-names>KJ</given-names></name><name><surname>Coban</surname><given-names>C</given-names></name><name><surname>Hemmi</surname><given-names>H</given-names></name><name><surname>Yamamoto</surname><given-names>M</given-names></name><name><surname>Terai</surname><given-names>K</given-names></name><name><surname>Matsuda</surname><given-names>M</given-names></name><name><surname>Inoue</surname><given-names>J</given-names></name><name><surname>Uematsu</surname><given-names>S</given-names></name><name><surname>Takeuchi</surname><given-names>O</given-names></name><name><surname>Akira</surname><given-names>S</given-names></name></person-group><article-title>Interferon-alpha induction through Toll-like receptors involves a direct interaction of IRF7 with MyD88 and TRAF6</article-title><source>Nat Immunol</source><year>2004</year><volume>5</volume><fpage>1061</fpage><lpage>8</lpage><pub-id pub-id-type="pmid">15361868</pub-id><pub-id pub-id-type="doi">10.1038/ni1118</pub-id></citation></ref><ref id="B35"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Miettinen</surname><given-names>M</given-names></name><name><surname>Sareneva</surname><given-names>T</given-names></name><name><surname>Julkunen</surname><given-names>I</given-names></name><name><surname>Matikainen</surname><given-names>S</given-names></name></person-group><article-title>IFNs activate toll-like receptor gene expression in viral infections</article-title><source>Genes Immun</source><year>2001</year><volume>2</volume><fpage>349</fpage><lpage>355</lpage><pub-id pub-id-type="pmid">11607792</pub-id><pub-id pub-id-type="doi">10.1038/sj.gene.6363791</pub-id></citation></ref><ref id="B36"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Zarember</surname><given-names>KA</given-names></name><name><surname>Godowski</surname><given-names>PJ</given-names></name></person-group><article-title>Tissue expression of human Toll-like receptors and differential regulation of Toll-like receptor mRNAs in leukocytes in response to microbes, their products, and cytokines</article-title><source>J Immunol</source><year>2002</year><volume>168</volume><fpage>554</fpage><lpage>561</lpage><comment>Erratum in: <italic>J Immunol </italic>169(2):1136.</comment><pub-id pub-id-type="pmid">11777946</pub-id></citation></ref><ref id="B37"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kokkinopoulos</surname><given-names>I</given-names></name><name><surname>Jordan</surname><given-names>WJ</given-names></name><name><surname>Ritter</surname><given-names>MA</given-names></name></person-group><article-title>Toll-like receptor mRNA expression patterns in human dendritic cells and monocytes</article-title><source>Mol Immunol</source><year>2005</year><volume>42</volume><fpage>957</fpage><lpage>968</lpage><pub-id pub-id-type="pmid">15829287</pub-id><pub-id pub-id-type="doi">10.1016/j.molimm.2004.09.037</pub-id></citation></ref><ref id="B38"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Siren</surname><given-names>J</given-names></name><name><surname>Pirhonen</surname><given-names>J</given-names></name><name><surname>Julkunen</surname><given-names>I</given-names></name><name><surname>Matikainen</surname><given-names>S</given-names></name></person-group><article-title>IFN-alpha regulates TLR-dependent gene expression of IFN-alpha, IFN-beta, IL-28, and IL-29</article-title><source>J Immunol</source><year>2005</year><volume>174</volume><fpage>1932</fpage><lpage>1937</lpage><pub-id pub-id-type="pmid">15699120</pub-id></citation></ref><ref id="B39"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Osterlund</surname><given-names>P</given-names></name><name><surname>Veckman</surname><given-names>V</given-names></name><name><surname>Siren</surname><given-names>J</given-names></name><name><surname>Klucher</surname><given-names>KM</given-names></name><name><surname>Hiscott</surname><given-names>J</given-names></name><name><surname>Matikainen</surname><given-names>S</given-names></name><name><surname>Julkunen</surname><given-names>I</given-names></name></person-group><article-title>Gene expression and antiviral activity of alpha/beta interferons and interleukin-29 in virus-infected human myeloid dendritic cells</article-title><source>J Virol</source><year>2005</year><volume>79</volume><fpage>9608</fpage><lpage>9617</lpage><pub-id pub-id-type="pmid">16014923</pub-id><pub-id pub-id-type="doi">10.1128/JVI.79.15.9608-9617.2005</pub-id></citation></ref><ref id="B40"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Yoneyama</surname><given-names>M</given-names></name><name><surname>Kikuchi</surname><given-names>M</given-names></name><name><surname>Natsukawa</surname><given-names>T</given-names></name><name><surname>Shinobu</surname><given-names>N</given-names></name><name><surname>Imaizumi</surname><given-names>T</given-names></name><name><surname>Miyagishi</surname><given-names>M</given-names></name><name><surname>Taira</surname><given-names>K</given-names></name><name><surname>Akira</surname><given-names>S</given-names></name><name><surname>Fujita</surname><given-names>T</given-names></name></person-group><article-title>The RNA helicase RIG-I has an essential function in double-stranded RNA-induced innate antiviral responses</article-title><source>Nat Immunol</source><year>2004</year><volume>5</volume><fpage>730</fpage><lpage>737</lpage><pub-id pub-id-type="pmid">15208624</pub-id><pub-id pub-id-type="doi">10.1038/ni1087</pub-id></citation></ref><ref id="B41"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Yoneyama</surname><given-names>M</given-names></name><name><surname>Kikuchi</surname><given-names>M</given-names></name><name><surname>Matsumoto</surname><given-names>K</given-names></name><name><surname>Imaizumi</surname><given-names>T</given-names></name><name><surname>Miyagishi</surname><given-names>M</given-names></name><name><surname>Taira</surname><given-names>K</given-names></name><name><surname>Foy</surname><given-names>E</given-names></name><name><surname>Loo</surname><given-names>YM</given-names></name><name><surname>Gale</surname><given-names>M</given-names><suffix>Jr</suffix></name><name><surname>Akira</surname><given-names>S</given-names></name><name><surname>Yonehara</surname><given-names>S</given-names></name><name><surname>Kato</surname><given-names>A</given-names></name><name><surname>Fujita</surname><given-names>T</given-names></name></person-group><article-title>Shared and unique functions of the DExD/H-Box helicases RIG-I, MDA5, and LGP2 in antiviral innate immunity</article-title><source>J Immunol</source><year>2005</year><volume>175</volume><fpage>2851</fpage><lpage>2858</lpage><pub-id pub-id-type="pmid">16116171</pub-id></citation></ref><ref id="B42"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kawai</surname><given-names>T</given-names></name><name><surname>Takahashi</surname><given-names>K</given-names></name><name><surname>Sato</surname><given-names>S</given-names></name><name><surname>Coban</surname><given-names>C</given-names></name><name><surname>Kumar</surname><given-names>H</given-names></name><name><surname>Kato</surname><given-names>H</given-names></name><name><surname>Ishii</surname><given-names>KJ</given-names></name><name><surname>Takeuchi</surname><given-names>O</given-names></name><name><surname>Akira</surname><given-names>S</given-names></name></person-group><article-title>IPS-1, an adaptor triggering RIG-I- and Mda5-mediated type I interferon induction</article-title><source>Nat Immunol</source><year>2005</year><volume>6</volume><fpage>981</fpage><lpage>988</lpage><pub-id pub-id-type="pmid">16127453</pub-id><pub-id pub-id-type="doi">10.1038/ni1243</pub-id></citation></ref><ref id="B43"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Taylor</surname><given-names>MW</given-names></name><name><surname>Grosse</surname><given-names>WM</given-names></name><name><surname>Schaley</surname><given-names>JE</given-names></name><name><surname>Sanda</surname><given-names>C</given-names></name><name><surname>Wu</surname><given-names>X</given-names></name><name><surname>Chien</surname><given-names>SC</given-names></name><name><surname>Smith</surname><given-names>F</given-names></name><name><surname>Wu</surname><given-names>TG</given-names></name><name><surname>Stephens</surname><given-names>M</given-names></name><name><surname>Ferris</surname><given-names>MW</given-names></name><name><surname>McClintick</surname><given-names>JN</given-names></name><name><surname>Jerome</surname><given-names>RE</given-names></name><name><surname>Edenberg</surname><given-names>HJ</given-names></name></person-group><article-title>Global effect of PEG-IFN-alpha and ribavirin on gene expression in PBMC <italic>in vitro</italic></article-title><source>J Interferon Cytokine Res</source><year>2004</year><volume>24</volume><fpage>107</fpage><lpage>118</lpage><pub-id pub-id-type="pmid">14980075</pub-id><pub-id pub-id-type="doi">10.1089/107999004322813354</pub-id></citation></ref><ref id="B44"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Nagorsen</surname><given-names>D</given-names></name><name><surname>Deola</surname><given-names>S</given-names></name><name><surname>Smith</surname><given-names>K</given-names></name><name><surname>Wang</surname><given-names>E</given-names></name><name><surname>Monsurro</surname><given-names>V</given-names></name><name><surname>Zanovello</surname><given-names>P</given-names></name><name><surname>Marincola</surname><given-names>FM</given-names></name><name><surname>Panelli</surname><given-names>MC</given-names></name></person-group><article-title>Polarized monocyte response to cytokine stimulation</article-title><source>Genome Biol</source><year>2005</year><volume>6</volume><fpage>R15</fpage><pub-id pub-id-type="pmid">15693944</pub-id><pub-id pub-id-type="doi">10.1186/gb-2005-6-2-r15</pub-id></citation></ref><ref id="B45"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kang</surname><given-names>DC</given-names></name><name><surname>Gopalkrishnan</surname><given-names>RV</given-names></name><name><surname>Wu</surname><given-names>Q</given-names></name><name><surname>Jankowsky</surname><given-names>E</given-names></name><name><surname>Pyle</surname><given-names>AM</given-names></name><name><surname>Fisher</surname><given-names>PB</given-names></name></person-group><article-title>MDA-5 An interferon-inducible putative RNA helicase with double-stranded RNA-dependent ATPase activity and melanoma growth-suppressive properties</article-title><source>Proc Natl Acad Sci USA</source><year>2002</year><volume>99</volume><fpage>637</fpage><lpage>642</lpage><pub-id pub-id-type="pmid">11805321</pub-id><pub-id pub-id-type="doi">10.1073/pnas.022637199</pub-id></citation></ref><ref id="B46"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Gorden</surname><given-names>KB</given-names></name><name><surname>Gorski</surname><given-names>KS</given-names></name><name><surname>Gibson</surname><given-names>SJ</given-names></name><name><surname>Kedl</surname><given-names>RM</given-names></name><name><surname>Kieper</surname><given-names>WC</given-names></name><name><surname>Qiu</surname><given-names>X</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name><name><surname>Alkan</surname><given-names>SS</given-names></name><name><surname>Vasilakos</surname><given-names>JP</given-names></name></person-group><article-title>Synthetic TLR agonists reveal functional differences between human TLR7 and TLR8</article-title><source>J Immunol</source><year>2005</year><volume>174</volume><fpage>1259</fpage><lpage>1268</lpage><pub-id pub-id-type="pmid">15661881</pub-id></citation></ref><ref id="B47"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Khabar</surname><given-names>KS</given-names></name><name><surname>Al-Haj</surname><given-names>L</given-names></name><name><surname>Al-Zoghaibi</surname><given-names>F</given-names></name><name><surname>Marie</surname><given-names>M</given-names></name><name><surname>Dhalla</surname><given-names>M</given-names></name><name><surname>Polyak</surname><given-names>SJ</given-names></name><name><surname>Williams</surname><given-names>BR</given-names></name></person-group><article-title>Expressed gene clusters associated with cellular sensitivity and resistance towards anti-viral and anti-proliferative actions of interferon</article-title><source>J Mol Biol</source><year>2004</year><volume>342</volume><fpage>833</fpage><lpage>846</lpage><pub-id pub-id-type="pmid">15342240</pub-id><pub-id pub-id-type="doi">10.1016/j.jmb.2004.07.065</pub-id></citation></ref><ref id="B48"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Der</surname><given-names>SD</given-names></name><name><surname>Zhou</surname><given-names>A</given-names></name><name><surname>Williams</surname><given-names>BR</given-names></name><name><surname>Silverman</surname><given-names>RH</given-names></name></person-group><article-title>Identification of genes differentially regulated by interferon alpha, beta, or gamma using oligonucleotide arrays</article-title><source>Proc Natl Acad Sci USA</source><year>1998</year><volume>95</volume><fpage>15623</fpage><lpage>15628</lpage><pub-id pub-id-type="pmid">9861020</pub-id><pub-id pub-id-type="doi">10.1073/pnas.95.26.15623</pub-id></citation></ref><ref id="B49"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Leaman</surname><given-names>DW</given-names></name><name><surname>Chawla-Sarkar</surname><given-names>M</given-names></name><name><surname>Jacobs</surname><given-names>B</given-names></name><name><surname>Vyas</surname><given-names>K</given-names></name><name><surname>Sun</surname><given-names>Y</given-names></name><name><surname>Ozdemir</surname><given-names>A</given-names></name><name><surname>Yi</surname><given-names>T</given-names></name><name><surname>Williams</surname><given-names>BR</given-names></name><name><surname>Borden</surname><given-names>EC</given-names></name></person-group><article-title>Novel growth and death related interferon-stimulated genes (ISGs) in melanoma: Greater potency of IFN-beta compared with IFN-alpha2</article-title><source>J Interferon Cytokine Res</source><year>2003</year><volume>23</volume><fpage>745</fpage><lpage>756</lpage><pub-id pub-id-type="pmid">14769151</pub-id><pub-id pub-id-type="doi">10.1089/107999003772084860</pub-id></citation></ref><ref id="B50"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Sarkar</surname><given-names>SN</given-names></name><name><surname>Sen</surname><given-names>GC</given-names></name></person-group><article-title>Novel functions of proteins encoded by viral stress-inducible genes</article-title><source>Pharmacol Ther</source><year>2004</year><volume>103</volume><fpage>245</fpage><lpage>259</lpage><pub-id pub-id-type="pmid">15464592</pub-id><pub-id pub-id-type="doi">10.1016/j.pharmthera.2004.07.007</pub-id></citation></ref><ref id="B51"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Stroncek</surname><given-names>DF</given-names></name><name><surname>Basil</surname><given-names>C</given-names></name><name><surname>Nagorsen</surname><given-names>D</given-names></name><name><surname>Deola</surname><given-names>S</given-names></name><name><surname>Arico</surname><given-names>E</given-names></name><name><surname>Smith</surname><given-names>K</given-names></name><name><surname>Wang</surname><given-names>E</given-names></name><name><surname>Marincola</surname><given-names>FM</given-names></name><name><surname>Panelli</surname><given-names>MC</given-names></name></person-group><article-title>Delayed polarization of mononuclear phagocyte transcriptional program by type I interferon isoforms</article-title><source>J Transl Med</source><year>2005</year><volume>3</volume><fpage>24</fpage><pub-id pub-id-type="pmid">15953390</pub-id><pub-id pub-id-type="doi">10.1186/1479-5876-3-24</pub-id></citation></ref><ref id="B52"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Gordien</surname><given-names>E</given-names></name><name><surname>Rosmorduc</surname><given-names>O</given-names></name><name><surname>Peltekian</surname><given-names>C</given-names></name><name><surname>Garreau</surname><given-names>F</given-names></name><name><surname>Brechot</surname><given-names>C</given-names></name><name><surname>Kremsdorf</surname><given-names>D</given-names></name></person-group><article-title>Inhibition of hepatitis B virus replication by the interferon-inducible MxA protein</article-title><source>J Virol</source><year>2001</year><volume>75</volume><fpage>2684</fpage><lpage>2691</lpage><pub-id pub-id-type="pmid">11222692</pub-id><pub-id pub-id-type="doi">10.1128/JVI.75.6.2684-2691.2001</pub-id></citation></ref><ref id="B53"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Sheehy</surname><given-names>AM</given-names></name><name><surname>Gaddis</surname><given-names>NC</given-names></name><name><surname>Choi</surname><given-names>JD</given-names></name><name><surname>Malim</surname><given-names>MH</given-names></name></person-group><article-title>Isolation of a human gene that inhibits HIV-1 infection and is suppressed by the viral Vif protein</article-title><source>Nature</source><year>2002</year><volume>418</volume><fpage>646</fpage><lpage>650</lpage><pub-id pub-id-type="pmid">12167863</pub-id><pub-id pub-id-type="doi">10.1038/nature00939</pub-id></citation></ref><ref id="B54"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Chin</surname><given-names>KC</given-names></name><name><surname>Cresswell</surname><given-names>P</given-names></name></person-group><article-title>Viperin (cig5), an IFN-inducible antiviral protein directly induced by human cytomegalovirus</article-title><source>Proc Natl Acad Sci U S A</source><year>2001</year><volume>98</volume><fpage>15125</fpage><lpage>15130</lpage><pub-id pub-id-type="pmid">11752458</pub-id><pub-id pub-id-type="doi">10.1073/pnas.011593298</pub-id></citation></ref><ref id="B55"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Castelli</surname><given-names>JC</given-names></name><name><surname>Hassel</surname><given-names>BA</given-names></name><name><surname>Maran</surname><given-names>A</given-names></name><name><surname>Paranjape</surname><given-names>J</given-names></name><name><surname>Hewitt</surname><given-names>JA</given-names></name><name><surname>Li</surname><given-names>XL</given-names></name><name><surname>Hsu</surname><given-names>YT</given-names></name><name><surname>Silverman</surname><given-names>RH</given-names></name><name><surname>Youle</surname><given-names>RJ</given-names></name></person-group><article-title>The role of 2'-5' oligoadenylate-activated ribonuclease L in apoptosis</article-title><source>Cell Death Differ</source><year>1998</year><fpage>313</fpage><lpage>320</lpage><pub-id pub-id-type="pmid">10200477</pub-id><pub-id pub-id-type="doi">10.1038/sj.cdd.4400352</pub-id></citation></ref><ref id="B56"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>de Veer</surname><given-names>MJ</given-names></name><name><surname>Holko</surname><given-names>M</given-names></name><name><surname>Frevel</surname><given-names>M</given-names></name><name><surname>Walker</surname><given-names>E</given-names></name><name><surname>Der</surname><given-names>S</given-names></name><name><surname>Paranjape</surname><given-names>JM</given-names></name><name><surname>Silverman</surname><given-names>RH</given-names></name><name><surname>Williams</surname><given-names>BR</given-names></name></person-group><article-title>Functional classification of interferon-stimulated genes identified using microarrays</article-title><source>J Leukoc Biol</source><year>2001</year><volume>69</volume><fpage>912</fpage><lpage>920</lpage><pub-id pub-id-type="pmid">11404376</pub-id></citation></ref><ref id="B57"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Papageorgiou</surname><given-names>A</given-names></name><name><surname>Lashinger</surname><given-names>L</given-names></name><name><surname>Millikan</surname><given-names>R</given-names></name><name><surname>Grossman</surname><given-names>HB</given-names></name><name><surname>Benedict</surname><given-names>W</given-names></name><name><surname>Dinney</surname><given-names>CP</given-names></name><name><surname>McConkey</surname><given-names>DJ</given-names></name></person-group><article-title>Role of tumor necrosis factor-related apoptosis-inducing ligand in interferon-induced apoptosis in human bladder cancer cells</article-title><source>Cancer Res</source><year>2004</year><volume>64</volume><fpage>8973</fpage><lpage>8979</lpage><pub-id pub-id-type="pmid">15604261</pub-id><pub-id pub-id-type="doi">10.1158/0008-5472.CAN-04-1909</pub-id></citation></ref><ref id="B58"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Keyser</surname><given-names>J</given-names></name><name><surname>Schultz</surname><given-names>J</given-names></name><name><surname>Ladell</surname><given-names>K</given-names></name><name><surname>Elzaouk</surname><given-names>L</given-names></name><name><surname>Heinzerling</surname><given-names>L</given-names></name><name><surname>Pavlovic</surname><given-names>J</given-names></name><name><surname>Moelling</surname><given-names>K</given-names></name></person-group><article-title>IP-10-encoding plasmid DNA therapy exhibits antitumor and antimetastatic efficiency</article-title><source>Exp Dermatol</source><year>2004</year><volume>13</volume><fpage>380</fpage><pub-id pub-id-type="pmid">15186325</pub-id><pub-id pub-id-type="doi">10.1111/j.0906-6705.2004.00191.x</pub-id></citation></ref><ref id="B59"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Li</surname><given-names>S</given-names></name><name><surname>Wilkinson</surname><given-names>M</given-names></name><name><surname>Xia</surname><given-names>X</given-names></name><name><surname>David</surname><given-names>M</given-names></name><name><surname>Xu</surname><given-names>L</given-names></name><name><surname>Purkel-Sutton</surname><given-names>A</given-names></name><name><surname>Bhardwaj</surname><given-names>A</given-names></name></person-group><article-title>Induction of IFN-regulated factors and antitumoral surveillance by transfected placebo plasmid DNA</article-title><source>Mol Ther</source><year>2005</year><volume>11</volume><fpage>112</fpage><lpage>119</lpage><pub-id pub-id-type="pmid">15585412</pub-id><pub-id pub-id-type="doi">10.1016/j.ymthe.2004.09.008</pub-id></citation></ref><ref id="B60"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Helbig</surname><given-names>KJ</given-names></name><name><surname>Lau</surname><given-names>DT</given-names></name><name><surname>Semendric</surname><given-names>L</given-names></name><name><surname>Harley</surname><given-names>HA</given-names></name><name><surname>Beard</surname><given-names>MR</given-names></name></person-group><article-title>Analysis of ISG expression in chronic hepatitis C identifies viperin as a potential antiviral effector</article-title><source>Hepatology</source><year>2005</year><volume>42</volume><fpage>702</fpage><lpage>710</lpage><pub-id pub-id-type="pmid">16108059</pub-id><pub-id pub-id-type="doi">10.1002/hep.20844</pub-id></citation></ref><ref id="B61"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Takaku</surname><given-names>T</given-names></name><name><surname>Ohyashiki</surname><given-names>JH</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Ohyashiki</surname><given-names>K</given-names></name></person-group><article-title>Estimating immunoregulatory gene networks in human herpesvirus type 6-infected T cells</article-title><source>Biochem Biophys Res Commun</source><year>2005</year><volume>336</volume><fpage>469</fpage><lpage>477</lpage><pub-id pub-id-type="pmid">16140273</pub-id><pub-id pub-id-type="doi">10.1016/j.bbrc.2005.08.104</pub-id></citation></ref><ref id="B62"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kim</surname><given-names>EJ</given-names></name><name><surname>Park</surname><given-names>JI</given-names></name><name><surname>Nelkin</surname><given-names>BD</given-names></name></person-group><article-title>IFI16 is an essential mediator of growth inhibition, but not differentiation, induced by the leukemia inhibitory factor/JAK/STAT pathway in medullary thyroid carcinoma cells</article-title><source>J Biol Chem</source><year>2005</year><volume>280</volume><fpage>4913</fpage><lpage>4920</lpage><pub-id pub-id-type="pmid">15572361</pub-id><pub-id pub-id-type="doi">10.1074/jbc.M410542200</pub-id></citation></ref><ref id="B63"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Ludlow</surname><given-names>LE</given-names></name><name><surname>Johnstone</surname><given-names>RW</given-names></name><name><surname>Clarke</surname><given-names>CJ</given-names></name></person-group><article-title>The HIN-200 family: More than interferon-inducible genes?</article-title><source>Exp Cell Res</source><year>2005</year><volume>308</volume><fpage>1</fpage><lpage>17</lpage><pub-id pub-id-type="pmid">15896773</pub-id><pub-id pub-id-type="doi">10.1016/j.yexcr.2005.03.032</pub-id></citation></ref><ref id="B64"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Chen</surname><given-names>GG</given-names></name><name><surname>Lai</surname><given-names>PB</given-names></name><name><surname>Ho</surname><given-names>RL</given-names></name><name><surname>Chan</surname><given-names>PK</given-names></name><name><surname>Xu</surname><given-names>H</given-names></name><name><surname>Wong</surname><given-names>J</given-names></name><name><surname>Lau</surname><given-names>WY</given-names></name></person-group><article-title>Reduction of double-stranded RNA-activated protein kinase in hepatocellular carcinoma associated with hepatitis B virus</article-title><source>J Med Virol</source><year>2004</year><volume>73</volume><fpage>187</fpage><lpage>194</lpage><pub-id pub-id-type="pmid">15122791</pub-id><pub-id pub-id-type="doi">10.1002/jmv.20074</pub-id></citation></ref><ref id="B65"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Keefe</surname><given-names>D</given-names></name><name><surname>Shi</surname><given-names>L</given-names></name><name><surname>Feske</surname><given-names>S</given-names></name><name><surname>Massol</surname><given-names>R</given-names></name><name><surname>Navarro</surname><given-names>F</given-names></name><name><surname>Kirchhausen</surname><given-names>T</given-names></name><name><surname>Lieberman</surname><given-names>J</given-names></name></person-group><article-title>Perforin triggers a plasma membrane-repair response that facilitates CTL induction of apoptosis</article-title><source>Immunity</source><year>2005</year><volume>23</volume><fpage>249</fpage><lpage>262</lpage><pub-id pub-id-type="pmid">16169498</pub-id><pub-id pub-id-type="doi">10.1016/j.immuni.2005.08.001</pub-id></citation></ref><ref id="B66"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Wenzel</surname><given-names>J</given-names></name><name><surname>Uerlich</surname><given-names>M</given-names></name><name><surname>Haller</surname><given-names>O</given-names></name><name><surname>Bieber</surname><given-names>T</given-names></name><name><surname>Tueting</surname><given-names>T</given-names></name></person-group><article-title>Enhanced type I interferon signaling and recruitment of chemokine receptor CXCR3-expressing lymphocytes into the skin following treatment with the TLR7-agonist imiquimod</article-title><source>J Cutan Pathol</source><year>2005</year><volume>32</volume><fpage>257</fpage><lpage>262</lpage><pub-id pub-id-type="pmid">15769273</pub-id><pub-id pub-id-type="doi">10.1111/j.0303-6987.2005.00297.x</pub-id></citation></ref><ref id="B67"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Urosevic</surname><given-names>M</given-names></name><name><surname>Oberholzer</surname><given-names>PA</given-names></name><name><surname>Maier</surname><given-names>T</given-names></name><name><surname>Hafner</surname><given-names>J</given-names></name><name><surname>Laine</surname><given-names>E</given-names></name><name><surname>Slade</surname><given-names>H</given-names></name><name><surname>Benninghoff</surname><given-names>B</given-names></name><name><surname>Burg</surname><given-names>G</given-names></name><name><surname>Dummer</surname><given-names>R</given-names></name></person-group><article-title>Imiquimod treatment induces expression of opioid growth factor receptor: A novel tumor antigen induced by interferon-alpha?</article-title><source>Clin Cancer Res</source><year>2004</year><volume>10</volume><fpage>4959</fpage><lpage>4970</lpage><pub-id pub-id-type="pmid">15297396</pub-id><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-04-0193</pub-id></citation></ref><ref id="B68"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Korman</surname><given-names>N</given-names></name><name><surname>Moy</surname><given-names>R</given-names></name><name><surname>Ling</surname><given-names>M</given-names></name><name><surname>Matheson</surname><given-names>R</given-names></name><name><surname>Smith</surname><given-names>S</given-names></name><name><surname>McKane</surname><given-names>S</given-names></name><name><surname>Lee</surname><given-names>JH</given-names></name></person-group><article-title>Dosing with 5% imiquimod cream 3 times per week for the treatment of actinic keratosis: Results of two phase 3, randomized, double-blind, parallel-group, placebo-controlled trials</article-title><source>Arch Dermatol</source><year>2005</year><volume>141</volume><fpage>467</fpage><lpage>473</lpage><pub-id pub-id-type="pmid">15837864</pub-id><pub-id pub-id-type="doi">10.1001/archderm.141.4.467</pub-id></citation></ref><ref id="B69"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Urosevic</surname><given-names>M</given-names></name><name><surname>Dummer</surname><given-names>R</given-names></name><name><surname>Conrad</surname><given-names>C</given-names></name><name><surname>Beyeler</surname><given-names>M</given-names></name><name><surname>Laine</surname><given-names>E</given-names></name><name><surname>Burg</surname><given-names>G</given-names></name><name><surname>Gilliet</surname><given-names>M</given-names></name></person-group><article-title>Disease-independent skin recruitment and activation of plasmacytoid predendritic cells following imiquimod treatment</article-title><source>J Natl Cancer Inst</source><year>2005</year><volume>97</volume><fpage>1143</fpage><lpage>1153</lpage><pub-id pub-id-type="pmid">16077073</pub-id></citation></ref><ref id="B70"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Matsumoto</surname><given-names>M</given-names></name><name><surname>Tanaka</surname><given-names>T</given-names></name><name><surname>Kaisho</surname><given-names>T</given-names></name><name><surname>Sanjo</surname><given-names>H</given-names></name><name><surname>Copeland</surname><given-names>NG</given-names></name><name><surname>Gilbert</surname><given-names>DJ</given-names></name><name><surname>Jenkins</surname><given-names>NA</given-names></name><name><surname>Akira</surname><given-names>S</given-names></name></person-group><article-title>A novel LPS-inducible C-type lectin is a transcriptional target of NF-IL6 in macrophages</article-title><source>J Immunol</source><year>1999</year><volume>163</volume><fpage>5039</fpage><lpage>5048</lpage><pub-id pub-id-type="pmid">10528209</pub-id></citation></ref><ref id="B71"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Walker</surname><given-names>MG</given-names></name></person-group><article-title>Z39Ig is co-expressed with activated macrophage genes</article-title><source>Biochim Biophys Acta</source><year>2002</year><volume>1574</volume><fpage>387</fpage><lpage>390</lpage><pub-id pub-id-type="pmid">11997108</pub-id></citation></ref><ref id="B72"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Ooi</surname><given-names>T</given-names></name><name><surname>Barnetson</surname><given-names>RS</given-names></name><name><surname>Zhuang</surname><given-names>L</given-names></name><name><surname>McKane</surname><given-names>S</given-names></name><name><surname>Lee</surname><given-names>JH</given-names></name><name><surname>Slade</surname><given-names>HB</given-names></name><name><surname>Halliday</surname><given-names>GM</given-names></name></person-group><article-title>Imiquimod-induced regression of actinic keratosis is associated with infiltration by T lymphocytes and dendritic cells: a randomized controlled trial</article-title><source>Br J Dermatol</source><year>2006</year><volume>154</volume><fpage>72</fpage><lpage>78</lpage><pub-id pub-id-type="pmid">16403097</pub-id><pub-id pub-id-type="doi">10.1111/j.1365-2133.2005.06932.x</pub-id></citation></ref><ref id="B73"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Michalopoulos</surname><given-names>P</given-names></name><name><surname>Yawalkar</surname><given-names>N</given-names></name><name><surname>Bronnimann</surname><given-names>M</given-names></name><name><surname>Kappeler</surname><given-names>A</given-names></name><name><surname>Braathen</surname><given-names>LR</given-names></name></person-group><article-title>Characterization of the cellular infiltrate during successful topical treatment of lentigo maligna with imiquimod</article-title><source>Br J Dermatol</source><year>2004</year><volume>151</volume><fpage>903</fpage><lpage>906</lpage><pub-id pub-id-type="pmid">15491436</pub-id><pub-id pub-id-type="doi">10.1111/j.1365-2133.2004.06176.x</pub-id></citation></ref><ref id="B74"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Wolf</surname><given-names>IH</given-names></name><name><surname>Cerroni</surname><given-names>L</given-names></name><name><surname>Kodama</surname><given-names>K</given-names></name><name><surname>Kerl</surname><given-names>H</given-names></name></person-group><article-title>Treatment of lentigo maligna (melanoma in situ) with the immune response modifier imiquimod</article-title><source>Arch Dermatol</source><year>2005</year><volume>141</volume><fpage>510</fpage><lpage>514</lpage><pub-id pub-id-type="pmid">15837872</pub-id><pub-id pub-id-type="doi">10.1001/archderm.141.4.510</pub-id></citation></ref><ref id="B75"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Bates</surname><given-names>EE</given-names></name><name><surname>Fournier</surname><given-names>N</given-names></name><name><surname>Garcia</surname><given-names>E</given-names></name><name><surname>Valladeau</surname><given-names>J</given-names></name><name><surname>Durand</surname><given-names>I</given-names></name><name><surname>Pin</surname><given-names>JJ</given-names></name><name><surname>Zurawski</surname><given-names>SM</given-names></name><name><surname>Patel</surname><given-names>S</given-names></name><name><surname>Abrams</surname><given-names>JS</given-names></name><name><surname>Lebecque</surname><given-names>S</given-names></name><name><surname>Garrone</surname><given-names>P</given-names></name><name><surname>Saeland</surname><given-names>S</given-names></name></person-group><article-title>APCs express DCIR, a novel C-type lectin surface receptor containing an immunoreceptor tyrosine-based inhibitory motif</article-title><source>J Immunol</source><year>1999</year><volume>163</volume><fpage>1973</fpage><lpage>1983</lpage><pub-id pub-id-type="pmid">10438934</pub-id></citation></ref><ref id="B76"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Ju</surname><given-names>XS</given-names></name><name><surname>Hacker</surname><given-names>C</given-names></name><name><surname>Scherer</surname><given-names>B</given-names></name><name><surname>Redecke</surname><given-names>V</given-names></name><name><surname>Berger</surname><given-names>T</given-names></name><name><surname>Schuler</surname><given-names>G</given-names></name><name><surname>Wagner</surname><given-names>H</given-names></name><name><surname>Lipford</surname><given-names>GB</given-names></name><name><surname>Zenke</surname><given-names>M</given-names></name></person-group><article-title>Immunoglobulin-like transcripts ILT2, ILT3 and ILT7 are expressed by human dendritic cells and down-regulated following activation</article-title><source>Gene</source><year>2004</year><volume>331</volume><fpage>159</fpage><lpage>164</lpage><pub-id pub-id-type="pmid">15094202</pub-id><pub-id pub-id-type="doi">10.1016/j.gene.2004.02.018</pub-id></citation></ref><ref id="B77"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Rissoan</surname><given-names>MC</given-names></name><name><surname>Duhen</surname><given-names>T</given-names></name><name><surname>Bridon</surname><given-names>JM</given-names></name><name><surname>Bendriss-Vermare</surname><given-names>N</given-names></name><name><surname>Peronne</surname><given-names>C</given-names></name><name><surname>de Saint Vis</surname><given-names>B</given-names></name><name><surname>Briere</surname><given-names>F</given-names></name><name><surname>Bates</surname><given-names>EE</given-names></name></person-group><article-title>Subtractive hybridization reveals the expression of immunoglobulinlike transcript 7, Eph-B1, granzyme B, and 3 novel transcripts in human plasmacytoid dendritic cells</article-title><source>Blood</source><year>2002</year><volume>100</volume><fpage>3295</fpage><lpage>3303</lpage><pub-id pub-id-type="pmid">12384430</pub-id><pub-id pub-id-type="doi">10.1182/blood-2002-02-0638</pub-id></citation></ref><ref id="B78"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Palamara</surname><given-names>F</given-names></name><name><surname>Meindl</surname><given-names>S</given-names></name><name><surname>Holcmann</surname><given-names>M</given-names></name><name><surname>Luhrs</surname><given-names>P</given-names></name><name><surname>Stingl</surname><given-names>G</given-names></name><name><surname>Sibilia</surname><given-names>M</given-names></name></person-group><article-title>Identification and characterization of pDC-like cells in normal mouse skin and melanomas treated with imiquimod</article-title><source>J Immunol</source><year>2004</year><volume>173</volume><fpage>3051</fpage><lpage>3061</lpage><pub-id pub-id-type="pmid">15322165</pub-id></citation></ref><ref id="B79"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Peiser</surname><given-names>M</given-names></name><name><surname>Grutzkau</surname><given-names>A</given-names></name><name><surname>Wanner</surname><given-names>R</given-names></name><name><surname>Kolde</surname><given-names>G</given-names></name></person-group><article-title>CD1a and CD1c cell sorting yields a homogeneous population of immature human Langerhans cells</article-title><source>J Immunol Methods</source><year>2003</year><volume>279</volume><fpage>41</fpage><lpage>53</lpage><pub-id pub-id-type="pmid">12969546</pub-id><pub-id pub-id-type="doi">10.1016/S0022-1759(03)00257-6</pub-id></citation></ref><ref id="B80"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Darmochwal-Kolarz</surname><given-names>D</given-names></name><name><surname>Rolinski</surname><given-names>J</given-names></name><name><surname>Buczkowski</surname><given-names>J</given-names></name><name><surname>Tabarkiewicz</surname><given-names>J</given-names></name><name><surname>Leszczynska-Gorzelak</surname><given-names>B</given-names></name><name><surname>Zych</surname><given-names>I</given-names></name><name><surname>Oleszczuk</surname><given-names>J</given-names></name></person-group><article-title>CD1c(+) immature myeloid dendritic cells are predominant in cord blood of healthy neonates</article-title><source>Immunol Lett</source><year>2004</year><volume>91</volume><fpage>71</fpage><lpage>74</lpage><pub-id pub-id-type="pmid">14757372</pub-id><pub-id pub-id-type="doi">10.1016/j.imlet.2003.11.006</pub-id></citation></ref><ref id="B81"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Suzuki</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>B</given-names></name><name><surname>Shivji</surname><given-names>GM</given-names></name><name><surname>Toto</surname><given-names>P</given-names></name><name><surname>Amerio</surname><given-names>P</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name><name><surname>Miller</surname><given-names>RL</given-names></name><name><surname>Sauder</surname><given-names>DN</given-names></name></person-group><article-title>Imiquimod, a topical immune response modifer, induces migration of Langerhans cells</article-title><source>J Investig Dermatol</source><year>2000</year><volume>114</volume><fpage>135</fpage><lpage>141</lpage><pub-id pub-id-type="pmid">10620129</pub-id><pub-id pub-id-type="doi">10.1046/j.1523-1747.2000.00833.x</pub-id></citation></ref><ref id="B82"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Verneris</surname><given-names>MR</given-names></name><name><surname>Karami</surname><given-names>M</given-names></name><name><surname>Baker</surname><given-names>J</given-names></name><name><surname>Jayaswal</surname><given-names>A</given-names></name><name><surname>Negrin</surname><given-names>RS</given-names></name></person-group><article-title>Role of NKG2D signaling in the cytotoxicity of activated and expanded CD8+ T cells</article-title><source>Blood</source><year>2004</year><volume>103</volume><fpage>3065</fpage><lpage>3072</lpage><pub-id pub-id-type="pmid">15070686</pub-id><pub-id pub-id-type="doi">10.1182/blood-2003-06-2125</pub-id></citation></ref><ref id="B83"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Pisegna</surname><given-names>S</given-names></name><name><surname>Zingoni</surname><given-names>A</given-names></name><name><surname>Pirozzi</surname><given-names>G</given-names></name><name><surname>Cinque</surname><given-names>B</given-names></name><name><surname>Cifone</surname><given-names>MG</given-names></name><name><surname>Morrone</surname><given-names>S</given-names></name><name><surname>Piccoli</surname><given-names>M</given-names></name><name><surname>Frati</surname><given-names>L</given-names></name><name><surname>Palmieri</surname><given-names>G</given-names></name><name><surname>Santoni</surname><given-names>A</given-names></name></person-group><article-title>Src-dependent Syk activation controls CD69-mediated signaling and function on human NK cells</article-title><source>J Immunol</source><year>2002</year><volume>169</volume><fpage>68</fpage><lpage>74</lpage><pub-id pub-id-type="pmid">12077230</pub-id></citation></ref><ref id="B84"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Barber</surname><given-names>DF</given-names></name><name><surname>Faure</surname><given-names>M</given-names></name><name><surname>Long</surname><given-names>EO</given-names></name></person-group><article-title>LFA-1 contributes an early signal for NK cell cytotoxicity</article-title><source>J Immunol</source><year>2004</year><volume>173</volume><fpage>3653</fpage><lpage>3659</lpage><pub-id pub-id-type="pmid">15356110</pub-id></citation></ref><ref id="B85"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Anikeeva</surname><given-names>N</given-names></name><name><surname>Somersalo</surname><given-names>K</given-names></name><name><surname>Sims</surname><given-names>TN</given-names></name><name><surname>Thomas</surname><given-names>VK</given-names></name><name><surname>Dustin</surname><given-names>ML</given-names></name><name><surname>Sykulev</surname><given-names>Y</given-names></name></person-group><article-title>Distinct role of lymphocyte function-associated antigen-1 in mediating effective cytolytic activity by cytotoxic T lymphocytes</article-title><source>Proc Natl Acad Sci USA</source><year>2005</year><volume>102</volume><fpage>6437</fpage><lpage>6442</lpage><pub-id pub-id-type="pmid">15851656</pub-id><pub-id pub-id-type="doi">10.1073/pnas.0502467102</pub-id></citation></ref><ref id="B86"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Cella</surname><given-names>M</given-names></name><name><surname>Fujikawa</surname><given-names>K</given-names></name><name><surname>Tassi</surname><given-names>I</given-names></name><name><surname>Kim</surname><given-names>S</given-names></name><name><surname>Latinis</surname><given-names>K</given-names></name><name><surname>Nishi</surname><given-names>S</given-names></name><name><surname>Yokoyama</surname><given-names>W</given-names></name><name><surname>Colonna</surname><given-names>M</given-names></name><name><surname>Swat</surname><given-names>W</given-names></name></person-group><article-title>Differential requirements for Vav proteins in DAP10- and ITAM-mediated NK cell cytotoxicity</article-title><source>J Exp Med</source><year>2004</year><volume>200</volume><fpage>817</fpage><lpage>823</lpage><pub-id pub-id-type="pmid">15365099</pub-id><pub-id pub-id-type="doi">10.1084/jem.20031847</pub-id></citation></ref><ref id="B87"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Groh</surname><given-names>V</given-names></name><name><surname>Rhinehart</surname><given-names>R</given-names></name><name><surname>Randolph-Habecker</surname><given-names>J</given-names></name><name><surname>Topp</surname><given-names>MS</given-names></name><name><surname>Riddell</surname><given-names>SR</given-names></name><name><surname>Spies</surname><given-names>T</given-names></name></person-group><article-title>Costimulation of CD8αβ T cells by NKG2D via engagement by MIC induced on virus-infected cells</article-title><source>Nat Immunol</source><year>2001</year><volume>2</volume><fpage>255</fpage><lpage>260</lpage><pub-id pub-id-type="pmid">11224526</pub-id><pub-id pub-id-type="doi">10.1038/85321</pub-id></citation></ref><ref id="B88"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Jinushi</surname><given-names>M</given-names></name><name><surname>Takehara</surname><given-names>T</given-names></name><name><surname>Kanto</surname><given-names>T</given-names></name><name><surname>Tatsumi</surname><given-names>T</given-names></name><name><surname>Groh</surname><given-names>V</given-names></name><name><surname>Spies</surname><given-names>T</given-names></name><name><surname>Miyagi</surname><given-names>T</given-names></name><name><surname>Suzuki</surname><given-names>T</given-names></name><name><surname>Sasaki</surname><given-names>Y</given-names></name><name><surname>Hayashi</surname><given-names>NT</given-names></name></person-group><article-title>Critical role of MHC class I-related chain A and B expression on IFNAα-stimulated dendritic cells in NK cell activation: Impairment in chronic hepatitis C virus infection</article-title><source>J Immunol</source><year>2003</year><volume>170</volume><fpage>1249</fpage><lpage>56</lpage><pub-id pub-id-type="pmid">12538683</pub-id></citation></ref><ref id="B89"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Waterhouse</surname><given-names>NJ</given-names></name><name><surname>Clarke</surname><given-names>CJ</given-names></name><name><surname>Sedelies</surname><given-names>KA</given-names></name><name><surname>Teng</surname><given-names>MW</given-names></name><name><surname>Trapani</surname><given-names>JA</given-names></name></person-group><article-title>Cytotoxic lymphocytes; Instigators of dramatic target cell death</article-title><source>Biochem Pharmacol</source><year>2004</year><volume>68</volume><fpage>1033</fpage><lpage>1040</lpage><pub-id pub-id-type="pmid">15313398</pub-id><pub-id pub-id-type="doi">10.1016/j.bcp.2004.05.043</pub-id></citation></ref><ref id="B90"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Wowk</surname><given-names>ME</given-names></name><name><surname>Trapani</surname><given-names>JA</given-names></name></person-group><article-title>Cytotoxic activity of the lymphocyte toxin granzyme B</article-title><source>Microbes Infect</source><year>2004</year><volume>6</volume><fpage>752</fpage><lpage>758</lpage><pub-id pub-id-type="pmid">15207822</pub-id><pub-id pub-id-type="doi">10.1016/j.micinf.2004.03.008</pub-id></citation></ref><ref id="B91"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Medley</surname><given-names>QG</given-names></name><name><surname>Kedersha</surname><given-names>N</given-names></name><name><surname>O'Brien</surname><given-names>S</given-names></name><name><surname>Tian</surname><given-names>Q</given-names></name><name><surname>Schlossman</surname><given-names>SF</given-names></name><name><surname>Streuli</surname><given-names>M</given-names></name><name><surname>Anderson</surname><given-names>P</given-names></name></person-group><article-title>Characterization of GMP-17, a granule membrane protein that moves to the plasma membrane of natural killer cells following target cell recognition</article-title><source>Proc Natl Acad Sci U S A</source><year>1996</year><volume>93</volume><fpage>685</fpage><lpage>689</lpage><pub-id pub-id-type="pmid">8570616</pub-id><pub-id pub-id-type="doi">10.1073/pnas.93.2.685</pub-id></citation></ref><ref id="B92"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Smyth</surname><given-names>MJ</given-names></name><name><surname>Cretney</surname><given-names>E</given-names></name><name><surname>Kelly</surname><given-names>JM</given-names></name><name><surname>Westwood</surname><given-names>JA</given-names></name><name><surname>Street</surname><given-names>SE</given-names></name><name><surname>Yagita</surname><given-names>H</given-names></name><name><surname>Takeda</surname><given-names>K</given-names></name><name><surname>van Dommelen</surname><given-names>SL</given-names></name><name><surname>Degli-Esposti</surname><given-names>MA</given-names></name><name><surname>Hayakawa</surname><given-names>Y</given-names></name></person-group><article-title>Activation of NK cell cytotoxicity</article-title><source>Mol Immunol</source><year>2005</year><volume>42</volume><fpage>501</fpage><lpage>510</lpage><pub-id pub-id-type="pmid">15607806</pub-id><pub-id pub-id-type="doi">10.1016/j.molimm.2004.07.034</pub-id></citation></ref><ref id="B93"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Ashton-Rickardt</surname><given-names>PG</given-names></name></person-group><article-title>The granule pathway of programmed cell death</article-title><source>Crit Rev Immunol</source><year>2005</year><volume>25</volume><fpage>161</fpage><lpage>182</lpage><pub-id pub-id-type="pmid">16048434</pub-id><pub-id pub-id-type="doi">10.1615/CritRevImmunol.v25.i3.10</pub-id></citation></ref><ref id="B94"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Denny</surname><given-names>MF</given-names></name><name><surname>Patai</surname><given-names>B</given-names></name><name><surname>Straus</surname><given-names>DB</given-names></name></person-group><article-title>Differential T-cell antigen receptor signaling mediated by the Src family kinases Lck and Fyn</article-title><source>Mol Cell Biol</source><year>2000</year><volume>20</volume><fpage>1426</fpage><lpage>1435</lpage><pub-id pub-id-type="pmid">10648627</pub-id><pub-id pub-id-type="doi">10.1128/MCB.20.4.1426-1435.2000</pub-id></citation></ref><ref id="B95"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kuhne</surname><given-names>MR</given-names></name><name><surname>Lin</surname><given-names>J</given-names></name><name><surname>Yablonski</surname><given-names>D</given-names></name><name><surname>Mollenauer</surname><given-names>MN</given-names></name><name><surname>Ehrlich</surname><given-names>LI</given-names></name><name><surname>Huppa</surname><given-names>J</given-names></name><name><surname>Davis</surname><given-names>MM</given-names></name><name><surname>Weiss</surname><given-names>A</given-names></name></person-group><article-title>Linker for activation of T cells, zeta-associated protein-70, and Src homology 2 domain-containing leukocyte protein-76 are required for TCR-induced microtubule-organizing center polarization</article-title><source>J Immunol</source><year>2003</year><volume>171</volume><fpage>860</fpage><lpage>866</lpage><pub-id pub-id-type="pmid">12847255</pub-id></citation></ref><ref id="B96"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Wherry</surname><given-names>EJ</given-names></name><name><surname>Ahmed</surname><given-names>R</given-names></name></person-group><article-title>Memory CD8 T-cell differentiation during viral infection</article-title><source>J Virol</source><year>2004</year><volume>78</volume><fpage>5535</fpage><lpage>5545</lpage><pub-id pub-id-type="pmid">15140950</pub-id><pub-id pub-id-type="doi">10.1128/JVI.78.11.5535-5545.2004</pub-id></citation></ref><ref id="B97"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Fuller</surname><given-names>MJ</given-names></name><name><surname>Hildeman</surname><given-names>DA</given-names></name><name><surname>Sabbaj</surname><given-names>S</given-names></name><name><surname>Gaddis</surname><given-names>DE</given-names></name><name><surname>Tebo</surname><given-names>AE</given-names></name><name><surname>Shang</surname><given-names>L</given-names></name><name><surname>Goepfert</surname><given-names>PA</given-names></name><name><surname>Zajac</surname><given-names>AJ</given-names></name></person-group><article-title>Cutting edge: Emergence of CD127high functionally competent memory T cells is compromised by high viral loads and inadequate T cell help</article-title><source>J Immunol</source><year>2005</year><volume>174</volume><fpage>5926</fpage><lpage>5930</lpage><pub-id pub-id-type="pmid">15879083</pub-id></citation></ref><ref id="B98"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Rodriguez</surname><given-names>MW</given-names></name><name><surname>Paquet</surname><given-names>AC</given-names></name><name><surname>Yang</surname><given-names>YH</given-names></name><name><surname>Erle</surname><given-names>DJ</given-names></name></person-group><article-title>Differential gene expression by integrin beta 7+ and beta 7-memory T helper cells</article-title><source>BMC Immunol</source><year>2004</year><volume>5</volume><fpage>13</fpage><pub-id pub-id-type="pmid">15236665</pub-id><pub-id pub-id-type="doi">10.1186/1471-2172-5-13</pub-id></citation></ref><ref id="B99"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Smith</surname><given-names>KJ</given-names></name><name><surname>Hamza</surname><given-names>S</given-names></name><name><surname>Skelton</surname><given-names>H</given-names></name></person-group><article-title>Topical imidazoquinoline therapy of cutaneous squamous cell carcinoma polarizes lymphoid and monocyte/macrophage populations to a Th1 and M1 cytokine pattern</article-title><source>Clin Exp Dermatol</source><year>2004</year><volume>29</volume><fpage>505</fpage><lpage>512</lpage><pub-id pub-id-type="pmid">15347337</pub-id><pub-id pub-id-type="doi">10.1111/j.1365-2230.2004.01593.x</pub-id></citation></ref><ref id="B100"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Bernstein</surname><given-names>DI</given-names></name><name><surname>Harrison</surname><given-names>CJ</given-names></name><name><surname>Tomai</surname><given-names>MA</given-names></name><name><surname>Miller</surname><given-names>RL</given-names></name></person-group><article-title>Daily or weekly therapy with resiquimod (R-848) reduces genital recurrences in herpes simplex virus-infected guinea pigs during and after treatment</article-title><source>J Infect Dis</source><year>2001</year><volume>183</volume><fpage>844</fpage><lpage>849</lpage><pub-id pub-id-type="pmid">11237799</pub-id><pub-id pub-id-type="doi">10.1086/319262</pub-id></citation></ref><ref id="B101"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Stockfleth</surname><given-names>E</given-names></name><name><surname>Christophers</surname><given-names>E</given-names></name><name><surname>Benninghoff</surname><given-names>B</given-names></name><name><surname>Sterry</surname><given-names>W</given-names></name></person-group><article-title>Low incidence of new actinic keratosis after topical 5% imiquimod cream treatment: A long-term follow-up study</article-title><source>Arch Dermatol</source><year>2004</year><volume>140</volume><fpage>1542</fpage><pub-id pub-id-type="pmid">15611446</pub-id><pub-id pub-id-type="doi">10.1001/archderm.140.12.1542-a</pub-id></citation></ref><ref id="B102"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Mohty</surname><given-names>M</given-names></name><name><surname>Vialle-Castellano</surname><given-names>A</given-names></name><name><surname>Nunes</surname><given-names>JA</given-names></name><name><surname>Isnardon</surname><given-names>D</given-names></name><name><surname>Olive</surname><given-names>D</given-names></name><name><surname>Gaugler</surname><given-names>B</given-names></name></person-group><article-title>IFN-alpha skews monocyte differentiation into Toll-like receptor 7-expressing dendritic cells with potent functional activities</article-title><source>J Immunol</source><year>2003</year><volume>171</volume><fpage>3385</fpage><lpage>3393</lpage><pub-id pub-id-type="pmid">14500632</pub-id></citation></ref></ref-list></back></article>