<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Cancer</journal-id><journal-title>BMC Cancer</journal-title><issn pub-type="epub">1471-2407</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">16029516</article-id><article-id pub-id-type="pmc">1183195</article-id><article-id pub-id-type="publisher-id">1471-2407-5-83</article-id><article-id pub-id-type="doi">10.1186/1471-2407-5-83</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>Application of serum SELDI proteomic patterns in diagnosis of lung cancer</article-title></title-group><contrib-group><contrib id="A1" equal-contrib="yes" corresp="yes" contrib-type="author"><name><surname>Yang</surname><given-names>Shuan-ying</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>yangshuanying@21cn.com</email></contrib><contrib id="A2" equal-contrib="yes" contrib-type="author"><name><surname>Xiao</surname><given-names>Xue-yuan</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>xyxiao@bnu.edu.cn</email></contrib><contrib id="A3" contrib-type="author"><name><surname>Zhang</surname><given-names>Wang-gang</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>zhangwanggang2003@yahoo.com</email></contrib><contrib id="A4" contrib-type="author"><name><surname>Zhang</surname><given-names>Li-juan</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>ljlj-1203@163.com</email></contrib><contrib id="A5" contrib-type="author"><name><surname>Zhang</surname><given-names>Wei</given-names></name><xref ref-type="aff" rid="I3">3</xref><email>zw0003@sina.com</email></contrib><contrib id="A6" contrib-type="author"><name><surname>Zhou</surname><given-names>Bin</given-names></name><xref ref-type="aff" rid="I3">3</xref><email>vvhdin@pub.xaonline.com</email></contrib><contrib id="A7" contrib-type="author"><name><surname>Chen</surname><given-names>Guoan</given-names></name><xref ref-type="aff" rid="I1">1</xref><xref ref-type="aff" rid="I4">4</xref><email>guoanche@umich.edu</email></contrib><contrib id="A8" corresp="yes" contrib-type="author"><name><surname>He</surname><given-names>Da-cheng</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>dhe@bnu.edu.cn</email></contrib></contrib-group><aff id="I1"><label>1</label>Department of Respiratory Medicine, Second Hospital of Xi'an Jiaotong University, Xi'an, 710004, China</aff><aff id="I2"><label>2</label>Key Laboratory for Cell Proliferation and Regulation Biology Ministry of Education, Bejing Normal University, Bejing, 100875, China</aff><aff id="I3"><label>3</label>General Thoracic Surgery, Second Hospital of Xi'an Jiaotong University, Xi'an, 710004, China</aff><aff id="I4"><label>4</label>Department of Surgery, University of Michigan Medical School, MI, 48109, USA</aff><pub-date pub-type="collection"><year>2005</year></pub-date><pub-date pub-type="epub"><day>20</day><month>7</month><year>2005</year></pub-date><volume>5</volume><fpage>83</fpage><lpage>83</lpage><ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2407/5/83"></ext-link><history><date date-type="received"><day>17</day><month>3</month><year>2005</year></date><date date-type="accepted"><day>20</day><month>7</month><year>2005</year></date></history><copyright-statement>Copyright Â© 2005 Yang et al; licensee BioMed Central Ltd.</copyright-statement><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0"></ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></license><abstract><sec><title><offsets xml_i="4106" xml_f="4116" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="4127" xml_f="4542" txt_i="22" txt_f="437">Currently, no satisfactory biomarkers are available to screen for lung cancer. Surface-Enhanced Laser Desorption/ionization Time-of- Flight Mass Spectrometry ProteinChip system (SELDI-TOF-MS) is one of the currently used techniques to identify biomarkers for cancers. The aim of this study is to explore the application of serum SELDI proteomic patterns to distinguish lung cancer patients from healthy individuals.</offsets></p></sec><sec sec-type="methods"><title><offsets xml_i="4583" xml_f="4590" txt_i="439" txt_f="446">Methods</offsets></title><p><offsets xml_i="4601" xml_f="5438" txt_i="447" txt_f="1284">A total of 208 serum samples, including 158 lung cancer patients and 50 healthy individuals, were randomly divided into a training set (including 11 sera from patients with stages I/II lung cancer, 63 from patients with stages III/IV lung cancer and 20 from healthy controls) and a blinded test set (including 43 sera from patients with stages I/II lung cancer, 41 from patients with stages III/IV lung cancer and 30 from healthy controls). All samples were analyzed by SELDI technology. The spectra were generated on weak cation exchange (WCX2) chips, and protein peaks clustering and classification analyses were made using Ciphergen Biomarker Wizard and Biomarker Pattern software, respectively. We additionally determined Cyfra21-1 and NSE in the 208 serum samples included in this study using an electrochemiluminescent immunoassay.</offsets></p></sec><sec><title><offsets xml_i="5460" xml_f="5467" txt_i="1286" txt_f="1293">Results</offsets></title><p><offsets xml_i="5478" xml_f="6209" txt_i="1294" txt_f="2022">Five protein peaks at 11493, 6429, 8245, 5335 and 2538 Da were automatically chosen as a biomarker pattern in the training set. When the SELDI marker pattern was tested with the blinded test set, it yielded a sensitivity of 86.9%, a specificity of 80.0% and a positive predictive value of 92.4%. The sensitivities provided by Cyfra21-1 and NSE used individually or in combination were significantly lower than that of the SELDI marker pattern (P &lt; 0.005 or 0.05, respectively). Based on the results of the test set, we found that the SELDI marker pattern showed a sensitivity of 91.4% in the detection of non-small cell lung cancers (NSCLC), which was significantly higher than that in the detection of small cell lung cancers (</offsets><italic><offsets xml_i="6217" xml_f="6219" txt_i="2022" txt_f="2024">P </offsets></italic><offsets xml_i="6228" xml_f="6332" txt_i="2024" txt_f="2125">&lt; 0.05); The pattern also had a sensitivity of 79.1% in the detection of lung cancers in stages I/II.</offsets></p></sec><sec><title><offsets xml_i="6354" xml_f="6364" txt_i="2127" txt_f="2137">Conclusion</offsets></title><p><offsets xml_i="6375" xml_f="6642" txt_i="2138" txt_f="2405">These results suggest that serum SELDI protein profiling can distinguish lung cancer patients, especially NSCLC patients, from normal subjects with relatively high sensitivity and specificity, and the SELDI-TOF-MS is a potential tool for the screening of lung cancer.</offsets></p></sec></abstract></article-meta></front><body><sec><title><offsets xml_i="6704" xml_f="6714" txt_i="2414" txt_f="2424">Background</offsets></title><p><offsets xml_i="6725" xml_f="6843" txt_i="2425" txt_f="2543">Lung cancer is, at present, the most common malignancy in the world and its overall 5-year survival rate is only 14% [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="6874" xml_f="6875" txt_i="2543" txt_f="2544">1</offsets></xref><offsets xml_i="6882" xml_f="7367" txt_i="2544" txt_f="3029">]. The poor prognosis is due largely to lack of sufficient screening and early diagnostic tools to physicians. Currently in clinic the screening and early diagnosis of lung cancer relies mainly on chest X-ray, low-dose computed tomography, bronchoscopy, sputum cytology, and tumor markers including carcinoembryonic antigen (CEA), cytokeratin-19 fragments (Cyfra21-1), carbohydrate antigen 19-9 (CA19-9), squamous cell carcinoma antigen (SCCAg) and neuron-specific enolase (NSE), etc [</offsets><xref ref-type="bibr" rid="B2"><offsets xml_i="7398" xml_f="7399" txt_i="3029" txt_f="3030">2</offsets></xref><offsets xml_i="7406" xml_f="7483" txt_i="3030" txt_f="3107">]. All these methods, however, lack adequate sensitivity and/or specificity [</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="7514" xml_f="7515" txt_i="3107" txt_f="3108">3</offsets></xref><offsets xml_i="7522" xml_f="7523" txt_i="3108" txt_f="3109">-</offsets><xref ref-type="bibr" rid="B6"><offsets xml_i="7554" xml_f="7555" txt_i="3109" txt_f="3110">6</offsets></xref><offsets xml_i="7562" xml_f="7755" txt_i="3110" txt_f="3303">]. Thus, it is urgent to search for better methods which provide more valuable information for screening and early diagnosis of lung cancer. Because of the marked heterogeneity of lung cancer [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="7786" xml_f="7787" txt_i="3303" txt_f="3304">5</offsets></xref><offsets xml_i="7794" xml_f="8036" txt_i="3304" txt_f="3546">], a panel of biomarkers for screening and diagnosis would be most appropriate. Surface-Enhanced Laser Desorption /ionization Time-of-Flight Mass Spectrometry (SELDI-TOF-MS), an innovative proteomic technology introduced by Hutchens and Yip [</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="8067" xml_f="8068" txt_i="3546" txt_f="3547">7</offsets></xref><offsets xml_i="8075" xml_f="8247" txt_i="3547" txt_f="3719">], has overcome many of the limitations of two-dimensional electrophoresis and Matrix-Assisted Laser Desorption/ionization Time-of-Flight Mass Spectrometry (MALDI-TOF-MS) [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="8278" xml_f="8279" txt_i="3719" txt_f="3720">8</offsets></xref><offsets xml_i="8286" xml_f="8287" txt_i="3720" txt_f="3721">,</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="8318" xml_f="8319" txt_i="3721" txt_f="3722">9</offsets></xref><offsets xml_i="8326" xml_f="8519" txt_i="3722" txt_f="3915">]. This is a high through-put technique for analysis of complex biological specimens such as serum. It can detect multiple protein changes simultaneously with high sensitivity and specificity [</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="8551" xml_f="8553" txt_i="3915" txt_f="3917">10</offsets></xref><offsets xml_i="8560" xml_f="8561" txt_i="3917" txt_f="3918">,</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="8593" xml_f="8595" txt_i="3918" txt_f="3920">11</offsets></xref><offsets xml_i="8602" xml_f="8727" txt_i="3920" txt_f="4045">]. Recently, SELDI has been successfully used to distinguish pancreatic, ovarian and prostate cancer patients from controls [</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="8758" xml_f="8759" txt_i="4045" txt_f="4046">9</offsets></xref><offsets xml_i="8766" xml_f="8767" txt_i="4046" txt_f="4047">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="8799" xml_f="8801" txt_i="4047" txt_f="4049">12</offsets></xref><offsets xml_i="8808" xml_f="8809" txt_i="4049" txt_f="4050">,</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="8841" xml_f="8843" txt_i="4050" txt_f="4052">13</offsets></xref><offsets xml_i="8850" xml_f="8900" txt_i="4052" txt_f="4102">], and detect markers of bladder cancer in urine [</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="8932" xml_f="8934" txt_i="4102" txt_f="4104">14</offsets></xref><offsets xml_i="8941" xml_f="8943" txt_i="4104" txt_f="4106">].</offsets></p><p><offsets xml_i="8950" xml_f="9109" txt_i="4107" txt_f="4266">The aim of the current study was to investigate the application of serum SELDI protein profiling to distinguish lung cancer patients from a healthy population.</offsets></p></sec><sec sec-type="methods"><title><offsets xml_i="9150" xml_f="9157" txt_i="4268" txt_f="4275">Methods</offsets></title><sec><title><offsets xml_i="9177" xml_f="9185" txt_i="4276" txt_f="4284">Patients</offsets></title><p><offsets xml_i="9196" xml_f="10397" txt_i="4285" txt_f="5486">A total of 208 serum samples including 158 pathologically confirmed lung cancer patients and 50 healthy subjects were collected from the Department of Respiratory and Thoracic Surgery of the Second Hospital of Xi'an Jiaotong University. Informed consent was obtained from every subject prior to the study. All patients with lung cancer were found to have no evidence of other disease. The distribution of clinical stages (UICC, 1997) was as follows: 13 cases were at stage I, 41 stage II, 58 stage III, 46 stage IV. Among these patients, 68 patients suffered from squamous cell carcinomas, 53 from adenocarcinomas, 35 from small cell cancers and 2 from bronchioloalveolar carcinomas. The average age of the patients (101 males, 57 females ranging from 28 to 79 years) was 56.8. The healthy controls (31 males, 19 females ranging from 30 to72 years) came from general physical examinations, and had an average age of 54.5. The two groups were matched for age, sex and smoking history. Two milliliters of whole blood were collected during fasting and stored within one hour at 4Â°C. The blood was later centrifuged for 20 min at 4000 rpm, distributed into 100 Î¼l aliquots, and stored at -80Â°C until used.</offsets></p></sec><sec><title><offsets xml_i="10419" xml_f="10442" txt_i="5488" txt_f="5511">SELDI protein profiling</offsets></title><p><offsets xml_i="10453" xml_f="11319" txt_i="5512" txt_f="6378">Five Î¼L of 10 mM HCl was applied to a weak cation exchange (WCX2) chip and placed at room temperature for 10 min. Chips were rinsed with deionized water in a conical tube and then put into a bioprocessor and washed with binding buffer (100 mM NaAc, pH4) with gentle shaking twice for 5 min. Five Î¼L of each serum and 10 Î¼L of 9 mol/L urea were combined and vortexed on ice. 5 Î¼L of this mixture was added to 60 Î¼L of binding buffer. 50 Î¼L of the serum mixture was applied to each spot and incubated on a shaker for 60 min. Chips were washed again with binding buffer with slight shaking 3 times. 200 Î¼L of 1 mM HEPES pH7.0 was added to each well. Wells were quickly rinsed and then removed and let dry. Once dry, 0.5 Î¼L of sinapinic acid (SPA) was applied to each spot twice. The arrays were allowed to air-dry and then stored in the dark at RT until SELDI analysis.</offsets></p></sec><sec><title><offsets xml_i="11341" xml_f="11354" txt_i="6380" txt_f="6393">Data analysis</offsets></title><p><offsets xml_i="11365" xml_f="12270" txt_i="6394" txt_f="7299">Before analysis, the data were randomly divided into two sets as follows: the training set consisted of 11 patients with stages I/II lung cancer, 63 patients with stages III/IV lung cancer and 20 healthy controls. The blinded test (in which the disease status was unrevealed) set consisted of 43 patients with stages I/II lung cancer, 41 patients with stages III/IV lung cancer and 30 healthy controls. The chips were placed in the Protein Biological system II-C mass spectrometer reader (Ciphergen Biosystems, Inc.) and TOF spectra were generated by averaging 128 laser shots with an intensity of 215 and a detector sensitivity of 9. The optimization range was from 3,000 to 50,000 Da, and a maximum of 200,000 Da. External calibration of the instrument was performed using the All-in-one peptide molecular mass standard (Ciphergen Biosystems, Inc.). We achieved a mass accuracy of 0.1% with this system.</offsets></p></sec><sec><title><offsets xml_i="12292" xml_f="12306" txt_i="7301" txt_f="7315">Peak detection</offsets></title><p><offsets xml_i="12317" xml_f="12643" txt_i="7316" txt_f="7642">Peak detection using Ciphergen Biomarker Wizard software 3.0.2 identified an average of 72 peaks/spectrum. Of the 72 peaks, 64 common peaks or clusters were generated from the training set. Eighteen of these proteins were found to have statistically differential expression levels between lung cancer and normal control sera (</offsets><italic><offsets xml_i="12651" xml_f="12653" txt_i="7642" txt_f="7644">P </offsets></italic><offsets xml_i="12662" xml_f="12669" txt_i="7644" txt_f="7648">&lt; 10</offsets><sup><offsets xml_i="12674" xml_f="12676" txt_i="7648" txt_f="7650">-4</offsets></sup><offsets xml_i="12682" xml_f="13027" txt_i="7650" txt_f="7995">). Peak detections involved baseline subtraction, mass accuracy calibration, and automatic peak detection. The settings used for our work were as follows: for peak detection the signal-to-noise ratio was 3, minimum peak threshold was 10%; for cluster completion, the cluster mass was 0.5% and the signal-to-noise ratio for the second pass was 1.</offsets></p></sec><sec><title><offsets xml_i="13049" xml_f="13077" txt_i="7997" txt_f="8025">Decision tree classification</offsets></title><p><offsets xml_i="13088" xml_f="13662" txt_i="8026" txt_f="8600">Construction of the decision tree classification algorithm was performed by Ciphergen Biomarker Pattern software version 5.0. Classification tree, selected Gini, split the data into two nodes using one rule at a time in the form of peak intensity. The splitting decisions in this case were based on the normalized intensity levels of peaks from SELDI protein expression profile. The process of splitting was continued until terminal nodes were created. After V-fold cross validation 50, the accuracy of each classification tree was then challenged with the blinded test set.</offsets></p></sec><sec><title><offsets xml_i="13684" xml_f="13720" txt_i="8602" txt_f="8638">Detection of serum Cyfra21-1 and NSE</offsets></title><p><offsets xml_i="13731" xml_f="14016" txt_i="8639" txt_f="8924">The two markers, Crfra21-1 and NSE, were measured in the 208 sera included in this study using an electrochemiluminescent immunoassay (ECLIA, Elecsys 2010 system, Roche Diagnostics, Switzerland). The cutoff values for Crfra21-1 and NSE, recommended by the manufacturers, were 3.3 ng ml</offsets><sup><offsets xml_i="14021" xml_f="14023" txt_i="8924" txt_f="8926">-1</offsets></sup><offsets xml_i="14029" xml_f="14043" txt_i="8926" txt_f="8940">and 16.3 ng ml</offsets><sup><offsets xml_i="14048" xml_f="14050" txt_i="8940" txt_f="8942">-1</offsets></sup><offsets xml_i="14056" xml_f="14071" txt_i="8942" txt_f="8957">, respectively.</offsets></p></sec><sec><title><offsets xml_i="14093" xml_f="14113" txt_i="8959" txt_f="8979">Statistical analysis</offsets></title><p><offsets xml_i="14124" xml_f="14213" txt_i="8980" txt_f="9069">Comparison of relative peak intensity levels between groups was made using the Student's </offsets><italic><offsets xml_i="14221" xml_f="14223" txt_i="9069" txt_f="9071">t </offsets></italic><offsets xml_i="14232" xml_f="14254" txt_i="9071" txt_f="9093">test and in all cases </offsets><italic><offsets xml_i="14262" xml_f="14264" txt_i="9093" txt_f="9095">P </offsets></italic><offsets xml_i="14273" xml_f="14280" txt_i="9095" txt_f="9099">&lt; 10</offsets><sup><offsets xml_i="14285" xml_f="14288" txt_i="9099" txt_f="9102">-4 </offsets></sup><offsets xml_i="14294" xml_f="14396" txt_i="9102" txt_f="9204">was considered statistically significant. Comparison of rates between groups was conducted using the Ï</offsets><sup><offsets xml_i="14401" xml_f="14403" txt_i="9204" txt_f="9206">2 </offsets></sup><offsets xml_i="14409" xml_f="14418" txt_i="9206" txt_f="9215">test and </offsets><italic><offsets xml_i="14426" xml_f="14428" txt_i="9215" txt_f="9217">P </offsets></italic><offsets xml_i="14437" xml_f="14488" txt_i="9217" txt_f="9265">&lt; 0.05 was regarded as a significant difference.</offsets></p></sec></sec><sec><title><offsets xml_i="14516" xml_f="14523" txt_i="9268" txt_f="9275">Results</offsets></title><sec><title><offsets xml_i="14543" xml_f="14558" txt_i="9276" txt_f="9291">Reproducibility</offsets></title><p><offsets xml_i="14569" xml_f="14926" txt_i="9292" txt_f="9649">The reproducibility of each SELDI proteinchip assay was determined by SELDI profiling of 10 aliquots of pooled normal serum. The average coefficient of variance (CV) based on 10 pooled normal human sera for intensities of 22 randomly chosen peaks was less than 20%. Little variation with day-to-day sampling and instrumentation or chip variations was found.</offsets></p></sec><sec><title><offsets xml_i="14948" xml_f="15008" txt_i="9651" txt_f="9711">Serum SELDI profiles of lung cancers versus healthy controls</offsets></title><p><offsets xml_i="15019" xml_f="15638" txt_i="9712" txt_f="10331">Using Ciphergen Biomarker pattern software to analyze the data derived from Ciphergen Biomarker wizard software, approximately 64 peaks per spectrum identified in the training set were determined with masses ranging from 3â30 kDa. We found that no single peak could adequately discriminate lung cancer sera from normal sera. Using all 64 peaks, a decision tree classification algorithm was built and five protein peaks at 11493, 8245, 5335, 6429 and 2538 Da were automatically selected as splitters. The 11,493 Da peak was used as the root node in the classification tree to divide the 94 samples into two groups (Fig. </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="15668" xml_f="15669" txt_i="10331" txt_f="10332">1</offsets></xref><offsets xml_i="15676" xml_f="16183" txt_i="10332" txt_f="10836">): the left node (node 2) included cases with peak intensity &lt; 2.018. The right node (node 6) contained the remaining with peak intensity = 2.018. The cases in each branch node were then reclassified at the next layer following the same process with 6429, 5335, 2538 and 8245 Da as splitters. This splitting process stops if terminal nodes for further splitting have no gain. Finally, all 94 cases in the training set were classified in the 7 terminal nodes, and a classification tree was obtained (Fig. </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="16213" xml_f="16214" txt_i="10836" txt_f="10837">1</offsets></xref><offsets xml_i="16221" xml_f="16311" txt_i="10837" txt_f="10927">). The tree correctly classified 95.9% of the lung cancer sera in the training set (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="16343" xml_f="16344" txt_i="10927" txt_f="10928">1</offsets></xref><offsets xml_i="16351" xml_f="16542" txt_i="10928" txt_f="11119">). The validity of this classification tree algorithm was then challenged with the test set and a total of 80.0% of controls and 86.9% of lung cancer samples were correctly identified (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="16574" xml_f="16575" txt_i="11119" txt_f="11120">1</offsets></xref><offsets xml_i="16582" xml_f="16765" txt_i="11120" txt_f="11303">). Based on the results of the test set we calculated the sensitivity of the SELDI marker pattern in the detection of lung cancers with different stages and pathological types (Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="16797" xml_f="16798" txt_i="11303" txt_f="11304">2</offsets></xref><offsets xml_i="16805" xml_f="16857" txt_i="11304" txt_f="11356">). The peaks at 11493 and 5335 Da are shown in Fig. </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="16887" xml_f="16888" txt_i="11356" txt_f="11357">2</offsets></xref><offsets xml_i="16895" xml_f="17111" txt_i="11357" txt_f="11573">. Aside from the 11493 Da peak, any of the other 5 peaks could have been used as the first node in the classification trees in the same way as 11493 Da, but their performance scores were inferior to the 11493Da peak.</offsets></p></sec><sec><title><offsets xml_i="17133" xml_f="17180" txt_i="11575" txt_f="11622">Discriminatory power of serum Cyfra21-1 and NSE</offsets></title><p><offsets xml_i="17191" xml_f="17197" txt_i="11623" txt_f="11629">Table </offsets><xref ref-type="table" rid="T3"><offsets xml_i="17229" xml_f="17230" txt_i="11629" txt_f="11630">3</offsets></xref><offsets xml_i="17237" xml_f="17469" txt_i="11630" txt_f="11862"> provides the results of sensitivities and specificities of Cyfra21-1 and NSE used individually and combined. We compared the diagnostic capacities of the SELDI marker pattern with Cyfra21-1 and NSE individually and combined (Table </offsets><xref ref-type="table" rid="T3"><offsets xml_i="17501" xml_f="17502" txt_i="11862" txt_f="11863">3</offsets></xref><offsets xml_i="17509" xml_f="17511" txt_i="11863" txt_f="11865">).</offsets></p></sec></sec><sec><title><offsets xml_i="17539" xml_f="17549" txt_i="11868" txt_f="11878">Discussion</offsets></title><p><offsets xml_i="17560" xml_f="18017" txt_i="11879" txt_f="12336">Currently, there are no satisfactory screening and early diagnostic strategies for lung cancer. SELDI is a high through-put technique used to generate protein expression profiles which, in combination with bioinformatics tools to extract information for biomarker discovery, has been essential in identifying novel protein biomarkers. Indeed, application of this technology has shown great potential for the early detection of ovarian and prostate cancers [</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="18049" xml_f="18051" txt_i="12336" txt_f="12338">10</offsets></xref><offsets xml_i="18058" xml_f="18059" txt_i="12338" txt_f="12339">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="18091" xml_f="18093" txt_i="12339" txt_f="12341">12</offsets></xref><offsets xml_i="18100" xml_f="18102" txt_i="12341" txt_f="12343">].</offsets></p><p><offsets xml_i="18109" xml_f="18160" txt_i="12344" txt_f="12395">Proteomic studies of lung cancer are still scarce [</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="18192" xml_f="18194" txt_i="12395" txt_f="12397">15</offsets></xref><offsets xml_i="18201" xml_f="18227" txt_i="12397" txt_f="12423">]. Recently, Xiao, et al [</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="18259" xml_f="18261" txt_i="12423" txt_f="12425">16</offsets></xref><offsets xml_i="18268" xml_f="19515" txt_i="12425" txt_f="13672">] reported that a proteomic panel consisting of three protein peaks yielded a sensitivity of 93.3% and specificity of 96.7% in distinguishing lung cancer patients from healthy controls. This study was, however, based on only 45 tumor samples. In the present study, we examined 158 serum samples from lung cancer patients and 50 from healthy individuals using the SELDI technique with the WCX2 proteinchip. The classification tree was constructed to distinguish lung cancer cases from healthy individuals using 5 protein peaks at 11493, 6429, 8245, 5335 and 2538 Da as a marker pattern. When the model was tested with the blinded test set, it yielded a sensitivity of 86.9%, specificity of 80.0%, and positive predictive value of 92.4% (73/79). For comparison, Cyfra21-1 and NSE were measured using ECLIA in our study. Although there is no statistical differences between the specificities of Cyfra21-1, NSE and the SELDI marker pattern, the sensitivity achieved by Cyfra21-1, NSE individually or in combination were significantly lower than that of the SELDI pattern. These results indicate that the SELDI pattern is distinctly superior to Cyfra21-1 and NSE individually or combined in distinguishing lung cancer patients from healthy individuals.</offsets></p><p><offsets xml_i="19522" xml_f="20059" txt_i="13673" txt_f="14210">Based on the results of the blinded test set, we found that the sensitivity of the SELDI marker pattern for NSCLCs was significantly higher than for SCLCs, indicating that the pattern may be more effective in discriminating NSCLC patients from healthy controls than SCLC patients. Similarly, the pattern also had a sensitivity of 79.1% in the detection of lung cancers with stages I/II, suggesting that the pattern might be better for early detection of lung cancer than any other single or panel of biomarkers currently used in clinic [</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="20091" xml_f="20093" txt_i="14210" txt_f="14212">17</offsets></xref><offsets xml_i="20100" xml_f="20101" txt_i="14212" txt_f="14213">,</offsets><xref ref-type="bibr" rid="B18"><offsets xml_i="20133" xml_f="20135" txt_i="14213" txt_f="14215">18</offsets></xref><offsets xml_i="20142" xml_f="20144" txt_i="14215" txt_f="14217">].</offsets></p><p><offsets xml_i="20151" xml_f="20561" txt_i="14218" txt_f="14628">To develop a broad biomarker panel for screening a diverse, high-risk population, both NSCLC and SCLC patients were chosen for our study. Due to the relatively fewer healthy control samples and the subgroup of patients with SCLC, our results require more samples to broaden and improve its diagnostic value. Furthermore, the five proteins included in the SELDI marker pattern will be identified by MALDI-MS-MS.</offsets></p></sec><sec><title><offsets xml_i="20583" xml_f="20593" txt_i="14630" txt_f="14640">Conclusion</offsets></title><p><offsets xml_i="20604" xml_f="20860" txt_i="14641" txt_f="14897">We have found that serum SELDI protein profiling can distinguish lung cancer patients, especially NSCLC patients, from healthy controls with relatively high sensitivity and specificity. The SELDI-TOF-MS is a potential tool for the screening of lung cancer.</offsets></p></sec><sec><title><offsets xml_i="20882" xml_f="20901" txt_i="14899" txt_f="14918">Competing interests</offsets></title><p><offsets xml_i="20912" xml_f="20972" txt_i="14919" txt_f="14979">The author(s) declare that they have no competing interests.</offsets></p></sec><sec><title><offsets xml_i="20994" xml_f="21016" txt_i="14981" txt_f="15003">Authors' contributions</offsets></title><p><offsets xml_i="21027" xml_f="21559" txt_i="15004" txt_f="15536">SY was responsible for the conception and design of this study, providing samples and clinical data, drafting and revising the article. XX contributed to the design of this study, performed statistical analysis and interpretation of the data. LZ provided technical support and some experiments. WZ and BZ provided in part study materials and medical aspects of the work. GC has been involved in discussion and revising this article. DH contributed to the conception of this study. All authors read and approved the final manuscript.</offsets></p></sec><sec><title><offsets xml_i="21581" xml_f="21604" txt_i="15538" txt_f="15561">Pre-publication history</offsets></title><p><offsets xml_i="21615" xml_f="21679" txt_i="15562" txt_f="15626">The pre-publication history for this paper can be accessed here:</offsets></p><p><ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2407/5/83/prepub"></ext-link></p></sec></body><back><ack><sec><title>Acknowledgements</title><p>This work was supported by grants from the Shaanxi Science and Technology Development Program [No.2004K13-G3 (3)]</p></sec></ack><ref-list><ref id="B1"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Spira</surname><given-names>A</given-names></name><name><surname>Ettinger</surname><given-names>DS</given-names></name></person-group><article-title>Multidisciplinary management of lung cancer</article-title><source>N Engl J Med</source><year>2004</year><volume>350</volume><fpage>379</fpage><lpage>392</lpage><pub-id pub-id-type="pmid">14736930</pub-id><pub-id pub-id-type="doi">10.1056/NEJMra035536</pub-id></citation></ref><ref id="B2"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Stieber</surname><given-names>P</given-names></name><name><surname>Aronsson</surname><given-names>AC</given-names></name><name><surname>Bialk</surname><given-names>P</given-names></name></person-group><article-title>Tumor markers in lung cancer: EGTM recommendations</article-title><source>Anticancer Res</source><year>1999</year><volume>19</volume><fpage>2817</fpage><lpage>2819</lpage><pub-id pub-id-type="pmid">10470248</pub-id></citation></ref><ref id="B3"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Swensen</surname><given-names>SJ</given-names></name><name><surname>Jett</surname><given-names>JR</given-names></name><name><surname>Hartman</surname><given-names>TE</given-names></name><name><surname>Midthun</surname><given-names>DE</given-names></name><name><surname>Sloan</surname><given-names>JA</given-names></name><name><surname>Sykes</surname><given-names>AM</given-names></name><name><surname>Aughenbaugh</surname><given-names>GL</given-names></name><name><surname>Clemens</surname><given-names>MA</given-names></name></person-group><article-title>Lung cancer screening with CT: Mayo clinic experience</article-title><source>Radiology</source><year>2003</year><volume>226</volume><fpage>756</fpage><lpage>761</lpage><pub-id pub-id-type="pmid">12601181</pub-id></citation></ref><ref id="B4"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Kulpa</surname><given-names>J</given-names></name><name><surname>Wojcik</surname><given-names>E</given-names></name><name><surname>Reinfuss</surname><given-names>M</given-names></name><name><surname>Kolodziejski</surname><given-names>L</given-names></name></person-group><article-title>Carcinoembryonic antigen, squamous cell carcinoma antigen, CYFRA21-1, and neuro-specific enolase in squamous cell lung cancer patients</article-title><source>Clin Chem</source><year>2002</year><volume>48</volume><fpage>1931</fpage><lpage>1937</lpage><pub-id pub-id-type="pmid">12406978</pub-id></citation></ref><ref id="B5"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Zhong</surname><given-names>L</given-names></name><name><surname>Peng</surname><given-names>X</given-names></name><name><surname>Hidalgo</surname><given-names>GE</given-names></name><name><surname>Doherty</surname><given-names>DE</given-names></name><name><surname>Stromberg</surname><given-names>AJ</given-names></name><name><surname>Hirschowitz</surname><given-names>EA</given-names></name></person-group><article-title>Identification of circulating antibodies to tumor-associated proteins for combined use as markers of non-small cell lung cancer</article-title><source>Proteomics</source><year>2004</year><volume>4</volume><fpage>1216</fpage><lpage>1225</lpage><pub-id pub-id-type="pmid">15049001</pub-id><pub-id pub-id-type="doi">10.1002/pmic.200200679</pub-id></citation></ref><ref id="B6"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Lam</surname><given-names>S</given-names></name><name><surname>Kennedy</surname><given-names>T</given-names></name><name><surname>Unger</surname><given-names>M</given-names></name><name><surname>Miller</surname><given-names>YE</given-names></name><name><surname>Gelmont</surname><given-names>D</given-names></name><name><surname>Rusch</surname><given-names>V</given-names></name><name><surname>Gipe</surname><given-names>B</given-names></name><name><surname>Howard</surname><given-names>D</given-names></name><name><surname>LeRiche</surname><given-names>JC</given-names></name><name><surname>Coldman</surname><given-names>A</given-names></name><name><surname>Gazdar</surname><given-names>AF</given-names></name></person-group><article-title>Localization of bronchial intraepithelial neoplastic lesions by fluorescence bronchoscopy</article-title><source>Chest</source><year>1998</year><volume>113</volume><fpage>696</fpage><lpage>702</lpage><pub-id pub-id-type="pmid">9515845</pub-id></citation></ref><ref id="B7"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Hutchen</surname><given-names>TW</given-names></name><name><surname>Yip</surname><given-names>TT</given-names></name></person-group><article-title>New desorption strategies for the mass spectrometric analysis of macromolecules</article-title><source>Rapid Commun Mass Spectrom</source><year>1993</year><volume>7</volume><fpage>576</fpage><lpage>580</lpage><pub-id pub-id-type="doi">10.1002/rcm.1290070703</pub-id></citation></ref><ref id="B8"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Wadsworth</surname><given-names>JT</given-names></name><name><surname>Somers</surname><given-names>KD</given-names></name><name><surname>Cazares</surname><given-names>LH</given-names></name><name><surname>Malik</surname><given-names>G</given-names></name><name><surname>Adam</surname><given-names>BL</given-names></name><name><surname>Stack</surname><given-names>BC</given-names><suffix>Jr</suffix></name><name><surname>Wright</surname><given-names>GL</given-names><suffix>Jr</suffix></name><name><surname>Semmes</surname><given-names>OJ</given-names></name></person-group><article-title>Serum protein profiles to identify head and neck cancer</article-title><source>Clin Cancer Res</source><year>2004</year><volume>10</volume><fpage>1625</fpage><lpage>1632</lpage><pub-id pub-id-type="pmid">15014013</pub-id></citation></ref><ref id="B9"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Rosty</surname><given-names>C</given-names></name><name><surname>Christa</surname><given-names>L</given-names></name><name><surname>Kuzdzal</surname><given-names>S</given-names></name><name><surname>Baldwin</surname><given-names>WM</given-names></name><name><surname>Zahurak</surname><given-names>ML</given-names></name><name><surname>Carnot</surname><given-names>F</given-names></name><name><surname>Chan</surname><given-names>DW</given-names></name><name><surname>Canto</surname><given-names>M</given-names></name><name><surname>Lillemoe</surname><given-names>KD</given-names></name><name><surname>Cameron</surname><given-names>JL</given-names></name><name><surname>Yeo</surname><given-names>CJ</given-names></name><name><surname>Hruban</surname><given-names>RH</given-names></name><name><surname>Goggins</surname><given-names>M</given-names></name></person-group><article-title>Identification of hepatocarcinoma-intestine-pancreas/pancreatitis â associated protein I as a biomarker for pancreatic ductal adenocarcinoma by protein biochip technology</article-title><source>Cancer Res</source><year>2002</year><volume>62</volume><fpage>1868</fpage><lpage>1875</lpage><pub-id pub-id-type="pmid">11912167</pub-id></citation></ref><ref id="B10"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Cazares</surname><given-names>LH</given-names></name><name><surname>Adam</surname><given-names>BL</given-names></name><name><surname>Ward</surname><given-names>MD</given-names></name><name><surname>Nasim</surname><given-names>S</given-names></name><name><surname>Schellhammer</surname><given-names>PF</given-names></name><name><surname>Semmes</surname><given-names>OJ</given-names></name><name><surname>Wright</surname><given-names>GL</given-names><suffix>Jr</suffix></name></person-group><article-title>Normal benign, preneoplastic, and malignant prostate cells have distinct protein expression profiles resolved by Surface Enhanced Laser Desorption/Ionization Mass Spectrometry</article-title><source>Clin Cancer Res</source><year>2002</year><volume>8</volume><fpage>2541</fpage><lpage>2552</lpage><pub-id pub-id-type="pmid">12171882</pub-id></citation></ref><ref id="B11"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Koopmann</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>White</surname><given-names>N</given-names></name><name><surname>Rosenzweig</surname><given-names>J</given-names></name><name><surname>Fedarko</surname><given-names>N</given-names></name><name><surname>Jagannath</surname><given-names>S</given-names></name><name><surname>Canto</surname><given-names>MI</given-names></name><name><surname>Yeo</surname><given-names>CJ</given-names></name><name><surname>Chan</surname><given-names>DW</given-names></name><name><surname>Goggins</surname><given-names>M</given-names></name></person-group><article-title>Serum diagnosis of pancreatic adenocarcinoma using Surface Enhanced Laser Desorption/Ionization Mass Spectrometry</article-title><source>Clin Cancer Res</source><year>2004</year><volume>10</volume><fpage>860</fpage><lpage>868</lpage><pub-id pub-id-type="pmid">14871961</pub-id></citation></ref><ref id="B12"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Petricoin</surname><given-names>EF</given-names></name><name><surname>Ardekani</surname><given-names>AM</given-names></name><name><surname>Hitt</surname><given-names>BA</given-names></name><name><surname>Levine</surname><given-names>PJ</given-names></name><name><surname>Fusaro</surname><given-names>VA</given-names></name><name><surname>Steinberg</surname><given-names>SM</given-names></name><name><surname>Mills</surname><given-names>GB</given-names></name><name><surname>Simone</surname><given-names>C</given-names></name><name><surname>Fishman</surname><given-names>DA</given-names></name><name><surname>Kohn</surname><given-names>EC</given-names></name><name><surname>Liotta</surname><given-names>LA</given-names></name></person-group><article-title>Use of proteomic patterns in serum to identify ovarian cancer</article-title><source>Lancet</source><year>2002</year><volume>359</volume><fpage>572</fpage><lpage>577</lpage><pub-id pub-id-type="pmid">11867112</pub-id><pub-id pub-id-type="doi">10.1016/S0140-6736(02)07746-2</pub-id></citation></ref><ref id="B13"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Adam</surname><given-names>BL</given-names></name><name><surname>Qu</surname><given-names>Y</given-names></name><name><surname>Davis</surname><given-names>JW</given-names></name><name><surname>Ward</surname><given-names>MD</given-names></name><name><surname>Clements</surname><given-names>MA</given-names></name><name><surname>Cazares</surname><given-names>LH</given-names></name><name><surname>Semmes</surname><given-names>OJ</given-names></name><name><surname>Schellhammer</surname><given-names>PF</given-names></name><name><surname>Yasui</surname><given-names>Y</given-names></name><name><surname>Feng</surname><given-names>Z</given-names></name><name><surname>Wright</surname><given-names>GL</given-names><suffix>Jr</suffix></name></person-group><article-title>Serum protein fingerprinting coupled with a pattern-matching algorithm distinguishes prostate cancer from benign prostate hyperplasia and healthy men</article-title><source>Cancer Res</source><year>2002</year><volume>62</volume><fpage>3609</fpage><lpage>3614</lpage><pub-id pub-id-type="pmid">12097261</pub-id></citation></ref><ref id="B14"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Vlahou</surname><given-names>A</given-names></name><name><surname>Schellhammer</surname><given-names>PF</given-names></name><name><surname>Mendrinos</surname><given-names>S</given-names></name><name><surname>Patel</surname><given-names>K</given-names></name><name><surname>Kondylis</surname><given-names>FI</given-names></name><name><surname>Gong</surname><given-names>L</given-names></name><name><surname>Nasim</surname><given-names>S</given-names></name><name><surname>Wright</surname><given-names>GL</given-names><suffix>Jr</suffix></name></person-group><article-title>Development of a novel proteomic approach for the detection of transitional cell carcinoma of the bladder in urine</article-title><source>Am J Pathol</source><year>2001</year><volume>158</volume><fpage>1491</fpage><lpage>1502</lpage><pub-id pub-id-type="pmid">11290567</pub-id></citation></ref><ref id="B15"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Alfonso</surname><given-names>P</given-names></name><name><surname>Catala</surname><given-names>M</given-names></name><name><surname>Rico-Morales</surname><given-names>ML</given-names></name><name><surname>Durante-Rodriguez</surname><given-names>G</given-names></name><name><surname>Moro-Rodriguez</surname><given-names>E</given-names></name><name><surname>Fernandez-Garcia</surname><given-names>H</given-names></name><name><surname>Escribano</surname><given-names>JM</given-names></name><name><surname>Alvarez-Fernandez</surname><given-names>E</given-names></name><name><surname>Garcia-Poblete</surname><given-names>E</given-names></name></person-group><article-title>Proteomic analysis of lung biopsies: Differential protein expression profile between peritumoral and tumoral tissue</article-title><source>Proteomics</source><year>2004</year><volume>4</volume><fpage>442</fpage><lpage>447</lpage><pub-id pub-id-type="pmid">14760715</pub-id><pub-id pub-id-type="doi">10.1002/pmic.200300647</pub-id></citation></ref><ref id="B16"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Xiao</surname><given-names>X</given-names></name><name><surname>Liu</surname><given-names>D</given-names></name><name><surname>Tang</surname><given-names>Y</given-names></name><name><surname>Guo</surname><given-names>F</given-names></name><name><surname>Xia</surname><given-names>L</given-names></name><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>He</surname><given-names>D</given-names></name></person-group><article-title>Development of proteomic patterns for detecting lung cancer</article-title><source>Dis Markers</source><year>2003</year><volume>19</volume><fpage>33</fpage><lpage>39</lpage><pub-id pub-id-type="pmid">14757945</pub-id></citation></ref><ref id="B17"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Luo</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>M</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name></person-group><article-title>Clinical significance in diagnosing lung cancer with the combined determination of serum tumor markers</article-title><source>J tumor Marker Oncology</source><year>2004</year><volume>19</volume><fpage>90</fpage><pub-id pub-id-type="doi">10.1159/000074964</pub-id></citation></ref><ref id="B18"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Gao</surname><given-names>Q</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Omoli</surname><given-names>H</given-names></name></person-group><article-title>Clinical value of a cancer marker cytokeratin 19 for diagnosis of lung cancer</article-title><source>J Chin Med Univ</source><year>1999</year><volume>28</volume><fpage>293</fpage><lpage>294</lpage></citation></ref></ref-list><sec sec-type="display-objects"><title>Figures and Tables</title><fig position="float" id="F1"><label>Figure 1</label><caption><p><bold>Classification of lung cancer <italic>vs</italic>. normal samples by the decision tree algorithm. </bold>The left branch node after the first layer is the cases of peak intensity under 2.018, the right one is over or equal to 2.018. The cutoff points for 8245, 6429 and 2538 Da were 1.574, 49.64 and 13.01, respectively. The cutoff points of mass 5335 Da were 0.288 (left) and 2.163 (right). N represents the number of samples. M represents the molecular weight.</p></caption><graphic xlink:href="1471-2407-5-83-1"></graphic></fig><fig position="float" id="F2"><label>Figure 2</label><caption><p><bold>Differential expressions of the SELDI peaks at 11493 (group A) and 5335Da (group B) in the comparisons of lung cancer and healthy control sera. </bold>a-c: squamous cell carcinomas; d-f: adenocarcinomas; g-i: small cell carcinomas; j-l, healthy controls. X-axis was molecular weight of peak; Y-axis was intensity of peak.</p></caption><graphic xlink:href="1471-2407-5-83-2"></graphic></fig><table-wrap position="float" id="T1"><label>Table 1</label><caption><p>Classification tree analysis of the lung cancer training and blinded test sets.</p></caption><table frame="hsides" rules="groups"><thead><tr><td align="center">Sets</td><td align="center">Groups</td><td align="center">Percentage correct</td><td align="center">Percentage misclassified</td></tr></thead><tbody><tr><td align="center">Training set</td><td align="center">Normal (n = 20)</td><td align="center">90.0 % (18/20)</td><td align="center">10.0% (2/20)</td></tr><tr><td></td><td align="center">Cancer (74)</td><td align="center">95.9% (71/74)</td><td align="center">4.1% (3/74)</td></tr><tr><td align="center">Blinded test set</td><td align="center">Normal (30)</td><td align="center">80.0% (24/30)</td><td align="center">20.0% (6/30)</td></tr><tr><td></td><td align="center">Cancer (84)</td><td align="center">86.9% (73/84)</td><td align="center">13.1% (11/84)</td></tr></tbody></table></table-wrap><table-wrap position="float" id="T2"><label>Table 2</label><caption><p>The sensitivity of the SELDI marker pattern in lung cancer patients with different stages and pathological types.</p></caption><table frame="hsides" rules="groups"><thead><tr><td align="left">Variables</td><td align="left">n</td><td align="left">Correct cases</td><td align="left">Error cases</td><td align="left">Sensitivity (%)</td><td align="left"><italic>P </italic>value</td></tr></thead><tbody><tr><td align="left">Clinical stages</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align="left">I/II</td><td align="left">43</td><td align="left">34</td><td align="left">9</td><td align="left">79.1</td><td></td></tr><tr><td align="left">III/IV</td><td align="left">41</td><td align="left">39</td><td align="left">2</td><td align="left">95.1</td><td align="left">&lt;0.05</td></tr><tr><td align="left">Pathological types</td><td></td><td></td><td></td><td></td><td></td></tr><tr><td align="left">NSCLC</td><td align="left">70</td><td align="left">64</td><td align="left">6</td><td align="left">91.4</td><td></td></tr><tr><td align="left">SCLC</td><td align="left">14</td><td align="left">9</td><td align="left">5</td><td align="left">64.3</td><td align="left">&lt;0.05</td></tr></tbody></table><table-wrap-foot><p>NSCLC: non-small cell lung cancer; SCLC: small cell lung cancer.</p></table-wrap-foot></table-wrap><table-wrap position="float" id="T3"><label>Table 3</label><caption><p>The sensitivities and specificities of the SELDI marker pattern, Cyfra21-1 and NSE.</p></caption><table frame="hsides" rules="groups"><thead><tr><td align="left">Items</td><td align="left">Pattern</td><td align="left">Cyfra21-1</td><td align="left">NSE</td><td align="left">Combination</td></tr></thead><tbody><tr><td align="left">Blinded test set</td><td></td><td></td><td></td><td></td></tr><tr><td align="left">Sensitivity (%)</td><td align="left">86.9 (73/84)</td><td align="left">44.0(37/84)<sup>a</sup></td><td align="left">36.9(31/84)<sup>a</sup></td><td align="left">63.1(53/84)<sup>a</sup></td></tr><tr><td align="left">Specificity (%)</td><td align="left">80.0 (24/30)</td><td align="left">70.0(21/30)</td><td align="left">73.3(22/30)</td><td align="left">53.3(16/30) <sup>b</sup></td></tr><tr><td align="left">Whole set</td><td></td><td></td><td></td><td></td></tr><tr><td align="left">Sensitivity (%)</td><td></td><td align="left">46.2 (73/158)</td><td align="left">35.4 (56/158)</td><td align="left">62.0 (98/158)</td></tr><tr><td align="left">Specificity (%)</td><td></td><td align="left">72.0 (36/50)</td><td align="left">74.0 (37/50)</td><td align="left">56.0 (28/50)</td></tr></tbody></table><table-wrap-foot><p>Pattern: SELDI marker pattern; Combination: combined use of Cyfra21-1 and NSE. <sup>a </sup>P &lt; 0.005 compared with the SELDI marker pattern; <sup>b </sup>P &lt; 0.05 compared with the SELDI marker pattern.</p></table-wrap-foot></table-wrap></sec></back></article>