<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Microbiol</journal-id><journal-title>BMC Microbiology</journal-title><issn pub-type="epub">1471-2180</issn><publisher><publisher-name>BioMed Central</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">18507832</article-id><article-id pub-id-type="pmc">2427034</article-id><article-id pub-id-type="publisher-id">1471-2180-8-81</article-id><article-id pub-id-type="doi">10.1186/1471-2180-8-81</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>Prevalence of A2143G mutation of <italic>H. pylori-23S rRNA </italic>in Chinese subjects with and without clarithromycin use history</article-title></title-group><contrib-group><contrib id="A1" contrib-type="author"><name><surname>Liu</surname><given-names>Zhuoqi</given-names></name><xref ref-type="aff" rid="I1">1</xref><xref ref-type="aff" rid="I2">2</xref><email>liuzhuoqi2406@163.com</email></contrib><contrib id="A2" contrib-type="author"><name><surname>Shen</surname><given-names>Jing</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>shenjing69@sina.com</email></contrib><contrib id="A3" contrib-type="author"><name><surname>Zhang</surname><given-names>Lian</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>zhanglmail@yahoo.com</email></contrib><contrib id="A4" contrib-type="author"><name><surname>Shen</surname><given-names>Lin</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>lin100@medmail.com.cn</email></contrib><contrib id="A5" contrib-type="author"><name><surname>Li</surname><given-names>Qiang</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>woodyqlee@yahoo.com.cn</email></contrib><contrib id="A6" contrib-type="author"><name><surname>Zhang</surname><given-names>Baozhen</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>zbz94@yahoo.com.cn</email></contrib><contrib id="A7" contrib-type="author"><name><surname>Zhou</surname><given-names>Jing</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>jane72@163.com</email></contrib><contrib id="A8" contrib-type="author"><name><surname>Gu</surname><given-names>Liankun</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>glkun@tom.com</email></contrib><contrib id="A9" contrib-type="author"><name><surname>Feng</surname><given-names>Guoshuang</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>glxfgsh@163.com</email></contrib><contrib id="A10" contrib-type="author"><name><surname>Ma</surname><given-names>Junling</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>13806491812@17288.com</email></contrib><contrib id="A11" corresp="yes" contrib-type="author"><name><surname>You</surname><given-names>Wei-Cheng</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>weichengyou@yahoo.com</email></contrib><contrib id="A12" corresp="yes" contrib-type="author"><name><surname>Deng</surname><given-names>Dajun</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>dengdajun@bjmu.edu.cn</email></contrib></contrib-group><aff id="I1"><label>1</label>Peking University School of Oncology and Beijing Institute for Cancer Research and Beijing Cancer Hospital, Hai-Dian District, Beijing, 100036, PR China</aff><aff id="I2"><label>2</label>Nanchang University Medical College, Nanchang, 330006, PR China</aff><pub-date pub-type="collection"><year>2008</year></pub-date><pub-date pub-type="epub"><day>28</day><month>5</month><year>2008</year></pub-date><volume>8</volume><fpage>81</fpage><lpage>81</lpage><ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2180/8/81"></ext-link><history><date date-type="received"><day>3</day><month>9</month><year>2007</year></date><date date-type="accepted"><day>28</day><month>5</month><year>2008</year></date></history><permissions><copyright-statement>Copyright © 2008 Liu et al; licensee BioMed Central Ltd.</copyright-statement><copyright-year>2008</copyright-year><copyright-holder>Liu et al; licensee BioMed Central Ltd.</copyright-holder><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0"></ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></license></permissions><abstract><sec><title><offsets xml_i="4725" xml_f="4735" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="4746" xml_f="4765" txt_i="22" txt_f="41">A2143G mutation of </offsets><italic><offsets xml_i="4773" xml_f="4782" txt_i="41" txt_f="50">23S rRNA </offsets></italic><offsets xml_i="4791" xml_f="4799" txt_i="50" txt_f="58">gene of </offsets><italic><offsets xml_i="4807" xml_f="4817" txt_i="58" txt_f="68">H. pylori </offsets></italic><offsets xml_i="4826" xml_f="4953" txt_i="68" txt_f="195">results in clarithromycin (CLR) resistance. To investigate the prevalence of the CLR resistance-related A2143G mutation of the </offsets><italic><offsets xml_i="4961" xml_f="4970" txt_i="195" txt_f="204">H. pylori</offsets></italic><offsets xml_i="4979" xml_f="4989" txt_i="204" txt_f="214">-specific </offsets><italic><offsets xml_i="4997" xml_f="5006" txt_i="214" txt_f="223">23S rRNA </offsets></italic><offsets xml_i="5015" xml_f="5357" txt_i="223" txt_f="565">gene in Chinese subjects with and without CLR use history, 307 subjects received the treatment with amoxicillin and omeprazole (OA) and 310 subjects received a placebo in 1995, and 153 subjects received a triple therapy with OA and CLR (OAC) in 2000. DNA was extracted from fasting gastric juice at the end of the intervention trial in 2003. </offsets><italic><offsets xml_i="5365" xml_f="5375" txt_i="565" txt_f="575">H. pylori </offsets></italic><offsets xml_i="5384" xml_f="5412" txt_i="575" txt_f="603">infection was determined by </offsets><italic><offsets xml_i="5420" xml_f="5429" txt_i="603" txt_f="612">H. pylori</offsets></italic><offsets xml_i="5438" xml_f="5448" txt_i="612" txt_f="622">-specific </offsets><italic><offsets xml_i="5456" xml_f="5465" txt_i="622" txt_f="631">23S rRNA </offsets></italic><offsets xml_i="5474" xml_f="5489" txt_i="631" txt_f="646">PCR, ELISA, and</offsets><sup><offsets xml_i="5494" xml_f="5496" txt_i="646" txt_f="648">13</offsets></sup><offsets xml_i="5502" xml_f="5546" txt_i="648" txt_f="692">C-urea breath test assays. Mutations of the </offsets><italic><offsets xml_i="5554" xml_f="5563" txt_i="692" txt_f="701">23S rRNA </offsets></italic><offsets xml_i="5572" xml_f="5606" txt_i="701" txt_f="735">gene were detected by RFLP assays.</offsets></p></sec><sec><title><offsets xml_i="5628" xml_f="5635" txt_i="737" txt_f="744">Results</offsets></title><p><offsets xml_i="5646" xml_f="5662" txt_i="745" txt_f="761">The presence of </offsets><italic><offsets xml_i="5670" xml_f="5679" txt_i="761" txt_f="770">23S rRNA </offsets></italic><offsets xml_i="5688" xml_f="5695" txt_i="770" txt_f="777">due to </offsets><italic><offsets xml_i="5703" xml_f="5713" txt_i="777" txt_f="787">H. pylori </offsets></italic><offsets xml_i="5722" xml_f="5892" txt_i="787" txt_f="957">infection in the OA group remained lower than that in the placebo group 7.3 yrs after OA-therapy [51.1% (157/307) vs. 83.9% (260/310), p = 0.0000]. In the OAC group, the </offsets><italic><offsets xml_i="5900" xml_f="5909" txt_i="957" txt_f="966">23S rRNA </offsets></italic><offsets xml_i="5918" xml_f="6018" txt_i="966" txt_f="1066">detection rate was 26.8% (41/153) three yrs after OAC-treatment. The A2143G mutation rate among the </offsets><italic><offsets xml_i="6026" xml_f="6034" txt_i="1066" txt_f="1074">23S rRNA</offsets></italic><offsets xml_i="6043" xml_f="6395" txt_i="1074" txt_f="1426">-positive subjects in the OAC group [31.7% (13/41)] was significantly higher than that in the OA group [10.2% (16/157)] and the placebo group [13.8% (36/260)]. The frequency of the AAGGG → CTTCA (2222–2226) and AACC → GAAG (2081–2084) sequence alterations in the OAC group was also significantly higher than those in the OA group and the placebo group.</offsets></p></sec><sec><title><offsets xml_i="6417" xml_f="6427" txt_i="1428" txt_f="1438">Conclusion</offsets></title><p><offsets xml_i="6438" xml_f="6583" txt_i="1439" txt_f="1584">Primary prevalence of the A2143G mutation was 10~14% among Chinese population without history of CLR therapy. Administration of CLR to eliminate </offsets><italic><offsets xml_i="6591" xml_f="6601" txt_i="1584" txt_f="1594">H. pylori </offsets></italic><offsets xml_i="6610" xml_f="6708" txt_i="1594" txt_f="1692">infection increased the prevalence of the A2143G mutation in Chinese subjects (32%) significantly.</offsets></p></sec></abstract></article-meta></front><body><sec><title><offsets xml_i="6770" xml_f="6780" txt_i="1701" txt_f="1711">Background</offsets></title><p><italic><offsets xml_i="6799" xml_f="6831" txt_i="1712" txt_f="1744">Helicobacter pylori (H. pylori) </offsets></italic><offsets xml_i="6840" xml_f="6899" txt_i="1744" txt_f="1803">infection occurs worldwide. The prevalence of infection by </offsets><italic><offsets xml_i="6907" xml_f="6917" txt_i="1803" txt_f="1813">H. pylori </offsets></italic><offsets xml_i="6926" xml_f="7073" txt_i="1813" txt_f="1960">varies greatly among countries, with over a 70% infection rate in many developing countries compared to 20% to 50% in most industrialized nations. </offsets><italic><offsets xml_i="7081" xml_f="7091" txt_i="1960" txt_f="1970">H. pylori </offsets></italic><offsets xml_i="7100" xml_f="7202" txt_i="1970" txt_f="2072">infection can lead to gastritis, duodenal and gastric ulcer, gastric carcinoma, and gastric lymphoma [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="7233" xml_f="7234" txt_i="2072" txt_f="2073">1</offsets></xref><offsets xml_i="7241" xml_f="7242" txt_i="2073" txt_f="2074">-</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="7273" xml_f="7274" txt_i="2074" txt_f="2075">3</offsets></xref><offsets xml_i="7281" xml_f="7351" txt_i="2075" txt_f="2145">]. Gastric cancer risks can be reduced by up to 60% by eradication of </offsets><italic><offsets xml_i="7359" xml_f="7369" txt_i="2145" txt_f="2155">H. pylori </offsets></italic><offsets xml_i="7378" xml_f="7488" txt_i="2155" txt_f="2265">infection through primary prevention, thus affecting approximately 800,000 gastric cancer patients worldwide [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="7519" xml_f="7520" txt_i="2265" txt_f="2266">4</offsets></xref><offsets xml_i="7527" xml_f="7528" txt_i="2266" txt_f="2267">,</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="7559" xml_f="7560" txt_i="2267" txt_f="2268">5</offsets></xref><offsets xml_i="7567" xml_f="7569" txt_i="2268" txt_f="2270">].</offsets></p><p><offsets xml_i="7576" xml_f="7680" txt_i="2271" txt_f="2375">Currently, a seven-day, triple-drug regimen has been recommended as one of the first-line therapies for </offsets><italic><offsets xml_i="7688" xml_f="7698" txt_i="2375" txt_f="2385">H. pylori </offsets></italic><offsets xml_i="7707" xml_f="7719" txt_i="2385" txt_f="2397">management [</offsets><xref ref-type="bibr" rid="B6"><offsets xml_i="7750" xml_f="7751" txt_i="2397" txt_f="2398">6</offsets></xref><offsets xml_i="7758" xml_f="7759" txt_i="2398" txt_f="2399">,</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="7790" xml_f="7791" txt_i="2399" txt_f="2400">7</offsets></xref><offsets xml_i="7798" xml_f="8031" txt_i="2400" txt_f="2633">]. This treatment includes omeprazole (a proton-pump inhibitor), clarithromycin (CLR), and amoxicillin. However, this therapy is being investigated because of increased eradication failures due to the prevalence of CLR-resistant (CLR</offsets><sup><offsets xml_i="8036" xml_f="8037" txt_i="2633" txt_f="2634">r</offsets></sup><offsets xml_i="8043" xml_f="8045" txt_i="2634" txt_f="2636">) </offsets><italic><offsets xml_i="8053" xml_f="8063" txt_i="2636" txt_f="2646">H. pylori </offsets></italic><offsets xml_i="8072" xml_f="8130" txt_i="2646" txt_f="2704">infections. Many studies have shown that between 0–50% of </offsets><italic><offsets xml_i="8138" xml_f="8148" txt_i="2704" txt_f="2714">H. pylori </offsets></italic><offsets xml_i="8157" xml_f="8174" txt_i="2714" txt_f="2731">isolates were CLR</offsets><sup><offsets xml_i="8179" xml_f="8180" txt_i="2731" txt_f="2732">r</offsets></sup><offsets xml_i="8186" xml_f="8286" txt_i="2732" txt_f="2832">, which leads to a need for long term assessment of the efficacy of CLR in the triple-drug regimen [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="8317" xml_f="8318" txt_i="2832" txt_f="2833">8</offsets></xref><offsets xml_i="8325" xml_f="8326" txt_i="2833" txt_f="2834">,</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="8357" xml_f="8358" txt_i="2834" txt_f="2835">9</offsets></xref><offsets xml_i="8365" xml_f="8476" txt_i="2835" txt_f="2946">]. It is well known that the abuse of macrolide antibiotics including CLR might lead to CLR resistant forms of </offsets><italic><offsets xml_i="8484" xml_f="8493" txt_i="2946" txt_f="2955">H. pylori</offsets></italic><offsets xml_i="8502" xml_f="8550" txt_i="2955" txt_f="3003">. Whether an optimal therapy containing CLR for </offsets><italic><offsets xml_i="8558" xml_f="8568" txt_i="3003" txt_f="3013">H. pylori </offsets></italic><offsets xml_i="8577" xml_f="8630" txt_i="3013" txt_f="3066">eradication would contribute to the prevalence of CLR</offsets><sup><offsets xml_i="8635" xml_f="8637" txt_i="3066" txt_f="3068">r </offsets></sup><italic><offsets xml_i="8651" xml_f="8661" txt_i="3068" txt_f="3078">H. pylori </offsets></italic><offsets xml_i="8670" xml_f="8735" txt_i="3078" txt_f="3143">infection was not well investigated among the Chinese population.</offsets></p><p><offsets xml_i="8742" xml_f="8934" txt_i="3144" txt_f="3336">From 1995 to 2003, a randomized, double blind and placebo-controlled intervention trial was conducted in a high-risk population for gastric cancer in Linqu County, Shandong Province of China [</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="8966" xml_f="8968" txt_i="3336" txt_f="3338">10</offsets></xref><offsets xml_i="8975" xml_f="9009" txt_i="3338" txt_f="3372">]. One of the three treatments of </offsets><italic><offsets xml_i="9017" xml_f="9027" txt_i="3372" txt_f="3382">H. pylori </offsets></italic><offsets xml_i="9036" xml_f="9219" txt_i="3382" txt_f="3565">infection in this study was a dual-drug therapy with omeprazole and amoxicillin (OA), since the triple-drug regimen was not available in 1995. This study focused on subjects who were </offsets><italic><offsets xml_i="9227" xml_f="9237" txt_i="3565" txt_f="3575">H. pylori </offsets></italic><offsets xml_i="9246" xml_f="9362" txt_i="3575" txt_f="3691">seropositive at baseline. In 2000, a triple therapy consisting of OA and CLR (OAC) was applied to subjects who were </offsets><italic><offsets xml_i="9370" xml_f="9380" txt_i="3691" txt_f="3701">H. pylori </offsets></italic><offsets xml_i="9389" xml_f="9443" txt_i="3701" txt_f="3755">seronegative at baseline and subsequently found to be </offsets><sup><offsets xml_i="9448" xml_f="9450" txt_i="3755" txt_f="3757">13</offsets></sup><offsets xml_i="9456" xml_f="9476" txt_i="3757" txt_f="3777">C-urea breath test (</offsets><sup><offsets xml_i="9481" xml_f="9483" txt_i="3777" txt_f="3779">13</offsets></sup><offsets xml_i="9489" xml_f="9644" txt_i="3779" txt_f="3934">C-UBT)-positive, according to instruction of the Ethic Committee of the Beijing Institute for Cancer Research and the U.S. National Cancer Institute (NCI).</offsets></p><p><offsets xml_i="9651" xml_f="9654" txt_i="3935" txt_f="3938">In </offsets><italic><offsets xml_i="9662" xml_f="9672" txt_i="3938" txt_f="3948">H. pylori </offsets></italic><offsets xml_i="9681" xml_f="9786" txt_i="3948" txt_f="4053">infection, resistance to CLR is mostly due to the presence of A2143G and A2142G/C point mutations of the </offsets><italic><offsets xml_i="9794" xml_f="9803" txt_i="4053" txt_f="4062">23S rRNA </offsets></italic><offsets xml_i="9812" xml_f="9818" txt_i="4062" txt_f="4068">gene [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="9849" xml_f="9850" txt_i="4068" txt_f="4069">8</offsets></xref><offsets xml_i="9857" xml_f="9858" txt_i="4069" txt_f="4070">,</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="9890" xml_f="9892" txt_i="4070" txt_f="4072">11</offsets></xref><offsets xml_i="9899" xml_f="9900" txt_i="4072" txt_f="4073">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="9932" xml_f="9934" txt_i="4073" txt_f="4075">12</offsets></xref><offsets xml_i="9941" xml_f="10047" txt_i="4075" txt_f="4181">]. We recently reported that the A2143G mutation was a late event in the development of CLR resistance of </offsets><italic><offsets xml_i="10055" xml_f="10071" txt_i="4181" txt_f="4197">H. pylori 26695 </offsets></italic><offsets xml_i="10080" xml_f="10081" txt_i="4197" txt_f="4198">[</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="10113" xml_f="10115" txt_i="4198" txt_f="4200">13</offsets></xref><offsets xml_i="10122" xml_f="10225" txt_i="4200" txt_f="4303">]. The global average mutation frequency for the A2143G and A2142G mutations causing CLR resistance of </offsets><italic><offsets xml_i="10233" xml_f="10243" txt_i="4303" txt_f="4313">H. pylori </offsets></italic><offsets xml_i="10252" xml_f="10288" txt_i="4313" txt_f="4349">were 69.8% and 11.7%, respectively [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="10319" xml_f="10320" txt_i="4349" txt_f="4350">8</offsets></xref><offsets xml_i="10327" xml_f="10405" txt_i="4350" txt_f="4428">]. Herein, we report the prevalence of the A2143G and A2142G mutations of the </offsets><italic><offsets xml_i="10413" xml_f="10422" txt_i="4428" txt_f="4437">H. pylori</offsets></italic><offsets xml_i="10431" xml_f="10441" txt_i="4437" txt_f="4447">-specific </offsets><italic><offsets xml_i="10449" xml_f="10458" txt_i="4447" txt_f="4456">23S rRNA </offsets></italic><offsets xml_i="10467" xml_f="10556" txt_i="4456" txt_f="4545">gene among these subjects treated with regimens with or without CLR after eradication of </offsets><italic><offsets xml_i="10564" xml_f="10574" txt_i="4545" txt_f="4555">H. pylori </offsets></italic><offsets xml_i="10583" xml_f="10593" txt_i="4555" txt_f="4565">infection.</offsets></p></sec><sec><title><offsets xml_i="10615" xml_f="10622" txt_i="4567" txt_f="4574">Results</offsets></title><p><offsets xml_i="10633" xml_f="10781" txt_i="4575" txt_f="4723">Genomic DNA was extracted from 770 of pre-fixed fasting gastric juice samples as described in the Method section. To determine both the presence of </offsets><italic><offsets xml_i="10789" xml_f="10799" txt_i="4723" txt_f="4733">H. pylori </offsets></italic><offsets xml_i="10808" xml_f="10839" txt_i="4733" txt_f="4764">infection and prevalence of CLR</offsets><sup><offsets xml_i="10844" xml_f="10845" txt_i="4764" txt_f="4765">r</offsets></sup><offsets xml_i="10851" xml_f="10855" txt_i="4765" txt_f="4769">, a </offsets><italic><offsets xml_i="10863" xml_f="10872" txt_i="4769" txt_f="4778">H. pylori</offsets></italic><offsets xml_i="10881" xml_f="10891" txt_i="4778" txt_f="4788">-specific </offsets><italic><offsets xml_i="10899" xml_f="10908" txt_i="4788" txt_f="4797">23S rRNA </offsets></italic><offsets xml_i="10917" xml_f="10942" txt_i="4797" txt_f="4822">PCR assay was developed. </offsets><italic><offsets xml_i="10950" xml_f="10960" txt_i="4822" txt_f="4832">H. pylori </offsets></italic><offsets xml_i="10969" xml_f="11073" txt_i="4832" txt_f="4936">infection was detected in 59.5% of analyzed subjects by the PCR assay as compared to 68.1% and 71.0% by </offsets><sup><offsets xml_i="11078" xml_f="11080" txt_i="4936" txt_f="4938">13</offsets></sup><offsets xml_i="11086" xml_f="11123" txt_i="4938" txt_f="4975">C-UBT and ELISA, respectively (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="11155" xml_f="11156" txt_i="4975" txt_f="4976">1</offsets></xref><offsets xml_i="11163" xml_f="11193" txt_i="4976" txt_f="5006">). If the combined results of </offsets><sup><offsets xml_i="11198" xml_f="11200" txt_i="5006" txt_f="5008">13</offsets></sup><offsets xml_i="11206" xml_f="11276" txt_i="5008" txt_f="5078">C-UBT and ELISA assays were used as the gold standard [both positive (</offsets><italic><offsets xml_i="11284" xml_f="11294" txt_i="5078" txt_f="5088">H. pylori </offsets></italic><offsets xml_i="11303" xml_f="11462" txt_i="5088" txt_f="5247">infection) or both negative (not infected)], the PCR assay's sensitivity and specificity was 83.4% and 75.5%, respectively. Its Youdens' index was 0.59 (Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="11494" xml_f="11495" txt_i="5247" txt_f="5248">2</offsets></xref><offsets xml_i="11502" xml_f="11504" txt_i="5248" txt_f="5250">).</offsets></p><table-wrap position="float" id="T1"><label><offsets xml_i="11552" xml_f="11559" txt_i="5251" txt_f="5258">Table 1</offsets></label><caption><p><offsets xml_i="11579" xml_f="11604" txt_i="5258" txt_f="5283">Comparisons of sex, age, </offsets><italic><offsets xml_i="11612" xml_f="11622" txt_i="5283" txt_f="5293">H. pylori </offsets></italic><offsets xml_i="11631" xml_f="11663" txt_i="5293" txt_f="5325">infection, and mutations of the </offsets><italic><offsets xml_i="11671" xml_f="11680" txt_i="5325" txt_f="5334">23S rRNA </offsets></italic><offsets xml_i="11689" xml_f="11743" txt_i="5334" txt_f="5388">gene between subjects with and without CLR use history</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><td align="left"><offsets xml_i="11822" xml_f="11828" txt_i="5389" txt_f="5395">Groups</offsets></td><td></td><td align="left"><offsets xml_i="11859" xml_f="11866" txt_i="5397" txt_f="5404">Placebo</offsets></td><td align="left"><offsets xml_i="11888" xml_f="11890" txt_i="5405" txt_f="5407">OA</offsets></td><td align="left"><offsets xml_i="11912" xml_f="11915" txt_i="5408" txt_f="5411">OAC</offsets></td><td align="left"><offsets xml_i="11937" xml_f="11944" txt_i="5412" txt_f="5419">(Total)</offsets></td></tr></thead><tbody><tr><td align="left"><offsets xml_i="11990" xml_f="12005" txt_i="5420" txt_f="5435">CLR use history</offsets></td><td></td><td align="left"><offsets xml_i="12036" xml_f="12037" txt_i="5437" txt_f="5438">-</offsets></td><td align="left"><offsets xml_i="12059" xml_f="12060" txt_i="5439" txt_f="5440">-</offsets></td><td align="left"><offsets xml_i="12082" xml_f="12083" txt_i="5441" txt_f="5442">+</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="12123" xml_f="12131" txt_i="5444" txt_f="5452">Case No.</offsets></td><td></td><td align="left"><offsets xml_i="12162" xml_f="12165" txt_i="5454" txt_f="5457">310</offsets></td><td align="left"><offsets xml_i="12187" xml_f="12190" txt_i="5458" txt_f="5461">307</offsets></td><td align="left"><offsets xml_i="12212" xml_f="12215" txt_i="5462" txt_f="5465">153</offsets></td><td align="left"><offsets xml_i="12237" xml_f="12240" txt_i="5466" txt_f="5469">770</offsets></td></tr><tr><td align="left"><offsets xml_i="12271" xml_f="12286" txt_i="5470" txt_f="5485">Sex ratio (M:F)</offsets></td><td></td><td align="left"><offsets xml_i="12317" xml_f="12323" txt_i="5487" txt_f="5493">1:1.00</offsets></td><td align="left"><offsets xml_i="12345" xml_f="12351" txt_i="5494" txt_f="5500">1:1.13</offsets></td><td align="left"><offsets xml_i="12373" xml_f="12379" txt_i="5501" txt_f="5507">1:0.82</offsets></td><td align="left"><offsets xml_i="12401" xml_f="12407" txt_i="5508" txt_f="5514">1:1.01</offsets></td></tr><tr><td align="left"><offsets xml_i="12438" xml_f="12459" txt_i="5515" txt_f="5536">Age (yrs) (Mean ± SD)</offsets></td><td></td><td align="left"><offsets xml_i="12490" xml_f="12500" txt_i="5538" txt_f="5548">45.3 ± 8.4</offsets></td><td align="left"><offsets xml_i="12522" xml_f="12532" txt_i="5549" txt_f="5559">45.0 ± 8.4</offsets></td><td align="left"><offsets xml_i="12554" xml_f="12564" txt_i="5560" txt_f="5570">45.7 ± 8.6</offsets></td><td align="left"><offsets xml_i="12586" xml_f="12596" txt_i="5571" txt_f="5581">45.2 ± 8.5</offsets></td></tr><tr><td align="left"><italic><offsets xml_i="12635" xml_f="12644" txt_i="5582" txt_f="5591">H. pylori</offsets></italic><offsets xml_i="12653" xml_f="12679" txt_i="5591" txt_f="5617">-positive rate in 2003 (%)</offsets></td><td align="left"><sup><offsets xml_i="12706" xml_f="12708" txt_i="5618" txt_f="5620">13</offsets></sup><offsets xml_i="12714" xml_f="12719" txt_i="5620" txt_f="5625">C-UBT</offsets></td><td align="left"><offsets xml_i="12741" xml_f="12755" txt_i="5626" txt_f="5640">90.6 (280/309)</offsets></td><td align="left"><offsets xml_i="12777" xml_f="12781" txt_i="5641" txt_f="5645">54.1</offsets><sup><offsets xml_i="12786" xml_f="12788" txt_i="5645" txt_f="5647">1 </offsets></sup><offsets xml_i="12794" xml_f="12803" txt_i="5647" txt_f="5656">(166/307)</offsets></td><td align="left"><offsets xml_i="12825" xml_f="12838" txt_i="5657" txt_f="5670">53.3 (81/152)</offsets></td><td align="left"><offsets xml_i="12860" xml_f="12874" txt_i="5671" txt_f="5685">68.1 (527/768)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="12914" xml_f="12919" txt_i="5687" txt_f="5692">ELISA</offsets></td><td align="left"><offsets xml_i="12941" xml_f="12955" txt_i="5693" txt_f="5707">92.5 (283/306)</offsets></td><td align="left"><offsets xml_i="12977" xml_f="12981" txt_i="5708" txt_f="5712">62.1</offsets><sup><offsets xml_i="12986" xml_f="12988" txt_i="5712" txt_f="5714">1 </offsets></sup><offsets xml_i="12994" xml_f="13003" txt_i="5714" txt_f="5723">(185/298)</offsets></td><td align="left"><offsets xml_i="13025" xml_f="13029" txt_i="5724" txt_f="5728">44.1</offsets><sup><offsets xml_i="13034" xml_f="13036" txt_i="5728" txt_f="5730">2 </offsets></sup><offsets xml_i="13042" xml_f="13050" txt_i="5730" txt_f="5738">(64/145)</offsets></td><td align="left"><offsets xml_i="13072" xml_f="13086" txt_i="5739" txt_f="5753">71.0 (532/749)</offsets></td></tr><tr><td></td><td align="left"><italic><offsets xml_i="13134" xml_f="13142" txt_i="5755" txt_f="5763">23S rRNA</offsets></italic></td><td align="left"><offsets xml_i="13173" xml_f="13187" txt_i="5764" txt_f="5778">83.9 (260/310)</offsets></td><td align="left"><offsets xml_i="13209" xml_f="13213" txt_i="5779" txt_f="5783">51.1</offsets><sup><offsets xml_i="13218" xml_f="13220" txt_i="5783" txt_f="5785">1 </offsets></sup><offsets xml_i="13226" xml_f="13235" txt_i="5785" txt_f="5794">(157/307)</offsets></td><td align="left"><offsets xml_i="13257" xml_f="13261" txt_i="5795" txt_f="5799">26.8</offsets><sup><offsets xml_i="13266" xml_f="13268" txt_i="5799" txt_f="5801">2 </offsets></sup><offsets xml_i="13274" xml_f="13282" txt_i="5801" txt_f="5809">(41/153)</offsets></td><td align="left"><offsets xml_i="13304" xml_f="13318" txt_i="5810" txt_f="5824">59.5 (458/770)</offsets></td></tr><tr><td align="left"><offsets xml_i="13349" xml_f="13378" txt_i="5825" txt_f="5854">Proportion of 2143G-positive </offsets><italic><offsets xml_i="13386" xml_f="13396" txt_i="5854" txt_f="5864">H. pylori </offsets></italic><offsets xml_i="13405" xml_f="13408" txt_i="5864" txt_f="5867">(%)</offsets></td><td align="left"><offsets xml_i="13430" xml_f="13439" txt_i="5868" txt_f="5877">Total CLR</offsets><sup><offsets xml_i="13444" xml_f="13445" txt_i="5877" txt_f="5878">r</offsets></sup></td><td align="left"><offsets xml_i="13473" xml_f="13486" txt_i="5879" txt_f="5892">13.8 (36/260)</offsets></td><td align="left"><offsets xml_i="13508" xml_f="13521" txt_i="5893" txt_f="5906">10.2 (16/157)</offsets></td><td align="left"><offsets xml_i="13543" xml_f="13547" txt_i="5907" txt_f="5911">31.7</offsets><sup><offsets xml_i="13552" xml_f="13554" txt_i="5911" txt_f="5913">3 </offsets></sup><offsets xml_i="13560" xml_f="13567" txt_i="5913" txt_f="5920">(13/41)</offsets></td><td align="left"><offsets xml_i="13589" xml_f="13602" txt_i="5921" txt_f="5934">14.2 (65/458)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="13642" xml_f="13645" txt_i="5936" txt_f="5939">CLR</offsets><sup><offsets xml_i="13650" xml_f="13659" txt_i="5939" txt_f="5944">r &amp; s</offsets></sup></td><td align="left"><offsets xml_i="13687" xml_f="13699" txt_i="5945" txt_f="5957">9.6 (25/260)</offsets></td><td align="left"><offsets xml_i="13721" xml_f="13733" txt_i="5958" txt_f="5970">6.4 (10/157)</offsets></td><td align="left"><offsets xml_i="13755" xml_f="13766" txt_i="5971" txt_f="5982">14.6 (6/41)</offsets></td><td align="left"><offsets xml_i="13788" xml_f="13800" txt_i="5983" txt_f="5995">9.0 (41/458)</offsets></td></tr><tr><td align="left"><offsets xml_i="13831" xml_f="13845" txt_i="5996" txt_f="6010">Proportion of </offsets><italic><offsets xml_i="13853" xml_f="13858" txt_i="6010" txt_f="6015">MboII</offsets></italic><offsets xml_i="13867" xml_f="13882" txt_i="6015" txt_f="6030">-RFLP-positive </offsets><italic><offsets xml_i="13890" xml_f="13900" txt_i="6030" txt_f="6040">H. pylori </offsets></italic><offsets xml_i="13909" xml_f="13912" txt_i="6040" txt_f="6043">(%)</offsets></td><td align="left"><offsets xml_i="13934" xml_f="13943" txt_i="6044" txt_f="6053">2222CTTCA</offsets></td><td align="left"><offsets xml_i="13965" xml_f="13976" txt_i="6054" txt_f="6065">0.4 (1/260)</offsets></td><td align="left"><offsets xml_i="13998" xml_f="14009" txt_i="6066" txt_f="6077">1.9 (3/157)</offsets></td><td align="left"><offsets xml_i="14031" xml_f="14034" txt_i="6078" txt_f="6081">4.9</offsets><sup><offsets xml_i="14039" xml_f="14041" txt_i="6081" txt_f="6083">4 </offsets></sup><offsets xml_i="14047" xml_f="14053" txt_i="6083" txt_f="6089">(2/41)</offsets></td><td align="left"><offsets xml_i="14075" xml_f="14086" txt_i="6090" txt_f="6101">1.3 (6/458)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="14126" xml_f="14134" txt_i="6103" txt_f="6111">2081GAAG</offsets></td><td align="left"><offsets xml_i="14156" xml_f="14167" txt_i="6112" txt_f="6123">1.9 (5/260)</offsets></td><td align="left"><offsets xml_i="14189" xml_f="14200" txt_i="6124" txt_f="6135">1.3 (2/157)</offsets></td><td align="left"><offsets xml_i="14222" xml_f="14226" txt_i="6136" txt_f="6140">14.6</offsets><sup><offsets xml_i="14231" xml_f="14233" txt_i="6140" txt_f="6142">5 </offsets></sup><offsets xml_i="14239" xml_f="14245" txt_i="6142" txt_f="6148">(4/41)</offsets></td><td align="left"><offsets xml_i="14267" xml_f="14279" txt_i="6149" txt_f="6161">2.4 (11/458)</offsets></td></tr></tbody></table><table-wrap-foot><p><sup><offsets xml_i="14330" xml_f="14331" txt_i="6162" txt_f="6163">1</offsets></sup><offsets xml_i="14337" xml_f="14381" txt_i="6163" txt_f="6204">: OA group vs. placebo group, p &lt; 0.0000;</offsets></p><p><sup><offsets xml_i="14393" xml_f="14394" txt_i="6205" txt_f="6206">2</offsets></sup><offsets xml_i="14400" xml_f="14440" txt_i="6206" txt_f="6243">: OAC group vs. OA group, p &lt; 0.0005;</offsets></p><p><sup><offsets xml_i="14452" xml_f="14453" txt_i="6244" txt_f="6245">3</offsets></sup><offsets xml_i="14459" xml_f="14574" txt_i="6245" txt_f="6360">: OAC group vs. OA group, OR = 4.09, 95%CI [1.64–10.23]; OAC group vs. placebo group, OR = 2.89, 95%CI [1.28–6.46];</offsets></p><p><sup><offsets xml_i="14586" xml_f="14587" txt_i="6361" txt_f="6362">4</offsets></sup><offsets xml_i="14593" xml_f="14708" txt_i="6362" txt_f="6477">: OAC group vs. OA group, OR = 2.63, 95%CI [0.30–20.29]; OAC group vs. placebo group, OR = 13.28, 95%CI [0.91–379];</offsets></p><p><sup><offsets xml_i="14720" xml_f="14721" txt_i="6478" txt_f="6479">5</offsets></sup><offsets xml_i="14727" xml_f="14842" txt_i="6479" txt_f="6594">: OAC group vs. OA group, OR = 8.38, 95%CI [1.25–68.84]; OAC group vs. placebo group, OR = 5.51, 95%CI [1.18–25.12]</offsets></p></table-wrap-foot></table-wrap><table-wrap position="float" id="T2"><label><offsets xml_i="14921" xml_f="14928" txt_i="6595" txt_f="6602">Table 2</offsets></label><caption><p><offsets xml_i="14948" xml_f="15006" txt_i="6602" txt_f="6660">Comparison of accuracy of various assays for detection of </offsets><italic><offsets xml_i="15014" xml_f="15024" txt_i="6660" txt_f="6670">H. pylori </offsets></italic><offsets xml_i="15033" xml_f="15042" txt_i="6670" txt_f="6679">infection</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><td align="left"><offsets xml_i="15121" xml_f="15127" txt_i="6680" txt_f="6686">Assays</offsets></td><td align="left"><offsets xml_i="15149" xml_f="15160" txt_i="6687" txt_f="6698">Sensitivity</offsets></td><td align="left"><offsets xml_i="15182" xml_f="15193" txt_i="6699" txt_f="6710">Specificity</offsets></td><td align="left"><offsets xml_i="15215" xml_f="15229" txt_i="6711" txt_f="6725">Youden's index</offsets><sup><offsets xml_i="15234" xml_f="15235" txt_i="6725" txt_f="6726">1</offsets></sup></td></tr></thead><tbody><tr><td align="left"><italic><offsets xml_i="15295" xml_f="15304" txt_i="6727" txt_f="6736">23S rRNA </offsets></italic><offsets xml_i="15313" xml_f="15316" txt_i="6736" txt_f="6739">PCR</offsets><sup><offsets xml_i="15321" xml_f="15322" txt_i="6739" txt_f="6740">2</offsets></sup></td><td align="left"><offsets xml_i="15350" xml_f="15365" txt_i="6741" txt_f="6756">83.4% (373/447)</offsets></td><td align="left"><offsets xml_i="15387" xml_f="15402" txt_i="6757" txt_f="6772">75.5% (114/151)</offsets></td><td align="left"><offsets xml_i="15424" xml_f="15428" txt_i="6773" txt_f="6777">0.59</offsets></td></tr><tr><td align="left"><sup><offsets xml_i="15464" xml_f="15466" txt_i="6778" txt_f="6780">13</offsets></sup><offsets xml_i="15472" xml_f="15477" txt_i="6780" txt_f="6785">C-UBT</offsets><sup><offsets xml_i="15482" xml_f="15483" txt_i="6785" txt_f="6786">3</offsets></sup></td><td align="left"><offsets xml_i="15511" xml_f="15526" txt_i="6787" txt_f="6802">94.9% (373/393)</offsets></td><td align="left"><offsets xml_i="15548" xml_f="15563" txt_i="6803" txt_f="6818">70.8% (114/161)</offsets></td><td align="left"><offsets xml_i="15585" xml_f="15589" txt_i="6819" txt_f="6823">0.66</offsets></td></tr><tr><td align="left"><offsets xml_i="15620" xml_f="15625" txt_i="6824" txt_f="6829">ELISA</offsets><sup><offsets xml_i="15630" xml_f="15631" txt_i="6829" txt_f="6830">4</offsets></sup></td><td align="left"><offsets xml_i="15659" xml_f="15674" txt_i="6831" txt_f="6846">95.2% (373/392)</offsets></td><td align="left"><offsets xml_i="15696" xml_f="15711" txt_i="6847" txt_f="6862">63.7% (114/179)</offsets></td><td align="left"><offsets xml_i="15733" xml_f="15737" txt_i="6863" txt_f="6867">0.59</offsets></td></tr></tbody></table><table-wrap-foot><p><sup><offsets xml_i="15788" xml_f="15789" txt_i="6868" txt_f="6869">1</offsets></sup><offsets xml_i="15795" xml_f="15840" txt_i="6869" txt_f="6914">: defined as "sensitivity + specificity - 1";</offsets></p><p><sup><offsets xml_i="15852" xml_f="15853" txt_i="6915" txt_f="6916">2</offsets></sup><offsets xml_i="15859" xml_f="15866" txt_i="6916" txt_f="6923">: both </offsets><sup><offsets xml_i="15871" xml_f="15873" txt_i="6923" txt_f="6925">13</offsets></sup><offsets xml_i="15879" xml_f="15936" txt_i="6925" txt_f="6982">C-UBT and ELISA were positive or negative for comparison;</offsets></p><p><sup><offsets xml_i="15948" xml_f="15949" txt_i="6983" txt_f="6984">3</offsets></sup><offsets xml_i="15955" xml_f="16017" txt_i="6984" txt_f="7046">: both PCR and ELISA were positive or negative for comparison;</offsets></p><p><sup><offsets xml_i="16029" xml_f="16030" txt_i="7047" txt_f="7048">4</offsets></sup><offsets xml_i="16036" xml_f="16051" txt_i="7048" txt_f="7063">: both PCR and </offsets><sup><offsets xml_i="16056" xml_f="16058" txt_i="7063" txt_f="7065">13</offsets></sup><offsets xml_i="16064" xml_f="16111" txt_i="7065" txt_f="7112">C-UBT were positive or negative for comparison.</offsets></p></table-wrap-foot></table-wrap><p><offsets xml_i="16149" xml_f="16201" txt_i="7113" txt_f="7165">Seven years after the OA treatment, the presence of </offsets><italic><offsets xml_i="16209" xml_f="16218" txt_i="7165" txt_f="7174">23S rRNA </offsets></italic><offsets xml_i="16227" xml_f="16234" txt_i="7174" txt_f="7181">due to </offsets><italic><offsets xml_i="16242" xml_f="16252" txt_i="7181" txt_f="7191">H. pylori </offsets></italic><offsets xml_i="16261" xml_f="16385" txt_i="7191" txt_f="7315">infection remained remarkably lower than that in the placebo group [51.1% (157/307) vs. 83.9% (260/310), p = 0.0000] (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="16417" xml_f="16418" txt_i="7315" txt_f="7316">1</offsets></xref><offsets xml_i="16425" xml_f="16490" txt_i="7316" txt_f="7381">). Three years after the triple therapy with CLR, the cumulative </offsets><italic><offsets xml_i="16498" xml_f="16508" txt_i="7381" txt_f="7391">H. pylori </offsets></italic><offsets xml_i="16517" xml_f="16582" txt_i="7391" txt_f="7456">infection rate in the OAC group was 26.8% (41/153) by PCR (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="16614" xml_f="16615" txt_i="7456" txt_f="7457">1</offsets></xref><offsets xml_i="16622" xml_f="16624" txt_i="7457" txt_f="7459">).</offsets></p><p><offsets xml_i="16631" xml_f="16662" txt_i="7460" txt_f="7491">To detect the prevalence of CLR</offsets><sup><offsets xml_i="16667" xml_f="16669" txt_i="7491" txt_f="7493">r </offsets></sup><italic><offsets xml_i="16683" xml_f="16693" txt_i="7493" txt_f="7503">H. pylori </offsets></italic><offsets xml_i="16702" xml_f="16715" txt_i="7503" txt_f="7516">infection, a </offsets><italic><offsets xml_i="16723" xml_f="16727" txt_i="7516" txt_f="7520">BsaI</offsets></italic><offsets xml_i="16736" xml_f="16803" txt_i="7520" txt_f="7587">-restriction PCR-RFLP assay was developed using the PCR product of </offsets><italic><offsets xml_i="16811" xml_f="16820" txt_i="7587" txt_f="7596">23S rRNA </offsets></italic><offsets xml_i="16829" xml_f="16888" txt_i="7596" txt_f="7655">for detection of the A2143G mutation as illustrated in Fig </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="16918" xml_f="16919" txt_i="7655" txt_f="7656">1</offsets></xref><offsets xml_i="16926" xml_f="17021" txt_i="7656" txt_f="7751">. The A2143G mutation was detected in 65 of the 458 PCR products, which showed the presence of </offsets><italic><offsets xml_i="17029" xml_f="17038" txt_i="7751" txt_f="7760">H. pylori</offsets></italic><offsets xml_i="17047" xml_f="17072" txt_i="7760" txt_f="7785">, by the RFLP assay (Fig </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="17102" xml_f="17104" txt_i="7785" txt_f="7787">2A</offsets></xref><offsets xml_i="17111" xml_f="17170" txt_i="7787" txt_f="7846">). In 41 of the 65 A2143G-positive cases (63.1%), both the </offsets><italic><offsets xml_i="17178" xml_f="17182" txt_i="7846" txt_f="7850">BsaI</offsets></italic><offsets xml_i="17191" xml_f="17235" txt_i="7850" txt_f="7894">-sensitive and -insensitive PCR products of </offsets><italic><offsets xml_i="17243" xml_f="17252" txt_i="7894" txt_f="7903">23S rRNA </offsets></italic><offsets xml_i="17261" xml_f="17382" txt_i="7903" txt_f="8024">were observed. For 12 representative samples, results of the PCR-RFLP assay were consistent with that of sequencing (Fig </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="17412" xml_f="17414" txt_i="8024" txt_f="8026">2B</offsets></xref><offsets xml_i="17421" xml_f="17423" txt_i="8026" txt_f="8028">).</offsets></p><fig position="float" id="F1"><label><offsets xml_i="17464" xml_f="17472" txt_i="8029" txt_f="8037">Figure 1</offsets></label><caption><p><bold><offsets xml_i="17498" xml_f="17559" txt_i="8037" txt_f="8098">Schematic diagram for detection of nucleotide alterations of </offsets><italic><offsets xml_i="17567" xml_f="17576" txt_i="8098" txt_f="8107">23S rRNA </offsets></italic><offsets xml_i="17585" xml_f="17603" txt_i="8107" txt_f="8125">by PCR-RFLP assays</offsets></bold><offsets xml_i="17610" xml_f="17616" txt_i="8125" txt_f="8131">. The </offsets><italic><offsets xml_i="17624" xml_f="17633" txt_i="8131" txt_f="8140">23S rRNA </offsets></italic><offsets xml_i="17642" xml_f="17654" txt_i="8140" txt_f="8152">sequence of </offsets><italic><offsets xml_i="17662" xml_f="17678" txt_i="8152" txt_f="8168">H. pylori 26695 </offsets></italic><offsets xml_i="17687" xml_f="17743" txt_i="8168" txt_f="8224">was illustrated as the wild-type sequence [GenBank No.: </offsets><ext-link ext-link-type="gen" xlink:href="U27270"><offsets xml_i="17793" xml_f="17799" txt_i="8224" txt_f="8230">U27270</offsets></ext-link><offsets xml_i="17810" xml_f="17812" txt_i="8230" txt_f="8232">].</offsets></p></caption><graphic xlink:href="1471-2180-8-81-1"></graphic></fig><fig position="float" id="F2"><label><offsets xml_i="17918" xml_f="17926" txt_i="8233" txt_f="8241">Figure 2</offsets></label><caption><p><bold><offsets xml_i="17952" xml_f="18043" txt_i="8241" txt_f="8332">Chromatograms of PCR-RFLP assays and sequencing for detection of nucleotide alterations of </offsets><italic><offsets xml_i="18051" xml_f="18059" txt_i="8332" txt_f="8340">23S rRNA</offsets></italic></bold><offsets xml_i="18075" xml_f="18077" txt_i="8340" txt_f="8342">. </offsets><italic><offsets xml_i="18085" xml_f="18101" txt_i="8342" txt_f="8358">H. Pylori 26695 </offsets></italic><offsets xml_i="18110" xml_f="18117" txt_i="8358" txt_f="8365">and CLR</offsets><sup><offsets xml_i="18122" xml_f="18123" txt_i="8365" txt_f="8366">r</offsets></sup><offsets xml_i="18129" xml_f="18195" txt_i="8366" txt_f="8432">-1 were used as negative and positive control of A2143G mutation. </offsets><italic><offsets xml_i="18203" xml_f="18208" txt_i="8432" txt_f="8437">BsaI </offsets></italic><offsets xml_i="18217" xml_f="18393" txt_i="8437" txt_f="8613">digestion of the PCR products of representative samples was displayed on 8% PAGE gel. The 289 bp A2143G-positive PCR products were cleaved into a 199 bp and a 90 bp fragments (</offsets><bold><offsets xml_i="18399" xml_f="18400" txt_i="8613" txt_f="8614">A</offsets></bold><offsets xml_i="18407" xml_f="18486" txt_i="8614" txt_f="8693">). The A2143G mutation was also confirmed by sequencing of the PCR products of </offsets><italic><offsets xml_i="18494" xml_f="18503" txt_i="8693" txt_f="8702">23S rRNA </offsets></italic><offsets xml_i="18512" xml_f="18513" txt_i="8702" txt_f="8703">(</offsets><bold><offsets xml_i="18519" xml_f="18520" txt_i="8703" txt_f="8704">B</offsets></bold><offsets xml_i="18527" xml_f="18560" txt_i="8704" txt_f="8737">, displayed 2140–2154 fragment). </offsets><italic><offsets xml_i="18568" xml_f="18584" txt_i="8737" txt_f="8753">H. Pylori 26695 </offsets></italic><offsets xml_i="18593" xml_f="18599" txt_i="8753" txt_f="8759">and a </offsets><italic><offsets xml_i="18607" xml_f="18613" txt_i="8759" txt_f="8765">2142G </offsets></italic><offsets xml_i="18622" xml_f="18691" txt_i="8765" txt_f="8834">clone were used as negative and positive control of A2142G mutation. </offsets><italic><offsets xml_i="18699" xml_f="18705" txt_i="8834" txt_f="8840">MboII </offsets></italic><offsets xml_i="18714" xml_f="18846" txt_i="8840" txt_f="8972">digestion of the PCR products of representative samples was displayed on 2% agarose gel. The 289 bp A2142G-positive PCR products of </offsets><italic><offsets xml_i="18854" xml_f="18860" txt_i="8972" txt_f="8978">2142G </offsets></italic><offsets xml_i="18869" xml_f="19070" txt_i="8978" txt_f="9179">were cleaved into an 182 bp and a 107 bp fragments. The PCR product of GJ2040 was cleaved into a 164 bp and a 125 bp fragments; and the product of GJ2111 was cleaved into 245 bp and 44 bp fragment(s) (</offsets><bold><offsets xml_i="19076" xml_f="19077" txt_i="9179" txt_f="9180">C</offsets></bold><offsets xml_i="19084" xml_f="19148" txt_i="9180" txt_f="9244">). The A2142G and other mutations were confirmed by sequencing (</offsets><bold><offsets xml_i="19154" xml_f="19155" txt_i="9244" txt_f="9245">D</offsets></bold><offsets xml_i="19162" xml_f="19173" txt_i="9245" txt_f="9256">). Two new </offsets><italic><offsets xml_i="19181" xml_f="19186" txt_i="9256" txt_f="9261">MboII</offsets></italic><offsets xml_i="19195" xml_f="19299" txt_i="9261" txt_f="9365">-sensitive sequences were characterized as CTTCA (2222–2226) for GJ2040 and GAAG (2081–2084) for GJ2111.</offsets></p></caption><graphic xlink:href="1471-2180-8-81-2"></graphic></fig><p><offsets xml_i="19371" xml_f="19408" txt_i="9366" txt_f="9403">To detect the A2142G mutation within </offsets><italic><offsets xml_i="19416" xml_f="19424" txt_i="9403" txt_f="9411">23S rRNA</offsets></italic><offsets xml_i="19433" xml_f="19437" txt_i="9411" txt_f="9415">, a </offsets><italic><offsets xml_i="19445" xml_f="19450" txt_i="9415" txt_f="9420">MboII</offsets></italic><offsets xml_i="19459" xml_f="19511" txt_i="9420" txt_f="9472">-restriction PCR-RFLP assay was also developed (Fig </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="19541" xml_f="19542" txt_i="9472" txt_f="9473">1</offsets></xref><offsets xml_i="19549" xml_f="19614" txt_i="9473" txt_f="9538">). No A2142G point mutation was observed among 458 subjects with </offsets><italic><offsets xml_i="19622" xml_f="19632" txt_i="9538" txt_f="9548">H. pylori </offsets></italic><offsets xml_i="19641" xml_f="19669" txt_i="9548" txt_f="9576">infection. However, two new </offsets><italic><offsets xml_i="19677" xml_f="19682" txt_i="9576" txt_f="9581">MboII</offsets></italic><offsets xml_i="19691" xml_f="19757" txt_i="9581" txt_f="9647">-RFLP patterns were detected in DNA samples from 17 subjects (Fig </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="19787" xml_f="19789" txt_i="9647" txt_f="9649">2C</offsets></xref><offsets xml_i="19796" xml_f="20144" txt_i="9649" txt_f="9997">): a pattern consisting of a 164 bp and a 125 bp fragment was observed from 11 subjects and another pattern consisting of a 245 bp and 44 bp fragment was detected from 6 cases. Sequencing showed that the former 11 cases had a sequence change of AACC → GAAG (2081–2084) and the latter 6 cases had a sequence change of AAGGG → CTTCA (2222–2226) (Fig </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="20174" xml_f="20176" txt_i="9997" txt_f="9999">2D</offsets></xref><offsets xml_i="20183" xml_f="20185" txt_i="9999" txt_f="10001">).</offsets></p><p><offsets xml_i="20192" xml_f="20253" txt_i="10002" txt_f="10063">To investigate the relationship between the prevalence of CLR</offsets><sup><offsets xml_i="20258" xml_f="20260" txt_i="10063" txt_f="10065">r </offsets></sup><italic><offsets xml_i="20274" xml_f="20284" txt_i="10065" txt_f="10075">H. pylori </offsets></italic><offsets xml_i="20293" xml_f="20403" txt_i="10075" txt_f="10185">infection and administration of CLR as a component of an optimal triple therapy, A2143G mutation rates within </offsets><italic><offsets xml_i="20411" xml_f="20420" txt_i="10185" txt_f="10194">23S rRNA </offsets></italic><offsets xml_i="20429" xml_f="20472" txt_i="10194" txt_f="10237">in each therapy group were analyzed (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="20504" xml_f="20505" txt_i="10237" txt_f="10238">1</offsets></xref><offsets xml_i="20512" xml_f="20935" txt_i="10238" txt_f="10661">). The presence of the A2143G mutation in the OAC group was 3.1 times higher than that in the OA group [31.7% vs. 10.2%, p = 0.0005; OR 4.09 (95% CI = 1.64–10.23)] and 2.3 times higher than that in the placebo group [31.7% vs. 13.8%, p = 0.0040; OR 2.89 (95% CI = 1.28–6.46)]. The difference of the A2143G mutation rate due to CLR was 21.5% between the OAC and OA groups. In the OAC group, the co-infection rate (14.6%) of </offsets><italic><offsets xml_i="20943" xml_f="20947" txt_i="10661" txt_f="10665">BsaI</offsets></italic><offsets xml_i="20956" xml_f="20969" txt_i="10665" txt_f="10678">-insensitive </offsets><italic><offsets xml_i="20977" xml_f="20987" txt_i="10678" txt_f="10688">H. pylori </offsets></italic><offsets xml_i="20996" xml_f="21025" txt_i="10688" txt_f="10717">among subjects infected with </offsets><italic><offsets xml_i="21033" xml_f="21037" txt_i="10717" txt_f="10721">BsaI</offsets></italic><offsets xml_i="21046" xml_f="21057" txt_i="10721" txt_f="10732">-sensitive </offsets><italic><offsets xml_i="21065" xml_f="21075" txt_i="10732" txt_f="10742">H. pylori </offsets></italic><offsets xml_i="21084" xml_f="21614" txt_i="10742" txt_f="11272">was higher than those in the OA group (6.4%) and the placebo group (9.6%) but not significant (p = 0.0838 and 0.3258, respectively). Moreover, the sequence change AAGGG → CTTCA (2222–2226) in the OAC group was 12.3 times of the placebo group (4.9% vs. 0.4%, p = 0.0497; OR 13.28, 95% CI 0.91–379); the AACC → GAAG (2081–2084) in the OAC group was 11.2 times of the OA group (14.6.7% vs. 1.3%, p = 0.018; OR 8.38, 95% CI 1.25–68.84) and 7.7 times of the placebo group (14.6% vs. 1.9%, p = 0.002; OR 5.51, 95% CI 1.18–25.12) (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="21646" xml_f="21647" txt_i="11272" txt_f="11273">1</offsets></xref><offsets xml_i="21654" xml_f="21656" txt_i="11273" txt_f="11275">).</offsets></p></sec><sec><title><offsets xml_i="21678" xml_f="21688" txt_i="11277" txt_f="11287">Discussion</offsets></title><p><offsets xml_i="21699" xml_f="21772" txt_i="11288" txt_f="11361">Among Chinese subjects enrolled in the long term population-based trial [</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="21804" xml_f="21806" txt_i="11361" txt_f="11363">10</offsets></xref><offsets xml_i="21813" xml_f="21863" txt_i="11363" txt_f="11413">], we found that the proportion of the primary CLR</offsets><sup><offsets xml_i="21868" xml_f="21870" txt_i="11413" txt_f="11415">r </offsets></sup><offsets xml_i="21876" xml_f="21906" txt_i="11415" txt_f="11445">related A2143G mutation among </offsets><italic><offsets xml_i="21914" xml_f="21922" txt_i="11445" txt_f="11453">23S rRNA</offsets></italic><offsets xml_i="21931" xml_f="22103" txt_i="11453" txt_f="11625">-positive subjects in the placebo group was 13.8% (36/260), which was similar to the average reported CLR-resistant prevalence of 10.5% (35/334) within China: 5.2% (5/96) [</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="22135" xml_f="22137" txt_i="11625" txt_f="11627">14</offsets></xref><offsets xml_i="22144" xml_f="22160" txt_i="11627" txt_f="11643">], 9.8% (4/41) [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="22192" xml_f="22194" txt_i="11643" txt_f="11645">13</offsets></xref><offsets xml_i="22201" xml_f="22219" txt_i="11645" txt_f="11663">], 13.5% (12/89) [</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="22251" xml_f="22253" txt_i="11663" txt_f="11665">15</offsets></xref><offsets xml_i="22260" xml_f="22283" txt_i="11665" txt_f="11688">], and 14.8% (16/108) [</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="22315" xml_f="22317" txt_i="11688" txt_f="11690">16</offsets></xref><offsets xml_i="22324" xml_f="22339" txt_i="11690" txt_f="11705">]. That the CLR</offsets><sup><offsets xml_i="22344" xml_f="22346" txt_i="11705" txt_f="11707">r </offsets></sup><offsets xml_i="22352" xml_f="22496" txt_i="11707" txt_f="11851">infection rate in the OA group was slightly lower than that in the placebo group (p = 0.274) indicated that the OA treatment for eradication of </offsets><italic><offsets xml_i="22504" xml_f="22514" txt_i="11851" txt_f="11861">H. pylori </offsets></italic><offsets xml_i="22523" xml_f="22565" txt_i="11861" txt_f="11903">might reduce the prevalence of primary CLR</offsets><sup><offsets xml_i="22570" xml_f="22572" txt_i="11903" txt_f="11905">r </offsets></sup><offsets xml_i="22578" xml_f="22611" txt_i="11905" txt_f="11938">among the treated subjects. Heep </offsets><italic><offsets xml_i="22619" xml_f="22624" txt_i="11938" txt_f="11943">et al</offsets></italic><offsets xml_i="22633" xml_f="22709" txt_i="11943" txt_f="12019">. reported that secondary resistance to CLR was up to 50% in a study of 554 </offsets><italic><offsets xml_i="22717" xml_f="22727" txt_i="12019" txt_f="12029">H. pylori </offsets></italic><offsets xml_i="22736" xml_f="22771" txt_i="12029" txt_f="12064">isolates after failure of therapy [</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="22803" xml_f="22805" txt_i="12064" txt_f="12066">17</offsets></xref><offsets xml_i="22812" xml_f="22899" txt_i="12066" txt_f="12153">]. It is well recognized that eradication therapy with CLR led to the prevalence of CLR</offsets><sup><offsets xml_i="22904" xml_f="22906" txt_i="12153" txt_f="12155">r </offsets></sup><italic><offsets xml_i="22920" xml_f="22930" txt_i="12155" txt_f="12165">H. pylori </offsets></italic><offsets xml_i="22939" xml_f="22950" txt_i="12165" txt_f="12176">infection [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="22981" xml_f="22982" txt_i="12176" txt_f="12177">8</offsets></xref><offsets xml_i="22989" xml_f="22990" txt_i="12177" txt_f="12178">,</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="23021" xml_f="23022" txt_i="12178" txt_f="12179">9</offsets></xref><offsets xml_i="23029" xml_f="23101" txt_i="12179" txt_f="12251">]. In the present study, we observed a 31.7% A2143G mutation rate among </offsets><italic><offsets xml_i="23109" xml_f="23119" txt_i="12251" txt_f="12261">H. pylori </offsets></italic><offsets xml_i="23128" xml_f="23643" txt_i="12261" txt_f="12776">infected subjects three years after treatment with CLR, which is significantly higher than that among subjects without CLR treatment in the OA and placebo groups (OR = 4.09 and 2.89, respectively). An optimal way to validate the exact effect of CLR use on the formation of the secondary CLR resistance was to compare the prevalence of the A2143G mutation among subjects before and after administration of CLR. Because of the lack of gastric juice samples before eradication therapy, the original mutation status of </offsets><italic><offsets xml_i="23651" xml_f="23660" txt_i="12776" txt_f="12785">23S rRNA </offsets></italic><offsets xml_i="23669" xml_f="23998" txt_i="12785" txt_f="13114">in samples was not available, thus such comparison could not be carried out in the present study. This is a limitation of our study. The strength of this study was that all of the subjects in the OAC group had the same CLR treatment history in 2000, and those in the OA and placebo group did not. Therefore, the prevalence of CLR</offsets><sup><offsets xml_i="24003" xml_f="24005" txt_i="13114" txt_f="13116">r </offsets></sup><italic><offsets xml_i="24019" xml_f="24029" txt_i="13116" txt_f="13126">H. pylori </offsets></italic><offsets xml_i="24038" xml_f="24234" txt_i="13126" txt_f="13322">infection among subjects with and without CLR use in the local area was reliably obtained. These results were useful for evaluation of the exact effect of administration of CLR for eradication of </offsets><italic><offsets xml_i="24242" xml_f="24252" txt_i="13322" txt_f="13332">H. pylori </offsets></italic><offsets xml_i="24261" xml_f="24325" txt_i="13332" txt_f="13396">on the prevalence of the A2143G mutation among Chinese subjects.</offsets></p><p><offsets xml_i="24332" xml_f="24373" txt_i="13397" txt_f="13438">Unlike indirect assays such as ELISA and </offsets><sup><offsets xml_i="24378" xml_f="24380" txt_i="13438" txt_f="13440">13</offsets></sup><offsets xml_i="24386" xml_f="24432" txt_i="13440" txt_f="13486">C-UBT, PCR assays are direct assays to detect </offsets><italic><offsets xml_i="24440" xml_f="24450" txt_i="13486" txt_f="13496">H. pylori </offsets></italic><offsets xml_i="24459" xml_f="24551" txt_i="13496" txt_f="13588">infection. Thus, these assays should result in higher specificity than the indirect assays. </offsets><italic><offsets xml_i="24559" xml_f="24569" txt_i="13588" txt_f="13598">H. pylori </offsets></italic><offsets xml_i="24578" xml_f="24741" txt_i="13598" txt_f="13761">colonizes on the apical surface of stomach's epithelial cells and sheds into gastric juice with the regular renewal of gastric mucosa. Several PCR assays based on </offsets><italic><offsets xml_i="24749" xml_f="24759" txt_i="13761" txt_f="13771">H. pylori </offsets></italic><offsets xml_i="24768" xml_f="24836" txt_i="13771" txt_f="13839">DNA from gastric juice/biopsy sample were reported for detection of </offsets><italic><offsets xml_i="24844" xml_f="24854" txt_i="13839" txt_f="13849">H. pylori </offsets></italic><offsets xml_i="24863" xml_f="24874" txt_i="13849" txt_f="13860">infection [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="24906" xml_f="24908" txt_i="13860" txt_f="13862">13</offsets></xref><offsets xml_i="24915" xml_f="24916" txt_i="13862" txt_f="13863">,</offsets><xref ref-type="bibr" rid="B18"><offsets xml_i="24948" xml_f="24950" txt_i="13863" txt_f="13865">18</offsets></xref><offsets xml_i="24957" xml_f="24958" txt_i="13865" txt_f="13866">,</offsets><xref ref-type="bibr" rid="B19"><offsets xml_i="24990" xml_f="24992" txt_i="13866" txt_f="13868">19</offsets></xref><offsets xml_i="24999" xml_f="25015" txt_i="13868" txt_f="13884">]. The possible </offsets><italic><offsets xml_i="25023" xml_f="25032" txt_i="13884" txt_f="13893">H. pylori</offsets></italic><offsets xml_i="25041" xml_f="25073" txt_i="13893" txt_f="13925">-specific target genes included </offsets><italic><offsets xml_i="25081" xml_f="25085" txt_i="13925" txt_f="13929">ureA</offsets></italic><offsets xml_i="25094" xml_f="25096" txt_i="13929" txt_f="13931">, </offsets><italic><offsets xml_i="25104" xml_f="25108" txt_i="13931" txt_f="13935">ureB</offsets></italic><offsets xml_i="25117" xml_f="25119" txt_i="13935" txt_f="13937">, </offsets><italic><offsets xml_i="25127" xml_f="25131" txt_i="13937" txt_f="13941">ureC</offsets></italic><offsets xml_i="25140" xml_f="25142" txt_i="13941" txt_f="13943">, </offsets><italic><offsets xml_i="25150" xml_f="25154" txt_i="13943" txt_f="13947">ureD</offsets></italic><offsets xml_i="25163" xml_f="25165" txt_i="13947" txt_f="13949">, </offsets><italic><offsets xml_i="25173" xml_f="25181" txt_i="13949" txt_f="13957">16S rRNA</offsets></italic><offsets xml_i="25190" xml_f="25196" txt_i="13957" txt_f="13963">, and </offsets><italic><offsets xml_i="25204" xml_f="25212" txt_i="13963" txt_f="13971">23S rRNA</offsets></italic><offsets xml_i="25221" xml_f="25242" txt_i="13971" txt_f="13992">. Point mutations of </offsets><italic><offsets xml_i="25250" xml_f="25259" txt_i="13992" txt_f="14001">23S rRNA </offsets></italic><offsets xml_i="25268" xml_f="25321" txt_i="14001" txt_f="14054">were closely related to formation of CLR resistance [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="25352" xml_f="25353" txt_i="14054" txt_f="14055">8</offsets></xref><offsets xml_i="25360" xml_f="25361" txt_i="14055" txt_f="14056">,</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="25393" xml_f="25395" txt_i="14056" txt_f="14058">11</offsets></xref><offsets xml_i="25402" xml_f="25403" txt_i="14058" txt_f="14059">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="25435" xml_f="25437" txt_i="14059" txt_f="14061">12</offsets></xref><offsets xml_i="25444" xml_f="25474" txt_i="14061" txt_f="14091">]. Because the PCR product of </offsets><italic><offsets xml_i="25482" xml_f="25491" txt_i="14091" txt_f="14100">23S rRNA </offsets></italic><offsets xml_i="25500" xml_f="25563" txt_i="14100" txt_f="14163">described in this study could be used as a biomarker to detect </offsets><italic><offsets xml_i="25571" xml_f="25581" txt_i="14163" txt_f="14173">H. pylori </offsets></italic><offsets xml_i="25590" xml_f="25745" txt_i="14173" txt_f="14328">infection and CLR resistance simultaneously, this assay has advantages over other PCR-based methodologies. However, the specificity and sensitivity of the </offsets><italic><offsets xml_i="25753" xml_f="25762" txt_i="14328" txt_f="14337">23S rRNA </offsets></italic><offsets xml_i="25771" xml_f="25872" txt_i="14337" txt_f="14438">PCR assay was not evaluated extensively. No single test is accepted as the standard for diagnosis of </offsets><italic><offsets xml_i="25880" xml_f="25890" txt_i="14438" txt_f="14448">H. pylori </offsets></italic><offsets xml_i="25899" xml_f="25910" txt_i="14448" txt_f="14459">infection [</offsets><xref ref-type="bibr" rid="B20"><offsets xml_i="25942" xml_f="25944" txt_i="14459" txt_f="14461">20</offsets></xref><offsets xml_i="25951" xml_f="25952" txt_i="14461" txt_f="14462">,</offsets><xref ref-type="bibr" rid="B21"><offsets xml_i="25984" xml_f="25986" txt_i="14462" txt_f="14464">21</offsets></xref><offsets xml_i="25993" xml_f="26032" txt_i="14464" txt_f="14503">]. In the present study, the status of </offsets><italic><offsets xml_i="26040" xml_f="26050" txt_i="14503" txt_f="14513">H. pylori </offsets></italic><offsets xml_i="26059" xml_f="26089" txt_i="14513" txt_f="14543">infection was detected by the </offsets><italic><offsets xml_i="26097" xml_f="26106" txt_i="14543" txt_f="14552">23S rRNA </offsets></italic><offsets xml_i="26115" xml_f="26126" txt_i="14552" txt_f="14563">PCR assay, </offsets><sup><offsets xml_i="26131" xml_f="26133" txt_i="14563" txt_f="14565">13</offsets></sup><offsets xml_i="26139" xml_f="26321" txt_i="14565" txt_f="14747">C-UBT, and ELISA for each subject, which provided an opportunity to assess the specificity and sensitivity of these assays. As expected, the highest specificity was observed for the </offsets><italic><offsets xml_i="26329" xml_f="26338" txt_i="14747" txt_f="14756">23S rRNA </offsets></italic><offsets xml_i="26347" xml_f="26407" txt_i="14756" txt_f="14816">PCR assay when both CUBT and ELISA assays were negative for </offsets><italic><offsets xml_i="26415" xml_f="26425" txt_i="14816" txt_f="14826">H. pylori </offsets></italic><offsets xml_i="26434" xml_f="26471" txt_i="14826" txt_f="14863">infection. The Youden's index of the </offsets><italic><offsets xml_i="26479" xml_f="26488" txt_i="14863" txt_f="14872">23S rRNA </offsets></italic><offsets xml_i="26497" xml_f="26569" txt_i="14872" txt_f="14944">PCR assay was similar to that of the ELISA assay but lower than that of </offsets><sup><offsets xml_i="26574" xml_f="26576" txt_i="14944" txt_f="14946">13</offsets></sup><offsets xml_i="26582" xml_f="26595" txt_i="14946" txt_f="14959">C-UBT (Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="26627" xml_f="26628" txt_i="14959" txt_f="14960">2</offsets></xref><offsets xml_i="26635" xml_f="26637" txt_i="14960" txt_f="14962">).</offsets></p><p><offsets xml_i="26644" xml_f="26687" txt_i="14963" txt_f="15006">Generally, PCR-based assays used to detect </offsets><italic><offsets xml_i="26695" xml_f="26705" txt_i="15006" txt_f="15016">H. pylori </offsets></italic><offsets xml_i="26714" xml_f="26791" txt_i="15016" txt_f="15093">infection are considered the most sensitive. However, the sensitivity of the </offsets><italic><offsets xml_i="26799" xml_f="26808" txt_i="15093" txt_f="15102">23S rRNA </offsets></italic><offsets xml_i="26817" xml_f="26842" txt_i="15102" txt_f="15127">PCR assay was lower than </offsets><sup><offsets xml_i="26847" xml_f="26849" txt_i="15127" txt_f="15129">13</offsets></sup><offsets xml_i="26855" xml_f="26899" txt_i="15129" txt_f="15173">C-UBT and ELISA in the present study (Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="26931" xml_f="26932" txt_i="15173" txt_f="15174">2</offsets></xref><offsets xml_i="26939" xml_f="27783" txt_i="15174" txt_f="16018">). Because the genomic DNA could not be extracted from gastric juice samples immediately after collection and that the shedded gastric epithelial cells easily self-decompose, the quality and amount of DNA extracted is variable between samples. In this study, fixative was added to gastric juice samples at the local clinics to prevent degeneration of DNA, and then the samples were stored at -20°C. These samples were shipped to the laboratory using dry ice and stored at -80°C. Although the fixative was removed with repeated washings before DNA extraction, it is possible that the quality of the DNA was adversely affected. The fixation process could be omitted if gastric juice samples can be stored at -80°C immediately after collection. However, it would be optimal if the DNA were isolated from fresh samples immediately after collection.</offsets></p><p><offsets xml_i="27790" xml_f="27793" txt_i="16019" txt_f="16022">CLR</offsets><sup><offsets xml_i="27798" xml_f="27800" txt_i="16022" txt_f="16024">r </offsets></sup><offsets xml_i="27806" xml_f="27884" txt_i="16024" txt_f="16102">results mainly from point mutations in the peptidyltransferase loop region of </offsets><italic><offsets xml_i="27892" xml_f="27901" txt_i="16102" txt_f="16111">23S rRNA </offsets></italic><offsets xml_i="27910" xml_f="27911" txt_i="16111" txt_f="16112">[</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="27942" xml_f="27943" txt_i="16112" txt_f="16113">8</offsets></xref><offsets xml_i="27950" xml_f="27951" txt_i="16113" txt_f="16114">,</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="27982" xml_f="27983" txt_i="16114" txt_f="16115">9</offsets></xref><offsets xml_i="27990" xml_f="27991" txt_i="16115" txt_f="16116">,</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="28023" xml_f="28025" txt_i="16116" txt_f="16118">11</offsets></xref><offsets xml_i="28032" xml_f="28033" txt_i="16118" txt_f="16119">-</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="28065" xml_f="28067" txt_i="16119" txt_f="16121">13</offsets></xref><offsets xml_i="28074" xml_f="28128" txt_i="16121" txt_f="16175">]. The most frequent mutations are A2143G and A2142G [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="28159" xml_f="28160" txt_i="16175" txt_f="16176">8</offsets></xref><offsets xml_i="28167" xml_f="28262" txt_i="16176" txt_f="16271">]. Other mutations such as A2142C, A2115G, G2141A, and T2717C might also be associated with CLR</offsets><sup><offsets xml_i="28267" xml_f="28268" txt_i="16271" txt_f="16272">r</offsets></sup><offsets xml_i="28274" xml_f="28338" txt_i="16272" txt_f="16336">. Recent reports demonstrated that the A2143G point mutation of </offsets><italic><offsets xml_i="28346" xml_f="28355" txt_i="16336" txt_f="16345">23S rRNA </offsets></italic><offsets xml_i="28364" xml_f="28410" txt_i="16345" txt_f="16391">were detected in average of 90% (44 of 49) CLR</offsets><sup><offsets xml_i="28415" xml_f="28417" txt_i="16391" txt_f="16393">r </offsets></sup><italic><offsets xml_i="28431" xml_f="28441" txt_i="16393" txt_f="16403">H. pylori </offsets></italic><offsets xml_i="28450" xml_f="28482" txt_i="16403" txt_f="16435">isolates from Chinese subjects [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="28514" xml_f="28516" txt_i="16435" txt_f="16437">13</offsets></xref><offsets xml_i="28523" xml_f="28524" txt_i="16437" txt_f="16438">-</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="28556" xml_f="28558" txt_i="16438" txt_f="16440">16</offsets></xref><offsets xml_i="28565" xml_f="28566" txt_i="16440" txt_f="16441">,</offsets><xref ref-type="bibr" rid="B22"><offsets xml_i="28598" xml_f="28600" txt_i="16441" txt_f="16443">22</offsets></xref><offsets xml_i="28607" xml_f="28680" txt_i="16443" txt_f="16516">]. In the present study, the A2143G mutation was detectable in 65 of 458 </offsets><italic><offsets xml_i="28688" xml_f="28697" txt_i="16516" txt_f="16525">23S rRNA </offsets></italic><offsets xml_i="28706" xml_f="28726" txt_i="16525" txt_f="16545">PCR products by the </offsets><italic><offsets xml_i="28734" xml_f="28738" txt_i="16545" txt_f="16549">BsaI</offsets></italic><offsets xml_i="28747" xml_f="28782" txt_i="16549" txt_f="16584">-restriction PCR-RFLP assay (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="28814" xml_f="28815" txt_i="16584" txt_f="16585">1</offsets></xref><offsets xml_i="28822" xml_f="28937" txt_i="16585" txt_f="16700">). No A2142G mutation was detected in these samples. These results suggest that the A2143G may be a very useful CLR</offsets><sup><offsets xml_i="28942" xml_f="28944" txt_i="16700" txt_f="16702">r </offsets></sup><offsets xml_i="28950" xml_f="29001" txt_i="16702" txt_f="16753">indicator for selection of optimal therapy against </offsets><italic><offsets xml_i="29009" xml_f="29019" txt_i="16753" txt_f="16763">H. pylori </offsets></italic><offsets xml_i="29028" xml_f="29047" txt_i="16763" txt_f="16782">infection in China.</offsets></p><p><offsets xml_i="29054" xml_f="29110" txt_i="16783" txt_f="16839">In 41 of the 65 A2143G-positive cases (63.1%), both the </offsets><italic><offsets xml_i="29118" xml_f="29122" txt_i="16839" txt_f="16843">BsaI</offsets></italic><offsets xml_i="29131" xml_f="29159" txt_i="16843" txt_f="16871">-sensitive and -insensitive </offsets><italic><offsets xml_i="29167" xml_f="29176" txt_i="16871" txt_f="16880">23S rRNA </offsets></italic><offsets xml_i="29185" xml_f="29272" txt_i="16880" txt_f="16967">PCR products was observed. Because the PCR products of the A2143G-positive control (CLR</offsets><sup><offsets xml_i="29277" xml_f="29278" txt_i="16967" txt_f="16968">r</offsets></sup><offsets xml_i="29284" xml_f="29305" txt_i="16968" txt_f="16989">-1) were digested by </offsets><italic><offsets xml_i="29313" xml_f="29318" txt_i="16989" txt_f="16994">BsaI </offsets></italic><offsets xml_i="29327" xml_f="29423" txt_i="16994" txt_f="17090">enzyme completely in each experiment, partial restriction could be excluded. Thus, multi-strain </offsets><italic><offsets xml_i="29431" xml_f="29441" txt_i="17090" txt_f="17100">H. pylori </offsets></italic><offsets xml_i="29450" xml_f="29573" txt_i="17100" txt_f="17223">infection likely happened among these cases. Sequencing results of 12 representative samples supported the deduction (Fig. </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="29603" xml_f="29605" txt_i="17223" txt_f="17225">2B</offsets></xref><offsets xml_i="29612" xml_f="29851" txt_i="17225" txt_f="17464">). Isolation of the sensitive and resistant strains from cultures might provide a direct evidence to show the mixed colonization. Because the experimental conditions at the local clinics in Linqu County was not adequate to support primary </offsets><italic><offsets xml_i="29859" xml_f="29869" txt_i="17464" txt_f="17474">H. pylori </offsets></italic><offsets xml_i="29878" xml_f="29900" txt_i="17474" txt_f="17496">culture, isolation of </offsets><italic><offsets xml_i="29908" xml_f="29918" txt_i="17496" txt_f="17506">H. pylori </offsets></italic><offsets xml_i="29927" xml_f="30004" txt_i="17506" txt_f="17583">from biopsies of stomach was not carried out in the clinical trial. The high </offsets><italic><offsets xml_i="30012" xml_f="30022" txt_i="17583" txt_f="17593">H. pylori </offsets></italic><offsets xml_i="30031" xml_f="30146" txt_i="17593" txt_f="17708">infection rate in the local area might contribute to the high co-infection rate of the CLR sensitive and resistant </offsets><italic><offsets xml_i="30154" xml_f="30164" txt_i="17708" txt_f="17718">H. pylori </offsets></italic><offsets xml_i="30173" xml_f="30181" txt_i="17718" txt_f="17726">strains.</offsets></p><p><offsets xml_i="30188" xml_f="30209" txt_i="17727" txt_f="17748">In addition, two new </offsets><italic><offsets xml_i="30217" xml_f="30222" txt_i="17748" txt_f="17753">MboII</offsets></italic><offsets xml_i="30231" xml_f="30673" txt_i="17753" txt_f="18195">-RFLP patterns were discovered in 17 subjects in the present study. Sequencing showed that 6 of the 17 cases had a sequence change of AAGGG → CTTCA (2222–2226) and that the remaining 11 cases had a sequence change of AACC → GAAG (2081–2084). The A2143G mutation was detected only in 3 of the 15 GAAG-positive cases (20%). Both the AAGGG → CTTCA (2222–2226) and AACC → GAAG (2081–2084) alterations are located around the conserved domain V of </offsets><italic><offsets xml_i="30681" xml_f="30690" txt_i="18195" txt_f="18204">23S rRNA </offsets></italic><offsets xml_i="30699" xml_f="30720" txt_i="18204" txt_f="18225">(2126–2712, equal to </offsets><italic><offsets xml_i="30728" xml_f="30745" txt_i="18225" txt_f="18242">Escherichia coli </offsets></italic><xref ref-type="bibr" rid="B23"><offsets xml_i="30786" xml_f="30788" txt_i="18242" txt_f="18244">23</offsets></xref><offsets xml_i="30795" xml_f="30796" txt_i="18244" txt_f="18245">,</offsets><xref ref-type="bibr" rid="B24"><offsets xml_i="30828" xml_f="30830" txt_i="18245" txt_f="18247">24</offsets></xref><offsets xml_i="30837" xml_f="30946" txt_i="18247" txt_f="18356">]. It was reported that the point mutation A2224G might not contribute to the CLR resistance among Japanese [</offsets><xref ref-type="bibr" rid="B27"><offsets xml_i="30978" xml_f="30980" txt_i="18356" txt_f="18358">27</offsets></xref><offsets xml_i="30987" xml_f="31210" txt_i="18358" txt_f="18581">]. In the present study, we observed that the presence of the AAGGG → CTTCA (2222–2226) and AACC → GAAG (2081–2084) sequence alterations in OAC group were significantly higher than those in the OA and placebo groups (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="31242" xml_f="31243" txt_i="18581" txt_f="18582">1</offsets></xref><offsets xml_i="31250" xml_f="31431" txt_i="18582" txt_f="18763">). The A2143G mutation was detected only in 2 of the 11 AACC → GAAG (2081–2084) cases (18.2%). Whether these alterations contribute to CLR resistance should be investigated further.</offsets></p></sec><sec><title><offsets xml_i="31453" xml_f="31463" txt_i="18765" txt_f="18775">Conclusion</offsets></title><p><offsets xml_i="31474" xml_f="31507" txt_i="18776" txt_f="18809">The primary prevalence of the CLR</offsets><sup><offsets xml_i="31512" xml_f="31513" txt_i="18809" txt_f="18810">r</offsets></sup><offsets xml_i="31519" xml_f="31555" txt_i="18810" txt_f="18846">-related A2143G mutation within the </offsets><italic><offsets xml_i="31563" xml_f="31572" txt_i="18846" txt_f="18855">23S rRNA </offsets></italic><offsets xml_i="31581" xml_f="31740" txt_i="18855" txt_f="19014">gene as determined by a novel PCR-RFLP assay is approximately 10~14% among Chinese subjects without CLR use history. Application of CLR-containing regimen for </offsets><italic><offsets xml_i="31748" xml_f="31758" txt_i="19014" txt_f="19024">H. pylori </offsets></italic><offsets xml_i="31767" xml_f="31861" txt_i="19024" txt_f="19118">eradication could increase risk of prevalence of the A2143G mutation significantly (OR = 3~4).</offsets></p></sec><sec sec-type="methods"><title><offsets xml_i="31902" xml_f="31909" txt_i="19120" txt_f="19127">Methods</offsets></title><sec><title><offsets xml_i="31929" xml_f="31985" txt_i="19128" txt_f="19184">Study population and collection of gastric juice samples</offsets></title><p><offsets xml_i="31996" xml_f="32197" txt_i="19185" txt_f="19386">All subjects were from those enrolled in the intervention trial. The experimental design and population of this intervention trial (trial no. NCI-OH-95-C-N029) have been described in detail elsewhere [</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="32229" xml_f="32231" txt_i="19386" txt_f="19388">10</offsets></xref><offsets xml_i="32238" xml_f="32619" txt_i="19388" txt_f="19769">]. Fasting gastric juice samples were taken by a side-tube connected to vacuum device within the gastroendoscope at the end of the intervention trial in 2003. All subjects with more than 5 mL of gastric juice were eligible for the present study. Seven hundred and seventy of gastric juice samples were collected during the endoscope examination. In the OA group, 307 subjects with </offsets><italic><offsets xml_i="32627" xml_f="32637" txt_i="19769" txt_f="19779">H. pylori </offsets></italic><offsets xml_i="32646" xml_f="32837" txt_i="19779" txt_f="19970">seropositive were given amoxicillin (1000 mg) and omeprazole (20 mg) to take twice daily for two weeks in 1995, while 310 subjects with seropositive were given placebo capsules as described [</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="32869" xml_f="32871" txt_i="19970" txt_f="19972">10</offsets></xref><offsets xml_i="32878" xml_f="32923" txt_i="19972" txt_f="20017">]. One hundred and fifty three subjects with </offsets><italic><offsets xml_i="32931" xml_f="32941" txt_i="20017" txt_f="20027">H. pylori </offsets></italic><offsets xml_i="32950" xml_f="33014" txt_i="20027" txt_f="20091">seronegative at baseline but were subsequently determined to be </offsets><sup><offsets xml_i="33019" xml_f="33021" txt_i="20091" txt_f="20093">13</offsets></sup><offsets xml_i="33027" xml_f="33243" txt_i="20093" txt_f="20309">C-UBT-positive received a first-line triple therapy of CLR (500 mg), amoxicillin (1000 mg) and omeprazole (20 mg) to take twice daily for two weeks, according to instruction of the Ethic Committee in 2000. Status of </offsets><italic><offsets xml_i="33251" xml_f="33261" txt_i="20309" txt_f="20319">H. pylori </offsets></italic><offsets xml_i="33270" xml_f="33315" txt_i="20319" txt_f="20364">infection of each individual was detected by </offsets><sup><offsets xml_i="33320" xml_f="33322" txt_i="20364" txt_f="20366">13</offsets></sup><offsets xml_i="33328" xml_f="33352" txt_i="20366" txt_f="20390">C-UBT and ELISA assays [</offsets><xref ref-type="bibr" rid="B25"><offsets xml_i="33384" xml_f="33386" txt_i="20390" txt_f="20392">25</offsets></xref><offsets xml_i="33393" xml_f="33394" txt_i="20392" txt_f="20393">,</offsets><xref ref-type="bibr" rid="B26"><offsets xml_i="33426" xml_f="33428" txt_i="20393" txt_f="20395">26</offsets></xref><offsets xml_i="33435" xml_f="33437" txt_i="20395" txt_f="20397">].</offsets></p></sec><sec><title><offsets xml_i="33459" xml_f="33511" txt_i="20399" txt_f="20451">Extraction of genomic DNA from gastric juice samples</offsets></title><p><offsets xml_i="33522" xml_f="33746" txt_i="20452" txt_f="20676">To avoid self-decomposition, the shedded gastric mucosa was fixed immediately after collection of the samples at a local clinic in Linqu County. All the collected gastric juice specimens (5–30 mL) were centrifuged at 10,000 </offsets><italic><offsets xml_i="33754" xml_f="33756" txt_i="20676" txt_f="20678">g </offsets></italic><offsets xml_i="33765" xml_f="34148" txt_i="20678" txt_f="21061">for 10 min. After centrifugation, the supernatant liquid was removed leaving ~1 mL residue. The sample was then resuspended in 2 mL of FAA fixative (ethanol, acetic acid, and 40% formaldehyde at volume proportion of 9:1:1) and frozen at -20°C. The collected samples were shipped with dry ice and stored at -80°C at Etiology Laboratory, Beijing Institute for Cancer Research, Beijing.</offsets></p><p><offsets xml_i="34155" xml_f="34306" txt_i="21062" txt_f="21213">For DNA extraction, the frozen samples were thawed, and the fixative was removed with 2 × 5 mL aliquots of PBS. The samples were centrifuged at 10,000 </offsets><italic><offsets xml_i="34314" xml_f="34316" txt_i="21213" txt_f="21215">g </offsets></italic><offsets xml_i="34325" xml_f="34860" txt_i="21215" txt_f="21750">for 10 min and the supernatant liquid was removed after each PBS wash. The sediment was then resuspended in 5 mL of 10% SDS containing hyaluronidase (final concentration, 20 U/mL) (Sigma, Inc, USA) and incubated at 37°C for two hours to digest mucus. Proteinase K (final concentration, 20 U/mL) (Merck, Inc, Germany) was then added, and the sample was incubated at 55°C for more than three days until the desquamated tissue was digested completely. The DNA in the samples was extracted following a standard phenol chloroform procedure.</offsets></p></sec><sec><title><offsets xml_i="34882" xml_f="34903" txt_i="21752" txt_f="21773">Amplification of the </offsets><italic><offsets xml_i="34911" xml_f="34920" txt_i="21773" txt_f="21782">H. pylori</offsets></italic><offsets xml_i="34929" xml_f="34954" txt_i="21782" txt_f="21807">-specific 23S rRNA by PCR</offsets></title><p><offsets xml_i="34965" xml_f="34986" txt_i="21808" txt_f="21829">A 289 bp fragment of </offsets><italic><offsets xml_i="34994" xml_f="35004" txt_i="21829" txt_f="21839">H. pylori </offsets></italic><offsets xml_i="35013" xml_f="35072" txt_i="21839" txt_f="21898">chromosomal DNA was amplified by PCR as reported recently [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="35104" xml_f="35106" txt_i="21898" txt_f="21900">13</offsets></xref><offsets xml_i="35113" xml_f="35129" txt_i="21900" txt_f="21916">]. Briefly, the </offsets><italic><offsets xml_i="35137" xml_f="35146" txt_i="21916" txt_f="21925">23S rRNA </offsets></italic><offsets xml_i="35155" xml_f="35163" txt_i="21925" txt_f="21933">gene of </offsets><italic><offsets xml_i="35171" xml_f="35181" txt_i="21933" txt_f="21943">H. pylori </offsets></italic><offsets xml_i="35190" xml_f="35200" txt_i="21943" txt_f="21953">[GenBank: </offsets><ext-link ext-link-type="gen" xlink:href="U27270"><offsets xml_i="35250" xml_f="35256" txt_i="21953" txt_f="21959">U27270</offsets></ext-link><offsets xml_i="35267" xml_f="35437" txt_i="21959" txt_f="22129">] was amplified using the following primers: 5'-GCA TGA ATG GCG TAA CGA GAT G-3' and 5'-CCC AGT CAA ACT ACC CAC CAA G-3' (corresponding to 2049–2070 and 2316–2337 of the </offsets><italic><offsets xml_i="35445" xml_f="35454" txt_i="22129" txt_f="22138">23S rRNA </offsets></italic><offsets xml_i="35463" xml_f="35540" txt_i="22138" txt_f="22215">gene, respectively). The 289 bp PCR products were verified on 2% agarose gel.</offsets></p></sec><sec><title><offsets xml_i="35562" xml_f="35573" txt_i="22217" txt_f="22228">RFLP assays</offsets></title><p><offsets xml_i="35584" xml_f="35607" txt_i="22229" txt_f="22252">The 289 bp amplicon of </offsets><italic><offsets xml_i="35615" xml_f="35624" txt_i="22252" txt_f="22261">23S rRNA </offsets></italic><offsets xml_i="35633" xml_f="35675" txt_i="22261" txt_f="22303">was digested with the restriction enzymes </offsets><italic><offsets xml_i="35683" xml_f="35688" txt_i="22303" txt_f="22308">BsaI </offsets></italic><offsets xml_i="35697" xml_f="35701" txt_i="22308" txt_f="22312">and </offsets><italic><offsets xml_i="35709" xml_f="35715" txt_i="22312" txt_f="22318">MboII </offsets></italic><offsets xml_i="35724" xml_f="35823" txt_i="22318" txt_f="22417">(New England Biolabs, USA) in order to detect A2143G and A2142G point mutations, respectively (Fig </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="35853" xml_f="35854" txt_i="22417" txt_f="22418">1</offsets></xref><offsets xml_i="35861" xml_f="35864" txt_i="22418" txt_f="22421">) [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="35896" xml_f="35898" txt_i="22421" txt_f="22423">13</offsets></xref><offsets xml_i="35905" xml_f="35906" txt_i="22423" txt_f="22424">,</offsets><xref ref-type="bibr" rid="B27"><offsets xml_i="35938" xml_f="35940" txt_i="22424" txt_f="22426">27</offsets></xref><offsets xml_i="35947" xml_f="36027" txt_i="22426" txt_f="22506">]. An 8% PAGE gel was used to observe the restriction products. The amplicon of </offsets><italic><offsets xml_i="36035" xml_f="36051" txt_i="22506" txt_f="22522">H. pylori 26695 </offsets></italic><offsets xml_i="36060" xml_f="36104" txt_i="22522" txt_f="22566">strain was used as negative control. The CLR</offsets><sup><offsets xml_i="36109" xml_f="36110" txt_i="22566" txt_f="22567">r</offsets></sup><offsets xml_i="36116" xml_f="36140" txt_i="22567" txt_f="22591">-1 strain selected from </offsets><italic><offsets xml_i="36148" xml_f="36164" txt_i="22591" txt_f="22607">H. pylori 26695 </offsets></italic><offsets xml_i="36173" xml_f="36255" txt_i="22607" txt_f="22689">[by CLR] contained an A2143G point mutation and was used as one positive control [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="36287" xml_f="36289" txt_i="22689" txt_f="22691">13</offsets></xref><offsets xml_i="36296" xml_f="36360" txt_i="22691" txt_f="22755">]. An A2142G point mutation was introduced into the amplicon of </offsets><italic><offsets xml_i="36368" xml_f="36384" txt_i="22755" txt_f="22771">H. pylori 26695 </offsets></italic><offsets xml_i="36393" xml_f="36498" txt_i="22771" txt_f="22876">by mutation PCR. One of the A2142G clones (confirmed by sequencing) was used as another positive control.</offsets></p></sec><sec><title><offsets xml_i="36520" xml_f="36534" txt_i="22878" txt_f="22892">DNA sequencing</offsets></title><p><offsets xml_i="36545" xml_f="36561" txt_i="22893" txt_f="22909">PCR products of </offsets><italic><offsets xml_i="36569" xml_f="36588" txt_i="22909" txt_f="22928">H. pylori 23S rRNA </offsets></italic><offsets xml_i="36597" xml_f="36696" txt_i="22928" txt_f="23027">were sequenced by an ABI Prism 377 sequencer (PE Biosystems, USA) with fluorescent dye terminators.</offsets></p></sec><sec><title><offsets xml_i="36718" xml_f="36738" txt_i="23029" txt_f="23049">Statistical analysis</offsets></title><p><offsets xml_i="36749" xml_f="36780" txt_i="23050" txt_f="23081">For statistical analysis, the χ</offsets><sup><offsets xml_i="36785" xml_f="36787" txt_i="23081" txt_f="23083">2 </offsets></sup><offsets xml_i="36793" xml_f="36854" txt_i="23083" txt_f="23144">test was used to evaluate the significance of differences of </offsets><italic><offsets xml_i="36862" xml_f="36872" txt_i="23144" txt_f="23154">H. pylori </offsets></italic><offsets xml_i="36881" xml_f="36898" txt_i="23154" txt_f="23171">infection and CLR</offsets><sup><offsets xml_i="36903" xml_f="36905" txt_i="23171" txt_f="23173">r </offsets></sup><offsets xml_i="36911" xml_f="37097" txt_i="23173" txt_f="23356">frequency between CLR and control groups. All p-values were two-sided and p &lt; 0.05 was considered statistically significant. All analyses were performed with SAS software version 8.0.</offsets></p></sec></sec><sec><title><offsets xml_i="37125" xml_f="37147" txt_i="23359" txt_f="23381">Authors' contributions</offsets></title><p><offsets xml_i="37158" xml_f="37324" txt_i="23382" txt_f="23548">ZL carried out experiments including DNA extraction, detection of 23S rRNA and point mutations by PCR-RFLP. JS setup the PCR-RFLP assay. LZ carried out the ELISA and </offsets><sup><offsets xml_i="37329" xml_f="37331" txt_i="23548" txt_f="23550">13</offsets></sup><offsets xml_i="37337" xml_f="37759" txt_i="23550" txt_f="23972">C-UBT assays. LS collected the gastric juice samples by endoscope. QL validated the quality of DNA from the samples. BZ, JZ, and LG pretreated the samples in the local clinic. GF performed the statistical analysis. JM collected patients' followup data. W-CY conceived the idea of the intervention trial and advised on the manuscript. DD conceived the idea of the present study, analyzed the data, and wrote the manuscript.</offsets></p></sec></body><back><ack><sec><title>Acknowledgements</title><p>This work was supported by grant# 7002008 from Beijing Natural Science Foundation. We appreciate Dr. Benjamin Legendre Jr. (Transgenomic, Inc.) for language editing.</p></sec></ack><ref-list><ref id="B1"><citation citation-type="book"><person-group person-group-type="author"><collab>IARC</collab></person-group><source>Monographs on the evaluation of carcinogenic risks to humans Vol 61 Schistosomes, liver flukes and Helicobacter pylori</source><year>1994</year><publisher-name>Lyon, France</publisher-name></citation></ref><ref id="B2"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Goodwin</surname><given-names>CS</given-names></name><name><surname>Armstring</surname><given-names>JA</given-names></name><name><surname>Marshall</surname><given-names>BJ</given-names></name></person-group><article-title><italic>Campylobacter pyloridis</italic>, gastritis and peptic ulceration</article-title><source>J Clin Pathol</source><year>1986</year><volume>39</volume><fpage>353</fpage><lpage>165</lpage><pub-id pub-id-type="pmid">3517070</pub-id><pub-id pub-id-type="doi">10.1136/jcp.39.4.353</pub-id></citation></ref><ref id="B3"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Edit</surname><given-names>S</given-names></name><name><surname>Stolte</surname><given-names>M</given-names></name><name><surname>Fischer</surname><given-names>R</given-names></name></person-group><article-title><italic>Helicobacter pylori </italic>gastritis and primary gastric non-Hodgkin's lymphomas</article-title><source>J Clin Pathol</source><year>1994</year><volume>47</volume><fpage>436</fpage><lpage>439</lpage><pub-id pub-id-type="pmid">8027397</pub-id><pub-id pub-id-type="doi">10.1136/jcp.47.5.436</pub-id></citation></ref><ref id="B4"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Sugano</surname><given-names>K</given-names></name></person-group><article-title>Prevention of gastric cancer: urgent need to implement <italic>Helicobacter pylori </italic>eradication therapy as a primary preventive measure in Japan</article-title><source>J Gastroenterol</source><year>2007</year><fpage>1</fpage><lpage>2</lpage><pub-id pub-id-type="pmid">17238016</pub-id><pub-id pub-id-type="doi">10.1007/s00535-006-1941-8</pub-id></citation></ref><ref id="B5"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Cheung</surname><given-names>TK</given-names></name><name><surname>Xia</surname><given-names>HHX</given-names></name><name><surname>Wong</surname><given-names>BCY</given-names></name></person-group><article-title><italic>Helicobacter pylori </italic>eradication for gastric cancer prevention</article-title><source>J Gastroenterol</source><year>2007</year><fpage>10</fpage><lpage>15</lpage><pub-id pub-id-type="pmid">17238019</pub-id><pub-id pub-id-type="doi">10.1007/s00535-006-1939-2</pub-id></citation></ref><ref id="B6"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Satoh</surname><given-names>K</given-names></name></person-group><article-title>Indications for <italic>Helicobacter pylori </italic>eradication therapy and first-line therapy regimen in Japan: recommendation by the Japanese Society for <italic>Helicobacter </italic>Research</article-title><source>J Gastroenterol</source><year>2002</year><volume>37</volume><fpage>34</fpage><lpage>38</lpage><pub-id pub-id-type="pmid">12109663</pub-id></citation></ref><ref id="B7"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Dupas</surname><given-names>JL</given-names></name></person-group><article-title>First line therapy for <italic>Helicobacter pylori </italic>eradication in France</article-title><source>Gastroenterol Clin Biol</source><year>2003</year><volume>27</volume><fpage>467</fpage><lpage>472</lpage><pub-id pub-id-type="pmid">12700504</pub-id></citation></ref><ref id="B8"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Mégraud</surname><given-names>F</given-names></name></person-group><article-title>H pylori antibiotic resistance: prevalence, importance, and advances in testing</article-title><source>Gut</source><year>2004</year><volume>53</volume><fpage>1374</fpage><lpage>1384</lpage><pub-id pub-id-type="pmid">15306603</pub-id><pub-id pub-id-type="doi">10.1136/gut.2003.022111</pub-id></citation></ref><ref id="B9"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Gerrits</surname><given-names>MM</given-names></name><name><surname>van Vliet</surname><given-names>AHM</given-names></name><name><surname>Kuipers</surname><given-names>EJ</given-names></name><name><surname>Kusters</surname><given-names>JG</given-names></name></person-group><article-title>Helicobacter pylori and antimicrobial resistance: molecular mechanisms and clinical implications</article-title><source>Lancet Infect Dis</source><year>2006</year><volume>6</volume><fpage>699</fpage><lpage>709</lpage><pub-id pub-id-type="pmid">17067919</pub-id><pub-id pub-id-type="doi">10.1016/S1473-3099(06)70627-2</pub-id></citation></ref><ref id="B10"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>You</surname><given-names>WC</given-names></name><name><surname>Brown</surname><given-names>LM</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Li</surname><given-names>JY</given-names></name><name><surname>Jin</surname><given-names>ML</given-names></name><name><surname>Chang</surname><given-names>YS</given-names></name><name><surname>Ma</surname><given-names>JL</given-names></name><name><surname>Pan</surname><given-names>KF</given-names></name><name><surname>Liu</surname><given-names>WD</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><name><surname>Crystal-Mansour</surname><given-names>S</given-names></name><name><surname>Pee</surname><given-names>D</given-names></name><name><surname>Blot</surname><given-names>WJ</given-names></name><name><surname>Fraumeni</surname><given-names>JF</given-names></name><name><surname>Xu</surname><given-names>GW</given-names></name><name><surname>Gail</surname><given-names>MH</given-names></name></person-group><article-title>Randomized double-blind factorial trial of three treatments to reduce the prevalence of precancerous gastric lesions</article-title><source>J Natl Cancer Inst</source><year>2006</year><volume>98</volume><fpage>974</fpage><lpage>983</lpage><pub-id pub-id-type="pmid">16849680</pub-id></citation></ref><ref id="B11"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Versalovic</surname><given-names>J</given-names></name><name><surname>Shortridge</surname><given-names>D</given-names></name><name><surname>Kibler</surname><given-names>K</given-names></name><name><surname>Griffy</surname><given-names>MV</given-names></name><name><surname>Beyer</surname><given-names>J</given-names></name><name><surname>Flamm</surname><given-names>RK</given-names></name><name><surname>Tanaka</surname><given-names>SK</given-names></name><name><surname>Graham</surname><given-names>DY</given-names></name><name><surname>Go</surname><given-names>MF</given-names></name></person-group><article-title>Mutations in 23S rRNA are associated with clarithromycin resistance in <italic>Helicobacter pylori</italic></article-title><source>Antimicrob Agents Chemother</source><year>1996</year><volume>40</volume><fpage>477</fpage><lpage>480</lpage><pub-id pub-id-type="pmid">8834903</pub-id></citation></ref><ref id="B12"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Stone</surname><given-names>GG</given-names></name><name><surname>Shortridge</surname><given-names>D</given-names></name><name><surname>Versalovic</surname><given-names>J</given-names></name><name><surname>Beyer</surname><given-names>J</given-names></name><name><surname>Flamm</surname><given-names>RK</given-names></name><name><surname>Graham</surname><given-names>DY</given-names></name><name><surname>Ghoneim</surname><given-names>AT</given-names></name><name><surname>Tanaka</surname><given-names>SK</given-names></name></person-group><article-title>A PCR-oligonucleotide ligation assay to determine the prevalence of 23S rRNA gene mutations clarithromycin-resistant <italic>Helicobacter pylori</italic></article-title><source>Antimicrob Agents Chemother</source><year>1997</year><volume>41</volume><fpage>712</fpage><lpage>714</lpage><pub-id pub-id-type="pmid">9056021</pub-id></citation></ref><ref id="B13"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Shen</surname><given-names>J</given-names></name><name><surname>Zhang</surname><given-names>J-Z</given-names></name><name><surname>Ke</surname><given-names>Y</given-names></name><name><surname>Deng</surname><given-names>D</given-names></name></person-group><article-title>Formation of A2143G mutation of <italic>23S rRNA </italic>in progression of clarithromycin-resistance in <italic>Helicobacter pylori </italic>26695</article-title><source>Microb Drug Resist</source><year>2005</year><volume>11</volume><fpage>100</fpage><lpage>106</lpage><pub-id pub-id-type="pmid">15910222</pub-id><pub-id pub-id-type="doi">10.1089/mdr.2005.11.100</pub-id></citation></ref><ref id="B14"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Pan</surname><given-names>ZJ</given-names></name><name><surname>Su</surname><given-names>WW</given-names></name><name><surname>Tytgat</surname><given-names>GN</given-names></name><name><surname>Dankert</surname><given-names>J</given-names></name><name><surname>Ende</surname><given-names>A van der</given-names></name></person-group><article-title>Assessment of clarithromycin-resistant <italic>Helicobacter pylori </italic>among patients in Shanghai and Guangzhou, China, by primer-mismatch PCR</article-title><source>J Clin Microbiol</source><year>2002</year><volume>40</volume><fpage>259</fpage><lpage>261</lpage><pub-id pub-id-type="pmid">11773126</pub-id><pub-id pub-id-type="doi">10.1128/JCM.40.1.259-261.2002</pub-id></citation></ref><ref id="B15"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Zheng</surname><given-names>X</given-names></name><name><surname>Hu</surname><given-names>F</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name></person-group><article-title>The prevalence and mechanism of <italic>Helicobacter pylori </italic>resistance to clarithromycin in Beijing</article-title><source>Zhonghua YiXue ZaZhi</source><year>2001</year><volume>81</volume><fpage>1413</fpage><lpage>1415</lpage><pub-id pub-id-type="pmid">11930609</pub-id></citation></ref><ref id="B16"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Yu</surname><given-names>JD</given-names></name><name><surname>Huang</surname><given-names>XL</given-names></name><name><surname>Chen</surname><given-names>FB</given-names></name><name><surname>Li</surname><given-names>ZY</given-names></name><name><surname>Zhang</surname><given-names>XP</given-names></name></person-group><article-title>Prevalence and mechanism of Helicobacter pylori resistance to clarithromycin in children</article-title><source>Zhonghua Er Ke Za Zhi</source><year>2004</year><volume>42</volume><fpage>850</fpage><lpage>853</lpage><pub-id pub-id-type="pmid">15631715</pub-id></citation></ref><ref id="B17"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Heep</surname><given-names>M</given-names></name><name><surname>Kist</surname><given-names>M</given-names></name><name><surname>Strobel</surname><given-names>S</given-names></name><name><surname>Beck</surname><given-names>D</given-names></name><name><surname>Lehn</surname><given-names>N</given-names></name></person-group><article-title>Secondary resistance among 554 isolates of <italic>Helicobacter pylori </italic>after failure of therapy</article-title><source>Eur J Clin Microbiol Infect Dis</source><year>2000</year><volume>19</volume><fpage>538</fpage><lpage>541</lpage><pub-id pub-id-type="pmid">10968325</pub-id><pub-id pub-id-type="doi">10.1007/s100960000288</pub-id></citation></ref><ref id="B18"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Maeda</surname><given-names>S</given-names></name><name><surname>Yoshida</surname><given-names>H</given-names></name><name><surname>Ogura</surname><given-names>K</given-names></name><name><surname>Kanai</surname><given-names>F</given-names></name><name><surname>Shiratori</surname><given-names>Y</given-names></name><name><surname>Omata</surname><given-names>M</given-names></name></person-group><article-title><italic>Helicobacter pylori </italic>specific nested PCR assay for the detection of <italic>23S rRNA </italic>mutation associated with clarithromycin resistance</article-title><source>Gut</source><year>1998</year><volume>43</volume><fpage>317</fpage><lpage>321</lpage><pub-id pub-id-type="pmid">9863474</pub-id></citation></ref><ref id="B19"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Akopyanz</surname><given-names>N</given-names></name><name><surname>Bukanov</surname><given-names>NO</given-names></name><name><surname>Westblom</surname><given-names>TU</given-names></name><name><surname>Berg</surname><given-names>DE</given-names></name></person-group><article-title>PCR-based RFLP analysis of DNA sequence diversity in the gastric pathogen <italic>Helicobacter pylori</italic></article-title><source>Nucleic Acids Res</source><year>1992</year><volume>20</volume><fpage>6221</fpage><lpage>6225</lpage><pub-id pub-id-type="pmid">1361982</pub-id><pub-id pub-id-type="doi">10.1093/nar/20.23.6221</pub-id></citation></ref><ref id="B20"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Krogfelt</surname><given-names>KA</given-names></name><name><surname>Lehours</surname><given-names>P</given-names></name><name><surname>Megraud</surname><given-names>F</given-names></name></person-group><article-title>Diagnosis of <italic>Helicobacter pylori </italic>infection</article-title><source>Helicobacter</source><year>2005</year><fpage>5</fpage><lpage>13</lpage><pub-id pub-id-type="pmid">16178965</pub-id><pub-id pub-id-type="doi">10.1111/j.1523-5378.2005.00341.x</pub-id></citation></ref><ref id="B21"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Vaira</surname><given-names>D</given-names></name><name><surname>Holton</surname><given-names>J</given-names></name><name><surname>Menegatti</surname><given-names>M</given-names></name><name><surname>Ricci</surname><given-names>C</given-names></name><name><surname>Gatta</surname><given-names>L</given-names></name><name><surname>Geminiani</surname><given-names>A</given-names></name><name><surname>Miglioli</surname><given-names>M</given-names></name></person-group><article-title>Review article: invasive and non-invasive tests for Helicobacter pylori infection</article-title><source>Aliment Pharmacol Ther</source><year>2000</year><fpage>13</fpage><lpage>22</lpage><pub-id pub-id-type="pmid">11050483</pub-id><pub-id pub-id-type="doi">10.1046/j.1365-2036.2000.00096.x</pub-id></citation></ref><ref id="B22"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Yang</surname><given-names>YJ</given-names></name><name><surname>Yang</surname><given-names>JC</given-names></name><name><surname>Jeng</surname><given-names>YM</given-names></name><name><surname>Chang</surname><given-names>MH</given-names></name><name><surname>Ni</surname><given-names>YH</given-names></name></person-group><article-title>Prevalence and rapid identification of clarithromycin-resistant <italic>Helicobacter pylori </italic>isolates in children</article-title><source>Pediatr Infect Dis J</source><year>2001</year><volume>20</volume><fpage>662</fpage><lpage>666</lpage><pub-id pub-id-type="pmid">11465837</pub-id><pub-id pub-id-type="doi">10.1097/00006454-200107000-00005</pub-id></citation></ref><ref id="B23"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Tsiodras</surname><given-names>S</given-names></name><name><surname>Gold</surname><given-names>HS</given-names></name><name><surname>Coakley</surname><given-names>EPG</given-names></name><name><surname>Wennersten</surname><given-names>C</given-names></name><name><surname>Moellering</surname><given-names>RC</given-names></name><name><surname>Eliopoulos</surname><given-names>GM</given-names></name></person-group><article-title>Diversity of domain V of 23S rRNA gene sequence in different enterococcus species</article-title><source>J Clin Microbiol</source><year>2000</year><volume>38</volume><fpage>3991</fpage><lpage>3993</lpage><pub-id pub-id-type="pmid">11060057</pub-id></citation></ref><ref id="B24"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Weisblum</surname><given-names>B</given-names></name></person-group><article-title>Erythromycin resistance by ribosome modification</article-title><source>Antimicrob Agents Chemother</source><year>1995</year><volume>39</volume><fpage>577</fpage><lpage>585</lpage><pub-id pub-id-type="pmid">7793855</pub-id></citation></ref><ref id="B25"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>You</surname><given-names>WC</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Gail</surname><given-names>MH</given-names></name><name><surname>Ma</surname><given-names>JL</given-names></name><name><surname>Chang</surname><given-names>YS</given-names></name><name><surname>Blot</surname><given-names>WJ</given-names></name><name><surname>Li</surname><given-names>JY</given-names></name><name><surname>Zhao</surname><given-names>CL</given-names></name><name><surname>Liu</surname><given-names>WD</given-names></name><name><surname>Li</surname><given-names>HQ</given-names></name><name><surname>Hu</surname><given-names>YR</given-names></name><name><surname>Bravo</surname><given-names>JC</given-names></name><name><surname>Correa</surname><given-names>P</given-names></name><name><surname>Xu</surname><given-names>GW</given-names></name><name><surname>Fraumeni</surname><given-names>JF</given-names></name></person-group><article-title><italic>Helicobacter pylori infection</italic>, garlic intake and precancerous lesions in a Chinese population at low risk of gastric cancer</article-title><source>Int J Epidemiol</source><year>1998</year><volume>27</volume><fpage>941</fpage><lpage>944</lpage><pub-id pub-id-type="pmid">10024185</pub-id><pub-id pub-id-type="doi">10.1093/ije/27.6.941</pub-id></citation></ref><ref id="B26"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Shen</surname><given-names>L</given-names></name><name><surname>Ma</surname><given-names>JL</given-names></name><name><surname>Pan</surname><given-names>KF</given-names></name><name><surname>Liu</surname><given-names>WD</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Xiao</surname><given-names>SD</given-names></name><name><surname>Lin</surname><given-names>SR</given-names></name><name><surname>Classen</surname><given-names>M</given-names></name><name><surname>You</surname><given-names>WC</given-names></name></person-group><article-title>Eradication of <italic>H pylori </italic>infection in a rural population: one-day quadruple therapy versus 7-day triple therapy</article-title><source>World J Gastroenterol</source><year>2006</year><volume>12</volume><fpage>3915</fpage><lpage>3918</lpage><pub-id pub-id-type="pmid">16804982</pub-id></citation></ref><ref id="B27"><citation citation-type="journal"><person-group person-group-type="author"><name><surname>Maeda</surname><given-names>S</given-names></name><name><surname>Yoshida</surname><given-names>H</given-names></name><name><surname>Ogura</surname><given-names>K</given-names></name><name><surname>Kanai</surname><given-names>F</given-names></name><name><surname>Shiratori</surname><given-names>Y</given-names></name><name><surname>Omata</surname><given-names>M</given-names></name></person-group><article-title><italic>Helicobacter pylori </italic>specific nested PCR assay for the detection of <italic>23S rRNA </italic>mutation associated with clarithromycin resistance</article-title><source>Gut</source><year>1998</year><volume>43</volume><fpage>317</fpage><lpage>321</lpage><pub-id pub-id-type="pmid">9863474</pub-id></citation></ref></ref-list></back></article>