<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Breast Cancer Res</journal-id><journal-title-group><journal-title>Breast Cancer Research : BCR</journal-title></journal-title-group><issn pub-type="ppub">1465-5411</issn><issn pub-type="epub">1465-542X</issn><publisher><publisher-name>BioMed Central</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">20649975</article-id><article-id pub-id-type="pmc">2949642</article-id><article-id pub-id-type="publisher-id">bcr2612</article-id><article-id pub-id-type="doi">10.1186/bcr2612</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>Predictive relevance of HOXB13 protein expression for tamoxifen benefit in breast cancer</article-title></title-group><contrib-group><contrib contrib-type="author" corresp="yes" id="A1"><name><surname>Jerevall</surname><given-names>Piiha-Lotta</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>piiha-lotta.jerevall@liu.se</email></contrib><contrib contrib-type="author" id="A2"><name><surname>Jansson</surname><given-names>Agneta</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>agneta.jansson@liu.se</email></contrib><contrib contrib-type="author" id="A3"><name><surname>Fornander</surname><given-names>Tommy</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>tommy.fornander@karolinska.se</email></contrib><contrib contrib-type="author" id="A4"><name><surname>Skoog</surname><given-names>Lambert</given-names></name><xref ref-type="aff" rid="I3">3</xref><email>lambert.skoog@karolinska.se</email></contrib><contrib contrib-type="author" id="A5"><name><surname>Nordenskjöld</surname><given-names>Bo</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>bo.nordenskjold@liu.se</email></contrib><contrib contrib-type="author" id="A6"><name><surname>Stål</surname><given-names>Olle</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>olle.stal@liu.se</email></contrib></contrib-group><aff id="I1"><label>1</label>Department of Clinical and Experimental Medicine, Division of Oncology, Faculty of Health Sciences, Linköping University, SE-58185 Linköping, Sweden</aff><aff id="I2"><label>2</label>Department of Oncology, Karolinska University Hospital, Stockholm South General Hospital, Sjukhusbacken 10, SE-11883 Stockholm, Sweden</aff><aff id="I3"><label>3</label>Department of Clinical Pathology and Cytology, Karolinska University Hospital, Solna, SE-17176 Stockholm, Sweden</aff><pub-date pub-type="ppub"><year>2010</year></pub-date><pub-date pub-type="epub"><day>22</day><month>7</month><year>2010</year></pub-date><volume>12</volume><issue>4</issue><fpage>R53</fpage><lpage>R53</lpage><history><date date-type="received"><day>15</day><month>3</month><year>2010</year></date><date date-type="rev-recd"><day>8</day><month>6</month><year>2010</year></date><date date-type="accepted"><day>22</day><month>7</month><year>2010</year></date></history><permissions><copyright-statement>Copyright ©2010 Jerevall et al.; licensee BioMed Central Ltd.</copyright-statement><copyright-year>2010</copyright-year><copyright-holder>Jerevall et al.; licensee BioMed Central Ltd.</copyright-holder><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"><license-p>This is an open access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</license-p></license></permissions><self-uri xlink:href="http://breast-cancer-research.com/content/12/4/R53"></self-uri><abstract><sec><title><offsets xml_i="4037" xml_f="4049" txt_i="11" txt_f="23">Introduction</offsets></title><p><offsets xml_i="4060" xml_f="4064" txt_i="24" txt_f="28">The </offsets><italic><offsets xml_i="4072" xml_f="4086" txt_i="28" txt_f="42">HOXB13:IL17BR </offsets></italic><offsets xml_i="4095" xml_f="4248" txt_i="42" txt_f="195">index has been identified to predict clinical outcome in the setting of adjuvant tamoxifen monotherapy of breast cancer. Further studies have shown that </offsets><italic><offsets xml_i="4256" xml_f="4263" txt_i="195" txt_f="202">HOXB13 </offsets></italic><offsets xml_i="4272" xml_f="4436" txt_i="202" txt_f="366">in particular can indicate benefit of prolonged tamoxifen treatment. Patients with high-expressing tumors did not benefit from prolonged treatment, suggesting that </offsets><italic><offsets xml_i="4444" xml_f="4451" txt_i="366" txt_f="373">HOXB13 </offsets></italic><offsets xml_i="4460" xml_f="4716" txt_i="373" txt_f="629">might be involved in tamoxifen resistance. No studies have been made regarding the HOXB13 protein levels in breast cancer. The aim of our study was to investigate whether tamoxifen benefit can be correlated to different levels of HOXB13 protein expression.</offsets></p></sec><sec><title><offsets xml_i="4738" xml_f="4745" txt_i="631" txt_f="638">Methods</offsets></title><p><offsets xml_i="4756" xml_f="4966" txt_i="639" txt_f="849">We used immunohistochemistry to analyze protein levels of HOXB13 in tumor samples from 912 postmenopausal node-negative breast cancer patients randomized to adjuvant tamoxifen therapy or no endocrine treatment.</offsets></p></sec><sec><title><offsets xml_i="4988" xml_f="4995" txt_i="851" txt_f="858">Results</offsets></title><p><offsets xml_i="5006" xml_f="5268" txt_i="859" txt_f="1121">Tamoxifen-treated patients with estrogen receptor-positive tumors expressing none or low levels of HOXB13 had a clear benefit from tamoxifen in terms of longer distant recurrence-free survival (DRFS) (hazard ratio = 0.38, 95% confidence interval = 0.23 to 0.60, </offsets><italic><offsets xml_i="5276" xml_f="5278" txt_i="1121" txt_f="1123">P </offsets></italic><offsets xml_i="5287" xml_f="5509" txt_i="1123" txt_f="1345">= 0.000048). However, for patients with a high or intermediate HOXB13 tumor expression, tamoxifen did not prolong the DRFS compared with the untreated patients (hazard ratio = 0.88, 95% confidence interval = 0.47 to 1.65, </offsets><italic><offsets xml_i="5517" xml_f="5519" txt_i="1345" txt_f="1347">P </offsets></italic><offsets xml_i="5528" xml_f="5642" txt_i="1347" txt_f="1461">= 0.69). Interaction between HOXB13 expression and benefit from tamoxifen was statistically significant for DRFS (</offsets><italic><offsets xml_i="5650" xml_f="5652" txt_i="1461" txt_f="1463">P </offsets></italic><offsets xml_i="5661" xml_f="5757" txt_i="1463" txt_f="1559">= 0.035). No prognostic value could be ascribed to HOXB13 among systemically untreated patients.</offsets></p></sec><sec><title><offsets xml_i="5779" xml_f="5790" txt_i="1561" txt_f="1572">Conclusions</offsets></title><p><offsets xml_i="5801" xml_f="5981" txt_i="1573" txt_f="1753">A high HOXB13 expression was associated with decreased benefit from tamoxifen, which indicates that HOXB13 protein level may be used as a predictive marker for tamoxifen treatment.</offsets></p></sec></abstract></article-meta></front><body><sec><title><offsets xml_i="6043" xml_f="6055" txt_i="1762" txt_f="1774">Introduction</offsets></title><p><offsets xml_i="6066" xml_f="6256" txt_i="1775" txt_f="1965">There have been several recent studies aimed at discovering novel biomarkers and gene signatures usable for predicting risk of recurrence and response to endocrine therapy of breast cancer [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="6287" xml_f="6288" txt_i="1965" txt_f="1966">1</offsets></xref><offsets xml_i="6295" xml_f="6296" txt_i="1966" txt_f="1967">-</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="6327" xml_f="6328" txt_i="1967" txt_f="1968">4</offsets></xref><offsets xml_i="6335" xml_f="6922" txt_i="1968" txt_f="2555">]. With the development of robust, reliable genetic markers for this purpose, it would be possible at an early stage to predict which patients would benefit from alternative hormonal therapies. Resulting gene signatures, based on genome-wide microarray analyses, are often very comprehensive and comprise a large number of genes. However, Ma and colleagues were able to show that their gene expression profiles of hormone receptor-positive invasive breast tumors could be reduced into a simple two-gene ratio predictive of tumor relapse in the setting of adjuvant tamoxifen monotherapy [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="6953" xml_f="6954" txt_i="2555" txt_f="2556">5</offsets></xref><offsets xml_i="6961" xml_f="6990" txt_i="2556" txt_f="2585">]. Subsequent studies of the </offsets><italic><offsets xml_i="6998" xml_f="7012" txt_i="2585" txt_f="2599">HOXB13:IL17BR </offsets></italic><offsets xml_i="7021" xml_f="7127" txt_i="2599" txt_f="2705">index have proven its significance in predicting risk of breast cancer recurrence and tamoxifen response [</offsets><xref ref-type="bibr" rid="B6"><offsets xml_i="7158" xml_f="7159" txt_i="2705" txt_f="2706">6</offsets></xref><offsets xml_i="7166" xml_f="7167" txt_i="2706" txt_f="2707">-</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="7198" xml_f="7199" txt_i="2707" txt_f="2708">9</offsets></xref><offsets xml_i="7206" xml_f="7208" txt_i="2708" txt_f="2710">].</offsets></p><p><offsets xml_i="7215" xml_f="7241" txt_i="2711" txt_f="2737">Our previous study of the </offsets><italic><offsets xml_i="7249" xml_f="7263" txt_i="2737" txt_f="2751">HOXB13:IL17BR </offsets></italic><offsets xml_i="7272" xml_f="7420" txt_i="2751" txt_f="2899">expression ratio indicated that the two genes individually could function as separate prognostic and treatment predictive markers in breast cancer [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="7451" xml_f="7452" txt_i="2899" txt_f="2900">8</offsets></xref><offsets xml_i="7459" xml_f="7476" txt_i="2900" txt_f="2917">]. Expression of </offsets><italic><offsets xml_i="7484" xml_f="7491" txt_i="2917" txt_f="2924">IL17BR </offsets></italic><offsets xml_i="7500" xml_f="7585" txt_i="2924" txt_f="3009">was inversely correlated to a number of factors related to a poor prognosis, whereas </offsets><italic><offsets xml_i="7593" xml_f="7600" txt_i="3009" txt_f="3016">HOXB13 </offsets></italic><offsets xml_i="7609" xml_f="7713" txt_i="3016" txt_f="3120">could predict recurrence in tamoxifen-treated patients. Patients with tumors expressing a high level of </offsets><italic><offsets xml_i="7721" xml_f="7728" txt_i="3120" txt_f="3127">HOXB13 </offsets></italic><offsets xml_i="7737" xml_f="7851" txt_i="3127" txt_f="3241">were more likely to be unresponsive to the therapy, suggesting that this gene is involved in tamoxifen resistance.</offsets></p><p><italic><offsets xml_i="7866" xml_f="7873" txt_i="3242" txt_f="3249">HOXB13 </offsets></italic><offsets xml_i="7882" xml_f="8136" txt_i="3249" txt_f="3503">is a member of the homeobox gene family, a group of genes encoding transcriptional regulators of cell growth and differentiation, predominantly during embryogenesis. Much is known about the function of the homeobox genes in these events, but the role of </offsets><italic><offsets xml_i="8144" xml_f="8151" txt_i="3503" txt_f="3510">HOXB13 </offsets></italic><offsets xml_i="8160" xml_f="8256" txt_i="3510" txt_f="3606">in breast cancer and endocrine resistance is only beginning to be elucidated. The expression of </offsets><italic><offsets xml_i="8264" xml_f="8271" txt_i="3606" txt_f="3613">HOXB13 </offsets></italic><offsets xml_i="8280" xml_f="8370" txt_i="3613" txt_f="3703">is known to be upregulated in breast cancer cells compared with normal breast epithelium [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="8401" xml_f="8402" txt_i="3703" txt_f="3704">5</offsets></xref><offsets xml_i="8409" xml_f="8410" txt_i="3704" txt_f="3705">,</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="8442" xml_f="8444" txt_i="3705" txt_f="3707">10</offsets></xref><offsets xml_i="8451" xml_f="8485" txt_i="3707" txt_f="3741">] and it has also been shown that </offsets><italic><offsets xml_i="8493" xml_f="8500" txt_i="3741" txt_f="3748">HOXB13 </offsets></italic><offsets xml_i="8509" xml_f="8595" txt_i="3748" txt_f="3834">is an estrogen-regulated gene negatively correlated to estrogen receptor (ER) status [</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="8627" xml_f="8629" txt_i="3834" txt_f="3836">11</offsets></xref><offsets xml_i="8636" xml_f="8637" txt_i="3836" txt_f="3837">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="8669" xml_f="8671" txt_i="3837" txt_f="3839">12</offsets></xref><offsets xml_i="8678" xml_f="8721" txt_i="3839" txt_f="3882">]. Wang and colleagues suggest that a high </offsets><italic><offsets xml_i="8729" xml_f="8743" txt_i="3882" txt_f="3896">HOXB13:IL17BR </offsets></italic><offsets xml_i="8752" xml_f="8845" txt_i="3896" txt_f="3989">index may indicate impaired ER signaling, which is known to predict resistance to tamoxifen [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="8877" xml_f="8879" txt_i="3989" txt_f="3991">12</offsets></xref><offsets xml_i="8886" xml_f="8888" txt_i="3991" txt_f="3993">].</offsets></p><p><offsets xml_i="8895" xml_f="9409" txt_i="3994" txt_f="4508">To our knowledge, there are no studies investigating the HOXB13 protein levels in breast cancer and its significance in predicting outcome after tamoxifen treatment. In the present study, we used immunohistochemistry to analyze the protein expression of HOXB13 in tumor samples from 912 postmenopausal breast cancer patients. The patients were participants in a randomized trial analyzing the benefit from adjuvant treatment with tamoxifen, which enabled us to investigate the treatment predictive value of HOXB13.</offsets></p></sec><sec sec-type="materials|methods"><title><offsets xml_i="9460" xml_f="9481" txt_i="4510" txt_f="4531">Materials and methods</offsets></title><sec><title><offsets xml_i="9501" xml_f="9509" txt_i="4532" txt_f="4540">Patients</offsets></title><p><offsets xml_i="9520" xml_f="9724" txt_i="4541" txt_f="4745">We analyzed tumor tissue from patients enrolled in a randomized tamoxifen trial comprising a total of 1,780 low-risk breast cancer patients, conducted in the Stockholm region in Sweden from 1976 to 1990 [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="9756" xml_f="9758" txt_i="4745" txt_f="4747">13</offsets></xref><offsets xml_i="9765" xml_f="10193" txt_i="4747" txt_f="5175">]. All patients were postmenopausal at the time of diagnosis, presented a tumor size ≤30 mm, and displayed no nodal involvement (N0). The patients were randomized to 2 years of endocrine treatment with tamoxifen (40 mg daily) or no adjuvant endocrine treatment. In 1983 a new trial was initiated; recurrence-free patients were, after 2 years of tamoxifen treatment, randomized to 3 years more of tamoxifen or no further therapy.</offsets></p><p><offsets xml_i="10200" xml_f="10359" txt_i="5176" txt_f="5335">Tumor samples of 912 women from the trial were available for the present study, and tumor and treatment characteristics of the patients are presented in Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="10391" xml_f="10392" txt_i="5335" txt_f="5336">1</offsets></xref><offsets xml_i="10399" xml_f="10927" txt_i="5336" txt_f="5864">. The clinicopathological characteristics in this subset were similar to those in the complete series of 1,780 patients in the trial, such as tumor size ≤20 mm (79% vs. 81%), positive ER status (78% vs. 80%), and tamoxifen treatment (52% vs. 50%). The standard procedure for tissue collection was fixation in 4% phosphate-buffered formalin. Follow-up data were collected from regional population registers and the Swedish Cause of Death Registry. The mean follow-up period for patients in the present investigation was 17 years.</offsets></p><table-wrap id="T1" position="float"><label><offsets xml_i="10975" xml_f="10982" txt_i="5865" txt_f="5872">Table 1</offsets></label><caption><p><offsets xml_i="11002" xml_f="11025" txt_i="5872" txt_f="5895">Patient characteristics</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th></th><th align="left"><offsets xml_i="11113" xml_f="11140" txt_i="5896" txt_f="5923">Patients in present study (</offsets><italic><offsets xml_i="11148" xml_f="11150" txt_i="5923" txt_f="5925">n </offsets></italic><offsets xml_i="11159" xml_f="11165" txt_i="5925" txt_f="5931">= 912)</offsets></th><th align="left"><offsets xml_i="11187" xml_f="11225" txt_i="5931" txt_f="5969">Patients with HOXB13 expression data (</offsets><italic><offsets xml_i="11233" xml_f="11235" txt_i="5969" txt_f="5971">n </offsets></italic><offsets xml_i="11244" xml_f="11250" txt_i="5971" txt_f="5977">= 866)</offsets></th><th align="left"><offsets xml_i="11272" xml_f="11289" txt_i="5977" txt_f="5994">Original cohort (</offsets><italic><offsets xml_i="11297" xml_f="11299" txt_i="5994" txt_f="5996">n </offsets></italic><offsets xml_i="11308" xml_f="11316" txt_i="5996" txt_f="6004">= 1,780)</offsets></th></tr></thead><tbody><tr><td align="left" colspan="4"><offsets xml_i="11374" xml_f="11388" txt_i="6004" txt_f="6018">Tumor diameter</offsets></td></tr><tr><td align="left"><offsets xml_i="11419" xml_f="11426" txt_i="6019" txt_f="6026"> ≤20 mm</offsets></td><td align="left"><offsets xml_i="11448" xml_f="11456" txt_i="6027" txt_f="6035">697 (79)</offsets></td><td align="left"><offsets xml_i="11478" xml_f="11486" txt_i="6036" txt_f="6044">661 (79)</offsets></td><td align="left"><offsets xml_i="11508" xml_f="11518" txt_i="6045" txt_f="6055">1,393 (81)</offsets></td></tr><tr><td align="left"><offsets xml_i="11549" xml_f="11559" txt_i="6056" txt_f="6063"> &gt;20 mm</offsets></td><td align="left"><offsets xml_i="11581" xml_f="11589" txt_i="6064" txt_f="6072">189 (21)</offsets></td><td align="left"><offsets xml_i="11611" xml_f="11619" txt_i="6073" txt_f="6081">181 (21)</offsets></td><td align="left"><offsets xml_i="11641" xml_f="11649" txt_i="6082" txt_f="6090">323 (19)</offsets></td></tr><tr><td align="left"><offsets xml_i="11680" xml_f="11692" txt_i="6091" txt_f="6103"> Unavailable</offsets></td><td align="left"><offsets xml_i="11714" xml_f="11716" txt_i="6104" txt_f="6106">26</offsets></td><td align="left"><offsets xml_i="11738" xml_f="11740" txt_i="6107" txt_f="6109">24</offsets></td><td align="left"><offsets xml_i="11762" xml_f="11764" txt_i="6110" txt_f="6112">64</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="11807" xml_f="11831" txt_i="6113" txt_f="6137">Estrogen receptor status</offsets></td></tr><tr><td align="left"><offsets xml_i="11862" xml_f="11871" txt_i="6138" txt_f="6147"> Positive</offsets></td><td align="left"><offsets xml_i="11893" xml_f="11901" txt_i="6148" txt_f="6156">686 (78)</offsets></td><td align="left"><offsets xml_i="11923" xml_f="11931" txt_i="6157" txt_f="6165">651 (77)</offsets></td><td align="left"><offsets xml_i="11953" xml_f="11963" txt_i="6166" txt_f="6176">1,183 (80)</offsets></td></tr><tr><td align="left"><offsets xml_i="11994" xml_f="12003" txt_i="6177" txt_f="6186"> Negative</offsets></td><td align="left"><offsets xml_i="12025" xml_f="12033" txt_i="6187" txt_f="6195">198 (22)</offsets></td><td align="left"><offsets xml_i="12055" xml_f="12063" txt_i="6196" txt_f="6204">190 (23)</offsets></td><td align="left"><offsets xml_i="12085" xml_f="12093" txt_i="6205" txt_f="6213">296 (20)</offsets></td></tr><tr><td align="left"><offsets xml_i="12124" xml_f="12136" txt_i="6214" txt_f="6226"> Unavailable</offsets></td><td align="left"><offsets xml_i="12158" xml_f="12160" txt_i="6227" txt_f="6229">28</offsets></td><td align="left"><offsets xml_i="12182" xml_f="12184" txt_i="6230" txt_f="6232">25</offsets></td><td align="left"><offsets xml_i="12206" xml_f="12209" txt_i="6233" txt_f="6236">301</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="12252" xml_f="12280" txt_i="6237" txt_f="6265">Progesterone receptor status</offsets></td></tr><tr><td align="left"><offsets xml_i="12311" xml_f="12320" txt_i="6266" txt_f="6275"> Positive</offsets></td><td align="left"><offsets xml_i="12342" xml_f="12350" txt_i="6276" txt_f="6284">415 (52)</offsets></td><td align="left"><offsets xml_i="12372" xml_f="12380" txt_i="6285" txt_f="6293">401 (52)</offsets></td><td align="left"><offsets xml_i="12402" xml_f="12410" txt_i="6294" txt_f="6302">590 (48)</offsets></td></tr><tr><td align="left"><offsets xml_i="12441" xml_f="12450" txt_i="6303" txt_f="6312"> Negative</offsets></td><td align="left"><offsets xml_i="12472" xml_f="12480" txt_i="6313" txt_f="6321">380 (48)</offsets></td><td align="left"><offsets xml_i="12502" xml_f="12510" txt_i="6322" txt_f="6330">366 (48)</offsets></td><td align="left"><offsets xml_i="12532" xml_f="12540" txt_i="6331" txt_f="6339">627 (52)</offsets></td></tr><tr><td align="left"><offsets xml_i="12571" xml_f="12583" txt_i="6340" txt_f="6352"> Unavailable</offsets></td><td align="left"><offsets xml_i="12605" xml_f="12608" txt_i="6353" txt_f="6356">117</offsets></td><td align="left"><offsets xml_i="12630" xml_f="12632" txt_i="6357" txt_f="6359">99</offsets></td><td align="left"><offsets xml_i="12654" xml_f="12657" txt_i="6360" txt_f="6363">563</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="12700" xml_f="12719" txt_i="6364" txt_f="6383">Tamoxifen treatment</offsets></td></tr><tr><td align="left"><offsets xml_i="12750" xml_f="12754" txt_i="6384" txt_f="6388"> Yes</offsets></td><td align="left"><offsets xml_i="12776" xml_f="12784" txt_i="6389" txt_f="6397">473 (52)</offsets></td><td align="left"><offsets xml_i="12806" xml_f="12814" txt_i="6398" txt_f="6406">447 (52)</offsets></td><td align="left"><offsets xml_i="12836" xml_f="12844" txt_i="6407" txt_f="6415">886 (50)</offsets></td></tr><tr><td align="left"><offsets xml_i="12875" xml_f="12878" txt_i="6416" txt_f="6419"> No</offsets></td><td align="left"><offsets xml_i="12900" xml_f="12908" txt_i="6420" txt_f="6428">439 (48)</offsets></td><td align="left"><offsets xml_i="12930" xml_f="12938" txt_i="6429" txt_f="6437">419 (48)</offsets></td><td align="left"><offsets xml_i="12960" xml_f="12968" txt_i="6438" txt_f="6446">894 (50)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="13014" xml_f="13255" txt_i="6447" txt_f="6688">Data presented as number of patients (%). Tumor and treatment characteristics of the patients included in the original cohort, patients in the present study and patients in the present study with successful HOXB13 protein expression scoring.</offsets></p></table-wrap-foot></table-wrap><p><offsets xml_i="13293" xml_f="13465" txt_i="6689" txt_f="6861">The study was approved by the local ethical committee at the Karolinska University Hospital. According to the approval, informed consent from the patients was not required.</offsets></p></sec><sec><title><offsets xml_i="13487" xml_f="13510" txt_i="6863" txt_f="6886">Hormone receptor status</offsets></title><p><offsets xml_i="13521" xml_f="13649" txt_i="6887" txt_f="7015">The status of the ER and the progesterone receptor (PR) was assessed retrospectively with immunohistochemistry using the Ventana</offsets><sup><offsets xml_i="13654" xml_f="13656" txt_i="7015" txt_f="7017">® </offsets></sup><offsets xml_i="13662" xml_f="14095" txt_i="7017" txt_f="7450">automated slide stainer (Ventana Medical Systems, S.A., Illkirch, France). Primary monoclonal antibodies were the CONFIRM™ mouse anti-ER antibody (clone 6F11) and the CONFIRM™ mouse anti-PR antibody (clone 16) from Ventana Medical Systems. The cut-off level was set to 10% positively stained tumor cell nuclei. In cases where immunohistochemical data for ER were missing (13%), the ER status determined in clinical routine practice [</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="14127" xml_f="14129" txt_i="7450" txt_f="7452">14</offsets></xref><offsets xml_i="14136" xml_f="14188" txt_i="7452" txt_f="7504">] was used with a cut-off level of 0.10 fmol/μg DNA.</offsets></p></sec><sec><title><offsets xml_i="14210" xml_f="14256" txt_i="7506" txt_f="7552">Tissue microarray and immunostaining of HOXB13</offsets></title><p><offsets xml_i="14267" xml_f="14856" txt_i="7553" txt_f="8142">Formalin-fixed and paraffin-embedded tissue blocks were selected as donor blocks for tissue microarray. Sections were cut from each donor block and were stained with hematoxylin and eosin. Three morphologically representative regions were chosen in each of the 912 tumor samples. Triplicate core tissue samples with a diameter of 0.8 mm were taken from selected areas of the tumors with a manual arrayer instrument (Beecher Instruments, Inc, Sun Prairie, WI, USA), and were arrayed into a paraffin block. Samples from human liver were included on each tissue block for orientation purpose.</offsets></p><p><offsets xml_i="14863" xml_f="16064" txt_i="8143" txt_f="9344">Sections (4 μm thick) from the tissue microarray blocks were cut and mounted onto frost-coated glass slides, deparaffinized in xylene and rehydrated through a graded alcohol series to distilled water. The slides were boiled in 10 mM citrate buffer with pH 6.0, in a pressure cooker for 3 minutes. After cooling to room temperature, the slides were washed and incubated for 5 minutes in 3% hydrogen peroxide and methanol, and were then incubated with serum-free protein block (Dako Sweden AB, Stockholm, Sweden). A mouse monoclonal antibody raised against amino acids 1 to 284 of human HOXB13 (F-9 sc-28333, lot number L1504; Santa Cruz Biotechnology, Inc., Heidelberg, Germany) was applied as primary antibody at a 1:50 dilution, and the slides were incubated at 4°C for 16 hours. As the secondary reagent, the EnVision™ + Dual Link System labeled with horseradish peroxidase (Dako Sweden AB) for 30 min at room temperature was used. The slides were incubated with 3,3'-diaminobenzidine tetrahydrochloride solution, counterstained with hematoxylin, and mounted. BT-474 cells were used as a positive control. All washing steps were performed in phosphate-buffered saline with 0.5% bovine serum albumin.</offsets></p></sec><sec><title><offsets xml_i="16086" xml_f="16128" txt_i="9346" txt_f="9388">Evaluation of immunohistochemical staining</offsets></title><p><offsets xml_i="16139" xml_f="16791" txt_i="9389" txt_f="10041">The stained tumor sections were evaluated independently by two investigators (P-LJ and OS) without knowledge about the clinicopathological data. The slides were examined using a Leica LB30T microscope (Leica Microsystems, Wetzlar, Germany) and were photographed using an Olympus SC20 digital camera (Olympus Europe GmbH, Hamburg, Germany). Nuclear staining intensity was graded as negative, weak, moderate or strong. In cases with different scoring results, a consensus score was reached after re-evaluation. In the survival analysis, weak or no staining was categorized as low expression and moderate to strong staining was considered high expression.</offsets></p></sec><sec><title><offsets xml_i="16813" xml_f="16849" txt_i="10043" txt_f="10079">Immunohistochemical analysis of HER2</offsets></title><p><offsets xml_i="16860" xml_f="17505" txt_i="10080" txt_f="10713">Formalin-fixed, paraffin-embedded tissue was stained for human epidermal growth factor receptor 2 (HER2) with the DAKO AO0485 polyclonal rabbit antibody according to the guidelines provided by the manufacturer. The slides were scored as follows: 0 = no staining or &lt;10% of the cells positive for membrane staining; 1 = weak or barely perceptible staining in &gt;10% of the cells, staining in only part of the cell membrane; 2 = weak to moderate staining in the whole membrane in &gt;10% of the cells; and 3 = strong staining in the whole membrane displayed by &gt;10% of the cells. In all cases, the scoring was limited to the invasive tumor.</offsets></p></sec><sec><title><offsets xml_i="17527" xml_f="17539" txt_i="10715" txt_f="10727">Western blot</offsets></title><p><offsets xml_i="17550" xml_f="17691" txt_i="10728" txt_f="10869">Lysates from the cell lines BT-474, SKBR3 and T47 D were used to determine the specificity of the HOXB13 antibody. Real-time PCR analysis of </offsets><italic><offsets xml_i="17699" xml_f="17706" txt_i="10869" txt_f="10876">HOXB13 </offsets></italic><offsets xml_i="17715" xml_f="17783" txt_i="10876" txt_f="10944">mRNA shows that SKBR3 and T47 D express very low, if any, levels of </offsets><italic><offsets xml_i="17791" xml_f="17797" txt_i="10944" txt_f="10950">HOXB13</offsets></italic><offsets xml_i="17806" xml_f="18599" txt_i="10950" txt_f="11743">, whereas BT-474 cells show high expression (unpublished data). The lysates were loaded on a 4 to 15% gradient precast gel (Criterion; Bio-Rad Laboratories AB, Sundbyberg, Sweden) and the proteins were then transferred to a polyvinylidene fluoride membrane. The membrane was incubated overnight with the antibody (1:750) or with antibody previously incubated with full-length recombinant HOXB13 protein in a 10-fold excess. Antibodies were detected with a commercial Enhanced Chemiluminescence Plus kit (GE Healthcare UK Ltd, Little Chalfont, UK). To control for equal loading, the membrane was stripped and then incubated with a rabbit monoclonal GAPDH antibody (1:1,000; Cell Signaling Technology, Inc., Danvers, MA, USA) and a secondary polyclonal goat-anti-rabbit antibody (1:2,000; Dako).</offsets></p></sec><sec><title><offsets xml_i="18621" xml_f="18641" txt_i="11745" txt_f="11765">Statistical analysis</offsets></title><p><offsets xml_i="18652" xml_f="19061" txt_i="11766" txt_f="12175">Survival curves were produced according to the life-table method described by Kaplan and Meier, and differences in survival were estimated with the log-rank test. Distant recurrence-free survival (DRFS) was defined as the time from diagnosis to distant recurrence or to death due to breast cancer. Breast cancer survival (BCS) was the time elapsed from diagnosis to the date of death related to breast cancer.</offsets></p><p><italic><offsets xml_i="19076" xml_f="19078" txt_i="12176" txt_f="12178">P </offsets></italic><offsets xml_i="19087" xml_f="19600" txt_i="12178" txt_f="12691">values for relationships between HOXB13 and other variables were assessed with a chi-square test for trend. Univariate analyses of recurrence rates were performed with Cox proportional hazard regression. Tests of interactions between treatment effect and HOXB13 analysis were performed by including product terms in the models, as well as other prognostic factors (tumor size and HER2). All statistical procedures are comprised in the statistical package Statistica 9.1 (StatSoft Scandinavia AB, Uppsala, Sweden).</offsets></p></sec></sec><sec><title><offsets xml_i="19628" xml_f="19635" txt_i="12694" txt_f="12701">Results</offsets></title><sec><title><offsets xml_i="19655" xml_f="19680" txt_i="12702" txt_f="12727">HOXB13 protein expression</offsets></title><p><offsets xml_i="19691" xml_f="19896" txt_i="12728" txt_f="12933">Immunohistochemical staining of HOXB13 was performed on tissue microarray slides comprising a total of 912 tumors from breast cancer patients. The flow of patients through the study is described in Figure </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="19926" xml_f="19927" txt_i="12933" txt_f="12934">1</offsets></xref><offsets xml_i="19934" xml_f="20326" txt_i="12934" txt_f="13326">. Grading of the protein expression was successful in 866 cases (95% of the total number of patients), of which 291 (33.6%) displayed negative staining, 317 (36.6%) weak staining, 212 (24.4%) moderate staining, and 46 (5.3%) strong staining. Expression of HOXB13 was located exclusively in the nuclei of the tumor cells. Examples of the different levels of immunostaining are shown in Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="20356" xml_f="20358" txt_i="13326" txt_f="13328">2a</offsets></xref><offsets xml_i="20365" xml_f="20369" txt_i="13328" txt_f="13332"> to </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="20399" xml_f="20401" txt_i="13332" txt_f="13334">2c</offsets></xref><offsets xml_i="20408" xml_f="20607" txt_i="13334" txt_f="13533">. Western blot analysis of the specificity of the antibody gave a single specific band at 34 kDa, which was not detectable when the antibody was pre-incubated with recombinant HOXB13 protein (Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="20637" xml_f="20639" txt_i="13533" txt_f="13535">2d</offsets></xref><offsets xml_i="20646" xml_f="20648" txt_i="13535" txt_f="13537">).</offsets></p><fig id="F1" position="float"><label><offsets xml_i="20689" xml_f="20697" txt_i="13538" txt_f="13546">Figure 1</offsets></label><caption><p><bold><offsets xml_i="20723" xml_f="20735" txt_i="13546" txt_f="13558">Study design</offsets></bold><offsets xml_i="20742" xml_f="20835" txt_i="13558" txt_f="13651">. Randomization and flow of patients included in the original study and in the present study.</offsets></p></caption><graphic xlink:href="bcr2612-1"></graphic></fig><fig id="F2" position="float"><label><offsets xml_i="20934" xml_f="20942" txt_i="13652" txt_f="13660">Figure 2</offsets></label><caption><p><bold><offsets xml_i="20968" xml_f="21003" txt_i="13660" txt_f="13695">HOXB13 grading and antibody control</offsets></bold><offsets xml_i="21010" xml_f="21051" txt_i="13695" txt_f="13736">. Tumor tissue immunostained for HOXB13: </offsets><bold><offsets xml_i="21057" xml_f="21061" txt_i="13736" txt_f="13740">(a) </offsets></bold><offsets xml_i="21068" xml_f="21074" txt_i="13740" txt_f="13746">weak, </offsets><bold><offsets xml_i="21080" xml_f="21084" txt_i="13746" txt_f="13750">(b) </offsets></bold><offsets xml_i="21091" xml_f="21104" txt_i="13750" txt_f="13763">moderate and </offsets><bold><offsets xml_i="21110" xml_f="21114" txt_i="13763" txt_f="13767">(c) </offsets></bold><offsets xml_i="21121" xml_f="21129" txt_i="13767" txt_f="13775">strong. </offsets><bold><offsets xml_i="21135" xml_f="21139" txt_i="13775" txt_f="13779">(d) </offsets></bold><offsets xml_i="21146" xml_f="21355" txt_i="13779" txt_f="13988">Western blot analysis of HOXB13 in protein lysates from SKBR3 cells, T47 D cells and BT-474 cells. Lane BT-474*, antibody was preincubated with recombinant HOXB13 protein prior to incubation with the membrane.</offsets></p></caption><graphic xlink:href="bcr2612-2"></graphic></fig></sec><sec><title><offsets xml_i="21435" xml_f="21516" txt_i="13990" txt_f="14071">Expression of HOXB13 in relation to prognosis and prediction of treatment outcome</offsets></title><p><offsets xml_i="21527" xml_f="21730" txt_i="14072" txt_f="14275">Among the patients included in the present study, there were 419 patients with HOXB13 protein expression data who did not receive any adjuvant endocrine treatment - 74% of which had ER-positive disease (</offsets><italic><offsets xml_i="21738" xml_f="21740" txt_i="14275" txt_f="14277">n </offsets></italic><offsets xml_i="21749" xml_f="22295" txt_i="14277" txt_f="14823">= 310, 16 patients had unknown ER status). For all of the endocrine-untreated patients, irrespective of ER status, neither the rate of distant recurrence nor the breast cancer-related survival differed between patients with different levels of HOXB13 protein expression in the tumor. Hazard ratios according to univariate Cox proportional hazard modeling for patients presenting high expression compared with patients presenting low expression were not statistically significant (DRFS hazard ratio = 0.89, 95% confidence interval = 0.58 to 1.36, </offsets><italic><offsets xml_i="22303" xml_f="22305" txt_i="14823" txt_f="14825">P </offsets></italic><offsets xml_i="22314" xml_f="22387" txt_i="14825" txt_f="14898">= 0.59; BCS hazard ratio = 0.76, 95% confidence interval = 0.46 to 1.25, </offsets><italic><offsets xml_i="22395" xml_f="22397" txt_i="14898" txt_f="14900">P </offsets></italic><offsets xml_i="22406" xml_f="22553" txt_i="14900" txt_f="15047">= 0.28). Restricting the analysis to the ER-positive group gave similar results (DRFS hazard ratio = 0.78, 95% confidence interval = 0.47 to 1.29, </offsets><italic><offsets xml_i="22561" xml_f="22563" txt_i="15047" txt_f="15049">P </offsets></italic><offsets xml_i="22572" xml_f="22645" txt_i="15049" txt_f="15122">= 0.32; BCS hazard ratio = 0.60, 95% confidence interval = 0.32 to 1.14, </offsets><italic><offsets xml_i="22653" xml_f="22655" txt_i="15122" txt_f="15124">P </offsets></italic><offsets xml_i="22664" xml_f="22672" txt_i="15124" txt_f="15132">= 0.11).</offsets></p><p><offsets xml_i="22679" xml_f="22723" txt_i="15133" txt_f="15177">Tamoxifen significantly increased the DRFS (</offsets><italic><offsets xml_i="22731" xml_f="22733" txt_i="15177" txt_f="15179">P </offsets></italic><offsets xml_i="22742" xml_f="22766" txt_i="15179" txt_f="15203">= 0.00002) and the BCS (</offsets><italic><offsets xml_i="22774" xml_f="22776" txt_i="15203" txt_f="15205">P </offsets></italic><offsets xml_i="22785" xml_f="22880" txt_i="15205" txt_f="15300">= 0.00008) in patients with ER-positive tumors expressing none or low levels of HOXB13 (Figure </offsets><xref ref-type="fig" rid="F3"><offsets xml_i="22910" xml_f="22914" txt_i="15300" txt_f="15304">3a,b</offsets></xref><offsets xml_i="22921" xml_f="23056" txt_i="15304" txt_f="15439">), whereas the patients with a moderate to high HOXB13 expression in their tumors did not benefit from the endocrine treatment (Figure </offsets><xref ref-type="fig" rid="F3"><offsets xml_i="23086" xml_f="23090" txt_i="15439" txt_f="15443">3c,d</offsets></xref><offsets xml_i="23097" xml_f="23203" txt_i="15443" txt_f="15549">). The interaction between HOXB13 expression and treatment effect was statistically significant for DRFS (</offsets><italic><offsets xml_i="23211" xml_f="23213" txt_i="15549" txt_f="15551">P </offsets></italic><offsets xml_i="23222" xml_f="23405" txt_i="15551" txt_f="15734">= 0.035; multivariate model adjusted for tumor size and HER2 status). Cox proportional hazard regressions for the tamoxifen benefit in the different patient groups are shown in Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="23437" xml_f="23438" txt_i="15734" txt_f="15735">2</offsets></xref><offsets xml_i="23445" xml_f="23446" txt_i="15735" txt_f="15736">.</offsets></p><table-wrap id="T2" position="float"><label><offsets xml_i="23494" xml_f="23501" txt_i="15737" txt_f="15744">Table 2</offsets></label><caption><p><offsets xml_i="23521" xml_f="23580" txt_i="15744" txt_f="15803">Estimation of the benefit from adjuvant tamoxifen treatment</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left"><offsets xml_i="23659" xml_f="23671" txt_i="15804" txt_f="15816">Tumor status</offsets></th><th align="center" colspan="6"><offsets xml_i="23707" xml_f="23756" txt_i="15816" txt_f="15865">Tamoxifen treatment versus no endocrine treatment</offsets></th></tr><tr><th></th><th colspan="6"><hr></hr></th></tr><tr><th></th><th align="center" colspan="3"><offsets xml_i="23858" xml_f="23876" txt_i="15865" txt_f="15883">Distant recurrence</offsets></th><th align="center" colspan="3"><offsets xml_i="23912" xml_f="23931" txt_i="15883" txt_f="15902">Breast cancer death</offsets></th></tr><tr><th></th><th colspan="3"><hr></hr></th><th colspan="3"><hr></hr></th></tr><tr><th></th><th align="left"><offsets xml_i="24049" xml_f="24070" txt_i="15902" txt_f="15923">Hazard ratio (95% CI)</offsets></th><th align="left"><italic><offsets xml_i="24100" xml_f="24102" txt_i="15923" txt_f="15925">P </offsets></italic><offsets xml_i="24111" xml_f="24116" txt_i="15925" txt_f="15930">value</offsets></th><th align="left"><italic><offsets xml_i="24146" xml_f="24147" txt_i="15930" txt_f="15931">P</offsets></italic><sub><bold><offsets xml_i="24167" xml_f="24178" txt_i="15931" txt_f="15942">interaction</offsets></bold></sub><offsets xml_i="24191" xml_f="24192" txt_i="15942" txt_f="15943">*</offsets></th><th align="left"><offsets xml_i="24214" xml_f="24235" txt_i="15943" txt_f="15964">Hazard ratio (95% CI)</offsets></th><th align="left"><italic><offsets xml_i="24265" xml_f="24267" txt_i="15964" txt_f="15966">P </offsets></italic><offsets xml_i="24276" xml_f="24281" txt_i="15966" txt_f="15971">value</offsets></th><th align="left"><italic><offsets xml_i="24311" xml_f="24312" txt_i="15971" txt_f="15972">P</offsets></italic><sub><bold><offsets xml_i="24332" xml_f="24343" txt_i="15972" txt_f="15983">interaction</offsets></bold></sub><offsets xml_i="24356" xml_f="24357" txt_i="15983" txt_f="15984">*</offsets></th></tr></thead><tbody><tr><td align="left" colspan="7"><offsets xml_i="24415" xml_f="24426" txt_i="15984" txt_f="15995">ER-positive</offsets></td></tr><tr><td align="left"><offsets xml_i="24457" xml_f="24479" txt_i="15996" txt_f="16018"> Low HOXB13 expression</offsets></td><td align="left"><offsets xml_i="24501" xml_f="24520" txt_i="16019" txt_f="16038">0.38 (0.23 to 0.60)</offsets></td><td align="left"><offsets xml_i="24542" xml_f="24550" txt_i="16039" txt_f="16047">0.000048</offsets></td><td></td><td align="left"><offsets xml_i="24581" xml_f="24600" txt_i="16049" txt_f="16068">0.35 (0.20 to 0.60)</offsets></td><td align="left"><offsets xml_i="24622" xml_f="24629" txt_i="16069" txt_f="16076">0.00016</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="24669" xml_f="24692" txt_i="16078" txt_f="16101"> High HOXB13 expression</offsets></td><td align="left"><offsets xml_i="24714" xml_f="24733" txt_i="16102" txt_f="16121">0.88 (0.47 to 1.65)</offsets></td><td align="left"><offsets xml_i="24755" xml_f="24759" txt_i="16122" txt_f="16126">0.69</offsets></td><td align="left"><offsets xml_i="24781" xml_f="24786" txt_i="16127" txt_f="16132">0.035</offsets></td><td align="left"><offsets xml_i="24808" xml_f="24827" txt_i="16133" txt_f="16152">0.84 (0.37 to 1.90)</offsets></td><td align="left"><offsets xml_i="24849" xml_f="24853" txt_i="16153" txt_f="16157">0.67</offsets></td><td align="left"><offsets xml_i="24875" xml_f="24880" txt_i="16158" txt_f="16163">0.060</offsets></td></tr><tr><td align="left" colspan="7"><offsets xml_i="24923" xml_f="24947" txt_i="16164" txt_f="16188">ER-positive, PR-positive</offsets></td></tr><tr><td align="left"><offsets xml_i="24978" xml_f="25000" txt_i="16189" txt_f="16211"> Low HOXB13 expression</offsets></td><td align="left"><offsets xml_i="25022" xml_f="25041" txt_i="16212" txt_f="16231">0.26 (0.14 to 0.49)</offsets></td><td align="left"><offsets xml_i="25063" xml_f="25071" txt_i="16232" txt_f="16240">0.000022</offsets></td><td></td><td align="left"><offsets xml_i="25102" xml_f="25121" txt_i="16242" txt_f="16261">0.24 (0.11 to 0.50)</offsets></td><td align="left"><offsets xml_i="25143" xml_f="25150" txt_i="16262" txt_f="16269">0.00015</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="25190" xml_f="25213" txt_i="16271" txt_f="16294"> High HOXB13 expression</offsets></td><td align="left"><offsets xml_i="25235" xml_f="25254" txt_i="16295" txt_f="16314">0.70 (0.31 to 1.56)</offsets></td><td align="left"><offsets xml_i="25276" xml_f="25280" txt_i="16315" txt_f="16319">0.38</offsets></td><td align="left"><offsets xml_i="25302" xml_f="25307" txt_i="16320" txt_f="16325">0.072</offsets></td><td align="left"><offsets xml_i="25329" xml_f="25348" txt_i="16326" txt_f="16345">0.82 (0.27 to 2.56)</offsets></td><td align="left"><offsets xml_i="25370" xml_f="25374" txt_i="16346" txt_f="16350">0.74</offsets></td><td align="left"><offsets xml_i="25396" xml_f="25401" txt_i="16351" txt_f="16356">0.059</offsets></td></tr><tr><td align="left" colspan="7"><offsets xml_i="25444" xml_f="25468" txt_i="16357" txt_f="16381">ER-positive, PR-negative</offsets></td></tr><tr><td align="left"><offsets xml_i="25499" xml_f="25521" txt_i="16382" txt_f="16404"> Low HOXB13 expression</offsets></td><td align="left"><offsets xml_i="25543" xml_f="25562" txt_i="16405" txt_f="16424">0.69 (0.31 to 1.56)</offsets></td><td align="left"><offsets xml_i="25584" xml_f="25588" txt_i="16425" txt_f="16429">0.37</offsets></td><td></td><td align="left"><offsets xml_i="25619" xml_f="25638" txt_i="16431" txt_f="16450">0.64 (0.26 to 1.56)</offsets></td><td align="left"><offsets xml_i="25660" xml_f="25664" txt_i="16451" txt_f="16455">0.33</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="25704" xml_f="25727" txt_i="16457" txt_f="16480"> High HOXB13 expression</offsets></td><td align="left"><offsets xml_i="25749" xml_f="25768" txt_i="16481" txt_f="16500">0.89 (0.31 to 2.54)</offsets></td><td align="left"><offsets xml_i="25790" xml_f="25794" txt_i="16501" txt_f="16505">0.83</offsets></td><td align="left"><offsets xml_i="25816" xml_f="25820" txt_i="16506" txt_f="16510">0.51</offsets></td><td align="left"><offsets xml_i="25842" xml_f="25861" txt_i="16511" txt_f="16530">0.60 (0.17 to 2.13)</offsets></td><td align="left"><offsets xml_i="25883" xml_f="25887" txt_i="16531" txt_f="16535">0.43</offsets></td><td align="left"><offsets xml_i="25909" xml_f="25913" txt_i="16536" txt_f="16540">0.98</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="25959" xml_f="26527" txt_i="16541" txt_f="17106">Cox regression analysis of distant recurrence rate and breast cancer-related deaths for patients with estrogen receptor (ER)-positive tumors, ER-positive and progesterone receptor (PR)-positive tumors, and ER-positive and PR-negative tumors in relation to HOXB13 protein expression. CI, confidence interval. *Tests whether there is a difference in the treatment response. The models included treatment (tamoxifen vs. no endocrine treatment), HOXB13 expression (high vs. low), an interaction variable, tumor size (&gt;20 mm vs. ≤20 mm) and HER2 status (3+ vs. 0 to 2+).</offsets></p></table-wrap-foot></table-wrap><fig id="F3" position="float"><label><offsets xml_i="26599" xml_f="26607" txt_i="17107" txt_f="17115">Figure 3</offsets></label><caption><p><bold><offsets xml_i="26633" xml_f="26726" txt_i="17115" txt_f="17208">Cumulative survival for patients with estrogen receptor-positive tumors, stratified by HOXB13</offsets></bold><offsets xml_i="26733" xml_f="26970" txt_i="17208" txt_f="17445">. Kaplan-Meier curves illustrating clinical outcome in terms of distant recurrence-free survival and breast cancer survival for patients with estrogen receptor (ER)-positive tumors undergoing tamoxifen treatment or no endocrine therapy: </offsets><bold><offsets xml_i="26976" xml_f="26985" txt_i="17445" txt_f="17454">(a), (b) </offsets></bold><offsets xml_i="26992" xml_f="27021" txt_i="17454" txt_f="17483">low or no HOXB13 expression; </offsets><bold><offsets xml_i="27027" xml_f="27036" txt_i="17483" txt_f="17492">(c), (d) </offsets></bold><offsets xml_i="27043" xml_f="27078" txt_i="17492" txt_f="17527">moderate or high HOXB13 expression.</offsets></p></caption><graphic xlink:href="bcr2612-3"></graphic></fig><p><offsets xml_i="27143" xml_f="27317" txt_i="17528" txt_f="17702">For patients with tumors positive for both ER and PR, survival analysis indicated that there was a significant benefit from tamoxifen if the HOXB13 expression was low (DRFS, </offsets><italic><offsets xml_i="27325" xml_f="27327" txt_i="17702" txt_f="17704">P </offsets></italic><offsets xml_i="27336" xml_f="27352" txt_i="17704" txt_f="17720">= 0.00001; BCS, </offsets><italic><offsets xml_i="27360" xml_f="27362" txt_i="17720" txt_f="17722">P </offsets></italic><offsets xml_i="27371" xml_f="27390" txt_i="17722" txt_f="17741">= 0.00004) (Figure </offsets><xref ref-type="fig" rid="F4"><offsets xml_i="27420" xml_f="27424" txt_i="17741" txt_f="17745">4a,b</offsets></xref><offsets xml_i="27431" xml_f="27608" txt_i="17745" txt_f="17922">). High levels of HOXB13 were still unfavorable for the patients; no difference in relapse-free survival was noted comparing the treated and untreated groups of patients (DRFS, </offsets><italic><offsets xml_i="27616" xml_f="27618" txt_i="17922" txt_f="17924">P </offsets></italic><offsets xml_i="27627" xml_f="27640" txt_i="17924" txt_f="17937">= 0.38; BCS, </offsets><italic><offsets xml_i="27648" xml_f="27650" txt_i="17937" txt_f="17939">P </offsets></italic><offsets xml_i="27659" xml_f="27675" txt_i="17939" txt_f="17955">= 0.74) (Figure </offsets><xref ref-type="fig" rid="F4"><offsets xml_i="27705" xml_f="27709" txt_i="17955" txt_f="17959">4c,d</offsets></xref><offsets xml_i="27716" xml_f="27719" txt_i="17959" txt_f="17962">). </offsets><italic><offsets xml_i="27727" xml_f="27729" txt_i="17962" txt_f="17964">P </offsets></italic><offsets xml_i="27738" xml_f="27837" txt_i="17964" txt_f="18063">values for the multivariate interaction analyses, however, did not reach statistical significance (</offsets><italic><offsets xml_i="27845" xml_f="27847" txt_i="18063" txt_f="18065">P </offsets></italic><offsets xml_i="27856" xml_f="27868" txt_i="18065" txt_f="18077">= 0.072 and </offsets><italic><offsets xml_i="27876" xml_f="27878" txt_i="18077" txt_f="18079">P </offsets></italic><offsets xml_i="27887" xml_f="27933" txt_i="18079" txt_f="18125">= 0.059 for DRFS and BCS, respectively; Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="27965" xml_f="27966" txt_i="18125" txt_f="18126">2</offsets></xref><offsets xml_i="27973" xml_f="28140" txt_i="18126" txt_f="18293">). Further analyses showed that patients with ER-positive but PR-negative tumors did not seem to benefit from tamoxifen, regardless of HOXB13 expression levels (Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="28172" xml_f="28173" txt_i="18293" txt_f="18294">2</offsets></xref><offsets xml_i="28180" xml_f="28182" txt_i="18294" txt_f="18296">).</offsets></p><fig id="F4" position="float"><label><offsets xml_i="28223" xml_f="28231" txt_i="18297" txt_f="18305">Figure 4</offsets></label><caption><p><bold><offsets xml_i="28257" xml_f="28380" txt_i="18305" txt_f="18428">Cumulative survival for patients with tumors expressing estrogen receptors and progesterone receptors, stratified by HOXB13</offsets></bold><offsets xml_i="28387" xml_f="28664" txt_i="18428" txt_f="18705">. Kaplan-Meier curves illustrating clinical outcome in terms of distant recurrence-free survival and breast cancer survival for patients with estrogen receptor (ER)-positive and progesterone receptor (PR)-positive tumors undergoing tamoxifen treatment or no endocrine therapy: </offsets><bold><offsets xml_i="28670" xml_f="28679" txt_i="18705" txt_f="18714">(a), (b) </offsets></bold><offsets xml_i="28686" xml_f="28715" txt_i="18714" txt_f="18743">low or no HOXB13 expression; </offsets><bold><offsets xml_i="28721" xml_f="28730" txt_i="18743" txt_f="18752">(c), (d) </offsets></bold><offsets xml_i="28737" xml_f="28772" txt_i="18752" txt_f="18787">moderate or high HOXB13 expression.</offsets></p></caption><graphic xlink:href="bcr2612-4"></graphic></fig></sec><sec><title><offsets xml_i="28852" xml_f="28887" txt_i="18789" txt_f="18824">Correlation with prognostic factors</offsets></title><p><offsets xml_i="28898" xml_f="29053" txt_i="18825" txt_f="18980">Analysis of possible relationships between HOXB13 protein expression and different tumor variables revealed correlations to HER2, ER and tumor size (Table </offsets><xref ref-type="table" rid="T3"><offsets xml_i="29085" xml_f="29086" txt_i="18980" txt_f="18981">3</offsets></xref><offsets xml_i="29093" xml_f="29143" txt_i="18981" txt_f="19031">). HOXB13 was inversely correlated to tumor size (</offsets><italic><offsets xml_i="29151" xml_f="29153" txt_i="19031" txt_f="19033">P </offsets></italic><offsets xml_i="29162" xml_f="29205" txt_i="19033" txt_f="19076">= 0.0048) and positively correlated to ER (</offsets><italic><offsets xml_i="29213" xml_f="29215" txt_i="19076" txt_f="19078">P </offsets></italic><offsets xml_i="29224" xml_f="29251" txt_i="19078" txt_f="19105">= 0.0022) and HER2 levels (</offsets><italic><offsets xml_i="29259" xml_f="29261" txt_i="19105" txt_f="19107">P </offsets></italic><offsets xml_i="29270" xml_f="29280" txt_i="19107" txt_f="19117">= 0.0023).</offsets></p><table-wrap id="T3" position="float"><label><offsets xml_i="29328" xml_f="29335" txt_i="19118" txt_f="19125">Table 3</offsets></label><caption><p><offsets xml_i="29355" xml_f="29463" txt_i="19125" txt_f="19233">Correlation between HOXB13 protein levels and prognostic factors analyzed with the chi-square test for trend</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th></th><th align="center" colspan="4"><offsets xml_i="29565" xml_f="29582" txt_i="19234" txt_f="19251">HOXB13 expression</offsets></th><th align="left"><italic><offsets xml_i="29612" xml_f="29614" txt_i="19251" txt_f="19253">P </offsets></italic><offsets xml_i="29623" xml_f="29628" txt_i="19253" txt_f="19258">value</offsets></th></tr><tr><th></th><th colspan="4"><hr></hr></th><th></th></tr><tr><th></th><th align="left"><offsets xml_i="29725" xml_f="29733" txt_i="19258" txt_f="19266">Negative</offsets></th><th align="left"><offsets xml_i="29755" xml_f="29758" txt_i="19266" txt_f="19269">Low</offsets></th><th align="left"><offsets xml_i="29780" xml_f="29792" txt_i="19269" txt_f="19281">Intermediate</offsets></th><th align="left"><offsets xml_i="29814" xml_f="29818" txt_i="19281" txt_f="19285">High</offsets></th><th></th></tr></thead><tbody><tr><td align="left" colspan="6"><offsets xml_i="29885" xml_f="29895" txt_i="19285" txt_f="19295">Tumor size</offsets></td></tr><tr><td align="left"><offsets xml_i="29926" xml_f="29933" txt_i="19296" txt_f="19303"> ≤20 mm</offsets></td><td align="left"><offsets xml_i="29955" xml_f="29965" txt_i="19304" txt_f="19314">205 (31.0)</offsets></td><td align="left"><offsets xml_i="29987" xml_f="29997" txt_i="19315" txt_f="19325">249 (37.7)</offsets></td><td align="left"><offsets xml_i="30019" xml_f="30029" txt_i="19326" txt_f="19336">168 (25.4)</offsets></td><td align="left"><offsets xml_i="30051" xml_f="30059" txt_i="19337" txt_f="19345">39 (5.9)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="30099" xml_f="30109" txt_i="19347" txt_f="19354"> &gt;20 mm</offsets></td><td align="left"><offsets xml_i="30131" xml_f="30140" txt_i="19355" txt_f="19364">78 (43.1)</offsets></td><td align="left"><offsets xml_i="30162" xml_f="30171" txt_i="19365" txt_f="19374">59 (32.6)</offsets></td><td align="left"><offsets xml_i="30193" xml_f="30202" txt_i="19375" txt_f="19384">37 (20.4)</offsets></td><td align="left"><offsets xml_i="30224" xml_f="30230" txt_i="19385" txt_f="19391">7 (3.9</offsets></td><td align="left"><offsets xml_i="30252" xml_f="30258" txt_i="19392" txt_f="19398">0.0048</offsets></td></tr><tr><td align="left" colspan="6"><offsets xml_i="30301" xml_f="30341" txt_i="19399" txt_f="19439">Human epidermal growth factor receptor 2</offsets></td></tr><tr><td align="left"><offsets xml_i="30372" xml_f="30374" txt_i="19440" txt_f="19442"> 0</offsets></td><td align="left"><offsets xml_i="30396" xml_f="30406" txt_i="19443" txt_f="19453">189 (38.4)</offsets></td><td align="left"><offsets xml_i="30428" xml_f="30438" txt_i="19454" txt_f="19464">172 (35.0)</offsets></td><td align="left"><offsets xml_i="30460" xml_f="30470" txt_i="19465" txt_f="19475">102 (20.7)</offsets></td><td align="left"><offsets xml_i="30492" xml_f="30500" txt_i="19476" txt_f="19484">29 (5.9)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="30540" xml_f="30543" txt_i="19486" txt_f="19489"> 1+</offsets></td><td align="left"><offsets xml_i="30565" xml_f="30574" txt_i="19490" txt_f="19499">38 (26.6)</offsets></td><td align="left"><offsets xml_i="30596" xml_f="30605" txt_i="19500" txt_f="19509">55 (38.5)</offsets></td><td align="left"><offsets xml_i="30627" xml_f="30636" txt_i="19510" txt_f="19519">45 (31.5)</offsets></td><td align="left"><offsets xml_i="30658" xml_f="30665" txt_i="19520" txt_f="19527">5 (3.5)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="30705" xml_f="30708" txt_i="19529" txt_f="19532"> 2+</offsets></td><td align="left"><offsets xml_i="30730" xml_f="30739" txt_i="19533" txt_f="19542">16 (20.5)</offsets></td><td align="left"><offsets xml_i="30761" xml_f="30770" txt_i="19543" txt_f="19552">32 (41.0)</offsets></td><td align="left"><offsets xml_i="30792" xml_f="30801" txt_i="19553" txt_f="19562">26 (33.3)</offsets></td><td align="left"><offsets xml_i="30823" xml_f="30830" txt_i="19563" txt_f="19570">4 (5.1)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="30870" xml_f="30873" txt_i="19572" txt_f="19575"> 3+</offsets></td><td align="left"><offsets xml_i="30895" xml_f="30904" txt_i="19576" txt_f="19585">27 (29.7)</offsets></td><td align="left"><offsets xml_i="30926" xml_f="30935" txt_i="19586" txt_f="19595">30 (33.0)</offsets></td><td align="left"><offsets xml_i="30957" xml_f="30966" txt_i="19596" txt_f="19605">27 (29.7)</offsets></td><td align="left"><offsets xml_i="30988" xml_f="30995" txt_i="19606" txt_f="19613">7 (7.7)</offsets></td><td align="left"><offsets xml_i="31017" xml_f="31023" txt_i="19614" txt_f="19620">0.0023</offsets></td></tr><tr><td align="left" colspan="6"><offsets xml_i="31066" xml_f="31084" txt_i="19621" txt_f="19639">Estrogen receptor*</offsets></td></tr><tr><td align="left"><offsets xml_i="31115" xml_f="31117" txt_i="19640" txt_f="19642"> 0</offsets></td><td align="left"><offsets xml_i="31139" xml_f="31148" txt_i="19643" txt_f="19652">63 (39.4)</offsets></td><td align="left"><offsets xml_i="31170" xml_f="31179" txt_i="19653" txt_f="19662">48 (30.0)</offsets></td><td align="left"><offsets xml_i="31201" xml_f="31210" txt_i="19663" txt_f="19672">34 (21.3)</offsets></td><td align="left"><offsets xml_i="31232" xml_f="31240" txt_i="19673" txt_f="19681">15 (9.4)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="31280" xml_f="31288" txt_i="19683" txt_f="19691"> 1 to 24</offsets></td><td align="left"><offsets xml_i="31310" xml_f="31318" txt_i="19692" txt_f="19700">5 (35.7)</offsets></td><td align="left"><offsets xml_i="31340" xml_f="31348" txt_i="19701" txt_f="19709">5 (35.7)</offsets></td><td align="left"><offsets xml_i="31370" xml_f="31378" txt_i="19710" txt_f="19718">4 (28.6)</offsets></td><td align="left"><offsets xml_i="31400" xml_f="31405" txt_i="19719" txt_f="19724">0 (0)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="31445" xml_f="31454" txt_i="19726" txt_f="19735"> 25 to 74</offsets></td><td align="left"><offsets xml_i="31476" xml_f="31485" txt_i="19736" txt_f="19745">69 (57.0)</offsets></td><td align="left"><offsets xml_i="31507" xml_f="31516" txt_i="19746" txt_f="19755">35 (28.9)</offsets></td><td align="left"><offsets xml_i="31538" xml_f="31547" txt_i="19756" txt_f="19765">16 (13.2)</offsets></td><td align="left"><offsets xml_i="31569" xml_f="31576" txt_i="19766" txt_f="19773">1 (0.8)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="31616" xml_f="31625" txt_i="19775" txt_f="19784"> 75 to 89</offsets></td><td align="left"><offsets xml_i="31647" xml_f="31656" txt_i="19785" txt_f="19794">63 (33.3)</offsets></td><td align="left"><offsets xml_i="31678" xml_f="31687" txt_i="19795" txt_f="19804">67 (35.5)</offsets></td><td align="left"><offsets xml_i="31709" xml_f="31718" txt_i="19805" txt_f="19814">47 (24.9)</offsets></td><td align="left"><offsets xml_i="31740" xml_f="31748" txt_i="19815" txt_f="19823">12 (6.4)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="31788" xml_f="31795" txt_i="19825" txt_f="19829"> &gt;90</offsets></td><td align="left"><offsets xml_i="31817" xml_f="31826" txt_i="19830" txt_f="19839">62 (22.1)</offsets></td><td align="left"><offsets xml_i="31848" xml_f="31858" txt_i="19840" txt_f="19850">110 (39.3)</offsets></td><td align="left"><offsets xml_i="31880" xml_f="31889" txt_i="19851" txt_f="19860">96 (34.3)</offsets></td><td align="left"><offsets xml_i="31911" xml_f="31919" txt_i="19861" txt_f="19869">12 (4.3)</offsets></td><td align="left"><offsets xml_i="31941" xml_f="31947" txt_i="19870" txt_f="19876">0.0022</offsets></td></tr><tr><td align="left" colspan="6"><offsets xml_i="31990" xml_f="32012" txt_i="19877" txt_f="19899">Progesterone receptor*</offsets></td></tr><tr><td align="left"><offsets xml_i="32043" xml_f="32045" txt_i="19900" txt_f="19902"> 0</offsets></td><td align="left"><offsets xml_i="32067" xml_f="32077" txt_i="19903" txt_f="19913">120 (39.9)</offsets></td><td align="left"><offsets xml_i="32099" xml_f="32108" txt_i="19914" txt_f="19923">92 (30.6)</offsets></td><td align="left"><offsets xml_i="32130" xml_f="32139" txt_i="19924" txt_f="19933">68 (22.6)</offsets></td><td align="left"><offsets xml_i="32161" xml_f="32169" txt_i="19934" txt_f="19942">21 (7.0)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="32209" xml_f="32217" txt_i="19944" txt_f="19952"> 1 to 24</offsets></td><td align="left"><offsets xml_i="32239" xml_f="32248" txt_i="19953" txt_f="19962">43 (42.6)</offsets></td><td align="left"><offsets xml_i="32270" xml_f="32279" txt_i="19963" txt_f="19972">29 (28.7)</offsets></td><td align="left"><offsets xml_i="32301" xml_f="32310" txt_i="19973" txt_f="19982">26 (25.7)</offsets></td><td align="left"><offsets xml_i="32332" xml_f="32339" txt_i="19983" txt_f="19990">3 (3.0)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="32379" xml_f="32388" txt_i="19992" txt_f="20001"> 25 to 74</offsets></td><td align="left"><offsets xml_i="32410" xml_f="32419" txt_i="20002" txt_f="20011">58 (29.3)</offsets></td><td align="left"><offsets xml_i="32441" xml_f="32450" txt_i="20012" txt_f="20021">87 (43.9)</offsets></td><td align="left"><offsets xml_i="32472" xml_f="32481" txt_i="20022" txt_f="20031">48 (24.2)</offsets></td><td align="left"><offsets xml_i="32503" xml_f="32510" txt_i="20032" txt_f="20039">5 (2.5)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="32550" xml_f="32559" txt_i="20041" txt_f="20050"> 75 to 89</offsets></td><td align="left"><offsets xml_i="32581" xml_f="32590" txt_i="20051" txt_f="20060">27 (24.1)</offsets></td><td align="left"><offsets xml_i="32612" xml_f="32621" txt_i="20061" txt_f="20070">42 (37.5)</offsets></td><td align="left"><offsets xml_i="32643" xml_f="32652" txt_i="20071" txt_f="20080">33 (29.5)</offsets></td><td align="left"><offsets xml_i="32674" xml_f="32682" txt_i="20081" txt_f="20089">10 (8.9)</offsets></td><td></td></tr><tr><td align="left"><offsets xml_i="32722" xml_f="32729" txt_i="20091" txt_f="20095"> &gt;90</offsets></td><td align="left"><offsets xml_i="32751" xml_f="32760" txt_i="20096" txt_f="20105">20 (36.4)</offsets></td><td align="left"><offsets xml_i="32782" xml_f="32791" txt_i="20106" txt_f="20115">21 (38.2)</offsets></td><td align="left"><offsets xml_i="32813" xml_f="32822" txt_i="20116" txt_f="20125">12 (21.8)</offsets></td><td align="left"><offsets xml_i="32844" xml_f="32851" txt_i="20126" txt_f="20133">2 (3.6)</offsets></td><td align="left"><offsets xml_i="32873" xml_f="32877" txt_i="20134" txt_f="20138">0.13</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="32923" xml_f="33029" txt_i="20139" txt_f="20245">Data presented as number of patients (%).*Percentage of positive nuclei stained with immunohistochemistry.</offsets></p></table-wrap-foot></table-wrap></sec></sec><sec><title><offsets xml_i="33088" xml_f="33098" txt_i="20248" txt_f="20258">Discussion</offsets></title><p><offsets xml_i="33109" xml_f="33320" txt_i="20259" txt_f="20470">The results from studies aiming to find molecular markers to predict treatment response in breast cancer demonstrate the importance of sensitive markers for optimizing and individualizing breast cancer therapy [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="33351" xml_f="33352" txt_i="20470" txt_f="20471">1</offsets></xref><offsets xml_i="33359" xml_f="33360" txt_i="20471" txt_f="20472">-</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="33391" xml_f="33392" txt_i="20472" txt_f="20473">4</offsets></xref><offsets xml_i="33399" xml_f="33455" txt_i="20473" txt_f="20529">]. Several studies have reported an association between </offsets><italic><offsets xml_i="33463" xml_f="33470" txt_i="20529" txt_f="20536">HOXB13 </offsets></italic><offsets xml_i="33479" xml_f="33517" txt_i="20536" txt_f="20574">mRNA expression and clinical outcome [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="33548" xml_f="33549" txt_i="20574" txt_f="20575">5</offsets></xref><offsets xml_i="33556" xml_f="33557" txt_i="20575" txt_f="20576">-</offsets><xref ref-type="bibr" rid="B9"><offsets xml_i="33588" xml_f="33589" txt_i="20576" txt_f="20577">9</offsets></xref><offsets xml_i="33596" xml_f="33597" txt_i="20577" txt_f="20578">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="33629" xml_f="33631" txt_i="20578" txt_f="20580">12</offsets></xref><offsets xml_i="33638" xml_f="33850" txt_i="20580" txt_f="20792">]. In the present study, 34% of the tumors were negative for HOXB13 protein expression - which correlates well to the proportion of tumors with undetectable mRNA expression in the study by Jansen and colleagues [</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="33881" xml_f="33882" txt_i="20792" txt_f="20793">7</offsets></xref><offsets xml_i="33889" xml_f="33891" txt_i="20793" txt_f="20795">].</offsets></p><p><offsets xml_i="33898" xml_f="34219" txt_i="20796" txt_f="21117">The present report is the first about HOXB13 protein expression and its potential as a predictor of response to endocrine treatment. In the systemically untreated group, HOXB13 could not predict the clinical outcome. This is in line with our previous findings, examining potential prognostic features of mRNA expression [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="34250" xml_f="34251" txt_i="21117" txt_f="21118">8</offsets></xref><offsets xml_i="34258" xml_f="34471" txt_i="21118" txt_f="21331">]. The clinical value of HOXB13 seems to be within its predictive power. As noted, a high expression of the protein results in less benefit from endocrine treatment with tamoxifen. Most of the previous studies of </offsets><italic><offsets xml_i="34479" xml_f="34486" txt_i="21331" txt_f="21338">HOXB13 </offsets></italic><offsets xml_i="34495" xml_f="34625" txt_i="21338" txt_f="21468">have included mainly lymph node-negative patients, but we have previously shown that the predictive feature of the two-gene ratio </offsets><italic><offsets xml_i="34633" xml_f="34647" txt_i="21468" txt_f="21482">HOXB13:IL17BR </offsets></italic><offsets xml_i="34656" xml_f="34711" txt_i="21482" txt_f="21537">is also applicable to patients with nodal involvement [</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="34742" xml_f="34743" txt_i="21537" txt_f="21538">8</offsets></xref><offsets xml_i="34750" xml_f="34934" txt_i="21538" txt_f="21722">]. Since the present study comprises a cohort of only lymph node-negative patients, it would be interesting to examine the HOXB13 protein expression in patients with nodal involvement.</offsets></p><p><offsets xml_i="34941" xml_f="35465" txt_i="21723" txt_f="22247">The causes of a high HOXB13 expression in breast tumors are still not completely clear. The gene encoding HOXB13 maps to chromosome 17q21, a region known to be amplified in breast cancer. The gene encoding HER2 also maps to this chromosomal location, which raises the question about a possible coamplification of the two genes. This is one potential reason for the positive correlation between the protein expressions we have seen in the present study, a result that is in line with a previous study by Wang and colleagues [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="35497" xml_f="35499" txt_i="22247" txt_f="22249">12</offsets></xref><offsets xml_i="35506" xml_f="35533" txt_i="22249" txt_f="22276">]. In that study, however, </offsets><italic><offsets xml_i="35541" xml_f="35548" txt_i="22276" txt_f="22283">HOXB13 </offsets></italic><offsets xml_i="35557" xml_f="35776" txt_i="22283" txt_f="22502">displayed an ER-dependent correlation with HER2 status, since the correlation was evident only in ER-positive tumors. In our study, this relationship was evident in both ER-positive tumors as well as ER-negative tumors.</offsets></p><p><offsets xml_i="35783" xml_f="35940" txt_i="22503" txt_f="22660">The correlation analysis of HOXB13 protein expression and other proteins in the present study revealed a positive correlation to ER; however, mRNA levels of </offsets><italic><offsets xml_i="35948" xml_f="35955" txt_i="22660" txt_f="22667">HOXB13 </offsets></italic><offsets xml_i="35964" xml_f="36016" txt_i="22667" txt_f="22719">have been reported to correlate negatively with ER [</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="36047" xml_f="36048" txt_i="22719" txt_f="22720">7</offsets></xref><offsets xml_i="36055" xml_f="36056" txt_i="22720" txt_f="22721">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="36088" xml_f="36090" txt_i="22721" txt_f="22723">12</offsets></xref><offsets xml_i="36097" xml_f="36140" txt_i="22723" txt_f="22766">]. There is also evidence pointing towards </offsets><italic><offsets xml_i="36148" xml_f="36155" txt_i="22766" txt_f="22773">HOXB13 </offsets></italic><offsets xml_i="36164" xml_f="36258" txt_i="22773" txt_f="22867">being an ER-responsive gene, with estradiol treatment resulting in a suppression of the gene [</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="36290" xml_f="36292" txt_i="22867" txt_f="22869">11</offsets></xref><offsets xml_i="36299" xml_f="36300" txt_i="22869" txt_f="22870">,</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="36332" xml_f="36334" txt_i="22870" txt_f="22872">12</offsets></xref><offsets xml_i="36341" xml_f="36442" txt_i="22872" txt_f="22973">]. Given that the intrinsic levels of estradiol are highly variable between different breast tumors [</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="36474" xml_f="36476" txt_i="22973" txt_f="22975">15</offsets></xref><offsets xml_i="36483" xml_f="36484" txt_i="22975" txt_f="22976">,</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="36516" xml_f="36518" txt_i="22976" txt_f="22978">16</offsets></xref><offsets xml_i="36525" xml_f="36589" txt_i="22978" txt_f="23042">] and that HOXB13 may be a coactivating factor in ER signaling [</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="36621" xml_f="36623" txt_i="23042" txt_f="23044">17</offsets></xref><offsets xml_i="36630" xml_f="37043" txt_i="23044" txt_f="23457">], one may speculate that high HOXB13 protein levels can contribute to increased ER activity despite a low estradiol concentration within the tumor. A decreased estradiol concentration can result from a lower intrinsic production, or can occur during treatment with aromatase inhibitors. In fact, it seems HOXB13 is not only a predictor of tamoxifen resistance, but is a marker of resistance to endocrine therapy </offsets><italic><offsets xml_i="37051" xml_f="37057" txt_i="23457" txt_f="23463">per se</offsets></italic><offsets xml_i="37066" xml_f="37108" txt_i="23463" txt_f="23505">. In a recent study by Ma and colleagues, </offsets><italic><offsets xml_i="37116" xml_f="37123" txt_i="23505" txt_f="23512">HOXB13 </offsets></italic><offsets xml_i="37132" xml_f="37232" txt_i="23512" txt_f="23612">gene expression was shown to predict benefit from treatment with the aromatase inhibitor letrozole [</offsets><xref ref-type="bibr" rid="B18"><offsets xml_i="37264" xml_f="37266" txt_i="23612" txt_f="23614">18</offsets></xref><offsets xml_i="37273" xml_f="37390" txt_i="23614" txt_f="23731">]. In the future, a study including both HOXB13, ER and estradiol concentrations in a set of tumors would be welcome.</offsets></p><p><offsets xml_i="37397" xml_f="37543" txt_i="23732" txt_f="23878">The mechanistic effects of HOXB13 in breast cancer are being investigated, but there are few studies published in this matter. The two-gene ratio </offsets><italic><offsets xml_i="37551" xml_f="37557" txt_i="23878" txt_f="23884">HOXB13</offsets></italic><offsets xml_i="37566" xml_f="37567" txt_i="23884" txt_f="23885">:</offsets><italic><offsets xml_i="37575" xml_f="37582" txt_i="23885" txt_f="23892">IL17BR </offsets></italic><offsets xml_i="37591" xml_f="37654" txt_i="23892" txt_f="23955">has been proposed as a marker for impaired ER gene regulation [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="37686" xml_f="37688" txt_i="23955" txt_f="23957">12</offsets></xref><offsets xml_i="37695" xml_f="37792" txt_i="23957" txt_f="24054">]. The presence of PR in a tumor is the most commonly used indicator of functional ER signaling [</offsets><xref ref-type="bibr" rid="B19"><offsets xml_i="37824" xml_f="37826" txt_i="24054" txt_f="24056">19</offsets></xref><offsets xml_i="37833" xml_f="38056" txt_i="24056" txt_f="24279">]; patients with tumors expressing both ER and PR benefit most from tamoxifen therapy. The worse prognosis when HOXB13 is high, which is seen in our study, shows that HOXB13 in fact has a predictive value in addition to PR.</offsets></p></sec><sec><title><offsets xml_i="38078" xml_f="38089" txt_i="24281" txt_f="24292">Conclusions</offsets></title><p><offsets xml_i="38100" xml_f="38450" txt_i="24293" txt_f="24643">The present study is the first about HOXB13 protein expression in breast cancer. We have shown that HOXB13 is predictive of response to adjuvant tamoxifen therapy, in terms of a longer DRFS and BCS for tamoxifen-treated breast cancer patients if the protein expression is low or absent. Nevertheless, further studies are needed to verify our results.</offsets></p></sec><sec><title><offsets xml_i="38472" xml_f="38485" txt_i="24645" txt_f="24658">Abbreviations</offsets></title><p><offsets xml_i="38496" xml_f="38694" txt_i="24659" txt_f="24857">BCS: breast cancer survival; DRFS: distant recurrence-free survival; ER: estrogen receptor; HER2: human epidermal growth factor receptor 2; PCR: polymerase chain reaction; PR: progesterone receptor.</offsets></p></sec><sec><title><offsets xml_i="38716" xml_f="38735" txt_i="24859" txt_f="24878">Competing interests</offsets></title><p><offsets xml_i="38746" xml_f="38804" txt_i="24879" txt_f="24937">The authors declare that they have no competing interests.</offsets></p></sec><sec><title><offsets xml_i="38826" xml_f="38848" txt_i="24939" txt_f="24961">Authors' contributions</offsets></title><p><offsets xml_i="38859" xml_f="39489" txt_i="24962" txt_f="25592">P-LJ participated in the study design, performed the laboratory work, evaluated the immunohistochemical staining, performed the statistical analyses, and wrote the manuscript. AJ participated in the study design, performed laboratory work, and provided critical revision of the manuscript. TF, LS, and BN provided the study material and clinical information, and collected laboratory data on the study patients. OS participated in the study design, evaluated the immunohistochemical staining, performed the statistical analyses, and provided critical revision of the manuscript. All authors read and approved the final manuscript.</offsets></p></sec></body><back><sec><title>Acknowledgements</title><p>The present study was supported by grants from the Swedish Cancer Society, the Swedish Research Council, and King Gustaf V Jubilee Fund.</p></sec><ref-list><ref id="B1"><mixed-citation publication-type="journal"><name><surname>Ma</surname><given-names>XJ</given-names></name><name><surname>Salunga</surname><given-names>R</given-names></name><name><surname>Dahiya</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Carney</surname><given-names>E</given-names></name><name><surname>Durbecq</surname><given-names>V</given-names></name><name><surname>Harris</surname><given-names>A</given-names></name><name><surname>Goss</surname><given-names>P</given-names></name><name><surname>Sotiriou</surname><given-names>C</given-names></name><name><surname>Erlander</surname><given-names>M</given-names></name><name><surname>Sgroi</surname><given-names>D</given-names></name><article-title>A five-gene molecular grade index and HOXB13:IL17BR are complementary prognostic factors in early stage breast cancer</article-title><source>Clin Cancer Res</source><year>2008</year><volume>14</volume><fpage>2601</fpage><lpage>2608</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-07-5026</pub-id><pub-id pub-id-type="pmid">18451222</pub-id></mixed-citation></ref><ref id="B2"><mixed-citation publication-type="journal"><name><surname>Paik</surname><given-names>S</given-names></name><name><surname>Shak</surname><given-names>S</given-names></name><name><surname>Tang</surname><given-names>G</given-names></name><name><surname>Kim</surname><given-names>C</given-names></name><name><surname>Baker</surname><given-names>J</given-names></name><name><surname>Cronin</surname><given-names>M</given-names></name><name><surname>Baehner</surname><given-names>FL</given-names></name><name><surname>Walker</surname><given-names>MG</given-names></name><name><surname>Watson</surname><given-names>D</given-names></name><name><surname>Park</surname><given-names>T</given-names></name><name><surname>Hiller</surname><given-names>W</given-names></name><name><surname>Fisher</surname><given-names>ER</given-names></name><name><surname>Wickerham</surname><given-names>DL</given-names></name><name><surname>Bryant</surname><given-names>J</given-names></name><name><surname>Wolmark</surname><given-names>N</given-names></name><article-title>A multigene assay to predict recurrence of tamoxifen-treated, node-negative breast cancer</article-title><source>N Engl J Med</source><year>2004</year><volume>351</volume><fpage>2817</fpage><lpage>2826</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa041588</pub-id><pub-id pub-id-type="pmid">15591335</pub-id></mixed-citation></ref><ref id="B3"><mixed-citation publication-type="journal"><name><surname>van 't Veer</surname><given-names>LJ</given-names></name><name><surname>Dai</surname><given-names>H</given-names></name><name><surname>van de Vijver</surname><given-names>MJ</given-names></name><name><surname>He</surname><given-names>YD</given-names></name><name><surname>Hart</surname><given-names>AA</given-names></name><name><surname>Mao</surname><given-names>M</given-names></name><name><surname>Peterse</surname><given-names>HL</given-names></name><name><surname>van der Kooy</surname><given-names>K</given-names></name><name><surname>Marton</surname><given-names>MJ</given-names></name><name><surname>Witteveen</surname><given-names>AT</given-names></name><name><surname>Schreiber</surname><given-names>GJ</given-names></name><name><surname>Kerkhoven</surname><given-names>RM</given-names></name><name><surname>Roberts</surname><given-names>C</given-names></name><name><surname>Linsley</surname><given-names>PS</given-names></name><name><surname>Bernards</surname><given-names>R</given-names></name><name><surname>Friend</surname><given-names>SH</given-names></name><article-title>Gene expression profiling predicts clinical outcome of breast cancer</article-title><source>Nature</source><year>2002</year><volume>415</volume><fpage>530</fpage><lpage>536</lpage><pub-id pub-id-type="doi">10.1038/415530a</pub-id><pub-id pub-id-type="pmid">11823860</pub-id></mixed-citation></ref><ref id="B4"><mixed-citation publication-type="journal"><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Klijn</surname><given-names>JG</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Sieuwerts</surname><given-names>AM</given-names></name><name><surname>Look</surname><given-names>MP</given-names></name><name><surname>Yang</surname><given-names>F</given-names></name><name><surname>Talantov</surname><given-names>D</given-names></name><name><surname>Timmermans</surname><given-names>M</given-names></name><name><surname>Meijer-van Gelder</surname><given-names>ME</given-names></name><name><surname>Yu</surname><given-names>J</given-names></name><name><surname>Jatkoe</surname><given-names>T</given-names></name><name><surname>Berns</surname><given-names>EM</given-names></name><name><surname>Atkins</surname><given-names>D</given-names></name><name><surname>Foekens</surname><given-names>JA</given-names></name><article-title>Gene-expression profiles to predict distant metastasis of lymph-node-negative primary breast cancer</article-title><source>Lancet</source><year>2005</year><volume>365</volume><fpage>671</fpage><lpage>679</lpage><pub-id pub-id-type="pmid">15721472</pub-id></mixed-citation></ref><ref id="B5"><mixed-citation publication-type="journal"><name><surname>Ma</surname><given-names>XJ</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Ryan</surname><given-names>PD</given-names></name><name><surname>Isakoff</surname><given-names>SJ</given-names></name><name><surname>Barmettler</surname><given-names>A</given-names></name><name><surname>Fuller</surname><given-names>A</given-names></name><name><surname>Muir</surname><given-names>B</given-names></name><name><surname>Mohapatra</surname><given-names>G</given-names></name><name><surname>Salunga</surname><given-names>R</given-names></name><name><surname>Tuggle</surname><given-names>JT</given-names></name><name><surname>Tran</surname><given-names>Y</given-names></name><name><surname>Tran</surname><given-names>D</given-names></name><name><surname>Tassin</surname><given-names>A</given-names></name><name><surname>Amon</surname><given-names>P</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Enright</surname><given-names>E</given-names></name><name><surname>Stecker</surname><given-names>K</given-names></name><name><surname>Estepa-Sabal</surname><given-names>E</given-names></name><name><surname>Smith</surname><given-names>B</given-names></name><name><surname>Younger</surname><given-names>J</given-names></name><name><surname>Balis</surname><given-names>U</given-names></name><name><surname>Michaelson</surname><given-names>J</given-names></name><name><surname>Bhan</surname><given-names>A</given-names></name><name><surname>Habin</surname><given-names>K</given-names></name><name><surname>Baer</surname><given-names>TM</given-names></name><name><surname>Brugge</surname><given-names>J</given-names></name><name><surname>Haber</surname><given-names>DA</given-names></name><name><surname>Erlander</surname><given-names>MG</given-names></name><name><surname>Sgroi</surname><given-names>DC</given-names></name><article-title>A two-gene expression ratio predicts clinical outcome in breast cancer patients treated with tamoxifen</article-title><source>Cancer Cell</source><year>2004</year><volume>5</volume><fpage>607</fpage><lpage>616</lpage><pub-id pub-id-type="doi">10.1016/j.ccr.2004.05.015</pub-id><pub-id pub-id-type="pmid">15193263</pub-id></mixed-citation></ref><ref id="B6"><mixed-citation publication-type="journal"><name><surname>Goetz</surname><given-names>MP</given-names></name><name><surname>Suman</surname><given-names>VJ</given-names></name><name><surname>Ingle</surname><given-names>JN</given-names></name><name><surname>Nibbe</surname><given-names>AM</given-names></name><name><surname>Visscher</surname><given-names>DW</given-names></name><name><surname>Reynolds</surname><given-names>CA</given-names></name><name><surname>Lingle</surname><given-names>WL</given-names></name><name><surname>Erlander</surname><given-names>M</given-names></name><name><surname>Ma</surname><given-names>XJ</given-names></name><name><surname>Sgroi</surname><given-names>DC</given-names></name><name><surname>Perez</surname><given-names>EA</given-names></name><name><surname>Couch</surname><given-names>FJ</given-names></name><article-title>A two-gene expression ratio of homeobox 13 and interleukin-17B receptor for prediction of recurrence and survival in women receiving adjuvant tamoxifen</article-title><source>Clin Cancer Res</source><year>2006</year><volume>12</volume><fpage>2080</fpage><lpage>2087</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-05-1263</pub-id><pub-id pub-id-type="pmid">16609019</pub-id></mixed-citation></ref><ref id="B7"><mixed-citation publication-type="journal"><name><surname>Jansen</surname><given-names>MP</given-names></name><name><surname>Sieuwerts</surname><given-names>AM</given-names></name><name><surname>Look</surname><given-names>MP</given-names></name><name><surname>Ritstier</surname><given-names>K</given-names></name><name><surname>Meijer-van Gelder</surname><given-names>ME</given-names></name><name><surname>van Staveren</surname><given-names>IL</given-names></name><name><surname>Klijn</surname><given-names>JG</given-names></name><name><surname>Foekens</surname><given-names>JA</given-names></name><name><surname>Berns</surname><given-names>EM</given-names></name><article-title>HOXB13-to-IL17BR expression ratio is related with tumor aggressiveness and response to tamoxifen of recurrent breast cancer: a retrospective study</article-title><source>J Clin Oncol</source><year>2007</year><volume>25</volume><fpage>662</fpage><lpage>668</lpage><pub-id pub-id-type="doi">10.1200/JCO.2006.07.3676</pub-id><pub-id pub-id-type="pmid">17308270</pub-id></mixed-citation></ref><ref id="B8"><mixed-citation publication-type="journal"><name><surname>Jerevall</surname><given-names>PL</given-names></name><name><surname>Brommesson</surname><given-names>S</given-names></name><name><surname>Strand</surname><given-names>C</given-names></name><name><surname>Gruvberger-Saal</surname><given-names>S</given-names></name><name><surname>Malmstrom</surname><given-names>P</given-names></name><name><surname>Nordenskjold</surname><given-names>B</given-names></name><name><surname>Wingren</surname><given-names>S</given-names></name><name><surname>Soderkvist</surname><given-names>P</given-names></name><name><surname>Ferno</surname><given-names>M</given-names></name><name><surname>Stal</surname><given-names>O</given-names></name><article-title>Exploring the two-gene ratio in breast cancer - independent roles for HOXB13 and IL17BR in prediction of clinical outcome</article-title><source>Breast Cancer Res Treat</source><year>2008</year><volume>107</volume><fpage>225</fpage><lpage>234</lpage><pub-id pub-id-type="doi">10.1007/s10549-007-9541-8</pub-id><pub-id pub-id-type="pmid">17453342</pub-id></mixed-citation></ref><ref id="B9"><mixed-citation publication-type="journal"><name><surname>Ma</surname><given-names>XJ</given-names></name><name><surname>Hilsenbeck</surname><given-names>SG</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Ding</surname><given-names>L</given-names></name><name><surname>Sgroi</surname><given-names>DC</given-names></name><name><surname>Bender</surname><given-names>RA</given-names></name><name><surname>Osborne</surname><given-names>CK</given-names></name><name><surname>Allred</surname><given-names>DC</given-names></name><name><surname>Erlander</surname><given-names>MG</given-names></name><article-title>The HOXB13:IL17BR expression index is a prognostic factor in early-stage breast cancer</article-title><source>J Clin Oncol</source><year>2006</year><volume>24</volume><fpage>4611</fpage><lpage>4619</lpage><pub-id pub-id-type="doi">10.1200/JCO.2006.06.6944</pub-id><pub-id pub-id-type="pmid">17008703</pub-id></mixed-citation></ref><ref id="B10"><mixed-citation publication-type="journal"><name><surname>Cantile</surname><given-names>M</given-names></name><name><surname>Pettinato</surname><given-names>G</given-names></name><name><surname>Procino</surname><given-names>A</given-names></name><name><surname>Feliciello</surname><given-names>I</given-names></name><name><surname>Cindolo</surname><given-names>L</given-names></name><name><surname>Cillo</surname><given-names>C</given-names></name><article-title>In vivo expression of the whole HOX gene network in human breast cancer</article-title><source>Eur J Cancer</source><year>2003</year><volume>39</volume><fpage>257</fpage><lpage>264</lpage><pub-id pub-id-type="doi">10.1016/S0959-8049(02)00599-3</pub-id><pub-id pub-id-type="pmid">12509959</pub-id></mixed-citation></ref><ref id="B11"><mixed-citation publication-type="journal"><name><surname>Rodriguez</surname><given-names>BA</given-names></name><name><surname>Cheng</surname><given-names>AS</given-names></name><name><surname>Yan</surname><given-names>PS</given-names></name><name><surname>Potter</surname><given-names>D</given-names></name><name><surname>Agosto-Perez</surname><given-names>FJ</given-names></name><name><surname>Shapiro</surname><given-names>CL</given-names></name><name><surname>Huang</surname><given-names>TH</given-names></name><article-title>Epigenetic repression of the estrogen-regulated Homeobox B13 gene in breast cancer</article-title><source>Carcinogenesis</source><year>2008</year><volume>29</volume><fpage>1459</fpage><lpage>1465</lpage><pub-id pub-id-type="doi">10.1093/carcin/bgn115</pub-id><pub-id pub-id-type="pmid">18499701</pub-id></mixed-citation></ref><ref id="B12"><mixed-citation publication-type="journal"><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Dahiya</surname><given-names>S</given-names></name><name><surname>Provencher</surname><given-names>H</given-names></name><name><surname>Muir</surname><given-names>B</given-names></name><name><surname>Carney</surname><given-names>E</given-names></name><name><surname>Coser</surname><given-names>K</given-names></name><name><surname>Shioda</surname><given-names>T</given-names></name><name><surname>Ma</surname><given-names>XJ</given-names></name><name><surname>Sgroi</surname><given-names>DC</given-names></name><article-title>The prognostic biomarkers HOXB13, IL17BR, and CHDH are regulated by estrogen in breast cancer</article-title><source>Clin Cancer Res</source><year>2007</year><volume>13</volume><fpage>6327</fpage><lpage>6334</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-07-0310</pub-id><pub-id pub-id-type="pmid">17975144</pub-id></mixed-citation></ref><ref id="B13"><mixed-citation publication-type="journal"><name><surname>Rutqvist</surname><given-names>LE</given-names></name><name><surname>Johansson</surname><given-names>H</given-names></name><article-title>Long-term follow-up of the randomized Stockholm trial on adjuvant tamoxifen among postmenopausal patients with early stage breast cancer</article-title><source>Acta Oncol</source><year>2007</year><volume>46</volume><fpage>133</fpage><lpage>145</lpage><pub-id pub-id-type="doi">10.1080/02841860601034834</pub-id><pub-id pub-id-type="pmid">17453361</pub-id></mixed-citation></ref><ref id="B14"><mixed-citation publication-type="journal"><name><surname>Wrange</surname><given-names>O</given-names></name><name><surname>Nordenskjöld</surname><given-names>B</given-names></name><name><surname>Gustafsson</surname><given-names>JA</given-names></name><article-title>Cytosol estradiol receptor in human mammary carcinoma: an assay based on isoelectric focusing in polyacrylamide gel</article-title><source>Anal Biochem</source><year>1978</year><volume>85</volume><fpage>461</fpage><lpage>475</lpage><pub-id pub-id-type="doi">10.1016/0003-2697(78)90243-9</pub-id><pub-id pub-id-type="pmid">565601</pub-id></mixed-citation></ref><ref id="B15"><mixed-citation publication-type="journal"><name><surname>Nagai</surname><given-names>R</given-names></name><name><surname>Kataoka</surname><given-names>M</given-names></name><name><surname>Kobayashi</surname><given-names>S</given-names></name><name><surname>Ishihara</surname><given-names>K</given-names></name><name><surname>Tobioka</surname><given-names>N</given-names></name><name><surname>Nakashima</surname><given-names>K</given-names></name><name><surname>Naruse</surname><given-names>M</given-names></name><name><surname>Saito</surname><given-names>K</given-names></name><name><surname>Sakuma</surname><given-names>S</given-names></name><article-title>Estrogen and progesterone receptors in human breast cancer with concomitant assay of plasma 17β-estradiol, progesterone, and prolactin levels</article-title><source>Cancer Res</source><year>1979</year><volume>39</volume><fpage>1834</fpage><lpage>1840</lpage><pub-id pub-id-type="pmid">427816</pub-id></mixed-citation></ref><ref id="B16"><mixed-citation publication-type="journal"><name><surname>van Landeghem</surname><given-names>AA</given-names></name><name><surname>Poortman</surname><given-names>J</given-names></name><name><surname>Nabuurs</surname><given-names>M</given-names></name><name><surname>Thijssen</surname><given-names>JH</given-names></name><article-title>Endogenous concentration and subcellular distribution of androgens in normal and malignant human breast tissue</article-title><source>Cancer Res</source><year>1985</year><volume>45</volume><fpage>2907</fpage><lpage>2912</lpage><pub-id pub-id-type="pmid">3157451</pub-id></mixed-citation></ref><ref id="B17"><mixed-citation publication-type="journal"><name><surname>Jung</surname><given-names>C</given-names></name><name><surname>Kim</surname><given-names>RS</given-names></name><name><surname>Zhang</surname><given-names>HJ</given-names></name><name><surname>Lee</surname><given-names>SJ</given-names></name><name><surname>Jeng</surname><given-names>MH</given-names></name><article-title>HOXB13 induces growth suppression of prostate cancer cells as a repressor of hormone-activated androgen receptor signaling</article-title><source>Cancer Res</source><year>2004</year><volume>64</volume><fpage>9185</fpage><lpage>9192</lpage><pub-id pub-id-type="doi">10.1158/0008-5472.CAN-04-1330</pub-id><pub-id pub-id-type="pmid">15604291</pub-id></mixed-citation></ref><ref id="B18"><mixed-citation publication-type="journal"><name><surname>Ma</surname><given-names>XJ</given-names></name><name><surname>Bandaru</surname><given-names>R</given-names></name><name><surname>Letzkus</surname><given-names>M</given-names></name><name><surname>Philips</surname><given-names>P</given-names></name><name><surname>Barrett</surname><given-names>JC</given-names></name><name><surname>Erlander</surname><given-names>M</given-names></name><name><surname>Goetz</surname><given-names>M</given-names></name><name><surname>Sgroi</surname><given-names>D</given-names></name><name><surname>Gardner</surname><given-names>H</given-names></name><name><surname>Baselga</surname><given-names>J</given-names></name><article-title>HOXB13 may predict response to neoadjuvant letrozole in patients with estrogen receptor-positive breast cancer [abstract]</article-title><source>Cancer Res</source><year>2009</year><volume>69</volume><issue>24 Suppl</issue><fpage>2120</fpage></mixed-citation></ref><ref id="B19"><mixed-citation publication-type="journal"><name><surname>Lapidus</surname><given-names>RG</given-names></name><name><surname>Nass</surname><given-names>SJ</given-names></name><name><surname>Davidson</surname><given-names>NE</given-names></name><article-title>The loss of estrogen and progesterone receptor gene expression in human breast cancer</article-title><source>J Mammary Gland Biol Neoplasia</source><year>1998</year><volume>3</volume><fpage>85</fpage><lpage>94</lpage><pub-id pub-id-type="doi">10.1023/A:1018778403001</pub-id><pub-id pub-id-type="pmid">10819507</pub-id></mixed-citation></ref></ref-list></back></article>