<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">PLoS One</journal-id><journal-id journal-id-type="iso-abbrev">PLoS ONE</journal-id><journal-id journal-id-type="publisher-id">plos</journal-id><journal-id journal-id-type="pmc">plosone</journal-id><journal-title-group><journal-title>PLoS ONE</journal-title></journal-title-group><issn pub-type="epub">1932-6203</issn><publisher><publisher-name>Public Library of Science</publisher-name><publisher-loc>San Francisco, USA</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">23468874</article-id><article-id pub-id-type="pmc">3585302</article-id><article-id pub-id-type="publisher-id">PONE-D-12-19184</article-id><article-id pub-id-type="doi">10.1371/journal.pone.0056710</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group><subj-group subj-group-type="Discipline-v2"><subject>Biology</subject><subj-group><subject>Computational Biology</subject><subj-group><subject>Sequence Analysis</subject></subj-group></subj-group><subj-group><subject>Ecology</subject><subj-group><subject>Microbial Ecology</subject></subj-group></subj-group><subj-group><subject>Microbiology</subject><subj-group><subject>Bacteriology</subject><subj-group><subject>Bacterial Taxonomy</subject></subj-group></subj-group><subj-group><subject>Applied Microbiology</subject><subject>Medical Microbiology</subject><subject>Microbial Ecology</subject></subj-group></subj-group></subj-group><subj-group subj-group-type="Discipline-v2"><subject>Medicine</subject><subj-group><subject>Clinical Research Design</subject><subj-group><subject>Clinical Trials</subject></subj-group></subj-group><subj-group><subject>Diagnostic Medicine</subject><subj-group><subject>Pathology</subject><subj-group><subject>Clinical Pathology</subject><subj-group><subject>Clinical Microbiology</subject></subj-group></subj-group></subj-group></subj-group><subj-group><subject>Gastroenterology and Hepatology</subject><subj-group><subject>Bacterial and Foodborne Illness</subject><subject>Pediatric Gastroenterology</subject></subj-group></subj-group><subj-group><subject>Pediatrics</subject><subj-group><subject>Pediatric Gastroenterology</subject></subj-group></subj-group></subj-group></article-categories><title-group><article-title>454 Pyrosequencing Analysis on Faecal Samples from a Randomized DBPC Trial of Colicky Infants Treated with <italic>Lactobacillus reuteri</italic> DSM 17938</article-title><alt-title alt-title-type="running-head">454-Pyrosequencing Analysis of Colicky Infants</alt-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Roos</surname><given-names>Stefan</given-names></name><xref ref-type="aff" rid="aff1">
<sup>1</sup>
</xref></contrib><contrib contrib-type="author"><name><surname>Dicksved</surname><given-names>Johan</given-names></name><xref ref-type="aff" rid="aff4">
<sup>4</sup>
</xref></contrib><contrib contrib-type="author"><name><surname>Tarasco</surname><given-names>Valentina</given-names></name><xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref></contrib><contrib contrib-type="author"><name><surname>Locatelli</surname><given-names>Emanuela</given-names></name><xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref></contrib><contrib contrib-type="author"><name><surname>Ricceri</surname><given-names>Fulvio</given-names></name><xref ref-type="aff" rid="aff3">
<sup>3</sup>
</xref></contrib><contrib contrib-type="author"><name><surname>Grandin</surname><given-names>Ulf</given-names></name><xref ref-type="aff" rid="aff5">
<sup>5</sup>
</xref></contrib><contrib contrib-type="author"><name><surname>Savino</surname><given-names>Francesco</given-names></name><xref ref-type="aff" rid="aff2">
<sup>2</sup>
</xref><xref ref-type="corresp" rid="cor1">
<sup>*</sup>
</xref></contrib></contrib-group><aff id="aff1">
<label>1</label>
<addr-line>Department of Microbiology, Swedish University of Agricultural Sciences, Uppsala, Sweden</addr-line>
</aff><aff id="aff2">
<label>2</label>
<addr-line>Department of Pediatrics, Regina Margherita Children Hospital, University of Turin, Torino, Italy</addr-line>
</aff><aff id="aff3">
<label>3</label>
<addr-line>Human Genetics Foundation–Molecular and Genetic Epidemiology Unit, Torino, Italy</addr-line>
</aff><aff id="aff4">
<label>4</label>
<addr-line>Department of Animal Nutrition and Management, Swedish University of Agricultural Sciences, Uppsala, Sweden</addr-line>
</aff><aff id="aff5">
<label>5</label>
<addr-line>Department of Aquatic Sciences and Assessment, Swedish University of Agricultural Sciences, Uppsala, Sweden</addr-line>
</aff><contrib-group><contrib contrib-type="editor"><name><surname>Quince</surname><given-names>Christopher</given-names></name><role>Editor</role><xref ref-type="aff" rid="edit1"></xref></contrib></contrib-group><aff id="edit1">
<addr-line>University of Glasgow, United Kingdom</addr-line>
</aff><author-notes><corresp id="cor1">* E-mail: <email>francesco.savino@unito.it</email></corresp><fn fn-type="COI-statement"><p><bold>Competing Interests: </bold>The University of Turin and Swedish University of Agricultural Sciences both received a donation from the manufacturer of L. reuteri DSM 17938, BioGaia AB, Lund, Sweden. There are no further patents, products in development or marketed products to declare. This does not alter the authors' adherence to all the PLOS ONE policies on sharing data and materials.</p></fn><fn fn-type="con"><p>Conceived and designed the experiments: FS SR. Performed the experiments: SR JD. Analyzed the data: VT EL FR. Contributed reagents/materials/analysis tools: SR JD UG. Wrote the paper: FS SR VT EL FR.</p></fn></author-notes><pub-date pub-type="collection"><year>2013</year></pub-date><pub-date pub-type="epub"><day>28</day><month>2</month><year>2013</year></pub-date><volume>8</volume><issue>2</issue><elocation-id>e56710</elocation-id><history><date date-type="received"><day>28</day><month>6</month><year>2012</year></date><date date-type="accepted"><day>16</day><month>1</month><year>2013</year></date></history><permissions><copyright-statement>© 2013 Roos et al</copyright-statement><copyright-year>2013</copyright-year><copyright-holder>Roos et al</copyright-holder><license xlink:href="http://creativecommons.org/licenses/by/4.0/"><license-p>This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are properly credited.</license-p></license></permissions><abstract><sec><title><offsets xml_i="6786" xml_f="6795" txt_i="11" txt_f="20">Objective</offsets></title><p><offsets xml_i="6806" xml_f="6949" txt_i="21" txt_f="164">To analyze the global microbial composition, using large-scale DNA sequencing of 16 S rRNA genes, in faecal samples from colicky infants given </offsets><italic><offsets xml_i="6957" xml_f="6967" txt_i="164" txt_f="174">L. reuteri</offsets></italic><offsets xml_i="6976" xml_f="6998" txt_i="174" txt_f="196"> DSM 17938 or placebo.</offsets></p></sec><sec><title><offsets xml_i="7020" xml_f="7027" txt_i="198" txt_f="205">Methods</offsets></title><p><offsets xml_i="7038" xml_f="7137" txt_i="206" txt_f="305">Twenty-nine colicky infants (age 10–60 days) were enrolled and randomly assigned to receive either </offsets><italic><offsets xml_i="7145" xml_f="7166" txt_i="305" txt_f="326">Lactobacillus reuteri</offsets></italic><offsets xml_i="7175" xml_f="7179" txt_i="326" txt_f="330"> (10</offsets><sup><offsets xml_i="7184" xml_f="7185" txt_i="330" txt_f="331">8</offsets></sup><offsets xml_i="7191" xml_f="7775" txt_i="331" txt_f="915"> cfu) or a placebo once daily for 21 days. Responders were defined as subjects with a decrease of 50% in daily crying time at day 21 compared with the starting point. The microbiota of faecal samples from day 1 and 21 were analyzed using 454 pyrosequencing. The primers: Bakt_341F and Bakt_805R, complemented with 454 adapters and sample specific barcodes were used for PCR amplification of the 16 S rRNA genes. The structure of the data was explored by using permutational multivariate analysis of variance and effects of different variables were visualized with ordination analysis.</offsets></p></sec><sec><title><offsets xml_i="7797" xml_f="7804" txt_i="917" txt_f="924">Results</offsets></title><p><offsets xml_i="7815" xml_f="7863" txt_i="925" txt_f="973">The infants’ faecal microbiota were composed of </offsets><italic><offsets xml_i="7871" xml_f="7885" txt_i="973" txt_f="987">Proteobacteria</offsets></italic><offsets xml_i="7894" xml_f="7896" txt_i="987" txt_f="989">, </offsets><italic><offsets xml_i="7904" xml_f="7914" txt_i="989" txt_f="999">Firmicutes</offsets></italic><offsets xml_i="7923" xml_f="7925" txt_i="999" txt_f="1001">, </offsets><italic><offsets xml_i="7933" xml_f="7947" txt_i="1001" txt_f="1015">Actinobacteria</offsets></italic><offsets xml_i="7956" xml_f="7961" txt_i="1015" txt_f="1020"> and </offsets><italic><offsets xml_i="7969" xml_f="7982" txt_i="1020" txt_f="1033">Bacteroidetes</offsets></italic><offsets xml_i="7991" xml_f="8120" txt_i="1033" txt_f="1162"> as the four main phyla. The composition of the microbiota in infants with colic had very high inter-individual variability with </offsets><italic><offsets xml_i="8128" xml_f="8138" txt_i="1162" txt_f="1172">Firmicutes</offsets></italic><offsets xml_i="8147" xml_f="8148" txt_i="1172" txt_f="1173">/</offsets><italic><offsets xml_i="8156" xml_f="8169" txt_i="1173" txt_f="1186">Bacteroidetes</offsets></italic><offsets xml_i="8178" xml_f="8311" txt_i="1186" txt_f="1319"> ratios varying from 4000 to 0.025. On an individual basis, the microbiota was, however, relatively stable over time. Treatment with </offsets><italic><offsets xml_i="8319" xml_f="8329" txt_i="1319" txt_f="1329">L. reuteri</offsets></italic><offsets xml_i="8338" xml_f="8526" txt_i="1329" txt_f="1517"> DSM 17938 did not change the global composition of the microbiota, but when comparing responders with non-responders the group responders had an increased relative abundance of the phyla </offsets><italic><offsets xml_i="8534" xml_f="8547" txt_i="1517" txt_f="1530">Bacteroidetes</offsets></italic><offsets xml_i="8556" xml_f="8567" txt_i="1530" txt_f="1541"> and genus </offsets><italic><offsets xml_i="8575" xml_f="8586" txt_i="1541" txt_f="1552">Bacteroides</offsets></italic><offsets xml_i="8595" xml_f="8747" txt_i="1552" txt_f="1704"> at day 21 compared with day 0. Furthermore, the phyla composition of the infants at day 21 could be divided into three enterotype groups, dominated by </offsets><italic><offsets xml_i="8755" xml_f="8765" txt_i="1704" txt_f="1714">Firmicutes</offsets></italic><offsets xml_i="8774" xml_f="8776" txt_i="1714" txt_f="1716">, </offsets><italic><offsets xml_i="8784" xml_f="8797" txt_i="1716" txt_f="1729">Bacteroidetes</offsets></italic><offsets xml_i="8806" xml_f="8812" txt_i="1729" txt_f="1735">, and </offsets><italic><offsets xml_i="8820" xml_f="8834" txt_i="1735" txt_f="1749">Actinobacteria</offsets></italic><offsets xml_i="8843" xml_f="8858" txt_i="1749" txt_f="1764">, respectively.</offsets></p></sec><sec><title><offsets xml_i="8880" xml_f="8890" txt_i="1766" txt_f="1776">Conclusion</offsets></title><p><offsets xml_i="8901" xml_f="8902" txt_i="1777" txt_f="1778">
</offsets><italic><offsets xml_i="8910" xml_f="8920" txt_i="1778" txt_f="1788">L. reuteri</offsets></italic><offsets xml_i="8929" xml_f="9022" txt_i="1788" txt_f="1881"> DSM 17938 did not affect the global composition of the microbiota. However, the increase of </offsets><italic><offsets xml_i="9030" xml_f="9043" txt_i="1881" txt_f="1894">Bacteroidetes</offsets></italic><offsets xml_i="9052" xml_f="9164" txt_i="1894" txt_f="2006"> in the responder infants indicated that a decrease in colicky symptoms was linked to changes of the microbiota.</offsets></p></sec><sec><title><offsets xml_i="9186" xml_f="9204" txt_i="2008" txt_f="2026">Trial Registration</offsets></title><p><offsets xml_i="9215" xml_f="9234" txt_i="2027" txt_f="2046">ClinicalTrials.gov </offsets><ext-link ext-link-type="uri" xlink:href="http://clinicaltrials.gov/ct2/show/NCT00893711"><offsets xml_i="9324" xml_f="9335" txt_i="2046" txt_f="2057">NCT00893711</offsets></ext-link><offsets xml_i="9346" xml_f="9347" txt_i="2057" txt_f="2058">
</offsets></p></sec></abstract><funding-group><funding-statement>This study was supported by the University of Turin (Italy)- Dipartimento di Scienze Pediatriche e dell'Adolescenza and the Swedish University of Agricultural Sciences, Department of Microbiology, Uppsala, Sweden, which both received a donation from the manufacturer of L. reuteri DSM 17938, BioGaia AB, Lund, Sweden. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</funding-statement></funding-group><counts><page-count count="5"></page-count></counts></article-meta></front><body><sec id="s1"><title><offsets xml_i="9982" xml_f="9994" txt_i="2067" txt_f="2079">Introduction</offsets></title><p><offsets xml_i="10005" xml_f="10461" txt_i="2080" txt_f="2536">Infantile colic is one of the most common problems in paediatric medicine. It is a clinical condition characterized by inconsolable crying, fussing and irritability in otherwise healthy newborn infants during the first 3 months of life. The diagnosis is clinical according to an old definition based on crying time, known as the “Wessel’s rule of three”: unexplained episodes of paroxysmal crying for more than 3 h/day for 3 days/week for at least 3 weeks </offsets><xref rid="pone.0056710-Wessel1" ref-type="bibr"><offsets xml_i="10510" xml_f="10513" txt_i="2536" txt_f="2539">[1]</offsets></xref><offsets xml_i="10520" xml_f="10689" txt_i="2539" txt_f="2708">. A recent study identified some perinatal predisposing factors including maternal education, smoking habits, cheese consumption, hostility scores and domestic violence </offsets><xref rid="pone.0056710-Yalin1" ref-type="bibr"><offsets xml_i="10737" xml_f="10740" txt_i="2708" txt_f="2711">[2]</offsets></xref><offsets xml_i="10747" xml_f="10899" txt_i="2711" txt_f="2863">. Despite, however, many studies conducted to determine the cause of this condition, its pathogenesis remains unclear and is most likely multifactorial </offsets><xref rid="pone.0056710-Savino1" ref-type="bibr"><offsets xml_i="10948" xml_f="10951" txt_i="2863" txt_f="2866">[3]</offsets></xref><offsets xml_i="10958" xml_f="10960" txt_i="2866" txt_f="2868">, </offsets><xref rid="pone.0056710-Savino2" ref-type="bibr"><offsets xml_i="11009" xml_f="11012" txt_i="2868" txt_f="2871">[4]</offsets></xref><offsets xml_i="11019" xml_f="11020" txt_i="2871" txt_f="2872">.</offsets></p><p><offsets xml_i="11027" xml_f="11227" txt_i="2873" txt_f="3073">Increasing importance has been recently assigned to the role of an aberrant gut microbiota in infants suffering from infantile colic, suggesting its influence on gut motor function and gas production </offsets><xref rid="pone.0056710-Savino3" ref-type="bibr"><offsets xml_i="11276" xml_f="11279" txt_i="3073" txt_f="3076">[5]</offsets></xref><offsets xml_i="11286" xml_f="11354" txt_i="3076" txt_f="3144"> and emphasizing the possible inflammatory origin of this condition </offsets><xref rid="pone.0056710-Rhoads1" ref-type="bibr"><offsets xml_i="11403" xml_f="11406" txt_i="3144" txt_f="3147">[6]</offsets></xref><offsets xml_i="11413" xml_f="11610" txt_i="3147" txt_f="3344">. Anaerobic Gram-negative bacteria producing hydrogen gas have been more frequently found in colicky infants, as well as lower numbers and specific colonization patterns of intestinal lactobacilli </offsets><xref rid="pone.0056710-Savino4" ref-type="bibr"><offsets xml_i="11659" xml_f="11662" txt_i="3344" txt_f="3347">[7]</offsets></xref><offsets xml_i="11669" xml_f="11703" txt_i="3347" txt_f="3381">. Coliform bacteria, particularly </offsets><italic><offsets xml_i="11711" xml_f="11727" txt_i="3381" txt_f="3397">Escherichia coli</offsets></italic><offsets xml_i="11736" xml_f="11951" txt_i="3397" txt_f="3612">, were found to be more abundant in the faeces of colicky infants, suggesting a role for colonic fermentation and consequent excessive intra-intestinal air production, aerophagia and pain, typical in these patients </offsets><xref rid="pone.0056710-Savino3" ref-type="bibr"><offsets xml_i="12000" xml_f="12003" txt_i="3612" txt_f="3615">[5]</offsets></xref><offsets xml_i="12010" xml_f="12012" txt_i="3615" txt_f="3617">, </offsets><xref rid="pone.0056710-Savino5" ref-type="bibr"><offsets xml_i="12061" xml_f="12064" txt_i="3617" txt_f="3620">[8]</offsets></xref><offsets xml_i="12071" xml_f="12163" txt_i="3620" txt_f="3712">. Further support for a role of the gut microbiota can be found in a study by Rhoads et al. </offsets><xref rid="pone.0056710-Rhoads1" ref-type="bibr"><offsets xml_i="12212" xml_f="12215" txt_i="3712" txt_f="3715">[6]</offsets></xref><offsets xml_i="12222" xml_f="12293" txt_i="3715" txt_f="3786">, in which elevated levels of fecal calprotectin and a higher level of </offsets><italic><offsets xml_i="12301" xml_f="12311" txt_i="3786" txt_f="3796">Klebsiella</offsets></italic><offsets xml_i="12320" xml_f="12386" txt_i="3796" txt_f="3862"> were demonstrated in crying infants, as well as in Pärtty et al. </offsets><xref rid="pone.0056710-Prtty1" ref-type="bibr"><offsets xml_i="12434" xml_f="12437" txt_i="3862" txt_f="3865">[9]</offsets></xref><offsets xml_i="12444" xml_f="12522" txt_i="3865" txt_f="3943"> where certain bifidobacteria were inversely associated with colicky symptoms.</offsets></p><p><offsets xml_i="12529" xml_f="12713" txt_i="3944" txt_f="4128">Probiotics are live micro-organisms which when administered in adequate amounts confer a health benefit on the host. Recent clinical work demonstrated that administration of probiotic </offsets><italic><offsets xml_i="12721" xml_f="12742" txt_i="4128" txt_f="4149">Lactobacillus reuteri</offsets></italic><offsets xml_i="12751" xml_f="12797" txt_i="4149" txt_f="4195"> improved colic symptoms in breastfed infants </offsets><xref rid="pone.0056710-Savino6" ref-type="bibr"><offsets xml_i="12846" xml_f="12850" txt_i="4195" txt_f="4199">[10]</offsets></xref><offsets xml_i="12857" xml_f="12858" txt_i="4199" txt_f="4200">–</offsets><xref rid="pone.0056710-Szajewska1" ref-type="bibr"><offsets xml_i="12910" xml_f="12914" txt_i="4200" txt_f="4204">[12]</offsets></xref><offsets xml_i="12921" xml_f="13151" txt_i="4204" txt_f="4434">. The probiotic may elicit its effect either through direct interaction with the host, or indirectly by affecting the host gastrointestinal microbiota and thereby the colicky symptoms. In fact, it was earlier shown that intake of </offsets><italic><offsets xml_i="13159" xml_f="13169" txt_i="4434" txt_f="4444">L. reuteri</offsets></italic><offsets xml_i="13178" xml_f="13218" txt_i="4444" txt_f="4484"> resulted in decreased faecal counts of </offsets><italic><offsets xml_i="13226" xml_f="13233" txt_i="4484" txt_f="4491">E. coli</offsets></italic><offsets xml_i="13242" xml_f="13275" txt_i="4491" txt_f="4524"> and an increase of lactobacilli </offsets><xref rid="pone.0056710-Savino6" ref-type="bibr"><offsets xml_i="13324" xml_f="13328" txt_i="4524" txt_f="4528">[10]</offsets></xref><offsets xml_i="13335" xml_f="13336" txt_i="4528" txt_f="4529">.</offsets></p><p><offsets xml_i="13343" xml_f="13719" txt_i="4530" txt_f="4906">Recent years have seen a revolution in methodology for the analysis of the composition of bacterial ecosystems. High throughput sequencing of 16 S rRNA genes PCR amplified from the bacteria present in the ecosystem allows mapping of the dominant microbiota, phylogenetic classification of the microbes and comparisons of microbial composition at different phylogenetic levels </offsets><xref rid="pone.0056710-Herlemann1" ref-type="bibr"><offsets xml_i="13771" xml_f="13775" txt_i="4906" txt_f="4910">[13]</offsets></xref><offsets xml_i="13782" xml_f="13783" txt_i="4910" txt_f="4911">–</offsets><xref rid="pone.0056710-Andersson1" ref-type="bibr"><offsets xml_i="13835" xml_f="13839" txt_i="4911" txt_f="4915">[16]</offsets></xref><offsets xml_i="13846" xml_f="14000" txt_i="4915" txt_f="5069">. The current study was performed to better understand the complex microbial communities of the colicky infant gut and the effect of supplementation with </offsets><italic><offsets xml_i="14008" xml_f="14018" txt_i="5069" txt_f="5079">L. reuteri</offsets></italic><offsets xml_i="14027" xml_f="14433" txt_i="5079" txt_f="5485">, known to improve colic symptoms, on the global microbiota. Crying patterns of infants were recorded and global faecal microbial composition was analysed at baseline and after 21 days of supplementation, using large-scale DNA sequencing of 16 S rRNA genes. The infants had a remarkably large inter-individual compositional variability and no correlation could be found with treatment. Thus, the probiotic </offsets><italic><offsets xml_i="14441" xml_f="14451" txt_i="5485" txt_f="5495">L. reuteri</offsets></italic><offsets xml_i="14460" xml_f="14571" txt_i="5495" txt_f="5606"> DSM 17938 was shown to not affect the global composition of the microbiota. However, there was an increase of </offsets><italic><offsets xml_i="14579" xml_f="14592" txt_i="5606" txt_f="5619">Bacteroidetes</offsets></italic><offsets xml_i="14601" xml_f="14766" txt_i="5619" txt_f="5784"> in the group responder infants, irrespective of treatment, indicating that a decrease in colicky symptoms was linked to changes of the global microbial composition.</offsets></p></sec><sec sec-type="materials|methods" id="s2"><title><offsets xml_i="14825" xml_f="14846" txt_i="5786" txt_f="5807">Materials and Methods</offsets></title><p><offsets xml_i="14857" xml_f="14992" txt_i="5808" txt_f="5943">The protocol for this trial and supporting CONSORT checklist are available as supporting information; see Checklist S1 and Protocol S1.</offsets></p><sec id="s2a"><title><offsets xml_i="15017" xml_f="15037" txt_i="5944" txt_f="5964">Subjects and Methods</offsets></title><p><offsets xml_i="15048" xml_f="15075" txt_i="5965" txt_f="5992">We have earlier shown that </offsets><italic><offsets xml_i="15083" xml_f="15104" txt_i="5992" txt_f="6013">Lactobacillus reuteri</offsets></italic><offsets xml_i="15113" xml_f="15149" txt_i="6013" txt_f="6049"> DSM 17938 relieves infantile colic </offsets><xref rid="pone.0056710-Savino6" ref-type="bibr"><offsets xml_i="15198" xml_f="15202" txt_i="6049" txt_f="6053">[10]</offsets></xref><offsets xml_i="15209" xml_f="15211" txt_i="6053" txt_f="6055">, </offsets><xref rid="pone.0056710-Savino7" ref-type="bibr"><offsets xml_i="15260" xml_f="15264" txt_i="6055" txt_f="6059">[11]</offsets></xref><offsets xml_i="15271" xml_f="15481" txt_i="6059" txt_f="6269">. The aim of this study was to evaluate the effect of administration of this bacterium on gut microbiota of the colicky infants and the analyses were done on samples collected during the study by Savino et al. </offsets><xref rid="pone.0056710-Savino6" ref-type="bibr"><offsets xml_i="15530" xml_f="15534" txt_i="6269" txt_f="6273">[10]</offsets></xref><offsets xml_i="15541" xml_f="16571" txt_i="6273" txt_f="7303">. This randomized, controlled, double-blind study has been performed on 50 breastfed infants consecutively recruited from general pediatricians and from outpatients at the Department of Pediatrics, University of Turin (Regina Margherita Children's Hospital) between March 2008 and August 2009 and in accordance with good clinical practice and the Declaration of Helsinki. All infants had a diagnosis of infantile colic according to Wessel’s criteria: episodes of crying lasting more than 3 h a day, for 3 or more days in the 1 week prior to enrolment. They were all born at term, adequate for gestational age, with a birth weight between 2500 and 4000 g, and were aged 10–60 days at recruitment. Only exclusively breastfed infants were enrolled in order to prevent variability in the intestinal microbiota caused by diet. Exclusion criteria were clinical evidence of chronic illness or gastrointestinal disorders, any intake of probiotics and/or antibiotics in the week preceding recruitment and mixed breast- and formula-feeding.</offsets></p><p><offsets xml_i="16578" xml_f="16795" txt_i="7304" txt_f="7521">The sample size was calculated post-hoc to find a clinically relevant difference in the reduction in daily average crying time of 50% between day 1 and day 21. With α = .05, β = .40, 13 patients were needed per group.</offsets></p><p><offsets xml_i="16802" xml_f="17061" txt_i="7522" txt_f="7781">For the specific analysis of the faecal microbiota, we evaluated 29 infants. The protocol was approved by the Ethics Committee of the Azienda Ospedaliera, OIRM S. Anna – Ospedale Mauriziano (Turin, Italy). Characteristics of enrolled subjects are reported in </offsets><xref ref-type="table" rid="pone-0056710-t001"><offsets xml_i="17108" xml_f="17115" txt_i="7781" txt_f="7788">Table 1</offsets></xref><offsets xml_i="17122" xml_f="17267" txt_i="7788" txt_f="7933">, and no differences were found between the two groups of subjects, besides the response to the treatment. Thus, the previous reported effect of </offsets><italic><offsets xml_i="17275" xml_f="17285" txt_i="7933" txt_f="7943">L. reuteri</offsets></italic><offsets xml_i="17294" xml_f="17305" txt_i="7943" txt_f="7954"> DSM 17938 </offsets><xref rid="pone.0056710-Savino6" ref-type="bibr"><offsets xml_i="17354" xml_f="17358" txt_i="7954" txt_f="7958">[10]</offsets></xref><offsets xml_i="17365" xml_f="17759" txt_i="7958" txt_f="8352"> was evident also in this subgroup of subjects. A written informed consent was obtained from both parents before the inclusion of infants. At enrolment, each infant underwent a medical examination and parents completed a questionnaire to obtain data about subjects, including also type of delivery (vaginally or caesarean). Infants were recruited and randomly assigned to receive the probiotic </offsets><italic><offsets xml_i="17767" xml_f="17777" txt_i="8352" txt_f="8362">L. reuteri</offsets></italic><offsets xml_i="17786" xml_f="18242" txt_i="8362" txt_f="8818"> DSM 17938 or placebo daily for 21 days. Patients in each of the two groups were defined as “responders” or “non responders”, a responder displaying a reduction of 50% of crying time on day 21. Faecal samples (10–15 g) were collected from each subject, directly from the diaper or anus, at enrolment and on day 21. Samples (blinded) were immediately placed at −20°C and then later stored at −70°C until analysis of the gut microbiota by 454 pyrosequencing.</offsets></p><table-wrap id="pone-0056710-t001" orientation="portrait" position="float"><object-id pub-id-type="doi"><offsets xml_i="18350" xml_f="18383" txt_i="8819" txt_f="8852">10.1371/journal.pone.0056710.t001</offsets></object-id><label><offsets xml_i="18402" xml_f="18409" txt_i="8852" txt_f="8859">Table 1</offsets></label><caption><title><offsets xml_i="18433" xml_f="18461" txt_i="8859" txt_f="8887">Characteristics of subjects.</offsets></title></caption><alternatives><graphic id="pone-0056710-t001-1" xlink:href="pone.0056710.t001"></graphic><table frame="hsides" rules="groups"><colgroup span="1"><col align="left" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col></colgroup><thead><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="18825" xml_f="18833" txt_i="8888" txt_f="8896">Variable</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="18879" xml_f="18887" txt_i="8897" txt_f="8905">Placebo </offsets><italic><offsets xml_i="18895" xml_f="18903" txt_i="8905" txt_f="8913">(n = 14)</offsets></italic><offsets xml_i="18912" xml_f="18913" txt_i="8913" txt_f="8914">
</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="18959" xml_f="18960" txt_i="8915" txt_f="8916">
</offsets><italic><offsets xml_i="18968" xml_f="18987" txt_i="8916" txt_f="8935">L. reuteri (n = 15)</offsets></italic><offsets xml_i="18996" xml_f="18997" txt_i="8935" txt_f="8936">
</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19043" xml_f="19044" txt_i="8937" txt_f="8938">
</offsets><italic><offsets xml_i="19052" xml_f="19053" txt_i="8938" txt_f="8939">P</offsets></italic><offsets xml_i="19062" xml_f="19068" txt_i="8939" txt_f="8945"> value</offsets></td></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="19138" xml_f="19173" txt_i="8946" txt_f="8981">Type of delivery (caesarean), n (%)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19219" xml_f="19227" txt_i="8982" txt_f="8990">3 (21.4)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19273" xml_f="19279" txt_i="8991" txt_f="8997">6 (40)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19325" xml_f="19330" txt_i="8998" txt_f="9003">0.427</offsets><xref ref-type="table-fn" rid="nt101"><offsets xml_i="19368" xml_f="19369" txt_i="9003" txt_f="9004">#</offsets></xref><offsets xml_i="19376" xml_f="19377" txt_i="9004" txt_f="9005">
</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="19432" xml_f="19443" txt_i="9006" txt_f="9017">Male, n (%)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19489" xml_f="19495" txt_i="9018" txt_f="9024">7 (50)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19541" xml_f="19547" txt_i="9025" txt_f="9031">9 (60)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19593" xml_f="19598" txt_i="9032" txt_f="9037">0.715</offsets><xref ref-type="table-fn" rid="nt101"><offsets xml_i="19636" xml_f="19637" txt_i="9037" txt_f="9038">#</offsets></xref><offsets xml_i="19644" xml_f="19645" txt_i="9038" txt_f="9039">
</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="19700" xml_f="19732" txt_i="9040" txt_f="9072">Age at entry (days), median (SD)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19778" xml_f="19789" txt_i="9073" txt_f="9084">29.6 (12.9)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19835" xml_f="19846" txt_i="9085" txt_f="9096">29.8 (11.7)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="19892" xml_f="19897" txt_i="9097" txt_f="9102">0.960</offsets><sup><offsets xml_i="19902" xml_f="19903" txt_i="9102" txt_f="9103">§</offsets></sup><offsets xml_i="19909" xml_f="19910" txt_i="9103" txt_f="9104">
</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="19965" xml_f="19982" txt_i="9105" txt_f="9122">Responders, n (%)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="20028" xml_f="20034" txt_i="9123" txt_f="9129">7 (50)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="20080" xml_f="20087" txt_i="9130" txt_f="9137">14 (93)</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="20133" xml_f="20138" txt_i="9138" txt_f="9143">0.013</offsets><sup><offsets xml_i="20143" xml_f="20144" txt_i="9143" txt_f="9144">#</offsets></sup><offsets xml_i="20150" xml_f="20151" txt_i="9144" txt_f="9145">
</offsets><xref ref-type="table-fn" rid="nt102"><offsets xml_i="20189" xml_f="20190" txt_i="9145" txt_f="9146">*</offsets></xref><offsets xml_i="20197" xml_f="20198" txt_i="9146" txt_f="9147">
</offsets></td></tr></tbody></table></alternatives><table-wrap-foot><fn id="nt101"><label><offsets xml_i="20278" xml_f="20279" txt_i="9148" txt_f="9149">#</offsets></label><p><offsets xml_i="20290" xml_f="20336" txt_i="9149" txt_f="9195">Fisher’s Exact Test; § Wilcoxon Rank Sum Test;</offsets></p></fn><fn id="nt102"><label><offsets xml_i="20367" xml_f="20368" txt_i="9196" txt_f="9197">*</offsets></label><p><offsets xml_i="20379" xml_f="20402" txt_i="9197" txt_f="9220">Significant difference.</offsets></p></fn></table-wrap-foot></table-wrap></sec><sec id="s2b"><title><offsets xml_i="20469" xml_f="20512" txt_i="9222" txt_f="9265">DNA Extraction, PCRs and 454 Pyrosequencing</offsets></title><p><offsets xml_i="20523" xml_f="20761" txt_i="9266" txt_f="9504">DNA was extracted from 250 mg of faeces using the MoBio Power Soil DNA Kit (Solana Beach, CA, USA) according to the manufacturer’s instruction with the modification that the bead-beating step was performed 2×45 s, at level 5 on a FastPrep</offsets><sup><offsets xml_i="20766" xml_f="20767" txt_i="9504" txt_f="9505">®</offsets></sup><offsets xml_i="20773" xml_f="20817" txt_i="9505" txt_f="9549">−24 device (MP Biomedicals, Solon, OH, USA).</offsets></p><p><offsets xml_i="20824" xml_f="21045" txt_i="9550" txt_f="9771">The 16S rRNA genes were PCR amplified using broad range bacterial primers. The primers: Bakt_341F (CCTACGGGNGGCWGGAG) and Bakt_805R (GACTACHVGGGTATCTAATCC) were complemented with 454 adapters and sample specific barcodes </offsets><xref rid="pone.0056710-Herlemann1" ref-type="bibr"><offsets xml_i="21097" xml_f="21101" txt_i="9771" txt_f="9775">[13]</offsets></xref><offsets xml_i="21108" xml_f="21945" txt_i="9775" txt_f="10612">. DNA was amplified with PCR (DreamTaq™ Mastermix, Thermo Fisher Scientific, Fermentas GmbH, St. Leon-Rot, Germany) under the following running conditions: initial denaturation at 95°C for 5 min, 30 cycles of 40 s at 95°C, 40s at 58°C, 1 min at 72°C, and a final elongation step for 7 min at 72°C. PCR products were confirmed using agarose gel electrophoresis and these were subsequently isolated from the gel and purified using GeneJET™ gel extraction kit (Thermo Fisher Scientific). Purified products were quantified using a NanoVue spectrophotometer (GE HealthCare, Uppsala, Sweden) and mixed into equimolar amounts. The mixed pool of PCR products was sequenced from the reverse primer direction at the Swedish Institute for Infectious Disease Control in Solna, using the Roche/454 GS Titanium technology platform (Branford, CT, USA).</offsets></p></sec><sec id="s2c"><title><offsets xml_i="21976" xml_f="21994" txt_i="10614" txt_f="10632">Taxonomic Analysis</offsets></title><p><offsets xml_i="22005" xml_f="22083" txt_i="10633" txt_f="10711">The sequence processing was mainly performed as described in Herlemann et al. </offsets><xref rid="pone.0056710-Herlemann1" ref-type="bibr"><offsets xml_i="22135" xml_f="22139" txt_i="10711" txt_f="10715">[13]</offsets></xref><offsets xml_i="22146" xml_f="22469" txt_i="10715" txt_f="11038">. In brief, sequences were checked for quality and those that were less than 300 bp in length excluding the primer sequence, contained incorrect primer sequences, or contained any ambiguous base were discarded. Remaining sequences were then subjected to complete linkage clustering using the pyrosequencing pipeline at RDP </offsets><xref rid="pone.0056710-Cole1" ref-type="bibr"><offsets xml_i="22516" xml_f="22520" txt_i="11038" txt_f="11042">[17]</offsets></xref><offsets xml_i="22527" xml_f="23092" txt_i="11042" txt_f="11607"> with a conservative 5% dissimilarity to define OTUs. The most abundant sequence from each OTU was selected as a representative sequence and was taxonomically classified by BLAST searching against a local BLAST database comprised of 600, 316 bacterial 16 S rRNA gene sequences longer than 1,200 bp with good Pintail scores from RDP v. 10.7. The OTU inherited the taxonomy (down to genus level) of the best scoring RDP hit fulfilling the criteria of ≥95% identity over an alignment of length ≥280 bp. Assignment of sequences to samples was based on the 5-bp barcode.</offsets></p></sec><sec id="s2d"><title><offsets xml_i="23123" xml_f="23143" txt_i="11609" txt_f="11629">Statistical Analyses</offsets></title><p><offsets xml_i="23154" xml_f="23489" txt_i="11630" txt_f="11965">Comparisons of characteristics of subjects were made using Fisher’s exact test and Wilcoxon rank sum test (SAS V9.2). Changes in microbial community composition during the intervention, for treatment type, response and type of birth, and interactions of these factors were tested using the “adonis” function in the vegan package for R </offsets><xref rid="pone.0056710-Oksanen1" ref-type="bibr"><offsets xml_i="23539" xml_f="23543" txt_i="11965" txt_f="11969">[18]</offsets></xref><offsets xml_i="23550" xml_f="23917" txt_i="11969" txt_f="12336">. Adonis is a permutational multivariate analysis of variance using a distance matrix as response variable. We used Euclidean distance and 999 permutations. The change of composition was also tested using principal component analysis (PCA) on a matrix on the difference in abundance of the different phyla or genus after and before treatment, using the vegan package </offsets><xref rid="pone.0056710-Oksanen1" ref-type="bibr"><offsets xml_i="23967" xml_f="23971" txt_i="12336" txt_f="12340">[18]</offsets></xref><offsets xml_i="23978" xml_f="24119" txt_i="12340" txt_f="12481">. Confidence ellipses around the centroids of the result categories were used to assess the differentiation of responders and non-responders </offsets><xref rid="pone.0056710-Oksanen1" ref-type="bibr"><offsets xml_i="24169" xml_f="24173" txt_i="12481" txt_f="12485">[18]</offsets></xref><offsets xml_i="24180" xml_f="24434" txt_i="12485" txt_f="12739">. In all multivariate analyses, data was transformed by taking the arcsine of the square root of the proportional abundance. Comparison of specific changes in relative abundance during the intervention period was done with a Mann Whitney test using Past </offsets><xref rid="pone.0056710-Hammer1" ref-type="bibr"><offsets xml_i="24483" xml_f="24487" txt_i="12739" txt_f="12743">[19]</offsets></xref><offsets xml_i="24494" xml_f="24496" txt_i="12743" txt_f="12745"> (</offsets><ext-link ext-link-type="uri" xlink:href="http://folk.uio.no/ohammer/past/"><offsets xml_i="24572" xml_f="24604" txt_i="12745" txt_f="12777">http://folk.uio.no/ohammer/past/</offsets></ext-link><offsets xml_i="24615" xml_f="24833" txt_i="12777" txt_f="12995">). We also explored the structure of the data from the two time points with PCA using Canoco 4.5 and the validity of clustering patterns was confirmed with a distance based MANOVA (Bray Curtis distance; Past software).</offsets></p></sec></sec><sec id="s3"><title><offsets xml_i="24869" xml_f="24891" txt_i="12998" txt_f="13020">Results and Discussion</offsets></title><p><offsets xml_i="24902" xml_f="25176" txt_i="13021" txt_f="13295">The 454 pyrosequencing generated in total 412 000 sequences and the number of sequences per sample were between 1593–21322 (average 6538, median 5997). Phylogenetic classification revealed that the prevalence of the four major phyla varied considerably between the samples (</offsets><xref ref-type="table" rid="pone-0056710-t002"><offsets xml_i="25223" xml_f="25230" txt_i="13295" txt_f="13302">Table 2</offsets></xref><offsets xml_i="25237" xml_f="25384" txt_i="13302" txt_f="13449">). All four were found at abundances varying from less than 2% to more than 50%, indicating that none of the phyla was discriminated by the method.</offsets></p><table-wrap id="pone-0056710-t002" orientation="portrait" position="float"><object-id pub-id-type="doi"><offsets xml_i="25492" xml_f="25525" txt_i="13450" txt_f="13483">10.1371/journal.pone.0056710.t002</offsets></object-id><label><offsets xml_i="25544" xml_f="25551" txt_i="13483" txt_f="13490">Table 2</offsets></label><caption><title><offsets xml_i="25575" xml_f="25627" txt_i="13490" txt_f="13542">Relative abundance of the main four bacterial phyla.</offsets></title></caption><alternatives><graphic id="pone-0056710-t002-2" xlink:href="pone.0056710.t002"></graphic><table frame="hsides" rules="groups"><colgroup span="1"><col align="left" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col></colgroup><thead><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="26026" xml_f="26031" txt_i="13543" txt_f="13548">Phyla</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26077" xml_f="26081" txt_i="13549" txt_f="13553">Mean</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26127" xml_f="26133" txt_i="13554" txt_f="13560">Median</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26179" xml_f="26182" txt_i="13561" txt_f="13564">Max</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26228" xml_f="26231" txt_i="13565" txt_f="13568">Min</offsets></td></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="26301" xml_f="26302" txt_i="13569" txt_f="13570">
</offsets><italic><offsets xml_i="26310" xml_f="26324" txt_i="13570" txt_f="13584">Proteobacteria</offsets></italic><offsets xml_i="26333" xml_f="26334" txt_i="13584" txt_f="13585">
</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26380" xml_f="26385" txt_i="13586" txt_f="13591">15.4%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26431" xml_f="26436" txt_i="13592" txt_f="13597">12.1%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26482" xml_f="26487" txt_i="13598" txt_f="13603">52.0%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26533" xml_f="26538" txt_i="13604" txt_f="13609">0.04%</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="26593" xml_f="26594" txt_i="13610" txt_f="13611">
</offsets><italic><offsets xml_i="26602" xml_f="26616" txt_i="13611" txt_f="13625">Actinobacteria</offsets></italic><offsets xml_i="26625" xml_f="26626" txt_i="13625" txt_f="13626">
</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26672" xml_f="26677" txt_i="13627" txt_f="13632">30.2%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26723" xml_f="26728" txt_i="13633" txt_f="13638">30.3%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26774" xml_f="26779" txt_i="13639" txt_f="13644">91.4%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26825" xml_f="26829" txt_i="13645" txt_f="13649">0.1%</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="26884" xml_f="26885" txt_i="13650" txt_f="13651">
</offsets><italic><offsets xml_i="26893" xml_f="26903" txt_i="13651" txt_f="13661">Firmicutes</offsets></italic><offsets xml_i="26912" xml_f="26913" txt_i="13661" txt_f="13662">
</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="26959" xml_f="26964" txt_i="13663" txt_f="13668">37.7%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27010" xml_f="27015" txt_i="13669" txt_f="13674">31.7%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27061" xml_f="27066" txt_i="13675" txt_f="13680">97.4%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27112" xml_f="27116" txt_i="13681" txt_f="13685">1.9%</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="27171" xml_f="27172" txt_i="13686" txt_f="13687">
</offsets><italic><offsets xml_i="27180" xml_f="27193" txt_i="13687" txt_f="13700">Bacteroidetes</offsets></italic><offsets xml_i="27202" xml_f="27203" txt_i="13700" txt_f="13701">
</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27249" xml_f="27254" txt_i="13702" txt_f="13707">16.3%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27300" xml_f="27304" txt_i="13708" txt_f="13712">0.3%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27350" xml_f="27355" txt_i="13713" txt_f="13718">78.6%</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="27401" xml_f="27413" txt_i="13719" txt_f="13731">Not detected</offsets></td></tr></tbody></table></alternatives></table-wrap><p><offsets xml_i="27470" xml_f="27518" txt_i="13732" txt_f="13780">The infants’ faecal microbiota were composed of </offsets><italic><offsets xml_i="27526" xml_f="27540" txt_i="13780" txt_f="13794">Proteobacteria</offsets></italic><offsets xml_i="27549" xml_f="27551" txt_i="13794" txt_f="13796">, </offsets><italic><offsets xml_i="27559" xml_f="27569" txt_i="13796" txt_f="13806">Firmicutes</offsets></italic><offsets xml_i="27578" xml_f="27580" txt_i="13806" txt_f="13808">, </offsets><italic><offsets xml_i="27588" xml_f="27602" txt_i="13808" txt_f="13822">Actinobacteria</offsets></italic><offsets xml_i="27611" xml_f="27616" txt_i="13822" txt_f="13827"> and </offsets><italic><offsets xml_i="27624" xml_f="27637" txt_i="13827" txt_f="13840">Bacteroidetes</offsets></italic><offsets xml_i="27646" xml_f="27671" txt_i="13840" txt_f="13865"> as the four main phyla (</offsets><xref ref-type="supplementary-material" rid="pone.0056710.s001"><offsets xml_i="27735" xml_f="27744" txt_i="13865" txt_f="13874">Figure S1</offsets></xref><offsets xml_i="27751" xml_f="27845" txt_i="13874" txt_f="13968">). The microbiota compositions were shown to have very high inter-individual variability with </offsets><italic><offsets xml_i="27853" xml_f="27863" txt_i="13968" txt_f="13978">Firmicutes</offsets></italic><offsets xml_i="27872" xml_f="27873" txt_i="13978" txt_f="13979">/</offsets><italic><offsets xml_i="27881" xml_f="27894" txt_i="13979" txt_f="13992">Bacteroidetes</offsets></italic><offsets xml_i="27903" xml_f="28024" txt_i="13992" txt_f="14113"> ratios varying from 0.025 to 4000 in line with previously published studies on gut microbiota establishment in neonates </offsets><xref rid="pone.0056710-Palmer1" ref-type="bibr"><offsets xml_i="28073" xml_f="28077" txt_i="14113" txt_f="14117">[20]</offsets></xref><offsets xml_i="28084" xml_f="28086" txt_i="14117" txt_f="14119">, </offsets><xref rid="pone.0056710-Adlerberth1" ref-type="bibr"><offsets xml_i="28139" xml_f="28143" txt_i="14119" txt_f="14123">[21]</offsets></xref><offsets xml_i="28150" xml_f="28151" txt_i="14123" txt_f="14124">
</offsets><monospace><offsets xml_i="28162" xml_f="28164" txt_i="14124" txt_f="14126">. </offsets></monospace><offsets xml_i="28176" xml_f="28279" txt_i="14126" txt_f="14229">On an individual basis, the microbiota was, however, for most individuals relatively stable over time (</offsets><xref ref-type="supplementary-material" rid="pone.0056710.s001"><offsets xml_i="28343" xml_f="28352" txt_i="14229" txt_f="14238">Figure S1</offsets></xref><offsets xml_i="28359" xml_f="28558" txt_i="14238" txt_f="14437">). Although the distribution between the phyla differs slightly from the present investigation, other studies have shown that composition of the microbial communities varies widely from baby to baby </offsets><xref rid="pone.0056710-Palmer1" ref-type="bibr"><offsets xml_i="28607" xml_f="28611" txt_i="14437" txt_f="14441">[20]</offsets></xref><offsets xml_i="28618" xml_f="28620" txt_i="14441" txt_f="14443">, </offsets><xref rid="pone.0056710-Abrahamsson1" ref-type="bibr"><offsets xml_i="28674" xml_f="28678" txt_i="14443" txt_f="14447">[22]</offsets></xref><offsets xml_i="28685" xml_f="28777" txt_i="14447" txt_f="14539">, but also that distinct structures of each individuals microbial community were noticeable </offsets><xref rid="pone.0056710-Palmer1" ref-type="bibr"><offsets xml_i="28826" xml_f="28830" txt_i="14539" txt_f="14543">[20]</offsets></xref><offsets xml_i="28837" xml_f="28838" txt_i="14543" txt_f="14544">.</offsets></p><p><offsets xml_i="28845" xml_f="28860" txt_i="14545" txt_f="14560">Treatment with </offsets><italic><offsets xml_i="28868" xml_f="28878" txt_i="14560" txt_f="14570">L. reuteri</offsets></italic><offsets xml_i="28887" xml_f="28955" txt_i="14570" txt_f="14638"> DSM 17938 did not affect the global composition of the microbiota (</offsets><xref ref-type="table" rid="pone-0056710-t003"><offsets xml_i="29002" xml_f="29009" txt_i="14638" txt_f="14645">Table 3</offsets></xref><offsets xml_i="29016" xml_f="29107" txt_i="14645" txt_f="14733">). However, when comparing responders (&gt;50% reduction in crying time; n = 21; 14 in the </offsets><italic><offsets xml_i="29115" xml_f="29125" txt_i="14733" txt_f="14743">L. reuteri</offsets></italic><offsets xml_i="29134" xml_f="29196" txt_i="14743" txt_f="14805"> group and 7 in placebo) with non-responders (n = 8; 1 in the </offsets><italic><offsets xml_i="29204" xml_f="29214" txt_i="14805" txt_f="14815">L. reuteri</offsets></italic><offsets xml_i="29223" xml_f="29333" txt_i="14815" txt_f="14925"> group and 7 in placebo) the change of composition at the phyla and genus levels differed between the groups (</offsets><xref ref-type="table" rid="pone-0056710-t003"><offsets xml_i="29380" xml_f="29387" txt_i="14925" txt_f="14932">Table 3</offsets></xref><offsets xml_i="29394" xml_f="29485" txt_i="14932" txt_f="15023">). Ordination analyses revealed that this difference depended on an increase of the phylum </offsets><italic><offsets xml_i="29493" xml_f="29506" txt_i="15023" txt_f="15036">Bacteroidetes</offsets></italic><offsets xml_i="29515" xml_f="29517" txt_i="15036" txt_f="15038"> (</offsets><xref ref-type="supplementary-material" rid="pone.0056710.s002"><offsets xml_i="29581" xml_f="29590" txt_i="15038" txt_f="15047">Figure S2</offsets></xref><offsets xml_i="29597" xml_f="29609" txt_i="15047" txt_f="15059">) and genus </offsets><italic><offsets xml_i="29617" xml_f="29628" txt_i="15059" txt_f="15070">Bacteroides</offsets></italic><offsets xml_i="29637" xml_f="29849" txt_i="15070" txt_f="15279"> (data not shown) during the intervention in the group of responders. Although some individuals only had a small change and the variation was large the difference was significant (p&lt;0.01) and the abundance of </offsets><italic><offsets xml_i="29857" xml_f="29870" txt_i="15279" txt_f="15292">Bacteroidetes</offsets></italic><offsets xml_i="29879" xml_f="29965" txt_i="15292" txt_f="15378"> increased on average with 8% in responders but decreased with 18% in non-responders (</offsets><xref ref-type="supplementary-material" rid="pone.0056710.s003"><offsets xml_i="30029" xml_f="30038" txt_i="15378" txt_f="15387">Figure S3</offsets></xref><offsets xml_i="30045" xml_f="30096" txt_i="15387" txt_f="15438">). This is the first time that a high abundance of </offsets><italic><offsets xml_i="30104" xml_f="30117" txt_i="15438" txt_f="15451">Bacteroidetes</offsets></italic><offsets xml_i="30126" xml_f="30233" txt_i="15451" txt_f="15558"> has been linked to a decrease in colicky symptoms, although increased diversity and relative abundance of </offsets><italic><offsets xml_i="30241" xml_f="30254" txt_i="15558" txt_f="15571">Bacteroidetes</offsets></italic><offsets xml_i="30263" xml_f="30268" txt_i="15571" txt_f="15576"> and </offsets><italic><offsets xml_i="30276" xml_f="30287" txt_i="15576" txt_f="15587">Bacteroides</offsets></italic><offsets xml_i="30296" xml_f="30364" txt_i="15587" txt_f="15655"> have been shown to correlate with absence of IgE-associated eczema </offsets><xref rid="pone.0056710-Abrahamsson1" ref-type="bibr"><offsets xml_i="30418" xml_f="30422" txt_i="15655" txt_f="15659">[22]</offsets></xref><offsets xml_i="30429" xml_f="30503" txt_i="15659" txt_f="15733"> and this is in line with a reported correlation between eczema and colic </offsets><xref rid="pone.0056710-Savino8" ref-type="bibr"><offsets xml_i="30552" xml_f="30556" txt_i="15733" txt_f="15737">[23]</offsets></xref><offsets xml_i="30563" xml_f="30738" txt_i="15737" txt_f="15912">. Those groups of bacteria are also indicators of a gastrointestinal microbial ecosystem that has maturated and participates in the breakdown of complex plant polysaccharides </offsets><xref rid="pone.0056710-Koenig1" ref-type="bibr"><offsets xml_i="30787" xml_f="30791" txt_i="15912" txt_f="15916">[24]</offsets></xref><offsets xml_i="30798" xml_f="30942" txt_i="15916" txt_f="16060">, which could improve a poor digestion. Furthermore, in a study on patients with severe systemic inflammatory response syndrome, Shimizu et al. </offsets><xref rid="pone.0056710-Shimizu1" ref-type="bibr"><offsets xml_i="30992" xml_f="30996" txt_i="16060" txt_f="16064">[25]</offsets></xref><offsets xml_i="31003" xml_f="31227" txt_i="16064" txt_f="16288"> reported that gastrointestinal dysmotility was associated with altered gut microbiota, and faecal analysis showed that patients with feeding intolerance had significantly lower numbers of total obligate anaerobes including </offsets><italic><offsets xml_i="31235" xml_f="31249" txt_i="16288" txt_f="16302">Bacteroidaceae</offsets></italic><offsets xml_i="31258" xml_f="31310" txt_i="16302" txt_f="16354"> than those in patients without feeding intolerance.</offsets></p><table-wrap id="pone-0056710-t003" orientation="portrait" position="float"><object-id pub-id-type="doi"><offsets xml_i="31418" xml_f="31451" txt_i="16355" txt_f="16388">10.1371/journal.pone.0056710.t003</offsets></object-id><label><offsets xml_i="31470" xml_f="31477" txt_i="16388" txt_f="16395">Table 3</offsets></label><caption><title><offsets xml_i="31501" xml_f="31729" txt_i="16395" txt_f="16623">Pseudo F-ratios from tests using “adonis” of effects in microbial community composition between treatment group, response of treatment, and type of delivery, on a matrix on the difference in abundance before and after treatment.</offsets></title></caption><alternatives><graphic id="pone-0056710-t003-3" xlink:href="pone.0056710.t003"></graphic><table frame="hsides" rules="groups"><colgroup span="1"><col align="left" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col><col align="center" span="1"></col></colgroup><thead><tr><td align="left" rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32139" xml_f="32144" txt_i="16625" txt_f="16630">Phyla</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32190" xml_f="32194" txt_i="16631" txt_f="16635">Taxa</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32240" xml_f="32243" txt_i="16636" txt_f="16639">OTU</offsets></td></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="32313" xml_f="32322" txt_i="16640" txt_f="16649">Treatment</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32368" xml_f="32372" txt_i="16650" txt_f="16654">1.02</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32418" xml_f="32422" txt_i="16655" txt_f="16659">0.81</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32468" xml_f="32472" txt_i="16660" txt_f="16664">0.99</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="32527" xml_f="32535" txt_i="16665" txt_f="16673">Response</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32581" xml_f="32586" txt_i="16674" txt_f="16679">4.09*</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32632" xml_f="32637" txt_i="16680" txt_f="16685">2.49*</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32683" xml_f="32687" txt_i="16686" txt_f="16690">1.30</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="32742" xml_f="32750" txt_i="16691" txt_f="16699">Delivery</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32796" xml_f="32800" txt_i="16700" txt_f="16704">0.89</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32846" xml_f="32850" txt_i="16705" txt_f="16709">0.82</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="32896" xml_f="32900" txt_i="16710" txt_f="16714">0.99</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="32955" xml_f="32975" txt_i="16715" txt_f="16735">Treatment×Responders</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33021" xml_f="33025" txt_i="16736" txt_f="16740">2.51</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33071" xml_f="33075" txt_i="16741" txt_f="16745">0.92</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33121" xml_f="33125" txt_i="16746" txt_f="16750">1.18</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="33180" xml_f="33198" txt_i="16751" txt_f="16769">Treatment×Delivery</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33244" xml_f="33248" txt_i="16770" txt_f="16774">0.56</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33294" xml_f="33298" txt_i="16775" txt_f="16779">0.44</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33344" xml_f="33348" txt_i="16780" txt_f="16784">0.51</offsets></td></tr><tr><td align="left" rowspan="1" colspan="1"><offsets xml_i="33403" xml_f="33422" txt_i="16785" txt_f="16804">Responders×Delivery</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33468" xml_f="33472" txt_i="16805" txt_f="16809">3.17</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33518" xml_f="33523" txt_i="16810" txt_f="16815">3.66.</offsets></td><td align="left" rowspan="1" colspan="1"><offsets xml_i="33569" xml_f="33573" txt_i="16816" txt_f="16820">2.22</offsets></td></tr></tbody></table></alternatives><table-wrap-foot><fn id="nt103"><p><offsets xml_i="33649" xml_f="33739" txt_i="16821" txt_f="16911">The analyses were performed on three different taxonomic resolutions: Phyla, Taxa and OTU.</offsets></p></fn><fn id="nt104"><p><offsets xml_i="33766" xml_f="33788" txt_i="16912" txt_f="16928">.p&lt;0.1, *p&lt;0.05.</offsets></p></fn></table-wrap-foot></table-wrap><p><offsets xml_i="33831" xml_f="33993" txt_i="16929" txt_f="17091">A principal component analysis on the microbial composition on day 21 revealed a pattern where three types of microbial compositions (enterotypes) could be seen (</offsets><xref ref-type="supplementary-material" rid="pone.0056710.s004"><offsets xml_i="34057" xml_f="34066" txt_i="17091" txt_f="17100">Figure S4</offsets></xref><offsets xml_i="34073" xml_f="34332" txt_i="17100" txt_f="17356">). This clustering pattern was also evident when samples from day 1 were included in the ordination analysis (data not shown). The validity of the clustering pattern was confirmed with a distance based MANOVA (Bray Curtis distance; p&lt;0.00001) In one type (</offsets><bold><offsets xml_i="34338" xml_f="34339" txt_i="17356" txt_f="17357">A</offsets></bold><offsets xml_i="34346" xml_f="34383" txt_i="17357" txt_f="17394">; n = 12), the relative abundance of </offsets><italic><offsets xml_i="34391" xml_f="34405" txt_i="17394" txt_f="17408">Actinobacteria</offsets></italic><offsets xml_i="34414" xml_f="34432" txt_i="17408" txt_f="17426"> was over 40% and </offsets><italic><offsets xml_i="34440" xml_f="34453" txt_i="17426" txt_f="17439">Bacteroidetes</offsets></italic><offsets xml_i="34462" xml_f="34504" txt_i="17439" txt_f="17481"> below 1% (11 of 12); in the second type (</offsets><bold><offsets xml_i="34510" xml_f="34511" txt_i="17481" txt_f="17482">B</offsets></bold><offsets xml_i="34518" xml_f="34554" txt_i="17482" txt_f="17518">; n = 9), the relative abundance of </offsets><italic><offsets xml_i="34562" xml_f="34575" txt_i="17518" txt_f="17531">Bacteroidetes</offsets></italic><offsets xml_i="34584" xml_f="34623" txt_i="17531" txt_f="17570"> was above 25%; and in the third type (</offsets><bold><offsets xml_i="34629" xml_f="34630" txt_i="17570" txt_f="17571">F</offsets></bold><offsets xml_i="34637" xml_f="34673" txt_i="17571" txt_f="17607">; n = 8), the relative abundance of </offsets><italic><offsets xml_i="34681" xml_f="34691" txt_i="17607" txt_f="17617">Firmicutes</offsets></italic><offsets xml_i="34700" xml_f="34716" txt_i="17617" txt_f="17633"> was above 45%, </offsets><italic><offsets xml_i="34724" xml_f="34738" txt_i="17633" txt_f="17647">Actinobacteria</offsets></italic><offsets xml_i="34747" xml_f="34761" txt_i="17647" txt_f="17661"> below 2% and </offsets><italic><offsets xml_i="34769" xml_f="34782" txt_i="17661" txt_f="17674">Bacteroidetes</offsets></italic><offsets xml_i="34791" xml_f="34872" txt_i="17674" txt_f="17755"> below 1% (7 of 8). Notably, 8 of the 9 type B samples, with a high abundance of </offsets><italic><offsets xml_i="34880" xml_f="34893" txt_i="17755" txt_f="17768">Bacteroidetes</offsets></italic><offsets xml_i="34902" xml_f="34996" txt_i="17768" txt_f="17862">, came from the group responders, and 7 of those came from infants with spontaneous delivery (</offsets><xref ref-type="supplementary-material" rid="pone.0056710.s004"><offsets xml_i="35060" xml_f="35069" txt_i="17862" txt_f="17871">Figure S4</offsets></xref><offsets xml_i="35076" xml_f="35171" txt_i="17871" txt_f="17966">). The existence of different host-microbial symbiotic states in adults was recently described </offsets><xref rid="pone.0056710-Arumugam1" ref-type="bibr"><offsets xml_i="35222" xml_f="35226" txt_i="17966" txt_f="17970">[14]</offsets></xref><offsets xml_i="35233" xml_f="35387" txt_i="17970" txt_f="18124"> and it is intriguing to see “enterotype-like” clustering in colicky infants and that this correlates to the improvement in colic symptoms to some extent.</offsets></p><p><offsets xml_i="35394" xml_f="35647" txt_i="18125" txt_f="18378">Vaginally delivery of the infants was overrepresented in enterotype group B and several other studies have reported that method of delivery influenced the microbiota in the first period of life. Caesarean section has been shown to delay colonization by </offsets><italic><offsets xml_i="35655" xml_f="35684" txt_i="18378" txt_f="18407">Escherichia coli, Bacteroides</offsets></italic><offsets xml_i="35693" xml_f="35698" txt_i="18407" txt_f="18412"> and </offsets><italic><offsets xml_i="35706" xml_f="35721" txt_i="18412" txt_f="18427">Bifidobacterium</offsets></italic><offsets xml_i="35730" xml_f="35731" txt_i="18427" txt_f="18428">
</offsets><xref rid="pone.0056710-Adlerberth2" ref-type="bibr"><offsets xml_i="35784" xml_f="35788" txt_i="18428" txt_f="18432">[26]</offsets></xref><offsets xml_i="35795" xml_f="35831" txt_i="18432" txt_f="18468"> and a negative correlation between </offsets><italic><offsets xml_i="35839" xml_f="35850" txt_i="18468" txt_f="18479">Bacteroides</offsets></italic><offsets xml_i="35859" xml_f="35900" txt_i="18479" txt_f="18520"> and cesarean section has been described </offsets><xref rid="pone.0056710-Fallani1" ref-type="bibr"><offsets xml_i="35950" xml_f="35954" txt_i="18520" txt_f="18524">[27]</offsets></xref><offsets xml_i="35961" xml_f="35984" txt_i="18524" txt_f="18547">. Recently Jost et al. </offsets><xref rid="pone.0056710-Jost1" ref-type="bibr"><offsets xml_i="36031" xml_f="36035" txt_i="18547" txt_f="18551">[28]</offsets></xref><offsets xml_i="36042" xml_f="36240" txt_i="18551" txt_f="18749"> performed a study, using pyrosequencing, to investigate the establishment of pioneer gut microbiota during the first month of life of vaginally and exclusively breastfed infants. An early onset of </offsets><italic><offsets xml_i="36248" xml_f="36259" txt_i="18749" txt_f="18760">Bacteroides</offsets></italic><offsets xml_i="36268" xml_f="36477" txt_i="18760" txt_f="18969"> was observed in 4 of 7 subjects, demonstrating that those strict anaerobes can reach high densities already during the first weeks of life. In this regards conflicting findings have been published previously </offsets><xref rid="pone.0056710-Turroni1" ref-type="bibr"><offsets xml_i="36527" xml_f="36531" txt_i="18969" txt_f="18973">[29]</offsets></xref><offsets xml_i="36538" xml_f="36539" txt_i="18973" txt_f="18974">.</offsets></p><p><offsets xml_i="36546" xml_f="37028" txt_i="18975" txt_f="19457">Our current knowledge of the intestinal microbiota and its role in infant colic is weak. The question is if infantile colic is a cause or consequence of an alteration in gut microbiota. Illustrating the complexity of the interaction, the gut microbiota has been described to be involved in the regulation of various host metabolic pathways, giving rise to interactive host-microbiota metabolic, signaling, and immune-inflammatory axes that physiologically link the gut to the brain </offsets><xref rid="pone.0056710-Nicholson1" ref-type="bibr"><offsets xml_i="37080" xml_f="37084" txt_i="19457" txt_f="19461">[30]</offsets></xref><offsets xml_i="37091" xml_f="37210" txt_i="19461" txt_f="19580">. The change of the composition of the microbiota detected in the responders could not be attributed to treatment with </offsets><italic><offsets xml_i="37218" xml_f="37228" txt_i="19580" txt_f="19590">L. reuteri</offsets></italic><offsets xml_i="37237" xml_f="37466" txt_i="19590" txt_f="19819">. Consequently, the mechanisms by which this bacterium reduces colic symptoms may be either by modulating of the microbiota in a way that is not detected by 454 pyrosequencing, or by a direct effect on the infant. Preidis et al. </offsets><xref rid="pone.0056710-Preidis1" ref-type="bibr"><offsets xml_i="37516" xml_f="37520" txt_i="19819" txt_f="19823">[31]</offsets></xref><offsets xml_i="37527" xml_f="37542" txt_i="19823" txt_f="19838"> reported that </offsets><italic><offsets xml_i="37550" xml_f="37560" txt_i="19838" txt_f="19848">L. reuteri</offsets></italic><offsets xml_i="37569" xml_f="37598" txt_i="19848" txt_f="19877"> DSM 17938, but not an other </offsets><italic><offsets xml_i="37606" xml_f="37616" txt_i="19877" txt_f="19887">L. reuteri</offsets></italic><offsets xml_i="37625" xml_f="37878" txt_i="19887" txt_f="20140"> strain, considerably increased enterocyte migration, proliferation and crypt height in an experimental model on neonatal mice. The possible role of these mechanisms in decreasing colic symptoms is not known, but they may represent novel mechanisms for </offsets><italic><offsets xml_i="37886" xml_f="37896" txt_i="20140" txt_f="20150">L. reuteri</offsets></italic><offsets xml_i="37905" xml_f="38012" txt_i="20150" txt_f="20257"> strain-specific effects. Although no effect on the microbiota could be attributed to supplementation with </offsets><italic><offsets xml_i="38020" xml_f="38030" txt_i="20257" txt_f="20267">L. reuteri</offsets></italic><offsets xml_i="38039" xml_f="38187" txt_i="20267" txt_f="20415"> DSM 17938, the results of the study may provide the basis for further investigation of the role of the intestinal microbiota in infants with colic.</offsets></p></sec><sec sec-type="supplementary-material" id="s4"><title><offsets xml_i="38251" xml_f="38273" txt_i="20417" txt_f="20439">Supporting Information</offsets></title><supplementary-material content-type="local-data" id="pone.0056710.s001"><label><offsets xml_i="38361" xml_f="38370" txt_i="20440" txt_f="20449">Figure S1</offsets></label><caption><p><offsets xml_i="38390" xml_f="38391" txt_i="20449" txt_f="20450">
</offsets><bold><offsets xml_i="38397" xml_f="38453" txt_i="20450" txt_f="20506">Bacterial composition of 29 infants at the phylum level.</offsets></bold><offsets xml_i="38460" xml_f="38694" txt_i="20506" txt_f="20740"> The first column in each pair shows the initial composition and the second the composition at day 21. The different phyla are indicated by different colors and the infants responding to the treatment as well as the individuals given </offsets><italic><offsets xml_i="38702" xml_f="38712" txt_i="20740" txt_f="20750">L. reuteri</offsets></italic><offsets xml_i="38721" xml_f="38747" txt_i="20750" txt_f="20776"> are marked in the figure.</offsets></p><p><offsets xml_i="38754" xml_f="38759" txt_i="20777" txt_f="20782">(EPS)</offsets></p></caption><media xlink:href="pone.0056710.s001.eps"><caption><p><offsets xml_i="38827" xml_f="38863" txt_i="20783" txt_f="20819">Click here for additional data file.</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="pone.0056710.s002"><label><offsets xml_i="38990" xml_f="38999" txt_i="20820" txt_f="20829">Figure S2</offsets></label><caption><p><offsets xml_i="39019" xml_f="39020" txt_i="20829" txt_f="20830">
</offsets><bold><offsets xml_i="39026" xml_f="39132" txt_i="20830" txt_f="20936">Principal component analysis on the difference in abundance of different phyla before and after treatment.</offsets></bold><offsets xml_i="39139" xml_f="39283" txt_i="20936" txt_f="21080"> Ellipses show 95% confidence around the centroids of the two outcomes of the treatment. Filled circles are responders and empty non-responders.</offsets></p><p><offsets xml_i="39290" xml_f="39295" txt_i="21081" txt_f="21086">(EPS)</offsets></p></caption><media xlink:href="pone.0056710.s002.eps"><caption><p><offsets xml_i="39363" xml_f="39399" txt_i="21087" txt_f="21123">Click here for additional data file.</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="pone.0056710.s003"><label><offsets xml_i="39526" xml_f="39535" txt_i="21124" txt_f="21133">Figure S3</offsets></label><caption><p><offsets xml_i="39555" xml_f="39556" txt_i="21133" txt_f="21134">
</offsets><bold><offsets xml_i="39562" xml_f="39594" txt_i="21134" txt_f="21166">Change in relative abundance of </offsets></bold><offsets xml_i="39601" xml_f="39602" txt_i="21166" txt_f="21167">
</offsets><bold><italic><offsets xml_i="39616" xml_f="39629" txt_i="21167" txt_f="21180">Bacteroidetes</offsets></italic></bold><offsets xml_i="39645" xml_f="39646" txt_i="21180" txt_f="21181">
</offsets><bold><offsets xml_i="39652" xml_f="39684" txt_i="21181" txt_f="21213"> during the intervention period.</offsets></bold><offsets xml_i="39691" xml_f="39804" txt_i="21213" txt_f="21326"> The group of responders had an increase of this bacterial phylum and differed from the group of non-responders (</offsets><italic><offsets xml_i="39812" xml_f="39813" txt_i="21326" txt_f="21327">P</offsets></italic><offsets xml_i="39822" xml_f="39851" txt_i="21327" txt_f="21356"> = 0.004; Mann Whitney test).</offsets></p><p><offsets xml_i="39858" xml_f="39863" txt_i="21357" txt_f="21362">(EPS)</offsets></p></caption><media xlink:href="pone.0056710.s003.eps"><caption><p><offsets xml_i="39931" xml_f="39967" txt_i="21363" txt_f="21399">Click here for additional data file.</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="pone.0056710.s004"><label><offsets xml_i="40094" xml_f="40103" txt_i="21400" txt_f="21409">Figure S4</offsets></label><caption><p><offsets xml_i="40123" xml_f="40124" txt_i="21409" txt_f="21410">
</offsets><bold><offsets xml_i="40130" xml_f="40169" txt_i="21410" txt_f="21449">PCA plot from faecal samples on day 21.</offsets></bold><offsets xml_i="40176" xml_f="40222" txt_i="21449" txt_f="21495"> Bacterial phyla indicated in circles: black, </offsets><italic><offsets xml_i="40230" xml_f="40244" txt_i="21495" txt_f="21509">Actinobacteria</offsets></italic><offsets xml_i="40253" xml_f="40260" txt_i="21509" txt_f="21516">; red, </offsets><italic><offsets xml_i="40268" xml_f="40281" txt_i="21516" txt_f="21529">Bacteroidetes</offsets></italic><offsets xml_i="40290" xml_f="40298" txt_i="21529" txt_f="21537">; blue, </offsets><italic><offsets xml_i="40306" xml_f="40320" txt_i="21537" txt_f="21551">Proteobacteria</offsets></italic><offsets xml_i="40329" xml_f="40342" txt_i="21551" txt_f="21564">, and green, </offsets><italic><offsets xml_i="40350" xml_f="40360" txt_i="21564" txt_f="21574">Firmicutes</offsets></italic><offsets xml_i="40369" xml_f="40542" txt_i="21574" txt_f="21747">. The small inlaid figure describes the features of the samples: purple, responders; green, non-responders; triangles, spontaneous delivery; and squares, caesarean delivery.</offsets></p><p><offsets xml_i="40549" xml_f="40554" txt_i="21748" txt_f="21753">(EPS)</offsets></p></caption><media xlink:href="pone.0056710.s004.eps"><caption><p><offsets xml_i="40622" xml_f="40658" txt_i="21754" txt_f="21790">Click here for additional data file.</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="pone.0056710.s005"><label><offsets xml_i="40785" xml_f="40797" txt_i="21791" txt_f="21803">Checklist S1</offsets></label><caption><p><offsets xml_i="40817" xml_f="40818" txt_i="21803" txt_f="21804">
</offsets><bold><offsets xml_i="40824" xml_f="40904" txt_i="21804" txt_f="21884">The CONSORT 2010 checklist shows all the information about the randomised trial.</offsets></bold><offsets xml_i="40911" xml_f="40912" txt_i="21884" txt_f="21885">
</offsets></p><p><offsets xml_i="40919" xml_f="40924" txt_i="21886" txt_f="21891">(DOC)</offsets></p></caption><media xlink:href="pone.0056710.s005.doc"><caption><p><offsets xml_i="40992" xml_f="41028" txt_i="21892" txt_f="21928">Click here for additional data file.</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="pone.0056710.s006"><label><offsets xml_i="41155" xml_f="41166" txt_i="21929" txt_f="21940">Protocol S1</offsets></label><caption><p><offsets xml_i="41186" xml_f="41187" txt_i="21940" txt_f="21941">
</offsets><bold><offsets xml_i="41193" xml_f="41208" txt_i="21941" txt_f="21956">Trial Protocol.</offsets></bold><offsets xml_i="41215" xml_f="41216" txt_i="21956" txt_f="21957">
</offsets></p><p><offsets xml_i="41223" xml_f="41228" txt_i="21958" txt_f="21963">(DOC)</offsets></p></caption><media xlink:href="pone.0056710.s006.doc"><caption><p><offsets xml_i="41296" xml_f="41332" txt_i="21964" txt_f="22000">Click here for additional data file.</offsets></p></caption></media></supplementary-material></sec></body><back><ack><p>We thank Lotta Jäderlund for skilled technical assistance.</p></ack><ref-list><title>References</title><ref id="pone.0056710-Wessel1"><label>1</label><mixed-citation publication-type="journal">
<name><surname>Wessel</surname><given-names>MA</given-names></name>, <name><surname>Cobb</surname><given-names>JC</given-names></name>, <name><surname>Jackson</surname><given-names>EB</given-names></name>, <name><surname>Harris</surname><given-names>GS</given-names></name>, <name><surname>Detwiler</surname><given-names>AC</given-names></name> (<year>1954</year>) <article-title>Paroxysmal fussing in infancy, sometimes called colic</article-title>. <source>Pediatrics</source>
<volume>14</volume>: <fpage>421</fpage>–<lpage>435</lpage>.<pub-id pub-id-type="pmid">13214956</pub-id></mixed-citation></ref><ref id="pone.0056710-Yalin1"><label>2</label><mixed-citation publication-type="journal">
<name><surname>Yalçin</surname><given-names>SS</given-names></name>, <name><surname>Orün</surname><given-names>E</given-names></name>, <name><surname>Mutlu</surname><given-names>B</given-names></name>, <name><surname>Madendağ</surname><given-names>Y</given-names></name>, <name><surname>Sinici</surname><given-names>I</given-names></name>, <etal>et al</etal> (<year>2010</year>) <article-title>Why are they having infant colic? A nested case-control study</article-title>. <source>Paediatr Perinat Epidemiol</source>
<volume>24</volume>: <fpage>584</fpage>–<lpage>596</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1111/j.1365–3016.2010.01150.x">10.1111/j.1365–3016.2010.01150.x</ext-link></comment>.<pub-id pub-id-type="pmid">20955236</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino1"><label>3</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name> (<year>2007</year>) <article-title>Focus on infantile colic</article-title>. <source>Acta Paediatr</source>
<volume>96</volume>: <fpage>1259</fpage>–<lpage>1264</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1111/j.1651–2227.2007.00428.x">10.1111/j.1651–2227.2007.00428.x</ext-link></comment>.<pub-id pub-id-type="pmid">17718777</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino2"><label>4</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Tarasco</surname><given-names>V</given-names></name> (<year>2010</year>) <article-title>New treatments for infant colic</article-title>. <source>Curr Opin Pediatr</source>
<volume>22</volume>: <fpage>791</fpage>–<lpage>797</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1097/MOP.0b013e32833fac24">10.1097/MOP.0b013e32833fac24</ext-link></comment>.<pub-id pub-id-type="pmid">20859207</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino3"><label>5</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Cordisco</surname><given-names>L</given-names></name>, <name><surname>Tarasco</surname><given-names>V</given-names></name>, <name><surname>Calabrese</surname><given-names>R</given-names></name>, <name><surname>Palumeri</surname><given-names>E</given-names></name>, <etal>et al</etal> (<year>2009</year>) <article-title>Molecular identification of coliform bacteria from colicky breastfed infants</article-title>. <source>Acta Paediatr</source>
<volume>98</volume>: <fpage>1582</fpage>–<lpage>1588</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1111/j.1651–2227.2009.01419.x">10.1111/j.1651–2227.2009.01419.x</ext-link></comment>.<pub-id pub-id-type="pmid">19604166</pub-id></mixed-citation></ref><ref id="pone.0056710-Rhoads1"><label>6</label><mixed-citation publication-type="journal">
<name><surname>Rhoads</surname><given-names>JM</given-names></name>, <name><surname>Fatheree</surname><given-names>NY</given-names></name>, <name><surname>Norori</surname><given-names>J</given-names></name>, <name><surname>Liu</surname><given-names>Y</given-names></name>, <name><surname>Lucke</surname><given-names>JF</given-names></name>, <etal>et al</etal> (<year>2009</year>) <article-title>Altered fecal microflora and increased fecal calprotectin in infants with colic</article-title>. <source>The Journal of Pediatrics</source>
<volume>155</volume>: <fpage>823</fpage>–<lpage>828.e1</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1016/j.jpeds.2009.05.012">10.1016/j.jpeds.2009.05.012</ext-link></comment>.<pub-id pub-id-type="pmid">19628216</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino4"><label>7</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Cresi</surname><given-names>F</given-names></name>, <name><surname>Pautasso</surname><given-names>S</given-names></name>, <name><surname>Palumeri</surname><given-names>E</given-names></name>, <name><surname>Tullio</surname><given-names>V</given-names></name>, <etal>et al</etal> (<year>2004</year>) <article-title>Intestinal microflora in breastfed colicky and non-colicky infants</article-title>. <source>Acta Paediatr</source>
<volume>93</volume>: <fpage>825</fpage>–<lpage>829</lpage>.<pub-id pub-id-type="pmid">15244234</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino5"><label>8</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Cordisco</surname><given-names>L</given-names></name>, <name><surname>Tarasco</surname><given-names>V</given-names></name>, <name><surname>Locatelli</surname><given-names>E</given-names></name>, <name><surname>Di Gioia</surname><given-names>D</given-names></name>, <etal>et al</etal> (<year>2011</year>) <article-title>Antagonistic effect of Lactobacillus strains against gas-producing coliforms isolated from colicky infants</article-title>. <source>BMC Microbiol</source>
<volume>11</volume>: <fpage>157</fpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1186/1471–2180–11–157">10.1186/1471–2180–11–157</ext-link></comment>.<pub-id pub-id-type="pmid">21718486</pub-id></mixed-citation></ref><ref id="pone.0056710-Prtty1"><label>9</label><mixed-citation publication-type="journal">
<name><surname>Pärtty</surname><given-names>A</given-names></name>, <name><surname>Kalliomäki</surname><given-names>M</given-names></name>, <name><surname>Endo</surname><given-names>A</given-names></name>, <name><surname>Salminen</surname><given-names>S</given-names></name>, <name><surname>Isolauri</surname><given-names>E</given-names></name> (<year>2012</year>) <article-title>Compositional development of Bifidobacterium and Lactobacillus microbiota is linked with crying and fussing in early infancy</article-title>. <source>PLoS ONE</source>
<volume>7</volume>: <fpage>e32495</fpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1371/journal.pone.0032495">10.1371/journal.pone.0032495</ext-link></comment>.<pub-id pub-id-type="pmid">22403665</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino6"><label>10</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Cordisco</surname><given-names>L</given-names></name>, <name><surname>Tarasco</surname><given-names>V</given-names></name>, <name><surname>Palumeri</surname><given-names>E</given-names></name>, <name><surname>Calabrese</surname><given-names>R</given-names></name>, <etal>et al</etal> (<year>2010</year>) <article-title>Lactobacillus reuteri DSM 17938 in infantile colic: a randomized, double-blind, placebo-controlled trial</article-title>. <source>Pediatrics</source>
<volume>126</volume>: <fpage>e526</fpage>–<lpage>e533</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1542/peds.2010–0433">10.1542/peds.2010–0433</ext-link></comment>.<pub-id pub-id-type="pmid">20713478</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino7"><label>11</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Pelle</surname><given-names>E</given-names></name>, <name><surname>Palumeri</surname><given-names>E</given-names></name>, <name><surname>Oggero</surname><given-names>R</given-names></name>, <name><surname>Miniero</surname><given-names>R</given-names></name> (<year>2007</year>) <article-title>Lactobacillus reuteri (American Type Culture Collection Strain 55730) versus simethicone in the treatment of infantile colic: a prospective randomized study</article-title>. <source>Pediatrics</source>
<volume>119</volume>: <fpage>e124</fpage>–<lpage>e130</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1542/peds.2006–1222">10.1542/peds.2006–1222</ext-link></comment>.<pub-id pub-id-type="pmid">17200238</pub-id></mixed-citation></ref><ref id="pone.0056710-Szajewska1"><label>12</label><mixed-citation publication-type="other">Szajewska H, Gyrczuk E, Horvath A (2012) Lactobacillus reuteri DSM 17938 for the Management of Infantile Colic in Breastfed Infants: A Randomized, Double-Blind, Placebo-Controlled Trial. The Journal of Pediatrics. doi:10.1016/j.jpeds.2012.08.004.</mixed-citation></ref><ref id="pone.0056710-Herlemann1"><label>13</label><mixed-citation publication-type="journal">
<name><surname>Herlemann</surname><given-names>DP</given-names></name>, <name><surname>Labrenz</surname><given-names>M</given-names></name>, <name><surname>Jürgens</surname><given-names>K</given-names></name>, <name><surname>Bertilsson</surname><given-names>S</given-names></name>, <name><surname>Waniek</surname><given-names>JJ</given-names></name>, <etal>et al</etal> (<year>2011</year>) <article-title>Transitions in bacterial communities along the 2000 km salinity gradient of the Baltic Sea</article-title>. <source>ISME J</source>
<volume>5</volume>: <fpage>1571</fpage>–<lpage>1579</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1038/ismej.2011.41">10.1038/ismej.2011.41</ext-link></comment>.<pub-id pub-id-type="pmid">21472016</pub-id></mixed-citation></ref><ref id="pone.0056710-Arumugam1"><label>14</label><mixed-citation publication-type="journal">
<name><surname>Arumugam</surname><given-names>M</given-names></name>, <name><surname>Raes</surname><given-names>J</given-names></name>, <name><surname>Pelletier</surname><given-names>E</given-names></name>, <name><surname>Le Paslier</surname><given-names>D</given-names></name>, <name><surname>Yamada</surname><given-names>T</given-names></name>, <etal>et al</etal> (<year>2011</year>) <article-title>Enterotypes of the human gut microbiome</article-title>. <source>Nature</source>
<volume>473</volume>: <fpage>174</fpage>–<lpage>180</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1038/nature09944">10.1038/nature09944</ext-link></comment>.<pub-id pub-id-type="pmid">21508958</pub-id></mixed-citation></ref><ref id="pone.0056710-DeFilippo1"><label>15</label><mixed-citation publication-type="journal">
<name><surname>De Filippo</surname><given-names>C</given-names></name>, <name><surname>Cavalieri</surname><given-names>D</given-names></name>, <name><surname>Di Paola</surname><given-names>M</given-names></name>, <name><surname>Ramazzotti</surname><given-names>M</given-names></name>, <name><surname>Poullet</surname><given-names>JB</given-names></name>, <etal>et al</etal> (<year>2010</year>) <article-title>Impact of diet in shaping gut microbiota revealed by a comparative study in children from Europe and rural Africa</article-title>. <source>Proc Natl Acad Sci USA</source>
<volume>107</volume>: <fpage>14691</fpage>–<lpage>14696</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1073/pnas.1005963107">10.1073/pnas.1005963107</ext-link></comment>.<pub-id pub-id-type="pmid">20679230</pub-id></mixed-citation></ref><ref id="pone.0056710-Andersson1"><label>16</label><mixed-citation publication-type="journal">
<name><surname>Andersson</surname><given-names>AF</given-names></name>, <name><surname>Lindberg</surname><given-names>M</given-names></name>, <name><surname>Jakobsson</surname><given-names>H</given-names></name>, <name><surname>Bäckhed</surname><given-names>F</given-names></name>, <name><surname>Nyrén</surname><given-names>P</given-names></name>, <etal>et al</etal> (<year>2008</year>) <article-title>Comparative analysis of human gut microbiota by barcoded pyrosequencing</article-title>. <source>PLoS ONE</source>
<volume>3</volume>: <fpage>e2836</fpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1371/journal.pone.0002836">10.1371/journal.pone.0002836</ext-link></comment>.<pub-id pub-id-type="pmid">18665274</pub-id></mixed-citation></ref><ref id="pone.0056710-Cole1"><label>17</label><mixed-citation publication-type="journal">
<name><surname>Cole</surname><given-names>JR</given-names></name>, <name><surname>Wang</surname><given-names>Q</given-names></name>, <name><surname>Cardenas</surname><given-names>E</given-names></name>, <name><surname>Fish</surname><given-names>J</given-names></name>, <name><surname>Chai</surname><given-names>B</given-names></name>, <etal>et al</etal> (<year>2009</year>) <article-title>The Ribosomal Database Project: improved alignments and new tools for rRNA analysis</article-title>. <source>Nucleic Acids Res</source>
<volume>37</volume>: <fpage>D141</fpage>–<lpage>D145</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1093/nar/gkn879">10.1093/nar/gkn879</ext-link></comment>.<pub-id pub-id-type="pmid">19004872</pub-id></mixed-citation></ref><ref id="pone.0056710-Oksanen1"><label>18</label><mixed-citation publication-type="other">Oksanen J, Blanchet GF, Kindt R, Legendre P, Minchin PR, <etal>et al</etal>. (2012) vegan: Community Ecology Package. R package version 2.0–5. <ext-link ext-link-type="uri" xlink:href="http://CRANR-projectorg/package=vegan">http://CRANR-projectorg/package=vegan</ext-link>.</mixed-citation></ref><ref id="pone.0056710-Hammer1"><label>19</label><mixed-citation publication-type="journal">
<name><surname>Hammer</surname><given-names>O</given-names></name>, <name><surname>Harper</surname><given-names>D</given-names></name>, <name><surname>Ryan</surname><given-names>PD</given-names></name> (<year>2001</year>) <article-title>PAST: Paleontological Statistics Software Package for Education and Data Analysis</article-title>. <source>Palaeontologia Electronica</source>
<volume>4</volume>: <fpage>1</fpage>–<lpage>9</lpage>.</mixed-citation></ref><ref id="pone.0056710-Palmer1"><label>20</label><mixed-citation publication-type="journal">
<name><surname>Palmer</surname><given-names>C</given-names></name>, <name><surname>Bik</surname><given-names>EM</given-names></name>, <name><surname>DiGiulio</surname><given-names>DB</given-names></name>, <name><surname>Relman</surname><given-names>DA</given-names></name>, <name><surname>Brown</surname><given-names>PO</given-names></name> (<year>2007</year>) <article-title>Development of the human infant intestinal microbiota</article-title>. <source>PLoS Biol</source>
<volume>5</volume>: <fpage>e177</fpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1371/journal.pbio.0050177">10.1371/journal.pbio.0050177</ext-link></comment>.<pub-id pub-id-type="pmid">17594176</pub-id></mixed-citation></ref><ref id="pone.0056710-Adlerberth1"><label>21</label><mixed-citation publication-type="journal">
<name><surname>Adlerberth</surname><given-names>I</given-names></name>, <name><surname>Wold</surname><given-names>AE</given-names></name> (<year>2009</year>) <article-title>Establishment of the gut microbiota in Western infants</article-title>. <source>Acta Paediatr</source>
<volume>98</volume>: <fpage>229</fpage>–<lpage>238</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1111/j.1651–2227.2008.01060.x">10.1111/j.1651–2227.2008.01060.x</ext-link></comment>.<pub-id pub-id-type="pmid">19143664</pub-id></mixed-citation></ref><ref id="pone.0056710-Abrahamsson1"><label>22</label><mixed-citation publication-type="journal">
<name><surname>Abrahamsson</surname><given-names>TRT</given-names></name>, <name><surname>Jakobsson</surname><given-names>HEH</given-names></name>, <name><surname>Andersson</surname><given-names>AFA</given-names></name>, <name><surname>Björkstén</surname><given-names>BB</given-names></name>, <name><surname>Engstrand</surname><given-names>LL</given-names></name>, <etal>et al</etal> (<year>2012</year>) <article-title>Low diversity of the gut microbiota in infants with atopic eczema</article-title>. <source>J Allergy Clin Immunol</source>
<volume>129</volume>: <fpage>434</fpage>–<lpage>432</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1016/j.jaci.2011.10.025">10.1016/j.jaci.2011.10.025</ext-link></comment>.<pub-id pub-id-type="pmid">22153774</pub-id></mixed-citation></ref><ref id="pone.0056710-Savino8"><label>23</label><mixed-citation publication-type="journal">
<name><surname>Savino</surname><given-names>F</given-names></name>, <name><surname>Castagno</surname><given-names>E</given-names></name>, <name><surname>Bretto</surname><given-names>R</given-names></name>, <name><surname>Brondello</surname><given-names>C</given-names></name>, <name><surname>Palumeri</surname><given-names>E</given-names></name>, <etal>et al</etal> (<year>2005</year>) <article-title>A prospective 10-year study on children who had severe infantile colic Vol</article-title>. <volume>94</volume>: <fpage>129</fpage>–<lpage>132</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1080/08035320510043691">10.1080/08035320510043691</ext-link></comment>.</mixed-citation></ref><ref id="pone.0056710-Koenig1"><label>24</label><mixed-citation publication-type="journal">
<name><surname>Koenig</surname><given-names>JE</given-names></name>, <name><surname>Spor</surname><given-names>A</given-names></name>, <name><surname>Scalfone</surname><given-names>N</given-names></name>, <name><surname>Fricker</surname><given-names>AD</given-names></name>, <name><surname>Stombaugh</surname><given-names>J</given-names></name>, <etal>et al</etal> (<year>2011</year>) <article-title>Succession of microbial consortia in the developing infant gut microbiome</article-title>. <source>Proc Natl Acad Sci USA</source>
<volume>108</volume> Suppl 1<fpage>4578</fpage>–<lpage>4585</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1073/pnas.1000081107">10.1073/pnas.1000081107</ext-link></comment>.<pub-id pub-id-type="pmid">20668239</pub-id></mixed-citation></ref><ref id="pone.0056710-Shimizu1"><label>25</label><mixed-citation publication-type="journal">
<name><surname>Shimizu</surname><given-names>K</given-names></name>, <name><surname>Ogura</surname><given-names>H</given-names></name>, <name><surname>Asahara</surname><given-names>T</given-names></name>, <name><surname>Nomoto</surname><given-names>K</given-names></name>, <name><surname>Morotomi</surname><given-names>M</given-names></name>, <etal>et al</etal> (<year>2011</year>) <article-title>Gastrointestinal dysmotility is associated with altered gut flora and septic mortality in patients with severe systemic inflammatory response syndrome: a preliminary study</article-title>. <source>Neurogastroenterol Motil</source>
<volume>23</volume>: <fpage>330</fpage>–<lpage>5–e157</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1111/j.1365–2982.2010.01653.x">10.1111/j.1365–2982.2010.01653.x</ext-link></comment>.<pub-id pub-id-type="pmid">21199173</pub-id></mixed-citation></ref><ref id="pone.0056710-Adlerberth2"><label>26</label><mixed-citation publication-type="journal">
<name><surname>Adlerberth</surname><given-names>I</given-names></name>, <name><surname>Strachan</surname><given-names>DP</given-names></name>, <name><surname>Matricardi</surname><given-names>PM</given-names></name>, <name><surname>Ahrné</surname><given-names>S</given-names></name>, <name><surname>Orfei</surname><given-names>L</given-names></name>, <etal>et al</etal> (<year>2007</year>) <article-title>Gut microbiota and development of atopic eczema in 3 European birth cohorts</article-title>. <source>J Allergy Clin Immunol</source>
<volume>120</volume>: <fpage>343</fpage>–<lpage>350</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1016/j.jaci.2007.05.018">10.1016/j.jaci.2007.05.018</ext-link></comment>.<pub-id pub-id-type="pmid">17604093</pub-id></mixed-citation></ref><ref id="pone.0056710-Fallani1"><label>27</label><mixed-citation publication-type="journal">
<name><surname>Fallani</surname><given-names>M</given-names></name>, <name><surname>Young</surname><given-names>D</given-names></name>, <name><surname>Scott</surname><given-names>J</given-names></name>, <name><surname>Norin</surname><given-names>E</given-names></name>, <name><surname>Amarri</surname><given-names>S</given-names></name>, <etal>et al</etal> (<year>2010</year>) <article-title>Intestinal Microbiota of 6-week-old Infants Across Europe: Geographic Influence Beyond Delivery Mode, Breast-feeding, and Antibiotics</article-title>. <source>Journal of Pediatric Gastroenterology and Nutrition</source>
<volume>51</volume>: <fpage>77</fpage>–<lpage>84</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1097/MPG.0b013e3181d1b11e">10.1097/MPG.0b013e3181d1b11e</ext-link></comment>.<pub-id pub-id-type="pmid">20479681</pub-id></mixed-citation></ref><ref id="pone.0056710-Jost1"><label>28</label><mixed-citation publication-type="journal">
<name><surname>Jost</surname><given-names>T</given-names></name>, <name><surname>Lacroix</surname><given-names>C</given-names></name>, <name><surname>Braegger</surname><given-names>CP</given-names></name>, <name><surname>Chassard</surname><given-names>C</given-names></name> (<year>2012</year>) <article-title>New insights in gut microbiota establishment in healthy breast fed neonates</article-title>. <source>PLoS ONE</source>
<volume>7</volume>: <fpage>e44595</fpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1371/journal.pone.0044595">10.1371/journal.pone.0044595</ext-link></comment>.<pub-id pub-id-type="pmid">22957008</pub-id></mixed-citation></ref><ref id="pone.0056710-Turroni1"><label>29</label><mixed-citation publication-type="journal">
<name><surname>Turroni</surname><given-names>F</given-names></name>, <name><surname>Peano</surname><given-names>C</given-names></name>, <name><surname>Pass</surname><given-names>DA</given-names></name>, <name><surname>Foroni</surname><given-names>E</given-names></name>, <name><surname>Severgnini</surname><given-names>M</given-names></name>, <etal>et al</etal> (<year>2012</year>) <article-title>Diversity of bifidobacteria within the infant gut microbiota</article-title>. <source>PLoS ONE</source>
<volume>7</volume>: <fpage>e36957</fpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1371/journal.pone.0036957">10.1371/journal.pone.0036957</ext-link></comment>.<pub-id pub-id-type="pmid">22606315</pub-id></mixed-citation></ref><ref id="pone.0056710-Nicholson1"><label>30</label><mixed-citation publication-type="journal">
<name><surname>Nicholson</surname><given-names>JK</given-names></name>, <name><surname>Holmes</surname><given-names>E</given-names></name>, <name><surname>Kinross</surname><given-names>J</given-names></name>, <name><surname>Burcelin</surname><given-names>R</given-names></name>, <name><surname>Gibson</surname><given-names>G</given-names></name>, <etal>et al</etal> (<year>2012</year>) <article-title>Host-gut microbiota metabolic interactions</article-title>. <source>Science</source>
<volume>336</volume>: <fpage>1262</fpage>–<lpage>1267</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1126/science.1223813">10.1126/science.1223813</ext-link></comment>.<pub-id pub-id-type="pmid">22674330</pub-id></mixed-citation></ref><ref id="pone.0056710-Preidis1"><label>31</label><mixed-citation publication-type="journal">
<name><surname>Preidis</surname><given-names>GA</given-names></name>, <name><surname>Saulnier</surname><given-names>DM</given-names></name>, <name><surname>Blutt</surname><given-names>SE</given-names></name>, <name><surname>Mistretta</surname><given-names>T-A</given-names></name>, <name><surname>Riehle</surname><given-names>KP</given-names></name>, <etal>et al</etal> (<year>2012</year>) <article-title>Probiotics stimulate enterocyte migration and microbial diversity in the neonatal mouse intestine</article-title>. <source>FASEB J</source>
<volume>26</volume>: <fpage>1960</fpage>–<lpage>1969</lpage>
<comment>doi:<ext-link ext-link-type="uri" xlink:href="http://dx.doi.org/10.1096/fj.10–177980">10.1096/fj.10–177980</ext-link></comment>.<pub-id pub-id-type="pmid">22267340</pub-id></mixed-citation></ref></ref-list></back></article>