<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article" xml:lang="en"><front><journal-meta><journal-id journal-id-type="nlm-ta">Breast Cancer Res</journal-id><journal-id journal-id-type="iso-abbrev">Breast Cancer Res</journal-id><journal-title-group><journal-title>Breast Cancer Research : BCR</journal-title></journal-title-group><issn pub-type="ppub">1465-5411</issn><issn pub-type="epub">1465-542X</issn><publisher><publisher-name>BioMed Central</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">23566419</article-id><article-id pub-id-type="pmc">3672758</article-id><article-id pub-id-type="publisher-id">bcr3409</article-id><article-id pub-id-type="doi">10.1186/bcr3409</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>A randomised controlled phase II trial of pre-operative celecoxib treatment reveals anti-tumour transcriptional response in primary breast cancer</article-title></title-group><contrib-group><contrib contrib-type="author" equal-contrib="yes" id="A1"><name><surname>Brandão</surname><given-names>Rita D</given-names></name><xref ref-type="aff" rid="I1">1</xref><xref ref-type="aff" rid="I2">2</xref><email>rita.brandao@maastrichtuniversity.nl</email></contrib><contrib contrib-type="author" corresp="yes" equal-contrib="yes" id="A2"><name><surname>Veeck</surname><given-names>Jürgen</given-names></name><xref ref-type="aff" rid="I2">2</xref><xref ref-type="aff" rid="I3">3</xref><xref ref-type="aff" rid="I4">4</xref><xref ref-type="aff" rid="I5">5</xref><email>juergen.veeck@rwth-aachen.de</email></contrib><contrib contrib-type="author" id="A3"><name><surname>Van de Vijver</surname><given-names>Koen K</given-names></name><xref ref-type="aff" rid="I2">2</xref><xref ref-type="aff" rid="I4">4</xref><email>koen.vande.vijver@mumc.nl</email></contrib><contrib contrib-type="author" id="A4"><name><surname>Lindsey</surname><given-names>Patrick</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>patrick.lindsey@maastrichtuniversity.nl</email></contrib><contrib contrib-type="author" id="A5"><name><surname>de Vries</surname><given-names>Bart</given-names></name><xref ref-type="aff" rid="I2">2</xref><xref ref-type="aff" rid="I4">4</xref><email>bart.de.vries@mumc.nl</email></contrib><contrib contrib-type="author" id="A6"><name><surname>van Elssen</surname><given-names>Catharina HMJ</given-names></name><xref ref-type="aff" rid="I6">6</xref><xref ref-type="aff" rid="I7">7</xref><email>j.vanelssen@maastrichtuniversity.nl</email></contrib><contrib contrib-type="author" id="A7"><name><surname>Blok</surname><given-names>Marinus J</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>rien.blok@mumc.nl</email></contrib><contrib contrib-type="author" id="A8"><name><surname>Keymeulen</surname><given-names>Kristien</given-names></name><xref ref-type="aff" rid="I2">2</xref><xref ref-type="aff" rid="I8">8</xref><email>k.keymeulen@mumc.nl</email></contrib><contrib contrib-type="author" id="A9"><name><surname>Ayoubi</surname><given-names>Torik</given-names></name><xref ref-type="aff" rid="I9">9</xref><email>torik.ayoubi@cme.vib-kuleuven.be</email></contrib><contrib contrib-type="author" id="A10"><name><surname>Smeets</surname><given-names>Hubert JM</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>bert.smeets@maastrichtuniversity.nl</email></contrib><contrib contrib-type="author" id="A11"><name><surname>Tjan-Heijnen</surname><given-names>Vivianne C</given-names></name><xref ref-type="aff" rid="I2">2</xref><xref ref-type="aff" rid="I3">3</xref><email>vcg.tjan.heijnen@mumc.nl</email></contrib><contrib contrib-type="author" id="A12"><name><surname>Hupperets</surname><given-names>Pierre S</given-names></name><xref ref-type="aff" rid="I2">2</xref><xref ref-type="aff" rid="I3">3</xref><email>pierre.hupperets@mumc.nl</email></contrib></contrib-group><aff id="I1"><label>1</label>Department of Clinical Genetics, Unit of Clinical Genomics, Maastricht University Medical Centre+, P. Debyelaan 25, Maastricht, 6229 HX, The Netherlands</aff><aff id="I2"><label>2</label>GROW - School for Oncology and Developmental Biology, Maastricht University Medical Centre+, P. Debyelaan 25, Maastricht, 6202 AZ, The Netherlands</aff><aff id="I3"><label>3</label>Department of Internal Medicine, Division of Medical Oncology, Maastricht University Medical Centre+, P. Debyelaan 25, Maastricht, 6229 HX, The Netherlands</aff><aff id="I4"><label>4</label>Department of Pathology, Maastricht University Medical Centre+, P. Debyelaan 25, Maastricht, 6229 HX, The Netherlands</aff><aff id="I5"><label>5</label>Institute of Pathology, RWTH Aachen University Hospital, Pauwelsstr. 30, Aachen, 52074, Germany</aff><aff id="I6"><label>6</label>Department of Internal Medicine, Division of Haematology, Maastricht University Medical Centre+, P. Debyelaan 25, Maastricht, 6229 HX, The Netherlands</aff><aff id="I7"><label>7</label>Pharmacell BV, Oxfordlaan 70, Maastricht, 6229 EV, The Netherlands</aff><aff id="I8"><label>8</label>Department of Surgery, Maastricht University Medical Centre+, P. Debyelaan 25, Maastricht, 6202 AZ, The Netherlands</aff><aff id="I9"><label>9</label>Saint James School of Medicine, Basic Science and Premedical Faculty, Kralendijk, Bonaire, Netherlands Antilles</aff><pub-date pub-type="ppub"><year>2013</year></pub-date><pub-date pub-type="epub"><day>8</day><month>4</month><year>2013</year></pub-date><volume>15</volume><issue>2</issue><fpage>R29</fpage><lpage>R29</lpage><history><date date-type="received"><day>20</day><month>8</month><year>2012</year></date><date date-type="rev-recd"><day>27</day><month>2</month><year>2013</year></date><date date-type="accepted"><day>3</day><month>4</month><year>2013</year></date></history><permissions><copyright-statement>Copyright © 2013 Brandão et al.; licensee BioMed Central Ltd.</copyright-statement><copyright-year>2013</copyright-year><copyright-holder>Brandão et al.; licensee BioMed Central Ltd.</copyright-holder><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"><license-p>This is an open access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</license-p></license></permissions><self-uri xlink:href="http://breast-cancer-research.com/content/15/2/R29"></self-uri><abstract><sec><title><offsets xml_i="6789" xml_f="6801" txt_i="11" txt_f="23">Introduction</offsets></title><p><offsets xml_i="6812" xml_f="7271" txt_i="24" txt_f="483">Cyclooxygenase-2 (COX-2) is frequently over-expressed in primary breast cancer. In transgenic breast cancer models, over-expression of COX-2 leads to tumour formation while COX-2 inhibition exerts anti-tumour effects in breast cancer cell lines. To further determine the effect of COX-2 inhibition in primary breast cancer, we aimed to identify transcriptional changes in breast cancer tissues of patients treated with the selective COX-2 inhibitor celecoxib.</offsets></p></sec><sec><title><offsets xml_i="7293" xml_f="7300" txt_i="485" txt_f="492">Methods</offsets></title><p><offsets xml_i="7311" xml_f="7498" txt_i="493" txt_f="680">In a single-centre double-blind phase II study, thirty-seven breast cancer patients were randomised to receive either pre-operative celecoxib (400 mg) twice daily for two to three weeks (</offsets><italic><offsets xml_i="7506" xml_f="7508" txt_i="680" txt_f="682">n </offsets></italic><offsets xml_i="7517" xml_f="7568" txt_i="682" txt_f="733">= 22) or a placebo according to the same schedule (</offsets><italic><offsets xml_i="7576" xml_f="7578" txt_i="733" txt_f="735">n </offsets></italic><offsets xml_i="7587" xml_f="8114" txt_i="735" txt_f="1262">= 15). Gene expression in fresh-frozen pre-surgical biopsies (before treatment) and surgical excision specimens (after treatment) was profiled by using Affymetrix arrays. Differentially expressed genes and altered pathways were bioinformatically identified. Expression of selected genes was validated by quantitative PCR (qPCR). Immunohistochemical protein expression analyses of the proliferation marker Ki-67, the apoptosis marker cleaved caspase-3 and the neo-angiogenesis marker CD34 served to evaluate biological response.</offsets></p></sec><sec><title><offsets xml_i="8136" xml_f="8143" txt_i="1264" txt_f="1271">Results</offsets></title><p><offsets xml_i="8154" xml_f="8587" txt_i="1272" txt_f="1705">We identified 972 and 586 significantly up- and down-regulated genes, respectively, in celecoxib-treated specimens. Significant expression changes in six out of eight genes could be validated by qPCR. Pathway analyses revealed over-representation of deregulated genes in the networks of proliferation, cell cycle, extracellular matrix biology, and inflammatory immune response. The Ki-67 mean change relative to baseline was -29.1% (</offsets><italic><offsets xml_i="8595" xml_f="8597" txt_i="1705" txt_f="1707">P </offsets></italic><offsets xml_i="8606" xml_f="8626" txt_i="1707" txt_f="1727">= 0.019) and -8.2% (</offsets><italic><offsets xml_i="8634" xml_f="8636" txt_i="1727" txt_f="1729">P </offsets></italic><offsets xml_i="8645" xml_f="8779" txt_i="1729" txt_f="1863">= 0.384) in the treatment and control arm, respectively. Between treatment groups, the change in Ki-67 was statistically significant (</offsets><italic><offsets xml_i="8787" xml_f="8789" txt_i="1863" txt_f="1865">P </offsets></italic><offsets xml_i="8798" xml_f="8936" txt_i="1865" txt_f="2003">= 0.029). Cleaved caspase-3 and CD34 expression were not significantly different between the celecoxib-treated and placebo-treated groups.</offsets></p></sec><sec><title><offsets xml_i="8958" xml_f="8969" txt_i="2005" txt_f="2016">Conclusions</offsets></title><p><offsets xml_i="8980" xml_f="9344" txt_i="2017" txt_f="2381">Short-term COX-2 inhibition by celecoxib induces transcriptional programs supporting anti-tumour activity in primary breast cancer tissue. The impact on proliferation-associated genes is reflected by a reduction of Ki-67 positive cells. Therefore, COX-2 inhibition should be considered as a treatment strategy for further clinical testing in primary breast cancer.</offsets></p></sec><sec><title><offsets xml_i="9366" xml_f="9384" txt_i="2383" txt_f="2401">Trial registration</offsets></title><p><offsets xml_i="9395" xml_f="9414" txt_i="2402" txt_f="2421">ClinicalTrials.gov </offsets><ext-link ext-link-type="uri" xlink:href="http://www.clinicaltrials.gov/ct2/show/NCT01695226"><offsets xml_i="9508" xml_f="9519" txt_i="2421" txt_f="2432">NCT01695226</offsets></ext-link><offsets xml_i="9530" xml_f="9531" txt_i="2432" txt_f="2433">.</offsets></p></sec></abstract></article-meta></front><body><sec sec-type="intro"><title><offsets xml_i="9610" xml_f="9622" txt_i="2442" txt_f="2454">Introduction</offsets></title><p><offsets xml_i="9633" xml_f="9767" txt_i="2455" txt_f="2589">Cancer development is associated with chronic immune activation, but the mechanisms behind this observation are not fully understood [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="9798" xml_f="9799" txt_i="2589" txt_f="2590">1</offsets></xref><offsets xml_i="9806" xml_f="9807" txt_i="2590" txt_f="2591">,</offsets><xref ref-type="bibr" rid="B2"><offsets xml_i="9838" xml_f="9839" txt_i="2591" txt_f="2592">2</offsets></xref><offsets xml_i="9846" xml_f="10008" txt_i="2592" txt_f="2754">]. In addition, the inflammatory processes that follow tumour formation provide a microenvironment in which the development of malignant disease may be enhanced [</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="10039" xml_f="10040" txt_i="2754" txt_f="2755">3</offsets></xref><offsets xml_i="10047" xml_f="10322" txt_i="2755" txt_f="3030">]. The involvement of chronic immune activation has been supported by several lines of evidence in which an association between non-steroidal anti-inflammatory drug (NSAID) consumption and decreased risk of cancer development, including breast cancer, has been demonstrated [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="10353" xml_f="10354" txt_i="3030" txt_f="3031">4</offsets></xref><offsets xml_i="10361" xml_f="10362" txt_i="3031" txt_f="3032">-</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="10394" xml_f="10396" txt_i="3032" txt_f="3034">11</offsets></xref><offsets xml_i="10403" xml_f="10539" txt_i="3034" txt_f="3170">]. However, the exact mechanisms by which NSAIDs exert inhibiting effects on tumour development have not yet been completely elucidated.</offsets></p><p><offsets xml_i="10546" xml_f="10739" txt_i="3171" txt_f="3364">Mediators of inflammatory responses, such as the cyclooxygenase (COX)-derived prostaglandins (PG), play an important role in tumour formation and provide a target for therapeutic intervention [</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="10770" xml_f="10771" txt_i="3364" txt_f="3365">3</offsets></xref><offsets xml_i="10778" xml_f="10956" txt_i="3365" txt_f="3543">]. PGs have important functions in every organ system and regulate a variety of physiological functions, such as immunity, maintenance of vascular integrity and bone metabolism [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="10988" xml_f="10990" txt_i="3543" txt_f="3545">12</offsets></xref><offsets xml_i="10997" xml_f="11126" txt_i="3545" txt_f="3674">]. Elevated COX expression in breast cancer was first suggested by the finding of elevated PG production in breast cancer cells [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="11158" xml_f="11160" txt_i="3674" txt_f="3676">13</offsets></xref><offsets xml_i="11167" xml_f="11228" txt_i="3676" txt_f="3737">]. To date, two different COX genes have been characterised, </offsets><italic><offsets xml_i="11236" xml_f="11242" txt_i="3737" txt_f="3743">COX-1 </offsets></italic><offsets xml_i="11251" xml_f="11255" txt_i="3743" txt_f="3747">and </offsets><italic><offsets xml_i="11263" xml_f="11276" txt_i="3747" txt_f="3760">COX-2. COX-1 </offsets></italic><offsets xml_i="11285" xml_f="11368" txt_i="3760" txt_f="3843">is constitutively expressed by almost all human cells and, therefore, differs from </offsets><italic><offsets xml_i="11376" xml_f="11382" txt_i="3843" txt_f="3849">COX-2 </offsets></italic><offsets xml_i="11391" xml_f="11510" txt_i="3849" txt_f="3968">expression which is normally absent but is inducible by a wide spectrum of growth factors, pro-inflammatory cytokines [</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="11542" xml_f="11544" txt_i="3968" txt_f="3970">14</offsets></xref><offsets xml_i="11551" xml_f="11552" txt_i="3970" txt_f="3971">,</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="11584" xml_f="11586" txt_i="3971" txt_f="3973">15</offsets></xref><offsets xml_i="11593" xml_f="11627" txt_i="3973" txt_f="4007">] and tumour-promoting compounds [</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="11659" xml_f="11661" txt_i="4007" txt_f="4009">16</offsets></xref><offsets xml_i="11668" xml_f="11669" txt_i="4009" txt_f="4010">,</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="11701" xml_f="11703" txt_i="4010" txt_f="4012">17</offsets></xref><offsets xml_i="11710" xml_f="11727" txt_i="4012" txt_f="4029">]. Consistently, </offsets><italic><offsets xml_i="11735" xml_f="11741" txt_i="4029" txt_f="4035">COX-2 </offsets></italic><offsets xml_i="11750" xml_f="11799" txt_i="4035" txt_f="4084">is abundantly expressed in breast cancer tissue [</offsets><xref ref-type="bibr" rid="B18"><offsets xml_i="11831" xml_f="11833" txt_i="4084" txt_f="4086">18</offsets></xref><offsets xml_i="11840" xml_f="11959" txt_i="4086" txt_f="4205">] and its enforced over-expression in mammary gland epithelia of transgenic mice results in breast tumour development [</offsets><xref ref-type="bibr" rid="B19"><offsets xml_i="11991" xml_f="11993" txt_i="4205" txt_f="4207">19</offsets></xref><offsets xml_i="12000" xml_f="12085" txt_i="4207" txt_f="4292">], suggesting that COX-2 might be an interesting therapeutic target in breast cancer.</offsets></p><p><offsets xml_i="12092" xml_f="12783" txt_i="4293" txt_f="4984">While several pre-clinical studies have indeed shown anti-tumour capacities of COX-2 inhibition, the treatment effects on primary breast cancer in the clinical setting remain elusive. Therefore, we aimed to determine for the first time transcriptional changes in primary breast cancer tissue of women with early breast cancer after treatment with the selective COX-2 inhibitor celecoxib. In our randomised controlled trial we found that upon celecoxib treatment numerous genes are differentially expressed in breast cancer tissues with an overall anti-tumour activity, suggesting that COX-2 inhibition should be further considered for clinical testing as a treatment option in breast cancer.</offsets></p></sec><sec sec-type="methods"><title><offsets xml_i="12824" xml_f="12831" txt_i="4986" txt_f="4993">Methods</offsets></title><sec><title><offsets xml_i="12851" xml_f="12876" txt_i="4994" txt_f="5019">Patients and study design</offsets></title><p><offsets xml_i="12887" xml_f="13571" txt_i="5020" txt_f="5704">The study was a double-blind, randomised, placebo-controlled phase II pre-surgical trial of celecoxib in early breast cancer. Exclusion criteria were: HIV, hepatitis B virus (HBV) or hepatitis C virus (HCV) positivity, known hypersensitivity to NSAIDs, patients already using NSAIDs or systemic use of corticosteroids. Informed consent was obtained prior to entering the trial and the Medical Ethics Committee of the Maastricht University Medical Centre+ (MUMC+) approved the study. We estimated that to test 18,500 genes at the 5% significance level and ensure 80% power, 23 samples were needed to detect differentially expressed genes by t-tests with a fold change of at least 1.5 (</offsets><ext-link ext-link-type="uri" xlink:href="http://bioinformatics.mdanderson.org/MicroarraySampleSize/"><offsets xml_i="13673" xml_f="13731" txt_i="5704" txt_f="5762">http://bioinformatics.mdanderson.org/MicroarraySampleSize/</offsets></ext-link><offsets xml_i="13742" xml_f="13850" txt_i="5762" txt_f="5870">). Initially, 45 patients were recruited between 2005 and 2007 and randomly allocated 2:1 to the treatment (</offsets><italic><offsets xml_i="13858" xml_f="13860" txt_i="5870" txt_f="5872">n </offsets></italic><offsets xml_i="13869" xml_f="13893" txt_i="5872" txt_f="5896">= 30) or placebo group (</offsets><italic><offsets xml_i="13901" xml_f="13903" txt_i="5896" txt_f="5898">n </offsets></italic><offsets xml_i="13912" xml_f="14234" txt_i="5898" txt_f="6220">= 15). Celecoxib was pre-surgically administered for two to three weeks at 400 mg twice daily, whereas patients in the control arm received a placebo on the same schedule. Eight patients allocated to the treatment arm dropped out because these patients were operated earlier, thus drug compliance was insufficient (Figure </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="14264" xml_f="14265" txt_i="6220" txt_f="6221">1</offsets></xref><offsets xml_i="14272" xml_f="14375" txt_i="6221" txt_f="6324">). Tumour histology was assessed according to criteria defined by the World Health Organization (WHO) [</offsets><xref ref-type="bibr" rid="B20"><offsets xml_i="14407" xml_f="14409" txt_i="6324" txt_f="6326">20</offsets></xref><offsets xml_i="14416" xml_f="14520" txt_i="6326" txt_f="6430">], while staging was performed according to the Union for International Cancer Control (UICC) criteria [</offsets><xref ref-type="bibr" rid="B21"><offsets xml_i="14552" xml_f="14554" txt_i="6430" txt_f="6432">21</offsets></xref><offsets xml_i="14561" xml_f="14663" txt_i="6432" txt_f="6534">]. Tumours were graded following the system of Bloom and Richardson, as modified by Elston and Ellis [</offsets><xref ref-type="bibr" rid="B22"><offsets xml_i="14695" xml_f="14697" txt_i="6534" txt_f="6536">22</offsets></xref><offsets xml_i="14704" xml_f="14754" txt_i="6536" txt_f="6586">]. Patient characteristics are described in Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="14786" xml_f="14787" txt_i="6586" txt_f="6587">1</offsets></xref><offsets xml_i="14794" xml_f="15160" txt_i="6587" txt_f="6953">. Importantly, in our study design patients acted as their own control, with a direct comparison of the final surgical specimen with the initial diagnostic biopsy. The inclusion of a placebo group served to observe a possible confounding impact of the disease and the experimental procedure, thereby allowing determination of a differential impact of celecoxib only.</offsets></p><fig id="F1" position="float"><label><offsets xml_i="15201" xml_f="15209" txt_i="6954" txt_f="6962">Figure 1</offsets></label><caption><p><bold><offsets xml_i="15235" xml_f="15270" txt_i="6962" txt_f="6997">Flow diagram of the presented study</offsets></bold><offsets xml_i="15277" xml_f="15570" txt_i="6997" txt_f="7290">. The design is a double-blind, randomised, controlled phase II trial of pre-operative celecoxib versus placebo in early breast cancer. Note that eight patients had discontinued intervention in the treatment arm. Gene expression profiling (GEP) has been performed from samples where indicated.</offsets></p></caption><graphic xlink:href="bcr3409-1"></graphic></fig><table-wrap id="T1" position="float"><label><offsets xml_i="15676" xml_f="15683" txt_i="7291" txt_f="7298">Table 1</offsets></label><caption><p><offsets xml_i="15703" xml_f="15766" txt_i="7298" txt_f="7361">Clinico-pathological characteristics of breast cancer patients.</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left"><offsets xml_i="15845" xml_f="15853" txt_i="7362" txt_f="7370">Variable</offsets></th><th align="left"><offsets xml_i="15875" xml_f="15889" txt_i="7370" txt_f="7384">Categorisation</offsets></th><th align="center"><offsets xml_i="15913" xml_f="15920" txt_i="7384" txt_f="7391">Control</offsets></th><th align="center"><offsets xml_i="15944" xml_f="15953" txt_i="7391" txt_f="7400">Celecoxib</offsets></th></tr></thead><tbody><tr><td></td><td></td><td align="center"><bold><offsets xml_i="16025" xml_f="16036" txt_i="7402" txt_f="7413">Number = 15</offsets></bold></td><td align="center"><bold><offsets xml_i="16073" xml_f="16084" txt_i="7414" txt_f="7425">Number = 22</offsets></bold></td></tr><tr><td colspan="4"><hr></hr></td></tr><tr><td align="left" colspan="2"><offsets xml_i="16173" xml_f="16194" txt_i="7427" txt_f="7448">Mean age ± SD (years)</offsets><sup><offsets xml_i="16199" xml_f="16200" txt_i="7448" txt_f="7449">a</offsets></sup></td><td align="center"><offsets xml_i="16230" xml_f="16237" txt_i="7450" txt_f="7457">57 ± 12</offsets></td><td align="center"><offsets xml_i="16261" xml_f="16268" txt_i="7458" txt_f="7465">51 ± 12</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="16311" xml_f="16335" txt_i="7466" txt_f="7490">Histological tumour type</offsets><sup><offsets xml_i="16340" xml_f="16341" txt_i="7490" txt_f="7491">b</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="16387" xml_f="16412" txt_i="7493" txt_f="7518">Invasive ductal carcinoma</offsets></td><td align="center"><offsets xml_i="16436" xml_f="16444" txt_i="7519" txt_f="7527">13 (87%)</offsets></td><td align="center"><offsets xml_i="16468" xml_f="16476" txt_i="7528" txt_f="7536">19 (86%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="16516" xml_f="16542" txt_i="7538" txt_f="7564">Invasive lobular carcinoma</offsets></td><td align="center"><offsets xml_i="16566" xml_f="16573" txt_i="7565" txt_f="7572">2 (13%)</offsets></td><td align="center"><offsets xml_i="16597" xml_f="16604" txt_i="7573" txt_f="7580">3 (14%)</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="16647" xml_f="16658" txt_i="7581" txt_f="7592">Tumour size</offsets><sup><offsets xml_i="16663" xml_f="16664" txt_i="7592" txt_f="7593">c</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="16710" xml_f="16713" txt_i="7595" txt_f="7598">pT1</offsets></td><td align="center"><offsets xml_i="16737" xml_f="16745" txt_i="7599" txt_f="7607">10 (67%)</offsets></td><td align="center"><offsets xml_i="16769" xml_f="16776" txt_i="7608" txt_f="7615">8 (36%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="16816" xml_f="16819" txt_i="7617" txt_f="7620">pT2</offsets></td><td align="center"><offsets xml_i="16843" xml_f="16850" txt_i="7621" txt_f="7628">5 (33%)</offsets></td><td align="center"><offsets xml_i="16874" xml_f="16882" txt_i="7629" txt_f="7637">13 (59%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="16922" xml_f="16925" txt_i="7639" txt_f="7642">pT3</offsets></td><td align="center"><offsets xml_i="16949" xml_f="16955" txt_i="7643" txt_f="7649">0 (0%)</offsets></td><td align="center"><offsets xml_i="16979" xml_f="16985" txt_i="7650" txt_f="7656">1 (5%)</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="17028" xml_f="17045" txt_i="7657" txt_f="7674">Lymph node status</offsets><sup><offsets xml_i="17050" xml_f="17051" txt_i="7674" txt_f="7675">c</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="17097" xml_f="17111" txt_i="7677" txt_f="7691">Negative (pN0)</offsets></td><td align="center"><offsets xml_i="17135" xml_f="17143" txt_i="7692" txt_f="7700">11 (73%)</offsets></td><td align="center"><offsets xml_i="17167" xml_f="17175" txt_i="7701" txt_f="7709">10 (45%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="17215" xml_f="17231" txt_i="7711" txt_f="7727">Positive (pN1-3)</offsets></td><td align="center"><offsets xml_i="17255" xml_f="17262" txt_i="7728" txt_f="7735">4 (27%)</offsets></td><td align="center"><offsets xml_i="17286" xml_f="17294" txt_i="7736" txt_f="7744">12 (55%)</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="17337" xml_f="17349" txt_i="7745" txt_f="7757">Tumour grade</offsets><sup><offsets xml_i="17354" xml_f="17355" txt_i="7757" txt_f="7758">d</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="17401" xml_f="17403" txt_i="7760" txt_f="7762">G1</offsets></td><td align="center"><offsets xml_i="17427" xml_f="17434" txt_i="7763" txt_f="7770">5 (33%)</offsets></td><td align="center"><offsets xml_i="17458" xml_f="17464" txt_i="7771" txt_f="7777">1 (5%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="17504" xml_f="17506" txt_i="7779" txt_f="7781">G2</offsets></td><td align="center"><offsets xml_i="17530" xml_f="17537" txt_i="7782" txt_f="7789">4 (27%)</offsets></td><td align="center"><offsets xml_i="17561" xml_f="17568" txt_i="7790" txt_f="7797">7 (32%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="17608" xml_f="17610" txt_i="7799" txt_f="7801">G3</offsets></td><td align="center"><offsets xml_i="17634" xml_f="17641" txt_i="7802" txt_f="7809">6 (40%)</offsets></td><td align="center"><offsets xml_i="17665" xml_f="17673" txt_i="7810" txt_f="7818">14 (63%)</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="17716" xml_f="17741" txt_i="7819" txt_f="7844">Oestrogen receptor status</offsets><sup><offsets xml_i="17746" xml_f="17747" txt_i="7844" txt_f="7845">e</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="17793" xml_f="17804" txt_i="7847" txt_f="7858">ER positive</offsets></td><td align="center"><offsets xml_i="17828" xml_f="17836" txt_i="7859" txt_f="7867">11 (73%)</offsets></td><td align="center"><offsets xml_i="17860" xml_f="17868" txt_i="7868" txt_f="7876">19 (86%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="17908" xml_f="17919" txt_i="7878" txt_f="7889">ER negative</offsets></td><td align="center"><offsets xml_i="17943" xml_f="17950" txt_i="7890" txt_f="7897">4 (27%)</offsets></td><td align="center"><offsets xml_i="17974" xml_f="17981" txt_i="7898" txt_f="7905">3 (14%)</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="18024" xml_f="18052" txt_i="7906" txt_f="7934">Progesterone receptor status</offsets><sup><offsets xml_i="18057" xml_f="18058" txt_i="7934" txt_f="7935">e</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="18104" xml_f="18116" txt_i="7937" txt_f="7949">PgR positive</offsets></td><td align="center"><offsets xml_i="18140" xml_f="18148" txt_i="7950" txt_f="7958">10 (67%)</offsets></td><td align="center"><offsets xml_i="18172" xml_f="18180" txt_i="7959" txt_f="7967">17 (77%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="18220" xml_f="18232" txt_i="7969" txt_f="7981">PgR negative</offsets></td><td align="center"><offsets xml_i="18256" xml_f="18263" txt_i="7982" txt_f="7989">5 (33%)</offsets></td><td align="center"><offsets xml_i="18287" xml_f="18294" txt_i="7990" txt_f="7997">5 (23%)</offsets></td></tr><tr><td align="left" colspan="4"><offsets xml_i="18337" xml_f="18348" txt_i="7998" txt_f="8009">HER2 status</offsets><sup><offsets xml_i="18353" xml_f="18354" txt_i="8009" txt_f="8010">f</offsets></sup></td></tr><tr><td></td><td align="left"><offsets xml_i="18400" xml_f="18413" txt_i="8012" txt_f="8025">HER2 positive</offsets></td><td align="center"><offsets xml_i="18437" xml_f="18444" txt_i="8026" txt_f="8033">3 (20%)</offsets></td><td align="center"><offsets xml_i="18468" xml_f="18475" txt_i="8034" txt_f="8041">3 (14%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="18515" xml_f="18528" txt_i="8043" txt_f="8056">HER2 negative</offsets></td><td align="center"><offsets xml_i="18552" xml_f="18560" txt_i="8057" txt_f="8065">12 (80%)</offsets></td><td align="center"><offsets xml_i="18584" xml_f="18592" txt_i="8066" txt_f="8074">18 (82%)</offsets></td></tr><tr><td></td><td align="left"><offsets xml_i="18632" xml_f="18639" txt_i="8076" txt_f="8083">Unknown</offsets></td><td align="center"><offsets xml_i="18663" xml_f="18669" txt_i="8084" txt_f="8090">0 (0%)</offsets></td><td align="center"><offsets xml_i="18693" xml_f="18699" txt_i="8091" txt_f="8097">1 (5%)</offsets></td></tr></tbody></table><table-wrap-foot><p><sup><offsets xml_i="18750" xml_f="18751" txt_i="8098" txt_f="8099">a</offsets></sup><offsets xml_i="18757" xml_f="18779" txt_i="8099" txt_f="8121">At time of diagnosis; </offsets><sup><offsets xml_i="18784" xml_f="18785" txt_i="8121" txt_f="8122">b</offsets></sup><offsets xml_i="18791" xml_f="18827" txt_i="8122" txt_f="8158">according to Tavassoli and Devilee [</offsets><xref ref-type="bibr" rid="B20"><offsets xml_i="18859" xml_f="18861" txt_i="8158" txt_f="8160">20</offsets></xref><offsets xml_i="18868" xml_f="18871" txt_i="8160" txt_f="8163">]; </offsets><sup><offsets xml_i="18876" xml_f="18877" txt_i="8163" txt_f="8164">c</offsets></sup><offsets xml_i="18883" xml_f="18917" txt_i="8164" txt_f="8198">according to Sobin and Wittekind [</offsets><xref ref-type="bibr" rid="B21"><offsets xml_i="18949" xml_f="18951" txt_i="8198" txt_f="8200">21</offsets></xref><offsets xml_i="18958" xml_f="18961" txt_i="8200" txt_f="8203">]; </offsets><sup><offsets xml_i="18966" xml_f="18967" txt_i="8203" txt_f="8204">d</offsets></sup><offsets xml_i="18973" xml_f="19004" txt_i="8204" txt_f="8235">according to Elston and Ellis [</offsets><xref ref-type="bibr" rid="B22"><offsets xml_i="19036" xml_f="19038" txt_i="8235" txt_f="8237">22</offsets></xref><offsets xml_i="19045" xml_f="19048" txt_i="8237" txt_f="8240">]; </offsets><sup><offsets xml_i="19053" xml_f="19054" txt_i="8240" txt_f="8241">e</offsets></sup><offsets xml_i="19060" xml_f="19143" txt_i="8241" txt_f="8321">positivity defined as &gt; 10% positive cells as analysed by immunohistochemistry [</offsets><xref ref-type="bibr" rid="B60"><offsets xml_i="19175" xml_f="19177" txt_i="8321" txt_f="8323">60</offsets></xref><offsets xml_i="19184" xml_f="19187" txt_i="8323" txt_f="8326">]; </offsets><sup><offsets xml_i="19192" xml_f="19193" txt_i="8326" txt_f="8327">f</offsets></sup><offsets xml_i="19199" xml_f="19346" txt_i="8327" txt_f="8474">positivity defined by immunohistochemistry (score 3+) or by dual-color FISH. Percentages may not sum-up to 100 due to rounding. FISH, fluorescence </offsets><italic><offsets xml_i="19354" xml_f="19362" txt_i="8474" txt_f="8482">in situ </offsets></italic><offsets xml_i="19371" xml_f="19385" txt_i="8482" txt_f="8496">hybridisation.</offsets></p></table-wrap-foot></table-wrap></sec><sec><title><offsets xml_i="19438" xml_f="19451" txt_i="8498" txt_f="8511">Biopsy method</offsets></title><p><offsets xml_i="19462" xml_f="20085" txt_i="8512" txt_f="9135">Before patient allocation to the groups, two to three core needle biopsies from the centre of the primary tumour were obtained using a 14-gauge needle under ultrasound guidance. At surgery, a central sample of the excised tumour was obtained. One part of the biopsies and surgical excision specimens was snap-frozen in liquid nitrogen within 30 minutes after removal and stored at -80°C until use, while the remaining part was formalin-fixed and paraffin-embedded (FFPE). All specimens subjected to gene expression analyses had a tumour cellularity of at least 80%, as determined by haematoxylin and eosin stained sections.</offsets></p></sec><sec><title><offsets xml_i="20107" xml_f="20189" txt_i="9137" txt_f="9219">RNA isolation, cRNA production and fragmentation, array hybridisation and scanning</offsets></title><p><offsets xml_i="20200" xml_f="21193" txt_i="9220" txt_f="10213">After homogenisation of fresh-frozen tissue specimens, total RNA was isolated using TRIzol reagent (Invitrogen, Carlsbad, CA, USA) according to the manufacturer's protocol. DNase treatment was performed with the RNase-Free DNase Set (Qiagen, Hilden, Germany) followed by purification of the RNA samples using the RNeasy Minikit (Qiagen). RNA quantity and purity were determined spectrophotometrically using the Nanodrop ND-1000 (Nanodrop Technologies, Wilmington, DE, USA) and RNA integrity was assessed by determining the RNA 28S/18S ratio using the Bioanalyzer 2100 (Agilent Technologies, Santa Clara, CA, USA). Biotinylated aRNA was synthesised and fragmented using the GeneChip IVT Express Kit from Affymetrix (Santa Clara, CA, USA). Hybridisation to Affymetrix Human Genome U133 Plus 2.0 arrays and subsequent scanning was performed following the manufacturer's guidelines using the GeneChip scanner 3000 (Affymetrix). Microarray datasets are publicly available at ArrayExpress database [</offsets><xref ref-type="bibr" rid="B23"><offsets xml_i="21225" xml_f="21227" txt_i="10213" txt_f="10215">23</offsets></xref><offsets xml_i="21234" xml_f="21270" txt_i="10215" txt_f="10251">] under accession number E-MTAB-566.</offsets></p></sec><sec><title><offsets xml_i="21292" xml_f="21316" txt_i="10253" txt_f="10277">Microarray data analysis</offsets></title><p><offsets xml_i="21327" xml_f="21549" txt_i="10278" txt_f="10500">Images of the arrays were quantified with GCOS software (Affymetrix). The chip description file (CDF) used for the analysis was an update created and freely distributed by the microarray lab of the University of Michigan [</offsets><xref ref-type="bibr" rid="B24"><offsets xml_i="21581" xml_f="21583" txt_i="10500" txt_f="10502">24</offsets></xref><offsets xml_i="21590" xml_f="22057" txt_i="10502" txt_f="10969">] based on Ensembl (version 10). The treatment effects on global gene expression were assessed by microarray analyses of subjects after celecoxib treatment versus subjects before treatment and controls (before and after treatment). Thus, expression changes have been adjusted for differences in the pre- and post-treatment specimens of the control group. The genes were analysed using a Gaussian linear regression including the hybridisation and labelling spikes and </offsets><italic><offsets xml_i="22065" xml_f="22070" txt_i="10969" txt_f="10974">COX-2</offsets></italic><offsets xml_i="22079" xml_f="22430" txt_i="10974" txt_f="11325">. The inference criterion used for comparing the models is their ability to predict the observed data, that is, models are compared directly through their minimised minus log-likelihood. When the numbers of parameters in models differ, they are penalised by adding the number of estimated parameters, a form of the Akaike information criterion (AIC) [</offsets><xref ref-type="bibr" rid="B25"><offsets xml_i="22462" xml_f="22464" txt_i="11325" txt_f="11327">25</offsets></xref><offsets xml_i="22471" xml_f="22788" txt_i="11327" txt_f="11644">]. For each gene, the group effect only after intervention was then added to the model. The gene under consideration was found to be differentially expressed if the AIC decreased compared to the model not containing this effect. All statistical analyses presented were performed using the freely available program R [</offsets><xref ref-type="bibr" rid="B26"><offsets xml_i="22820" xml_f="22822" txt_i="11644" txt_f="11646">26</offsets></xref><offsets xml_i="22829" xml_f="22876" txt_i="11646" txt_f="11693">] and the publicly available library 'growth' [</offsets><xref ref-type="bibr" rid="B27"><offsets xml_i="22908" xml_f="22910" txt_i="11693" txt_f="11695">27</offsets></xref><offsets xml_i="22917" xml_f="22919" txt_i="11695" txt_f="11697">].</offsets></p></sec><sec><title><offsets xml_i="22941" xml_f="22989" txt_i="11699" txt_f="11747">Validation of microarrays by quantitative RT-PCR</offsets></title><p><offsets xml_i="23000" xml_f="23187" txt_i="11748" txt_f="11935">Quantitative RT-PCR (qPCR) was performed for selected genes on all 74 samples in order to validate the results obtained in the microarray study. As a reference, TATA box binding protein (</offsets><italic><offsets xml_i="23195" xml_f="23198" txt_i="11935" txt_f="11938">TBP</offsets></italic><offsets xml_i="23207" xml_f="23235" txt_i="11938" txt_f="11966">) and RNA, 18S ribosomal 1 (</offsets><italic><offsets xml_i="23243" xml_f="23249" txt_i="11966" txt_f="11972">RN18S1</offsets></italic><offsets xml_i="23258" xml_f="23379" txt_i="11972" txt_f="12093">) were included as housekeeping genes. The list of genes and the primers that were used are described in Additional file </offsets><xref ref-type="supplementary-material" rid="S1"><offsets xml_i="23428" xml_f="23429" txt_i="12093" txt_f="12094">1</offsets></xref><offsets xml_i="23436" xml_f="23919" txt_i="12094" txt_f="12577">, Table S1. Excess biotinylated aRNA (before the fragmentation step) was used for validation, after cDNA synthesis with M-MuLV Reverse Transcriptase (Finnzymes, Espoo, Finland) and oligo(dT), (Invitrogen) using the SensiMix SYBR Kit (Quantace, London, UK) following the manufacturer's protocol. qPCRs were run on the 7900HT system (Applied Biosystems, Singapore). Results were analysed using a Gaussian linear regression similar to microarray data. Expression of housekeeping genes (</offsets><italic><offsets xml_i="23927" xml_f="23931" txt_i="12577" txt_f="12581">TBP </offsets></italic><offsets xml_i="23940" xml_f="23944" txt_i="12581" txt_f="12585">and </offsets><italic><offsets xml_i="23952" xml_f="23957" txt_i="12585" txt_f="12590">GAPDH</offsets></italic><offsets xml_i="23966" xml_f="23972" txt_i="12590" txt_f="12596">) and </offsets><italic><offsets xml_i="23980" xml_f="23986" txt_i="12596" txt_f="12602">COX-2 </offsets></italic><offsets xml_i="23995" xml_f="24137" txt_i="12602" txt_f="12744">were included during the analysis. The AIC was used to assess whether there was a difference between the controls and patients (group effect).</offsets></p></sec><sec><title><offsets xml_i="24159" xml_f="24192" txt_i="12746" txt_f="12779">Analysis of functional categories</offsets></title><p><offsets xml_i="24203" xml_f="24455" txt_i="12780" txt_f="13032">Significantly altered genes that were found to have a fold-change difference of at least 10% were classified into categories of biological processes and molecular functions using DAVID (Database for Annotation, Visualisation and Integrated Discovery) [</offsets><xref ref-type="bibr" rid="B28"><offsets xml_i="24487" xml_f="24489" txt_i="13032" txt_f="13034">28</offsets></xref><offsets xml_i="24496" xml_f="24513" txt_i="13034" txt_f="13051">] and PathVisio [</offsets><xref ref-type="bibr" rid="B29"><offsets xml_i="24545" xml_f="24547" txt_i="13051" txt_f="13053">29</offsets></xref><offsets xml_i="24554" xml_f="24756" txt_i="13053" txt_f="13255">]. DAVID and PathVisio analyses for pathway enrichment were performed first for all significantly altered genes, then for up- and down-regulated genes separately. DAVID results are listed together with </offsets><italic><offsets xml_i="24764" xml_f="24765" txt_i="13255" txt_f="13256">P</offsets></italic><offsets xml_i="24774" xml_f="24885" txt_i="13256" txt_f="13367">-values corrected with the Bonferroni method, as this was the most conservative method used by the software. A </offsets><italic><offsets xml_i="24893" xml_f="24894" txt_i="13367" txt_f="13368">P</offsets></italic><offsets xml_i="24903" xml_f="25248" txt_i="13368" txt_f="13713">-value of 0.05 was used as a cut-off value. In both analyses we filtered the pathways or gene ontology (GO) terms for which less than five and more than 150 genes were found, as those were either too specific or too general for our analysis. In PathVisio, the gene database Hs_Derby_20110601.bridge and the pathway collection from WikiPathways [</offsets><xref ref-type="bibr" rid="B30"><offsets xml_i="25280" xml_f="25282" txt_i="13713" txt_f="13715">30</offsets></xref><offsets xml_i="25289" xml_f="25407" txt_i="13715" txt_f="13833">] were used to obtain a ranked list of pathways with differentially expressed genes. PathVisio results were sorted by </offsets><italic><offsets xml_i="25415" xml_f="25416" txt_i="13833" txt_f="13834">Z</offsets></italic><offsets xml_i="25425" xml_f="25532" txt_i="13834" txt_f="13941">-score, which is the standard statistical test under the hypergeometric distribution. Only pathways with a </offsets><italic><offsets xml_i="25540" xml_f="25541" txt_i="13941" txt_f="13942">Z</offsets></italic><offsets xml_i="25550" xml_f="25587" txt_i="13942" txt_f="13979">-score above 3, which corresponds to </offsets><italic><offsets xml_i="25595" xml_f="25596" txt_i="13979" txt_f="13980">P</offsets></italic><offsets xml_i="25605" xml_f="25647" txt_i="13980" txt_f="14022">-values of 0.0013 or lower, were selected.</offsets></p></sec><sec><title><offsets xml_i="25669" xml_f="25693" txt_i="14024" txt_f="14048">Tissue marker expression</offsets></title><p><offsets xml_i="25704" xml_f="25865" txt_i="14049" txt_f="14210">Biomarkers of response were assessed by immunohistochemistry on FFPE tissues. Cell proliferation was assessed using the MIB1 mouse monoclonal antibody to Ki-67 [</offsets><xref ref-type="bibr" rid="B31"><offsets xml_i="25897" xml_f="25899" txt_i="14210" txt_f="14212">31</offsets></xref><offsets xml_i="25906" xml_f="25988" txt_i="14212" txt_f="14294">]. Measurement of apoptosis was performed by cleaved caspase-3 (Asp175) staining [</offsets><xref ref-type="bibr" rid="B32"><offsets xml_i="26020" xml_f="26022" txt_i="14294" txt_f="14296">32</offsets></xref><offsets xml_i="26029" xml_f="26348" txt_i="14296" txt_f="14609">]. Tumour cells/section were counted for Ki-67 (1,000 cells) and apoptotic index (3,000 cells). The apoptotic index was expressed as a percentage of the number of cells displaying apoptotic bodies, scoring 0 if &lt; 0.5%, +1 if between 0.5% and 2%, and +3 if &gt; 2%. Samples were also assessed for expression of CD34 [</offsets><xref ref-type="bibr" rid="B33"><offsets xml_i="26380" xml_f="26382" txt_i="14609" txt_f="14611">33</offsets></xref><offsets xml_i="26389" xml_f="26418" txt_i="14611" txt_f="14640">] using the Chalkley method [</offsets><xref ref-type="bibr" rid="B34"><offsets xml_i="26450" xml_f="26452" txt_i="14640" txt_f="14642">34</offsets></xref><offsets xml_i="26459" xml_f="26461" txt_i="14642" txt_f="14644">].</offsets></p></sec><sec><title><offsets xml_i="26483" xml_f="26503" txt_i="14646" txt_f="14666">Statistical analyses</offsets></title><p><offsets xml_i="26514" xml_f="27044" txt_i="14667" txt_f="15197">Ki-67 values at baseline and time of surgery were expressed as geometric mean proportions of the baseline and transformed into percentage changes. Ki-67 changes within groups were compared using a paired t-test, differences in Ki-67 and CD34 between groups by using an unpaired t-test, and the Mann-Whitney test was used for group comparisons. Differences in caspase-3 expression were assessed by the Freeman-Halton extension of the Fisher's exact probability test. All tests were performed two-sided at the 5% significance level.</offsets></p></sec></sec><sec sec-type="results"><title><offsets xml_i="27091" xml_f="27098" txt_i="15200" txt_f="15207">Results</offsets></title><sec><title><offsets xml_i="27118" xml_f="27188" txt_i="15208" txt_f="15278">Celecoxib treatment changes gene expression in breast carcinoma tissue</offsets></title><p><offsets xml_i="27199" xml_f="27539" txt_i="15279" txt_f="15619">Two to three weeks of celecoxib treatment significantly altered the expression of 1,558 genes in breast cancer tissues, of which 972 genes were up- and 586 genes were down-regulated after treatment and adjustment to control tissue gene expression. The 50 most strongly up-regulated and down-regulated genes are presented in Additional file </offsets><xref ref-type="supplementary-material" rid="S2"><offsets xml_i="27588" xml_f="27589" txt_i="15619" txt_f="15620">2</offsets></xref><offsets xml_i="27596" xml_f="27627" txt_i="15620" txt_f="15651">, Table S2 and Additional file </offsets><xref ref-type="supplementary-material" rid="S3"><offsets xml_i="27676" xml_f="27677" txt_i="15651" txt_f="15652">3</offsets></xref><offsets xml_i="27684" xml_f="27815" txt_i="15652" txt_f="15783">, Table S3, respectively. For all of the selected genes, except two, significant expression changes were confirmed by qPCR (Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="27845" xml_f="27846" txt_i="15783" txt_f="15784">2</offsets></xref><offsets xml_i="27853" xml_f="27855" txt_i="15784" txt_f="15786">).</offsets></p><fig id="F2" position="float"><label><offsets xml_i="27896" xml_f="27904" txt_i="15787" txt_f="15795">Figure 2</offsets></label><caption><p><bold><offsets xml_i="27930" xml_f="28054" txt_i="15795" txt_f="15919">qPCR validation of selected genes differentially expressed in celecoxib-treated samples as determined by microarray analysis</offsets></bold><offsets xml_i="28061" xml_f="28253" txt_i="15919" txt_f="16111">. Fold-change and the 95% CI (error bars) are shown. Expression of six out of eight genes analysed (indicated by asterisks) was significantly changed in agreement with the microarray analysis.</offsets></p></caption><graphic xlink:href="bcr3409-2"></graphic></fig><p><offsets xml_i="28318" xml_f="28463" txt_i="16112" txt_f="16257">The list of the biological pathways, in which significantly changed genes were over-represented, identified by DAVID analysis, is shown in Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="28495" xml_f="28496" txt_i="16257" txt_f="16258">2</offsets></xref><offsets xml_i="28503" xml_f="28569" txt_i="16258" txt_f="16324">. Biological processes identified by PathVisio are shown in Table </offsets><xref ref-type="table" rid="T3"><offsets xml_i="28601" xml_f="28602" txt_i="16324" txt_f="16325">3</offsets></xref><offsets xml_i="28609" xml_f="29089" txt_i="16325" txt_f="16805">. In both approaches down-regulated genes were consistently over-represented in cell cycle-related processes and proliferation. Up-regulated genes were over-represented in extracellular matrix (ECM) organisation, cell adhesion, and blood vessel development in DAVID analyses, whereas PathVisio results suggested an implication of further tumour suppressive pathways, for example, complement activation, senescence and autophagy, and transforming growth factor-β (TGF-β) signaling.</offsets></p><table-wrap id="T2" position="float"><label><offsets xml_i="29137" xml_f="29144" txt_i="16806" txt_f="16813">Table 2</offsets></label><caption><p><offsets xml_i="29164" xml_f="29280" txt_i="16813" txt_f="16929">Over-represented functional categories within genes up- and/or down-regulated after celecoxib treatment using DAVID.</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left"><offsets xml_i="29359" xml_f="29364" txt_i="16930" txt_f="16935">GO-ID</offsets></th><th align="left"><offsets xml_i="29386" xml_f="29390" txt_i="16935" txt_f="16939">Term</offsets></th><th align="center"><offsets xml_i="29414" xml_f="29419" txt_i="16939" txt_f="16944">Count</offsets></th><th align="center"><offsets xml_i="29443" xml_f="29458" txt_i="16944" txt_f="16959">Fold enrichment</offsets></th><th align="center"><bold><italic><offsets xml_i="29496" xml_f="29498" txt_i="16959" txt_f="16961">P-</offsets></italic><offsets xml_i="29507" xml_f="29512" txt_i="16961" txt_f="16966">value</offsets></bold><sup><offsets xml_i="29524" xml_f="29525" txt_i="16966" txt_f="16967">a</offsets></sup></th></tr></thead><tbody><tr><td align="left" colspan="5"><bold><offsets xml_i="29595" xml_f="29607" txt_i="16967" txt_f="16979">Up-regulated</offsets></bold></td></tr><tr><td colspan="5"><hr></hr></td></tr><tr><td align="left"><offsets xml_i="29684" xml_f="29691" txt_i="16981" txt_f="16988">0030198</offsets></td><td align="left"><offsets xml_i="29713" xml_f="29746" txt_i="16989" txt_f="17022">Extracellular matrix organisation</offsets></td><td align="center"><offsets xml_i="29770" xml_f="29772" txt_i="17023" txt_f="17025">28</offsets></td><td align="center"><offsets xml_i="29796" xml_f="29801" txt_i="17026" txt_f="17031">4.657</offsets></td><td align="center"><offsets xml_i="29825" xml_f="29833" txt_i="17032" txt_f="17040">6.35E-08</offsets></td></tr><tr><td align="left"><offsets xml_i="29864" xml_f="29871" txt_i="17041" txt_f="17048">0009611</offsets></td><td align="left"><offsets xml_i="29893" xml_f="29913" txt_i="17049" txt_f="17069">Response to wounding</offsets></td><td align="center"><offsets xml_i="29937" xml_f="29939" txt_i="17070" txt_f="17072">70</offsets></td><td align="center"><offsets xml_i="29963" xml_f="29968" txt_i="17073" txt_f="17078">2.337</offsets></td><td align="center"><offsets xml_i="29992" xml_f="30000" txt_i="17079" txt_f="17087">1.32E-07</offsets></td></tr><tr><td align="left"><offsets xml_i="30031" xml_f="30038" txt_i="17088" txt_f="17095">0007155</offsets></td><td align="left"><offsets xml_i="30060" xml_f="30073" txt_i="17096" txt_f="17109">Cell adhesion</offsets></td><td align="center"><offsets xml_i="30097" xml_f="30099" txt_i="17110" txt_f="17112">83</offsets></td><td align="center"><offsets xml_i="30123" xml_f="30128" txt_i="17113" txt_f="17118">2.108</offsets></td><td align="center"><offsets xml_i="30152" xml_f="30160" txt_i="17119" txt_f="17127">3.24E-07</offsets></td></tr><tr><td align="left"><offsets xml_i="30191" xml_f="30198" txt_i="17128" txt_f="17135">0022610</offsets></td><td align="left"><offsets xml_i="30220" xml_f="30239" txt_i="17136" txt_f="17155">Biological adhesion</offsets></td><td align="center"><offsets xml_i="30263" xml_f="30265" txt_i="17156" txt_f="17158">83</offsets></td><td align="center"><offsets xml_i="30289" xml_f="30294" txt_i="17159" txt_f="17164">2.105</offsets></td><td align="center"><offsets xml_i="30318" xml_f="30326" txt_i="17165" txt_f="17173">3.46E-07</offsets></td></tr><tr><td align="left"><offsets xml_i="30357" xml_f="30364" txt_i="17174" txt_f="17181">0001944</offsets></td><td align="left"><offsets xml_i="30386" xml_f="30409" txt_i="17182" txt_f="17205">Vasculature development</offsets></td><td align="center"><offsets xml_i="30433" xml_f="30435" txt_i="17206" txt_f="17208">43</offsets></td><td align="center"><offsets xml_i="30459" xml_f="30464" txt_i="17209" txt_f="17214">2.951</offsets></td><td align="center"><offsets xml_i="30488" xml_f="30496" txt_i="17215" txt_f="17223">1.20E-06</offsets></td></tr><tr><td align="left"><offsets xml_i="30527" xml_f="30534" txt_i="17224" txt_f="17231">0001568</offsets></td><td align="left"><offsets xml_i="30556" xml_f="30580" txt_i="17232" txt_f="17256">Blood vessel development</offsets></td><td align="center"><offsets xml_i="30604" xml_f="30606" txt_i="17257" txt_f="17259">42</offsets></td><td align="center"><offsets xml_i="30630" xml_f="30635" txt_i="17260" txt_f="17265">2.953</offsets></td><td align="center"><offsets xml_i="30659" xml_f="30667" txt_i="17266" txt_f="17274">1.96E-06</offsets></td></tr><tr><td align="left"><offsets xml_i="30698" xml_f="30705" txt_i="17275" txt_f="17282">0043062</offsets></td><td align="left"><offsets xml_i="30727" xml_f="30763" txt_i="17283" txt_f="17319">Extracellular structure organisation</offsets></td><td align="center"><offsets xml_i="30787" xml_f="30789" txt_i="17320" txt_f="17322">33</offsets></td><td align="center"><offsets xml_i="30813" xml_f="30818" txt_i="17323" txt_f="17328">3.502</offsets></td><td align="center"><offsets xml_i="30842" xml_f="30850" txt_i="17329" txt_f="17337">2.35E-06</offsets></td></tr><tr><td align="left"><offsets xml_i="30881" xml_f="30888" txt_i="17338" txt_f="17345">0051270</offsets></td><td align="left"><offsets xml_i="30910" xml_f="30935" txt_i="17346" txt_f="17371">Regulation of cell motion</offsets></td><td align="center"><offsets xml_i="30959" xml_f="30961" txt_i="17372" txt_f="17374">34</offsets></td><td align="center"><offsets xml_i="30985" xml_f="30990" txt_i="17375" txt_f="17380">3.047</offsets></td><td align="center"><offsets xml_i="31014" xml_f="31022" txt_i="17381" txt_f="17389">4.89E-05</offsets></td></tr><tr><td align="left"><offsets xml_i="31053" xml_f="31060" txt_i="17390" txt_f="17397">0001501</offsets></td><td align="left"><offsets xml_i="31082" xml_f="31109" txt_i="17398" txt_f="17425">Skeletal system development</offsets></td><td align="center"><offsets xml_i="31133" xml_f="31135" txt_i="17426" txt_f="17428">46</offsets></td><td align="center"><offsets xml_i="31159" xml_f="31164" txt_i="17429" txt_f="17434">2.510</offsets></td><td align="center"><offsets xml_i="31188" xml_f="31196" txt_i="17435" txt_f="17443">5.23E-05</offsets></td></tr><tr><td align="left"><offsets xml_i="31227" xml_f="31234" txt_i="17444" txt_f="17451">0030334</offsets></td><td align="left"><offsets xml_i="31256" xml_f="31284" txt_i="17452" txt_f="17480">Regulation of cell migration</offsets></td><td align="center"><offsets xml_i="31308" xml_f="31310" txt_i="17481" txt_f="17483">31</offsets></td><td align="center"><offsets xml_i="31334" xml_f="31339" txt_i="17484" txt_f="17489">3.173</offsets></td><td align="center"><offsets xml_i="31363" xml_f="31371" txt_i="17490" txt_f="17498">9.15E-05</offsets></td></tr><tr><td align="left"><offsets xml_i="31402" xml_f="31409" txt_i="17499" txt_f="17506">0040012</offsets></td><td align="left"><offsets xml_i="31431" xml_f="31455" txt_i="17507" txt_f="17531">Regulation of locomotion</offsets></td><td align="center"><offsets xml_i="31479" xml_f="31481" txt_i="17532" txt_f="17534">33</offsets></td><td align="center"><offsets xml_i="31505" xml_f="31510" txt_i="17535" txt_f="17540">2.973</offsets></td><td align="center"><offsets xml_i="31534" xml_f="31542" txt_i="17541" txt_f="17549">1.51E-04</offsets></td></tr><tr><td align="left"><offsets xml_i="31573" xml_f="31580" txt_i="17550" txt_f="17557">0042127</offsets></td><td align="left"><offsets xml_i="31602" xml_f="31634" txt_i="17558" txt_f="17590">Regulation of cell proliferation</offsets></td><td align="center"><offsets xml_i="31658" xml_f="31660" txt_i="17591" txt_f="17593">83</offsets></td><td align="center"><offsets xml_i="31684" xml_f="31689" txt_i="17594" txt_f="17599">1.834</offsets></td><td align="center"><offsets xml_i="31713" xml_f="31721" txt_i="17600" txt_f="17608">2.27E-04</offsets></td></tr><tr><td align="left"><offsets xml_i="31752" xml_f="31759" txt_i="17609" txt_f="17616">0042060</offsets></td><td align="left"><offsets xml_i="31781" xml_f="31794" txt_i="17617" txt_f="17630">Wound healing</offsets></td><td align="center"><offsets xml_i="31818" xml_f="31820" txt_i="17631" txt_f="17633">32</offsets></td><td align="center"><offsets xml_i="31844" xml_f="31849" txt_i="17634" txt_f="17639">2.944</offsets></td><td align="center"><offsets xml_i="31873" xml_f="31881" txt_i="17640" txt_f="17648">3.14E-04</offsets></td></tr><tr><td align="left"><offsets xml_i="31912" xml_f="31919" txt_i="17649" txt_f="17656">0007160</offsets></td><td align="left"><offsets xml_i="31941" xml_f="31961" txt_i="17657" txt_f="17677">Cell-matrix adhesion</offsets></td><td align="center"><offsets xml_i="31985" xml_f="31987" txt_i="17678" txt_f="17680">20</offsets></td><td align="center"><offsets xml_i="32011" xml_f="32016" txt_i="17681" txt_f="17686">3.931</offsets></td><td align="center"><offsets xml_i="32040" xml_f="32048" txt_i="17687" txt_f="17695">1.42E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="32079" xml_f="32086" txt_i="17696" txt_f="17703">0048514</offsets></td><td align="left"><offsets xml_i="32108" xml_f="32134" txt_i="17704" txt_f="17730">Blood vessel morphogenesis</offsets></td><td align="center"><offsets xml_i="32158" xml_f="32160" txt_i="17731" txt_f="17733">33</offsets></td><td align="center"><offsets xml_i="32184" xml_f="32189" txt_i="17734" txt_f="17739">2.692</offsets></td><td align="center"><offsets xml_i="32213" xml_f="32221" txt_i="17740" txt_f="17748">1.56E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="32252" xml_f="32259" txt_i="17749" txt_f="17756">0031589</offsets></td><td align="left"><offsets xml_i="32281" xml_f="32304" txt_i="17757" txt_f="17780">Cell-substrate adhesion</offsets></td><td align="center"><offsets xml_i="32328" xml_f="32330" txt_i="17781" txt_f="17783">21</offsets></td><td align="center"><offsets xml_i="32354" xml_f="32359" txt_i="17784" txt_f="17789">3.745</offsets></td><td align="center"><offsets xml_i="32383" xml_f="32391" txt_i="17790" txt_f="17798">1.57E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="32422" xml_f="32429" txt_i="17799" txt_f="17806">0006928</offsets></td><td align="left"><offsets xml_i="32451" xml_f="32462" txt_i="17807" txt_f="17818">Cell motion</offsets></td><td align="center"><offsets xml_i="32486" xml_f="32488" txt_i="17819" txt_f="17821">53</offsets></td><td align="center"><offsets xml_i="32512" xml_f="32517" txt_i="17822" txt_f="17827">1.984</offsets></td><td align="center"><offsets xml_i="32541" xml_f="32549" txt_i="17828" txt_f="17836">7.97E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="32580" xml_f="32587" txt_i="17837" txt_f="17844">0001525</offsets></td><td align="left"></td><td align="center"><offsets xml_i="32633" xml_f="32635" txt_i="17846" txt_f="17848">25</offsets></td><td align="center"><offsets xml_i="32659" xml_f="32664" txt_i="17849" txt_f="17854">2.922</offsets></td><td align="center"><offsets xml_i="32688" xml_f="32696" txt_i="17855" txt_f="17863">1.15E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="32727" xml_f="32734" txt_i="17864" txt_f="17871">0009612</offsets></td><td align="left"><offsets xml_i="32756" xml_f="32787" txt_i="17872" txt_f="17903">Response to mechanical stimulus</offsets></td><td align="center"><offsets xml_i="32811" xml_f="32813" txt_i="17904" txt_f="17906">14</offsets></td><td align="center"><offsets xml_i="32837" xml_f="32842" txt_i="17907" txt_f="17912">4.324</offsets></td><td align="center"><offsets xml_i="32866" xml_f="32874" txt_i="17913" txt_f="17921">3.91E-02</offsets></td></tr><tr><td colspan="5"><hr></hr></td></tr><tr><td align="left" colspan="5"><bold><offsets xml_i="32962" xml_f="32976" txt_i="17923" txt_f="17937">Down-regulated</offsets></bold></td></tr><tr><td colspan="5"><hr></hr></td></tr><tr><td align="left"><offsets xml_i="33053" xml_f="33060" txt_i="17939" txt_f="17946">0006259</offsets></td><td align="left"><offsets xml_i="33082" xml_f="33103" txt_i="17947" txt_f="17968">DNA metabolic process</offsets></td><td align="center"><offsets xml_i="33127" xml_f="33129" txt_i="17969" txt_f="17971">52</offsets></td><td align="center"><offsets xml_i="33153" xml_f="33158" txt_i="17972" txt_f="17977">3.018</offsets></td><td align="center"><offsets xml_i="33182" xml_f="33190" txt_i="17978" txt_f="17986">5.10E-09</offsets></td></tr><tr><td align="left"><offsets xml_i="33221" xml_f="33228" txt_i="17987" txt_f="17994">0007049</offsets></td><td align="left"><offsets xml_i="33250" xml_f="33260" txt_i="17995" txt_f="18005">Cell cycle</offsets></td><td align="center"><offsets xml_i="33284" xml_f="33286" txt_i="18006" txt_f="18008">64</offsets></td><td align="center"><offsets xml_i="33310" xml_f="33315" txt_i="18009" txt_f="18014">2.411</offsets></td><td align="center"><offsets xml_i="33339" xml_f="33347" txt_i="18015" txt_f="18023">1.85E-07</offsets></td></tr><tr><td align="left"><offsets xml_i="33378" xml_f="33385" txt_i="18024" txt_f="18031">0006260</offsets></td><td align="left"><offsets xml_i="33407" xml_f="33422" txt_i="18032" txt_f="18047">DNA replication</offsets></td><td align="center"><offsets xml_i="33446" xml_f="33448" txt_i="18048" txt_f="18050">28</offsets></td><td align="center"><offsets xml_i="33472" xml_f="33477" txt_i="18051" txt_f="18056">4.387</offsets></td><td align="center"><offsets xml_i="33501" xml_f="33509" txt_i="18057" txt_f="18065">3.98E-07</offsets></td></tr><tr><td align="left"><offsets xml_i="33540" xml_f="33547" txt_i="18066" txt_f="18073">0022402</offsets></td><td align="left"><offsets xml_i="33569" xml_f="33587" txt_i="18074" txt_f="18092">Cell cycle process</offsets></td><td align="center"><offsets xml_i="33611" xml_f="33613" txt_i="18093" txt_f="18095">50</offsets></td><td align="center"><offsets xml_i="33637" xml_f="33642" txt_i="18096" txt_f="18101">2.569</offsets></td><td align="center"><offsets xml_i="33666" xml_f="33674" txt_i="18102" txt_f="18110">4.12E-06</offsets></td></tr><tr><td align="left"><offsets xml_i="33705" xml_f="33712" txt_i="18111" txt_f="18118">0000278</offsets></td><td align="left"><offsets xml_i="33734" xml_f="33752" txt_i="18119" txt_f="18137">Mitotic cell cycle</offsets></td><td align="center"><offsets xml_i="33776" xml_f="33778" txt_i="18138" txt_f="18140">37</offsets></td><td align="center"><offsets xml_i="33802" xml_f="33807" txt_i="18141" txt_f="18146">2.898</offsets></td><td align="center"><offsets xml_i="33831" xml_f="33839" txt_i="18147" txt_f="18155">3.70E-05</offsets></td></tr><tr><td align="left"><offsets xml_i="33870" xml_f="33877" txt_i="18156" txt_f="18163">0000279</offsets></td><td align="left"><offsets xml_i="33899" xml_f="33906" txt_i="18164" txt_f="18171">M phase</offsets></td><td align="center"><offsets xml_i="33930" xml_f="33932" txt_i="18172" txt_f="18174">32</offsets></td><td align="center"><offsets xml_i="33956" xml_f="33961" txt_i="18175" txt_f="18180">2.832</offsets></td><td align="center"><offsets xml_i="33985" xml_f="33993" txt_i="18181" txt_f="18189">6.92E-04</offsets></td></tr><tr><td align="left"><offsets xml_i="34024" xml_f="34031" txt_i="18190" txt_f="18197">0051276</offsets></td><td align="left"><offsets xml_i="34053" xml_f="34076" txt_i="18198" txt_f="18221">Chromosome organisation</offsets></td><td align="center"><offsets xml_i="34100" xml_f="34102" txt_i="18222" txt_f="18224">40</offsets></td><td align="center"><offsets xml_i="34126" xml_f="34131" txt_i="18225" txt_f="18230">2.461</offsets></td><td align="center"><offsets xml_i="34155" xml_f="34163" txt_i="18231" txt_f="18239">7.18E-04</offsets></td></tr><tr><td align="left"><offsets xml_i="34194" xml_f="34201" txt_i="18240" txt_f="18247">0006974</offsets></td><td align="left"><offsets xml_i="34223" xml_f="34254" txt_i="18248" txt_f="18279">Response to DNA damage stimulus</offsets></td><td align="center"><offsets xml_i="34278" xml_f="34280" txt_i="18280" txt_f="18282">34</offsets></td><td align="center"><offsets xml_i="34304" xml_f="34309" txt_i="18283" txt_f="18288">2.656</offsets></td><td align="center"><offsets xml_i="34333" xml_f="34341" txt_i="18289" txt_f="18297">1.20E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="34372" xml_f="34379" txt_i="18298" txt_f="18305">0000087</offsets></td><td align="left"><offsets xml_i="34401" xml_f="34430" txt_i="18306" txt_f="18335">M phase of mitotic cell cycle</offsets></td><td align="center"><offsets xml_i="34454" xml_f="34456" txt_i="18336" txt_f="18338">25</offsets></td><td align="center"><offsets xml_i="34480" xml_f="34485" txt_i="18339" txt_f="18344">3.229</offsets></td><td align="center"><offsets xml_i="34509" xml_f="34517" txt_i="18345" txt_f="18353">1.76E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="34548" xml_f="34555" txt_i="18354" txt_f="18361">0022613</offsets></td><td align="left"><offsets xml_i="34577" xml_f="34613" txt_i="18362" txt_f="18398">Ribonucleoprotein complex biogenesis</offsets></td><td align="center"><offsets xml_i="34637" xml_f="34639" txt_i="18399" txt_f="18401">22</offsets></td><td align="center"><offsets xml_i="34663" xml_f="34668" txt_i="18402" txt_f="18407">3.544</offsets></td><td align="center"><offsets xml_i="34692" xml_f="34700" txt_i="18408" txt_f="18416">2.07E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="34731" xml_f="34738" txt_i="18417" txt_f="18424">0000280</offsets></td><td align="left"><offsets xml_i="34760" xml_f="34776" txt_i="18425" txt_f="18441">Nuclear division</offsets></td><td align="center"><offsets xml_i="34800" xml_f="34802" txt_i="18442" txt_f="18444">24</offsets></td><td align="center"><offsets xml_i="34826" xml_f="34831" txt_i="18445" txt_f="18450">3.156</offsets></td><td align="center"><offsets xml_i="34855" xml_f="34863" txt_i="18451" txt_f="18459">4.53E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="34894" xml_f="34901" txt_i="18460" txt_f="18467">0007067</offsets></td><td align="left"><offsets xml_i="34923" xml_f="34930" txt_i="18468" txt_f="18475">Mitosis</offsets></td><td align="center"><offsets xml_i="34954" xml_f="34956" txt_i="18476" txt_f="18478">24</offsets></td><td align="center"><offsets xml_i="34980" xml_f="34985" txt_i="18479" txt_f="18484">3.156</offsets></td><td align="center"><offsets xml_i="35009" xml_f="35017" txt_i="18485" txt_f="18493">4.53E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="35048" xml_f="35055" txt_i="18494" txt_f="18501">0022403</offsets></td><td align="left"><offsets xml_i="35077" xml_f="35093" txt_i="18502" txt_f="18518">Cell cycle phase</offsets></td><td align="center"><offsets xml_i="35117" xml_f="35119" txt_i="18519" txt_f="18521">35</offsets></td><td align="center"><offsets xml_i="35143" xml_f="35148" txt_i="18522" txt_f="18527">2.454</offsets></td><td align="center"><offsets xml_i="35172" xml_f="35180" txt_i="18528" txt_f="18536">4.81E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="35211" xml_f="35218" txt_i="18537" txt_f="18544">0042254</offsets></td><td align="left"><offsets xml_i="35240" xml_f="35259" txt_i="18545" txt_f="18564">Ribosome biogenesis</offsets></td><td align="center"><offsets xml_i="35283" xml_f="35285" txt_i="18565" txt_f="18567">17</offsets></td><td align="center"><offsets xml_i="35309" xml_f="35314" txt_i="18568" txt_f="18573">4.062</offsets></td><td align="center"><offsets xml_i="35338" xml_f="35346" txt_i="18574" txt_f="18582">8.27E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="35377" xml_f="35384" txt_i="18583" txt_f="18590">0048285</offsets></td><td align="left"><offsets xml_i="35406" xml_f="35423" txt_i="18591" txt_f="18608">Organelle fission</offsets></td><td align="center"><offsets xml_i="35447" xml_f="35449" txt_i="18609" txt_f="18611">24</offsets></td><td align="center"><offsets xml_i="35473" xml_f="35478" txt_i="18612" txt_f="18617">3.031</offsets></td><td align="center"><offsets xml_i="35502" xml_f="35510" txt_i="18618" txt_f="18626">9.01E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="35541" xml_f="35548" txt_i="18627" txt_f="18634">0006281</offsets></td><td align="left"><offsets xml_i="35570" xml_f="35580" txt_i="18635" txt_f="18645">DNA repair</offsets></td><td align="center"><offsets xml_i="35604" xml_f="35606" txt_i="18646" txt_f="18648">27</offsets></td><td align="center"><offsets xml_i="35630" xml_f="35635" txt_i="18649" txt_f="18654">2.785</offsets></td><td align="center"><offsets xml_i="35659" xml_f="35667" txt_i="18655" txt_f="18663">9.53E-03</offsets></td></tr><tr><td align="left"><offsets xml_i="35698" xml_f="35705" txt_i="18664" txt_f="18671">0006396</offsets></td><td align="left"><offsets xml_i="35727" xml_f="35741" txt_i="18672" txt_f="18686">RNA processing</offsets></td><td align="center"><offsets xml_i="35765" xml_f="35767" txt_i="18687" txt_f="18689">41</offsets></td><td align="center"><offsets xml_i="35791" xml_f="35796" txt_i="18690" txt_f="18695">2.185</offsets></td><td align="center"><offsets xml_i="35820" xml_f="35828" txt_i="18696" txt_f="18704">1.01E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="35859" xml_f="35866" txt_i="18705" txt_f="18712">0031396</offsets></td><td align="left"><offsets xml_i="35888" xml_f="35924" txt_i="18713" txt_f="18749">Regulation of protein ubiquitination</offsets></td><td align="center"><offsets xml_i="35948" xml_f="35950" txt_i="18750" txt_f="18752">15</offsets></td><td align="center"><offsets xml_i="35974" xml_f="35979" txt_i="18753" txt_f="18758">4.434</offsets></td><td align="center"><offsets xml_i="36003" xml_f="36011" txt_i="18759" txt_f="18767">1.30E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="36042" xml_f="36049" txt_i="18768" txt_f="18775">0031397</offsets></td><td align="left"><offsets xml_i="36071" xml_f="36116" txt_i="18776" txt_f="18821">Negative regulation of protein ubiquitination</offsets></td><td align="center"><offsets xml_i="36140" xml_f="36142" txt_i="18822" txt_f="18824">13</offsets></td><td align="center"><offsets xml_i="36166" xml_f="36171" txt_i="18825" txt_f="18830">5.177</offsets></td><td align="center"><offsets xml_i="36195" xml_f="36203" txt_i="18831" txt_f="18839">1.31E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="36234" xml_f="36241" txt_i="18840" txt_f="18847">0031145</offsets></td><td align="left"><offsets xml_i="36263" xml_f="36357" txt_i="18848" txt_f="18942">Anaphase-promoting complex-dependent proteasomal ubiquitin-dependent protein catabolic process</offsets></td><td align="center"><offsets xml_i="36381" xml_f="36383" txt_i="18943" txt_f="18945">12</offsets></td><td align="center"><offsets xml_i="36407" xml_f="36412" txt_i="18946" txt_f="18951">5.461</offsets></td><td align="center"><offsets xml_i="36436" xml_f="36444" txt_i="18952" txt_f="18960">2.01E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="36475" xml_f="36482" txt_i="18961" txt_f="18968">0016071</offsets></td><td align="left"><offsets xml_i="36504" xml_f="36526" txt_i="18969" txt_f="18991">mRNA metabolic process</offsets></td><td align="center"><offsets xml_i="36550" xml_f="36552" txt_i="18992" txt_f="18994">31</offsets></td><td align="center"><offsets xml_i="36576" xml_f="36581" txt_i="18995" txt_f="19000">2.428</offsets></td><td align="center"><offsets xml_i="36605" xml_f="36613" txt_i="19001" txt_f="19009">2.52E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="36644" xml_f="36651" txt_i="19010" txt_f="19017">0034470</offsets></td><td align="left"><offsets xml_i="36673" xml_f="36689" txt_i="19018" txt_f="19034">ncRNA processing</offsets></td><td align="center"><offsets xml_i="36713" xml_f="36715" txt_i="19035" txt_f="19037">20</offsets></td><td align="center"><offsets xml_i="36739" xml_f="36744" txt_i="19038" txt_f="19043">3.133</offsets></td><td align="center"><offsets xml_i="36768" xml_f="36776" txt_i="19044" txt_f="19052">4.28E-02</offsets></td></tr><tr><td align="left"><offsets xml_i="36807" xml_f="36814" txt_i="19053" txt_f="19060">0051439</offsets></td><td align="left"><offsets xml_i="36836" xml_f="36909" txt_i="19061" txt_f="19134">Regulation of ubiquitin-protein ligase activity during mitotic cell cycle</offsets></td><td align="center"><offsets xml_i="36933" xml_f="36935" txt_i="19135" txt_f="19137">12</offsets></td><td align="center"><offsets xml_i="36959" xml_f="36964" txt_i="19138" txt_f="19143">4.986</offsets></td><td align="center"><offsets xml_i="36988" xml_f="36996" txt_i="19144" txt_f="19152">4.85E-02</offsets></td></tr></tbody></table><table-wrap-foot><p><sup><offsets xml_i="37047" xml_f="37048" txt_i="19153" txt_f="19154">a</offsets></sup><italic><offsets xml_i="37062" xml_f="37064" txt_i="19154" txt_f="19156">P-</offsets></italic><offsets xml_i="37073" xml_f="37172" txt_i="19156" txt_f="19255">values shown were corrected for multiple testing using the Bonferroni method; categories ranked by </offsets><italic><offsets xml_i="37180" xml_f="37182" txt_i="19255" txt_f="19257">P-</offsets></italic><offsets xml_i="37191" xml_f="37225" txt_i="19257" txt_f="19291">value; functional categories with </offsets><italic><offsets xml_i="37233" xml_f="37235" txt_i="19291" txt_f="19293">P-</offsets></italic><offsets xml_i="37244" xml_f="37336" txt_i="19293" txt_f="19382">values &lt; 0.05 and at least five, but less than 150 genes meeting the criterion are shown.</offsets></p></table-wrap-foot></table-wrap><table-wrap id="T3" position="float"><label><offsets xml_i="37415" xml_f="37422" txt_i="19383" txt_f="19390">Table 3</offsets></label><caption><p><offsets xml_i="37442" xml_f="37528" txt_i="19390" txt_f="19476">List of local pathways in PathVisio significantly affected by the celecoxib treatment.</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left"><offsets xml_i="37607" xml_f="37616" txt_i="19477" txt_f="19486">MAPP Name</offsets></th><th align="center"><offsets xml_i="37640" xml_f="37646" txt_i="19486" txt_f="19492">number</offsets><break></break><offsets xml_i="37661" xml_f="37668" txt_i="19492" txt_f="19499">changed</offsets></th><th align="center"><offsets xml_i="37692" xml_f="37698" txt_i="19499" txt_f="19505">number</offsets><break></break><offsets xml_i="37713" xml_f="37721" txt_i="19505" txt_f="19513">measured</offsets></th><th align="center"><offsets xml_i="37745" xml_f="37754" txt_i="19513" txt_f="19522">% changed</offsets></th><th align="center"><bold><italic><offsets xml_i="37792" xml_f="37793" txt_i="19522" txt_f="19523">Z</offsets></italic><offsets xml_i="37802" xml_f="37808" txt_i="19523" txt_f="19529">-score</offsets></bold><sup><offsets xml_i="37820" xml_f="37821" txt_i="19529" txt_f="19530">a</offsets></sup></th></tr></thead><tbody><tr><td align="left" colspan="5"><bold><offsets xml_i="37891" xml_f="37903" txt_i="19530" txt_f="19542">Up-regulated</offsets></bold></td></tr><tr><td colspan="5"><hr></hr></td></tr><tr><td align="left"><offsets xml_i="37980" xml_f="38015" txt_i="19544" txt_f="19579">Complement and coagulation cascades</offsets></td><td align="center"><offsets xml_i="38039" xml_f="38041" txt_i="19580" txt_f="19582">15</offsets></td><td align="center"><offsets xml_i="38065" xml_f="38067" txt_i="19583" txt_f="19585">50</offsets></td><td align="center"><offsets xml_i="38091" xml_f="38097" txt_i="19586" txt_f="19592">30.00%</offsets></td><td align="center"><offsets xml_i="38121" xml_f="38125" txt_i="19593" txt_f="19597">5.65</offsets></td></tr><tr><td align="left"><offsets xml_i="38156" xml_f="38180" txt_i="19598" txt_f="19622">Senescence and autophagy</offsets></td><td align="center"><offsets xml_i="38204" xml_f="38206" txt_i="19623" txt_f="19625">18</offsets></td><td align="center"><offsets xml_i="38230" xml_f="38233" txt_i="19626" txt_f="19629">100</offsets></td><td align="center"><offsets xml_i="38257" xml_f="38263" txt_i="19630" txt_f="19636">18.00%</offsets></td><td align="center"><offsets xml_i="38287" xml_f="38291" txt_i="19637" txt_f="19641">3.61</offsets></td></tr><tr><td align="left"><offsets xml_i="38322" xml_f="38351" txt_i="19642" txt_f="19671">RANKL/RANK signalling pathway</offsets></td><td align="center"><offsets xml_i="38375" xml_f="38377" txt_i="19672" txt_f="19674">11</offsets></td><td align="center"><offsets xml_i="38401" xml_f="38403" txt_i="19675" txt_f="19677">50</offsets></td><td align="center"><offsets xml_i="38427" xml_f="38433" txt_i="19678" txt_f="19684">22.00%</offsets></td><td align="center"><offsets xml_i="38457" xml_f="38461" txt_i="19685" txt_f="19689">3.58</offsets></td></tr><tr><td align="left"><offsets xml_i="38492" xml_f="38532" txt_i="19690" txt_f="19730">Complement activation, classical pathway</offsets></td><td align="center"><offsets xml_i="38556" xml_f="38557" txt_i="19731" txt_f="19732">5</offsets></td><td align="center"><offsets xml_i="38581" xml_f="38583" txt_i="19733" txt_f="19735">15</offsets></td><td align="center"><offsets xml_i="38607" xml_f="38613" txt_i="19736" txt_f="19742">33.33%</offsets></td><td align="center"><offsets xml_i="38637" xml_f="38641" txt_i="19743" txt_f="19747">3.55</offsets></td></tr><tr><td align="left"><offsets xml_i="38672" xml_f="38697" txt_i="19748" txt_f="19773">Matrix metalloproteinases</offsets></td><td align="center"><offsets xml_i="38721" xml_f="38722" txt_i="19774" txt_f="19775">7</offsets></td><td align="center"><offsets xml_i="38746" xml_f="38748" txt_i="19776" txt_f="19778">29</offsets></td><td align="center"><offsets xml_i="38772" xml_f="38778" txt_i="19779" txt_f="19785">24.14%</offsets></td><td align="center"><offsets xml_i="38802" xml_f="38806" txt_i="19786" txt_f="19790">3.14</offsets></td></tr><tr><td align="left"><offsets xml_i="38837" xml_f="38864" txt_i="19791" txt_f="19818">TGF beta signalling pathway</offsets></td><td align="center"><offsets xml_i="38888" xml_f="38890" txt_i="19819" txt_f="19821">18</offsets></td><td align="center"><offsets xml_i="38914" xml_f="38917" txt_i="19822" txt_f="19825">113</offsets></td><td align="center"><offsets xml_i="38941" xml_f="38947" txt_i="19826" txt_f="19832">15.93%</offsets></td><td align="center"><offsets xml_i="38971" xml_f="38975" txt_i="19833" txt_f="19837">3.03</offsets></td></tr><tr><td align="left"><offsets xml_i="39006" xml_f="39035" txt_i="19838" txt_f="19867">Inflammatory response pathway</offsets></td><td align="center"><offsets xml_i="39059" xml_f="39060" txt_i="19868" txt_f="19869">7</offsets></td><td align="center"><offsets xml_i="39084" xml_f="39086" txt_i="19870" txt_f="19872">30</offsets></td><td align="center"><offsets xml_i="39110" xml_f="39116" txt_i="19873" txt_f="19879">23.33%</offsets></td><td align="center"><offsets xml_i="39140" xml_f="39144" txt_i="19880" txt_f="19884">3.03</offsets></td></tr><tr><td align="left"><offsets xml_i="39175" xml_f="39191" txt_i="19885" txt_f="19901">Oxidative stress</offsets></td><td align="center"><offsets xml_i="39215" xml_f="39216" txt_i="19902" txt_f="19903">6</offsets></td><td align="center"><offsets xml_i="39240" xml_f="39242" txt_i="19904" txt_f="19906">24</offsets></td><td align="center"><offsets xml_i="39266" xml_f="39272" txt_i="19907" txt_f="19913">25.00%</offsets></td><td align="center"><offsets xml_i="39296" xml_f="39300" txt_i="19914" txt_f="19918">3.01</offsets></td></tr><tr><td align="left"><offsets xml_i="39331" xml_f="39343" txt_i="19919" txt_f="19931">Adipogenesis</offsets></td><td align="center"><offsets xml_i="39367" xml_f="39369" txt_i="19932" txt_f="19934">19</offsets></td><td align="center"><offsets xml_i="39393" xml_f="39396" txt_i="19935" txt_f="19938">125</offsets></td><td align="center"><offsets xml_i="39420" xml_f="39426" txt_i="19939" txt_f="19945">15.20%</offsets></td><td align="center"><offsets xml_i="39450" xml_f="39454" txt_i="19946" txt_f="19950">2.89</offsets></td></tr><tr><td align="left"><offsets xml_i="39485" xml_f="39510" txt_i="19951" txt_f="19976">Endochondral ossification</offsets></td><td align="center"><offsets xml_i="39534" xml_f="39536" txt_i="19977" txt_f="19979">11</offsets></td><td align="center"><offsets xml_i="39560" xml_f="39562" txt_i="19980" txt_f="19982">61</offsets></td><td align="center"><offsets xml_i="39586" xml_f="39592" txt_i="19983" txt_f="19989">18.03%</offsets></td><td align="center"><offsets xml_i="39616" xml_f="39620" txt_i="19990" txt_f="19994">2.82</offsets></td></tr><tr><td align="left"><offsets xml_i="39651" xml_f="39675" txt_i="19995" txt_f="20019">TWEAK signalling pathway</offsets></td><td align="center"><offsets xml_i="39699" xml_f="39700" txt_i="20020" txt_f="20021">8</offsets></td><td align="center"><offsets xml_i="39724" xml_f="39726" txt_i="20022" txt_f="20024">40</offsets></td><td align="center"><offsets xml_i="39750" xml_f="39756" txt_i="20025" txt_f="20031">20.00%</offsets></td><td align="center"><offsets xml_i="39780" xml_f="39784" txt_i="20032" txt_f="20036">2.73</offsets></td></tr><tr><td align="left"><offsets xml_i="39815" xml_f="39831" txt_i="20037" txt_f="20053">SREBP signalling</offsets></td><td align="center"><offsets xml_i="39855" xml_f="39856" txt_i="20054" txt_f="20055">6</offsets></td><td align="center"><offsets xml_i="39880" xml_f="39882" txt_i="20056" txt_f="20058">28</offsets></td><td align="center"><offsets xml_i="39906" xml_f="39912" txt_i="20059" txt_f="20065">21.43%</offsets></td><td align="center"><offsets xml_i="39936" xml_f="39940" txt_i="20066" txt_f="20070">2.56</offsets></td></tr><tr><td colspan="5"><hr></hr></td></tr><tr><td align="left"><bold><offsets xml_i="40016" xml_f="40030" txt_i="20072" txt_f="20086">Down-regulated</offsets></bold></td><td></td><td></td><td></td><td></td></tr><tr><td colspan="5"><hr></hr></td></tr><tr><td align="left"><offsets xml_i="40143" xml_f="40153" txt_i="20092" txt_f="20102">Cell cycle</offsets></td><td align="center"><offsets xml_i="40177" xml_f="40179" txt_i="20103" txt_f="20105">17</offsets></td><td align="center"><offsets xml_i="40203" xml_f="40205" txt_i="20106" txt_f="20108">95</offsets></td><td align="center"><offsets xml_i="40229" xml_f="40235" txt_i="20109" txt_f="20115">17.89%</offsets></td><td align="center"><offsets xml_i="40259" xml_f="40263" txt_i="20116" txt_f="20120">6.62</offsets></td></tr><tr><td align="left"><offsets xml_i="40294" xml_f="40309" txt_i="20121" txt_f="20136">DNA replication</offsets></td><td align="center"><offsets xml_i="40333" xml_f="40334" txt_i="20137" txt_f="20138">9</offsets></td><td align="center"><offsets xml_i="40358" xml_f="40360" txt_i="20139" txt_f="20141">41</offsets></td><td align="center"><offsets xml_i="40384" xml_f="40390" txt_i="20142" txt_f="20148">21.95%</offsets></td><td align="center"><offsets xml_i="40414" xml_f="40418" txt_i="20149" txt_f="20153">5.61</offsets></td></tr><tr><td align="left"><offsets xml_i="40449" xml_f="40470" txt_i="20154" txt_f="20175">Nucleotide metabolism</offsets></td><td align="center"><offsets xml_i="40494" xml_f="40495" txt_i="20176" txt_f="20177">5</offsets></td><td align="center"><offsets xml_i="40519" xml_f="40521" txt_i="20178" txt_f="20180">17</offsets></td><td align="center"><offsets xml_i="40545" xml_f="40551" txt_i="20181" txt_f="20187">29.41%</offsets></td><td align="center"><offsets xml_i="40575" xml_f="40579" txt_i="20188" txt_f="20192">5.12</offsets></td></tr><tr><td align="left"><offsets xml_i="40610" xml_f="40632" txt_i="20193" txt_f="20215">Proteasome degradation</offsets></td><td align="center"><offsets xml_i="40656" xml_f="40658" txt_i="20216" txt_f="20218">10</offsets></td><td align="center"><offsets xml_i="40682" xml_f="40684" txt_i="20219" txt_f="20221">57</offsets></td><td align="center"><offsets xml_i="40708" xml_f="40714" txt_i="20222" txt_f="20228">17.54%</offsets></td><td align="center"><offsets xml_i="40738" xml_f="40742" txt_i="20229" txt_f="20233">4.97</offsets></td></tr><tr><td align="left"><offsets xml_i="40773" xml_f="40797" txt_i="20234" txt_f="20258">Electron transport chain</offsets></td><td align="center"><offsets xml_i="40821" xml_f="40823" txt_i="20259" txt_f="20261">10</offsets></td><td align="center"><offsets xml_i="40847" xml_f="40849" txt_i="20262" txt_f="20264">81</offsets></td><td align="center"><offsets xml_i="40873" xml_f="40879" txt_i="20265" txt_f="20271">12.35%</offsets></td><td align="center"><offsets xml_i="40903" xml_f="40907" txt_i="20272" txt_f="20276">3.61</offsets></td></tr><tr><td align="left"><offsets xml_i="40938" xml_f="40964" txt_i="20277" txt_f="20303">G1 to S cell cycle control</offsets></td><td align="center"><offsets xml_i="40988" xml_f="40989" txt_i="20304" txt_f="20305">7</offsets></td><td align="center"><offsets xml_i="41013" xml_f="41015" txt_i="20306" txt_f="20308">67</offsets></td><td align="center"><offsets xml_i="41039" xml_f="41045" txt_i="20309" txt_f="20315">10.45%</offsets></td><td align="center"><offsets xml_i="41069" xml_f="41073" txt_i="20316" txt_f="20320">2.51</offsets></td></tr></tbody></table><table-wrap-foot><p><sup><offsets xml_i="41124" xml_f="41125" txt_i="20321" txt_f="20322">a</offsets></sup><italic><offsets xml_i="41139" xml_f="41140" txt_i="20322" txt_f="20323">Z</offsets></italic><offsets xml_i="41149" xml_f="41247" txt_i="20323" txt_f="20421">-score is the standard statistical test under the hypergeometric distribution; pathways ranked by </offsets><italic><offsets xml_i="41255" xml_f="41256" txt_i="20421" txt_f="20422">Z</offsets></italic><offsets xml_i="41265" xml_f="41273" txt_i="20422" txt_f="20430">-score. </offsets><italic><offsets xml_i="41281" xml_f="41282" txt_i="20430" txt_f="20431">Z</offsets></italic><offsets xml_i="41291" xml_f="41326" txt_i="20431" txt_f="20463">-score &gt; 1.96 (corresponding to </offsets><italic><offsets xml_i="41334" xml_f="41335" txt_i="20463" txt_f="20464">P</offsets></italic><offsets xml_i="41344" xml_f="41461" txt_i="20464" txt_f="20581">-value = 0.05) and at least five, but less than 150 genes meeting the criterion are shown. TGF, tumour growth factor.</offsets></p></table-wrap-foot></table-wrap></sec><sec><title><offsets xml_i="41514" xml_f="41564" txt_i="20583" txt_f="20633">Effects of celecoxib on cell cycle gene expression</offsets></title><p><offsets xml_i="41575" xml_f="41811" txt_i="20634" txt_f="20870">DAVID and PathVisio consistently identified biological processes referring to regulation of cell cycle and proliferation, in particular among those genes down-regulated after treatment. An important downstream DNA damage response gene, </offsets><italic><offsets xml_i="41819" xml_f="41827" txt_i="20870" txt_f="20878">GADD45A </offsets></italic><offsets xml_i="41836" xml_f="41924" txt_i="20878" txt_f="20966">(Growth arrest and DNA-damage-inducible, alpha), was significantly up-regulated (Figure </offsets><xref ref-type="fig" rid="F3"><offsets xml_i="41954" xml_f="41955" txt_i="20966" txt_f="20967">3</offsets></xref><offsets xml_i="41962" xml_f="42089" txt_i="20967" txt_f="21094">). Consistent with a putative activation of the G2/M checkpoint and cell cycle arrest due to DNA damage response, target genes </offsets><italic><offsets xml_i="42097" xml_f="42103" txt_i="21094" txt_f="21100">CCNB1 </offsets></italic><offsets xml_i="42112" xml_f="42148" txt_i="21100" txt_f="21136">(G2/mitotic-specific cyclin-B1) and </offsets><italic><offsets xml_i="42156" xml_f="42162" txt_i="21136" txt_f="21142">CCNB2 </offsets></italic><offsets xml_i="42171" xml_f="42253" txt_i="21142" txt_f="21224">(G2/mitotic-specific cyclin-B2) were significantly down-regulated after treatment.</offsets></p><fig id="F3" position="float"><label><offsets xml_i="42294" xml_f="42302" txt_i="21225" txt_f="21233">Figure 3</offsets></label><caption><p><bold><offsets xml_i="42328" xml_f="42390" txt_i="21233" txt_f="21295">Effects of celecoxib treatment on cell cycle and proliferation</offsets></bold><offsets xml_i="42397" xml_f="42854" txt_i="21295" txt_f="21752">. Contributed map from GenMAPP software with an overview of the pathways and genes involved in cell cycle regulation. The expression fold-changes of each gene are indicated next to the gene box. Genes highlighted in red and green in the left box half represent genes with fold-changes increased and decreased, respectively. Red colour in the right box half indicates a significant change. Grey boxes correspond to genes that were not analysed in the arrays.</offsets></p></caption><graphic xlink:href="bcr3409-3"></graphic></fig></sec><sec><title><offsets xml_i="42934" xml_f="42989" txt_i="21754" txt_f="21809">Effects of celecoxib on ECM degradation gene expression</offsets></title><p><offsets xml_i="43000" xml_f="43279" txt_i="21810" txt_f="22089">The majority of the matrix metalloproteinase (MMP) family members have been associated with tumour progression. The conversion of pro-MMP to active MMP-2 requires membrane type MT1-MMP (MMP-14), a trans-membrane protein that is activated intracellularly by the convertase FURIN [</offsets><xref ref-type="bibr" rid="B35"><offsets xml_i="43311" xml_f="43313" txt_i="22089" txt_f="22091">35</offsets></xref><offsets xml_i="43320" xml_f="43369" txt_i="22091" txt_f="22140">]. The down-regulation of the protein convertase </offsets><italic><offsets xml_i="43377" xml_f="43383" txt_i="22140" txt_f="22146">FURIN </offsets></italic><offsets xml_i="43392" xml_f="43562" txt_i="22146" txt_f="22316">in the celecoxib-treated group potentially leads to less activation of MT1-MMP. Additionally, the effect of MMP-2 on proteolysis was inhibited either by up-regulation of </offsets><italic><offsets xml_i="43570" xml_f="43589" txt_i="22316" txt_f="22335">TIMP1, TIMP2, TIMP3</offsets></italic><offsets xml_i="43598" xml_f="43606" txt_i="22335" txt_f="22343">, or by </offsets><italic><offsets xml_i="43614" xml_f="43619" txt_i="22343" txt_f="22348">RECK </offsets></italic><offsets xml_i="43628" xml_f="43636" txt_i="22348" txt_f="22356">(Figure </offsets><xref ref-type="fig" rid="F4"><offsets xml_i="43666" xml_f="43667" txt_i="22356" txt_f="22357">4</offsets></xref><offsets xml_i="43674" xml_f="43694" txt_i="22357" txt_f="22377">). The up-regulated </offsets><italic><offsets xml_i="43702" xml_f="43707" txt_i="22377" txt_f="22382">RECK </offsets></italic><offsets xml_i="43716" xml_f="43948" txt_i="22382" txt_f="22614">exerts inhibitory effects on the conversion of pro-MMP-2 to MMP-2 and on the activation of pro-MMP-9 to MMP-9. In summary, our data suggest that degradation of ECM proteins was significantly inhibited in the celecoxib-treated group.</offsets></p><fig id="F4" position="float"><label><offsets xml_i="43989" xml_f="43997" txt_i="22615" txt_f="22623">Figure 4</offsets></label><caption><p><bold><offsets xml_i="44023" xml_f="44097" txt_i="22623" txt_f="22697">Effects of celecoxib treatment on extracellular matrix protein degradation</offsets></bold><offsets xml_i="44104" xml_f="44574" txt_i="22697" txt_f="23167">. Map designed on GenMAPP software with an overview of the genes involved in the extracellular matrix protein degradation process. The expression fold-changes of each gene are indicated next to the gene box. Genes highlighted in red and green in the left box half represent genes with fold-changes increased and decreased, respectively. Red colour in the right box half indicates a significant change. Grey boxes correspond to genes that were not analysed in the arrays.</offsets></p></caption><graphic xlink:href="bcr3409-4"></graphic></fig></sec><sec><title><offsets xml_i="44654" xml_f="44752" txt_i="23169" txt_f="23267">Recruitment of tumour-infiltrating leukocytes to breast carcinoma tissue after celecoxib treatment</offsets></title><p><offsets xml_i="44763" xml_f="44891" txt_i="23268" txt_f="23396">Breast cancer tissue of celecoxib-treated patients showed a significantly increased expression of MHC class II genes, including </offsets><italic><offsets xml_i="44899" xml_f="44907" txt_i="23396" txt_f="23404">HLA-DRα </offsets></italic><offsets xml_i="44916" xml_f="44920" txt_i="23404" txt_f="23408">and </offsets><italic><offsets xml_i="44928" xml_f="44943" txt_i="23408" txt_f="23423">HLA-DRβ2, CD74 </offsets></italic><offsets xml_i="44952" xml_f="44987" txt_i="23423" txt_f="23458">(MHC class II invariant chain) and </offsets><italic><offsets xml_i="44995" xml_f="45001" txt_i="23458" txt_f="23464">HLA-DM</offsets></italic><offsets xml_i="45010" xml_f="45020" txt_i="23464" txt_f="23474">, but not </offsets><italic><offsets xml_i="45028" xml_f="45035" txt_i="23474" txt_f="23481">HLA-DQ </offsets></italic><offsets xml_i="45044" xml_f="45048" txt_i="23481" txt_f="23485">and </offsets><italic><offsets xml_i="45056" xml_f="45064" txt_i="23485" txt_f="23493">HLA-DOA </offsets></italic><offsets xml_i="45073" xml_f="45094" txt_i="23493" txt_f="23514">[see Additional file </offsets><xref ref-type="supplementary-material" rid="S4"><offsets xml_i="45143" xml_f="45144" txt_i="23514" txt_f="23515">4</offsets></xref><offsets xml_i="45151" xml_f="45337" txt_i="23515" txt_f="23701">, Table S4]. MHC class I gene expression was not significantly changed. Subsequent to the increased expression of HLA-class II genes, co-stimulatory markers of antigen-presenting cells (</offsets><italic><offsets xml_i="45345" xml_f="45349" txt_i="23701" txt_f="23705">CD83</offsets></italic><offsets xml_i="45358" xml_f="45401" txt_i="23705" txt_f="23748">) and the monocyte differentiation antigen </offsets><italic><offsets xml_i="45409" xml_f="45414" txt_i="23748" txt_f="23753">CD14 </offsets></italic><offsets xml_i="45423" xml_f="45613" txt_i="23753" txt_f="23943">were up-regulated after celecoxib treatment. Infiltration of antigen-presenting cells was supported by increased expression of the Pattern Recognition Receptors (PRR), Toll-like receptor-2 (</offsets><italic><offsets xml_i="45621" xml_f="45625" txt_i="23943" txt_f="23947">TLR2</offsets></italic><offsets xml_i="45634" xml_f="45646" txt_i="23947" txt_f="23959">) and MD-1 (</offsets><italic><offsets xml_i="45654" xml_f="45658" txt_i="23959" txt_f="23963">LY86</offsets></italic><offsets xml_i="45667" xml_f="45719" txt_i="23963" txt_f="24015">). Gene expression of the classical B cell markers, </offsets><italic><offsets xml_i="45727" xml_f="45732" txt_i="24015" txt_f="24020">CD20 </offsets></italic><offsets xml_i="45741" xml_f="45745" txt_i="24020" txt_f="24024">and </offsets><italic><offsets xml_i="45753" xml_f="45757" txt_i="24024" txt_f="24028">CD19</offsets></italic><offsets xml_i="45766" xml_f="46015" txt_i="24028" txt_f="24277">, was not altered, although there was increased expression of immunoglobulin J chain. As myeloid-derived suppressor cells (MDSC) immune suppressive cells also express MHC class II molecules, and their presence correlates with COX-2 over-expression [</offsets><xref ref-type="bibr" rid="B36"><offsets xml_i="46047" xml_f="46049" txt_i="24277" txt_f="24279">36</offsets></xref><offsets xml_i="46056" xml_f="46192" txt_i="24279" txt_f="24415">], induction of MDSC signalling was investigated. However, MDSC induction seems unlikely, since the expression of MDSC-signalling genes </offsets><italic><offsets xml_i="46200" xml_f="46205" txt_i="24415" txt_f="24420">ARG1 </offsets></italic><offsets xml_i="46214" xml_f="46218" txt_i="24420" txt_f="24424">and </offsets><italic><offsets xml_i="46226" xml_f="46231" txt_i="24424" txt_f="24429">NOS2 </offsets></italic><offsets xml_i="46240" xml_f="46483" txt_i="24429" txt_f="24672">is not altered. Moreover, expression of important effector molecules, such as granzymes and perforin, was not affected. The increased infiltration of leukocytes observed in the breast tumour seems restricted to macrophages and dendritic cells.</offsets></p></sec><sec><title><offsets xml_i="46505" xml_f="46542" txt_i="24674" txt_f="24711">Change of tissue biomarker expression</offsets></title><p><offsets xml_i="46553" xml_f="47147" txt_i="24712" txt_f="25306">To confirm the transcriptional changes, we determined expression of protein markers for proliferation, apoptosis, and neo-angiogenesis. The proliferation marker Ki-67 was assessed on paired pre- and post-treatment tissues. Due to a lack of further tissue, apoptotic marker cleaved caspase-3 and neo-angiogenesis marker CD34 were assessed only on post-treatment tissues. Baseline Ki-67 positivity in the control group (geometric mean: 10.0%; 95% CI: 5.5 to 18.3) was not significantly different from baseline Ki-67 positivity in the treatment group (geometric mean: 13.4%; 95% CI: 9.8 to 18.3) (</offsets><italic><offsets xml_i="47155" xml_f="47157" txt_i="25306" txt_f="25308">P </offsets></italic><offsets xml_i="47166" xml_f="47277" txt_i="25308" txt_f="25419">= 0.915). The change in Ki-67 is shown for individual patients according to treatment or control arm in Figure </offsets><xref ref-type="fig" rid="F5"><offsets xml_i="47307" xml_f="47308" txt_i="25419" txt_f="25420">5</offsets></xref><offsets xml_i="47315" xml_f="47432" txt_i="25420" txt_f="25537">. The geometric mean change in Ki-67 relative to baseline in the treatment arm was -29.1% (95% CI: -40.1% to -16.2%, </offsets><italic><offsets xml_i="47440" xml_f="47442" txt_i="25537" txt_f="25539">P </offsets></italic><offsets xml_i="47451" xml_f="47525" txt_i="25539" txt_f="25613">= 0.019), whereas in the control arm it was -8.2% (95% CI: -23.1 to 9.5%, </offsets><italic><offsets xml_i="47533" xml_f="47535" txt_i="25613" txt_f="25615">P </offsets></italic><offsets xml_i="47544" xml_f="47620" txt_i="25615" txt_f="25691">= 0.384). There was a significant change difference between the two groups (</offsets><italic><offsets xml_i="47628" xml_f="47630" txt_i="25691" txt_f="25693">P </offsets></italic><offsets xml_i="47639" xml_f="47741" txt_i="25693" txt_f="25795">= 0.029). In contrast, the apoptotic index was not significantly different in post-treatment tissues (</offsets><italic><offsets xml_i="47749" xml_f="47751" txt_i="25795" txt_f="25797">P </offsets></italic><offsets xml_i="47760" xml_f="48042" txt_i="25797" txt_f="26079">= 0.231). The number of CD34 positive cells was slightly higher in celecoxib-treated tissues, but this was statistically insignificant. The geometric means (95% CI) of the 'Chalkley mean' value were 6.8 (5.4 to 8.5) in the control group and 7.7 (6.8 to 8.8) in the treatment group (</offsets><italic><offsets xml_i="48050" xml_f="48052" txt_i="26079" txt_f="26081">P </offsets></italic><offsets xml_i="48061" xml_f="48070" txt_i="26081" txt_f="26090">= 0.376).</offsets></p><fig id="F5" position="float"><label><offsets xml_i="48111" xml_f="48119" txt_i="26091" txt_f="26099">Figure 5</offsets></label><caption><p><bold><offsets xml_i="48145" xml_f="48203" txt_i="26099" txt_f="26157">Effects of celecoxib treatment on Ki-67 protein expression</offsets></bold><offsets xml_i="48210" xml_f="48212" txt_i="26157" txt_f="26159">. </offsets><bold><offsets xml_i="48218" xml_f="48222" txt_i="26159" txt_f="26163">(A) </offsets></bold><offsets xml_i="48229" xml_f="48392" txt_i="26163" txt_f="26326">Examples of immunohistochemical staining of nuclear Ki-67 protein on breast cancer tissues yielding a high score (left) and low score (right). Scale bar = 100 μm. </offsets><bold><offsets xml_i="48398" xml_f="48402" txt_i="26326" txt_f="26330">(B) </offsets></bold><offsets xml_i="48409" xml_f="48621" txt_i="26330" txt_f="26542">Shown are the Ki-67 scores from individual patients in the control arm (plot on left-hand side) and treatment arm (plot on right-hand side). Geometric means in the control group were statistically not different (</offsets><italic><offsets xml_i="48629" xml_f="48631" txt_i="26542" txt_f="26544">P </offsets></italic><offsets xml_i="48640" xml_f="48728" txt_i="26544" txt_f="26632">= 0.384), while the geometric mean after celecoxib treatment was significantly reduced (</offsets><italic><offsets xml_i="48736" xml_f="48738" txt_i="26632" txt_f="26634">P </offsets></italic><offsets xml_i="48747" xml_f="48865" txt_i="26634" txt_f="26752">= 0.019). Also, the change of the means between both groups was significantly greater in the celecoxib-treated group (</offsets><italic><offsets xml_i="48873" xml_f="48875" txt_i="26752" txt_f="26754">P </offsets></italic><offsets xml_i="48884" xml_f="48893" txt_i="26754" txt_f="26763">= 0.029).</offsets></p></caption><graphic xlink:href="bcr3409-5"></graphic></fig></sec></sec><sec sec-type="discussion"><title><offsets xml_i="49001" xml_f="49011" txt_i="26766" txt_f="26776">Discussion</offsets></title><p><offsets xml_i="49022" xml_f="49740" txt_i="26777" txt_f="27495">In this study, we analysed the transcriptional changes seen in primary breast cancer tissue following short-term celecoxib treatment. To accomplish this, we used global gene expression profiles from paired pre- and post-treatment specimens. After adjustment to the control group, we identified a large number of differentially expressed genes after treatment that are involved in the regulation of cancer-associated pathways, such as cell cycle and proliferation, ECM biology, and inflammatory response, amongst others. Most convincingly, COX-2 inhibition induced gene expression patterns indicative of a decelerated cell cycle and reduced proliferation. Celecoxib may induce G2/M arrest by p53 activation, leading to </offsets><italic><offsets xml_i="49748" xml_f="49756" txt_i="27495" txt_f="27503">GADD45A </offsets></italic><offsets xml_i="49765" xml_f="49871" txt_i="27503" txt_f="27609">up-regulation, which in turn inhibits cyclin-B1 and cyclin-B2 expression and promotes G2/M arrest (Figure </offsets><xref ref-type="fig" rid="F3"><offsets xml_i="49901" xml_f="49902" txt_i="27609" txt_f="27610">3</offsets></xref><offsets xml_i="49909" xml_f="50011" txt_i="27610" txt_f="27712">). A G2/M arrest is mainly forced after DNA damage to enable the initiation of DNA repair mechanisms [</offsets><xref ref-type="bibr" rid="B37"><offsets xml_i="50043" xml_f="50045" txt_i="27712" txt_f="27714">37</offsets></xref><offsets xml_i="50052" xml_f="50158" txt_i="27714" txt_f="27820">]. Our finding is in line with previous studies investigating the effects of celecoxib on cancerous cells </offsets><italic><offsets xml_i="50166" xml_f="50174" txt_i="27820" txt_f="27828">in vitro</offsets></italic><offsets xml_i="50183" xml_f="50305" txt_i="27828" txt_f="27950">. Dvory-Sobol and colleagues demonstrated that celecoxib induces G2/M arrest associated with cyclin-B1 down-regulation in </offsets><italic><offsets xml_i="50313" xml_f="50318" txt_i="27950" txt_f="27955">K-RAS</offsets></italic><offsets xml_i="50327" xml_f="50353" txt_i="27955" txt_f="27981">-transformed enterocytes [</offsets><xref ref-type="bibr" rid="B38"><offsets xml_i="50385" xml_f="50387" txt_i="27981" txt_f="27983">38</offsets></xref><offsets xml_i="50394" xml_f="50519" txt_i="27983" txt_f="28108">], and in the COX-2 expressing murine breast cancer cell line MCa-35, celecoxib induced a G2/M arrest followed by apoptosis [</offsets><xref ref-type="bibr" rid="B39"><offsets xml_i="50551" xml_f="50553" txt_i="28108" txt_f="28110">39</offsets></xref><offsets xml_i="50560" xml_f="50783" txt_i="28110" txt_f="28333">]. Interestingly, equally treated lung cancer A549 cells lacking COX-2 expression showed increased DNA damage, but low levels of apoptosis in these cells suggested a selective effect of celecoxib on COX-2 expressing cells [</offsets><xref ref-type="bibr" rid="B39"><offsets xml_i="50815" xml_f="50817" txt_i="28333" txt_f="28335">39</offsets></xref><offsets xml_i="50824" xml_f="50921" txt_i="28335" txt_f="28432">]. Celecoxib seems to increase DNA damage in irradiated cells, enhancing their radiosensitivity [</offsets><xref ref-type="bibr" rid="B40"><offsets xml_i="50953" xml_f="50955" txt_i="28432" txt_f="28434">40</offsets></xref><offsets xml_i="50962" xml_f="51378" txt_i="28434" txt_f="28850">]. However, the mechanisms behind increased DNA damage in celecoxib-treated tumour cells remains poorly understood. Our results are further consistent with a study of colorectal cancer (CRC) cell lines in which more than 1,000 genes were identified as differentially expressed after celecoxib treatment, clustering of which revealed significant changes of cell cycle control, apoptosis, and lymphocyte infiltration [</offsets><xref ref-type="bibr" rid="B41"><offsets xml_i="51410" xml_f="51412" txt_i="28850" txt_f="28852">41</offsets></xref><offsets xml_i="51419" xml_f="51550" txt_i="28852" txt_f="28983">]. Also in a study on primary CRC, celecoxib-induced gene expression changes significantly interfered with proliferation pathways [</offsets><xref ref-type="bibr" rid="B42"><offsets xml_i="51582" xml_f="51584" txt_i="28983" txt_f="28985">42</offsets></xref><offsets xml_i="51591" xml_f="51804" txt_i="28985" txt_f="29198">]. In summary, we have confirmed a positive treatment effect of COX-2 inhibition on cell proliferation-related transcriptional programs in primary breast carcinomas, as has been previously demonstrated by several </offsets><italic><offsets xml_i="51812" xml_f="51821" txt_i="29198" txt_f="29207">in vitro </offsets></italic><offsets xml_i="51830" xml_f="51834" txt_i="29207" txt_f="29211">and </offsets><italic><offsets xml_i="51842" xml_f="51850" txt_i="29211" txt_f="29219">in vivo </offsets></italic><offsets xml_i="51859" xml_f="51867" txt_i="29219" txt_f="29227">studies.</offsets></p><p><offsets xml_i="51874" xml_f="52236" txt_i="29228" txt_f="29590">Disruption of the basement membrane is a hallmark of malignancy. Degradative enzymes, such as MMPs, are produced by tumour cells and by resident and infiltrating cells as a response to the tumour, and contribute to matrix degradation and facilitate tumour invasion. MMP-2, MMP-9, and other members of the MMP family have been associated with tumour progression [</offsets><xref ref-type="bibr" rid="B43"><offsets xml_i="52268" xml_f="52270" txt_i="29590" txt_f="29592">43</offsets></xref><offsets xml_i="52277" xml_f="52418" txt_i="29592" txt_f="29733">]. In particular, MMP-2 and MMP-9 activity appears to be inhibited by celecoxib in our study, the first by up-regulation of MMP antagonists (</offsets><italic><offsets xml_i="52426" xml_f="52438" txt_i="29733" txt_f="29745">TIMP1, TIMP2</offsets></italic><offsets xml_i="52447" xml_f="52453" txt_i="29745" txt_f="29751">, and </offsets><italic><offsets xml_i="52461" xml_f="52466" txt_i="29751" txt_f="29756">TIMP3</offsets></italic><offsets xml_i="52475" xml_f="52529" txt_i="29756" txt_f="29810">), the latter by up-regulation of the MMP-9 inhibitor </offsets><italic><offsets xml_i="52537" xml_f="52541" txt_i="29810" txt_f="29814">RECK</offsets></italic><offsets xml_i="52550" xml_f="52734" txt_i="29814" txt_f="29998">. An involvement of selective COX-2 inhibition in matrix stability by decreasing MMP activity and tumour invasiveness has been previously demonstrated in breast and CRC cancer models [</offsets><xref ref-type="bibr" rid="B44"><offsets xml_i="52766" xml_f="52768" txt_i="29998" txt_f="30000">44</offsets></xref><offsets xml_i="52775" xml_f="52776" txt_i="30000" txt_f="30001">,</offsets><xref ref-type="bibr" rid="B45"><offsets xml_i="52808" xml_f="52810" txt_i="30001" txt_f="30003">45</offsets></xref><offsets xml_i="52817" xml_f="52863" txt_i="30003" txt_f="30049">], thus being in good agreement with our data.</offsets></p><p><offsets xml_i="52870" xml_f="53031" txt_i="30050" txt_f="30211">Several lines of evidence demonstrated that immune cell infiltration in tumours is enhanced by celecoxib treatment, which is associated with a better prognosis [</offsets><xref ref-type="bibr" rid="B46"><offsets xml_i="53063" xml_f="53065" txt_i="30211" txt_f="30213">46</offsets></xref><offsets xml_i="53072" xml_f="53073" txt_i="30213" txt_f="30214">,</offsets><xref ref-type="bibr" rid="B47"><offsets xml_i="53105" xml_f="53107" txt_i="30214" txt_f="30216">47</offsets></xref><offsets xml_i="53114" xml_f="53491" txt_i="30216" txt_f="30593">]. In our study, increased infiltration of antigen-presenting cells is supported by gene expression data whereas other immune cells of both the innate and adaptive immune system do not seem to be affected by celecoxib treatment. Most up-regulated genes within this category belong to MHC class II. Comparable data on MHC class I and II induction have been reported by Lönnroth </offsets><italic><offsets xml_i="53499" xml_f="53504" txt_i="30593" txt_f="30598">et al</offsets></italic><offsets xml_i="53513" xml_f="53546" txt_i="30598" txt_f="30631">. in CRC patients using a NSAID [</offsets><xref ref-type="bibr" rid="B47"><offsets xml_i="53578" xml_f="53580" txt_i="30631" txt_f="30633">47</offsets></xref><offsets xml_i="53587" xml_f="53589" txt_i="30633" txt_f="30635">].</offsets></p><p><offsets xml_i="53596" xml_f="54068" txt_i="30636" txt_f="31108">In order to investigate whether observed gene expression changes after COX-2 inhibition have translated to a biologically relevant effect, we analysed protein markers for proliferation, apoptosis, and neo-angiogenesis in primary tissues. Suppression of the proliferation marker Ki-67 has been previously reported as a surrogate marker for decreased aromatase activity in oestrogen receptor (ER) positive breast cancer treated with aromatase inhibitors (AI) for two weeks [</offsets><xref ref-type="bibr" rid="B48"><offsets xml_i="54100" xml_f="54102" txt_i="31108" txt_f="31110">48</offsets></xref><offsets xml_i="54109" xml_f="54110" txt_i="31110" txt_f="31111">,</offsets><xref ref-type="bibr" rid="B49"><offsets xml_i="54142" xml_f="54144" txt_i="31111" txt_f="31113">49</offsets></xref><offsets xml_i="54151" xml_f="54233" txt_i="31113" txt_f="31195">]. Since COX-2 expression is positively correlated with tumour aromatase content [</offsets><xref ref-type="bibr" rid="B50"><offsets xml_i="54265" xml_f="54267" txt_i="31195" txt_f="31197">50</offsets></xref><offsets xml_i="54274" xml_f="54408" txt_i="31197" txt_f="31331">], we were interested whether COX-2 inhibition would also lead to a reduction in Ki-67 positivity. As expected from previous studies [</offsets><xref ref-type="bibr" rid="B51"><offsets xml_i="54440" xml_f="54442" txt_i="31331" txt_f="31333">51</offsets></xref><offsets xml_i="54449" xml_f="54450" txt_i="31333" txt_f="31334">,</offsets><xref ref-type="bibr" rid="B52"><offsets xml_i="54482" xml_f="54484" txt_i="31334" txt_f="31336">52</offsets></xref><offsets xml_i="54491" xml_f="54868" txt_i="31336" txt_f="31713">], Ki-67 was not significantly reduced in the control arm. In contrast, the celecoxib arm showed a significant suppression of Ki-67, confirming the reduced proliferation observed in our gene expression data and suggesting an indirect treatment effect on aromatase activity. Although the Ki-67 suppressive effect was only modest (-29%) as compared to the AI anastrozole (-75%) [</offsets><xref ref-type="bibr" rid="B49"><offsets xml_i="54900" xml_f="54902" txt_i="31713" txt_f="31715">49</offsets></xref><offsets xml_i="54909" xml_f="55006" txt_i="31715" txt_f="31812">], it was similar to the Ki-67 suppression achieved with another AI, that is, raloxifene (-24%) [</offsets><xref ref-type="bibr" rid="B52"><offsets xml_i="55038" xml_f="55040" txt_i="31812" txt_f="31814">52</offsets></xref><offsets xml_i="55047" xml_f="55445" txt_i="31814" txt_f="32212">]. Other than Ki-67, caspase-3 and CD34 were not significantly changed after celecoxib treatment, although this has to be interpreted cautiously due to the lack of baseline data. However, in a previous neo-adjuvant study in breast cancer, two weeks of celecoxib did not result in a biological response of proliferation and apoptosis, as determined by Ki-67 staining and TUNEL assays, respectively [</offsets><xref ref-type="bibr" rid="B53"><offsets xml_i="55477" xml_f="55479" txt_i="32212" txt_f="32214">53</offsets></xref><offsets xml_i="55486" xml_f="55971" txt_i="32214" txt_f="32699">]. Notably, the referenced study analysed fewer patients and used half the drug dose that we applied. Taken together, we hypothesise that two weeks of COX-2 inhibition may not be sufficient to translate all transcriptional activation to a measurable biological phenotype. In particular, suppression of blood vessel development may take effect only after longer drug exposure, which should be taken into consideration when designing future clinical trials of COX-2 inhibition in cancer.</offsets></p><p><offsets xml_i="55978" xml_f="56070" txt_i="32700" txt_f="32792">In breast cancer, COX-2 over-expression is positively associated with HER2 over-expression [</offsets><xref ref-type="bibr" rid="B54"><offsets xml_i="56102" xml_f="56104" txt_i="32792" txt_f="32794">54</offsets></xref><offsets xml_i="56111" xml_f="56148" txt_i="32794" txt_f="32831">] and with tumour aromatase content [</offsets><xref ref-type="bibr" rid="B50"><offsets xml_i="56180" xml_f="56182" txt_i="32831" txt_f="32833">50</offsets></xref><offsets xml_i="56189" xml_f="56477" txt_i="32833" txt_f="33121">]. Therefore, COX-2 inhibition might prove beneficial, especially in combination with trastuzumab in HER2 positive breast cancer or with AI in hormone receptor (HR) positive disease. In trastuzumab-refractory metastatic breast cancer COX-2 inhibition was previously shown to be inactive [</offsets><xref ref-type="bibr" rid="B55"><offsets xml_i="56509" xml_f="56511" txt_i="33121" txt_f="33123">55</offsets></xref><offsets xml_i="56518" xml_f="56734" txt_i="33123" txt_f="33339">]. However, improved efficacy and endpoint benefits of celecoxib in combination with AI were reported in post-menopausal metastatic breast cancer, although these were pronounced only in tamoxifen resistant patients [</offsets><xref ref-type="bibr" rid="B56"><offsets xml_i="56766" xml_f="56768" txt_i="33339" txt_f="33341">56</offsets></xref><offsets xml_i="56775" xml_f="56776" txt_i="33341" txt_f="33342">,</offsets><xref ref-type="bibr" rid="B57"><offsets xml_i="56808" xml_f="56810" txt_i="33342" txt_f="33344">57</offsets></xref><offsets xml_i="56817" xml_f="56987" txt_i="33344" txt_f="33514">]. Although promising, further studies are needed in order to elucidate a clinical benefit of COX-2 inhibition in combination with other drugs in breast cancer treatment.</offsets></p><p><offsets xml_i="56994" xml_f="57369" txt_i="33515" txt_f="33890">The strength of our study is the trial design, which allows patients to act as their own control, and the inclusion of a placebo group, which served to exclude potentially confounding effects by the disease and the experimental procedures. It would be interesting to sub-analyse the gene expression and Ki-67 in stratified breast cancer subtypes, as for example, HR positive </offsets><italic><offsets xml_i="57377" xml_f="57384" txt_i="33890" txt_f="33897">versus </offsets></italic><offsets xml_i="57393" xml_f="57490" txt_i="33897" txt_f="33994">negative disease, but this is hindered by the low number of HR negative patients in the control (</offsets><italic><offsets xml_i="57498" xml_f="57500" txt_i="33994" txt_f="33996">n </offsets></italic><offsets xml_i="57509" xml_f="57537" txt_i="33996" txt_f="34024">= 5) and the treatment arm (</offsets><italic><offsets xml_i="57545" xml_f="57547" txt_i="34024" txt_f="34026">n </offsets></italic><offsets xml_i="57556" xml_f="58011" txt_i="34026" txt_f="34481">= 1). Of note, however, there is a trend of bias towards poorer prognostic factors in the celecoxib arm, which could not have been predicted or changed before the end of the treatment procedure. Although this bias must be considered a limitation of the study, we speculate that the observed anti-tumour transcriptional response in this arm may have been under-estimated, and thus could become even more evident by comparing clinically highly similar arms.</offsets></p><p><offsets xml_i="58018" xml_f="58581" txt_i="34482" txt_f="35045">Unfortunately, eight patients dropped out of the study due to early surgery, all of them from the treatment arm, resulting in a reduced statistical power to detect differentially expressed genes. However, in our microarray analysis we found more differentially expressed genes than expected by chance alone (1,558 observed versus 920 expected). After identification of the differentially expressed genes, pathway analyses were performed in order to identify genes clustering within the same biological pathway, as those genes are very likely to be true positives.</offsets></p><p><offsets xml_i="58588" xml_f="58829" txt_i="35046" txt_f="35287">Large meta-analyses have recently shown that the non-selective COX inhibitor aspirin is protective against cancer development. Regular intake of aspirin significantly reduced the risk of several cancers, including breast cancer (OR = 0.88) [</offsets><xref ref-type="bibr" rid="B58"><offsets xml_i="58861" xml_f="58863" txt_i="35287" txt_f="35289">58</offsets></xref><offsets xml_i="58870" xml_f="58981" txt_i="35289" txt_f="35400">], and three years onwards of daily aspirin intake reduced cancer incidence in both women and men (OR = 0.75) [</offsets><xref ref-type="bibr" rid="B59"><offsets xml_i="59013" xml_f="59015" txt_i="35400" txt_f="35402">59</offsets></xref><offsets xml_i="59022" xml_f="59503" txt_i="35402" txt_f="35883">]. These studies suggest that selective COX-2 inhibitors may have similar effects on cancer incidence albeit with the advantage of causing less adverse side effects associated with aspirin use, such as gastrointestinal bleeding. COX-1 is expressed constitutively in many different tissues, whereas COX-2 is conditionally induced, such as in inflammation, suggesting that selective COX-2 inhibition might prove more successful in cancer prevention than non-selective COX inhibitors.</offsets></p></sec><sec sec-type="conclusions"><title><offsets xml_i="59548" xml_f="59559" txt_i="35885" txt_f="35896">Conclusions</offsets></title><p><offsets xml_i="59570" xml_f="59903" txt_i="35897" txt_f="36230">Our clinical trial provides substantial evidence for an anti-tumour activity of celecoxib based on global transcriptional changes and suppression of Ki-67 protein in primary breast cancer tissues, encouraging further clinical trials of celecoxib or its derivative molecules in breast cancer. Our study confirms results from previous </offsets><italic><offsets xml_i="59911" xml_f="59920" txt_i="36230" txt_f="36239">in vitro </offsets></italic><offsets xml_i="59929" xml_f="59933" txt_i="36239" txt_f="36243">and </offsets><italic><offsets xml_i="59941" xml_f="59949" txt_i="36243" txt_f="36251">in vivo </offsets></italic><offsets xml_i="59958" xml_f="60635" txt_i="36251" txt_f="36928">studies as we found a large number of cell-cycle and proliferation-associated genes to be differentially expressed in celecoxib-treated primary breast cancer tissues. Our aim was to determine the global transcriptional response to short-term COX-2 inhibition in primary tumours, and our results indicate that proliferation and ECM biology are significantly influenced, possibly underlying the proposed anti-tumour activity. Future trials of COX-2 inhibition considering other primary endpoints, such as pathological or clinical response, should take into account that effects of a transcriptional response may need a longer time to translate into a measurable clinical benefit.</offsets></p></sec><sec><title><offsets xml_i="60657" xml_f="60670" txt_i="36930" txt_f="36943">Abbreviations</offsets></title><p><offsets xml_i="60681" xml_f="61267" txt_i="36944" txt_f="37530">AI: aromatase inhibitor; AIC: Akaike information criterion; COX-2: cyclooxygenase-2; CRC: colorectal cancer; DAVID: Database for Annotation, Visualisation and Integrated Discovery; ECM: extracellular matrix; ER: oestrogen receptor; FFPE: formalin-fixed and paraffin-embedded; GEP: gene expression profiling; GO: gene ontology; HR: hormone receptor; MDSC: myeloid-derived suppressor cells; MMP: matrix metalloproteinase; NSAID: non-steroidal anti-inflammatory drug; PG: prostaglandin; qPCR: quantitative polymerase chain reaction; RT-PCR: reverse transcriptase polymerase chain reaction.</offsets></p></sec><sec><title><offsets xml_i="61289" xml_f="61308" txt_i="37532" txt_f="37551">Competing interests</offsets></title><p><offsets xml_i="61319" xml_f="61377" txt_i="37552" txt_f="37610">The authors declare that they have no competing interests.</offsets></p></sec><sec><title><offsets xml_i="61399" xml_f="61421" txt_i="37612" txt_f="37634">Authors' contributions</offsets></title><p><offsets xml_i="61432" xml_f="62729" txt_i="37635" txt_f="38932">RDB performed microarray data analyses, participated in qPCR validation, data interpretation and co-drafted the manuscript. JV participated in qPCR validation, data interpretation, statistical analyses of immunohistochemical results and co-drafted the manuscript. KKVdV performed pathological review and immunohistochemical scoring, participated in data interpretation and critically revised the manuscript. PL provided statistical analyses of microarray data and critically revised the manuscript. BdV provided support in immunohistochemical scoring and critically revised the manuscript. CHMJvE performed immunohistochemical staining, participated in data interpretation and critically revised the manuscript. MJB provided support for the data mining and critically revised the manuscript. KK provided surgical support in tissue acquisition and critically revised the manuscript. TA performed microarray analyses and critically revised the manuscript. HJMS participated in the conception and design of the study and critically revised the manuscript. VCTH participated in the conception and design of the study and critically revised the manuscript. PSH conceived and coordinated the study and critically revised the manuscript. All authors read and approved the final version of the manuscript.</offsets></p></sec><sec sec-type="supplementary-material"><title><offsets xml_i="62785" xml_f="62807" txt_i="38934" txt_f="38956">Supplementary Material</offsets></title><supplementary-material content-type="local-data" id="S1"><caption><title><offsets xml_i="62889" xml_f="62906" txt_i="38957" txt_f="38974">Additional file 1</offsets></title><p><bold><offsets xml_i="62923" xml_f="62985" txt_i="38975" txt_f="39037">Table S1 showing the primer sequences used for qPCR validation</offsets></bold><offsets xml_i="62992" xml_f="62993" txt_i="39037" txt_f="39038">.</offsets></p></caption><media xlink:href="bcr3409-S1.DOC"><caption><p><offsets xml_i="63054" xml_f="63073" txt_i="39039" txt_f="39058">Click here for file</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="S2"><caption><title><offsets xml_i="63194" xml_f="63211" txt_i="39059" txt_f="39076">Additional file 2</offsets></title><p><bold><offsets xml_i="63228" xml_f="63305" txt_i="39077" txt_f="39154">Table S2 showing the 50 most significantly up-regulated genes after treatment</offsets></bold><offsets xml_i="63312" xml_f="63313" txt_i="39154" txt_f="39155">.</offsets></p></caption><media xlink:href="bcr3409-S2.DOC"><caption><p><offsets xml_i="63374" xml_f="63393" txt_i="39156" txt_f="39175">Click here for file</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="S3"><caption><title><offsets xml_i="63514" xml_f="63531" txt_i="39176" txt_f="39193">Additional file 3</offsets></title><p><bold><offsets xml_i="63548" xml_f="63627" txt_i="39194" txt_f="39273">Table S3 showing the 50 most significantly down-regulated genes after treatment</offsets></bold><offsets xml_i="63634" xml_f="63635" txt_i="39273" txt_f="39274">.</offsets></p></caption><media xlink:href="bcr3409-S3.DOC"><caption><p><offsets xml_i="63696" xml_f="63715" txt_i="39275" txt_f="39294">Click here for file</offsets></p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="S4"><caption><title><offsets xml_i="63836" xml_f="63853" txt_i="39295" txt_f="39312">Additional file 4</offsets></title><p><bold><offsets xml_i="63870" xml_f="63968" txt_i="39313" txt_f="39411">Table S4 showing genes of the immune surveillance mechanism and their deregulation after treatment</offsets></bold><offsets xml_i="63975" xml_f="63976" txt_i="39411" txt_f="39412">.</offsets></p></caption><media xlink:href="bcr3409-S4.DOC"><caption><p><offsets xml_i="64037" xml_f="64056" txt_i="39413" txt_f="39432">Click here for file</offsets></p></caption></media></supplementary-material></sec></body><back><sec><title>Acknowledgements</title><p>We are grateful to the patients who participated in this trial. The expert technical help of Sabina Vanherle is greatly appreciated. This work was supported by a grant obtained by CORE Fund - 2002 (financed by Pharmacia &amp; Upjohn; Pfizer since 2003) and the Fundação para a Ciência e Tecnologia (SFRH/BD/32386/2006 to R.D.B), co-financed by the POPH and European Community Fund.</p></sec><ref-list><ref id="B1"><mixed-citation publication-type="journal"><name><surname>O'Byrne</surname><given-names>KJ</given-names></name><name><surname>Dalgleish</surname><given-names>AG</given-names></name><article-title>Chronic immune activation and inflammation as the cause of malignancy</article-title><source>Br J Cancer</source><year>2001</year><volume>84</volume><fpage>473</fpage><lpage>483</lpage><pub-id pub-id-type="pmid">11506482</pub-id></mixed-citation></ref><ref id="B2"><mixed-citation publication-type="journal"><name><surname>Grivennikov</surname><given-names>SI</given-names></name><name><surname>Greten</surname><given-names>FR</given-names></name><name><surname>Karin</surname><given-names>M</given-names></name><article-title>Immunity, inflammation, and cancer</article-title><source>Cell</source><year></year><volume>140</volume><fpage>883</fpage><lpage>899</lpage><pub-id pub-id-type="doi">10.1016/j.cell.2010.01.025</pub-id><pub-id pub-id-type="pmid">20303878</pub-id></mixed-citation></ref><ref id="B3"><mixed-citation publication-type="journal"><name><surname>Wang</surname><given-names>D</given-names></name><name><surname>DuBois</surname><given-names>RN</given-names></name><article-title>Eicosanoids and cancer</article-title><source>Nat Rev Cancer</source><year>2010</year><volume>10</volume><fpage>181</fpage><lpage>193</lpage><pub-id pub-id-type="doi">10.1038/nrc2809</pub-id><pub-id pub-id-type="pmid">20168319</pub-id></mixed-citation></ref><ref id="B4"><mixed-citation publication-type="journal"><name><surname>Friedman</surname><given-names>G</given-names></name><name><surname>Ury</surname><given-names>H</given-names></name><article-title>Initial screening for carcinogenicity of commonly used drugs</article-title><source>J Natl Cancer Inst</source><year>1980</year><volume>65</volume><fpage>723</fpage><lpage>733</lpage><pub-id pub-id-type="pmid">6932525</pub-id></mixed-citation></ref><ref id="B5"><mixed-citation publication-type="journal"><name><surname>Harris</surname><given-names>RE</given-names></name><name><surname>Namboodiri</surname><given-names>KK</given-names></name><name><surname>Farrar</surname><given-names>WB</given-names></name><article-title>Nonsteroidal antiinflammatory drugs and breast cancer</article-title><source>Epidemiology</source><year>1996</year><volume>7</volume><fpage>203</fpage><lpage>205</lpage><pub-id pub-id-type="doi">10.1097/00001648-199603000-00017</pub-id><pub-id pub-id-type="pmid">8834563</pub-id></mixed-citation></ref><ref id="B6"><mixed-citation publication-type="journal"><name><surname>Thun</surname><given-names>MJ</given-names></name><name><surname>Namboodiri</surname><given-names>MM</given-names></name><name><surname>Heath</surname><given-names>CW</given-names></name><article-title>Aspirin use and reduced risk of fatal colon cancer</article-title><source>N Engl J Med</source><year>1991</year><volume>325</volume><fpage>1593</fpage><lpage>1596</lpage><pub-id pub-id-type="doi">10.1056/NEJM199112053252301</pub-id><pub-id pub-id-type="pmid">1669840</pub-id></mixed-citation></ref><ref id="B7"><mixed-citation publication-type="journal"><name><surname>Giardiello</surname><given-names>FM</given-names></name><name><surname>Hamilton</surname><given-names>SR</given-names></name><name><surname>Krush</surname><given-names>AJ</given-names></name><name><surname>Piantadosi</surname><given-names>S</given-names></name><name><surname>Hylind</surname><given-names>LM</given-names></name><name><surname>Celano</surname><given-names>P</given-names></name><name><surname>Booker</surname><given-names>SV</given-names></name><name><surname>Robinson</surname><given-names>CR</given-names></name><name><surname>Offerhaus</surname><given-names>GJ</given-names></name><article-title>Treatment of colonic and rectal adenomas with sulindac in familial adenomatous polyposis</article-title><source>N Engl J Med</source><year>1993</year><volume>328</volume><fpage>1313</fpage><lpage>1316</lpage><pub-id pub-id-type="doi">10.1056/NEJM199305063281805</pub-id><pub-id pub-id-type="pmid">8385741</pub-id></mixed-citation></ref><ref id="B8"><mixed-citation publication-type="journal"><name><surname>Giovannucci</surname><given-names>E</given-names></name><name><surname>Egan</surname><given-names>KM</given-names></name><name><surname>Hunter</surname><given-names>DJ</given-names></name><name><surname>Stampfer</surname><given-names>MJ</given-names></name><name><surname>Colditz</surname><given-names>GA</given-names></name><name><surname>Willett</surname><given-names>WC</given-names></name><name><surname>Speizer</surname><given-names>FE</given-names></name><article-title>Aspirin and the risk of colorectal cancer in women</article-title><source>N Engl J Med</source><year>1995</year><volume>333</volume><fpage>609</fpage><lpage>614</lpage><pub-id pub-id-type="doi">10.1056/NEJM199509073331001</pub-id><pub-id pub-id-type="pmid">7637720</pub-id></mixed-citation></ref><ref id="B9"><mixed-citation publication-type="journal"><name><surname>Schreinemachers</surname><given-names>DM</given-names></name><name><surname>Everson</surname><given-names>RB</given-names></name><article-title>Aspirin use and lung, colon, and breast cancer incidence in a prospective study</article-title><source>Epidemiology</source><year>1994</year><volume>5</volume><fpage>138</fpage><lpage>146</lpage><pub-id pub-id-type="doi">10.1097/00001648-199403000-00003</pub-id><pub-id pub-id-type="pmid">8172988</pub-id></mixed-citation></ref><ref id="B10"><mixed-citation publication-type="journal"><name><surname>Sharpe</surname><given-names>CR</given-names></name><name><surname>Collet</surname><given-names>JP</given-names></name><name><surname>McNutt</surname><given-names>M</given-names></name><name><surname>Belzile</surname><given-names>E</given-names></name><name><surname>Boivin</surname><given-names>J-F</given-names></name><name><surname>Hanley</surname><given-names>JA</given-names></name><article-title>Nested case-control study of the effects of non-steroidal anti-inflammatory drugs on breast cancer risk and stage</article-title><source>Br J Cancer</source><year>2000</year><volume>83</volume><fpage>112</fpage><lpage>120</lpage><pub-id pub-id-type="pmid">10883678</pub-id></mixed-citation></ref><ref id="B11"><mixed-citation publication-type="journal"><name><surname>Baron</surname><given-names>JA</given-names></name><name><surname>Sandler</surname><given-names>RS</given-names></name><name><surname>Bresalier</surname><given-names>RS</given-names></name><name><surname>Quan</surname><given-names>H</given-names></name><name><surname>Riddell</surname><given-names>R</given-names></name><name><surname>Lanas</surname><given-names>A</given-names></name><name><surname>Bolognese</surname><given-names>JA</given-names></name><name><surname>Oxenius</surname><given-names>B</given-names></name><name><surname>Horgan</surname><given-names>K</given-names></name><name><surname>Loftus</surname><given-names>S</given-names></name><name><surname>Morton</surname><given-names>DG</given-names></name><collab>APPROVe Trial Investigators</collab><article-title>A randomized trial of rofecoxib for the chemoprevention of colorectal adenomas</article-title><source>Gastroenterology</source><year>2006</year><volume>131</volume><fpage>1674</fpage><lpage>1682</lpage><pub-id pub-id-type="doi">10.1053/j.gastro.2006.08.079</pub-id><pub-id pub-id-type="pmid">17087947</pub-id></mixed-citation></ref><ref id="B12"><mixed-citation publication-type="journal"><name><surname>Funk</surname><given-names>CD</given-names></name><article-title>Prostaglandins and leukotrienes: advances in eicosanoid biology</article-title><source>Science</source><year>2001</year><volume>294</volume><fpage>1871</fpage><lpage>1875</lpage><pub-id pub-id-type="doi">10.1126/science.294.5548.1871</pub-id><pub-id pub-id-type="pmid">11729303</pub-id></mixed-citation></ref><ref id="B13"><mixed-citation publication-type="journal"><name><surname>Bennett</surname><given-names>A</given-names></name><name><surname>McDonald</surname><given-names>AM</given-names></name><name><surname>Stamford</surname><given-names>IF</given-names></name><name><surname>Charlier</surname><given-names>EM</given-names></name><name><surname>Simpson</surname><given-names>JS</given-names></name><name><surname>Zebro</surname><given-names>T</given-names></name><article-title>Prostaglandins and breast cancer</article-title><source>Lancet</source><year>1977</year><volume>310</volume><fpage>624</fpage><lpage>626</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(77)92496-5</pub-id><pub-id pub-id-type="pmid">71446</pub-id></mixed-citation></ref><ref id="B14"><mixed-citation publication-type="journal"><name><surname>Smith</surname><given-names>CJ</given-names></name><name><surname>Sun</surname><given-names>D</given-names></name><name><surname>Hoegler</surname><given-names>C</given-names></name><name><surname>Roth</surname><given-names>BS</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Zhao</surname><given-names>G</given-names></name><name><surname>Xu</surname><given-names>XB</given-names></name><name><surname>Kobari</surname><given-names>Y</given-names></name><name><surname>Pritchard</surname><given-names>K</given-names><suffix>Jr</suffix></name><name><surname>Sessa</surname><given-names>WC</given-names></name><name><surname>Hintze</surname><given-names>TH</given-names></name><article-title>Reduced gene expression of vascular endothelial NO synthase and cyclooxygenase-1 in heart failure</article-title><source>Circ Res</source><year>1996</year><volume>78</volume><fpage>58</fpage><lpage>64</lpage><pub-id pub-id-type="doi">10.1161/01.RES.78.1.58</pub-id><pub-id pub-id-type="pmid">8603506</pub-id></mixed-citation></ref><ref id="B15"><mixed-citation publication-type="journal"><name><surname>Smith</surname><given-names>WL</given-names></name><name><surname>DeWitt</surname><given-names>DL</given-names></name><name><surname>Garavito</surname><given-names>RM</given-names></name><article-title>Cyclooxygenases: structural, cellular, and molecular biology</article-title><source>Annu Rev Biochem</source><year>2000</year><volume>69</volume><fpage>145</fpage><lpage>182</lpage><pub-id pub-id-type="doi">10.1146/annurev.biochem.69.1.145</pub-id><pub-id pub-id-type="pmid">10966456</pub-id></mixed-citation></ref><ref id="B16"><mixed-citation publication-type="journal"><name><surname>Kujubu</surname><given-names>DA</given-names></name><name><surname>Fletcher</surname><given-names>BS</given-names></name><name><surname>Varnum</surname><given-names>BC</given-names></name><name><surname>Lim</surname><given-names>RW</given-names></name><name><surname>Herschman</surname><given-names>HR</given-names></name><article-title>TIS10, a phorbol ester tumor promoter-inducible mRNA from Swiss 3T3 cells, encodes a novel prostaglandin synthase/cyclooxygenase homologue</article-title><source>J Biol Chem</source><year>1991</year><volume>266</volume><fpage>12866</fpage><lpage>12872</lpage><pub-id pub-id-type="pmid">1712772</pub-id></mixed-citation></ref><ref id="B17"><mixed-citation publication-type="journal"><name><surname>Xie</surname><given-names>WL</given-names></name><name><surname>Chipman</surname><given-names>JG</given-names></name><name><surname>Robertson</surname><given-names>DL</given-names></name><name><surname>Erikson</surname><given-names>RL</given-names></name><name><surname>Simmons</surname><given-names>DL</given-names></name><article-title>Expression of a mitogen-responsive gene encoding prostaglendin synthase is regulated by mRNA splicing</article-title><source>Proc Natl Acad Sci USA</source><year>1991</year><volume>88</volume><fpage>2692</fpage><lpage>2696</lpage><pub-id pub-id-type="doi">10.1073/pnas.88.7.2692</pub-id><pub-id pub-id-type="pmid">1849272</pub-id></mixed-citation></ref><ref id="B18"><mixed-citation publication-type="journal"><name><surname>Soslow</surname><given-names>RA</given-names></name><name><surname>Dannenberg</surname><given-names>AJ</given-names></name><name><surname>Rush</surname><given-names>D</given-names></name><name><surname>Woerner</surname><given-names>BM</given-names></name><name><surname>Khan</surname><given-names>KN</given-names></name><name><surname>Masferrer</surname><given-names>J</given-names></name><name><surname>Koki</surname><given-names>AT</given-names></name><article-title>COX-2 is expressed in human pulmonary, colonic, and mammary tumors</article-title><source>Cancer</source><year>2000</year><volume>89</volume><fpage>2637</fpage><lpage>2645</lpage><pub-id pub-id-type="doi">10.1002/1097-0142(20001215)89:12&lt;2637::AID-CNCR17&gt;3.0.CO;2-B</pub-id><pub-id pub-id-type="pmid">11135226</pub-id></mixed-citation></ref><ref id="B19"><mixed-citation publication-type="journal"><name><surname>Liu</surname><given-names>CH</given-names></name><name><surname>Chang</surname><given-names>SH</given-names></name><name><surname>Narko</surname><given-names>K</given-names></name><name><surname>Trifan</surname><given-names>OC</given-names></name><name><surname>Wu</surname><given-names>MT</given-names></name><name><surname>Smith</surname><given-names>E</given-names></name><name><surname>Haudenschild</surname><given-names>C</given-names></name><name><surname>Lane</surname><given-names>TF</given-names></name><name><surname>Hla</surname><given-names>T</given-names></name><article-title>Overexpression of cyclooxygenase-2 Is sufficient to induce tumorigenesis in transgenic mice</article-title><source>J Biol Chem</source><year>2001</year><volume>276</volume><fpage>18563</fpage><lpage>18569</lpage><pub-id pub-id-type="doi">10.1074/jbc.M010787200</pub-id><pub-id pub-id-type="pmid">11278747</pub-id></mixed-citation></ref><ref id="B20"><mixed-citation publication-type="book"><name><surname>Tavassoli</surname><given-names>FA</given-names></name><name><surname>Devilee</surname><given-names>P</given-names></name><source>Pathology and Genetics of Tumours of the Breast and Female Genital Organs</source><year>2003</year><publisher-name>Lyon: IARC Press</publisher-name></mixed-citation></ref><ref id="B21"><mixed-citation publication-type="book"><name><surname>Sobin</surname><given-names>LH</given-names></name><name><surname>Wittekind</surname><given-names>C</given-names></name><source>TNM Classification of Malignant Tumours</source><year>2002</year><edition>6</edition><publisher-name>New York: Wiley-Liss</publisher-name></mixed-citation></ref><ref id="B22"><mixed-citation publication-type="journal"><name><surname>Elston</surname><given-names>EW</given-names></name><name><surname>Ellis</surname><given-names>IO</given-names></name><article-title>Method for grading breast cancer</article-title><source>J Clin Pathol</source><year>1993</year><volume>46</volume><fpage>189</fpage><lpage>190</lpage><pub-id pub-id-type="pmid">8459046</pub-id></mixed-citation></ref><ref id="B23"><mixed-citation publication-type="other"><article-title>ArrayExpress</article-title><ext-link ext-link-type="uri" xlink:href="http://www.ebi.ac.uk/arrayexpress">http://www.ebi.ac.uk/arrayexpress</ext-link></mixed-citation></ref><ref id="B24"><mixed-citation publication-type="journal"><name><surname>Dai</surname><given-names>M</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name><name><surname>Boyd</surname><given-names>AD</given-names></name><name><surname>Kostov</surname><given-names>G</given-names></name><name><surname>Athey</surname><given-names>B</given-names></name><name><surname>Jones</surname><given-names>EG</given-names></name><name><surname>Bunney</surname><given-names>WE</given-names></name><name><surname>Myers</surname><given-names>RM</given-names></name><name><surname>Speed</surname><given-names>TP</given-names></name><name><surname>Akil</surname><given-names>H</given-names></name><name><surname>Watson</surname><given-names>SJ</given-names></name><name><surname>Meng</surname><given-names>F</given-names></name><article-title>Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data</article-title><source>Nucleic Acids Res</source><year>2005</year><volume>33</volume><fpage>e175</fpage><pub-id pub-id-type="doi">10.1093/nar/gni179</pub-id><pub-id pub-id-type="pmid">16284200</pub-id></mixed-citation></ref><ref id="B25"><mixed-citation publication-type="book"><name><surname>Akaike</surname><given-names>H</given-names></name><person-group person-group-type="editor">Petrov B, Csàki F</person-group><article-title>Information theory and an extension of the maximum likelihood principle</article-title><source>Second International Symposium of Inference Theory</source><year>1973</year><publisher-name>Budapest: Akadémiai Kiadó</publisher-name><fpage>267</fpage><lpage>281</lpage></mixed-citation></ref><ref id="B26"><mixed-citation publication-type="journal"><name><surname>Ihaka</surname><given-names>R</given-names></name><name><surname>Gentleman</surname><given-names>R</given-names></name><article-title>R: a language for data analysis and graphics</article-title><source>Journal of Computational and Graphical Statistics</source><year>1996</year><volume>5</volume><fpage>299</fpage><lpage>314</lpage></mixed-citation></ref><ref id="B27"><mixed-citation publication-type="book"><name><surname>Lindsey</surname><given-names>J</given-names></name><source>Models for Repeated Measurements</source><year>1999</year><edition>2</edition><publisher-name>Oxford: Oxford University Press</publisher-name></mixed-citation></ref><ref id="B28"><mixed-citation publication-type="journal"><name><surname>Dennis</surname><given-names>G</given-names></name><name><surname>Sherman</surname><given-names>B</given-names></name><name><surname>Hosack</surname><given-names>D</given-names></name><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Gao</surname><given-names>W</given-names></name><name><surname>Lane</surname><given-names>HC</given-names></name><name><surname>Lempicki</surname><given-names>R</given-names></name><article-title>DAVID: Database for Annotation, Visualization, and Integrated Discovery</article-title><source>Genome Biol</source><year>2003</year><volume>4</volume><fpage>3</fpage><pub-id pub-id-type="doi">10.1186/gb-2003-4-5-p3</pub-id></mixed-citation></ref><ref id="B29"><mixed-citation publication-type="journal"><name><surname>van Iersel</surname><given-names>M</given-names></name><name><surname>Kelder</surname><given-names>T</given-names></name><name><surname>Pico</surname><given-names>A</given-names></name><name><surname>Hanspers</surname><given-names>K</given-names></name><name><surname>Coort</surname><given-names>S</given-names></name><name><surname>Conklin</surname><given-names>B</given-names></name><name><surname>Evelo</surname><given-names>C</given-names></name><article-title>Presenting and exploring biological pathways with PathVisio</article-title><source>BMC Bioinformatics</source><year>2008</year><volume>9</volume><fpage>399</fpage><pub-id pub-id-type="doi">10.1186/1471-2105-9-399</pub-id><pub-id pub-id-type="pmid">18817533</pub-id></mixed-citation></ref><ref id="B30"><mixed-citation publication-type="journal"><name><surname>Pico</surname><given-names>AR</given-names></name><name><surname>Kelder</surname><given-names>T</given-names></name><name><surname>van Iersel</surname><given-names>MP</given-names></name><name><surname>Hanspers</surname><given-names>K</given-names></name><name><surname>Conklin</surname><given-names>BR</given-names></name><name><surname>Evelo</surname><given-names>C</given-names></name><article-title>WikiPathways: pathway editing for the people</article-title><source>PLoS Biol</source><year>2008</year><volume>6</volume><fpage>e184</fpage><pub-id pub-id-type="doi">10.1371/journal.pbio.0060184</pub-id><pub-id pub-id-type="pmid">18651794</pub-id></mixed-citation></ref><ref id="B31"><mixed-citation publication-type="journal"><name><surname>Harper-Wynne</surname><given-names>C</given-names></name><name><surname>Ross</surname><given-names>G</given-names></name><name><surname>Sacks</surname><given-names>N</given-names></name><name><surname>Salter</surname><given-names>J</given-names></name><name><surname>Nasiri</surname><given-names>N</given-names></name><name><surname>Iqbal</surname><given-names>J</given-names></name><name><surname>A'Hern</surname><given-names>R</given-names></name><name><surname>Dowsett</surname><given-names>M</given-names></name><article-title>Effects of the aromatase inhibitor letrozole on normal breast epithelial cell proliferation and metabolic indices in postmenopausal women: a pilot study for breast cancer prevention</article-title><source>Cancer Epidemiol Biomarkers Prev</source><year>2002</year><volume>11</volume><fpage>614</fpage><lpage>621</lpage><pub-id pub-id-type="pmid">12101108</pub-id></mixed-citation></ref><ref id="B32"><mixed-citation publication-type="journal"><name><surname>Kaiser</surname><given-names>CL</given-names></name><name><surname>Chapman</surname><given-names>BJ</given-names></name><name><surname>Guidi</surname><given-names>JL</given-names></name><name><surname>Terry</surname><given-names>CE</given-names></name><name><surname>Mangiardi</surname><given-names>DA</given-names></name><name><surname>Cotanche</surname><given-names>DA</given-names></name><article-title>Comparison of activated caspase detection methods in the gentamicin-treated chick cochlea</article-title><source>Hear Res</source><year>2008</year><volume>240</volume><fpage>1</fpage><lpage>11</lpage><pub-id pub-id-type="doi">10.1016/j.heares.2008.03.003</pub-id><pub-id pub-id-type="pmid">18487027</pub-id></mixed-citation></ref><ref id="B33"><mixed-citation publication-type="journal"><name><surname>da Silva</surname><given-names>BB</given-names></name><name><surname>Pires</surname><given-names>CG</given-names></name><name><surname>dos Santos</surname><given-names>AR</given-names></name><name><surname>Castro-Leão</surname><given-names>AHd</given-names></name><name><surname>Alencar</surname><given-names>AP</given-names></name><name><surname>Lopes-Costa</surname><given-names>PV</given-names></name><article-title>Effects of raloxifene on Ki-67 and CD34 antigen expression in breast cancer</article-title><source>Gynecol Obstet Invest</source><year>2009</year><volume>67</volume><fpage>103</fpage><lpage>108</lpage><pub-id pub-id-type="doi">10.1159/000165512</pub-id><pub-id pub-id-type="pmid">18953168</pub-id></mixed-citation></ref><ref id="B34"><mixed-citation publication-type="journal"><name><surname>Chalkley</surname><given-names>H</given-names></name><article-title>Method for the quantitative morphologic analysis of tissues</article-title><source>J Natl Cancer Inst</source><year>1943</year><volume>4</volume><fpage>47</fpage><lpage>53</lpage></mixed-citation></ref><ref id="B35"><mixed-citation publication-type="journal"><name><surname>Sato</surname><given-names>H</given-names></name><name><surname>Kinoshita</surname><given-names>T</given-names></name><name><surname>Takino</surname><given-names>T</given-names></name><name><surname>Nakayama</surname><given-names>K</given-names></name><name><surname>Seiki</surname><given-names>M</given-names></name><article-title>Activation of a recombinant membrane type 1-matrix metalloproteinase (MT1-MMP) by furin and its interaction with tissue inhibitor of metalloproteinases (TIMP)-2</article-title><source>FEBS Lett</source><year>1996</year><volume>393</volume><fpage>101</fpage><lpage>104</lpage><pub-id pub-id-type="doi">10.1016/0014-5793(96)00861-7</pub-id><pub-id pub-id-type="pmid">8804434</pub-id></mixed-citation></ref><ref id="B36"><mixed-citation publication-type="journal"><name><surname>Talmadge</surname><given-names>JE</given-names></name><article-title>Pathways mediating the expansion and immunosuppressive activity of myeloid-derived suppressor cells and their relevance to cancer therapy</article-title><source>Clin Cancer Res</source><year>2007</year><volume>13</volume><fpage>5243</fpage><lpage>5248</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-07-0182</pub-id><pub-id pub-id-type="pmid">17875751</pub-id></mixed-citation></ref><ref id="B37"><mixed-citation publication-type="journal"><name><surname>Callegari</surname><given-names>AJ</given-names></name><name><surname>Kelly</surname><given-names>TJ</given-names></name><article-title>Shedding light on the DNA damage checkpoint</article-title><source>Cell Cycle</source><year>2007</year><volume>6</volume><fpage>660</fpage><lpage>666</lpage><pub-id pub-id-type="doi">10.4161/cc.6.6.3984</pub-id><pub-id pub-id-type="pmid">17387276</pub-id></mixed-citation></ref><ref id="B38"><mixed-citation publication-type="journal"><name><surname>Dvory-Sobol</surname><given-names>H</given-names></name><name><surname>Cohen-Noyman</surname><given-names>E</given-names></name><name><surname>Kazanov</surname><given-names>D</given-names></name><name><surname>Figer</surname><given-names>A</given-names></name><name><surname>Birkenfeld</surname><given-names>S</given-names></name><name><surname>Madar-Shapiro</surname><given-names>L</given-names></name><name><surname>Benamouzig</surname><given-names>R</given-names></name><name><surname>Arber</surname><given-names>N</given-names></name><article-title>Celecoxib leads to G2/M arrest by induction of p21 and down-regulation of cyclin B1 expression in a p53-independent manner</article-title><source>Eur J Cancer</source><year>2006</year><volume>42</volume><fpage>422</fpage><lpage>426</lpage><pub-id pub-id-type="doi">10.1016/j.ejca.2005.11.009</pub-id><pub-id pub-id-type="pmid">16426843</pub-id></mixed-citation></ref><ref id="B39"><mixed-citation publication-type="journal"><name><surname>Liu</surname><given-names>W</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Keng</surname><given-names>P</given-names></name><name><surname>Finkelstein</surname><given-names>J</given-names></name><name><surname>Hu</surname><given-names>D</given-names></name><name><surname>Liang</surname><given-names>L</given-names></name><name><surname>Guo</surname><given-names>M</given-names></name><name><surname>Fenton</surname><given-names>B</given-names></name><name><surname>Okunieff</surname><given-names>P</given-names></name><name><surname>Ding</surname><given-names>I</given-names></name><article-title>Combination of radiation and celebrex (celecoxib) reduce mammary and lung tumor growth</article-title><source>Am J Clin Oncol</source><year>2003</year><volume>26</volume><fpage>103</fpage><lpage>109</lpage><pub-id pub-id-type="pmid">12576935</pub-id></mixed-citation></ref><ref id="B40"><mixed-citation publication-type="journal"><name><surname>Dittmann</surname><given-names>KH</given-names></name><name><surname>Mayer</surname><given-names>C</given-names></name><name><surname>Ohneseit</surname><given-names>PA</given-names></name><name><surname>Raju</surname><given-names>U</given-names></name><name><surname>Andratschke</surname><given-names>NH</given-names></name><name><surname>Milas</surname><given-names>L</given-names></name><name><surname>Rodemann</surname><given-names>HP</given-names></name><article-title>Celecoxib induced tumor cell radiosensitization by inhibiting radiation induced nuclear EGFR transport and DNA-repair: a COX-2 independent mechanism</article-title><source>Int J Radiat Oncol Biol Phys</source><year>2008</year><volume>70</volume><fpage>203</fpage><lpage>212</lpage><pub-id pub-id-type="doi">10.1016/j.ijrobp.2007.08.065</pub-id><pub-id pub-id-type="pmid">17996386</pub-id></mixed-citation></ref><ref id="B41"><mixed-citation publication-type="journal"><name><surname>Fatima</surname><given-names>N</given-names></name><name><surname>Yi</surname><given-names>M</given-names></name><name><surname>Ajaz</surname><given-names>S</given-names></name><name><surname>Stephens</surname><given-names>RM</given-names></name><name><surname>Stauffer</surname><given-names>S</given-names></name><name><surname>Greenwald</surname><given-names>P</given-names></name><name><surname>Munroe</surname><given-names>DJ</given-names></name><name><surname>Ali</surname><given-names>IU</given-names></name><article-title>Altered gene expression profiles define pathways in colorectal cancer cell lines affected by celecoxib</article-title><source>Cancer Epidemiol Biomarkers Prev</source><year>2008</year><volume>17</volume><fpage>3051</fpage><lpage>3061</lpage><pub-id pub-id-type="doi">10.1158/1055-9965.EPI-08-0261</pub-id><pub-id pub-id-type="pmid">18957522</pub-id></mixed-citation></ref><ref id="B42"><mixed-citation publication-type="journal"><name><surname>Auman</surname><given-names>JT</given-names></name><name><surname>Church</surname><given-names>R</given-names></name><name><surname>Lee</surname><given-names>SY</given-names></name><name><surname>Watson</surname><given-names>MA</given-names></name><name><surname>Fleshman</surname><given-names>JW</given-names></name><name><surname>McLeod</surname><given-names>HL</given-names></name><article-title>Celecoxib pre-treatment in human colorectal adenocarcinoma patients is associated with gene expression alterations suggestive of diminished cellular proliferation</article-title><source>Eur J Cancer</source><year>2008</year><volume>44</volume><fpage>1754</fpage><lpage>1760</lpage><pub-id pub-id-type="doi">10.1016/j.ejca.2008.05.010</pub-id><pub-id pub-id-type="pmid">18653328</pub-id></mixed-citation></ref><ref id="B43"><mixed-citation publication-type="journal"><name><surname>Orlichenko</surname><given-names>L</given-names></name><name><surname>Radisky</surname><given-names>D</given-names></name><article-title>Matrix metalloproteinases stimulate epithelial-mesenchymal transition during tumor development</article-title><source>Clin Exp Metastasis</source><year>2008</year><volume>25</volume><fpage>593</fpage><lpage>600</lpage><pub-id pub-id-type="doi">10.1007/s10585-008-9143-9</pub-id><pub-id pub-id-type="pmid">18286378</pub-id></mixed-citation></ref><ref id="B44"><mixed-citation publication-type="journal"><name><surname>Wang</surname><given-names>S</given-names></name><name><surname>Liu</surname><given-names>Q</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Liu</surname><given-names>K</given-names></name><name><surname>Yu</surname><given-names>P</given-names></name><name><surname>Liu</surname><given-names>K</given-names></name><name><surname>Luan</surname><given-names>J</given-names></name><name><surname>Duan</surname><given-names>H</given-names></name><name><surname>Lu</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>F</given-names></name><name><surname>Wu</surname><given-names>E</given-names></name><name><surname>Yagasaki</surname><given-names>K</given-names></name><name><surname>Zhang</surname><given-names>G</given-names></name><article-title>Suppression of growth, migration and invasion of highly-metastatic human breast cancer cells by berbamine and its molecular mechanisms of action</article-title><source>Mol Cancer</source><year>2009</year><volume>8</volume><fpage>81</fpage><pub-id pub-id-type="doi">10.1186/1476-4598-8-81</pub-id><pub-id pub-id-type="pmid">19796390</pub-id></mixed-citation></ref><ref id="B45"><mixed-citation publication-type="journal"><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Chen</surname><given-names>W</given-names></name><name><surname>Xie</surname><given-names>X</given-names></name><name><surname>He</surname><given-names>Y</given-names></name><name><surname>Bai</surname><given-names>X</given-names></name><article-title>Celecoxib inhibits tumor growth and angiogenesis in an orthotopic implantation tumor model of human colon cancer</article-title><source>Exp Oncol</source><year>2008</year><volume>30</volume><fpage>42</fpage><lpage>51</lpage><pub-id pub-id-type="pmid">18438340</pub-id></mixed-citation></ref><ref id="B46"><mixed-citation publication-type="journal"><name><surname>Lang</surname><given-names>S</given-names></name><name><surname>Lauffer</surname><given-names>L</given-names></name><name><surname>Clausen</surname><given-names>C</given-names></name><name><surname>Lohr</surname><given-names>I</given-names></name><name><surname>Schmitt</surname><given-names>B</given-names></name><name><surname>Holzel</surname><given-names>D</given-names></name><name><surname>Wollenberg</surname><given-names>B</given-names></name><name><surname>Gires</surname><given-names>O</given-names></name><name><surname>Kastenbauer</surname><given-names>E</given-names></name><name><surname>Zeidler</surname><given-names>R</given-names></name><article-title>Impaired monocyte function in cancer patients: restoration with a cyclooxygenase-2 inhibitor</article-title><source>FASEB J</source><year>2003</year><volume>17</volume><fpage>286</fpage><lpage>288</lpage><pub-id pub-id-type="pmid">12490541</pub-id></mixed-citation></ref><ref id="B47"><mixed-citation publication-type="journal"><name><surname>Lonnroth</surname><given-names>C</given-names></name><name><surname>Andersson</surname><given-names>M</given-names></name><name><surname>Arvidsson</surname><given-names>A</given-names></name><name><surname>Nordgren</surname><given-names>S</given-names></name><name><surname>Brevinge</surname><given-names>H</given-names></name><name><surname>Lagerstedt</surname><given-names>K</given-names></name><name><surname>Lundholm</surname><given-names>K</given-names></name><article-title>Preoperative treatment with a non-steroidal anti-inflammatory drug (NSAID) increases tumor tissue infiltration of seemingly activated immune cells in colorectal cancer</article-title><source>Cancer Immun</source><year>2008</year><volume>8</volume><fpage>5</fpage><pub-id pub-id-type="pmid">18307280</pub-id></mixed-citation></ref><ref id="B48"><mixed-citation publication-type="journal"><name><surname>Harper-Wynne</surname><given-names>CL</given-names></name><name><surname>Sacks</surname><given-names>NP</given-names></name><name><surname>Shenton</surname><given-names>K</given-names></name><name><surname>MacNeill</surname><given-names>FA</given-names></name><name><surname>Sauven</surname><given-names>P</given-names></name><name><surname>Laidlaw</surname><given-names>IJ</given-names></name><name><surname>Rayter</surname><given-names>Z</given-names></name><name><surname>Miall</surname><given-names>S</given-names></name><name><surname>Howes</surname><given-names>A</given-names></name><name><surname>Salter</surname><given-names>J</given-names></name><name><surname>Hills</surname><given-names>MJ</given-names></name><name><surname>Lowe</surname><given-names>FM</given-names></name><name><surname>A'Hern</surname><given-names>R</given-names></name><name><surname>Nasiri</surname><given-names>N</given-names></name><name><surname>Doody</surname><given-names>D</given-names></name><name><surname>Iqbal</surname><given-names>J</given-names></name><name><surname>Dowsett</surname><given-names>M</given-names></name><article-title>Comparison of the systemic and intratumoral effects of tamoxifen and the aromatase inhibitor vorozole in postmenopausal patients with primary breast cancer</article-title><source>J Clin Oncol</source><year>2002</year><volume>20</volume><fpage>1026</fpage><lpage>1035</lpage><pub-id pub-id-type="doi">10.1200/JCO.20.4.1026</pub-id><pub-id pub-id-type="pmid">11844826</pub-id></mixed-citation></ref><ref id="B49"><mixed-citation publication-type="journal"><name><surname>Smith</surname><given-names>IE</given-names></name><name><surname>Dowsett</surname><given-names>M</given-names></name><name><surname>Ebbs</surname><given-names>SR</given-names></name><name><surname>Dixon</surname><given-names>JM</given-names></name><name><surname>Skene</surname><given-names>A</given-names></name><name><surname>Blohmer</surname><given-names>JU</given-names></name><name><surname>Ashley</surname><given-names>SE</given-names></name><name><surname>Francis</surname><given-names>S</given-names></name><name><surname>Boeddinghaus</surname><given-names>I</given-names></name><name><surname>Walsh</surname><given-names>G</given-names></name><article-title>Neoadjuvant treatment of postmenopausal breast cancer with anastrozole, tamoxifen, or both in combination: the Immediate Preoperative Anastrozole, Tamoxifen, or Combined with Tamoxifen (IMPACT) multicenter double-blind randomized trial</article-title><source>J Clin Oncol</source><year>2005</year><volume>23</volume><fpage>5108</fpage><lpage>5116</lpage><pub-id pub-id-type="doi">10.1200/JCO.2005.04.005</pub-id><pub-id pub-id-type="pmid">15998903</pub-id></mixed-citation></ref><ref id="B50"><mixed-citation publication-type="journal"><name><surname>Brueggemeier</surname><given-names>R</given-names></name><name><surname>Quinn</surname><given-names>A</given-names></name><name><surname>Parrett</surname><given-names>M</given-names></name><name><surname>Joarder</surname><given-names>F</given-names></name><name><surname>Harris</surname><given-names>R</given-names></name><name><surname>Robertson</surname><given-names>F</given-names></name><article-title>Correlation of aromatase and cyclooxygenase gene expression in human breast cancer specimens</article-title><source>Cancer Lett</source><year>1999</year><volume>140</volume><fpage>27</fpage><lpage>35</lpage><pub-id pub-id-type="doi">10.1016/S0304-3835(99)00050-6</pub-id><pub-id pub-id-type="pmid">10403538</pub-id></mixed-citation></ref><ref id="B51"><mixed-citation publication-type="journal"><name><surname>Dowsett</surname><given-names>M</given-names></name><name><surname>Ebbs</surname><given-names>SR</given-names></name><name><surname>Dixon</surname><given-names>JM</given-names></name><name><surname>Skene</surname><given-names>A</given-names></name><name><surname>Griffith</surname><given-names>C</given-names></name><name><surname>Boeddinghaus</surname><given-names>I</given-names></name><name><surname>Salter</surname><given-names>J</given-names></name><name><surname>Detre</surname><given-names>S</given-names></name><name><surname>Hills</surname><given-names>M</given-names></name><name><surname>Ashley</surname><given-names>S</given-names></name><name><surname>Francis</surname><given-names>S</given-names></name><name><surname>Walsh</surname><given-names>G</given-names></name><name><surname>Smith</surname><given-names>IE</given-names></name><article-title>Biomarker changes during neoadjuvant anastrozole, tamoxifen, or the combination: influence of hormonal status and HER-2 in breast cancer - a study from the IMPACT trialists</article-title><source>J Clin Oncol</source><year>2005</year><volume>23</volume><fpage>2477</fpage><lpage>2492</lpage><pub-id pub-id-type="doi">10.1200/JCO.2005.07.559</pub-id><pub-id pub-id-type="pmid">15767642</pub-id></mixed-citation></ref><ref id="B52"><mixed-citation publication-type="journal"><name><surname>Dowsett</surname><given-names>M</given-names></name><name><surname>Bundred</surname><given-names>NJ</given-names></name><name><surname>Decensi</surname><given-names>A</given-names></name><name><surname>Sainsbury</surname><given-names>RC</given-names></name><name><surname>Lu</surname><given-names>Y</given-names></name><name><surname>Hills</surname><given-names>MJ</given-names></name><name><surname>Cohen</surname><given-names>FJ</given-names></name><name><surname>Veronesi</surname><given-names>P</given-names></name><name><surname>O'Brien</surname><given-names>ME</given-names></name><name><surname>Scott</surname><given-names>T</given-names></name><name><surname>Muchmore</surname><given-names>DB</given-names></name><article-title>Effect of raloxifene on breast cancer cell Ki67 and apoptosis: a double-blind, placebo-controlled, randomized clinical trial in postmenopausal patients</article-title><source>Cancer Epidemiol Biomarkers Prev</source><year>2001</year><volume>10</volume><fpage>961</fpage><lpage>966</lpage><pub-id pub-id-type="pmid">11535548</pub-id></mixed-citation></ref><ref id="B53"><mixed-citation publication-type="journal"><name><surname>Martin</surname><given-names>LA</given-names></name><name><surname>Davies</surname><given-names>G</given-names></name><name><surname>Weigel</surname><given-names>M</given-names></name><name><surname>Betambeau</surname><given-names>N</given-names></name><name><surname>Hills</surname><given-names>M</given-names></name><name><surname>Salter</surname><given-names>J</given-names></name><name><surname>Walsh</surname><given-names>G</given-names></name><name><surname>A'Hern</surname><given-names>R</given-names></name><name><surname>Dowsett</surname><given-names>M</given-names></name><article-title>Pre-surgical study of the biological effects of the selective cyclo-oxygenase-2 inhibitor celecoxib in patients with primary breast cancer</article-title><source>Breast Cancer Res Treat</source><year>2010</year><volume>123</volume><fpage>829</fpage><lpage>836</lpage><pub-id pub-id-type="doi">10.1007/s10549-010-1100-z</pub-id><pub-id pub-id-type="pmid">20697803</pub-id></mixed-citation></ref><ref id="B54"><mixed-citation publication-type="journal"><name><surname>Subbaramaiah</surname><given-names>K</given-names></name><name><surname>Norton</surname><given-names>L</given-names></name><name><surname>Gerald</surname><given-names>W</given-names></name><name><surname>Dannenberg</surname><given-names>AJ</given-names></name><article-title>Cyclooxygenase-2 is overexpressed in HER-2/neu-positive breast cancer: evidence for involvement of AP-1 and PEA3</article-title><source>J Biol Chem</source><year>2002</year><volume>277</volume><fpage>18649</fpage><lpage>18657</lpage><pub-id pub-id-type="doi">10.1074/jbc.M111415200</pub-id><pub-id pub-id-type="pmid">11901151</pub-id></mixed-citation></ref><ref id="B55"><mixed-citation publication-type="journal"><name><surname>Dang</surname><given-names>CT</given-names></name><name><surname>Dannenberg</surname><given-names>AJ</given-names></name><name><surname>Subbaramaiah</surname><given-names>K</given-names></name><name><surname>Dickler</surname><given-names>MN</given-names></name><name><surname>Moasser</surname><given-names>MM</given-names></name><name><surname>Seidman</surname><given-names>AD</given-names></name><name><surname>D'Andrea</surname><given-names>GM</given-names></name><name><surname>Theodoulou</surname><given-names>M</given-names></name><name><surname>Panageas</surname><given-names>KS</given-names></name><name><surname>Norton</surname><given-names>L</given-names></name><name><surname>Hudis</surname><given-names>CA</given-names></name><article-title>Phase II study of celecoxib and trastuzumab in metastatic breast cancer patients who have progressed after prior trastuzumab-based treatments</article-title><source>Clin Cancer Res</source><year>2004</year><volume>10</volume><fpage>4062</fpage><lpage>4067</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-03-0463</pub-id><pub-id pub-id-type="pmid">15217939</pub-id></mixed-citation></ref><ref id="B56"><mixed-citation publication-type="journal"><name><surname>Dirix</surname><given-names>LY</given-names></name><name><surname>Ignacio</surname><given-names>J</given-names></name><name><surname>Nag</surname><given-names>S</given-names></name><name><surname>Bapsy</surname><given-names>P</given-names></name><name><surname>Gomez</surname><given-names>H</given-names></name><name><surname>Raghunadharao</surname><given-names>D</given-names></name><name><surname>Paridaens</surname><given-names>R</given-names></name><name><surname>Jones</surname><given-names>S</given-names></name><name><surname>Falcon</surname><given-names>S</given-names></name><name><surname>Carpentieri</surname><given-names>M</given-names></name><name><surname>Abbattista</surname><given-names>A</given-names></name><name><surname>Lobelle</surname><given-names>JP</given-names></name><article-title>Treatment of advanced hormone-sensitive breast cancer in postmenopausal women with exemestane alone or in combination with celecoxib</article-title><source>J Clin Oncol</source><year>2008</year><volume>26</volume><fpage>1253</fpage><lpage>1259</lpage><pub-id pub-id-type="doi">10.1200/JCO.2007.13.3744</pub-id><pub-id pub-id-type="pmid">18323548</pub-id></mixed-citation></ref><ref id="B57"><mixed-citation publication-type="journal"><name><surname>Falandry</surname><given-names>C</given-names></name><name><surname>Canney</surname><given-names>PA</given-names></name><name><surname>Freyer</surname><given-names>G</given-names></name><name><surname>Dirix</surname><given-names>LY</given-names></name><article-title>Role of combination therapy with aromatase and cyclooxygenase-2 inhibitors in patients with metastatic breast cancer</article-title><source>Ann Oncol</source><year>2009</year><volume>20</volume><fpage>615</fpage><lpage>620</lpage><pub-id pub-id-type="doi">10.1093/annonc/mdn693</pub-id><pub-id pub-id-type="pmid">19254941</pub-id></mixed-citation></ref><ref id="B58"><mixed-citation publication-type="journal"><name><surname>Algra</surname><given-names>AM</given-names></name><name><surname>Rothwell</surname><given-names>PM</given-names></name><article-title>Effects of regular aspirin on long-term cancer incidence and metastasis: a systematic comparison of evidence from observational studies versus randomised trials</article-title><source>Lancet Oncol</source><year>2012</year><volume>13</volume><fpage>518</fpage><lpage>527</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(12)70112-2</pub-id><pub-id pub-id-type="pmid">22440112</pub-id></mixed-citation></ref><ref id="B59"><mixed-citation publication-type="journal"><name><surname>Rothwell</surname><given-names>PM</given-names></name><name><surname>Price</surname><given-names>JF</given-names></name><name><surname>Fowkes</surname><given-names>FG</given-names></name><name><surname>Zanchetti</surname><given-names>A</given-names></name><name><surname>Roncaglioni</surname><given-names>MC</given-names></name><name><surname>Tognoni</surname><given-names>G</given-names></name><name><surname>Lee</surname><given-names>R</given-names></name><name><surname>Belch</surname><given-names>JF</given-names></name><name><surname>Wilson</surname><given-names>M</given-names></name><name><surname>Mehta</surname><given-names>Z</given-names></name><name><surname>Meade</surname><given-names>TW</given-names></name><article-title>Short-term effects of daily aspirin on cancer incidence, mortality, and non-vascular death: analysis of the time course of risks and benefits in 51 randomised controlled trials</article-title><source>Lancet</source><year>2012</year><volume>379</volume><fpage>1602</fpage><lpage>1612</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(11)61720-0</pub-id><pub-id pub-id-type="pmid">22440946</pub-id></mixed-citation></ref><ref id="B60"><mixed-citation publication-type="journal"><name><surname>Snead</surname><given-names>D</given-names></name><name><surname>Bell</surname><given-names>J</given-names></name><name><surname>Dixon</surname><given-names>A</given-names></name><name><surname>Nicholson</surname><given-names>R</given-names></name><name><surname>Elston</surname><given-names>C</given-names></name><name><surname>Blamey</surname><given-names>R</given-names></name><name><surname>Ellis</surname><given-names>I</given-names></name><article-title>Methodology of immunohistological detection of oestrogen receptor in human breast carcinoma in formalin-fixed, paraffin-embedded tissue: a comparison with frozen section methodology</article-title><source>Histopathology</source><year>1993</year><volume>23</volume><fpage>233</fpage><lpage>238</lpage><pub-id pub-id-type="doi">10.1111/j.1365-2559.1993.tb01195.x</pub-id><pub-id pub-id-type="pmid">8225241</pub-id></mixed-citation></ref></ref-list></back></article>