<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article" xml:lang="en"><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Cancer</journal-id><journal-id journal-id-type="iso-abbrev">BMC Cancer</journal-id><journal-title-group><journal-title>BMC Cancer</journal-title></journal-title-group><issn pub-type="epub">1471-2407</issn><publisher><publisher-name>BioMed Central</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">22876876</article-id><article-id pub-id-type="pmc">3442969</article-id><article-id pub-id-type="publisher-id">1471-2407-12-349</article-id><article-id pub-id-type="doi">10.1186/1471-2407-12-349</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>Effect of <italic>KRAS</italic> codon13 mutations in patients with advanced colorectal cancer (advanced CRC) under oxaliplatin containing chemotherapy. Results from a translational study of the AIO colorectal study group</article-title></title-group><contrib-group><contrib contrib-type="author" corresp="yes" id="A1"><name><surname>Reinacher-Schick</surname><given-names>Anke</given-names></name><xref ref-type="aff" rid="I1">1</xref><xref ref-type="aff" rid="I9">9</xref><xref ref-type="aff" rid="I10">10</xref><email>Anke.Reinacher@rub.de</email></contrib><contrib contrib-type="author" id="A2"><name><surname>Schulmann</surname><given-names>Karsten</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>karsten.schulmann@rub.de</email></contrib><contrib contrib-type="author" id="A3"><name><surname>Modest</surname><given-names>Dominik P</given-names></name><xref ref-type="aff" rid="I2">2</xref><email>dominik.modest@med.uni-muenchen.de</email></contrib><contrib contrib-type="author" id="A4"><name><surname>Bruns</surname><given-names>Nina</given-names></name><xref ref-type="aff" rid="I1">1</xref><email>nina.bruns@web.de</email></contrib><contrib contrib-type="author" id="A5"><name><surname>Graeven</surname><given-names>Ulrich</given-names></name><xref ref-type="aff" rid="I3">3</xref><email>ullrich.graeven@mariahilf.de</email></contrib><contrib contrib-type="author" id="A6"><name><surname>Jaworska</surname><given-names>Malgorzata</given-names></name><xref ref-type="aff" rid="I4">4</xref><email>malgorzata.jaworska@rub.de</email></contrib><contrib contrib-type="author" id="A7"><name><surname>Greil</surname><given-names>Richard</given-names></name><xref ref-type="aff" rid="I5">5</xref><email>greil@salk.at</email></contrib><contrib contrib-type="author" id="A8"><name><surname>Porschen</surname><given-names>Rainer</given-names></name><xref ref-type="aff" rid="I6">6</xref><email>rainer.porschen@klinikum-bremen-ost.de</email></contrib><contrib contrib-type="author" id="A9"><name><surname>Arnold</surname><given-names>Dirk</given-names></name><xref ref-type="aff" rid="I7">7</xref><email>d.arnold@uke.de</email></contrib><contrib contrib-type="author" id="A10"><name><surname>Schmiegel</surname><given-names>Wolff</given-names></name><xref ref-type="aff" rid="I1">1</xref><xref ref-type="aff" rid="I8">8</xref><xref ref-type="aff" rid="I9">9</xref><email>wolff.schmiegel@rub.de</email></contrib><contrib contrib-type="author" id="A11"><name><surname>Tannapfel</surname><given-names>Andrea</given-names></name><xref ref-type="aff" rid="I4">4</xref><email>andrea.tannapfel@rub.de</email></contrib></contrib-group><aff id="I1"><label>1</label>Department of Internal Medicine, Knappschaftskrankenhaus, Ruhr-University Bochum, Bochum, Germany</aff><aff id="I2"><label>2</label>Department of Internal Medicine III, Klinikum der Universität, München, Germany</aff><aff id="I3"><label>3</label>Department of Hematology, Oncology and Gastroenterology, Kliniken Maria Hilf, Mönchengladbach, Germany</aff><aff id="I4"><label>4</label>Institute of Pathology, Ruhr-University Bochum, Bochum, Germany</aff><aff id="I5"><label>5</label>3rd Medical Department with Hematology and Medical Oncology, Oncologic Centre Paracelsus Medical University, Salzburg, Austria</aff><aff id="I6"><label>6</label>Klinikum Bremen-Ost, Bremen, Germany</aff><aff id="I7"><label>7</label>Hubertus Wald Cancer Center, University Hospital Hamburg, Hamburg, Germany</aff><aff id="I8"><label>8</label>Department of Gastroenterology &amp; Hepatology, Berufsgenossenschaftliches Klinikum Bergmannsheil, Ruhr-University Bochum, Bochum, Germany</aff><aff id="I9"><label>9</label>Center for Clinical Studies in Oncology within PURE, Ruhr-University Bochum, Bochum, Germany</aff><aff id="I10"><label>10</label>Medical Department, Knappschaftskrankenhaus, Ruhr-University Bochum, In der Schornau 23-25, Bochum, 44892, Germany</aff><pub-date pub-type="collection"><year>2012</year></pub-date><pub-date pub-type="epub"><day>9</day><month>8</month><year>2012</year></pub-date><volume>12</volume><fpage>349</fpage><lpage>349</lpage><history><date date-type="received"><day>18</day><month>9</month><year>2011</year></date><date date-type="accepted"><day>26</day><month>6</month><year>2012</year></date></history><permissions><copyright-statement>Copyright ©2012 Reinacher-Schick et al.; licensee BioMed Central Ltd.</copyright-statement><copyright-year>2012</copyright-year><copyright-holder>Reinacher-Schick et al.; licensee BioMed Central Ltd.</copyright-holder><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/2.0"><license-p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</license-p></license></permissions><self-uri xlink:href="http://www.biomedcentral.com/1471-2407/12/349"></self-uri><abstract><sec><title><offsets xml_i="6054" xml_f="6064" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="6075" xml_f="6100" txt_i="22" txt_f="47">To evaluate the value of </offsets><italic><offsets xml_i="6108" xml_f="6112" txt_i="47" txt_f="51">KRAS</offsets></italic><offsets xml_i="6121" xml_f="6247" txt_i="51" txt_f="177"> codon 13 mutations in patients with advanced colorectal cancer (advanced CRC) treated with oxaliplatin and fluoropyrimidines.</offsets></p></sec><sec><title><offsets xml_i="6269" xml_f="6276" txt_i="179" txt_f="186">Methods</offsets></title><p><offsets xml_i="6287" xml_f="6464" txt_i="187" txt_f="364">Tumor specimens from 201 patients with advanced CRC from a randomized, phase III trial comparing oxaliplatin/5-FU vs. oxaliplatin/capecitabine were retrospectively analyzed for </offsets><italic><offsets xml_i="6472" xml_f="6476" txt_i="364" txt_f="368">KRAS</offsets></italic><offsets xml_i="6485" xml_f="6632" txt_i="368" txt_f="515"> mutations. Mutation data were correlated to response data (Overall response rate, ORR), progression-free survival (PFS) and overall survival (OS).</offsets></p></sec><sec><title><offsets xml_i="6654" xml_f="6661" txt_i="517" txt_f="524">Results</offsets></title><p><offsets xml_i="6672" xml_f="6703" txt_i="525" txt_f="556">201 patients were analysed for </offsets><italic><offsets xml_i="6711" xml_f="6715" txt_i="556" txt_f="560">KRAS</offsets></italic><offsets xml_i="6724" xml_f="6776" txt_i="560" txt_f="612"> mutation (61.2% males; mean age 64.2 ± 8.6 years). </offsets><italic><offsets xml_i="6784" xml_f="6788" txt_i="612" txt_f="616">KRAS</offsets></italic><offsets xml_i="6797" xml_f="7046" txt_i="616" txt_f="865"> mutations were identified in 36.3% of tumors (28.8% in codon 12, 7.4% in codon 13). The ORR in codon 13 patients compared to codon 12 and wild type patients was significantly lower (p = 0.008). There was a tendency for a better overall survival in </offsets><italic><offsets xml_i="7054" xml_f="7058" txt_i="865" txt_f="869">KRAS</offsets></italic><offsets xml_i="7067" xml_f="7171" txt_i="869" txt_f="973"> wild type patients compared to mutants (p = 0.085). PFS in all patients was not different in the three </offsets><italic><offsets xml_i="7179" xml_f="7183" txt_i="973" txt_f="977">KRAS</offsets></italic><offsets xml_i="7192" xml_f="7381" txt_i="977" txt_f="1166"> genetic groups (p = 0.72). However, we found a marked difference in PFS between patients with codon 12 and 13 mutant tumors treated with infusional 5-FU versus capecitabine based regimens.</offsets></p></sec><sec><title><offsets xml_i="7403" xml_f="7414" txt_i="1168" txt_f="1179">Conclusions</offsets></title><p><offsets xml_i="7425" xml_f="7459" txt_i="1180" txt_f="1214">Our data suggest that the type of </offsets><italic><offsets xml_i="7467" xml_f="7471" txt_i="1214" txt_f="1218">KRAS</offsets></italic><offsets xml_i="7480" xml_f="7666" txt_i="1218" txt_f="1404"> mutation may be of clinical relevance under oxaliplatin combination chemotherapies without the addition of monoclonal antibodies in particular when overall response rates are important.</offsets></p></sec><sec><title><offsets xml_i="7688" xml_f="7713" txt_i="1406" txt_f="1431">Trial registration number</offsets></title><p><offsets xml_i="7724" xml_f="7735" txt_i="1432" txt_f="1443">2002-04-017</offsets></p></sec></abstract><kwd-group><kwd>Codon 13 mutation</kwd><kwd>Colorectal cancer</kwd><kwd><italic>KRAS</italic></kwd><kwd>Oxaliplatin</kwd><kwd>Prognosis</kwd></kwd-group></article-meta></front><body><sec><title><offsets xml_i="7950" xml_f="7960" txt_i="1452" txt_f="1462">Background</offsets></title><p><offsets xml_i="7971" xml_f="8093" txt_i="1463" txt_f="1585">The oncogene KRAS belongs to the protein family of small G-proteins and is mutated in 35-40% of colorectal cancers (CRC) [</offsets><xref ref-type="bibr" rid="B1"><offsets xml_i="8124" xml_f="8125" txt_i="1585" txt_f="1586">1</offsets></xref><offsets xml_i="8132" xml_f="8135" txt_i="1586" txt_f="1589">]. </offsets><italic><offsets xml_i="8143" xml_f="8146" txt_i="1589" txt_f="1592">RAS</offsets></italic><offsets xml_i="8155" xml_f="8293" txt_i="1592" txt_f="1730"> mutations are considered early events in colon carcinogenesis and are well conserved between primary tumor and corresponding metastases [</offsets><xref ref-type="bibr" rid="B2"><offsets xml_i="8324" xml_f="8325" txt_i="1730" txt_f="1731">2</offsets></xref><offsets xml_i="8332" xml_f="8335" txt_i="1731" txt_f="1734">]. </offsets><italic><offsets xml_i="8343" xml_f="8347" txt_i="1734" txt_f="1738">KRAS</offsets></italic><offsets xml_i="8356" xml_f="8447" txt_i="1738" txt_f="1829"> mutations tested routinely include six mutations in codon 12 and one mutation in codon 13.</offsets></p><p><italic><offsets xml_i="8462" xml_f="8466" txt_i="1830" txt_f="1834">KRAS</offsets></italic><offsets xml_i="8475" xml_f="8646" txt_i="1834" txt_f="2005"> has been studied extensively as a prognostic marker in CRC, but results are still conflicting. Overall, there seems a tendency towards inferior outcome for patients with </offsets><italic><offsets xml_i="8654" xml_f="8658" txt_i="2005" txt_f="2009">KRAS</offsets></italic><offsets xml_i="8667" xml_f="8715" txt_i="2009" txt_f="2057"> mutant tumors even in large randomized trials [</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="8746" xml_f="8747" txt_i="2057" txt_f="2058">3</offsets></xref><offsets xml_i="8754" xml_f="8755" txt_i="2058" txt_f="2059">-</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="8786" xml_f="8787" txt_i="2059" txt_f="2060">7</offsets></xref><offsets xml_i="8794" xml_f="8810" txt_i="2060" txt_f="2076">]. Importantly, </offsets><italic><offsets xml_i="8818" xml_f="8822" txt_i="2076" txt_f="2080">KRAS</offsets></italic><offsets xml_i="8831" xml_f="9146" txt_i="2080" txt_f="2395"> has recently been identified as a strong predictive marker for patients with advanced CRC under anti-EGFR-treatment. Various studies demonstrated that while patients with tumors expressing wild type KRAS may benefit from anti-EGFR (epidermal growth factor receptor)-antibody treatment, patients carrying a mutated </offsets><italic><offsets xml_i="9154" xml_f="9158" txt_i="2395" txt_f="2399">KRAS</offsets></italic><offsets xml_i="9167" xml_f="9181" txt_i="2399" txt_f="2413"> gene do not [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="9212" xml_f="9213" txt_i="2413" txt_f="2414">5</offsets></xref><offsets xml_i="9220" xml_f="9221" txt_i="2414" txt_f="2415">,</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="9252" xml_f="9253" txt_i="2415" txt_f="2416">8</offsets></xref><offsets xml_i="9260" xml_f="9261" txt_i="2416" txt_f="2417">-</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="9293" xml_f="9295" txt_i="2417" txt_f="2419">10</offsets></xref><offsets xml_i="9302" xml_f="9398" txt_i="2419" txt_f="2515">]. In contrast, two large trials using oxaliplatin-based chemotherapy-backbones did not confirm </offsets><italic><offsets xml_i="9406" xml_f="9410" txt_i="2515" txt_f="2519">KRAS</offsets></italic><offsets xml_i="9419" xml_f="9498" txt_i="2519" txt_f="2598"> wild type to be a powerful predictor of treatment efficacy in metastatic CRC [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="9529" xml_f="9530" txt_i="2598" txt_f="2599">4</offsets></xref><offsets xml_i="9537" xml_f="9538" txt_i="2599" txt_f="2600">,</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="9570" xml_f="9572" txt_i="2600" txt_f="2602">11</offsets></xref><offsets xml_i="9579" xml_f="9672" txt_i="2602" txt_f="2695">]. However, the studies published until recently did not differentiate between the different </offsets><italic><offsets xml_i="9680" xml_f="9684" txt_i="2695" txt_f="2699">KRAS</offsets></italic><offsets xml_i="9693" xml_f="9823" txt_i="2699" txt_f="2829"> mutations and other histopathological or molecular features of their patients. The clinical observation, that some patients with </offsets><italic><offsets xml_i="9831" xml_f="9835" txt_i="2829" txt_f="2833">KRAS</offsets></italic><offsets xml_i="9844" xml_f="9958" txt_i="2833" txt_f="2947"> mutation may respond to anti-EGFR-antibody therapy, as well as experimental data, that the biological effects of </offsets><italic><offsets xml_i="9966" xml_f="9970" txt_i="2947" txt_f="2951">KRAS</offsets></italic><offsets xml_i="9979" xml_f="10049" txt_i="2951" txt_f="3021"> mutations may differ, was addressed recently by de Roock and Tejpar [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="10081" xml_f="10083" txt_i="3021" txt_f="3023">12</offsets></xref><offsets xml_i="10090" xml_f="10091" txt_i="3023" txt_f="3024">,</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="10123" xml_f="10125" txt_i="3024" txt_f="3026">13</offsets></xref><offsets xml_i="10132" xml_f="10190" txt_i="3026" txt_f="3084">]. They reported that patients with codon 13 mutations in </offsets><italic><offsets xml_i="10198" xml_f="10202" txt_i="3084" txt_f="3088">KRAS</offsets></italic><offsets xml_i="10211" xml_f="10449" txt_i="3088" txt_f="3326"> exhibit a worse overall prognosis with short overall survival times under standard chemotherapy, but may benefit from anti-EGFR-antibody therapy similar to wild type patients. These observations prompted us to look for the effect of the </offsets><italic><offsets xml_i="10457" xml_f="10461" txt_i="3326" txt_f="3330">KRAS</offsets></italic><offsets xml_i="10470" xml_f="10714" txt_i="3330" txt_f="3574"> mutational status in correlation with response and survival data in patients with advanced colorectal cancer receiving oxaliplatin containing chemotherapy from a prospective randomized multicenter phase III trial of the German AIO study group.</offsets></p></sec><sec sec-type="methods"><title><offsets xml_i="10755" xml_f="10762" txt_i="3576" txt_f="3583">Methods</offsets></title><sec><title><offsets xml_i="10782" xml_f="10790" txt_i="3584" txt_f="3592">Patients</offsets></title><p><offsets xml_i="10801" xml_f="11355" txt_i="3593" txt_f="4147">All patients participated in a prospective randomized phase III first-line palliative chemotherapy trial of advanced CRC of the AIO colorectal study group (Arbeitsgemeinschaft Internistische Onkologie of the German Cancer Society). The study was performed according to the Helsinki declaration and it was approved by the ethics review board of the Central Hospital Bremen and of the Medical Faculty of the Ruhr-University Bochum. The study design, patient characteristics, treatment plans and results of the clinical trial have been reported previously [</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="11387" xml_f="11389" txt_i="4147" txt_f="4149">14</offsets></xref><offsets xml_i="11396" xml_f="11541" txt_i="4149" txt_f="4294">]. Briefly, a total of 474 patients were randomized to be treated with either 5-FU/folinic acid (FA) and oxaliplatin (FUFOX: oxaliplatin, 50 mg/m</offsets><sup><offsets xml_i="11546" xml_f="11547" txt_i="4294" txt_f="4295">2</offsets></sup><offsets xml_i="11553" xml_f="11567" txt_i="4295" txt_f="4309">; FA, 500 mg/m</offsets><sup><offsets xml_i="11572" xml_f="11573" txt_i="4309" txt_f="4310">2</offsets></sup><offsets xml_i="11579" xml_f="11608" txt_i="4310" txt_f="4339">; continuous 5-FU, 2,000 mg/m</offsets><sup><offsets xml_i="11613" xml_f="11614" txt_i="4339" txt_f="4340">2</offsets></sup><offsets xml_i="11620" xml_f="11718" txt_i="4340" txt_f="4438">/22 h; on day 1, 8, 15, 22; q day 36) or capecitabine and oxaliplatin (CAPOX: oxaliplatin, 70 mg/m</offsets><sup><offsets xml_i="11723" xml_f="11724" txt_i="4438" txt_f="4439">2</offsets></sup><offsets xml_i="11730" xml_f="11775" txt_i="4439" txt_f="4484"> on day 1 and 8; capecitabine, 2 × 1,000 mg/m</offsets><sup><offsets xml_i="11780" xml_f="11781" txt_i="4484" txt_f="4485">2</offsets></sup><offsets xml_i="11787" xml_f="12325" txt_i="4485" txt_f="5023">/day consecutively for 2 weeks, q day 22). No clinical factor was found to be predictive for definition of a subgroup of patients benefiting more or less from each fluoropyrimidine backbone. We here present data on a subcohort of 201 patients (42.4%) with available formalin-fixed paraffin-embedded tissue. Samples were retrieved from pathologists responsible for first diagnosis, pseudononymized and forwarded to the Institute of Pathology of the Ruhr-University Bochum, which was blinded to treatment allocation and prognostic outcomes.</offsets></p></sec><sec><title><offsets xml_i="12347" xml_f="12383" txt_i="5025" txt_f="5061">DNA extraction and mutation analysis</offsets></title><p><offsets xml_i="12394" xml_f="12861" txt_i="5062" txt_f="5522">DNA was extracted from anonymized formalin-fixed paraffin embedded tissue samples. For each patient, five 10-μm sections were prepared. An additional representative 1-μm section was deparaffinized, stained with H&amp;E, and analyzed for detailed morphology. Regions displaying tumor cellularity of &gt;70% were marked and macrodissected. Tissue was extracted using QIAmp DNA Mini kit (Qiagen, Hilden, Germany). Real-Time PCR amplification for the most frequent seven </offsets><italic><offsets xml_i="12869" xml_f="12873" txt_i="5522" txt_f="5526">KRAS</offsets></italic><offsets xml_i="12882" xml_f="13052" txt_i="5526" txt_f="5693"> mutations was performed using commercially available kits from DxS Ltd. (Manchester, UK) according to the manufacturer’s instructions. This kit detects &gt;95% of known </offsets><italic><offsets xml_i="13060" xml_f="13064" txt_i="5693" txt_f="5697">KRAS</offsets></italic><offsets xml_i="13073" xml_f="13156" txt_i="5697" txt_f="5780"> mutations. Laboratory staff was blinded to the patient data and clinical outcome. </offsets><italic><offsets xml_i="13164" xml_f="13168" txt_i="5780" txt_f="5784">BRAF</offsets></italic><offsets xml_i="13177" xml_f="13297" txt_i="5784" txt_f="5904"> mutation analysis to detect the V600E substitution was performed by RT-PCR (see supplemental file for further details).</offsets></p></sec><sec><title><offsets xml_i="13319" xml_f="13339" txt_i="5906" txt_f="5926">Statistical analysis</offsets></title><p><offsets xml_i="13350" xml_f="13412" txt_i="5927" txt_f="5989">Chi-square test was used to evaluate the associations between </offsets><italic><offsets xml_i="13420" xml_f="13424" txt_i="5989" txt_f="5993">KRAS</offsets></italic><offsets xml_i="13433" xml_f="13755" txt_i="5993" txt_f="6315"> status and other dichotomous variables. The analysis of progression-free survival (PFS) and overall survival (OS) was done using the Kaplan-Meier method and differences between subgroups were calculated by log-rank test. Data have been analyzed using SPSS 18.0 (Munich, Germany). All tests were two sided. For all tests, </offsets><italic><offsets xml_i="13763" xml_f="13764" txt_i="6315" txt_f="6316">p</offsets></italic><offsets xml_i="13773" xml_f="13818" txt_i="6316" txt_f="6358"> values &lt;0.05 were considered significant.</offsets></p></sec></sec><sec sec-type="results"><title><offsets xml_i="13865" xml_f="13872" txt_i="6361" txt_f="6368">Results</offsets></title><sec><title><offsets xml_i="13892" xml_f="13915" txt_i="6369" txt_f="6392">Patient characteristics</offsets></title><p><offsets xml_i="13926" xml_f="13978" txt_i="6393" txt_f="6445">The subcohort of patients successfully analyzed for </offsets><italic><offsets xml_i="13986" xml_f="13990" txt_i="6445" txt_f="6449">KRAS</offsets></italic><offsets xml_i="13999" xml_f="14559" txt_i="6449" txt_f="7009"> mutations consisted of 201 patients (61.2% males with a mean age of 64.2 ± 8.6 years, range 35 – 82). Colon cancer was diagnosed in 131 (65.2%) patients while rectal cancer was diagnosed as primary tumor in 61 (30.3%) patients. Localization was not known for 9 cases (4.5%). 105 (52.2%) patients were treated with FUFOX, whereas 96 (47.7%) patients received CAPOX. 115 (57.2%) patients had synchronous metastases, whereas 59 (29.4%) had metachronous metastases. Whether metastases were synchronous or metachronous was not known for 27 ( 13.4%) patients (Table</offsets><xref ref-type="table" rid="T1"><offsets xml_i="14591" xml_f="14592" txt_i="7009" txt_f="7010">1</offsets></xref><offsets xml_i="14599" xml_f="14856" txt_i="7010" txt_f="7267">). The subcohort reported here was representative of the complete study cohort with respect to age, gender, treatment plans (i.e. percentage of patients with FUFOX and CAPOX respectively). Median PFS and OS of the total ITT (intention-to-treat) population [</offsets><xref ref-type="bibr" rid="B14"><offsets xml_i="14888" xml_f="14890" txt_i="7267" txt_f="7269">14</offsets></xref><offsets xml_i="14897" xml_f="15011" txt_i="7269" txt_f="7383">] and the biomarker subpopulation were fully comparable (PFS under FUFOX was 8.0 in the ITT cohort vs. 7.8 in the </offsets><italic><offsets xml_i="15019" xml_f="15023" txt_i="7383" txt_f="7387">KRAS</offsets></italic><offsets xml_i="15032" xml_f="15132" txt_i="7387" txt_f="7487"> cohort; PFS under CAPOX was 7.1 vs. 7.0; OS under FUFOX was 18.8 in the ITT cohort vs. 17.5 in the </offsets><italic><offsets xml_i="15140" xml_f="15144" txt_i="7487" txt_f="7491">KRAS</offsets></italic><offsets xml_i="15153" xml_f="15202" txt_i="7491" txt_f="7540"> subcohort and OS under CAPOX was 16.8 vs. 18.4).</offsets></p><table-wrap position="float" id="T1"><label><offsets xml_i="15250" xml_f="15257" txt_i="7541" txt_f="7548">Table 1</offsets></label><caption><p><offsets xml_i="15277" xml_f="15331" txt_i="7548" txt_f="7602">Baseline characteristics of the investigated subcohort</offsets></p></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="right"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="top"><tr><th align="right"><bold><offsets xml_i="15539" xml_f="15559" txt_i="7603" txt_f="7623">Biomarker population</offsets></bold></th><th align="center"><bold><offsets xml_i="15596" xml_f="15597" txt_i="7623" txt_f="7624">N</offsets></bold></th><th align="center"><bold><offsets xml_i="15634" xml_f="15635" txt_i="7624" txt_f="7625">%</offsets></bold></th></tr></thead><tbody valign="top"><tr><td align="left" valign="bottom"><bold><offsets xml_i="15723" xml_f="15726" txt_i="7625" txt_f="7628">Age</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="15782" xml_f="15783" txt_i="7629" txt_f="7630"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="15832" xml_f="15833" txt_i="7631" txt_f="7632"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="15896" xml_f="15912" txt_i="7633" txt_f="7649">Mean 64.15 years</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="15968" xml_f="15969" txt_i="7650" txt_f="7651"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16018" xml_f="16019" txt_i="7652" txt_f="7653"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="16082" xml_f="16099" txt_i="7654" txt_f="7671">Range 35–82 years</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16155" xml_f="16156" txt_i="7672" txt_f="7673"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16205" xml_f="16206" txt_i="7674" txt_f="7675"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="16269" xml_f="16274" txt_i="7676" txt_f="7681">≤60 y</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16330" xml_f="16332" txt_i="7682" txt_f="7684">58</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="16388" xml_f="16392" txt_i="7685" txt_f="7689">28.9</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="16455" xml_f="16463" txt_i="7690" txt_f="7695">&gt;60 y</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16519" xml_f="16522" txt_i="7696" txt_f="7699">143</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="16578" xml_f="16582" txt_i="7700" txt_f="7704">71.1</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="16644" xml_f="16647" txt_i="7705" txt_f="7708">Sex</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16703" xml_f="16704" txt_i="7709" txt_f="7710"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16753" xml_f="16754" txt_i="7711" txt_f="7712"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="16817" xml_f="16823" txt_i="7713" txt_f="7719">Female</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="16879" xml_f="16881" txt_i="7720" txt_f="7722">78</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="16937" xml_f="16941" txt_i="7723" txt_f="7727">38.8</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="17004" xml_f="17008" txt_i="7728" txt_f="7732">Male</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17064" xml_f="17067" txt_i="7733" txt_f="7736">123</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="17123" xml_f="17127" txt_i="7737" txt_f="7741">61.2</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="17189" xml_f="17202" txt_i="7742" txt_f="7755">Treatment arm</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17258" xml_f="17259" txt_i="7756" txt_f="7757"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17308" xml_f="17309" txt_i="7758" txt_f="7759"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="17372" xml_f="17377" txt_i="7760" txt_f="7765">FUFOX</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17433" xml_f="17436" txt_i="7766" txt_f="7769">105</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="17492" xml_f="17496" txt_i="7770" txt_f="7774">52.2</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="17559" xml_f="17564" txt_i="7775" txt_f="7780">CAPOX</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17620" xml_f="17622" txt_i="7781" txt_f="7783">96</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="17678" xml_f="17682" txt_i="7784" txt_f="7788">47.8</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="17744" xml_f="17773" txt_i="7789" txt_f="7818">Localization of primary tumor</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17829" xml_f="17830" txt_i="7819" txt_f="7820"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="17879" xml_f="17880" txt_i="7821" txt_f="7822"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="17943" xml_f="17948" txt_i="7823" txt_f="7828">Colon</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18004" xml_f="18007" txt_i="7829" txt_f="7832">131</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="18063" xml_f="18067" txt_i="7833" txt_f="7837">65.2</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="18130" xml_f="18136" txt_i="7838" txt_f="7844">Rectum</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18192" xml_f="18194" txt_i="7845" txt_f="7847">61</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="18250" xml_f="18254" txt_i="7848" txt_f="7852">30.3</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="18317" xml_f="18321" txt_i="7853" txt_f="7857">n.k.</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18377" xml_f="18378" txt_i="7858" txt_f="7859">9</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="18434" xml_f="18437" txt_i="7860" txt_f="7863">4,5</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="18499" xml_f="18527" txt_i="7864" txt_f="7892">T stage at initial diagnosis</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18583" xml_f="18584" txt_i="7893" txt_f="7894"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18633" xml_f="18634" txt_i="7895" txt_f="7896"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="18697" xml_f="18699" txt_i="7897" txt_f="7899">T1</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18755" xml_f="18756" txt_i="7900" txt_f="7901">1</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="18812" xml_f="18815" txt_i="7902" txt_f="7905">0.4</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="18878" xml_f="18880" txt_i="7906" txt_f="7908">T2</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="18936" xml_f="18938" txt_i="7909" txt_f="7911">15</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="18994" xml_f="18997" txt_i="7912" txt_f="7915">7.5</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="19060" xml_f="19062" txt_i="7916" txt_f="7918">T3</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="19118" xml_f="19121" txt_i="7919" txt_f="7922">132</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="19177" xml_f="19181" txt_i="7923" txt_f="7927">65.7</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="19244" xml_f="19246" txt_i="7928" txt_f="7930">T4</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="19302" xml_f="19304" txt_i="7931" txt_f="7933">49</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="19360" xml_f="19364" txt_i="7934" txt_f="7938">24.4</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="19427" xml_f="19431" txt_i="7939" txt_f="7943">n.k.</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="19487" xml_f="19488" txt_i="7944" txt_f="7945">4</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="19544" xml_f="19547" txt_i="7946" txt_f="7949">2.0</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="19609" xml_f="19637" txt_i="7950" txt_f="7978">N stage at initial diagnosis</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="19693" xml_f="19694" txt_i="7979" txt_f="7980"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="19743" xml_f="19744" txt_i="7981" txt_f="7982"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="19807" xml_f="19809" txt_i="7983" txt_f="7985">N0</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="19865" xml_f="19867" txt_i="7986" txt_f="7988">46</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="19923" xml_f="19927" txt_i="7989" txt_f="7993">22.9</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="19990" xml_f="19992" txt_i="7994" txt_f="7996">N1</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20048" xml_f="20050" txt_i="7997" txt_f="7999">57</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="20106" xml_f="20110" txt_i="8000" txt_f="8004">28.3</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="20173" xml_f="20175" txt_i="8005" txt_f="8007">N2</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20231" xml_f="20233" txt_i="8008" txt_f="8010">89</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="20289" xml_f="20293" txt_i="8011" txt_f="8015">44.3</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="20356" xml_f="20360" txt_i="8016" txt_f="8020">n.k.</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20416" xml_f="20417" txt_i="8021" txt_f="8022">9</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="20473" xml_f="20476" txt_i="8023" txt_f="8026">4.5</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="20538" xml_f="20566" txt_i="8027" txt_f="8055">M stage at initial diagnosis</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20622" xml_f="20623" txt_i="8056" txt_f="8057"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20672" xml_f="20673" txt_i="8058" txt_f="8059"> </offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="20736" xml_f="20739" txt_i="8060" txt_f="8063">M0*</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20795" xml_f="20797" txt_i="8064" txt_f="8066">59</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="20853" xml_f="20857" txt_i="8067" txt_f="8071">29.4</offsets><hr></hr></td></tr><tr><td align="right" valign="bottom"><bold><offsets xml_i="20920" xml_f="20922" txt_i="8072" txt_f="8074">M1</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="20978" xml_f="20981" txt_i="8075" txt_f="8078">115</offsets><hr></hr></td><td align="char" valign="bottom" char="."><offsets xml_i="21037" xml_f="21041" txt_i="8079" txt_f="8083">57.2</offsets><hr></hr></td></tr><tr><td align="right"><bold><offsets xml_i="21088" xml_f="21092" txt_i="8084" txt_f="8088">n.k.</offsets></bold></td><td align="center"><offsets xml_i="21123" xml_f="21125" txt_i="8089" txt_f="8091">27</offsets></td><td align="char" char="."><offsets xml_i="21156" xml_f="21160" txt_i="8092" txt_f="8096">13.4</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="21206" xml_f="21354" txt_i="8097" txt_f="8245">*M0 indicates metachronous metastatic disease at initial diagnosis of CRC, M1 indicates synchronous metastatic disease at initial diagnosis of CRC. </offsets><italic><offsets xml_i="21362" xml_f="21366" txt_i="8245" txt_f="8249">n.k.</offsets></italic><offsets xml_i="21375" xml_f="21419" txt_i="8249" txt_f="8293"> = not known – there were no data available.</offsets></p></table-wrap-foot></table-wrap></sec><sec><title><offsets xml_i="21472" xml_f="21490" txt_i="8295" txt_f="8313">Mutation frequency</offsets></title><p><italic><offsets xml_i="21509" xml_f="21513" txt_i="8314" txt_f="8318">KRAS</offsets></italic><offsets xml_i="21522" xml_f="21594" txt_i="8318" txt_f="8390"> mutations could be successfully analyzed in 201 samples. We identified </offsets><italic><offsets xml_i="21602" xml_f="21606" txt_i="8390" txt_f="8394">KRAS</offsets></italic><offsets xml_i="21615" xml_f="21666" txt_i="8394" txt_f="8445"> mutations in 73/201 (36.3%) tumors. 58 (28.9%) of </offsets><italic><offsets xml_i="21674" xml_f="21678" txt_i="8445" txt_f="8449">KRAS</offsets></italic><offsets xml_i="21687" xml_f="21775" txt_i="8449" txt_f="8537"> mutations were located in codon 12, whereas 15 (7.46%) were found in codon 13(see Table</offsets><xref ref-type="table" rid="T2"><offsets xml_i="21807" xml_f="21808" txt_i="8537" txt_f="8538">2</offsets></xref><offsets xml_i="21815" xml_f="21842" txt_i="8538" txt_f="8565">). The three most frequent </offsets><italic><offsets xml_i="21850" xml_f="21854" txt_i="8565" txt_f="8569">KRAS</offsets></italic><offsets xml_i="21863" xml_f="22030" txt_i="8569" txt_f="8727"> alterations in our samples were c.35 G &gt; A (G12D, n = 25; 12.4%), c.38 G &gt; A (G13D, n = 15; 7.46%), and c.35 G &gt; T (G12V, n = 13, 6.46%). In 128 patients no </offsets><italic><offsets xml_i="22038" xml_f="22042" txt_i="8727" txt_f="8731">KRAS</offsets></italic><offsets xml_i="22051" xml_f="22130" txt_i="8731" txt_f="8810"> mutation was found (63.6%). In these patients we detected 13 mutations in the </offsets><italic><offsets xml_i="22138" xml_f="22142" txt_i="8810" txt_f="8814">BRAF</offsets></italic><offsets xml_i="22151" xml_f="22193" txt_i="8814" txt_f="8856"> gene (V600E) (10% of wild type patients).</offsets></p><table-wrap position="float" id="T2"><label><offsets xml_i="22241" xml_f="22248" txt_i="8857" txt_f="8864">Table 2</offsets></label><caption><p><bold><offsets xml_i="22274" xml_f="22275" txt_i="8864" txt_f="8865"> </offsets><italic><offsets xml_i="22283" xml_f="22287" txt_i="8865" txt_f="8869">KRAS</offsets></italic><offsets xml_i="22296" xml_f="22297" txt_i="8869" txt_f="8870"> </offsets></bold><bold><offsets xml_i="22310" xml_f="22360" txt_i="8870" txt_f="8920">mutation frequency in the investigated subcohort (</offsets></bold><bold><offsets xml_i="22373" xml_f="22374" txt_i="8920" txt_f="8921"> </offsets><italic><offsets xml_i="22382" xml_f="22386" txt_i="8921" txt_f="8925">KRAS</offsets></italic><offsets xml_i="22395" xml_f="22396" txt_i="8925" txt_f="8926"> </offsets></bold><bold><offsets xml_i="22409" xml_f="22432" txt_i="8926" txt_f="8949">Wild type: 128 (63.7%),</offsets></bold><bold><offsets xml_i="22445" xml_f="22446" txt_i="8949" txt_f="8950"> </offsets><italic><offsets xml_i="22454" xml_f="22458" txt_i="8950" txt_f="8954">KRAS</offsets></italic><offsets xml_i="22467" xml_f="22468" txt_i="8954" txt_f="8955"> </offsets></bold><bold><offsets xml_i="22481" xml_f="22501" txt_i="8955" txt_f="8975">Mutation 73 (36.3%))</offsets></bold></p></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="left"></col><col align="left"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="top"><tr><th align="left"><offsets xml_i="22756" xml_f="22757" txt_i="8976" txt_f="8977"> </offsets></th><th align="left"><offsets xml_i="22779" xml_f="22780" txt_i="8977" txt_f="8978"> </offsets></th><th align="left"><offsets xml_i="22802" xml_f="22803" txt_i="8978" txt_f="8979"> </offsets></th><th align="center"><bold><offsets xml_i="22833" xml_f="22869" txt_i="8979" txt_f="9015">relative frequency of occurrence (%)</offsets></bold></th><th align="center"><bold><offsets xml_i="22906" xml_f="22938" txt_i="9015" txt_f="9047">absolute frequency of occurrence</offsets></bold></th></tr></thead><tbody valign="top"><tr><td colspan="2" align="left" valign="bottom"><bold><offsets xml_i="23038" xml_f="23056" txt_i="9047" txt_f="9065">CODON 12 mutation:</offsets></bold><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="23110" xml_f="23111" txt_i="9066" txt_f="9067"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23160" xml_f="23161" txt_i="9068" txt_f="9069"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23210" xml_f="23211" txt_i="9070" txt_f="9071"> </offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><offsets xml_i="23279" xml_f="23295" txt_i="9072" txt_f="9088">Aspartate (G12D)</offsets><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="23342" xml_f="23352" txt_i="9089" txt_f="9096">c.35G&gt;A</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23401" xml_f="23405" txt_i="9097" txt_f="9101">12,4</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23454" xml_f="23456" txt_i="9102" txt_f="9104">25</offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><offsets xml_i="23524" xml_f="23537" txt_i="9105" txt_f="9118">Valine (G12V)</offsets><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="23584" xml_f="23594" txt_i="9119" txt_f="9126">c.35G&gt;T</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23643" xml_f="23646" txt_i="9127" txt_f="9130">6,5</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23695" xml_f="23697" txt_i="9131" txt_f="9133">13</offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><offsets xml_i="23765" xml_f="23779" txt_i="9134" txt_f="9148">Alanine (G12A)</offsets><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="23826" xml_f="23836" txt_i="9149" txt_f="9156">c.35G&gt;C</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23885" xml_f="23888" txt_i="9157" txt_f="9160">3,4</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="23937" xml_f="23938" txt_i="9161" txt_f="9162">7</offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><offsets xml_i="24006" xml_f="24021" txt_i="9163" txt_f="9178">Cysteine (G12C)</offsets><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="24068" xml_f="24078" txt_i="9179" txt_f="9186">c.34G&gt;T</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24127" xml_f="24128" txt_i="9187" txt_f="9188">3</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24177" xml_f="24178" txt_i="9189" txt_f="9190">6</offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><offsets xml_i="24246" xml_f="24259" txt_i="9191" txt_f="9204">Serine (G12S)</offsets><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="24306" xml_f="24316" txt_i="9205" txt_f="9212">c.34G&gt;A</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24365" xml_f="24366" txt_i="9213" txt_f="9214">3</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24415" xml_f="24416" txt_i="9215" txt_f="9216">6</offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><offsets xml_i="24484" xml_f="24499" txt_i="9217" txt_f="9232">Arginine (G12R)</offsets><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="24546" xml_f="24556" txt_i="9233" txt_f="9240">c.34G&gt;C</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24605" xml_f="24608" txt_i="9241" txt_f="9244">0,5</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24657" xml_f="24658" txt_i="9245" txt_f="9246">1</offsets><hr></hr></td></tr><tr><td colspan="2" align="left" valign="bottom"><bold><offsets xml_i="24732" xml_f="24750" txt_i="9247" txt_f="9265">CODON 13 mutation:</offsets></bold><hr></hr></td><td align="left" valign="bottom"><offsets xml_i="24804" xml_f="24805" txt_i="9266" txt_f="9267"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24854" xml_f="24855" txt_i="9268" txt_f="9269"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="24904" xml_f="24905" txt_i="9270" txt_f="9271"> </offsets><hr></hr></td></tr><tr><td colspan="2" align="left"><offsets xml_i="24957" xml_f="24973" txt_i="9272" txt_f="9288">Aspartate (G13D)</offsets></td><td align="left"><offsets xml_i="24995" xml_f="25005" txt_i="9289" txt_f="9296">c.38G&gt;A</offsets></td><td align="center"><offsets xml_i="25029" xml_f="25032" txt_i="9297" txt_f="9300">7,5</offsets></td><td align="center"><offsets xml_i="25056" xml_f="25058" txt_i="9301" txt_f="9303">15</offsets></td></tr></tbody></table></table-wrap></sec><sec><title><offsets xml_i="25115" xml_f="25162" txt_i="9305" txt_f="9352">Correlation between mutations and response rate</offsets></title><p><offsets xml_i="25173" xml_f="25244" txt_i="9353" txt_f="9424">Tumor response evaluation was available for 201 patients. Grouping all </offsets><italic><offsets xml_i="25252" xml_f="25256" txt_i="9424" txt_f="9428">KRAS</offsets></italic><offsets xml_i="25265" xml_f="25445" txt_i="9428" txt_f="9608"> mutations together, mutated tumors were associated with a significantly lower response rate (RR; defined as partial or complete remission by RECIST) as compared to tumors without </offsets><italic><offsets xml_i="25453" xml_f="25457" txt_i="9608" txt_f="9612">KRAS</offsets></italic><offsets xml_i="25466" xml_f="25484" txt_i="9612" txt_f="9630"> mutations (44.4% </offsets><italic><offsets xml_i="25492" xml_f="25495" txt_i="9630" txt_f="9633">vs.</offsets></italic><offsets xml_i="25504" xml_f="25721" txt_i="9633" txt_f="9850"> 63.0%, p = 0.012). When patients with codon 13 mutated tumors were analysed separately the overall response rate in this cohort was 23% as compared to 49% in codon 12 mutated tumors and 63% in wild type tumors (Table</offsets><xref ref-type="table" rid="T3"><offsets xml_i="25753" xml_f="25754" txt_i="9850" txt_f="9851">3</offsets></xref><offsets xml_i="25761" xml_f="25886" txt_i="9851" txt_f="9976">, p = 0.008). Disease control rates (DCR) were 77%, 81% and 88%, respectively, which was not statistically significant (Table</offsets><xref ref-type="table" rid="T3"><offsets xml_i="25918" xml_f="25919" txt_i="9976" txt_f="9977">3</offsets></xref><offsets xml_i="25926" xml_f="25938" txt_i="9977" txt_f="9989">, p = 0.29).</offsets></p><table-wrap position="float" id="T3"><label><offsets xml_i="25986" xml_f="25993" txt_i="9990" txt_f="9997">Table 3</offsets></label><caption><p><bold><offsets xml_i="26019" xml_f="26063" txt_i="9997" txt_f="10041">Tumor response assessment and correlation to</offsets></bold><bold><offsets xml_i="26076" xml_f="26077" txt_i="10041" txt_f="10042"> </offsets><italic><offsets xml_i="26085" xml_f="26089" txt_i="10042" txt_f="10046">KRAS</offsets></italic><offsets xml_i="26098" xml_f="26099" txt_i="10046" txt_f="10047"> </offsets></bold><bold><offsets xml_i="26112" xml_f="26129" txt_i="10047" txt_f="10064">mutational status</offsets></bold></p></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="top"><tr><th align="left"><bold><offsets xml_i="26420" xml_f="26432" txt_i="10065" txt_f="10077">All patients</offsets></bold></th><th align="center"><bold><offsets xml_i="26469" xml_f="26472" txt_i="10077" txt_f="10080">All</offsets></bold></th><th align="center"><bold><offsets xml_i="26509" xml_f="26511" txt_i="10080" txt_f="10082">WT</offsets></bold></th><th align="center"><bold><offsets xml_i="26548" xml_f="26565" txt_i="10082" txt_f="10099">Codon 12 mutation</offsets></bold></th><th align="center"><bold><offsets xml_i="26602" xml_f="26619" txt_i="10099" txt_f="10116">Codon 13 mutation</offsets></bold></th><th align="center"><bold><offsets xml_i="26656" xml_f="26663" txt_i="10116" txt_f="10123">p-value</offsets></bold></th></tr></thead><tbody valign="top"><tr><td rowspan="2" align="left" valign="bottom"><bold><offsets xml_i="26763" xml_f="26779" txt_i="10123" txt_f="10139">No. of patients%</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="26841" xml_f="26844" txt_i="10140" txt_f="10143">201</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="26906" xml_f="26909" txt_i="10144" txt_f="10147">128</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="26971" xml_f="26973" txt_i="10148" txt_f="10150">58</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27035" xml_f="27037" txt_i="10151" txt_f="10153">15</offsets></bold><hr></hr></td><td rowspan="2" align="center" valign="bottom"><offsets xml_i="27105" xml_f="27106" txt_i="10154" txt_f="10155"> </offsets><hr></hr></td></tr><tr><td align="center" valign="bottom"><bold><offsets xml_i="27170" xml_f="27173" txt_i="10156" txt_f="10159">100</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="27229" xml_f="27233" txt_i="10160" txt_f="10164">63.6</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="27282" xml_f="27286" txt_i="10165" txt_f="10169">28.9</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="27335" xml_f="27338" txt_i="10170" txt_f="10173">7.5</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="27400" xml_f="27404" txt_i="10174" txt_f="10178">ORR%</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27466" xml_f="27468" txt_i="10179" txt_f="10181">55</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27530" xml_f="27532" txt_i="10182" txt_f="10184">63</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27594" xml_f="27596" txt_i="10185" txt_f="10187">49</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27658" xml_f="27660" txt_i="10188" txt_f="10190">23</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27722" xml_f="27727" txt_i="10191" txt_f="10196">0.008</offsets></bold><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="27796" xml_f="27802" txt_i="10197" txt_f="10203">95% CI</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="27864" xml_f="27871" txt_i="10204" txt_f="10211">(49–61)</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="27927" xml_f="27934" txt_i="10212" txt_f="10219">(54–71)</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="27983" xml_f="27990" txt_i="10220" txt_f="10227">(37–62)</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="28039" xml_f="28045" txt_i="10228" txt_f="10234">(8–51)</offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="28094" xml_f="28106" txt_i="10235" txt_f="10247">(Chi-Square)</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="28168" xml_f="28172" txt_i="10248" txt_f="10252">DCR%</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="28234" xml_f="28236" txt_i="10253" txt_f="10255">85</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="28298" xml_f="28300" txt_i="10256" txt_f="10258">88</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="28362" xml_f="28364" txt_i="10259" txt_f="10261">81</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="28426" xml_f="28428" txt_i="10262" txt_f="10264">77</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="28490" xml_f="28494" txt_i="10265" txt_f="10269">0.29</offsets></bold><hr></hr></td></tr><tr><td align="left"><bold><offsets xml_i="28547" xml_f="28553" txt_i="10270" txt_f="10276">95% CI</offsets></bold></td><td align="center"><bold><offsets xml_i="28590" xml_f="28597" txt_i="10277" txt_f="10284">(80–90)</offsets></bold></td><td align="center"><offsets xml_i="28628" xml_f="28635" txt_i="10285" txt_f="10292">(80–92)</offsets></td><td align="center"><offsets xml_i="28659" xml_f="28666" txt_i="10293" txt_f="10300">(68–89)</offsets></td><td align="center"><offsets xml_i="28690" xml_f="28697" txt_i="10301" txt_f="10308">(49–93)</offsets></td><td align="center"><offsets xml_i="28721" xml_f="28733" txt_i="10309" txt_f="10321">(Chi-Square)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="28779" xml_f="28893" txt_i="10322" txt_f="10436">All patients. Percentages based on non-missing data, p-values for WT vs Codon 12 mutations vs codon 13 mutations; </offsets><italic><offsets xml_i="28901" xml_f="28903" txt_i="10436" txt_f="10438">WT</offsets></italic><offsets xml_i="28912" xml_f="28924" txt_i="10438" txt_f="10450"> wild type; </offsets><italic><offsets xml_i="28932" xml_f="28935" txt_i="10450" txt_f="10453">ORR</offsets></italic><offsets xml_i="28944" xml_f="28968" txt_i="10453" txt_f="10477"> overall response rate, </offsets><italic><offsets xml_i="28976" xml_f="28979" txt_i="10477" txt_f="10480">DCR</offsets></italic><offsets xml_i="28988" xml_f="29010" txt_i="10480" txt_f="10502"> disease control rate.</offsets></p></table-wrap-foot></table-wrap></sec><sec><title><offsets xml_i="29063" xml_f="29122" txt_i="10504" txt_f="10563">Correlation between mutations and progression-free survival</offsets></title><p><offsets xml_i="29133" xml_f="29235" txt_i="10564" txt_f="10666">During follow-up, 170 of 201 evaluable patients had progressed. The median PFS in all patients of the </offsets><italic><offsets xml_i="29243" xml_f="29247" txt_i="10666" txt_f="10670">KRAS</offsets></italic><offsets xml_i="29256" xml_f="29318" txt_i="10670" txt_f="10732"> subcohort was not statistically different in relation to the </offsets><italic><offsets xml_i="29326" xml_f="29330" txt_i="10732" txt_f="10736">KRAS</offsets></italic><offsets xml_i="29339" xml_f="29462" txt_i="10736" txt_f="10859"> mutational status (wild type: 7.5 months, mutation codon 12: 8.2 months, mutation codon 13: 10.0 months; p = 0.71) (Figure</offsets><xref ref-type="fig" rid="F1"><offsets xml_i="29492" xml_f="29493" txt_i="10859" txt_f="10860">1</offsets></xref><offsets xml_i="29500" xml_f="30052" txt_i="10860" txt_f="11412">). However, when analysing the two treatment arms separately, we found a substantial, non-significant, difference in PFS in codon 13 mutated tumors versus codon 12 mutated tumors. While median PFS was as low as 6.1 months for codon 13 patients receiving infusional 5-FU, median PFS was 13.3 months in patients treated with capecitabine (HR: 2.52, p = 0.22). Patients with codon 12 mutations showed a trend towards the opposite effect: median PFS was 7.0 months under CAPOX therapy while median PFS was 9.9 months under FUFOX (HR: 0.62, p = 0.12) (Table</offsets><xref ref-type="table" rid="T4"><offsets xml_i="30084" xml_f="30085" txt_i="11412" txt_f="11413">4</offsets></xref><offsets xml_i="30092" xml_f="30094" txt_i="11413" txt_f="11415">).</offsets></p><fig id="F1" position="float"><label><offsets xml_i="30135" xml_f="30144" txt_i="11416" txt_f="11425">Figure 1 </offsets></label><caption><p><bold><offsets xml_i="30170" xml_f="30208" txt_i="11425" txt_f="11463">Progression-free survival according to</offsets></bold><bold><offsets xml_i="30221" xml_f="30222" txt_i="11463" txt_f="11464"> </offsets><italic><offsets xml_i="30230" xml_f="30234" txt_i="11464" txt_f="11468">KRAS</offsets></italic><offsets xml_i="30243" xml_f="30244" txt_i="11468" txt_f="11469"> </offsets></bold><bold><offsets xml_i="30257" xml_f="30264" txt_i="11469" txt_f="11476">status.</offsets></bold></p></caption><graphic xlink:href="1471-2407-12-349-1"></graphic></fig><table-wrap position="float" id="T4"><label><offsets xml_i="30386" xml_f="30393" txt_i="11477" txt_f="11484">Table 4</offsets></label><caption><p><bold><offsets xml_i="30419" xml_f="30463" txt_i="11484" txt_f="11528">PFS (progression free survival) according to</offsets></bold><bold><offsets xml_i="30476" xml_f="30477" txt_i="11528" txt_f="11529"> </offsets><italic><offsets xml_i="30485" xml_f="30489" txt_i="11529" txt_f="11533">KRAS</offsets></italic><offsets xml_i="30498" xml_f="30499" txt_i="11533" txt_f="11534"> </offsets></bold><bold><offsets xml_i="30512" xml_f="30557" txt_i="11534" txt_f="11579">mutation for the two different treatment arms</offsets></bold></p></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="center"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="top"><tr><th align="left" valign="bottom"><bold><offsets xml_i="30812" xml_f="30815" txt_i="11580" txt_f="11583">PFS</offsets></bold><hr></hr></th><th align="center" valign="bottom"><bold><offsets xml_i="30877" xml_f="30882" txt_i="11583" txt_f="11588">FUFOX</offsets></bold><hr></hr></th><th align="center" valign="bottom"><bold><offsets xml_i="30944" xml_f="30949" txt_i="11588" txt_f="11593">CAPOX</offsets></bold><hr></hr></th><th align="center" valign="bottom"><bold><offsets xml_i="31011" xml_f="31029" txt_i="11593" txt_f="11611">p-value (log rank)</offsets></bold><hr></hr></th></tr><tr><th align="left"><offsets xml_i="31076" xml_f="31077" txt_i="11611" txt_f="11612"> </offsets></th><th align="center"><bold><offsets xml_i="31107" xml_f="31114" txt_i="11612" txt_f="11619">n = 105</offsets></bold></th><th align="center"><bold><offsets xml_i="31151" xml_f="31157" txt_i="11619" txt_f="11625">n = 96</offsets></bold></th><th align="center"><bold><offsets xml_i="31194" xml_f="31205" txt_i="11625" txt_f="11636">HR (95% CI)</offsets></bold></th></tr></thead><tbody valign="top"><tr><td align="left" valign="bottom"><bold><offsets xml_i="31293" xml_f="31305" txt_i="11636" txt_f="11648">All patients</offsets></bold><hr></hr></td><td align="char" valign="bottom" char="."><bold><offsets xml_i="31374" xml_f="31384" txt_i="11649" txt_f="11659">8.2 months</offsets></bold><hr></hr></td><td align="char" valign="bottom" char="."><bold><offsets xml_i="31453" xml_f="31463" txt_i="11660" txt_f="11670">6.4 months</offsets></bold><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="31523" xml_f="31527" txt_i="11671" txt_f="11675">0.18</offsets></bold><break></break><bold><offsets xml_i="31555" xml_f="31571" txt_i="11675" txt_f="11691">0.81 (0.61-1.09)</offsets></bold><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="31640" xml_f="31642" txt_i="11692" txt_f="11694">WT</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="31704" xml_f="31707" txt_i="11695" txt_f="11698">8.1</offsets></bold><offsets xml_i="31714" xml_f="31720" txt_i="11698" txt_f="11704">months</offsets><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="31775" xml_f="31778" txt_i="11705" txt_f="11708">6.4</offsets></bold><offsets xml_i="31785" xml_f="31791" txt_i="11708" txt_f="11714">months</offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="31844" xml_f="31848" txt_i="11715" txt_f="11719">0.29</offsets></bold><hr></hr></td></tr><tr><td align="left" valign="bottom"><offsets xml_i="31911" xml_f="31912" txt_i="11720" txt_f="11721"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="31961" xml_f="31962" txt_i="11722" txt_f="11723"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="32011" xml_f="32012" txt_i="11724" txt_f="11725"> </offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="32065" xml_f="32069" txt_i="11726" txt_f="11730">0.82</offsets></bold><offsets xml_i="32076" xml_f="32088" txt_i="11730" txt_f="11742"> (0.56-1.19)</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="32150" xml_f="32167" txt_i="11743" txt_f="11760">Codon 12 mutation</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="32229" xml_f="32232" txt_i="11761" txt_f="11764">9.9</offsets></bold><offsets xml_i="32239" xml_f="32245" txt_i="11764" txt_f="11770">months</offsets><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="32300" xml_f="32303" txt_i="11771" txt_f="11774">7.0</offsets></bold><offsets xml_i="32310" xml_f="32316" txt_i="11774" txt_f="11780">months</offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="32369" xml_f="32373" txt_i="11781" txt_f="11785">0.12</offsets></bold><hr></hr></td></tr><tr><td align="left" valign="bottom"><offsets xml_i="32436" xml_f="32437" txt_i="11786" txt_f="11787"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="32486" xml_f="32487" txt_i="11788" txt_f="11789"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="32536" xml_f="32537" txt_i="11790" txt_f="11791"> </offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="32590" xml_f="32594" txt_i="11792" txt_f="11796">0.62</offsets></bold><offsets xml_i="32601" xml_f="32613" txt_i="11796" txt_f="11808"> (0.33-1.14)</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="32675" xml_f="32692" txt_i="11809" txt_f="11826">Codon 13 mutation</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="32754" xml_f="32757" txt_i="11827" txt_f="11830">6.1</offsets></bold><offsets xml_i="32764" xml_f="32770" txt_i="11830" txt_f="11836">months</offsets><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="32825" xml_f="32829" txt_i="11837" txt_f="11841">13.3</offsets></bold><offsets xml_i="32836" xml_f="32842" txt_i="11841" txt_f="11847">months</offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="32895" xml_f="32899" txt_i="11848" txt_f="11852">0.22</offsets></bold><hr></hr></td></tr><tr><td align="left"><offsets xml_i="32946" xml_f="32947" txt_i="11853" txt_f="11854"> </offsets></td><td align="center"><offsets xml_i="32971" xml_f="32972" txt_i="11855" txt_f="11856"> </offsets></td><td align="center"><offsets xml_i="32996" xml_f="32997" txt_i="11857" txt_f="11858"> </offsets></td><td align="left"><bold><offsets xml_i="33025" xml_f="33029" txt_i="11859" txt_f="11863">2.52</offsets></bold><offsets xml_i="33036" xml_f="33049" txt_i="11863" txt_f="11876"> (0.54-11.70)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="33095" xml_f="33100" txt_i="11877" txt_f="11882">PFS, </offsets><italic><offsets xml_i="33108" xml_f="33110" txt_i="11882" txt_f="11884">HR</offsets></italic><offsets xml_i="33119" xml_f="33152" txt_i="11884" txt_f="11917"> hazard ratio by cox regression, </offsets><italic><offsets xml_i="33160" xml_f="33162" txt_i="11917" txt_f="11919">CI</offsets></italic><offsets xml_i="33171" xml_f="33193" txt_i="11919" txt_f="11941"> confidence interval, </offsets><italic><offsets xml_i="33201" xml_f="33203" txt_i="11941" txt_f="11943">WT</offsets></italic><offsets xml_i="33212" xml_f="33223" txt_i="11943" txt_f="11954"> wild type.</offsets></p></table-wrap-foot></table-wrap></sec><sec><title><offsets xml_i="33276" xml_f="33326" txt_i="11956" txt_f="12006">Correlation between mutations and overall survival</offsets></title><p><offsets xml_i="33337" xml_f="33516" txt_i="12007" txt_f="12186">During follow-up 135 of 201 evaluated patients had died. We observed a trend towards a better survival time in patients with wild type tumors compared to those with a mutation of </offsets><italic><offsets xml_i="33524" xml_f="33528" txt_i="12186" txt_f="12190">KRAS</offsets></italic><offsets xml_i="33537" xml_f="33573" txt_i="12190" txt_f="12226">. The overall survival of wild type </offsets><italic><offsets xml_i="33581" xml_f="33585" txt_i="12226" txt_f="12230">KRAS</offsets></italic><offsets xml_i="33594" xml_f="33791" txt_i="12230" txt_f="12427"> patients was 19.2 months, for patients with codon 12 mutations 15.6 months and for patients with codon 13 mutations 16.5 months. These differences were of marginal significance (p = 0.085) (Figure</offsets><xref ref-type="fig" rid="F2"><offsets xml_i="33821" xml_f="33822" txt_i="12427" txt_f="12428">2</offsets></xref><offsets xml_i="33829" xml_f="33831" txt_i="12428" txt_f="12430">).</offsets></p><fig id="F2" position="float"><label><offsets xml_i="33872" xml_f="33881" txt_i="12431" txt_f="12440">Figure 2 </offsets></label><caption><p><offsets xml_i="33901" xml_f="33943" txt_i="12440" txt_f="12482">Overall survival according to KRAS status.</offsets></p></caption><graphic xlink:href="1471-2407-12-349-2"></graphic></fig><p><offsets xml_i="34017" xml_f="34142" txt_i="12483" txt_f="12608">Evaluating the different treatment arms separately, we found comparable survival times without significant differences (Table</offsets><xref ref-type="table" rid="T5"><offsets xml_i="34174" xml_f="34175" txt_i="12608" txt_f="12609">5</offsets></xref><offsets xml_i="34182" xml_f="34244" txt_i="12609" txt_f="12671">). Since the survival curves seemed to separate for wild type </offsets><italic><offsets xml_i="34252" xml_f="34256" txt_i="12671" txt_f="12675">KRAS</offsets></italic><offsets xml_i="34265" xml_f="34277" txt_i="12675" txt_f="12687"> and mutant </offsets><italic><offsets xml_i="34285" xml_f="34289" txt_i="12687" txt_f="12691">KRAS</offsets></italic><offsets xml_i="34298" xml_f="34587" txt_i="12691" txt_f="12980"> patients at 14 months we hypothesised that this difference was most likely caused by the influence of post-study treatment. 71 out of 128 (55.4%) wild type patients received further lines of therapy while 32 out of 128 wild type patients (25%) received cetuximab. Out of 73 patients with </offsets><italic><offsets xml_i="34595" xml_f="34599" txt_i="12980" txt_f="12984">KRAS</offsets></italic><offsets xml_i="34608" xml_f="34970" txt_i="12984" txt_f="13346"> mutations 42 were treated in further lines (57.5%) with 10 patients receiving cetuximab (13.7%). When analysing post progression survival (PPS) in these patients receiving second and further line therapies, patients receiving cetuximab had a significant better overall survival when compared to those patients under irinotecan therapy only, irrespective of the </offsets><italic><offsets xml_i="34978" xml_f="34982" txt_i="13346" txt_f="13350">KRAS</offsets></italic><offsets xml_i="34991" xml_f="35007" txt_i="13350" txt_f="13366"> status. PPS in </offsets><italic><offsets xml_i="35015" xml_f="35019" txt_i="13366" txt_f="13370">KRAS</offsets></italic><offsets xml_i="35028" xml_f="35158" txt_i="13370" txt_f="13497"> wild type patients: 32.2 months when cetuximab was given and 18.8 months when cetuximab was not included (p &lt; 0.001). PPS for </offsets><italic><offsets xml_i="35166" xml_f="35170" txt_i="13497" txt_f="13501">KRAS</offsets></italic><offsets xml_i="35179" xml_f="35271" txt_i="13501" txt_f="13593"> mutant patients: 27.9 months under cetuximab and 16.9 months without cetuximab (p = 0.032).</offsets></p><table-wrap position="float" id="T5"><label><offsets xml_i="35319" xml_f="35326" txt_i="13594" txt_f="13601">Table 5</offsets></label><caption><p><bold><offsets xml_i="35352" xml_f="35386" txt_i="13601" txt_f="13635">OS (overall survival) according to</offsets></bold><bold><offsets xml_i="35399" xml_f="35400" txt_i="13635" txt_f="13636"> </offsets><italic><offsets xml_i="35408" xml_f="35412" txt_i="13636" txt_f="13640">KRAS</offsets></italic><offsets xml_i="35421" xml_f="35422" txt_i="13640" txt_f="13641"> </offsets></bold><bold><offsets xml_i="35435" xml_f="35480" txt_i="13641" txt_f="13686">mutation for the two different treatment arms</offsets></bold></p></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="center"></col><col align="center"></col><col align="center"></col></colgroup><thead><tr><th align="left" valign="bottom"><bold><offsets xml_i="35722" xml_f="35724" txt_i="13687" txt_f="13689">OS</offsets></bold><hr></hr></th><th align="center" valign="bottom"><bold><offsets xml_i="35786" xml_f="35791" txt_i="13689" txt_f="13694">FUFOX</offsets></bold><hr></hr></th><th align="center" valign="bottom"><bold><offsets xml_i="35853" xml_f="35858" txt_i="13694" txt_f="13699">CAPOX</offsets></bold><hr></hr></th><th align="center" valign="bottom"><bold><offsets xml_i="35920" xml_f="35938" txt_i="13699" txt_f="13717">p-value (log rank)</offsets></bold><hr></hr></th></tr><tr><th align="left"><offsets xml_i="35985" xml_f="35986" txt_i="13717" txt_f="13718"> </offsets></th><th align="center"><bold><offsets xml_i="36016" xml_f="36023" txt_i="13718" txt_f="13725">n = 105</offsets></bold></th><th align="center"><bold><offsets xml_i="36060" xml_f="36066" txt_i="13725" txt_f="13731">n = 96</offsets></bold></th><th align="center"><bold><offsets xml_i="36103" xml_f="36114" txt_i="13731" txt_f="13742">HR (95% CI)</offsets></bold></th></tr></thead><tbody valign="top"><tr><td align="left" valign="bottom"><bold><offsets xml_i="36202" xml_f="36204" txt_i="13742" txt_f="13744">WT</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="36266" xml_f="36270" txt_i="13745" txt_f="13749">24.2</offsets></bold><offsets xml_i="36277" xml_f="36283" txt_i="13749" txt_f="13755">months</offsets><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="36338" xml_f="36342" txt_i="13756" txt_f="13760">18.9</offsets></bold><offsets xml_i="36349" xml_f="36355" txt_i="13760" txt_f="13766">months</offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="36408" xml_f="36412" txt_i="13767" txt_f="13771">0.31</offsets></bold><hr></hr></td></tr><tr><td align="left" valign="bottom"><offsets xml_i="36475" xml_f="36476" txt_i="13772" txt_f="13773"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="36525" xml_f="36526" txt_i="13774" txt_f="13775"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="36575" xml_f="36576" txt_i="13776" txt_f="13777"> </offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="36629" xml_f="36633" txt_i="13778" txt_f="13782">0.79</offsets></bold><offsets xml_i="36640" xml_f="36652" txt_i="13782" txt_f="13794"> (0.51-1.24)</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="36714" xml_f="36722" txt_i="13795" txt_f="13803">Codon 12</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="36784" xml_f="36788" txt_i="13804" txt_f="13808">15.6</offsets></bold><offsets xml_i="36795" xml_f="36801" txt_i="13808" txt_f="13814">months</offsets><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="36856" xml_f="36860" txt_i="13815" txt_f="13819">15.5</offsets></bold><offsets xml_i="36867" xml_f="36873" txt_i="13819" txt_f="13825">months</offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="36926" xml_f="36930" txt_i="13826" txt_f="13830">0.54</offsets></bold><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="36999" xml_f="37007" txt_i="13831" txt_f="13839">mutation</offsets></bold><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="37063" xml_f="37064" txt_i="13840" txt_f="13841"> </offsets><hr></hr></td><td align="center" valign="bottom"><offsets xml_i="37113" xml_f="37114" txt_i="13842" txt_f="13843"> </offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="37167" xml_f="37171" txt_i="13844" txt_f="13848">0.83</offsets></bold><offsets xml_i="37178" xml_f="37190" txt_i="13848" txt_f="13860"> (0.45-1.53)</offsets><hr></hr></td></tr><tr><td align="left" valign="bottom"><bold><offsets xml_i="37252" xml_f="37260" txt_i="13861" txt_f="13869">Codon 13</offsets></bold><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="37322" xml_f="37326" txt_i="13870" txt_f="13874">16.1</offsets></bold><offsets xml_i="37333" xml_f="37339" txt_i="13874" txt_f="13880">months</offsets><hr></hr></td><td align="center" valign="bottom"><bold><offsets xml_i="37394" xml_f="37398" txt_i="13881" txt_f="13885">16.5</offsets></bold><offsets xml_i="37405" xml_f="37411" txt_i="13885" txt_f="13891">months</offsets><hr></hr></td><td align="left" valign="bottom"><bold><offsets xml_i="37464" xml_f="37468" txt_i="13892" txt_f="13896">0.62</offsets></bold><hr></hr></td></tr><tr><td align="left"><bold><offsets xml_i="37521" xml_f="37529" txt_i="13897" txt_f="13905">mutation</offsets></bold></td><td align="center"><offsets xml_i="37560" xml_f="37561" txt_i="13906" txt_f="13907"> </offsets></td><td align="center"><offsets xml_i="37585" xml_f="37586" txt_i="13908" txt_f="13909"> </offsets></td><td align="left"><bold><offsets xml_i="37614" xml_f="37618" txt_i="13910" txt_f="13914">1.39</offsets></bold><offsets xml_i="37625" xml_f="37637" txt_i="13914" txt_f="13926"> (0.37-5.37)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="37683" xml_f="37687" txt_i="13927" txt_f="13931">OS, </offsets><italic><offsets xml_i="37695" xml_f="37697" txt_i="13931" txt_f="13933">HR</offsets></italic><offsets xml_i="37706" xml_f="37739" txt_i="13933" txt_f="13966"> hazard ratio by cox regression, </offsets><italic><offsets xml_i="37747" xml_f="37749" txt_i="13966" txt_f="13968">CI</offsets></italic><offsets xml_i="37758" xml_f="37779" txt_i="13968" txt_f="13989"> confidence interval.</offsets></p></table-wrap-foot></table-wrap></sec></sec><sec><title><offsets xml_i="37838" xml_f="37864" txt_i="13992" txt_f="14018">Discussion and conclusions</offsets></title><p><offsets xml_i="37875" xml_f="37911" txt_i="14019" txt_f="14055">We assessed the prognostic value of </offsets><italic><offsets xml_i="37919" xml_f="37923" txt_i="14055" txt_f="14059">KRAS</offsets></italic><offsets xml_i="37932" xml_f="38443" txt_i="14059" txt_f="14570"> codon 12 and codon 13 mutations in tumor tissue from patients with advanced CRC recruited into a phase III clinical trial using CAPOX or FUFOX treatment regimens. This is the first randomized phase III trial retrospectively investigating the role of codon 13 mutations in advanced CRC patients treated with oxaliplatin combination chemotherapy only, without the addition of a monoclonal antibody. While the overall response rate in codon 13 patients was significantly lower, PFS was not different in the three </offsets><italic><offsets xml_i="38451" xml_f="38455" txt_i="14570" txt_f="14574">KRAS</offsets></italic><offsets xml_i="38464" xml_f="38868" txt_i="14574" txt_f="14978"> mutational groups. Interestingly, we found a substantial difference in PFS between patients with codon 12 and 13 mutant tumors when looking at infusional 5-FU versus capecitabine based regimens. Patients with codon 13 mutations seem to benefit more in terms of PFS from the oral capecitabine based protocols. Moreover, there was a strong trend towards better overall survival in patients with wild type </offsets><italic><offsets xml_i="38876" xml_f="38880" txt_i="14978" txt_f="14982">KRAS</offsets></italic><offsets xml_i="38889" xml_f="38913" txt_i="14982" txt_f="15006"> compared to all mutant </offsets><italic><offsets xml_i="38921" xml_f="38925" txt_i="15006" txt_f="15010">KRAS</offsets></italic><offsets xml_i="38934" xml_f="39042" txt_i="15010" txt_f="15118"> patients. Lastly, when analysing OS in patients who received second and further line therapy we found that </offsets><italic><offsets xml_i="39050" xml_f="39054" txt_i="15118" txt_f="15122">KRAS</offsets></italic><offsets xml_i="39063" xml_f="39078" txt_i="15122" txt_f="15137"> wild type and </offsets><italic><offsets xml_i="39086" xml_f="39090" txt_i="15137" txt_f="15141">KRAS</offsets></italic><offsets xml_i="39099" xml_f="39200" txt_i="15141" txt_f="15242"> mutant patients alike showed a significantly higher OS post progression when treated with cetuximab.</offsets></p><p><offsets xml_i="39207" xml_f="39290" txt_i="15243" txt_f="15326">A number of studies have looked at the potential prognostic or predictive value of </offsets><italic><offsets xml_i="39298" xml_f="39302" txt_i="15326" txt_f="15330">KRAS</offsets></italic><offsets xml_i="39311" xml_f="39421" txt_i="15330" txt_f="15440"> mutations on response rates and survival in patients with CRC and several studies found a negative impact of </offsets><italic><offsets xml_i="39429" xml_f="39433" txt_i="15440" txt_f="15444">KRAS</offsets></italic><offsets xml_i="39442" xml_f="39467" txt_i="15444" txt_f="15469"> mutations on prognosis [</offsets><xref ref-type="bibr" rid="B3"><offsets xml_i="39498" xml_f="39499" txt_i="15469" txt_f="15470">3</offsets></xref><offsets xml_i="39506" xml_f="39507" txt_i="15470" txt_f="15471">,</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="39538" xml_f="39539" txt_i="15471" txt_f="15472">7</offsets></xref><offsets xml_i="39546" xml_f="39547" txt_i="15472" txt_f="15473">,</offsets><xref ref-type="bibr" rid="B15"><offsets xml_i="39579" xml_f="39581" txt_i="15473" txt_f="15475">15</offsets></xref><offsets xml_i="39588" xml_f="39589" txt_i="15475" txt_f="15476">-</offsets><xref ref-type="bibr" rid="B17"><offsets xml_i="39621" xml_f="39623" txt_i="15476" txt_f="15478">17</offsets></xref><offsets xml_i="39630" xml_f="39632" txt_i="15478" txt_f="15480">].</offsets></p><p><offsets xml_i="39639" xml_f="39880" txt_i="15481" txt_f="15722">Recently, a number of randomized trials have included translational research programs to evaluate certain target genes and their role as prognostic or predictive markers in patients with metastatic disease. Thereby, the mutational status of </offsets><italic><offsets xml_i="39888" xml_f="39892" txt_i="15722" txt_f="15726">KRAS</offsets></italic><offsets xml_i="39901" xml_f="40004" txt_i="15726" txt_f="15829"> has now been established as a strong predictive marker of resistance to anti-EGFR-antibody treatment [</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="40035" xml_f="40036" txt_i="15829" txt_f="15830">5</offsets></xref><offsets xml_i="40043" xml_f="40044" txt_i="15830" txt_f="15831">,</offsets><xref ref-type="bibr" rid="B8"><offsets xml_i="40075" xml_f="40076" txt_i="15831" txt_f="15832">8</offsets></xref><offsets xml_i="40083" xml_f="40084" txt_i="15832" txt_f="15833">,</offsets><xref ref-type="bibr" rid="B10"><offsets xml_i="40116" xml_f="40118" txt_i="15833" txt_f="15835">10</offsets></xref><offsets xml_i="40125" xml_f="40188" txt_i="15835" txt_f="15898">], although some trials could not fully confirm these results [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="40219" xml_f="40220" txt_i="15898" txt_f="15899">4</offsets></xref><offsets xml_i="40227" xml_f="40228" txt_i="15899" txt_f="15900">,</offsets><xref ref-type="bibr" rid="B11"><offsets xml_i="40260" xml_f="40262" txt_i="15900" txt_f="15902">11</offsets></xref><offsets xml_i="40269" xml_f="40309" txt_i="15902" txt_f="15942">]. We still do not exactly know whether </offsets><italic><offsets xml_i="40317" xml_f="40321" txt_i="15942" txt_f="15946">KRAS</offsets></italic><offsets xml_i="40330" xml_f="40505" txt_i="15946" txt_f="16121"> mutations influence the response to other treatment regimens such as standard chemotherapy or bevacizumab combinations. While bevacizumab efficacy seems independent from the </offsets><italic><offsets xml_i="40513" xml_f="40517" txt_i="16121" txt_f="16125">KRAS</offsets></italic><offsets xml_i="40526" xml_f="40535" txt_i="16125" txt_f="16134"> status [</offsets><xref ref-type="bibr" rid="B18"><offsets xml_i="40567" xml_f="40569" txt_i="16134" txt_f="16136">18</offsets></xref><offsets xml_i="40576" xml_f="40648" txt_i="16136" txt_f="16208">], the activity of certain chemotherapeutic agents may be influenced by </offsets><italic><offsets xml_i="40656" xml_f="40660" txt_i="16208" txt_f="16212">KRAS</offsets></italic><offsets xml_i="40669" xml_f="40768" txt_i="16212" txt_f="16311"> mutations. There, patients seem to do worse under oxaliplatin combinations when carrying a mutant </offsets><italic><offsets xml_i="40776" xml_f="40780" txt_i="16311" txt_f="16315">KRAS</offsets></italic><offsets xml_i="40789" xml_f="40824" txt_i="16315" txt_f="16350"> gene within their primary cancer [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="40855" xml_f="40856" txt_i="16350" txt_f="16351">4</offsets></xref><offsets xml_i="40863" xml_f="40864" txt_i="16351" txt_f="16352">,</offsets><xref ref-type="bibr" rid="B5"><offsets xml_i="40895" xml_f="40896" txt_i="16352" txt_f="16353">5</offsets></xref><offsets xml_i="40903" xml_f="40904" txt_i="16353" txt_f="16354">,</offsets><xref ref-type="bibr" rid="B7"><offsets xml_i="40935" xml_f="40936" txt_i="16354" txt_f="16355">7</offsets></xref><offsets xml_i="40943" xml_f="40944" txt_i="16355" txt_f="16356">,</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="40976" xml_f="40978" txt_i="16356" txt_f="16358">16</offsets></xref><offsets xml_i="40985" xml_f="40986" txt_i="16358" txt_f="16359">,</offsets><xref ref-type="bibr" rid="B19"><offsets xml_i="41018" xml_f="41020" txt_i="16359" txt_f="16361">19</offsets></xref><offsets xml_i="41027" xml_f="41028" txt_i="16361" txt_f="16362">,</offsets><xref ref-type="bibr" rid="B20"><offsets xml_i="41060" xml_f="41062" txt_i="16362" txt_f="16364">20</offsets></xref><offsets xml_i="41069" xml_f="41217" txt_i="16364" txt_f="16512">]. For example, in the recently reported COIN study patients under oxaliplatin/5-FU combinations showed a median PFS of 8.6 months in the wild type </offsets><italic><offsets xml_i="41225" xml_f="41229" txt_i="16512" txt_f="16516">KRAS</offsets></italic><offsets xml_i="41238" xml_f="41290" txt_i="16516" txt_f="16568"> cohort while median PFS was only 6.9 months in the </offsets><italic><offsets xml_i="41298" xml_f="41302" txt_i="16568" txt_f="16572">KRAS</offsets></italic><offsets xml_i="41311" xml_f="41327" txt_i="16572" txt_f="16588"> mutant cohort [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="41358" xml_f="41359" txt_i="16588" txt_f="16589">4</offsets></xref><offsets xml_i="41366" xml_f="41398" txt_i="16589" txt_f="16621">]. Similarily, ORR was lower in </offsets><italic><offsets xml_i="41406" xml_f="41410" txt_i="16621" txt_f="16625">KRAS</offsets></italic><offsets xml_i="41419" xml_f="41550" txt_i="16625" txt_f="16756"> mutant patients receiving chemotherapy with oxaliplatin only (41% vs. 50%). Furthermore, another recent small study evaluated the </offsets><italic><offsets xml_i="41558" xml_f="41562" txt_i="16756" txt_f="16760">KRAS</offsets></italic><offsets xml_i="41571" xml_f="41718" txt_i="16760" txt_f="16907"> status in 66 patients receiving a second line chemotherapy with oxaliplatin and infusional 5-FU refractory to 5-FU/irinotecan based chemotherapy [</offsets><xref ref-type="bibr" rid="B16"><offsets xml_i="41750" xml_f="41752" txt_i="16907" txt_f="16909">16</offsets></xref><offsets xml_i="41759" xml_f="41933" txt_i="16909" txt_f="17083">]. This study found a significantly lower response rate (7% vs. 27%, p = 0.026) and significantly shorter median PFS (3.1 vs. 5.2 months, p = 0.007) for patients with mutant </offsets><italic><offsets xml_i="41941" xml_f="41945" txt_i="17083" txt_f="17087">KRAS</offsets></italic><offsets xml_i="41954" xml_f="41998" txt_i="17087" txt_f="17131"> tumors compared to patients with wild type </offsets><italic><offsets xml_i="42006" xml_f="42010" txt_i="17131" txt_f="17135">KRAS</offsets></italic><offsets xml_i="42019" xml_f="42064" txt_i="17135" txt_f="17180"> tumors under oxaliplatin containing therapy.</offsets></p><p><offsets xml_i="42071" xml_f="42148" txt_i="17181" txt_f="17258">Very recently, experimental and some clinical reports suggested that not all </offsets><italic><offsets xml_i="42156" xml_f="42160" txt_i="17258" txt_f="17262">KRAS</offsets></italic><offsets xml_i="42169" xml_f="42194" txt_i="17262" txt_f="17287"> mutations behave alike [</offsets><xref ref-type="bibr" rid="B12"><offsets xml_i="42226" xml_f="42228" txt_i="17287" txt_f="17289">12</offsets></xref><offsets xml_i="42235" xml_f="42236" txt_i="17289" txt_f="17290">,</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="42268" xml_f="42270" txt_i="17290" txt_f="17292">13</offsets></xref><offsets xml_i="42277" xml_f="42278" txt_i="17292" txt_f="17293">,</offsets><xref ref-type="bibr" rid="B21"><offsets xml_i="42310" xml_f="42312" txt_i="17293" txt_f="17295">21</offsets></xref><offsets xml_i="42319" xml_f="42401" txt_i="17295" txt_f="17377">]. In fact, there is evidence that patients carrying mutations in codon 13 of the </offsets><italic><offsets xml_i="42409" xml_f="42413" txt_i="17377" txt_f="17381">KRAS</offsets></italic><offsets xml_i="42422" xml_f="42595" txt_i="17381" txt_f="17554"> oncogene which is found in about 8% of patients with advanced CRC have a substantially worse overall prognosis but may, on the other hand, benefit from anti-EGFR-treatment.</offsets></p><p><offsets xml_i="42602" xml_f="43070" txt_i="17555" txt_f="18023">In our study we found a similar rate of codon 13 mutations as described before. The overall response rate in patients with codon 13 mutations in our analysis was as low as 23%, significantly lower than in patients with wild type or codon 12 mutations. The CRYSTAL- and the OPUS-studies alike found low response rates in patients with codon 13 mutations treated with combination chemotherapy only (17% for irinotecan combinations and 33% for oxaliplatin combinations) [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="43102" xml_f="43104" txt_i="18023" txt_f="18025">13</offsets></xref><offsets xml_i="43111" xml_f="43240" txt_i="18025" txt_f="18154">]. In contrast to previous reports, ORR was substantially higher in this codon 13 mutant patient cohort when cetuximab was added.</offsets></p><p><offsets xml_i="43247" xml_f="43583" txt_i="18155" txt_f="18491">Our study was conducted using combination chemotherapy with oxaliplatin in first line treatment without the addition of monoclonal antibodies. Interestingly, the PFS and OS in our codon 13 cohort compared to the other mutated groups and other previously published works was rather long with a PFS of 10 months and an OS of 16.5 months [</offsets><xref ref-type="bibr" rid="B13"><offsets xml_i="43615" xml_f="43617" txt_i="18491" txt_f="18493">13</offsets></xref><offsets xml_i="43624" xml_f="43906" txt_i="18493" txt_f="18775">]. We do not know why PFS and OS of the codon 13 cohort in our study was prolonged when response rates were as low as 23%. Either low patient numbers or a yet unknown functional mechanism of codon 13 mutated KRAS proteins within colon cancers may be responsible for the observation.</offsets></p><p><offsets xml_i="43913" xml_f="44001" txt_i="18776" txt_f="18864">The present analysis also suggests that there may be an interaction between the type of </offsets><italic><offsets xml_i="44009" xml_f="44013" txt_i="18864" txt_f="18868">KRAS</offsets></italic><offsets xml_i="44022" xml_f="44391" txt_i="18868" txt_f="19237"> mutation and the mode of application of 5-FU, i.e. whether administered intravenously or orally as capecitabine. In particular, patients with codon 13 mutations showed longer median PFS intervals when receiving capecitabine compared to infusional 5-FU. Although the overall efficacy of infusional 5-FU and capecitabine in advanced CRC has been found to be comparable [</offsets><xref ref-type="bibr" rid="B22"><offsets xml_i="44423" xml_f="44425" txt_i="19237" txt_f="19239">22</offsets></xref><offsets xml_i="44432" xml_f="44614" txt_i="19239" txt_f="19421">] there may be some patient subpopulations or treatment regimen where the mode of application of the fluoropyrimidine is more crucial. For example, anti-EGFR antibodies in wild type </offsets><italic><offsets xml_i="44622" xml_f="44626" txt_i="19421" txt_f="19425">KRAS</offsets></italic><offsets xml_i="44635" xml_f="44759" txt_i="19425" txt_f="19549"> patients may only be active when infusional 5-FU regimens are used, but not when capecitabine based protocols are applied [</offsets><xref ref-type="bibr" rid="B4"><offsets xml_i="44790" xml_f="44791" txt_i="19549" txt_f="19550">4</offsets></xref><offsets xml_i="44798" xml_f="44800" txt_i="19550" txt_f="19552">].</offsets></p><p><offsets xml_i="44807" xml_f="44835" txt_i="19553" txt_f="19581">The potential resistance of </offsets><italic><offsets xml_i="44843" xml_f="44847" txt_i="19581" txt_f="19585">KRAS</offsets></italic><offsets xml_i="44856" xml_f="45948" txt_i="19585" txt_f="20677"> mutated tumors to oxaliplatin containing regimens seems interesting and may reveal more general mechanisms of drug resistance in cancers. Oxaliplatin belongs to the platinum containing compounds like cisplatinum and carboplatinum. Metabolites of platinum compounds interact with DNA and form crosslinks. In addition, platinum-DNA-adducts strongly inhibit DNA polymerases and therefore act antineoplastic. Some authors have studied in cell culture systems and preclinical models the influence of oncogenic RAS mutations on the activity of platinum compounds and found that the nucleotide excision repair protein ERCC-1 may be upregulated through activated RAS. ERCC-1 may subsequently activate DNA repair capacity and thus mediate platinum resistance. A recent study evaluated the role of ERCC-1 mRNA levels in 191 patients treated with FOLFOX within the CONFIRM1 and CONFIRM2 studies. Low ERCC-1 gene expression was correlated with higher response rates to FOLFOX chemotherapy and better overall survival. In contrast patients with high ERCC-1 did not have benefit from FOLFOX chemotherapy [</offsets><xref ref-type="bibr" rid="B23"><offsets xml_i="45980" xml_f="45982" txt_i="20677" txt_f="20679">23</offsets></xref><offsets xml_i="45989" xml_f="46119" txt_i="20679" txt_f="20809">]. Of note, we have examined the expression of ERCC-1 by immunohistochemistry in our cohort and found no obvious correlation with </offsets><italic><offsets xml_i="46127" xml_f="46131" txt_i="20809" txt_f="20813">KRAS</offsets></italic><offsets xml_i="46140" xml_f="46331" txt_i="20813" txt_f="21004"> status, response rates or survival and ERCC-1 expression (data not shown). The analysis of the ERCC-1 expression levels by RT-PCR as reported by other studies has not been performed so far [</offsets><xref ref-type="bibr" rid="B24"><offsets xml_i="46363" xml_f="46365" txt_i="21004" txt_f="21006">24</offsets></xref><offsets xml_i="46372" xml_f="46374" txt_i="21006" txt_f="21008">].</offsets></p><p><offsets xml_i="46381" xml_f="46485" txt_i="21009" txt_f="21113">Median overall survival after first progression correlated with cetuximab-treatment in patients bearing </offsets><italic><offsets xml_i="46493" xml_f="46497" txt_i="21113" txt_f="21117">KRAS</offsets></italic><offsets xml_i="46506" xml_f="46686" txt_i="21117" txt_f="21297"> wild type and mutant tumors alike. The role of codon 13 mutation in this setting appears minor since there were only two patients with a codon 13 mutated tumor in the cohort with </offsets><italic><offsets xml_i="46694" xml_f="46698" txt_i="21297" txt_f="21301">KRAS</offsets></italic><offsets xml_i="46707" xml_f="47222" txt_i="21301" txt_f="21816"> mutations of whom we know about cetuximab application. However, there are at least two limitations to our analysis. First, numbers are low in particular in the cetuximab group and secondly addition of cetuximab was not randomized for. There seems to be a selection of patients with good performance status and good prognosis who received third line therapy compared to those who did not. Therefore, we can not draw definite conclusions from our analysis whether anti-EGFR antibodies are effective in patients with </offsets><italic><offsets xml_i="47230" xml_f="47234" txt_i="21816" txt_f="21820">KRAS</offsets></italic><offsets xml_i="47243" xml_f="47266" txt_i="21820" txt_f="21843"> G13D mutations or not.</offsets></p><p><offsets xml_i="47273" xml_f="47473" txt_i="21844" txt_f="22044">We presently can not draw final conclusions regarding patient management from this study. It remains unclear whether chemotherapy backbones with irinotecan are less prone to interactions with mutated </offsets><italic><offsets xml_i="47481" xml_f="47485" txt_i="22044" txt_f="22048">KRAS</offsets></italic><offsets xml_i="47494" xml_f="47811" txt_i="22048" txt_f="22365"> because data regarding this issue are conflicting. Independent validation of the findings is essential. Therefore, the standardized, thorough and comprehensive collection of tissue and blood samples of all trial patients within independent cancer tissue banks should be a major goal of modern clinical cancer trials.</offsets></p></sec><sec><title><offsets xml_i="47833" xml_f="47837" txt_i="22367" txt_f="22371">Misc</offsets></title><p><offsets xml_i="47848" xml_f="47915" txt_i="22372" txt_f="22439">Anke Reinacher-Schick and Karsten Schulmann are contributed equally</offsets></p></sec><sec><title><offsets xml_i="47937" xml_f="47956" txt_i="22441" txt_f="22460">Competing interests</offsets></title><p><offsets xml_i="47967" xml_f="48749" txt_i="22461" txt_f="23243">D. Arnold: Honoraria from Roche, Sanofi-Aventis. U. Graeven: Honoraria from Roche, Amgen, Sanofi-Aventis, Merck-Serono; Advisory Board Roche, Amgen. A. Reinacher-Schick: Honoraria from Amgen, Roche, Pfizer, Sanofi-Aventis; Advisory board member: Amgen, Roche, Pfizer; Studies sponsored by: Roche, Sanofi-Aventis. W. Schmiegel: Honoraria from Merck-Serono, Roche, Abott, Amgen, Astra-Zeneca, Pfizer, Falk; Advisory board member: Astra-Zeneca, Roche, Amgen; Studies sponsored by: Novartis, Amgen, Roche. K. Schulmann: Honoraria from Astra-Zeneca, Amgen, Falk; Travel support: Pfizer, Merck-Serono, Novartis. A. Tannapfel: Honoraria from Amgen, Roche, Pfizer, Sanofi-Aventis. Studies sponsored by: Roche, Sanofi-Aventis. The other authors declare that they have no competing interests.</offsets></p></sec><sec><title><offsets xml_i="48771" xml_f="48793" txt_i="23245" txt_f="23267">Authors’ contributions</offsets></title><p><offsets xml_i="48804" xml_f="49879" txt_i="23268" txt_f="24343">ARS and KS: analysis and interpretation of data, drafting of the manuscript, revising the manuscript critically, final approval of manuscript. DPM: interpretation of data, biostatistic analysis, revising the manuscript critically, final approval of manuscript. NB: acquisition of data, analysis and interpretation of data, final approval of manuscript. UG: conception and design, recruitment of patients, revising the manuscript critically, final approval of manuscript. MJ: acquisition of data, analysis and interpretation of data, final approval of manuscript. RG: recruitment of patients, revising the manuscript critically, final approval of manuscript. RP and WS: recruitment of patients, conception and design, revising the manuscript critically, final approval of manuscript. DA: conception and design, revising the manuscript critically, final approval of manuscript. AT: conception and design, data acquisition, analysis and interpretation of data, revising the manuscript critically, final approval of manuscript. All authors read and approved the final manuscript.</offsets></p></sec><sec><title><offsets xml_i="49901" xml_f="49924" txt_i="24345" txt_f="24368">Pre-publication history</offsets></title><p><offsets xml_i="49935" xml_f="49999" txt_i="24369" txt_f="24433">The pre-publication history for this paper can be accessed here:</offsets></p><p><ext-link ext-link-type="uri" xlink:href="http://www.biomedcentral.com/1471-2407/12/349/prepub"><offsets xml_i="50102" xml_f="50154" txt_i="24434" txt_f="24486">http://www.biomedcentral.com/1471-2407/12/349/prepub</offsets></ext-link></p></sec></body><back><sec><title>Acknowledgments</title><p>The authors wish to thank the AIO colorectal cancer study group for approval of the biomarker study, participating AIO study centers and their pathologists for providing tissue blocks, patients for providing written consent for biomarker studies, Petra Freitag for administrative regarding tissue acquisition and Sabine Geiger for DNA isolation. This work was funded by the Protein Research Unit Ruhr within Europe, PURE, from the Ministry of Science and Technology, Northrhine-Westfalia, Germany.</p></sec><ref-list><ref id="B1"><mixed-citation publication-type="journal"><name><surname>Bamford</surname><given-names>S</given-names></name><name><surname>Dawson</surname><given-names>E</given-names></name><name><surname>Forbes</surname><given-names>S</given-names></name><etal></etal><article-title>The COSMIC (Catalogue of Somatic Mutations in Cancer) database and website</article-title><source>Br J Cancer</source><year>2004</year><volume>91</volume><issue>2</issue><fpage>355</fpage><lpage>358</lpage><pub-id pub-id-type="pmid">15188009</pub-id></mixed-citation></ref><ref id="B2"><mixed-citation publication-type="journal"><name><surname>Etienne-Grimaldi</surname><given-names>MC</given-names></name><name><surname>Formento</surname><given-names>JL</given-names></name><name><surname>Francoual</surname><given-names>M</given-names></name><etal></etal><article-title>KRAS mutations and treatment outcome in colorectal cancer patients receiving exclusive fluoropyrimidine therapy</article-title><source>Clin Cancer Res</source><year>2008</year><volume>14</volume><issue>15</issue><fpage>4830</fpage><lpage>4835</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-07-4906</pub-id><pub-id pub-id-type="pmid">18676755</pub-id></mixed-citation></ref><ref id="B3"><mixed-citation publication-type="journal"><name><surname>Andreyev</surname><given-names>HJ</given-names></name><name><surname>Norman</surname><given-names>AR</given-names></name><name><surname>Cunningham</surname><given-names>D</given-names></name><etal></etal><article-title>RAS mutations in patients with colorectal cancer: the multicenter "RASCAL" study</article-title><source>J Natl Cancer Inst</source><year>1998</year><volume>90</volume><issue>9</issue><fpage>675</fpage><lpage>684</lpage><pub-id pub-id-type="doi">10.1093/jnci/90.9.675</pub-id><pub-id pub-id-type="pmid">9586664</pub-id></mixed-citation></ref><ref id="B4"><mixed-citation publication-type="journal"><name><surname>Maughan</surname><given-names>TS</given-names></name><name><surname>Adams</surname><given-names>RA</given-names></name><name><surname>Smith</surname><given-names>CG</given-names></name><etal></etal><article-title>Addition of cetuximab to oxaliplatin-based first-line combination chemotherapy for treatment of advanced colorectal cancer: results of the randomised phase 3 MRC COIN trial</article-title><source>Lancet</source><year>2011</year><volume>377</volume><issue>9783</issue><fpage>2103</fpage><lpage>2114</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(11)60613-2</pub-id><pub-id pub-id-type="pmid">21641636</pub-id></mixed-citation></ref><ref id="B5"><mixed-citation publication-type="journal"><name><surname>Bokemeyer</surname><given-names>C</given-names></name><name><surname>Bondarenko</surname><given-names>I</given-names></name><name><surname>Makhson</surname><given-names>A</given-names></name><etal></etal><article-title>Fluorouracil, leucovorin, and oxaliplatin with and without cetuximab in the first-line treatment of metastatic colorectal cancer</article-title><source>J Clin Oncol</source><year>2009</year><volume>27</volume><issue>5</issue><fpage>663</fpage><lpage>671</lpage><pub-id pub-id-type="doi">10.1200/JCO.2008.20.8397</pub-id><pub-id pub-id-type="pmid">19114683</pub-id></mixed-citation></ref><ref id="B6"><mixed-citation publication-type="journal"><name><surname>Punt</surname><given-names>CJ</given-names></name><name><surname>Tol</surname><given-names>J</given-names></name><name><surname>Rodenburg</surname><given-names>CJ</given-names></name><article-title>Randomized phase III study of capecitabine, oxaliplatin, and bevacizumab with or without cetuximab in advanced colorectal cancer (ACC), the CAIRO2 study of the Dutch Colorectal Cancer Group (DCCG)</article-title><source>J Clin Oncol</source><year>2008</year><volume>26</volume><comment>May 20 suppl; abstr BA401</comment></mixed-citation></ref><ref id="B7"><mixed-citation publication-type="journal"><name><surname>Richman</surname><given-names>SD</given-names></name><name><surname>Seymour</surname><given-names>MT</given-names></name><name><surname>Chambers</surname><given-names>P</given-names></name><etal></etal><article-title>KRAS and BRAF Mutations in Advanced Colorectal Cancer Are Associated With Poor Prognosis but Do Not Preclude Benefit From Oxaliplatin or Irinotecan: Results From the MRC FOCUS Trial</article-title><source>JCO Dec</source><year>2009</year><volume>10</volume><fpage>5931</fpage><lpage>5937</lpage></mixed-citation></ref><ref id="B8"><mixed-citation publication-type="journal"><name><surname>Amado</surname><given-names>RG</given-names></name><name><surname>Wolf</surname><given-names>M</given-names></name><name><surname>Peeters</surname><given-names>M</given-names></name><etal></etal><article-title>Wild type KRAS is required for panitumumab efficacy in patients with metastatic colorectal cancer</article-title><source>J Clin Oncol</source><year>2008</year><volume>26</volume><fpage>1626</fpage><lpage>1634</lpage><pub-id pub-id-type="doi">10.1200/JCO.2007.14.7116</pub-id><pub-id pub-id-type="pmid">18316791</pub-id></mixed-citation></ref><ref id="B9"><mixed-citation publication-type="journal"><name><surname>Lièvre</surname><given-names>A</given-names></name><name><surname>Bachet</surname><given-names>JB</given-names></name><name><surname>Boige</surname><given-names>V</given-names></name><etal></etal><article-title>KRAS mutations as an independent prognostic factor in patients with advanced colorectal cancer treated with cetuximab</article-title><source>J Clin Oncol</source><year>2008</year><volume>26</volume><issue>3</issue><fpage>374</fpage><lpage>379</lpage><pub-id pub-id-type="doi">10.1200/JCO.2007.12.5906</pub-id><pub-id pub-id-type="pmid">18202412</pub-id></mixed-citation></ref><ref id="B10"><mixed-citation publication-type="journal"><name><surname>Van Cutsem</surname><given-names>E</given-names></name><name><surname>Köhne</surname><given-names>CH</given-names></name><name><surname>Hitre</surname><given-names>E</given-names></name><etal></etal><article-title>Cetuximab and chemotherapy as initial treatment for metastatic colorectal cancer</article-title><source>N Engl J Med</source><year>2009</year><volume>360</volume><issue>14</issue><fpage>1408</fpage><lpage>1417</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa0805019</pub-id><pub-id pub-id-type="pmid">19339720</pub-id></mixed-citation></ref><ref id="B11"><mixed-citation publication-type="journal"><name><surname>Tveit</surname><given-names>K</given-names></name><name><surname>Guren</surname><given-names>T</given-names></name><name><surname>Glimelius</surname><given-names>B</given-names></name><etal></etal><article-title>Randomized phase III study of 5-fluorouracil/folinate/oxaliplatin given continuously or intermittently with or without cetuximab. as first-line treatment of metastatic colorectal cancer: The NORDIC VII Study (NCT00145314)</article-title><year>2010</year><volume>ESMO</volume><fpage># LBA20</fpage></mixed-citation></ref><ref id="B12"><mixed-citation publication-type="journal"><name><surname>De Roock</surname><given-names>W</given-names></name><name><surname>Claes</surname><given-names>B</given-names></name><name><surname>Bernascon</surname><given-names>D</given-names></name><etal></etal><article-title>Effects of KRAS, BRAF, NRAS, and PIK3CA mutations on the efficacy of cetuximab plus chemotherapy in chemotherapy-refractory metastatic colorectal cancer: a retrospective consortium analysis</article-title><source>Lancet Oncol</source><year>2011</year><volume>12</volume><issue>6</issue><fpage>594</fpage><lpage>603</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(10)70209-6</pub-id><pub-id pub-id-type="pmid">21163703</pub-id></mixed-citation></ref><ref id="B13"><mixed-citation publication-type="journal"><name><surname>Tejpar</surname><given-names>S</given-names></name><name><surname>Bokemeyer</surname><given-names>C</given-names></name><name><surname>Celik</surname><given-names>I</given-names></name><etal></etal><article-title>Influence of KRAS G13D mutations on outcome in patients with metastatic colorectal cancer (mCRC) treated with first-line chemotherapy with or without cetuximab</article-title><source>J Clin Oncol</source><year>2011</year><volume>29</volume><comment>suppl; abstr 3511</comment></mixed-citation></ref><ref id="B14"><mixed-citation publication-type="journal"><name><surname>Porschen</surname><given-names>R</given-names></name><name><surname>Arkenau</surname><given-names>HT</given-names></name><name><surname>Kubicka</surname><given-names>S</given-names></name><etal></etal><article-title>AIO Colorectal Study Group. Phase III study of capecitabine plus oxaliplatin compared with fluorouracil and leucovorin plus oxaliplatin in metastatic colorectal cancer: a final report of the AIO Colorectal Study Group</article-title><source>J Clin Oncol</source><year>2007</year><volume>25</volume><issue>27</issue><fpage>4217</fpage><lpage>4223</lpage><pub-id pub-id-type="doi">10.1200/JCO.2006.09.2684</pub-id><pub-id pub-id-type="pmid">17548840</pub-id></mixed-citation></ref><ref id="B15"><mixed-citation publication-type="journal"><name><surname>Bazan</surname><given-names>V</given-names></name><name><surname>Migliavacca</surname><given-names>M</given-names></name><name><surname>Zanna</surname><given-names>I</given-names></name><etal></etal><article-title>Specific codon 13 KRAS mutations are predictive of clinical outcome in colorectal cancer patients, whereas codon 12 KRAS mutations are associated with mucinous histotype</article-title><source>Ann Oncol</source><year>2002</year><volume>13</volume><issue>9</issue><fpage>1438</fpage><lpage>1446</lpage><pub-id pub-id-type="doi">10.1093/annonc/mdf226</pub-id><pub-id pub-id-type="pmid">12196370</pub-id></mixed-citation></ref><ref id="B16"><mixed-citation publication-type="other"><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>El-Khoueiry</surname><given-names>A</given-names></name><name><surname>Yang</surname><given-names>D</given-names></name><etal></etal><article-title>KRAS mutation status associated with clinical outcome in metastatic colorectal cancer patients treated with 5-fluorouracil/oxaliplatin</article-title><year>2009</year><comment>Gastrointestinal Cancers Symposium, abstr 340</comment></mixed-citation></ref><ref id="B17"><mixed-citation publication-type="journal"><name><surname>Moosmann</surname><given-names>N</given-names></name><name><surname>Fischer von Weikersthal</surname><given-names>L</given-names></name><name><surname>Vehling-Kaiser</surname><given-names>U</given-names></name><etal></etal><article-title>Cetuximab Plus Capecitabine and Irinotecan Compared With Cetuximab Plus Capecitabine and Oxaliplatin As First-Line Treatment for Patients With MetastaticColorectal Cancer: AIO KRK-0104—A Randomized Trial of the German AIO CRC Study Group</article-title><source>J Clin Oncol</source><year>2011</year><volume>29</volume><fpage>1050</fpage><lpage>1058</lpage><pub-id pub-id-type="doi">10.1200/JCO.2010.31.1936</pub-id><pub-id pub-id-type="pmid">21300933</pub-id></mixed-citation></ref><ref id="B18"><mixed-citation publication-type="journal"><name><surname>Ince</surname><given-names>WL</given-names></name><name><surname>Jubb</surname><given-names>AM</given-names></name><name><surname>Holden</surname><given-names>SN</given-names></name><etal></etal><article-title>Association of KRAS, b-raf, and p53 status with the treatment effect of bevacizumab</article-title><source>J Natl Cancer Inst</source><year>2005</year><volume>97</volume><issue>13</issue><fpage>981</fpage><lpage>989</lpage><pub-id pub-id-type="doi">10.1093/jnci/dji174</pub-id><pub-id pub-id-type="pmid">15998951</pub-id></mixed-citation></ref><ref id="B19"><mixed-citation publication-type="journal"><name><surname>Ocvirk</surname><given-names>J</given-names></name><name><surname>Brodowicz</surname><given-names>T</given-names></name><name><surname>Wrba</surname><given-names>F</given-names></name><etal></etal><article-title>Cetuximab plus FOLFOX6 or FOLFIRI in metastatic colorectal cancer: CECOG trial</article-title><source>World J Gastroenterol</source><year>2010</year><volume>16</volume><issue>25</issue><fpage>3133</fpage><lpage>43</lpage><pub-id pub-id-type="doi">10.3748/wjg.v16.i25.3133</pub-id><pub-id pub-id-type="pmid">20593498</pub-id></mixed-citation></ref><ref id="B20"><mixed-citation publication-type="journal"><name><surname>Reinacher-Schick</surname><given-names>A</given-names></name><name><surname>Arnold</surname><given-names>D</given-names></name><name><surname>Kubicka</surname><given-names>S</given-names></name><etal></etal><article-title>Impact of KRAS status on survival in patients (pts.) with metastatic colorectal cancer (MCRC) undergoing Bevacizumab (BEV) containing chemotherapy regimen – Analysis of the AIO Colorectal Cancer Study Group</article-title><source>Ann Oncol</source><year>2010</year><volume>21</volume><issue>8</issue><comment>abstr 584PD</comment></mixed-citation></ref><ref id="B21"><mixed-citation publication-type="journal"><name><surname>Stinchcombe</surname><given-names>TE</given-names></name><name><surname>Der</surname><given-names>CJ</given-names></name><article-title>Are all KRAS mutations created equal?</article-title><source>Lancet Oncol</source><year>2011</year><volume>12</volume><issue>8</issue><fpage>717</fpage><lpage>718</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(11)70200-5</pub-id><pub-id pub-id-type="pmid">21782508</pub-id></mixed-citation></ref><ref id="B22"><mixed-citation publication-type="journal"><name><surname>Arkenau</surname><given-names>HT</given-names></name><name><surname>Arnold</surname><given-names>D</given-names></name><name><surname>Cassidy</surname><given-names>J</given-names></name><etal></etal><article-title>Efficacy of oxaliplatin plus capecitabine or infusional fluorouracil/leucovorin in patients with metastatic colorectal cancer: a pooled analysis of randomized trials</article-title><source>J Clin Oncol</source><year>2008</year><volume>26</volume><issue>36</issue><fpage>5910</fpage><lpage>5917</lpage><pub-id pub-id-type="doi">10.1200/JCO.2008.16.7759</pub-id><pub-id pub-id-type="pmid">19018087</pub-id></mixed-citation></ref><ref id="B23"><mixed-citation publication-type="journal"><name><surname>Lenz</surname><given-names>HJ</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Shi</surname><given-names>MM</given-names></name><etal></etal><article-title>ERCC-1 gene expression levels and outcome to FOLFOX chemotherapy in patients enrolled in CONFIRM1 and CONFIRM2</article-title><source>J Clin Oncol</source><year>2008</year><volume>26</volume><comment>May 20 suppl; abstr 4131</comment></mixed-citation></ref><ref id="B24"><mixed-citation publication-type="journal"><name><surname>Friboulet</surname><given-names>L</given-names></name><name><surname>Barrios-Gonzales</surname><given-names>D</given-names></name><name><surname>Commo</surname><given-names>F</given-names></name><etal></etal><article-title>Molecular Characteristics of ERCC1-Negative versus ERCC1-Positive Tumors in Resected NSCLC</article-title><source>Clin Cancer Res</source><year>2011</year><volume>17</volume><issue>17</issue><fpage>5562</fpage><lpage>5572</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-11-0790</pub-id><pub-id pub-id-type="pmid">21750204</pub-id></mixed-citation></ref></ref-list></back></article>