<article article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Br J Cancer</journal-id><journal-id journal-id-type="iso-abbrev">Br. J. Cancer</journal-id><journal-title-group><journal-title>British Journal of Cancer</journal-title></journal-title-group><issn pub-type="ppub">0007-0920</issn><issn pub-type="epub">1532-1827</issn><publisher><publisher-name>Nature Publishing Group</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">23169292</article-id><article-id pub-id-type="pmc">3566829</article-id><article-id pub-id-type="pii">bjc2012495</article-id><article-id pub-id-type="doi">10.1038/bjc.2012.495</article-id><article-categories><subj-group subj-group-type="heading"><subject>Genetics &amp; Genomics</subject></subj-group></article-categories><title-group><article-title>EGFR pathway biomarkers in erlotinib-treated patients with advanced pancreatic cancer: translational results from the randomised, crossover phase 3 trial AIO-PK0104</article-title><alt-title alt-title-type="running">EGFR pathway in advanced pancreatic cancer</alt-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Boeck</surname><given-names>S</given-names></name><xref ref-type="aff" rid="aff1">1</xref><xref ref-type="corresp" rid="caf1">*</xref></contrib><contrib contrib-type="author"><name><surname>Jung</surname><given-names>A</given-names></name><xref ref-type="aff" rid="aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Laubender</surname><given-names>R P</given-names></name><xref ref-type="aff" rid="aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Neumann</surname><given-names>J</given-names></name><xref ref-type="aff" rid="aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Egg</surname><given-names>R</given-names></name><xref ref-type="aff" rid="aff1">1</xref><xref ref-type="aff" rid="aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Goritschan</surname><given-names>C</given-names></name><xref ref-type="aff" rid="aff1">1</xref><xref ref-type="aff" rid="aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Vehling-Kaiser</surname><given-names>U</given-names></name><xref ref-type="aff" rid="aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Winkelmann</surname><given-names>C</given-names></name><xref ref-type="aff" rid="aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Fischer von Weikersthal</surname><given-names>L</given-names></name><xref ref-type="aff" rid="aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Clemens</surname><given-names>M R</given-names></name><xref ref-type="aff" rid="aff7">7</xref></contrib><contrib contrib-type="author"><name><surname>Gauler</surname><given-names>T C</given-names></name><xref ref-type="aff" rid="aff8">8</xref></contrib><contrib contrib-type="author"><name><surname>Märten</surname><given-names>A</given-names></name><xref ref-type="aff" rid="aff9">9</xref></contrib><contrib contrib-type="author"><name><surname>Klein</surname><given-names>S</given-names></name><xref ref-type="aff" rid="aff10">10</xref></contrib><contrib contrib-type="author"><name><surname>Kojouharoff</surname><given-names>G</given-names></name><xref ref-type="aff" rid="aff11">11</xref></contrib><contrib contrib-type="author"><name><surname>Barner</surname><given-names>M</given-names></name><xref ref-type="aff" rid="aff12">12</xref></contrib><contrib contrib-type="author"><name><surname>Geissler</surname><given-names>M</given-names></name><xref ref-type="aff" rid="aff13">13</xref></contrib><contrib contrib-type="author"><name><surname>Greten</surname><given-names>T F</given-names></name><xref ref-type="aff" rid="aff14">14</xref></contrib><contrib contrib-type="author"><name><surname>Mansmann</surname><given-names>U</given-names></name><xref ref-type="aff" rid="aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Kirchner</surname><given-names>T</given-names></name><xref ref-type="aff" rid="aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Heinemann</surname><given-names>V</given-names></name><xref ref-type="aff" rid="aff1">1</xref></contrib><aff id="aff1"><label>1</label><institution>Department of Internal Medicine III and Comprehensive Cancer Center, Klinikum Grosshadern, Ludwig-Maximilians-University of Munich</institution>, Marchioninistr 15, Munich D-81377, <country>Germany</country></aff><aff id="aff2"><label>2</label><institution>Department of Pathology, Ludwig-Maximilians-University of Munich</institution>, Munich, <country>Germany</country></aff><aff id="aff3"><label>3</label><institution>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University of Munich</institution>, Munich, <country>Germany</country></aff><aff id="aff4"><label>4</label><institution>Practice for Medical Oncology</institution>, Landshut, <country>Germany</country></aff><aff id="aff5"><label>5</label><institution>Department of Internal Medicine, Krankenhaus Lutherstadt-Wittenberg, Lutherstadt-Wittenberg, Germany</institution></aff><aff id="aff6"><label>6</label><institution>Department of Oncology, Gesundheitszentrum St Marien GmbH</institution>, Amberg, <country>Germany</country></aff><aff id="aff7"><label>7</label><institution>Department of Internal Medicine I, Klinikum Mutterhaus Trier, Trier, Germany</institution></aff><aff id="aff8"><label>8</label><institution>Department of Medicine (Cancer Research), West German Cancer Center, University of Duisburg-Essen, Essen, Germany</institution></aff><aff id="aff9"><label>9</label><institution>Department of Surgery, University of Heidelberg</institution>, Heidelberg, <country>Germany</country></aff><aff id="aff10"><label>10</label><institution>Department of Internal Medicine IV, Klinikum Bayreuth, Bayreuth, Germany</institution></aff><aff id="aff11"><label>11</label><institution>Practice for Medical Oncology</institution>, Darmstadt, <country>Germany</country></aff><aff id="aff12"><label>12</label><institution>Department of Internal Medicine, Evangelisches Krankenhaus Köln-Weyertal, Köln, Germany</institution></aff><aff id="aff13"><label>13</label><institution>Department of Gastroenterology and Oncology, Klinikum Esslingen, Esslingen, Germany</institution></aff><aff id="aff14"><label>14</label><institution>Department of Gastroenterology, Hepatology and Endocrinology, Medical School Hannover</institution>, Hannover, <country>Germany</country></aff></contrib-group><author-notes><corresp id="caf1"><label>*</label>E-mail: <email>stefan.boeck@med.uni-muenchen.de</email></corresp></author-notes><pub-date pub-type="ppub"><day>05</day><month>02</month><year>2013</year></pub-date><pub-date pub-type="epub"><day>20</day><month>11</month><year>2012</year></pub-date><volume>108</volume><issue>2</issue><fpage>469</fpage><lpage>476</lpage><history><date date-type="received"><day>30</day><month>08</month><year>2012</year></date><date date-type="rev-recd"><day>15</day><month>10</month><year>2012</year></date><date date-type="accepted"><day>16</day><month>10</month><year>2012</year></date></history><permissions><copyright-statement>Copyright © 2013 Cancer Research UK</copyright-statement><copyright-year>2013</copyright-year><copyright-holder>Cancer Research UK</copyright-holder><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by-nc-sa/3.0/"><license-p>From twelve months after its original publication, this work is licensed under the Creative Commons Attribution-NonCommercial-Share Alike 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-sa/3.0/</license-p></license></permissions><abstract><sec><title><offsets xml_i="7828" xml_f="7839" txt_i="11" txt_f="22">Background:</offsets></title><p><offsets xml_i="7850" xml_f="8064" txt_i="23" txt_f="237">We aimed to identify molecular epidermal growth factor receptor (EGFR) tissue biomarkers in pancreatic cancer (PC) patients treated with the anti-EGFR agent erlotinib within the phase 3 randomised AIO-PK0104 study.</offsets></p></sec><sec><title><offsets xml_i="8086" xml_f="8094" txt_i="239" txt_f="247">Methods:</offsets></title><p><offsets xml_i="8105" xml_f="8599" txt_i="248" txt_f="742">AIO-PK0104 was a multicenter trial comparing gemcitabine/erlotinib followed by capecitabine with capecitabine/erlotinib followed by gemcitabine in advanced PC; primary study end point was the time-to-treatment failure after first- and second-line therapy (TTF2). Translational analyses were performed for KRAS exon 2 mutations, EGFR expression, PTEN expression, the EGFR intron 1 and exon 13 R497K polymorphism (PM). Biomarker data were correlated with TTF, overall survival (OS) and skin rash.</offsets></p></sec><sec><title><offsets xml_i="8621" xml_f="8629" txt_i="744" txt_f="752">Results:</offsets></title><p><offsets xml_i="8640" xml_f="8964" txt_i="753" txt_f="1077">Archival tumour tissue was available from 208 (74%) of the randomised patients. The KRAS mutations were found in 70% (121 out of 173) of patients and exclusively occurred in codon 12. The EGFR overexpression was detected in 89 out of 181 patients (49%) by immunohistochemistry (IHC), and 77 out of 166 patients (46%) had an </offsets><italic><offsets xml_i="8972" xml_f="8976" txt_i="1077" txt_f="1081">EGFR</offsets></italic><offsets xml_i="8985" xml_f="9021" txt_i="1081" txt_f="1117"> gene amplification by fluorescence </offsets><italic><offsets xml_i="9029" xml_f="9036" txt_i="1117" txt_f="1124">in-situ</offsets></italic><offsets xml_i="9045" xml_f="9201" txt_i="1124" txt_f="1280"> hybridisation (FISH); 30 out of 171 patients (18%) had a loss of PTEN expression, which was associated with an inferior TTF1 (first-line therapy; HR 0.61, </offsets><italic><offsets xml_i="9209" xml_f="9210" txt_i="1280" txt_f="1281">P</offsets></italic><offsets xml_i="9219" xml_f="9245" txt_i="1281" txt_f="1307">=0.02) and TTF2 (HR 0.66, </offsets><italic><offsets xml_i="9253" xml_f="9254" txt_i="1307" txt_f="1308">P</offsets></italic><offsets xml_i="9263" xml_f="9339" txt_i="1308" txt_f="1384">=0.04). The KRAS wild-type status was associated with improved OS (HR 1.68, </offsets><italic><offsets xml_i="9347" xml_f="9348" txt_i="1384" txt_f="1385">P</offsets></italic><offsets xml_i="9357" xml_f="9592" txt_i="1385" txt_f="1620">=0.005); no significant OS correlation was found for EGFR–IHC (HR 0.96), EGFR–FISH (HR 1.22), PTEN–IHC (HR 0.77), intron 1 (HR 0.91) or exon 13 R497K PM (HR 0.83). None of the six biomarkers correlated with the occurrence of skin rash.</offsets></p></sec><sec><title><offsets xml_i="9614" xml_f="9625" txt_i="1622" txt_f="1633">Conclusion:</offsets></title><p><offsets xml_i="9636" xml_f="9824" txt_i="1634" txt_f="1822">The KRAS wild-type was associated with an improved OS in erlotinib-treated PC patients in this phase 3 study; it remains to be defined whether this association is prognostic or predictive.</offsets></p></sec></abstract><kwd-group><kwd>EGFR</kwd><kwd>erlotinib</kwd><kwd>capecitabine</kwd><kwd>gemcitabine</kwd><kwd>pancreatic cancer</kwd></kwd-group></article-meta></front><body><p><offsets xml_i="10008" xml_f="10177" txt_i="1831" txt_f="2000">During the last decade, several molecular agents targeting different pathways have been investigated for the treatment of patients with advanced pancreatic cancer (PC) (</offsets><xref ref-type="bibr" rid="bib26"><offsets xml_i="10211" xml_f="10234" txt_i="2000" txt_f="2023">Stathis and Moore, 2010</offsets></xref><offsets xml_i="10241" xml_f="10508" txt_i="2023" txt_f="2290">). However, only the oral epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor erlotinib has shown a statistically significant (but clinically moderate) OS benefit when added to standard chemotherapy with gemcitabine in a large randomised phase 3 trial (</offsets><xref ref-type="bibr" rid="bib20"><offsets xml_i="10542" xml_f="10548" txt_i="2290" txt_f="2296">Moore </offsets><italic><offsets xml_i="10556" xml_f="10561" txt_i="2296" txt_f="2301">et al</offsets></italic><offsets xml_i="10570" xml_f="10576" txt_i="2301" txt_f="2307">, 2007</offsets></xref><offsets xml_i="10583" xml_f="10585" txt_i="2307" txt_f="2309">; </offsets><xref ref-type="bibr" rid="bib26"><offsets xml_i="10619" xml_f="10642" txt_i="2309" txt_f="2332">Stathis and Moore, 2010</offsets></xref><offsets xml_i="10649" xml_f="10913" txt_i="2332" txt_f="2596">). Erlotinib is a small molecule that targets the erbB1/HER1 receptor (EGFR) by inhibiting intracellular receptor transphosphorylation, thereby blocking downstream signal transduction pathways such as the PI3K–AKT–PTEN network or the RAS–RAF–MAPK–MEK–ERK cascade (</offsets><xref ref-type="bibr" rid="bib22"><offsets xml_i="10947" xml_f="10950" txt_i="2596" txt_f="2599">Ng </offsets><italic><offsets xml_i="10958" xml_f="10963" txt_i="2599" txt_f="2604">et al</offsets></italic><offsets xml_i="10972" xml_f="10978" txt_i="2604" txt_f="2610">, 2002</offsets></xref><offsets xml_i="10985" xml_f="10987" txt_i="2610" txt_f="2612">).</offsets></p><p><offsets xml_i="10994" xml_f="11193" txt_i="2613" txt_f="2812">In advanced PC, several clinical (e.g. stage of disease or performance status) and biochemical (e.g. serum tumour marker CA 19–9) factors with a prognostic significance have already been identified (</offsets><xref ref-type="bibr" rid="bib2"><offsets xml_i="11226" xml_f="11232" txt_i="2812" txt_f="2818">Boeck </offsets><italic><offsets xml_i="11240" xml_f="11245" txt_i="2818" txt_f="2823">et al</offsets></italic><offsets xml_i="11254" xml_f="11260" txt_i="2823" txt_f="2829">, 2007</offsets></xref><offsets xml_i="11267" xml_f="11269" txt_i="2829" txt_f="2831">, </offsets><xref ref-type="bibr" rid="bib3"><offsets xml_i="11302" xml_f="11306" txt_i="2831" txt_f="2835">2010</offsets></xref><offsets xml_i="11313" xml_f="12115" txt_i="2835" txt_f="3637">). In contrast, validated molecular prognostic markers or even predictive biomarkers for the treatment efficacy of biological agents (such as erlotinib or cetuximab) are still lacking in PC. All phase 3 trials investigating biologicals in PC have included unselected patient populations and up to now, only limited data have been published on translational substudies from large multicenter phase 3 trials. In other disease entities such as colorectal cancer (CRC) or non-small-cell lung cancer (NSCLC)—where anti-EGFR treatment strategies also have been studied widely—several predictive biomarkers for agents such as cetuximab, panitumumab, erlotinib or gefitinib were identified including mutations in the KRAS gene, activating mutations in EGFR exons 18 to 21 as well as the EGFR expression level (</offsets><xref ref-type="bibr" rid="bib11"><offsets xml_i="12149" xml_f="12159" txt_i="3637" txt_f="3647">Heinemann </offsets><italic><offsets xml_i="12167" xml_f="12172" txt_i="3647" txt_f="3652">et al</offsets></italic><offsets xml_i="12181" xml_f="12187" txt_i="3652" txt_f="3658">, 2009</offsets></xref><offsets xml_i="12194" xml_f="12196" txt_i="3658" txt_f="3660">; </offsets><xref ref-type="bibr" rid="bib13"><offsets xml_i="12230" xml_f="12256" txt_i="3660" txt_f="3686">Heist and Christiani, 2009</offsets></xref><offsets xml_i="12263" xml_f="12265" txt_i="3686" txt_f="3688">; </offsets><xref ref-type="bibr" rid="bib24"><offsets xml_i="12299" xml_f="12306" txt_i="3688" txt_f="3695">Pirker </offsets><italic><offsets xml_i="12314" xml_f="12319" txt_i="3695" txt_f="3700">et al</offsets></italic><offsets xml_i="12328" xml_f="12334" txt_i="3700" txt_f="3706">, 2012</offsets></xref><offsets xml_i="12341" xml_f="12414" txt_i="3706" txt_f="3779">). In advanced PC, only the authors of the erlotinib pivotal PA.3 trial (</offsets><italic><offsets xml_i="12422" xml_f="12423" txt_i="3779" txt_f="3780">n</offsets></italic><offsets xml_i="12432" xml_f="12509" txt_i="3780" txt_f="3857">=569) recently reported a translational biomarker analysis on KRAS mutation (</offsets><italic><offsets xml_i="12517" xml_f="12518" txt_i="3857" txt_f="3858">n</offsets></italic><offsets xml_i="12527" xml_f="12537" txt_i="3858" txt_f="3868">=117) and </offsets><italic><offsets xml_i="12545" xml_f="12549" txt_i="3868" txt_f="3872">EGFR</offsets></italic><offsets xml_i="12558" xml_f="12577" txt_i="3872" txt_f="3891"> gene copy number (</offsets><italic><offsets xml_i="12585" xml_f="12586" txt_i="3891" txt_f="3892">n</offsets></italic><offsets xml_i="12595" xml_f="12650" txt_i="3892" txt_f="3947">=107) in a subset of their study patients. Within this </offsets><italic><offsets xml_i="12658" xml_f="12666" txt_i="3947" txt_f="3955">post-hoc</offsets></italic><offsets xml_i="12675" xml_f="12725" txt_i="3955" txt_f="4005"> analysis, neither KRAS mutational status nor the </offsets><italic><offsets xml_i="12733" xml_f="12737" txt_i="4005" txt_f="4009">EGFR</offsets></italic><offsets xml_i="12746" xml_f="12854" txt_i="4009" txt_f="4117"> gene copy number was predictive for a survival benefit from the combination of gemcitabine plus erlotinib (</offsets><xref ref-type="bibr" rid="bib6"><offsets xml_i="12887" xml_f="12903" txt_i="4117" txt_f="4133">Da Cunha Santos </offsets><italic><offsets xml_i="12911" xml_f="12916" txt_i="4133" txt_f="4138">et al</offsets></italic><offsets xml_i="12925" xml_f="12931" txt_i="4138" txt_f="4144">, 2010</offsets></xref><offsets xml_i="12938" xml_f="12940" txt_i="4144" txt_f="4146">).</offsets></p><p><offsets xml_i="12947" xml_f="13182" txt_i="4147" txt_f="4382">Previous reports investigating potential prognostic and/or predictive molecular biomarkers of the EGFR pathway in PC were mainly derived from small, often retrospective single-centre studies and not from prospective randomised trials (</offsets><xref ref-type="bibr" rid="bib15"><offsets xml_i="13216" xml_f="13226" txt_i="4382" txt_f="4392">Immervoll </offsets><italic><offsets xml_i="13234" xml_f="13239" txt_i="4392" txt_f="4397">et al</offsets></italic><offsets xml_i="13248" xml_f="13254" txt_i="4397" txt_f="4403">, 2006</offsets></xref><offsets xml_i="13261" xml_f="13263" txt_i="4403" txt_f="4405">; </offsets><xref ref-type="bibr" rid="bib18"><offsets xml_i="13297" xml_f="13301" txt_i="4405" txt_f="4409">Lee </offsets><italic><offsets xml_i="13309" xml_f="13314" txt_i="4409" txt_f="4414">et al</offsets></italic><offsets xml_i="13323" xml_f="13329" txt_i="4414" txt_f="4420">, 2007</offsets></xref><offsets xml_i="13336" xml_f="13338" txt_i="4420" txt_f="4422">; </offsets><xref ref-type="bibr" rid="bib28"><offsets xml_i="13372" xml_f="13378" txt_i="4422" txt_f="4428">Tzeng </offsets><italic><offsets xml_i="13386" xml_f="13391" txt_i="4428" txt_f="4433">et al</offsets></italic><offsets xml_i="13400" xml_f="13407" txt_i="4433" txt_f="4440">, 2007b</offsets></xref><offsets xml_i="13414" xml_f="13416" txt_i="4440" txt_f="4442">; </offsets><xref ref-type="bibr" rid="bib5"><offsets xml_i="13449" xml_f="13454" txt_i="4442" txt_f="4447">Chen </offsets><italic><offsets xml_i="13462" xml_f="13467" txt_i="4447" txt_f="4452">et al</offsets></italic><offsets xml_i="13476" xml_f="13482" txt_i="4452" txt_f="4458">, 2010</offsets></xref><offsets xml_i="13489" xml_f="13597" txt_i="4458" txt_f="4566">). In those studies, the presence of a KRAS codon 12 mutation was found to be a negative prognostic factor (</offsets><xref ref-type="bibr" rid="bib18"><offsets xml_i="13631" xml_f="13635" txt_i="4566" txt_f="4570">Lee </offsets><italic><offsets xml_i="13643" xml_f="13648" txt_i="4570" txt_f="4575">et al</offsets></italic><offsets xml_i="13657" xml_f="13663" txt_i="4575" txt_f="4581">, 2007</offsets></xref><offsets xml_i="13670" xml_f="13672" txt_i="4581" txt_f="4583">; </offsets><xref ref-type="bibr" rid="bib5"><offsets xml_i="13705" xml_f="13710" txt_i="4583" txt_f="4588">Chen </offsets><italic><offsets xml_i="13718" xml_f="13723" txt_i="4588" txt_f="4593">et al</offsets></italic><offsets xml_i="13732" xml_f="13738" txt_i="4593" txt_f="4599">, 2010</offsets></xref><offsets xml_i="13745" xml_f="13829" txt_i="4599" txt_f="4683">), whereas an increased EGFR gene copy number did not seem to have an impact on OS (</offsets><xref ref-type="bibr" rid="bib18"><offsets xml_i="13863" xml_f="13867" txt_i="4683" txt_f="4687">Lee </offsets><italic><offsets xml_i="13875" xml_f="13880" txt_i="4687" txt_f="4692">et al</offsets></italic><offsets xml_i="13889" xml_f="13895" txt_i="4692" txt_f="4698">, 2007</offsets></xref><offsets xml_i="13902" xml_f="13904" txt_i="4698" txt_f="4700">; </offsets><xref ref-type="bibr" rid="bib28"><offsets xml_i="13938" xml_f="13944" txt_i="4700" txt_f="4706">Tzeng </offsets><italic><offsets xml_i="13952" xml_f="13957" txt_i="4706" txt_f="4711">et al</offsets></italic><offsets xml_i="13966" xml_f="13973" txt_i="4711" txt_f="4718">, 2007b</offsets></xref><offsets xml_i="13980" xml_f="14051" txt_i="4718" txt_f="4789">). Of note, the incidence of activating mutations in exon 18–21 of the </offsets><italic><offsets xml_i="14059" xml_f="14063" txt_i="4789" txt_f="4793">EGFR</offsets></italic><offsets xml_i="14072" xml_f="14184" txt_i="4793" txt_f="4902"> gene as well as in the BRAF proto-oncogene seemed to be very low (&lt;2%) in pancreatic ductal adenocarcinoma (</offsets><xref ref-type="bibr" rid="bib15"><offsets xml_i="14218" xml_f="14228" txt_i="4902" txt_f="4912">Immervoll </offsets><italic><offsets xml_i="14236" xml_f="14241" txt_i="4912" txt_f="4917">et al</offsets></italic><offsets xml_i="14250" xml_f="14256" txt_i="4917" txt_f="4923">, 2006</offsets></xref><offsets xml_i="14263" xml_f="14265" txt_i="4923" txt_f="4925">; </offsets><xref ref-type="bibr" rid="bib18"><offsets xml_i="14299" xml_f="14303" txt_i="4925" txt_f="4929">Lee </offsets><italic><offsets xml_i="14311" xml_f="14316" txt_i="4929" txt_f="4934">et al</offsets></italic><offsets xml_i="14325" xml_f="14331" txt_i="4934" txt_f="4940">, 2007</offsets></xref><offsets xml_i="14338" xml_f="14520" txt_i="4940" txt_f="5122">). The tumour-suppressor gene and AKT-inhibitor PTEN may also have a role in the sensitivity to anti-EGFR agents, as suggested by several translational analyses from metastatic CRC (</offsets><xref ref-type="bibr" rid="bib19"><offsets xml_i="14554" xml_f="14563" txt_i="5122" txt_f="5131">Loupakis </offsets><italic><offsets xml_i="14571" xml_f="14576" txt_i="5131" txt_f="5136">et al</offsets></italic><offsets xml_i="14585" xml_f="14591" txt_i="5136" txt_f="5142">, 2009</offsets></xref><offsets xml_i="14598" xml_f="14745" txt_i="5142" txt_f="5289">). In PC, no data on the prognostic or predictive role of PTEN (specifically in patients receiving anti-EGFR treatment) have been reported to date.</offsets></p><p><offsets xml_i="14752" xml_f="14797" txt_i="5290" txt_f="5335">Other groups focused on the investigation of </offsets><italic><offsets xml_i="14805" xml_f="14809" txt_i="5335" txt_f="5339">EGFR</offsets></italic><offsets xml_i="14818" xml_f="14860" txt_i="5339" txt_f="5381"> gene polymorphisms that may regulate the </offsets><italic><offsets xml_i="14868" xml_f="14872" txt_i="5381" txt_f="5385">EGFR</offsets></italic><offsets xml_i="14881" xml_f="15041" txt_i="5385" txt_f="5545"> gene transcription level and also could have a prognostic and/or predictive impact on outcome parameters. In addition, hypothesis-generating data suggest that </offsets><italic><offsets xml_i="15049" xml_f="15053" txt_i="5545" txt_f="5549">EGFR</offsets></italic><offsets xml_i="15062" xml_f="15332" txt_i="5549" txt_f="5819"> gene polymorphisms may also be associated with the occurrence of skin rash, an important side-effect of anti-EGFR treatment known to correlate with survival. In pre-clinical and early clinical studies, a highly polymorphic CA dinucleotide repeat in the intron 1 of the </offsets><italic><offsets xml_i="15340" xml_f="15344" txt_i="5819" txt_f="5823">EGFR</offsets></italic><offsets xml_i="15353" xml_f="15424" txt_i="5823" txt_f="5894"> gene was found to have a prognostic role after PC surgery, to mediate </offsets><italic><offsets xml_i="15432" xml_f="15440" txt_i="5894" txt_f="5902">in-vitro</offsets></italic><offsets xml_i="15449" xml_f="15524" txt_i="5902" txt_f="5977"> response to erlotinib, and to correlate with the occurrence of skin rash (</offsets><xref ref-type="bibr" rid="bib1"><offsets xml_i="15557" xml_f="15564" txt_i="5977" txt_f="5984">Amador </offsets><italic><offsets xml_i="15572" xml_f="15577" txt_i="5984" txt_f="5989">et al</offsets></italic><offsets xml_i="15586" xml_f="15592" txt_i="5989" txt_f="5995">, 2004</offsets></xref><offsets xml_i="15599" xml_f="15601" txt_i="5995" txt_f="5997">; </offsets><xref ref-type="bibr" rid="bib27"><offsets xml_i="15635" xml_f="15641" txt_i="5997" txt_f="6003">Tzeng </offsets><italic><offsets xml_i="15649" xml_f="15654" txt_i="6003" txt_f="6008">et al</offsets></italic><offsets xml_i="15663" xml_f="15670" txt_i="6008" txt_f="6015">, 2007a</offsets></xref><offsets xml_i="15677" xml_f="15679" txt_i="6015" txt_f="6017">; </offsets><xref ref-type="bibr" rid="bib14"><offsets xml_i="15713" xml_f="15719" txt_i="6017" txt_f="6023">Huang </offsets><italic><offsets xml_i="15727" xml_f="15732" txt_i="6023" txt_f="6028">et al</offsets></italic><offsets xml_i="15741" xml_f="15747" txt_i="6028" txt_f="6034">, 2009</offsets></xref><offsets xml_i="15754" xml_f="15756" txt_i="6034" txt_f="6036">; </offsets><xref ref-type="bibr" rid="bib10"><offsets xml_i="15790" xml_f="15797" txt_i="6036" txt_f="6043">Frolov </offsets><italic><offsets xml_i="15805" xml_f="15810" txt_i="6043" txt_f="6048">et al</offsets></italic><offsets xml_i="15819" xml_f="15825" txt_i="6048" txt_f="6054">, 2010</offsets></xref><offsets xml_i="15832" xml_f="15834" txt_i="6054" txt_f="6056">; </offsets><xref ref-type="bibr" rid="bib17"><offsets xml_i="15868" xml_f="15880" txt_i="6056" txt_f="6068">Klinghammer </offsets><italic><offsets xml_i="15888" xml_f="15893" txt_i="6068" txt_f="6073">et al</offsets></italic><offsets xml_i="15902" xml_f="15908" txt_i="6073" txt_f="6079">, 2010</offsets></xref><offsets xml_i="15915" xml_f="15966" txt_i="6079" txt_f="6130">). In addition, for the R497K PM in exon 13 of the </offsets><italic><offsets xml_i="15974" xml_f="15978" txt_i="6130" txt_f="6134">EGFR</offsets></italic><offsets xml_i="15987" xml_f="16068" txt_i="6134" txt_f="6215"> gene a prognostic role was previously suggested in patients with CRC and NSCLC (</offsets><xref ref-type="bibr" rid="bib31"><offsets xml_i="16102" xml_f="16107" txt_i="6215" txt_f="6220">Wang </offsets><italic><offsets xml_i="16115" xml_f="16120" txt_i="6220" txt_f="6225">et al</offsets></italic><offsets xml_i="16129" xml_f="16135" txt_i="6225" txt_f="6231">, 2007</offsets></xref><offsets xml_i="16142" xml_f="16144" txt_i="6231" txt_f="6233">; </offsets><xref ref-type="bibr" rid="bib25"><offsets xml_i="16178" xml_f="16185" txt_i="6233" txt_f="6240">Sasaki </offsets><italic><offsets xml_i="16193" xml_f="16198" txt_i="6240" txt_f="6245">et al</offsets></italic><offsets xml_i="16207" xml_f="16213" txt_i="6245" txt_f="6251">, 2009</offsets></xref><offsets xml_i="16220" xml_f="16222" txt_i="6251" txt_f="6253">).</offsets></p><p><offsets xml_i="16229" xml_f="16519" txt_i="6254" txt_f="6544">The aim of this explorative translational biomarker study based on the prospective AIO-PK0104 trial was: first, to assess the feasibility of translational research in PC multicenter phase 3 studies; second, to determine the frequency of alterations of components of the EGFR pathway and of </offsets><italic><offsets xml_i="16527" xml_f="16531" txt_i="6544" txt_f="6548">EGFR</offsets></italic><offsets xml_i="16540" xml_f="16741" txt_i="6548" txt_f="6749"> gene polymorphisms within the study population and third, to correlate biomarker data with the efficacy (time-to-treatment failure and OS) and safety end points (skin rash) from the clinical data set.</offsets></p><sec sec-type="materials|methods"><title><offsets xml_i="16786" xml_f="16807" txt_i="6750" txt_f="6771">Materials and methods</offsets></title><sec><title><offsets xml_i="16827" xml_f="16859" txt_i="6772" txt_f="6804">Translational patient population</offsets></title><p><offsets xml_i="16870" xml_f="17197" txt_i="6805" txt_f="7132">For the AIO-PK0104 phase 3 trial adult patients (aged 18–75 years) with a histologically or cytologically confirmed diagnosis of treatment-naïve, advanced, exocrine PC (stage III and IV) were eligible. Overall, 281 patients were randomised and 274 patients were classified eligible for the intention-to-treat (ITT) population (</offsets><xref ref-type="bibr" rid="bib12"><offsets xml_i="17231" xml_f="17241" txt_i="7132" txt_f="7142">Heinemann </offsets><italic><offsets xml_i="17249" xml_f="17254" txt_i="7142" txt_f="7147">et al</offsets></italic><offsets xml_i="17263" xml_f="17269" txt_i="7147" txt_f="7153">, 2012</offsets></xref><offsets xml_i="17276" xml_f="18061" txt_i="7153" txt_f="7938">). Archival formalin-fixed paraffin-embedded (FFPE) tissue, which was obtained during routine diagnostic procedures, was requested retrospectively from the participating centres/pathologists for this translational study. Cytological specimens were not suitable for this analysis. The FFPE histological tissue was accepted independent of its origin, for example surgical or biopsy specimens from primary pancreatic tumour, lymph nodes or distant metastases. The study had approval of the ethical committees in all participating German centres and patients gave written informed consent before any study-specific procedure. This study was conducted according to GCP/ICH guidelines and according to the Declaration of Helsinki and was registered at ClinicalTrials.gov, number NCT00440167.</offsets></p></sec><sec><title><offsets xml_i="18083" xml_f="18122" txt_i="7940" txt_f="7979">Analyses of molecular tissue biomarkers</offsets></title><p><offsets xml_i="18133" xml_f="18812" txt_i="7980" txt_f="8659">The translational analyses from AIO-PK0104 were performed centrally at the University of Munich, Department of Pathology (Max-Borst Laboratory for Cancer Research) by JN, AJ and TK. All obtained FFPE tumour blocks were checked for quality, tissue integrity and tumour content (HE staining) by a pathologist (JN) in a blinded manner. Tumour tissue was enriched by hand-guided microdissection using scalpel blades. Genomic DNA was extracted using QIAamp DNA FFPE micro or DNAeasy micro Kits (Qiagen, Hilden, Germany) according to the manufacturer's instructions. Such DNA was used as a template in PCRs together with Hot Star Taq DNA Polymerase (Qiagen) and gene-specific primers (</offsets><xref ref-type="supplementary-material" rid="sup1"><offsets xml_i="18863" xml_f="18885" txt_i="8659" txt_f="8681">Supplementary Table S1</offsets></xref><offsets xml_i="18892" xml_f="18894" txt_i="8681" txt_f="8683">).</offsets></p><sec><title><offsets xml_i="18910" xml_f="18930" txt_i="8684" txt_f="8704">KRAS exon 2 mutation</offsets></title><p><offsets xml_i="18941" xml_f="19197" txt_i="8705" txt_f="8961">Mutations in codons 12 and 13 of the KRAS proto-oncogene were investigated by pyrosequencing using KRAS exon 2-specific primers and PyroMark Gold kits (Qiagen). Pyrosequencing was performed employing a PyroMark Q24 device (Qiagen) as described previously (</offsets><xref ref-type="bibr" rid="bib21"><offsets xml_i="19231" xml_f="19239" txt_i="8961" txt_f="8969">Neumann </offsets><italic><offsets xml_i="19247" xml_f="19252" txt_i="8969" txt_f="8974">et al</offsets></italic><offsets xml_i="19261" xml_f="19267" txt_i="8974" txt_f="8980">, 2009</offsets></xref><offsets xml_i="19274" xml_f="19276" txt_i="8980" txt_f="8982">).</offsets></p></sec><sec><title><offsets xml_i="19298" xml_f="19341" txt_i="8984" txt_f="9027">Epidermal growth factor receptor expression</offsets></title><p><offsets xml_i="19352" xml_f="19783" txt_i="9028" txt_f="9459">Protein expression of the EGFR was analysed using a monoclonal mouse anti-EGFR antibody (pre-diluted CONFIRM EGFR antibody, clone 5B7; Ventana Medical Systems, Tucson, AZ, USA) and was performed on a Ventana Benchmark XT autostainer using the XT UltraView diaminobenzidine kit (Ventana Medical Systems). EGFR–IHC (immunohistochemistry) scoring (0–3) was performed blinded by JN and RE based on a previously published score for PC (</offsets><xref ref-type="bibr" rid="bib29"><offsets xml_i="19817" xml_f="19822" txt_i="9459" txt_f="9464">Ueda </offsets><italic><offsets xml_i="19830" xml_f="19835" txt_i="9464" txt_f="9469">et al</offsets></italic><offsets xml_i="19844" xml_f="19850" txt_i="9469" txt_f="9475">, 2004</offsets></xref><offsets xml_i="19857" xml_f="20102" txt_i="9475" txt_f="9720">). The following staining parameters were taken into account: percentage of positive cells, membrane or cytoplasmic staining, complete or incomplete staining. For this study, only samples with a score of 2 or 3 were judged as EGFR–IHC positive (</offsets><xref ref-type="bibr" rid="bib29"><offsets xml_i="20136" xml_f="20141" txt_i="9720" txt_f="9725">Ueda </offsets><italic><offsets xml_i="20149" xml_f="20154" txt_i="9725" txt_f="9730">et al</offsets></italic><offsets xml_i="20163" xml_f="20169" txt_i="9730" txt_f="9736">, 2004</offsets></xref><offsets xml_i="20176" xml_f="20178" txt_i="9736" txt_f="9738">).</offsets></p><p><offsets xml_i="20185" xml_f="20189" txt_i="9739" txt_f="9743">The </offsets><italic><offsets xml_i="20197" xml_f="20201" txt_i="9743" txt_f="9747">EGFR</offsets></italic><offsets xml_i="20210" xml_f="20259" txt_i="9747" txt_f="9796"> gene amplification was analysed by fluorescence </offsets><italic><offsets xml_i="20267" xml_f="20274" txt_i="9796" txt_f="9803">in-situ</offsets></italic><offsets xml_i="20283" xml_f="20706" txt_i="9803" txt_f="10226"> hybridisation (FISH) assays, which were carried out on 4 μm tissue sections containing representative tumour cells. The FISH assay was performed with the SPEC EGFR/CEN 7 Dual Color Probe (Zytovision, Bremerhaven, Germany) according to the manufacturer's instructions. Analyses were done by one observer (RE) using a fluorescence microscope: a minimum of 100 tumour cell nuclei was scored according to the number of green (</offsets><italic><offsets xml_i="20714" xml_f="20718" txt_i="10226" txt_f="10230">EGFR</offsets></italic><offsets xml_i="20727" xml_f="20838" txt_i="10230" txt_f="10341"> gene) and red (centromere region of chromosome 7) signals observed and grouped into six different categories (</offsets><xref ref-type="bibr" rid="bib4"><offsets xml_i="20871" xml_f="20880" txt_i="10341" txt_f="10350">Cappuzzo </offsets><italic><offsets xml_i="20888" xml_f="20893" txt_i="10350" txt_f="10355">et al</offsets></italic><offsets xml_i="20902" xml_f="20908" txt_i="10355" txt_f="10361">, 2005</offsets></xref><offsets xml_i="20915" xml_f="20917" txt_i="10361" txt_f="10363">; </offsets><xref ref-type="bibr" rid="bib6"><offsets xml_i="20950" xml_f="20966" txt_i="10363" txt_f="10379">Da Cunha Santos </offsets><italic><offsets xml_i="20974" xml_f="20979" txt_i="10379" txt_f="10384">et al</offsets></italic><offsets xml_i="20988" xml_f="20994" txt_i="10384" txt_f="10390">, 2010</offsets></xref><offsets xml_i="21001" xml_f="21084" txt_i="10390" txt_f="10473">). Samples with polysomy high or amplification were regarded as EGFR–FISH positive.</offsets></p></sec><sec><title><offsets xml_i="21106" xml_f="21125" txt_i="10475" txt_f="10494">The PTEN expression</offsets></title><p><offsets xml_i="21136" xml_f="21905" txt_i="10495" txt_f="11264">For the PTEN–IHC analyses, a monoclonal mouse anti-human PTEN antibody was used (clone 6H2.1; DAKO, Hamburg, Germany). After heat-induced epitope retrieval using Pro Taqs IV Antigen Enhancers (Quartett, Berlin, Germany) the primary antibody was incubated for 1 h at a dilution of 1 : 30. The slides were then washed and subsequently developed by addition of the detection system (ImmPRESS Reagent Kit Anti-MOUSE Ig, Vector Laboratories, Burlingame, CA, USA). The AEC+system (DAKO) was used as chromogen and the slides were then counterstained with hematoxylin (Vector Laboratories). To exclude nonspecific staining, system controls were included. As there are still no validated scoring systems to interpret staining for PTEN-IHC in PC, we adopted a score developed by </offsets><xref ref-type="bibr" rid="bib19"><offsets xml_i="21939" xml_f="21948" txt_i="11264" txt_f="11273">Loupakis </offsets><italic><offsets xml_i="21956" xml_f="21961" txt_i="11273" txt_f="11278">et al</offsets></italic><offsets xml_i="21970" xml_f="21977" txt_i="11278" txt_f="11285"> (2009)</offsets></xref><offsets xml_i="21984" xml_f="22176" txt_i="11285" txt_f="11477"> in CRC. For the PTEN–IHC scoring of 0–6 (performed blinded by JN and CG), the two parameters percentage of positive tumour cells and staining intensity were selected as described previously (</offsets><xref ref-type="bibr" rid="bib19"><offsets xml_i="22210" xml_f="22219" txt_i="11477" txt_f="11486">Loupakis </offsets><italic><offsets xml_i="22227" xml_f="22232" txt_i="11486" txt_f="11491">et al</offsets></italic><offsets xml_i="22241" xml_f="22247" txt_i="11491" txt_f="11497">, 2009</offsets></xref><offsets xml_i="22254" xml_f="22319" txt_i="11497" txt_f="11562">). Samples were defined as PTEN–IHC positive if the score was ⩾4.</offsets></p></sec><sec><title><italic><offsets xml_i="22349" xml_f="22353" txt_i="11564" txt_f="11568">EGFR</offsets></italic><offsets xml_i="22362" xml_f="22403" txt_i="11568" txt_f="11609"> gene polymorphism (intron 1 and exon 13)</offsets></title><p><offsets xml_i="22414" xml_f="22422" txt_i="11610" txt_f="11618">The (CA)</offsets><sub><italic><offsets xml_i="22435" xml_f="22436" txt_i="11618" txt_f="11619">n</offsets></italic></sub><offsets xml_i="22451" xml_f="22795" txt_i="11619" txt_f="11960"> polymorphism found in intron 1 of the EGFR as well as the R497K (G&gt;A) PM in exon 13 of the EGFR were investigated by subjecting PCR products to restriction fragment length polymorphism analysis on a Genetic Analyzer 3130 (Applied Biosystems, Darmstadt, Germany) (intron 1) and by pyrosequencing on a Pyromark Q24 device (Qiagen) (exon 13) (</offsets><xref ref-type="bibr" rid="bib31"><offsets xml_i="22829" xml_f="22834" txt_i="11960" txt_f="11965">Wang </offsets><italic><offsets xml_i="22842" xml_f="22847" txt_i="11965" txt_f="11970">et al</offsets></italic><offsets xml_i="22856" xml_f="22862" txt_i="11970" txt_f="11976">, 2007</offsets></xref><offsets xml_i="22869" xml_f="22871" txt_i="11976" txt_f="11978">; </offsets><xref ref-type="bibr" rid="bib27"><offsets xml_i="22905" xml_f="22911" txt_i="11978" txt_f="11984">Tzeng </offsets><italic><offsets xml_i="22919" xml_f="22924" txt_i="11984" txt_f="11989">et al</offsets></italic><offsets xml_i="22933" xml_f="22940" txt_i="11989" txt_f="11996">, 2007a</offsets></xref><offsets xml_i="22947" xml_f="23106" txt_i="11996" txt_f="12155">). For the statistical correlation the number of CA repeats was determined for both alleles separately but also the sum of both repeat numbers was calculated (</offsets><xref ref-type="bibr" rid="bib27"><offsets xml_i="23140" xml_f="23146" txt_i="12155" txt_f="12161">Tzeng </offsets><italic><offsets xml_i="23154" xml_f="23159" txt_i="12161" txt_f="12166">et al</offsets></italic><offsets xml_i="23168" xml_f="23175" txt_i="12166" txt_f="12173">, 2007a</offsets></xref><offsets xml_i="23182" xml_f="23184" txt_i="12173" txt_f="12175">; </offsets><xref ref-type="bibr" rid="bib10"><offsets xml_i="23218" xml_f="23225" txt_i="12175" txt_f="12182">Frolov </offsets><italic><offsets xml_i="23233" xml_f="23238" txt_i="12182" txt_f="12187">et al</offsets></italic><offsets xml_i="23247" xml_f="23253" txt_i="12187" txt_f="12193">, 2010</offsets></xref><offsets xml_i="23260" xml_f="23509" txt_i="12193" txt_f="12442">). For the EGFR exon 13 R497K PM, homozygous G on both alleles (Arg/Arg) was scored as wild-type sequence, whereas heterogeneous GA (Arg/Lys) or homogeneous A (Lys/Lys) on the two alleles were classified to be single-nucleotide polymorphisms (SNP) (</offsets><xref ref-type="bibr" rid="bib31"><offsets xml_i="23543" xml_f="23548" txt_i="12442" txt_f="12447">Wang </offsets><italic><offsets xml_i="23556" xml_f="23561" txt_i="12447" txt_f="12452">et al</offsets></italic><offsets xml_i="23570" xml_f="23576" txt_i="12452" txt_f="12458">, 2007</offsets></xref><offsets xml_i="23583" xml_f="23585" txt_i="12458" txt_f="12460">).</offsets></p></sec></sec><sec><title><offsets xml_i="23613" xml_f="23633" txt_i="12463" txt_f="12483">Statistical analyses</offsets></title><p><offsets xml_i="23644" xml_f="24113" txt_i="12484" txt_f="12953">All statistical analyses for the translational study of the AIO–PK0104 trial were performed centrally at the University of Munich, Institute of Medical Informatics, Biometry and Epidemiology by RPL and UM. Translational biomarker data were correlated with efficacy (time-to-treatment failure and OS) and safety study end points (skin rash) using univariate analyses. As appropriate, biomarker results were handled as dichotomous (e.g. KRAS mutation, EGFR–FISH positive </offsets><italic><offsets xml_i="24121" xml_f="24123" txt_i="12953" txt_f="12955">vs</offsets></italic><offsets xml_i="24132" xml_f="24360" txt_i="12955" txt_f="13183"> EGFR–FISH negative) or continuous variables (e.g. linear-scoring system 0–6 for PTEN–IHC). Time-to-event end points were analysed with the Kaplan–Meier method; differences were compared using the log-rank test with a two-sided </offsets><italic><offsets xml_i="24368" xml_f="24369" txt_i="13183" txt_f="13184">P</offsets></italic><offsets xml_i="24378" xml_f="24441" txt_i="13184" txt_f="13244">-value of &lt;0.05 being regarded as statistically significant.</offsets></p></sec></sec><sec sec-type="results"><title><offsets xml_i="24488" xml_f="24495" txt_i="13247" txt_f="13254">Results</offsets></title><sec><title><offsets xml_i="24515" xml_f="24538" txt_i="13255" txt_f="13278">Patient characteristics</offsets></title><p><offsets xml_i="24549" xml_f="24773" txt_i="13279" txt_f="13503">The FFPE tumour blocks were available from 208 of the 281 randomised patients (74%). The main reasons for not obtaining FFPE tissue in the remaining 73 cases were missing tissue according to the study centre or pathologist (</offsets><italic><offsets xml_i="24781" xml_f="24782" txt_i="13503" txt_f="13504">n</offsets></italic><offsets xml_i="24791" xml_f="24836" txt_i="13504" txt_f="13549">=46), disease confirmation by cytology only (</offsets><italic><offsets xml_i="24844" xml_f="24845" txt_i="13549" txt_f="13550">n</offsets></italic><offsets xml_i="24854" xml_f="24917" txt_i="13550" txt_f="13613">=13) or refusal of the pathologist to provide archival tissue (</offsets><italic><offsets xml_i="24925" xml_f="24926" txt_i="13613" txt_f="13614">n</offsets></italic><offsets xml_i="24935" xml_f="25117" txt_i="13614" txt_f="13796">=7). The ITT study population consisted of 274 eligible patients, and 206 patients were eligible for the translational population; detailed patient characteristics are summarised in </offsets><xref rid="tbl1" ref-type="table"><offsets xml_i="25151" xml_f="25158" txt_i="13796" txt_f="13803">Table 1</offsets></xref><offsets xml_i="25165" xml_f="25376" txt_i="13803" txt_f="14014">. With regard to important baseline parameters (e.g. age, gender, stage of disease, performance status), no significant imbalances between the ITT population and the translational study population were apparent.</offsets></p></sec><sec><title><offsets xml_i="25398" xml_f="25460" txt_i="14016" txt_f="14078">Frequency of alternations in molecular EGFR pathway biomarkers</offsets></title><p><offsets xml_i="25471" xml_f="25478" txt_i="14079" txt_f="14086">Within </offsets><xref rid="tbl2" ref-type="table"><offsets xml_i="25512" xml_f="25519" txt_i="14086" txt_f="14093">Table 2</offsets></xref><offsets xml_i="25526" xml_f="25715" txt_i="14093" txt_f="14282">, each of the six analysed markers was categorised as a dichotomous variable and a correlation between selected baseline patient characteristics and molecular marker results were performed.</offsets></p><sec><title><offsets xml_i="25731" xml_f="25770" txt_i="14283" txt_f="14322">KRAS exon 2 mutations (codon 12 and 13)</offsets></title><p><offsets xml_i="25781" xml_f="25975" txt_i="14323" txt_f="14514">The KRAS mutation analysis was technically successful in 173 cases, and a KRAS wildtype was detected in 52 FFPE tumour samples (30%). All mutations were within codon 12, with c.35G&gt;A-p.G12D (</offsets><italic><offsets xml_i="25983" xml_f="25984" txt_i="14514" txt_f="14515">n</offsets></italic><offsets xml_i="25993" xml_f="26037" txt_i="14515" txt_f="14556">=82 out of 121, 68%) and c.35G&gt;T-p.G12V (</offsets><italic><offsets xml_i="26045" xml_f="26046" txt_i="14556" txt_f="14557">n</offsets></italic><offsets xml_i="26055" xml_f="26269" txt_i="14557" txt_f="14771">=26 out of 121, 22%) being the two most frequent mutations observed. The distribution of clinical patient characteristics did not differ significantly between the two subgroups of KRAS mutation and KRAS wild-type (</offsets><xref rid="tbl2" ref-type="table"><offsets xml_i="26303" xml_f="26310" txt_i="14771" txt_f="14778">Table 2</offsets></xref><offsets xml_i="26317" xml_f="26394" txt_i="14778" txt_f="14855">); however, the median pre-treatment CA 19–9 level was significantly higher (</offsets><italic><offsets xml_i="26402" xml_f="26403" txt_i="14855" txt_f="14856">P</offsets></italic><offsets xml_i="26412" xml_f="26480" txt_i="14856" txt_f="14924">=0.03) in the KRAS mutant compared with the KRAS wild-type subgroup.</offsets></p></sec><sec><title><offsets xml_i="26502" xml_f="26537" txt_i="14926" txt_f="14961">Expression by IHC and fluorescence </offsets><italic><offsets xml_i="26545" xml_f="26552" txt_i="14961" txt_f="14968">in-situ</offsets></italic><offsets xml_i="26561" xml_f="26575" txt_i="14968" txt_f="14982"> hybridisation</offsets></title><p><offsets xml_i="26586" xml_f="26792" txt_i="14983" txt_f="15189">EGFR–IHC was successfully performed in 181 cases, with 89 cases (49%) being classified as EGFR–IHC positive. The EGFR gene amplification was analysed in 166 patients, 77 (46%) of them were found to have an </offsets><italic><offsets xml_i="26800" xml_f="26804" txt_i="15189" txt_f="15193">EGFR</offsets></italic><offsets xml_i="26813" xml_f="26928" txt_i="15193" txt_f="15308"> gene overexpression by FISH (EGFR–FISH positive). No correlation between EGFR–IHC and –FISH results was observed (</offsets><italic><offsets xml_i="26936" xml_f="26937" txt_i="15308" txt_f="15309">P</offsets></italic><offsets xml_i="26946" xml_f="27080" txt_i="15309" txt_f="15443">=0.97; detailed data not shown). The distribution of relevant baseline characteristics between the two subgroups of EGFR–IHC positive </offsets><italic><offsets xml_i="27088" xml_f="27090" txt_i="15443" txt_f="15445">vs</offsets></italic><offsets xml_i="27099" xml_f="27141" txt_i="15445" txt_f="15487"> EGFR–IHC negative and EGFR–FISH positive </offsets><italic><offsets xml_i="27149" xml_f="27151" txt_i="15487" txt_f="15489">vs</offsets></italic><offsets xml_i="27160" xml_f="27197" txt_i="15489" txt_f="15526"> EGFR–FISH negative is summarised in </offsets><xref rid="tbl2" ref-type="table"><offsets xml_i="27231" xml_f="27238" txt_i="15526" txt_f="15533">Table 2</offsets></xref><offsets xml_i="27245" xml_f="27260" txt_i="15533" txt_f="15548">, respectively.</offsets></p></sec><sec><title><offsets xml_i="27282" xml_f="27308" txt_i="15550" txt_f="15576">The PTEN expression by IHC</offsets></title><p><offsets xml_i="27319" xml_f="27531" txt_i="15577" txt_f="15789">The PTEN–IHC data were obtained from 171 patients, and a loss of PTEN (score 0–3) was detected in 30 cases (18%). A detailed distribution of the results for the continuous PTEN score (0–6) is included in Table 4.</offsets></p></sec><sec><title><offsets xml_i="27553" xml_f="27597" txt_i="15791" txt_f="15835">Epidermal growth factor receptor intron 1 PM</offsets></title><p><offsets xml_i="27608" xml_f="27681" txt_i="15836" txt_f="15909">The median number of CA repeats (sum of both alleles) in intron 1 of the </offsets><italic><offsets xml_i="27689" xml_f="27693" txt_i="15909" txt_f="15913">EGFR</offsets></italic><offsets xml_i="27702" xml_f="27883" txt_i="15913" txt_f="16094"> gene was 34 (186 evaluable patients), with a range from 28 to 41. For dichotomising the EGFR intron 1 variable, we applied the previously reported cutoff point of 36 (according to </offsets><xref ref-type="bibr" rid="bib27"><offsets xml_i="27917" xml_f="27923" txt_i="16094" txt_f="16100">Tzeng </offsets><italic><offsets xml_i="27931" xml_f="27936" txt_i="16100" txt_f="16105">et al</offsets></italic><offsets xml_i="27945" xml_f="27952" txt_i="16105" txt_f="16112">, 2007a</offsets></xref><offsets xml_i="27959" xml_f="27961" txt_i="16112" txt_f="16114">; </offsets><xref ref-type="bibr" rid="bib10"><offsets xml_i="27995" xml_f="28002" txt_i="16114" txt_f="16121">Frolov </offsets><italic><offsets xml_i="28010" xml_f="28015" txt_i="16121" txt_f="16126">et al</offsets></italic><offsets xml_i="28024" xml_f="28030" txt_i="16126" txt_f="16132">, 2010</offsets></xref><offsets xml_i="28037" xml_f="28039" txt_i="16132" txt_f="16134">; </offsets><xref rid="tbl2" ref-type="table"><offsets xml_i="28073" xml_f="28080" txt_i="16134" txt_f="16141">Table 2</offsets></xref><offsets xml_i="28087" xml_f="28153" txt_i="16141" txt_f="16207">). No significant correlation between the EGFR expression by IHC (</offsets><italic><offsets xml_i="28161" xml_f="28162" txt_i="16207" txt_f="16208">P</offsets></italic><offsets xml_i="28171" xml_f="28187" txt_i="16208" txt_f="16224">=0.26) or FISH (</offsets><italic><offsets xml_i="28195" xml_f="28196" txt_i="16224" txt_f="16225">P</offsets></italic><offsets xml_i="28205" xml_f="28267" txt_i="16225" txt_f="16287">=0.31) and the EGFR intron 1 PM was observed (data not shown).</offsets></p></sec><sec><title><offsets xml_i="28289" xml_f="28338" txt_i="16289" txt_f="16338">Epidermal growth factor receptor exon 13 R497K PM</offsets></title><p><offsets xml_i="28349" xml_f="28600" txt_i="16339" txt_f="16590">In 194 patient samples, the analysis of the EGFR R497K PM was technically successful: 112 (58%) showed the GG wild-type sequence, whereas 82 had either the GA or AA SNP. Also, for the exon 13 PM, no correlation with the EGFR expression by either IHC (</offsets><italic><offsets xml_i="28608" xml_f="28609" txt_i="16590" txt_f="16591">P</offsets></italic><offsets xml_i="28618" xml_f="28634" txt_i="16591" txt_f="16607">=0.80) or FISH (</offsets><italic><offsets xml_i="28642" xml_f="28643" txt_i="16607" txt_f="16608">P</offsets></italic><offsets xml_i="28652" xml_f="28826" txt_i="16608" txt_f="16782">=0.39) was apparent (data not shown). Patients with a GG wild-type sequence were more likely to have locally advanced disease compared with patients with a GA or AA SNP (21% </offsets><italic><offsets xml_i="28834" xml_f="28836" txt_i="16782" txt_f="16784">vs</offsets></italic><offsets xml_i="28845" xml_f="28850" txt_i="16784" txt_f="16789"> 9%, </offsets><italic><offsets xml_i="28858" xml_f="28859" txt_i="16789" txt_f="16790">P</offsets></italic><offsets xml_i="28868" xml_f="28876" txt_i="16790" txt_f="16798">=0.017; </offsets><xref rid="tbl2" ref-type="table"><offsets xml_i="28910" xml_f="28917" txt_i="16798" txt_f="16805">Table 2</offsets></xref><offsets xml_i="28924" xml_f="28926" txt_i="16805" txt_f="16807">).</offsets></p></sec></sec><sec><title><offsets xml_i="28954" xml_f="29027" txt_i="16810" txt_f="16883">Correlation of molecular EGFR pathway biomarkers with efficacy end points</offsets></title><p><xref rid="tbl3" ref-type="table"><offsets xml_i="29072" xml_f="29079" txt_i="16884" txt_f="16891">Table 3</offsets></xref><offsets xml_i="29086" xml_f="29369" txt_i="16891" txt_f="17174"> summarises the correlation of biomarker results (as dichotomous variables) with the study end points time-to-treatment failure for first-line (TTF1) and first and second-line therapy (TTF2) as well as OS. A significant correlation with TTF1 and TTF2 was observed for PTEN (HR 0.61, </offsets><italic><offsets xml_i="29377" xml_f="29378" txt_i="17174" txt_f="17175">P</offsets></italic><offsets xml_i="29387" xml_f="29406" txt_i="17175" txt_f="17194">=0.02 and HR 0.66, </offsets><italic><offsets xml_i="29414" xml_f="29415" txt_i="17194" txt_f="17195">P</offsets></italic><offsets xml_i="29424" xml_f="29553" txt_i="17195" txt_f="17324">=0.04, respectively). The only biomarker that had a statistically significant impact on OS in this univariate analysis (HR 1.68, </offsets><italic><offsets xml_i="29561" xml_f="29562" txt_i="17324" txt_f="17325">P</offsets></italic><offsets xml_i="29571" xml_f="29643" txt_i="17325" txt_f="17397">=0.005) was KRAS; however, no significant difference for TTF1 (HR 1.23, </offsets><italic><offsets xml_i="29651" xml_f="29652" txt_i="17397" txt_f="17398">P</offsets></italic><offsets xml_i="29661" xml_f="29686" txt_i="17398" txt_f="17423">=0.22) or TTF2 (HR 1.34, </offsets><italic><offsets xml_i="29694" xml_f="29695" txt_i="17423" txt_f="17424">P</offsets></italic><offsets xml_i="29704" xml_f="29759" txt_i="17424" txt_f="17479">=0.09) was observed for patients with a KRAS wild-type </offsets><italic><offsets xml_i="29767" xml_f="29769" txt_i="17479" txt_f="17481">vs</offsets></italic><offsets xml_i="29778" xml_f="29816" txt_i="17481" txt_f="17519"> a KRAS-mutated status, respectively (</offsets><xref rid="tbl3" ref-type="table"><offsets xml_i="29850" xml_f="29857" txt_i="17519" txt_f="17526">Table 3</offsets></xref><offsets xml_i="29864" xml_f="29917" txt_i="17526" txt_f="17579">). The Kaplan–Meier plot for KRAS and OS is shown in </offsets><xref ref-type="fig" rid="fig1"><offsets xml_i="29949" xml_f="29957" txt_i="17579" txt_f="17587">Figure 1</offsets></xref><offsets xml_i="29964" xml_f="30185" txt_i="17587" txt_f="17808">. The four biomarkers that were analysable as continuous variables were correlated with TTF1, TTF2 and OS with regard to each of their expression levels (if applicable, the lowest level/score thereby was set as HR=1; see </offsets><xref rid="tbl4" ref-type="table"><offsets xml_i="30219" xml_f="30226" txt_i="17808" txt_f="17815">Table 4</offsets></xref><offsets xml_i="30233" xml_f="30668" txt_i="17815" txt_f="18250">). None of the investigated markers reached the level of statistical significance in these analyses; however a nonsignificant—but at least consistent—trend for a prolonged TTF1, TTF2 and OS was evident for increasing levels of the PTEN score. Of note, none of the six investigated markers showed a significant correlation with objective response (to first-line treatment) as determined by RECIST, version 1.0 (detailed data not shown).</offsets></p></sec><sec><title><offsets xml_i="30690" xml_f="30753" txt_i="18252" txt_f="18315">Correlation of molecular EGFR pathway biomarkers with skin rash</offsets></title><p><offsets xml_i="30764" xml_f="30984" txt_i="18316" txt_f="18536">None of the six molecular biomarkers correlated with the occurrence of skin rash (any grade I–IV): the corresponding odds ratios for each marker (dichotomous and/or continuous variable, as appropriate) are summarised in </offsets><xref rid="tbl5" ref-type="table"><offsets xml_i="31018" xml_f="31025" txt_i="18536" txt_f="18543">Table 5</offsets></xref><offsets xml_i="31032" xml_f="31033" txt_i="18543" txt_f="18544">.</offsets></p></sec></sec><sec sec-type="discussion"><title><offsets xml_i="31083" xml_f="31093" txt_i="18547" txt_f="18557">Discussion</offsets></title><p><offsets xml_i="31104" xml_f="31551" txt_i="18558" txt_f="19005">AIO-PK0104 is one of the first phase 3 trials in advanced PC that was accompanied by a large translational research project. To date, only limited data on EGFR pathway biomarkers are available for PC, specifically in patients being treated with anti-EGFR agents such as erlotinib or cetuximab. In contrast to the biomarker results of the PA.3 study, we found a higher rate of KRAS wild-type patients within our translational study population (30% </offsets><italic><offsets xml_i="31559" xml_f="31561" txt_i="19005" txt_f="19007">vs</offsets></italic><offsets xml_i="31570" xml_f="31692" txt_i="19007" txt_f="19129"> 21%), and a KRAS wild-type status was the only analysed marker associated with an improved OS in our patient population (</offsets><xref ref-type="bibr" rid="bib6"><offsets xml_i="31725" xml_f="31741" txt_i="19129" txt_f="19145">Da Cunha Santos </offsets><italic><offsets xml_i="31749" xml_f="31754" txt_i="19145" txt_f="19150">et al</offsets></italic><offsets xml_i="31763" xml_f="31769" txt_i="19150" txt_f="19156">, 2010</offsets></xref><xref ref-type="supplementary-material" rid="sup1"><offsets xml_i="31827" xml_f="31849" txt_i="19156" txt_f="19178">Supplementary Table S2</offsets></xref><offsets xml_i="31856" xml_f="31970" txt_i="19178" txt_f="19292">) still remain conflicting to some extent, as there are studies that found no correlation at all of KRAS with OS (</offsets><xref ref-type="bibr" rid="bib6"><offsets xml_i="32003" xml_f="32019" txt_i="19292" txt_f="19308">Da Cunha Santos </offsets><italic><offsets xml_i="32027" xml_f="32032" txt_i="19308" txt_f="19313">et al</offsets></italic><offsets xml_i="32041" xml_f="32047" txt_i="19313" txt_f="19319">, 2010</offsets></xref><offsets xml_i="32054" xml_f="32143" txt_i="19319" txt_f="19408">), whereas others support the assumption that KRAS serves as prognostic biomarker in PC (</offsets><xref ref-type="bibr" rid="bib18"><offsets xml_i="32177" xml_f="32181" txt_i="19408" txt_f="19412">Lee </offsets><italic><offsets xml_i="32189" xml_f="32194" txt_i="19412" txt_f="19417">et al</offsets></italic><offsets xml_i="32203" xml_f="32209" txt_i="19417" txt_f="19423">, 2007</offsets></xref><offsets xml_i="32216" xml_f="32218" txt_i="19423" txt_f="19425">; </offsets><xref ref-type="bibr" rid="bib5"><offsets xml_i="32251" xml_f="32256" txt_i="19425" txt_f="19430">Chen </offsets><italic><offsets xml_i="32264" xml_f="32269" txt_i="19430" txt_f="19435">et al</offsets></italic><offsets xml_i="32278" xml_f="32284" txt_i="19435" txt_f="19441">, 2010</offsets></xref><offsets xml_i="32291" xml_f="32293" txt_i="19441" txt_f="19443">; </offsets><xref ref-type="bibr" rid="bib23"><offsets xml_i="32327" xml_f="32333" txt_i="19443" txt_f="19449">Ogura </offsets><italic><offsets xml_i="32341" xml_f="32346" txt_i="19449" txt_f="19454">et al</offsets></italic><offsets xml_i="32355" xml_f="32361" txt_i="19454" txt_f="19460">, 2012</offsets></xref><offsets xml_i="32368" xml_f="32643" txt_i="19460" txt_f="19735">). In contrast, recent data from a retrospective, non-randomised single-centre analysis suggest that KRAS may rather be a predictive marker for erlotinib efficacy than a prognostic factor; this information obviously needs to be verified by a well-designed prospective study (</offsets><xref ref-type="bibr" rid="bib16"><offsets xml_i="32677" xml_f="32681" txt_i="19735" txt_f="19739">Kim </offsets><italic><offsets xml_i="32689" xml_f="32694" txt_i="19739" txt_f="19744">et al</offsets></italic><offsets xml_i="32703" xml_f="32709" txt_i="19744" txt_f="19750">, 2011</offsets></xref><offsets xml_i="32716" xml_f="32718" txt_i="19750" txt_f="19752">).</offsets></p><p><offsets xml_i="32725" xml_f="32861" txt_i="19753" txt_f="19889">On the basis of our data, no correlation with efficacy end points was found for EGFR–IHC, EGFR–FISH, EGFR intron 1 or exon 13 R497K PM (</offsets><xref rid="tbl3" ref-type="table"><offsets xml_i="32895" xml_f="32902" txt_i="19889" txt_f="19896">Table 3</offsets></xref><offsets xml_i="32909" xml_f="33146" txt_i="19896" txt_f="20133">). Regarding the expression of PTEN, a trend for a worse outcome with regard to TTF1, TTF2 and OS was apparent for patients with PTEN-deficient tumours (when analysing PTEN as a dichotomous or as a continuous variable, respectively; see </offsets><xref rid="tbl3" ref-type="table"><offsets xml_i="33180" xml_f="33188" txt_i="20133" txt_f="20141">Tables 3</offsets></xref><offsets xml_i="33195" xml_f="33200" txt_i="20141" txt_f="20146"> and </offsets><xref rid="tbl4" ref-type="table"><offsets xml_i="33234" xml_f="33235" txt_i="20146" txt_f="20147">4</offsets></xref><offsets xml_i="33242" xml_f="33360" txt_i="20147" txt_f="20265">). Thus, the loss of the tumour-suppressor PTEN may perhaps represent a prognostically unfavourable event also in PC (</offsets><xref ref-type="bibr" rid="bib19"><offsets xml_i="33394" xml_f="33403" txt_i="20265" txt_f="20274">Loupakis </offsets><italic><offsets xml_i="33411" xml_f="33416" txt_i="20274" txt_f="20279">et al</offsets></italic><offsets xml_i="33425" xml_f="33431" txt_i="20279" txt_f="20285">, 2009</offsets></xref><offsets xml_i="33438" xml_f="33440" txt_i="20285" txt_f="20287">; </offsets><xref ref-type="bibr" rid="bib9"><offsets xml_i="33473" xml_f="33478" txt_i="20287" txt_f="20292">Feng </offsets><italic><offsets xml_i="33486" xml_f="33491" txt_i="20292" txt_f="20297">et al</offsets></italic><offsets xml_i="33500" xml_f="33506" txt_i="20297" txt_f="20303">, 2011</offsets></xref><offsets xml_i="33513" xml_f="34110" txt_i="20303" txt_f="20900">). Therefore, one might conclude that the expression of the EGFR itself (determined either by IHC or FISH) does not serve as a relevant biomarker in advanced PC and that future translational research should focus on EGFR downstream signalling networks (such as the PI3K–AKT–PTEN and the RAS–MAPK–MEK–ERK cascade) as prognostic and/or predictive targets in patients treated with anti-EGFR agents. Furthermore, we were not able to confirm data from previous reports, suggesting an association of two EGFR PMs (intron 1 and exon 13 R497K) with efficacy outcome parameters or skin rash, respectively (</offsets><xref ref-type="bibr" rid="bib31"><offsets xml_i="34144" xml_f="34149" txt_i="20900" txt_f="20905">Wang </offsets><italic><offsets xml_i="34157" xml_f="34162" txt_i="20905" txt_f="20910">et al</offsets></italic><offsets xml_i="34171" xml_f="34177" txt_i="20910" txt_f="20916">, 2007</offsets></xref><offsets xml_i="34184" xml_f="34186" txt_i="20916" txt_f="20918">; </offsets><xref ref-type="bibr" rid="bib27"><offsets xml_i="34220" xml_f="34226" txt_i="20918" txt_f="20924">Tzeng </offsets><italic><offsets xml_i="34234" xml_f="34239" txt_i="20924" txt_f="20929">et al</offsets></italic><offsets xml_i="34248" xml_f="34255" txt_i="20929" txt_f="20936">, 2007a</offsets></xref><offsets xml_i="34262" xml_f="34264" txt_i="20936" txt_f="20938">; </offsets><xref ref-type="bibr" rid="bib14"><offsets xml_i="34298" xml_f="34304" txt_i="20938" txt_f="20944">Huang </offsets><italic><offsets xml_i="34312" xml_f="34317" txt_i="20944" txt_f="20949">et al</offsets></italic><offsets xml_i="34326" xml_f="34332" txt_i="20949" txt_f="20955">, 2009</offsets></xref><offsets xml_i="34339" xml_f="34341" txt_i="20955" txt_f="20957">; </offsets><xref ref-type="bibr" rid="bib25"><offsets xml_i="34375" xml_f="34382" txt_i="20957" txt_f="20964">Sasaki </offsets><italic><offsets xml_i="34390" xml_f="34395" txt_i="20964" txt_f="20969">et al</offsets></italic><offsets xml_i="34404" xml_f="34410" txt_i="20969" txt_f="20975">, 2009</offsets></xref><offsets xml_i="34417" xml_f="34666" txt_i="20975" txt_f="21224">). Of note, most of the existing evidence on the role of these two EGFR PMs was derived from diseases other than PC, and the exciting hypothesis-generating pre-clinical data in PC could—at least in part—not be transferred into the clinical setting (</offsets><xref ref-type="bibr" rid="bib27"><offsets xml_i="34700" xml_f="34706" txt_i="21224" txt_f="21230">Tzeng </offsets><italic><offsets xml_i="34714" xml_f="34719" txt_i="21230" txt_f="21235">et al</offsets></italic><offsets xml_i="34728" xml_f="34735" txt_i="21235" txt_f="21242">, 2007a</offsets></xref><offsets xml_i="34742" xml_f="34744" txt_i="21242" txt_f="21244">; </offsets><xref ref-type="bibr" rid="bib10"><offsets xml_i="34778" xml_f="34785" txt_i="21244" txt_f="21251">Frolov </offsets><italic><offsets xml_i="34793" xml_f="34798" txt_i="21251" txt_f="21256">et al</offsets></italic><offsets xml_i="34807" xml_f="34813" txt_i="21256" txt_f="21262">, 2010</offsets></xref><offsets xml_i="34820" xml_f="34822" txt_i="21262" txt_f="21264">).</offsets></p><p><offsets xml_i="34829" xml_f="35703" txt_i="21265" txt_f="22139">One main limitation for translational research in PC arises from the fact that histological tumour tissue often is insufficient for detailed molecular analyses. Furthermore, different methods often require a sufficient amount of adequate tissue/tumour cells (e.g. FISH analyses), and thus it remains a challenge to obtain a full analysis data set with complete results for all tissue samples submitted to molecular analysis. We were able to obtain archival FFPE tissue from 208 of the 281 randomised patients; however, the overall sample size of our ‘complete biomarker measurements set' was 138 patients only. These aspects on tissue quality and technical performance thus also might significantly affect biostatistical results; this potential bias also was the main reason why the authors decided not to perform multivariate analyses within this explorative investigation.</offsets></p><p><offsets xml_i="35710" xml_f="35845" txt_i="22140" txt_f="22275">Skin rash as a frequent side-effect developing during erlotinib treatment is known to correlate with a longer survival in advanced PC (</offsets><xref ref-type="bibr" rid="bib20"><offsets xml_i="35879" xml_f="35885" txt_i="22275" txt_f="22281">Moore </offsets><italic><offsets xml_i="35893" xml_f="35898" txt_i="22281" txt_f="22286">et al</offsets></italic><offsets xml_i="35907" xml_f="35913" txt_i="22286" txt_f="22292">, 2007</offsets></xref><offsets xml_i="35920" xml_f="36197" txt_i="22292" txt_f="22569">). Thus, a molecular marker that could serve as a (pre-treatment) predictor for rash would be a valuable tool for treatment selection. However, none of the six EGFR pathway biomarker analyses within this translational substudy showed a correlation with the occurrence of rash (</offsets><xref rid="tbl5" ref-type="table"><offsets xml_i="36231" xml_f="36238" txt_i="22569" txt_f="22576">Table 5</offsets></xref><offsets xml_i="36245" xml_f="36247" txt_i="22576" txt_f="22578">).</offsets></p><p><offsets xml_i="36254" xml_f="36514" txt_i="22579" txt_f="22839">The question arises how to move forward in PC translational research: most of the data that increased our understanding of molecular resistance and molecular predictors for efficacy of anti-EGFR agents were derived from well-conducted trials in CRC and NSCLC (</offsets><xref ref-type="bibr" rid="bib7"><offsets xml_i="36547" xml_f="36556" txt_i="22839" txt_f="22848">De Roock </offsets><italic><offsets xml_i="36564" xml_f="36569" txt_i="22848" txt_f="22853">et al</offsets></italic><offsets xml_i="36578" xml_f="36584" txt_i="22853" txt_f="22859">, 2010</offsets></xref><offsets xml_i="36591" xml_f="36593" txt_i="22859" txt_f="22861">; </offsets><xref ref-type="bibr" rid="bib32"><offsets xml_i="36627" xml_f="36635" txt_i="22861" txt_f="22869">Wheeler </offsets><italic><offsets xml_i="36643" xml_f="36648" txt_i="22869" txt_f="22874">et al</offsets></italic><offsets xml_i="36657" xml_f="36663" txt_i="22874" txt_f="22880">, 2010</offsets></xref><offsets xml_i="36670" xml_f="37124" txt_i="22880" txt_f="23334">). Nevertheless, a translation of these results into PC will not be appropriate, mainly because of the fact that PC represents a genetically complex malignancy, which is based on multifactorial aberrations. Besides the EGFR pathway, molecular predictors for the efficacy of cytotoxic agents such as gemcitabine (e.g. hENT1) seem promising and a new avenue of research also identified peritumoral stroma (e.g. SPARC) as a relevant target in PC treatment (</offsets><xref ref-type="bibr" rid="bib8"><offsets xml_i="37157" xml_f="37165" txt_i="23334" txt_f="23342">Farrell </offsets><italic><offsets xml_i="37173" xml_f="37178" txt_i="23342" txt_f="23347">et al</offsets></italic><offsets xml_i="37187" xml_f="37193" txt_i="23347" txt_f="23353">, 2009</offsets></xref><offsets xml_i="37200" xml_f="37202" txt_i="23353" txt_f="23355">; </offsets><xref ref-type="bibr" rid="bib30"><offsets xml_i="37236" xml_f="37245" txt_i="23355" txt_f="23364">Von Hoff </offsets><italic><offsets xml_i="37253" xml_f="37258" txt_i="23364" txt_f="23369">et al</offsets></italic><offsets xml_i="37267" xml_f="37273" txt_i="23369" txt_f="23375">, 2011</offsets></xref><offsets xml_i="37280" xml_f="37622" txt_i="23375" txt_f="23717">). Most importantly, translational research projects should be included in all well-designed ongoing and future PC trials: only with a prospective collection of adequate tissue, the use of standardised and comparable molecular methods and innovative biostatistical models prognostic and/or predictive biomarkers may also be established in PC.</offsets></p><p><offsets xml_i="37629" xml_f="38101" txt_i="23718" txt_f="24190">In conclusion, the explorative translational substudy on AIO-PK0104 demonstrated that translational research with FFPE tumour tissue is feasible also in PC multicenter phase 3 trials and that KRAS and PTEN may have a role as biomarkers associated with treatment outcome in erlotinib-treated patients with advanced PC. Validation of these results within other PC randomised trials (investigating anti-EGFR and other novel targeted and cytotoxic agents) is urgently awaited.</offsets></p></sec></body><back><ack><p>We thank all patients and their families, nurses, study coordinators and investigators for their participation in the AIO-PK0104 study. In addition, the active commitment of the pathologists providing FFPE archival tumour tissue is gratefully acknowledged because they enabled this translational study. This study was funded by Roche Pharma AG, Germany and Sturm-Stiftung, City of Munich, Germany.</p></ack><fn-group><fn><p><xref ref-type="supplementary-material" rid="sup1">Supplementary Information</xref> accompanies the paper on British Journal of Cancer website (http://www.nature.com/bjc)</p></fn><fn><p>This work is published under the standard license to publish agreement. After 12 months the work will become freely available and the license terms will switch to a Creative Commons Attribution-NonCommercial-Share Alike 3.0 Unported License.</p></fn><fn><p>Previous presentation: 35th ESMO Congress, 8–12 October 2010, Milan, Italy and 47th ASCO Annual Meeting, 3–7 June 2011, Chicago, IL.</p></fn></fn-group><notes><p>Stefan Boeck and Michael R Clemens received Honoraria for scientific presentations, Research funding and Travel grants from Roche. Volker Heinemann was a consultant in Roche, and received Honoraria for scientific presentations and Research funding from Roche. The remaining authors declare no conflict of interest.</p></notes><ref-list><ref id="bib1"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Amador</surname><given-names>ML</given-names></name><name><surname>Oppenheimer</surname><given-names>D</given-names></name><name><surname>Perea</surname><given-names>S</given-names></name><name><surname>Maitra</surname><given-names>A</given-names></name><name><surname>Cusatis</surname><given-names>G</given-names></name><name><surname>Iacobuzio-Donahue</surname><given-names>C</given-names></name><name><surname>Baker</surname><given-names>SD</given-names></name><name><surname>Ashfaq</surname><given-names>R</given-names></name><name><surname>Takimoto</surname><given-names>C</given-names></name><name><surname>Forastiere</surname><given-names>A</given-names></name><name><surname>Hidalgo</surname><given-names>M</given-names></name></person-group><year>2004</year><article-title>An epidermal growth factor receptor intron 1 polymorphism mediates response to epidermal growth factor receptor inhibitors</article-title><source>Cancer Res</source><volume>64</volume><supplement>(24</supplement><fpage>9139</fpage><lpage>9143</lpage><pub-id pub-id-type="pmid">15604284</pub-id></mixed-citation></ref><ref id="bib2"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Boeck</surname><given-names>S</given-names></name><name><surname>Hinke</surname><given-names>A</given-names></name><name><surname>Wilkowski</surname><given-names>R</given-names></name><name><surname>Heinemann</surname><given-names>V</given-names></name></person-group><year>2007</year><article-title>Importance of performance status for treatment outcome in advanced pancreatic cancer</article-title><source>World J Gastroenterol</source><volume>13</volume><supplement>(2</supplement><fpage>224</fpage><lpage>227</lpage><pub-id pub-id-type="pmid">17226900</pub-id></mixed-citation></ref><ref id="bib3"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Boeck</surname><given-names>S</given-names></name><name><surname>Haas</surname><given-names>M</given-names></name><name><surname>Laubender</surname><given-names>RP</given-names></name><name><surname>Kullmann</surname><given-names>F</given-names></name><name><surname>Klose</surname><given-names>C</given-names></name><name><surname>Bruns</surname><given-names>CJ</given-names></name><name><surname>Wilkowski</surname><given-names>R</given-names></name><name><surname>Stieber</surname><given-names>P</given-names></name><name><surname>Holdenrieder</surname><given-names>S</given-names></name><name><surname>Buchner</surname><given-names>H</given-names></name><name><surname>Mansmann</surname><given-names>U</given-names></name><name><surname>Heinemann</surname><given-names>V</given-names></name></person-group><year>2010</year><article-title>Application of a time-varying covariate model to the analysis of CA 19-9 as serum biomarker in patients with advanced pancreatic cancer</article-title><source>Clin Cancer Res</source><volume>16</volume><supplement>(3</supplement><fpage>986</fpage><lpage>994</lpage><pub-id pub-id-type="pmid">20103662</pub-id></mixed-citation></ref><ref id="bib4"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cappuzzo</surname><given-names>F</given-names></name><name><surname>Hirsch</surname><given-names>FR</given-names></name><name><surname>Rossi</surname><given-names>E</given-names></name><name><surname>Bartolini</surname><given-names>S</given-names></name><name><surname>Ceresoli</surname><given-names>GL</given-names></name><name><surname>Bemis</surname><given-names>L</given-names></name><name><surname>Haney</surname><given-names>J</given-names></name><name><surname>Witta</surname><given-names>S</given-names></name><name><surname>Danenberg</surname><given-names>K</given-names></name><name><surname>Domenichini</surname><given-names>I</given-names></name><name><surname>Ludovini</surname><given-names>V</given-names></name><name><surname>Magrini</surname><given-names>E</given-names></name><name><surname>Gregorc</surname><given-names>V</given-names></name><name><surname>Doglioni</surname><given-names>C</given-names></name><name><surname>Sidoni</surname><given-names>A</given-names></name><name><surname>Tonato</surname><given-names>M</given-names></name><name><surname>Franklin</surname><given-names>WA</given-names></name><name><surname>Crino</surname><given-names>L</given-names></name><name><surname>Bunn</surname><given-names>PA</given-names></name><name><surname>Varella-Garcia</surname><given-names>M</given-names></name></person-group><year>2005</year><article-title>Epidermal growth factor receptor gene and protein and gefitinib sensitivity in non-small-cell lung cancer</article-title><source>J Natl Cancer Inst</source><volume>97</volume><supplement>(9</supplement><fpage>643</fpage><lpage>655</lpage><pub-id pub-id-type="pmid">15870435</pub-id></mixed-citation></ref><ref id="bib5"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Tu</surname><given-names>H</given-names></name><name><surname>Meng</surname><given-names>ZQ</given-names></name><name><surname>Chen</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>P</given-names></name><name><surname>Liu</surname><given-names>LM</given-names></name></person-group><year>2010</year><article-title>K-ras mutational status predicts poor prognosis in unresectable pancreatic cancer</article-title><source>Eur J Surg Oncol</source><volume>36</volume><supplement>(7</supplement><fpage>657</fpage><lpage>662</lpage><pub-id pub-id-type="pmid">20542658</pub-id></mixed-citation></ref><ref id="bib6"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Da Cunha Santos</surname><given-names>G</given-names></name><name><surname>Dhani</surname><given-names>N</given-names></name><name><surname>Tu</surname><given-names>D</given-names></name><name><surname>Chin</surname><given-names>K</given-names></name><name><surname>Ludkovski</surname><given-names>O</given-names></name><name><surname>Kamel-Reid</surname><given-names>S</given-names></name><name><surname>Squire</surname><given-names>J</given-names></name><name><surname>Parulekar</surname><given-names>W</given-names></name><name><surname>Moore</surname><given-names>MJ</given-names></name><name><surname>Tsao</surname><given-names>MS</given-names></name></person-group><year>2010</year><article-title>Molecular predictors of outcome in a phase 3 study of gemcitabine and erlotinib therapy in patients with advanced pancreatic cancer: National Cancer Institute of Canada Clinical Trials Group Study PA.3</article-title><source>Cancer</source><volume>116</volume><supplement>(24</supplement><fpage>5599</fpage><lpage>5607</lpage><pub-id pub-id-type="pmid">20824720</pub-id></mixed-citation></ref><ref id="bib7"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>De Roock</surname><given-names>W</given-names></name><name><surname>Jonker</surname><given-names>DJ</given-names></name><name><surname>Di Nicolantonio</surname><given-names>F</given-names></name><name><surname>Sartore-Bianchi</surname><given-names>A</given-names></name><name><surname>Tu</surname><given-names>D</given-names></name><name><surname>Siena</surname><given-names>S</given-names></name><name><surname>Lamba</surname><given-names>S</given-names></name><name><surname>Arena</surname><given-names>S</given-names></name><name><surname>Frattini</surname><given-names>M</given-names></name><name><surname>Piessevaux</surname><given-names>H</given-names></name><name><surname>Van Cutsem</surname><given-names>E</given-names></name><name><surname>O'Callaghan</surname><given-names>CJ</given-names></name><name><surname>Khambata-Ford</surname><given-names>S</given-names></name><name><surname>Zalcberg</surname><given-names>JR</given-names></name><name><surname>Simes</surname><given-names>J</given-names></name><name><surname>Karapetis</surname><given-names>CS</given-names></name><name><surname>Bardelli</surname><given-names>A</given-names></name><name><surname>Tejpar</surname><given-names>S</given-names></name></person-group><year>2010</year><article-title>Association of KRAS p.G13D mutation with outcome in patients with chemotherapy-refractory metastatic colorectal cancer treated with cetuximab</article-title><source>JAMA</source><volume>304</volume><supplement>(16</supplement><fpage>1812</fpage><lpage>1820</lpage><pub-id pub-id-type="pmid">20978259</pub-id></mixed-citation></ref><ref id="bib8"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Farrell</surname><given-names>JJ</given-names></name><name><surname>Elsaleh</surname><given-names>H</given-names></name><name><surname>Garcia</surname><given-names>M</given-names></name><name><surname>Lai</surname><given-names>R</given-names></name><name><surname>Ammar</surname><given-names>A</given-names></name><name><surname>Regine</surname><given-names>WF</given-names></name><name><surname>Abrams</surname><given-names>R</given-names></name><name><surname>Benson</surname><given-names>AB</given-names></name><name><surname>Macdonald</surname><given-names>J</given-names></name><name><surname>Cass</surname><given-names>CE</given-names></name><name><surname>Dicker</surname><given-names>AP</given-names></name><name><surname>Mackey</surname><given-names>JR</given-names></name></person-group><year>2009</year><article-title>Human equilibrative nucleoside transporter 1 levels predict response to gemcitabine in patients with pancreatic cancer</article-title><source>Gastroenterology</source><volume>136</volume><supplement>(1</supplement><fpage>187</fpage><lpage>195</lpage><pub-id pub-id-type="pmid">18992248</pub-id></mixed-citation></ref><ref id="bib9"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Feng</surname><given-names>C</given-names></name><name><surname>Yao</surname><given-names>R</given-names></name><name><surname>Huang</surname><given-names>F</given-names></name><name><surname>Liu</surname><given-names>X</given-names></name><name><surname>Nie</surname><given-names>W</given-names></name></person-group><year>2011</year><article-title>High level of PTEN expression and its association with low-grade liver metastases and satisfactory patient survival in pancreatic cancer</article-title><source>Arch Med Res</source><volume>42</volume><supplement>(7</supplement><fpage>584</fpage><lpage>588</lpage><pub-id pub-id-type="pmid">22036936</pub-id></mixed-citation></ref><ref id="bib10"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Frolov</surname><given-names>A</given-names></name><name><surname>Liles</surname><given-names>JS</given-names></name><name><surname>Kossenkov</surname><given-names>AV</given-names></name><name><surname>Tzeng</surname><given-names>CW</given-names></name><name><surname>Jhala</surname><given-names>N</given-names></name><name><surname>Kulesza</surname><given-names>P</given-names></name><name><surname>Varadarajulu</surname><given-names>S</given-names></name><name><surname>Eloubeidi</surname><given-names>M</given-names></name><name><surname>Heslin</surname><given-names>MJ</given-names></name><name><surname>Arnoletti</surname><given-names>JP</given-names></name></person-group><year>2010</year><article-title>Epidermal growth factor receptor (EGFR) intron 1 polymorphism and clinical outcome in pancreatic adenocarcinoma</article-title><source>Am J Surg</source><volume>200</volume><supplement>(3</supplement><fpage>398</fpage><lpage>405</lpage><pub-id pub-id-type="pmid">20409526</pub-id></mixed-citation></ref><ref id="bib11"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Heinemann</surname><given-names>V</given-names></name><name><surname>Stintzing</surname><given-names>S</given-names></name><name><surname>Kirchner</surname><given-names>T</given-names></name><name><surname>Boeck</surname><given-names>S</given-names></name><name><surname>Jung</surname><given-names>A</given-names></name></person-group><year>2009</year><article-title>Clinical relevance of EGFR- and KRAS-status in colorectal cancer patients treated with monoclonal antibodies directed against the EGFR</article-title><source>Cancer Treat Rev</source><volume>35</volume><supplement>(3</supplement><fpage>262</fpage><lpage>271</lpage><pub-id pub-id-type="pmid">19117687</pub-id></mixed-citation></ref><ref id="bib12"><mixed-citation publication-type="other"><person-group person-group-type="author"><name><surname>Heinemann</surname><given-names>V</given-names></name><name><surname>Vehling-Kaiser</surname><given-names>U</given-names></name><name><surname>Waldschmidt</surname><given-names>D</given-names></name><name><surname>Kettner</surname><given-names>E</given-names></name><name><surname>Märten</surname><given-names>A</given-names></name><name><surname>Winkelmann</surname><given-names>C</given-names></name><name><surname>Klein</surname><given-names>S</given-names></name><name><surname>Kojouharoff</surname><given-names>G</given-names></name><name><surname>Gauler</surname><given-names>TC</given-names></name><name><surname>Fischer von Weikersthal</surname><given-names>L</given-names></name><name><surname>Clemens</surname><given-names>MR</given-names></name><name><surname>Geissler</surname><given-names>M</given-names></name><name><surname>Greten</surname><given-names>TF</given-names></name><name><surname>Hegewisch-Becker</surname><given-names>S</given-names></name><name><surname>Rubanov</surname><given-names>O</given-names></name><name><surname>Baake</surname><given-names>G</given-names></name><name><surname>Höhler</surname><given-names>T</given-names></name><name><surname>Ko</surname><given-names>YD</given-names></name><name><surname>Jung</surname><given-names>A</given-names></name><name><surname>Neugebauer</surname><given-names>S</given-names></name><name><surname>Boeck</surname><given-names>S</given-names></name></person-group><year>2012</year><article-title>Gemcitabine plus erlotinib followed by capecitabine versus capecitabine plus erlotinib followed by gemcitabine in advanced pancreatic cancer: final results of a randomised phase 3 trial of the ‘Arbeitsgemeinschaft Internistische Onkologie' (AIO-PK0104)</article-title><source>Gut</source>e-pub ahead of print</mixed-citation></ref><ref id="bib13"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Heist</surname><given-names>RS</given-names></name><name><surname>Christiani</surname><given-names>D</given-names></name></person-group><year>2009</year><article-title>EGFR-targeted therapies in lung cancer: predictors of response and toxicity</article-title><source>Pharmacogenomics</source><volume>10</volume><supplement>(1</supplement><fpage>59</fpage><lpage>68</lpage><pub-id pub-id-type="pmid">19102716</pub-id></mixed-citation></ref><ref id="bib14"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Huang</surname><given-names>CL</given-names></name><name><surname>Yang</surname><given-names>CH</given-names></name><name><surname>Yeh</surname><given-names>KH</given-names></name><name><surname>Hu</surname><given-names>FC</given-names></name><name><surname>Chen</surname><given-names>KY</given-names></name><name><surname>Shih</surname><given-names>JY</given-names></name><name><surname>Lin</surname><given-names>ZZ</given-names></name><name><surname>Yu</surname><given-names>CJ</given-names></name><name><surname>Cheng</surname><given-names>AL</given-names></name><name><surname>Yang</surname><given-names>PC</given-names></name></person-group><year>2009</year><article-title>EGFR intron 1 dinucleotide repeat polymorphism is associated with the occurrence of skin rash with gefitinib treatment</article-title><source>Lung Cancer</source><volume>64</volume><supplement>(3</supplement><fpage>346</fpage><lpage>351</lpage><pub-id pub-id-type="pmid">18995924</pub-id></mixed-citation></ref><ref id="bib15"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Immervoll</surname><given-names>H</given-names></name><name><surname>Hoem</surname><given-names>D</given-names></name><name><surname>Kugarajh</surname><given-names>K</given-names></name><name><surname>Steine</surname><given-names>SJ</given-names></name><name><surname>Molven</surname><given-names>A</given-names></name></person-group><year>2006</year><article-title>Molecular analysis of the EGFR-RAS-RAF pathway in pancreatic ductal adenocarcinomas: lack of mutations in the BRAF and EGFR genes</article-title><source>Virchows Arch</source><volume>448</volume><supplement>(6</supplement><fpage>788</fpage><lpage>796</lpage><pub-id pub-id-type="pmid">16598499</pub-id></mixed-citation></ref><ref id="bib16"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kim</surname><given-names>ST</given-names></name><name><surname>Lim do</surname><given-names>H</given-names></name><name><surname>Jang</surname><given-names>KT</given-names></name><name><surname>Lim</surname><given-names>T</given-names></name><name><surname>Lee</surname><given-names>J</given-names></name><name><surname>Choi</surname><given-names>YL</given-names></name><name><surname>Jang</surname><given-names>HL</given-names></name><name><surname>Yi</surname><given-names>JH</given-names></name><name><surname>Baek</surname><given-names>KK</given-names></name><name><surname>Park</surname><given-names>SH</given-names></name><name><surname>Park</surname><given-names>YS</given-names></name><name><surname>Lim</surname><given-names>HY</given-names></name><name><surname>Kang</surname><given-names>WK</given-names></name><name><surname>Park</surname><given-names>JO</given-names></name></person-group><year>2011</year><article-title>Impact of KRAS mutations on clinical outcomes in pancreatic cancer patients treated with first-line gemcitabine-based chemotherapy</article-title><source>Mol Cancer Ther</source><volume>10</volume><supplement>(10</supplement><fpage>1993</fpage><lpage>1999</lpage><pub-id pub-id-type="pmid">21862683</pub-id></mixed-citation></ref><ref id="bib17"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Klinghammer</surname><given-names>K</given-names></name><name><surname>Knödler</surname><given-names>M</given-names></name><name><surname>Schmittel</surname><given-names>A</given-names></name><name><surname>Budach</surname><given-names>V</given-names></name><name><surname>Keilholz</surname><given-names>U</given-names></name><name><surname>Tinhofer</surname><given-names>I</given-names></name></person-group><year>2010</year><article-title>Association of epidermal growth factor receptor polymorphism, skin toxicity, and outcome in patients with squamous cell carcinoma of the head and neck receiving cetuximab-docetaxel treatment</article-title><source>Clin Cancer Res</source><volume>16</volume><supplement>(1</supplement><fpage>304</fpage><lpage>310</lpage><pub-id pub-id-type="pmid">20028750</pub-id></mixed-citation></ref><ref id="bib18"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lee</surname><given-names>J</given-names></name><name><surname>Jang</surname><given-names>KT</given-names></name><name><surname>Ki</surname><given-names>CS</given-names></name><name><surname>Lim</surname><given-names>T</given-names></name><name><surname>Park</surname><given-names>YS</given-names></name><name><surname>Lim</surname><given-names>HY</given-names></name><name><surname>Choi</surname><given-names>DW</given-names></name><name><surname>Kang</surname><given-names>WK</given-names></name><name><surname>Park</surname><given-names>K</given-names></name><name><surname>Park</surname><given-names>JO</given-names></name></person-group><year>2007</year><article-title>Impact of epidermal growth factor receptor (EGFR) kinase mutations, EGFR gene amplifications, and KRAS mutations on survival of pancreatic adenocarcinoma</article-title><source>Cancer</source><volume>109</volume><supplement>(8</supplement><fpage>1561</fpage><lpage>1569</lpage><pub-id pub-id-type="pmid">17354229</pub-id></mixed-citation></ref><ref id="bib19"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Loupakis</surname><given-names>F</given-names></name><name><surname>Pollina</surname><given-names>L</given-names></name><name><surname>Stasi</surname><given-names>I</given-names></name><name><surname>Ruzzo</surname><given-names>A</given-names></name><name><surname>Scartozzi</surname><given-names>M</given-names></name><name><surname>Santini</surname><given-names>D</given-names></name><name><surname>Masi</surname><given-names>G</given-names></name><name><surname>Graziano</surname><given-names>F</given-names></name><name><surname>Cremolini</surname><given-names>C</given-names></name><name><surname>Rulli</surname><given-names>E</given-names></name><name><surname>Canestrari</surname><given-names>E</given-names></name><name><surname>Funel</surname><given-names>N</given-names></name><name><surname>Schiavon</surname><given-names>G</given-names></name><name><surname>Petrini</surname><given-names>I</given-names></name><name><surname>Magnani</surname><given-names>M</given-names></name><name><surname>Tonini</surname><given-names>G</given-names></name><name><surname>Campani</surname><given-names>D</given-names></name><name><surname>Floriani</surname><given-names>I</given-names></name><name><surname>Cascinu</surname><given-names>S</given-names></name><name><surname>Falcone</surname><given-names>A</given-names></name></person-group><year>2009</year><article-title>PTEN expression and KRAS mutations on primary tumors and metastases in the prediction of benefit from cetuximab plus irinotecan for patients with metastatic colorectal cancer</article-title><source>J Clin Oncol</source><volume>27</volume><supplement>(16</supplement><fpage>2622</fpage><lpage>2629</lpage><pub-id pub-id-type="pmid">19398573</pub-id></mixed-citation></ref><ref id="bib20"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Moore</surname><given-names>MJ</given-names></name><name><surname>Goldstein</surname><given-names>D</given-names></name><name><surname>Hamm</surname><given-names>J</given-names></name><name><surname>Figer</surname><given-names>A</given-names></name><name><surname>Hecht</surname><given-names>JR</given-names></name><name><surname>Gallinger</surname><given-names>S</given-names></name><name><surname>Au</surname><given-names>HJ</given-names></name><name><surname>Murawa</surname><given-names>P</given-names></name><name><surname>Walde</surname><given-names>D</given-names></name><name><surname>Wolff</surname><given-names>RA</given-names></name><name><surname>Campos</surname><given-names>D</given-names></name><name><surname>Lim</surname><given-names>R</given-names></name><name><surname>Ding</surname><given-names>K</given-names></name><name><surname>Clark</surname><given-names>G</given-names></name><name><surname>Voskoglou-Nomikos</surname><given-names>T</given-names></name><name><surname>Ptasynski</surname><given-names>M</given-names></name><name><surname>Parulekar</surname><given-names>W</given-names></name></person-group><year>2007</year><article-title>Erlotinib plus gemcitabine compared with gemcitabine alone in patients with advanced pancreatic cancer. A phase III trial of the National Cancer Institute of Canada Clinical trials group</article-title><source>J Clin Oncol</source><volume>25</volume><supplement>(15</supplement><fpage>1960</fpage><lpage>1966</lpage><pub-id pub-id-type="pmid">17452677</pub-id></mixed-citation></ref><ref id="bib21"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Neumann</surname><given-names>J</given-names></name><name><surname>Zeindl-Eberhart</surname><given-names>E</given-names></name><name><surname>Kirchner</surname><given-names>T</given-names></name><name><surname>Jung</surname><given-names>A</given-names></name></person-group><year>2009</year><article-title>Frequency and type of KRAS mutations in routine diagnostic analysis of metastatic colorectal cancer</article-title><source>Pathol Res Pract</source><volume>205</volume><supplement>(12</supplement><fpage>858</fpage><lpage>862</lpage><pub-id pub-id-type="pmid">19679400</pub-id></mixed-citation></ref><ref id="bib22"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ng</surname><given-names>SS</given-names></name><name><surname>Tsao</surname><given-names>MS</given-names></name><name><surname>Nicklee</surname><given-names>T</given-names></name><name><surname>Hedley</surname><given-names>DW</given-names></name></person-group><year>2002</year><article-title>Effects of the epidermal growth factor receptor inhibitor OSI-774, Tarceva, on downstream signaling pathways and apoptosis in human pancreatic adenocarcinoma</article-title><source>Mol Cancer Ther</source><volume>1</volume><supplement>(10</supplement><fpage>777</fpage><lpage>783</lpage><pub-id pub-id-type="pmid">12492110</pub-id></mixed-citation></ref><ref id="bib23"><mixed-citation publication-type="other"><person-group person-group-type="author"><name><surname>Ogura</surname><given-names>T</given-names></name><name><surname>Yamao</surname><given-names>K</given-names></name><name><surname>Hara</surname><given-names>K</given-names></name><name><surname>Mizuno</surname><given-names>N</given-names></name><name><surname>Hijioka</surname><given-names>S</given-names></name><name><surname>Imaoka</surname><given-names>H</given-names></name><name><surname>Sawaki</surname><given-names>A</given-names></name><name><surname>Niwa</surname><given-names>Y</given-names></name><name><surname>Tajika</surname><given-names>M</given-names></name><name><surname>Kondo</surname><given-names>S</given-names></name><name><surname>Tanaka</surname><given-names>T</given-names></name><name><surname>Shimizu</surname><given-names>Y</given-names></name><name><surname>Bhatia</surname><given-names>V</given-names></name><name><surname>Higuchi</surname><given-names>K</given-names></name><name><surname>Hosoda</surname><given-names>W</given-names></name><name><surname>Yatabe</surname><given-names>Y</given-names></name></person-group><year>2012</year><article-title>Prognostic value of K-ras mutation status and subtypes in endoscopic ultrasound-guided fine-needle aspiration specimens from patients with unresectable pancreatic cancer</article-title><source>J Gastroenterol</source>e-pub ahead of print</mixed-citation></ref><ref id="bib24"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Pirker</surname><given-names>R</given-names></name><name><surname>Pereira</surname><given-names>JR</given-names></name><name><surname>von Pawel</surname><given-names>J</given-names></name><name><surname>Krzakowski</surname><given-names>M</given-names></name><name><surname>Ramlau</surname><given-names>R</given-names></name><name><surname>Park</surname><given-names>K</given-names></name><name><surname>de Marinis</surname><given-names>F</given-names></name><name><surname>Eberhardt</surname><given-names>WE</given-names></name><name><surname>Paz-Ares</surname><given-names>L</given-names></name><name><surname>Störkel</surname><given-names>S</given-names></name><name><surname>Schumacher</surname><given-names>KM</given-names></name><name><surname>von Heydebreck</surname><given-names>A</given-names></name><name><surname>Celik</surname><given-names>I</given-names></name><name><surname>O'Byrne</surname><given-names>KJ</given-names></name></person-group><year>2012</year><article-title>EGFR expression as a predictor of survival for first-line chemotherapy plus cetuximab in patients with advanced non-small-cell lung cancer: analysis of data from the phase 3 FLEX study</article-title><source>Lancet Oncol</source><volume>13</volume><supplement>(1</supplement><fpage>33</fpage><lpage>42</lpage><pub-id pub-id-type="pmid">22056021</pub-id></mixed-citation></ref><ref id="bib25"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Sasaki</surname><given-names>H</given-names></name><name><surname>Okuda</surname><given-names>K</given-names></name><name><surname>Shimizu</surname><given-names>S</given-names></name><name><surname>Takada</surname><given-names>M</given-names></name><name><surname>Kawahara</surname><given-names>M</given-names></name><name><surname>Kitahara</surname><given-names>N</given-names></name><name><surname>Okumura</surname><given-names>M</given-names></name><name><surname>Matsumura</surname><given-names>A</given-names></name><name><surname>Iuchi</surname><given-names>K</given-names></name><name><surname>Kawaguchi</surname><given-names>T</given-names></name><name><surname>Kubo</surname><given-names>A</given-names></name><name><surname>Kawano</surname><given-names>O</given-names></name><name><surname>Yukiue</surname><given-names>H</given-names></name><name><surname>Yano</surname><given-names>M</given-names></name><name><surname>Fujii</surname><given-names>Y</given-names></name></person-group><year>2009</year><article-title>EGFR R497K polymorphism is a favorable prognostic factor for advanced lung cancer</article-title><source>J Cancer Res Clin Oncol</source><volume>135</volume><supplement>(2</supplement><fpage>313</fpage><lpage>318</lpage><pub-id pub-id-type="pmid">18726117</pub-id></mixed-citation></ref><ref id="bib26"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stathis</surname><given-names>A</given-names></name><name><surname>Moore</surname><given-names>MJ</given-names></name></person-group><year>2010</year><article-title>Advanced pancreatic carcinoma: current treatment and future challenges</article-title><source>Nat Rev Clin Oncol</source><volume>7</volume><supplement>(3</supplement><fpage>163</fpage><lpage>172</lpage><pub-id pub-id-type="pmid">20101258</pub-id></mixed-citation></ref><ref id="bib27"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tzeng</surname><given-names>CW</given-names></name><name><surname>Frolov</surname><given-names>A</given-names></name><name><surname>Frolova</surname><given-names>N</given-names></name><name><surname>Jhala</surname><given-names>NC</given-names></name><name><surname>Howard</surname><given-names>JH</given-names></name><name><surname>Vickers</surname><given-names>SM</given-names></name><name><surname>Buchsbaum</surname><given-names>DJ</given-names></name><name><surname>Heslin</surname><given-names>MJ</given-names></name><name><surname>Arnoletti</surname><given-names>JP</given-names></name></person-group><year>2007a</year><article-title>Pancreatic cancer epidermal growth factor receptor (EGFR) intron 1 polymorphism influences postoperative patient survival and <italic>in vitro</italic> erlotinib response</article-title><source>Ann Surg Oncol</source><volume>14</volume><supplement>(7</supplement><fpage>2150</fpage><lpage>2158</lpage><pub-id pub-id-type="pmid">17453292</pub-id></mixed-citation></ref><ref id="bib28"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tzeng</surname><given-names>CW</given-names></name><name><surname>Frolov</surname><given-names>A</given-names></name><name><surname>Frolova</surname><given-names>N</given-names></name><name><surname>Jhala</surname><given-names>NC</given-names></name><name><surname>Howard</surname><given-names>JH</given-names></name><name><surname>Vickers</surname><given-names>SM</given-names></name><name><surname>Buchsbaum</surname><given-names>DJ</given-names></name><name><surname>Heslin</surname><given-names>MJ</given-names></name><name><surname>Arnoletti</surname><given-names>JP</given-names></name></person-group><year>2007b</year><article-title>EGFR genomic gain and aberrant pathway signaling in pancreatic cancer patients</article-title><source>J Surg Res</source><volume>143</volume><supplement>(1</supplement><fpage>20</fpage><lpage>26</lpage><pub-id pub-id-type="pmid">17950068</pub-id></mixed-citation></ref><ref id="bib29"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ueda</surname><given-names>S</given-names></name><name><surname>Ogata</surname><given-names>S</given-names></name><name><surname>Tsuda</surname><given-names>H</given-names></name><name><surname>Kawarabayashi</surname><given-names>N</given-names></name><name><surname>Kimura</surname><given-names>M</given-names></name><name><surname>Sugiura</surname><given-names>Y</given-names></name><name><surname>Tamai</surname><given-names>S</given-names></name><name><surname>Matsubara</surname><given-names>O</given-names></name><name><surname>Hatsuse</surname><given-names>K</given-names></name><name><surname>Mochizuki</surname><given-names>H</given-names></name></person-group><year>2004</year><article-title>The correlation between cytoplasmic overexpression of epidermal growth factor receptor and tumor aggressiveness: poor prognosis in patients with pancreatic ductal adenocarcinoma</article-title><source>Pancreas</source><volume>29</volume><supplement>(1</supplement><fpage>e1</fpage><lpage>e8</lpage><pub-id pub-id-type="pmid">15211117</pub-id></mixed-citation></ref><ref id="bib30"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Von Hoff</surname><given-names>DD</given-names></name><name><surname>Ramanathan</surname><given-names>RK</given-names></name><name><surname>Borad</surname><given-names>MJ</given-names></name><name><surname>Laheru</surname><given-names>DA</given-names></name><name><surname>Smith</surname><given-names>LS</given-names></name><name><surname>Wood</surname><given-names>TE</given-names></name><name><surname>Korn</surname><given-names>RL</given-names></name><name><surname>Desai</surname><given-names>N</given-names></name><name><surname>Trieu</surname><given-names>V</given-names></name><name><surname>Iglesias</surname><given-names>JL</given-names></name><name><surname>Zhang</surname><given-names>H</given-names></name><name><surname>Soon-Shiong</surname><given-names>P</given-names></name><name><surname>Shi</surname><given-names>T</given-names></name><name><surname>Rajeshkumar</surname><given-names>NV</given-names></name><name><surname>Maitra</surname><given-names>A</given-names></name><name><surname>Hidalgo</surname><given-names>M</given-names></name></person-group><year>2011</year><article-title>Gemcitabine plus nab-paclitaxel is an active regimen in patients with advanced pancreatic cancer: a phase I/II trial</article-title><source>J Clin Oncol</source><volume>29</volume><supplement>(34</supplement><fpage>4548</fpage><lpage>4554</lpage><pub-id pub-id-type="pmid">21969517</pub-id></mixed-citation></ref><ref id="bib31"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>WS</given-names></name><name><surname>Chen</surname><given-names>PM</given-names></name><name><surname>Chiou</surname><given-names>TJ</given-names></name><name><surname>Liu</surname><given-names>JH</given-names></name><name><surname>Lin</surname><given-names>JK</given-names></name><name><surname>Lin</surname><given-names>TC</given-names></name><name><surname>Wang</surname><given-names>HS</given-names></name><name><surname>Su</surname><given-names>Y</given-names></name></person-group><year>2007</year><article-title>Epidermal growth factor receptor R497K polymorphism is a favorable prognostic factor for patients with colorectal carcinoma</article-title><source>Clin Cancer Res</source><volume>13</volume><supplement>(12</supplement><fpage>3597</fpage><lpage>3604</lpage><pub-id pub-id-type="pmid">17575224</pub-id></mixed-citation></ref><ref id="bib32"><mixed-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wheeler</surname><given-names>DL</given-names></name><name><surname>Dunn</surname><given-names>EF</given-names></name><name><surname>Harari</surname><given-names>PM</given-names></name></person-group><year>2010</year><article-title>Understanding resistance to EGFR inhibitors-impact on future treatment strategies</article-title><source>Nat Rev Clin Oncol</source><volume>7</volume><supplement>(9</supplement><fpage>493</fpage><lpage>507</lpage><pub-id pub-id-type="pmid">20551942</pub-id></mixed-citation></ref></ref-list><sec sec-type="supplementary-material" id="sup1"><title>Supplementary Material</title><supplementary-material content-type="local-data" id="xob1"><label>Supplementary Tables</label><media mimetype="application" mime-subtype="msword" xlink:href="bjc2012495x1.doc"><caption><p>Click here for additional data file.</p></caption></media></supplementary-material></sec></back><floats-group><fig id="fig1"><label>Figure 1</label><caption><p>Correlation between KRAS mutation status and OS. (<italic>n</italic>=173, 157 events). Black line, KRAS wild-type, Grey line, KRAS mutation.</p></caption><graphic xlink:href="bjc2012495f1"></graphic></fig><table-wrap id="tbl1"><label>Table 1</label><caption><title>Baseline patient characteristics: Intention-to-treat population (<italic>n</italic>=274) and translational study population (<italic>n</italic>=206)</title></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="char" char=""></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="bottom"><tr><th align="char" valign="top" charoff="50"> </th><th colspan="4" align="center" valign="top" charoff="50"><bold>Intention-to-treat</bold><hr></hr></th><th colspan="4" align="center" valign="top" charoff="50"><bold>Translational</bold><hr></hr></th></tr><tr><th align="char" valign="top" charoff="50"> </th><th colspan="2" align="center" valign="top" charoff="50"><bold>Gem+E=&gt;Cap (</bold><italic><bold>n</bold></italic><bold>=143)</bold><hr></hr></th><th colspan="2" align="center" valign="top" charoff="50"><bold>Cap+E=&gt;Gem (</bold><italic><bold>n</bold></italic><bold>=131)</bold><hr></hr></th><th colspan="2" align="center" valign="top" charoff="50"><bold>Gem+E=&gt;Cap (</bold><italic><bold>n</bold></italic><bold>=113)</bold><hr></hr></th><th colspan="2" align="center" valign="top" charoff="50"><bold>Cap+E=&gt;Gem (</bold><italic><bold>n</bold></italic><bold>=93)</bold><hr></hr></th></tr><tr><th align="center" valign="top" charoff="50"><bold>Parameter</bold></th><th align="center" valign="top" charoff="50"><bold>No.</bold></th><th align="center" valign="top" charoff="50"><bold>%</bold></th><th align="center" valign="top" charoff="50"><bold>No.</bold></th><th align="center" valign="top" charoff="50"><bold>%</bold></th><th align="center" valign="top" charoff="50"><bold>No.</bold></th><th align="center" valign="top" charoff="50"><bold>%</bold></th><th align="center" valign="top" charoff="50"><bold>No.</bold></th><th align="center" valign="top" charoff="50"><bold>%</bold></th></tr></thead><tbody valign="top"><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>Age (years)</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Median</td><td align="center" valign="top" charoff="50">65</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">63</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">65</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">64</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="char" valign="top" charoff="50"> Range<hr></hr></td><td align="center" valign="top" charoff="50">32–78<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">38–75<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">32–75<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">42–75<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>Gender</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Male</td><td align="center" valign="top" charoff="50">82</td><td align="center" valign="top" charoff="50">57</td><td align="center" valign="top" charoff="50">83</td><td align="center" valign="top" charoff="50">63</td><td align="center" valign="top" charoff="50">64</td><td align="center" valign="top" charoff="50">57</td><td align="center" valign="top" charoff="50">58</td><td align="center" valign="top" charoff="50">62</td></tr><tr><td align="char" valign="top" charoff="50"> Female<hr></hr></td><td align="center" valign="top" charoff="50">61<hr></hr></td><td align="center" valign="top" charoff="50">43<hr></hr></td><td align="center" valign="top" charoff="50">48<hr></hr></td><td align="center" valign="top" charoff="50">37<hr></hr></td><td align="center" valign="top" charoff="50">49<hr></hr></td><td align="center" valign="top" charoff="50">43<hr></hr></td><td align="center" valign="top" charoff="50">35<hr></hr></td><td align="center" valign="top" charoff="50">38<hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>Stage of disease</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Locally advanced</td><td align="center" valign="top" charoff="50">21</td><td align="center" valign="top" charoff="50">15</td><td align="center" valign="top" charoff="50">22</td><td align="center" valign="top" charoff="50">17</td><td align="center" valign="top" charoff="50">17</td><td align="center" valign="top" charoff="50">15</td><td align="center" valign="top" charoff="50">16</td><td align="center" valign="top" charoff="50">17</td></tr><tr><td align="char" valign="top" charoff="50"> Metastatic<hr></hr></td><td align="center" valign="top" charoff="50">122<hr></hr></td><td align="center" valign="top" charoff="50">85<hr></hr></td><td align="center" valign="top" charoff="50">109<hr></hr></td><td align="center" valign="top" charoff="50">83<hr></hr></td><td align="center" valign="top" charoff="50">96<hr></hr></td><td align="center" valign="top" charoff="50">85<hr></hr></td><td align="center" valign="top" charoff="50">77<hr></hr></td><td align="center" valign="top" charoff="50">83<hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>Performance status</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> KPS 60–80%</td><td align="center" valign="top" charoff="50">50</td><td align="center" valign="top" charoff="50">35</td><td align="center" valign="top" charoff="50">49</td><td align="center" valign="top" charoff="50">33</td><td align="center" valign="top" charoff="50">37</td><td align="center" valign="top" charoff="50">33</td><td align="center" valign="top" charoff="50">38</td><td align="center" valign="top" charoff="50">41</td></tr><tr><td align="char" valign="top" charoff="50"> KPS 90–100%</td><td align="center" valign="top" charoff="50">85</td><td align="center" valign="top" charoff="50">59</td><td align="center" valign="top" charoff="50">79</td><td align="center" valign="top" charoff="50">60</td><td align="center" valign="top" charoff="50">70</td><td align="center" valign="top" charoff="50">62</td><td align="center" valign="top" charoff="50">55</td><td align="center" valign="top" charoff="50">59</td></tr><tr><td align="char" valign="top" charoff="50"> Missing</td><td align="center" valign="top" charoff="50">8</td><td align="center" valign="top" charoff="50">6</td><td align="center" valign="top" charoff="50">3</td><td align="center" valign="top" charoff="50">2</td><td align="center" valign="top" charoff="50">6</td><td align="center" valign="top" charoff="50">5</td><td align="center" valign="top" charoff="50">0</td><td align="center" valign="top" charoff="50">0</td></tr><tr><td align="char" valign="top" charoff="50">Previous surgery<hr></hr></td><td align="center" valign="top" charoff="50">8<hr></hr></td><td align="center" valign="top" charoff="50">6<hr></hr></td><td align="center" valign="top" charoff="50">17<hr></hr></td><td align="center" valign="top" charoff="50">13<hr></hr></td><td align="center" valign="top" charoff="50">7<hr></hr></td><td align="center" valign="top" charoff="50">6<hr></hr></td><td align="center" valign="top" charoff="50">13<hr></hr></td><td align="center" valign="top" charoff="50">14<hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>Weight loss during 3 months before randomisation (kg)</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Median</td><td align="center" valign="top" charoff="50">5</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">7</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">5</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">7</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="char" valign="top" charoff="50"> Range<hr></hr></td><td align="center" valign="top" charoff="50">0–47<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">0–45<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">0–47<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">0–45<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>Baseline CA 19–9 (U ml<sup>−1</sup>)<xref ref-type="fn" rid="t1-fn2">a</xref></bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Median</td><td align="center" valign="top" charoff="50">1999</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1756</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1999</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1565</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="char" valign="top" charoff="50"> Range<hr></hr></td><td align="center" valign="top" charoff="50">1–700 000<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">1–1 000 000<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">1–700 000<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50">1–1 000 000<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>TTF1 (months)</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Median</td><td align="center" valign="top" charoff="50">3.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">2.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">3.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">2.2</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="char" valign="top" charoff="50"> <italic>P</italic>-value<hr></hr></td><td align="center" valign="top" charoff="50"><italic>P</italic>=0.0034<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"><italic>P</italic>=0.0089<xref ref-type="fn" rid="t1-fn3">b</xref><hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>TTF2 (months)</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Median</td><td align="center" valign="top" charoff="50">4.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">4.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">4.1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">4.0</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="char" valign="top" charoff="50"> <italic>P</italic>-value<hr></hr></td><td align="center" valign="top" charoff="50"><italic>P</italic>=1.0<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"><italic>P</italic>=0.85<xref ref-type="fn" rid="t1-fn3">b</xref><hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td colspan="9" align="left" valign="top" charoff="50"><bold>OS (months)</bold><hr></hr></td></tr><tr><td align="char" valign="top" charoff="50"> Median</td><td align="center" valign="top" charoff="50">6.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">6.9</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">5.7</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">6.7</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="char" valign="top" charoff="50"> <italic>P</italic>-value</td><td align="center" valign="top" charoff="50"><italic>P</italic>=0.90</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"><italic>P</italic>=0.57<xref ref-type="fn" rid="t1-fn3">b</xref></td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr></tbody></table><table-wrap-foot><fn id="t1-fn1"><p>Abbreviations: Cap=capecitabine; E=erlotinib; Gem=gemcitabine; KPS=Karnofsky performance status; OS=overall survival; TTF1=time-to-treatment failure (first-line therapy); TTF2=time-to-treatment failure (first-line and second-line therapy).</p></fn><fn id="t1-fn2"><label>a</label><p><italic>n</italic>=245 out of 274.</p></fn><fn id="t1-fn3"><label>b</label><p>Exploratory analysis.</p></fn></table-wrap-foot></table-wrap><table-wrap id="tbl2"><label>Table 2</label><caption><title>Selected patient characteristics and molecular marker results (dichotomous variables) in the translational study population (<italic>n</italic>=206)</title></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col><col align="char" char="."></col></colgroup><thead valign="bottom"><tr><th align="left" valign="top" charoff="50"> </th><th colspan="3" align="center" valign="top" char="." charoff="50"><bold>KRAS (</bold><italic><bold>n</bold></italic><bold>=173)</bold><hr></hr></th><th colspan="3" align="center" valign="top" char="." charoff="50"><bold>EGFR–IHC (</bold><italic><bold>n</bold></italic><bold>=181)</bold><hr></hr></th><th colspan="3" align="center" valign="top" char="." charoff="50"><bold>EGFR–FISH (</bold><italic><bold>n</bold></italic><bold>=166)</bold><hr></hr></th><th colspan="3" align="center" valign="top" char="." charoff="50"><bold>PTEN–IHC (</bold><italic><bold>n</bold></italic><bold>=171)</bold><hr></hr></th><th colspan="3" align="center" valign="top" char="." charoff="50"><bold>EGFR intron 1 PM (</bold><italic><bold>n</bold></italic><bold>=186)</bold><hr></hr></th><th colspan="3" align="center" valign="top" char="." charoff="50"><bold>EGFR R497K PM (</bold><italic><bold>n</bold></italic><bold>=194)</bold><hr></hr></th></tr><tr><th align="left" valign="top" charoff="50"><bold>Parameter</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Wild-type,</bold>
<italic><bold>n</bold></italic><bold>=52 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Mutated,</bold>
<italic><bold>n</bold></italic><bold>=121 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>P</bold></italic><bold>-value</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Negative,</bold>
<italic><bold>n</bold></italic><bold>=92 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Positive,</bold>
<italic><bold>n</bold></italic><bold>=89 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>P</bold></italic><bold>-value</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Negative,</bold>
<italic><bold>n</bold></italic><bold>=89 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Positive,</bold>
<italic><bold>n</bold></italic><bold>=77 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>P</bold></italic><bold>-value</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Negative,</bold>
<italic><bold>n</bold></italic><bold>=30 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><bold>Positive,</bold>
<italic><bold>n</bold></italic><bold>=141 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>P</bold></italic><bold>-value</bold></th><th align="center" valign="top" char="." charoff="50"><bold>CA repeats&lt;36,</bold>
<italic><bold>n</bold></italic><bold>=107 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><bold>CA repeats⩾36,</bold>
<italic><bold>n</bold></italic><bold>=79 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>P</bold></italic><bold>-value</bold></th><th align="center" valign="top" char="." charoff="50"><bold>GG (wild-type),</bold>
<italic><bold>n</bold></italic><bold>=112 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><bold>GA/AA,</bold>
<italic><bold>n</bold></italic><bold>=82 (%)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>P</bold></italic><bold>-value</bold></th></tr></thead><tbody valign="top"><tr><td align="left" valign="top" charoff="50">Age (years), median<hr></hr></td><td align="char" valign="top" char="." charoff="50">65<hr></hr></td><td align="char" valign="top" char="." charoff="50">64<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.96<hr></hr></td><td align="char" valign="top" char="." charoff="50">65<hr></hr></td><td align="char" valign="top" char="." charoff="50">63<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.97<hr></hr></td><td align="char" valign="top" char="." charoff="50">65<hr></hr></td><td align="char" valign="top" char="." charoff="50">63<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.34<hr></hr></td><td align="char" valign="top" char="." charoff="50">66<hr></hr></td><td align="char" valign="top" char="." charoff="50">63<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.08<hr></hr></td><td align="char" valign="top" char="." charoff="50">66<hr></hr></td><td align="char" valign="top" char="." charoff="50">62<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.49<hr></hr></td><td align="char" valign="top" char="." charoff="50">63<hr></hr></td><td align="char" valign="top" char="." charoff="50">66<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.30<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"><italic>Gender</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.87</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.45</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>1.00</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.54</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.55</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.38</italic></td></tr><tr><td align="left" valign="top" charoff="50"> Male</td><td align="char" valign="top" char="." charoff="50">60</td><td align="char" valign="top" char="." charoff="50">61</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">62</td><td align="char" valign="top" char="." charoff="50">56</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">61</td><td align="char" valign="top" char="." charoff="50">60</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">53</td><td align="char" valign="top" char="." charoff="50">60</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">58</td><td align="char" valign="top" char="." charoff="50">63</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">56</td><td align="char" valign="top" char="." charoff="50">63</td><td align="char" valign="top" char="." charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Female<hr></hr></td><td align="char" valign="top" char="." charoff="50">40<hr></hr></td><td align="char" valign="top" char="." charoff="50">39<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">38<hr></hr></td><td align="char" valign="top" char="." charoff="50">44<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">39<hr></hr></td><td align="char" valign="top" char="." charoff="50">40<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">47<hr></hr></td><td align="char" valign="top" char="." charoff="50">40<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">42<hr></hr></td><td align="char" valign="top" char="." charoff="50">37<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">44<hr></hr></td><td align="char" valign="top" char="." charoff="50">37<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>Stage of disease</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.27</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.24</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.42</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.42</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.55</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.017</italic></td></tr><tr><td align="left" valign="top" charoff="50"> Locally advanced</td><td align="char" valign="top" char="." charoff="50">21</td><td align="char" valign="top" char="." charoff="50">14</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">21</td><td align="char" valign="top" char="." charoff="50">13</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">15</td><td align="char" valign="top" char="." charoff="50">19</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">10</td><td align="char" valign="top" char="." charoff="50">17</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">18</td><td align="char" valign="top" char="." charoff="50">14</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">21</td><td align="char" valign="top" char="." charoff="50">9</td><td align="char" valign="top" char="." charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Metastatic<hr></hr></td><td align="char" valign="top" char="." charoff="50">79<hr></hr></td><td align="char" valign="top" char="." charoff="50">86<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">79<hr></hr></td><td align="char" valign="top" char="." charoff="50">87<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">85<hr></hr></td><td align="char" valign="top" char="." charoff="50">81<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">90<hr></hr></td><td align="char" valign="top" char="." charoff="50">83<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">82<hr></hr></td><td align="char" valign="top" char="." charoff="50">86<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">79<hr></hr></td><td align="char" valign="top" char="." charoff="50">91<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>Performance status</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.39</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.54</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.19</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.09</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.64</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.29</italic></td></tr><tr><td align="left" valign="top" charoff="50"> KPS 60–80%</td><td align="char" valign="top" char="." charoff="50">31</td><td align="char" valign="top" char="." charoff="50">39</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">40</td><td align="char" valign="top" char="." charoff="50">35</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">42</td><td align="char" valign="top" char="." charoff="50">31</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">52</td><td align="char" valign="top" char="." charoff="50">34</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">38</td><td align="char" valign="top" char="." charoff="50">35</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">41</td><td align="char" valign="top" char="." charoff="50">33</td><td align="char" valign="top" char="." charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> KPS 90–100%<hr></hr></td><td align="char" valign="top" char="." charoff="50">69<hr></hr></td><td align="char" valign="top" char="." charoff="50">61<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">60<hr></hr></td><td align="char" valign="top" char="." charoff="50">65<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">58<hr></hr></td><td align="char" valign="top" char="." charoff="50">69<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">48<hr></hr></td><td align="char" valign="top" char="." charoff="50">66<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">62<hr></hr></td><td align="char" valign="top" char="." charoff="50">65<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">59<hr></hr></td><td align="char" valign="top" char="." charoff="50">67<hr></hr></td><td align="char" valign="top" char="." charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">Previous surgery<hr></hr></td><td align="char" valign="top" char="." charoff="50">15<hr></hr></td><td align="char" valign="top" char="." charoff="50">8<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.16<hr></hr></td><td align="char" valign="top" char="." charoff="50">14<hr></hr></td><td align="char" valign="top" char="." charoff="50">5<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.04<hr></hr></td><td align="char" valign="top" char="." charoff="50">9<hr></hr></td><td align="char" valign="top" char="." charoff="50">11<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.80<hr></hr></td><td align="char" valign="top" char="." charoff="50">13<hr></hr></td><td align="char" valign="top" char="." charoff="50">10<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.53<hr></hr></td><td align="char" valign="top" char="." charoff="50">14<hr></hr></td><td align="char" valign="top" char="." charoff="50">6<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.10<hr></hr></td><td align="char" valign="top" char="." charoff="50">12<hr></hr></td><td align="char" valign="top" char="." charoff="50">9<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.64<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">CA 19–9 (U ml<sup>−1</sup>), median<hr></hr></td><td align="char" valign="top" char="." charoff="50">1206<hr></hr></td><td align="char" valign="top" char="." charoff="50">2000<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.03<hr></hr></td><td align="char" valign="top" char="." charoff="50">1475<hr></hr></td><td align="char" valign="top" char="." charoff="50">1878<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.97<hr></hr></td><td align="char" valign="top" char="." charoff="50">1257<hr></hr></td><td align="char" valign="top" char="." charoff="50">1878<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.73<hr></hr></td><td align="char" valign="top" char="." charoff="50">2207<hr></hr></td><td align="char" valign="top" char="." charoff="50">1321<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.26<hr></hr></td><td align="char" valign="top" char="." charoff="50">1369<hr></hr></td><td align="char" valign="top" char="." charoff="50">3700<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.19<hr></hr></td><td align="char" valign="top" char="." charoff="50">1449<hr></hr></td><td align="char" valign="top" char="." charoff="50">1829<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.81<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>Treatment arm</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.51</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.007</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.76</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.026</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>0.24</italic></td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50"><italic>1.00</italic></td></tr><tr><td align="left" valign="top" charoff="50"> Gem+E⩾Cap</td><td align="char" valign="top" char="." charoff="50">58</td><td align="char" valign="top" char="." charoff="50">52</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">43</td><td align="char" valign="top" char="." charoff="50">36</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">53</td><td align="char" valign="top" char="." charoff="50">56</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">33</td><td align="char" valign="top" char="." charoff="50">57</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">50</td><td align="char" valign="top" char="." charoff="50">59</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">55</td><td align="char" valign="top" char="." charoff="50">55</td><td align="char" valign="top" char="." charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Cap+E⩾Gem</td><td align="char" valign="top" char="." charoff="50">42</td><td align="char" valign="top" char="." charoff="50">48</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">57</td><td align="char" valign="top" char="." charoff="50">64</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">47</td><td align="char" valign="top" char="." charoff="50">44</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">67</td><td align="char" valign="top" char="." charoff="50">43</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">50</td><td align="char" valign="top" char="." charoff="50">41</td><td align="char" valign="top" char="." charoff="50"> </td><td align="char" valign="top" char="." charoff="50">45</td><td align="char" valign="top" char="." charoff="50">45</td><td align="char" valign="top" char="." charoff="50"> </td></tr></tbody></table><table-wrap-foot><fn id="t2-fn1"><p>Abbreviations: Cap=capecitabine; E=erlotinib; EGFR=epidermal growth factor receptor; FISH=fluorescence <italic>in-situ</italic> hybridisation; Gem=gemcitabine; IHC=immunohistochemistry; KPS=Karnofsky performance status; KRAS=Kirsten rat sarcoma viral oncogene; PM=polymorphism; PTEN=Phosphatase and tensin homolog.</p></fn></table-wrap-foot></table-wrap><table-wrap id="tbl3"><label>Table 3</label><caption><title>Correlation of biomarker results (dichotomous variables) with efficacy parameters: TTF1, TTF2 and OS</title></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="left"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="bottom"><tr><th align="left" valign="top" charoff="50"> </th><th align="left" valign="top" charoff="50"> </th><th colspan="3" align="center" valign="top" charoff="50"><bold>Median TTF1</bold></th><th colspan="3" align="center" valign="top" charoff="50"><bold>Median TTF2</bold></th><th colspan="3" align="center" valign="top" charoff="50"><bold>Median OS</bold></th></tr></thead><tbody valign="top"><tr><td align="left" valign="top" charoff="50"><bold>Biomarker (alteration)</bold><hr></hr></td><td align="left" valign="top" charoff="50"><italic><bold>n</bold></italic><hr></hr></td><td align="center" valign="top" charoff="50"><bold>Mo.</bold><hr></hr></td><td align="center" valign="top" charoff="50"><bold>HR (95% CI)</bold><hr></hr></td><td align="center" valign="top" charoff="50"><italic><bold>P</bold></italic><hr></hr></td><td align="center" valign="top" charoff="50"><bold>Mo.</bold><hr></hr></td><td align="center" valign="top" charoff="50"><bold>HR (95% CI)</bold><hr></hr></td><td align="center" valign="top" charoff="50"><italic><bold>P</bold></italic><hr></hr></td><td align="center" valign="top" charoff="50"><bold>Mo.</bold><hr></hr></td><td align="center" valign="top" charoff="50"><bold>HR (95% CI)</bold><hr></hr></td><td align="center" valign="top" charoff="50"><italic><bold>P</bold></italic><hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>KRAS</italic></td><td align="char" valign="top" char="." charoff="50">173</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Wild type</td><td align="char" valign="top" char="." charoff="50">52</td><td align="char" valign="top" char="." charoff="50">2.9</td><td align="char" valign="top" char="(" charoff="50">1.23 (0.88–1.71)</td><td align="char" valign="top" char="." charoff="50">0.22</td><td align="char" valign="top" char="." charoff="50">4.2</td><td align="char" valign="top" char="(" charoff="50">1.34 (0.96–1.87)</td><td align="char" valign="top" char="." charoff="50">0.09</td><td align="char" valign="top" char="." charoff="50">7.9</td><td align="char" valign="top" char="(" charoff="50">1.68 (1.17–2.41)</td><td align="char" valign="top" char="." charoff="50"><bold>0.005</bold></td></tr><tr><td align="left" valign="top" charoff="50"> Mutation<hr></hr></td><td align="char" valign="top" char="." charoff="50">121<hr></hr></td><td align="char" valign="top" char="." charoff="50">2.2<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">4<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">5.7<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>EGFR–IHC</italic></td><td align="char" valign="top" char="." charoff="50">181</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Negative</td><td align="char" valign="top" char="." charoff="50">92</td><td align="char" valign="top" char="." charoff="50">2.2</td><td align="char" valign="top" char="(" charoff="50">0.76 (0.56–1.02)</td><td align="char" valign="top" char="." charoff="50">0.07</td><td align="char" valign="top" char="." charoff="50">3.9</td><td align="char" valign="top" char="(" charoff="50">0.82 (0.60–1.11)</td><td align="char" valign="top" char="." charoff="50">0.19</td><td align="char" valign="top" char="." charoff="50">6.7</td><td align="char" valign="top" char="(" charoff="50">0.96 (0.71–1.31)</td><td align="char" valign="top" char="." charoff="50">0.8</td></tr><tr><td align="left" valign="top" charoff="50"> Positive<hr></hr></td><td align="char" valign="top" char="." charoff="50">89<hr></hr></td><td align="char" valign="top" char="." charoff="50">3.1<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">4.1<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">6.9<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>EGFR–FISH</italic></td><td align="char" valign="top" char="." charoff="50">166</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Negative</td><td align="char" valign="top" char="." charoff="50">89</td><td align="char" valign="top" char="." charoff="50">2.5</td><td align="char" valign="top" char="(" charoff="50">0.90 (0.66–1.23)</td><td align="char" valign="top" char="." charoff="50">0.51</td><td align="char" valign="top" char="." charoff="50">4.1</td><td align="char" valign="top" char="(" charoff="50">1.03 (0.75–1.42)</td><td align="char" valign="top" char="." charoff="50">0.84</td><td align="char" valign="top" char="." charoff="50">6.9</td><td align="char" valign="top" char="(" charoff="50">1.22 (0.87–1.70)</td><td align="char" valign="top" char="." charoff="50">0.25</td></tr><tr><td align="left" valign="top" charoff="50"> Positive<hr></hr></td><td align="char" valign="top" char="." charoff="50">77<hr></hr></td><td align="char" valign="top" char="." charoff="50">2.8<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">4.1<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">6.7<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>PTEN–IHC</italic></td><td align="char" valign="top" char="." charoff="50">171</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Negative</td><td align="char" valign="top" char="." charoff="50">30</td><td align="char" valign="top" char="." charoff="50">2</td><td align="char" valign="top" char="(" charoff="50">0.61 (0.41–0.92)</td><td align="char" valign="top" char="." charoff="50"><bold>0.02</bold></td><td align="char" valign="top" char="." charoff="50">3</td><td align="char" valign="top" char="(" charoff="50">0.66 (0.44–0.98)</td><td align="char" valign="top" char="." charoff="50"><bold>0.04</bold></td><td align="char" valign="top" char="." charoff="50">4.5</td><td align="char" valign="top" char="(" charoff="50">0.77 (0.51–1.17)</td><td align="char" valign="top" char="." charoff="50">0.22</td></tr><tr><td align="left" valign="top" charoff="50"> Positive<hr></hr></td><td align="char" valign="top" char="." charoff="50">141<hr></hr></td><td align="char" valign="top" char="." charoff="50">2.4<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">4.1<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">6.8<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>EGFR intron 1 PM</italic></td><td align="char" valign="top" char="." charoff="50">186</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> CA repeats&lt;36</td><td align="char" valign="top" char="." charoff="50">107</td><td align="char" valign="top" char="." charoff="50">2.6</td><td align="char" valign="top" char="(" charoff="50">0.91 (0.67–1.22)</td><td align="char" valign="top" char="." charoff="50">0.52</td><td align="char" valign="top" char="." charoff="50">4.1</td><td align="char" valign="top" char="(" charoff="50">0.84 (0.62–1.14)</td><td align="char" valign="top" char="." charoff="50">0.25</td><td align="char" valign="top" char="." charoff="50">6.7</td><td align="char" valign="top" char="(" charoff="50">0.91 (0.67–1.24)</td><td align="char" valign="top" char="." charoff="50">0.55</td></tr><tr><td align="left" valign="top" charoff="50"> CA repeats⩾36<hr></hr></td><td align="char" valign="top" char="." charoff="50">79<hr></hr></td><td align="char" valign="top" char="." charoff="50">2.8<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">4.4<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="char" valign="top" char="." charoff="50">6.9<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic></italic></td><td align="char" valign="top" char="." charoff="50">194</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> GG</td><td align="char" valign="top" char="." charoff="50">112</td><td align="char" valign="top" char="." charoff="50">2.4</td><td align="char" valign="top" char="(" charoff="50">0.88 (0.65–1.18)</td><td align="char" valign="top" char="." charoff="50">0.38</td><td align="char" valign="top" char="." charoff="50">4.1</td><td align="char" valign="top" char="(" charoff="50">0.90 (0.67–1.21)</td><td align="char" valign="top" char="." charoff="50">0.48</td><td align="char" valign="top" char="." charoff="50">5.8</td><td align="char" valign="top" char="(" charoff="50">0.83 (0.61–1.13)</td><td align="char" valign="top" char="." charoff="50">0.23</td></tr><tr><td align="left" valign="top" charoff="50"> GA/AA</td><td align="char" valign="top" char="." charoff="50">82</td><td align="char" valign="top" char="." charoff="50">2.8</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="." charoff="50">4.2</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="." charoff="50">7.3</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr></tbody></table><table-wrap-foot><fn id="t3-fn1"><p>Abbreviations: CI=confidence interval; FISH=fluorescence <italic>in-situ</italic> hybridisation; HR=hazard ratio; IHC=immunohistochemistry; KRAS=Kirsten rat sarcoma viral oncogene; Mo.=months; OS=overall survival; PM=polymorphism; PTEN=Phosphatase and tensin homolog; TTF1=time-to-treatment failure 1; TTF2=time-to-treatment failure 2. Bold entries indicate statistically significant <italic>P</italic>-values (<italic>P</italic>&lt;0.05).</p></fn></table-wrap-foot></table-wrap><table-wrap id="tbl4"><label>Table 4</label><caption><title>Correlation of selected biomarker results (analysable as continuous variables) with efficacy parameters: TTF1, TTF2, and OS</title></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="char" char="."></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col><col align="center"></col></colgroup><thead valign="bottom"><tr><th align="left" valign="top" charoff="50"> </th><th align="center" valign="top" char="." charoff="50"> </th><th colspan="2" align="center" valign="top" charoff="50"><bold>TTF1</bold><hr></hr></th><th colspan="2" align="center" valign="top" charoff="50"><bold>TTF2</bold><hr></hr></th><th colspan="2" align="center" valign="top" charoff="50"><bold>OS</bold><hr></hr></th></tr><tr><th align="left" valign="top" charoff="50"><bold>Biomarker (alteration)</bold></th><th align="center" valign="top" char="." charoff="50"><italic><bold>n</bold></italic></th><th align="center" valign="top" charoff="50"><bold>HR (95% CI)</bold></th><th align="center" valign="top" charoff="50"><italic><bold>P</bold></italic></th><th align="center" valign="top" charoff="50"><bold>HR (95% CI)</bold></th><th align="center" valign="top" charoff="50"><italic><bold>P</bold></italic></th><th align="center" valign="top" charoff="50"><bold>HR (95% CI)</bold></th><th align="center" valign="top" charoff="50"><italic><bold>P</bold></italic></th></tr></thead><tbody valign="top"><tr><td align="left" valign="top" charoff="50"><italic>EGFR–IHC score</italic></td><td align="char" valign="top" char="." charoff="50"><italic>181</italic></td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 0</td><td align="char" valign="top" char="." charoff="50">7</td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 1</td><td align="char" valign="top" char="." charoff="50">85</td><td align="char" valign="top" char="(" charoff="50">0.91 (0.42–1.97)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.92 (0.43–2.00)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">1.03 (0.47–2.25)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 2</td><td align="char" valign="top" char="." charoff="50">48</td><td align="char" valign="top" char="(" charoff="50">0.63 (0.28–1.40)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.70 (0.31–1.55)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.84 (0.38–1.88)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 3<hr></hr></td><td align="char" valign="top" char="." charoff="50">41<hr></hr></td><td align="char" valign="top" char="(" charoff="50">0.79 (0.35–1.78)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.22<hr></hr></td><td align="char" valign="top" char="(" charoff="50">0.85 (0.38–1.92)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.48<hr></hr></td><td align="char" valign="top" char="(" charoff="50">1.21 (0.54–2.73)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.46<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>EGFR–FISH score</italic></td><td align="char" valign="top" char="." charoff="50"><italic>166</italic></td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Trisomy low</td><td align="char" valign="top" char="." charoff="50">4</td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Trisomy high</td><td align="char" valign="top" char="." charoff="50">23</td><td align="char" valign="top" char="(" charoff="50">0.77 (0.27–2.25)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">1.07 (0.37–3.16)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">2.36 (0.66–8.36)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Polysomy low</td><td align="char" valign="top" char="." charoff="50">62</td><td align="char" valign="top" char="(" charoff="50">0.81 (0.29–2.24)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">1.59 (0.57–4.42)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">3.01 (0.89–10.14)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Polysomy high</td><td align="char" valign="top" char="." charoff="50">74</td><td align="char" valign="top" char="(" charoff="50">0.72 (0.26–1.98)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">1.43 (0.52–3.97)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">3.28 (0.96–11.19)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> Amplification<hr></hr></td><td align="char" valign="top" char="." charoff="50">3<hr></hr></td><td align="char" valign="top" char="(" charoff="50">0.92 (0.17–5.07)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.94<hr></hr></td><td align="char" valign="top" char="(" charoff="50">1.43 (0.26–7.87)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.59<hr></hr></td><td align="char" valign="top" char="(" charoff="50">2.72 (0.27–27.26)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.28<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50"><italic>PTEN–IHC score</italic></td><td align="char" valign="top" char="." charoff="50"><italic>171</italic></td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 0</td><td align="char" valign="top" char="." charoff="50">16</td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">1</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 1</td><td align="char" valign="top" char="." charoff="50">0</td><td align="center" valign="top" charoff="50">—</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">—</td><td align="center" valign="top" charoff="50"> </td><td align="center" valign="top" charoff="50">—</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 2</td><td align="char" valign="top" char="." charoff="50">7</td><td align="char" valign="top" char="(" charoff="50">0.77 (0.32–1.89)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.69 (0.28–1.68)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.87 (0.34–2.24)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 3</td><td align="char" valign="top" char="." charoff="50">7</td><td align="char" valign="top" char="(" charoff="50">0.75 (0.31–1.84)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.61 (0.25–1.50)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.87 (0.34–2.23)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 4</td><td align="char" valign="top" char="." charoff="50">95</td><td align="char" valign="top" char="(" charoff="50">0.59 (0.34–1.01)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.59 (0.34–1.02)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.84 (0.49–1.43)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 5</td><td align="char" valign="top" char="." charoff="50">36</td><td align="char" valign="top" char="(" charoff="50">0.45 (0.24–0.83)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.43 (0.23–0.79)</td><td align="center" valign="top" charoff="50"> </td><td align="char" valign="top" char="(" charoff="50">0.60 (0.32–1.11)</td><td align="center" valign="top" charoff="50"> </td></tr><tr><td align="left" valign="top" charoff="50"> 6<hr></hr></td><td align="char" valign="top" char="." charoff="50">10<hr></hr></td><td align="char" valign="top" char="(" charoff="50">0.37 (0.17–0.83)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.09<hr></hr></td><td align="char" valign="top" char="(" charoff="50">0.34 (0.15–0.77)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.07<hr></hr></td><td align="char" valign="top" char="(" charoff="50">0.38 (0.16–0.88)<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.17<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">EGFR intron 1 PM</td><td align="char" valign="top" char="." charoff="50">186</td><td align="char" valign="top" char="(" charoff="50">1.01 (0.95–1.07)</td><td align="char" valign="top" char="." charoff="50">0.7</td><td align="char" valign="top" char="(" charoff="50">1.00 (0.94–1.06)</td><td align="char" valign="top" char="." charoff="50">0.93</td><td align="char" valign="top" char="(" charoff="50">1.02 (0.96–1.08)</td><td align="char" valign="top" char="." charoff="50">0.54</td></tr></tbody></table><table-wrap-foot><fn id="t4-fn1"><p>Abbreviations: CI=confidence interval; FISH=fluorescence <italic>in-situ</italic> hybridisation; HR=hazard ratio; IHC=immunohistochemistry; OS=overall survival; PM=polymorphism; PTEN=Phosphatase and tensin homolog; TTF1=time-to-treatment failure 1; TTF2=time-to-treatment failure 2.</p></fn></table-wrap-foot></table-wrap><table-wrap id="tbl5"><label>Table 5</label><caption><title>Correlation of molecular EGFR pathway biomarkers with the occurrence of skin rash (any grades, I–IV)</title></caption><table frame="hsides" rules="groups" border="1"><colgroup><col align="left"></col><col align="left"></col><col align="center"></col><col align="center"></col><col align="left"></col><col align="center"></col></colgroup><thead valign="bottom"><tr><th align="left" valign="top" charoff="50"> </th><th align="left" valign="top" charoff="50"> </th><th colspan="2" align="center" valign="top" charoff="50"><bold>Dichotomous variable</bold></th><th colspan="2" align="center" valign="top" charoff="50"><bold>Continuous variable</bold></th></tr></thead><tbody valign="top"><tr><td align="left" valign="top" charoff="50"><bold>Biomarker</bold><hr></hr></td><td align="left" valign="top" charoff="50"><italic><bold>n</bold></italic><hr></hr></td><td align="center" valign="top" charoff="50"><bold>Odds ratio</bold><hr></hr></td><td align="center" valign="top" charoff="50"><italic><bold>P</bold></italic><hr></hr></td><td align="center" valign="top" charoff="50"><bold>Odds ratio</bold><hr></hr></td><td align="center" valign="top" charoff="50"><italic><bold>P</bold></italic><hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">KRAS<hr></hr></td><td align="char" valign="top" char="." charoff="50">163<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.96<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.59<hr></hr></td><td align="center" valign="top" charoff="50">—<hr></hr></td><td align="center" valign="top" charoff="50"> <hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">EGFR–IHC<hr></hr></td><td align="char" valign="top" char="." charoff="50">170<hr></hr></td><td align="char" valign="top" char="." charoff="50">1.09<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.22<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.97<xref ref-type="fn" rid="t5-fn2">a</xref><hr></hr></td><td align="char" valign="top" char="." charoff="50">0.37<xref ref-type="fn" rid="t5-fn2">a</xref><hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">EGFR–FISH<hr></hr></td><td align="char" valign="top" char="." charoff="50">155<hr></hr></td><td align="char" valign="top" char="." charoff="50">1.03<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.73<hr></hr></td><td align="char" valign="top" char="." charoff="50">1.18<xref ref-type="fn" rid="t5-fn2">a</xref><hr></hr></td><td align="char" valign="top" char="." charoff="50">0.74<xref ref-type="fn" rid="t5-fn2">a</xref><hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">PTEN–IHC<hr></hr></td><td align="char" valign="top" char="." charoff="50">161<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.91<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.33<hr></hr></td><td align="char" valign="top" char="." charoff="50">1.09<xref ref-type="fn" rid="t5-fn2">a</xref><hr></hr></td><td align="char" valign="top" char="." charoff="50">0.42<xref ref-type="fn" rid="t5-fn2">a</xref><hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">EGFR intron 1 PM<hr></hr></td><td align="char" valign="top" char="." charoff="50">176<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.93<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.29<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.99<hr></hr></td><td align="char" valign="top" char="." charoff="50">0.31<hr></hr></td></tr><tr><td align="left" valign="top" charoff="50">EGFR R497K PM</td><td align="char" valign="top" char="." charoff="50">182</td><td align="char" valign="top" char="." charoff="50">1.1</td><td align="char" valign="top" char="." charoff="50">0.19</td><td align="center" valign="top" charoff="50">—</td><td align="center" valign="top" charoff="50"> </td></tr></tbody></table><table-wrap-foot><fn id="t5-fn1"><p>Abbreviations: EGFR=epidermal growth factor receptor; FISH=fluorescence <italic>in-situ</italic> hybridisation; IHC=immunohistochemistry; KRAS=Kirsten rat sarcoma viral oncogene; PM=polymorphism; PTEN=Phosphatase and tensin homolog.</p></fn><fn id="t5-fn2"><label>a</label><p>Single odds ratio for lowest <italic>vs</italic> highest score; global <italic>P</italic>-value.</p></fn></table-wrap-foot></table-wrap></floats-group></article>