<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Hum Genomics</journal-id><journal-id journal-id-type="iso-abbrev">Hum. Genomics</journal-id><journal-title-group><journal-title>Human Genomics</journal-title></journal-title-group><issn pub-type="ppub">1473-9542</issn><issn pub-type="epub">1479-7364</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">26742487</article-id><article-id pub-id-type="pmc">4705764</article-id><article-id pub-id-type="publisher-id">59</article-id><article-id pub-id-type="doi">10.1186/s40246-015-0059-6</article-id><article-categories><subj-group subj-group-type="heading"><subject>Primary Research</subject></subj-group></article-categories><title-group><article-title>Intragraft transcriptional profiling of renal transplant patients with tubular dysfunction reveals mechanisms underlying graft injury and recovery</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Azevedo</surname><given-names>Hátylas</given-names></name><xref ref-type="aff" rid="Aff1"></xref></contrib><contrib contrib-type="author"><name><surname>Renesto</surname><given-names>Paulo Guilherme</given-names></name><xref ref-type="aff" rid="Aff3"></xref></contrib><contrib contrib-type="author"><name><surname>Chinen</surname><given-names>Rogério</given-names></name><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author"><name><surname>Naka</surname><given-names>Erika</given-names></name><xref ref-type="aff" rid="Aff3"></xref></contrib><contrib contrib-type="author"><name><surname>de Matos</surname><given-names>Ana Cristina Carvalho</given-names></name><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author"><name><surname>Cenedeze</surname><given-names>Marcos Antônio</given-names></name><xref ref-type="aff" rid="Aff3"></xref></contrib><contrib contrib-type="author"><name><surname>Moreira-Filho</surname><given-names>Carlos Alberto</given-names></name><xref ref-type="aff" rid="Aff1"></xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Câmara</surname><given-names>Niels Olsen Saraiva</given-names></name><address><email>niels@icb.usp.br</email></address><xref ref-type="aff" rid="Aff2"></xref><xref ref-type="aff" rid="Aff3"></xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Pacheco-Silva</surname><given-names>Alvaro</given-names></name><address><email>alvaro.pfilho@einstein.br</email></address><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><aff id="Aff1"><label></label>Department of Pediatrics, Faculdade de Medicina da Universidade de São Paulo (FMUSP), São Paulo, Brazil </aff><aff id="Aff2"><label></label>Laboratory of Transplantation Immunobiology, Department of Immunology, Institute of Biomedical Sciences, Universidade de São Paulo (USP), São Paulo, Brazil </aff><aff id="Aff3"><label></label>Laboratory of Clinical and Experimental Immunology, Nephrology Division, Universidade Federal de São Paulo (UNIFESP), São Paulo, Brazil </aff><aff id="Aff4"><label></label>Instituto Israelita de Ensino e Pesquisa Albert Einstein, Hospital Albert Einstein, São Paulo, Brazil </aff></contrib-group><pub-date pub-type="epub"><day>7</day><month>1</month><year>2016</year></pub-date><pub-date pub-type="pmc-release"><day>7</day><month>1</month><year>2016</year></pub-date><pub-date pub-type="collection"><year>2016</year></pub-date><volume>10</volume><elocation-id>2</elocation-id><history><date date-type="received"><day>17</day><month>11</month><year>2015</year></date><date date-type="accepted"><day>30</day><month>12</month><year>2015</year></date></history><permissions><copyright-statement>© Azevedo et al. 2016</copyright-statement><license license-type="OpenAccess"><license-p>
<bold>Open Access</bold>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</ext-link>) applies to the data made available in this article, unless otherwise stated.</license-p></license></permissions><abstract id="Abs1"><sec><title><offsets xml_i="4986" xml_f="4996" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="5007" xml_f="5374" txt_i="22" txt_f="389">Proximal tubular dysfunction (PTD) is associated with a decreased long-term graft survival in renal transplant patients and can be detected by the elevation of urinary tubular proteins. This study investigated transcriptional changes in biopsies from renal transplant patients with PTD to disclose molecular mechanisms underlying graft injury and functional recovery.</offsets></p></sec><sec><title><offsets xml_i="5396" xml_f="5403" txt_i="391" txt_f="398">Methods</offsets></title><p><offsets xml_i="5414" xml_f="6203" txt_i="399" txt_f="1188">Thirty-three renal transplant patients with high urinary levels of retinol-binding protein, a biomarker of PTD, were enrolled in the study. The initial immunosuppressive scheme included azathioprine, cyclosporine, and steroids. After randomization, 18 patients (group 2) had their treatment modified by reducing cyclosporine dosage and substituting azathioprine for mycophenolate mofetil, while the other 15 patients (group 1) remained under the initial scheme. Patients were biopsied at enrollment and after 12 months of follow-up, and paired comparisons were performed between their intragraft gene expression profiles. The differential transcriptome profiles were analyzed by constructing gene co-expression networks and identifying enriched functions and central nodes in each network.</offsets></p></sec><sec><title><offsets xml_i="6225" xml_f="6232" txt_i="1190" txt_f="1197">Results</offsets></title><p><offsets xml_i="6243" xml_f="6641" txt_i="1198" txt_f="1596">Only the alternative immunosuppressive scheme used in group 2 ameliorated renal function and tubular proteinuria after 12 months of follow-up. Intragraft molecular changes observed in group 2 were linked to autophagy, extracellular matrix, and adaptive immunity. Conversely, gene expression changes in group 1 were related to fibrosis, endocytosis, ubiquitination, and endoplasmic reticulum stress.</offsets></p></sec><sec><title><offsets xml_i="6663" xml_f="6673" txt_i="1598" txt_f="1608">Conclusion</offsets></title><p><offsets xml_i="6684" xml_f="6884" txt_i="1609" txt_f="1809">These results suggest that molecular networks associated with the control of endocytosis, autophagy, protein overload, fibrosis, and adaptive immunity may be involved in improvement of graft function.</offsets></p></sec><sec><title><offsets xml_i="6906" xml_f="6939" txt_i="1811" txt_f="1844">Electronic supplementary material</offsets></title><p><offsets xml_i="6950" xml_f="7089" txt_i="1845" txt_f="1984">The online version of this article (doi:10.1186/s40246-015-0059-6) contains supplementary material, which is available to authorized users.</offsets></p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Network analysis</kwd><kwd>Kidney transplantation</kwd><kwd>Genomics</kwd><kwd>Proximal tubular dysfunction</kwd><kwd>Transcriptional profiling</kwd></kwd-group><funding-group><award-group><funding-source><institution>FAPESP</institution></funding-source><award-id>2011/50761-2</award-id><award-id>2009/53443-1 and 2011/50761-2</award-id><principal-award-recipient><name><surname>Moreira-Filho</surname><given-names>Carlos Alberto</given-names></name><name><surname>Câmara</surname><given-names>Niels Olsen Saraiva</given-names></name></principal-award-recipient></award-group><award-group><funding-source><institution>Conselho Nacional de Desenvolvimento Cientifico e Tecnológico-CNPq </institution></funding-source><award-id>307626/2014-8</award-id><principal-award-recipient><name><surname>Câmara</surname><given-names>Niels Olsen Saraiva</given-names></name></principal-award-recipient></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2016</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1" sec-type="introduction"><title><offsets xml_i="8317" xml_f="8329" txt_i="1993" txt_f="2005">Introduction</offsets></title><p><offsets xml_i="8340" xml_f="8593" txt_i="2006" txt_f="2259">Proximal tubular dysfunction (PTD) is characterized by proteinuria, aminoaciduria, and glucosuria. It is associated with a decreased long-term graft survival in renal transplant patients and can be detected by the elevation of urinary tubular proteins [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="8625" xml_f="8626" txt_i="2259" txt_f="2260">1</offsets></xref><offsets xml_i="8633" xml_f="8635" txt_i="2260" txt_f="2262">, </offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="8667" xml_f="8668" txt_i="2262" txt_f="2263">2</offsets></xref><offsets xml_i="8675" xml_f="8677" txt_i="2263" txt_f="2265">].</offsets></p><p><offsets xml_i="8684" xml_f="8760" txt_i="2266" txt_f="2342">Low-molecular-weight proteins (LMWP) are prominent urine biomarkers of PTD [</offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="8792" xml_f="8793" txt_i="2342" txt_f="2343">3</offsets></xref><offsets xml_i="8800" xml_f="8801" txt_i="2343" txt_f="2344">–</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="8833" xml_f="8834" txt_i="2344" txt_f="2345">7</offsets></xref><offsets xml_i="8841" xml_f="9121" txt_i="2345" txt_f="2625">], as they are cleared by glomerular filtration and almost totally reabsorbed by tubular epithelial cells. Specifically, increased urinary levels of the LMWP urinary retinol-binding protein (uRBP) have been associated with tubular injury and fibrosis after renal transplantation [</offsets><xref ref-type="bibr" rid="CR8"><offsets xml_i="9153" xml_f="9154" txt_i="2625" txt_f="2626">8</offsets></xref><offsets xml_i="9161" xml_f="9163" txt_i="2626" txt_f="2628">, </offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="9195" xml_f="9196" txt_i="2628" txt_f="2629">9</offsets></xref><offsets xml_i="9203" xml_f="9325" txt_i="2629" txt_f="2751">]. Therefore, measuring uRBP levels may contribute to the detection of patients at a higher risk for renal function loss [</offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="9358" xml_f="9360" txt_i="2751" txt_f="2753">10</offsets></xref><offsets xml_i="9367" xml_f="9369" txt_i="2753" txt_f="2755">, </offsets><xref ref-type="bibr" rid="CR11"><offsets xml_i="9402" xml_f="9404" txt_i="2755" txt_f="2757">11</offsets></xref><offsets xml_i="9411" xml_f="9413" txt_i="2757" txt_f="2759">].</offsets></p><p><offsets xml_i="9420" xml_f="9727" txt_i="2760" txt_f="3067">The early detection of transplanted patients with kidney dysfunction helps optimizing their immunosuppression protocols in the attempt to improve graft outcomes. For instance, PTD is found in heart transplant patients with a progressive worsening on renal function due to cyclosporine (CsA) nephrotoxicity [</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="9760" xml_f="9762" txt_i="3067" txt_f="3069">12</offsets></xref><offsets xml_i="9769" xml_f="9906" txt_i="3069" txt_f="3206">]. In parallel, immunosuppression with mycophenolate mofetil (MYF) is associated with less acute rejection after kidney transplantation [</offsets><xref ref-type="bibr" rid="CR13"><offsets xml_i="9939" xml_f="9941" txt_i="3206" txt_f="3208">13</offsets></xref><offsets xml_i="9948" xml_f="10237" txt_i="3208" txt_f="3497">], offering a therapeutic alternative for patients with renal dysfunction caused by CsA. However, a large number of transplant patients in developing countries still receive maintenance immunosuppression regimens containing low-cost drugs, including CsA, azathioprine (AZA), and steroids [</offsets><xref ref-type="bibr" rid="CR14"><offsets xml_i="10270" xml_f="10272" txt_i="3497" txt_f="3499">14</offsets></xref><offsets xml_i="10279" xml_f="10491" txt_i="3499" txt_f="3711">]. Hence, the follow-up of renal transplant patients diagnosed with PTD allows the adoption of tailored immunosuppression regimens and provides a window of opportunity for investigating renal injury and recovery.</offsets></p><p><offsets xml_i="10498" xml_f="10943" txt_i="3712" txt_f="4157">Here, we followed up renal transplant patients with elevated uRBP levels that were submitted to specific immunosuppression regimens for 12 months. Kidney biopsies were performed to investigate intragraft transcriptional profiles and to correlate their molecular changes with renal function outcomes. Through this approach, we identified some potential molecular mechanisms associated with graft function improvement in renal transplant patients.</offsets></p></sec><sec id="Sec2" sec-type="results"><title><offsets xml_i="10994" xml_f="11001" txt_i="4159" txt_f="4166">Results</offsets></title><sec id="Sec3"><title><offsets xml_i="11031" xml_f="11161" txt_i="4167" txt_f="4297">Assessment of renal dysfunction in transplanted patients and association between tubular proteinuria and glomerular function rates</offsets></title><p><offsets xml_i="11172" xml_f="11178" txt_i="4298" txt_f="4304">Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="11212" xml_f="11213" txt_i="4304" txt_f="4305">1</offsets></xref><offsets xml_i="11220" xml_f="11365" txt_i="4305" txt_f="4450"> summarizes the patients’ baseline clinical and laboratory features. None of these characteristics showed statistically significant differences (</offsets><italic><offsets xml_i="11373" xml_f="11374" txt_i="4450" txt_f="4451">p</offsets></italic><offsets xml_i="11383" xml_f="12102" txt_i="4451" txt_f="5167"> &lt; 0.05) between group 1 and group 2 patients. The analysis of renal histology in our cohort revealed no significant differences in the baseline Banff classification scores between the groups and also no significant differences in the Banff scores within the same group after 12 months of follow-up. At enrollment, the biopsies of three patients from group 1 showed no evidence of interstitial fibrosis and tubular atrophy (IF/TA), while nine, three, and zero biopsies showed mild, moderate, and severe IF/TA, respectively. At the end of the follow-up period, all biopsies from group 1 showed some evidence of IF/TA, with eight biopsies at mild, two at moderate, and two at severe IF/TA classification, respectively.</offsets><table-wrap id="Tab1"><label><offsets xml_i="12131" xml_f="12138" txt_i="5167" txt_f="5174">Table 1</offsets></label><caption><p><offsets xml_i="12158" xml_f="12264" txt_i="5174" txt_f="5280">Baseline clinical and laboratory characteristics of the renal transplant patients enrolled in this study (</offsets><italic><offsets xml_i="12272" xml_f="12273" txt_i="5280" txt_f="5281">n</offsets></italic><offsets xml_i="12282" xml_f="12288" txt_i="5281" txt_f="5287"> = 33)</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th><offsets xml_i="12354" xml_f="12363" txt_i="5288" txt_f="5297">Variables</offsets></th><th><offsets xml_i="12372" xml_f="12379" txt_i="5297" txt_f="5304">Group 1</offsets><sup><offsets xml_i="12384" xml_f="12385" txt_i="5304" txt_f="5305">a</offsets></sup><offsets xml_i="12391" xml_f="12393" txt_i="5305" txt_f="5307"> (</offsets><italic><offsets xml_i="12401" xml_f="12402" txt_i="5307" txt_f="5308">n</offsets></italic><offsets xml_i="12411" xml_f="12417" txt_i="5308" txt_f="5314"> = 15)</offsets></th><th><offsets xml_i="12426" xml_f="12433" txt_i="5314" txt_f="5321">Group 2</offsets><sup><offsets xml_i="12438" xml_f="12439" txt_i="5321" txt_f="5322">b</offsets></sup><offsets xml_i="12445" xml_f="12447" txt_i="5322" txt_f="5324"> (</offsets><italic><offsets xml_i="12455" xml_f="12456" txt_i="5324" txt_f="5325">n</offsets></italic><offsets xml_i="12465" xml_f="12471" txt_i="5325" txt_f="5331"> = 18)</offsets></th><th><offsets xml_i="12480" xml_f="12481" txt_i="5331" txt_f="5332">
</offsets><italic><offsets xml_i="12489" xml_f="12490" txt_i="5332" txt_f="5333">p</offsets></italic><offsets xml_i="12499" xml_f="12505" txt_i="5333" txt_f="5339"> value</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="12538" xml_f="12566" txt_i="5339" txt_f="5367">Recipient age ± SD, in years</offsets></td><td><offsets xml_i="12575" xml_f="12586" txt_i="5368" txt_f="5379">44.9 ± 11.2</offsets></td><td><offsets xml_i="12595" xml_f="12605" txt_i="5380" txt_f="5390">45.4 ± 9.9</offsets></td><td><offsets xml_i="12614" xml_f="12618" txt_i="5391" txt_f="5395">0.79</offsets></td></tr><tr><td><offsets xml_i="12636" xml_f="12664" txt_i="5396" txt_f="5424">Recipient weight ± SD, in kg</offsets></td><td><offsets xml_i="12673" xml_f="12684" txt_i="5425" txt_f="5436">67.5 ± 13.8</offsets></td><td><offsets xml_i="12693" xml_f="12704" txt_i="5437" txt_f="5448">69.6 ± 14.7</offsets></td><td><offsets xml_i="12713" xml_f="12717" txt_i="5449" txt_f="5453">0.35</offsets></td></tr><tr><td><offsets xml_i="12735" xml_f="12751" txt_i="5454" txt_f="5470">Recipient male, </offsets><italic><offsets xml_i="12759" xml_f="12760" txt_i="5470" txt_f="5471">N</offsets></italic><offsets xml_i="12769" xml_f="12773" txt_i="5471" txt_f="5475"> (%)</offsets></td><td><offsets xml_i="12782" xml_f="12793" txt_i="5476" txt_f="5487">11 (73.3 %)</offsets></td><td><offsets xml_i="12802" xml_f="12813" txt_i="5488" txt_f="5499">10 (55.6 %)</offsets></td><td><offsets xml_i="12822" xml_f="12826" txt_i="5500" txt_f="5504">0.34</offsets></td></tr><tr><td><offsets xml_i="12844" xml_f="12861" txt_i="5505" txt_f="5522">Deceased donors, </offsets><italic><offsets xml_i="12869" xml_f="12870" txt_i="5522" txt_f="5523">N</offsets></italic><offsets xml_i="12879" xml_f="12883" txt_i="5523" txt_f="5527"> (%)</offsets></td><td><offsets xml_i="12892" xml_f="12900" txt_i="5528" txt_f="5536">6 (40 %)</offsets></td><td><offsets xml_i="12909" xml_f="12919" txt_i="5537" txt_f="5547">6 (33.3 %)</offsets></td><td><offsets xml_i="12928" xml_f="12932" txt_i="5548" txt_f="5552">0.69</offsets></td></tr><tr><td><offsets xml_i="12950" xml_f="12970" txt_i="5553" txt_f="5573">Haploidentical HLA, </offsets><italic><offsets xml_i="12978" xml_f="12979" txt_i="5573" txt_f="5574">N</offsets></italic><offsets xml_i="12988" xml_f="12992" txt_i="5574" txt_f="5578"> (%)</offsets></td><td><offsets xml_i="13001" xml_f="13011" txt_i="5579" txt_f="5589">5 (33.3 %)</offsets></td><td><offsets xml_i="13020" xml_f="13031" txt_i="5590" txt_f="5601">10 (55.6 %)</offsets></td><td><offsets xml_i="13040" xml_f="13044" txt_i="5602" txt_f="5606">0.20</offsets></td></tr><tr><td><offsets xml_i="13062" xml_f="13098" txt_i="5607" txt_f="5643">Time post-transplant ± SD, in months</offsets></td><td><offsets xml_i="13107" xml_f="13118" txt_i="5644" txt_f="5655">93.3 ± 35.7</offsets></td><td><offsets xml_i="13127" xml_f="13138" txt_i="5656" txt_f="5667">92.4 ± 33.1</offsets></td><td><offsets xml_i="13147" xml_f="13151" txt_i="5668" txt_f="5672">0.94</offsets></td></tr><tr><td><offsets xml_i="13169" xml_f="13201" txt_i="5673" txt_f="5705">Systemic arterial hypertension, </offsets><italic><offsets xml_i="13209" xml_f="13210" txt_i="5705" txt_f="5706">N</offsets></italic><offsets xml_i="13219" xml_f="13223" txt_i="5706" txt_f="5710"> (%)</offsets></td><td><offsets xml_i="13232" xml_f="13243" txt_i="5711" txt_f="5722">13 (86.7 %)</offsets></td><td><offsets xml_i="13252" xml_f="13263" txt_i="5723" txt_f="5734">16 (88.9 %)</offsets></td><td><offsets xml_i="13272" xml_f="13276" txt_i="5735" txt_f="5739">0.84</offsets></td></tr><tr><td><offsets xml_i="13294" xml_f="13313" txt_i="5740" txt_f="5759">Diabetes mellitus, </offsets><italic><offsets xml_i="13321" xml_f="13322" txt_i="5759" txt_f="5760">N</offsets></italic><offsets xml_i="13331" xml_f="13335" txt_i="5760" txt_f="5764"> (%)</offsets></td><td><offsets xml_i="13344" xml_f="13354" txt_i="5765" txt_f="5775">2 (13.3 %)</offsets></td><td><offsets xml_i="13363" xml_f="13372" txt_i="5776" txt_f="5785">1 (5.6 %)</offsets></td><td><offsets xml_i="13381" xml_f="13385" txt_i="5786" txt_f="5790">0.44</offsets></td></tr><tr><td><offsets xml_i="13403" xml_f="13441" txt_i="5791" txt_f="5829">Serum creatinine levels ± SD, in mg/dL</offsets></td><td><offsets xml_i="13450" xml_f="13461" txt_i="5830" txt_f="5841">1.59 ± 0.28</offsets></td><td><offsets xml_i="13470" xml_f="13481" txt_i="5842" txt_f="5853">1.67 ± 0.33</offsets></td><td><offsets xml_i="13490" xml_f="13494" txt_i="5854" txt_f="5858">0.46</offsets></td></tr><tr><td><offsets xml_i="13512" xml_f="13537" txt_i="5859" txt_f="5884">uRBP levels ± SD, in mg/L</offsets></td><td><offsets xml_i="13546" xml_f="13557" txt_i="5885" txt_f="5896">2.79 ± 2.55</offsets></td><td><offsets xml_i="13566" xml_f="13577" txt_i="5897" txt_f="5908">2.72 ± 2.24</offsets></td><td><offsets xml_i="13586" xml_f="13590" txt_i="5909" txt_f="5913">0.88</offsets></td></tr><tr><td><offsets xml_i="13608" xml_f="13628" txt_i="5914" txt_f="5934">Banff classification</offsets><sup><offsets xml_i="13633" xml_f="13634" txt_i="5934" txt_f="5935">c</offsets></sup><offsets xml_i="13640" xml_f="13654" txt_i="5935" txt_f="5949"> at enrollment</offsets></td><td><offsets xml_i="13663" xml_f="13714" txt_i="5950" txt_f="6001">No evidence (3), mild (9), moderate (3), severe (0)</offsets></td><td><offsets xml_i="13723" xml_f="13774" txt_i="6002" txt_f="6053">No evidence (2), mild (9), moderate (5), severe (2)</offsets></td><td><offsets xml_i="13783" xml_f="13787" txt_i="6054" txt_f="6058">0.64</offsets></td></tr><tr><td><offsets xml_i="13805" xml_f="13825" txt_i="6059" txt_f="6079">Banff classification</offsets><sup><offsets xml_i="13830" xml_f="13831" txt_i="6079" txt_f="6080">c</offsets></sup><offsets xml_i="13837" xml_f="13853" txt_i="6080" txt_f="6096"> after 12 months</offsets></td><td><offsets xml_i="13862" xml_f="13913" txt_i="6097" txt_f="6148">No evidence (0), mild (8), moderate (2), severe (2)</offsets></td><td><offsets xml_i="13922" xml_f="13973" txt_i="6149" txt_f="6200">No evidence (3), mild (7), moderate (6), severe (1)</offsets></td><td><offsets xml_i="13982" xml_f="13986" txt_i="6201" txt_f="6205">0.25</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="14032" xml_f="14033" txt_i="6206" txt_f="6207">
</offsets><sup><offsets xml_i="14038" xml_f="14039" txt_i="6207" txt_f="6208">a</offsets></sup><offsets xml_i="14045" xml_f="14148" txt_i="6208" txt_f="6311">Group 1 patients: initial immunosuppression scheme with cyclosporine, corticosteroids, and azathioprine</offsets></p><p><offsets xml_i="14155" xml_f="14156" txt_i="6312" txt_f="6313">
</offsets><sup><offsets xml_i="14161" xml_f="14162" txt_i="6313" txt_f="6314">b</offsets></sup><offsets xml_i="14168" xml_f="14291" txt_i="6314" txt_f="6437">Group 2 patients: alternative immunosuppression scheme with reduced levels of cyclosporine and use of mycophenolate mofetil</offsets></p><p><offsets xml_i="14298" xml_f="14299" txt_i="6438" txt_f="6439">
</offsets><sup><offsets xml_i="14304" xml_f="14305" txt_i="6439" txt_f="6440">c</offsets></sup><offsets xml_i="14311" xml_f="14420" txt_i="6440" txt_f="6549">Banff classification: mild, moderate, or severe evidence of interstitial fibrosis and tubular atrophy (IF/TA)</offsets></p></table-wrap-foot></table-wrap></p><p><offsets xml_i="14462" xml_f="14838" txt_i="6551" txt_f="6927">For group 2, the biopsies of two patients at enrollment showed no evidence of IF/TA, while nine, five, and two biopsies showed mild, moderate, and severe IF/TA, respectively. At the end of the follow-up period, three biopsies from group 2 showed no evidence of IF/TA, while the biopsies of seven, six, and one patients displayed mild, moderate, and severe IF/TA, respectively.</offsets></p><p><offsets xml_i="14845" xml_f="14852" txt_i="6928" txt_f="6935">Figure </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="14884" xml_f="14885" txt_i="6935" txt_f="6936">1</offsets></xref><offsets xml_i="14892" xml_f="15231" txt_i="6936" txt_f="7275"> illustrates serum creatinine levels, estimated creatinine clearance, and uRBP levels at 0 and 12 months of follow-up. Interestingly, only patients from group 2 significantly improved these renal function parameters. Serum creatinine levels increased in group 1 after 12 months of follow-up, indicating a worsening in renal function (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="15263" xml_f="15265" txt_i="7275" txt_f="7277">1a</offsets></xref><offsets xml_i="15272" xml_f="15352" txt_i="7277" txt_f="7357">). In parallel, creatinine clearance calculated by either Cockcroft-Gault (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="15384" xml_f="15386" txt_i="7357" txt_f="7359">1b</offsets></xref><offsets xml_i="15393" xml_f="15449" txt_i="7359" txt_f="7415">) or modification of diet in renal disease (MDRD) (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="15481" xml_f="15483" txt_i="7415" txt_f="7417">1c</offsets></xref><offsets xml_i="15490" xml_f="15698" txt_i="7417" txt_f="7625">) equations was only significantly improved in patients submitted to the alternative immunosuppression protocol (group 2). uRBP concentration was also decreased in patients from group 2 after 12 months (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="15730" xml_f="15732" txt_i="7625" txt_f="7627">1d</offsets></xref><offsets xml_i="15739" xml_f="15741" txt_i="7627" txt_f="7629">, </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="15773" xml_f="15774" txt_i="7629" txt_f="7630">e</offsets></xref><offsets xml_i="15781" xml_f="15783" txt_i="7630" txt_f="7632">).</offsets><fig id="Fig1"><label><offsets xml_i="15805" xml_f="15811" txt_i="7632" txt_f="7638">Fig. 1</offsets></label><caption><p><offsets xml_i="15831" xml_f="15923" txt_i="7638" txt_f="7730">Measurement of serum creatinine, creatinine clearance, and urinary retinol-binding protein (</offsets><italic><offsets xml_i="15931" xml_f="15935" txt_i="7730" txt_f="7734">uRBP</offsets></italic><offsets xml_i="15944" xml_f="15954" txt_i="7734" txt_f="7744">) levels. </offsets><bold><offsets xml_i="15960" xml_f="15961" txt_i="7744" txt_f="7745">a</offsets></bold><offsets xml_i="15968" xml_f="16071" txt_i="7745" txt_f="7848"> Serum creatinine levels (mg/dL) at the beginning and at the end of the follow-up period of 12 months. </offsets><bold><offsets xml_i="16077" xml_f="16078" txt_i="7848" txt_f="7849">b</offsets></bold><offsets xml_i="16085" xml_f="16158" txt_i="7849" txt_f="7922"> Creatinine clearance (mL/min) calculated using Cockcroft-Gault formula. </offsets><bold><offsets xml_i="16164" xml_f="16165" txt_i="7922" txt_f="7923">c</offsets></bold><offsets xml_i="16172" xml_f="16234" txt_i="7923" txt_f="7985"> Creatinine clearance (mL/min) calculated using MDRD formula. </offsets><bold><offsets xml_i="16240" xml_f="16241" txt_i="7985" txt_f="7986">d</offsets></bold><offsets xml_i="16248" xml_f="16296" txt_i="7986" txt_f="8034"> uRBP levels (mg/L) from all enrolled patients. </offsets><bold><offsets xml_i="16302" xml_f="16303" txt_i="8034" txt_f="8035">e</offsets></bold><offsets xml_i="16310" xml_f="16391" txt_i="8035" txt_f="8116"> uRBP levels (mg/L) from patients used for intragraft gene expression profiling. </offsets><bold><offsets xml_i="16397" xml_f="16398" txt_i="8116" txt_f="8117">f</offsets></bold><offsets xml_i="16405" xml_f="16556" txt_i="8117" txt_f="8268"> Comparison between very high and high uRBP levels at t0 and t12 post-enrollment periods. Statistical significance was assessed using paired Student’s </offsets><italic><offsets xml_i="16564" xml_f="16565" txt_i="8268" txt_f="8269">t</offsets></italic><offsets xml_i="16574" xml_f="16585" txt_i="8269" txt_f="8280"> test with </offsets><italic><offsets xml_i="16593" xml_f="16594" txt_i="8280" txt_f="8281">p</offsets></italic><offsets xml_i="16603" xml_f="16795" txt_i="8281" txt_f="8470"> &lt; 0.05. Renal function parameters were significantly altered in the groups treated with azathioprine (*AZA group comparison) or mycophenolate mofetil (**MYF group comparison) for 12 months</offsets></p></caption><graphic xlink:href="40246_2015_59_Fig1_HTML" id="MO1"></graphic></fig></p><p><offsets xml_i="16887" xml_f="17240" txt_i="8472" txt_f="8819">The distribution of uRBP values relative to estimated glomerular function rate (eGFR) equally showed a significant increase in RBP proteinuria in patients whose eGFR values ranged from 30 to 45 mL/min (with moderate kidney function reduction). uRBP levels in patients with eGFR &lt;30 mL/min did not differ to those with eGFR values &gt;45 mL/min (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="17272" xml_f="17274" txt_i="8819" txt_f="8821">2a</offsets></xref><offsets xml_i="17281" xml_f="17283" txt_i="8821" txt_f="8823">).</offsets><fig id="Fig2"><label><offsets xml_i="17305" xml_f="17311" txt_i="8823" txt_f="8829">Fig. 2</offsets></label><caption><p><offsets xml_i="17331" xml_f="17486" txt_i="8829" txt_f="8984">Distribution of uRBP levels according to glomerular filtration rates and hierarchical clustering analysis of samples from different ranges of uRBP levels. </offsets><bold><offsets xml_i="17492" xml_f="17493" txt_i="8984" txt_f="8985">a</offsets></bold><offsets xml_i="17500" xml_f="17704" txt_i="8985" txt_f="9189"> Histograms showing uRBP values according to eGFR levels at enrollment (t0) and after 12 months of follow-up (t12). uRBP levels are expressed as the mean ± SEM. The estimated glomerular filtration rates (</offsets><italic><offsets xml_i="17712" xml_f="17716" txt_i="9189" txt_f="9193">eGFR</offsets></italic><offsets xml_i="17725" xml_f="17769" txt_i="9193" txt_f="9237">) were calculated using the MDRD formula. **</offsets><italic><offsets xml_i="17777" xml_f="17778" txt_i="9237" txt_f="9238">p</offsets></italic><offsets xml_i="17787" xml_f="17801" txt_i="9238" txt_f="9249"> &lt; 0.05, **</offsets><italic><offsets xml_i="17809" xml_f="17810" txt_i="9249" txt_f="9250">p</offsets></italic><offsets xml_i="17819" xml_f="17909" txt_i="9250" txt_f="9337"> &lt; 0.001 compared to eGFR at 30–45 mL/min, using ANOVA followed by Tukey’s correction. </offsets><bold><offsets xml_i="17915" xml_f="17916" txt_i="9337" txt_f="9338">b</offsets></bold><offsets xml_i="17923" xml_f="18022" txt_i="9338" txt_f="9437"> Best-fit slope of the linear regression of the delta values from urinary retinol-binding protein (</offsets><italic><offsets xml_i="18030" xml_f="18034" txt_i="9437" txt_f="9441">uRBP</offsets></italic><offsets xml_i="18043" xml_f="18155" txt_i="9441" txt_f="9553">) levels and eGFR between t0 and t12. The estimated eGFR was calculated using Cockcroft-Gault or MDRD formulas. </offsets><bold><offsets xml_i="18161" xml_f="18162" txt_i="9553" txt_f="9554">c</offsets></bold><offsets xml_i="18169" xml_f="18233" txt_i="9554" txt_f="9618"> Hierarchical clustering analysis of samples in the high group (</offsets><italic><offsets xml_i="18241" xml_f="18243" txt_i="9618" txt_f="9620">HG</offsets></italic><offsets xml_i="18252" xml_f="18307" txt_i="9620" txt_f="9675">, uRBP levels from 0.4 to 1 mg/L) and very high group (</offsets><italic><offsets xml_i="18315" xml_f="18318" txt_i="9675" txt_f="9678">VHG</offsets></italic><offsets xml_i="18327" xml_f="18357" txt_i="9678" txt_f="9708">, uRBP ≥ 1 mg/L) at t0 and t12</offsets></p></caption><graphic xlink:href="40246_2015_59_Fig2_HTML" id="MO2"></graphic></fig></p><p><offsets xml_i="18449" xml_f="18719" txt_i="9710" txt_f="9980">A linear regression analysis was conducted to examine the association between delta values (t12–t0) for uRBP and eGFR over 12 months of follow-up. We found a small but significant correlation between delta values for uRBP and eGFR calculated via either Cockcroft-Gault (</offsets><italic><offsets xml_i="18727" xml_f="18728" txt_i="9980" txt_f="9981">r</offsets></italic><sup><offsets xml_i="18742" xml_f="18743" txt_i="9981" txt_f="9982">2</offsets></sup><offsets xml_i="18749" xml_f="18760" txt_i="9982" txt_f="9993"> = 0.2062, </offsets><italic><offsets xml_i="18768" xml_f="18769" txt_i="9993" txt_f="9994">p</offsets></italic><offsets xml_i="18778" xml_f="18798" txt_i="9994" txt_f="10014"> = 0.0103) or MDRD (</offsets><italic><offsets xml_i="18806" xml_f="18807" txt_i="10014" txt_f="10015">r</offsets></italic><sup><offsets xml_i="18821" xml_f="18822" txt_i="10015" txt_f="10016">2</offsets></sup><offsets xml_i="18828" xml_f="18839" txt_i="10016" txt_f="10027"> = 0.2880, </offsets><italic><offsets xml_i="18847" xml_f="18848" txt_i="10027" txt_f="10028">p</offsets></italic><offsets xml_i="18857" xml_f="18898" txt_i="10028" txt_f="10069"> = 0.0019) formulas, as depicted in Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="18930" xml_f="18932" txt_i="10069" txt_f="10071">2b</offsets></xref><offsets xml_i="18939" xml_f="19070" txt_i="10071" txt_f="10202">. This analysis showed that around 25 % of the variance in eGFR values was significantly explained by the variation in uRBP values.</offsets></p></sec><sec id="Sec4"><title><offsets xml_i="19102" xml_f="19218" txt_i="10204" txt_f="10320">Network enrichment analysis reveals intragraft molecular changes associated with specific immunosuppression regimens</offsets></title><p><offsets xml_i="19229" xml_f="19362" txt_i="10321" txt_f="10454">The lists of differentially expressed (DE) genes obtained in each comparison and their fold changes are displayed in Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="19398" xml_f="19399" txt_i="10454" txt_f="10455">1</offsets></xref><offsets xml_i="19406" xml_f="19788" txt_i="10455" txt_f="10837">. These DE genes were used to construct co-expression gene networks, and centrality measures were calculated for all nodes in order to determine the central nodes in each network. The centrality measures calculated were degree, which denotes the number of interactions a node has, and betweenness, which represents the fraction of shortest paths that passes through a specific node.</offsets></p><p><offsets xml_i="19795" xml_f="19947" txt_i="10838" txt_f="10990">These central nodes were then categorized in three subclasses of hubs, high-hubs, and bottlenecks, using the information from the scatterplots in Figs. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="19979" xml_f="19981" txt_i="10990" txt_f="10992">3b</offsets></xref><offsets xml_i="19988" xml_f="19990" txt_i="10992" txt_f="10994">, </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="20022" xml_f="20024" txt_i="10994" txt_f="10996">4b</offsets></xref><offsets xml_i="20031" xml_f="20033" txt_i="10996" txt_f="10998">, </offsets><xref rid="Fig5" ref-type="fig"><offsets xml_i="20065" xml_f="20067" txt_i="10998" txt_f="11000">5b</offsets></xref><offsets xml_i="20074" xml_f="20080" txt_i="11000" txt_f="11006">, and </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="20112" xml_f="20114" txt_i="11006" txt_f="11008">6b</offsets></xref><offsets xml_i="20121" xml_f="20466" txt_i="11008" txt_f="11353">. High-hubs, hubs, and bottlenecks are displayed in the scatterplots respectively at the up-right, down-right, and up-left quadrants, due to their differences in degree and betweenness values. This classification is in accordance with previous studies showing that node subclasses exert important functions in the context of health and disease [</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="20499" xml_f="20501" txt_i="11353" txt_f="11355">15</offsets></xref><offsets xml_i="20508" xml_f="20509" txt_i="11355" txt_f="11356">–</offsets><xref ref-type="bibr" rid="CR17"><offsets xml_i="20542" xml_f="20544" txt_i="11356" txt_f="11358">17</offsets></xref><offsets xml_i="20551" xml_f="20653" txt_i="11358" txt_f="11460">]. For instance, hub genes regulate many other genes and tend to be essential in biological networks [</offsets><xref ref-type="bibr" rid="CR18"><offsets xml_i="20686" xml_f="20688" txt_i="11460" txt_f="11462">18</offsets></xref><offsets xml_i="20695" xml_f="20810" txt_i="11462" txt_f="11577">] and their node degrees increase in tumor networks, suggesting their association with gain of regulatory control [</offsets><xref ref-type="bibr" rid="CR19"><offsets xml_i="20843" xml_f="20845" txt_i="11577" txt_f="11579">19</offsets></xref><offsets xml_i="20852" xml_f="20936" txt_i="11579" txt_f="11663">]. In addition, high-hubs were linked to mechanisms underlying refractory epilepsy [</offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="20969" xml_f="20971" txt_i="11663" txt_f="11665">20</offsets></xref><offsets xml_i="20978" xml_f="21036" txt_i="11665" txt_f="11723">] and bottleneck genes were correlated with essentiality [</offsets><xref ref-type="bibr" rid="CR21"><offsets xml_i="21069" xml_f="21071" txt_i="11723" txt_f="11725">21</offsets></xref><offsets xml_i="21078" xml_f="21174" txt_i="11725" txt_f="11821">] in biological networks. The high-hubs, hubs, and bottlenecks are described in Additional file </offsets><xref rid="MOESM2" ref-type="media"><offsets xml_i="21210" xml_f="21211" txt_i="11821" txt_f="11822">2</offsets></xref><offsets xml_i="21218" xml_f="21219" txt_i="11822" txt_f="11823">.</offsets><fig id="Fig3"><label><offsets xml_i="21241" xml_f="21247" txt_i="11823" txt_f="11829">Fig. 3</offsets></label><caption><p><offsets xml_i="21267" xml_f="21675" txt_i="11829" txt_f="12237">Analysis of the intragraft transcriptional network related to the alternative immunosuppression scheme (group 2 patients) after 12 months of follow-up. Network genes were obtained by performing paired comparisons between the intragraft transcriptional profiles (t12 versus t0) of patients submitted to the alternative immunosuppression scheme and by subsequently inputting those genes in the GeneMania tool. </offsets><bold><offsets xml_i="21681" xml_f="21682" txt_i="12237" txt_f="12238">a</offsets></bold><offsets xml_i="21689" xml_f="21761" txt_i="12238" txt_f="12310"> KEGG categories showing enrichment in functions for the network nodes. </offsets><bold><offsets xml_i="21767" xml_f="21768" txt_i="12310" txt_f="12311">b</offsets></bold><offsets xml_i="21775" xml_f="21952" txt_i="12311" txt_f="12488"> Scatterplot of degree versus betweenness centralities for nodes obtained in the transcriptional network analysis. High-hubs, hubs, and bottlenecks are depicted respectively in </offsets><italic><offsets xml_i="21960" xml_f="21970" txt_i="12488" txt_f="12498">royal blue</offsets></italic><offsets xml_i="21979" xml_f="21981" txt_i="12498" txt_f="12500">, </offsets><italic><offsets xml_i="21989" xml_f="21992" txt_i="12500" txt_f="12503">red</offsets></italic><offsets xml_i="22001" xml_f="22007" txt_i="12503" txt_f="12509">, and </offsets><italic><offsets xml_i="22015" xml_f="22030" txt_i="12509" txt_f="12524">oil blue colors</offsets></italic><offsets xml_i="22039" xml_f="22041" txt_i="12524" txt_f="12526">. </offsets><bold><offsets xml_i="22047" xml_f="22048" txt_i="12526" txt_f="12527">c</offsets></bold><offsets xml_i="22055" xml_f="22230" txt_i="12527" txt_f="12702"> Transcriptional subnetwork containing the most central nodes in the group 2 network. Differentially expressed (DE) genes and DE-related genes are represented respectively as </offsets><italic><offsets xml_i="22238" xml_f="22251" txt_i="12702" txt_f="12715">gray diamonds</offsets></italic><offsets xml_i="22260" xml_f="22265" txt_i="12715" txt_f="12720"> and </offsets><italic><offsets xml_i="22273" xml_f="22285" txt_i="12720" txt_f="12732">blue circles</offsets></italic><offsets xml_i="22294" xml_f="22508" txt_i="12732" txt_f="12946"> in the network. Node sizes are based on betweenness centrality measurements. Genes previously associated with the keywords “autophagy,” “extracellular matrix,” “cell adhesion,” and “autoimmunity” are displayed in </offsets><italic><offsets xml_i="22516" xml_f="22527" txt_i="12946" txt_f="12957">green color</offsets></italic><offsets xml_i="22536" xml_f="22537" txt_i="12957" txt_f="12958">
</offsets></p></caption><graphic xlink:href="40246_2015_59_Fig3_HTML" id="MO3"></graphic></fig><fig id="Fig4"><label><offsets xml_i="22644" xml_f="22650" txt_i="12959" txt_f="12965">Fig. 4</offsets></label><caption><p><offsets xml_i="22670" xml_f="23072" txt_i="12965" txt_f="13367">Analysis of the intragraft transcriptional network related to the standard immunosuppression scheme (group 1 patients) after 12 months of follow-up. Network genes were obtained by performing paired comparisons between the intragraft transcriptional profiles (t12 versus t0) of patients submitted to the standard immunosuppression scheme and by subsequently inputting those genes in the GeneMania tool. </offsets><bold><offsets xml_i="23078" xml_f="23079" txt_i="13367" txt_f="13368">a</offsets></bold><offsets xml_i="23086" xml_f="23158" txt_i="13368" txt_f="13440"> KEGG categories showing enrichment in functions for the network nodes. </offsets><bold><offsets xml_i="23164" xml_f="23165" txt_i="13440" txt_f="13441">b</offsets></bold><offsets xml_i="23172" xml_f="23349" txt_i="13441" txt_f="13618"> Scatterplot of degree versus betweenness centralities for nodes obtained in the transcriptional network analysis. High-hubs, hubs, and bottlenecks are depicted respectively in </offsets><italic><offsets xml_i="23357" xml_f="23367" txt_i="13618" txt_f="13628">royal blue</offsets></italic><offsets xml_i="23376" xml_f="23378" txt_i="13628" txt_f="13630">, </offsets><italic><offsets xml_i="23386" xml_f="23389" txt_i="13630" txt_f="13633">red</offsets></italic><offsets xml_i="23398" xml_f="23404" txt_i="13633" txt_f="13639">, and </offsets><italic><offsets xml_i="23412" xml_f="23427" txt_i="13639" txt_f="13654">oil blue colors</offsets></italic><offsets xml_i="23436" xml_f="23438" txt_i="13654" txt_f="13656">. </offsets><bold><offsets xml_i="23444" xml_f="23445" txt_i="13656" txt_f="13657">c</offsets></bold><offsets xml_i="23452" xml_f="23627" txt_i="13657" txt_f="13832"> Transcriptional subnetwork containing the most central nodes in the group 1 network. Differentially expressed (DE) genes and DE-related genes are represented respectively as </offsets><italic><offsets xml_i="23635" xml_f="23648" txt_i="13832" txt_f="13845">gray diamonds</offsets></italic><offsets xml_i="23657" xml_f="23662" txt_i="13845" txt_f="13850"> and </offsets><italic><offsets xml_i="23670" xml_f="23682" txt_i="13850" txt_f="13862">blue circles</offsets></italic><offsets xml_i="23691" xml_f="23878" txt_i="13862" txt_f="14049"> in the network. Node sizes are based on betweenness centrality measurements. Genes previously associated with the keywords “endocytosis,” “fibrosis,” and “inflammation” are displayed in </offsets><italic><offsets xml_i="23886" xml_f="23897" txt_i="14049" txt_f="14060">beige color</offsets></italic><offsets xml_i="23906" xml_f="23907" txt_i="14060" txt_f="14061">
</offsets></p></caption><graphic xlink:href="40246_2015_59_Fig4_HTML" id="MO4"></graphic></fig><fig id="Fig5"><label><offsets xml_i="24014" xml_f="24020" txt_i="14062" txt_f="14068">Fig. 5</offsets></label><caption><p><offsets xml_i="24040" xml_f="24350" txt_i="14068" txt_f="14378">Analysis of the intragraft transcriptional network related to proximal tubular dysfunction (PTD) right after enrollment (t0). Network genes were obtained by comparing the intragraft transcriptional profiles of patients with elevated uRBP levels and by subsequently inputting those genes in the GeneMania tool. </offsets><bold><offsets xml_i="24356" xml_f="24357" txt_i="14378" txt_f="14379">a</offsets></bold><offsets xml_i="24364" xml_f="24436" txt_i="14379" txt_f="14451"> KEGG categories showing enrichment in functions for the network nodes. </offsets><bold><offsets xml_i="24442" xml_f="24443" txt_i="14451" txt_f="14452">b</offsets></bold><offsets xml_i="24450" xml_f="24627" txt_i="14452" txt_f="14629"> Scatterplot of degree versus betweenness centralities for nodes obtained in the transcriptional network analysis. High-hubs, hubs, and bottlenecks are depicted respectively in </offsets><italic><offsets xml_i="24635" xml_f="24645" txt_i="14629" txt_f="14639">royal blue</offsets></italic><offsets xml_i="24654" xml_f="24656" txt_i="14639" txt_f="14641">, </offsets><italic><offsets xml_i="24664" xml_f="24667" txt_i="14641" txt_f="14644">red</offsets></italic><offsets xml_i="24676" xml_f="24682" txt_i="14644" txt_f="14650">, and </offsets><italic><offsets xml_i="24690" xml_f="24705" txt_i="14650" txt_f="14665">oil blue colors</offsets></italic><offsets xml_i="24714" xml_f="24716" txt_i="14665" txt_f="14667">. </offsets><bold><offsets xml_i="24722" xml_f="24723" txt_i="14667" txt_f="14668">c</offsets></bold><offsets xml_i="24730" xml_f="24900" txt_i="14668" txt_f="14838"> Transcriptional subnetwork containing the most central nodes in the t0 network. Differentially expressed (DE) genes and DE-related genes are represented respectively as </offsets><italic><offsets xml_i="24908" xml_f="24921" txt_i="14838" txt_f="14851">gray diamonds</offsets></italic><offsets xml_i="24930" xml_f="24935" txt_i="14851" txt_f="14856"> and </offsets><italic><offsets xml_i="24943" xml_f="24955" txt_i="14856" txt_f="14868">blue circles</offsets></italic><offsets xml_i="24964" xml_f="25185" txt_i="14868" txt_f="15089"> in the network. Node sizes are based on betweenness centrality measurements. Genes previously associated with the keywords “immune response,” “T regulatory cells,” “autophagy,” or “ubiquitin-proteasome” are displayed in </offsets><italic><offsets xml_i="25193" xml_f="25205" txt_i="15089" txt_f="15101">orange color</offsets></italic><offsets xml_i="25214" xml_f="25215" txt_i="15101" txt_f="15102">
</offsets></p></caption><graphic xlink:href="40246_2015_59_Fig5_HTML" id="MO5"></graphic></fig><fig id="Fig6"><label><offsets xml_i="25322" xml_f="25328" txt_i="15103" txt_f="15109">Fig. 6</offsets></label><caption><p><offsets xml_i="25348" xml_f="25665" txt_i="15109" txt_f="15426">Analysis of the intragraft transcriptional network related to proximal tubular dysfunction (PTD) after 12 months of follow-up (t12). Network genes were obtained by comparing the intragraft transcriptional profiles of patients with elevated uRBP levels and by subsequently inputting those genes in the GeneMania tool. </offsets><bold><offsets xml_i="25671" xml_f="25672" txt_i="15426" txt_f="15427">a</offsets></bold><offsets xml_i="25679" xml_f="25751" txt_i="15427" txt_f="15499"> KEGG categories showing enrichment in functions for the network nodes. </offsets><bold><offsets xml_i="25757" xml_f="25758" txt_i="15499" txt_f="15500">b</offsets></bold><offsets xml_i="25765" xml_f="25942" txt_i="15500" txt_f="15677"> Scatterplot of degree versus betweenness centralities for nodes obtained in the transcriptional network analysis. High-hubs, hubs, and bottlenecks are depicted respectively in </offsets><italic><offsets xml_i="25950" xml_f="25960" txt_i="15677" txt_f="15687">royal blue</offsets></italic><offsets xml_i="25969" xml_f="25971" txt_i="15687" txt_f="15689">, </offsets><italic><offsets xml_i="25979" xml_f="25982" txt_i="15689" txt_f="15692">red</offsets></italic><offsets xml_i="25991" xml_f="25997" txt_i="15692" txt_f="15698">, and </offsets><italic><offsets xml_i="26005" xml_f="26020" txt_i="15698" txt_f="15713">oil blue colors</offsets></italic><offsets xml_i="26029" xml_f="26031" txt_i="15713" txt_f="15715">. </offsets><bold><offsets xml_i="26037" xml_f="26038" txt_i="15715" txt_f="15716">c</offsets></bold><offsets xml_i="26045" xml_f="26216" txt_i="15716" txt_f="15887"> Transcriptional subnetwork containing the most central nodes in the t12 network. Differentially expressed (DE) genes and DE-related genes are represented respectively as </offsets><italic><offsets xml_i="26224" xml_f="26237" txt_i="15887" txt_f="15900">gray diamonds</offsets></italic><offsets xml_i="26246" xml_f="26251" txt_i="15900" txt_f="15905"> and </offsets><italic><offsets xml_i="26259" xml_f="26271" txt_i="15905" txt_f="15917">blue circles</offsets></italic><offsets xml_i="26280" xml_f="26452" txt_i="15917" txt_f="16089"> in the network. Node sizes are based on betweenness centrality measurements. Genes previously associated with the keywords “fibrosis” and “tissue repair” are displayed in </offsets><italic><offsets xml_i="26460" xml_f="26472" txt_i="16089" txt_f="16101">purple color</offsets></italic><offsets xml_i="26481" xml_f="26482" txt_i="16101" txt_f="16102">
</offsets></p></caption><graphic xlink:href="40246_2015_59_Fig6_HTML" id="MO6"></graphic></fig></p><p><offsets xml_i="26574" xml_f="26708" txt_i="16104" txt_f="16238">Enriched biological functions were revealed among central nodes in the subnetworks. These biological functions are displayed in Figs. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="26740" xml_f="26742" txt_i="16238" txt_f="16240">3a</offsets></xref><offsets xml_i="26749" xml_f="26751" txt_i="16240" txt_f="16242">, </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="26783" xml_f="26785" txt_i="16242" txt_f="16244">4a</offsets></xref><offsets xml_i="26792" xml_f="26794" txt_i="16244" txt_f="16246">, </offsets><xref rid="Fig5" ref-type="fig"><offsets xml_i="26826" xml_f="26828" txt_i="16246" txt_f="16248">5a</offsets></xref><offsets xml_i="26835" xml_f="26841" txt_i="16248" txt_f="16254">, and </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="26873" xml_f="26875" txt_i="16254" txt_f="16256">6a</offsets></xref><offsets xml_i="26882" xml_f="26996" txt_i="16256" txt_f="16370">. Biological functions enriched by the genes from group 1 and group 2 comparisons are shown respectively in Figs. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="27028" xml_f="27029" txt_i="16370" txt_f="16371">3</offsets></xref><offsets xml_i="27036" xml_f="27041" txt_i="16371" txt_f="16376"> and </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="27073" xml_f="27074" txt_i="16376" txt_f="16377">4</offsets></xref><offsets xml_i="27081" xml_f="27568" txt_i="16377" txt_f="16864">. Genes derived from the group 1 comparison (group 1 network) were related to endocytosis, ubiquitin-mediated proteolysis, endoplasmic reticulum stress, TGF-β pathway, and adherens junctions. Conversely, genes from the group 2 comparison (group 2 network) were linked to phagosome, antigen processing and presentation, cell adhesion, autoimmunity, allograft rejection, and graft-versus-host disease. The subnetworks containing the most central nodes in each group are displayed in Figs. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="27600" xml_f="27602" txt_i="16864" txt_f="16866">3c</offsets></xref><offsets xml_i="27609" xml_f="27613" txt_i="16866" txt_f="16870"> to </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="27645" xml_f="27647" txt_i="16870" txt_f="16872">6c</offsets></xref><offsets xml_i="27654" xml_f="27785" txt_i="16872" txt_f="17003">. Genes associated with specific keywords in each case were marked in a different color, as described in the legend of each figure.</offsets></p></sec><sec id="Sec5"><title><offsets xml_i="27817" xml_f="27912" txt_i="17005" txt_f="17100">Analysis of the intragraft molecular changes associated with higher retinol-binding proteinuria</offsets></title><p><offsets xml_i="27923" xml_f="28120" txt_i="17101" txt_f="17298">We also associated the increased tubular proteinuria with a specific gene expression pattern, considering the differences in uRBP levels between the very high group (VHG) and high group (HG) (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="28152" xml_f="28154" txt_i="17298" txt_f="17300">1f</offsets></xref><offsets xml_i="28161" xml_f="28334" txt_i="17300" txt_f="17473">). In addition, a hierarchical clustering analysis on the DE gene subsets was conducted to verify if the samples in each group showed similar gene expression patterns (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="28366" xml_f="28368" txt_i="17473" txt_f="17475">2c</offsets></xref><offsets xml_i="28375" xml_f="28490" txt_i="17475" txt_f="17590">). With this approach, we confirmed that patients in the HG and VHG clustered together at t0 and t12, respectively.</offsets></p><p><offsets xml_i="28497" xml_f="28598" txt_i="17591" txt_f="17692">Biological functions overrepresented by network genes from t0 and t12 comparisons are shown in Figs. </offsets><xref rid="Fig5" ref-type="fig"><offsets xml_i="28630" xml_f="28631" txt_i="17692" txt_f="17693">5</offsets></xref><offsets xml_i="28638" xml_f="28643" txt_i="17693" txt_f="17698"> and </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="28675" xml_f="28676" txt_i="17698" txt_f="17699">6</offsets></xref><offsets xml_i="28683" xml_f="29068" txt_i="17699" txt_f="18084">. Genes derived from the former comparison (t0 network) are associated with osteoclast differentiation, protein digestion and absorption, and Toll-like, B cell, and T cell receptor pathways. On the other hand, genes derived from the latter comparison (t12 network) were linked to neuroactive ligand-receptor interaction, pathways in cancer, diabetes, and glycosaminoglycan degradation.</offsets></p></sec></sec><sec id="Sec6" sec-type="discussion"><title><offsets xml_i="29128" xml_f="29138" txt_i="18087" txt_f="18097">Discussion</offsets></title><sec id="Sec7"><title><offsets xml_i="29168" xml_f="29318" txt_i="18098" txt_f="18248">Patients submitted to the alternative immunosuppression scheme show better renal function outcomes and a survival-related intragraft molecular profile</offsets></title><p><offsets xml_i="29329" xml_f="29699" txt_i="18249" txt_f="18619">In the study cohort, patients that had their immunosuppression scheme altered by reducing the cyclosporine dosage and replacing AZA by MYF (group 2) improved serum creatinine and uRBP levels after 12 months of follow-up. Therefore, we hypothesized that distinct renal function outcomes could reflect intragraft molecular differences between patients from groups 1 and 2.</offsets></p><p><offsets xml_i="29706" xml_f="29735" txt_i="18620" txt_f="18649">In the group 1 network (Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="29767" xml_f="29768" txt_i="18649" txt_f="18650">3</offsets></xref><offsets xml_i="29775" xml_f="29981" txt_i="18650" txt_f="18856">), the most overrepresented function was endocytosis. Therefore, endocytosis disturbance may be involved in proteinuria, as tubular injury is linked to the inhibition of LMWP receptor-mediated endocytosis [</offsets><xref ref-type="bibr" rid="CR22"><offsets xml_i="30014" xml_f="30016" txt_i="18856" txt_f="18858">22</offsets></xref><offsets xml_i="30023" xml_f="30125" txt_i="18858" txt_f="18960">]. Moreover, tubular atrophy biomarkers are associated with endocytosis in renal transplant patients [</offsets><xref ref-type="bibr" rid="CR23"><offsets xml_i="30158" xml_f="30160" txt_i="18960" txt_f="18962">23</offsets></xref><offsets xml_i="30167" xml_f="30413" txt_i="18962" txt_f="19208">]. Besides endocytosis, genes in the group 1 network were related to ubiquitination and endoplasmic reticulum stress. Endoplasmic reticulum stress is activated by unfolded protein accumulation and induces fibrosis in proteinuric kidney diseases [</offsets><xref ref-type="bibr" rid="CR24"><offsets xml_i="30446" xml_f="30448" txt_i="19208" txt_f="19210">24</offsets></xref><offsets xml_i="30455" xml_f="30457" txt_i="19210" txt_f="19212">, </offsets><xref ref-type="bibr" rid="CR25"><offsets xml_i="30490" xml_f="30492" txt_i="19212" txt_f="19214">25</offsets></xref><offsets xml_i="30499" xml_f="30501" txt_i="19214" txt_f="19216">].</offsets></p><p><offsets xml_i="30508" xml_f="30537" txt_i="19217" txt_f="19246">In the group 2 network (Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="30569" xml_f="30570" txt_i="19246" txt_f="19247">4</offsets></xref><offsets xml_i="30577" xml_f="30816" txt_i="19247" txt_f="19486">), the most overrepresented function was phagosome, a type of vesicle formed during phagocytosis and autophagy. Interestingly, autophagy plays a role in tissue homeostasis, as damaged structures are degraded by autolysosomes after injury [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="30849" xml_f="30851" txt_i="19486" txt_f="19488">26</offsets></xref><offsets xml_i="30858" xml_f="30860" txt_i="19488" txt_f="19490">, </offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="30893" xml_f="30895" txt_i="19490" txt_f="19492">27</offsets></xref><offsets xml_i="30902" xml_f="31022" txt_i="19492" txt_f="19612">]. Moreover, the role of autophagy as a protective mechanism against ischemic injury was investigated in tubular cells [</offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="31055" xml_f="31057" txt_i="19612" txt_f="19614">28</offsets></xref><offsets xml_i="31064" xml_f="31133" txt_i="19614" txt_f="19683">] and autophagy activation by MYF was able to prolong cell survival [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="31166" xml_f="31168" txt_i="19683" txt_f="19685">29</offsets></xref><xref ref-type="bibr" rid="CR30"><offsets xml_i="31208" xml_f="31210" txt_i="19685" txt_f="19687">30</offsets></xref><offsets xml_i="31217" xml_f="31219" txt_i="19687" txt_f="19689">].</offsets></p><p><offsets xml_i="31226" xml_f="31443" txt_i="19690" txt_f="19907">Other overrepresented functions in the group 2 network were antigen presentation, autoimmune diseases, and allograft rejection. Two of these genes, Igfpb4 and Il10ra, were already associated with graft function loss [</offsets><xref ref-type="bibr" rid="CR31"><offsets xml_i="31476" xml_f="31478" txt_i="19907" txt_f="19909">31</offsets></xref><offsets xml_i="31485" xml_f="31601" txt_i="19909" txt_f="20025">]. Moreover, the up-regulated gene Klrb1 disclosed the existence of tolerance mechanisms related to FoxP3+ T cells [</offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="31634" xml_f="31636" txt_i="20025" txt_f="20027">32</offsets></xref><offsets xml_i="31643" xml_f="31856" txt_i="20027" txt_f="20240">]. Genes coding for class I and II major histocompatibility molecules (MHC) were also found in the group 2 network, highlighting the role of HLA-G (MHC class I) for protecting renal transplants from autoimmunity [</offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="31889" xml_f="31891" txt_i="20240" txt_f="20242">33</offsets></xref><offsets xml_i="31898" xml_f="31974" txt_i="20242" txt_f="20318">] and the function of Hla-dpb1 (MHC class II) in renal allograft rejection [</offsets><xref ref-type="bibr" rid="CR34"><offsets xml_i="32007" xml_f="32009" txt_i="20318" txt_f="20320">34</offsets></xref><offsets xml_i="32016" xml_f="32018" txt_i="20320" txt_f="20322">].</offsets></p><p><offsets xml_i="32025" xml_f="32260" txt_i="20323" txt_f="20558">Finally, cell adhesion and extracellular matrix functions were overrepresented by genes in the group 2 network. Serping1 and Mmp12, for example, were already associated with parenchyma deterioration in allografts undergoing rejection [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="32293" xml_f="32295" txt_i="20558" txt_f="20560">35</offsets></xref><offsets xml_i="32302" xml_f="32304" txt_i="20560" txt_f="20562">, </offsets><xref ref-type="bibr" rid="CR36"><offsets xml_i="32337" xml_f="32339" txt_i="20562" txt_f="20564">36</offsets></xref><offsets xml_i="32346" xml_f="32587" txt_i="20564" txt_f="20805">]. Therefore, the alternative immunosuppression regimen may exert part of its protective effects by regulating the extracellular matrix, considering that MYF reduces fibroblast infiltration, collagen deposition, and ECM synthesis in kidney [</offsets><xref ref-type="bibr" rid="CR37"><offsets xml_i="32620" xml_f="32622" txt_i="20805" txt_f="20807">37</offsets></xref><offsets xml_i="32629" xml_f="32631" txt_i="20807" txt_f="20809">].</offsets></p></sec><sec id="Sec8"><title><offsets xml_i="32663" xml_f="32796" txt_i="20811" txt_f="20944">Patients with higher retinol-binding proteinuria have intragraft transcriptional changes associated with injury and repair mechanisms</offsets></title><p><offsets xml_i="32807" xml_f="33139" txt_i="20945" txt_f="21277">Increased uRBP levels were significantly correlated with lower creatinine clearance in the cohort, corroborating the relevance of uRBP levels for assessing the extent of glomerular dysfunction after renal transplantation. Thus, we conducted a network analysis to identify molecular events correlated with higher tubular proteinuria.</offsets></p><p><offsets xml_i="33146" xml_f="33205" txt_i="21278" txt_f="21337">Intragraft transcriptional changes in the t0 network (Fig. </offsets><xref rid="Fig5" ref-type="fig"><offsets xml_i="33237" xml_f="33238" txt_i="21337" txt_f="21338">5</offsets></xref><offsets xml_i="33245" xml_f="33454" txt_i="21338" txt_f="21547">) reflected molecular mechanisms of immune-mediated allograft injury. Indeed, intragraft gene expression changes during transplant rejection are associated with immune response and precede the onset of IF/TA [</offsets><xref ref-type="bibr" rid="CR38"><offsets xml_i="33487" xml_f="33489" txt_i="21547" txt_f="21549">38</offsets></xref><offsets xml_i="33496" xml_f="33498" txt_i="21549" txt_f="21551">, </offsets><xref ref-type="bibr" rid="CR39"><offsets xml_i="33531" xml_f="33533" txt_i="21551" txt_f="21553">39</offsets></xref><offsets xml_i="33540" xml_f="33722" txt_i="21553" txt_f="21735">]. In parallel, a gene subset in the t0 network was associated with the Foxp3+ T regulatory cell phenotype. The high-hub Satb1, for example, inhibits the transcription factor Foxp3 [</offsets><xref ref-type="bibr" rid="CR40"><offsets xml_i="33755" xml_f="33757" txt_i="21735" txt_f="21737">40</offsets></xref><offsets xml_i="33764" xml_f="33878" txt_i="21737" txt_f="21851">], and Srrm1, the most up-regulated gene in the t0 comparison (fold change = 23.9), is part of the Foxp3 network [</offsets><xref ref-type="bibr" rid="CR41"><offsets xml_i="33911" xml_f="33913" txt_i="21851" txt_f="21853">41</offsets></xref><offsets xml_i="33920" xml_f="33922" txt_i="21853" txt_f="21855">].</offsets></p><p><offsets xml_i="33929" xml_f="34148" txt_i="21856" txt_f="22075">Central nodes in the t0 network such as Trappc8, Agrp, Syk, and Baz2b were also linked to autophagy and ubiquitin-proteasome functions. As a matter of fact, autophagy components interplay with ubiquitination processes [</offsets><xref ref-type="bibr" rid="CR42"><offsets xml_i="34181" xml_f="34183" txt_i="22075" txt_f="22077">42</offsets></xref><offsets xml_i="34190" xml_f="34311" txt_i="22077" txt_f="22198">], and autophagy exerts protective functions by inhibiting endoplasmic reticulum stress and ubiquitin-proteasome system [</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="34344" xml_f="34346" txt_i="22198" txt_f="22200">43</offsets></xref><offsets xml_i="34353" xml_f="34462" txt_i="22200" txt_f="22309">]. Hence, the imbalance of these two protein degradation systems may be involved in kidney graft dysfunction.</offsets></p><p><offsets xml_i="34469" xml_f="34539" txt_i="22310" txt_f="22380">In the t12 network, central nodes were associated with fibrosis (Fig. </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="34571" xml_f="34572" txt_i="22380" txt_f="22381">6</offsets></xref><offsets xml_i="34579" xml_f="34612" txt_i="22381" txt_f="22414">), like Irf3, Myod1, and Crybb3 [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="34645" xml_f="34647" txt_i="22414" txt_f="22416">44</offsets></xref><offsets xml_i="34654" xml_f="34655" txt_i="22416" txt_f="22417">–</offsets><xref ref-type="bibr" rid="CR46"><offsets xml_i="34688" xml_f="34690" txt_i="22417" txt_f="22419">46</offsets></xref><offsets xml_i="34697" xml_f="35026" txt_i="22419" txt_f="22748">]. Moreover, genes encoding actin and myosin, biomarkers of fibrosis, were highly up-regulated in the VHG (fold change around 300). Notably, the most overrepresented function in the t12 network was “neuroactive ligand-receptor interaction”, highlighting the role of neuronal genes for the response against ischemic renal injury [</offsets><xref ref-type="bibr" rid="CR47"><offsets xml_i="35059" xml_f="35061" txt_i="22748" txt_f="22750">47</offsets></xref><offsets xml_i="35068" xml_f="35131" txt_i="22750" txt_f="22813">] and maintenance of an epithelial phenotype in tubular cells [</offsets><xref ref-type="bibr" rid="CR48"><offsets xml_i="35164" xml_f="35166" txt_i="22813" txt_f="22815">48</offsets></xref><offsets xml_i="35173" xml_f="35175" txt_i="22815" txt_f="22817">].</offsets></p></sec></sec><sec id="Sec9" sec-type="conclusion"><title><offsets xml_i="35235" xml_f="35246" txt_i="22820" txt_f="22831">Conclusions</offsets></title><p><offsets xml_i="35257" xml_f="35914" txt_i="22832" txt_f="23489">This study showed that increased retinol-binding proteinuria is associated with kidney function worsening in renal transplant patients. Patients under the alternative immunosuppressive scheme (group 2) improved renal function after 12 months of follow-up. The examination of their intragraft transcriptional profiles revealed mechanisms linked to graft injury and recovery. These results can be useful for further studying the mechanisms underlying graft injury and functional recovery. In particular, the role of genes participating in ubiquitination, autophagy, protein overload, and neuroactive ligand-receptor interaction should be further investigated.</offsets></p></sec><sec id="Sec10" sec-type="materials|methods"><title><offsets xml_i="35976" xml_f="35983" txt_i="23491" txt_f="23498">Methods</offsets></title><sec id="Sec11"><title><offsets xml_i="36014" xml_f="36022" txt_i="23499" txt_f="23507">Patients</offsets></title><p><offsets xml_i="36033" xml_f="36434" txt_i="23508" txt_f="23906">We studied a cohort of 33 subjects who underwent renal transplantation in the Kidney Hospital at the Federal University of São Paulo. Patients with at least 1 year of transplant surgery were checked for uRBP levels during a 3-month follow-up. Patients with uRBP &gt;0.4 mg/L in three consecutive monthly measurements were diagnosed with PTD, according to studies establishing this upper normal limit [</offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="36466" xml_f="36467" txt_i="23906" txt_f="23907">2</offsets></xref><offsets xml_i="36474" xml_f="36476" txt_i="23907" txt_f="23909">, </offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="36509" xml_f="36511" txt_i="23909" txt_f="23911">12</offsets></xref><offsets xml_i="36518" xml_f="36646" txt_i="23911" txt_f="24039">], and sequentially included in this study. The study was approved by the Research Ethics Committee at UNIFESP (number 0420/05).</offsets></p></sec><sec id="Sec12"><title><offsets xml_i="36679" xml_f="36700" txt_i="24041" txt_f="24062">Clinical study design</offsets></title><p><offsets xml_i="36711" xml_f="36889" txt_i="24063" txt_f="24241">All patients enrolled were initially (t0) in use of CsA, prednisone, and AZA. They were randomized at enrollment for a prospective treatment with the standard protocol (group 1, </offsets><italic><offsets xml_i="36897" xml_f="36898" txt_i="24241" txt_f="24242">n</offsets></italic><offsets xml_i="36907" xml_f="37001" txt_i="24242" txt_f="24336"> = 15) or with an alternative protocol with reduced CsA dosage and MYF introduction (group 2, </offsets><italic><offsets xml_i="37009" xml_f="37010" txt_i="24336" txt_f="24337">n</offsets></italic><offsets xml_i="37019" xml_f="37322" txt_i="24337" txt_f="24640"> = 18). Patients were followed up for 12 months (t12) to determine if modifying their immunosuppression regimens could result in better renal function outcomes. All patients were submitted to graft biopsy at t0 and t12, and each biopsy was individually assessed in the transcriptomic profiling analysis.</offsets></p></sec><sec id="Sec13"><title><offsets xml_i="37355" xml_f="37383" txt_i="24642" txt_f="24670">Kidney histological analysis</offsets></title><p><offsets xml_i="37394" xml_f="37846" txt_i="24671" txt_f="25123">Histological findings were evaluated by a pathologist using optical microscopy. Biopsies were interpreted following the Banff classification for kidney allograft pathology: each biopsy was categorized according to classification grades of interstitial fibrosis and tubular atrophy (IF/TA), which establishes the degree (I, II, and III) of interstitial fibrosis (mild, moderate, severe) and tubular atrophy (mild to moderate) for each evaluated sample [</offsets><xref ref-type="bibr" rid="CR49"><offsets xml_i="37879" xml_f="37881" txt_i="25123" txt_f="25125">49</offsets></xref><offsets xml_i="37888" xml_f="37961" txt_i="25125" txt_f="25198">]. Histological differences between the groups were determined using the </offsets><italic><offsets xml_i="37969" xml_f="37970" txt_i="25198" txt_f="25199">χ</offsets></italic><sup><offsets xml_i="37984" xml_f="37985" txt_i="25199" txt_f="25200">2</offsets></sup><offsets xml_i="37991" xml_f="38027" txt_i="25200" txt_f="25236"> test for the categorical variables.</offsets></p></sec><sec id="Sec14"><title><offsets xml_i="38060" xml_f="38085" txt_i="25238" txt_f="25263">Renal function evaluation</offsets></title><p><offsets xml_i="38096" xml_f="38313" txt_i="25264" txt_f="25481">Serum creatinine, creatinine clearance, and uRBP levels were measured in patients at enrollment and at the end of the follow-up period. Serum creatinine and uRBP levels were measured according to described elsewhere [</offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="38346" xml_f="38348" txt_i="25481" txt_f="25483">10</offsets></xref><offsets xml_i="38355" xml_f="38971" txt_i="25483" txt_f="26099">]. The glomerular function rate (eGFR) estimation was based on serum creatinine concentration and calculated using the Cockcroft-Gault and the modification of diet in renal disease (MDRD) formulas. The prediction of creatinine clearance by the Cockcroft-Gault equation was calculated as (140 − age) × body weight/plasma creatinine × 72 (×0.85 if female), and the MDRD estimate was determined as 175 × plasma creatinine − 1.154 × age − 0.203 (×0.742 if female; ×1.21 if black). Serum creatinine levels are represented in mg/dL, creatinine clearance in mL/min, and uRBP levels in mg/L. Statistical analysis (Student’s </offsets><italic><offsets xml_i="38979" xml_f="38980" txt_i="26099" txt_f="26100">t</offsets></italic><offsets xml_i="38989" xml_f="39002" txt_i="26100" txt_f="26113"> tests, with </offsets><italic><offsets xml_i="39010" xml_f="39011" txt_i="26113" txt_f="26114">p</offsets></italic><offsets xml_i="39020" xml_f="39069" txt_i="26114" txt_f="26160"> &lt; 0.05) was performed using GraphPad Prism 5.</offsets></p></sec><sec id="Sec15"><title><offsets xml_i="39102" xml_f="39165" txt_i="26162" txt_f="26225">Comparative analysis of the intragraft gene expression profiles</offsets></title><p><offsets xml_i="39176" xml_f="39948" txt_i="26226" txt_f="26998">To investigate transcriptional changes related to specific immunosuppressive regimens, we performed paired comparisons (t12 versus t0) between biopsies from the same patients in group 1 (group 1 comparison) and in group 2 (group 2 comparison). A second analysis was performed to determine transcriptional changes associated with increased retinol-binding proteinuria. Patients were divided into subgroups: patients with uRBP values above 1.0 mg/L were classified in the very high group (VHG), whereas patients with uRBP levels above 0.4 mg/L but less than 1.0 mg/L were classified in the high group (HG). The uRBP cutoff level of 1.0 mg/L was selected because renal transplant patients with uRBP levels ≥1 mg/L are at a much higher risk for kidney function deterioration [</offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="39981" xml_f="39983" txt_i="26998" txt_f="27000">10</offsets></xref><offsets xml_i="39990" xml_f="40167" txt_i="27000" txt_f="27177">]. Moreover, the average uRBP levels in transplant patients with an estimated GFR less than 60 mL/min (i.e., with moderate to severe renal damage) were also higher than 1 mg/L [</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="40200" xml_f="40202" txt_i="27177" txt_f="27179">12</offsets></xref><offsets xml_i="40209" xml_f="40363" txt_i="27179" txt_f="27333">]. The comparative analyses were done between the transcriptional profiles of patients from the VHG and HG at t0 (t0 comparison) and t12 (t12 comparison).</offsets></p></sec><sec id="Sec16"><title><offsets xml_i="40396" xml_f="40434" txt_i="27335" txt_f="27373">Oligonucleotide microarray experiments</offsets></title><p><offsets xml_i="40445" xml_f="40508" txt_i="27374" txt_f="27437">Microarray experiments were performed as previously described [</offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="40541" xml_f="40543" txt_i="27437" txt_f="27439">50</offsets></xref><offsets xml_i="40550" xml_f="40776" txt_i="27439" txt_f="27665">]. Only samples with a RNA integrity number greater than 7 were employed in microarray experiments. Gene expression profiles were assessed using Agilent whole human genome 4x44K oligonucleotide microarrays. The R environment (</offsets><ext-link ext-link-type="uri" xlink:href="http://www.r-project.org"><offsets xml_i="40844" xml_f="40868" txt_i="27665" txt_f="27689">http://www.r-project.org</offsets></ext-link><offsets xml_i="40879" xml_f="41123" txt_i="27689" txt_f="27933">) was used to filter and pre-process the data. The mean of the probes for each gene was calculated, and the signal intensities were log2 transformed. These logarithmic values were input in the TmeV software to perform the statistical analyses [</offsets><xref ref-type="bibr" rid="CR51"><offsets xml_i="41156" xml_f="41158" txt_i="27933" txt_f="27935">51</offsets></xref><offsets xml_i="41165" xml_f="41439" txt_i="27935" txt_f="28209">]. No further normalization step was performed after visual inspection of the data distribution across the samples. The complete data set is available at the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus, through the accession number GSE48250.</offsets></p></sec><sec id="Sec17"><title><offsets xml_i="41472" xml_f="41495" txt_i="28211" txt_f="28234">Bioinformatics workflow</offsets></title><p><offsets xml_i="41506" xml_f="41624" txt_i="28235" txt_f="28353">The analyses described below were performed to obtain further information on the comparative gene expression profiles.</offsets><list list-type="simple"><list-item><label><offsets xml_i="41667" xml_f="41669" txt_i="28353" txt_f="28355">i)</offsets></label><p><italic><offsets xml_i="41688" xml_f="41708" txt_i="28355" txt_f="28375">Statistical testing.</offsets></italic><offsets xml_i="41717" xml_f="41845" txt_i="28375" txt_f="28503"> Differentially expressed (DE) genes were found by paired (group 1 and group 2 comparisons) and unpaired (VHG × HG comparisons) </offsets><italic><offsets xml_i="41853" xml_f="41854" txt_i="28503" txt_f="28504">t</offsets></italic><offsets xml_i="41863" xml_f="41876" txt_i="28504" txt_f="28517"> tests, with </offsets><italic><offsets xml_i="41884" xml_f="41885" txt_i="28517" txt_f="28518">p</offsets></italic><offsets xml_i="41894" xml_f="42011" txt_i="28518" txt_f="28632"> &lt; 0.05 as the significance threshold. The false discovery rate (FDR) control was initially applied to adjust the </offsets><italic><offsets xml_i="42019" xml_f="42020" txt_i="28632" txt_f="28633">p</offsets></italic><offsets xml_i="42029" xml_f="42425" txt_i="28633" txt_f="29029"> values for multiple comparisons, but only few significant genes were obtained using this procedure. Therefore, a less stringent approach was used by analyzing the data with no FDR adjustment, with the understanding that false positives were not restrained a priori. In addition, the FDR corrections were not used in order to obtain a relevant number of genes for the network enrichment analysis.</offsets></p></list-item><list-item><label><offsets xml_i="42459" xml_f="42462" txt_i="29030" txt_f="29033">ii)</offsets></label><p><italic><offsets xml_i="42481" xml_f="42514" txt_i="29033" txt_f="29066">Hierarchical clustering analysis.</offsets></italic><offsets xml_i="42523" xml_f="42564" txt_i="29066" txt_f="29107"> Hierarchical clustering (HCL) analysis [</offsets><xref ref-type="bibr" rid="CR52"><offsets xml_i="42597" xml_f="42599" txt_i="29107" txt_f="29109">52</offsets></xref><offsets xml_i="42606" xml_f="42971" txt_i="29109" txt_f="29474">] was performed using TmeV. HCL was used to group DE genes based on their expression similarities across the samples. The average distance clustering method was employed, using sample tree selection and sample leaf order optimization. The distance metric used was the Pearson correlation, and HCL was performed only in the significant genes to reduce cluster noise.</offsets></p></list-item><list-item><label><offsets xml_i="43005" xml_f="43009" txt_i="29475" txt_f="29479">iii)</offsets></label><p><italic><offsets xml_i="43028" xml_f="43064" txt_i="29479" txt_f="29515">Gene co-expression network analysis.</offsets></italic><offsets xml_i="43073" xml_f="43203" txt_i="29515" txt_f="29645"> The differential transcriptomic datasets were used to generate the gene co-expression networks. The Cytoscape plug-in GeneMANIA [</offsets><xref ref-type="bibr" rid="CR53"><offsets xml_i="43236" xml_f="43238" txt_i="29645" txt_f="29647">53</offsets></xref><offsets xml_i="43245" xml_f="43728" txt_i="29647" txt_f="30130">] was used to predict DE gene interactions. Networks were generated using information from the co-expression category in GeneMANIA. Genes that co-express with DE genes (DE-related genes) were also included in the networks to study the interactions between DE genes and other co-expressed genes. To analyze the centrality of the nodes (genes) contained in the networks, the node centrality parameters “degree” and “betweenness” were calculated using the Cytoscape plug-in CentiScaPe [</offsets><xref ref-type="bibr" rid="CR54"><offsets xml_i="43761" xml_f="43763" txt_i="30130" txt_f="30132">54</offsets></xref><offsets xml_i="43770" xml_f="44012" txt_i="30132" txt_f="30374">]. Node degree is a local structure measure in networks that determines the number of edges linked to a node. Conversely, betweenness centrality is a global structure measure that defines the fraction of shortest paths passing through a node.</offsets></p></list-item><list-item><label><offsets xml_i="44046" xml_f="44049" txt_i="30375" txt_f="30378">iv)</offsets></label><p><italic><offsets xml_i="44068" xml_f="44119" txt_i="30378" txt_f="30429">Identification of high-hubs, hubs, and bottlenecks.</offsets></italic><offsets xml_i="44128" xml_f="44760" txt_i="30429" txt_f="31061"> Scatterplots were constructed using degree and betweenness values for each node in GraphPad Prism®5. These scatterplots allowed node hierarchy categorization in high-hubs, hubs, and bottlenecks. This categorization takes in account gene localization in different quadrants of the graph. High-hubs are placed in the up-right quadrant due to their higher degree and betweenness values. Conversely, hubs are located in the down-right quadrant, as they present high degree but lower betweenness values compared to high-hubs. Finally, bottlenecks are located in the up-left quadrant, as they show high betweenness but low degree values.</offsets></p></list-item><list-item><label><offsets xml_i="44794" xml_f="44796" txt_i="31062" txt_f="31064">v)</offsets></label><p><italic><offsets xml_i="44815" xml_f="44845" txt_i="31064" txt_f="31094">Functional enrichment analysis</offsets></italic><offsets xml_i="44854" xml_f="45047" txt_i="31094" txt_f="31287">. Overrepresented biological functions were searched in the differential transcriptomic datasets using FunNet, a bioinformatics tool that performs functional profiling of gene expression data [</offsets><xref ref-type="bibr" rid="CR55"><offsets xml_i="45080" xml_f="45082" txt_i="31287" txt_f="31289">55</offsets></xref><offsets xml_i="45089" xml_f="45091" txt_i="31289" txt_f="31291">].</offsets></p></list-item><list-item><label><offsets xml_i="45125" xml_f="45128" txt_i="31292" txt_f="31295">vi)</offsets></label><p><italic><offsets xml_i="45147" xml_f="45198" txt_i="31295" txt_f="31346">Analysis of subnetworks derived from central nodes.</offsets></italic><offsets xml_i="45207" xml_f="45258" txt_i="31346" txt_f="31397"> Subnetworks were built with the aid of Cytoscape [</offsets><xref ref-type="bibr" rid="CR56"><offsets xml_i="45291" xml_f="45293" txt_i="31397" txt_f="31399">56</offsets></xref><offsets xml_i="45300" xml_f="45459" txt_i="31399" txt_f="31558">], using the central nodes identified in each comparison. Semantic relationships were identified between genes and keywords with the text mining tool GenClip [</offsets><xref ref-type="bibr" rid="CR57"><offsets xml_i="45492" xml_f="45494" txt_i="31558" txt_f="31560">57</offsets></xref><offsets xml_i="45501" xml_f="45791" txt_i="31560" txt_f="31850">]. We searched for relationships using the keywords “immune response”, “T regulatory cells”, “autophagy”, “ubiquitin-proteasome”, “endocytosis”, “fibrosis”, “inflammation”, “extracellular matrix”, “cell adhesion”, and “autoimmunity”. These relationships were highlighted in each subnetwork.</offsets></p></list-item></list></p></sec></sec></body><back><app-group><app id="App1"><sec id="Sec18"><title>Additional files</title><p><media position="anchor" xlink:href="40246_2015_59_MOESM1_ESM.pdf" id="MOESM1"><label>Additional file 1:</label><caption><p>
<bold>Differentially expressed genes obtained in each comparison.</bold> This table contains the differentially expressed (DE) genes obtained in each comparison and their fold changes. Statistical analysis using unpaired and paired <italic>t</italic> tests identified 250, 434, 417, and 593 DE genes according to the respective comparisons: t0 (very high versus high uRBP levels), t12 (very high versus high uRBP levels), group 1 (t12 × t0), and group 2 (t12 × t0).</p></caption></media><media position="anchor" xlink:href="40246_2015_59_MOESM2_ESM.pdf" id="MOESM2"><label>Additional file 2:</label><caption><p>
<bold>High-hub, hub, and bottleneck genes identified in each comparison.</bold> Hubs were defined as highly connected nodes according to node degree values. High-hubs are top-ranked hubs presenting also high betweenness centrality values. Bottleneck genes were classified as nodes with high betweenness centrality but low node degree values.</p></caption></media></p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>AZA</term><def><p>azathioprine</p></def></def-item><def-item><term>CsA</term><def><p>cyclosporine</p></def></def-item><def-item><term>DE</term><def><p>differentially expressed</p></def></def-item><def-item><term>eGFR</term><def><p>estimated glomerular function rate</p></def></def-item><def-item><term>HCL</term><def><p>hierarchical clustering</p></def></def-item><def-item><term>HG</term><def><p>high group</p></def></def-item><def-item><term>IF/TA</term><def><p>interstitial fibrosis and tubular atrophy</p></def></def-item><def-item><term>LMWP</term><def><p>low-molecular-weight proteins</p></def></def-item><def-item><term>MDRD</term><def><p>modification of diet in renal diseases</p></def></def-item><def-item><term>MYF</term><def><p>mycophenolate mofetil</p></def></def-item><def-item><term>PTD</term><def><p>proximal tubular dysfunction</p></def></def-item><def-item><term>uRBP</term><def><p>urinary retinol-binding protein</p></def></def-item><def-item><term>VHG</term><def><p>very high group</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Competing interests</bold></p><p>The authors declare that they have no competing interests.</p></fn><fn><p><bold>Authors’ contributions</bold></p><p>PGR, RC, NOSC, and APS designed the study. RC, PGR, ACCM, and APS performed the clinical procedures. EN, MAC, PGR, and HA performed the RNA extraction and microarray experiments. HA, PGR, RC, CAMF, NOSC, and APS analyzed the data. HA, CAMF, NOSC, and APS drafted the manuscript. All authors read and approved the final manuscript.</p></fn></fn-group><ack><title>Acknowledgements</title><p>We would like to thank the following funding agencies for supporting this study: Fundação de Amparo à Pesquisa do Estado de São Paulo-FAPESP (grants 2009/53443-1, 2011/50761-2, 2012/02270-2) and Conselho Nacional de Desenvolvimento Cientifico e Tecnológico-CNPq (grants 307626/2014-8 and INCT Complex Fluids) and NAP e-Science USP.</p></ack><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Pinsky</surname><given-names>BW</given-names></name><name><surname>Lentine</surname><given-names>KL</given-names></name><name><surname>Ercole</surname><given-names>PR</given-names></name><name><surname>Salvalaggio</surname><given-names>PR</given-names></name><name><surname>Burroughs</surname><given-names>TE</given-names></name><name><surname>Schnitzler</surname><given-names>MA</given-names></name></person-group><article-title>Predicting long-term graft survival in adult kidney transplant recipients</article-title><source>Saudi J Kidney Dis Transpl.</source><year>2012</year><volume>23</volume><fpage>693</fpage><lpage>700</lpage><pub-id pub-id-type="doi">10.4103/1319-2442.98112</pub-id><pub-id pub-id-type="pmid">22805379</pub-id></element-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Camara</surname><given-names>NO</given-names></name><name><surname>Silva</surname><given-names>MS</given-names></name><name><surname>Nishida</surname><given-names>S</given-names></name><name><surname>Pereira</surname><given-names>AB</given-names></name><name><surname>Pacheco-Silva</surname><given-names>A</given-names></name></person-group><article-title>Proximal tubular dysfunction is associated with chronic allograft nephropathy and decreased long-term renal-graft survival</article-title><source>Transplantation</source><year>2004</year><volume>78</volume><fpage>269</fpage><lpage>275</lpage><pub-id pub-id-type="doi">10.1097/01.TP.0000128333.46949.A4</pub-id><pub-id pub-id-type="pmid">15280689</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Nauta</surname><given-names>FL</given-names></name><name><surname>Bakker</surname><given-names>SJ</given-names></name><name><surname>van Oeveren</surname><given-names>W</given-names></name><name><surname>Navis</surname><given-names>G</given-names></name><name><surname>van der Heide</surname><given-names>JJ</given-names></name><name><surname>van Goor</surname><given-names>H</given-names></name><etal></etal></person-group><article-title>Albuminuria, proteinuria, and novel urine biomarkers as predictors of long-term allograft outcomes in kidney transplant recipients</article-title><source>Am J Kidney Dis.</source><year>2011</year><volume>57</volume><fpage>733</fpage><lpage>43</lpage><pub-id pub-id-type="doi">10.1053/j.ajkd.2010.12.022</pub-id><pub-id pub-id-type="pmid">21458900</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Woitas</surname><given-names>RP</given-names></name><name><surname>Stoffel-Wagner</surname><given-names>B</given-names></name><name><surname>Poege</surname><given-names>U</given-names></name><name><surname>Schiedermaier</surname><given-names>P</given-names></name><name><surname>Spengler</surname><given-names>U</given-names></name><name><surname>Sauerbruch</surname><given-names>T</given-names></name></person-group><article-title>Low-molecular weight proteins as markers for glomerular filtration rate</article-title><source>Clin Chem.</source><year>2001</year><volume>47</volume><fpage>2179</fpage><lpage>80</lpage><pub-id pub-id-type="pmid">11719489</pub-id></element-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>van den Brand</surname><given-names>JA</given-names></name><name><surname>Hofstra</surname><given-names>JM</given-names></name><name><surname>Wetzels</surname><given-names>JF</given-names></name></person-group><article-title>Low-molecular-weight proteins as prognostic markers in idiopathic membranous nephropathy</article-title><source>Clin J Am Soc Nephrol</source><year>2011</year><volume>6</volume><fpage>2846</fpage><lpage>2853</lpage><pub-id pub-id-type="doi">10.2215/CJN.04020411</pub-id><pub-id pub-id-type="pmid">22157712</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ball</surname><given-names>ST</given-names></name><name><surname>Lapsley</surname><given-names>M</given-names></name><name><surname>Norden</surname><given-names>AG</given-names></name><name><surname>Cairns</surname><given-names>TD</given-names></name><name><surname>Palmer</surname><given-names>AB</given-names></name><name><surname>Taube</surname><given-names>DH</given-names></name></person-group><article-title>Urinary retinol binding protein in Indo-Asian patients with idiopathic interstitial nephritis</article-title><source>QJM.</source><year>2003</year><volume>96</volume><fpage>363</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1093/qjmed/hcg052</pub-id><pub-id pub-id-type="pmid">12702785</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Jaconi</surname><given-names>S</given-names></name><name><surname>Saurat</surname><given-names>JH</given-names></name><name><surname>Siegenthaler</surname><given-names>G</given-names></name></person-group><article-title>Analysis of normal and truncated holo- and apo-retinol-binding protein (RBP) in human serum: altered ratios in chronic renal failure</article-title><source>Eur J Endocrinol</source><year>1996</year><volume>134</volume><fpage>576</fpage><lpage>582</lpage><pub-id pub-id-type="doi">10.1530/eje.0.1340576</pub-id><pub-id pub-id-type="pmid">8664978</pub-id></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Schaub</surname><given-names>S</given-names></name><name><surname>Mayr</surname><given-names>M</given-names></name><name><surname>Hönger</surname><given-names>G</given-names></name><name><surname>Bestland</surname><given-names>J</given-names></name><name><surname>Steiger</surname><given-names>J</given-names></name><name><surname>Regeniter</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Detection of subclinical tubular injury after renal transplantation: comparison of urine protein analysis with allograft histopathology</article-title><source>Transplantation.</source><year>2007</year><volume>84</volume><fpage>104</fpage><lpage>12</lpage><pub-id pub-id-type="doi">10.1097/01.tp.0000268808.39401.e8</pub-id><pub-id pub-id-type="pmid">17627245</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Pallet</surname><given-names>N</given-names></name><name><surname>Chauvet</surname><given-names>S</given-names></name><name><surname>Chassé</surname><given-names>JF</given-names></name><name><surname>Vincent</surname><given-names>M</given-names></name><name><surname>Avillach</surname><given-names>P</given-names></name><name><surname>Levi</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>Urinary retinol binding protein is a marker of the extent of interstitial kidney fibrosis</article-title><source>PLoS One.</source><year>2014</year><volume>9</volume><pub-id pub-id-type="doi">10.1371/journal.pone.0084708</pub-id><pub-id pub-id-type="pmid">24416268</pub-id></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>de Matos</surname><given-names>AC</given-names></name><name><surname>Câmara</surname><given-names>NO</given-names></name><name><surname>de Oliveira</surname><given-names>AF</given-names></name><name><surname>Franco</surname><given-names>MF</given-names></name><name><surname>Moura</surname><given-names>LA</given-names></name><name><surname>Nishida</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Functional and morphologic evaluation of kidney proximal tubuli and correlation with renal allograft prognosis</article-title><source>Transpl Int.</source><year>2010</year><volume>23</volume><fpage>493</fpage><lpage>9</lpage><pub-id pub-id-type="doi">10.1111/j.1432-2277.2009.01005.x</pub-id><pub-id pub-id-type="pmid">19929858</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Amer</surname><given-names>H</given-names></name><name><surname>Lieske</surname><given-names>JC</given-names></name><name><surname>Rule</surname><given-names>AD</given-names></name><name><surname>Kremers</surname><given-names>WK</given-names></name><name><surname>Larson</surname><given-names>TS</given-names></name><name><surname>Franco Palacios</surname><given-names>CR</given-names></name><etal></etal></person-group><article-title>Urine high and low molecular weight proteins one-year post-kidney transplant: relationship to histology and graft survival</article-title><source>Am J Transplant.</source><year>2013</year><volume>13</volume><fpage>676</fpage><lpage>84</lpage><pub-id pub-id-type="doi">10.1111/ajt.12044</pub-id><pub-id pub-id-type="pmid">23414180</pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chinen</surname><given-names>R</given-names></name><name><surname>Câmara</surname><given-names>NO</given-names></name><name><surname>Nishida</surname><given-names>S</given-names></name><name><surname>Silva</surname><given-names>MS</given-names></name><name><surname>Rodrigues</surname><given-names>DA</given-names></name><name><surname>Pereira</surname><given-names>AB</given-names></name><etal></etal></person-group><article-title>Determination of renal function in long-term heart transplant patients by measurement of urinary retinol-binding protein levels</article-title><source>Braz J MedBiol Res.</source><year>2006</year><volume>39</volume><fpage>1305</fpage><lpage>13</lpage><pub-id pub-id-type="doi">10.1590/S0100-879X2006001000006</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wüthrich</surname><given-names>RP</given-names></name><name><surname>Weinreich</surname><given-names>T</given-names></name><name><surname>Schwarzkopf</surname><given-names>AK</given-names></name><name><surname>Candinas</surname><given-names>D</given-names></name><name><surname>Binswanger</surname><given-names>U</given-names></name></person-group><article-title>Postmarketing evaluation of mycophenolate mofetil-based triple therapy immunosuppression compared with a conventional azathioprine-based regimen reveals enhanced efficacy and early pharmacoeconomic benefit after renal transplantation</article-title><source>Transplant Proc</source><year>1998</year><volume>30</volume><fpage>4096</fpage><lpage>4097</lpage><pub-id pub-id-type="doi">10.1016/S0041-1345(98)01355-4</pub-id><pub-id pub-id-type="pmid">9865310</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Akoh</surname><given-names>JA</given-names></name></person-group><article-title>Renal transplantation in developing countries</article-title><source>Saudi J Kidney Dis Transpl</source><year>2011</year><volume>22</volume><fpage>637</fpage><lpage>650</lpage><pub-id pub-id-type="pmid">21743206</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ntemka</surname><given-names>A</given-names></name><name><surname>Iliadis</surname><given-names>F</given-names></name><name><surname>Papanikolaou</surname><given-names>N</given-names></name><name><surname>Grekas</surname><given-names>D</given-names></name></person-group><article-title>Network-centric analysis of genetic predisposition in diabetic nephropathy</article-title><source>Hippokratia</source><year>2011</year><volume>15</volume><fpage>232</fpage><lpage>237</lpage><pub-id pub-id-type="pmid">22435020</pub-id></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>McDermott</surname><given-names>JE</given-names></name><name><surname>Taylor</surname><given-names>RC</given-names></name><name><surname>Yoon</surname><given-names>H</given-names></name><name><surname>Heffron</surname><given-names>F</given-names></name></person-group><article-title>Bottlenecks and hubs in inferred networks are important for virulence in Salmonella typhimurium</article-title><source>J Comput Biol</source><year>2009</year><volume>16</volume><fpage>169</fpage><lpage>180</lpage><pub-id pub-id-type="doi">10.1089/cmb.2008.04TT</pub-id><pub-id pub-id-type="pmid">19178137</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hsu</surname><given-names>CW</given-names></name><name><surname>Juan</surname><given-names>HF</given-names></name><name><surname>Huang</surname><given-names>HC</given-names></name></person-group><article-title>Characterization of microRNA-regulated protein-protein interaction network</article-title><source>Proteomics</source><year>2008</year><volume>8</volume><fpage>1975</fpage><lpage>1979</lpage><pub-id pub-id-type="doi">10.1002/pmic.200701004</pub-id><pub-id pub-id-type="pmid">18491312</pub-id></element-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Benson</surname><given-names>M</given-names></name><name><surname>Breitling</surname><given-names>R</given-names></name></person-group><article-title>Network theory to understand microarray studies of complex diseases</article-title><source>Curr Mol Med</source><year>2006</year><volume>6</volume><fpage>695</fpage><lpage>701</lpage><pub-id pub-id-type="doi">10.2174/156652406778195044</pub-id><pub-id pub-id-type="pmid">17022739</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Reddy</surname><given-names>A</given-names></name><name><surname>Huang</surname><given-names>CC</given-names></name><name><surname>Liu</surname><given-names>H</given-names></name><name><surname>Delisi</surname><given-names>C</given-names></name><name><surname>Nevalainen</surname><given-names>MT</given-names></name><name><surname>Szalma</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Robust gene network analysis reveals alteration of the Stat5a network as a hallmark of prostate cancer</article-title><source>Genome Inform.</source><year>2010</year><volume>24</volume><fpage>139</fpage><lpage>53</lpage><pub-id pub-id-type="pmid">22081596</pub-id></element-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bando</surname><given-names>SY</given-names></name><name><surname>Silva</surname><given-names>FN</given-names></name><name><surname>Costa Lda</surname><given-names>F</given-names></name><name><surname>Silva</surname><given-names>AV</given-names></name><name><surname>Pimentel-Silva</surname><given-names>LR</given-names></name><name><surname>Castro</surname><given-names>LH</given-names></name><etal></etal></person-group><article-title>Complex network analysis of CA3 transcriptome reveals pathogenic and compensatory pathways in refractory temporal lobe epilepsy</article-title><source>Plos One</source><year>2013</year><volume>8</volume><fpage>e79913</fpage><pub-id pub-id-type="doi">10.1371/journal.pone.0079913</pub-id><pub-id pub-id-type="pmid">24278214</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Yu</surname><given-names>H</given-names></name><name><surname>Kim</surname><given-names>PM</given-names></name><name><surname>Sprecher</surname><given-names>E</given-names></name><name><surname>Trifonov</surname><given-names>V</given-names></name><name><surname>Gerstein</surname><given-names>M</given-names></name></person-group><article-title>The importance of bottlenecks in protein networks: correlation with gene essentiality and expression dynamics</article-title><source>PLoS Comput Biol</source><year>2007</year><volume>3</volume><fpage>e59</fpage><pub-id pub-id-type="doi">10.1371/journal.pcbi.0030059</pub-id><pub-id pub-id-type="pmid">17447836</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lebeau</surname><given-names>C</given-names></name><name><surname>Debelle</surname><given-names>FD</given-names></name><name><surname>Arlt</surname><given-names>VM</given-names></name><name><surname>Pozdzik</surname><given-names>A</given-names></name><name><surname>De Prez</surname><given-names>EG</given-names></name><name><surname>Phillips</surname><given-names>DH</given-names></name><etal></etal></person-group><article-title>Early proximal tubule injury in experimental aristolochic acid nephropathy: functional and histological studies</article-title><source>Nephrol Dial Transplant.</source><year>2005</year><volume>20</volume><fpage>2321</fpage><lpage>32</lpage><pub-id pub-id-type="doi">10.1093/ndt/gfi042</pub-id><pub-id pub-id-type="pmid">16077141</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Le Roux</surname><given-names>S</given-names></name><name><surname>Devys</surname><given-names>A</given-names></name><name><surname>Girard</surname><given-names>C</given-names></name><name><surname>Harb</surname><given-names>J</given-names></name><name><surname>Hourmant</surname><given-names>M</given-names></name></person-group><article-title>Biomarkers for the diagnosis of the stable kidney transplant and chronic transplant injury using the ProtoArray® technology</article-title><source>Transplant Proc</source><year>2010</year><volume>42</volume><fpage>3475</fpage><lpage>3481</lpage><pub-id pub-id-type="doi">10.1016/j.transproceed.2010.09.006</pub-id><pub-id pub-id-type="pmid">21094800</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tanjore</surname><given-names>H</given-names></name><name><surname>Lawson</surname><given-names>WE</given-names></name><name><surname>Blackwell</surname><given-names>TS</given-names></name></person-group><article-title>Endoplasmic reticulum stress as a pro-fibrotic stimulus</article-title><source>Biochim Biophys Acta</source><year>1832</year><volume>2013</volume><fpage>940</fpage><lpage>947</lpage></element-citation></ref><ref id="CR25"><label>25.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cybulsky</surname><given-names>AV</given-names></name></person-group><article-title>The intersecting roles of endoplasmic reticulum stress, ubiquitin- proteasome system, and autophagy in the pathogenesis of proteinuric kidney disease</article-title><source>Kidney Int</source><year>2013</year><volume>84</volume><fpage>25</fpage><lpage>33</lpage><pub-id pub-id-type="doi">10.1038/ki.2012.390</pub-id><pub-id pub-id-type="pmid">23254900</pub-id></element-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Heymann</surname><given-names>D</given-names></name></person-group><article-title>Autophagy: a protective mechanism in response to stress and inflammation</article-title><source>Curr Opin Investig Drugs</source><year>2006</year><volume>7</volume><fpage>443</fpage><lpage>450</lpage><pub-id pub-id-type="pmid">16729721</pub-id></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Deretic</surname><given-names>V</given-names></name></person-group><article-title>Autophagosome and phagosome</article-title><source>Methods Mol Biol</source><year>2008</year><volume>445</volume><fpage>1</fpage><lpage>10</lpage><pub-id pub-id-type="doi">10.1007/978-1-59745-157-4_1</pub-id><pub-id pub-id-type="pmid">18425440</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Jiang</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>K</given-names></name><name><surname>Luo</surname><given-names>J</given-names></name><name><surname>Dong</surname><given-names>Z</given-names></name></person-group><article-title>Autophagy is a renoprotective mechanism during in vitro hypoxia and in vivo ischemia-reperfusion injury</article-title><source>Am J Pathol</source><year>2010</year><volume>176</volume><fpage>1181</fpage><lpage>1192</lpage><pub-id pub-id-type="doi">10.2353/ajpath.2010.090594</pub-id><pub-id pub-id-type="pmid">20075199</pub-id></element-citation></ref><ref id="CR29"><label>29.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Dohi</surname><given-names>E</given-names></name><name><surname>Tanaka</surname><given-names>S</given-names></name><name><surname>Seki</surname><given-names>T</given-names></name><name><surname>Miyagi</surname><given-names>T</given-names></name><name><surname>Hide</surname><given-names>I</given-names></name><name><surname>Takahashi</surname><given-names>T</given-names></name><etal></etal></person-group><article-title>Hypoxic stress activates chaperone-mediated autophagy and modulates neuronal cell survival</article-title><source>Neurochem Int.</source><year>2012</year><volume>60</volume><fpage>431</fpage><lpage>42</lpage><pub-id pub-id-type="doi">10.1016/j.neuint.2012.01.020</pub-id><pub-id pub-id-type="pmid">22306777</pub-id></element-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Seto</surname><given-names>S</given-names></name><name><surname>Tsujimura</surname><given-names>K</given-names></name><name><surname>Koide</surname><given-names>Y</given-names></name></person-group><article-title>Coronin-1a inhibits autophagosome formation around Mycobacterium tuberculosis-containing phagosomes and assists mycobacterial survival in macrophages</article-title><source>Cell Microbiol</source><year>2012</year><volume>14</volume><fpage>710</fpage><lpage>727</lpage><pub-id pub-id-type="doi">10.1111/j.1462-5822.2012.01754.x</pub-id><pub-id pub-id-type="pmid">22256790</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Maluf</surname><given-names>DG</given-names></name><name><surname>Mas</surname><given-names>VR</given-names></name><name><surname>Archer</surname><given-names>KJ</given-names></name><name><surname>Yanek</surname><given-names>K</given-names></name><name><surname>Gibney</surname><given-names>EM</given-names></name><name><surname>King</surname><given-names>AL</given-names></name><etal></etal></person-group><article-title>Molecular pathways involved in loss of kidney graft function with tubular atrophy and interstitial fibrosis</article-title><source>Mol Med.</source><year>2008</year><volume>14</volume><fpage>276</fpage><lpage>85</lpage><pub-id pub-id-type="doi">10.2119/2007-00111.Maluf</pub-id><pub-id pub-id-type="pmid">18286166</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Pesenacker</surname><given-names>AM</given-names></name><name><surname>Bending</surname><given-names>D</given-names></name><name><surname>Ursu</surname><given-names>S</given-names></name><name><surname>Wu</surname><given-names>Q</given-names></name><name><surname>Nistala</surname><given-names>K</given-names></name><name><surname>Wedderburn</surname><given-names>LR</given-names></name></person-group><article-title>CD161 defines the subset of FoxP3+ T cells capable of producing proinflammatory cytokines</article-title><source>Blood.</source><year>2013</year><volume>121</volume><fpage>2647</fpage><lpage>58</lpage><pub-id pub-id-type="doi">10.1182/blood-2012-08-443473</pub-id><pub-id pub-id-type="pmid">23355538</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rouas-Freiss</surname><given-names>N</given-names></name><name><surname>LeMaoult</surname><given-names>J</given-names></name><name><surname>Moreau</surname><given-names>P</given-names></name><name><surname>Dausset</surname><given-names>J</given-names></name><name><surname>Carosella</surname><given-names>ED</given-names></name></person-group><article-title>HLA-G in transplantation: a relevant molecule for inhibition of graft rejection?</article-title><source>Am J Transplant</source><year>2003</year><volume>3</volume><fpage>11</fpage><lpage>16</lpage><pub-id pub-id-type="doi">10.1034/j.1600-6143.2003.30103.x</pub-id><pub-id pub-id-type="pmid">12492704</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Jolly</surname><given-names>EC</given-names></name><name><surname>Key</surname><given-names>T</given-names></name><name><surname>Rasheed</surname><given-names>H</given-names></name><name><surname>Morgan</surname><given-names>H</given-names></name><name><surname>Butler</surname><given-names>A</given-names></name><name><surname>Pritchard</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>Preformed donor HLA-DP-specific antibodies mediate acute and chronic antibody-mediated rejection following renal transplantation</article-title><source>Am J Transplant.</source><year>2012</year><volume>12</volume><fpage>2845</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1111/j.1600-6143.2012.04172.x</pub-id><pub-id pub-id-type="pmid">22759079</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ling</surname><given-names>XB</given-names></name><name><surname>Sigdel</surname><given-names>TK</given-names></name><name><surname>Lau</surname><given-names>K</given-names></name><name><surname>Ying</surname><given-names>L</given-names></name><name><surname>Lau</surname><given-names>I</given-names></name><name><surname>Schilling</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Integrative urinary peptidomics in renal transplantation identifies biomarkers for acute rejection</article-title><source>J Am Soc Nephrol.</source><year>2010</year><volume>21</volume><fpage>646</fpage><lpage>53</lpage><pub-id pub-id-type="doi">10.1681/ASN.2009080876</pub-id><pub-id pub-id-type="pmid">20150539</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Famulski</surname><given-names>KS</given-names></name><name><surname>Kayser</surname><given-names>D</given-names></name><name><surname>Einecke</surname><given-names>G</given-names></name><name><surname>Allanach</surname><given-names>K</given-names></name><name><surname>Badr</surname><given-names>D</given-names></name><name><surname>Venner</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Alternative macrophage activation-associated transcripts in T-cell-mediated rejection of mouse kidney allografts</article-title><source>Am J Transplant.</source><year>2010</year><volume>10</volume><fpage>490</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1111/j.1600-6143.2009.02983.x</pub-id><pub-id pub-id-type="pmid">20121742</pub-id></element-citation></ref><ref id="CR37"><label>37.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Badid</surname><given-names>C</given-names></name><name><surname>Vincent</surname><given-names>M</given-names></name><name><surname>McGregor</surname><given-names>B</given-names></name><name><surname>Melin</surname><given-names>M</given-names></name><name><surname>Hadj-Aissa</surname><given-names>A</given-names></name><name><surname>Veysseyre</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>Mycophenolate mofetil reduces myofibroblast infiltration and collagen III deposition in rat remnant kidney</article-title><source>Kidney Int.</source><year>2000</year><volume>58</volume><fpage>51</fpage><lpage>61</lpage><pub-id pub-id-type="doi">10.1046/j.1523-1755.2000.00140.x</pub-id><pub-id pub-id-type="pmid">10886549</pub-id></element-citation></ref><ref id="CR38"><label>38.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bradley</surname><given-names>SP</given-names></name><name><surname>Pahari</surname><given-names>M</given-names></name><name><surname>Elias</surname><given-names>G</given-names></name><name><surname>Uknis</surname><given-names>ME</given-names></name><name><surname>Misra</surname><given-names>MV</given-names></name><name><surname>Rastellini</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>Intragraft gene expression profile during acute cellular rejection in clinical small bowel transplantation: a case report</article-title><source>Transplant Proc.</source><year>2006</year><volume>38</volume><fpage>1740</fpage><lpage>1</lpage><pub-id pub-id-type="doi">10.1016/j.transproceed.2006.05.054</pub-id><pub-id pub-id-type="pmid">16908268</pub-id></element-citation></ref><ref id="CR39"><label>39.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Scherer</surname><given-names>A</given-names></name><name><surname>Gwinner</surname><given-names>W</given-names></name><name><surname>Mengel</surname><given-names>M</given-names></name><name><surname>Kirsch</surname><given-names>T</given-names></name><name><surname>Raulf</surname><given-names>F</given-names></name><name><surname>Szustakowski</surname><given-names>JD</given-names></name><etal></etal></person-group><article-title>Transcriptome changes in renal allograft protocol biopsies at 3 months precede the onset of interstitial fibrosis/tubular atrophy (IF/TA) at 6 months</article-title><source>Nephrol Dial Transplant.</source><year>2009</year><volume>24</volume><fpage>2567</fpage><lpage>75</lpage><pub-id pub-id-type="doi">10.1093/ndt/gfp183</pub-id><pub-id pub-id-type="pmid">19398767</pub-id></element-citation></ref><ref id="CR40"><label>40.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Grzanka</surname><given-names>J</given-names></name><name><surname>Leveson-Gower</surname><given-names>D</given-names></name><name><surname>Golab</surname><given-names>K</given-names></name><name><surname>Wang</surname><given-names>XJ</given-names></name><name><surname>Marek-Trzonkowska</surname><given-names>N</given-names></name><name><surname>Krzystyniak</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>FoxP3, Helios, and SATB1: roles and relationships in regulatory T cells</article-title><source>Int Immunopharmacol.</source><year>2013</year><volume>16</volume><fpage>343</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1016/j.intimp.2013.02.004</pub-id><pub-id pub-id-type="pmid">23428911</pub-id></element-citation></ref><ref id="CR41"><label>41.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rudra</surname><given-names>D</given-names></name></person-group><article-title>deRoos P, Chaudhry A, Niec RE, Arvey A, Samstein RM, et al. Transcription factor Foxp3 and its protein partners form a complex regulatory network</article-title><source>Nat Immunol</source><year>2012</year><volume>13</volume><fpage>1009</fpage><lpage>10</lpage><pub-id pub-id-type="doi">10.1038/ni.2402</pub-id></element-citation></ref><ref id="CR42"><label>42.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Behrends</surname><given-names>C</given-names></name><name><surname>Sowa</surname><given-names>ME</given-names></name><name><surname>Gygi</surname><given-names>SP</given-names></name><name><surname>Harper</surname><given-names>JW</given-names></name></person-group><article-title>Network organization of the human autophagy system</article-title><source>Nature</source><year>2010</year><volume>466</volume><fpage>68</fpage><lpage>76</lpage><pub-id pub-id-type="doi">10.1038/nature09204</pub-id><pub-id pub-id-type="pmid">20562859</pub-id></element-citation></ref><ref id="CR43"><label>43.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ding</surname><given-names>WX</given-names></name><name><surname>Ni</surname><given-names>HM</given-names></name><name><surname>Gao</surname><given-names>W</given-names></name><name><surname>Yoshimori</surname><given-names>T</given-names></name><name><surname>Stolz</surname><given-names>DB</given-names></name><name><surname>Ron</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Linking of autophagy to ubiquitin-proteasome system is important for the regulation of endoplasmic reticulum stress and cell viability</article-title><source>Am J Pathol.</source><year>2007</year><volume>171</volume><fpage>513</fpage><lpage>24</lpage><pub-id pub-id-type="doi">10.2353/ajpath.2007.070188</pub-id><pub-id pub-id-type="pmid">17620365</pub-id></element-citation></ref><ref id="CR44"><label>44.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tsushima</surname><given-names>K</given-names></name><name><surname>Osawa</surname><given-names>T</given-names></name><name><surname>Yanai</surname><given-names>H</given-names></name><name><surname>Nakajima</surname><given-names>A</given-names></name><name><surname>Takaoka</surname><given-names>A</given-names></name><name><surname>Manabe</surname><given-names>I</given-names></name><etal></etal></person-group><article-title>IRF3 regulates cardiac fibrosis but not hypertrophy in mice during angiotensin II-induced hypertension</article-title><source>FASEB J.</source><year>2011</year><volume>25</volume><fpage>1531</fpage><lpage>43</lpage><pub-id pub-id-type="doi">10.1096/fj.10-174615</pub-id><pub-id pub-id-type="pmid">21266535</pub-id></element-citation></ref><ref id="CR45"><label>45.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Walker</surname><given-names>JL</given-names></name><name><surname>Zhai</surname><given-names>N</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name><name><surname>Bleaken</surname><given-names>BM</given-names></name><name><surname>Wolff</surname><given-names>I</given-names></name><name><surname>Gerhart</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Unique precursors for the mesenchymal cells involved in injury response and fibrosis</article-title><source>Proc Natl Acad Sci U S A.</source><year>2010</year><volume>107</volume><fpage>13730</fpage><lpage>5</lpage><pub-id pub-id-type="doi">10.1073/pnas.0910382107</pub-id><pub-id pub-id-type="pmid">20634425</pub-id></element-citation></ref><ref id="CR46"><label>46.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stacchiotti</surname><given-names>A</given-names></name><name><surname>Rezzani</surname><given-names>R</given-names></name><name><surname>Angoscini</surname><given-names>P</given-names></name><name><surname>Rodella</surname><given-names>L</given-names></name><name><surname>Bianchi</surname><given-names>R</given-names></name></person-group><article-title>Small heat shock proteins expression in rat kidneys treated with cyclosporine A alone and combined with melatonin</article-title><source>Histochem J</source><year>2002</year><volume>34</volume><fpage>305</fpage><lpage>312</lpage><pub-id pub-id-type="doi">10.1023/A:1023378525471</pub-id><pub-id pub-id-type="pmid">12769262</pub-id></element-citation></ref><ref id="CR47"><label>47.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Abbate</surname><given-names>M</given-names></name><name><surname>Brown</surname><given-names>D</given-names></name><name><surname>Bonventre</surname><given-names>JV</given-names></name></person-group><article-title>Expression of NCAM recapitulates tubulogenic development in kidneys recovering from acute ischemia</article-title><source>Am J Physiol</source><year>1999</year><volume>277</volume><fpage>F454</fpage><lpage>F463</lpage><pub-id pub-id-type="pmid">10484529</pub-id></element-citation></ref><ref id="CR48"><label>48.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bozic</surname><given-names>M</given-names></name><name><surname>de Rooij</surname><given-names>J</given-names></name><name><surname>Parisi</surname><given-names>E</given-names></name><name><surname>Ortega</surname><given-names>MR</given-names></name><name><surname>Fernandez</surname><given-names>E</given-names></name><name><surname>Valdivielso</surname><given-names>JM</given-names></name></person-group><article-title>Glutamatergic signaling maintains the epithelial phenotype of proximal tubular cells</article-title><source>J Am Soc Nephrol.</source><year>2011</year><volume>22</volume><fpage>1099</fpage><lpage>111</lpage><pub-id pub-id-type="doi">10.1681/ASN.2010070701</pub-id><pub-id pub-id-type="pmid">21597037</pub-id></element-citation></ref><ref id="CR49"><label>49.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Solez</surname><given-names>K</given-names></name><name><surname>Colvin</surname><given-names>RB</given-names></name><name><surname>Racusen</surname><given-names>LC</given-names></name><name><surname>Sis</surname><given-names>B</given-names></name><name><surname>Halloran</surname><given-names>PF</given-names></name><name><surname>Birk</surname><given-names>PE</given-names></name><etal></etal></person-group><article-title>Banff ’05 meeting report: differential diagnosis of chronic allograft injury and elimination of chronic allograft nephropathy</article-title><source>Am J Transplant.</source><year>2007</year><volume>7</volume><fpage>518</fpage><lpage>26</lpage><pub-id pub-id-type="doi">10.1111/j.1600-6143.2006.01688.x</pub-id><pub-id pub-id-type="pmid">17352710</pub-id></element-citation></ref><ref id="CR50"><label>50.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Correa-Costa</surname><given-names>M</given-names></name><name><surname>Azevedo</surname><given-names>H</given-names></name><name><surname>Amano</surname><given-names>MT</given-names></name><name><surname>Gonçalves</surname><given-names>GM</given-names></name><name><surname>Hyane</surname><given-names>MI</given-names></name><name><surname>Cenedeze</surname><given-names>MA</given-names></name><etal></etal></person-group><article-title>Transcriptome analysis of renal ischemia/reperfusion injury and its modulation by ischemic pre-conditioning or hemin treatment</article-title><source>PLoS One.</source><year>2012</year><volume>7</volume><pub-id pub-id-type="doi">10.1371/journal.pone.0049569</pub-id><pub-id pub-id-type="pmid">23166714</pub-id></element-citation></ref><ref id="CR51"><label>51.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Saeed</surname><given-names>AI</given-names></name><name><surname>Sharov</surname><given-names>V</given-names></name><name><surname>White</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Liang</surname><given-names>W</given-names></name><name><surname>Bhagabati</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>TM4, a free, open-source system for microarray data management and analysis</article-title><source>Biotechniques.</source><year>2003</year><volume>34</volume><fpage>374</fpage><lpage>8</lpage><pub-id pub-id-type="pmid">12613259</pub-id></element-citation></ref><ref id="CR52"><label>52.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Eisen</surname><given-names>MB</given-names></name><name><surname>Spellman</surname><given-names>PT</given-names></name><name><surname>Brown</surname><given-names>PO</given-names></name><name><surname>Botstein</surname><given-names>D</given-names></name></person-group><article-title>Cluster analysis and display of genome-wide expression patterns</article-title><source>Proc Natl Acad Sci U S A</source><year>1998</year><volume>95</volume><fpage>14863</fpage><lpage>14868</lpage><pub-id pub-id-type="doi">10.1073/pnas.95.25.14863</pub-id><pub-id pub-id-type="pmid">9843981</pub-id></element-citation></ref><ref id="CR53"><label>53.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Montojo</surname><given-names>J</given-names></name><name><surname>Zuberi</surname><given-names>K</given-names></name><name><surname>Rodriguez</surname><given-names>H</given-names></name><name><surname>Kazi</surname><given-names>F</given-names></name><name><surname>Wright</surname><given-names>G</given-names></name><name><surname>Donaldson</surname><given-names>SL</given-names></name><etal></etal></person-group><article-title>GeneMANIA Cytoscape plugin: fast gene function predictions on the desktop</article-title><source>Bioinformatics.</source><year>2010</year><volume>26</volume><fpage>2927</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1093/bioinformatics/btq562</pub-id><pub-id pub-id-type="pmid">20926419</pub-id></element-citation></ref><ref id="CR54"><label>54.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Scardoni</surname><given-names>G</given-names></name><name><surname>Petterlini</surname><given-names>M</given-names></name><name><surname>Laudanna</surname><given-names>C</given-names></name></person-group><article-title>Analyzing biological network parameters with CentiScaPe</article-title><source>Bioinformatics</source><year>2009</year><volume>25</volume><fpage>2857</fpage><lpage>2859</lpage><pub-id pub-id-type="doi">10.1093/bioinformatics/btp517</pub-id><pub-id pub-id-type="pmid">19729372</pub-id></element-citation></ref><ref id="CR55"><label>55.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Prifti</surname><given-names>E</given-names></name><name><surname>Zucker</surname><given-names>JD</given-names></name><name><surname>Clement</surname><given-names>K</given-names></name><name><surname>Henegar</surname><given-names>C</given-names></name></person-group><article-title>FunNet: an integrative tool for exploring transcriptional interactions</article-title><source>Bioinformatics</source><year>2008</year><volume>24</volume><fpage>2636</fpage><lpage>2638</lpage><pub-id pub-id-type="doi">10.1093/bioinformatics/btn492</pub-id><pub-id pub-id-type="pmid">18799481</pub-id></element-citation></ref><ref id="CR56"><label>56.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Smoot</surname><given-names>M</given-names></name><name><surname>Keiichiro</surname><given-names>O</given-names></name><name><surname>Ruscheinski</surname><given-names>J</given-names></name><name><surname>Wang</surname><given-names>PL</given-names></name><name><surname>Ideker</surname><given-names>T</given-names></name></person-group><article-title>Cytoscape 2.8: new features for data integration and network visualization</article-title><source>Bioinformatics</source><year>2011</year><volume>27</volume><fpage>431</fpage><lpage>432</lpage><pub-id pub-id-type="doi">10.1093/bioinformatics/btq675</pub-id><pub-id pub-id-type="pmid">21149340</pub-id></element-citation></ref><ref id="CR57"><label>57.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Huang</surname><given-names>ZX</given-names></name><name><surname>Tian</surname><given-names>HY</given-names></name><name><surname>Hu</surname><given-names>ZF</given-names></name><name><surname>Zhou</surname><given-names>YB</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Yao</surname><given-names>KT</given-names></name></person-group><article-title>GenCLiP: a software program for clustering gene lists by literature profiling and constructing gene co-occurrence networks related to custom keywords</article-title><source>BMC Bioinformatics.</source><year>2008</year><volume>9</volume><fpage>308</fpage><pub-id pub-id-type="doi">10.1186/1471-2105-9-308</pub-id><pub-id pub-id-type="pmid">18620599</pub-id></element-citation></ref></ref-list></back></article>