<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Clin Epigenetics</journal-id><journal-id journal-id-type="iso-abbrev">Clin Epigenetics</journal-id><journal-title-group><journal-title>Clinical Epigenetics</journal-title></journal-title-group><issn pub-type="ppub">1868-7075</issn><issn pub-type="epub">1868-7083</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">26052356</article-id><article-id pub-id-type="pmc">4457092</article-id><article-id pub-id-type="publisher-id">92</article-id><article-id pub-id-type="doi">10.1186/s13148-015-0092-2</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>MicroRNA expression profiling predicts clinical outcome of carboplatin/paclitaxel-based therapy in metastatic melanoma treated on the ECOG-ACRIN trial E2603</article-title></title-group><contrib-group><contrib contrib-type="author" equal-contrib="yes"><name><surname>Villaruz</surname><given-names>Liza C.</given-names></name><address><email>Villaruzl@upmc.edu</email></address><xref ref-type="aff" rid="Aff1"></xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Huang</surname><given-names>Grace</given-names></name><address><email>graceworkhuang@gmail.com</email></address><xref ref-type="aff" rid="Aff2"></xref></contrib><contrib contrib-type="author"><name><surname>Romkes</surname><given-names>Marjorie</given-names></name><address><email>romkes@pitt.edu</email></address><xref ref-type="aff" rid="Aff1"></xref></contrib><contrib contrib-type="author"><name><surname>Kirkwood</surname><given-names>John M.</given-names></name><address><email>kirkwoodjm@upmc.edu</email></address><xref ref-type="aff" rid="Aff1"></xref></contrib><contrib contrib-type="author"><name><surname>Buch</surname><given-names>Shama C.</given-names></name><address><email>shamabuch@gmail.com</email></address><xref ref-type="aff" rid="Aff2"></xref></contrib><contrib contrib-type="author"><name><surname>Nukui</surname><given-names>Tomoko</given-names></name><address><email>tonst2@pitt.edu</email></address><xref ref-type="aff" rid="Aff2"></xref></contrib><contrib contrib-type="author"><name><surname>Flaherty</surname><given-names>Keith T.</given-names></name><address><email>kflaherty@partners.org</email></address><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author"><name><surname>Lee</surname><given-names>Sandra J.</given-names></name><address><email>sjlee@jimmy.harvard.edu</email></address><xref ref-type="aff" rid="Aff5"></xref></contrib><contrib contrib-type="author"><name><surname>Wilson</surname><given-names>Melissa A.</given-names></name><address><email>melissawilson@gmail.com</email></address><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff6"></xref></contrib><contrib contrib-type="author"><name><surname>Nathanson</surname><given-names>Katherine L.</given-names></name><address><email>knathans@exchange.upenn.edu</email></address><xref ref-type="aff" rid="Aff3"></xref></contrib><contrib contrib-type="author"><name><surname>Benos</surname><given-names>Panayiotis V.</given-names></name><address><email>benos@pitt.edu</email></address><xref ref-type="aff" rid="Aff2"></xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Tawbi</surname><given-names>Hussein A.</given-names></name><address><email>tawbih@upmc.edu</email></address><xref ref-type="aff" rid="Aff1"></xref></contrib><aff id="Aff1"><label></label>University of Pittsburgh Cancer Institute, Pittsburgh, PA USA </aff><aff id="Aff2"><label></label>University of Pittsburgh School of Medicine, Pittsburgh, PA USA </aff><aff id="Aff3"><label></label>University of Pennsylvania, Philadelphia, PA USA </aff><aff id="Aff4"><label></label>Massachusetts General Hospital, Boston, MA USA </aff><aff id="Aff5"><label></label>Dana Farber Cancer Institute, Boston, MA USA </aff><aff id="Aff6"><label></label>New York University, New York, NY USA </aff></contrib-group><pub-date pub-type="epub"><day>4</day><month>6</month><year>2015</year></pub-date><pub-date pub-type="pmc-release"><day>4</day><month>6</month><year>2015</year></pub-date><pub-date pub-type="collection"><year>2015</year></pub-date><volume>7</volume><issue>1</issue><elocation-id>58</elocation-id><history><date date-type="received"><day>12</day><month>1</month><year>2015</year></date><date date-type="accepted"><day>27</day><month>5</month><year>2015</year></date></history><permissions><copyright-statement>© Villaruz et al. 2015</copyright-statement><license license-type="open-access"><license-p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0">http://creativecommons.org/licenses/by/4.0</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</ext-link>) applies to the data made available in this article, unless otherwise stated.</license-p></license></permissions><abstract id="Abs1"><sec><title><offsets xml_i="5646" xml_f="5656" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="5667" xml_f="5959" txt_i="22" txt_f="314">Carboplatin/paclitaxel (CP), with or without sorafenib, result in objective response rates of 18–20 % in unselected chemotherapy-naïve patients. Molecular predictors of survival and response to CP-based chemotherapy in metastatic melanoma (MM) are critical to improving the therapeutic index.</offsets></p><p><offsets xml_i="5966" xml_f="6252" txt_i="315" txt_f="601">Intergroup trial E2603 randomized MM patients to CP with or without sorafenib. Expression data were collected from pre-treatment formalin-fixed paraffin-embedded (FFPE) tumor tissues from 115 of 823 patients enrolled on E2603. The selected patients were balanced across treatment arms, </offsets><italic><offsets xml_i="6260" xml_f="6264" txt_i="601" txt_f="605">BRAF</offsets></italic><offsets xml_i="6273" xml_f="6808" txt_i="605" txt_f="1140"> status, and clinical outcome. We generated data using Nanostring array (microRNA (miRNA) expression) and DNA-mediated annealing, selection, extension and ligation (DASL)/Illumina microarrays (HT12 v4) (mRNA expression) with protocols optimized for FFPE samples. Integrative computational analysis was performed using a novel Tree-guided Recursive Cluster Selection (T-ReCS) [1] algorithm to select the most informative features/genes, followed by TargetScan miRNA target prediction (Human v6.2) and mirConnX [2] for network inference.</offsets></p></sec><sec><title><offsets xml_i="6830" xml_f="6837" txt_i="1142" txt_f="1149">Results</offsets></title><p><offsets xml_i="6848" xml_f="6866" txt_i="1150" txt_f="1168">T-ReCS identified </offsets><italic><offsets xml_i="6874" xml_f="6880" txt_i="1168" txt_f="1174">PLXNB1</offsets></italic><offsets xml_i="6889" xml_f="6956" txt_i="1174" txt_f="1241"> as negatively associated with progression-free survival (PFS) and </offsets><italic><offsets xml_i="6964" xml_f="6974" txt_i="1241" txt_f="1251">miR-659-3p</offsets></italic><offsets xml_i="6983" xml_f="7037" txt_i="1251" txt_f="1305"> as the primary miRNA associated positively with PFS. </offsets><italic><offsets xml_i="7045" xml_f="7048" txt_i="1305" txt_f="1308">miR</offsets></italic><offsets xml_i="7057" xml_f="7058" txt_i="1308" txt_f="1309">-</offsets><italic><offsets xml_i="7066" xml_f="7069" txt_i="1309" txt_f="1312">659</offsets></italic><offsets xml_i="7078" xml_f="7079" txt_i="1312" txt_f="1313">-</offsets><italic><offsets xml_i="7087" xml_f="7089" txt_i="1313" txt_f="1315">3p</offsets></italic><offsets xml_i="7098" xml_f="7173" txt_i="1315" txt_f="1390"> was differentially expressed based on PFS but not based on treatment arm, </offsets><italic><offsets xml_i="7181" xml_f="7185" txt_i="1390" txt_f="1394">BRAF</offsets></italic><offsets xml_i="7194" xml_f="7198" txt_i="1394" txt_f="1398"> or </offsets><italic><offsets xml_i="7206" xml_f="7210" txt_i="1398" txt_f="1402">NRAS</offsets></italic><offsets xml_i="7219" xml_f="7288" txt_i="1402" txt_f="1471"> status. Dichotomized by median PFS (less vs greater than 4 months), </offsets><italic><offsets xml_i="7296" xml_f="7306" txt_i="1471" txt_f="1481">miR-659-3p</offsets></italic><offsets xml_i="7315" xml_f="7361" txt_i="1481" txt_f="1527"> expression was significantly different. High </offsets><italic><offsets xml_i="7369" xml_f="7379" txt_i="1527" txt_f="1537">miR-659-3p</offsets></italic><offsets xml_i="7388" xml_f="7513" txt_i="1537" txt_f="1662"> expression distinguished patients with responsive disease (complete or partial response) from patients with stable disease. </offsets><italic><offsets xml_i="7521" xml_f="7531" txt_i="1662" txt_f="1672">miR-659-3p</offsets></italic><offsets xml_i="7540" xml_f="7572" txt_i="1672" txt_f="1704"> predicted gene targets include </offsets><italic><offsets xml_i="7580" xml_f="7584" txt_i="1704" txt_f="1708">NFIX</offsets></italic><offsets xml_i="7593" xml_f="7650" txt_i="1708" txt_f="1765">, which is a transcription factor known to interact with </offsets><italic><offsets xml_i="7658" xml_f="7663" txt_i="1765" txt_f="1770">c-Jun</offsets></italic><offsets xml_i="7672" xml_f="7677" txt_i="1770" txt_f="1775"> and </offsets><italic><offsets xml_i="7685" xml_f="7689" txt_i="1775" txt_f="1779">AP-1</offsets></italic><offsets xml_i="7698" xml_f="7753" txt_i="1779" txt_f="1834"> in the context of developmental processes and disease.</offsets></p></sec><sec><title><offsets xml_i="7775" xml_f="7786" txt_i="1836" txt_f="1847">Conclusions</offsets></title><p><offsets xml_i="7797" xml_f="7840" txt_i="1848" txt_f="1891">This novel integrative analysis implicates </offsets><italic><offsets xml_i="7848" xml_f="7858" txt_i="1891" txt_f="1901">miR-659-3p</offsets></italic><offsets xml_i="7867" xml_f="8004" txt_i="1901" txt_f="2038"> as a candidate predictive biomarker for MM patients treated with platinum-based chemotherapy and may serve to improve patient selection.</offsets></p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Melanoma</kwd><kwd>MicroRNAs</kwd><kwd>Chemotherapy</kwd><kwd>Response</kwd><kwd>Biomarkers</kwd><kwd>Predictive</kwd></kwd-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2015</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1" sec-type="introduction"><title><offsets xml_i="8442" xml_f="8452" txt_i="2047" txt_f="2057">Background</offsets></title><p><offsets xml_i="8463" xml_f="8612" txt_i="2058" txt_f="2207">The incidence of melanoma has increased progressively over the last 30 years with an estimated 76,100 new cases in 2014 and a projected 9700 deaths (</offsets><ext-link ext-link-type="uri" xlink:href="http://seer.cancer.gov/statfacts/html/melan.html"><offsets xml_i="8704" xml_f="8752" txt_i="2207" txt_f="2255">http://seer.cancer.gov/statfacts/html/melan.html</offsets></ext-link><offsets xml_i="8763" xml_f="8906" txt_i="2255" txt_f="2398">). The prognosis of metastatic disease is historically poor, with 5-year survival rates of less than 6 % and median survival of 6 to 9 months [</offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="8938" xml_f="8939" txt_i="2398" txt_f="2399">3</offsets></xref><offsets xml_i="8946" xml_f="8948" txt_i="2399" txt_f="2401">, </offsets><xref ref-type="bibr" rid="CR4"><offsets xml_i="8980" xml_f="8981" txt_i="2401" txt_f="2402">4</offsets></xref><offsets xml_i="8988" xml_f="9106" txt_i="2402" txt_f="2520">]. Until 2010, alkylating agents were the mainstay of treatment but associated with response rates less than 10–15 % [</offsets><xref ref-type="bibr" rid="CR5"><offsets xml_i="9138" xml_f="9139" txt_i="2520" txt_f="2521">5</offsets></xref><offsets xml_i="9146" xml_f="9148" txt_i="2521" txt_f="2523">, </offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="9180" xml_f="9181" txt_i="2523" txt_f="2524">6</offsets></xref><offsets xml_i="9188" xml_f="9372" txt_i="2524" txt_f="2708">]. Platinum-based chemotherapy with carboplatin and paclitaxel (CP) is associated with response rates of 18–20 % in chemotherapy-naïve patients and 11–12 % in the second-line setting [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="9404" xml_f="9405" txt_i="2708" txt_f="2709">7</offsets></xref><offsets xml_i="9412" xml_f="9414" txt_i="2709" txt_f="2711">, </offsets><xref ref-type="bibr" rid="CR8"><offsets xml_i="9446" xml_f="9447" txt_i="2711" txt_f="2712">8</offsets></xref><offsets xml_i="9454" xml_f="9747" txt_i="2712" txt_f="3005">]. Significant therapeutic advances have been made in recent years with FDA approval of the monoclonal antibody to CTLA-4, ipilimumab, which results in durable responses and improved overall survival (OS) in patients with metastatic melanoma (MM) compared with peptide vaccine or dacarbazine [</offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="9779" xml_f="9780" txt_i="3005" txt_f="3006">9</offsets></xref><offsets xml_i="9787" xml_f="9975" txt_i="3006" txt_f="3194">]. Antibody therapies against PD-1 and PD-L1 have also been associated with durable responses in MM, and the first agent targeting this pathway was granted FDA approval in September 2014 [</offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="10008" xml_f="10010" txt_i="3194" txt_f="3196">10</offsets></xref><offsets xml_i="10017" xml_f="10018" txt_i="3196" txt_f="3197">–</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="10051" xml_f="10053" txt_i="3197" txt_f="3199">12</offsets></xref><offsets xml_i="10060" xml_f="10099" txt_i="3199" txt_f="3238">]. Finally, the selective targeting of </offsets><italic><offsets xml_i="10107" xml_f="10111" txt_i="3238" txt_f="3242">BRAF</offsets></italic><offsets xml_i="10120" xml_f="10141" txt_i="3242" txt_f="3263"> activating mutations</offsets><italic><offsets xml_i="10149" xml_f="10150" txt_i="3263" txt_f="3264">,</offsets></italic><offsets xml_i="10159" xml_f="10334" txt_i="3264" txt_f="3439"> which occur in about half of melanomas, with vemurafenib, dabrafenib, and trametinib, is associated with marked clinical activity and improved OS compared with chemotherapy [</offsets><xref ref-type="bibr" rid="CR13"><offsets xml_i="10367" xml_f="10369" txt_i="3439" txt_f="3441">13</offsets></xref><offsets xml_i="10376" xml_f="10378" txt_i="3441" txt_f="3443">, </offsets><xref ref-type="bibr" rid="CR14"><offsets xml_i="10411" xml_f="10413" txt_i="3443" txt_f="3445">14</offsets></xref><offsets xml_i="10420" xml_f="10422" txt_i="3445" txt_f="3447">].</offsets></p><p><offsets xml_i="10429" xml_f="11310" txt_i="3448" txt_f="4329">Despite the recent successes of immunotherapy and targeted therapy, chemotherapy continues to be a significant modality utilized worldwide in MM, especially after progression on immunotherapy and/or antitumor targeted agents. Recalcitrance of melanoma to chemotherapy is a significant issue due to limited response rates and lack of predictive biomarkers of chemotherapy benefit. Identification of biomarkers of response and survival for patients treated with chemotherapy is critical to allow more refined application of this modality and improving treatment outcomes. MicroRNAs (miRNAs) are endogenous non-coding RNAs approximately 22 nucleotides in length that regulate gene expression at the post-transcriptional level and play an important role in the regulation of tumor suppressor genes, oncogenes, and genes involved in the epigenetic machinery and cancer cell metastasis [</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="11343" xml_f="11345" txt_i="4329" txt_f="4331">15</offsets></xref><offsets xml_i="11352" xml_f="11355" txt_i="4331" txt_f="4334">]. </offsets><italic><offsets xml_i="11363" xml_f="11366" txt_i="4334" txt_f="4337">miR</offsets></italic><offsets xml_i="11375" xml_f="11376" txt_i="4337" txt_f="4338">-</offsets><italic><offsets xml_i="11384" xml_f="11387" txt_i="4338" txt_f="4341">15a</offsets></italic><offsets xml_i="11396" xml_f="11398" txt_i="4341" txt_f="4343">, </offsets><italic><offsets xml_i="11406" xml_f="11409" txt_i="4343" txt_f="4346">miR</offsets></italic><offsets xml_i="11418" xml_f="11419" txt_i="4346" txt_f="4347">-</offsets><italic><offsets xml_i="11427" xml_f="11429" txt_i="4347" txt_f="4349">16</offsets></italic><offsets xml_i="11438" xml_f="11448" txt_i="4349" txt_f="4359">, and the </offsets><italic><offsets xml_i="11456" xml_f="11459" txt_i="4359" txt_f="4362">let</offsets></italic><offsets xml_i="11468" xml_f="11469" txt_i="4362" txt_f="4363">-</offsets><italic><offsets xml_i="11477" xml_f="11478" txt_i="4363" txt_f="4364">7</offsets></italic><offsets xml_i="11487" xml_f="11550" txt_i="4364" txt_f="4427"> family function predominately as tumor suppressors, while the </offsets><italic><offsets xml_i="11558" xml_f="11561" txt_i="4427" txt_f="4430">miR</offsets></italic><offsets xml_i="11570" xml_f="11571" txt_i="4430" txt_f="4431">-</offsets><italic><offsets xml_i="11579" xml_f="11581" txt_i="4431" txt_f="4433">17</offsets></italic><offsets xml_i="11590" xml_f="11591" txt_i="4433" txt_f="4434">-</offsets><italic><offsets xml_i="11599" xml_f="11601" txt_i="4434" txt_f="4436">92</offsets></italic><offsets xml_i="11610" xml_f="11643" txt_i="4436" txt_f="4469"> cluster functions as oncogenes [</offsets><xref ref-type="bibr" rid="CR16"><offsets xml_i="11676" xml_f="11678" txt_i="4469" txt_f="4471">16</offsets></xref><offsets xml_i="11685" xml_f="11686" txt_i="4471" txt_f="4472">–</offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="11719" xml_f="11721" txt_i="4472" txt_f="4474">20</offsets></xref><offsets xml_i="11728" xml_f="11771" txt_i="4474" txt_f="4517">]. Differential expression of four miRNAs (</offsets><italic><offsets xml_i="11779" xml_f="11786" txt_i="4517" txt_f="4524">miR-205</offsets></italic><offsets xml_i="11795" xml_f="11797" txt_i="4524" txt_f="4526">, </offsets><italic><offsets xml_i="11805" xml_f="11812" txt_i="4526" txt_f="4533">miR-23b</offsets></italic><offsets xml_i="11821" xml_f="11823" txt_i="4533" txt_f="4535">, </offsets><italic><offsets xml_i="11831" xml_f="11839" txt_i="4535" txt_f="4543">miR-146a</offsets></italic><offsets xml_i="11848" xml_f="11854" txt_i="4543" txt_f="4549">, and </offsets><italic><offsets xml_i="11862" xml_f="11869" txt_i="4549" txt_f="4556">miR-155</offsets></italic><offsets xml_i="11878" xml_f="11960" txt_i="4556" txt_f="4638">) distinguishes melanoma cell lines from nevi and primary or metastatic melanoma [</offsets><xref ref-type="bibr" rid="CR19"><offsets xml_i="11993" xml_f="11995" txt_i="4638" txt_f="4640">19</offsets></xref><offsets xml_i="12002" xml_f="12218" txt_i="4640" txt_f="4856">]. Differential miRNA expression also distinguishes among common melanoma histologic subtypes (superficial spreading melanoma vs nodular melanoma) and benign nevi compared with either primary or metastatic melanoma [</offsets><xref ref-type="bibr" rid="CR21"><offsets xml_i="12251" xml_f="12253" txt_i="4856" txt_f="4858">21</offsets></xref><offsets xml_i="12260" xml_f="12261" txt_i="4858" txt_f="4859">–</offsets><xref ref-type="bibr" rid="CR24"><offsets xml_i="12294" xml_f="12296" txt_i="4859" txt_f="4861">24</offsets></xref><offsets xml_i="12303" xml_f="12416" txt_i="4861" txt_f="4974">]. Of greatest importance, miRNAs have been shown to have prognostic implications in melanoma. Downregulation of </offsets><italic><offsets xml_i="12424" xml_f="12432" txt_i="4974" txt_f="4982">miR-125b</offsets></italic><offsets xml_i="12441" xml_f="12582" txt_i="4982" txt_f="5123"> in patients with stage T2 primary cutaneous melanoma is associated with the presence of micrometastatic disease to the sentinel lymph node [</offsets><xref ref-type="bibr" rid="CR25"><offsets xml_i="12615" xml_f="12617" txt_i="5123" txt_f="5125">25</offsets></xref><offsets xml_i="12624" xml_f="12845" txt_i="5125" txt_f="5346">]. Analysis of primary and metastatic melanoma specimens shows that overexpression of an 18-miRNA signature is significantly correlated with longer survival (defined as a post-recurrence survival of more than 18 months) [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="12878" xml_f="12880" txt_i="5346" txt_f="5348">26</offsets></xref><offsets xml_i="12887" xml_f="13096" txt_i="5348" txt_f="5557">], indicating the value of miRNAs as candidate prognostic biomarkers. To date, there have been no studies examining miRNAs as candidate predictive biomarkers of response to chemotherapy in metastatic melanoma.</offsets></p><p><offsets xml_i="13103" xml_f="13891" txt_i="5558" txt_f="6346">The Eastern Cooperative Oncology Group (ECOG) Study, E2603, was a phase III randomized controlled clinical trial evaluating the addition of the unselective multi-tyrosine kinase inhibitor, sorafenib, to platinum-based chemotherapy in patients with previously untreated MM. This phase III trial was the first to examine targeted therapy in melanoma but failed to meet its primary endpoint of demonstrating an OS benefit in MM patients treated with CP alone vs CP with sorafenib. The response rates and progression-free survival (PFS) were similar in either treatment arm. In the present study, our primary objective was to identify molecular predictors of survival and response to CP-based chemotherapy in the MM patients treated on E2603. We applied a novel method for cluster selection [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="13923" xml_f="13924" txt_i="6346" txt_f="6347">1</offsets></xref><offsets xml_i="13931" xml_f="14441" txt_i="6347" txt_f="6857">] to miRNA and gene expression data in this well-defined patient population in an effort to identify miRNAs as candidate biomarkers of prognosis and response to platinum-based chemotherapy. We then performed novel integrative analysis to construct a proposed genome-wide regulatory network, which lends insight into the role of our candidate miRNA biomarkers in the landscape of MM signaling. To our knowledge, our study is the first to utilize miRNAs as predictive biomarkers for therapy response in melanoma.</offsets></p></sec><sec id="Sec2" sec-type="results"><title><offsets xml_i="14492" xml_f="14514" txt_i="6859" txt_f="6881">Results and discussion</offsets></title><p><offsets xml_i="14525" xml_f="14741" txt_i="6882" txt_f="7098">High-quality miRNA and mRNA expression data were generated from legacy samples from 115 MM patients treated on E2603. The clinicopathologic features for the patients included in this analysis are summarized in Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="14775" xml_f="14776" txt_i="7098" txt_f="7099">1</offsets></xref><offsets xml_i="14783" xml_f="14876" txt_i="7099" txt_f="7192">. The median PFS for this cohort of patients was 4.4 months and the median OS was 9.8 months.</offsets><table-wrap id="Tab1"><label><offsets xml_i="14905" xml_f="14912" txt_i="7192" txt_f="7199">Table 1</offsets></label><caption><p><offsets xml_i="14932" xml_f="15002" txt_i="7199" txt_f="7269">Characteristics of patients treated on E2603 included in this analysis</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th></th><th><offsets xml_i="15077" xml_f="15078" txt_i="7270" txt_f="7271">
</offsets><italic><offsets xml_i="15086" xml_f="15087" txt_i="7271" txt_f="7272">n</offsets></italic><offsets xml_i="15096" xml_f="15102" txt_i="7272" txt_f="7278"> = 115</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="15135" xml_f="15149" txt_i="7278" txt_f="7292">Characteristic</offsets></td><td><offsets xml_i="15158" xml_f="15159" txt_i="7293" txt_f="7294">
</offsets><italic><offsets xml_i="15167" xml_f="15168" txt_i="7294" txt_f="7295">n</offsets></italic><offsets xml_i="15177" xml_f="15181" txt_i="7295" txt_f="7299"> (%)</offsets></td></tr><tr><td><offsets xml_i="15199" xml_f="15207" txt_i="7300" txt_f="7308">Male sex</offsets></td><td><offsets xml_i="15216" xml_f="15225" txt_i="7309" txt_f="7318">77 (67.0)</offsets></td></tr><tr><td><offsets xml_i="15243" xml_f="15268" txt_i="7319" txt_f="7344">Median age, years (range)</offsets></td><td><offsets xml_i="15277" xml_f="15287" txt_i="7345" txt_f="7355">59 (23–82)</offsets></td></tr><tr><td><offsets xml_i="15305" xml_f="15315" txt_i="7356" txt_f="7366">AJCC stage</offsets></td><td></td></tr><tr><td><offsets xml_i="15342" xml_f="15365" txt_i="7368" txt_f="7391"> Unresectable stage III</offsets></td><td><offsets xml_i="15374" xml_f="15383" txt_i="7392" txt_f="7401">16 (13.9)</offsets></td></tr><tr><td><offsets xml_i="15401" xml_f="15409" txt_i="7402" txt_f="7410"> M1a/M1b</offsets></td><td><offsets xml_i="15418" xml_f="15427" txt_i="7411" txt_f="7420">41 (35.7)</offsets></td></tr><tr><td><offsets xml_i="15445" xml_f="15449" txt_i="7421" txt_f="7425"> M1c</offsets></td><td><offsets xml_i="15458" xml_f="15467" txt_i="7426" txt_f="7435">58 (50.4)</offsets></td></tr><tr><td><offsets xml_i="15485" xml_f="15506" txt_i="7436" txt_f="7457">Serum LDH at baseline</offsets></td><td></td></tr><tr><td><offsets xml_i="15533" xml_f="15540" txt_i="7459" txt_f="7466"> Normal</offsets></td><td><offsets xml_i="15549" xml_f="15558" txt_i="7467" txt_f="7476">65 (56.5)</offsets></td></tr><tr><td><offsets xml_i="15576" xml_f="15589" txt_i="7477" txt_f="7490"> Above normal</offsets></td><td><offsets xml_i="15598" xml_f="15607" txt_i="7491" txt_f="7500">47 (40.9)</offsets></td></tr><tr><td><offsets xml_i="15625" xml_f="15648" txt_i="7501" txt_f="7524">ECOG performance status</offsets></td><td></td></tr><tr><td><offsets xml_i="15675" xml_f="15677" txt_i="7526" txt_f="7528"> 0</offsets></td><td><offsets xml_i="15686" xml_f="15695" txt_i="7529" txt_f="7538">67 (58.3)</offsets></td></tr><tr><td><offsets xml_i="15713" xml_f="15715" txt_i="7539" txt_f="7541"> 1</offsets></td><td><offsets xml_i="15724" xml_f="15733" txt_i="7542" txt_f="7551">48 (41.7)</offsets></td></tr><tr><td><offsets xml_i="15751" xml_f="15773" txt_i="7552" txt_f="7574">Prior systemic therapy</offsets></td><td></td></tr><tr><td><offsets xml_i="15800" xml_f="15805" txt_i="7576" txt_f="7581"> None</offsets></td><td><offsets xml_i="15814" xml_f="15823" txt_i="7582" txt_f="7591">52 (45.2)</offsets></td></tr><tr><td><offsets xml_i="15841" xml_f="15866" txt_i="7592" txt_f="7617"> Interferon, IL-2, GM-CSF</offsets></td><td><offsets xml_i="15875" xml_f="15884" txt_i="7618" txt_f="7627">59 (51.3)</offsets></td></tr><tr><td><offsets xml_i="15902" xml_f="15926" txt_i="7628" txt_f="7652"> Investigational therapy</offsets></td><td><offsets xml_i="15935" xml_f="15942" txt_i="7653" txt_f="7660">4 (3.5)</offsets></td></tr><tr><td><offsets xml_i="15960" xml_f="15973" txt_i="7661" txt_f="7674">Treatment arm</offsets></td><td></td></tr><tr><td><offsets xml_i="16000" xml_f="16023" txt_i="7676" txt_f="7699"> Carboplatin/paclitaxel</offsets></td><td><offsets xml_i="16032" xml_f="16041" txt_i="7700" txt_f="7709">58 (50.4)</offsets></td></tr><tr><td><offsets xml_i="16059" xml_f="16092" txt_i="7710" txt_f="7743"> Carboplatin/paclitaxel/sorafenib</offsets></td><td><offsets xml_i="16101" xml_f="16110" txt_i="7744" txt_f="7753">57 (49.6)</offsets></td></tr><tr><td><offsets xml_i="16128" xml_f="16129" txt_i="7754" txt_f="7755">
</offsets><italic><offsets xml_i="16137" xml_f="16141" txt_i="7755" txt_f="7759">BRAF</offsets></italic><offsets xml_i="16150" xml_f="16157" txt_i="7759" txt_f="7766"> status</offsets></td><td></td></tr><tr><td><offsets xml_i="16184" xml_f="16191" txt_i="7768" txt_f="7775"> Mutant</offsets></td><td><offsets xml_i="16200" xml_f="16209" txt_i="7776" txt_f="7785">48 (41.8)</offsets></td></tr><tr><td><offsets xml_i="16227" xml_f="16233" txt_i="7786" txt_f="7792"> V600E</offsets></td><td><offsets xml_i="16242" xml_f="16251" txt_i="7793" txt_f="7802">34 (29.6)</offsets></td></tr><tr><td><offsets xml_i="16269" xml_f="16275" txt_i="7803" txt_f="7809"> V600K</offsets></td><td><offsets xml_i="16284" xml_f="16291" txt_i="7810" txt_f="7817">8 (7.0)</offsets></td></tr><tr><td><offsets xml_i="16309" xml_f="16315" txt_i="7818" txt_f="7824"> V600R</offsets></td><td><offsets xml_i="16324" xml_f="16331" txt_i="7825" txt_f="7832">3 (2.6)</offsets></td></tr><tr><td><offsets xml_i="16349" xml_f="16355" txt_i="7833" txt_f="7839"> V600D</offsets></td><td><offsets xml_i="16364" xml_f="16371" txt_i="7840" txt_f="7847">1 (0.9)</offsets></td></tr><tr><td><offsets xml_i="16389" xml_f="16395" txt_i="7848" txt_f="7854"> K601E</offsets></td><td><offsets xml_i="16404" xml_f="16411" txt_i="7855" txt_f="7862">2 (1.7)</offsets></td></tr><tr><td><offsets xml_i="16429" xml_f="16439" txt_i="7863" txt_f="7873"> Wild-type</offsets></td><td><offsets xml_i="16448" xml_f="16457" txt_i="7874" txt_f="7883">52 (45.2)</offsets></td></tr><tr><td><offsets xml_i="16475" xml_f="16483" txt_i="7884" txt_f="7892"> Unknown</offsets></td><td><offsets xml_i="16492" xml_f="16501" txt_i="7893" txt_f="7902">15 (13.0)</offsets></td></tr><tr><td><offsets xml_i="16519" xml_f="16520" txt_i="7903" txt_f="7904">
</offsets><italic><offsets xml_i="16528" xml_f="16532" txt_i="7904" txt_f="7908">NRAS</offsets></italic><offsets xml_i="16541" xml_f="16548" txt_i="7908" txt_f="7915"> status</offsets></td><td></td></tr><tr><td><offsets xml_i="16575" xml_f="16582" txt_i="7917" txt_f="7924"> Mutant</offsets></td><td><offsets xml_i="16591" xml_f="16600" txt_i="7925" txt_f="7934">22 (19.1)</offsets></td></tr><tr><td><offsets xml_i="16618" xml_f="16623" txt_i="7935" txt_f="7940"> G13R</offsets></td><td><offsets xml_i="16632" xml_f="16639" txt_i="7941" txt_f="7948">2 (1.7)</offsets></td></tr><tr><td><offsets xml_i="16657" xml_f="16662" txt_i="7949" txt_f="7954"> G13C</offsets></td><td><offsets xml_i="16671" xml_f="16678" txt_i="7955" txt_f="7962">1 (0.9)</offsets></td></tr><tr><td><offsets xml_i="16696" xml_f="16701" txt_i="7963" txt_f="7968"> Q61R</offsets></td><td><offsets xml_i="16710" xml_f="16717" txt_i="7969" txt_f="7976">9 (7.8)</offsets></td></tr><tr><td><offsets xml_i="16735" xml_f="16740" txt_i="7977" txt_f="7982"> Q61K</offsets></td><td><offsets xml_i="16749" xml_f="16756" txt_i="7983" txt_f="7990">8 (7.0)</offsets></td></tr><tr><td><offsets xml_i="16774" xml_f="16779" txt_i="7991" txt_f="7996"> D61L</offsets></td><td><offsets xml_i="16788" xml_f="16795" txt_i="7997" txt_f="8004">2 (1.7)</offsets></td></tr><tr><td><offsets xml_i="16813" xml_f="16823" txt_i="8005" txt_f="8015"> Wild-type</offsets></td><td><offsets xml_i="16832" xml_f="16841" txt_i="8016" txt_f="8025">78 (67.8)</offsets></td></tr><tr><td><offsets xml_i="16859" xml_f="16867" txt_i="8026" txt_f="8034"> Unknown</offsets></td><td><offsets xml_i="16876" xml_f="16885" txt_i="8035" txt_f="8044">15 (13.0)</offsets></td></tr><tr><td><offsets xml_i="16903" xml_f="16911" txt_i="8045" txt_f="8053">Survival</offsets></td><td></td></tr><tr><td><offsets xml_i="16938" xml_f="16964" txt_i="8055" txt_f="8081"> Median OS, months (range)</offsets></td><td><offsets xml_i="16973" xml_f="16987" txt_i="8082" txt_f="8096">9.8 (0.5–56.9)</offsets></td></tr><tr><td><offsets xml_i="17005" xml_f="17032" txt_i="8097" txt_f="8124"> Median PFS, months (range)</offsets></td><td><offsets xml_i="17041" xml_f="17055" txt_i="8125" txt_f="8139">4.4 (0.5–42.1)</offsets></td></tr></tbody></table></table-wrap></p><sec id="Sec3"><title><offsets xml_i="17120" xml_f="17193" txt_i="8141" txt_f="8214">PLXNB1 and RAD23 are associated with shortened PFS in metastatic melanoma</offsets></title><p><offsets xml_i="17204" xml_f="17291" txt_i="8215" txt_f="8302">Among the single genes selected with Tree-guided Recursive Cluster Selection (T-ReCS), </offsets><italic><offsets xml_i="17299" xml_f="17305" txt_i="8302" txt_f="8308">PLXNB1</offsets></italic><offsets xml_i="17314" xml_f="17410" txt_i="8308" txt_f="8404">, a known tumor suppressor of melanoma, had the most significant negative association with PFS (</offsets><italic><offsets xml_i="17418" xml_f="17419" txt_i="8404" txt_f="8405">p</offsets></italic><offsets xml_i="17428" xml_f="17444" txt_i="8405" txt_f="8421"> = 0.0001; Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="17476" xml_f="17477" txt_i="8421" txt_f="8422">1</offsets></xref><offsets xml_i="17484" xml_f="17488" txt_i="8422" txt_f="8426">a) [</offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="17521" xml_f="17523" txt_i="8426" txt_f="8428">27</offsets></xref><offsets xml_i="17530" xml_f="17532" txt_i="8428" txt_f="8430">, </offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="17565" xml_f="17567" txt_i="8430" txt_f="8432">28</offsets></xref><offsets xml_i="17574" xml_f="17595" txt_i="8432" txt_f="8453">]. T-ReCS found gene </offsets><italic><offsets xml_i="17603" xml_f="17608" txt_i="8453" txt_f="8458">RAD23</offsets></italic><offsets xml_i="17617" xml_f="17639" txt_i="8458" txt_f="8480"> in the same group as </offsets><italic><offsets xml_i="17647" xml_f="17660" txt_i="8480" txt_f="8493">PLXNB1. RAD23</offsets></italic><offsets xml_i="17669" xml_f="17804" txt_i="8493" txt_f="8628">, a component of the nucleotide excision repair mechanism, was also negatively associated with PFS. In the other cluster, T-ReCS found </offsets><italic><offsets xml_i="17812" xml_f="17817" txt_i="8628" txt_f="8633">BCMO1</offsets></italic><offsets xml_i="17826" xml_f="17939" txt_i="8633" txt_f="8746">, which encodes a key enzyme in beta-carotene metabolism to vitamin A and is thus important for skin protection. </offsets><italic><offsets xml_i="17947" xml_f="17952" txt_i="8746" txt_f="8751">BCMO1</offsets></italic><offsets xml_i="17961" xml_f="18024" txt_i="8751" txt_f="8814"> was part of a very significant group variable, which included </offsets><italic><offsets xml_i="18032" xml_f="18041" txt_i="8814" txt_f="8823">LOC389936</offsets></italic><offsets xml_i="18050" xml_f="18110" txt_i="8823" txt_f="8883">, a shRNA construct with no known association with melanoma.</offsets><fig id="Fig1"><label><offsets xml_i="18132" xml_f="18138" txt_i="8883" txt_f="8889">Fig. 1</offsets></label><caption><p><offsets xml_i="18158" xml_f="18201" txt_i="8889" txt_f="8932">T-ReCS identified two major mRNA clusters (</offsets><bold><offsets xml_i="18207" xml_f="18208" txt_i="8932" txt_f="8933">a</offsets></bold><offsets xml_i="18215" xml_f="18256" txt_i="8933" txt_f="8974">) and one major cluster of three miRNAs (</offsets><bold><offsets xml_i="18262" xml_f="18263" txt_i="8974" txt_f="8975">b</offsets></bold><offsets xml_i="18270" xml_f="18420" txt_i="8975" txt_f="9125">) as predictive of progression-free survival (PFS), represented at heatmaps. Patients are ordered from shortest to longest progression-free survival (</offsets><bold><offsets xml_i="18426" xml_f="18427" txt_i="9125" txt_f="9126">c</offsets></bold><offsets xml_i="18434" xml_f="18435" txt_i="9126" txt_f="9127">)</offsets></p></caption><graphic xlink:href="13148_2015_92_Fig1_HTML" id="MO1"></graphic></fig></p></sec><sec id="Sec4"><title><offsets xml_i="18552" xml_f="18617" txt_i="9130" txt_f="9195">miR-659-3p is associated with improved PFS in metastatic melanoma</offsets></title><p><offsets xml_i="18628" xml_f="18646" txt_i="9196" txt_f="9214">T-ReCS identified </offsets><italic><offsets xml_i="18654" xml_f="18664" txt_i="9214" txt_f="9224">miR-659-3p</offsets></italic><offsets xml_i="18673" xml_f="18727" txt_i="9224" txt_f="9278"> as the primary miRNA associated positively with PFS (</offsets><italic><offsets xml_i="18735" xml_f="18736" txt_i="9278" txt_f="9279">p</offsets></italic><offsets xml_i="18745" xml_f="18760" txt_i="9279" txt_f="9294"> = 0.008). The </offsets><italic><offsets xml_i="18768" xml_f="18778" txt_i="9294" txt_f="9304">miR-659-3p</offsets></italic><offsets xml_i="18787" xml_f="18810" txt_i="9304" txt_f="9327"> cluster also included </offsets><italic><offsets xml_i="18818" xml_f="18828" txt_i="9327" txt_f="9337">miR-219-3p</offsets></italic><offsets xml_i="18837" xml_f="18842" txt_i="9337" txt_f="9342"> and </offsets><italic><offsets xml_i="18850" xml_f="18860" txt_i="9342" txt_f="9352">miR-519-5p</offsets></italic><offsets xml_i="18869" xml_f="18876" txt_i="9352" txt_f="9359"> (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="18908" xml_f="18910" txt_i="9359" txt_f="9361">1b</offsets></xref><offsets xml_i="18917" xml_f="18975" txt_i="9361" txt_f="9419">), which had less significant or no association with PFS (</offsets><italic><offsets xml_i="18983" xml_f="18984" txt_i="9419" txt_f="9420">p</offsets></italic><offsets xml_i="18993" xml_f="19006" txt_i="9420" txt_f="9433"> = 0.036 and </offsets><italic><offsets xml_i="19014" xml_f="19015" txt_i="9433" txt_f="9434">p</offsets></italic><offsets xml_i="19024" xml_f="19210" txt_i="9434" txt_f="9620"> = 0.12, respectively). We examined the Cox regression coefficients of this selected cluster to interrogate our results. Indeed, all of these miRNAs were positively associated with PFS. </offsets><italic><offsets xml_i="19218" xml_f="19228" txt_i="9620" txt_f="9630">miR-659-3p</offsets></italic><offsets xml_i="19237" xml_f="19342" txt_i="9630" txt_f="9735"> was not differentially expressed based on the treatment arm (CP with sorafenib vs CP without sorafenib; </offsets><italic><offsets xml_i="19350" xml_f="19351" txt_i="9735" txt_f="9736">p</offsets></italic><offsets xml_i="19360" xml_f="19369" txt_i="9736" txt_f="9745"> = 0.6), </offsets><italic><offsets xml_i="19377" xml_f="19381" txt_i="9745" txt_f="9749">BRAF</offsets></italic><offsets xml_i="19390" xml_f="19429" txt_i="9749" txt_f="9788"> mutation status (mutant vs wild-type; </offsets><italic><offsets xml_i="19437" xml_f="19438" txt_i="9788" txt_f="9789">p</offsets></italic><offsets xml_i="19447" xml_f="19459" txt_i="9789" txt_f="9801"> = 0.34) or </offsets><italic><offsets xml_i="19467" xml_f="19471" txt_i="9801" txt_f="9805">NRAS</offsets></italic><offsets xml_i="19480" xml_f="19519" txt_i="9805" txt_f="9844"> mutation status (mutant vs wild-type; </offsets><italic><offsets xml_i="19527" xml_f="19528" txt_i="9844" txt_f="9845">p</offsets></italic><offsets xml_i="19537" xml_f="19707" txt_i="9845" txt_f="10015"> = 0.55). We dichotomized PFS less than 4 months vs greater than 4 months, which is the median PFS of patients treated with CP with or without sorafenib on E2603. Higher </offsets><italic><offsets xml_i="19715" xml_f="19725" txt_i="10015" txt_f="10025">miR-659-3p</offsets></italic><offsets xml_i="19734" xml_f="19781" txt_i="10025" txt_f="10072"> expression remained prognostic of longer PFS (</offsets><italic><offsets xml_i="19789" xml_f="19790" txt_i="10072" txt_f="10073">p</offsets></italic><offsets xml_i="19799" xml_f="19970" txt_i="10073" txt_f="10244"> = 0.03). Further partitioning of the dataset into terciles or quantiles did not yield any significant results, presumably due to sample size. We also noticed that higher </offsets><italic><offsets xml_i="19978" xml_f="19988" txt_i="10244" txt_f="10254">miR-659-3p</offsets></italic><offsets xml_i="19997" xml_f="20211" txt_i="10254" txt_f="10468"> expression was generally observed in patients who achieved an objective response (complete or partial response) as opposed to patients with stable disease by Response Evaluation Criteria In Solid Tumors (RECIST) (</offsets><italic><offsets xml_i="20219" xml_f="20220" txt_i="10468" txt_f="10469">p</offsets></italic><offsets xml_i="20229" xml_f="20238" txt_i="10469" txt_f="10478"> = 0.04.)</offsets></p></sec><sec id="Sec5"><title><offsets xml_i="20270" xml_f="20293" txt_i="10480" txt_f="10503">miR-659-3p gene targets</offsets></title><p><italic><offsets xml_i="20312" xml_f="20322" txt_i="10504" txt_f="10514">miR-659-3p</offsets></italic><offsets xml_i="20331" xml_f="20432" txt_i="10514" txt_f="10615"> gene targets were inferred from TargetScan Human v6.2 and from the mirConnX prior information file [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="20465" xml_f="20467" txt_i="10615" txt_f="10617">29</offsets></xref><offsets xml_i="20474" xml_f="20558" txt_i="10617" txt_f="10701">]. The list of predicted targets included genes relevant to the MAP kinase pathway (</offsets><italic><offsets xml_i="20566" xml_f="20570" txt_i="10701" txt_f="10705">BRAF</offsets></italic><offsets xml_i="20579" xml_f="20584" txt_i="10705" txt_f="10710"> and </offsets><italic><offsets xml_i="20592" xml_f="20596" txt_i="10710" txt_f="10714">NRAS</offsets></italic><offsets xml_i="20605" xml_f="20713" txt_i="10714" txt_f="10822">), genes relevant to the PI3K/MTOR, FGFR, and RAC1 pathways, as well as genes involved in DNA repair (Table </offsets><xref rid="Tab2" ref-type="table"><offsets xml_i="20747" xml_f="20748" txt_i="10822" txt_f="10823">2</offsets></xref><offsets xml_i="20755" xml_f="20762" txt_i="10823" txt_f="10830">; Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="20794" xml_f="20795" txt_i="10830" txt_f="10831">2</offsets></xref><offsets xml_i="20802" xml_f="20896" txt_i="10831" txt_f="10925">b). However, the expression of most of those genes was not significantly anti-correlated with </offsets><italic><offsets xml_i="20904" xml_f="20914" txt_i="10925" txt_f="10935">miR-659-3p</offsets></italic><offsets xml_i="20923" xml_f="21128" txt_i="10935" txt_f="11140"> expression. This might be explained by the fact that each individual miRNA target usually decreases the expression of the host gene by a small amount. However, we did find a significant anti-correlation (</offsets><italic><offsets xml_i="21136" xml_f="21137" txt_i="11140" txt_f="11141">p</offsets></italic><offsets xml_i="21146" xml_f="21166" txt_i="11141" txt_f="11158"> &lt; 0.01) between </offsets><italic><offsets xml_i="21174" xml_f="21184" txt_i="11158" txt_f="11168">miR-659-3p</offsets></italic><offsets xml_i="21193" xml_f="21220" txt_i="11168" txt_f="11195"> and its predicted targets </offsets><italic><offsets xml_i="21228" xml_f="21232" txt_i="11195" txt_f="11199">COG6</offsets></italic><offsets xml_i="21241" xml_f="21243" txt_i="11199" txt_f="11201">, </offsets><italic><offsets xml_i="21251" xml_f="21256" txt_i="11201" txt_f="11206">ENOX2</offsets></italic><offsets xml_i="21265" xml_f="21267" txt_i="11206" txt_f="11208">, </offsets><italic><offsets xml_i="21275" xml_f="21280" txt_i="11208" txt_f="11213">HCFC2</offsets></italic><offsets xml_i="21289" xml_f="21291" txt_i="11213" txt_f="11215">, </offsets><italic><offsets xml_i="21299" xml_f="21303" txt_i="11215" txt_f="11219">NFIX</offsets></italic><offsets xml_i="21312" xml_f="21314" txt_i="11219" txt_f="11221">, </offsets><italic><offsets xml_i="21322" xml_f="21326" txt_i="11221" txt_f="11225">SDC1</offsets></italic><offsets xml_i="21335" xml_f="21341" txt_i="11225" txt_f="11231">, and </offsets><italic><offsets xml_i="21349" xml_f="21356" txt_i="11231" txt_f="11238">TSC22D2</offsets></italic><offsets xml_i="21365" xml_f="21372" txt_i="11238" txt_f="11245"> (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="21404" xml_f="21406" txt_i="11245" txt_f="11247">2a</offsets></xref><offsets xml_i="21413" xml_f="21673" txt_i="11247" txt_f="11507">). NFIX (nuclear factor 1 X-type), a member of the NF-1 family, is particularly important since it has been implicated in many developmental and biological processes and diseases, and the role of the NF-1 pathway is emerging as a potential therapeutic target [</offsets><xref ref-type="bibr" rid="CR30"><offsets xml_i="21706" xml_f="21708" txt_i="11507" txt_f="11509">30</offsets></xref><offsets xml_i="21715" xml_f="21716" txt_i="11509" txt_f="11510">–</offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="21749" xml_f="21751" txt_i="11510" txt_f="11512">33</offsets></xref><offsets xml_i="21758" xml_f="21822" txt_i="11512" txt_f="11576">]. Studies have also shown that it interacts with the oncogenes </offsets><italic><offsets xml_i="21830" xml_f="21835" txt_i="11576" txt_f="11581">c-Jun</offsets></italic><offsets xml_i="21844" xml_f="21849" txt_i="11581" txt_f="11586"> and </offsets><italic><offsets xml_i="21857" xml_f="21861" txt_i="11586" txt_f="11590">AP-1</offsets></italic><offsets xml_i="21870" xml_f="21872" txt_i="11590" txt_f="11592"> [</offsets><xref ref-type="bibr" rid="CR34"><offsets xml_i="21905" xml_f="21907" txt_i="11592" txt_f="11594">34</offsets></xref><offsets xml_i="21914" xml_f="21916" txt_i="11594" txt_f="11596">].</offsets><table-wrap id="Tab2"><label><offsets xml_i="21945" xml_f="21952" txt_i="11596" txt_f="11603">Table 2</offsets></label><caption><p><offsets xml_i="21972" xml_f="22000" txt_i="11603" txt_f="11631">Gene targets of interest of </offsets><italic><offsets xml_i="22008" xml_f="22018" txt_i="11631" txt_f="11641">miR-659-3p</offsets></italic><offsets xml_i="22027" xml_f="22066" txt_i="11641" txt_f="11680"> identified using TargetScan Human V6.2</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th><offsets xml_i="22132" xml_f="22148" txt_i="11681" txt_f="11697">Relevant pathway</offsets></th><th><offsets xml_i="22157" xml_f="22168" txt_i="11697" txt_f="11708">Gene target</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="22201" xml_f="22205" txt_i="11708" txt_f="11712">MAPK</offsets></td><td><offsets xml_i="22214" xml_f="22218" txt_i="11713" txt_f="11717">BRAF</offsets></td></tr><tr><td></td><td><offsets xml_i="22245" xml_f="22249" txt_i="11719" txt_f="11723">NRAS</offsets></td></tr><tr><td></td><td><offsets xml_i="22276" xml_f="22280" txt_i="11725" txt_f="11729">MAP7</offsets></td></tr><tr><td></td><td><offsets xml_i="22307" xml_f="22312" txt_i="11731" txt_f="11736">MAPK9</offsets></td></tr><tr><td><offsets xml_i="22330" xml_f="22339" txt_i="11737" txt_f="11746">PI3K/MTOR</offsets></td><td><offsets xml_i="22348" xml_f="22355" txt_i="11747" txt_f="11754">PIK3C2A</offsets></td></tr><tr><td></td><td><offsets xml_i="22382" xml_f="22387" txt_i="11756" txt_f="11761">FBXW2</offsets></td></tr><tr><td></td><td><offsets xml_i="22414" xml_f="22420" txt_i="11763" txt_f="11769">RICTOR</offsets></td></tr><tr><td><offsets xml_i="22438" xml_f="22442" txt_i="11770" txt_f="11774">FGFR</offsets></td><td><offsets xml_i="22451" xml_f="22456" txt_i="11775" txt_f="11780">FGF12</offsets></td></tr><tr><td></td><td><offsets xml_i="22483" xml_f="22488" txt_i="11782" txt_f="11787">FGF18</offsets></td></tr><tr><td><offsets xml_i="22506" xml_f="22510" txt_i="11788" txt_f="11792">RAC1</offsets></td><td><offsets xml_i="22519" xml_f="22523" txt_i="11793" txt_f="11797">RAC1</offsets></td></tr><tr><td><offsets xml_i="22541" xml_f="22551" txt_i="11798" txt_f="11808">DNA repair</offsets></td><td><offsets xml_i="22560" xml_f="22566" txt_i="11809" txt_f="11815">PARP16</offsets></td></tr><tr><td></td><td><offsets xml_i="22593" xml_f="22601" txt_i="11817" txt_f="11825">TP53INP1</offsets></td></tr><tr><td></td><td><offsets xml_i="22628" xml_f="22633" txt_i="11827" txt_f="11832">CDK19</offsets></td></tr><tr><td></td><td><offsets xml_i="22660" xml_f="22665" txt_i="11834" txt_f="11839">ABCC1</offsets></td></tr></tbody></table></table-wrap><fig id="Fig2"><label><offsets xml_i="22726" xml_f="22732" txt_i="11840" txt_f="11846">Fig. 2</offsets></label><caption><p><offsets xml_i="22752" xml_f="22820" txt_i="11846" txt_f="11914">Clustergram of genes whose expression was inversely correlated with </offsets><italic><offsets xml_i="22828" xml_f="22838" txt_i="11914" txt_f="11924">miR-659-3p</offsets></italic><offsets xml_i="22847" xml_f="22875" txt_i="11924" txt_f="11952"> expression in our dataset (</offsets><bold><offsets xml_i="22881" xml_f="22882" txt_i="11952" txt_f="11953">a</offsets></bold><offsets xml_i="22889" xml_f="22914" txt_i="11953" txt_f="11978">). Target predictions of </offsets><italic><offsets xml_i="22922" xml_f="22932" txt_i="11978" txt_f="11988">miR-659-3p</offsets></italic><offsets xml_i="22941" xml_f="23004" txt_i="11988" txt_f="12051"> identified utilizing mirConnX software for network inference (</offsets><bold><offsets xml_i="23010" xml_f="23011" txt_i="12051" txt_f="12052">b</offsets></bold><offsets xml_i="23018" xml_f="23110" txt_i="12052" txt_f="12144">). Several oncogenes known to be involved in melanoma disease mechanisms are highlighted in </offsets><italic><offsets xml_i="23118" xml_f="23122" txt_i="12144" txt_f="12148">gold</offsets></italic><offsets xml_i="23131" xml_f="23158" txt_i="12148" txt_f="12175">. Genes are represented by </offsets><italic><offsets xml_i="23166" xml_f="23173" txt_i="12175" txt_f="12182">squares</offsets></italic><offsets xml_i="23182" xml_f="23194" txt_i="12182" txt_f="12194">, miRNAs by </offsets><italic><offsets xml_i="23202" xml_f="23211" txt_i="12194" txt_f="12203">triangles</offsets></italic><offsets xml_i="23220" xml_f="23251" txt_i="12203" txt_f="12234">, and transcription factors by </offsets><italic><offsets xml_i="23259" xml_f="23278" txt_i="12234" txt_f="12253">circles. Left panel</offsets></italic><offsets xml_i="23287" xml_f="23312" txt_i="12253" txt_f="12278">, anti-correlated genes; </offsets><italic><offsets xml_i="23320" xml_f="23332" txt_i="12278" txt_f="12290">center panel</offsets></italic><offsets xml_i="23341" xml_f="23379" txt_i="12290" txt_f="12328">, genes in the KEGG melanoma pathway; </offsets><italic><offsets xml_i="23387" xml_f="23403" txt_i="12328" txt_f="12344">top right corner</offsets></italic><offsets xml_i="23412" xml_f="23440" txt_i="12344" txt_f="12372">, other cancer-related genes</offsets></p></caption><graphic xlink:href="13148_2015_92_Fig2_HTML" id="MO2"></graphic></fig></p></sec><sec id="Sec6"><title><offsets xml_i="23557" xml_f="23607" txt_i="12375" txt_f="12425">miR-331-3p distinguishes OS in metastatic melanoma</offsets></title><p><offsets xml_i="23618" xml_f="23675" txt_i="12426" txt_f="12483">Running T-ReCS with OS as the outcome variable, we found </offsets><italic><offsets xml_i="23683" xml_f="23693" txt_i="12483" txt_f="12493">miR-331-3p</offsets></italic><offsets xml_i="23702" xml_f="23787" txt_i="12493" txt_f="12578"> as the primary miRNA positively associated with OS, while its cluster also included </offsets><italic><offsets xml_i="23795" xml_f="23816" txt_i="12578" txt_f="12599">miR-93-5p. miR-331-3p</offsets></italic><offsets xml_i="23825" xml_f="23930" txt_i="12599" txt_f="12704"> was not differentially expressed based on the treatment arm (CP with sorafenib vs CP without sorafenib; </offsets><italic><offsets xml_i="23938" xml_f="23939" txt_i="12704" txt_f="12705">p</offsets></italic><offsets xml_i="23948" xml_f="23959" txt_i="12705" txt_f="12716"> = 0.706), </offsets><italic><offsets xml_i="23967" xml_f="23971" txt_i="12716" txt_f="12720">BRAF</offsets></italic><offsets xml_i="23980" xml_f="24019" txt_i="12720" txt_f="12759"> mutation status (mutant vs wild-type; </offsets><italic><offsets xml_i="24027" xml_f="24028" txt_i="12759" txt_f="12760">p</offsets></italic><offsets xml_i="24037" xml_f="24051" txt_i="12760" txt_f="12774"> = 0.1909) or </offsets><italic><offsets xml_i="24059" xml_f="24063" txt_i="12774" txt_f="12778">NRAS</offsets></italic><offsets xml_i="24072" xml_f="24111" txt_i="12778" txt_f="12817"> mutation status (mutant vs wild-type; </offsets><italic><offsets xml_i="24119" xml_f="24120" txt_i="12817" txt_f="12818">p</offsets></italic><offsets xml_i="24129" xml_f="24138" txt_i="12818" txt_f="12827"> = 0.52).</offsets></p></sec></sec><sec id="Sec7" sec-type="conclusion"><title><offsets xml_i="24198" xml_f="24209" txt_i="12830" txt_f="12841">Conclusions</offsets></title><p><offsets xml_i="24220" xml_f="24363" txt_i="12842" txt_f="12985">We have utilized a novel integrative analysis of miRNA and gene expression data to identify molecular biomarkers of PFS and response and found </offsets><italic><offsets xml_i="24371" xml_f="24381" txt_i="12985" txt_f="12995">miR-659-3p</offsets></italic><offsets xml_i="24390" xml_f="24575" txt_i="12995" txt_f="13180"> expression to be positively associated with longer PFS, distinguishing patients with disease that is responsive to CP-based therapy. Characterization of the functional significance of </offsets><italic><offsets xml_i="24583" xml_f="24593" txt_i="13180" txt_f="13190">miR-659-3p</offsets></italic><offsets xml_i="24602" xml_f="24752" txt_i="13190" txt_f="13340"> has been limited to the neuroscience literature, where it has been shown to bind a common genetic variant of and regulate the expression of the gene </offsets><italic><offsets xml_i="24760" xml_f="24771" txt_i="13340" txt_f="13351">progranulin</offsets></italic><offsets xml_i="24780" xml_f="24853" txt_i="13351" txt_f="13424">, which may significantly increase the risk for frontotemporal dementia [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="24886" xml_f="24888" txt_i="13424" txt_f="13426">35</offsets></xref><offsets xml_i="24895" xml_f="24929" txt_i="13426" txt_f="13460">]. The functional significance of </offsets><italic><offsets xml_i="24937" xml_f="24947" txt_i="13460" txt_f="13470">miR-659-3p</offsets></italic><offsets xml_i="24956" xml_f="25094" txt_i="13470" txt_f="13608"> in cancer and melanoma is as of yet unknown. Of particular significance in our study is the identification of gene targets predicted for </offsets><italic><offsets xml_i="25102" xml_f="25112" txt_i="13608" txt_f="13618">miR-659-3p</offsets></italic><offsets xml_i="25121" xml_f="25228" txt_i="13618" txt_f="13725"> and the construction of a proposed regulatory network which may lend insight into the mechanistic role of </offsets><italic><offsets xml_i="25236" xml_f="25246" txt_i="13725" txt_f="13735">miR-659-3p</offsets></italic><offsets xml_i="25255" xml_f="25306" txt_i="13735" txt_f="13786"> in response to cytotoxic chemotherapy in melanoma.</offsets></p><p><offsets xml_i="25313" xml_f="25345" txt_i="13787" txt_f="13819">Gene expression analysis showed </offsets><italic><offsets xml_i="25353" xml_f="25359" txt_i="13819" txt_f="13825">PLXNB1</offsets></italic><offsets xml_i="25368" xml_f="25410" txt_i="13825" txt_f="13867"> to be associated with foreshortened PFS. </offsets><italic><offsets xml_i="25418" xml_f="25424" txt_i="13867" txt_f="13873">PLXNB1</offsets></italic><offsets xml_i="25433" xml_f="25600" txt_i="13873" txt_f="14040"> has previously been shown to be lost in MM and in deeply invasive primary tumors and is strongly inhibited by MAP kinase signaling in melanoma cells and melanocytes [</offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="25633" xml_f="25635" txt_i="14040" txt_f="14042">27</offsets></xref><offsets xml_i="25642" xml_f="25644" txt_i="14042" txt_f="14044">, </offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="25677" xml_f="25679" txt_i="14044" txt_f="14046">28</offsets></xref><offsets xml_i="25686" xml_f="25913" txt_i="14046" txt_f="14273">]. In preclinical models, plexin B1 suppressed proliferation, enhanced migration, stimulated Akt activation and, of particular interest, has been reported to render melanoma cell lines resistant to cisplatin-induced apoptosis [</offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="25946" xml_f="25948" txt_i="14273" txt_f="14275">28</offsets></xref><offsets xml_i="25955" xml_f="26169" txt_i="14275" txt_f="14489">]. Plexin B1 has been shown to inhibit c-Met in response to its ligand hepatocyte growth factor (HGF) and is predicted to be a classic tumor suppressor protein in melanomas in which progression is c-Met dependent [</offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="26202" xml_f="26204" txt_i="14489" txt_f="14491">28</offsets></xref><offsets xml_i="26211" xml_f="26214" txt_i="14491" txt_f="14494">]. </offsets><italic><offsets xml_i="26222" xml_f="26227" txt_i="14494" txt_f="14499">RAD23</offsets></italic><offsets xml_i="26236" xml_f="26475" txt_i="14499" txt_f="14738"> was also identified in our model as associated with shorter PFS and is a component of the protein complex that specifically complements the nucleotide excision repair defect of xeroderma pigmentosum group C (XP-c) cell extracts in vitro [</offsets><xref ref-type="bibr" rid="CR36"><offsets xml_i="26508" xml_f="26510" txt_i="14738" txt_f="14740">36</offsets></xref><offsets xml_i="26517" xml_f="26533" txt_i="14740" txt_f="14756">]. Furthermore, </offsets><italic><offsets xml_i="26541" xml_f="26546" txt_i="14756" txt_f="14761">RAD23</offsets></italic><offsets xml_i="26555" xml_f="26606" txt_i="14761" txt_f="14812"> has been shown to be downregulated in glutathione </offsets><italic><offsets xml_i="26614" xml_f="26615" txt_i="14812" txt_f="14813">S</offsets></italic><offsets xml_i="26624" xml_f="26708" txt_i="14813" txt_f="14897">-transferase M1 (GSTM1)-null melanomas in patients reporting histories of sunburns [</offsets><xref ref-type="bibr" rid="CR37"><offsets xml_i="26741" xml_f="26743" txt_i="14897" txt_f="14899">37</offsets></xref><offsets xml_i="26750" xml_f="26921" txt_i="14899" txt_f="15070">]. RAD23 has been implicated as a biomarker of response to epigenetic therapy, in particular HDAC inhibition, through a regulatory circuit that involves RAD23B and HDAC6 [</offsets><xref ref-type="bibr" rid="CR38"><offsets xml_i="26954" xml_f="26956" txt_i="15070" txt_f="15072">38</offsets></xref><offsets xml_i="26963" xml_f="26965" txt_i="15072" txt_f="15074">, </offsets><xref ref-type="bibr" rid="CR39"><offsets xml_i="26998" xml_f="27000" txt_i="15074" txt_f="15076">39</offsets></xref><offsets xml_i="27007" xml_f="27009" txt_i="15076" txt_f="15078">].</offsets></p><p><offsets xml_i="27016" xml_f="27042" txt_i="15079" txt_f="15105">Predicted gene targets of </offsets><italic><offsets xml_i="27050" xml_f="27060" txt_i="15105" txt_f="15115">miR-659-3p</offsets></italic><offsets xml_i="27069" xml_f="27166" txt_i="15115" txt_f="15212"> selected in our analysis are genes important to the MAP kinase signaling pathway, in particular </offsets><italic><offsets xml_i="27174" xml_f="27178" txt_i="15212" txt_f="15216">BRAF</offsets></italic><offsets xml_i="27187" xml_f="27192" txt_i="15216" txt_f="15221"> and </offsets><italic><offsets xml_i="27200" xml_f="27204" txt_i="15221" txt_f="15225">NRAS</offsets></italic><offsets xml_i="27213" xml_f="27526" txt_i="15225" txt_f="15538">, and genes relevant to the PI3K/MTOR, FGFR, and RAC1 pathways, as well as genes involved in DNA repair. BRAF, a member of the RAF kinase family of serine/threonine protein kinases, is mutated in 40–50 % of melanomas and results most frequently from a valine (V) to glutamic acid (E) substitution at residue 600 (</offsets><italic><offsets xml_i="27534" xml_f="27538" txt_i="15538" txt_f="15542">BRAF</offsets></italic><sup><italic><offsets xml_i="27560" xml_f="27565" txt_i="15542" txt_f="15547">V600E</offsets></italic></sup><offsets xml_i="27580" xml_f="27792" txt_i="15547" txt_f="15759">), rendering mutant BRAF protein which no longer requires dimerization for its activity, and is highly predictive of therapeutic response to the specific BRAF inhibitors, vemurafenib and dabrafenib. Mutations in </offsets><italic><offsets xml_i="27800" xml_f="27804" txt_i="15759" txt_f="15763">NRAS</offsets></italic><offsets xml_i="27813" xml_f="27937" txt_i="15763" txt_f="15887"> occur in 15–20 % of melanomas and result in constitutive activation of the NRAS protein and enhanced MAP kinase signaling [</offsets><xref ref-type="bibr" rid="CR40"><offsets xml_i="27970" xml_f="27972" txt_i="15887" txt_f="15889">40</offsets></xref><offsets xml_i="27979" xml_f="27980" txt_i="15889" txt_f="15890">–</offsets><xref ref-type="bibr" rid="CR42"><offsets xml_i="28013" xml_f="28015" txt_i="15890" txt_f="15892">42</offsets></xref><offsets xml_i="28022" xml_f="28086" txt_i="15892" txt_f="15956">]. FGFR1 signaling has been implicated in melanoma progression [</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="28119" xml_f="28121" txt_i="15956" txt_f="15958">43</offsets></xref><offsets xml_i="28128" xml_f="28190" txt_i="15958" txt_f="16020">]; introduction of antisense oligonucleotides targeted toward </offsets><italic><offsets xml_i="28198" xml_f="28203" txt_i="16020" txt_f="16025">FGFR1</offsets></italic><offsets xml_i="28212" xml_f="28305" txt_i="16025" txt_f="16118"> into metastatic cell lines results in decreased proliferation and signs of differentiation [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="28338" xml_f="28340" txt_i="16118" txt_f="16120">44</offsets></xref><offsets xml_i="28347" xml_f="28349" txt_i="16120" txt_f="16122">, </offsets><xref ref-type="bibr" rid="CR45"><offsets xml_i="28382" xml_f="28384" txt_i="16122" txt_f="16124">45</offsets></xref><offsets xml_i="28391" xml_f="28424" txt_i="16124" txt_f="16157">]; and injection of an antisense </offsets><italic><offsets xml_i="28432" xml_f="28437" txt_i="16157" txt_f="16162">FGFR1</offsets></italic><offsets xml_i="28446" xml_f="28580" txt_i="16162" txt_f="16296"> construct into primary and metastatic melanomas grown in nude mice results in inhibition of tumor growth and induction of apoptosis [</offsets><xref ref-type="bibr" rid="CR46"><offsets xml_i="28613" xml_f="28615" txt_i="16296" txt_f="16298">46</offsets></xref><offsets xml_i="28622" xml_f="28624" txt_i="16298" txt_f="16300">, </offsets><xref ref-type="bibr" rid="CR47"><offsets xml_i="28657" xml_f="28659" txt_i="16300" txt_f="16302">47</offsets></xref><offsets xml_i="28666" xml_f="28791" txt_i="16302" txt_f="16427">]. RAC1, a member of the Rho family of GTPases, is involved in cellular proliferation, survival, motility, and invasiveness [</offsets><xref ref-type="bibr" rid="CR48"><offsets xml_i="28824" xml_f="28826" txt_i="16427" txt_f="16429">48</offsets></xref><offsets xml_i="28833" xml_f="28840" txt_i="16429" txt_f="16436">]. The </offsets><italic><offsets xml_i="28848" xml_f="28852" txt_i="16436" txt_f="16440">RAC1</offsets></italic><offsets xml_i="28861" xml_f="28992" txt_i="16440" txt_f="16571"> mutation resulting in the substitution of proline for serine at residue 29 is the most frequently mutated amino acid in MM, after </offsets><italic><offsets xml_i="29000" xml_f="29004" txt_i="16571" txt_f="16575">V600</offsets></italic><offsets xml_i="29013" xml_f="29027" txt_i="16575" txt_f="16589"> mutations in </offsets><italic><offsets xml_i="29035" xml_f="29039" txt_i="16589" txt_f="16593">BRAF</offsets></italic><offsets xml_i="29048" xml_f="29102" txt_i="16593" txt_f="16647"> and the glutamine (Q) substitution at position 61 in </offsets><italic><offsets xml_i="29110" xml_f="29120" txt_i="16647" txt_f="16657">NRAS. RAC1</offsets></italic><offsets xml_i="29129" xml_f="29268" txt_i="16657" txt_f="16796"> mutations are observed in 4 % of all melanomas and 9 % of sun-exposed melanomas and may occur either independently or in conjunction with </offsets><italic><offsets xml_i="29276" xml_f="29280" txt_i="16796" txt_f="16800">BRAF</offsets></italic><offsets xml_i="29289" xml_f="29294" txt_i="16800" txt_f="16805"> and </offsets><italic><offsets xml_i="29302" xml_f="29306" txt_i="16805" txt_f="16809">NRAS</offsets></italic><offsets xml_i="29315" xml_f="29418" txt_i="16809" txt_f="16912"> mutations, suggesting growth advantages distinct from activation of the MAP kinase signaling pathway [</offsets><xref ref-type="bibr" rid="CR49"><offsets xml_i="29451" xml_f="29453" txt_i="16912" txt_f="16914">49</offsets></xref><offsets xml_i="29460" xml_f="29462" txt_i="16914" txt_f="16916">, </offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="29495" xml_f="29497" txt_i="16916" txt_f="16918">50</offsets></xref><offsets xml_i="29504" xml_f="29763" txt_i="16918" txt_f="17177">]. RAC1 and other members of the Rho family of GTPases have been implicated as key modulators of sensitivity to DNA-damaging agents; in particular, chemical inhibition of RAC1 has been shown to sensitize drug-resistant ovarian cancer cell lines to cisplatin [</offsets><xref ref-type="bibr" rid="CR51"><offsets xml_i="29796" xml_f="29798" txt_i="17177" txt_f="17179">51</offsets></xref><offsets xml_i="29805" xml_f="29806" txt_i="17179" txt_f="17180">–</offsets><xref ref-type="bibr" rid="CR53"><offsets xml_i="29839" xml_f="29841" txt_i="17180" txt_f="17182">53</offsets></xref><offsets xml_i="29848" xml_f="29850" txt_i="17182" txt_f="17184">].</offsets></p><p><offsets xml_i="29857" xml_f="29887" txt_i="17185" txt_f="17215">While the association between </offsets><italic><offsets xml_i="29895" xml_f="29905" txt_i="17215" txt_f="17225">miR-659-3p</offsets></italic><offsets xml_i="29914" xml_f="30055" txt_i="17225" txt_f="17366"> and these predicted genes targets is intriguing, it is important to note that our gene expression data do not show strong correlations with </offsets><italic><offsets xml_i="30063" xml_f="30073" txt_i="17366" txt_f="17376">miR-659-3p</offsets></italic><offsets xml_i="30082" xml_f="30196" txt_i="17376" txt_f="17490"> and at this point, these findings are not confirmed. These findings do, however, suggest biological relevance of </offsets><italic><offsets xml_i="30204" xml_f="30214" txt_i="17490" txt_f="17500">miR-659-3p</offsets></italic><offsets xml_i="30223" xml_f="30401" txt_i="17500" txt_f="17678"> in the context of known oncogenic drivers in melanoma, an association which requires further investigation in future studies. We did find a significant anti-correlation between </offsets><italic><offsets xml_i="30409" xml_f="30419" txt_i="17678" txt_f="17688">miR-659-3p</offsets></italic><offsets xml_i="30428" xml_f="30455" txt_i="17688" txt_f="17715"> and a small set of genes. </offsets><italic><offsets xml_i="30463" xml_f="30467" txt_i="17715" txt_f="17719">NFIX</offsets></italic><offsets xml_i="30476" xml_f="30701" txt_i="17719" txt_f="17944">, which has been implicated in many developmental and biological processes and diseases, is one of them and has been shown to interact with known oncogenes. This interaction may represent a clue as to functional relevance of </offsets><italic><offsets xml_i="30709" xml_f="30719" txt_i="17944" txt_f="17954">miR-659-3p</offsets></italic><offsets xml_i="30728" xml_f="30761" txt_i="17954" txt_f="17987"> and requires further validation.</offsets></p><p><offsets xml_i="30768" xml_f="30812" txt_i="17988" txt_f="18032">Our study identified an association between </offsets><italic><offsets xml_i="30820" xml_f="30830" txt_i="18032" txt_f="18042">miR-659-3p</offsets></italic><offsets xml_i="30839" xml_f="30898" txt_i="18042" txt_f="18101"> and the clinical activity of CP-based therapy implicating </offsets><italic><offsets xml_i="30906" xml_f="30916" txt_i="18101" txt_f="18111">miR-659-3p</offsets></italic><offsets xml_i="30925" xml_f="31118" txt_i="18111" txt_f="18304"> as a candidate predictive biomarker in patients with metastatic melanoma. This analysis did not include untreated MM patients, and therefore, we are unable to evaluate the prognostic value of </offsets><italic><offsets xml_i="31126" xml_f="31136" txt_i="18304" txt_f="18314">miR-659-3p</offsets></italic><offsets xml_i="31145" xml_f="31204" txt_i="18314" txt_f="18373">, i.e., we are unable to assess if the association between </offsets><italic><offsets xml_i="31212" xml_f="31222" txt_i="18373" txt_f="18383">miR-659-3p</offsets></italic><offsets xml_i="31231" xml_f="32304" txt_i="18383" txt_f="19456"> and PFS persists in the absence of therapy. These data were generated from formalin-fixed paraffin-embedded (FFPE) legacy samples from a phase III multicenter clinical trial that represents a limitation of this study. Recovery of RNA from archival FFPE tissues can be challenging due to strand breakage and cross-linking by formalin fixation, which may inhibit polymerase in PCR-based molecular assays. We overcame this challenge by utilizing technology optimized for FFPE samples. Specifically, the miRNA expression assay used in this study allows for direct digital detection and counting of miRNAs in a single reaction without the need for an amplification step. Another limitation is that this bio-analysis represents a subset of the 823 patients enrolled on E2603. Archival tissue was not compulsory in E2603, and therefore, genotyping information is available for only 179 patients treated on this trial. It is notable that the patients included in this analysis were similar to the overall E2603 population and the population of patients for whom genotyping informa</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="32336" xml_f="32337" txt_i="19456" txt_f="19457">7</offsets></xref><offsets xml_i="32344" xml_f="32346" txt_i="19457" txt_f="19459">, </offsets><xref ref-type="bibr" rid="CR54"><offsets xml_i="32379" xml_f="32381" txt_i="19459" txt_f="19461">54</offsets></xref><offsets xml_i="32388" xml_f="32390" txt_i="19461" txt_f="19463">].</offsets></p><p><offsets xml_i="32397" xml_f="32438" txt_i="19464" txt_f="19505">This novel integrative analysis suggests </offsets><italic><offsets xml_i="32446" xml_f="32456" txt_i="19505" txt_f="19515">miR-659-3p</offsets></italic><offsets xml_i="32465" xml_f="32775" txt_i="19515" txt_f="19825"> as a candidate biomarker of response in MM patients treated with platinum-based chemotherapy and may serve to improve patient selection. To our knowledge, our study is the first to identify a miRNA species as a predictor of therapy outcome in metastatic melanoma. While our proposed regulatory network embeds </offsets><italic><offsets xml_i="32783" xml_f="32793" txt_i="19825" txt_f="19835">miR-659-3p</offsets></italic><offsets xml_i="32802" xml_f="33086" txt_i="19835" txt_f="20119"> in the landscape of melanoma signaling, functional studies are paramount to the validation of this miRNA as a mediator of melanoma signaling. The value of the proposed regulatory network is to inform upon pathways which should be interrogated as we continue to elucidate the role of </offsets><italic><offsets xml_i="33094" xml_f="33104" txt_i="20119" txt_f="20129">miR-659-3p</offsets></italic><offsets xml_i="33113" xml_f="33191" txt_i="20129" txt_f="20207"> as a predictive biomarker. Independent validation of the predictive value of </offsets><italic><offsets xml_i="33199" xml_f="33209" txt_i="20207" txt_f="20217">miR-659-3p</offsets></italic><offsets xml_i="33218" xml_f="33320" txt_i="20217" txt_f="20319"> is currently ongoing in an independent cohort of MM patients treated with carboplatin and paclitaxel.</offsets></p></sec><sec id="Sec8" sec-type="materials|methods"><title><offsets xml_i="33381" xml_f="33388" txt_i="20321" txt_f="20328">Methods</offsets></title><sec id="Sec9"><title><offsets xml_i="33418" xml_f="33443" txt_i="20329" txt_f="20354">Patients and study design</offsets></title><p><offsets xml_i="33454" xml_f="33714" txt_i="20355" txt_f="20615">E2603 enrolled patients with pathologically confirmed advanced unresectable or metastatic melanoma without brain involvement, who had no previous treatment with chemotherapeutic or MAPK pathway targeting agents. The study design has been previously described [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="33746" xml_f="33747" txt_i="20615" txt_f="20616">7</offsets></xref><offsets xml_i="33754" xml_f="34398" txt_i="20616" txt_f="21260">]. Key inclusion criteria were the presence of measurable disease by RECIST version 1.0, at least 18 years of age, ECOG performance status of 0 or 1, and satisfactory baseline organ function. Pre-treatment specimen collection was conducted as part of the clinical trial and analyzed with institutional review board approval. From the 823 patients enrolled on the E2603 clinical trial, 620 archival pre-treatment samples were available for analysis. One hundred and twenty pre-treatment formalin fixed paraffin-embedded (FFPE) tumor tissues from 115 patients were selected to ensure a balanced representation of patients based on treatment arm, </offsets><italic><offsets xml_i="34406" xml_f="34410" txt_i="21260" txt_f="21264">BRAF</offsets></italic><offsets xml_i="34419" xml_f="34530" txt_i="21264" txt_f="21375"> status, and clinical outcome (PFS less than 4 months vs greater than 4 months) were included in this analysis.</offsets></p></sec><sec id="Sec10"><title><offsets xml_i="34563" xml_f="34599" txt_i="21377" txt_f="21413">Treatment and assessment of response</offsets></title><p><offsets xml_i="34610" xml_f="34753" txt_i="21414" txt_f="21557">The patients were randomized in a double-blind fashion to receive CP, either with or without sorafenib. Paclitaxel was administered at 225 mg/m</offsets><sup><offsets xml_i="34758" xml_f="34759" txt_i="21557" txt_f="21558">2</offsets></sup><offsets xml_i="34765" xml_f="34936" txt_i="21558" txt_f="21729"> over 3 h followed by carboplatin AUC 6 IV over 30 min on day 1 of each 21-day cycle during cycles 1–4. During cycles 5 through 10, paclitaxel was administered at 175 mg/m</offsets><sup><offsets xml_i="34941" xml_f="34942" txt_i="21729" txt_f="21730">2</offsets></sup><offsets xml_i="34948" xml_f="35423" txt_i="21730" txt_f="22205"> IV over 3 h followed by carboplatin AUC 5 IV over 30 min. Sorafenib 400 mg or placebo was administered twice daily by mouth on days 2 through 19 of each 21-day cycle. Upon completion of 10 cycles of chemotherapy, sorafenib 400 mg twice daily by mouth or placebo was administered daily until disease progression or intolerable toxicity. Tumor response was assessed every 2 cycles during cycles 1 through 10 and then every 3 cycles. Response was defined by RECIST version 1.0.</offsets></p></sec><sec id="Sec11"><title><offsets xml_i="35456" xml_f="35474" txt_i="22207" txt_f="22225">Mutation profiling</offsets></title><p><offsets xml_i="35485" xml_f="35675" txt_i="22226" txt_f="22416">Genotyping was performed using a custom iPlex Sequenom panel interrogating 74 mutations in 13 genes in the Perelman School of Medicine Molecular Profiling Facility, as previously described [</offsets><xref ref-type="bibr" rid="CR54"><offsets xml_i="35708" xml_f="35710" txt_i="22416" txt_f="22418">54</offsets></xref><offsets xml_i="35717" xml_f="35719" txt_i="22418" txt_f="22420">].</offsets></p></sec><sec id="Sec12"><title><offsets xml_i="35752" xml_f="35766" txt_i="22422" txt_f="22436">RNA extraction</offsets></title><p><offsets xml_i="35777" xml_f="36111" txt_i="22437" txt_f="22771">Total RNA was isolated from melanoma tissues using the PerfectPure RNA Tissue kit (5Prime Inc., MD, USA) at the University of Pittsburgh Cancer Institute (UPCI). RNA was quantified using Ribogreen RNA quantitation Kit (Molecular Probes, Eugene OR). RNA quality was also evaluated by RNA integrity number using the Agilent Bioanalyzer.</offsets></p></sec><sec id="Sec13"><title><offsets xml_i="36144" xml_f="36210" txt_i="22773" txt_f="22839">miRNA microarray expression profiling and gene expression analysis</offsets></title><p><offsets xml_i="36221" xml_f="37280" txt_i="22840" txt_f="23899">Profiling of miRNA levels was performed at the UPCI utilizing technology optimized for FFPE samples: Nanostring technology (Ncounter Human v2 miRNA Expression Assay) for miRNA analysis of 800 human miRNAs was derived from miRBase v.18. Total RNA was used as input for nCounter miRNA sample preparation reactions. All sample preparation and processing were performed according to the manufacturer’s protocol. Following ligation, sample preparation reactions were purified and diluted, and hybridization reactions were performed by incubation at 65 °C for a minimum of 18 h. Hybridized probes were purified and counted on the nCounter Prep Station and Digital Analyzer (NanoString) following the manufacturer’s instructions. For each assay, a high-density scan (600 fields of view) was performed. Whole-genome gene expression analysis was also carried out using DNA-mediated annealing, selection, extension, and ligation (DASL)/Illumina HT-12 v4 Expression BeadChip, a technology optimized for FFPE samples, at the UPCI according to the manufacturer’s protocol.</offsets></p></sec><sec id="Sec14"><title><offsets xml_i="37313" xml_f="37386" txt_i="23901" txt_f="23974">Statistical analysis for mRNA and miRNA biomarkers predictive of survival</offsets></title><p><offsets xml_i="37397" xml_f="37487" txt_i="23975" txt_f="24065">We performed feature selection at the cluster level utilizing a novel algorithm (T-ReCS) [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="37519" xml_f="37520" txt_i="24065" txt_f="24066">1</offsets></xref><offsets xml_i="37527" xml_f="37528" txt_i="24066" txt_f="24067">]</offsets><italic><offsets xml_i="37536" xml_f="37537" txt_i="24067" txt_f="24068">.</offsets></italic><offsets xml_i="37546" xml_f="38452" txt_i="24068" txt_f="24974"> Briefly, starting from a standard feature selection method against a target variable (in our case, PFS or OS), T-ReCS expands the single features into clusters of features according to two statistical criteria that maintain the prediction accuracy. The clusters are defined dynamically as groups of similarly expressed genes. T-ReCS works in two steps. First, it partitions the feature space into a tree structure with the root being the set of all variables (mRNA or miRNA genes, respectively), while each leaf is a single variable. Each internal node in the tree represents a cluster of variables, its descendants, whose values (expression) are similar. The lower in the tree a node is, the more similar the patterns of its members are. For our implementation, we use the tree produced by our Recursive K-means Spectral Clustering (ReKS), which was previously developed for heterogeneous clinical data [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="38485" xml_f="38487" txt_i="24974" txt_f="24976">29</offsets></xref><offsets xml_i="38494" xml_f="38910" txt_i="24976" txt_f="25392">]. The second step of T-ReCS is feature selection operating on the tree structure iteratively. Starting with a set of variables (tree leafs) selected with any feature selection method, it ascends the tree one level at a time and statistically assesses whether the group variables (internal nodes) retain the same predictive properties. For statistical assessment, T-ReCS uses two conditional independence tests with </offsets><italic><offsets xml_i="38918" xml_f="38919" txt_i="25392" txt_f="25393">p</offsets></italic><offsets xml_i="38928" xml_f="39044" txt_i="25393" txt_f="25509"> value thresholds determined by cross-validation. For survival target variables, T-ReCS uses survival MMPC (sMMPC) [</offsets><xref ref-type="bibr" rid="CR55"><offsets xml_i="39077" xml_f="39079" txt_i="25509" txt_f="25511">55</offsets></xref><offsets xml_i="39086" xml_f="39237" txt_i="25511" txt_f="25662">] for single-variable selection. The association between selected genes and survival was tested using Cox regression (Cox proportional hazards model) [</offsets><xref ref-type="bibr" rid="CR56"><offsets xml_i="39270" xml_f="39272" txt_i="25662" txt_f="25664">56</offsets></xref><offsets xml_i="39279" xml_f="39281" txt_i="25664" txt_f="25666">].</offsets></p></sec><sec id="Sec15"><title><offsets xml_i="39314" xml_f="39380" txt_i="25668" txt_f="25734">Regulatory associations between predictive mRNA and miRNA features</offsets></title><p><offsets xml_i="39391" xml_f="39593" txt_i="25735" txt_f="25937">A regulatory network was constructed from the mRNA and miRNA genes and their clusters that were selected as characteristic of PFS. The mRNA targets of miRNAs were predicted using TargetScan Human v6.2 [</offsets><xref ref-type="bibr" rid="CR57"><offsets xml_i="39626" xml_f="39628" txt_i="25937" txt_f="25939">57</offsets></xref><offsets xml_i="39635" xml_f="39771" txt_i="25939" txt_f="26075">], while transcription factor (TF)→gene and TF→miRNA regulatory interactions were obtained from the prior (static) network of mirConnX [</offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="39803" xml_f="39804" txt_i="26075" txt_f="26076">2</offsets></xref><offsets xml_i="39811" xml_f="40143" txt_i="26076" txt_f="26408">]. mirConnX is a user-friendly web interface for inferring, displaying, and parsing mRNA and miRNA gene regulatory networks. mirConnX infers regulatory networks by combining expression correlation values (dynamic network) with regulatory interactions obtained from the literature and from computational predictions (static network).</offsets></p></sec></sec></body><back><glossary><title>Abbreviations</title><def-list><def-item><term>CP</term><def><p>carboplatin/Paclitaxel</p></def></def-item><def-item><term>FFPE</term><def><p>formalin-fixed paraffin-embedded</p></def></def-item><def-item><term>MM</term><def><p>metastatic melanoma</p></def></def-item><def-item><term>OS</term><def><p>overall survival</p></def></def-item><def-item><term>PFS</term><def><p>progression-free survival</p></def></def-item></def-list></glossary><fn-group><fn><p>Liza C. Villaruz and Grace Huang contributed equally to this work.</p></fn><fn><p><bold>Competing interests</bold></p><p>Liza C. Villaruz, Grace Huang, Marjorie Romkes, Shama Buch, Tomoko Nukui, Keith Flaherty, Sandra Lee, Melissa A. Wilson, Katherine L. Nathanson, Panayiotis V. Beno, and Hussein A. Tawbi declare that they have no competing interests. John Kirkwood reports personal fees from BMS, Merck, GSK, Celgene, Vical , and Ziopharm.</p></fn><fn><p><bold>Authors’ contributions</bold></p><p>LCV is responsible for the concept, design, and conduct of the study and manuscript preparation. GH performed the bioinformatics analysis of the mRNA and microRNA expression data in the lab of PVB. MR who performed the gene expression and microRNA profiling. JMK is the Chairman of the Melanoma Committee of ECOG-ACRIN. SCB performed the gene expression and microRNA profiling in MR’s lab. TN performed the gene expression and microRNA profiling in MR’s lab. KTF is the principal investigator of ECOG-ACRIN Cancer Research Group intergroup trial E2603. SJL is the ECOG-ACRIN biostatistician responsible for the E2603 clinical data. MAW performed mutation profiling in the lab of KLN. KLN performed mutation profiling. PVB performed the bioinformatics analysis of the mRNA and microRNA expression data. HAT provided oversight and mentorship in the concept, design, and conduct of the study and manuscript preparation. All authors read and approved the final manuscript.</p></fn></fn-group><ack><title>Acknowledgements</title><p>This study was conducted by the ECOG-ACRIN Cancer Research Group (Robert L. Comis, MD and Mitchell D. Schnall, MD, PhD, Group Co-Chairs) and supported in part by Public Health Service Grants CA180820, CA180794, CA180844, CA15488, and CA18086; the UPCI Skin Cancer SPORE (P50 CA121973-02) with the UPCI Cancer Biomarkers Facility (P30CA047904); the National Institutes of Health grant (R01LM012087) and the Conquer Cancer Foundation of ASCO Young Investigator Award and the National Cancer Institute, National Institutes of Health, and the Department of Health and Human Services. Its content is solely the responsibility of the authors and does not necessarily represent the official views of the National Cancer Institute, National Institutes of Health, and the Department of Health and Human Services.</p></ack><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><mixed-citation publication-type="other">Huang GT, Tsamardinos I, Raghu V, Kaminski N, Benos PV. T-ReCS: stable selection of dynamically formed groups of features with application to prediction of clinical outcomes. In: Pacific Symposium in Biocomputing. USA: Big Island, Hawai'i; 2015</mixed-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Huang</surname><given-names>GT</given-names></name><name><surname>Athanassiou</surname><given-names>C</given-names></name><name><surname>Benos</surname><given-names>PV</given-names></name></person-group><article-title>mirConnX: condition-specific mRNA-microRNA network integrator</article-title><source>Nucleic Acids Res.</source><year>2011</year><volume>39</volume><fpage>W416</fpage><lpage>23</lpage><pub-id pub-id-type="doi">10.1093/nar/gkr276</pub-id><pub-id pub-id-type="pmid">21558324</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cummins</surname><given-names>D</given-names></name><name><surname>Cummins</surname><given-names>JM</given-names></name><name><surname>Pantle</surname><given-names>H</given-names></name><name><surname>Silverman</surname><given-names>MA</given-names></name><name><surname>Leonard</surname><given-names>AL</given-names></name></person-group><article-title>Cutaneous malignant melanoma</article-title><source>Mayo Clin Proc.</source><year>2006</year><volume>81</volume><fpage>500</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.4065/81.4.500</pub-id><pub-id pub-id-type="pmid">16610570</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Barth</surname><given-names>A</given-names></name><name><surname>Wanek</surname><given-names>LA</given-names></name><name><surname>Morton</surname><given-names>DL</given-names></name></person-group><article-title>Prognostic factors in 1,521 melanoma patients with distant metastases</article-title><source>J Am Coll Surg.</source><year>1995</year><volume>181</volume><fpage>193</fpage><lpage>201</lpage><pub-id pub-id-type="pmid">7670677</pub-id></element-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Middleton</surname><given-names>MR</given-names></name><name><surname>Grob</surname><given-names>JJ</given-names></name><name><surname>Aaronson</surname><given-names>N</given-names></name><name><surname>Fierlbeck</surname><given-names>G</given-names></name><name><surname>Tilgen</surname><given-names>W</given-names></name><name><surname>Seiter</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Randomized phase III study of temozolomide versus dacarbazine in the treatment of patients with advanced metastatic malignant melanoma</article-title><source>J Clin Oncol.</source><year>2000</year><volume>18</volume><fpage>158</fpage><lpage>66</lpage><pub-id pub-id-type="pmid">10623706</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Patel</surname><given-names>PM</given-names></name><name><surname>Suciu</surname><given-names>S</given-names></name><name><surname>Mortier</surname><given-names>L</given-names></name><name><surname>Kruit</surname><given-names>WH</given-names></name><name><surname>Robert</surname><given-names>C</given-names></name><name><surname>Schadendorf</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Extended schedule, escalated dose temozolomide versus dacarbazine in stage IV melanoma: final results of a randomised phase III study (EORTC 18032)</article-title><source>Eur J Cancer.</source><year>2011</year><volume>47</volume><fpage>1476</fpage><lpage>83</lpage><pub-id pub-id-type="doi">10.1016/j.ejca.2011.04.030</pub-id><pub-id pub-id-type="pmid">21600759</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Flaherty KT</surname><given-names>LS</given-names></name><name><surname>Schuchter</surname><given-names>LM</given-names></name><name><surname>Flaherty</surname><given-names>LE</given-names></name><name><surname>Wright</surname><given-names>JJ</given-names></name><name><surname>Leming</surname><given-names>PD</given-names></name><name><surname>Kirkwood</surname><given-names>JM</given-names></name></person-group><article-title>Final results of E2603: a double-blind, randomized phase III trial comparing carboplatin/paclitaxel with or without sorafenib in metastatic melanoma</article-title><source>J Clin Oncol</source><year>2010</year><volume>28</volume><fpage>Abstract 8511</fpage></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hauschild</surname><given-names>A</given-names></name><name><surname>Agarwala</surname><given-names>SS</given-names></name><name><surname>Trefzer</surname><given-names>U</given-names></name><name><surname>Hogg</surname><given-names>D</given-names></name><name><surname>Robert</surname><given-names>C</given-names></name><name><surname>Hersey</surname><given-names>P</given-names></name><etal></etal></person-group><article-title>Results of a phase III, randomized, placebo-controlled study of sorafenib in combination with carboplatin and paclitaxel as second-line treatment in patients with unresectable stage III or stage IV melanoma</article-title><source>J Clin Oncol.</source><year>2009</year><volume>27</volume><fpage>2823</fpage><lpage>30</lpage><pub-id pub-id-type="doi">10.1200/JCO.2007.15.7636</pub-id><pub-id pub-id-type="pmid">19349552</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hodi</surname><given-names>FS</given-names></name><name><surname>O'Day</surname><given-names>SJ</given-names></name><name><surname>McDermott</surname><given-names>DF</given-names></name><name><surname>Weber</surname><given-names>RW</given-names></name><name><surname>Sosman</surname><given-names>JA</given-names></name><name><surname>Haanen</surname><given-names>JB</given-names></name><etal></etal></person-group><article-title>Improved survival with ipilimumab in patients with metastatic melanoma</article-title><source>N Engl J Med.</source><year>2010</year><volume>363</volume><fpage>711</fpage><lpage>23</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa1003466</pub-id><pub-id pub-id-type="pmid">20525992</pub-id></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hamid</surname><given-names>O</given-names></name><name><surname>Robert</surname><given-names>C</given-names></name><name><surname>Daud</surname><given-names>A</given-names></name><name><surname>Hodi</surname><given-names>FS</given-names></name><name><surname>Hwu</surname><given-names>WJ</given-names></name><name><surname>Kefford</surname><given-names>R</given-names></name><etal></etal></person-group><article-title>Safety and tumor responses with lambrolizumab (anti-PD-1) in melanoma</article-title><source>N Engl J Med.</source><year>2013</year><volume>369</volume><fpage>134</fpage><lpage>44</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa1305133</pub-id><pub-id pub-id-type="pmid">23724846</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Topalian</surname><given-names>SL</given-names></name><name><surname>Hodi</surname><given-names>FS</given-names></name><name><surname>Brahmer</surname><given-names>JR</given-names></name><name><surname>Gettinger</surname><given-names>SN</given-names></name><name><surname>Smith</surname><given-names>DC</given-names></name><name><surname>McDermott</surname><given-names>DF</given-names></name><etal></etal></person-group><article-title>Safety, activity, and immune correlates of anti-PD-1 antibody in cancer</article-title><source>N Engl J Med.</source><year>2012</year><volume>366</volume><fpage>2443</fpage><lpage>54</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa1200690</pub-id><pub-id pub-id-type="pmid">22658127</pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Brahmer</surname><given-names>JR</given-names></name><name><surname>Tykodi</surname><given-names>SS</given-names></name><name><surname>Chow</surname><given-names>LQ</given-names></name><name><surname>Hwu</surname><given-names>WJ</given-names></name><name><surname>Topalian</surname><given-names>SL</given-names></name><name><surname>Hwu</surname><given-names>P</given-names></name><etal></etal></person-group><article-title>Safety and activity of anti-PD-L1 antibody in patients with advanced cancer</article-title><source>N Engl J Med.</source><year>2012</year><volume>366</volume><fpage>2455</fpage><lpage>65</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa1200694</pub-id><pub-id pub-id-type="pmid">22658128</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chapman</surname><given-names>PB</given-names></name><name><surname>Hauschild</surname><given-names>A</given-names></name><name><surname>Robert</surname><given-names>C</given-names></name><name><surname>Haanen</surname><given-names>JB</given-names></name><name><surname>Ascierto</surname><given-names>P</given-names></name><name><surname>Larkin</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Improved survival with vemurafenib in melanoma with BRAF V600E mutation</article-title><source>N Engl J Med.</source><year>2011</year><volume>364</volume><fpage>2507</fpage><lpage>16</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa1103782</pub-id><pub-id pub-id-type="pmid">21639808</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hauschild</surname><given-names>A</given-names></name><name><surname>Grob</surname><given-names>JJ</given-names></name><name><surname>Demidov</surname><given-names>LV</given-names></name><name><surname>Jouary</surname><given-names>T</given-names></name><name><surname>Gutzmer</surname><given-names>R</given-names></name><name><surname>Millward</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Dabrafenib in BRAF-mutated metastatic melanoma: a multicentre, open-label, phase 3 randomised controlled trial</article-title><source>Lancet.</source><year>2012</year><volume>380</volume><fpage>358</fpage><lpage>65</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(12)60868-X</pub-id><pub-id pub-id-type="pmid">22735384</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Davalos</surname><given-names>V</given-names></name><name><surname>Esteller</surname><given-names>M</given-names></name></person-group><article-title>MicroRNAs and cancer epigenetics: a macrorevolution</article-title><source>Curr Opin Oncol.</source><year>2010</year><volume>22</volume><fpage>35</fpage><lpage>45</lpage><pub-id pub-id-type="doi">10.1097/CCO.0b013e328333dcbb</pub-id><pub-id pub-id-type="pmid">19907325</pub-id></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Calin</surname><given-names>GA</given-names></name><name><surname>Sevignani</surname><given-names>C</given-names></name><name><surname>Dumitru</surname><given-names>CD</given-names></name><name><surname>Hyslop</surname><given-names>T</given-names></name><name><surname>Noch</surname><given-names>E</given-names></name><name><surname>Yendamuri</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers</article-title><source>Proc Natl Acad Sci U S A.</source><year>2004</year><volume>101</volume><fpage>2999</fpage><lpage>3004</lpage><pub-id pub-id-type="doi">10.1073/pnas.0307323101</pub-id><pub-id pub-id-type="pmid">14973191</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Johnson</surname><given-names>SM</given-names></name><name><surname>Grosshans</surname><given-names>H</given-names></name><name><surname>Shingara</surname><given-names>J</given-names></name><name><surname>Byrom</surname><given-names>M</given-names></name><name><surname>Jarvis</surname><given-names>R</given-names></name><name><surname>Cheng</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>RAS is regulated by the let-7 microRNA family</article-title><source>Cell.</source><year>2005</year><volume>120</volume><fpage>635</fpage><lpage>47</lpage><pub-id pub-id-type="doi">10.1016/j.cell.2005.01.014</pub-id><pub-id pub-id-type="pmid">15766527</pub-id></element-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Calin</surname><given-names>GA</given-names></name><name><surname>Dumitru</surname><given-names>CD</given-names></name><name><surname>Shimizu</surname><given-names>M</given-names></name><name><surname>Bichi</surname><given-names>R</given-names></name><name><surname>Zupo</surname><given-names>S</given-names></name><name><surname>Noch</surname><given-names>E</given-names></name><etal></etal></person-group><article-title>Frequent deletions and down-regulation of micro-RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia</article-title><source>Proc Natl Acad Sci U S A.</source><year>2002</year><volume>99</volume><fpage>15524</fpage><lpage>9</lpage><pub-id pub-id-type="doi">10.1073/pnas.242606799</pub-id><pub-id pub-id-type="pmid">12434020</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tagawa</surname><given-names>H</given-names></name><name><surname>Seto</surname><given-names>M</given-names></name></person-group><article-title>A microRNA cluster as a target of genomic amplification in malignant lymphoma</article-title><source>Leukemia.</source><year>2005</year><volume>19</volume><fpage>2013</fpage><lpage>6</lpage><pub-id pub-id-type="doi">10.1038/sj.leu.2403942</pub-id><pub-id pub-id-type="pmid">16167061</pub-id></element-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hayashita</surname><given-names>Y</given-names></name><name><surname>Osada</surname><given-names>H</given-names></name><name><surname>Tatematsu</surname><given-names>Y</given-names></name><name><surname>Yamada</surname><given-names>H</given-names></name><name><surname>Yanagisawa</surname><given-names>K</given-names></name><name><surname>Tomida</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>A polycistronic microRNA cluster, miR-17-92, is overexpressed in human lung cancers and enhances cell proliferation</article-title><source>Cancer Res.</source><year>2005</year><volume>65</volume><fpage>9628</fpage><lpage>32</lpage><pub-id pub-id-type="doi">10.1158/0008-5472.CAN-05-2352</pub-id><pub-id pub-id-type="pmid">16266980</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Philippidou</surname><given-names>D</given-names></name><name><surname>Schmitt</surname><given-names>M</given-names></name><name><surname>Moser</surname><given-names>D</given-names></name><name><surname>Margue</surname><given-names>C</given-names></name><name><surname>Nazarov</surname><given-names>PV</given-names></name><name><surname>Muller</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Signatures of microRNAs and selected microRNA target genes in human melanoma</article-title><source>Cancer Res.</source><year>2010</year><volume>70</volume><fpage>4163</fpage><lpage>73</lpage><pub-id pub-id-type="doi">10.1158/0008-5472.CAN-09-4512</pub-id><pub-id pub-id-type="pmid">20442294</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Poliseno</surname><given-names>L</given-names></name><name><surname>Haimovic</surname><given-names>A</given-names></name><name><surname>Segura</surname><given-names>MF</given-names></name><name><surname>Hanniford</surname><given-names>D</given-names></name><name><surname>Christos</surname><given-names>PJ</given-names></name><name><surname>Darvishian</surname><given-names>F</given-names></name><etal></etal></person-group><article-title>Histology-specific microRNA alterations in melanoma</article-title><source>J Invest Dermatol.</source><year>2012</year><volume>132</volume><fpage>1860</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1038/jid.2011.451</pub-id><pub-id pub-id-type="pmid">22551973</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Xu</surname><given-names>Y</given-names></name><name><surname>Brenn</surname><given-names>T</given-names></name><name><surname>Brown</surname><given-names>ER</given-names></name><name><surname>Doherty</surname><given-names>V</given-names></name><name><surname>Melton</surname><given-names>DW</given-names></name></person-group><article-title>Differential expression of microRNAs during melanoma progression: miR-200c, miR-205 and miR-211 are downregulated in melanoma and act as tumour suppressors</article-title><source>Br J Cancer.</source><year>2012</year><volume>106</volume><fpage>553</fpage><lpage>61</lpage><pub-id pub-id-type="doi">10.1038/bjc.2011.568</pub-id><pub-id pub-id-type="pmid">22223089</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kozubek</surname><given-names>J</given-names></name><name><surname>Ma</surname><given-names>Z</given-names></name><name><surname>Fleming</surname><given-names>E</given-names></name><name><surname>Duggan</surname><given-names>T</given-names></name><name><surname>Wu</surname><given-names>R</given-names></name><name><surname>Shin</surname><given-names>DG</given-names></name><etal></etal></person-group><article-title>In-depth characterization of microRNA transcriptome in melanoma</article-title><source>PLoS One.</source><year>2013</year><volume>8</volume><pub-id pub-id-type="doi">10.1371/journal.pone.0072699</pub-id><pub-id pub-id-type="pmid">24023765</pub-id></element-citation></ref><ref id="CR25"><label>25.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Glud</surname><given-names>M</given-names></name><name><surname>Rossing</surname><given-names>M</given-names></name><name><surname>Hother</surname><given-names>C</given-names></name><name><surname>Holst</surname><given-names>L</given-names></name><name><surname>Hastrup</surname><given-names>N</given-names></name><name><surname>Nielsen</surname><given-names>FC</given-names></name><etal></etal></person-group><article-title>Downregulation of miR-125b in metastatic cutaneous malignant melanoma</article-title><source>Melanoma Res.</source><year>2010</year><volume>20</volume><fpage>479</fpage><lpage>84</lpage><pub-id pub-id-type="doi">10.1097/CMR.0b013e32833e32a1</pub-id><pub-id pub-id-type="pmid">20827223</pub-id></element-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Segura</surname><given-names>MF</given-names></name><name><surname>Belitskaya-Levy</surname><given-names>I</given-names></name><name><surname>Rose</surname><given-names>AE</given-names></name><name><surname>Zakrzewski</surname><given-names>J</given-names></name><name><surname>Gaziel</surname><given-names>A</given-names></name><name><surname>Hanniford</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Melanoma microRNA signature predicts post-recurrence survival</article-title><source>Clin Cancer Res.</source><year>2010</year><volume>16</volume><fpage>1577</fpage><lpage>86</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-09-2721</pub-id><pub-id pub-id-type="pmid">20179230</pub-id></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Argast</surname><given-names>GM</given-names></name><name><surname>Croy</surname><given-names>CH</given-names></name><name><surname>Couts</surname><given-names>KL</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Litman</surname><given-names>E</given-names></name><name><surname>Chan</surname><given-names>DC</given-names></name><etal></etal></person-group><article-title>Plexin B1 is repressed by oncogenic B-Raf signaling and functions as a tumor suppressor in melanoma cells</article-title><source>Oncogene.</source><year>2009</year><volume>28</volume><fpage>2697</fpage><lpage>709</lpage><pub-id pub-id-type="doi">10.1038/onc.2009.133</pub-id><pub-id pub-id-type="pmid">19483722</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stevens</surname><given-names>L</given-names></name><name><surname>McClelland</surname><given-names>L</given-names></name><name><surname>Fricke</surname><given-names>A</given-names></name><name><surname>Williamson</surname><given-names>M</given-names></name><name><surname>Kuo</surname><given-names>I</given-names></name><name><surname>Scott</surname><given-names>G</given-names></name></person-group><article-title>Plexin B1 suppresses c-Met in melanoma: a role for plexin B1 as a tumor-suppressor protein through regulation of c-Met</article-title><source>J Invest Dermatol.</source><year>2010</year><volume>130</volume><fpage>1636</fpage><lpage>45</lpage><pub-id pub-id-type="doi">10.1038/jid.2010.13</pub-id><pub-id pub-id-type="pmid">20164843</pub-id></element-citation></ref><ref id="CR29"><label>29.</label><mixed-citation publication-type="other">Huang GT, Cunningham KI, Benos PV, Chennubhotla CS. Spectral clustering strategies for heterogeneous disease expression data. Pac Symp Biocomput. 2013;212–223.</mixed-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Holmfeldt</surname><given-names>P</given-names></name><name><surname>Pardieck</surname><given-names>J</given-names></name><name><surname>Saulsberry</surname><given-names>AC</given-names></name><name><surname>Nandakumar</surname><given-names>SK</given-names></name><name><surname>Finkelstein</surname><given-names>D</given-names></name><name><surname>Gray</surname><given-names>JT</given-names></name><etal></etal></person-group><article-title>Nfix is a novel regulator of murine hematopoietic stem and progenitor cell survival</article-title><source>Blood.</source><year>2013</year><volume>122</volume><fpage>2987</fpage><lpage>96</lpage><pub-id pub-id-type="doi">10.1182/blood-2013-04-493973</pub-id><pub-id pub-id-type="pmid">24041575</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Martynoga</surname><given-names>B</given-names></name><name><surname>Mateo</surname><given-names>JL</given-names></name><name><surname>Zhou</surname><given-names>B</given-names></name><name><surname>Andersen</surname><given-names>J</given-names></name><name><surname>Achimastou</surname><given-names>A</given-names></name><name><surname>Urban</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>Epigenomic enhancer annotation reveals a key role for NFIX in neural stem cell quiescence</article-title><source>Genes Dev.</source><year>2013</year><volume>27</volume><fpage>1769</fpage><lpage>86</lpage><pub-id pub-id-type="doi">10.1101/gad.216804.113</pub-id><pub-id pub-id-type="pmid">23964093</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Dixon</surname><given-names>C</given-names></name><name><surname>Harvey</surname><given-names>TJ</given-names></name><name><surname>Smith</surname><given-names>AG</given-names></name><name><surname>Gronostajski</surname><given-names>RM</given-names></name><name><surname>Bailey</surname><given-names>TL</given-names></name><name><surname>Piper</surname><given-names>M</given-names></name></person-group><article-title>Nuclear factor one X regulates Bobby sox during development of the mouse forebrain</article-title><source>Cell Mol Neurobiol.</source><year>2013</year><volume>33</volume><fpage>867</fpage><lpage>73</lpage><pub-id pub-id-type="doi">10.1007/s10571-013-9961-4</pub-id><pub-id pub-id-type="pmid">23852417</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Pistocchi</surname><given-names>A</given-names></name><name><surname>Gaudenzi</surname><given-names>G</given-names></name><name><surname>Foglia</surname><given-names>E</given-names></name><name><surname>Monteverde</surname><given-names>S</given-names></name><name><surname>Moreno-Fortuny</surname><given-names>A</given-names></name><name><surname>Pianca</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Conserved and divergent functions of Nfix in skeletal muscle development during vertebrate evolution</article-title><source>Development.</source><year>2013</year><volume>140</volume><fpage>1528</fpage><lpage>36</lpage><pub-id pub-id-type="doi">10.1242/dev.076315</pub-id><pub-id pub-id-type="pmid">23482488</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ravichandran</surname><given-names>V</given-names></name><name><surname>Sabath</surname><given-names>BF</given-names></name><name><surname>Jensen</surname><given-names>PN</given-names></name><name><surname>Houff</surname><given-names>SA</given-names></name><name><surname>Major</surname><given-names>EO</given-names></name></person-group><article-title>Interactions between c-Jun, nuclear factor 1, and JC virus promoter sequences: implications for viral tropism</article-title><source>J Virol.</source><year>2006</year><volume>80</volume><fpage>10506</fpage><lpage>13</lpage><pub-id pub-id-type="doi">10.1128/JVI.01355-06</pub-id><pub-id pub-id-type="pmid">16928756</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rademakers</surname><given-names>R</given-names></name><name><surname>Eriksen</surname><given-names>JL</given-names></name><name><surname>Baker</surname><given-names>M</given-names></name><name><surname>Robinson</surname><given-names>T</given-names></name><name><surname>Ahmed</surname><given-names>Z</given-names></name><name><surname>Lincoln</surname><given-names>SJ</given-names></name><etal></etal></person-group><article-title>Common variation in the miR-659 binding-site of GRN is a major risk factor for TDP43-positive frontotemporal dementia</article-title><source>Hum Mol Genet.</source><year>2008</year><volume>17</volume><fpage>3631</fpage><lpage>42</lpage><pub-id pub-id-type="doi">10.1093/hmg/ddn257</pub-id><pub-id pub-id-type="pmid">18723524</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Budden</surname><given-names>T</given-names></name><name><surname>Bowden</surname><given-names>NA</given-names></name></person-group><article-title>The role of altered nucleotide excision repair and UVB-induced DNA damage in melanomagenesis</article-title><source>Int J Mol Sci.</source><year>2013</year><volume>14</volume><fpage>1132</fpage><lpage>51</lpage><pub-id pub-id-type="doi">10.3390/ijms14011132</pub-id><pub-id pub-id-type="pmid">23303275</pub-id></element-citation></ref><ref id="CR37"><label>37.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Steinberg</surname><given-names>ML</given-names></name><name><surname>Hubbard</surname><given-names>K</given-names></name><name><surname>Utti</surname><given-names>C</given-names></name><name><surname>Clas</surname><given-names>B</given-names></name><name><surname>Hwang</surname><given-names>BJ</given-names></name><name><surname>Hill</surname><given-names>HZ</given-names></name><etal></etal></person-group><article-title>Patterns of persistent DNA damage associated with sun exposure and the glutathione S-transferase M1 genotype in melanoma patients</article-title><source>Photochem Photobiol.</source><year>2009</year><volume>85</volume><fpage>379</fpage><lpage>86</lpage><pub-id pub-id-type="doi">10.1111/j.1751-1097.2008.00455.x</pub-id><pub-id pub-id-type="pmid">19161404</pub-id></element-citation></ref><ref id="CR38"><label>38.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>New</surname><given-names>M</given-names></name><name><surname>Olzscha</surname><given-names>H</given-names></name><name><surname>Liu</surname><given-names>G</given-names></name><name><surname>Khan</surname><given-names>O</given-names></name><name><surname>Stimson</surname><given-names>L</given-names></name><name><surname>McGouran</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>A regulatory circuit that involves HR23B and HDAC6 governs the biological response to HDAC inhibitors</article-title><source>Cell Death Differ.</source><year>2013</year><volume>20</volume><fpage>1306</fpage><lpage>16</lpage><pub-id pub-id-type="doi">10.1038/cdd.2013.47</pub-id><pub-id pub-id-type="pmid">23703321</pub-id></element-citation></ref><ref id="CR39"><label>39.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Khan</surname><given-names>O</given-names></name><name><surname>Fotheringham</surname><given-names>S</given-names></name><name><surname>Wood</surname><given-names>V</given-names></name><name><surname>Stimson</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Pezzella</surname><given-names>F</given-names></name><etal></etal></person-group><article-title>HR23B is a biomarker for tumor sensitivity to HDAC inhibitor-based therapy</article-title><source>Proc Natl Acad Sci U S A.</source><year>2010</year><volume>107</volume><fpage>6532</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1073/pnas.0913912107</pub-id><pub-id pub-id-type="pmid">20308564</pub-id></element-citation></ref><ref id="CR40"><label>40.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lovly</surname><given-names>CM</given-names></name><name><surname>Dahlman</surname><given-names>KB</given-names></name><name><surname>Fohn</surname><given-names>LE</given-names></name><name><surname>Su</surname><given-names>Z</given-names></name><name><surname>Dias-Santagata</surname><given-names>D</given-names></name><name><surname>Hicks</surname><given-names>DJ</given-names></name><etal></etal></person-group><article-title>Routine multiplex mutational profiling of melanomas enables enrollment in genotype-driven therapeutic trials</article-title><source>PLoS One.</source><year>2012</year><volume>7</volume><pub-id pub-id-type="doi">10.1371/journal.pone.0035309</pub-id><pub-id pub-id-type="pmid">22536370</pub-id></element-citation></ref><ref id="CR41"><label>41.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Long</surname><given-names>GV</given-names></name><name><surname>Menzies</surname><given-names>AM</given-names></name><name><surname>Nagrial</surname><given-names>AM</given-names></name><name><surname>Haydu</surname><given-names>LE</given-names></name><name><surname>Hamilton</surname><given-names>AL</given-names></name><name><surname>Mann</surname><given-names>GJ</given-names></name><etal></etal></person-group><article-title>Prognostic and clinicopathologic associations of oncogenic BRAF in metastatic melanoma</article-title><source>J Clin Oncol.</source><year>2011</year><volume>29</volume><fpage>1239</fpage><lpage>46</lpage><pub-id pub-id-type="doi">10.1200/JCO.2010.32.4327</pub-id><pub-id pub-id-type="pmid">21343559</pub-id></element-citation></ref><ref id="CR42"><label>42.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Curtin</surname><given-names>JA</given-names></name><name><surname>Fridlyand</surname><given-names>J</given-names></name><name><surname>Kageshita</surname><given-names>T</given-names></name><name><surname>Patel</surname><given-names>HN</given-names></name><name><surname>Busam</surname><given-names>KJ</given-names></name><name><surname>Kutzner</surname><given-names>H</given-names></name><etal></etal></person-group><article-title>Distinct sets of genetic alterations in melanoma</article-title><source>N Engl J Med.</source><year>2005</year><volume>353</volume><fpage>2135</fpage><lpage>47</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa050092</pub-id><pub-id pub-id-type="pmid">16291983</pub-id></element-citation></ref><ref id="CR43"><label>43.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Gartside</surname><given-names>MG</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Ibrahimi</surname><given-names>OA</given-names></name><name><surname>Byron</surname><given-names>SA</given-names></name><name><surname>Curtis</surname><given-names>AV</given-names></name><name><surname>Wellens</surname><given-names>CL</given-names></name><etal></etal></person-group><article-title>Loss-of-function fibroblast growth factor receptor-2 mutations in melanoma</article-title><source>Mol Cancer Res.</source><year>2009</year><volume>7</volume><fpage>41</fpage><lpage>54</lpage><pub-id pub-id-type="doi">10.1158/1541-7786.MCR-08-0021</pub-id><pub-id pub-id-type="pmid">19147536</pub-id></element-citation></ref><ref id="CR44"><label>44.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Becker</surname><given-names>D</given-names></name><name><surname>Lee</surname><given-names>PL</given-names></name><name><surname>Rodeck</surname><given-names>U</given-names></name><name><surname>Herlyn</surname><given-names>M</given-names></name></person-group><article-title>Inhibition of the fibroblast growth factor receptor 1 (FGFR-1) gene in human melanocytes and malignant melanomas leads to inhibition of proliferation and signs indicative of differentiation</article-title><source>Oncogene.</source><year>1992</year><volume>7</volume><fpage>2303</fpage><lpage>13</lpage><pub-id pub-id-type="pmid">1437152</pub-id></element-citation></ref><ref id="CR45"><label>45.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Xerri</surname><given-names>L</given-names></name><name><surname>Battyani</surname><given-names>Z</given-names></name><name><surname>Grob</surname><given-names>JJ</given-names></name><name><surname>Parc</surname><given-names>P</given-names></name><name><surname>Hassoun</surname><given-names>J</given-names></name><name><surname>Bonerandi</surname><given-names>JJ</given-names></name><etal></etal></person-group><article-title>Expression of FGF1 and FGFR1 in human melanoma tissues</article-title><source>Melanoma Res.</source><year>1996</year><volume>6</volume><fpage>223</fpage><lpage>30</lpage><pub-id pub-id-type="doi">10.1097/00008390-199606000-00005</pub-id><pub-id pub-id-type="pmid">8819125</pub-id></element-citation></ref><ref id="CR46"><label>46.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Becker</surname><given-names>D</given-names></name></person-group><article-title>Antisense targeting of basic fibroblast growth factor and fibroblast growth factor receptor-1 in human melanomas blocks intratumoral angiogenesis and tumor growth</article-title><source>Nat Med.</source><year>1997</year><volume>3</volume><fpage>887</fpage><lpage>93</lpage><pub-id pub-id-type="doi">10.1038/nm0897-887</pub-id><pub-id pub-id-type="pmid">9256280</pub-id></element-citation></ref><ref id="CR47"><label>47.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Valesky</surname><given-names>M</given-names></name><name><surname>Spang</surname><given-names>AJ</given-names></name><name><surname>Fisher</surname><given-names>GW</given-names></name><name><surname>Farkas</surname><given-names>DL</given-names></name><name><surname>Becker</surname><given-names>D</given-names></name></person-group><article-title>Noninvasive dynamic fluorescence imaging of human melanomas reveals that targeted inhibition of bFGF or FGFR-1 in melanoma cells blocks tumor growth by apoptosis</article-title><source>Mol Med.</source><year>2002</year><volume>8</volume><fpage>103</fpage><lpage>12</lpage><pub-id pub-id-type="pmid">12080186</pub-id></element-citation></ref><ref id="CR48"><label>48.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mack</surname><given-names>NA</given-names></name><name><surname>Whalley</surname><given-names>HJ</given-names></name><name><surname>Castillo-Lluva</surname><given-names>S</given-names></name><name><surname>Malliri</surname><given-names>A</given-names></name></person-group><article-title>The diverse roles of Rac signaling in tumorigenesis</article-title><source>Cell cycle (Georgetown, Tex)</source><year>2011</year><volume>10</volume><fpage>1571</fpage><lpage>81</lpage><pub-id pub-id-type="doi">10.4161/cc.10.10.15612</pub-id></element-citation></ref><ref id="CR49"><label>49.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Krauthammer</surname><given-names>M</given-names></name><name><surname>Kong</surname><given-names>Y</given-names></name><name><surname>Ha</surname><given-names>BH</given-names></name><name><surname>Evans</surname><given-names>P</given-names></name><name><surname>Bacchiocchi</surname><given-names>A</given-names></name><name><surname>McCusker</surname><given-names>JP</given-names></name><etal></etal></person-group><article-title>Exome sequencing identifies recurrent somatic RAC1 mutations in melanoma</article-title><source>Nat Genet.</source><year>2012</year><volume>44</volume><fpage>1006</fpage><lpage>14</lpage><pub-id pub-id-type="doi">10.1038/ng.2359</pub-id><pub-id pub-id-type="pmid">22842228</pub-id></element-citation></ref><ref id="CR50"><label>50.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hodis</surname><given-names>E</given-names></name><name><surname>Watson</surname><given-names>IR</given-names></name><name><surname>Kryukov</surname><given-names>GV</given-names></name><name><surname>Arold</surname><given-names>ST</given-names></name><name><surname>Imielinski</surname><given-names>M</given-names></name><name><surname>Theurillat</surname><given-names>JP</given-names></name><etal></etal></person-group><article-title>A landscape of driver mutations in melanoma</article-title><source>Cell.</source><year>2012</year><volume>150</volume><fpage>251</fpage><lpage>63</lpage><pub-id pub-id-type="doi">10.1016/j.cell.2012.06.024</pub-id><pub-id pub-id-type="pmid">22817889</pub-id></element-citation></ref><ref id="CR51"><label>51.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ho</surname><given-names>H</given-names></name><name><surname>Aruri</surname><given-names>J</given-names></name><name><surname>Kapadia</surname><given-names>R</given-names></name><name><surname>Mehr</surname><given-names>H</given-names></name><name><surname>White</surname><given-names>MA</given-names></name><name><surname>Ganesan</surname><given-names>AK</given-names></name></person-group><article-title>RhoJ regulates melanoma chemoresistance by suppressing pathways that sense DNA damage</article-title><source>Cancer Res.</source><year>2012</year><volume>72</volume><fpage>5516</fpage><lpage>28</lpage><pub-id pub-id-type="doi">10.1158/0008-5472.CAN-12-0775</pub-id><pub-id pub-id-type="pmid">22971344</pub-id></element-citation></ref><ref id="CR52"><label>52.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bid</surname><given-names>HK</given-names></name><name><surname>Roberts</surname><given-names>RD</given-names></name><name><surname>Manchanda</surname><given-names>PK</given-names></name><name><surname>Houghton</surname><given-names>PJ</given-names></name></person-group><article-title>RAC1: an emerging therapeutic option for targeting cancer angiogenesis and metastasis</article-title><source>Mol Cancer Ther.</source><year>2013</year><volume>12</volume><fpage>1925</fpage><lpage>34</lpage><pub-id pub-id-type="doi">10.1158/1535-7163.MCT-13-0164</pub-id><pub-id pub-id-type="pmid">24072884</pub-id></element-citation></ref><ref id="CR53"><label>53.</label><element-citation publication-type="book"><person-group person-group-type="author"><name><surname>Goncharov</surname><given-names>K</given-names></name><name><surname>Zeineldin</surname><given-names>R</given-names></name></person-group><article-title>Sensitizing resistant ovarian cancer to chemotherapy through inhibition of small GTPases [abstract]</article-title><source>Proceedings of the 103rd Annual Meeting of the American Association for Cancer Research</source><year>2012</year><publisher-loc>Chicago</publisher-loc><publisher-name>AACR</publisher-name></element-citation></ref><ref id="CR54"><label>54.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wilson</surname><given-names>MA</given-names></name><name><surname>Zhao</surname><given-names>F</given-names></name><name><surname>Letrero</surname><given-names>R</given-names></name><name><surname>D'Andrea</surname><given-names>K</given-names></name><name><surname>Rimm</surname><given-names>DL</given-names></name><name><surname>Kirkwood</surname><given-names>JM</given-names></name><etal></etal></person-group><article-title>Correlation of somatic mutations and clinical outcome in melanoma patients treated with carboplatin, paclitaxel, and sorafenib</article-title><source>Clin Cancer Res.</source><year>2014</year><volume>20</volume><fpage>3328</fpage><lpage>37</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-14-0093</pub-id><pub-id pub-id-type="pmid">24714776</pub-id></element-citation></ref><ref id="CR55"><label>55.</label><mixed-citation publication-type="other">Lagani V, Tsamardinos I. Biomarker signature identification in “omics” data with multi-class outcome. Computational Structural Biotechnol J. 2013;6</mixed-citation></ref><ref id="CR56"><label>56.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cox</surname><given-names>DR</given-names></name></person-group><article-title>Regression models and life-tables</article-title><source>J R Stat Soc B.</source><year>1972</year><volume>34</volume><fpage>187</fpage><lpage>220</lpage></element-citation></ref><ref id="CR57"><label>57.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lewis</surname><given-names>BP</given-names></name><name><surname>Shih</surname><given-names>IH</given-names></name><name><surname>Jones-Rhoades</surname><given-names>MW</given-names></name><name><surname>Bartel</surname><given-names>DP</given-names></name><name><surname>Burge</surname><given-names>CB</given-names></name></person-group><article-title>Prediction of mammalian microRNA targets</article-title><source>Cell.</source><year>2003</year><volume>115</volume><fpage>787</fpage><lpage>98</lpage><pub-id pub-id-type="doi">10.1016/S0092-8674(03)01018-3</pub-id><pub-id pub-id-type="pmid">14697198</pub-id></element-citation></ref></ref-list></back></article>