<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Cancer</journal-id><journal-id journal-id-type="iso-abbrev">BMC Cancer</journal-id><journal-title-group><journal-title>BMC Cancer</journal-title></journal-title-group><issn pub-type="epub">1471-2407</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">26674353</article-id><article-id pub-id-type="pmc">4682219</article-id><article-id pub-id-type="publisher-id">1959</article-id><article-id pub-id-type="doi">10.1186/s12885-015-1959-5</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>Comparison of the Roche cobas® 4800 and Digene Hybrid Capture® 2 HPV tests for primary cervical cancer screening in the HPV FOCAL trial</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Cook</surname><given-names>Darrel A.</given-names></name><xref ref-type="aff" rid="Aff1"></xref><xref ref-type="aff" rid="Aff2"></xref></contrib><contrib contrib-type="author"><name><surname>Mei</surname><given-names>Wendy</given-names></name><xref ref-type="aff" rid="Aff3"></xref></contrib><contrib contrib-type="author"><name><surname>Smith</surname><given-names>Laurie W.</given-names></name><xref ref-type="aff" rid="Aff1"></xref></contrib><contrib contrib-type="author"><name><surname>van Niekerk</surname><given-names>Dirk J.</given-names></name><xref ref-type="aff" rid="Aff1"></xref><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author"><name><surname>Ceballos</surname><given-names>Kathy</given-names></name><xref ref-type="aff" rid="Aff1"></xref><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author"><name><surname>Franco</surname><given-names>Eduardo L.</given-names></name><xref ref-type="aff" rid="Aff5"></xref></contrib><contrib contrib-type="author"><name><surname>Coldman</surname><given-names>Andrew J.</given-names></name><xref ref-type="aff" rid="Aff1"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author"><name><surname>Ogilvie</surname><given-names>Gina S.</given-names></name><xref ref-type="aff" rid="Aff2"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Krajden</surname><given-names>Mel</given-names></name><address><phone>604-707-2421</phone><email>mel.krajden@bccdc.ca</email></address><xref ref-type="aff" rid="Aff2"></xref><xref ref-type="aff" rid="Aff3"></xref><xref ref-type="aff" rid="Aff4"></xref></contrib><aff id="Aff1"><label></label>BC Cancer Agency, 675 West 10th Ave, Vancouver, BC V5Z 1L3 Canada </aff><aff id="Aff2"><label></label>BC Centre for Disease Control, 655 West 12th Ave, Vancouver, BC V5Z 4R4 Canada </aff><aff id="Aff3"><label></label>Lower Mainland Pathology and Laboratory Medicine, 655 West 12th Ave, Vancouver, BC V5Z 4R4 Canada </aff><aff id="Aff4"><label></label>University of British Columbia, 2329 West Mall, Vancouver, BC V6T 1Z4 Canada </aff><aff id="Aff5"><label></label>McGill University, 845 Rue Sherbrooke O, Montréal, QC Canada </aff></contrib-group><pub-date pub-type="epub"><day>16</day><month>12</month><year>2015</year></pub-date><pub-date pub-type="pmc-release"><day>16</day><month>12</month><year>2015</year></pub-date><pub-date pub-type="collection"><year>2015</year></pub-date><volume>15</volume><elocation-id>968</elocation-id><history><date date-type="received"><day>24</day><month>7</month><year>2015</year></date><date date-type="accepted"><day>30</day><month>11</month><year>2015</year></date></history><permissions><copyright-statement>© Cook et al. 2015</copyright-statement><license license-type="OpenAccess"><license-p>
<bold>Open Access</bold>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</ext-link>) applies to the data made available in this article, unless otherwise stated.</license-p></license></permissions><abstract id="Abs1"><sec><title><offsets xml_i="4946" xml_f="4956" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="4967" xml_f="5296" txt_i="22" txt_f="351">HPV FOCAL is a randomized trial (ISRCTN79347302, registered 20 Apr 2007) comparing high-risk (hr) HPV testing vs. liquid-based cytology (LBC) for cervical cancer screening of women aged 25–65. We compared the Digene Hybrid Capture® 2 High-Risk HPV DNA Test® (HC2) and the Roche cobas® 4800 HPV Test (COBAS) for primary screening.</offsets></p></sec><sec><title><offsets xml_i="5318" xml_f="5325" txt_i="353" txt_f="360">Methods</offsets></title><p><offsets xml_i="5336" xml_f="5343" txt_i="361" txt_f="368">Women (</offsets><italic><offsets xml_i="5351" xml_f="5352" txt_i="368" txt_f="369">n</offsets></italic><offsets xml_i="5361" xml_f="5537" txt_i="369" txt_f="545"> = 6,172) were screened at baseline by HC2 and COBAS and by LBC 24 months later. We assessed HPV genotyping and reflex LBC for colposcopy triage of baseline HPV positive women.</offsets></p></sec><sec><title><offsets xml_i="5559" xml_f="5566" txt_i="547" txt_f="554">Results</offsets></title><p><offsets xml_i="5577" xml_f="6474" txt_i="555" txt_f="1452">Overall HC2/COBAS agreement was 96.1 % (kappa 0.75) and positive agreement was 77.5 %. Baseline CIN2 and CIN3+ rates based on HPV screening were 8.6/1,000 and 6.6/1,000 respectively; 24 month rates were 0.7/1,000 and 0.4/1,000 (LBC screening). HC2 and COBAS were concordant positive for 91 % of round 1 CIN2 and 98 % of CIN3+. CIN3+ was significantly associated with HPV 16 (Odds Ratio [OR] 5.11; 95 % confidence interval [CI] 2.30, 11.37), but not HPV 18 (OR 2.62; 95 % CI 0.73, 9.49), vs. non-HPV 16/18 HPV at baseline. There was no significant association between HPV genotype and CIN2. CIN3+ was significantly more likely for high-grade (OR 5.99; 95 % CI 2.53, 14.18), but not low-grade (OR 0.54; 95 % CI 0.20, 1.49), vs. negative LBC. No significant association was observed between LBC grade and CIN2. HPV 16 and 18 were associated with 33 % of CIN2 and 68 % of CIN3+ identified at baseline.</offsets></p></sec><sec><title><offsets xml_i="6496" xml_f="6507" txt_i="1454" txt_f="1465">Conclusions</offsets></title><p><offsets xml_i="6518" xml_f="6785" txt_i="1466" txt_f="1733">For hrHPV positive women, abnormal reflex LBC is appropriate for colposcopy triage. In addition, immediate referral of women with HPV 16/18 and normal cytology may allow for earlier detection of CIN2+ lesions which would not be detected until after follow-up testing.</offsets></p></sec><sec><title><offsets xml_i="6807" xml_f="6840" txt_i="1735" txt_f="1768">Electronic supplementary material</offsets></title><p><offsets xml_i="6851" xml_f="6990" txt_i="1769" txt_f="1908">The online version of this article (doi:10.1186/s12885-015-1959-5) contains supplementary material, which is available to authorized users.</offsets></p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Cervical cancer screening tests</kwd><kwd>Human papillomavirus DNA tests</kwd><kwd>Papanicolaou test</kwd><kwd>Colposcopy triage</kwd><kwd>Cervical intraepithelial neoplasia</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/http://dx.doi.org/10.13039/100007013</institution-id><institution>F. Hoffman-La Roche (CH)</institution></institution-wrap></funding-source><award-id>ᅟ</award-id><principal-award-recipient><name><surname>Coldman</surname><given-names>Andrew J.</given-names></name></principal-award-recipient></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2015</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1" sec-type="introduction"><title><offsets xml_i="7937" xml_f="7947" txt_i="1917" txt_f="1927">Background</offsets></title><p><offsets xml_i="7958" xml_f="8083" txt_i="1928" txt_f="2053">Persistent infection with high-risk (hr) human papillomavirus (HPV) genotypes is recognized as the cause of cervical cancer [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="8115" xml_f="8116" txt_i="2053" txt_f="2054">1</offsets></xref><offsets xml_i="8123" xml_f="8125" txt_i="2054" txt_f="2056">, </offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="8157" xml_f="8158" txt_i="2056" txt_f="2057">2</offsets></xref><offsets xml_i="8165" xml_f="8367" txt_i="2057" txt_f="2259">] and many screening programs are shifting from traditional Pap cytology to hrHPV testing. Screening for hrHPV has higher sensitivity relative to the Pap for detection of cervical precancerous lesions [</offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="8399" xml_f="8400" txt_i="2259" txt_f="2260">3</offsets></xref><offsets xml_i="8407" xml_f="8656" txt_i="2260" txt_f="2509">] (i.e., cervical intraepithelial neoplasia grade two [CIN2] or CIN2 and worse [CIN2+]). In addition, hrHPV testing has a negative predictive value for CIN2+ of greater than 99 %, enabling screening intervals to be extended up to 5 years or longer [</offsets><xref ref-type="bibr" rid="CR4"><offsets xml_i="8688" xml_f="8689" txt_i="2509" txt_f="2510">4</offsets></xref><offsets xml_i="8696" xml_f="8697" txt_i="2510" txt_f="2511">–</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="8729" xml_f="8730" txt_i="2511" txt_f="2512">6</offsets></xref><offsets xml_i="8737" xml_f="8913" txt_i="2512" txt_f="2688">]. A number of hrHPV screening tests are commercially available, many of which incorporate partial genotyping in addition to generating a qualitative positive/negative result [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="8945" xml_f="8946" txt_i="2688" txt_f="2689">7</offsets></xref><offsets xml_i="8953" xml_f="9044" txt_i="2689" txt_f="2780">]. Since HPV 16 and 18 infections account for at least 70 % of cervical cancers worldwide [</offsets><xref ref-type="bibr" rid="CR8"><offsets xml_i="9076" xml_f="9077" txt_i="2780" txt_f="2781">8</offsets></xref><offsets xml_i="9084" xml_f="9276" txt_i="2781" txt_f="2973">], specific detection of these two genotypes provides useful additional information for colposcopy triage of hrHPV positive women who are more likely to have or develop pre-cancerous lesions [</offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="9308" xml_f="9309" txt_i="2973" txt_f="2974">9</offsets></xref><offsets xml_i="9316" xml_f="9318" txt_i="2974" txt_f="2976">, </offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="9351" xml_f="9353" txt_i="2976" txt_f="2978">10</offsets></xref><offsets xml_i="9360" xml_f="9362" txt_i="2978" txt_f="2980">].</offsets></p><p><offsets xml_i="9369" xml_f="9534" txt_i="2981" txt_f="3146">A challenge is that hrHPV testing identifies women with both persistent and transient HPV infections, with most of the latter clearing spontaneously within 2 years [</offsets><xref ref-type="bibr" rid="CR11"><offsets xml_i="9567" xml_f="9569" txt_i="3146" txt_f="3148">11</offsets></xref><offsets xml_i="9576" xml_f="9797" txt_i="3148" txt_f="3369">]. For that reason it is undesirable to refer all hrHPV positive women to colposcopy since treatment of lesions which are likely to spontaneously regress is associated with increased health system costs, patient anxiety [</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="9830" xml_f="9832" txt_i="3369" txt_f="3371">12</offsets></xref><offsets xml_i="9839" xml_f="9883" txt_i="3371" txt_f="3415">] and the potential for reproductive harms [</offsets><xref ref-type="bibr" rid="CR13"><offsets xml_i="9916" xml_f="9918" txt_i="3415" txt_f="3417">13</offsets></xref><offsets xml_i="9925" xml_f="9927" txt_i="3417" txt_f="3419">, </offsets><xref ref-type="bibr" rid="CR14"><offsets xml_i="9960" xml_f="9962" txt_i="3419" txt_f="3421">14</offsets></xref><offsets xml_i="9969" xml_f="10395" txt_i="3421" txt_f="3847">]. An efficient triage mechanism is required so that women at the highest risk of developing rapidly progressive cervical disease are referred to colposcopy, whereas those at lower risk could be more closely followed for clearance of transient infections, with colposcopy referral only for those hrHPV infections which persist. Several triage approaches for hrHPV positive women have been assessed, including reflex cytology [</offsets><xref ref-type="bibr" rid="CR5"><offsets xml_i="10427" xml_f="10428" txt_i="3847" txt_f="3848">5</offsets></xref><offsets xml_i="10435" xml_f="10463" txt_i="3848" txt_f="3876">], p16/Ki67 immunostaining [</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="10496" xml_f="10498" txt_i="3876" txt_f="3878">15</offsets></xref><offsets xml_i="10505" xml_f="10530" txt_i="3878" txt_f="3903">], HPV 16/18 genotyping [</offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="10562" xml_f="10563" txt_i="3903" txt_f="3904">9</offsets></xref><offsets xml_i="10570" xml_f="10572" txt_i="3904" txt_f="3906">, </offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="10605" xml_f="10607" txt_i="3906" txt_f="3908">10</offsets></xref><offsets xml_i="10614" xml_f="10676" txt_i="3908" txt_f="3970">] and the detection of methylation of HPV and/or human genes [</offsets><xref ref-type="bibr" rid="CR16"><offsets xml_i="10709" xml_f="10711" txt_i="3970" txt_f="3972">16</offsets></xref><offsets xml_i="10718" xml_f="10720" txt_i="3972" txt_f="3974">, </offsets><xref ref-type="bibr" rid="CR17"><offsets xml_i="10753" xml_f="10755" txt_i="3974" txt_f="3976">17</offsets></xref><offsets xml_i="10762" xml_f="10764" txt_i="3976" txt_f="3978">].</offsets></p><p><offsets xml_i="10771" xml_f="11088" txt_i="3979" txt_f="4296">In this study, we compared the performance of the Digene Hybrid Capture® 2 High-Risk HPV DNA Test® (HC2) to that of the Roche cobas® 4800 HPV Test (COBAS) for primary cervical screening within the HPV FOCAL randomized trial. We assessed CIN outcomes following reflex cytology and HPV genotyping for colposcopy triage.</offsets></p></sec><sec id="Sec2" sec-type="materials|methods"><title><offsets xml_i="11149" xml_f="11156" txt_i="4298" txt_f="4305">Methods</offsets></title><sec id="Sec3"><title><offsets xml_i="11186" xml_f="11202" txt_i="4306" txt_f="4322">Study population</offsets></title><p><offsets xml_i="11213" xml_f="11271" txt_i="4323" txt_f="4381">The HPV FOCAL Trial design has been described previously [</offsets><xref ref-type="bibr" rid="CR18"><offsets xml_i="11304" xml_f="11306" txt_i="4381" txt_f="4383">18</offsets></xref><offsets xml_i="11313" xml_f="11321" txt_i="4383" txt_f="4391">] (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="11353" xml_f="11354" txt_i="4391" txt_f="4392">1</offsets></xref><offsets xml_i="11361" xml_f="12802" txt_i="4392" txt_f="5833">). Briefly, the trial is a three-arm, randomized controlled trial (ISRCTN79347302, registered 20 Apr 2007) to establish the efficacy of hrHPV screening (using HC2) with liquid-based cytology (LBC) triage of HC2 positive women (Intervention and Safety Arms) compared to LBC screening with HC2 triage of atypical squamous cells of undetermined significance (ASCUS) (Control Arm) in women aged 25–65 years. The Safety Arm was included to verify the safety of the Intervention Arm 4 year screening interval by re-screening baseline HC2 negative women with LBC at 2 years, comparable to the current standard of care in British Columbia (BC), conventional cytology every 2 years. Women in the Safety Arm are the subjects of the present analysis. At baseline, HC2 positive women with LBC ≥ASCUS were referred immediately to colposcopy, whereas those HC2 positive/LBC negative for intraepithelial lesions or malignancy (NILM) were re-tested by HC2 and LBC at 12 months. Those persistently HC2 positive and/or LBC ≥ASCUS were then referred to colposcopy. Visible lesions were biopsied and if none were seen, endocervical curettage was performed for histological analysis. Baseline HC2 negative women, together with those who cleared hrHPV and were LBC NILM at 12 months, had their Safety Arm exit screen at 24 months. Round 1 refers to the baseline screen together with the 12-month follow-up and Round 2 refers to the exit screen at 24 months (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="12834" xml_f="12835" txt_i="5833" txt_f="5834">1</offsets></xref><offsets xml_i="12842" xml_f="12844" txt_i="5834" txt_f="5836">).</offsets><fig id="Fig1"><label><offsets xml_i="12866" xml_f="12872" txt_i="5836" txt_f="5842">Fig. 1</offsets></label><caption><p><offsets xml_i="12892" xml_f="12916" txt_i="5842" txt_f="5866">HPV FOCAL Trial Design. </offsets><italic><offsets xml_i="12924" xml_f="12937" txt_i="5866" txt_f="5879">Abbreviations</offsets></italic><offsets xml_i="12946" xml_f="12948" txt_i="5879" txt_f="5881">: </offsets><italic><offsets xml_i="12956" xml_f="12959" txt_i="5881" txt_f="5884">HPV</offsets></italic><offsets xml_i="12968" xml_f="13019" txt_i="5884" txt_f="5935"> Digene Hybrid Capture® 2 high-risk HPV DNA test®, </offsets><italic><offsets xml_i="13027" xml_f="13030" txt_i="5935" txt_f="5938">LBC</offsets></italic><offsets xml_i="13039" xml_f="13063" txt_i="5938" txt_f="5962"> liquid-based cytology, </offsets><italic><offsets xml_i="13071" xml_f="13076" txt_i="5962" txt_f="5967">ASCUS</offsets></italic><offsets xml_i="13085" xml_f="13138" txt_i="5967" txt_f="6020"> atypical squamous cells, undetermined significance, </offsets><italic><offsets xml_i="13146" xml_f="13150" txt_i="6020" txt_f="6024">ASCH</offsets></italic><offsets xml_i="13159" xml_f="13212" txt_i="6024" txt_f="6077"> atypical squamous cells, cannot exclude high-grade, </offsets><italic><offsets xml_i="13220" xml_f="13224" txt_i="6077" txt_f="6081">LSIL</offsets></italic><offsets xml_i="13233" xml_f="13275" txt_i="6081" txt_f="6123"> low-grade squamous intraepithelial lesion</offsets></p></caption><graphic xlink:href="12885_2015_1959_Fig1_HTML" id="MO1"></graphic></fig></p><p><offsets xml_i="13369" xml_f="13535" txt_i="6125" txt_f="6291">All trial subjects provided written informed consent and the study was approved by the University of British Columbia/BC Cancer Agency Clinical Research Ethics Board.</offsets></p></sec><sec id="Sec4"><title><offsets xml_i="13567" xml_f="13585" txt_i="6293" txt_f="6311">Laboratory testing</offsets></title><p><offsets xml_i="13596" xml_f="15880" txt_i="6312" txt_f="8596">Cervical specimens collected in PreservCyt® (Hologic Inc., Bedford MA, USA) by participating family physicians in Metro Vancouver and Greater Victoria BC were used for both hrHPV testing and LBC. For Safety Arm enrollment and 12 month follow-up specimens, separate aliquots for HC2 (6 mL) and COBAS (2 mL) were removed from the PreservCyt® vial prior to any other processing. At the 24 month exit screen, aliquots for reflex HC2 testing were obtained after cytology processing. At enrollment, all specimens were tested by HC2 (Qiagen, Gaithersburg MD, USA), COBAS (Roche Molecular Systems, Pleasanton CA, USA) and the Roche Linear Array HPV Genotyping Test (LA; Roche Molecular Systems). HC2 generates a qualitative result for the presence or absence of one or more of 13 hrHPV genotypes (HPV 16, 18, 31, 33, 35, 45, 51, 52, 56, 58, 59 and 68). We used a relative light unit/cutoff (RLU/CO) ratio of ≥1.0 to indicate a positive HC2 result. Specimens which had no visible cell pellet at the preparation stage were rejected as inadequate and reported as HC2 invalid. COBAS generates individual qualitative results for HPV 16, HPV 18 and a pool of 12 other hr (OHR) HPV genotypes (HPV 31, 33, 35, 45, 51, 52, 56, 58, 59, 66 and 68). LA generates individual qualitative results for the same 14 hrHPV genotypes as COBAS together with 23 additional HPV genotypes which are defined for the purposes of this study as low-risk (lr) HPV. LA was performed using residual extracts from COBAS testing. COBAS and LA were performed blinded at the time of collection and were not used for subject management. Specimens with initially invalid COBAS results were re-tested to confirm the invalid result. LBC slides were prepared and stained for all subjects at enrollment but were not screened and interpreted unless the subject was HC2 positive. COBAS and LA results were unblinded and compared with HC2 after Safety Arm women exited the trial at 2 years. LBC slides were then screened and interpreted retrospectively for women with baseline HC2 negative/COBAS positive and HC2 negative/LA positive results. For the latter group, specimens positive by LA for at least one hrHPV genotype were classified as hrHPV positive and those positive for only lrHPV genotype(s) were classified as lrHPV positive.</offsets></p></sec><sec id="Sec5"><title><offsets xml_i="15912" xml_f="15932" txt_i="8598" txt_f="8618">Statistical analysis</offsets></title><p><offsets xml_i="15943" xml_f="16153" txt_i="8619" txt_f="8829">Overall agreement, Cohen’s kappa, and positive and negative agreements with 95 % confidence intervals (CI) were calculated for enrollment HC2 vs. COBAS results. Positive agreement was calculated by the formula </offsets><italic><offsets xml_i="16161" xml_f="16162" txt_i="8829" txt_f="8830">p</offsets></italic><sub><italic><offsets xml_i="16184" xml_f="16187" txt_i="8830" txt_f="8833">pos</offsets></italic></sub><offsets xml_i="16202" xml_f="16204" txt_i="8833" txt_f="8835"> 
</offsets><italic><offsets xml_i="16212" xml_f="16230" txt_i="8835" txt_f="8853">= (2a)/(N + a - d)</offsets></italic><offsets xml_i="16239" xml_f="16266" txt_i="8853" txt_f="8880"> and negative agreement by </offsets><italic><offsets xml_i="16274" xml_f="16275" txt_i="8880" txt_f="8881">p</offsets></italic><sub><italic><offsets xml_i="16297" xml_f="16300" txt_i="8881" txt_f="8884">neg</offsets></italic></sub><offsets xml_i="16315" xml_f="16317" txt_i="8884" txt_f="8886"> 
</offsets><italic><offsets xml_i="16325" xml_f="16343" txt_i="8886" txt_f="8904">= (2d)/(N – a + d)</offsets></italic><offsets xml_i="16352" xml_f="16360" txt_i="8904" txt_f="8912">, where </offsets><italic><offsets xml_i="16368" xml_f="16369" txt_i="8912" txt_f="8913">N</offsets></italic><offsets xml_i="16378" xml_f="16401" txt_i="8913" txt_f="8936"> is the number tested, </offsets><italic><offsets xml_i="16409" xml_f="16410" txt_i="8936" txt_f="8937">a</offsets></italic><offsets xml_i="16419" xml_f="16462" txt_i="8937" txt_f="8980"> is the number positive by both tests, and </offsets><italic><offsets xml_i="16470" xml_f="16471" txt_i="8980" txt_f="8981">d</offsets></italic><offsets xml_i="16480" xml_f="16519" txt_i="8981" txt_f="9020"> is the number negative by both tests [</offsets><xref ref-type="bibr" rid="CR19"><offsets xml_i="16552" xml_f="16554" txt_i="9020" txt_f="9022">19</offsets></xref><offsets xml_i="16561" xml_f="16673" txt_i="9022" txt_f="9134">]. Genotyping agreements (overall and positive) and Cohen’s kappa with 95 % CI were calculated for COBAS vs. LA.</offsets></p><p><offsets xml_i="16680" xml_f="17848" txt_i="9135" txt_f="10303">CIN diagnoses were obtained from the centralized provincial cervical screening registry, which includes all screening and histopathology results for BC women. CIN2 and CIN3+ rates with 95 % CI, stratified by baseline HC2 and COBAS results, per 1,000 women screened were calculated and 95 % CI for rates assumed a Poisson distribution. Round 1 included CIN lesions identified within 2 years after the colposcopy referral date, which could have occurred at baseline or after 12 months follow-up, while round 2 included CIN lesions identified within 2 years of the colposcopy referral date following the exit screen. If a subject had more than one colposcopic biopsy, the result for the highest grade CIN lesion identified was used for analysis. The analyses in this paper were based only on CIN lesions identified among women referred to colposcopy as per the FOCAL trial protocol. For trial quality assurance purposes, periodic audits of the provincial registry were conducted to identify CIN2+ lesions among trial subjects whose colposcopy referrals were not based on the trial protocol. Any additional CIN2+ identified by the audits were not included in the analyses.</offsets></p><p><offsets xml_i="17855" xml_f="18503" txt_i="10304" txt_f="10952">A multivariate logistic regression model was fitted to the data to test the hypothesis regarding the relationship between HPV genotype and reflex LBC results and the likelihood of a CIN outcome. Since a given subject may be positive for more than one hrHPV genotype, subjects were classified hierarchically for analysis: for HPV 16, the subject was HPV 16 positive but could also be positive for HPV 18 and OHR HPV genotypes; for HPV 18, the subject was HPV 16 negative, HPV 18 positive, and could also be positive for OHR HPV genotypes; for OHR HPV, the subject was negative for HPV 16 and 18 and positive for one or more of the OHR HPV genotypes.</offsets></p><p><offsets xml_i="18510" xml_f="18876" txt_i="10953" txt_f="11319">The Median score and Mann-Whitney U tests were used to calculate p values for the median HC2 RLU/CO and COBAS cycle threshold (Ct) values for concordant vs. discordant hrHPV positive specimens. The significance of the differences between proportions used a Z score test. All statistical calculations were performed using SAS v.9.3 (SAS Institute Inc., Cary NC, USA).</offsets></p></sec></sec><sec id="Sec6" sec-type="results"><title><offsets xml_i="18933" xml_f="18940" txt_i="11322" txt_f="11329">Results</offsets></title><sec id="Sec7"><title><offsets xml_i="18970" xml_f="19000" txt_i="11330" txt_f="11360">HC2 and COBAS positivity rates</offsets></title><p><offsets xml_i="19011" xml_f="19420" txt_i="11361" txt_f="11770">Among 6,214 women enrolled to the Safety Arm, 10 (0.16 %) and 11 (0.18 %) respectively had invalid HC2 and COBAS results, and 21 (0.34 %) had one or more missing HC2, COBAS or LA results. The present analysis includes the 6,172 (99.3 %) women with valid baseline HC2, COBAS and LA results. Overall HC2/COBAS agreement was 96.1 % (95 % CI: 95.6 – 96.6) and Cohen’s kappa was 0.75 (95 % CI: 0.72 – 0.79) (Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="19454" xml_f="19455" txt_i="11770" txt_f="11771">1</offsets></xref><offsets xml_i="19462" xml_f="19990" txt_i="11771" txt_f="12299">). Positive and negative agreements were 77.5 % (95 % CI: 74.7 – 80.3) and 97.9 % (95 % CI: 97.6 – 98.2) respectively. The respective baseline HC2 and COBAS positivity rates were 8.4 % (516/6,172; 95 % CI: 7.7 – 9.1) and 8.8 % (542/6,172; 95 % CI: 8.1 – 9.5). HC2 positivity rates for those 25–29, 30–34, 35–49 and ≥50 years, were 26.6 % (122/458), 15.1 % (84/556), 7.5 % (213/2,859) and 4.2 % (97/2,299) respectively; the corresponding COBAS rates were 27.9 % (128/458), 17.4 % (97/556), 7.7 % (219/2,859) and 4.3 % (98/2,299).</offsets><table-wrap id="Tab1"><label><offsets xml_i="20019" xml_f="20026" txt_i="12299" txt_f="12306">Table 1</offsets></label><caption><p><offsets xml_i="20046" xml_f="20094" txt_i="12306" txt_f="12354">Digene HC2 and Roche COBAS agreement at baseline</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3" colspan="2"></th><th colspan="9"><offsets xml_i="20205" xml_f="20210" txt_i="12355" txt_f="12360">COBAS</offsets></th></tr><tr><th colspan="3"><offsets xml_i="20240" xml_f="20248" txt_i="12360" txt_f="12368">All ages</offsets></th><th colspan="3"><offsets xml_i="20269" xml_f="20284" txt_i="12368" txt_f="12380">Age &lt;30 year</offsets></th><th colspan="3"><offsets xml_i="20305" xml_f="20317" txt_i="12380" txt_f="12392">Age ≥30 year</offsets></th></tr><tr><th><offsets xml_i="20335" xml_f="20343" txt_i="12392" txt_f="12400">Positive</offsets></th><th><offsets xml_i="20352" xml_f="20360" txt_i="12400" txt_f="12408">Negative</offsets></th><th><offsets xml_i="20369" xml_f="20374" txt_i="12408" txt_f="12413">Total</offsets></th><th><offsets xml_i="20383" xml_f="20391" txt_i="12413" txt_f="12421">Positive</offsets></th><th><offsets xml_i="20400" xml_f="20408" txt_i="12421" txt_f="12429">Negative</offsets></th><th><offsets xml_i="20417" xml_f="20422" txt_i="12429" txt_f="12434">Total</offsets></th><th><offsets xml_i="20431" xml_f="20439" txt_i="12434" txt_f="12442">Positive</offsets></th><th><offsets xml_i="20448" xml_f="20456" txt_i="12442" txt_f="12450">Negative</offsets></th><th><offsets xml_i="20465" xml_f="20470" txt_i="12450" txt_f="12455">Total</offsets></th></tr></thead><tbody><tr><td rowspan="3"><offsets xml_i="20515" xml_f="20518" txt_i="12455" txt_f="12458">HC2</offsets></td><td><offsets xml_i="20527" xml_f="20535" txt_i="12459" txt_f="12467">Positive</offsets></td><td><offsets xml_i="20544" xml_f="20547" txt_i="12468" txt_f="12471">410</offsets></td><td><offsets xml_i="20556" xml_f="20559" txt_i="12472" txt_f="12475">106</offsets></td><td><offsets xml_i="20568" xml_f="20571" txt_i="12476" txt_f="12479">516</offsets></td><td><offsets xml_i="20580" xml_f="20583" txt_i="12480" txt_f="12483">108</offsets></td><td><offsets xml_i="20592" xml_f="20594" txt_i="12484" txt_f="12486">14</offsets></td><td><offsets xml_i="20603" xml_f="20606" txt_i="12487" txt_f="12490">122</offsets></td><td><offsets xml_i="20615" xml_f="20618" txt_i="12491" txt_f="12494">302</offsets></td><td><offsets xml_i="20627" xml_f="20629" txt_i="12495" txt_f="12497">92</offsets></td><td><offsets xml_i="20638" xml_f="20641" txt_i="12498" txt_f="12501">394</offsets></td></tr><tr><td><offsets xml_i="20659" xml_f="20667" txt_i="12502" txt_f="12510">Negative</offsets></td><td><offsets xml_i="20676" xml_f="20679" txt_i="12511" txt_f="12514">132</offsets></td><td><offsets xml_i="20688" xml_f="20693" txt_i="12515" txt_f="12520">5,524</offsets></td><td><offsets xml_i="20702" xml_f="20707" txt_i="12521" txt_f="12526">5,656</offsets></td><td><offsets xml_i="20716" xml_f="20718" txt_i="12527" txt_f="12529">20</offsets></td><td><offsets xml_i="20727" xml_f="20730" txt_i="12530" txt_f="12533">316</offsets></td><td><offsets xml_i="20739" xml_f="20742" txt_i="12534" txt_f="12537">336</offsets></td><td><offsets xml_i="20751" xml_f="20754" txt_i="12538" txt_f="12541">112</offsets></td><td><offsets xml_i="20763" xml_f="20768" txt_i="12542" txt_f="12547">5,208</offsets></td><td><offsets xml_i="20777" xml_f="20782" txt_i="12548" txt_f="12553">5,320</offsets></td></tr><tr><td><offsets xml_i="20800" xml_f="20805" txt_i="12554" txt_f="12559">Total</offsets></td><td><offsets xml_i="20814" xml_f="20817" txt_i="12560" txt_f="12563">542</offsets></td><td><offsets xml_i="20826" xml_f="20831" txt_i="12564" txt_f="12569">5,630</offsets></td><td><offsets xml_i="20840" xml_f="20845" txt_i="12570" txt_f="12575">6,172</offsets></td><td><offsets xml_i="20854" xml_f="20857" txt_i="12576" txt_f="12579">128</offsets></td><td><offsets xml_i="20866" xml_f="20869" txt_i="12580" txt_f="12583">330</offsets></td><td><offsets xml_i="20878" xml_f="20881" txt_i="12584" txt_f="12587">458</offsets></td><td><offsets xml_i="20890" xml_f="20893" txt_i="12588" txt_f="12591">414</offsets></td><td><offsets xml_i="20902" xml_f="20907" txt_i="12592" txt_f="12597">5,300</offsets></td><td><offsets xml_i="20916" xml_f="20921" txt_i="12598" txt_f="12603">5,714</offsets></td></tr><tr><td colspan="2"><offsets xml_i="20951" xml_f="20978" txt_i="12604" txt_f="12631">Overall agreement (95 % CI)</offsets></td><td colspan="3"><offsets xml_i="20999" xml_f="21018" txt_i="12632" txt_f="12651">96.1 % (95.6, 96.6)</offsets></td><td colspan="3"><offsets xml_i="21039" xml_f="21058" txt_i="12652" txt_f="12671">92.6 % (89.7, 94.7)</offsets></td><td colspan="3"><offsets xml_i="21079" xml_f="21098" txt_i="12672" txt_f="12691">96.4 % (95.9, 96.9)</offsets></td></tr><tr><td colspan="2"><offsets xml_i="21128" xml_f="21143" txt_i="12692" txt_f="12707">Kappa (95 % CI)</offsets></td><td colspan="3"><offsets xml_i="21164" xml_f="21181" txt_i="12708" txt_f="12725">0.75 (0.72, 0.79)</offsets></td><td colspan="3"><offsets xml_i="21202" xml_f="21219" txt_i="12726" txt_f="12743">0.81 (0.75, 0.87)</offsets></td><td colspan="3"><offsets xml_i="21240" xml_f="21257" txt_i="12744" txt_f="12761">0.73 (0.69, 0.76)</offsets></td></tr><tr><td colspan="2"><offsets xml_i="21287" xml_f="21315" txt_i="12762" txt_f="12790">Positive agreement (95 % CI)</offsets></td><td colspan="3"><offsets xml_i="21336" xml_f="21355" txt_i="12791" txt_f="12810">77.5 % (74.7, 80.3)</offsets></td><td colspan="3"><offsets xml_i="21376" xml_f="21395" txt_i="12811" txt_f="12830">86.4 % (81.9, 90.9)</offsets></td><td colspan="3"><offsets xml_i="21416" xml_f="21435" txt_i="12831" txt_f="12850">74.8 % (71.4, 78.1)</offsets></td></tr><tr><td colspan="2"><offsets xml_i="21465" xml_f="21493" txt_i="12851" txt_f="12879">Negative agreement (95 % CI)</offsets></td><td colspan="3"><offsets xml_i="21514" xml_f="21533" txt_i="12880" txt_f="12899">97.9 % (97.6, 98.2)</offsets></td><td colspan="3"><offsets xml_i="21554" xml_f="21573" txt_i="12900" txt_f="12919">94.9 % (93.2, 96.6)</offsets></td><td colspan="3"><offsets xml_i="21594" xml_f="21613" txt_i="12920" txt_f="12939">98.1 % (97.8, 98.3)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="21659" xml_f="21660" txt_i="12940" txt_f="12941">
</offsets><italic><offsets xml_i="21668" xml_f="21681" txt_i="12941" txt_f="12954">Abbreviations</offsets></italic><offsets xml_i="21690" xml_f="21692" txt_i="12954" txt_f="12956">: </offsets><italic><offsets xml_i="21700" xml_f="21703" txt_i="12956" txt_f="12959">HC2</offsets></italic><offsets xml_i="21712" xml_f="21763" txt_i="12959" txt_f="13010"> Digene Hybrid Capture® 2 high-risk HPV DNA test®, </offsets><italic><offsets xml_i="21771" xml_f="21776" txt_i="13010" txt_f="13015">COBAS</offsets></italic><offsets xml_i="21785" xml_f="21814" txt_i="13015" txt_f="13044"> Roche cobas® 4800 HPV test, </offsets><italic><offsets xml_i="21822" xml_f="21824" txt_i="13044" txt_f="13046">CI</offsets></italic><offsets xml_i="21833" xml_f="21853" txt_i="13046" txt_f="13066"> confidence interval</offsets></p></table-wrap-foot></table-wrap></p></sec><sec id="Sec8"><title><offsets xml_i="21920" xml_f="21936" txt_i="13069" txt_f="13085">COBAS genotyping</offsets></title><p><offsets xml_i="21947" xml_f="22091" txt_i="13086" txt_f="13230">COBAS genotyping demonstrated 2.1 % (128/6,172) HPV 16 positive, 0.7 % (46/6,171) HPV 18 positive and 7.1 % (437/6,171) OHR HPV positive (Table </offsets><xref rid="Tab2" ref-type="table"><offsets xml_i="22125" xml_f="22126" txt_i="13230" txt_f="13231">2</offsets></xref><offsets xml_i="22133" xml_f="22319" txt_i="13231" txt_f="13417">). COBAS and LA overall genotyping agreements ranged from 97.9 to 99.8 %, positive agreements from 85.7 to 87.0 %, and the kappa’s (0.85 – 0.87) indicated excellent genotyping agreement.</offsets><table-wrap id="Tab2"><label><offsets xml_i="22348" xml_f="22355" txt_i="13417" txt_f="13424">Table 2</offsets></label><caption><p><offsets xml_i="22375" xml_f="22436" txt_i="13424" txt_f="13485">Roche COBAS and linear array genotyping agreement at baseline</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"><offsets xml_i="22514" xml_f="22526" txt_i="13486" txt_f="13498">Linear array</offsets></th><th colspan="15"><offsets xml_i="22548" xml_f="22553" txt_i="13498" txt_f="13503">COBAS</offsets></th></tr><tr><th colspan="5"><offsets xml_i="22583" xml_f="22590" txt_i="13503" txt_f="13510">HPV 16 </offsets><italic><offsets xml_i="22598" xml_f="22599" txt_i="13510" txt_f="13511">n</offsets></italic><offsets xml_i="22608" xml_f="22616" txt_i="13511" txt_f="13519"> = 6,172</offsets></th><th colspan="5"><offsets xml_i="22637" xml_f="22644" txt_i="13519" txt_f="13526">HPV 18 </offsets><italic><offsets xml_i="22652" xml_f="22653" txt_i="13526" txt_f="13527">n</offsets></italic><offsets xml_i="22662" xml_f="22670" txt_i="13527" txt_f="13535"> = 6,171</offsets><sup><offsets xml_i="22675" xml_f="22676" txt_i="13535" txt_f="13536">a</offsets></sup><offsets xml_i="22682" xml_f="22683" txt_i="13536" txt_f="13537">
</offsets></th><th colspan="5"><offsets xml_i="22704" xml_f="22724" txt_i="13537" txt_f="13557">Other high risk HPV </offsets><italic><offsets xml_i="22732" xml_f="22733" txt_i="13557" txt_f="13558">n</offsets></italic><offsets xml_i="22742" xml_f="22750" txt_i="13558" txt_f="13566"> = 6,171</offsets><sup><offsets xml_i="22755" xml_f="22756" txt_i="13566" txt_f="13567">a</offsets></sup><offsets xml_i="22762" xml_f="22763" txt_i="13567" txt_f="13568">
</offsets></th></tr><tr><th><offsets xml_i="22781" xml_f="22784" txt_i="13568" txt_f="13571">Pos</offsets></th><th><offsets xml_i="22793" xml_f="22796" txt_i="13571" txt_f="13574">Neg</offsets></th><th><offsets xml_i="22805" xml_f="22832" txt_i="13574" txt_f="13601">Overall agreement (95 % CI)</offsets></th><th><offsets xml_i="22841" xml_f="22856" txt_i="13601" txt_f="13616">Kappa (95 % CI)</offsets></th><th><offsets xml_i="22865" xml_f="22888" txt_i="13616" txt_f="13639">Pos agreement (95 % CI)</offsets></th><th><offsets xml_i="22897" xml_f="22900" txt_i="13639" txt_f="13642">Pos</offsets></th><th><offsets xml_i="22909" xml_f="22912" txt_i="13642" txt_f="13645">Neg</offsets></th><th><offsets xml_i="22921" xml_f="22948" txt_i="13645" txt_f="13672">Overall agreement (95 % CI)</offsets></th><th><offsets xml_i="22957" xml_f="22972" txt_i="13672" txt_f="13687">Kappa (95 % CI)</offsets></th><th><offsets xml_i="22981" xml_f="23004" txt_i="13687" txt_f="13710">Pos agreement (95 % CI)</offsets></th><th><offsets xml_i="23013" xml_f="23016" txt_i="13710" txt_f="13713">Pos</offsets></th><th><offsets xml_i="23025" xml_f="23028" txt_i="13713" txt_f="13716">Neg</offsets></th><th><offsets xml_i="23037" xml_f="23064" txt_i="13716" txt_f="13743">Overall agreement (95 % CI)</offsets></th><th><offsets xml_i="23073" xml_f="23088" txt_i="13743" txt_f="13758">Kappa (95 % CI)</offsets></th><th><offsets xml_i="23097" xml_f="23120" txt_i="13758" txt_f="13781">Pos agreement (95 % CI)</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="23153" xml_f="23156" txt_i="13781" txt_f="13784">Pos</offsets></td><td><offsets xml_i="23165" xml_f="23168" txt_i="13785" txt_f="13788">114</offsets></td><td><offsets xml_i="23177" xml_f="23179" txt_i="13789" txt_f="13791">20</offsets></td><td rowspan="2"><offsets xml_i="23200" xml_f="23220" txt_i="13792" txt_f="13812">99.4 % (99.2 – 99.6)</offsets></td><td rowspan="2"><offsets xml_i="23241" xml_f="23259" txt_i="13813" txt_f="13831">0.87 (0.82 – 0.91)</offsets></td><td rowspan="2"><offsets xml_i="23280" xml_f="23300" txt_i="13832" txt_f="13852">87.0 % (82.7 – 91.4)</offsets></td><td><offsets xml_i="23309" xml_f="23311" txt_i="13853" txt_f="13855">42</offsets></td><td><offsets xml_i="23320" xml_f="23321" txt_i="13856" txt_f="13857">9</offsets></td><td rowspan="2"><offsets xml_i="23342" xml_f="23362" txt_i="13858" txt_f="13878">99.8 % (99.6 – 99.9)</offsets></td><td rowspan="2"><offsets xml_i="23383" xml_f="23401" txt_i="13879" txt_f="13897">0.86 (0.79 – 0.94)</offsets></td><td rowspan="2"><offsets xml_i="23422" xml_f="23442" txt_i="13898" txt_f="13918">86.6 % (79.4 – 93.8)</offsets></td><td><offsets xml_i="23451" xml_f="23454" txt_i="13919" txt_f="13922">388</offsets></td><td><offsets xml_i="23463" xml_f="23465" txt_i="13923" txt_f="13925">80</offsets></td><td rowspan="2"><offsets xml_i="23486" xml_f="23506" txt_i="13926" txt_f="13946">97.9 % (97.5 – 98.2)</offsets></td><td rowspan="2"><offsets xml_i="23527" xml_f="23545" txt_i="13947" txt_f="13965">0.85 (0.82 – 0.87)</offsets></td><td rowspan="2"><offsets xml_i="23566" xml_f="23586" txt_i="13966" txt_f="13986">85.7 % (83.3 – 88.2)</offsets></td></tr><tr><td><offsets xml_i="23604" xml_f="23607" txt_i="13987" txt_f="13990">Neg</offsets></td><td><offsets xml_i="23616" xml_f="23618" txt_i="13991" txt_f="13993">14</offsets></td><td><offsets xml_i="23627" xml_f="23632" txt_i="13994" txt_f="13999">6,024</offsets></td><td><offsets xml_i="23641" xml_f="23642" txt_i="14000" txt_f="14001">4</offsets></td><td><offsets xml_i="23651" xml_f="23656" txt_i="14002" txt_f="14007">6,116</offsets></td><td><offsets xml_i="23665" xml_f="23667" txt_i="14008" txt_f="14010">49</offsets></td><td><offsets xml_i="23676" xml_f="23681" txt_i="14011" txt_f="14016">5,654</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="23727" xml_f="23728" txt_i="14017" txt_f="14018">
</offsets><sup><offsets xml_i="23733" xml_f="23734" txt_i="14018" txt_f="14019">a</offsets></sup><offsets xml_i="23740" xml_f="23871" txt_i="14019" txt_f="14150">One COBAS HPV 16 positive sample had invalid HPV 18 and Other High Risk HPV results (i.e., no beta-globin detected in the specimen)</offsets></p><p><offsets xml_i="23878" xml_f="23879" txt_i="14151" txt_f="14152">
</offsets><italic><offsets xml_i="23887" xml_f="23900" txt_i="14152" txt_f="14165">Abbreviations</offsets></italic><offsets xml_i="23909" xml_f="23911" txt_i="14165" txt_f="14167">: </offsets><italic><offsets xml_i="23919" xml_f="23924" txt_i="14167" txt_f="14172">COBAS</offsets></italic><offsets xml_i="23933" xml_f="23962" txt_i="14172" txt_f="14201"> Roche cobas® 4800 HPV test, </offsets><italic><offsets xml_i="23970" xml_f="23972" txt_i="14201" txt_f="14203">CI</offsets></italic><offsets xml_i="23981" xml_f="24001" txt_i="14203" txt_f="14223"> confidence interval</offsets></p></table-wrap-foot></table-wrap></p></sec><sec id="Sec9"><title><offsets xml_i="24068" xml_f="24100" txt_i="14226" txt_f="14258">HC2 and COBAS result discordance</offsets></title><p><offsets xml_i="24111" xml_f="24289" txt_i="14259" txt_f="14437">Discordant HC2/COBAS results were observed among women of all ages, but were more common among those 25–29 years (34/458; 7.4 %) than those 30 years and older (204/5,714; 3.6 %; </offsets><italic><offsets xml_i="24297" xml_f="24298" txt_i="14437" txt_f="14438">p</offsets></italic><offsets xml_i="24307" xml_f="24438" txt_i="14438" txt_f="14566"> &lt;0.0001). HC2+/COBAS+ specimens had significantly higher median HC2 RLU/CO values compared to those HC2+/COBAS– (54.9 vs. 5.9; </offsets><italic><offsets xml_i="24446" xml_f="24447" txt_i="14566" txt_f="14567">p</offsets></italic><offsets xml_i="24456" xml_f="24566" txt_i="14567" txt_f="14674"> &lt;0.0001). Similarly, HC2+/COBAS+ specimens had significantly lower median COBAS Ct values (30.2 vs. 38.3; </offsets><italic><offsets xml_i="24574" xml_f="24575" txt_i="14674" txt_f="14675">p</offsets></italic><offsets xml_i="24584" xml_f="24696" txt_i="14675" txt_f="14784"> &lt;0.0001) than HC2–/COBAS+ specimens. Median Ct values were similar when stratified by COBAS genotype (Table </offsets><xref rid="Tab3" ref-type="table"><offsets xml_i="24730" xml_f="24731" txt_i="14784" txt_f="14785">3</offsets></xref><offsets xml_i="24738" xml_f="24884" txt_i="14785" txt_f="14931">). Reflex LBC was abnormal for 36.3 % of those HC2+/COBAS+, 23.6 % of HC2+/COBAS–, 4.5 % of HC2–/COBAS+ and 0.5 % of HC2–/COBAS– (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="24920" xml_f="24921" txt_i="14931" txt_f="14932">1</offsets></xref><offsets xml_i="24928" xml_f="24940" txt_i="14932" txt_f="14944">: Table S1).</offsets><table-wrap id="Tab3"><label><offsets xml_i="24969" xml_f="24976" txt_i="14944" txt_f="14951">Table 3</offsets></label><caption><p><offsets xml_i="24996" xml_f="25081" txt_i="14951" txt_f="15036">RLU/CO and Ct values for concordant and discordant digene HC2 and Roche COBAS results</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"></th><th><offsets xml_i="25168" xml_f="25174" txt_i="15037" txt_f="15043">RLU/CO</offsets></th><th><offsets xml_i="25183" xml_f="25191" txt_i="15043" txt_f="15051">Ct (all)</offsets></th><th><offsets xml_i="25200" xml_f="25216" txt_i="15051" txt_f="15067">Ct (HPV 16 only)</offsets></th><th><offsets xml_i="25225" xml_f="25241" txt_i="15067" txt_f="15083">Ct (HPV 18 only)</offsets></th><th><offsets xml_i="25250" xml_f="25263" txt_i="15083" txt_f="15096">Ct (OHR only)</offsets></th></tr><tr><th><offsets xml_i="25281" xml_f="25296" txt_i="15096" txt_f="15111">Median (Q1, Q3)</offsets></th><th><offsets xml_i="25305" xml_f="25320" txt_i="15111" txt_f="15126">Median (Q1, Q3)</offsets></th><th><offsets xml_i="25329" xml_f="25344" txt_i="15126" txt_f="15141">Median (Q1, Q3)</offsets></th><th><offsets xml_i="25353" xml_f="25368" txt_i="15141" txt_f="15156">Median (Q1, Q3)</offsets></th><th><offsets xml_i="25377" xml_f="25392" txt_i="15156" txt_f="15171">Median (Q1, Q3)</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="25425" xml_f="25436" txt_i="15171" txt_f="15182">HC2+/COBAS+</offsets></td><td char="(" align="char"><offsets xml_i="25467" xml_f="25484" txt_i="15183" txt_f="15200">54.9 (8.6, 368.4)</offsets></td><td><offsets xml_i="25493" xml_f="25510" txt_i="15201" txt_f="15218">30.2 (26.6, 34.2)</offsets></td><td><offsets xml_i="25519" xml_f="25536" txt_i="15219" txt_f="15236">29.4 (26.6, 34.2)</offsets></td><td><offsets xml_i="25545" xml_f="25562" txt_i="15237" txt_f="15254">30.7 (27.2, 35.0)</offsets></td><td><offsets xml_i="25571" xml_f="25588" txt_i="15255" txt_f="15272">30.3 (26.3, 33.8)</offsets></td></tr><tr><td><offsets xml_i="25606" xml_f="25617" txt_i="15273" txt_f="15284">HC2+/COBAS−</offsets></td><td char="(" align="char"><offsets xml_i="25648" xml_f="25663" txt_i="15285" txt_f="15300">5.9 (2.7, 16.4)</offsets></td><td><offsets xml_i="25672" xml_f="25673" txt_i="15301" txt_f="15302">–</offsets></td><td><offsets xml_i="25682" xml_f="25683" txt_i="15303" txt_f="15304">–</offsets></td><td><offsets xml_i="25692" xml_f="25693" txt_i="15305" txt_f="15306">–</offsets></td><td><offsets xml_i="25702" xml_f="25703" txt_i="15307" txt_f="15308">–</offsets></td></tr><tr><td><offsets xml_i="25721" xml_f="25732" txt_i="15309" txt_f="15320">HC2−/COBAS+</offsets></td><td char="(" align="char"><offsets xml_i="25763" xml_f="25777" txt_i="15321" txt_f="15335">0.3 (0.2, 0.5)</offsets></td><td><offsets xml_i="25786" xml_f="25803" txt_i="15336" txt_f="15353">38.3 (36.8, 39.1)</offsets></td><td><offsets xml_i="25812" xml_f="25829" txt_i="15354" txt_f="15371">39.0 (38.0, 40.0)</offsets></td><td><offsets xml_i="25838" xml_f="25855" txt_i="15372" txt_f="15389">38.1 (37.0, 38.8)</offsets></td><td><offsets xml_i="25864" xml_f="25881" txt_i="15390" txt_f="15407">38.2 (36.7, 38.9)</offsets></td></tr><tr><td><offsets xml_i="25899" xml_f="25910" txt_i="15408" txt_f="15419">HC2−/COBAS−</offsets></td><td char="(" align="char"><offsets xml_i="25941" xml_f="25955" txt_i="15420" txt_f="15434">0.1 (0.1, 0.2)</offsets></td><td><offsets xml_i="25964" xml_f="25965" txt_i="15435" txt_f="15436">–</offsets></td><td><offsets xml_i="25974" xml_f="25975" txt_i="15437" txt_f="15438">–</offsets></td><td><offsets xml_i="25984" xml_f="25985" txt_i="15439" txt_f="15440">–</offsets></td><td><offsets xml_i="25994" xml_f="25995" txt_i="15441" txt_f="15442">–</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="26041" xml_f="26042" txt_i="15443" txt_f="15444">
</offsets><italic><offsets xml_i="26050" xml_f="26063" txt_i="15444" txt_f="15457">Abbreviations</offsets></italic><offsets xml_i="26072" xml_f="26074" txt_i="15457" txt_f="15459">: </offsets><italic><offsets xml_i="26082" xml_f="26085" txt_i="15459" txt_f="15462">HC2</offsets></italic><offsets xml_i="26094" xml_f="26145" txt_i="15462" txt_f="15513"> Digene Hybrid Capture® 2 high-risk HPV DNA test®, </offsets><italic><offsets xml_i="26153" xml_f="26158" txt_i="15513" txt_f="15518">COBAS</offsets></italic><offsets xml_i="26167" xml_f="26196" txt_i="15518" txt_f="15547"> Roche cobas® 4800 HPV test, </offsets><italic><offsets xml_i="26204" xml_f="26210" txt_i="15547" txt_f="15553">RLU/CO</offsets></italic><offsets xml_i="26219" xml_f="26248" txt_i="15553" txt_f="15582"> relative light unit/cutoff, </offsets><italic><offsets xml_i="26256" xml_f="26258" txt_i="15582" txt_f="15584">Ct</offsets></italic><offsets xml_i="26267" xml_f="26285" txt_i="15584" txt_f="15602"> cycle threshold, </offsets><italic><offsets xml_i="26293" xml_f="26295" txt_i="15602" txt_f="15604">Q1</offsets></italic><offsets xml_i="26304" xml_f="26338" txt_i="15604" txt_f="15638"> first quartile, Q3 third quartile</offsets></p></table-wrap-foot></table-wrap></p></sec><sec id="Sec10"><title><offsets xml_i="26406" xml_f="26475" txt_i="15641" txt_f="15710">Correlation of HC2 and COBAS results with linear array and reflex LBC</offsets></title><p><offsets xml_i="26486" xml_f="26578" txt_i="15711" txt_f="15803">LA confirmed hrHPV genotypes in 96.3 % of HC2+/COBAS+ enrollment specimens (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="26614" xml_f="26615" txt_i="15803" txt_f="15804">1</offsets></xref><offsets xml_i="26622" xml_f="26746" txt_i="15804" txt_f="15928">: Table S1). For those with discordant results, HC2+/COBAS− were less likely to contain hrHPV genotypes (12.3 % vs. 68.9 %; </offsets><italic><offsets xml_i="26754" xml_f="26755" txt_i="15928" txt_f="15929">p</offsets></italic><offsets xml_i="26764" xml_f="26845" txt_i="15929" txt_f="16007"> &lt;0.0001) and more likely to contain only lrHPV genotypes (52.8 % vs. 12.1 %; </offsets><italic><offsets xml_i="26853" xml_f="26854" txt_i="16007" txt_f="16008">p</offsets></italic><offsets xml_i="26863" xml_f="27046" txt_i="16008" txt_f="16188"> &lt;0.0001) than those HC2−/COBAS+. There was no significant difference in the overall abnormal reflex LBC proportions for those with discordant HC2/COBAS results (23.6 % vs. 4.5 %; </offsets><italic><offsets xml_i="27054" xml_f="27055" txt_i="16188" txt_f="16189">p</offsets></italic><offsets xml_i="27064" xml_f="27151" txt_i="16189" txt_f="16276"> = 0.47), nor for those discordant specimens containing only lrHPV (26.8 % vs. 18.8 %; </offsets><italic><offsets xml_i="27159" xml_f="27160" txt_i="16276" txt_f="16277">p</offsets></italic><offsets xml_i="27169" xml_f="27244" txt_i="16277" txt_f="16352"> = 0.75). Of interest, specimens containing multiple hrHPV (43 % vs. 28 %; </offsets><italic><offsets xml_i="27252" xml_f="27253" txt_i="16352" txt_f="16353">p</offsets></italic><offsets xml_i="27262" xml_f="27297" txt_i="16353" txt_f="16388"> = 0.002) and lrHPV (42 % vs. 9 %; </offsets><italic><offsets xml_i="27305" xml_f="27306" txt_i="16388" txt_f="16389">p</offsets></italic><offsets xml_i="27315" xml_f="27449" txt_i="16389" txt_f="16523"> = 0.06) genotypes by LA were more likely to have abnormal LBC compared to those with a single hr- or lrHPV genotype (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="27485" xml_f="27486" txt_i="16523" txt_f="16524">1</offsets></xref><offsets xml_i="27493" xml_f="27505" txt_i="16524" txt_f="16536">: Table S2).</offsets></p></sec><sec id="Sec11"><title><offsets xml_i="27538" xml_f="27551" txt_i="16538" txt_f="16551">CIN detection</offsets></title><p><offsets xml_i="27562" xml_f="27660" txt_i="16552" txt_f="16650">Overall round 1 and 2 CIN2 and CIN3+ rates classified by HC2 and COBAS results are shown in Table </offsets><xref rid="Tab4" ref-type="table"><offsets xml_i="27694" xml_f="27695" txt_i="16650" txt_f="16651">4</offsets></xref><offsets xml_i="27702" xml_f="27974" txt_i="16651" txt_f="16923">. At round 1, the respective rates were 8.6/1,000 (95 % CI: 6.3 – 10.9) and 6.6/1,000 (95 % CI: 4.6 – 8.7), whereas at round 2 (LBC screening, with HC2 triage of ASCUS), the overall rates were 0.7/1,000 (95 % CI: 0.0 – 1.4) and 0.4/1,000 (95 % CI: 0.0 – 0.9) respectively.</offsets><table-wrap id="Tab4"><label><offsets xml_i="28003" xml_f="28010" txt_i="16923" txt_f="16930">Table 4</offsets></label><caption><p><offsets xml_i="28030" xml_f="28112" txt_i="16930" txt_f="17012">Rounds 1 and 2 pathology stratified by baseline Digene HC2 and Roche COBAS results</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="3"><offsets xml_i="28190" xml_f="28208" txt_i="17013" txt_f="17031">Baseline HC2/COBAS</offsets></th><th colspan="7"><offsets xml_i="28229" xml_f="28269" txt_i="17031" txt_f="17071">Round 1 (Baseline + 12 months follow-up)</offsets></th><th colspan="7"><offsets xml_i="28290" xml_f="28309" txt_i="17071" txt_f="17090">Round 2 (24 months)</offsets></th></tr><tr><th rowspan="2"><offsets xml_i="28339" xml_f="28347" txt_i="17090" txt_f="17098">Cytology</offsets></th><th rowspan="2"><offsets xml_i="28368" xml_f="28369" txt_i="17098" txt_f="17099">N</offsets></th><th rowspan="2"><offsets xml_i="28390" xml_f="28414" txt_i="17099" txt_f="17123">Colpo Referred/Completed</offsets></th><th colspan="2"><offsets xml_i="28435" xml_f="28439" txt_i="17123" txt_f="17127">CIN2</offsets></th><th colspan="2"><offsets xml_i="28460" xml_f="28465" txt_i="17127" txt_f="17132">CIN3+</offsets></th><th rowspan="2"><offsets xml_i="28486" xml_f="28494" txt_i="17132" txt_f="17140">Cytology</offsets></th><th rowspan="2"><offsets xml_i="28515" xml_f="28516" txt_i="17140" txt_f="17141">N</offsets></th><th rowspan="2"><offsets xml_i="28537" xml_f="28561" txt_i="17141" txt_f="17165">Colpo Referred/Completed</offsets></th><th colspan="2"><offsets xml_i="28582" xml_f="28586" txt_i="17165" txt_f="17169">CIN2</offsets></th><th colspan="2"><offsets xml_i="28607" xml_f="28612" txt_i="17169" txt_f="17174">CIN3+</offsets></th></tr><tr><th><offsets xml_i="28630" xml_f="28631" txt_i="17174" txt_f="17175">N</offsets></th><th><offsets xml_i="28640" xml_f="28644" txt_i="17175" txt_f="17179">Rate</offsets><sup><offsets xml_i="28649" xml_f="28650" txt_i="17179" txt_f="17180">a</offsets></sup><offsets xml_i="28656" xml_f="28666" txt_i="17180" txt_f="17190"> (95 % CI)</offsets></th><th><offsets xml_i="28675" xml_f="28676" txt_i="17190" txt_f="17191">N</offsets></th><th><offsets xml_i="28685" xml_f="28689" txt_i="17191" txt_f="17195">Rate</offsets><sup><offsets xml_i="28694" xml_f="28695" txt_i="17195" txt_f="17196">a</offsets></sup><offsets xml_i="28701" xml_f="28711" txt_i="17196" txt_f="17206"> (95 % CI)</offsets></th><th><offsets xml_i="28720" xml_f="28721" txt_i="17206" txt_f="17207">N</offsets></th><th><offsets xml_i="28730" xml_f="28734" txt_i="17207" txt_f="17211">Rate</offsets><sup><offsets xml_i="28739" xml_f="28740" txt_i="17211" txt_f="17212">a</offsets></sup><offsets xml_i="28746" xml_f="28756" txt_i="17212" txt_f="17222"> (95 % CI)</offsets></th><th><offsets xml_i="28765" xml_f="28766" txt_i="17222" txt_f="17223">N</offsets></th><th><offsets xml_i="28775" xml_f="28779" txt_i="17223" txt_f="17227">Rate</offsets><sup><offsets xml_i="28784" xml_f="28785" txt_i="17227" txt_f="17228">a</offsets></sup><offsets xml_i="28791" xml_f="28801" txt_i="17228" txt_f="17238"> (95 % CI)</offsets></th></tr></thead><tbody><tr><td rowspan="7"><offsets xml_i="28846" xml_f="28853" txt_i="17238" txt_f="17245">Pos/Pos</offsets></td><td><offsets xml_i="28862" xml_f="28867" txt_i="17246" txt_f="17251">UNSAT</offsets></td><td><offsets xml_i="28876" xml_f="28877" txt_i="17252" txt_f="17253">1</offsets></td><td><offsets xml_i="28886" xml_f="28889" txt_i="17254" txt_f="17257">1/1</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="28934" xml_f="28939" txt_i="17262" txt_f="17267">UNSAT</offsets></td><td><offsets xml_i="28948" xml_f="28949" txt_i="17268" txt_f="17269">1</offsets></td><td><offsets xml_i="28958" xml_f="28961" txt_i="17270" txt_f="17273">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="29015" xml_f="29019" txt_i="17278" txt_f="17282">NILM</offsets></td><td><offsets xml_i="29028" xml_f="29031" txt_i="17283" txt_f="17286">260</offsets></td><td><offsets xml_i="29040" xml_f="29047" txt_i="17287" txt_f="17294">152/143</offsets></td><td><offsets xml_i="29056" xml_f="29058" txt_i="17295" txt_f="17297">22</offsets></td><td></td><td><offsets xml_i="29076" xml_f="29078" txt_i="17299" txt_f="17301">16</offsets></td><td></td><td><offsets xml_i="29096" xml_f="29100" txt_i="17303" txt_f="17307">NILM</offsets></td><td><offsets xml_i="29109" xml_f="29112" txt_i="17308" txt_f="17311">208</offsets></td><td><offsets xml_i="29121" xml_f="29124" txt_i="17312" txt_f="17315">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="29178" xml_f="29183" txt_i="17320" txt_f="17325">ASCUS</offsets></td><td><offsets xml_i="29192" xml_f="29194" txt_i="17326" txt_f="17328">35</offsets></td><td><offsets xml_i="29203" xml_f="29208" txt_i="17329" txt_f="17334">35/33</offsets></td><td><offsets xml_i="29217" xml_f="29218" txt_i="17335" txt_f="17336">6</offsets></td><td></td><td><offsets xml_i="29236" xml_f="29237" txt_i="17338" txt_f="17339">2</offsets></td><td></td><td><offsets xml_i="29255" xml_f="29260" txt_i="17341" txt_f="17346">ASCUS</offsets></td><td><offsets xml_i="29269" xml_f="29270" txt_i="17347" txt_f="17348">4</offsets></td><td><offsets xml_i="29279" xml_f="29282" txt_i="17349" txt_f="17352">3/3</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="29336" xml_f="29340" txt_i="17357" txt_f="17361">LSIL</offsets></td><td><offsets xml_i="29349" xml_f="29351" txt_i="17362" txt_f="17364">68</offsets></td><td><offsets xml_i="29360" xml_f="29365" txt_i="17365" txt_f="17370">68/63</offsets></td><td><offsets xml_i="29374" xml_f="29375" txt_i="17371" txt_f="17372">9</offsets></td><td></td><td><offsets xml_i="29393" xml_f="29394" txt_i="17374" txt_f="17375">4</offsets></td><td></td><td><offsets xml_i="29412" xml_f="29416" txt_i="17377" txt_f="17381">LSIL</offsets></td><td><offsets xml_i="29425" xml_f="29426" txt_i="17382" txt_f="17383">6</offsets></td><td><offsets xml_i="29435" xml_f="29438" txt_i="17384" txt_f="17387">6/6</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="29492" xml_f="29496" txt_i="17392" txt_f="17396">ASCH</offsets></td><td><offsets xml_i="29505" xml_f="29507" txt_i="17397" txt_f="17399">18</offsets></td><td><offsets xml_i="29516" xml_f="29521" txt_i="17400" txt_f="17405">18/15</offsets></td><td><offsets xml_i="29530" xml_f="29531" txt_i="17406" txt_f="17407">4</offsets></td><td></td><td><offsets xml_i="29549" xml_f="29550" txt_i="17409" txt_f="17410">5</offsets></td><td></td><td><offsets xml_i="29568" xml_f="29572" txt_i="17412" txt_f="17416">ASCH</offsets></td><td><offsets xml_i="29581" xml_f="29582" txt_i="17417" txt_f="17418">2</offsets></td><td><offsets xml_i="29591" xml_f="29594" txt_i="17419" txt_f="17422">2/2</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="29648" xml_f="29652" txt_i="17427" txt_f="17431">HSIL</offsets></td><td><offsets xml_i="29661" xml_f="29663" txt_i="17432" txt_f="17434">28</offsets></td><td><offsets xml_i="29672" xml_f="29677" txt_i="17435" txt_f="17440">28/25</offsets></td><td><offsets xml_i="29686" xml_f="29687" txt_i="17441" txt_f="17442">7</offsets></td><td></td><td><offsets xml_i="29705" xml_f="29707" txt_i="17444" txt_f="17446">13</offsets></td><td></td><td><offsets xml_i="29725" xml_f="29729" txt_i="17448" txt_f="17452">HSIL</offsets></td><td><offsets xml_i="29738" xml_f="29739" txt_i="17453" txt_f="17454">1</offsets></td><td><offsets xml_i="29748" xml_f="29751" txt_i="17455" txt_f="17458">1/1</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="29805" xml_f="29810" txt_i="17463" txt_f="17468">Total</offsets></td><td><offsets xml_i="29819" xml_f="29822" txt_i="17469" txt_f="17472">410</offsets></td><td><offsets xml_i="29831" xml_f="29838" txt_i="17473" txt_f="17480">302/280</offsets></td><td><offsets xml_i="29847" xml_f="29849" txt_i="17481" txt_f="17483">48</offsets></td><td><offsets xml_i="29858" xml_f="29874" txt_i="17484" txt_f="17500">7.8 (5.6 – 10.0)</offsets></td><td><offsets xml_i="29883" xml_f="29885" txt_i="17501" txt_f="17503">40</offsets></td><td><offsets xml_i="29894" xml_f="29909" txt_i="17504" txt_f="17519">6.5 (4.5 – 8.5)</offsets></td><td><offsets xml_i="29918" xml_f="29923" txt_i="17520" txt_f="17525">Total</offsets></td><td><offsets xml_i="29932" xml_f="29935" txt_i="17526" txt_f="17529">222</offsets></td><td><offsets xml_i="29944" xml_f="29949" txt_i="17530" txt_f="17535">13/13</offsets></td><td><offsets xml_i="29958" xml_f="29959" txt_i="17536" txt_f="17537">0</offsets></td><td><offsets xml_i="29968" xml_f="29969" txt_i="17538" txt_f="17539">–</offsets></td><td><offsets xml_i="29978" xml_f="29979" txt_i="17540" txt_f="17541">0</offsets></td><td><offsets xml_i="29988" xml_f="29989" txt_i="17542" txt_f="17543">–</offsets></td></tr><tr><td rowspan="7"><offsets xml_i="30019" xml_f="30026" txt_i="17544" txt_f="17551">Pos/Neg</offsets></td><td><offsets xml_i="30035" xml_f="30040" txt_i="17552" txt_f="17557">UNSAT</offsets></td><td><offsets xml_i="30049" xml_f="30050" txt_i="17558" txt_f="17559">0</offsets></td><td><offsets xml_i="30059" xml_f="30062" txt_i="17560" txt_f="17563">0/0</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="30107" xml_f="30112" txt_i="17568" txt_f="17573">UNSAT</offsets></td><td><offsets xml_i="30121" xml_f="30122" txt_i="17574" txt_f="17575">1</offsets></td><td><offsets xml_i="30131" xml_f="30134" txt_i="17576" txt_f="17579">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30188" xml_f="30192" txt_i="17584" txt_f="17588">NILM</offsets></td><td><offsets xml_i="30201" xml_f="30203" txt_i="17589" txt_f="17591">81</offsets></td><td><offsets xml_i="30212" xml_f="30217" txt_i="17592" txt_f="17597">27/23</offsets></td><td><offsets xml_i="30226" xml_f="30227" txt_i="17598" txt_f="17599">4</offsets></td><td></td><td></td><td></td><td><offsets xml_i="30263" xml_f="30267" txt_i="17603" txt_f="17607">NILM</offsets></td><td><offsets xml_i="30276" xml_f="30278" txt_i="17608" txt_f="17610">75</offsets></td><td><offsets xml_i="30287" xml_f="30290" txt_i="17611" txt_f="17614">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30344" xml_f="30349" txt_i="17619" txt_f="17624">ASCUS</offsets></td><td><offsets xml_i="30358" xml_f="30360" txt_i="17625" txt_f="17627">10</offsets></td><td><offsets xml_i="30369" xml_f="30373" txt_i="17628" txt_f="17632">10/9</offsets></td><td><offsets xml_i="30382" xml_f="30383" txt_i="17633" txt_f="17634">1</offsets></td><td></td><td></td><td></td><td><offsets xml_i="30419" xml_f="30424" txt_i="17638" txt_f="17643">ASCUS</offsets></td><td><offsets xml_i="30433" xml_f="30434" txt_i="17644" txt_f="17645">2</offsets></td><td><offsets xml_i="30443" xml_f="30446" txt_i="17646" txt_f="17649">1/1</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30500" xml_f="30504" txt_i="17654" txt_f="17658">LSIL</offsets></td><td><offsets xml_i="30513" xml_f="30515" txt_i="17659" txt_f="17661">13</offsets></td><td><offsets xml_i="30524" xml_f="30529" txt_i="17662" txt_f="17667">13/12</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="30574" xml_f="30578" txt_i="17672" txt_f="17676">LSIL</offsets></td><td><offsets xml_i="30587" xml_f="30588" txt_i="17677" txt_f="17678">4</offsets></td><td><offsets xml_i="30597" xml_f="30600" txt_i="17679" txt_f="17682">4/4</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30654" xml_f="30658" txt_i="17687" txt_f="17691">ASCH</offsets></td><td><offsets xml_i="30667" xml_f="30668" txt_i="17692" txt_f="17693">2</offsets></td><td><offsets xml_i="30677" xml_f="30680" txt_i="17694" txt_f="17697">2/2</offsets></td><td></td><td></td><td><offsets xml_i="30707" xml_f="30708" txt_i="17700" txt_f="17701">1</offsets></td><td></td><td><offsets xml_i="30726" xml_f="30730" txt_i="17703" txt_f="17707">ASCH</offsets></td><td><offsets xml_i="30739" xml_f="30740" txt_i="17708" txt_f="17709">0</offsets></td><td><offsets xml_i="30749" xml_f="30752" txt_i="17710" txt_f="17713">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30806" xml_f="30810" txt_i="17718" txt_f="17722">HSIL</offsets></td><td><offsets xml_i="30819" xml_f="30820" txt_i="17723" txt_f="17724">0</offsets></td><td><offsets xml_i="30829" xml_f="30832" txt_i="17725" txt_f="17728">0/0</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="30877" xml_f="30881" txt_i="17733" txt_f="17737">HSIL</offsets></td><td><offsets xml_i="30890" xml_f="30891" txt_i="17738" txt_f="17739">0</offsets></td><td><offsets xml_i="30900" xml_f="30903" txt_i="17740" txt_f="17743">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30957" xml_f="30962" txt_i="17748" txt_f="17753">Total</offsets></td><td><offsets xml_i="30971" xml_f="30974" txt_i="17754" txt_f="17757">106</offsets></td><td><offsets xml_i="30983" xml_f="30988" txt_i="17758" txt_f="17763">52/46</offsets></td><td><offsets xml_i="30997" xml_f="30998" txt_i="17764" txt_f="17765">5</offsets><sup><offsets xml_i="31003" xml_f="31004" txt_i="17765" txt_f="17766">b</offsets></sup><offsets xml_i="31010" xml_f="31011" txt_i="17766" txt_f="17767">
</offsets></td><td><offsets xml_i="31020" xml_f="31035" txt_i="17768" txt_f="17783">0.8 (0.1 – 1.5)</offsets></td><td><offsets xml_i="31044" xml_f="31045" txt_i="17784" txt_f="17785">1</offsets><sup><offsets xml_i="31050" xml_f="31051" txt_i="17785" txt_f="17786">c</offsets></sup><offsets xml_i="31057" xml_f="31058" txt_i="17786" txt_f="17787">
</offsets></td><td><offsets xml_i="31067" xml_f="31082" txt_i="17788" txt_f="17803">0.2 (0.0 – 0.5)</offsets></td><td><offsets xml_i="31091" xml_f="31096" txt_i="17804" txt_f="17809">Total</offsets></td><td><offsets xml_i="31105" xml_f="31107" txt_i="17810" txt_f="17812">82</offsets></td><td><offsets xml_i="31116" xml_f="31119" txt_i="17813" txt_f="17816">5/5</offsets></td><td><offsets xml_i="31128" xml_f="31129" txt_i="17817" txt_f="17818">0</offsets></td><td><offsets xml_i="31138" xml_f="31139" txt_i="17819" txt_f="17820">–</offsets></td><td><offsets xml_i="31148" xml_f="31149" txt_i="17821" txt_f="17822">0</offsets></td><td><offsets xml_i="31158" xml_f="31159" txt_i="17823" txt_f="17824">–</offsets></td></tr><tr><td rowspan="7"><offsets xml_i="31189" xml_f="31196" txt_i="17825" txt_f="17832">Neg/Pos</offsets></td><td><offsets xml_i="31205" xml_f="31210" txt_i="17833" txt_f="17838">UNSAT</offsets></td><td><offsets xml_i="31219" xml_f="31220" txt_i="17839" txt_f="17840">3</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="31274" xml_f="31279" txt_i="17846" txt_f="17851">UNSAT</offsets></td><td><offsets xml_i="31288" xml_f="31289" txt_i="17852" txt_f="17853">1</offsets></td><td><offsets xml_i="31298" xml_f="31301" txt_i="17854" txt_f="17857">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="31355" xml_f="31359" txt_i="17862" txt_f="17866">NILM</offsets></td><td><offsets xml_i="31368" xml_f="31371" txt_i="17867" txt_f="17870">123</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="31425" xml_f="31429" txt_i="17876" txt_f="17880">NILM</offsets></td><td><offsets xml_i="31438" xml_f="31441" txt_i="17881" txt_f="17884">117</offsets></td><td><offsets xml_i="31450" xml_f="31453" txt_i="17885" txt_f="17888">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="31507" xml_f="31512" txt_i="17893" txt_f="17898">ASCUS</offsets></td><td><offsets xml_i="31521" xml_f="31522" txt_i="17899" txt_f="17900">1</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="31576" xml_f="31581" txt_i="17906" txt_f="17911">ASCUS</offsets></td><td><offsets xml_i="31590" xml_f="31591" txt_i="17912" txt_f="17913">0</offsets></td><td><offsets xml_i="31600" xml_f="31603" txt_i="17914" txt_f="17917">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="31657" xml_f="31661" txt_i="17922" txt_f="17926">LSIL</offsets></td><td><offsets xml_i="31670" xml_f="31671" txt_i="17927" txt_f="17928">5</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="31725" xml_f="31729" txt_i="17934" txt_f="17938">LSIL</offsets></td><td><offsets xml_i="31738" xml_f="31739" txt_i="17939" txt_f="17940">2</offsets></td><td><offsets xml_i="31748" xml_f="31751" txt_i="17941" txt_f="17944">2/2</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="31805" xml_f="31809" txt_i="17949" txt_f="17953">ASCH</offsets></td><td><offsets xml_i="31818" xml_f="31819" txt_i="17954" txt_f="17955">0</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="31873" xml_f="31877" txt_i="17961" txt_f="17965">ASCH</offsets></td><td><offsets xml_i="31886" xml_f="31887" txt_i="17966" txt_f="17967">0</offsets></td><td><offsets xml_i="31896" xml_f="31899" txt_i="17968" txt_f="17971">0/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="31953" xml_f="31957" txt_i="17976" txt_f="17980">HSIL</offsets></td><td><offsets xml_i="31966" xml_f="31967" txt_i="17981" txt_f="17982">0</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="32021" xml_f="32025" txt_i="17988" txt_f="17992">HSIL</offsets></td><td><offsets xml_i="32034" xml_f="32035" txt_i="17993" txt_f="17994">1</offsets></td><td><offsets xml_i="32044" xml_f="32047" txt_i="17995" txt_f="17998">1/1</offsets></td><td><offsets xml_i="32056" xml_f="32057" txt_i="17999" txt_f="18000">1</offsets></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="32102" xml_f="32107" txt_i="18004" txt_f="18009">Total</offsets></td><td><offsets xml_i="32116" xml_f="32119" txt_i="18010" txt_f="18013">132</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="32173" xml_f="32178" txt_i="18019" txt_f="18024">Total</offsets></td><td><offsets xml_i="32187" xml_f="32190" txt_i="18025" txt_f="18028">121</offsets></td><td><offsets xml_i="32199" xml_f="32202" txt_i="18029" txt_f="18032">3/3</offsets></td><td><offsets xml_i="32211" xml_f="32212" txt_i="18033" txt_f="18034">1</offsets><sup><offsets xml_i="32217" xml_f="32218" txt_i="18034" txt_f="18035">d</offsets></sup><offsets xml_i="32224" xml_f="32225" txt_i="18035" txt_f="18036">
</offsets></td><td><offsets xml_i="32234" xml_f="32249" txt_i="18037" txt_f="18052">0.2 (0.0 – 0.5)</offsets></td><td><offsets xml_i="32258" xml_f="32259" txt_i="18053" txt_f="18054">0</offsets></td><td><offsets xml_i="32268" xml_f="32269" txt_i="18055" txt_f="18056">-</offsets></td></tr><tr><td rowspan="9"><offsets xml_i="32299" xml_f="32306" txt_i="18057" txt_f="18064">Neg/Neg</offsets></td><td><offsets xml_i="32315" xml_f="32320" txt_i="18065" txt_f="18070">UNSAT</offsets></td><td><offsets xml_i="32329" xml_f="32331" txt_i="18071" txt_f="18073">12</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="32385" xml_f="32390" txt_i="18079" txt_f="18084">UNSAT</offsets></td><td><offsets xml_i="32399" xml_f="32401" txt_i="18085" txt_f="18087">24</offsets></td><td><offsets xml_i="32410" xml_f="32413" txt_i="18088" txt_f="18091">6/5</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="32467" xml_f="32475" txt_i="18096" txt_f="18104">Not done</offsets></td><td><offsets xml_i="32484" xml_f="32488" txt_i="18105" txt_f="18109">4901</offsets></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="32614" xml_f="32618" txt_i="18122" txt_f="18126">NILM</offsets></td><td><offsets xml_i="32627" xml_f="32630" txt_i="18127" txt_f="18130">585</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="32684" xml_f="32688" txt_i="18136" txt_f="18140">NILM</offsets></td><td><offsets xml_i="32697" xml_f="32701" txt_i="18141" txt_f="18145">4957</offsets></td><td><offsets xml_i="32710" xml_f="32713" txt_i="18146" txt_f="18149">1/0</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="32767" xml_f="32772" txt_i="18154" txt_f="18159">ASCUS</offsets></td><td><offsets xml_i="32781" xml_f="32783" txt_i="18160" txt_f="18162">17</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="32837" xml_f="32842" txt_i="18168" txt_f="18173">ASCUS</offsets></td><td><offsets xml_i="32851" xml_f="32853" txt_i="18174" txt_f="18176">31</offsets></td><td><offsets xml_i="32862" xml_f="32865" txt_i="18177" txt_f="18180">2/2</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="32919" xml_f="32931" txt_i="18185" txt_f="18197">Other atypia</offsets></td><td><offsets xml_i="32940" xml_f="32941" txt_i="18198" txt_f="18199">0</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="32995" xml_f="33007" txt_i="18205" txt_f="18217">Other atypia</offsets></td><td><offsets xml_i="33016" xml_f="33017" txt_i="18218" txt_f="18219">6</offsets></td><td><offsets xml_i="33026" xml_f="33029" txt_i="18220" txt_f="18223">6/6</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="33083" xml_f="33087" txt_i="18228" txt_f="18232">LSIL</offsets></td><td><offsets xml_i="33096" xml_f="33097" txt_i="18233" txt_f="18234">8</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="33151" xml_f="33155" txt_i="18240" txt_f="18244">LSIL</offsets></td><td><offsets xml_i="33164" xml_f="33166" txt_i="18245" txt_f="18247">38</offsets></td><td><offsets xml_i="33175" xml_f="33180" txt_i="18248" txt_f="18253">38/36</offsets></td><td><offsets xml_i="33189" xml_f="33190" txt_i="18254" txt_f="18255">2</offsets></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="33235" xml_f="33239" txt_i="18259" txt_f="18263">ASCH</offsets></td><td><offsets xml_i="33248" xml_f="33249" txt_i="18264" txt_f="18265">1</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="33303" xml_f="33307" txt_i="18271" txt_f="18275">ASCH</offsets></td><td><offsets xml_i="33316" xml_f="33317" txt_i="18276" txt_f="18277">4</offsets></td><td><offsets xml_i="33326" xml_f="33329" txt_i="18278" txt_f="18281">4/4</offsets></td><td></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="33383" xml_f="33387" txt_i="18286" txt_f="18290">HSIL</offsets></td><td><offsets xml_i="33396" xml_f="33397" txt_i="18291" txt_f="18292">0</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="33451" xml_f="33455" txt_i="18298" txt_f="18302">HSIL</offsets></td><td><offsets xml_i="33464" xml_f="33465" txt_i="18303" txt_f="18304">5</offsets></td><td><offsets xml_i="33474" xml_f="33477" txt_i="18305" txt_f="18308">5/5</offsets></td><td><offsets xml_i="33486" xml_f="33487" txt_i="18309" txt_f="18310">1</offsets></td><td></td><td><offsets xml_i="33505" xml_f="33506" txt_i="18312" txt_f="18313">2</offsets></td><td></td></tr><tr><td><offsets xml_i="33533" xml_f="33538" txt_i="18315" txt_f="18320">Total</offsets></td><td><offsets xml_i="33547" xml_f="33551" txt_i="18321" txt_f="18325">5524</offsets></td><td></td><td></td><td></td><td></td><td></td><td><offsets xml_i="33605" xml_f="33610" txt_i="18331" txt_f="18336">Total</offsets></td><td><offsets xml_i="33619" xml_f="33623" txt_i="18337" txt_f="18341">5065</offsets></td><td><offsets xml_i="33632" xml_f="33637" txt_i="18342" txt_f="18347">62/58</offsets></td><td><offsets xml_i="33646" xml_f="33647" txt_i="18348" txt_f="18349">3</offsets><sup><offsets xml_i="33652" xml_f="33653" txt_i="18349" txt_f="18350">e</offsets></sup><offsets xml_i="33659" xml_f="33660" txt_i="18350" txt_f="18351">
</offsets></td><td><offsets xml_i="33669" xml_f="33684" txt_i="18352" txt_f="18367">0.6 (0.0 – 1.3)</offsets></td><td><offsets xml_i="33693" xml_f="33694" txt_i="18368" txt_f="18369">2</offsets><sup><offsets xml_i="33699" xml_f="33700" txt_i="18369" txt_f="18370">f</offsets></sup><offsets xml_i="33706" xml_f="33707" txt_i="18370" txt_f="18371">
</offsets></td><td><offsets xml_i="33716" xml_f="33731" txt_i="18372" txt_f="18387">0.4 (0.0 – 0.9)</offsets></td></tr><tr><td><offsets xml_i="33749" xml_f="33754" txt_i="18388" txt_f="18393">Total</offsets></td><td></td><td><offsets xml_i="33772" xml_f="33776" txt_i="18395" txt_f="18399">6172</offsets></td><td><offsets xml_i="33785" xml_f="33792" txt_i="18400" txt_f="18407">354/326</offsets></td><td><offsets xml_i="33801" xml_f="33803" txt_i="18408" txt_f="18410">53</offsets></td><td><offsets xml_i="33812" xml_f="33828" txt_i="18411" txt_f="18427">8.6 (6.3 – 10.9)</offsets></td><td><offsets xml_i="33837" xml_f="33839" txt_i="18428" txt_f="18430">41</offsets></td><td><offsets xml_i="33848" xml_f="33863" txt_i="18431" txt_f="18446">6.6 (4.6 – 8.7)</offsets></td><td><offsets xml_i="33872" xml_f="33877" txt_i="18447" txt_f="18452">Total</offsets></td><td><offsets xml_i="33886" xml_f="33890" txt_i="18453" txt_f="18457">5490</offsets></td><td><offsets xml_i="33899" xml_f="33904" txt_i="18458" txt_f="18463">83/79</offsets></td><td><offsets xml_i="33913" xml_f="33914" txt_i="18464" txt_f="18465">4</offsets></td><td><offsets xml_i="33923" xml_f="33938" txt_i="18466" txt_f="18481">0.7 (0.0 – 1.4)</offsets></td><td><offsets xml_i="33947" xml_f="33948" txt_i="18482" txt_f="18483">2</offsets></td><td><offsets xml_i="33957" xml_f="33972" txt_i="18484" txt_f="18499">0.4 (0.0 – 0.9)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="34018" xml_f="34019" txt_i="18500" txt_f="18501">
</offsets><sup><offsets xml_i="34024" xml_f="34025" txt_i="18501" txt_f="18502">a</offsets></sup><offsets xml_i="34031" xml_f="34062" txt_i="18502" txt_f="18533">Rate per 1,000 women screened. </offsets><sup><offsets xml_i="34067" xml_f="34068" txt_i="18533" txt_f="18534">b</offsets></sup><offsets xml_i="34074" xml_f="34180" txt_i="18534" txt_f="18640">By Linear Array, two were HPV 52 positive, one HPV 56 positive, one HPV 66 positive and one HPV negative. </offsets><sup><offsets xml_i="34185" xml_f="34186" txt_i="18640" txt_f="18641">c</offsets></sup><offsets xml_i="34192" xml_f="34259" txt_i="18641" txt_f="18708">Linear Array was HPV negative; a LEEP one month later showed CIN2. </offsets><sup><offsets xml_i="34264" xml_f="34265" txt_i="18708" txt_f="18709">d</offsets></sup><offsets xml_i="34271" xml_f="34347" txt_i="18709" txt_f="18785">Linear Array was HPV 16 and HPV 51 positive at both baseline and 24 months. </offsets><sup><offsets xml_i="34352" xml_f="34353" txt_i="18785" txt_f="18786">e</offsets></sup><offsets xml_i="34359" xml_f="34595" txt_i="18786" txt_f="19022">By Linear Array, two were HPV negative at baseline; at 24 months one was HPV 66 positive and the second was HPV 35 and HPV 58 positive. The third case was HPV 66 positive at baseline and HPV 16, HPV 18 and HPV 66 positive at 24 months. </offsets><sup><offsets xml_i="34600" xml_f="34601" txt_i="19022" txt_f="19023">f</offsets></sup><offsets xml_i="34607" xml_f="34766" txt_i="19023" txt_f="19182">By Linear Array, one was HPV 16 and HPV 52 positive at baseline and HPV 52 positive at 24 months; the second was HPV 16 positive at both baseline and 24 months</offsets></p><p><offsets xml_i="34773" xml_f="34774" txt_i="19183" txt_f="19184">
</offsets><italic><offsets xml_i="34782" xml_f="34795" txt_i="19184" txt_f="19197">Abbreviations</offsets></italic><offsets xml_i="34804" xml_f="34806" txt_i="19197" txt_f="19199">: </offsets><italic><offsets xml_i="34814" xml_f="34817" txt_i="19199" txt_f="19202">HC2</offsets></italic><offsets xml_i="34826" xml_f="34877" txt_i="19202" txt_f="19253"> Digene Hybrid Capture® 2 high-risk HPV DNA test®, </offsets><italic><offsets xml_i="34885" xml_f="34890" txt_i="19253" txt_f="19258">COBAS</offsets></italic><offsets xml_i="34899" xml_f="34928" txt_i="19258" txt_f="19287"> Roche cobas® 4800 HPV test, </offsets><italic><offsets xml_i="34936" xml_f="34941" txt_i="19287" txt_f="19292">UNSAT</offsets></italic><offsets xml_i="34950" xml_f="34973" txt_i="19292" txt_f="19315"> smear unsatisfactory; </offsets><italic><offsets xml_i="34981" xml_f="34985" txt_i="19315" txt_f="19319">NILM</offsets></italic><offsets xml_i="34994" xml_f="35048" txt_i="19319" txt_f="19373"> negative for intraepithelial lesions and malignancy, </offsets><italic><offsets xml_i="35056" xml_f="35061" txt_i="19373" txt_f="19378">ASCUS</offsets></italic><offsets xml_i="35070" xml_f="35122" txt_i="19378" txt_f="19430"> atypical squamous cells undetermined significance, </offsets><italic><offsets xml_i="35130" xml_f="35134" txt_i="19430" txt_f="19434">ASCH</offsets></italic><offsets xml_i="35143" xml_f="35197" txt_i="19434" txt_f="19488"> atypical squamous cells, cannot rule out high-grade, </offsets><italic><offsets xml_i="35205" xml_f="35209" txt_i="19488" txt_f="19492">LSIL</offsets></italic><offsets xml_i="35218" xml_f="35262" txt_i="19492" txt_f="19536"> low-grade squamous intraepithelial lesion, </offsets><italic><offsets xml_i="35270" xml_f="35274" txt_i="19536" txt_f="19540">HSIL</offsets></italic><offsets xml_i="35283" xml_f="35328" txt_i="19540" txt_f="19585"> high-grade squamous intraepithelial lesion, </offsets><italic><offsets xml_i="35336" xml_f="35339" txt_i="19585" txt_f="19588">CIN</offsets></italic><offsets xml_i="35348" xml_f="35385" txt_i="19588" txt_f="19625"> cervical intraepithelial neoplasia, </offsets><italic><offsets xml_i="35393" xml_f="35397" txt_i="19625" txt_f="19629">LEEP</offsets></italic><offsets xml_i="35406" xml_f="35446" txt_i="19629" txt_f="19669"> loop electrosurgical excision procedure</offsets></p></table-wrap-foot></table-wrap></p><p><offsets xml_i="35488" xml_f="36864" txt_i="19671" txt_f="21047">Of the round 1 CIN2 and CIN3+, 48/53 (90.6 %) and 40/41 (97.6 %) respectively were concordant HC2+/COBAS+. The CIN3 identified among HC2+/COBAS− discordant subjects was HPV negative by LA. This subject underwent a loop electrosurgical excision procedure (LEEP) 1 month following the biopsy, at which time CIN2 was identified, suggesting that this lesion may have been destined to regress spontaneously. Immunohistochemistry for p16 was not performed on either the biopsy or the LEEP specimen. Due to the study blinding, it was not possible to obtain histopathology at round 1 for discordant HC2−/COBAS+ subjects, who were not referred to colposcopy unless they had abnormal LBC at round 2, 24 months later. At the 24 month exit LBC screen, among the enrollment HC2−/COBAS+ subjects, one CIN2 was identified, and at baseline, this subject was HPV 16 and HPV 51 positive by both COBAS and LA with NILM LBC. At round 2, LBC had progressed to high-grade squamous intraepithelial lesion and the subject was still HPV 16 and HPV 51 positive by both COBAS and LA. Of the two CIN3 at round 2 among enrollment HC2−/COBAS− subjects, one was LA HPV 16 and HPV 52 positive at enrollment and HPV 52 positive at exit; the second was HPV 16 positive at both enrollment and exit. This suggests that these lesions may have been present at baseline but were not detected by either HC2 or COBAS.</offsets></p><p><offsets xml_i="36871" xml_f="37336" txt_i="21048" txt_f="21513">Of the 94 CIN2+ identified in Round 1, 52 (55 %) had LBC ≥ASCUS at baseline; of the remaining subjects with NILM LBC at baseline, 16 (17 %) were HPV 16 or 18 positive. Therefore, if both abnormal LBC and presence of HPV 16/18 genotypes had been used for colposcopy referral, 68/94 (72 %) CIN2+ cases would likely have been identified at the baseline screen, with the remainder identified when 12 month follow-up testing indicated persistent non-HPV 16/18 infection.</offsets></p><p><offsets xml_i="37343" xml_f="37679" txt_i="21514" txt_f="21850">Regular audits of the provincial registry for CIN2+ lesions that were not based on a FOCAL trial colposcopy referral identified one adenocarcinoma and one endometrial cancer, both diagnosed ~20 months after the baseline screens (HC2, COBAS and LA were negative for both cases). Both subjects underwent hysterectomy and exited the trial.</offsets></p></sec><sec id="Sec12"><title><offsets xml_i="37712" xml_f="37742" txt_i="21852" txt_f="21882">HC2 clearance and CIN outcomes</offsets></title><p><offsets xml_i="37753" xml_f="37760" txt_i="21883" txt_f="21890">Figure </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="37792" xml_f="37793" txt_i="21890" txt_f="21891">2</offsets></xref><offsets xml_i="37800" xml_f="38365" txt_i="21891" txt_f="22456"> illustrates the round 1 HPV clearance rates and CIN outcomes for the 516 baseline HC2 positive women. Of those HC2+/LBC NILM who attended the 12 month follow-up screen, 177/320 (55.3 %) were persistently HC2 positive, among whom 26/53 (49.1 %) of the round 1 CIN2 and 16/41 (39.0 %) of the round 1 CIN3+ were identified. LBC progression from NILM at baseline to ≥ASCUS at 12 months was not associated with a higher likelihood of having CIN2+ vs. those without LBC progression. There was no CIN2+ among the 143 women who cleared their hrHPV infections by 12 months.</offsets><fig id="Fig2"><label><offsets xml_i="38387" xml_f="38393" txt_i="22456" txt_f="22462">Fig. 2</offsets></label><caption><p><offsets xml_i="38413" xml_f="38501" txt_i="22462" txt_f="22550">Round 1 CIN Outcomes and HPV Clearance Rates for Baseline Digene HC2 Positive Subjects. </offsets><italic><offsets xml_i="38509" xml_f="38522" txt_i="22550" txt_f="22563">Abbreviations</offsets></italic><offsets xml_i="38531" xml_f="38533" txt_i="22563" txt_f="22565">: </offsets><italic><offsets xml_i="38541" xml_f="38544" txt_i="22565" txt_f="22568">HC2</offsets></italic><offsets xml_i="38553" xml_f="38604" txt_i="22568" txt_f="22619"> Digene Hybrid Capture® 2 high-risk HPV DNA test®, </offsets><italic><offsets xml_i="38612" xml_f="38615" txt_i="22619" txt_f="22622">LBC</offsets></italic><offsets xml_i="38624" xml_f="38648" txt_i="22622" txt_f="22646"> liquid-based cytology, </offsets><italic><offsets xml_i="38656" xml_f="38661" txt_i="22646" txt_f="22651">ASCUS</offsets></italic><offsets xml_i="38670" xml_f="38723" txt_i="22651" txt_f="22704"> atypical squamous cells, undetermined significance, </offsets><italic><offsets xml_i="38731" xml_f="38735" txt_i="22704" txt_f="22708">NILM</offsets></italic><offsets xml_i="38744" xml_f="38796" txt_i="22708" txt_f="22760"> negative for intraepithelial lesions and malignancy</offsets></p></caption><graphic xlink:href="12885_2015_1959_Fig2_HTML" id="MO2"></graphic></fig></p></sec><sec id="Sec13"><title><offsets xml_i="38916" xml_f="38966" txt_i="22763" txt_f="22813">Colposcopy triage by reflex LBC and HPV genotyping</offsets></title><p><offsets xml_i="38977" xml_f="39180" txt_i="22814" txt_f="23017">Since HC2−/COBAS+ women were not referred to colposcopy at round 1, assessment of the contributions of HPV genotype and reflex LBC to CIN2 and CIN3+ outcomes was performed only for HC2+/COBAS+ subjects (</offsets><italic><offsets xml_i="39188" xml_f="39189" txt_i="23017" txt_f="23018">n</offsets></italic><offsets xml_i="39198" xml_f="39430" txt_i="23018" txt_f="23250"> = 410). For CIN2, by COBAS, 11/48 (23 %) were HPV 16 positive; 7/48 (15 %) were HPV 18 positive; and 32/48 (67 %) were positive only for non-HPV 16/18 OHR HPV. For CIN2, there were no significant differences based on HPV genotype (</offsets><italic><offsets xml_i="39438" xml_f="39439" txt_i="23250" txt_f="23251">p</offsets></italic><offsets xml_i="39448" xml_f="39464" txt_i="23251" txt_f="23267"> = 0.60) (Table </offsets><xref rid="Tab5" ref-type="table"><offsets xml_i="39498" xml_f="39499" txt_i="23267" txt_f="23268">5</offsets></xref><offsets xml_i="39506" xml_f="39817" txt_i="23268" txt_f="23579">). For CIN3+, 23/40 (58 %) were HPV 16 positive, 6/40 (15 %) were HPV 18 positive and 13/40 (33 %) were positive only for non-HPV 16/18 OHR HPV. HPV 16 positive subjects were significantly more likely to have CIN3+ compared to those positive for non-HPV 16 genotypes (odds ratio [OR] 5.11, 95 % CI 2.30, 11.37; </offsets><italic><offsets xml_i="39825" xml_f="39826" txt_i="23579" txt_f="23580">p</offsets></italic><offsets xml_i="39835" xml_f="40299" txt_i="23580" txt_f="24044"> = 0.0003), but there was no significant difference for HPV 18 positive vs. non-HPV 16/18 OHR positive subjects (OR 2.62; 95 % CI 0.73, 9.49). HPV 16 positive subjects had a higher proportion of CIN3+ (58 % [23/41]) vs. 23 % [11/48] for CIN2), whereas the proportions were the same for HPV 18 (15 % [7/48] vs. 15 % [6/40]). Conversely, subjects positive only for non-HPV 16/18 OHR HPV had a higher CIN2 proportion compared to CIN3+ (67 % [32/48] vs. 33 % [13/40]).</offsets><table-wrap id="Tab5"><label><offsets xml_i="40328" xml_f="40335" txt_i="24044" txt_f="24051">Table 5</offsets></label><caption><p><offsets xml_i="40355" xml_f="40443" txt_i="24051" txt_f="24139">Multivariate analysis for association of HPV genotype and reflex LBC with CIN2 and CIN3+</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th rowspan="2"></th><th colspan="2"><offsets xml_i="40542" xml_f="40546" txt_i="24140" txt_f="24144">CIN2</offsets></th><th colspan="2"><offsets xml_i="40567" xml_f="40572" txt_i="24144" txt_f="24149">CIN3+</offsets></th></tr><tr><th><offsets xml_i="40590" xml_f="40591" txt_i="24149" txt_f="24150">
</offsets><italic><offsets xml_i="40599" xml_f="40600" txt_i="24150" txt_f="24151">p</offsets></italic><offsets xml_i="40609" xml_f="40615" txt_i="24151" txt_f="24157"> value</offsets></th><th><offsets xml_i="40624" xml_f="40644" txt_i="24157" txt_f="24177">Odds ratio (95 % CI)</offsets></th><th><offsets xml_i="40653" xml_f="40654" txt_i="24177" txt_f="24178">
</offsets><italic><offsets xml_i="40662" xml_f="40663" txt_i="24178" txt_f="24179">p</offsets></italic><offsets xml_i="40672" xml_f="40678" txt_i="24179" txt_f="24185"> value</offsets></th><th><offsets xml_i="40687" xml_f="40707" txt_i="24185" txt_f="24205">Odds ratio (95 % CI)</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="40740" xml_f="40752" txt_i="24205" txt_f="24217">HPV genotype</offsets></td><td><offsets xml_i="40761" xml_f="40765" txt_i="24218" txt_f="24222">0.60</offsets></td><td></td><td><offsets xml_i="40783" xml_f="40784" txt_i="24224" txt_f="24225">
</offsets><bold><offsets xml_i="40790" xml_f="40796" txt_i="24225" txt_f="24231">0.0003</offsets></bold><offsets xml_i="40803" xml_f="40804" txt_i="24231" txt_f="24232">
</offsets></td><td></td></tr><tr><td><offsets xml_i="40831" xml_f="40849" txt_i="24234" txt_f="24252"> 16 vs. non-16 OHR</offsets></td><td></td><td><offsets xml_i="40867" xml_f="40884" txt_i="24254" txt_f="24271">0.71 (0.33, 1.53)</offsets></td><td></td><td><offsets xml_i="40902" xml_f="40903" txt_i="24273" txt_f="24274">
</offsets><bold><offsets xml_i="40909" xml_f="40927" txt_i="24274" txt_f="24292">5.11 (2.30, 11.37)</offsets></bold><offsets xml_i="40934" xml_f="40935" txt_i="24292" txt_f="24293">
</offsets></td></tr><tr><td><offsets xml_i="40953" xml_f="40974" txt_i="24294" txt_f="24315"> 18 vs. non-16/18 OHR</offsets></td><td></td><td><offsets xml_i="40992" xml_f="41009" txt_i="24317" txt_f="24334">1.21 (0.42, 3.50)</offsets></td><td></td><td><offsets xml_i="41027" xml_f="41044" txt_i="24336" txt_f="24353">2.62 (0.73, 9.49)</offsets></td></tr><tr><td><offsets xml_i="41062" xml_f="41072" txt_i="24354" txt_f="24364">Reflex LBC</offsets></td><td><offsets xml_i="41081" xml_f="41085" txt_i="24365" txt_f="24369">0.15</offsets></td><td></td><td><offsets xml_i="41103" xml_f="41104" txt_i="24371" txt_f="24372">
</offsets><bold><offsets xml_i="41110" xml_f="41120" txt_i="24372" txt_f="24379">&lt;0.0001</offsets></bold><offsets xml_i="41127" xml_f="41128" txt_i="24379" txt_f="24380">
</offsets></td><td></td></tr><tr><td><offsets xml_i="41155" xml_f="41166" txt_i="24382" txt_f="24393"> High-grade</offsets><sup><offsets xml_i="41171" xml_f="41172" txt_i="24393" txt_f="24394">a</offsets></sup><offsets xml_i="41178" xml_f="41187" txt_i="24394" txt_f="24403"> vs. NILM</offsets></td><td></td><td><offsets xml_i="41205" xml_f="41222" txt_i="24405" txt_f="24422">2.23 (0.96, 5.17)</offsets></td><td></td><td><offsets xml_i="41240" xml_f="41241" txt_i="24424" txt_f="24425">
</offsets><bold><offsets xml_i="41247" xml_f="41265" txt_i="24425" txt_f="24443">5.99 (2.53, 14.18)</offsets></bold><offsets xml_i="41272" xml_f="41273" txt_i="24443" txt_f="24444">
</offsets></td></tr><tr><td><offsets xml_i="41291" xml_f="41301" txt_i="24445" txt_f="24455"> Low-grade</offsets><sup><offsets xml_i="41306" xml_f="41307" txt_i="24455" txt_f="24456">b</offsets></sup><offsets xml_i="41313" xml_f="41322" txt_i="24456" txt_f="24465"> vs. NILM</offsets></td><td></td><td><offsets xml_i="41340" xml_f="41357" txt_i="24467" txt_f="24484">1.02 (0.50, 2.08)</offsets></td><td></td><td><offsets xml_i="41375" xml_f="41392" txt_i="24486" txt_f="24503">0.54 (0.20, 1.49)</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="41438" xml_f="41439" txt_i="24504" txt_f="24505">
</offsets><sup><offsets xml_i="41444" xml_f="41445" txt_i="24505" txt_f="24506">a</offsets></sup><offsets xml_i="41451" xml_f="41477" txt_i="24506" txt_f="24532">High-grade = ASCH + HSIL. </offsets><sup><offsets xml_i="41482" xml_f="41483" txt_i="24532" txt_f="24533">b</offsets></sup><offsets xml_i="41489" xml_f="41513" txt_i="24533" txt_f="24557">Low-grade = ASCUS + LSIL</offsets></p><p><offsets xml_i="41520" xml_f="41521" txt_i="24558" txt_f="24559">
</offsets><italic><offsets xml_i="41529" xml_f="41542" txt_i="24559" txt_f="24572">Abbreviations</offsets></italic><offsets xml_i="41551" xml_f="41553" txt_i="24572" txt_f="24574">: </offsets><italic><offsets xml_i="41561" xml_f="41564" txt_i="24574" txt_f="24577">OHR</offsets></italic><offsets xml_i="41573" xml_f="41595" txt_i="24577" txt_f="24599"> other high-risk HPV, </offsets><italic><offsets xml_i="41603" xml_f="41606" txt_i="24599" txt_f="24602">LBC</offsets></italic><offsets xml_i="41615" xml_f="41639" txt_i="24602" txt_f="24626"> liquid-based cytology, </offsets><italic><offsets xml_i="41647" xml_f="41651" txt_i="24626" txt_f="24630">NILM</offsets></italic><offsets xml_i="41660" xml_f="41714" txt_i="24630" txt_f="24684"> negative for intraepithelial lesions and malignancy, </offsets><italic><offsets xml_i="41722" xml_f="41725" txt_i="24684" txt_f="24687">CIN</offsets></italic><offsets xml_i="41734" xml_f="41771" txt_i="24687" txt_f="24724"> cervical intraepithelial neoplasia, </offsets><italic><offsets xml_i="41779" xml_f="41784" txt_i="24724" txt_f="24729">ASCUS</offsets></italic><offsets xml_i="41793" xml_f="41845" txt_i="24729" txt_f="24781"> atypical squamous cells undetermined significance, </offsets><italic><offsets xml_i="41853" xml_f="41857" txt_i="24781" txt_f="24785">ASCH</offsets></italic><offsets xml_i="41866" xml_f="41920" txt_i="24785" txt_f="24839"> atypical squamous cells, cannot rule out high-grade; </offsets><italic><offsets xml_i="41928" xml_f="41932" txt_i="24839" txt_f="24843">LSIL</offsets></italic><offsets xml_i="41941" xml_f="41985" txt_i="24843" txt_f="24887"> low-grade squamous intraepithelial lesion, </offsets><italic><offsets xml_i="41993" xml_f="41997" txt_i="24887" txt_f="24891">HSIL</offsets></italic><offsets xml_i="42006" xml_f="42049" txt_i="24891" txt_f="24934"> high-grade squamous intraepithelial lesion</offsets></p><p><offsets xml_i="42056" xml_f="42111" txt_i="24935" txt_f="24990">Bolded type indicates statistically significant results</offsets></p></table-wrap-foot></table-wrap></p><p><offsets xml_i="42153" xml_f="42280" txt_i="24992" txt_f="25119">For reflex LBC, those with high-grade vs. NILM LBC were significantly more likely to have CIN3+ (OR 5.99, 95 % CI 2.53, 14.18; </offsets><italic><offsets xml_i="42288" xml_f="42289" txt_i="25119" txt_f="25120">p</offsets></italic><offsets xml_i="42298" xml_f="42476" txt_i="25120" txt_f="25295"> &lt;0.0001) compared to those with low-grade vs. NILM LBC. In contrast, there was no significant difference for CIN2 for those with either high-grade or low-grade vs. NILM LBC (</offsets><italic><offsets xml_i="42484" xml_f="42485" txt_i="25295" txt_f="25296">p</offsets></italic><offsets xml_i="42494" xml_f="42589" txt_i="25296" txt_f="25391"> = 0.15). There were no significant interactions between HPV genotype and LBC (data not shown).</offsets></p></sec></sec><sec id="Sec14" sec-type="discussion"><title><offsets xml_i="42650" xml_f="42660" txt_i="25394" txt_f="25404">Discussion</offsets></title><p><offsets xml_i="42671" xml_f="43603" txt_i="25405" txt_f="26337">In this sub-study of the HPV FOCAL Trial Safety Arm, we demonstrated that over a 2 year screening interval with hrHPV screening at baseline followed by LBC screening at 24 months, hrHPV testing at baseline identified 94 % of the CIN2+ lesions detected over the two screening rounds. Very low CIN2+ rates were observed at the exit LBC screen, which supports the safety of hrHPV vs. LBC screening every 2 years. It is important to note that in this study women were managed only on their HC2 and corresponding cytology results (COBAS results were blinded). Despite the occurrence of discordant baseline HC2 and COBAS results, 43/44 round 1 CIN3+ were among those who were both HC2 and COBAS positive. If women had been managed on both HC2 and COBAS results, it is possible that additional CIN lesions might have been identified at baseline among those HC2−/COBAS+. In agreement with our findings, a comparison study by Cuzick et al. [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="43635" xml_f="43636" txt_i="26337" txt_f="26338">7</offsets></xref><offsets xml_i="43643" xml_f="44313" txt_i="26338" txt_f="27008">] of the performance of six HPV assays in a population referred to colposcopy on the basis of abnormal cytology, showed that both HC2 and COBAS were positive for all CIN3+ cases. However, in that study HPV positive/cytology normal women, among whom greater discordance between various HPV assays and CIN detection might be expected, were not referred to colposcopy. The full impact of discordant HPV results among cytologically normal women remains under investigation. It is not unexpected that a given hrHPV assay may miss a potentially significant CIN, as other trials, including ALTS, ARTISTIC and POBASCAM, noted that a proportion of CIN2+ test negative for hrHPV [</offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="44346" xml_f="44348" txt_i="27008" txt_f="27010">20</offsets></xref><offsets xml_i="44355" xml_f="44356" txt_i="27010" txt_f="27011">–</offsets><xref ref-type="bibr" rid="CR22"><offsets xml_i="44389" xml_f="44391" txt_i="27011" txt_f="27013">22</offsets></xref><offsets xml_i="44398" xml_f="44730" txt_i="27013" txt_f="27345">]. Of note, co-testing would have identified the CIN3+ that was not detected by COBAS at baseline, since the LBC result was atypical squamous cells, cannot rule out high-grade, which would have resulted in a colposcopy referral. Some studies have shown that co-testing has a higher sensitivity for CIN2+ than hrHPV screening alone [</offsets><xref ref-type="bibr" rid="CR23"><offsets xml_i="44763" xml_f="44765" txt_i="27345" txt_f="27347">23</offsets></xref><offsets xml_i="44772" xml_f="44774" txt_i="27347" txt_f="27349">, </offsets><xref ref-type="bibr" rid="CR24"><offsets xml_i="44807" xml_f="44809" txt_i="27349" txt_f="27351">24</offsets></xref><offsets xml_i="44816" xml_f="44870" txt_i="27351" txt_f="27405">], whereas others have not demonstrated a difference [</offsets><xref ref-type="bibr" rid="CR25"><offsets xml_i="44903" xml_f="44905" txt_i="27405" txt_f="27407">25</offsets></xref><offsets xml_i="44912" xml_f="44914" txt_i="27407" txt_f="27409">, </offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="44947" xml_f="44949" txt_i="27409" txt_f="27411">26</offsets></xref><offsets xml_i="44956" xml_f="44958" txt_i="27411" txt_f="27413">].</offsets></p><p><offsets xml_i="44965" xml_f="45223" txt_i="27414" txt_f="27672">The round 1 CIN2 and CIN3+ rates based on primary hrHPV screening (8.6/1,000 and 6.6/1,000 respectively) were higher than in the HPV FOCAL Control Arm (LBC screening, with HC2 triage of ASCUS) which had round 1 rates of 6.0/1,000 and 5.0/1,000 respectively [</offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="45256" xml_f="45258" txt_i="27672" txt_f="27674">27</offsets></xref><offsets xml_i="45265" xml_f="45464" txt_i="27674" txt_f="27873">]. The CIN2+ rate in this study (15.2/1,000) was also higher than the 2011 CIN2+ rate of 6.3/1,000 for women aged 20–69 in the BC provincial cervical screening program (using conventional cytology) [</offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="45497" xml_f="45499" txt_i="27873" txt_f="27875">28</offsets></xref><offsets xml_i="45506" xml_f="45658" txt_i="27875" txt_f="28027">]. The round 1 CIN rates in our study are similar to the verification bias-adjusted rates in the ATHENA trial (CIN2: 8.6 vs. 8.0; CIN3+: 6.6 vs. 10.0) [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="45691" xml_f="45693" txt_i="28027" txt_f="28029">29</offsets></xref><offsets xml_i="45700" xml_f="46172" txt_i="28029" txt_f="28501">]. The FOCAL trial did not adjust for verification bias, but there are some differences in the study populations, e.g., the mean age of FOCAL vs. ATHENA subjects (46 vs. 40 years) and those who ever smoked (36.4 % vs. 29.3 %). The FOCAL trial had a higher proportion of individuals with Asian ethnic background (12.5 % vs. 1.6 %), whereas ATHENA had a higher proportion of Black/African American individuals (14.0 %) compared to FOCAL (3.3 %, which includes Aboriginals) [</offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="46205" xml_f="46207" txt_i="28501" txt_f="28503">27</offsets></xref><offsets xml_i="46214" xml_f="46216" txt_i="28503" txt_f="28505">, </offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="46249" xml_f="46251" txt_i="28505" txt_f="28507">29</offsets></xref><offsets xml_i="46258" xml_f="46260" txt_i="28507" txt_f="28509">].</offsets></p><p><offsets xml_i="46267" xml_f="46349" txt_i="28510" txt_f="28592">While our study had a lower COBAS positivity rate than ATHENA (8.8 % vs. 12.6 %) [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="46382" xml_f="46384" txt_i="28592" txt_f="28594">29</offsets></xref><offsets xml_i="46391" xml_f="46565" txt_i="28594" txt_f="28768">], in both studies HPV 16 (2.1 % vs. 2.8 %) was approximately three times more commonly detected than HPV 18 (0.7 % vs. 1.0 %). These data are consistent with other studies [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="46597" xml_f="46598" txt_i="28768" txt_f="28769">7</offsets></xref><offsets xml_i="46605" xml_f="46782" txt_i="28769" txt_f="28946">], including a recent BC population-based HPV prevalence survey among unvaccinated women aged 15–69 years attending cervical screening (HPV 16: 2.7 %; HPV 18: 1.1 %, using LA) [</offsets><xref ref-type="bibr" rid="CR30"><offsets xml_i="46815" xml_f="46817" txt_i="28946" txt_f="28948">30</offsets></xref><offsets xml_i="46824" xml_f="47187" txt_i="28948" txt_f="29311">]. Our study identified similar HPV 16- but higher HPV 18-attributable CIN2 and CIN3+ proportions compared to ATHENA. We identified HPV 16 in 23 % (11/48) of CIN2 vs. 30 % (57/192) for ATHENA, and 58 % (23/40) vs. 50 % (153/305) for CIN3+. We identified HPV 18 in 15 % (7/40) of CIN2 compared to 3 % (6/192) for ATHENA, and 15 % (6/40) vs. 8 % (25/305) for CIN3+.</offsets></p><p><offsets xml_i="47194" xml_f="47821" txt_i="29312" txt_f="29939">For concordant HC2/COBAS positive subjects, CIN3+ was significantly more likely when LBC was high-grade vs. NILM. Reflex LBC dysplasia was not significantly different for CIN2. CIN3+ was significantly more likely for HPV 16 vs. non-HPV 16 HPV positive subjects, while no significant differences in CIN3+ were found for HPV 18 vs. non-HPV 16/18 OHR genotypes. There was no significant difference between the CIN2-associated HPV genotypes. These findings are in agreement with the ATHENA trial which concluded that a combination of HPV 16/18 genotyping and reflex cytology is a suitable triage strategy for hrHPV positive women [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="47854" xml_f="47856" txt_i="29939" txt_f="29941">26</offsets></xref><offsets xml_i="47863" xml_f="47865" txt_i="29941" txt_f="29943">].</offsets></p><p><offsets xml_i="47872" xml_f="48304" txt_i="29944" txt_f="30376">Specimens with discordant HC2/COBAS results had RLU/CO ratios and Ct values significantly closer to the test limits of detection than concordant positive specimens. It seems reasonable that specimens likely to have lower viral loads (based on low RLU/CO ratios or high Ct values) would be more likely to produce discordant results with different assays. Furthermore, some of these discordant results likely reflect false-positives [</offsets><xref ref-type="bibr" rid="CR31"><offsets xml_i="48337" xml_f="48339" txt_i="30376" txt_f="30378">31</offsets></xref><offsets xml_i="48346" xml_f="48484" txt_i="30378" txt_f="30516">], which are less likely to be consistent between assays. This is supported by the lower CIN2+ rates among women with HC2+/COBAS− results.</offsets></p><p><offsets xml_i="48491" xml_f="48653" txt_i="30517" txt_f="30679">Our finding that 45 % of HC2 positive women with NILM LBC at baseline cleared their hrHPV infections after 12 months is consistent with the published literature [</offsets><xref ref-type="bibr" rid="CR11"><offsets xml_i="48686" xml_f="48688" txt_i="30679" txt_f="30681">11</offsets></xref><offsets xml_i="48695" xml_f="48697" txt_i="30681" txt_f="30683">, </offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="48730" xml_f="48732" txt_i="30683" txt_f="30685">32</offsets></xref><offsets xml_i="48739" xml_f="48741" txt_i="30685" txt_f="30687">, </offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="48774" xml_f="48776" txt_i="30687" txt_f="30689">33</offsets></xref><offsets xml_i="48783" xml_f="49032" txt_i="30689" txt_f="30938">]. The high rate of HPV clearance supports recommendations that when LBC is used for triage, an appropriate way to manage hrHPV positive women with NILM cytology is by re-testing for HPV clearance after a short follow-up period, e.g., 12–18 months [</offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="49065" xml_f="49067" txt_i="30938" txt_f="30940">33</offsets></xref><offsets xml_i="49074" xml_f="49076" txt_i="30940" txt_f="30942">, </offsets><xref ref-type="bibr" rid="CR34"><offsets xml_i="49109" xml_f="49111" txt_i="30942" txt_f="30944">34</offsets></xref><offsets xml_i="49118" xml_f="49239" txt_i="30944" txt_f="31065">]. However, this may result in a high rate of loss to follow-up of these women, some of whom will have underlying CIN2+ [</offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="49271" xml_f="49272" txt_i="31065" txt_f="31066">9</offsets></xref><offsets xml_i="49279" xml_f="49281" txt_i="31066" txt_f="31068">, </offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="49314" xml_f="49316" txt_i="31068" txt_f="31070">20</offsets></xref><offsets xml_i="49323" xml_f="49325" txt_i="31070" txt_f="31072">, </offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="49358" xml_f="49360" txt_i="31072" txt_f="31074">26</offsets></xref><offsets xml_i="49367" xml_f="49525" txt_i="31074" txt_f="31232">]. A triage test which efficiently identifies a high likelihood of CIN2+ at the time of the initial positive HPV screen test would be of considerable benefit.</offsets></p><p><offsets xml_i="49532" xml_f="49843" txt_i="31233" txt_f="31544">Specimens containing only lrHPV types frequently had low-grade LBC abnormalities [11 % (57/516) of HC2 and 3 % (17/542) of COBAS positive results] which may have implications for selection of a primary HPV screening assay if LBC is to be used for triage. Since lrHPV’s have a negligible association with CIN2+ [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="49876" xml_f="49878" txt_i="31544" txt_f="31546">35</offsets></xref><offsets xml_i="49885" xml_f="50394" txt_i="31546" txt_f="32055">], these individuals would not benefit from colposcopy referral. Full genotyping of all hrHPV positives will become increasingly feasible with the availability of automated commercial genotyping assays and this information could be used to prioritize women for colposcopy. However, cost considerations may preclude its use to help identify women with false positive hrHPV screening results. This emphasizes the need for better triage tests to avoid colposcopy over-referral and the associated potential harms.</offsets></p><p><offsets xml_i="50401" xml_f="51985" txt_i="32056" txt_f="33640">An important limitation of this study was the inability to obtain CIN outcomes at round 1 for women with HC2−/COBAS+ discordant results, since baseline COBAS results were blinded until after study exit. This precluded comparison of the sensitivity and specificity of the HC2 and COBAS assays. In addition, had these women been referred to colposcopy at baseline, it is possible that additional CIN lesions would have been detected, as was observed for the HC2+/COBAS− group. Furthermore, no hrHPV testing was performed at the 24 month study exit except for HC2 triage of women with ASCUS LBC. Although one CIN2 was identified at round 2 among baseline HC2−/COBAS+ women, it is possible that the use of only LBC at round 2 was not sufficiently sensitive to demonstrate whether this group of women had underlying CIN lesions not detected by HC2 at baseline. A future analysis of the FOCAL Intervention Arm where women were screened with HC2 and COBAS at both enrollment and the 48 month exit screen may provide further insight. However, Intervention Arm COBAS enrollment results were also blinded and not used for subject management, but both HC2 and COBAS were used for colposcopy referral at the 48 month exit screen. Furthermore, there is an opportunity to follow these women for significant CIN outcomes over longer time periods once they exit the trial and return to the centralized cervical screening program. Long term follow-up of Swedescreen subjects after a baseline hrHPV positive screen showed that incidence rate ratios for CIN2+ remained high for greater than nine years [</offsets><xref ref-type="bibr" rid="CR36"><offsets xml_i="52018" xml_f="52020" txt_i="33640" txt_f="33642">36</offsets></xref><offsets xml_i="52027" xml_f="52101" txt_i="33642" txt_f="33716">], especially for those with HPV 16, 18, 31 and 33 identified at baseline.</offsets></p><p><offsets xml_i="52108" xml_f="52465" txt_i="33717" txt_f="34074">Another limitation relates to the retrospective cytology interpretations. Reflex LBC for HC2 positive women was performed as it would be in a real setting, whereas LBC slides for HC2 negative, COBAS and/or LA positive women were examined approximately two years later. This could have biased the LBC interpretations relative to those conducted in real-time.</offsets></p></sec><sec id="Sec15" sec-type="conclusion"><title><offsets xml_i="52520" xml_f="52531" txt_i="34076" txt_f="34087">Conclusions</offsets></title><p><offsets xml_i="52542" xml_f="53647" txt_i="34088" txt_f="35193">This study demonstrated that over a 2 year screening interval with hrHPV testing at baseline, followed by LBC screening at 24 months, 94 % of CIN2+ detected over the two screening rounds were hrHPV positive at baseline. Women were managed at round 1 only on their HC2 and corresponding LBC results, but 43/44 CIN3+ were detected among those HC2+/COBAS+. It is possible that additional CIN cases would have been detected among HC2−/COBAS+ subjects if COBAS results had also been used for subject management. Of the CIN2+ identified in Round 1, 55 % had abnormal cytology and an additional 17 % were HPV 16 or 18 positive at baseline, demonstrating the added benefit of immediate referral of women with these genotypes to potentially identify CIN2+ earlier than by follow-up testing of those with NILM cytology. Therefore, the results of this study support the use of reflex LBC together with HPV16/18 genotyping for colposcopy triage of hrHPV positive women.  However, as the cohort of HPV vaccinated women reaches cervical screening age, the benefits of HPV 16/18 genotyping for triage will be diminished.</offsets></p></sec></body><back><app-group><app id="App1"><sec id="Sec16"><title>Additional file</title><p><media position="anchor" xlink:href="12885_2015_1959_MOESM1_ESM.docx" id="MOESM1"><label>Additional file 1:</label><caption><p>
<bold>Table S1.</bold> Baseline Digene HC2/Roche COBAS Results Stratified by LA and Cytology. <bold>Table S2</bold>. Baseline Digene HC2 and Roche COBAS Abnormal Cytology Rates Stratified by Linear Array Genotyping. (DOCX 16 kb)</p></caption></media></p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>ASCUS</term><def><p>Atypical squamous cells, undetermined significance</p></def></def-item><def-item><term>BC</term><def><p>British Columbia</p></def></def-item><def-item><term>CI</term><def><p>Confidence interval</p></def></def-item><def-item><term>CIN</term><def><p>Cervical intraepithelial neoplasia</p></def></def-item><def-item><term>COBAS</term><def><p>Roche cobas® 4800 HPV test</p></def></def-item><def-item><term>Ct</term><def><p>Cycle threshold</p></def></def-item><def-item><term>HC2</term><def><p>Digene Hybrid Capture® 2 HPV test</p></def></def-item><def-item><term>HPV</term><def><p>Human papillomavirus</p></def></def-item><def-item><term>hr</term><def><p>High-risk</p></def></def-item><def-item><term>LA</term><def><p>Roche linear array HPV genotyping test</p></def></def-item><def-item><term>LBC</term><def><p>Liquid-based cytology</p></def></def-item><def-item><term>LEEP</term><def><p>Loop electrosurgical excision procedure</p></def></def-item><def-item><term>lr</term><def><p>Low-risk</p></def></def-item><def-item><term>NILM</term><def><p>Negative for intraepithelial lesions or malignancy</p></def></def-item><def-item><term>OHR</term><def><p>Other high-risk</p></def></def-item><def-item><term>OR</term><def><p>Odds ratio</p></def></def-item><def-item><term>Q1</term><def><p>First quartile</p></def></def-item><def-item><term>Q3</term><def><p>Third quartile</p></def></def-item><def-item><term>RLU/CO</term><def><p>Relative light units/cutoff</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Competing interests</bold></p><p>MK has received research grant funding via his institution from Roche Molecular Systems, Boehringer Ingelheim, Merck, Siemens Healthcare Diagnostics and Hologic Inc. ELF has served as an occasional advisor to companies involved with HPV diagnostics (Roche Molecular Systems, Becton Dickinson, Qiagen, Gen-Probe), HPV vaccines (Merck, GlaxoSmithKline), and cervical cancer cytology (Ikonisys, Cytyc). All other authors declare that they have no competing interests.</p></fn><fn><p><bold>Authors’ contributions</bold></p><p>DAC, LWS, DJVN, KC, ELF, AJC, GSO and MK conceived and designed the study together with the study sponsor, Roche Molecular Systems. WM carried out the COBAS and HC2 HPV testing. DJVN and KC carried out the cytology and histopathology interpretations. DAC acquired, analysed and interpreted the data, and drafted the manuscript. ELF, AJC and GSO provided advice on statistical analyses. DAC, WM, LWS, DJVN, KC, ELF, AJC, GSO and MK reviewed the draft manuscript, revised it critically for important intellectual content, and read and approved the final submitted manuscript.</p></fn></fn-group><ack><title>Acknowledgements</title><p>The HPV FOCAL Trial was supported by a grant from the Canadian Institutes of Health Research (Grant No. MCT82072). This HPV FOCAL Trial Roche adjunct study was funded by Roche Molecular Systems (RMS), Pleasanton, CA, USA. RMS and the FOCAL trial investigators together designed the adjunct study, but RMS had no input into the design of the original FOCAL Trial. RMS was provided an opportunity to critically review this manuscript but did not influence the decision whether to publish the data.</p><p>The authors wish to acknowledge the assistance of Keith Duncan, Steve Sung and Mona Cheng in assembling the study data and Lovedeep Gondara for statistical analysis. We acknowledge the excellent technical skills of the staff of the BCCDC Clinical Trials Unit and the BCCA Cervical Cancer Screening Laboratory who performed all the HPV and LBC screening. Thanks are due to the FOCAL Study Centre staff for their diligence in following up with women to minimize loss to follow-up.</p></ack><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Walboomers</surname><given-names>JM</given-names></name><name><surname>Jacobs</surname><given-names>MV</given-names></name><name><surname>Manos</surname><given-names>MM</given-names></name><name><surname>Bosch</surname><given-names>FX</given-names></name><name><surname>Kummer</surname><given-names>JA</given-names></name><name><surname>Shah</surname><given-names>KV</given-names></name><etal></etal></person-group><article-title>Human papillomavirus is a necessary cause of invasive cervical cancer worldwide</article-title><source>J Pathol</source><year>1999</year><volume>189</volume><fpage>12</fpage><lpage>9</lpage><pub-id pub-id-type="doi">10.1002/(SICI)1096-9896(199909)189:1&lt;12::AID-PATH431&gt;3.0.CO;2-F</pub-id><pub-id pub-id-type="pmid">10451482</pub-id></element-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Muñoz</surname><given-names>N</given-names></name><name><surname>Bosch</surname><given-names>FX</given-names></name><name><surname>De Sanjose</surname><given-names>S</given-names></name><name><surname>Herrero</surname><given-names>R</given-names></name><name><surname>Castellsague</surname><given-names>X</given-names></name><name><surname>Shah</surname><given-names>KV</given-names></name><etal></etal></person-group><article-title>Epidemiologic classification of human papillomavirus types associated with cervical cancer</article-title><source>N Engl J Med</source><year>2003</year><volume>348</volume><fpage>518</fpage><lpage>27</lpage><pub-id pub-id-type="doi">10.1056/NEJMoa021641</pub-id><pub-id pub-id-type="pmid">12571259</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Arbyn</surname><given-names>M</given-names></name><name><surname>Ronco</surname><given-names>G</given-names></name><name><surname>Anttila</surname><given-names>A</given-names></name><name><surname>Meijer</surname><given-names>CJLM</given-names></name><name><surname>Poljak</surname><given-names>M</given-names></name><name><surname>Ogilvie</surname><given-names>G</given-names></name><etal></etal></person-group><article-title>Evidence regarding human papillomavirus testing in secondary prevention of cervical cancer</article-title><source>Vaccine</source><year>2012</year><volume>30</volume><issue>Suppl 5</issue><fpage>F88</fpage><lpage>99</lpage><pub-id pub-id-type="doi">10.1016/j.vaccine.2012.06.095</pub-id><pub-id pub-id-type="pmid">23199969</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Gage</surname><given-names>JC</given-names></name><name><surname>Schiffman</surname><given-names>M</given-names></name><name><surname>Katki</surname><given-names>HA</given-names></name><name><surname>Castle</surname><given-names>PE</given-names></name><name><surname>Fetterman</surname><given-names>B</given-names></name><name><surname>Wentzensen</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>Reassurance against future risk of precancer and cancer conferred by a negative human papillomavirus test</article-title><source>J Natl Cancer Inst</source><year>2014</year><volume>106</volume><fpage>dju153</fpage><pub-id pub-id-type="doi">10.1093/jnci/dju153</pub-id><pub-id pub-id-type="pmid">25038467</pub-id></element-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ronco</surname><given-names>G</given-names></name><name><surname>Dillner</surname><given-names>J</given-names></name><name><surname>Elfström</surname><given-names>KM</given-names></name><name><surname>Tunesi</surname><given-names>S</given-names></name><name><surname>Snijders</surname><given-names>PJF</given-names></name><name><surname>Arbyn</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Efficacy of HPV-based screening for prevention of invasive cervical cancer: follow-up of four European randomised controlled trials</article-title><source>Lancet</source><year>2013</year><volume>383</volume><fpage>524</fpage><lpage>32</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(13)62218-7</pub-id><pub-id pub-id-type="pmid">24192252</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Vink</surname><given-names>MA</given-names></name><name><surname>Bogaards</surname><given-names>JA</given-names></name><name><surname>Meijer</surname><given-names>CJLM</given-names></name><name><surname>Berkhof</surname><given-names>J</given-names></name></person-group><article-title>Primary human papillomavirus DNA screening for cervical cancer prevention: can the screening interval be safely extended?</article-title><source>Int J Cancer</source><year>2015</year><volume>137</volume><fpage>420</fpage><lpage>27</lpage><pub-id pub-id-type="doi">10.1002/ijc.29381</pub-id><pub-id pub-id-type="pmid">25484263</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cuzick</surname><given-names>J</given-names></name><name><surname>Cadman</surname><given-names>L</given-names></name><name><surname>Mesher</surname><given-names>D</given-names></name><name><surname>Austin</surname><given-names>J</given-names></name><name><surname>Ashdown-Barr</surname><given-names>L</given-names></name><name><surname>Ho</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Comparing the performance of six human papillomavirus tests in a screening population</article-title><source>Br J Cancer</source><year>2013</year><volume>108</volume><fpage>908</fpage><lpage>13</lpage><pub-id pub-id-type="doi">10.1038/bjc.2013.22</pub-id><pub-id pub-id-type="pmid">23370211</pub-id></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Muñoz</surname><given-names>N</given-names></name><name><surname>Bosch</surname><given-names>FX</given-names></name><name><surname>Castellsagué</surname><given-names>X</given-names></name><name><surname>Díaz</surname><given-names>M</given-names></name><name><surname>de Sanjose</surname><given-names>S</given-names></name><name><surname>Hammouda</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Against which human papillomavirus types shall we vaccinate and screen? The international perspective</article-title><source>Int J Cancer</source><year>2004</year><volume>111</volume><fpage>278</fpage><lpage>85</lpage><pub-id pub-id-type="doi">10.1002/ijc.20244</pub-id><pub-id pub-id-type="pmid">15197783</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Castle</surname><given-names>PE</given-names></name><name><surname>Stoler</surname><given-names>MH</given-names></name><name><surname>Wright</surname><given-names>TC</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Wright</surname><given-names>TL</given-names></name><name><surname>Behrens</surname><given-names>CM</given-names></name></person-group><article-title>Performance of carcinogenic human papillomavirus (HPV) testing and HPV16 or HPV18 genotyping for cervical cancer screening of women aged 25 years and older: a subanalysis of the ATHENA study</article-title><source>Lancet Oncol</source><year>2011</year><volume>12</volume><fpage>880</fpage><lpage>90</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(11)70188-7</pub-id><pub-id pub-id-type="pmid">21865084</pub-id></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cox</surname><given-names>JT</given-names></name><name><surname>Castle</surname><given-names>PE</given-names></name><name><surname>Behrens</surname><given-names>CM</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Wright</surname><given-names>TC</given-names></name><name><surname>Cuzick</surname><given-names>J</given-names></name></person-group><article-title>Comparison of cervical cancer screening strategies incorporating different combinations of cytology, HPV testing, and genotyping for HPV 16/18: results from the ATHENA HPV study</article-title><source>Am J Obstet Gynecol</source><year>2013</year><volume>208</volume><fpage>184.e1</fpage><lpage>84.e11</lpage><pub-id pub-id-type="doi">10.1016/j.ajog.2012.11.020</pub-id><pub-id pub-id-type="pmid">23174289</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rositch</surname><given-names>AF</given-names></name><name><surname>Koshiol</surname><given-names>J</given-names></name><name><surname>Hudgens</surname><given-names>MG</given-names></name><name><surname>Razzaghi</surname><given-names>H</given-names></name><name><surname>Backes</surname><given-names>DM</given-names></name><name><surname>Pimenta</surname><given-names>JM</given-names></name><etal></etal></person-group><article-title>Patterns of persistent genital human papillomavirus infection among women worldwide: a literature review and meta-analysis</article-title><source>Int J Cancer</source><year>2013</year><volume>133</volume><fpage>1271</fpage><lpage>85</lpage><pub-id pub-id-type="doi">10.1002/ijc.27828</pub-id><pub-id pub-id-type="pmid">22961444</pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Saslow</surname><given-names>D</given-names></name><name><surname>Solomon</surname><given-names>D</given-names></name><name><surname>Lawson</surname><given-names>HW</given-names></name><name><surname>Killackey</surname><given-names>M</given-names></name><name><surname>Kulasingam</surname><given-names>SL</given-names></name><name><surname>Cain</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>American Cancer Society, American Society for Colposcopy and Cervical Pathology and American Society for Clinical Pathology screening guidelines for the prevention and early detection of cervical cancer</article-title><source>Am J Clin Pathol</source><year>2012</year><volume>137</volume><fpage>516</fpage><lpage>42</lpage><pub-id pub-id-type="doi">10.1309/AJCPTGD94EVRSJCG</pub-id><pub-id pub-id-type="pmid">22431528</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kyrgiou</surname><given-names>M</given-names></name><name><surname>Koliopoulos</surname><given-names>G</given-names></name><name><surname>Martin-Hirsch</surname><given-names>P</given-names></name><name><surname>Arbyn</surname><given-names>M</given-names></name><name><surname>Prendiville</surname><given-names>W</given-names></name><name><surname>Paraskevaidis</surname><given-names>E</given-names></name></person-group><article-title>Obstetric outcomes after conservative treatment for intraepithelial or early invasive cervical lesions: systematic review and meta-analysis</article-title><source>Lancet</source><year>2006</year><volume>367</volume><fpage>489</fpage><lpage>98</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(06)68181-6</pub-id><pub-id pub-id-type="pmid">16473126</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Castanon</surname><given-names>A</given-names></name><name><surname>Landy</surname><given-names>R</given-names></name><name><surname>Brocklehurst</surname><given-names>P</given-names></name><name><surname>Evans</surname><given-names>H</given-names></name><name><surname>Peebles</surname><given-names>D</given-names></name><name><surname>Singh</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>Risk of preterm delivery with increasing depth of excision for cervical intraepithelial neoplasia in England: nested case–control study</article-title><source>BMJ</source><year>2014</year><volume>349</volume><fpage>g6223</fpage><pub-id pub-id-type="doi">10.1136/bmj.g6223</pub-id><pub-id pub-id-type="pmid">25378384</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ikenberg</surname><given-names>H</given-names></name><name><surname>Bergeron</surname><given-names>C</given-names></name><name><surname>Schmidt</surname><given-names>D</given-names></name><name><surname>Griesser</surname><given-names>H</given-names></name><name><surname>Alameda</surname><given-names>F</given-names></name><name><surname>Angeloni</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>Screening for cervical cancer precursors with p16/Ki-67 dual-stained cytology: results of the PALMS study</article-title><source>J Natl Cancer Inst</source><year>2013</year><volume>105</volume><fpage>1550</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1093/jnci/djt235</pub-id><pub-id pub-id-type="pmid">24096620</pub-id></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Snellenberg</surname><given-names>S</given-names></name><name><surname>Strooper</surname><given-names>LM</given-names></name><name><surname>Hesselink</surname><given-names>A</given-names></name><name><surname>Meijer</surname><given-names>C</given-names></name><name><surname>Snijders</surname><given-names>P</given-names></name><name><surname>Heideman</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Development of a multiplex methylation-specific PCR as candidate triage test for women with an HPV-positive cervical scrape</article-title><source>BMC Cancer</source><year>2012</year><volume>12</volume><fpage>551</fpage><pub-id pub-id-type="doi">10.1186/1471-2407-12-551</pub-id><pub-id pub-id-type="pmid">23176198</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Brentnall</surname><given-names>AR</given-names></name><name><surname>Vasiljević</surname><given-names>N</given-names></name><name><surname>Scibior-Bentkowska</surname><given-names>D</given-names></name><name><surname>Cadman</surname><given-names>L</given-names></name><name><surname>Austin</surname><given-names>J</given-names></name><name><surname>Szarewski</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>A DNA methylation classifier of cervical precancer based on human papillomavirus and human genes</article-title><source>Int J Cancer</source><year>2014</year><volume>135</volume><fpage>1425</fpage><lpage>32</lpage><pub-id pub-id-type="doi">10.1002/ijc.28790</pub-id><pub-id pub-id-type="pmid">24535756</pub-id></element-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ogilvie</surname><given-names>G</given-names></name><name><surname>van Niekerk</surname><given-names>D</given-names></name><name><surname>Krajden</surname><given-names>M</given-names></name><name><surname>Martin</surname><given-names>R</given-names></name><name><surname>Ehlen</surname><given-names>T</given-names></name><name><surname>Ceballos</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>A randomized controlled trial of Human Papillomavirus (HPV) testing for cervical cancer screening: trial design and preliminary results (HPV FOCAL Trial)</article-title><source>BMC Cancer</source><year>2010</year><volume>10</volume><fpage>111</fpage><pub-id pub-id-type="doi">10.1186/1471-2407-10-111</pub-id><pub-id pub-id-type="pmid">20334685</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><mixed-citation publication-type="other">Cunningham M. More than Just the Kappa Coefficient: A Program to Fully Characterize Inter-Rater Reliability between Two Raters. SAS Global Forum 2009. Available from <ext-link ext-link-type="uri" xlink:href="http://support.sas.com/resources/papers/proceedings09/242-2009.pdf">http://support.sas.com/resources/papers/proceedings09/242-2009.pdf</ext-link>. Accessed April 27, 2015.</mixed-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kitchener</surname><given-names>HC</given-names></name><name><surname>Almonte</surname><given-names>M</given-names></name><name><surname>Thomson</surname><given-names>C</given-names></name><name><surname>Wheeler</surname><given-names>P</given-names></name><name><surname>Sargent</surname><given-names>A</given-names></name><name><surname>Stoykova</surname><given-names>B</given-names></name><etal></etal></person-group><article-title>HPV testing in combination with liquid-based cytology in primary cervical screening (ARTISTIC): a randomised controlled trial</article-title><source>Lancet Oncol</source><year>2009</year><volume>10</volume><fpage>672</fpage><lpage>82</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(09)70156-1</pub-id><pub-id pub-id-type="pmid">19540162</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rijkaart</surname><given-names>DC</given-names></name><name><surname>Berkhof</surname><given-names>J</given-names></name><name><surname>Rozendaal</surname><given-names>L</given-names></name><name><surname>van Kemenade</surname><given-names>FJ</given-names></name><name><surname>Bulkmans</surname><given-names>NWJ</given-names></name><name><surname>Heideman</surname><given-names>DAM</given-names></name><etal></etal></person-group><article-title>Human papillomavirus testing for the detection of high-grade cervical intraepithelial neoplasia and cancer: final results of the POBASCAM randomised controlled trial</article-title><source>Lancet Oncol</source><year>2012</year><volume>13</volume><fpage>78</fpage><lpage>88</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(11)70296-0</pub-id><pub-id pub-id-type="pmid">22177579</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Castle</surname><given-names>PE</given-names></name><name><surname>Cox</surname><given-names>JT</given-names></name><name><surname>Jeronimo</surname><given-names>J</given-names></name><name><surname>Solomon</surname><given-names>D</given-names></name><name><surname>Wheeler</surname><given-names>CM</given-names></name><name><surname>Gravitt</surname><given-names>PE</given-names></name><etal></etal></person-group><article-title>An analysis of high-risk human papillomavirus DNA-negative cervical precancers in the ASCUS-LSIL triage study (ALTS)</article-title><source>Obstet Gynecol</source><year>2008</year><volume>111</volume><fpage>847</fpage><lpage>56</lpage><pub-id pub-id-type="doi">10.1097/AOG.0b013e318168460b</pub-id><pub-id pub-id-type="pmid">18378743</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bouchard-Fortier</surname><given-names>G</given-names></name><name><surname>Hajifathalian</surname><given-names>K</given-names></name><name><surname>McKnight</surname><given-names>MD</given-names></name><name><surname>Zacharias</surname><given-names>DG</given-names></name><name><surname>Gonzalez-Gonzalez</surname><given-names>LA</given-names></name></person-group><article-title>Co-testing for detection of high-grade cervical intraepithelial neoplasia and cancer compared with cytology alone: a meta-analysis of randomized controlled trials</article-title><source>J Public Health</source><year>2014</year><volume>36</volume><fpage>46</fpage><lpage>55</lpage><pub-id pub-id-type="doi">10.1093/pubmed/fdt057</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Blatt</surname><given-names>AJ</given-names></name><name><surname>Kennedy</surname><given-names>R</given-names></name><name><surname>Luff</surname><given-names>RD</given-names></name><name><surname>Austin</surname><given-names>RM</given-names></name><name><surname>Rabin</surname><given-names>DS</given-names></name></person-group><article-title>Comparison of cervical cancer screening results among 256,648 women in multiple clinical practices</article-title><source>Cancer Cytopathol</source><year>2015</year><volume>123</volume><fpage>282</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1002/cncy.21544</pub-id><pub-id pub-id-type="pmid">25864682</pub-id></element-citation></ref><ref id="CR25"><label>25.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Elfström</surname><given-names>KM</given-names></name><name><surname>Smelov</surname><given-names>V</given-names></name><name><surname>Johansson</surname><given-names>ALV</given-names></name><name><surname>Eklund</surname><given-names>C</given-names></name><name><surname>Nauclér</surname><given-names>P</given-names></name><name><surname>Arnheim-Dahlström</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Long term duration of protective effect for HPV negative women: follow-up of primary HPV screening randomised controlled trial</article-title><source>BMJ</source><year>2014</year><volume>348</volume><fpage>g130</fpage><pub-id pub-id-type="doi">10.1136/bmj.g130</pub-id><pub-id pub-id-type="pmid">24435414</pub-id></element-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wright</surname><given-names>TC</given-names></name><name><surname>Stoler</surname><given-names>MH</given-names></name><name><surname>Behrens</surname><given-names>CM</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name><name><surname>Zhang</surname><given-names>G</given-names></name><name><surname>Wright</surname><given-names>TL</given-names></name></person-group><article-title>Primary cervical cancer screening with human papillomavirus: end of study results from the ATHENA study using HPV as the first-line screening test</article-title><source>Gynecol Oncol</source><year>2015</year><volume>136</volume><fpage>189</fpage><lpage>97</lpage><pub-id pub-id-type="doi">10.1016/j.ygyno.2014.11.076</pub-id><pub-id pub-id-type="pmid">25579108</pub-id></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ogilvie</surname><given-names>GS</given-names></name><name><surname>Krajden</surname><given-names>M</given-names></name><name><surname>van Niekerk</surname><given-names>DJ</given-names></name><name><surname>Martin</surname><given-names>RE</given-names></name><name><surname>Ehlen</surname><given-names>TG</given-names></name><name><surname>Ceballos</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>Primary cervical cancer screening with HPV testing compared with liquid-based cytology: results of round 1 of a randomised controlled trial - the HPV FOCAL Study</article-title><source>Br J Cancer</source><year>2012</year><volume>107</volume><fpage>1917</fpage><lpage>24</lpage><pub-id pub-id-type="doi">10.1038/bjc.2012.489</pub-id><pub-id pub-id-type="pmid">23169286</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><mixed-citation publication-type="other">British Columbia Cancer Agency. Cervical Cancer Screening Program 2012 Annual Report. Available from <ext-link ext-link-type="uri" xlink:href="http://www.screeningbc.ca/NR/rdonlyres/F2D89BB9-3B33-4DBD-BE76-975A220B0C7B/69596/CCSPAnnualReport2012PrintVersion.pdf">http://www.screeningbc.ca/NR/rdonlyres/F2D89BB9-3B33-4DBD-BE76-975A220B0C7B/69596/CCSPAnnualReport2012PrintVersion.pdf</ext-link>. Accessed April 27, 2015.</mixed-citation></ref><ref id="CR29"><label>29.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wright</surname><given-names>TC</given-names></name><name><surname>Stoler</surname><given-names>MH</given-names></name><name><surname>Behrens</surname><given-names>CM</given-names></name><name><surname>Apple</surname><given-names>R</given-names></name><name><surname>Derion</surname><given-names>T</given-names></name><name><surname>Wright</surname><given-names>TL</given-names></name></person-group><article-title>The ATHENA human papillomavirus study: design, methods, and baseline results</article-title><source>Am J Obstet Gynecol</source><year>2011</year><volume>46</volume><fpage>e1</fpage><lpage>46</lpage></element-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ogilvie</surname><given-names>GS</given-names></name><name><surname>Cook</surname><given-names>DA</given-names></name><name><surname>Taylor</surname><given-names>DL</given-names></name><name><surname>Rank</surname><given-names>C</given-names></name><name><surname>Kan</surname><given-names>L</given-names></name><name><surname>Yu</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Population-based evaluation of type-specific HPV prevalence among women in British Columbia, Canada</article-title><source>Vaccine</source><year>2013</year><volume>31</volume><fpage>1129</fpage><lpage>33</lpage><pub-id pub-id-type="doi">10.1016/j.vaccine.2012.09.085</pub-id><pub-id pub-id-type="pmid">23273510</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cubie</surname><given-names>HA</given-names></name><name><surname>Cuschieri</surname><given-names>K</given-names></name></person-group><article-title>Understanding HPV tests and their appropriate applications</article-title><source>Cytopathology</source><year>2013</year><volume>24</volume><fpage>289</fpage><lpage>308</lpage><pub-id pub-id-type="pmid">23998275</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bulkmans</surname><given-names>NWJ</given-names></name><name><surname>Berkhof</surname><given-names>J</given-names></name><name><surname>Bulk</surname><given-names>S</given-names></name><name><surname>Bleeker</surname><given-names>MCG</given-names></name><name><surname>van Kemenade</surname><given-names>FJ</given-names></name><name><surname>Rozendaal</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>High-risk HPV type-specific clearance rates in cervical screening</article-title><source>Br J Cancer</source><year>2007</year><volume>96</volume><fpage>1419</fpage><lpage>24</lpage><pub-id pub-id-type="pmid">17342094</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Naucler</surname><given-names>P</given-names></name><name><surname>Ryd</surname><given-names>W</given-names></name><name><surname>Törnberg</surname><given-names>S</given-names></name><name><surname>Strand</surname><given-names>A</given-names></name><name><surname>Wadell</surname><given-names>G</given-names></name><name><surname>Elfgren</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>Efficacy of HPV DNA testing with cytology triage and/or repeat HPV DNA testing in primary cervical cancer screening</article-title><source>J Natl Cancer Inst</source><year>2009</year><volume>101</volume><fpage>88</fpage><lpage>99</lpage><pub-id pub-id-type="doi">10.1093/jnci/djn444</pub-id><pub-id pub-id-type="pmid">19141778</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rodríguez</surname><given-names>AC</given-names></name><name><surname>Schiffman</surname><given-names>M</given-names></name><name><surname>Herrero</surname><given-names>R</given-names></name><name><surname>Wacholder</surname><given-names>S</given-names></name><name><surname>Hildesheim</surname><given-names>A</given-names></name><name><surname>Castle</surname><given-names>PE</given-names></name><etal></etal></person-group><article-title>Rapid clearance of human papillomavirus and implications for clinical focus on persistent infections</article-title><source>J Natl Cancer Inst</source><year>2008</year><volume>100</volume><fpage>513</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1093/jnci/djn044</pub-id><pub-id pub-id-type="pmid">18364507</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Castle</surname><given-names>PE</given-names></name><name><surname>Hunt</surname><given-names>WC</given-names></name><name><surname>Langsfeld</surname><given-names>E</given-names></name><name><surname>Wheeler</surname><given-names>CM</given-names></name><name><surname>Committee</surname><given-names>NMHPR</given-names></name></person-group><article-title>Three-year risk of cervical precancer and cancer after the detection of low-risk human papillomavirus genotypes targeted by a commercial test</article-title><source>Obstet Gynecol</source><year>2014</year><volume>123</volume><fpage>49</fpage><lpage>56</lpage><pub-id pub-id-type="doi">10.1097/AOG.0000000000000013</pub-id><pub-id pub-id-type="pmid">24463663</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Smelov</surname><given-names>V</given-names></name><name><surname>Elfström</surname><given-names>KM</given-names></name><name><surname>Johansson</surname><given-names>ALV</given-names></name><name><surname>Eklund</surname><given-names>C</given-names></name><name><surname>Naucler</surname><given-names>P</given-names></name><name><surname>Arnheim-Dahlström</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Long-term HPV type-specific risks of high-grade cervical intraepithelial lesions: a 14-year follow-up of a randomized primary HPV screening trial</article-title><source>Int J Cancer</source><year>2015</year><volume>136</volume><fpage>1171</fpage><lpage>80</lpage><pub-id pub-id-type="doi">10.1002/ijc.29085</pub-id><pub-id pub-id-type="pmid">25043476</pub-id></element-citation></ref></ref-list></back></article>