<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Hepatology</journal-id><journal-id journal-id-type="iso-abbrev">Hepatology</journal-id><journal-id journal-id-type="doi">10.1002/(ISSN)1527-3350</journal-id><journal-id journal-id-type="publisher-id">HEP</journal-id><journal-title-group><journal-title>Hepatology (Baltimore, Md.)</journal-title></journal-title-group><issn pub-type="ppub">0270-9139</issn><issn pub-type="epub">1527-3350</issn><publisher><publisher-name>John Wiley and Sons Inc.</publisher-name><publisher-loc>Hoboken</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">28128852</article-id><article-id pub-id-type="pmc">5573922</article-id><article-id pub-id-type="doi">10.1002/hep.29081</article-id><article-id pub-id-type="publisher-id">HEP29081</article-id><article-categories><subj-group subj-group-type="overline"><subject>Original Article</subject></subj-group><subj-group subj-group-type="heading"><subject>Original Articles</subject><subj-group subj-group-type="heading"><subject>Viral Hepatitis</subject></subj-group></subj-group></article-categories><title-group><article-title>Glecaprevir and pibrentasvir for 12 weeks for hepatitis C virus genotype 1 infection and prior direct‐acting antiviral treatment</article-title><alt-title alt-title-type="right-running-head">Poordad et al.</alt-title></title-group><contrib-group><contrib id="hep29081-cr-0001" contrib-type="author" corresp="yes"><name><surname>Poordad</surname><given-names>Fred</given-names></name><address><email>poordad@txliver.com</email></address><xref ref-type="aff" rid="hep29081-aff-0001">
<sup>1</sup>
</xref></contrib><contrib id="hep29081-cr-0002" contrib-type="author"><name><surname>Felizarta</surname><given-names>Franco</given-names></name><xref ref-type="aff" rid="hep29081-aff-0002">
<sup>2</sup>
</xref></contrib><contrib id="hep29081-cr-0003" contrib-type="author"><name><surname>Asatryan</surname><given-names>Armen</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib><contrib id="hep29081-cr-0004" contrib-type="author"><name><surname>Sulkowski</surname><given-names>Mark S.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0004">
<sup>4</sup>
</xref></contrib><contrib id="hep29081-cr-0005" contrib-type="author"><name><surname>Reindollar</surname><given-names>Robert W.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0005">
<sup>5</sup>
</xref></contrib><contrib id="hep29081-cr-0006" contrib-type="author"><name><surname>Landis</surname><given-names>Charles S.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0006">
<sup>6</sup>
</xref></contrib><contrib id="hep29081-cr-0007" contrib-type="author"><name><surname>Gordon</surname><given-names>Stuart C.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0007">
<sup>7</sup>
</xref></contrib><contrib id="hep29081-cr-0008" contrib-type="author"><name><surname>Flamm</surname><given-names>Steven L.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0008">
<sup>8</sup>
</xref></contrib><contrib id="hep29081-cr-0009" contrib-type="author"><name><surname>Fried</surname><given-names>Michael W.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0009">
<sup>9</sup>
</xref></contrib><contrib id="hep29081-cr-0010" contrib-type="author"><name><surname>Bernstein</surname><given-names>David E.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0010">
<sup>10</sup>
</xref></contrib><contrib id="hep29081-cr-0011" contrib-type="author"><name><surname>Lin</surname><given-names>Chih‐Wei</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib><contrib id="hep29081-cr-0012" contrib-type="author"><name><surname>Liu</surname><given-names>Ran</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib><contrib id="hep29081-cr-0013" contrib-type="author"><name><surname>Lovell</surname><given-names>Sandra S.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib><contrib id="hep29081-cr-0014" contrib-type="author"><name><surname>Ng</surname><given-names>Teresa I.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib><contrib id="hep29081-cr-0015" contrib-type="author"><name><surname>Kort</surname><given-names>Jens</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib><contrib id="hep29081-cr-0016" contrib-type="author"><name><surname>Mensa</surname><given-names>Federico J.</given-names></name><xref ref-type="aff" rid="hep29081-aff-0003">
<sup>3</sup>
</xref></contrib></contrib-group><aff id="hep29081-aff-0001">
<label><sup>1</sup></label>
<named-content content-type="organisation-division">Texas Liver Institute</named-content>
<institution>University of Texas Health Science Center</institution>
<named-content content-type="city">San Antonio</named-content>
<named-content content-type="country-part">TX</named-content>
</aff><aff id="hep29081-aff-0002">
<label><sup>2</sup></label>
<institution>Private Practice</institution>
<named-content content-type="city">Bakersfield</named-content>
<named-content content-type="country-part">CA</named-content>
</aff><aff id="hep29081-aff-0003">
<label><sup>3</sup></label>
<institution>AbbVie Inc</institution>
<named-content content-type="city">North Chicago</named-content>
<named-content content-type="country-part">IL</named-content>
</aff><aff id="hep29081-aff-0004">
<label><sup>4</sup></label>
<institution>Johns Hopkins University School of Medicine</institution>
<named-content content-type="city">Baltimore</named-content>
<named-content content-type="country-part">MD</named-content>
</aff><aff id="hep29081-aff-0005">
<label><sup>5</sup></label>
<institution>Piedmont Healthcare/Carolinas Center for Liver Disease</institution>
<named-content content-type="city">Statesville</named-content>
<named-content content-type="country-part">NC</named-content>
</aff><aff id="hep29081-aff-0006">
<label><sup>6</sup></label>
<named-content content-type="organisation-division">Division of Gastroenterology</named-content>
<institution>University of Washington</institution>
<named-content content-type="city">Seattle</named-content>
<named-content content-type="country-part">WA</named-content>
</aff><aff id="hep29081-aff-0007">
<label><sup>7</sup></label>
<institution>Henry Ford Health System</institution>
<named-content content-type="city">Detroit</named-content>
<named-content content-type="country-part">MI</named-content>
</aff><aff id="hep29081-aff-0008">
<label><sup>8</sup></label>
<institution>Northwestern Feinberg School of Medicine</institution>
<named-content content-type="city">Chicago</named-content>
<named-content content-type="country-part">IL</named-content>
</aff><aff id="hep29081-aff-0009">
<label><sup>9</sup></label>
<institution>University of North Carolina</institution>
<named-content content-type="city">Chapel Hill</named-content>
<named-content content-type="country-part">NC</named-content>
</aff><aff id="hep29081-aff-0010">
<label><sup>10</sup></label>
<institution>North Shore University Hospital</institution>
<named-content content-type="city">Manhasset</named-content>
<named-content content-type="country-part">NY</named-content>
</aff><author-notes><corresp id="correspondenceTo"><label>*</label><bold>ADDRESS CORRESPONDENCE AND REPRINT REQUESTS TO</bold>:<break></break>Fred Poordad, M.D.<break></break>Texas Liver Institute<break></break>University of Texas Health Science Center<break></break>607 Camden St., Suite 101<break></break>San Antonio, TX 78215<break></break>E‐mail: <email>poordad@txliver.com</email><break></break>Tel: +1‐210‐253‐3426.<break></break></corresp></author-notes><pub-date pub-type="epub"><day>10</day><month>4</month><year>2017</year></pub-date><pub-date pub-type="ppub"><month>8</month><year>2017</year></pub-date><volume>66</volume><issue>2</issue><issue-id pub-id-type="doi">10.1002/hep.v66.2</issue-id><fpage>389</fpage><lpage>397</lpage><history><date date-type="received"><day>25</day><month>10</month><year>2016</year></date><date date-type="rev-recd"><day>08</day><month>12</month><year>2016</year></date><date date-type="accepted"><day>23</day><month>1</month><year>2017</year></date></history><permissions><copyright-statement content-type="article-copyright">© 2017 The Authors. H<sc>epatology</sc> published by Wiley Periodicals, Inc., on behalf of the American Association for the Study of Liver Diseases.</copyright-statement><license license-type="creativeCommonsBy-nc-nd"><license-p>This is an open access article under the terms of the <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution‐NonCommercial‐NoDerivs</ext-link> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</license-p></license></permissions><self-uri content-type="pdf" xlink:type="simple" xlink:href="file:HEP-66-389.pdf"></self-uri><abstract><p><offsets xml_i="9300" xml_f="10885" txt_i="11" txt_f="1596">Although direct‐acting antiviral (DAA) therapies for chronic hepatitis C virus (HCV) infection have demonstrated high rates of sustained virologic response, virologic failure may still occur, potentially leading to the emergence of viral resistance, which can decrease the effectiveness of subsequent treatment. Treatment options for patients who failed previous DAA‐containing regimens, particularly those with nonstructural protein 5A inhibitors, are limited and remain an area of unmet medical need. This phase 2, open‐label study (MAGELLAN‐1) evaluated the efficacy and safety of glecaprevir (GLE) + pibrentasvir (PIB) ± ribavirin (RBV) in HCV genotype 1–infected patients with prior virologic failure to HCV DAA‐containing therapy. A total of 50 patients without cirrhosis were randomized to three arms: 200 mg GLE + 80 mg PIB (arm A), 300 mg GLE + 120 mg PIB with 800 mg once‐daily RBV (arm B), or 300 mg GLE + 120 mg PIB without RBV (arm C). By intent‐to‐treat analysis, sustained virologic response at posttreatment week 12 was achieved in 100% (6/6, 95% confidence interval 61‐100), 95% (21/22, 95% confidence interval 78‐99), and 86% (19/22, 95% confidence interval 67‐95) of patients in arms A, B, and C, respectively. Virologic failure occurred in no patients in arm A and in 1 patient each in arms B and C (two patients were lost to follow‐up in arm C). The majority of adverse events were mild in severity; no serious adverse events related to study drug and no relevant laboratory abnormalities in alanine aminotransferase, total bilirubin, or hemoglobin were observed. </offsets><italic><offsets xml_i="10893" xml_f="10903" txt_i="1596" txt_f="1606">Conclusion</offsets></italic><offsets xml_i="10912" xml_f="11123" txt_i="1606" txt_f="1817">: The combination of GLE and PIB was highly efficacious and well tolerated in patients with HCV genotype 1 infection and prior failure of DAA‐containing therapy; RBV coadministration did not improve efficacy. (H</offsets><sc><offsets xml_i="11127" xml_f="11136" txt_i="1817" txt_f="1826">epatology</offsets></sc><offsets xml_i="11141" xml_f="11159" txt_i="1826" txt_f="1844"> 2017;66:389–397).</offsets></p></abstract><funding-group><award-group><funding-source>AbbVie
</funding-source></award-group></funding-group><counts><fig-count count="2"></fig-count><table-count count="3"></table-count><page-count count="9"></page-count><word-count count="4968"></word-count></counts><custom-meta-group><custom-meta><meta-name>source-schema-version-number</meta-name><meta-value>2.0</meta-value></custom-meta><custom-meta><meta-name>component-id</meta-name><meta-value>hep29081</meta-value></custom-meta><custom-meta><meta-name>cover-date</meta-name><meta-value>August 2017</meta-value></custom-meta><custom-meta><meta-name>details-of-publishers-convertor</meta-name><meta-value>Converter:WILEY_ML3GV2_TO_NLMPMC version:5.1.8 mode:remove_FC converted:29.08.2017</meta-value></custom-meta></custom-meta-group></article-meta><notes><fn-group><fn id="hep29081-note-0001"><p>Potential conflict of interest: Dr. Reindollar is on the speakers' bureau for AbbVie. Dr. Asatryan is employed by and owns stock in AbbVie. Dr. Kort is employed by and owns stock in AbbVie. Dr. Landis received grants from Gilead and AbbVie. Dr. Sulkowski consults, advises, and received grants from AbbVie, Gilead, and Merck. He consults and advises Janssen, Trek, and Cocrystal. Dr. Mensa is employed by and owns stock in AbbVie. Dr. Lovell is employed by and owns stock in AbbVie. Dr. Felizarta is on the speakers' bureau and received grants from AbbVie, Gilead, Janssen, and Merck. Dr. Lin is employed by and owns stock in AbbVie. Dr. Liu is employed by and owns stock in AbbVie. Dr. Ng is employed by and owns stock in AbbVie. Dr. Fried consults and received grants from AbbVie, Bristol‐Myers Squibb, Gilead, and Merck. He owns stock in Target. Dr. Bernstein consults and received grants from AbbVie. Dr. Flamm consults, advises, is on the speakers' bureau, and received grants from AbbVie and Gilead. He consults, advises, and is on the speakers' bureau for Merck. Dr. Poordad consults, advises, is on the speakers' bureau, and received grants from Gilead, Merck, GlaxoSmithKline, and Vertex. He consults, advises, and received grants from AbbVie, Achillion, Anadys, Biolex, Boehringer‐Ingelheim, Bristol‐Myers Squibb, GlobeImmune, Idenix, and Novartis. He is on the speakers' bureau and received grants from Genentech. He consults and advises Tibotec/Janssen and Theravance. He is on the speakers' bureau for Kadmon, Onyx/Bayer, and Salix. He received grants from Idera, Intercept, Janssen, Medarex, Medtronic, Santaris, Scynexis, and ZymoGenetics. Dr. Gordon advises and received grants from AbbVie, Bristol‐Myers Squibb, Intercept, Gilead, and Merck. He advises CVS Caremark. He received grants from Conatus, CymaBay, and Exalenz.</p></fn><fn id="hep29081-note-0101"><p>Supported by AbbVie.</p></fn></fn-group></notes></front><body><def-list list-content="abbreviations" id="hep29081-lp-0001"><title><offsets xml_i="14027" xml_f="14040" txt_i="1852" txt_f="1865">Abbreviations</offsets></title><def-item><term id="hep29081-li-0001"><offsets xml_i="14086" xml_f="14088" txt_i="1866" txt_f="1868">CI</offsets></term><def id="hep29081-li-0002"><p><offsets xml_i="14125" xml_f="14144" txt_i="1868" txt_f="1887">confidence interval</offsets></p></def></def-item><def-item><term id="hep29081-li-0003"><offsets xml_i="14203" xml_f="14206" txt_i="1888" txt_f="1891">DAA</offsets></term><def id="hep29081-li-0004"><p><offsets xml_i="14243" xml_f="14266" txt_i="1891" txt_f="1914">direct‐acting antiviral</offsets></p></def></def-item><def-item><term id="hep29081-li-0005"><offsets xml_i="14325" xml_f="14329" txt_i="1915" txt_f="1919">EC50</offsets></term><def id="hep29081-li-0006"><p><offsets xml_i="14366" xml_f="14402" txt_i="1919" txt_f="1955">half‐maximal effective concentration</offsets></p></def></def-item><def-item><term id="hep29081-li-0007"><offsets xml_i="14461" xml_f="14464" txt_i="1956" txt_f="1959">GLE</offsets></term><def id="hep29081-li-0008"><p><offsets xml_i="14501" xml_f="14512" txt_i="1959" txt_f="1970">glecaprevir</offsets></p></def></def-item><def-item><term id="hep29081-li-0009"><offsets xml_i="14571" xml_f="14573" txt_i="1971" txt_f="1973">GT</offsets></term><def id="hep29081-li-0010"><p><offsets xml_i="14610" xml_f="14618" txt_i="1973" txt_f="1981">genotype</offsets></p></def></def-item><def-item><term id="hep29081-li-0011"><offsets xml_i="14677" xml_f="14680" txt_i="1982" txt_f="1985">HCV</offsets></term><def id="hep29081-li-0012"><p><offsets xml_i="14717" xml_f="14734" txt_i="1985" txt_f="2002">hepatitis C virus</offsets></p></def></def-item><def-item><term id="hep29081-li-0013"><offsets xml_i="14793" xml_f="14796" txt_i="2003" txt_f="2006">ITT</offsets></term><def id="hep29081-li-0014"><p><offsets xml_i="14833" xml_f="14848" txt_i="2006" txt_f="2021">intent‐to‐treat</offsets></p></def></def-item><def-item><term id="hep29081-li-0015"><offsets xml_i="14907" xml_f="14911" txt_i="2022" txt_f="2026">mITT</offsets></term><def id="hep29081-li-0016"><p><offsets xml_i="14948" xml_f="14960" txt_i="2026" txt_f="2038">modified ITT</offsets></p></def></def-item><def-item><term id="hep29081-li-0017"><offsets xml_i="15019" xml_f="15021" txt_i="2039" txt_f="2041">NS</offsets></term><def id="hep29081-li-0018"><p><offsets xml_i="15058" xml_f="15071" txt_i="2041" txt_f="2054">nonstructural</offsets></p></def></def-item><def-item><term id="hep29081-li-0019"><offsets xml_i="15130" xml_f="15132" txt_i="2055" txt_f="2057">PI</offsets></term><def id="hep29081-li-0020"><p><offsets xml_i="15169" xml_f="15187" txt_i="2057" txt_f="2075">protease inhibitor</offsets></p></def></def-item><def-item><term id="hep29081-li-0021"><offsets xml_i="15246" xml_f="15249" txt_i="2076" txt_f="2079">PIB</offsets></term><def id="hep29081-li-0022"><p><offsets xml_i="15286" xml_f="15298" txt_i="2079" txt_f="2091">pibrentasvir</offsets></p></def></def-item><def-item><term id="hep29081-li-0023"><offsets xml_i="15357" xml_f="15360" txt_i="2092" txt_f="2095">RBV</offsets></term><def id="hep29081-li-0024"><p><offsets xml_i="15397" xml_f="15406" txt_i="2095" txt_f="2104">ribavirin</offsets></p></def></def-item><def-item><term id="hep29081-li-0025"><offsets xml_i="15465" xml_f="15470" txt_i="2105" txt_f="2110">SVR12</offsets></term><def id="hep29081-li-0026"><p><offsets xml_i="15507" xml_f="15560" txt_i="2110" txt_f="2163">sustained virologic response at posttreatment week 12</offsets></p></def></def-item></def-list><p><offsets xml_i="15595" xml_f="15760" txt_i="2164" txt_f="2329">Hepatitis C virus (HCV) genotype (GT) 1 is the most common among HCV GTs globally, accounting for approximately 46% of an estimated 185 million infections worldwide.</offsets><xref rid="hep29081-bib-0001" ref-type="ref"><offsets xml_i="15805" xml_f="15806" txt_i="2329" txt_f="2330">1</offsets></xref><offsets xml_i="15813" xml_f="15815" txt_i="2330" txt_f="2332">, </offsets><xref rid="hep29081-bib-0002" ref-type="ref"><offsets xml_i="15860" xml_f="15861" txt_i="2332" txt_f="2333">2</offsets></xref><offsets xml_i="15868" xml_f="16004" txt_i="2333" txt_f="2469"> Although the treatment landscape for HCV has rapidly evolved with highly effective and safe treatments for patients with GT1 infection,</offsets><xref rid="hep29081-bib-0003" ref-type="ref"><offsets xml_i="16049" xml_f="16050" txt_i="2469" txt_f="2470">3</offsets></xref><offsets xml_i="16057" xml_f="16059" txt_i="2470" txt_f="2472">, </offsets><xref rid="hep29081-bib-0004" ref-type="ref"><offsets xml_i="16104" xml_f="16105" txt_i="2472" txt_f="2473">4</offsets></xref><offsets xml_i="16112" xml_f="16255" txt_i="2473" txt_f="2616"> as direct‐acting antiviral (DAA) agents are used extensively, the number of patients with virologic failure of DAA regimens continues to grow.</offsets><xref rid="hep29081-bib-0005" ref-type="ref"><offsets xml_i="16300" xml_f="16301" txt_i="2616" txt_f="2617">5</offsets></xref><offsets xml_i="16308" xml_f="16465" txt_i="2617" txt_f="2774"> Virologic failure of DAAs often results from baseline resistance‐associated polymorphisms or resistance‐associated substitutions that emerge during therapy.</offsets><xref rid="hep29081-bib-0004" ref-type="ref"><offsets xml_i="16510" xml_f="16511" txt_i="2774" txt_f="2775">4</offsets></xref><offsets xml_i="16518" xml_f="16520" txt_i="2775" txt_f="2777">, </offsets><xref rid="hep29081-bib-0005" ref-type="ref"><offsets xml_i="16565" xml_f="16566" txt_i="2777" txt_f="2778">5</offsets></xref><offsets xml_i="16573" xml_f="16575" txt_i="2778" txt_f="2780">, </offsets><xref rid="hep29081-bib-0006" ref-type="ref"><offsets xml_i="16620" xml_f="16621" txt_i="2780" txt_f="2781">6</offsets></xref><offsets xml_i="16628" xml_f="16630" txt_i="2781" txt_f="2783">, </offsets><xref rid="hep29081-bib-0007" ref-type="ref"><offsets xml_i="16675" xml_f="16676" txt_i="2783" txt_f="2784">7</offsets></xref><offsets xml_i="16683" xml_f="16685" txt_i="2784" txt_f="2786">, </offsets><xref rid="hep29081-bib-0008" ref-type="ref"><offsets xml_i="16730" xml_f="16731" txt_i="2786" txt_f="2787">8</offsets></xref><offsets xml_i="16738" xml_f="16888" txt_i="2787" txt_f="2937"> Variants within the HCV nonstructural protein 5A (NS5A) region substantially increase the risk of virologic failure for many DAA‐containing regimens,</offsets><xref rid="hep29081-bib-0009" ref-type="ref"><offsets xml_i="16933" xml_f="16934" txt_i="2937" txt_f="2938">9</offsets></xref><offsets xml_i="16941" xml_f="16943" txt_i="2938" txt_f="2940">, </offsets><xref rid="hep29081-bib-0010" ref-type="ref"><offsets xml_i="16988" xml_f="16990" txt_i="2940" txt_f="2942">10</offsets></xref><offsets xml_i="16997" xml_f="17114" txt_i="2942" txt_f="3059"> and there are currently no treatments specifically indicated for either NS5A or NS5B inhibitor–experienced patients.</offsets></p><p><offsets xml_i="17121" xml_f="17234" txt_i="3060" txt_f="3173">DAA treatment failure is a growing concern given the long‐term persistence of NS5A resistance‐associated variants</offsets><xref rid="hep29081-bib-0011" ref-type="ref"><offsets xml_i="17279" xml_f="17281" txt_i="3173" txt_f="3175">11</offsets></xref><offsets xml_i="17288" xml_f="17290" txt_i="3175" txt_f="3177">, </offsets><xref rid="hep29081-bib-0012" ref-type="ref"><offsets xml_i="17335" xml_f="17337" txt_i="3177" txt_f="3179">12</offsets></xref><offsets xml_i="17344" xml_f="17483" txt_i="3179" txt_f="3318"> and suboptimal treatment response rates in patients with resistance‐associated baseline polymorphisms or treatment‐emergent substitutions.</offsets><xref rid="hep29081-bib-0009" ref-type="ref"><offsets xml_i="17528" xml_f="17529" txt_i="3318" txt_f="3319">9</offsets></xref><offsets xml_i="17536" xml_f="17538" txt_i="3319" txt_f="3321">, </offsets><xref rid="hep29081-bib-0013" ref-type="ref"><offsets xml_i="17583" xml_f="17585" txt_i="3321" txt_f="3323">13</offsets></xref><offsets xml_i="17592" xml_f="17594" txt_i="3323" txt_f="3325">, </offsets><xref rid="hep29081-bib-0014" ref-type="ref"><offsets xml_i="17639" xml_f="17641" txt_i="3325" txt_f="3327">14</offsets></xref><offsets xml_i="17648" xml_f="17902" txt_i="3327" txt_f="3581"> Retreatment strategies using 24 weeks of ledipasvir/sofosbuvir have demonstrated a 60% sustained virologic response at posttreatment week 12 (SVR12) rate among patients with baseline NS5A polymorphisms and 33% in patients with NS5A Y93H/N polymorphisms.</offsets><xref rid="hep29081-bib-0014" ref-type="ref"><offsets xml_i="17947" xml_f="17949" txt_i="3581" txt_f="3583">14</offsets></xref><offsets xml_i="17956" xml_f="18186" txt_i="3583" txt_f="3813"> Similarly, the combination regimen of elbasvir and grazoprevir has reduced efficacy (70%) in GT1a‐infected patients with baseline NS5A resistance‐associated polymorphisms at elbasvir‐specific positions (e.g., M28, Q30, L31, Y93).</offsets><xref rid="hep29081-bib-0009" ref-type="ref"><offsets xml_i="18231" xml_f="18232" txt_i="3813" txt_f="3814">9</offsets></xref><offsets xml_i="18239" xml_f="18241" txt_i="3814" txt_f="3816">, </offsets><xref rid="hep29081-bib-0015" ref-type="ref"><offsets xml_i="18286" xml_f="18288" txt_i="3816" txt_f="3818">15</offsets></xref><offsets xml_i="18295" xml_f="18545" txt_i="3818" txt_f="4068"> In addition, the combination of sofosbuvir/velpatasvir has reduced potency to HCV GT1a NS5A variants at position Y93, including the common Y93H variant, which confers a 609‐fold increase in half‐maximal effective concentration (EC50) to velpatasvir.</offsets><xref rid="hep29081-bib-0016" ref-type="ref"><offsets xml_i="18590" xml_f="18592" txt_i="4068" txt_f="4070">16</offsets></xref><offsets xml_i="18599" xml_f="18779" txt_i="4070" txt_f="4250"> Therefore, treatments with a high barrier to viral resistance that maintain potency against viral variants, particularly for patients previously treated with HCV DAAs, are needed.</offsets></p><p><offsets xml_i="18786" xml_f="19170" txt_i="4251" txt_f="4635">Current recommended retreatment strategies for patients with prior failure of NS3/4A protease inhibitor (PI)–containing DAA regimens include the NS5B nucleotide analogue inhibitor sofosbuvir plus an NS5A inhibitor (ledipasvir, velpatasvir, or daclatasvir) for 12 weeks or the combination of the NS3/4A PI grazoprevir plus the NS5A inhibitor elbasvir with ribavirin (RBV) for 16 weeks.</offsets><xref rid="hep29081-bib-0016" ref-type="ref"><offsets xml_i="19215" xml_f="19217" txt_i="4635" txt_f="4637">16</offsets></xref><offsets xml_i="19224" xml_f="19560" txt_i="4637" txt_f="4973"> Patients with baseline NS5A variants, either preexisting or the result of treatment emergence from prior exposure to an NS5A inhibitor, have proven more difficult to cure with approved DAA regimens; thus, longer treatment durations, addition of RBV, and the addition of a third or fourth DAA have been examined to maximize SVR12 rates.</offsets><xref rid="hep29081-bib-0014" ref-type="ref"><offsets xml_i="19605" xml_f="19607" txt_i="4973" txt_f="4975">14</offsets></xref><offsets xml_i="19614" xml_f="19616" txt_i="4975" txt_f="4977">, </offsets><xref rid="hep29081-bib-0017" ref-type="ref"><offsets xml_i="19661" xml_f="19663" txt_i="4977" txt_f="4979">17</offsets></xref><offsets xml_i="19670" xml_f="19672" txt_i="4979" txt_f="4981">, </offsets><xref rid="hep29081-bib-0018" ref-type="ref"><offsets xml_i="19717" xml_f="19719" txt_i="4981" txt_f="4983">18</offsets></xref><offsets xml_i="19726" xml_f="19728" txt_i="4983" txt_f="4985">, </offsets><xref rid="hep29081-bib-0019" ref-type="ref"><offsets xml_i="19773" xml_f="19775" txt_i="4985" txt_f="4987">19</offsets></xref><offsets xml_i="19782" xml_f="19784" txt_i="4987" txt_f="4989">, </offsets><xref rid="hep29081-bib-0020" ref-type="ref"><offsets xml_i="19829" xml_f="19831" txt_i="4989" txt_f="4991">20</offsets></xref><offsets xml_i="19838" xml_f="19839" txt_i="4991" txt_f="4992">
</offsets></p><p><offsets xml_i="19846" xml_f="20031" txt_i="4993" txt_f="5178">Glecaprevir (GLE; formerly ABT‐493; identified by AbbVie and Enanta) is an HCV NS3/4A PI that has potent pangenotypic antiviral activity. GLE does not inhibit human proteases, exhibits </offsets><italic><offsets xml_i="20039" xml_f="20047" txt_i="5178" txt_f="5186">in vitro</offsets></italic><offsets xml_i="20056" xml_f="20271" txt_i="5186" txt_f="5398"> EC50 values ≤5 nanomolar across all major HCV GTs, and demonstrates &lt;5‐fold loss of activity against common GT1 variants at key resistance‐associated positions of R155 and D168 to currently available NS3/4A PIs.</offsets><xref rid="hep29081-bib-0006" ref-type="ref"><offsets xml_i="20316" xml_f="20317" txt_i="5398" txt_f="5399">6</offsets></xref><offsets xml_i="20324" xml_f="20326" txt_i="5399" txt_f="5401">, </offsets><xref rid="hep29081-bib-0021" ref-type="ref"><offsets xml_i="20371" xml_f="20373" txt_i="5401" txt_f="5403">21</offsets></xref><offsets xml_i="20380" xml_f="20625" txt_i="5403" txt_f="5648"> Pibrentasvir (PIB; formerly ABT‐530) is an HCV NS5A inhibitor with EC50 values ≤5 picomolar across all major HCV GTs; it maintains high potency against common NS5A resistance‐associated variants, including GT1a Y93H (6.7‐fold increase in EC50),</offsets><xref rid="hep29081-bib-0022" ref-type="ref"><offsets xml_i="20670" xml_f="20672" txt_i="5648" txt_f="5650">22</offsets></xref><offsets xml_i="20679" xml_f="20897" txt_i="5650" txt_f="5868"> which has been associated with reduced susceptibility to other NS5A inhibitors, such as ledipasvir (3,294‐fold increase in EC50), daclatasvir (1,600‐fold increase in EC50), and velpatasvir (609‐fold increase in EC50).</offsets><xref rid="hep29081-bib-0023" ref-type="ref"><offsets xml_i="20942" xml_f="20944" txt_i="5868" txt_f="5870">23</offsets></xref><offsets xml_i="20951" xml_f="20952" txt_i="5870" txt_f="5871">
</offsets></p><p><offsets xml_i="20959" xml_f="21312" txt_i="5872" txt_f="6225">In part 1 of the MAGELLAN‐1 study, we evaluated the efficacy and safety of GLE + PIB for 12 weeks, with or without RBV, in patients with prior treatment failure of HCV regimens containing an NS5A inhibitor and/or NS3/4A PI with or without NS5B inhibitors. The impacts of baseline polymorphisms and RBV coadministration on SVR12 rates were also assessed.</offsets></p><sec id="hep29081-sec-0002"><title><offsets xml_i="21351" xml_f="21372" txt_i="6226" txt_f="6247">Materials and Methods</offsets></title><sec id="hep29081-sec-0003"><title><offsets xml_i="21415" xml_f="21442" txt_i="6248" txt_f="6275">STUDY OVERVIEW AND REGIMENS</offsets></title><p><offsets xml_i="21453" xml_f="21738" txt_i="6276" txt_f="6561">The MAGELLAN‐1 (NCT02446717) study was a phase 2, randomized, open‐label, multicenter study that assessed the efficacy and safety of GLE + PIB in HCV GT1‐infected patients with prior DAA treatment experience. Patients were initially randomized 1:1:1 into three arms (A, B, and C; Fig. </offsets><xref rid="hep29081-fig-0001" ref-type="fig"><offsets xml_i="21783" xml_f="21784" txt_i="6561" txt_f="6562">1</offsets></xref><offsets xml_i="21791" xml_f="22051" txt_i="6562" txt_f="6822">). Patients enrolled in arm A were treated for 12 weeks with GLE (200 mg once daily) + PIB (80 mg once daily); however, a protocol amendment was implemented to halt enrollment (with 6 patients enrolled) to optimize doses of GLE and PIB for further development.</offsets><xref rid="hep29081-bib-0024" ref-type="ref"><offsets xml_i="22096" xml_f="22098" txt_i="6822" txt_f="6824">24</offsets></xref><offsets xml_i="22105" xml_f="23062" txt_i="6824" txt_f="7781"> Subsequent to this protocol amendment, the remaining enrolled patients were randomized 1:1 into arm B or C for treatment with GLE (300 mg once daily) + PIB (120 mg once daily) with RBV (arm B; 800 mg once daily) or without RBV (arm C) for 12 weeks. Patients were stratified by HCV subtype (1b or non‐1b) and previous DAA class (NS5A inhibitor‐experienced, NS3/4A PI‐experienced but NS5A inhibitor‐naive, or other). For the purpose of analysis, previous HCV treatment experience was deemed cumulative (i.e., a patient exposed to NS3/4A PI and subsequently to NS5A inhibitor was considered NS5A and NS3/4A PI‐experienced). All patients signed informed consent, and the study was conducted in accordance with its protocol (designed and sponsored by AbbVie), the Good Clinical Practice Guidelines, and the ethical principles set forth in the Declaration of Helsinki, with independent ethics committee or institutional review board approval for all study sites.</offsets></p><fig fig-type="Figure" xml:lang="en" id="hep29081-fig-0001" orientation="portrait" position="float"><label><offsets xml_i="23173" xml_f="23181" txt_i="7782" txt_f="7790">Figure 1</offsets></label><caption><p><offsets xml_i="23201" xml_f="23788" txt_i="7790" txt_f="8377">MAGELLAN‐1, part 1, clinical trial design schematic. In part 1 of the MAGELLAN‐1 study, patients were randomized 1:1:1 into three treatment arms, stratified by HCV subtype (1b or non‐1b) and previous DAA classification (NS5A inhibitor–experienced, NS3/4A PI–experienced but NS5A inhibitor‐naive, or other). Enrollment in arm A was halted by protocol amendment after 6 patients were randomized to that arm (see Materials and Methods). In total, 50 patients were enrolled to receive GLE + PIB ± RBV, once daily, for 12 weeks. The primary endpoint was the proportion of patients with SVR12.</offsets></p></caption><graphic id="nlm-graphic-1" xlink:href="HEP-66-389-g001"></graphic></fig></sec><sec id="hep29081-sec-0004"><title><offsets xml_i="23916" xml_f="23962" txt_i="8379" txt_f="8425">PATIENT POPULATION, CRITERIA, AND STUDY DESIGN</offsets></title><p><offsets xml_i="23973" xml_f="25062" txt_i="8426" txt_f="9509">Patients were adults, 18‐70 years, without cirrhosis but with chronic HCV GT1 infection and treatment‐experienced with a prior DAA‐containing regimen. Patients had to have completed past DAA treatment at least 1 month prior to screening visit, with the outcome of prior HCV treatment being either on‐treatment virologic failure or posttreatment relapse. Plasma samples for HCV genotyping were collected at screening and assessed with the Versant HCV Genotype Inno LiPA Assay, version 2.0 or higher. The absence of cirrhosis (METAVIR score ≤3, Ishak score ≤4) was determined by one of the following: liver biopsy within 24 months prior to (or during) screening, transient elastography (FibroScan) result of &lt;12.5 kPa within 6 months prior to (or during) screening, or a screening FibroTest score of ≤0.48 with an aspartate aminotransferase to platelet ratio index &lt;1. Patients coinfected with hepatitis B virus, human immunodeficiency virus, or more than one HCV GT at screening were excluded. Key eligibility criteria and definitions of prior treatment responses are provided in the </offsets><ext-link ext-link-type="uri" xlink:href="http://onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo"><offsets xml_i="25167" xml_f="25189" txt_i="9509" txt_f="9531">Supporting Information</offsets></ext-link><offsets xml_i="25200" xml_f="25201" txt_i="9531" txt_f="9532">.</offsets></p></sec><sec id="hep29081-sec-0005"><title><offsets xml_i="25246" xml_f="25289" txt_i="9534" txt_f="9577">EFFICACY, VIROLOGIC, AND SAFETY ASSESSMENTS</offsets></title><p><offsets xml_i="25300" xml_f="26199" txt_i="9578" txt_f="10474">Plasma samples for HCV RNA measurements were collected at screening; treatment days 1 and 3 and weeks 1, 2, 4, 6, 8, 10, and 12 (or early discontinuation); and posttreatment weeks 2, 4, 8, 12, and 24 (or early discontinuation) and assessed using the COBAS AmpliPrep/ COBAS TaqMan HCV Quantitative Test, version 2.0. The primary efficacy endpoint was the percentage of patients who achieved SVR12 (HCV RNA &lt;15 IU/mL) in the intent‐to‐treat (ITT) population, defined as all randomized patients who received at least one dose of study drug. A modified ITT (mITT) analysis was also conducted, excluding all nonvirologic failures (i.e., patients lost to follow‐up or early discontinuation). Two‐sided confidence intervals (CIs) were determined at a significance level of 0.05 using the Wilson score method for binomial proportions. Statistical summaries were performed using SAS software, version 9.3.</offsets></p><p><offsets xml_i="26206" xml_f="27657" txt_i="10475" txt_f="11926">Viral sequences from the baseline plasma sample for each patient were analyzed by next‐generation sequencing (Illumina MiSeq) to identify NS3 or NS5A polymorphisms at detection thresholds of 2% and 15%. For patients who had virologic failure within the study, the first samples with HCV RNA ≥1,000 IU/mL collected at or after the time of virologic failure were also analyzed by next‐generation sequencing at the same thresholds. Sequences were compared to the corresponding baseline and reference sequences to identify amino acid substitutions that could be associated with resistance to components of the therapy. For HCV resistance analysis, a polymorphism was defined as a baseline amino acid difference relative to the appropriate subtype‐specific reference sequence; despite DAA‐experienced patients having had prior HCV antiviral therapy, a patient's baseline amino acid variants in this study were considered polymorphisms as the patient's HCV amino acid sequences prior to all previous therapies are unknown. A substitution was defined as a treatment‐emergent amino acid sequence different from the patient's baseline viral sequence. An amino acid variant was considered an amino acid change due to a baseline polymorphism or treatment‐emergent substitution. Detailed information on the collection of plasma samples, HCV RNA measurement, virologic‐failure criteria, and amino acid variants included in resistance analysis are available in the </offsets><ext-link ext-link-type="uri" xlink:href="http://onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo"><offsets xml_i="27762" xml_f="27784" txt_i="11926" txt_f="11948">Supporting Information</offsets></ext-link><offsets xml_i="27795" xml_f="27796" txt_i="11948" txt_f="11949">.</offsets></p><p><offsets xml_i="27803" xml_f="28155" txt_i="11950" txt_f="12302">Safety assessments were based on the safety population (same as the ITT population). Safety and tolerability assessments were conducted at screening and throughout the study and included monitoring vital signs, physical examinations, adverse events, and clinical chemistry and hematology tests. Adverse events were recorded up to 30 days posttreatment.</offsets></p></sec></sec><sec id="hep29081-sec-0006"><title><offsets xml_i="28206" xml_f="28213" txt_i="12305" txt_f="12312">Results</offsets></title><p><offsets xml_i="28224" xml_f="28628" txt_i="12313" txt_f="12717">A total of 91 patients with HCV GT1 were screened in the United States; 50 were randomized and received at least one dose of study drug. Patients not randomized due to abnormal laboratory values had elevated alanine aminotransferase, aspartate aminotransferase, direct bilirubin, or low platelet count. Among randomized patients, 82% were male, 34% reported black race, and 84% had GT1a infection (Table </offsets><xref rid="hep29081-tbl-0001" ref-type="table-wrap"><offsets xml_i="28680" xml_f="28681" txt_i="12717" txt_f="12718">1</offsets></xref><offsets xml_i="28688" xml_f="29351" txt_i="12718" txt_f="13381">). The majority of patients (66%) had prior treatment failure of regimens containing multiple DAAs; 50% had been previously treated with an NS5A inhibitor, 84% with an NS3/4A PI, and 54% with an NS5B polymerase inhibitor. Of 27 patients with NS5B polymerase inhibitor exposure, 56% had prior treatment with a nucleotide analogue inhibitor, 33% had treatment with non‐nucleoside inhibitors, and 11% had exposure to both. The most common prior DAA‐containing regimens were boceprevir plus pegylated interferon/RBV (n = 10), telaprevir plus pegylated interferon/RBV (n = 8), ledipasvir/sofosbuvir (n = 8), and simeprevir plus sofosbuvir with or without RBV (n = 8) (</offsets><ext-link ext-link-type="uri" xlink:href="http://onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo"><offsets xml_i="29456" xml_f="29475" txt_i="13381" txt_f="13400">Supporting Table S1</offsets></ext-link><offsets xml_i="29486" xml_f="29843" txt_i="13400" txt_f="13757">). Overall, 21/50 patients had prior failure to NS3/4A PI + pegylated interferon/RBV; 5 of those 21 patients were also experienced with NS5A or NS5B inhibitors within the interferon‐containing regimen or within other regimens. The type of prior DAA experience (NS3/4A PI only or NS5A inhibitor only or both) was well balanced across treatment groups (Table </offsets><xref rid="hep29081-tbl-0001" ref-type="table-wrap"><offsets xml_i="29895" xml_f="29896" txt_i="13757" txt_f="13758">1</offsets></xref><offsets xml_i="29903" xml_f="30395" txt_i="13758" txt_f="14250">). Next‐generation sequencing identified baseline polymorphisms in NS3 and/or NS5A in 86% (43/50) of patients at a detection threshold of 2% and in 80% (40/50) of patients at a threshold of 15%. At a 15% detection threshold, the majority of patients had at least one baseline polymorphism across all treatment groups; although no patients in arm A had polymorphisms in both NS3 and NS5A targets, 27%(6/22) and 41% (9/22) had polymorphisms in both targets in arms B and C, respectively (Table </offsets><xref rid="hep29081-tbl-0001" ref-type="table-wrap"><offsets xml_i="30447" xml_f="30448" txt_i="14250" txt_f="14251">1</offsets></xref><offsets xml_i="30455" xml_f="31265" txt_i="14251" txt_f="15061">). In addition, baseline NS5A polymorphisms were detected in 50% (3/6), 50% (11/22), and 55% (12/22) of patients in arms A, B, and C, respectively, at a detection threshold of 15%. Although a detection threshold of 15% is commonly accepted as clinically relevant, lower thresholds of detection give greater insight into amino acid changes that may emerge in the majority of the population after prior therapy with DAA‐containing regimens. At a more sensitive 2% detection threshold, the most common polymorphisms in NS3 were at amino acid positions Q80 (n = 23), R155 (n = 4), D168 (n = 4), and I/V170 (n = 4); and those for NS5A were at positions Q30 (n = 14), Y93 (n = 8), L31 (n = 7), M28 (n = 6), Q54 (n = 6), and H/P58 (n = 5). Baseline amino acid polymorphisms detected at a threshold of 2% are shown in </offsets><ext-link ext-link-type="uri" xlink:href="http://onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo"><offsets xml_i="31370" xml_f="31389" txt_i="15061" txt_f="15080">Supporting Table S2</offsets></ext-link><offsets xml_i="31400" xml_f="31492" txt_i="15080" txt_f="15172">, and the prevalence of specific polymorphisms detected at both 2% and 15% is summarized in </offsets><ext-link ext-link-type="uri" xlink:href="http://onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo"><offsets xml_i="31597" xml_f="31616" txt_i="15172" txt_f="15191">Supporting Table S3</offsets></ext-link><offsets xml_i="31627" xml_f="31628" txt_i="15191" txt_f="15192">.</offsets></p><table-wrap id="hep29081-tbl-0001" xml:lang="en" orientation="portrait" position="float"><label><offsets xml_i="31728" xml_f="31735" txt_i="15193" txt_f="15200">Table 1</offsets></label><caption><p><offsets xml_i="31755" xml_f="31804" txt_i="15200" txt_f="15249">Patient Demographics and Baseline Characteristics</offsets></p></caption><table frame="hsides" rules="groups"><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><thead valign="bottom"><tr valign="bottom" style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="32355" xml_f="32369" txt_i="15250" txt_f="15264">Characteristic</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="32433" xml_f="32469" txt_i="15264" txt_f="15300">Arm A GLE+PIB (200 mg + 80 mg) n = 6</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="32533" xml_f="32586" txt_i="15300" txt_f="15353">Arm B GLE+PIB (300 mg + 120 mg) + RBV (800 mg) n = 22</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="32650" xml_f="32688" txt_i="15353" txt_f="15391">Arm C GLE+PIB (300 mg + 120 mg) n = 22</offsets></th></tr></thead><tbody valign="top"><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="32837" xml_f="32848" txt_i="15391" txt_f="15402">Male, n (%)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="32916" xml_f="32922" txt_i="15403" txt_f="15409">3 (50)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="32990" xml_f="32997" txt_i="15410" txt_f="15417">20 (91)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33065" xml_f="33072" txt_i="15418" txt_f="15425">18 (82)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="33193" xml_f="33204" txt_i="15426" txt_f="15437">Black race,</offsets><xref ref-type="fn" rid="hep29081-note-0002"><offsets xml_i="33249" xml_f="33250" txt_i="15437" txt_f="15438">a</offsets></xref><offsets xml_i="33257" xml_f="33263" txt_i="15438" txt_f="15444"> n (%)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33331" xml_f="33337" txt_i="15445" txt_f="15451">2 (33)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33405" xml_f="33411" txt_i="15452" txt_f="15458">5 (23)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33479" xml_f="33486" txt_i="15459" txt_f="15466">10 (45)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="33607" xml_f="33632" txt_i="15467" txt_f="15492">Age, median years (range)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33700" xml_f="33710" txt_i="15493" txt_f="15503">59 (39‐61)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33778" xml_f="33788" txt_i="15504" txt_f="15514">56 (39‐64)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="33856" xml_f="33866" txt_i="15515" txt_f="15525">59 (46‐70)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="33987" xml_f="34005" txt_i="15526" txt_f="15544">HCV subtype, n (%)</offsets></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="34328" xml_f="34330" txt_i="15548" txt_f="15550">1a</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="34398" xml_f="34404" txt_i="15551" txt_f="15557">4 (67)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="34472" xml_f="34479" txt_i="15558" txt_f="15565">20 (91)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="34547" xml_f="34554" txt_i="15566" txt_f="15573">18 (82)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="34700" xml_f="34702" txt_i="15574" txt_f="15576">1b</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="34770" xml_f="34776" txt_i="15577" txt_f="15583">2 (33)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="34844" xml_f="34849" txt_i="15584" txt_f="15589">2 (9)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="34917" xml_f="34923" txt_i="15590" txt_f="15596">4 (18)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="35044" xml_f="35084" txt_i="15597" txt_f="15637">Treatment experience by DAA class, n (%)</offsets></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="35407" xml_f="35432" txt_i="15641" txt_f="15666">NS5A‐experienced/PI‐naive</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="35493" xml_f="35494" txt_i="15667" txt_f="15668">0</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="35562" xml_f="35568" txt_i="15669" txt_f="15675">4 (18)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="35636" xml_f="35642" txt_i="15676" txt_f="15682">4 (18)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="35788" xml_f="35813" txt_i="15683" txt_f="15708">NS5A‐naive/PI‐experienced</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="35881" xml_f="35887" txt_i="15709" txt_f="15715">3 (50)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="35955" xml_f="35962" txt_i="15716" txt_f="15723">11 (50)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36030" xml_f="36037" txt_i="15724" txt_f="15731">11 (50)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="36183" xml_f="36214" txt_i="15732" txt_f="15763">NS5A‐experienced/PI‐experienced</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36282" xml_f="36288" txt_i="15764" txt_f="15770">3 (50)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36356" xml_f="36362" txt_i="15771" txt_f="15777">7 (32)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36430" xml_f="36436" txt_i="15778" txt_f="15784">7 (32)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="36582" xml_f="36598" txt_i="15785" txt_f="15801">BMI, median kg/m</offsets><sup><offsets xml_i="36603" xml_f="36604" txt_i="15801" txt_f="15802">2</offsets></sup><offsets xml_i="36610" xml_f="36618" txt_i="15802" txt_f="15810"> (range)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36686" xml_f="36696" txt_i="15811" txt_f="15821">27 (25‐37)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36764" xml_f="36774" txt_i="15822" txt_f="15832">28 (22‐34)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="36842" xml_f="36852" txt_i="15833" txt_f="15843">28 (19‐37)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="36998" xml_f="36999" txt_i="15844" txt_f="15845">
</offsets><italic><offsets xml_i="37007" xml_f="37012" txt_i="15845" txt_f="15850">IL28B</offsets></italic><offsets xml_i="37021" xml_f="37044" txt_i="15850" txt_f="15873"> non‐CC genotype, n (%)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="37112" xml_f="37118" txt_i="15874" txt_f="15880">4 (67)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="37186" xml_f="37193" txt_i="15881" txt_f="15888">16 (73)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="37261" xml_f="37268" txt_i="15889" txt_f="15896">19 (86)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="37414" xml_f="37433" txt_i="15897" txt_f="15916">HCV RNA, median log</offsets><sub><offsets xml_i="37438" xml_f="37440" txt_i="15916" txt_f="15918">10</offsets></sub><offsets xml_i="37446" xml_f="37460" txt_i="15918" txt_f="15932"> IU/mL (range)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="37528" xml_f="37541" txt_i="15933" txt_f="15946">6.1 (5.6‐6.7)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="37609" xml_f="37622" txt_i="15947" txt_f="15960">6.7 (5.0‐7.3)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="37690" xml_f="37703" txt_i="15961" txt_f="15974">6.6 (5.5‐7.2)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="37849" xml_f="37880" txt_i="15975" txt_f="16006">HCV RNA ≥6,000,000 IU/mL, n (%)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="37941" xml_f="37942" txt_i="16007" txt_f="16008">0</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="38010" xml_f="38017" txt_i="16009" txt_f="16016">11 (50)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="38085" xml_f="38092" txt_i="16017" txt_f="16024">10 (46)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="38213" xml_f="38243" txt_i="16025" txt_f="16055">Baseline fibrosis stage, n (%)</offsets></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="38566" xml_f="38571" txt_i="16059" txt_f="16064">F0‐F1</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="38639" xml_f="38645" txt_i="16065" txt_f="16071">4 (67)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="38713" xml_f="38720" txt_i="16072" txt_f="16079">17 (77)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="38788" xml_f="38795" txt_i="16080" txt_f="16087">11 (50)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="38941" xml_f="38943" txt_i="16088" txt_f="16090">F2</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="39011" xml_f="39017" txt_i="16091" txt_f="16097">1 (17)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="39078" xml_f="39079" txt_i="16098" txt_f="16099">0</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="39147" xml_f="39153" txt_i="16100" txt_f="16106">6 (27)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="39299" xml_f="39301" txt_i="16107" txt_f="16109">F3</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="39369" xml_f="39375" txt_i="16110" txt_f="16116">1 (17)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="39443" xml_f="39449" txt_i="16117" txt_f="16123">5 (23)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="39517" xml_f="39523" txt_i="16124" txt_f="16130">5 (23)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" valign="top" rowspan="1" colspan="1"><offsets xml_i="39644" xml_f="39667" txt_i="16131" txt_f="16154">Baseline polymorphisms,</offsets><xref ref-type="fn" rid="hep29081-note-0003"><offsets xml_i="39712" xml_f="39713" txt_i="16154" txt_f="16155">b</offsets></xref><offsets xml_i="39720" xml_f="39726" txt_i="16155" txt_f="16161"> n (%)</offsets></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="40049" xml_f="40079" txt_i="16165" txt_f="16195">Any polymorphism (NS3 or NS5A)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40147" xml_f="40153" txt_i="16196" txt_f="16202">5 (83)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40221" xml_f="40228" txt_i="16203" txt_f="16210">18 (82)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40296" xml_f="40303" txt_i="16211" txt_f="16218">17 (77)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="40449" xml_f="40457" txt_i="16219" txt_f="16227">NS3 only</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40525" xml_f="40531" txt_i="16228" txt_f="16234">2 (33)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40599" xml_f="40605" txt_i="16235" txt_f="16241">7 (32)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40673" xml_f="40679" txt_i="16242" txt_f="16248">5 (23)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="40825" xml_f="40834" txt_i="16249" txt_f="16258">NS5A only</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40902" xml_f="40908" txt_i="16259" txt_f="16265">3 (50)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="40976" xml_f="40982" txt_i="16266" txt_f="16272">5 (23)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="41050" xml_f="41056" txt_i="16273" txt_f="16279">3 (14)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="41202" xml_f="41219" txt_i="16280" txt_f="16297">Both NS3 and NS5A</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="41280" xml_f="41281" txt_i="16298" txt_f="16299">0</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="41349" xml_f="41355" txt_i="16300" txt_f="16306">6 (27)</offsets></td><td align="char" char="(" valign="top" rowspan="1" colspan="1"><offsets xml_i="41423" xml_f="41429" txt_i="16307" txt_f="16313">9 (41)</offsets></td></tr></tbody></table><table-wrap-foot><fn id="hep29081-note-0002"><label><offsets xml_i="41507" xml_f="41508" txt_i="16314" txt_f="16315">a</offsets></label><p><offsets xml_i="41519" xml_f="41542" txt_i="16315" txt_f="16338">Race was self‐reported.</offsets></p></fn><fn id="hep29081-note-0003"><label><offsets xml_i="41586" xml_f="41587" txt_i="16339" txt_f="16340">b</offsets></label><p><offsets xml_i="41598" xml_f="41815" txt_i="16340" txt_f="16557">Polymorphisms were detected at a 15% detection threshold with next‐generation sequencing. “Only” indicates total number of patients with baseline polymorphisms within the indicated target and none in the other target.</offsets></p></fn><fn id="hep29081-note-0004"><p><offsets xml_i="41855" xml_f="41869" txt_i="16558" txt_f="16572">Abbreviation: </offsets><italic><offsets xml_i="41877" xml_f="41882" txt_i="16572" txt_f="16577">IL28B</offsets></italic><offsets xml_i="41891" xml_f="41909" txt_i="16577" txt_f="16595">, interleukin 28B.</offsets></p></fn></table-wrap-foot></table-wrap><p><offsets xml_i="41952" xml_f="42490" txt_i="16596" txt_f="17134">Overall, by ITT analysis, SVR12 was achieved in 92% (46/50, 95% CI 81‐97) of patients treated with GLE + PIB with or without RBV for 12 weeks. In the halted arm A, in which patients received the lower dose of 200 mg GLE and 80 mg PIB, 100% (6/6, 95% CI 61‐100) of patients achieved SVR12. In arm B (300 mg GLE + 120 mg PIB + 800 mg RBV for 12 weeks) SVR12 was achieved in 95% (21/22, 95% CI 78‐99) of patients, and among patients in arm C (300 mg GLE + 120 mg PIB for 12 weeks), 86% (19/22, 95% CI 67‐95) of patients achieved SVR12 (Fig. </offsets><xref rid="hep29081-fig-0002" ref-type="fig"><offsets xml_i="42535" xml_f="42536" txt_i="17134" txt_f="17135">2</offsets></xref><offsets xml_i="42543" xml_f="42974" txt_i="17135" txt_f="17566">). The rates of virologic failure were identical (1/22, 5%) with or without administration of RBV (arm B versus arm C). The two patients in arm C who did not achieve SVR12 were lost to follow‐up; however, both patients had undetectable HCV RNA at posttreatment week 8. By mITT analysis, excluding patients who failed to achieve SVR12 for nonvirologic reasons, SVR12 rates were 100%, 95%, and 95% for arms A, B, and C, respectively.</offsets></p><fig fig-type="Figure" xml:lang="en" id="hep29081-fig-0002" orientation="portrait" position="float"><label><offsets xml_i="43085" xml_f="43093" txt_i="17567" txt_f="17575">Figure 2</offsets></label><caption><p><offsets xml_i="43113" xml_f="43685" txt_i="17575" txt_f="18147">Sustained virologic response in the ITT and mITT populations. Individual SVR12 rates for arm A (blue), arm B (green), and arm C (gray‐blue) are shown for the ITT and mITT populations. The ITT population was all patients that received at least one dose of study drug (n = 50), while the mITT population excluded all patients who did not achieve SVR for reasons other than virologic failure. Whiskers represent the 95% CI using the Wilson score method. Both patients lost to follow‐up had nondetectable HCV RNA at posttreatment week 8. Abbreviation: LTFU, lost‐to‐follow‐up.</offsets></p></caption><graphic id="nlm-graphic-3" xlink:href="HEP-66-389-g002"></graphic></fig><p><offsets xml_i="43775" xml_f="43966" txt_i="18148" txt_f="18339">Two confirmed virologic failures were observed; both patients were compliant and had on‐treatment drug levels of GLE and PIB that were consistent with those observed in other patients (Table </offsets><xref rid="hep29081-tbl-0002" ref-type="table-wrap"><offsets xml_i="44018" xml_f="44019" txt_i="18339" txt_f="18340">2</offsets></xref><offsets xml_i="44026" xml_f="45181" txt_i="18340" txt_f="19495">). The patient in arm C (300 mg GLE + 120 mg PIB) had on‐treatment HCV breakthrough at week 8; using a 2% detection threshold, this patient had baseline polymorphisms in both NS3 (Y56H and D168A/T) and NS5A (M28V, Q30R, and H58C). At the time of virologic failure, this patient had treatment‐emergent substitutions of V36M in NS3 and M28G in NS5A. Of note, although GLE and PIB levels in this patient were within expected therapeutic ranges, the patient had Crohn's disease, was receiving immunosuppressive therapy, and had a prior ileocolectomy. The other patient who had virologic failure was a posttreatment week 4 relapse in arm B (300 mg GLE + 120 mg PIB + 800 mg RBV) and had baseline NS5A polymorphisms L31M and H58D, with no polymorphisms in NS3. At the time of relapse, this patient had treatment‐emergent substitutions of A156V in NS3 and Q30R in NS5A. Overall, in patients with baseline polymorphisms in NS3 only, NS5A only, or both NS3 and NS5A at a 15% detection threshold, SVR12 rates were 100% (14/14), 91% (10/11), and 93% (14/15), respectively; SVR12 was achieved by 100% (10/10) of patients with no baseline polymorphisms in NS3 or NS5A.</offsets></p><table-wrap id="hep29081-tbl-0002" xml:lang="en" orientation="portrait" position="float"><label><offsets xml_i="45281" xml_f="45288" txt_i="19496" txt_f="19503">Table 2</offsets></label><caption><p><offsets xml_i="45308" xml_f="45374" txt_i="19503" txt_f="19569">Characteristics of Patients with Virologic Breakthrough or Relapse</offsets></p></caption><table frame="hsides" rules="groups"><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><thead valign="bottom"><tr valign="bottom" style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46195" xml_f="46206" txt_i="19570" txt_f="19581">Subgenotype</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46270" xml_f="46285" txt_i="19581" txt_f="19596">Prior treatment</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46349" xml_f="46366" txt_i="19596" txt_f="19613">12‐Week treatment</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46430" xml_f="46452" txt_i="19613" txt_f="19635">Reason for nonresponse</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46516" xml_f="46536" txt_i="19635" txt_f="19655">Time point sequenced</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46600" xml_f="46612" txt_i="19655" txt_f="19667">NS3 variants</offsets><xref ref-type="fn" rid="hep29081-note-0005"><offsets xml_i="46657" xml_f="46658" txt_i="19667" txt_f="19668">a</offsets></xref><offsets xml_i="46665" xml_f="46666" txt_i="19668" txt_f="19669">
</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="46730" xml_f="46743" txt_i="19669" txt_f="19682">NS5A variants</offsets><xref ref-type="fn" rid="hep29081-note-0005"><offsets xml_i="46788" xml_f="46789" txt_i="19682" txt_f="19683">a</offsets></xref><offsets xml_i="46796" xml_f="46797" txt_i="19683" txt_f="19684">
</offsets></th></tr></thead><tbody valign="top"><tr valign="top" style="border-bottom:solid 1px #000000"><td rowspan="2" align="left" valign="top" colspan="1"><offsets xml_i="46946" xml_f="46948" txt_i="19684" txt_f="19686">1a</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47009" xml_f="47013" txt_i="19687" txt_f="19691">DCV;</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47074" xml_f="47089" txt_i="19692" txt_f="19707">GLE + PIB + RBV</offsets></td><td rowspan="2" align="left" valign="top" colspan="1"><offsets xml_i="47148" xml_f="47155" txt_i="19708" txt_f="19715">Relapse</offsets><xref ref-type="fn" rid="hep29081-note-0006"><offsets xml_i="47200" xml_f="47201" txt_i="19715" txt_f="19716">b</offsets></xref><offsets xml_i="47208" xml_f="47209" txt_i="19716" txt_f="19717">
</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47270" xml_f="47278" txt_i="19718" txt_f="19726">Baseline</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47339" xml_f="47343" txt_i="19727" txt_f="19731">None</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47404" xml_f="47420" txt_i="19732" txt_f="19748">L31M, H58D (26%)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47543" xml_f="47551" txt_i="19749" txt_f="19757">TVR + PR</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47612" xml_f="47636" txt_i="19758" txt_f="19782">300 mg + 120 mg + 800 mg</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47697" xml_f="47701" txt_i="19783" txt_f="19787">PTW4</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47762" xml_f="47773" txt_i="19788" txt_f="19799">A156V (91%)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="47834" xml_f="47850" txt_i="19800" txt_f="19816">Q30R, L31M, H58D</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td rowspan="2" align="left" valign="top" colspan="1"><offsets xml_i="47971" xml_f="47973" txt_i="19817" txt_f="19819">1a</offsets></td><td rowspan="2" align="center" valign="top" colspan="1"><offsets xml_i="48034" xml_f="48058" txt_i="19820" txt_f="19844">OBV + PTV/RTV + DSV +RBV</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48119" xml_f="48128" txt_i="19845" txt_f="19854">GLE + PIB</offsets></td><td rowspan="2" align="left" valign="top" colspan="1"><offsets xml_i="48187" xml_f="48199" txt_i="19855" txt_f="19867">Breakthrough</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48260" xml_f="48268" txt_i="19868" txt_f="19876">Baseline</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48329" xml_f="48356" txt_i="19877" txt_f="19904">Y56H (5%), D168A/T (94%/3%)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48417" xml_f="48450" txt_i="19905" txt_f="19938">M28V (3%), Q30R (98%), H58C (99%)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48573" xml_f="48588" txt_i="19939" txt_f="19954">300 mg + 120 mg</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48649" xml_f="48655" txt_i="19955" txt_f="19961">Week 8</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48716" xml_f="48738" txt_i="19962" txt_f="19984">V36M (6%), Y56H, D168A</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="48799" xml_f="48821" txt_i="19985" txt_f="20007">M28G, Q30R (99%), H58C</offsets></td></tr></tbody></table><table-wrap-foot><fn id="hep29081-note-0005"><label><offsets xml_i="48899" xml_f="48900" txt_i="20008" txt_f="20009">a</offsets></label><p><offsets xml_i="48911" xml_f="48986" txt_i="20009" txt_f="20084">Variants due to baseline polymorphisms or treatment‐emergent substitutions.</offsets></p></fn><fn id="hep29081-note-0006"><label><offsets xml_i="49030" xml_f="49031" txt_i="20085" txt_f="20086">b</offsets></label><p><offsets xml_i="49042" xml_f="49138" txt_i="20086" txt_f="20182">This patient had Crohn's disease, was on immunosuppressant therapy, and had prior ileocolectomy.</offsets></p></fn><fn id="hep29081-note-0007"><p><offsets xml_i="49178" xml_f="49411" txt_i="20183" txt_f="20413">Variants were detected at a 2% detection threshold with next‐generation sequencing. Only variants with prevalence ≥2% are listed; variants with prevalence &gt;99% within a patient's viral population do not have the prevalence listed.</offsets></p></fn><fn id="hep29081-note-0008"><p><offsets xml_i="49451" xml_f="49641" txt_i="20414" txt_f="20604">Abbreviations: DCV, daclatasvir; DSV, dasabuvir; OBV, ombitasvir; PR, pegylated interferon plus ribavirin; PTV/RTV, ritonavir‐boosted paritaprevir; PTW, post‐treatment week; TVR, telaprevir.</offsets></p></fn></table-wrap-foot></table-wrap><p><offsets xml_i="49684" xml_f="49858" txt_i="20605" txt_f="20779">Adverse events were mostly mild in severity and reported in 84% of patients. Adverse events occurring in ≥10% of patients were headache, fatigue, nausea, and insomnia (Table </offsets><xref rid="hep29081-tbl-0003" ref-type="table-wrap"><offsets xml_i="49910" xml_f="49911" txt_i="20779" txt_f="20780">3</offsets></xref><offsets xml_i="49918" xml_f="50497" txt_i="20780" txt_f="21347">); such events were more common in the RBV‐containing arm. Two treatment‐emergent serious adverse events were reported, neither of which was deemed related to study drug by the investigator (fractured femur and breast cancer). No patient prematurely discontinued treatment due to adverse events. Clinical chemistry and hematology revealed no significant on‐treatment abnormalities in alanine aminotransferase (&gt;3 × upper limit of normal), aspartate aminotransferase (&gt;3 × upper limit of normal), hemoglobin (&lt;10 g/dL), or total bilirubin (&gt;3 × upper limit of normal).</offsets></p><table-wrap id="hep29081-tbl-0003" xml:lang="en" orientation="portrait" position="float"><label><offsets xml_i="50597" xml_f="50604" txt_i="21348" txt_f="21355">Table 3</offsets></label><caption><p><offsets xml_i="50624" xml_f="50638" txt_i="21355" txt_f="21369">Adverse Events</offsets></p></caption><table frame="hsides" rules="groups"><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><col style="border-right:solid 1px #000000border-bottom:solid 1px #000000" span="1"></col><thead valign="bottom"><tr valign="bottom" style="border-bottom:solid 1px #000000"><th align="left" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="51189" xml_f="51201" txt_i="21370" txt_f="21382">Event, n (%)</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="51265" xml_f="51297" txt_i="21382" txt_f="21414">GLE + PIB (200 mg + 80 mg) n = 6</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="51361" xml_f="51409" txt_i="21414" txt_f="21462">GLE + PIB (300 mg + 80 mg) + RBV (800 mg) n = 22</offsets></th><th align="center" valign="bottom" rowspan="1" colspan="1"><offsets xml_i="51473" xml_f="51507" txt_i="21462" txt_f="21496">GLE + PIB (300 mg + 120 mg) n = 22</offsets></th></tr></thead><tbody valign="top"><tr valign="top" style="border-bottom:solid 1px #000000"><td colspan="4" align="left" style="border-bottom:solid 1px #000000" valign="top" rowspan="1"><offsets xml_i="51696" xml_f="51710" txt_i="21496" txt_f="21510">Adverse events</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="51856" xml_f="51859" txt_i="21511" txt_f="21514">Any</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="51920" xml_f="51928" txt_i="21515" txt_f="21523">5 (83.3)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="51989" xml_f="51998" txt_i="21524" txt_f="21533">19 (86.4)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52059" xml_f="52068" txt_i="21534" txt_f="21543">18 (81.8)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="52214" xml_f="52221" txt_i="21544" txt_f="21551">Serious</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52282" xml_f="52290" txt_i="21552" txt_f="21560">1 (16.7)</offsets><xref ref-type="fn" rid="hep29081-note-0010"><offsets xml_i="52335" xml_f="52336" txt_i="21560" txt_f="21561">a</offsets></xref><offsets xml_i="52343" xml_f="52344" txt_i="21561" txt_f="21562">
</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52405" xml_f="52412" txt_i="21563" txt_f="21570">1 (4.5)</offsets><xref ref-type="fn" rid="hep29081-note-0011"><offsets xml_i="52457" xml_f="52458" txt_i="21570" txt_f="21571">b</offsets></xref><offsets xml_i="52465" xml_f="52466" txt_i="21571" txt_f="21572">
</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52527" xml_f="52528" txt_i="21573" txt_f="21574">0</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="52674" xml_f="52701" txt_i="21575" txt_f="21602">Discontinuation as a result</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52762" xml_f="52763" txt_i="21603" txt_f="21604">0</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52824" xml_f="52825" txt_i="21605" txt_f="21606">0</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="52886" xml_f="52887" txt_i="21607" txt_f="21608">0</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td colspan="4" align="left" style="border-bottom:solid 1px #000000" valign="top" rowspan="1"><offsets xml_i="53048" xml_f="53069" txt_i="21609" txt_f="21630">Common adverse events</offsets><xref ref-type="fn" rid="hep29081-note-0012"><offsets xml_i="53114" xml_f="53115" txt_i="21630" txt_f="21631">c</offsets></xref><offsets xml_i="53122" xml_f="53123" txt_i="21631" txt_f="21632">
</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="53269" xml_f="53277" txt_i="21633" txt_f="21641">Headache</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="53338" xml_f="53346" txt_i="21642" txt_f="21650">1 (16.7)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="53407" xml_f="53415" txt_i="21651" txt_f="21659">5 (22.7)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="53476" xml_f="53484" txt_i="21660" txt_f="21668">8 (36.4)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="53630" xml_f="53637" txt_i="21669" txt_f="21676">Fatigue</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="53698" xml_f="53706" txt_i="21677" txt_f="21685">1 (16.7)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="53767" xml_f="53775" txt_i="21686" txt_f="21694">8 (36.4)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="53836" xml_f="53844" txt_i="21695" txt_f="21703">4 (18.2)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="53990" xml_f="53996" txt_i="21704" txt_f="21710">Nausea</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="54057" xml_f="54065" txt_i="21711" txt_f="21719">1 (16.7)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="54126" xml_f="54134" txt_i="21720" txt_f="21728">6 (27.3)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="54195" xml_f="54203" txt_i="21729" txt_f="21737">3 (13.6)</offsets></td></tr><tr valign="top" style="border-bottom:solid 1px #000000"><td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1"><offsets xml_i="54349" xml_f="54357" txt_i="21738" txt_f="21746">Insomnia</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="54418" xml_f="54419" txt_i="21747" txt_f="21748">0</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="54480" xml_f="54488" txt_i="21749" txt_f="21757">6 (27.3)</offsets></td><td align="center" valign="top" rowspan="1" colspan="1"><offsets xml_i="54549" xml_f="54550" txt_i="21758" txt_f="21759">0</offsets></td></tr></tbody></table><table-wrap-foot><fn id="hep29081-note-0009"><p><offsets xml_i="54624" xml_f="54911" txt_i="21760" txt_f="22047">Serious adverse events were defined as events resulting in hospitalization or prolongation of hospitalization, persistent or clinically significant disability or incapacity, or death or that were life‐threatening or required medical or surgical intervention to prevent a serious outcome.</offsets></p></fn><fn id="hep29081-note-0010"><label><offsets xml_i="54955" xml_f="54956" txt_i="22048" txt_f="22049">a</offsets></label><p><offsets xml_i="54967" xml_f="54981" txt_i="22049" txt_f="22063">Breast cancer.</offsets></p></fn><fn id="hep29081-note-0011"><label><offsets xml_i="55025" xml_f="55026" txt_i="22064" txt_f="22065">b</offsets></label><p><offsets xml_i="55037" xml_f="55053" txt_i="22065" txt_f="22081">Fractured femur.</offsets></p></fn><fn id="hep29081-note-0012"><label><offsets xml_i="55097" xml_f="55098" txt_i="22082" txt_f="22083">c</offsets></label><p><offsets xml_i="55109" xml_f="55146" txt_i="22083" txt_f="22117">Occurring in &gt;10% of all patients.</offsets></p></fn></table-wrap-foot></table-wrap></sec><sec id="hep29081-sec-0007"><title><offsets xml_i="55227" xml_f="55237" txt_i="22119" txt_f="22129">Discussion</offsets></title><p><offsets xml_i="55248" xml_f="56096" txt_i="22130" txt_f="22978">The once‐daily regimen of GLE and PIB was well tolerated with no serious adverse events related to study drug, no discontinuations due to adverse events, and no relevant laboratory abnormalities. It also resulted in high rates of SVR with or without coadministration of RBV in patients with HCV GT1 infection and prior DAA therapy experience. Overall, the DAA‐experienced population in the MAGELLAN‐1 study had broad representation of baseline NS3 and NS5A polymorphisms, including polymorphisms at key NS5A positions M28, Q30, L31, H58, and Y93 that confer resistance to earlier‐generation NS5A inhibitors. Additionally, all 8 patients previously treated with ledipasvir/sofosbuvir achieved SVR12 despite the presence of the NS5A resistance‐associated Y93H/N polymorphism in 5 patients and multiple NS5A polymorphisms in 4 patients. This confirms </offsets><italic><offsets xml_i="56104" xml_f="56112" txt_i="22978" txt_f="22986">in vitro</offsets></italic><offsets xml_i="56121" xml_f="56183" txt_i="22986" txt_f="23048"> data that variants at the Y93 position are susceptible to PIB</offsets><xref rid="hep29081-bib-0022" ref-type="ref"><offsets xml_i="56228" xml_f="56230" txt_i="23048" txt_f="23050">22</offsets></xref><offsets xml_i="56237" xml_f="56575" txt_i="23050" txt_f="23388"> and suggests GLE plus PIB is an effective treatment for those with a baseline polymorphism or treatment‐emergent substitution at this position. Furthermore, 14 of 15 (93%) patients who failed a prior dual (NS3/4A PI plus NS5A inhibitor) or triple (NS3/4A PI plus NS5A inhibitor plus NS5B polymerase inhibitor) DAA regimen achieved SVR12.</offsets></p><p><offsets xml_i="56582" xml_f="56846" txt_i="23389" txt_f="23653">The addition of RBV to the GLE + PIB regimen had no apparent impact on response as the rates of virologic failure were identical in arms B and arm C by mITT analysis. All 5 patients who modified RBV dose achieved SVR12, similar to findings with other DAA regimens.</offsets><xref rid="hep29081-bib-0025" ref-type="ref"><offsets xml_i="56891" xml_f="56893" txt_i="23653" txt_f="23655">25</offsets></xref><offsets xml_i="56900" xml_f="56902" txt_i="23655" txt_f="23657">, </offsets><xref rid="hep29081-bib-0026" ref-type="ref"><offsets xml_i="56947" xml_f="56949" txt_i="23657" txt_f="23659">26</offsets></xref><offsets xml_i="56956" xml_f="56958" txt_i="23659" txt_f="23661">, </offsets><xref rid="hep29081-bib-0027" ref-type="ref"><offsets xml_i="57003" xml_f="57005" txt_i="23661" txt_f="23663">27</offsets></xref><offsets xml_i="57012" xml_f="58475" txt_i="23663" txt_f="25126"> However, this study did not have a large enough sample size for sufficient statistical power to confirm the impact of RBV on SVR. The patient with virologic relapse in the RBV‐containing arm had prior treatment with two different therapeutic regimens: daclatasvir alone and telaprevir plus pegylated interferon with RBV. This patient had L31M and H58D in NS5A at baseline; at the time of virologic failure at posttreatment week 4, a Q30R substitution emerged in this patient in addition to L31M and H58D from baseline. The 1 other patient in this study with a baseline H58D mutation in NS5A also had baseline M28V and Q30R polymorphisms (instead of L31M) in the same target, with an additional Q80K polymorphism in NS3; and this patient achieved SVR12. The patient with virologic breakthrough was on immunosuppressive therapy for Crohn's disease and had prior ileocolectomy; it is unclear whether this contributed to virologic failure. This patient had three NS5A baseline polymorphisms (M28V, Q30R, and H58C) and NS3 baseline polymorphisms at amino acid positions Y56 and D168 that were maintained until virologic failure. While 3 other patients with baseline D168 polymorphisms achieved SVR12, this was the only patient with a baseline Y56 polymorphism. The NS3 V36M and NS5A M28G substitutions emerged in this patient after virologic failure. Both patients who had virologic failure were reported as compliant and had drug exposures similar to other patients.</offsets></p><p><offsets xml_i="58482" xml_f="59642" txt_i="25127" txt_f="26284">No clinically significant laboratory abnormalities were reported in hemoglobin levels, including in the RBV‐containing arm. Furthermore, no clinically significant chemistry values were observed, including on‐treatment elevations in total bilirubin or alanine aminotransferase &gt;3 times the upper limit of normal after alanine aminotransferase normalization or nadir. Although no serious adverse events related to study drug were reported in either arm, the addition of RBV did not reduce the rate of virologic failure but did increase the rate of adverse events. One potential limitation of this study was that RBV coadministration was 800 mg daily, regardless of patient weight, i.e., below the conventional 1,000 or 1,200 mg weight‐based RBV dosing. Because of this, it is unclear whether weight‐based RBV coadministration could have increased the efficacy in arm B. By corollary, it is also likely that a conventional weight‐based RBV dose would have resulted in a further increased rate of adverse events for these patients when compared to that seen with the lower dose of RBV in arm B as increasing side effects with higher RBV dose is well documented.</offsets><xref rid="hep29081-bib-0028" ref-type="ref"><offsets xml_i="59687" xml_f="59689" txt_i="26284" txt_f="26286">28</offsets></xref><offsets xml_i="59696" xml_f="60112" txt_i="26286" txt_f="26702"> However, regardless of comparison between treatment arms, the low rate of virologic failure in the RBV‐sparing arm suggests that the response rate is already near maximal, and the addition of RBV may not impact this. The patient population enrolled here is considered inherently difficult to cure, owing to increased prevalence of baseline resistance‐associated polymorphisms likely stemming from prior DAA failure.</offsets><xref rid="hep29081-bib-0015" ref-type="ref"><offsets xml_i="60157" xml_f="60159" txt_i="26702" txt_f="26704">15</offsets></xref><offsets xml_i="60166" xml_f="60168" txt_i="26704" txt_f="26706">, </offsets><xref rid="hep29081-bib-0029" ref-type="ref"><offsets xml_i="60213" xml_f="60215" txt_i="26706" txt_f="26708">29</offsets></xref><offsets xml_i="60222" xml_f="60224" txt_i="26708" txt_f="26710">, </offsets><xref rid="hep29081-bib-0030" ref-type="ref"><offsets xml_i="60269" xml_f="60271" txt_i="26710" txt_f="26712">30</offsets></xref><offsets xml_i="60278" xml_f="60280" txt_i="26712" txt_f="26714">, </offsets><xref rid="hep29081-bib-0031" ref-type="ref"><offsets xml_i="60325" xml_f="60327" txt_i="26714" txt_f="26716">31</offsets></xref><offsets xml_i="60334" xml_f="60336" txt_i="26716" txt_f="26718">, </offsets><xref rid="hep29081-bib-0032" ref-type="ref"><offsets xml_i="60381" xml_f="60383" txt_i="26718" txt_f="26720">32</offsets></xref><offsets xml_i="60390" xml_f="60675" txt_i="26720" txt_f="27005"> As such, additional study is required in this diverse patient population to further confirm efficacy of the regimen in GT1 patients with prior failure of DAA therapy, including patients with prior exposure to DAA‐containing therapies and concomitant cirrhosis, who were excluded here.</offsets></p><p><offsets xml_i="60682" xml_f="61427" txt_i="27006" txt_f="27751">In summary, the combination of GLE and PIB showed potent antiviral activity, regardless of the presence of one or more baseline resistance‐associated polymorphisms and irrespective of previous DAA‐containing treatment regimens, resulting in high SVR12 rates in patients without cirrhosis but with HCV GT1 infection. This suggests that the combination of GLE and PIB is highly effective in this population, which currently has limited treatment options. Based on these findings, larger and more diverse patient groups are being evaluated in phase 3 studies to confirm the safety and efficacy of the RBV‐free coformulation of GLE/PIB (300 mg/120 mg) in all six major HCV GTs, including patients with prior DAA experience and compensated cirrhosis.</offsets></p></sec><sec sec-type="supplementary-material"><title><offsets xml_i="61483" xml_f="61505" txt_i="27753" txt_f="27775">Supporting information</offsets></title><supplementary-material content-type="local-data"><p><offsets xml_i="61566" xml_f="61616" txt_i="27776" txt_f="27826">Additional Supporting Information may be found at </offsets><ext-link ext-link-type="uri" xlink:href="http://onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo"><offsets xml_i="61721" xml_f="61775" txt_i="27826" txt_f="27880">onlinelibrary.wiley.com/doi/10.1002/hep.29081/suppinfo</offsets></ext-link><offsets xml_i="61786" xml_f="61787" txt_i="27880" txt_f="27881">.</offsets></p></supplementary-material><supplementary-material content-type="local-data"><caption><p><offsets xml_i="61878" xml_f="61900" txt_i="27882" txt_f="27904">Supporting Information</offsets></p></caption><media xlink:href="HEP-66-389-s001.pdf"><caption><p><offsets xml_i="61966" xml_f="62002" txt_i="27905" txt_f="27941">Click here for additional data file.</offsets></p></caption></media></supplementary-material></sec></body><back><ack id="hep29081-sec-0008"><title>Acknowledgment</title><p>We thank AbbVie employees Wei Liu, Tami Pilot‐Matias, Christine Collins, Rakesh Tripathi, Gretja Schnell, Preethi Krishnan, Jill M. Beyer, Thomas J. Reisch, Tanya Dekhtyar, and Michelle L. Irvin for assistance provided in the design and data analysis of the MAGELLAN‐I study, as well as Douglas E. Dylla for scientific insights and critical review. AbbVie funded the study and participated in the collection, analysis, and interpretation of the data and in the writing, review, and approval of the manuscript. Medical writing was provided by Ryan J. Bourgo of AbbVie. These persons received no compensation apart from usual salary for their contributions.</p></ack><ref-list content-type="cited-references"><title>REFERENCES</title><ref id="hep29081-bib-0001"><label>1</label><mixed-citation publication-type="journal" id="hep29081-cit-0001">
<string-name>
<surname>Gower</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Estes</surname>
<given-names>C</given-names>
</string-name>, 
<string-name>
<surname>Blach</surname>
<given-names>S</given-names>
</string-name>, 
<string-name>
<surname>Razavi‐Shearer</surname>
<given-names>K</given-names>
</string-name>, 
<string-name>
<surname>Razavi</surname>
<given-names>H</given-names>
</string-name>. <article-title>Global epidemiology and genotype distribution of the hepatitis C virus infection</article-title>. <source>J Hepatol</source>
<year>2014</year>;<volume>61</volume>(<issue>1 Suppl.</issue>):<fpage>S45</fpage>‐<lpage>S57</lpage>.
<pub-id pub-id-type="pmid">25086286</pub-id></mixed-citation></ref><ref id="hep29081-bib-0002"><label>2</label><mixed-citation publication-type="journal" id="hep29081-cit-0002">
<string-name>
<surname>Messina</surname>
<given-names>JP</given-names>
</string-name>, 
<string-name>
<surname>Humphreys</surname>
<given-names>I</given-names>
</string-name>, 
<string-name>
<surname>Flaxman</surname>
<given-names>A</given-names>
</string-name>, 
<string-name>
<surname>Brown</surname>
<given-names>A</given-names>
</string-name>, 
<string-name>
<surname>Cooke</surname>
<given-names>GS</given-names>
</string-name>, 
<string-name>
<surname>Pybus</surname>
<given-names>OG</given-names>
</string-name>, et al. <article-title>Global distribution and prevalence of hepatitis C virus genotypes</article-title>. <source>Hepatology</source>
<year>2015</year>;<volume>61</volume>:<fpage>77</fpage>‐<lpage>87</lpage>.
<pub-id pub-id-type="pmid">25069599</pub-id></mixed-citation></ref><ref id="hep29081-bib-0003"><label>3</label><mixed-citation publication-type="journal" id="hep29081-cit-0003">
<string-name>
<surname>Asselah</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>Boyer</surname>
<given-names>N</given-names>
</string-name>, 
<string-name>
<surname>Saadoun</surname>
<given-names>D</given-names>
</string-name>, 
<string-name>
<surname>Martinot‐Peignoux</surname>
<given-names>M</given-names>
</string-name>, 
<string-name>
<surname>Marcellin</surname>
<given-names>P</given-names>
</string-name>. <article-title>Direct‐acting antivirals for the treatment of hepatitis C virus infection: optimizing current IFN‐free treatment and future perspectives</article-title>. <source>Liver Int</source>
<year>2016</year>;<volume>36</volume>(<issue>Suppl. 1</issue>):<fpage>47</fpage>‐<lpage>57</lpage>.
<pub-id pub-id-type="pmid">26725897</pub-id></mixed-citation></ref><ref id="hep29081-bib-0004"><label>4</label><mixed-citation publication-type="journal" id="hep29081-cit-0004">
<string-name>
<surname>Pawlotsky</surname>
<given-names>JM</given-names>
</string-name>. <article-title>New hepatitis C therapies: the toolbox, strategies, and challenges</article-title>. <source>Gastroenterology</source>
<year>2014</year>;<volume>146</volume>:<fpage>1176</fpage>‐<lpage>1192</lpage>.
<pub-id pub-id-type="pmid">24631495</pub-id></mixed-citation></ref><ref id="hep29081-bib-0005"><label>5</label><mixed-citation publication-type="journal" id="hep29081-cit-0005">
<string-name>
<surname>Shah</surname>
<given-names>ND</given-names>
</string-name>, 
<string-name>
<surname>Fried</surname>
<given-names>MW</given-names>
</string-name>. <article-title>Treatment options of patients with chronic hepatitis C who have failed prior therapy</article-title>. <source>Clin Liver Dis</source>
<year>2016</year>;<volume>7</volume>:<fpage>40</fpage>‐<lpage>44</lpage>.
</mixed-citation></ref><ref id="hep29081-bib-0006"><label>6</label><mixed-citation publication-type="journal" id="hep29081-cit-0006">
<string-name>
<surname>Ahmed</surname>
<given-names>A</given-names>
</string-name>, 
<string-name>
<surname>Felmlee</surname>
<given-names>DJ</given-names>
</string-name>. <article-title>Mechanisms of hepatitis C viral resistance to direct acting antivirals</article-title>. <source>Viruses</source>
<year>2015</year>;<volume>7</volume>:<fpage>6716</fpage>‐<lpage>6729</lpage>.
<pub-id pub-id-type="pmid">26694454</pub-id></mixed-citation></ref><ref id="hep29081-bib-0007"><label>7</label><mixed-citation publication-type="journal" id="hep29081-cit-0007">
<string-name>
<surname>Horner</surname>
<given-names>SM</given-names>
</string-name>, 
<string-name>
<surname>Naggie</surname>
<given-names>S</given-names>
</string-name>. <article-title>Successes and challenges on the road to cure hepatitis C</article-title>. <source>PLoS Pathog</source>
<year>2015</year>;<volume>11</volume>:<fpage>e1004854</fpage>.
<pub-id pub-id-type="pmid">26087286</pub-id></mixed-citation></ref><ref id="hep29081-bib-0008"><label>8</label><mixed-citation publication-type="journal" id="hep29081-cit-0008">
<string-name>
<surname>Poveda</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Wyles</surname>
<given-names>DL</given-names>
</string-name>, 
<string-name>
<surname>Mena</surname>
<given-names>Á</given-names>
</string-name>, 
<string-name>
<surname>Pedreira</surname>
<given-names>JD</given-names>
</string-name>, 
<string-name>
<surname>Castro‐Iglesias</surname>
<given-names>Á</given-names>
</string-name>, 
<string-name>
<surname>Cachay</surname>
<given-names>E</given-names>
</string-name>. <article-title>Update on hepatitis C virus resistance to direct‐acting antiviral agents</article-title>. <source>Antiviral Res</source>
<year>2014</year>;<volume>108</volume>:<fpage>181</fpage>‐<lpage>191</lpage>.
<pub-id pub-id-type="pmid">24911972</pub-id></mixed-citation></ref><ref id="hep29081-bib-0009"><label>9</label><mixed-citation publication-type="journal" id="hep29081-cit-0009">
<string-name>
<surname>Jacobson</surname>
<given-names>IM</given-names>
</string-name>, 
<string-name>
<surname>Asante‐Appiah</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Wong</surname>
<given-names>P</given-names>
</string-name>, 
<string-name>
<surname>Black</surname>
<given-names>TA</given-names>
</string-name>, 
<string-name>
<surname>Howe</surname>
<given-names>AY</given-names>
</string-name>, 
<string-name>
<surname>Wahl</surname>
<given-names>J</given-names>
</string-name>, et al. <article-title>Prevalence and impact of baseline NS5A resistance‐associated variants (RAVs) on the efficacy of elbasvir/grazoprevir (EBR/GZR) against GT1a infection</article-title>. <source>Hepatology</source>
<year>2015</year>;<volume>62</volume>:<fpage>1393</fpage>‐<lpage>1394A</lpage>.
</mixed-citation></ref><ref id="hep29081-bib-0010"><label>10</label><mixed-citation publication-type="journal" id="hep29081-cit-0010">
<string-name>
<surname>Sarrazin</surname>
<given-names>C</given-names>
</string-name>, 
<string-name>
<surname>Dvory‐Sobol</surname>
<given-names>H</given-names>
</string-name>, 
<string-name>
<surname>Svarovskaia</surname>
<given-names>ES</given-names>
</string-name>, 
<string-name>
<surname>Doehle</surname>
<given-names>B</given-names>
</string-name>, 
<string-name>
<surname>Martin</surname>
<given-names>R</given-names>
</string-name>, 
<string-name>
<surname>Zeuzem</surname>
<given-names>S</given-names>
</string-name>, et al. <article-title>The prevalence and the effect of HCV NS5A resistance associated variants in subjects with compensated cirrhosis treated with ledipasvir/sofosbuvir ± RBV</article-title>. <source>J Hepatol</source>
<year>2015</year>;<volume>62</volume>:<fpage>S620</fpage>.
</mixed-citation></ref><ref id="hep29081-bib-0011"><label>11</label><mixed-citation publication-type="journal" id="hep29081-cit-0011">
<string-name>
<surname>Dvory‐Sobol</surname>
<given-names>H</given-names>
</string-name>, 
<string-name>
<surname>Wyles</surname>
<given-names>D</given-names>
</string-name>, 
<string-name>
<surname>Ouyang</surname>
<given-names>W</given-names>
</string-name>, 
<string-name>
<surname>Chodavarapu</surname>
<given-names>K</given-names>
</string-name>, 
<string-name>
<surname>McNally</surname>
<given-names>J</given-names>
</string-name>, 
<string-name>
<surname>Cheng</surname>
<given-names>W</given-names>
</string-name>, et al. <article-title>Long‐term persistance of HCV NS5A variants after treatment with NS5A inhibitor ledipasvir</article-title>. <source>J Hepatol</source>
<year>2015</year>;<volume>62</volume>:<fpage>S221</fpage>.
</mixed-citation></ref><ref id="hep29081-bib-0012"><label>12</label><mixed-citation publication-type="journal" id="hep29081-cit-0012">
<string-name>
<surname>Krishnan</surname>
<given-names>P</given-names>
</string-name>, 
<string-name>
<surname>Tripathi</surname>
<given-names>R</given-names>
</string-name>, 
<string-name>
<surname>Schnell</surname>
<given-names>G</given-names>
</string-name>, 
<string-name>
<surname>Reisch</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>Beyer</surname>
<given-names>J</given-names>
</string-name>, 
<string-name>
<surname>Dekhtyar</surname>
<given-names>T</given-names>
</string-name>, et al. <article-title>Long‐term follow‐up of treatment‐emergent resistance‐associated variants in NS3, NS5A, and NS5B with paritaprevir/r‐, ombitasvir‐, and dasabuvir‐based regimens</article-title>. <source>J Hepatol</source>
<year>2015</year>;<volume>62</volume>:<fpage>S220</fpage>.
</mixed-citation></ref><ref id="hep29081-bib-0013"><label>13</label><mixed-citation publication-type="journal" id="hep29081-cit-0013">
<string-name>
<surname>Zeuzem</surname>
<given-names>S</given-names>
</string-name>, 
<string-name>
<surname>Ghalib</surname>
<given-names>R</given-names>
</string-name>, 
<string-name>
<surname>Reddy</surname>
<given-names>KR</given-names>
</string-name>, 
<string-name>
<surname>Pockros</surname>
<given-names>PJ</given-names>
</string-name>, 
<string-name>
<surname>Ben Ari</surname>
<given-names>Z</given-names>
</string-name>, 
<string-name>
<surname>Zhao</surname>
<given-names>Y</given-names>
</string-name>, et al. <article-title>Grazoprevir‐elbasvir combination therapy for treatment‐naive cirrhotic and noncirrhotic patients with chronic HCV genotype 1, 4, or 6 infection: a randomized trial</article-title>. <source>Ann Intern Med</source>
<year>2015</year>;<volume>163</volume>:<fpage>1</fpage>‐<lpage>13</lpage>.
<pub-id pub-id-type="pmid">25909356</pub-id></mixed-citation></ref><ref id="hep29081-bib-0014"><label>14</label><mixed-citation publication-type="journal" id="hep29081-cit-0014">
<string-name>
<surname>Lawitz</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Flamm</surname>
<given-names>S</given-names>
</string-name>, 
<string-name>
<surname>Yang</surname>
<given-names>JC</given-names>
</string-name>, 
<string-name>
<surname>Pang</surname>
<given-names>PS</given-names>
</string-name>, 
<string-name>
<surname>Zhu</surname>
<given-names>Y</given-names>
</string-name>, 
<string-name>
<surname>Svarovskaia</surname>
<given-names>E</given-names>
</string-name>, et al. <article-title>Retreatment of patients who failed 8 or 12 weeks of ledipasvir/sofosbuvir‐based regimens with ledipasvir/sofosbuvir for 24 weeks</article-title>. <source>J Hepatol</source>
<year>2015</year>;<volume>62</volume>:<fpage>S192</fpage>.
</mixed-citation></ref><ref id="hep29081-bib-0015"><label>15</label><mixed-citation publication-type="book" id="hep29081-cit-0015">
<source>ZEPATIER (elbasvir and grazoprevir) tablets, for oral use [package insert]</source>. Revised February 2017. <publisher-loc>Whitehouse Station, NJ</publisher-loc>: <publisher-name>Merck &amp; Co</publisher-name>.
</mixed-citation></ref><ref id="hep29081-bib-0016"><label>16</label><mixed-citation publication-type="miscellaneous" id="hep29081-cit-0016">
<collab collab-type="authors">AASLD‐IDSA</collab>
. <article-title>Recommendations for testing, managing, and treating hepatitis C</article-title>. <year>2016</year> Available at <ext-link ext-link-type="uri" xlink:href="http://www.hcvguidelines.org">http://www.hcvguidelines.org</ext-link>. Accessed December 8, 2016.
</mixed-citation></ref><ref id="hep29081-bib-0017"><label>17</label><mixed-citation publication-type="journal" id="hep29081-cit-0017">
<string-name>
<surname>Buti</surname>
<given-names>M</given-names>
</string-name>, 
<string-name>
<surname>Gordon</surname>
<given-names>SC</given-names>
</string-name>, 
<string-name>
<surname>Zuckerman</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Lawitz</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Calleja</surname>
<given-names>JL</given-names>
</string-name>, 
<string-name>
<surname>Hofer</surname>
<given-names>H</given-names>
</string-name>, et al. <article-title>Grazoprevir, elbasvir, and ribavirin for chronic hepatitis C virus genotype 1 infection after failure of pegylated interferon and ribavirin with an earlier‐generation protease inhibitor: final 24‐week results from C‐SALVAGE</article-title>. <source>Clin Infect Dis</source>
<year>2016</year>;<volume>62</volume>:<fpage>32</fpage>‐<lpage>36</lpage>.
<pub-id pub-id-type="pmid">26371152</pub-id></mixed-citation></ref><ref id="hep29081-bib-0018"><label>18</label><mixed-citation publication-type="journal" id="hep29081-cit-0018">
<string-name>
<surname>Gane</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Shiffman</surname>
<given-names>ML</given-names>
</string-name>, 
<string-name>
<surname>Etzkorn</surname>
<given-names>K</given-names>
</string-name>, 
<string-name>
<surname>Morelli</surname>
<given-names>G</given-names>
</string-name>, 
<string-name>
<surname>Stedman</surname>
<given-names>CA</given-names>
</string-name>, 
<string-name>
<surname>Davis</surname>
<given-names>MN</given-names>
</string-name>, et al. <article-title>Sofosbuvir/velpatasvir in combination with ribavirin for 24 weeks is effective retreatment for patients who failed prior NS5A containing DAA regimens: results of the GS‐US‐342–1553 study</article-title>. <source>J Hepatol</source>
<year>2016</year>;<volume>64</volume>(<issue>Suppl.</issue>):<fpage>S147</fpage>‐<lpage>S148</lpage>.
</mixed-citation></ref><ref id="hep29081-bib-0019"><label>19</label><mixed-citation publication-type="book" id="hep29081-cit-0019">
<string-name>
<surname>Lawitz</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Poordad</surname>
<given-names>F</given-names>
</string-name>, 
<string-name>
<surname>Gutierrez</surname>
<given-names>JA</given-names>
</string-name>, 
<string-name>
<surname>Wells</surname>
<given-names>JT</given-names>
</string-name>, 
<string-name>
<surname>Landaverde</surname>
<given-names>CE</given-names>
</string-name>, 
<string-name>
<surname>Reiling</surname>
<given-names>JR</given-names>
</string-name>, et al. <chapter-title>C‐SWIFT retreatment (part B): 12 weeks of elbasvir/grazoprevir with sofosbuvir and ribavirin successfully treated G1‐infected subjects who failed short‐duration all‐oral therapy</chapter-title> Presented at: <source>The Liver Meeting (AASLD) 2015</source>. November 13‐17, <year>2015</year>; <publisher-loc>San Francisco, CA, USA</publisher-loc>.
</mixed-citation></ref><ref id="hep29081-bib-0020"><label>20</label><mixed-citation publication-type="book" id="hep29081-cit-0020">
<string-name>
<surname>Poordad</surname>
<given-names>F</given-names>
</string-name>, 
<string-name>
<surname>Bennett</surname>
<given-names>M</given-names>
</string-name>, 
<string-name>
<surname>Sepe</surname>
<given-names>TE</given-names>
</string-name>, 
<string-name>
<surname>Cohen</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Reindollar</surname>
<given-names>RW</given-names>
</string-name>, 
<string-name>
<surname>Everson</surname>
<given-names>GT</given-names>
</string-name>, et al. <chapter-title>QUARTZ‐I: retreatment of HCV genotype 1 DAA‐failures with ombitasvir/paritaprevir/r, dasabuvir, and sofosbuvir</chapter-title> Presented at: <source>The Liver Meeting (AASLD) 2015</source>. November 13‐17, <year>2015</year>; <publisher-loc>San Francisco, CA, USA</publisher-loc>.
</mixed-citation></ref><ref id="hep29081-bib-0021"><label>21</label><mixed-citation publication-type="book" id="hep29081-cit-0021">
<string-name>
<surname>Ng</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>Reisch</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>Middleton</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>McDaniel</surname>
<given-names>K</given-names>
</string-name>, 
<string-name>
<surname>Kempf</surname>
<given-names>D</given-names>
</string-name>, 
<string-name>
<surname>Lu</surname>
<given-names>L</given-names>
</string-name>, et al. <chapter-title>ABT‐493, a potent HCV NS3/4A protease inhibitor with broad genotype coverage</chapter-title>
<source>Poster #636 at: Twenty‐first Annual Conference on Retroviruses and Opportunistic Infections (CROI)</source>, March 3‐6, <year>2014</year>; <publisher-loc>Boston, MA, USA</publisher-loc>.
</mixed-citation></ref><ref id="hep29081-bib-0022"><label>22</label><mixed-citation publication-type="journal" id="hep29081-cit-0022">
<string-name>
<surname>Ng</surname>
<given-names>TI</given-names>
</string-name>, 
<string-name>
<surname>Krishnan</surname>
<given-names>P</given-names>
</string-name>, 
<string-name>
<surname>Pilot‐Matias</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>Kati</surname>
<given-names>W</given-names>
</string-name>, 
<string-name>
<surname>Schnell</surname>
<given-names>G</given-names>
</string-name>, 
<string-name>
<surname>Beyer</surname>
<given-names>J</given-names>
</string-name>, et al. <article-title>In vitro antiviral activity and resistance profile of the next generation hepatitis C virus NS5A inhibitor pibrentasvir</article-title>. <source>Antimicrob Agents Chemother</source>
<year>2017</year> Feb 13. pii: AAC.02558‐16. doi: 10.1128/AAC.02558‐16. [Epub ahead of print]
</mixed-citation></ref><ref id="hep29081-bib-0023"><label>23</label><mixed-citation publication-type="journal" id="hep29081-cit-0023">
<string-name>
<surname>Gao</surname>
<given-names>M</given-names>
</string-name>. <article-title>Antiviral activity and resistance of HCV NS5A replication complex inhibitors</article-title>. <source>Curr Opin Virol</source>
<year>2013</year>;<volume>3</volume>:<fpage>514</fpage>‐<lpage>520</lpage>.
<pub-id pub-id-type="pmid">23896281</pub-id></mixed-citation></ref><ref id="hep29081-bib-0024"><label>24</label><mixed-citation publication-type="journal" id="hep29081-cit-0024">
<string-name>
<surname>Gane</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Poordad</surname>
<given-names>F</given-names>
</string-name>, 
<string-name>
<surname>Wang</surname>
<given-names>S</given-names>
</string-name>, 
<string-name>
<surname>Asatryan</surname>
<given-names>A</given-names>
</string-name>, 
<string-name>
<surname>Kwo</surname>
<given-names>PY</given-names>
</string-name>, 
<string-name>
<surname>Lalezari</surname>
<given-names>J</given-names>
</string-name>, et al. <article-title>High efficacy of ABT‐493 and ABT‐530 in patients with HCV genotype 1 or 3 infection and compensated cirrhosis</article-title>. <source>Gastroenterology</source>
<year>2016</year>;<volume>151</volume>:<fpage>651</fpage>‐<lpage>659</lpage>.
<pub-id pub-id-type="pmid">27456384</pub-id></mixed-citation></ref><ref id="hep29081-bib-0025"><label>25</label><mixed-citation publication-type="journal" id="hep29081-cit-0025">
<string-name>
<surname>Ferenci</surname>
<given-names>P</given-names>
</string-name>, 
<string-name>
<surname>Bernstein</surname>
<given-names>D</given-names>
</string-name>, 
<string-name>
<surname>Lalezari</surname>
<given-names>J</given-names>
</string-name>, 
<string-name>
<surname>Cohen</surname>
<given-names>D</given-names>
</string-name>, 
<string-name>
<surname>Luo</surname>
<given-names>Y</given-names>
</string-name>, 
<string-name>
<surname>Cooper</surname>
<given-names>C</given-names>
</string-name>, et al. <article-title>ABT‐450/r‐Ombitasvir and Dasabuvir with or without Ribavirin for HCV</article-title>. <source>N Engl J Med</source>
<year>2014</year>; <volume>370</volume>:<fpage>1983</fpage>‐<lpage>1992</lpage>.
<pub-id pub-id-type="pmid">24795200</pub-id></mixed-citation></ref><ref id="hep29081-bib-0026"><label>26</label><mixed-citation publication-type="book" id="hep29081-cit-0026">
<string-name>
<surname>Hassanein</surname>
<given-names>T</given-names>
</string-name>, 
<string-name>
<surname>Vierling</surname>
<given-names>JM</given-names>
</string-name>, 
<string-name>
<surname>Reddy</surname>
<given-names>KR</given-names>
</string-name>, 
<string-name>
<surname>Cohen</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Morelli</surname>
<given-names>G</given-names>
</string-name>, 
<string-name>
<surname>Mantry</surname>
<given-names>PS</given-names>
</string-name>, et al. <chapter-title>RUBY‐I, cohort 2: treatment of HCV genotype 1 infection in patients with severe or end‐stage renal disease, including patients with cirrhosis</chapter-title>
<source>Presented at Asian Pacific Association for the Study of the Liver (APASL) Meeting 2016</source>. June 12, <year>2016</year>; <publisher-loc>Kaohsiung, Taiwan</publisher-loc>.
</mixed-citation></ref><ref id="hep29081-bib-0027"><label>27</label><mixed-citation publication-type="journal" id="hep29081-cit-0027">
<string-name>
<surname>Poordad</surname>
<given-names>F</given-names>
</string-name>, 
<string-name>
<surname>Hezode</surname>
<given-names>C</given-names>
</string-name>, 
<string-name>
<surname>Trinh</surname>
<given-names>R</given-names>
</string-name>, 
<string-name>
<surname>Kowdley</surname>
<given-names>KV</given-names>
</string-name>, 
<string-name>
<surname>Zeuzem</surname>
<given-names>S</given-names>
</string-name>, 
<string-name>
<surname>Agarwal</surname>
<given-names>K</given-names>
</string-name>, et al. <article-title>ABT‐450/r‐ombitasvir and dasabuvir with ribavirin for hepatitis C with cirrhosis</article-title>. <source>N Engl J Med</source>
<year>2014</year>;<volume>370</volume>:<fpage>1973</fpage>‐<lpage>1982</lpage>.
<pub-id pub-id-type="pmid">24725237</pub-id></mixed-citation></ref><ref id="hep29081-bib-0028"><label>28</label><mixed-citation publication-type="journal" id="hep29081-cit-0028">
<string-name>
<surname>Lindahl</surname>
<given-names>K</given-names>
</string-name>, 
<string-name>
<surname>Stahle</surname>
<given-names>L</given-names>
</string-name>, 
<string-name>
<surname>Bruchfeld</surname>
<given-names>A</given-names>
</string-name>, 
<string-name>
<surname>Schvarcz</surname>
<given-names>R</given-names>
</string-name>. <article-title>High‐dose ribavirin in combination with standard dose peginterferon for treatment of patients with chronic hepatitis C</article-title>. <source>Hepatology</source>
<year>2005</year>;<volume>41</volume>:<fpage>275</fpage>‐<lpage>279</lpage>.
<pub-id pub-id-type="pmid">15660393</pub-id></mixed-citation></ref><ref id="hep29081-bib-0029"><label>29</label><mixed-citation publication-type="book" id="hep29081-cit-0029">
<source>DAKLINZA (daclatasvir) tablets [package insert]</source>. Revised February 2017. <publisher-loc>Princeton, NJ</publisher-loc>: <publisher-name>Bristol‐Meyers Squibb</publisher-name>.
</mixed-citation></ref><ref id="hep29081-bib-0030"><label>30</label><mixed-citation publication-type="journal" id="hep29081-cit-0030">
<string-name>
<surname>Fried</surname>
<given-names>MW</given-names>
</string-name>, 
<string-name>
<surname>Buti</surname>
<given-names>M</given-names>
</string-name>, 
<string-name>
<surname>Dore</surname>
<given-names>GJ</given-names>
</string-name>, 
<string-name>
<surname>Flisiak</surname>
<given-names>R</given-names>
</string-name>, 
<string-name>
<surname>Ferenci</surname>
<given-names>P</given-names>
</string-name>, 
<string-name>
<surname>Jacobson</surname>
<given-names>I</given-names>
</string-name>, et al. <article-title>Once‐daily simeprevir (TMC435) with pegylated interferon and ribavirin in treatment‐naive genotype 1 hepatitis C: the randomized PILLAR study</article-title>. <source>Hepatology</source>
<year>2013</year>;<volume>58</volume>:<fpage>1918</fpage>‐<lpage>1929</lpage>.
<pub-id pub-id-type="pmid">23907700</pub-id></mixed-citation></ref><ref id="hep29081-bib-0031"><label>31</label><mixed-citation publication-type="book" id="hep29081-cit-0031">
<string-name>
<surname>Wang</surname>
<given-names>G</given-names>
</string-name>, 
<string-name>
<surname>Reeves</surname>
<given-names>J</given-names>
</string-name>, 
<string-name>
<surname>Terrault</surname>
<given-names>N</given-names>
</string-name>, 
<string-name>
<surname>Lim</surname>
<given-names>J</given-names>
</string-name>, 
<string-name>
<surname>Morelli</surname>
<given-names>G</given-names>
</string-name>, 
<string-name>
<surname>Kuo</surname>
<given-names>A</given-names>
</string-name>, et al. <chapter-title>Prevalence and impact of baseline resistance‐associated variants (RAVs) on the efficacy of ledipasvir/sofosbuvir or simeprevir/sofosbuvir against GT1 HCV infection: HCV‐TARGET interim analysis</chapter-title>
<source>Presented at: The International Liver Congress (EASL)</source>
<year>2016</year> April 14‐17; <publisher-loc>Barcelona, Spain</publisher-loc>.
</mixed-citation></ref><ref id="hep29081-bib-0032"><label>32</label><mixed-citation publication-type="journal" id="hep29081-cit-0032">
<string-name>
<surname>Zeuzem</surname>
<given-names>S</given-names>
</string-name>, 
<string-name>
<surname>Rockstroh</surname>
<given-names>JK</given-names>
</string-name>, 
<string-name>
<surname>Kwo</surname>
<given-names>PY</given-names>
</string-name>, 
<string-name>
<surname>Roth</surname>
<given-names>D</given-names>
</string-name>, 
<string-name>
<surname>Lawitz</surname>
<given-names>E</given-names>
</string-name>, 
<string-name>
<surname>Sulkowski</surname>
<given-names>MS</given-names>
</string-name>, et al. <article-title>Predictors of response to grazoprevir/elbasvir among HCV genotype 1 (GT1)–infected patients: integrated analysis of phase 2‐3 trials</article-title>. <source>Hepatology</source>
<year>2015</year>;<volume>62</volume>:<fpage>554A</fpage>.
</mixed-citation></ref></ref-list></back></article>