<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Parasit Vectors</journal-id><journal-id journal-id-type="iso-abbrev">Parasit Vectors</journal-id><journal-title-group><journal-title>Parasites &amp; Vectors</journal-title></journal-title-group><issn pub-type="epub">1756-3305</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">29530088</article-id><article-id pub-id-type="pmc">5848565</article-id><article-id pub-id-type="publisher-id">2739</article-id><article-id pub-id-type="doi">10.1186/s13071-018-2739-2</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Investigations on the interplays between <italic>Schistosoma mansoni</italic>, praziquantel and the gut microbiome</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Schneeberger</surname><given-names>Pierre H. H.</given-names></name><address><email>pierre.schneeberger@swisstph.ch</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Coulibaly</surname><given-names>Jean T.</given-names></name><address><email>couljeanvae@yahoo.fr</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff4">4</xref><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Panic</surname><given-names>Gordana</given-names></name><address><email>gordana.panic@swisstph.ch</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Daubenberger</surname><given-names>Claudia</given-names></name><address><email>claudia.daubenberger@swisstph.ch</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Gueuning</surname><given-names>Morgan</given-names></name><address><email>morgan.gueuning@agroscope.admin.ch</email></address><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Frey</surname><given-names>Jürg E.</given-names></name><address><email>juerg.frey@agroscope.admin.ch</email></address><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Keiser</surname><given-names>Jennifer</given-names></name><address><email>jennifer.keiser@swisstph.ch</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 0587 0574</institution-id><institution-id institution-id-type="GRID">grid.416786.a</institution-id><institution>Department of Medical Parasitology and Infection Biology, </institution><institution>Swiss Tropical and Public Health Institute, </institution></institution-wrap>Basel, Switzerland </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 1937 0642</institution-id><institution-id institution-id-type="GRID">grid.6612.3</institution-id><institution>University of Basel, </institution></institution-wrap>Basel, Switzerland </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 4681 910X</institution-id><institution-id institution-id-type="GRID">grid.417771.3</institution-id><institution>Department of Methods Development and Analytics, </institution><institution>Agroscope, </institution></institution-wrap>Wädenswil, Switzerland </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2176 6353</institution-id><institution-id institution-id-type="GRID">grid.410694.e</institution-id><institution>Unité de Formation et de Recherche Biosciences, Université Felix Houphouët-Boigny, </institution></institution-wrap>Abidjan, Côte d’Ivoire </aff><aff id="Aff5"><label>5</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 0697 1172</institution-id><institution-id institution-id-type="GRID">grid.462846.a</institution-id><institution>Centre Suisse de Recherches Scientifiques en Côte d’Ivoire, </institution></institution-wrap>Abidjan, Côte d’Ivoire </aff></contrib-group><pub-date pub-type="epub"><day>12</day><month>3</month><year>2018</year></pub-date><pub-date pub-type="pmc-release"><day>12</day><month>3</month><year>2018</year></pub-date><pub-date pub-type="collection"><year>2018</year></pub-date><volume>11</volume><elocation-id>168</elocation-id><history><date date-type="received"><day>5</day><month>12</month><year>2017</year></date><date date-type="accepted"><day>22</day><month>2</month><year>2018</year></date></history><permissions><copyright-statement>© The Author(s). 2018</copyright-statement><license license-type="OpenAccess"><license-p><bold>Open Access</bold>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</ext-link>) applies to the data made available in this article, unless otherwise stated.</license-p></license></permissions><abstract id="Abs1"><sec><title><offsets xml_i="6129" xml_f="6139" txt_i="11" txt_f="21">Background</offsets></title><p id="Par1"><offsets xml_i="6160" xml_f="6688" txt_i="22" txt_f="550">Schistosomiasis is a neglected tropical disease burdening millions of people. One drug, praziquantel, is currently used for treatment and control. Clinically relevant drug resistance has not yet been described, but there is considerable heterogeneity in treatment outcomes, ranging from cure to only moderate egg reduction rates. The objectives of this study are to investigate potential worm-induced dysbacteriosis of the gut microbiota and to assess whether a specific microbiome profile could influence praziquantel response.</offsets></p></sec><sec><title><offsets xml_i="6710" xml_f="6717" txt_i="552" txt_f="559">Methods</offsets></title><p id="Par2"><offsets xml_i="6738" xml_f="6765" txt_i="560" txt_f="587">Using V3 and V4 regions of </offsets><italic><offsets xml_i="6773" xml_f="6776" txt_i="587" txt_f="590">16S</offsets></italic><offsets xml_i="6785" xml_f="6835" txt_i="590" txt_f="640"> rRNA genes, we screened the gut microbiota of 34 </offsets><italic><offsets xml_i="6843" xml_f="6862" txt_i="640" txt_f="659">Schistosoma mansoni</offsets></italic><offsets xml_i="6871" xml_f="7082" txt_i="659" txt_f="870"> infected and uninfected children from Côte d’Ivoire. From each infected child one pre-treatment, one 24-hour and one 21-day follow-up sample after administering 60 mg/kg praziquantel or placebo, were collected.</offsets></p></sec><sec><title><offsets xml_i="7104" xml_f="7111" txt_i="872" txt_f="879">Results</offsets></title><p id="Par3"><offsets xml_i="7132" xml_f="7313" txt_i="880" txt_f="1061">Overall taxonomic profiling and diversity indicators were found to be close to a “healthy” gut structure in all children. Slight overall compositional changes were observed between </offsets><italic><offsets xml_i="7321" xml_f="7331" txt_i="1061" txt_f="1071">S. mansoni</offsets></italic><offsets xml_i="7340" xml_f="7570" txt_i="1071" txt_f="1301">-infected and non-infected children. Praziquantel treatment was not linked to a major shift in the gut taxonomic profiles, thus reinforcing the good safety profile of the drug by ruling out off-targets effects on the gut microbes.</offsets><italic><offsets xml_i="7578" xml_f="7581" txt_i="1301" txt_f="1304">16S</offsets></italic><offsets xml_i="7590" xml_f="7608" txt_i="1304" txt_f="1322"> rRNA gene of the </offsets><italic><offsets xml_i="7616" xml_f="7631" txt_i="1322" txt_f="1337">Fusobacteriales</offsets></italic><offsets xml_i="7640" xml_f="7796" txt_i="1337" txt_f="1493"> order was significantly more abundant in cured individuals, both at baseline and 24 hours post-treatment. A real-time qPCR confirmed the over-abundance of </offsets><italic><offsets xml_i="7804" xml_f="7817" txt_i="1493" txt_f="1506">Fusobacterium</offsets></italic><offsets xml_i="7826" xml_f="7851" txt_i="1506" txt_f="1531"> spp. in cured children. </offsets><italic><offsets xml_i="7859" xml_f="7872" txt_i="1531" txt_f="1544">Fusobacterium</offsets></italic><offsets xml_i="7881" xml_f="7945" txt_i="1544" txt_f="1608"> spp. abundance could also be correlated with treatment induced </offsets><italic><offsets xml_i="7953" xml_f="7963" txt_i="1608" txt_f="1618">S. mansoni</offsets></italic><offsets xml_i="7972" xml_f="7987" txt_i="1618" txt_f="1633"> egg-reduction.</offsets></p></sec><sec><title><offsets xml_i="8009" xml_f="8020" txt_i="1635" txt_f="1646">Conclusions</offsets></title><p id="Par4"><offsets xml_i="8041" xml_f="8075" txt_i="1647" txt_f="1681">Our study suggests that neither a </offsets><italic><offsets xml_i="8083" xml_f="8093" txt_i="1681" txt_f="1691">S. mansoni</offsets></italic><offsets xml_i="8102" xml_f="8235" txt_i="1691" txt_f="1824"> infection nor praziquantel administration triggers a significant effect on the microbial composition and that a higher abundance of </offsets><italic><offsets xml_i="8243" xml_f="8256" txt_i="1824" txt_f="1837">Fusobacterium</offsets></italic><offsets xml_i="8265" xml_f="8361" txt_i="1837" txt_f="1933"> spp., before treatment, is associated with higher efficacy of praziquantel in the treatment of </offsets><italic><offsets xml_i="8369" xml_f="8379" txt_i="1933" txt_f="1943">S. mansoni</offsets></italic><offsets xml_i="8388" xml_f="8400" txt_i="1943" txt_f="1955"> infections.</offsets></p></sec><sec><title><offsets xml_i="8422" xml_f="8440" txt_i="1957" txt_f="1975">Trial registration</offsets></title><p id="Par5"><offsets xml_i="8461" xml_f="8520" txt_i="1976" txt_f="2035">International Standard Randomised Controlled Trial, number </offsets><ext-link ext-link-type="uri" xlink:href="http://www.isrctn.com/ISRCTN15280205"><offsets xml_i="8600" xml_f="8614" txt_i="2035" txt_f="2049">ISRCTN15280205</offsets></ext-link><offsets xml_i="8625" xml_f="8626" txt_i="2049" txt_f="2050">.</offsets></p></sec><sec><title><offsets xml_i="8648" xml_f="8681" txt_i="2052" txt_f="2085">Electronic supplementary material</offsets></title><p><offsets xml_i="8692" xml_f="8827" txt_i="2086" txt_f="2221">The online version of this article (10.1186/s13071-018-2739-2) contains supplementary material, which is available to authorized users.</offsets></p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd><italic>Schistosoma mansoni</italic></kwd><kwd>Praziquantel</kwd><kwd>Gut microbiome</kwd><kwd>Infectious disease</kwd><kwd>Microbiome-parasite interaction</kwd><kwd>Microbiome-drug interaction</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/100010663</institution-id><institution>H2020 European Research Council</institution></institution-wrap></funding-source><award-id>614739</award-id><principal-award-recipient><name><surname>Keiser</surname><given-names>Jennifer</given-names></name></principal-award-recipient></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2018</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title><offsets xml_i="9762" xml_f="9772" txt_i="2230" txt_f="2240">Background</offsets></title><p id="Par6"><offsets xml_i="9793" xml_f="9900" txt_i="2241" txt_f="2348">Schistosomiasis belongs to the group of neglected tropical diseases caused by parasitic worms of the genus </offsets><italic><offsets xml_i="9908" xml_f="9919" txt_i="2348" txt_f="2359">Schistosoma</offsets></italic><offsets xml_i="9928" xml_f="9930" txt_i="2359" txt_f="2361"> [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="9962" xml_f="9963" txt_i="2361" txt_f="2362">1</offsets></xref><offsets xml_i="9970" xml_f="10097" txt_i="2362" txt_f="2489">]. Schistosomes have accompanied mankind for thousands of years and - still today - they are pervasive where poverty prevails [</offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="10129" xml_f="10130" txt_i="2489" txt_f="2490">2</offsets></xref><offsets xml_i="10137" xml_f="10139" txt_i="2490" txt_f="2492">, </offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="10171" xml_f="10172" txt_i="2492" txt_f="2493">3</offsets></xref><offsets xml_i="10179" xml_f="10278" txt_i="2493" txt_f="2592">]. Recent estimates suggest that approximately 200 million people are affected by schistosomiasis [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="10310" xml_f="10311" txt_i="2592" txt_f="2593">1</offsets></xref><offsets xml_i="10318" xml_f="10464" txt_i="2593" txt_f="2739">]. Left untreated, the disease becomes chronic and debilitating and is therefore known as one of the diseases that perpetuate the “poverty trap” [</offsets><xref ref-type="bibr" rid="CR4"><offsets xml_i="10496" xml_f="10497" txt_i="2739" txt_f="2740">4</offsets></xref><offsets xml_i="10504" xml_f="10506" txt_i="2740" txt_f="2742">, </offsets><xref ref-type="bibr" rid="CR5"><offsets xml_i="10538" xml_f="10539" txt_i="2742" txt_f="2743">5</offsets></xref><offsets xml_i="10546" xml_f="10549" txt_i="2743" txt_f="2746">]. </offsets><italic><offsets xml_i="10557" xml_f="10576" txt_i="2746" txt_f="2765">Schistosoma mansoni</offsets></italic><offsets xml_i="10585" xml_f="10777" txt_i="2765" txt_f="2957"> inhabits the mesenteric veins of the gut and, by releasing eggs, triggers a host cellular immune response, causing a wide range of clinical manifestations including serious gut inflammation [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="10809" xml_f="10810" txt_i="2957" txt_f="2958">1</offsets></xref><offsets xml_i="10817" xml_f="10926" txt_i="2958" txt_f="3067">]. Similarly, to other parasitic worms, they interact and affect the same environment as the gut microbiota [</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="10958" xml_f="10959" txt_i="3067" txt_f="3068">6</offsets></xref><offsets xml_i="10966" xml_f="10968" txt_i="3068" txt_f="3070">, </offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="11000" xml_f="11001" txt_i="3070" txt_f="3071">7</offsets></xref><offsets xml_i="11008" xml_f="11010" txt_i="3071" txt_f="3073">].</offsets></p><p id="Par7"><offsets xml_i="11027" xml_f="11311" txt_i="3074" txt_f="3358">Treatment options against schistosomiasis are limited to only one drug, namely, praziquantel. Developed in the mid-1970s, praziquantel remains poorly characterized. Its mechanism of action is not well understood and pharmacokinetic/dynamic relationships have not yet been determined [</offsets><xref ref-type="bibr" rid="CR8"><offsets xml_i="11343" xml_f="11344" txt_i="3358" txt_f="3359">8</offsets></xref><offsets xml_i="11351" xml_f="11353" txt_i="3359" txt_f="3361">, </offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="11385" xml_f="11386" txt_i="3361" txt_f="3362">9</offsets></xref><offsets xml_i="11393" xml_f="11533" txt_i="3362" txt_f="3502">]. Remarkably, there is considerable heterogeneity in treatment outcomes, yet the link to any type of resistance mechanism remains elusive [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="11565" xml_f="11566" txt_i="3502" txt_f="3503">1</offsets></xref><offsets xml_i="11573" xml_f="11575" txt_i="3503" txt_f="3505">, </offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="11608" xml_f="11610" txt_i="3505" txt_f="3507">10</offsets></xref><offsets xml_i="11617" xml_f="11619" txt_i="3507" txt_f="3509">, </offsets><xref ref-type="bibr" rid="CR11"><offsets xml_i="11652" xml_f="11654" txt_i="3509" txt_f="3511">11</offsets></xref><offsets xml_i="11661" xml_f="11898" txt_i="3511" txt_f="3748">]. Hence, it is possible that the efficacy of praziquantel is dependent on a variety of factors, including - but not limited - to hosts genetic background, differences in the drug disposition and bioavailability, and the gut microbiota [</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="11930" xml_f="11931" txt_i="3748" txt_f="3749">6</offsets></xref><offsets xml_i="11938" xml_f="11940" txt_i="3749" txt_f="3751">, </offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="11972" xml_f="11973" txt_i="3751" txt_f="3752">7</offsets></xref><offsets xml_i="11980" xml_f="11982" txt_i="3752" txt_f="3754">].</offsets></p><p id="Par8"><offsets xml_i="11999" xml_f="12168" txt_i="3755" txt_f="3924">Several studies have aimed at characterizing the microbiome from different parts of the human body and have shown considerable variation from one individual to another [</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="12201" xml_f="12203" txt_i="3924" txt_f="3926">12</offsets></xref><offsets xml_i="12210" xml_f="12211" txt_i="3926" txt_f="3927">–</offsets><xref ref-type="bibr" rid="CR14"><offsets xml_i="12244" xml_f="12246" txt_i="3927" txt_f="3929">14</offsets></xref><offsets xml_i="12253" xml_f="12382" txt_i="3929" txt_f="4058">]. The gut microbiome is composed of hundreds of different microorganisms, including eukaryotic parasites, bacteria and viruses [</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="12415" xml_f="12417" txt_i="4058" txt_f="4060">15</offsets></xref><offsets xml_i="12424" xml_f="12425" txt_i="4060" txt_f="4061">–</offsets><xref ref-type="bibr" rid="CR17"><offsets xml_i="12458" xml_f="12460" txt_i="4061" txt_f="4063">17</offsets></xref><offsets xml_i="12467" xml_f="12652" txt_i="4063" txt_f="4248">]. Bacteria are the most predominant type, with approximately 300 to 1000 different species in the intestine, and account for a large majority of the genetic material present in stool [</offsets><xref ref-type="bibr" rid="CR16"><offsets xml_i="12685" xml_f="12687" txt_i="4248" txt_f="4250">16</offsets></xref><offsets xml_i="12694" xml_f="12696" txt_i="4250" txt_f="4252">, </offsets><xref ref-type="bibr" rid="CR18"><offsets xml_i="12729" xml_f="12731" txt_i="4252" txt_f="4254">18</offsets></xref><offsets xml_i="12738" xml_f="12740" txt_i="4254" txt_f="4256">, </offsets><xref ref-type="bibr" rid="CR19"><offsets xml_i="12773" xml_f="12775" txt_i="4256" txt_f="4258">19</offsets></xref><offsets xml_i="12782" xml_f="12964" txt_i="4258" txt_f="4440">]. Recent studies have shown that specific composition of the microbiota, i.e. species diversity as well as relative abundance, modulates the metabolism and disposal of xenobiotics [</offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="12997" xml_f="12999" txt_i="4440" txt_f="4442">20</offsets></xref><offsets xml_i="13006" xml_f="13007" txt_i="4442" txt_f="4443">–</offsets><xref ref-type="bibr" rid="CR24"><offsets xml_i="13040" xml_f="13042" txt_i="4443" txt_f="4445">24</offsets></xref><offsets xml_i="13049" xml_f="13051" txt_i="4445" txt_f="4447">].</offsets></p><p id="Par9"><offsets xml_i="13068" xml_f="13123" txt_i="4448" txt_f="4503">In this study, we investigate the associations between </offsets><italic><offsets xml_i="13131" xml_f="13141" txt_i="4503" txt_f="4513">S. mansoni</offsets></italic><offsets xml_i="13150" xml_f="13496" txt_i="4513" txt_f="4859">, praziquantel and the gut microbial composition in the framework of a randomized, controlled, dose-finding and pharmacokinetic trial of praziquantel in preschool-aged and school-aged children. Samples were analysed to explore differences in microbial composition between (i) infected and non-infected children, (ii) those receiving praziquantel </offsets><italic><offsets xml_i="13504" xml_f="13510" txt_i="4859" txt_f="4865">versus</offsets></italic><offsets xml_i="13519" xml_f="13648" txt_i="4865" txt_f="4994"> placebo, and (iii) those for which treatment with praziquantel resulted either in successful or unsuccessful clearance of worms.</offsets></p></sec><sec id="Sec2"><title><offsets xml_i="13680" xml_f="13687" txt_i="4996" txt_f="5003">Methods</offsets></title><sec id="Sec3"><title><offsets xml_i="13717" xml_f="13734" txt_i="5004" txt_f="5021">Sample collection</offsets></title><p id="Par10"><offsets xml_i="13756" xml_f="13907" txt_i="5022" txt_f="5173">The stool samples used in this study were sourced from a praziquantel dose-finding clinical trial in school-aged and pre-school-aged in Côte d’Ivoire [</offsets><xref ref-type="bibr" rid="CR25"><offsets xml_i="13940" xml_f="13942" txt_i="5173" txt_f="5175">25</offsets></xref><offsets xml_i="13949" xml_f="14117" txt_i="5175" txt_f="5343">]. Briefly, the clinical trial was executed between November 2014 and February 2015 in five villages located in the health district of Azaguié, southern Côte d’Ivoire. </offsets><italic><offsets xml_i="14125" xml_f="14144" txt_i="5343" txt_f="5362">Schistosoma mansoni</offsets></italic><offsets xml_i="14153" xml_f="14173" txt_i="5362" txt_f="5382">-positive children (</offsets><italic><offsets xml_i="14181" xml_f="14182" txt_i="5382" txt_f="5383">n</offsets></italic><offsets xml_i="14191" xml_f="14527" txt_i="5383" txt_f="5719"> = 317; 2–15 years) as confirmed by quadruplicate Kato-Katz smears from 2 stool samples were eligible to participate in this trial and randomized to 20, 40 or 60 mg/kg praziquantel or placebo. For this study, a sub-set of samples from 34 children treated with 60 mg/kg were selected for microbial analysis. Children had been tested for </offsets><italic><offsets xml_i="14535" xml_f="14555" txt_i="5719" txt_f="5739">Ascaris lumbricoides</offsets></italic><offsets xml_i="14564" xml_f="14579" txt_i="5739" txt_f="5754">, hookworm and </offsets><italic><offsets xml_i="14587" xml_f="14606" txt_i="5754" txt_f="5773">Trichuris trichuria</offsets></italic><offsets xml_i="14615" xml_f="15228" txt_i="5773" txt_f="6386">, and those who were co-infected with either of these parasites were not included in the subset of 34 children analysed in this study. Children who received anthelminthic or antimalarial treatment, or presented symptoms of a systemic disease 4 weeks prior to this study were excluded. From each child, stool samples were collected prior to treatment (labelled with “A”), 24 hours post-treatment (labelled with “B”) and 3 weeks post-treatment (labelled with “C”), and a total of 96 stool samples were analysed. Samples from each of these time-points were selected according to four different conditions (see Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="15262" xml_f="15263" txt_i="6386" txt_f="6387">1</offsets></xref><offsets xml_i="15270" xml_f="15304" txt_i="6387" txt_f="6421">): (1) non-infected children; (2) </offsets><italic><offsets xml_i="15312" xml_f="15322" txt_i="6421" txt_f="6431">S. mansoni</offsets></italic><offsets xml_i="15331" xml_f="15415" txt_i="6431" txt_f="6515"> infection and cure of children following 60 mg/kg praziquantel administration; (3) </offsets><italic><offsets xml_i="15423" xml_f="15433" txt_i="6515" txt_f="6525">S. mansoni</offsets></italic><offsets xml_i="15442" xml_f="15551" txt_i="6525" txt_f="6634"> infection and children not cured following 60 mg/kg praziquantel administration; (4) children infected with </offsets><italic><offsets xml_i="15559" xml_f="15569" txt_i="6634" txt_f="6644">S. mansoni</offsets></italic><offsets xml_i="15578" xml_f="15604" txt_i="6644" txt_f="6670"> receiving a placebo. The </offsets><italic><offsets xml_i="15612" xml_f="15623" txt_i="6670" txt_f="6681">S. mansoni-</offsets></italic><offsets xml_i="15632" xml_f="15922" txt_i="6681" txt_f="6971">positive groups (Groups 2–4) were further stratified according to infection intensity as defined by the World Health Organization (WHO) guidelines for classification of schistosomiasis: (i) high (over 400 eggs per gram (epg) of stool); (ii) moderate (100–399 epg); or (iii) low (1–99 epg) [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="15955" xml_f="15957" txt_i="6971" txt_f="6973">26</offsets></xref><offsets xml_i="15964" xml_f="16044" txt_i="6973" txt_f="7053">]. We included children characterized by different infection intensities (Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="16078" xml_f="16079" txt_i="7053" txt_f="7054">1</offsets></xref><offsets xml_i="16086" xml_f="16088" txt_i="7054" txt_f="7056">).</offsets><table-wrap id="Tab1"><label><offsets xml_i="16117" xml_f="16124" txt_i="7056" txt_f="7063">Table 1</offsets></label><caption><p><offsets xml_i="16144" xml_f="16398" txt_i="7063" txt_f="7317">Summary of participants investigated in this study. Each child provided a sample before, 24 hours after and 3 weeks after treatment with single dose (60 mg/kg) dose of praziquantel. For the placebo controls, samples were collected at the same time points</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th><italic><offsets xml_i="16472" xml_f="16482" txt_i="7318" txt_f="7328">S. mansoni</offsets></italic><offsets xml_i="16491" xml_f="16501" txt_i="7328" txt_f="7338"> infection</offsets></th><th><offsets xml_i="16510" xml_f="16535" txt_i="7338" txt_f="7363">Infection intensity (epg)</offsets></th><th><offsets xml_i="16544" xml_f="16561" txt_i="7363" txt_f="7380">Treatment outcome</offsets></th><th><offsets xml_i="16570" xml_f="16589" txt_i="7380" txt_f="7399">No. of participants</offsets></th><th><offsets xml_i="16598" xml_f="16616" txt_i="7399" txt_f="7417">Sample identifiers</offsets></th><th><offsets xml_i="16625" xml_f="16635" txt_i="7417" txt_f="7427">Group name</offsets></th></tr></thead><tbody><tr><td><offsets xml_i="16668" xml_f="16676" txt_i="7427" txt_f="7435">Negative</offsets></td><td><offsets xml_i="16685" xml_f="16686" txt_i="7436" txt_f="7437">0</offsets></td><td><offsets xml_i="16695" xml_f="16709" txt_i="7438" txt_f="7452">Not applicable</offsets></td><td><offsets xml_i="16718" xml_f="16719" txt_i="7453" txt_f="7454">6</offsets></td><td><offsets xml_i="16728" xml_f="16749" txt_i="7455" txt_f="7476">1; 12; 13; 14; 15; 25</offsets></td><td><offsets xml_i="16758" xml_f="16766" txt_i="7477" txt_f="7485">Controls</offsets></td></tr><tr><td rowspan="9"><offsets xml_i="16796" xml_f="16804" txt_i="7486" txt_f="7494">Positive</offsets></td><td><offsets xml_i="16813" xml_f="16817" txt_i="7495" txt_f="7499">1–99</offsets></td><td rowspan="3"><offsets xml_i="16838" xml_f="16842" txt_i="7500" txt_f="7504">Cure</offsets></td><td><offsets xml_i="16851" xml_f="16852" txt_i="7505" txt_f="7506">0</offsets></td><td></td><td rowspan="3"><offsets xml_i="16882" xml_f="16886" txt_i="7508" txt_f="7512">Cure</offsets></td></tr><tr><td><offsets xml_i="16904" xml_f="16911" txt_i="7513" txt_f="7520">100–399</offsets></td><td><offsets xml_i="16920" xml_f="16921" txt_i="7521" txt_f="7522">5</offsets></td><td><offsets xml_i="16930" xml_f="16948" txt_i="7523" txt_f="7541">11; 17; 22; 26; 28</offsets></td></tr><tr><td><offsets xml_i="16966" xml_f="16974" txt_i="7542" txt_f="7547">&gt; 400</offsets></td><td><offsets xml_i="16983" xml_f="16984" txt_i="7548" txt_f="7549">4</offsets></td><td><offsets xml_i="16993" xml_f="17005" txt_i="7550" txt_f="7562">2; 7; 16; 27</offsets></td></tr><tr><td><offsets xml_i="17023" xml_f="17027" txt_i="7563" txt_f="7567">1–99</offsets></td><td rowspan="3"><offsets xml_i="17048" xml_f="17057" txt_i="7568" txt_f="7577">Not cured</offsets></td><td><offsets xml_i="17066" xml_f="17067" txt_i="7578" txt_f="7579">3</offsets></td><td><offsets xml_i="17076" xml_f="17086" txt_i="7580" txt_f="7590">19; 23; 32</offsets></td><td rowspan="3"><offsets xml_i="17107" xml_f="17116" txt_i="7591" txt_f="7600">Not cured</offsets></td></tr><tr><td><offsets xml_i="17134" xml_f="17141" txt_i="7601" txt_f="7608">100–399</offsets></td><td><offsets xml_i="17150" xml_f="17151" txt_i="7609" txt_f="7610">3</offsets></td><td><offsets xml_i="17160" xml_f="17170" txt_i="7611" txt_f="7621">10; 30; 31</offsets></td></tr><tr><td><offsets xml_i="17188" xml_f="17196" txt_i="7622" txt_f="7627">&gt; 400</offsets></td><td><offsets xml_i="17205" xml_f="17206" txt_i="7628" txt_f="7629">2</offsets></td><td><offsets xml_i="17215" xml_f="17219" txt_i="7630" txt_f="7634">5; 8</offsets></td></tr><tr><td><offsets xml_i="17237" xml_f="17241" txt_i="7635" txt_f="7639">1–99</offsets></td><td rowspan="3"><offsets xml_i="17262" xml_f="17269" txt_i="7640" txt_f="7647">Placebo</offsets></td><td><offsets xml_i="17278" xml_f="17279" txt_i="7648" txt_f="7649">3</offsets></td><td><offsets xml_i="17288" xml_f="17298" txt_i="7650" txt_f="7660">20; 29; 34</offsets></td><td rowspan="3"><offsets xml_i="17319" xml_f="17326" txt_i="7661" txt_f="7668">Placebo</offsets></td></tr><tr><td><offsets xml_i="17344" xml_f="17351" txt_i="7669" txt_f="7676">100–399</offsets></td><td><offsets xml_i="17360" xml_f="17361" txt_i="7677" txt_f="7678">5</offsets></td><td><offsets xml_i="17370" xml_f="17390" txt_i="7679" txt_f="7699">6; 18; 21; 24; 33; 1</offsets></td></tr><tr><td><offsets xml_i="17408" xml_f="17416" txt_i="7700" txt_f="7705">&gt; 400</offsets></td><td><offsets xml_i="17425" xml_f="17426" txt_i="7706" txt_f="7707">3</offsets></td><td><offsets xml_i="17435" xml_f="17442" txt_i="7708" txt_f="7715">3; 4; 9</offsets></td></tr></tbody></table><table-wrap-foot><p><italic><offsets xml_i="17496" xml_f="17508" txt_i="7716" txt_f="7728">Abbreviation</offsets></italic><offsets xml_i="17517" xml_f="17519" txt_i="7728" txt_f="7730">: </offsets><italic><offsets xml_i="17527" xml_f="17530" txt_i="7730" txt_f="7733">epg</offsets></italic><offsets xml_i="17539" xml_f="17559" txt_i="7733" txt_f="7753"> eggs per gram stool</offsets></p></table-wrap-foot></table-wrap></p><p id="Par11"><offsets xml_i="17612" xml_f="17827" txt_i="7755" txt_f="7970">The subset of children was selected based on the availability of (i) complete metadata and demographic information, (ii) all parasitology test results, and (iii) the three stool samples, at all three sampling times.</offsets></p><p id="Par12"><offsets xml_i="17845" xml_f="18058" txt_i="7971" txt_f="8184">Pre-schoolers (3–5 year-old) had a Body Mass Index (BMI) ranging from 12.7 to 21 and school-aged children (5–13 year-old) had a BMI ranging from 14.7 to 32.2. Detailed information is summarized in Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="18094" xml_f="18095" txt_i="8184" txt_f="8185">1</offsets></xref><offsets xml_i="18102" xml_f="18358" txt_i="8185" txt_f="8441">: Table S1. According to thresholds set by the WHO, growth indicators indicate that nutritional stunting is not occurring in the subset of children involved in this study and that the potential impact of malnutrition on the gut microbiota can be ruled out.</offsets></p></sec><sec id="Sec4"><title><offsets xml_i="18390" xml_f="18403" txt_i="8443" txt_f="8456">DNA isolation</offsets></title><p id="Par13"><offsets xml_i="18425" xml_f="18922" txt_i="8457" txt_f="8954">For each sample, 150 mg of faecal material was used to isolate genomic DNA. DNA isolation was conducted with the QIAamp DNA Stool kits (Qiagen, Hilden, Germany) according to the manufacturers’ recommendations. Extracted DNA was eluted in 50 μl of the elution buffer and used for downstream analyses. Concentrations were measured with a Qubit 3.0 fluorometer using a dsDNA High-sensitivity Assay kit (Thermo Fisher Scientific, Waltham, MA, USA). All concentrations are indicated in Additional file </offsets><xref rid="MOESM2" ref-type="media"><offsets xml_i="18958" xml_f="18959" txt_i="8954" txt_f="8955">2</offsets></xref><offsets xml_i="18966" xml_f="18977" txt_i="8955" txt_f="8966">: Table S2.</offsets></p></sec><sec id="Sec5"><title><italic><offsets xml_i="19017" xml_f="19020" txt_i="8968" txt_f="8971">16S</offsets></italic><offsets xml_i="19029" xml_f="19070" txt_i="8971" txt_f="9012"> amplicon polymerase chain reaction (PCR)</offsets></title><p id="Par14"><offsets xml_i="19092" xml_f="19233" txt_i="9013" txt_f="9154">Isolated DNA (2.5 μl) was used to perform amplification of the V3-V4 region using the following primer pair, according to Klindworth et al. [</offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="19266" xml_f="19268" txt_i="9154" txt_f="9156">27</offsets></xref><offsets xml_i="19275" xml_f="19452" txt_i="9156" txt_f="9333">]: forward (5'-TCG TCG GCA GCG TCA GAT GTG TAT AAG AGA CAG CCT ACG GGN GGC WGC AG-3'); reverse (5'-GTC TCG TGG GCT CGG AGA TGT GTA TAA GAG ACA GGA CTA CHV GGG TAT CTA ATC C-3').</offsets></p><p id="Par15"><offsets xml_i="19470" xml_f="19992" txt_i="9334" txt_f="9856">The reaction was performed in 25 μl reaction volumes using the 2X KAPA HiFi HotStar ReadyMix (KAPA Biosystems, Boston, MA, USA). The thermocycler was set to the following parameters: 95 °C for 3 min, 25 cycles of 95 °C (30 s), 55 °C (30 s) and 72 °C (30 s), one additional step at 72 °C (5 min) and finally set on hold indefinitely at 4 °C. The quality of the amplified product was controlled visually on a 1% agarose gel. The amplicons were purified with an AMPure XP beads (Beckman-Coulter, Fullerton, CA, USA) protocol.</offsets></p></sec><sec id="Sec6"><title><offsets xml_i="20024" xml_f="20034" txt_i="9858" txt_f="9868">Sequencing</offsets></title><p id="Par16"><offsets xml_i="20056" xml_f="21064" txt_i="9869" txt_f="10877">The barcoding PCR was performed using primer pairs from the Nextera XT Index kit (Illumina, San Diego, CA, USA) to produce 96 amplicon pools with different tags. The amplification reaction was conducted with the same reagents and the thermocycler was set to the following parameters: 95 °C for 3 min, 20 cycles of 95 °C (30 s), 55 °C (30 s), 72 °C (30 s), one additional step at 72 °C (5 min) and a final step at 4 °C until further processing. Similarly to the first PCR, amplified products were cleaned with AMPure XP beads. The quality of the product was assessed using a 1% agarose gel and the quantification was performed using a Qubit 3.0 Fluorimeter (Thermo Fisher Scientific, Waltham, MA, USA) and the corresponding High-sensitivity dsDNA HS Assay Kit. The 96 amplicon samples were pooled together in an equimolar way and loaded on a cartridge on the Illumina MiSeq sequencing system (Illumina, USA). MiSeq Reagent Kit V3 (2 × 300 bp) sequencing reagents (Illumina, USA) were used for this experiment.</offsets></p></sec><sec id="Sec7"><title><italic><offsets xml_i="21104" xml_f="21117" txt_i="10879" txt_f="10892">Fusobacterium</offsets></italic><offsets xml_i="21126" xml_f="21165" txt_i="10892" txt_f="10931"> spp. quantitative real-time PCR (qPCR)</offsets></title><p id="Par17"><italic><offsets xml_i="21195" xml_f="21208" txt_i="10932" txt_f="10945">Fusobacterium</offsets></italic><offsets xml_i="21217" xml_f="21304" txt_i="10945" txt_f="11032"> spp. was amplified using the primers and Taqman probes as described in Martin et al. [</offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="21337" xml_f="21339" txt_i="11032" txt_f="11034">28</offsets></xref><offsets xml_i="21346" xml_f="21358" txt_i="11034" txt_f="11046">]. DNA from </offsets><italic><offsets xml_i="21366" xml_f="21389" txt_i="11046" txt_f="11069">Fusobacterium nucleatum</offsets></italic><offsets xml_i="21398" xml_f="21602" txt_i="11069" txt_f="11273"> was obtained from the German Collection of Microorganisms and Cell Cultures (DSMZ). Briefly, 2 μl of isolated DNA was used to perform the amplification of a conserved region of the 16S rDNA of the genus </offsets><italic><offsets xml_i="21610" xml_f="21623" txt_i="11273" txt_f="11286">Fusobacterium</offsets></italic><offsets xml_i="21632" xml_f="21644" txt_i="11286" txt_f="11298"> (including </offsets><italic><offsets xml_i="21652" xml_f="21664" txt_i="11298" txt_f="11310">F. nucleatum</offsets></italic><offsets xml_i="21673" xml_f="21675" txt_i="11310" txt_f="11312">, </offsets><italic><offsets xml_i="21683" xml_f="21699" txt_i="11312" txt_f="11328">F. periodonticum</offsets></italic><offsets xml_i="21708" xml_f="21710" txt_i="11328" txt_f="11330">, </offsets><italic><offsets xml_i="21718" xml_f="21727" txt_i="11330" txt_f="11339">F. alocis</offsets></italic><offsets xml_i="21736" xml_f="21741" txt_i="11339" txt_f="11344"> and </offsets><italic><offsets xml_i="21749" xml_f="21758" txt_i="11344" txt_f="11353">F. simiae</offsets></italic><offsets xml_i="21767" xml_f="22326" txt_i="11353" txt_f="11912">). The experiment was conducted in 20 μl reaction volumes using the KAPA HiFi Universal kit (KAPA Biosystems, USA). Thermocycling conditions were set as follows: 50 °C for 2 min, 95 °C (10 min) and 40 cycles of 15 s at 95 °C and 1 min at 60 °C. Amplification was performed on a ViiA 7 Real-Time PCR System (Applied Biosystems, Foster City, USA). All samples were tested in triplicates. For normalization, total DNA was measured using a Qubit 3.0 fluorometer combined with a dsDNA High-sensitivity Assay kit (Thermo Fisher Scientific, Waltham, MA, USA). QPCR C</offsets><sub><offsets xml_i="22331" xml_f="22332" txt_i="11912" txt_f="11913">t</offsets></sub><offsets xml_i="22338" xml_f="22408" txt_i="11913" txt_f="11983"> values were normalized by input material using the following formula:</offsets></p><p id="Par18"><offsets xml_i="22426" xml_f="22427" txt_i="11984" txt_f="11985">
</offsets><disp-formula id="Equa"><alternatives><tex-math id="M1"><offsets xml_i="22483" xml_f="22916" txt_i="11985" txt_f="12418">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ {\mathrm{Ct}}_{\mathrm{measured}}-{\log}_{\mathrm{efficiency}}\times \left(\mathrm{dilution}\ \mathrm{factor}\right)={\mathrm{Ct}}_{\mathrm{norm}} $$\end{document}</offsets></tex-math><mml:math id="M2" display="block"><mml:msub><mml:mi><offsets xml_i="22979" xml_f="22981" txt_i="12418" txt_f="12420">Ct</offsets></mml:mi><mml:mtext><offsets xml_i="23001" xml_f="23009" txt_i="12420" txt_f="12428">measured</offsets></mml:mtext></mml:msub><mml:mo><offsets xml_i="23040" xml_f="23041" txt_i="12428" txt_f="12429">−</offsets></mml:mo><mml:msub><mml:mo><offsets xml_i="23068" xml_f="23071" txt_i="12429" txt_f="12432">log</offsets></mml:mo><mml:mtext><offsets xml_i="23091" xml_f="23101" txt_i="12432" txt_f="12442">efficiency</offsets></mml:mtext></mml:msub><mml:mo><offsets xml_i="23132" xml_f="23133" txt_i="12442" txt_f="12443">×</offsets></mml:mo><mml:mfenced close=")" open="("><mml:mtext><offsets xml_i="23185" xml_f="23200" txt_i="12443" txt_f="12458">dilution factor</offsets></mml:mtext></mml:mfenced><mml:mo><offsets xml_i="23234" xml_f="23235" txt_i="12458" txt_f="12459">=</offsets></mml:mo><mml:msub><mml:mi><offsets xml_i="23262" xml_f="23264" txt_i="12459" txt_f="12461">Ct</offsets></mml:mi><mml:mi><offsets xml_i="23281" xml_f="23285" txt_i="12461" txt_f="12465">norm</offsets></mml:mi></mml:msub></mml:math><graphic xlink:href="13071_2018_2739_Article_Equa.gif" position="anchor"></graphic></alternatives></disp-formula><offsets xml_i="23429" xml_f="23430" txt_i="12465" txt_f="12466">
</offsets></p><p id="Par19"><offsets xml_i="23448" xml_f="23449" txt_i="12467" txt_f="12468">C</offsets><sub><offsets xml_i="23454" xml_f="23455" txt_i="12468" txt_f="12469">t</offsets></sub><offsets xml_i="23461" xml_f="23589" txt_i="12469" txt_f="12597"> values were further converted into concentrations and subsequently into a theoretical copy numbers using the following formula:</offsets><disp-formula id="Equb"><alternatives><tex-math id="M3"><offsets xml_i="23645" xml_f="24188" txt_i="12597" txt_f="13140">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ \left({\mathrm{Exp}}^{\left(\mathrm{Ctnorm}\hbox{-} \mathrm{b}\right)/\mathrm{a}\Big)}\times \mathrm{NA}\right)/\left({\mathrm{Fusobacterium}}_{\mathrm{bp}}\times \left(1\times {10}^{{{}^{\hat{\mkern6mu}}}^9}\right)\times {\mathrm{MW}}_{\mathrm{bp}}\right) $$\end{document}</offsets></tex-math><mml:math id="M4" display="block"><mml:mfenced close=")" open="("><mml:mrow><mml:msup><mml:mi><offsets xml_i="24293" xml_f="24296" txt_i="13140" txt_f="13143">Exp</offsets></mml:mi><mml:mrow><mml:mfenced close=")" open="("><mml:mrow><mml:mtext><offsets xml_i="24368" xml_f="24374" txt_i="13143" txt_f="13149">Ctnorm</offsets></mml:mtext><mml:mo><offsets xml_i="24394" xml_f="24395" txt_i="13149" txt_f="13150">‐</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="24433" xml_f="24434" txt_i="13150" txt_f="13151">b</offsets></mml:mi></mml:mrow></mml:mfenced><mml:mo><offsets xml_i="24476" xml_f="24477" txt_i="13151" txt_f="13152">/</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="24515" xml_f="24516" txt_i="13152" txt_f="13153">a</offsets></mml:mi><mml:mo stretchy="true"><offsets xml_i="24549" xml_f="24550" txt_i="13153" txt_f="13154">)</offsets></mml:mo></mml:mrow></mml:msup><mml:mo><offsets xml_i="24589" xml_f="24590" txt_i="13154" txt_f="13155">×</offsets></mml:mo><mml:mi><offsets xml_i="24607" xml_f="24609" txt_i="13155" txt_f="13157">NA</offsets></mml:mi></mml:mrow></mml:mfenced><mml:mo><offsets xml_i="24651" xml_f="24652" txt_i="13157" txt_f="13158">/</offsets></mml:mo><mml:mfenced close=")" open="("><mml:mrow><mml:msub><mml:mtext><offsets xml_i="24724" xml_f="24737" txt_i="13158" txt_f="13171">Fusobacterium</offsets></mml:mtext><mml:mi><offsets xml_i="24757" xml_f="24759" txt_i="13171" txt_f="13173">bp</offsets></mml:mi></mml:msub><mml:mo><offsets xml_i="24787" xml_f="24788" txt_i="13173" txt_f="13174">×</offsets></mml:mo><mml:mfenced close=")" open="("><mml:mrow><mml:mn><offsets xml_i="24847" xml_f="24848" txt_i="13174" txt_f="13175">1</offsets></mml:mn><mml:mo><offsets xml_i="24865" xml_f="24866" txt_i="13175" txt_f="13176">×</offsets></mml:mo><mml:msup><mml:mn><offsets xml_i="24893" xml_f="24895" txt_i="13176" txt_f="13178">10</offsets></mml:mn><mml:msup><mml:msup><mml:mrow></mml:mrow><mml:mo><offsets xml_i="24953" xml_f="24954" txt_i="13178" txt_f="13179">^</offsets></mml:mo></mml:msup><mml:mn><offsets xml_i="24982" xml_f="24983" txt_i="13179" txt_f="13180">9</offsets></mml:mn></mml:msup></mml:msup></mml:mrow></mml:mfenced><mml:mo><offsets xml_i="25047" xml_f="25048" txt_i="13180" txt_f="13181">×</offsets></mml:mo><mml:msub><mml:mi><offsets xml_i="25075" xml_f="25077" txt_i="13181" txt_f="13183">MW</offsets></mml:mi><mml:mi><offsets xml_i="25094" xml_f="25096" txt_i="13183" txt_f="13185">bp</offsets></mml:mi></mml:msub></mml:mrow></mml:mfenced></mml:math><graphic xlink:href="13071_2018_2739_Article_Equb.gif" position="anchor"></graphic></alternatives></disp-formula></p><p id="Par20"><offsets xml_i="25283" xml_f="25348" txt_i="13186" txt_f="13251">All resulting values were compared using the Mann-Whitney U-test.</offsets></p></sec><sec id="Sec8"><title><offsets xml_i="25380" xml_f="25420" txt_i="13253" txt_f="13293">Data processing and statistical analysis</offsets></title><p id="Par21"><offsets xml_i="25442" xml_f="25489" txt_i="13294" txt_f="13341">Raw datasets were fed into the QIIME pipeline [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="25522" xml_f="25524" txt_i="13341" txt_f="13343">29</offsets></xref><offsets xml_i="25531" xml_f="25780" txt_i="13343" txt_f="13592">] with standard OTU (operational taxonomic unit) picking parameters in closed-reference mode. Alpha and beta diversities were computed using a rarefaction depth of 10,800 sequences. Resulting abundance tables were analysed using the LefSe pipeline [</offsets><xref ref-type="bibr" rid="CR30"><offsets xml_i="25813" xml_f="25815" txt_i="13592" txt_f="13594">30</offsets></xref><offsets xml_i="25822" xml_f="25996" txt_i="13594" txt_f="13768">]. Briefly, a first step compared the relative abundance of all identified taxa between population groups using the Kruskal-Wallis test. In case of statistical significance (</offsets><italic><offsets xml_i="26004" xml_f="26005" txt_i="13768" txt_f="13769">P</offsets></italic><offsets xml_i="26014" xml_f="26359" txt_i="13769" txt_f="14111"> &lt; 0.05), a pairwise Wilcoxon test was conducted to analyse whether the feature is evenly distributed among individuals of the same group but with different demographic characteristics (e.g. different age, sex). All variables passing both tests are ranked using a Linear Discriminant Analysis model by their relative differences among groups.</offsets></p></sec></sec><sec id="Sec9"><title><offsets xml_i="26397" xml_f="26404" txt_i="14114" txt_f="14121">Results</offsets></title><sec id="Sec10"><title><offsets xml_i="26435" xml_f="26525" txt_i="14122" txt_f="14212">Composition of the gut microbiota in pre-treatment samples at the phylum and family levels</offsets></title><p id="Par22"><offsets xml_i="26547" xml_f="26823" txt_i="14213" txt_f="14489">Pre-processing, including filtering and de-noising of the sequence datasets resulted in the analysis of a total of 4,160,032 sequences for 34 baseline samples with an average of 122,353 (Standard deviation: 74,507) sequences per sample. Results, presented at the phylum (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="26855" xml_f="26857" txt_i="14489" txt_f="14491">1a</offsets></xref><offsets xml_i="26864" xml_f="26890" txt_i="14491" txt_f="14517">) and at the family (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="26922" xml_f="26924" txt_i="14517" txt_f="14519">1b</offsets></xref><offsets xml_i="26931" xml_f="27038" txt_i="14519" txt_f="14626">) level of resolution are stratified according to infection intensities (negative, low, moderate and high).</offsets><fig id="Fig1"><label><offsets xml_i="27060" xml_f="27066" txt_i="14626" txt_f="14632">Fig. 1</offsets></label><caption><p><offsets xml_i="27086" xml_f="27284" txt_i="14632" txt_f="14830">Composition of the microbiota of pre-treatment samples at two taxonomic levels. This bar chart shows the composition of the most abundant bacterial groups of each patient, both at the phylum level (</offsets><bold><offsets xml_i="27290" xml_f="27291" txt_i="14830" txt_f="14831">a</offsets></bold><offsets xml_i="27298" xml_f="27332" txt_i="14831" txt_f="14865">) as well as at the family level (</offsets><bold><offsets xml_i="27338" xml_f="27339" txt_i="14865" txt_f="14866">b</offsets></bold><offsets xml_i="27346" xml_f="27347" txt_i="14866" txt_f="14867">)</offsets></p></caption><graphic xlink:href="13071_2018_2739_Fig1_HTML" id="MO1"></graphic></fig></p><p id="Par23"><offsets xml_i="27452" xml_f="27481" txt_i="14869" txt_f="14898">Across all groups, the phyla </offsets><italic><offsets xml_i="27489" xml_f="27499" txt_i="14898" txt_f="14908">Firmicutes</offsets></italic><offsets xml_i="27508" xml_f="27513" txt_i="14908" txt_f="14913"> and </offsets><italic><offsets xml_i="27521" xml_f="27534" txt_i="14913" txt_f="14926">Bacteroidetes</offsets></italic><offsets xml_i="27543" xml_f="27767" txt_i="14926" txt_f="15147"> were the most abundant (&gt; 50%), except in one sample from the low-intensity infection group (sample 34A) and two from the high-intensity infection group (samples 5A and 8A). The minimum and maximum observed abundance of </offsets><italic><offsets xml_i="27775" xml_f="27785" txt_i="15147" txt_f="15157">Firmicutes</offsets></italic><offsets xml_i="27794" xml_f="27838" txt_i="15157" txt_f="15201"> were 5.9% and 86.5%, respectively. For the </offsets><italic><offsets xml_i="27846" xml_f="27859" txt_i="15201" txt_f="15214">Bacteroidetes</offsets></italic><offsets xml_i="27868" xml_f="27921" txt_i="15214" txt_f="15267">, the range spanned from 1.9% up to 64.2%. The ratio </offsets><italic><offsets xml_i="27929" xml_f="27939" txt_i="15267" txt_f="15277">Firmicutes</offsets></italic><offsets xml_i="27948" xml_f="27954" txt_i="15277" txt_f="15283"> over </offsets><italic><offsets xml_i="27962" xml_f="27975" txt_i="15283" txt_f="15296">Bacteroidetes</offsets></italic><offsets xml_i="27984" xml_f="28180" txt_i="15296" txt_f="15492"> remained above one across all but 6 samples (samples 6A, 18A, 19A, 23A, 24A and 30A). All 6 samples with ratios lower than one were found among the group of schistosome-infected individuals. The </offsets><italic><offsets xml_i="28188" xml_f="28202" txt_i="15492" txt_f="15506">Proteobacteria</offsets></italic><offsets xml_i="28211" xml_f="28493" txt_i="15506" txt_f="15788"> phylum was the third most abundant taxon, with a median of 7.71% across all samples. While the relative abundance of this group was low in the control group (non-infected children), with a median of 3.69% (minimum of 1%; maximum of 20%), the median value in children infected with </offsets><italic><offsets xml_i="28501" xml_f="28511" txt_i="15788" txt_f="15798">S. mansoni</offsets></italic><offsets xml_i="28520" xml_f="28579" txt_i="15798" txt_f="15857"> was 9.04% (minimum of 0.9%; maximum of 87%). The ratio of </offsets><italic><offsets xml_i="28587" xml_f="28597" txt_i="15857" txt_f="15867">Firmicutes</offsets></italic><offsets xml_i="28606" xml_f="28612" txt_i="15867" txt_f="15873"> over </offsets><italic><offsets xml_i="28620" xml_f="28634" txt_i="15873" txt_f="15887">Proteobacteria</offsets></italic><offsets xml_i="28643" xml_f="28764" txt_i="15887" txt_f="16008"> was above one in all except for 6 samples from the infected group (samples 5A, 8A, 22A, 24A, 29A and 34A). The ratio of </offsets><italic><offsets xml_i="28772" xml_f="28785" txt_i="16008" txt_f="16021">Bacteroidetes</offsets></italic><offsets xml_i="28794" xml_f="28800" txt_i="16021" txt_f="16027"> over </offsets><italic><offsets xml_i="28808" xml_f="28822" txt_i="16027" txt_f="16041">Proteobacteria</offsets></italic><offsets xml_i="28831" xml_f="28908" txt_i="16041" txt_f="16118"> was below one in the same samples, along with 6 additional samples from the </offsets><italic><offsets xml_i="28916" xml_f="28927" txt_i="16118" txt_f="16129">Schistosoma</offsets></italic><offsets xml_i="28936" xml_f="29054" txt_i="16129" txt_f="16247">-positive group (samples 7A, 10A, 20A, 28A, 32A and 33A). A final observation at the phylum level was the presence of </offsets><italic><offsets xml_i="29062" xml_f="29076" txt_i="16247" txt_f="16261">Actinobacteria</offsets></italic><offsets xml_i="29085" xml_f="29090" txt_i="16261" txt_f="16266"> and </offsets><italic><offsets xml_i="29098" xml_f="29113" txt_i="16266" txt_f="16281">Verrucomicrobia</offsets></italic><offsets xml_i="29122" xml_f="29207" txt_i="16281" txt_f="16366"> with a median of 0.9% and 0.1%, respectively. At the family level, the abundance of </offsets><italic><offsets xml_i="29215" xml_f="29233" txt_i="16366" txt_f="16384">Enterobacteriaceae</offsets></italic><offsets xml_i="29242" xml_f="29269" txt_i="16384" txt_f="16411">, a subgroup of the phylum </offsets><italic><offsets xml_i="29277" xml_f="29291" txt_i="16411" txt_f="16425">Proteobacteria</offsets></italic><offsets xml_i="29300" xml_f="29552" txt_i="16425" txt_f="16677"> that contains numerous pathogens, reached a maximum of 20% in the control group (minimum of 2.6%) whereas in infected children, the abundance of the same family was as high as 87.6% of the overall composition. The abundance of members from the family </offsets><italic><offsets xml_i="29560" xml_f="29574" txt_i="16677" txt_f="16691">Prevotellaceae</offsets></italic><offsets xml_i="29583" xml_f="29663" txt_i="16691" txt_f="16771"> was highly heterogeneous among the different groups, ranging from 1.6 to 56.2%.</offsets></p></sec><sec id="Sec11"><title><offsets xml_i="29696" xml_f="29732" txt_i="16773" txt_f="16809">Comparison of the gut microbiota in </offsets><italic><offsets xml_i="29740" xml_f="29750" txt_i="16809" txt_f="16819">S. mansoni</offsets></italic><offsets xml_i="29759" xml_f="29794" txt_i="16819" txt_f="16854">-infected and non-infected children</offsets></title><p id="Par24"><offsets xml_i="29816" xml_f="29921" txt_i="16855" txt_f="16960">Differences between infected and non-infected individuals were explored using the LefSe pipeline. Figure </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="29953" xml_f="29954" txt_i="16960" txt_f="16961">2</offsets></xref><offsets xml_i="29961" xml_f="30059" txt_i="16961" txt_f="17059"> shows the abundances of the different taxonomic groups between infected and non-infected samples.</offsets><fig id="Fig2"><label><offsets xml_i="30081" xml_f="30087" txt_i="17059" txt_f="17065">Fig. 2</offsets></label><caption><p><offsets xml_i="30107" xml_f="30176" txt_i="17065" txt_f="17134">The gut microbiome composition differences in samples collected from </offsets><italic><offsets xml_i="30184" xml_f="30194" txt_i="17134" txt_f="17144">S. mansoni</offsets></italic><offsets xml_i="30203" xml_f="30248" txt_i="17144" txt_f="17189"> negative and positive children at baseline. </offsets><bold><offsets xml_i="30254" xml_f="30255" txt_i="17189" txt_f="17190">a</offsets></bold><offsets xml_i="30262" xml_f="30409" txt_i="17190" txt_f="17337"> Quantitative representation of the differences in the gut microbial composition of both groups using a bar chart representation and an LDA model (</offsets><italic><offsets xml_i="30417" xml_f="30418" txt_i="17337" txt_f="17338">P</offsets></italic><offsets xml_i="30427" xml_f="30440" txt_i="17338" txt_f="17348"> &lt; 0.05). </offsets><bold><offsets xml_i="30446" xml_f="30447" txt_i="17348" txt_f="17349">b</offsets></bold><offsets xml_i="30454" xml_f="30553" txt_i="17349" txt_f="17448"> Taxonomic representation of significantly different relative abundances. Discriminative features (</offsets><italic><offsets xml_i="30561" xml_f="30562" txt_i="17448" txt_f="17449">P</offsets></italic><offsets xml_i="30571" xml_f="30623" txt_i="17449" txt_f="17498"> &lt; 0.05) are observed at various taxonomic levels</offsets></p></caption><graphic xlink:href="13071_2018_2739_Fig2_HTML" id="MO2"></graphic></fig></p><p id="Par25"><offsets xml_i="30728" xml_f="30809" txt_i="17500" txt_f="17581">Differentiating abundances between these groups are largely observable among the </offsets><italic><offsets xml_i="30817" xml_f="30832" txt_i="17581" txt_f="17596">Proteobacteria.</offsets></italic><offsets xml_i="30841" xml_f="30866" txt_i="17596" txt_f="17621"> Bacteria from the genus </offsets><italic><offsets xml_i="30874" xml_f="30884" txt_i="17621" txt_f="17631">Klebsiella</offsets></italic><offsets xml_i="30893" xml_f="30898" txt_i="17631" txt_f="17636"> and </offsets><italic><offsets xml_i="30906" xml_f="30928" txt_i="17636" txt_f="17658">Enterobacter arachidis</offsets></italic><offsets xml_i="30937" xml_f="30997" txt_i="17658" txt_f="17718"> were significantly more abundant in children infected with </offsets><italic><offsets xml_i="31005" xml_f="31015" txt_i="17718" txt_f="17728">S. mansoni</offsets></italic><offsets xml_i="31024" xml_f="31026" txt_i="17728" txt_f="17730"> (</offsets><italic><offsets xml_i="31034" xml_f="31035" txt_i="17730" txt_f="17731">P</offsets></italic><offsets xml_i="31044" xml_f="31101" txt_i="17731" txt_f="17785"> &lt; 0.05). Over-abundance of members from the families </offsets><italic><offsets xml_i="31109" xml_f="31124" txt_i="17785" txt_f="17800">Cerasicoccaceae</offsets></italic><offsets xml_i="31133" xml_f="31135" txt_i="17800" txt_f="17802">, </offsets><italic><offsets xml_i="31143" xml_f="31161" txt_i="17802" txt_f="17820">Anaeroplasmataceae</offsets></italic><offsets xml_i="31170" xml_f="31172" txt_i="17820" txt_f="17822">, </offsets><italic><offsets xml_i="31180" xml_f="31198" txt_i="17822" txt_f="17840">Campylobacteraceae</offsets></italic><offsets xml_i="31207" xml_f="31212" txt_i="17840" txt_f="17845"> and </offsets><italic><offsets xml_i="31220" xml_f="31234" txt_i="17845" txt_f="17859">Peptococcaceae</offsets></italic><offsets xml_i="31243" xml_f="31262" txt_i="17859" txt_f="17878">, and of the genus </offsets><italic><offsets xml_i="31270" xml_f="31284" txt_i="17878" txt_f="17892">Fructobacillus</offsets></italic><offsets xml_i="31293" xml_f="31385" txt_i="17892" txt_f="17984"> seem to be significantly linked to the faecal microbiome of schistosome-negative children (</offsets><italic><offsets xml_i="31393" xml_f="31394" txt_i="17984" txt_f="17985">P</offsets></italic><offsets xml_i="31403" xml_f="31415" txt_i="17985" txt_f="17994"> &lt; 0.05).</offsets></p></sec><sec id="Sec12"><title><offsets xml_i="31448" xml_f="31527" txt_i="17996" txt_f="18075">Comparison of the gut microbiota diversity among the different treatment groups</offsets></title><p id="Par26"><offsets xml_i="31549" xml_f="31557" txt_i="18076" txt_f="18084">In Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="31589" xml_f="31590" txt_i="18084" txt_f="18085">3</offsets></xref><offsets xml_i="31597" xml_f="31713" txt_i="18085" txt_f="18201">, we present the different indicators commonly used to assess microbial diversity, obtained with the QIIME pipeline.</offsets><fig id="Fig3"><label><offsets xml_i="31735" xml_f="31741" txt_i="18201" txt_f="18207">Fig. 3</offsets></label><caption><p><offsets xml_i="31761" xml_f="31870" txt_i="18207" txt_f="18316">Indicators of microbial diversity in control and infected samples, both before and 24 hours after treatment. </offsets><bold><offsets xml_i="31876" xml_f="31877" txt_i="18316" txt_f="18317">a</offsets></bold><offsets xml_i="31884" xml_f="31927" txt_i="18317" txt_f="18360"> Faith’s Phylogenetic Diversity indicator. </offsets><bold><offsets xml_i="31933" xml_f="31934" txt_i="18360" txt_f="18361">b</offsets></bold><bold><offsets xml_i="31947" xml_f="31948" txt_i="18361" txt_f="18362">c</offsets></bold><offsets xml_i="31955" xml_f="31977" txt_i="18362" txt_f="18384"> The Shannon’s index. </offsets><bold><offsets xml_i="31983" xml_f="31984" txt_i="18384" txt_f="18385">d</offsets></bold><offsets xml_i="31991" xml_f="32008" txt_i="18385" txt_f="18402"> The Chao 1 index</offsets></p></caption><graphic xlink:href="13071_2018_2739_Fig3_HTML" id="MO3"></graphic></fig></p><p id="Par27"><offsets xml_i="32113" xml_f="32718" txt_i="18404" txt_f="19009">Although the diversity was lower in infected children, the range of diversity indicators was more heterogeneous in that group than in uninfected children. The variability in control samples at both sampling times (0 h and 24 h) ranged between 25.9–50.9, 419.1–742.1, 4.4–6.51 and 681.3–1147.8 for phylogenetic diversity, observed species, Shannon’s index and Chao 1 index, respectively. Identical indicators range between 19.2–50.1, 217.8–837.8, 4.5–7.0 and 352.6–1285 in samples from infected children, both at baseline and 24 h post-treatment. We did not observe any age-related differences (pre-school </offsets><italic><offsets xml_i="32726" xml_f="32728" txt_i="19009" txt_f="19011">vs</offsets></italic><offsets xml_i="32737" xml_f="32760" txt_i="19011" txt_f="19034"> school-aged children).</offsets></p></sec><sec id="Sec13"><title><offsets xml_i="32793" xml_f="32870" txt_i="19036" txt_f="19113">Association between praziquantel treatment and the gut microbiota composition</offsets></title><p id="Par28"><offsets xml_i="32892" xml_f="33254" txt_i="19114" txt_f="19476">For the 34 samples collected 24 hours after treatment, sequencing resulted in the analysis of a total of 4,595,622 reads, with an average of 135,165 curated reads (Standard deviation: 81,880) per sample. To elucidate the potential off-target effects of praziquantel on the microbiota composition, 24-hour samples from those who received praziquantel (non-cured) </offsets><italic><offsets xml_i="33262" xml_f="33268" txt_i="19476" txt_f="19482">versus</offsets></italic><offsets xml_i="33277" xml_f="33402" txt_i="19482" txt_f="19607"> those that received placebo were analysed. The taxonomic representation of the differential abundances is presented in Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="33434" xml_f="33435" txt_i="19607" txt_f="19608">4</offsets></xref><offsets xml_i="33442" xml_f="33443" txt_i="19608" txt_f="19609">.</offsets><fig id="Fig4"><label><offsets xml_i="33465" xml_f="33471" txt_i="19609" txt_f="19615">Fig. 4</offsets></label><caption><p><offsets xml_i="33491" xml_f="33551" txt_i="19615" txt_f="19675">Differences in the gut microbiota of children infected with </offsets><italic><offsets xml_i="33559" xml_f="33569" txt_i="19675" txt_f="19685">S. mansoni</offsets></italic><offsets xml_i="33578" xml_f="33852" txt_i="19685" txt_f="19959"> after administration of placebo or praziquantel. This cladogram shows the differences in bacterial composition of infected children receiving a placebo treatment (= Placebo) or infected children with failed treatment receiving a unique dose of 60 mg/kg praziquantel (= Low)</offsets></p></caption><graphic xlink:href="13071_2018_2739_Fig4_HTML" id="MO4"></graphic></fig></p><p id="Par29"><offsets xml_i="33957" xml_f="34055" txt_i="19961" txt_f="20059">Comparison of both treatment groups highlighted a few differences including a higher abundance of </offsets><italic><offsets xml_i="34063" xml_f="34070" txt_i="20059" txt_f="20066">Bacilli</offsets></italic><offsets xml_i="34079" xml_f="34084" txt_i="20066" txt_f="20071"> and </offsets><italic><offsets xml_i="34092" xml_f="34107" txt_i="20071" txt_f="20086">Erysipelotrichi</offsets></italic><offsets xml_i="34116" xml_f="34221" txt_i="20086" txt_f="20191"> taxa in volunteers who received praziquantel while some bacteria from the ML615J_28 order and the genus </offsets><italic><offsets xml_i="34229" xml_f="34243" txt_i="20191" txt_f="20205">Actinobacillus</offsets></italic><offsets xml_i="34252" xml_f="34293" txt_i="20205" txt_f="20246"> were more abundant in the placebo group.</offsets></p></sec><sec id="Sec14"><title><offsets xml_i="34326" xml_f="34420" txt_i="20248" txt_f="20342">Comparison of the gut microbiota of individuals with different praziquantel treatment outcomes</offsets></title><p id="Par30"><offsets xml_i="34442" xml_f="34591" txt_i="20343" txt_f="20492">Infected children were categorized after administration of a single dose of praziquantel in two groups, namely, (i) cured and (ii) not cured. Figure </offsets><xref rid="Fig5" ref-type="fig"><offsets xml_i="34623" xml_f="34624" txt_i="20492" txt_f="20493">5</offsets></xref><offsets xml_i="34631" xml_f="34684" txt_i="20493" txt_f="20546"> summarizes the main differences between both groups.</offsets><fig id="Fig5"><label><offsets xml_i="34706" xml_f="34712" txt_i="20546" txt_f="20552">Fig. 5</offsets></label><caption><p><offsets xml_i="34732" xml_f="34783" txt_i="20552" txt_f="20603">Comparison of microbiome in the treatment failures </offsets><italic><offsets xml_i="34791" xml_f="34797" txt_i="20603" txt_f="20609">versus</offsets></italic><offsets xml_i="34806" xml_f="34836" txt_i="20609" txt_f="20639"> successful treatment groups. </offsets><bold><offsets xml_i="34842" xml_f="34843" txt_i="20639" txt_f="20640">a</offsets></bold><offsets xml_i="34850" xml_f="34890" txt_i="20640" txt_f="20680"> Before administration of praziquantel. </offsets><bold><offsets xml_i="34896" xml_f="34897" txt_i="20680" txt_f="20681">b</offsets></bold><offsets xml_i="34904" xml_f="35094" txt_i="20681" txt_f="20871"> 24 hours after administration of 60 mg/kg praziquantel. Using the same statistical tests did not reveal any differences of the same groups for the 3 weeks follow-up samples (data not shown)</offsets></p></caption><graphic xlink:href="13071_2018_2739_Fig5_HTML" id="MO5"></graphic></fig></p><p id="Par31"><offsets xml_i="35199" xml_f="35307" txt_i="20873" txt_f="20981">For samples collected at baseline, before treatment, there was an overabundance of members from the classes </offsets><italic><offsets xml_i="35315" xml_f="35330" txt_i="20981" txt_f="20996">Fusobacteriales</offsets></italic><offsets xml_i="35339" xml_f="35341" txt_i="20996" txt_f="20998">, </offsets><italic><offsets xml_i="35349" xml_f="35362" txt_i="20998" txt_f="21011">Rickettsiales</offsets></italic><offsets xml_i="35371" xml_f="35376" txt_i="21011" txt_f="21016"> and </offsets><italic><offsets xml_i="35384" xml_f="35396" txt_i="21016" txt_f="21028">Neisseriales</offsets></italic><offsets xml_i="35405" xml_f="35579" txt_i="21028" txt_f="21202"> in volunteers with high treatment efficacy. In contrast, individuals that were not cured, harbored in their microbiome a higher abundance of bacteria belonging to the genus </offsets><italic><offsets xml_i="35587" xml_f="35596" txt_i="21202" txt_f="21211">Weissella</offsets></italic><offsets xml_i="35605" xml_f="35676" txt_i="21211" txt_f="21282">. There was a significantly higher abundance of members from the class </offsets><italic><offsets xml_i="35684" xml_f="35699" txt_i="21282" txt_f="21297">Fusobacteriales</offsets></italic><offsets xml_i="35708" xml_f="35747" txt_i="21297" txt_f="21336"> as well as of members from the genera </offsets><italic><offsets xml_i="35755" xml_f="35766" txt_i="21336" txt_f="21347">Clostridium</offsets></italic><offsets xml_i="35775" xml_f="35780" txt_i="21347" txt_f="21352"> and </offsets><italic><offsets xml_i="35788" xml_f="35802" txt_i="21352" txt_f="21366">Actinobacillus</offsets></italic><offsets xml_i="35811" xml_f="36001" txt_i="21366" txt_f="21556"> in cured volunteers, 24 hours after administration of praziquantel. At this time point, in children characterized by treatment failure, there was an overabundance of members from the class </offsets><italic><offsets xml_i="36009" xml_f="36016" txt_i="21556" txt_f="21563">Bacilli</offsets></italic><offsets xml_i="36025" xml_f="36075" txt_i="21563" txt_f="21613">. We used qPCR to confirm the results observed by </offsets><italic><offsets xml_i="36083" xml_f="36086" txt_i="21613" txt_f="21616">16S</offsets></italic><offsets xml_i="36095" xml_f="36115" txt_i="21616" txt_f="21636"> PCR for the family </offsets><italic><offsets xml_i="36123" xml_f="36139" txt_i="21636" txt_f="21652">Fusobacteriaceae</offsets></italic><offsets xml_i="36148" xml_f="36182" txt_i="21652" txt_f="21686">, by selecting an assay to detect </offsets><italic><offsets xml_i="36190" xml_f="36203" txt_i="21686" txt_f="21699">Fusobacterium</offsets></italic><offsets xml_i="36212" xml_f="36284" txt_i="21699" txt_f="21771"> spp., the most-represented genus from this family in our dataset (Fig. </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="36316" xml_f="36317" txt_i="21771" txt_f="21772">6</offsets></xref><offsets xml_i="36324" xml_f="36326" txt_i="21772" txt_f="21774">).</offsets><fig id="Fig6"><label><offsets xml_i="36348" xml_f="36354" txt_i="21774" txt_f="21780">Fig. 6</offsets></label><caption><p><italic><offsets xml_i="36382" xml_f="36395" txt_i="21780" txt_f="21793">Fusobacterium</offsets></italic><offsets xml_i="36404" xml_f="36416" txt_i="21793" txt_f="21805"> spp. qPCR. </offsets><bold><offsets xml_i="36422" xml_f="36423" txt_i="21805" txt_f="21806">a</offsets></bold><offsets xml_i="36430" xml_f="36451" txt_i="21806" txt_f="21827"> The copy number for </offsets><italic><offsets xml_i="36459" xml_f="36472" txt_i="21827" txt_f="21840">Fusobacterium</offsets></italic><offsets xml_i="36481" xml_f="36542" txt_i="21840" txt_f="21901"> spp. before treatment both in cured and non-cured children. </offsets><bold><offsets xml_i="36548" xml_f="36549" txt_i="21901" txt_f="21902">b</offsets></bold><offsets xml_i="36556" xml_f="36664" txt_i="21902" txt_f="22010"> The variation in copies number over the 24 hours post-treatment period in both cured and non-cured children</offsets></p></caption><graphic xlink:href="13071_2018_2739_Fig6_HTML" id="MO6"></graphic></fig></p><p id="Par32"><offsets xml_i="36769" xml_f="36813" txt_i="22012" txt_f="22056">The measured qPCR efficiency, on the DNA of </offsets><italic><offsets xml_i="36821" xml_f="36834" txt_i="22056" txt_f="22069">Fusobacterium</offsets></italic><offsets xml_i="36843" xml_f="36844" txt_i="22069" txt_f="22070">
</offsets><italic><offsets xml_i="36852" xml_f="36862" txt_i="22070" txt_f="22080"> nucleatum</offsets></italic><offsets xml_i="36871" xml_f="36952" txt_i="22080" txt_f="22161"> was 91.5%. Fluorimeter quantification of qPCR input is shown in Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="36988" xml_f="36989" txt_i="22161" txt_f="22162">1</offsets></xref><offsets xml_i="36996" xml_f="37008" txt_i="22162" txt_f="22174">: Table S1. </offsets><italic><offsets xml_i="37016" xml_f="37029" txt_i="22174" txt_f="22187">Fusobacterium</offsets></italic><offsets xml_i="37038" xml_f="37111" txt_i="22187" txt_f="22260"> spp. is significantly more abundant in cured children before treatment (</offsets><italic><offsets xml_i="37119" xml_f="37120" txt_i="22260" txt_f="22261">P</offsets></italic><offsets xml_i="37129" xml_f="37180" txt_i="22261" txt_f="22312"> = 0.018). The median normalized cycle threshold (C</offsets><sub><offsets xml_i="37185" xml_f="37186" txt_i="22312" txt_f="22313">t</offsets></sub><offsets xml_i="37192" xml_f="37422" txt_i="22313" txt_f="22543">) increase was +2.42 cycles between pre- and 24 hours post-treatment in fully cured children, equivalent to a median decrease in copy number is of -11,091 copies. In the population of children for which the treatment failed, the C</offsets><sub><offsets xml_i="37427" xml_f="37428" txt_i="22543" txt_f="22544">t</offsets></sub><offsets xml_i="37434" xml_f="37522" txt_i="22544" txt_f="22632"> value remained stable over the 24 h period, with a median of -0.17 cycles. The median C</offsets><sub><offsets xml_i="37527" xml_f="37528" txt_i="22632" txt_f="22633">t</offsets></sub><offsets xml_i="37534" xml_f="37643" txt_i="22633" txt_f="22742"> was 29.37 and 29.72 in the cured and non-cured groups, respectively, in the 24 hours post-treatment samples.</offsets></p><p id="Par33"><offsets xml_i="37661" xml_f="37678" txt_i="22743" txt_f="22760">We compared the C</offsets><sub><offsets xml_i="37683" xml_f="37684" txt_i="22760" txt_f="22761">t</offsets></sub><offsets xml_i="37690" xml_f="37783" txt_i="22761" txt_f="22854"> values in placebo samples, to see whether associations exist between egg reduction rate and </offsets><italic><offsets xml_i="37791" xml_f="37804" txt_i="22854" txt_f="22867">Fusobacterium</offsets></italic><offsets xml_i="37813" xml_f="37872" txt_i="22867" txt_f="22926"> spp. abundance. In the samples that showed an increase in </offsets><italic><offsets xml_i="37880" xml_f="37893" txt_i="22926" txt_f="22939">Fusobacterium</offsets></italic><offsets xml_i="37902" xml_f="37937" txt_i="22939" txt_f="22974"> spp. abundance over the 24 hours (</offsets><italic><offsets xml_i="37945" xml_f="37946" txt_i="22974" txt_f="22975">n</offsets></italic><offsets xml_i="37955" xml_f="38083" txt_i="22975" txt_f="23103"> = 5 samples; median = +1.21 cycle), the percentage of remaining eggs at follow-up was 66% while in those who had a decrease in </offsets><italic><offsets xml_i="38091" xml_f="38104" txt_i="23103" txt_f="23116">Fusobacterium</offsets></italic><offsets xml_i="38113" xml_f="38130" txt_i="23116" txt_f="23133"> spp. abundance (</offsets><italic><offsets xml_i="38138" xml_f="38139" txt_i="23133" txt_f="23134">n</offsets></italic><offsets xml_i="38148" xml_f="38259" txt_i="23134" txt_f="23245"> = 6 samples; median = -3.02 cycle), the average percentage of remaining eggs in the follow-up samples was 34%.</offsets></p></sec></sec><sec id="Sec15"><title><offsets xml_i="38298" xml_f="38308" txt_i="23248" txt_f="23258">Discussion</offsets></title><p id="Par34"><offsets xml_i="38330" xml_f="38406" txt_i="23259" txt_f="23335">In this study, we describe for the first time the gut microbiome changes in </offsets><italic><offsets xml_i="38414" xml_f="38424" txt_i="23335" txt_f="23345">S. mansoni</offsets></italic><offsets xml_i="38433" xml_f="38581" txt_i="23345" txt_f="23493">-infected children and their potential impact on treatment outcome based on oral praziquantel application in an endemic rural area in Côte d’Ivoire.</offsets></p><sec id="Sec16"><title><offsets xml_i="38608" xml_f="38667" txt_i="23494" txt_f="23553">Associations between schistosomiasis and the gut microbiome</offsets></title><p id="Par35"><offsets xml_i="38689" xml_f="38776" txt_i="23554" txt_f="23641">Four bacterial phyla are considered to dominate in the human gut microbiome, including </offsets><italic><offsets xml_i="38784" xml_f="38794" txt_i="23641" txt_f="23651">Firmicutes</offsets></italic><offsets xml_i="38803" xml_f="38805" txt_i="23651" txt_f="23653">, </offsets><italic><offsets xml_i="38813" xml_f="38826" txt_i="23653" txt_f="23666">Bacteroidetes</offsets></italic><offsets xml_i="38835" xml_f="38837" txt_i="23666" txt_f="23668">, </offsets><italic><offsets xml_i="38845" xml_f="38859" txt_i="23668" txt_f="23682">Actinobacteria</offsets></italic><offsets xml_i="38868" xml_f="38873" txt_i="23682" txt_f="23687"> and </offsets><italic><offsets xml_i="38881" xml_f="38895" txt_i="23687" txt_f="23701">Proteobacteria</offsets></italic><offsets xml_i="38904" xml_f="38906" txt_i="23701" txt_f="23703"> [</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="38939" xml_f="38941" txt_i="23703" txt_f="23705">12</offsets></xref><offsets xml_i="38948" xml_f="38950" txt_i="23705" txt_f="23707">, </offsets><xref ref-type="bibr" rid="CR31"><offsets xml_i="38983" xml_f="38985" txt_i="23707" txt_f="23709">31</offsets></xref><offsets xml_i="38992" xml_f="39139" txt_i="23709" txt_f="23856">]. The comparison of the gut microbiota profiles of children from the health district of Azaguié in Côte d’Ivoire, whether they were infected with </offsets><italic><offsets xml_i="39147" xml_f="39157" txt_i="23856" txt_f="23866">S. mansoni</offsets></italic><offsets xml_i="39166" xml_f="39383" txt_i="23866" txt_f="24083"> or not, confirmed this finding. Abundance ratios of these four dominating phyla are usually good indicators for dysbiosis and have been linked to various health conditions, including colitis and metabolic disorders [</offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="39416" xml_f="39418" txt_i="24083" txt_f="24085">32</offsets></xref><offsets xml_i="39425" xml_f="39427" txt_i="24085" txt_f="24087">, </offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="39460" xml_f="39462" txt_i="24087" txt_f="24089">33</offsets></xref><offsets xml_i="39469" xml_f="39613" txt_i="24089" txt_f="24233">]. Relative abundances of the different bacterial phyla depicted in this study correspond to those expected in normal human gut structure, with </offsets><italic><offsets xml_i="39621" xml_f="39631" txt_i="24233" txt_f="24243">Firmicutes</offsets></italic><offsets xml_i="39640" xml_f="39646" txt_i="24243" txt_f="24246"> &gt; </offsets><italic><offsets xml_i="39654" xml_f="39667" txt_i="24246" txt_f="24259">Bacteroidetes</offsets></italic><offsets xml_i="39676" xml_f="39682" txt_i="24259" txt_f="24262"> &gt; </offsets><italic><offsets xml_i="39690" xml_f="39704" txt_i="24262" txt_f="24276">Proteobacteria</offsets></italic><offsets xml_i="39713" xml_f="39773" txt_i="24276" txt_f="24336">, except for six children infected with schistosomes. While </offsets><italic><offsets xml_i="39781" xml_f="39795" txt_i="24336" txt_f="24350">Proteobacteria</offsets></italic><offsets xml_i="39804" xml_f="39892" txt_i="24350" txt_f="24438"> are commensals of the gut microbiome, they are usually present in lower abundance than </offsets><italic><offsets xml_i="39900" xml_f="39910" txt_i="24438" txt_f="24448">Firmicutes</offsets></italic><offsets xml_i="39919" xml_f="39924" txt_i="24448" txt_f="24453"> and </offsets><italic><offsets xml_i="39932" xml_f="39945" txt_i="24453" txt_f="24466">Bacteroidetes</offsets></italic><offsets xml_i="39954" xml_f="39979" txt_i="24466" txt_f="24491"> in healthy individuals [</offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="40012" xml_f="40014" txt_i="24491" txt_f="24493">32</offsets></xref><offsets xml_i="40021" xml_f="40270" txt_i="24493" txt_f="24742">]. An outgrowth of bacteria from this taxon is typical of a major dysbiosis with clinical relevance, as several bacterial species from this phylum have been linked with pathogenesis in humans, resulting in either severe disease burden and/or death [</offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="40303" xml_f="40305" txt_i="24742" txt_f="24744">32</offsets></xref><offsets xml_i="40312" xml_f="40314" txt_i="24744" txt_f="24746">, </offsets><xref ref-type="bibr" rid="CR34"><offsets xml_i="40347" xml_f="40349" txt_i="24746" txt_f="24748">34</offsets></xref><offsets xml_i="40356" xml_f="40357" txt_i="24748" txt_f="24749">–</offsets><xref ref-type="bibr" rid="CR36"><offsets xml_i="40390" xml_f="40392" txt_i="24749" txt_f="24751">36</offsets></xref><offsets xml_i="40399" xml_f="40467" txt_i="24751" txt_f="24819">]. In this specific population, the cumulated relative abundance of </offsets><italic><offsets xml_i="40475" xml_f="40485" txt_i="24819" txt_f="24829">Firmicutes</offsets></italic><offsets xml_i="40494" xml_f="40499" txt_i="24829" txt_f="24834"> and </offsets><italic><offsets xml_i="40507" xml_f="40520" txt_i="24834" txt_f="24847">Bacteroidetes</offsets></italic><offsets xml_i="40529" xml_f="40737" txt_i="24847" txt_f="25055"> represents more than 50% of the total bacterial composition in all but three samples. Interestingly, all three children, characterized by an untypical microbiome present a clear dysbiosis towards the phylum </offsets><italic><offsets xml_i="40745" xml_f="40759" txt_i="25055" txt_f="25069">Proteobacteria</offsets></italic><offsets xml_i="40768" xml_f="41055" txt_i="25069" txt_f="25356"> and in all cases, the clinical assessment showed an outcome of clinical relevance, be it blood in stool and splenomegaly, simple splenomegaly or vomiting within three hours after praziquantel administration (data not shown). At the family level, we observed that members from the clade </offsets><italic><offsets xml_i="41063" xml_f="41076" txt_i="25356" txt_f="25369">Prevotellacae</offsets></italic><offsets xml_i="41085" xml_f="41294" txt_i="25369" txt_f="25578">, another important group for healthy gut functionality and which has been linked to various chronic diseases (e.g. rheumatoid arthritis or obesity), has a highly heterogeneous distribution among our samples [</offsets><xref ref-type="bibr" rid="CR37"><offsets xml_i="41327" xml_f="41329" txt_i="25578" txt_f="25580">37</offsets></xref><offsets xml_i="41336" xml_f="41338" txt_i="25580" txt_f="25582">, </offsets><xref ref-type="bibr" rid="CR38"><offsets xml_i="41371" xml_f="41373" txt_i="25582" txt_f="25584">38</offsets></xref><offsets xml_i="41380" xml_f="41382" txt_i="25584" txt_f="25586">].</offsets></p><p id="Par36"><offsets xml_i="41400" xml_f="41603" txt_i="25587" txt_f="25790">Diversity indices commonly used in ecological research, including the phylogenetic diversity, the number of observed species, the Shannon’s and the Chao 1 indices, are accepted indicators for dysbiosis [</offsets><xref ref-type="bibr" rid="CR39"><offsets xml_i="41636" xml_f="41638" txt_i="25790" txt_f="25792">39</offsets></xref><offsets xml_i="41645" xml_f="41646" txt_i="25792" txt_f="25793">–</offsets><xref ref-type="bibr" rid="CR42"><offsets xml_i="41679" xml_f="41681" txt_i="25793" txt_f="25795">42</offsets></xref><offsets xml_i="41688" xml_f="41825" txt_i="25795" txt_f="25932">]. In our study, we observed a trend that these different indicators show an overall lower bacterial diversity in children infected with </offsets><italic><offsets xml_i="41833" xml_f="41843" txt_i="25932" txt_f="25942">S. mansoni</offsets></italic><offsets xml_i="41852" xml_f="42241" txt_i="25942" txt_f="26331">. However, the inter-individual diversity varies more in this group and could reflect a less stable gut microbiota in infected children. This result fills a gap in our knowledge of associations of intestinal parasites and the gut microbiota structure. To date it has been shown that different intestinal parasites might increase, decrease or have no effect on the gut microbial diversity [</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="42273" xml_f="42274" txt_i="26331" txt_f="26332">6</offsets></xref><offsets xml_i="42281" xml_f="42283" txt_i="26332" txt_f="26334">, </offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="42315" xml_f="42316" txt_i="26334" txt_f="26335">7</offsets></xref><offsets xml_i="42323" xml_f="42325" txt_i="26335" txt_f="26337">, </offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="42358" xml_f="42360" txt_i="26337" txt_f="26339">43</offsets></xref><offsets xml_i="42367" xml_f="42369" txt_i="26339" txt_f="26341">].</offsets></p><p id="Par37"><offsets xml_i="42387" xml_f="42408" txt_i="26342" txt_f="26363">Associations between </offsets><italic><offsets xml_i="42416" xml_f="42426" txt_i="26363" txt_f="26373">S. mansoni</offsets></italic><offsets xml_i="42435" xml_f="42849" txt_i="26373" txt_f="26787"> infection and the usual indicators of a healthy gut microbiome are relatively subtle. While we observe modifications that seem to be correlated with the presence of schistosomes, both in the phyla ratios and diversity indicators, the same modifications are not distributed homogeneously among all infected patients. This suggests that the gut microbiota stability is affected to some extent by the infection with </offsets><italic><offsets xml_i="42857" xml_f="42867" txt_i="26787" txt_f="26797">S. mansoni</offsets></italic><offsets xml_i="42876" xml_f="43184" txt_i="26797" txt_f="27105">, but that other factors, including the host’s specific genetic makeup in combination with environmental factors, such as nutrition, play a greater role in the composition of the gut microbiome, in this context. This observation is in partial agreement with the conclusion of the recent study of Kay et al. [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="43217" xml_f="43219" txt_i="27105" txt_f="27107">44</offsets></xref><offsets xml_i="43226" xml_f="43257" txt_i="27107" txt_f="27138">], stating that infection with </offsets><italic><offsets xml_i="43265" xml_f="43288" txt_i="27138" txt_f="27161">Schistosoma haematobium</offsets></italic><offsets xml_i="43297" xml_f="43399" txt_i="27161" txt_f="27263">, although being a bladder infecting schistosome, is associated with variations in the gut microbiota.</offsets></p></sec><sec id="Sec17"><title><offsets xml_i="43432" xml_f="43491" txt_i="27265" txt_f="27324">Effect of praziquantel administration on the gut microbiome</offsets></title><p id="Par38"><offsets xml_i="43513" xml_f="43790" txt_i="27325" txt_f="27602">Adverse effects from the orally administered praziquantel are usually mild or moderate and there are gut-related side effects of praziquantel (abdominal pain, cramps, with or without nausea and vomiting) that could potentially be related to a drug-induced microbial dysbiosis [</offsets><xref ref-type="bibr" rid="CR45"><offsets xml_i="43823" xml_f="43825" txt_i="27602" txt_f="27604">45</offsets></xref><offsets xml_i="43832" xml_f="43987" txt_i="27604" txt_f="27759">]. We compared the gut microbiotas of children 24 hours after administration of either a placebo or a high single praziquantel dose (60 mg/kg). The clades </offsets><italic><offsets xml_i="43995" xml_f="44010" txt_i="27759" txt_f="27774">Erysipelotrichi</offsets></italic><offsets xml_i="44019" xml_f="44024" txt_i="27774" txt_f="27779"> and </offsets><italic><offsets xml_i="44032" xml_f="44039" txt_i="27779" txt_f="27786">Bacilli</offsets></italic><offsets xml_i="44048" xml_f="44134" txt_i="27786" txt_f="27872"> were overabundant in stools produced by praziquantel-treated children. For the clade </offsets><italic><offsets xml_i="44142" xml_f="44157" txt_i="27872" txt_f="27887">Erysipelotrichi</offsets></italic><offsets xml_i="44166" xml_f="44323" txt_i="27887" txt_f="28044">, this information correlates with already published data, which states that this clade might be overabundant in patients receiving an antibiotic treatment [</offsets><xref ref-type="bibr" rid="CR46"><offsets xml_i="44356" xml_f="44358" txt_i="28044" txt_f="28046">46</offsets></xref><offsets xml_i="44365" xml_f="44396" txt_i="28046" txt_f="28077">]. Some members from the class </offsets><italic><offsets xml_i="44404" xml_f="44411" txt_i="28077" txt_f="28084">Bacilli</offsets></italic><offsets xml_i="44420" xml_f="44492" txt_i="28084" txt_f="28156"> were shown to be potentially highly efficient xenobiotic metabolizers [</offsets><xref ref-type="bibr" rid="CR47"><offsets xml_i="44525" xml_f="44527" txt_i="28156" txt_f="28158">47</offsets></xref><offsets xml_i="44534" xml_f="44536" txt_i="28158" txt_f="28160">, </offsets><xref ref-type="bibr" rid="CR48"><offsets xml_i="44569" xml_f="44571" txt_i="28160" txt_f="28162">48</offsets></xref><offsets xml_i="44578" xml_f="44800" txt_i="28162" txt_f="28384">]. However, since we did not study these bacteria at a lower taxonomic level, we cannot draw a more precise conclusion about this specific effect. At the time of this study, no published data were available linking either </offsets><italic><offsets xml_i="44808" xml_f="44823" txt_i="28384" txt_f="28399">Erysipelotrichi</offsets></italic><offsets xml_i="44832" xml_f="44836" txt_i="28399" txt_f="28403"> or </offsets><italic><offsets xml_i="44844" xml_f="44851" txt_i="28403" txt_f="28410">Bacilli</offsets></italic><offsets xml_i="44860" xml_f="45344" txt_i="28410" txt_f="28894"> clades to gastrointestinal complications. Therefore, since the structural changes in microbiota composition remain moderate and restricted to these two clades, we hypothesize that they do not contribute to the gut-related side effects of praziquantel administration. We can therefore conclude that praziquantel-related side effects are not related to indirect effects on the gut microbiome and that they are mostly explained by other factors, potentially including expulsion of dead </offsets><italic><offsets xml_i="45352" xml_f="45362" txt_i="28894" txt_f="28904">S. mansoni</offsets></italic><offsets xml_i="45371" xml_f="45423" txt_i="28904" txt_f="28956"> adult worms, host’s genetics and/or degradation of </offsets><italic><offsets xml_i="45431" xml_f="45441" txt_i="28956" txt_f="28966">S. mansoni</offsets></italic><offsets xml_i="45450" xml_f="45522" txt_i="28966" txt_f="29038"> eggs triggering the host’s inflammatory response, as previously shown [</offsets><xref ref-type="bibr" rid="CR49"><offsets xml_i="45555" xml_f="45557" txt_i="29038" txt_f="29040">49</offsets></xref><offsets xml_i="45564" xml_f="45566" txt_i="29040" txt_f="29042">, </offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="45599" xml_f="45601" txt_i="29042" txt_f="29044">50</offsets></xref><offsets xml_i="45608" xml_f="45774" txt_i="29044" txt_f="29210">]. This observation, while based on a restricted amount of samples, gives new insights on conclusions made in a previous study on off-target effects of praziquantel [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="45807" xml_f="45809" txt_i="29210" txt_f="29212">44</offsets></xref><offsets xml_i="45816" xml_f="45850" txt_i="29212" txt_f="29246">]. While the study of Kay et al. [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="45883" xml_f="45885" txt_i="29246" txt_f="29248">44</offsets></xref><offsets xml_i="45892" xml_f="46171" txt_i="29248" txt_f="29527">] shows that praziquantel does not have an effect on the gut microbiota in the long term (12 weeks post-administration), we complement this information by stating that praziquantel does in fact have a slight short-term effect (24 hours post-administration) on the gut microbiota.</offsets></p></sec><sec id="Sec18"><title><offsets xml_i="46204" xml_f="46248" txt_i="29529" txt_f="29573">Differences in the microbiome of successful </offsets><italic><offsets xml_i="46256" xml_f="46262" txt_i="29573" txt_f="29579">versus</offsets></italic><offsets xml_i="46271" xml_f="46289" txt_i="29579" txt_f="29597"> failed treatments</offsets></title><p id="Par39"><offsets xml_i="46311" xml_f="47082" txt_i="29598" txt_f="30366">As a final part of this study, we investigated potential bacterial factors in the gut microbiome that might explain the variations in treatment response to praziquantel. For this purpose, we stratified the samples from infected children into two groups based on the cure rates observed within the 3-week follow-up period. Of note, non-cured children showed on average an egg reduction rate of &lt; 60%, which is way below the reference drug efficacy value of ≥ 90% set by WHO, hence praziquantel did not fulfil the criteria of clinical efficacy in these children. However, key pharmacokinetic parameters (AUC, Cmax) did not show a significant difference between cured and not cured children (data not shown) hence we can rule out that children did not take the treatment.</offsets></p><p id="Par40"><offsets xml_i="47100" xml_f="47202" txt_i="30367" txt_f="30469">At baseline, the main differences were driven partly by endosymbionts of plant cells from the classes </offsets><italic><offsets xml_i="47210" xml_f="47223" txt_i="30469" txt_f="30482">Rickettsiales</offsets></italic><offsets xml_i="47232" xml_f="47237" txt_i="30482" txt_f="30487"> and </offsets><italic><offsets xml_i="47245" xml_f="47257" txt_i="30487" txt_f="30499">Neisseriales</offsets></italic><offsets xml_i="47266" xml_f="47404" txt_i="30499" txt_f="30637">. For both classes, there is no specific literature describing their potential to modulate xenobiotics metabolism. Members from the class </offsets><italic><offsets xml_i="47412" xml_f="47427" txt_i="30637" txt_f="30652">Fusobacteriales</offsets></italic><offsets xml_i="47436" xml_f="47767" txt_i="30652" txt_f="30983"> were also significantly more abundant in the successfully treated group at baseline. Twenty-four hours post-treatment, it is interesting to note that members from the same class were still significantly more abundant in the treatment success group. The qPCR assays confirmed a significant difference in the abundance of the genus </offsets><italic><offsets xml_i="47775" xml_f="47788" txt_i="30983" txt_f="30996">Fusobacterium</offsets></italic><offsets xml_i="47797" xml_f="47911" txt_i="30996" txt_f="31110">, a genus populated with potential pathogens, in the pre-treatment samples, between cured and non-cured patients [</offsets><xref ref-type="bibr" rid="CR51"><offsets xml_i="47944" xml_f="47946" txt_i="31110" txt_f="31112">51</offsets></xref><offsets xml_i="47953" xml_f="48118" txt_i="31112" txt_f="31277">]. This genus is particularly interesting as it includes species known to have inflammatory properties and species that have been linked to chronic disease, such as </offsets><italic><offsets xml_i="48126" xml_f="48139" txt_i="31277" txt_f="31290">Fusobacterium</offsets></italic><offsets xml_i="48148" xml_f="48149" txt_i="31290" txt_f="31291">
</offsets><italic><offsets xml_i="48157" xml_f="48167" txt_i="31291" txt_f="31301"> nucleatum</offsets></italic><offsets xml_i="48176" xml_f="48258" txt_i="31301" txt_f="31383">, that could be closely associated with the worm-triggered inflammatory response [</offsets><xref ref-type="bibr" rid="CR52"><offsets xml_i="48291" xml_f="48293" txt_i="31383" txt_f="31385">52</offsets></xref><offsets xml_i="48300" xml_f="48302" txt_i="31385" txt_f="31387">, </offsets><xref ref-type="bibr" rid="CR53"><offsets xml_i="48335" xml_f="48337" txt_i="31387" txt_f="31389">53</offsets></xref><offsets xml_i="48344" xml_f="48378" txt_i="31389" txt_f="31423">]. Our results indicate that high </offsets><italic><offsets xml_i="48386" xml_f="48399" txt_i="31423" txt_f="31436">Fusobacterium</offsets></italic><offsets xml_i="48408" xml_f="48607" txt_i="31436" txt_f="31635"> spp. abundance before treatment and high decrease rate of the bacteria over the first 24 hours post-treatment is correlated with a better outcome of praziquantel treatment. This could indicate that </offsets><italic><offsets xml_i="48615" xml_f="48628" txt_i="31635" txt_f="31648">Fusobacterium</offsets></italic><offsets xml_i="48637" xml_f="48657" txt_i="31648" txt_f="31668"> spp. abundance and </offsets><italic><offsets xml_i="48665" xml_f="48675" txt_i="31668" txt_f="31678">S. mansoni</offsets></italic><offsets xml_i="48684" xml_f="48751" txt_i="31678" txt_f="31745"> infection are dynamically linked and that preliminary presence of </offsets><italic><offsets xml_i="48759" xml_f="48772" txt_i="31745" txt_f="31758">Fusobacterium</offsets></italic><offsets xml_i="48781" xml_f="48889" txt_i="31758" txt_f="31866"> spp. might condition the outcome of schistosomiasis treatment while being independent from the drug itself.</offsets></p></sec><sec id="Sec19"><title><offsets xml_i="48922" xml_f="48933" txt_i="31868" txt_f="31879">Limitations</offsets></title><p id="Par41"><offsets xml_i="48955" xml_f="49430" txt_i="31880" txt_f="32355">While this study does show preliminary results on gut composition related differences that may explain discrepancies in treatment outcome with praziquantel, it remains based on this specific paediatric population and a larger cohort is needed to confirm these results. For instance, additional patients with low-intensity of infection and high cure rate would be needed to draw significant conclusions as to what pertains the impact of different intensity of infections with </offsets><italic><offsets xml_i="49438" xml_f="49448" txt_i="32355" txt_f="32365">S. mansoni</offsets></italic><offsets xml_i="49457" xml_f="49655" txt_i="32365" txt_f="32563"> on the gut microbiota. Also, additional investigations are required to fully understand the potential role and mechanism of action of these factors on praziquantel efficacy against schistosomiasis.</offsets></p></sec></sec><sec id="Sec20"><title><offsets xml_i="49694" xml_f="49705" txt_i="32566" txt_f="32577">Conclusions</offsets></title><p id="Par42"><offsets xml_i="49727" xml_f="49794" txt_i="32578" txt_f="32645">This study highlighted three aspects of the interrelations between </offsets><italic><offsets xml_i="49802" xml_f="49812" txt_i="32645" txt_f="32655">S. mansoni</offsets></italic><offsets xml_i="49821" xml_f="49892" txt_i="32655" txt_f="32726">, praziquantel and the gut microbiota, namely; (i) the associations of </offsets><italic><offsets xml_i="49900" xml_f="49910" txt_i="32726" txt_f="32736">S. mansoni</offsets></italic><offsets xml_i="49919" xml_f="50154" txt_i="32736" txt_f="32971"> and the usual indicators for a healthy gut microbiome seem to lay more in subtle modifications than in major compositional shifts. While we observed some modifications in the gut microbiome that seem to be specific to the presence of </offsets><italic><offsets xml_i="50162" xml_f="50172" txt_i="32971" txt_f="32981">S. mansoni</offsets></italic><offsets xml_i="50181" xml_f="50954" txt_i="32981" txt_f="33754">, the same modifications were not distributed evenly among infected patients; (ii) praziquantel administration had, similarly, relatively moderate effects on the gut microbiome. The observed effects on the microbiota are limited to two bacterial groups for which there is, so far, no proof of pathogenesis in human in the literature, but rather associations with responses to drug treatment. Therefore, we conclude that the gastrointestinal side effects observed in praziquantel administration are not related to an off-target effect of the drug on the microbial communities in the gut; and (iii) we were also able to highlight a bacterial taxon that could potentially play a role in the variations observed for schistosomiasis treatment outcomes with praziquantel, namely </offsets><italic><offsets xml_i="50962" xml_f="50975" txt_i="33754" txt_f="33767">Fusobacterium</offsets></italic><offsets xml_i="50984" xml_f="51067" txt_i="33767" txt_f="33850"> spp. Further investigations are needed to understand and characterize the role of </offsets><italic><offsets xml_i="51075" xml_f="51088" txt_i="33850" txt_f="33863">Fusobacterium</offsets></italic><offsets xml_i="51097" xml_f="51157" txt_i="33863" txt_f="33923"> spp. and their role in oral praziquantel treatment success.</offsets></p></sec><sec sec-type="supplementary-material"><title><offsets xml_i="51213" xml_f="51229" txt_i="33925" txt_f="33941">Additional files</offsets></title><sec id="Sec21"><p><offsets xml_i="51256" xml_f="51257" txt_i="33942" txt_f="33943">
</offsets><supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="13071_2018_2739_MOESM1_ESM.xlsx"><label><offsets xml_i="51378" xml_f="51396" txt_i="33943" txt_f="33961">Additional file 1:</offsets></label><caption><p><bold><offsets xml_i="51422" xml_f="51431" txt_i="33961" txt_f="33970">Table S1.</offsets></bold><offsets xml_i="51438" xml_f="51517" txt_i="33970" txt_f="34049"> Additional information about the children included in this study. (XLSX 12 kb)</offsets></p></caption></media></supplementary-material><offsets xml_i="51564" xml_f="51565" txt_i="34050" txt_f="34051">
</offsets><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="13071_2018_2739_MOESM2_ESM.xlsx"><label><offsets xml_i="51686" xml_f="51704" txt_i="34051" txt_f="34069">Additional file 2:</offsets></label><caption><p><bold><offsets xml_i="51730" xml_f="51739" txt_i="34069" txt_f="34078">Table S2.</offsets></bold><offsets xml_i="51746" xml_f="51874" txt_i="34078" txt_f="34206"> Barcodes and DNA concentrations. Summary table showing the pooling barcodes and pre-sequencing DNA concentrations. (XLSX 10 kb)</offsets></p></caption></media></supplementary-material><offsets xml_i="51921" xml_f="51922" txt_i="34207" txt_f="34208">
</offsets></p></sec></sec></body><back><fn-group><fn><p><bold>Electronic supplementary material</bold></p><p>The online version of this article (10.1186/s13071-018-2739-2) contains supplementary material, which is available to authorized users.</p></fn></fn-group><ack><title>Acknowledgements</title><p>Not applicable.</p><sec id="FPar1"><title>Funding</title><p id="Par43">Funding was obtained by the European Research Council (ERC-2013-CoG 614739-A_HERO).</p></sec><sec id="FPar2"><title>Availability of data and materials</title><p id="Par44">Sequencing datasets are available upon a request to the corresponding author.</p></sec></ack><notes notes-type="author-contribution"><title>Authors’ contributions</title><p>Conceived and designed the experiments: PHHS. Performed the experiments: PHHS, JTC, GP and MG. Analyzed the data: PHHS and JK. Contributed to reagents/materials/analysis tools: JEF, CD and JK. Contributed significantly to the manuscript: PHHS, JTC, GP, CD, MG, JEF and JK. All authors read and approved the final manuscript.</p></notes><notes notes-type="COI-statement"><sec id="FPar3"><title>Ethics approval and consent to participate</title><p id="Par45">Signed assent was obtained from children in addition to a written informed consent from a parent or legal guardian. Participation was voluntary and children had the right to withdraw from the study at any given point in time with no further obligations. Ethical approval for the study was obtained by the National Ethics Committee of the Ministry of Health in Côte d’Ivoire (CNER, reference no. 037/MSLS/CNER-dkn) and the Ethical Committee of Northwestern and Central Switzerland (EKNZ, reference no. 162/2014).</p></sec><sec id="FPar4"><title>Consent for publication</title><p id="Par46">Consent to publish from the participant (or legal parent or guardian for children) to report individual patient data was obtained.</p></sec><sec id="FPar5"><title>Competing interests</title><p id="Par47">The authors declare that they have no competing interests.</p></sec><sec id="FPar6"><title>Publisher’s Note</title><p id="Par48">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></sec></notes><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Gryseels</surname><given-names>B</given-names></name><name><surname>Polman</surname><given-names>K</given-names></name><name><surname>Clerinx</surname><given-names>J</given-names></name><name><surname>Kestens</surname><given-names>L</given-names></name></person-group><article-title>Human schistosomiasis</article-title><source>Lancet</source><year>2006</year><volume>368</volume><issue>9541</issue><fpage>1106</fpage><lpage>1118</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(06)69440-3</pub-id><pub-id pub-id-type="pmid">16997665</pub-id></element-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hotez</surname><given-names>PJ</given-names></name><name><surname>Fenwick</surname><given-names>A</given-names></name><name><surname>Savioli</surname><given-names>L</given-names></name><name><surname>Molyneux</surname><given-names>DH</given-names></name></person-group><article-title>Rescuing the bottom billion through control of neglected tropical diseases</article-title><source>Lancet</source><year>2009</year><volume>373</volume><issue>9674</issue><fpage>1570</fpage><lpage>1575</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(09)60233-6</pub-id><pub-id pub-id-type="pmid">19410718</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Abou-Zeid</surname><given-names>AHA</given-names></name><name><surname>Abkar</surname><given-names>TA</given-names></name><name><surname>Mohamed</surname><given-names>RO</given-names></name></person-group><article-title>Schistosomiasis and soil-transmitted helminths among an adult population in a war affected area, Southern Kordofan state, Sudan</article-title><source>Parasit Vectors</source><year>2012</year><volume>5</volume><issue>1</issue><fpage>1</fpage><pub-id pub-id-type="doi">10.1186/1756-3305-5-133</pub-id><pub-id pub-id-type="pmid">22212459</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Murray</surname><given-names>CJ</given-names></name><name><surname>Barber</surname><given-names>RM</given-names></name><name><surname>Foreman</surname><given-names>KJ</given-names></name><name><surname>Abbasoglu Ozgoren</surname><given-names>A</given-names></name><name><surname>Abd-Allah</surname><given-names>F</given-names></name><name><surname>Abera</surname><given-names>SF</given-names></name><etal></etal></person-group><article-title>Global, regional, and national disability-adjusted life years (DALYs) for 306 diseases and injuries and healthy life expectancy (HALE) for 188 countries, 1990–2013: quantifying the epidemiological transition</article-title><source>Lancet</source><year>2015</year><volume>386</volume><issue>10009</issue><fpage>2145</fpage><lpage>2191</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(15)61340-X</pub-id><pub-id pub-id-type="pmid">26321261</pub-id></element-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>King</surname><given-names>CH</given-names></name></person-group><article-title>Parasites and poverty: the case of schistosomiasis</article-title><source>Acta Trop</source><year>2010</year><volume>113</volume><issue>2</issue><fpage>95</fpage><lpage>104</lpage><pub-id pub-id-type="doi">10.1016/j.actatropica.2009.11.012</pub-id><pub-id pub-id-type="pmid">19962954</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mutapi</surname><given-names>F</given-names></name></person-group><article-title>The gut microbiome in the helminth infected host</article-title><source>Trends Parasitol</source><year>2015</year><volume>31</volume><issue>9</issue><fpage>405</fpage><lpage>406</lpage><pub-id pub-id-type="doi">10.1016/j.pt.2015.06.003</pub-id><pub-id pub-id-type="pmid">26142923</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Glendinning</surname><given-names>L</given-names></name><name><surname>Nausch</surname><given-names>N</given-names></name><name><surname>Free</surname><given-names>A</given-names></name><name><surname>Taylor</surname><given-names>DW</given-names></name><name><surname>Mutapi</surname><given-names>F</given-names></name></person-group><article-title>The microbiota and helminths: sharing the same niche in the human host</article-title><source>Parasitology</source><year>2014</year><volume>141</volume><issue>10</issue><fpage>1255</fpage><lpage>1271</lpage><pub-id pub-id-type="doi">10.1017/S0031182014000699</pub-id><pub-id pub-id-type="pmid">24901211</pub-id></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Doenhoff</surname><given-names>MJ</given-names></name><name><surname>Cioli</surname><given-names>D</given-names></name><name><surname>Utzinger</surname><given-names>J</given-names></name></person-group><article-title>Praziquantel: mechanisms of action, resistance and new derivatives for schistosomiasis</article-title><source>Curr Opin Infect Dis</source><year>2008</year><volume>21</volume><issue>6</issue><fpage>659</fpage><lpage>667</lpage><pub-id pub-id-type="doi">10.1097/QCO.0b013e328318978f</pub-id><pub-id pub-id-type="pmid">18978535</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Gönnert</surname><given-names>R</given-names></name><name><surname>Andrews</surname><given-names>P</given-names></name></person-group><article-title>Praziquantel, a new broad-spectrum antischistosomal agent</article-title><source>Parasitol Res</source><year>1977</year><volume>52</volume><issue>2</issue><fpage>129</fpage><lpage>150</lpage></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>King</surname><given-names>CH</given-names></name><name><surname>Muchiri</surname><given-names>EM</given-names></name><name><surname>Ouma</surname><given-names>JH</given-names></name></person-group><article-title>Evidence against rapid emergence of praziquantel resistance in <italic>Schistosoma haematobium,</italic> Kenya</article-title><source>Emerg Infect Dis</source><year>2000</year><volume>6</volume><issue>6</issue><fpage>585</fpage><pub-id pub-id-type="doi">10.3201/eid0606.000606</pub-id><pub-id pub-id-type="pmid">11076716</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name><name><surname>Liang</surname><given-names>Y-S</given-names></name></person-group><article-title>Susceptibility or resistance of praziquantel in human schistosomiasis: a review</article-title><source>Parasitol Res</source><year>2012</year><volume>111</volume><issue>5</issue><fpage>1871</fpage><lpage>1877</lpage><pub-id pub-id-type="doi">10.1007/s00436-012-3151-z</pub-id><pub-id pub-id-type="pmid">23052781</pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Arumugam</surname><given-names>M</given-names></name><name><surname>Raes</surname><given-names>J</given-names></name><name><surname>Pelletier</surname><given-names>E</given-names></name><name><surname>Le Paslier</surname><given-names>D</given-names></name><name><surname>Yamada</surname><given-names>T</given-names></name><name><surname>Mende</surname><given-names>DR</given-names></name><etal></etal></person-group><article-title>Enterotypes of the human gut microbiome</article-title><source>Nature</source><year>2011</year><volume>473</volume><issue>7346</issue><fpage>174</fpage><lpage>180</lpage><pub-id pub-id-type="doi">10.1038/nature09944</pub-id><pub-id pub-id-type="pmid">21508958</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>HMP</surname><given-names>C</given-names></name></person-group><article-title>Structure, function and diversity of the healthy human microbiome</article-title><source>Nature</source><year>2012</year><volume>486</volume><issue>7402</issue><fpage>207</fpage><lpage>214</lpage><pub-id pub-id-type="doi">10.1038/nature11234</pub-id><pub-id pub-id-type="pmid">22699609</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Turnbaugh</surname><given-names>PJ</given-names></name><name><surname>Ley</surname><given-names>RE</given-names></name><name><surname>Hamady</surname><given-names>M</given-names></name><name><surname>Fraser-Liggett</surname><given-names>C</given-names></name><name><surname>Knight</surname><given-names>R</given-names></name><name><surname>Gordon</surname><given-names>JI</given-names></name></person-group><article-title>The human microbiome project: exploring the microbial part of ourselves in a changing world</article-title><source>Nature</source><year>2007</year><volume>449</volume><issue>7164</issue><fpage>804</fpage><pub-id pub-id-type="doi">10.1038/nature06244</pub-id><pub-id pub-id-type="pmid">17943116</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lozupone</surname><given-names>CA</given-names></name><name><surname>Stombaugh</surname><given-names>JI</given-names></name><name><surname>Gordon</surname><given-names>JI</given-names></name><name><surname>Jansson</surname><given-names>JK</given-names></name><name><surname>Knight</surname><given-names>R</given-names></name></person-group><article-title>Diversity, stability and resilience of the human gut microbiota</article-title><source>Nature</source><year>2012</year><volume>489</volume><issue>7415</issue><fpage>220</fpage><lpage>230</lpage><pub-id pub-id-type="doi">10.1038/nature11550</pub-id><pub-id pub-id-type="pmid">22972295</pub-id></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stephen</surname><given-names>AM</given-names></name><name><surname>Cummings</surname><given-names>JH</given-names></name></person-group><article-title>The microbial contribution to human faecal mass</article-title><source>J Med Microbiol</source><year>1980</year><volume>13</volume><issue>1</issue><fpage>45</fpage><lpage>56</lpage><pub-id pub-id-type="doi">10.1099/00222615-13-1-45</pub-id><pub-id pub-id-type="pmid">7359576</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><mixed-citation publication-type="other">Schneeberger PH, Becker SL, Pothier JF, Duffy B, N'Goran EK, Beuret C, et al. Metagenomic diagnostics for the simultaneous detection of multiple pathogens in human stool specimens from Côte d'Ivoire: a proof-of-concept study. Infect Genet Evol. 2016;40:389–97.</mixed-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Guarner</surname><given-names>F</given-names></name><name><surname>Malagelada</surname><given-names>J-R</given-names></name></person-group><article-title>Gut flora in health and disease</article-title><source>Lancet</source><year>2003</year><volume>361</volume><issue>9356</issue><fpage>512</fpage><lpage>519</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(03)12489-0</pub-id><pub-id pub-id-type="pmid">12583961</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Sears</surname><given-names>CL</given-names></name></person-group><article-title>A dynamic partnership: Celebrating our gut flora</article-title><source>Anaerobe</source><year>2005</year><volume>11</volume><issue>5</issue><fpage>247</fpage><lpage>251</lpage><pub-id pub-id-type="doi">10.1016/j.anaerobe.2005.05.001</pub-id><pub-id pub-id-type="pmid">16701579</pub-id></element-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cheng</surname><given-names>X-Y</given-names></name><name><surname>Tian</surname><given-names>X-L</given-names></name><name><surname>Wang</surname><given-names>Y-S</given-names></name><name><surname>Lin</surname><given-names>R-M</given-names></name><name><surname>Mao</surname><given-names>Z-C</given-names></name><name><surname>Chen</surname><given-names>N</given-names></name><name><surname>Xie</surname><given-names>B-Y</given-names></name></person-group><article-title>Metagenomic analysis of the pinewood nematode microbiome reveals a symbiotic relationship critical for xenobiotics degradation</article-title><source>Sci Rep</source><year>2013</year><volume>3</volume><fpage>1869</fpage><pub-id pub-id-type="doi">10.1038/srep01869</pub-id><pub-id pub-id-type="pmid">23694939</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Clayton</surname><given-names>TA</given-names></name><name><surname>Baker</surname><given-names>D</given-names></name><name><surname>Lindon</surname><given-names>JC</given-names></name><name><surname>Everett</surname><given-names>JR</given-names></name><name><surname>Nicholson</surname><given-names>JK</given-names></name></person-group><article-title>Pharmacometabonomic identification of a significant host-microbiome metabolic interaction affecting human drug metabolism</article-title><source>Proc Natl Acad Sci USA</source><year>2009</year><volume>106</volume><issue>34</issue><fpage>14728</fpage><lpage>14733</lpage><pub-id pub-id-type="doi">10.1073/pnas.0904489106</pub-id><pub-id pub-id-type="pmid">19667173</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Maurice</surname><given-names>CF</given-names></name><name><surname>Haiser</surname><given-names>HJ</given-names></name><name><surname>Turnbaugh</surname><given-names>PJ</given-names></name></person-group><article-title>Xenobiotics shape the physiology and gene expression of the active human gut microbiome</article-title><source>Cell</source><year>2013</year><volume>152</volume><issue>1–2</issue><fpage>39</fpage><lpage>50</lpage><pub-id pub-id-type="doi">10.1016/j.cell.2012.10.052</pub-id><pub-id pub-id-type="pmid">23332745</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Nicholson</surname><given-names>JK</given-names></name><name><surname>Holmes</surname><given-names>E</given-names></name><name><surname>Wilson</surname><given-names>ID</given-names></name></person-group><article-title>Gut microorganisms, mammalian metabolism and personalized health care</article-title><source>Nat Rev Microbiol</source><year>2005</year><volume>3</volume><issue>5</issue><fpage>431</fpage><lpage>438</lpage><pub-id pub-id-type="doi">10.1038/nrmicro1152</pub-id><pub-id pub-id-type="pmid">15821725</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wilson</surname><given-names>ID</given-names></name><name><surname>Nicholson</surname><given-names>JK</given-names></name></person-group><article-title>The role of gut microbiota in drug response</article-title><source>Curr Pharm Des</source><year>2009</year><volume>15</volume><fpage>1519</fpage><lpage>1523</lpage><pub-id pub-id-type="doi">10.2174/138161209788168173</pub-id><pub-id pub-id-type="pmid">19442168</pub-id></element-citation></ref><ref id="CR25"><label>25.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Coulibaly</surname><given-names>JT</given-names></name><name><surname>Panic</surname><given-names>G</given-names></name><name><surname>Silué</surname><given-names>KD</given-names></name><name><surname>Kovač</surname><given-names>J</given-names></name><name><surname>Hattendorf</surname><given-names>J</given-names></name><name><surname>Keiser</surname><given-names>J</given-names></name></person-group><article-title>Efficacy and safety of praziquantel in preschool-aged and school-aged children infected with <italic>Schistosoma mansoni</italic>: a randomised controlled, parallel-group, dose-ranging, phase 2 trial</article-title><source>Lancet Glob Health</source><year>2017</year><volume>5</volume><issue>7</issue><fpage>e688</fpage><lpage>e698</lpage><pub-id pub-id-type="doi">10.1016/S2214-109X(17)30187-0</pub-id><pub-id pub-id-type="pmid">28619227</pub-id></element-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><collab>WHO Expert Committee</collab></person-group><article-title>Prevention and control of schistosomiasis and soil-transmitted helminthiasis</article-title><source>World Health Organ Tech Rep Ser</source><year>2002</year><volume>912</volume><issue>i-vi</issue><fpage>1</fpage><lpage>57</lpage></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Klindworth</surname><given-names>A</given-names></name><name><surname>Pruesse</surname><given-names>E</given-names></name><name><surname>Schweer</surname><given-names>T</given-names></name><name><surname>Peplies</surname><given-names>J</given-names></name><name><surname>Quast</surname><given-names>C</given-names></name><name><surname>Horn</surname><given-names>M</given-names></name><name><surname>Glöckner</surname><given-names>FO</given-names></name></person-group><article-title>Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</article-title><source>Nucleic Acids Res</source><year>2012</year><volume>40</volume><issue>1</issue><fpage>e1</fpage><pub-id pub-id-type="doi">10.1093/nar/gkr839</pub-id><pub-id pub-id-type="pmid">22009681</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Martin</surname><given-names>FE</given-names></name><name><surname>Nadkarni</surname><given-names>MA</given-names></name><name><surname>Jacques</surname><given-names>NA</given-names></name><name><surname>Hunter</surname><given-names>N</given-names></name></person-group><article-title>Quantitative microbiological study of human carious dentine by culture and real-time PCR: association of anaerobes with histopathological changes in chronic pulpitis</article-title><source>J Clin Microbiol</source><year>2002</year><volume>40</volume><issue>5</issue><fpage>1698</fpage><lpage>1704</lpage><pub-id pub-id-type="doi">10.1128/JCM.40.5.1698-1704.2002</pub-id><pub-id pub-id-type="pmid">11980945</pub-id></element-citation></ref><ref id="CR29"><label>29.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Caporaso</surname><given-names>JG</given-names></name><name><surname>Kuczynski</surname><given-names>J</given-names></name><name><surname>Stombaugh</surname><given-names>J</given-names></name><name><surname>Bittinger</surname><given-names>K</given-names></name><name><surname>Bushman</surname><given-names>FD</given-names></name><name><surname>Costello</surname><given-names>EK</given-names></name><etal></etal></person-group><article-title>QIIME allows analysis of high-throughput community sequencing data</article-title><source>Nat Methods</source><year>2010</year><volume>7</volume><issue>5</issue><fpage>335</fpage><lpage>336</lpage><pub-id pub-id-type="doi">10.1038/nmeth.f.303</pub-id><pub-id pub-id-type="pmid">20383131</pub-id></element-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Segata</surname><given-names>N</given-names></name><name><surname>Izard</surname><given-names>J</given-names></name><name><surname>Waldron</surname><given-names>L</given-names></name><name><surname>Gevers</surname><given-names>D</given-names></name><name><surname>Miropolsky</surname><given-names>L</given-names></name><name><surname>Garrett</surname><given-names>WS</given-names></name><name><surname>Huttenhower</surname><given-names>C</given-names></name></person-group><article-title>Metagenomic biomarker discovery and explanation</article-title><source>Genome Biol</source><year>2011</year><volume>12</volume><issue>6</issue><fpage>1</fpage><pub-id pub-id-type="doi">10.1186/gb-2011-12-6-r60</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tap</surname><given-names>J</given-names></name><name><surname>Mondot</surname><given-names>S</given-names></name><name><surname>Levenez</surname><given-names>F</given-names></name><name><surname>Pelletier</surname><given-names>E</given-names></name><name><surname>Caron</surname><given-names>C</given-names></name><name><surname>Furet</surname><given-names>JP</given-names></name><etal></etal></person-group><article-title>Towards the human intestinal microbiota phylogenetic core</article-title><source>Environ Microbiol</source><year>2009</year><volume>11</volume><issue>10</issue><fpage>2574</fpage><lpage>2584</lpage><pub-id pub-id-type="doi">10.1111/j.1462-2920.2009.01982.x</pub-id><pub-id pub-id-type="pmid">19601958</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Shin</surname><given-names>N-R</given-names></name><name><surname>Whon</surname><given-names>TW</given-names></name><name><surname>Bae</surname><given-names>J-W</given-names></name></person-group><article-title><italic>Proteobacteria</italic>: microbial signature of dysbiosis in gut microbiota</article-title><source>Trends Biotechnol</source><year>2015</year><volume>33</volume><issue>9</issue><fpage>496</fpage><lpage>503</lpage><pub-id pub-id-type="doi">10.1016/j.tibtech.2015.06.011</pub-id><pub-id pub-id-type="pmid">26210164</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tilg</surname><given-names>H</given-names></name><name><surname>Kaser</surname><given-names>A</given-names></name></person-group><article-title>Gut microbiome, obesity, and metabolic dysfunction</article-title><source>J Clin Invest</source><year>2011</year><volume>121</volume><issue>6</issue><fpage>2126</fpage><lpage>2132</lpage><pub-id pub-id-type="doi">10.1172/JCI58109</pub-id><pub-id pub-id-type="pmid">21633181</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Carter</surname><given-names>AO</given-names></name><name><surname>Borczyk</surname><given-names>AA</given-names></name><name><surname>Carlson</surname><given-names>JA</given-names></name><name><surname>Harvey</surname><given-names>B</given-names></name><name><surname>Hockin</surname><given-names>JC</given-names></name><name><surname>Karmali</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>A severe outbreak of <italic>Escherichia coli</italic> O157: H7-associated hemorrhagic colitis in a nursing home</article-title><source>N Engl J Med</source><year>1987</year><volume>317</volume><issue>24</issue><fpage>1496</fpage><lpage>1500</lpage><pub-id pub-id-type="doi">10.1056/NEJM198712103172403</pub-id><pub-id pub-id-type="pmid">3317047</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Steele</surname><given-names>AD</given-names></name><name><surname>Burgess</surname><given-names>DCH</given-names></name><name><surname>Diaz</surname><given-names>Z</given-names></name><name><surname>Carey</surname><given-names>ME</given-names></name><name><surname>Zaidi</surname><given-names>AK</given-names></name></person-group><article-title>Challenges and opportunities for typhoid fever control: a call for coordinated action</article-title><source>Clin Infect Dis</source><year>2016</year><volume>62</volume><issue>Suppl. 1</issue><fpage>4</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1093/cid/civ976</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Swerdlow</surname><given-names>DL</given-names></name><name><surname>Woodruff</surname><given-names>BA</given-names></name><name><surname>Brady</surname><given-names>RC</given-names></name><name><surname>Griffin</surname><given-names>PM</given-names></name><name><surname>Tippen</surname><given-names>S</given-names></name><name><surname>Donnell</surname><given-names>HD</given-names></name><etal></etal></person-group><article-title>A waterborne outbreak in Missouri of <italic>Escherichia coli</italic> O157: H7 associated with bloody diarrhea and death</article-title><source>Ann Intern Med</source><year>1992</year><volume>117</volume><issue>10</issue><fpage>812</fpage><lpage>819</lpage><pub-id pub-id-type="doi">10.7326/0003-4819-117-10-812</pub-id><pub-id pub-id-type="pmid">1416555</pub-id></element-citation></ref><ref id="CR37"><label>37.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bernard</surname><given-names>NJ</given-names></name></person-group><article-title>Rheumatoid arthritis: <italic>Prevotella copri</italic> associated with new-onset untreated RA</article-title><source>Nat Rev Rheumatol</source><year>2014</year><volume>10</volume><issue>1</issue><fpage>2</fpage><lpage>2</lpage><pub-id pub-id-type="doi">10.1038/nrrheum.2013.187</pub-id><pub-id pub-id-type="pmid">24275964</pub-id></element-citation></ref><ref id="CR38"><label>38.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kinross</surname><given-names>JM</given-names></name><name><surname>Darzi</surname><given-names>AW</given-names></name><name><surname>Nicholson</surname><given-names>JK</given-names></name></person-group><article-title>Gut microbiome-host interactions in health and disease</article-title><source>Genome Med</source><year>2011</year><volume>3</volume><issue>3</issue><fpage>14</fpage><pub-id pub-id-type="doi">10.1186/gm228</pub-id><pub-id pub-id-type="pmid">21392406</pub-id></element-citation></ref><ref id="CR39"><label>39.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chao</surname><given-names>A</given-names></name><name><surname>Shen</surname><given-names>T-J</given-names></name></person-group><article-title>Nonparametric estimation of Shannon’s index of diversity when there are unseen species in sample</article-title><source>Environ Ecol Stat</source><year>2003</year><volume>10</volume><issue>4</issue><fpage>429</fpage><lpage>443</lpage><pub-id pub-id-type="doi">10.1023/A:1026096204727</pub-id></element-citation></ref><ref id="CR40"><label>40.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Faith</surname><given-names>DP</given-names></name><name><surname>Baker</surname><given-names>AM</given-names></name></person-group><article-title>Phylogenetic diversity (PD) and biodiversity conservation: some bioinformatics challenges</article-title><source>Evol Bioinform Online</source><year>2006</year><volume>2</volume><fpage>121</fpage><lpage>128</lpage></element-citation></ref><ref id="CR41"><label>41.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Peet</surname><given-names>RK</given-names></name></person-group><article-title>The measurement of species diversity</article-title><source>Annu Rev Ecol Syst</source><year>1974</year><volume>5</volume><fpage>285</fpage><lpage>307</lpage><pub-id pub-id-type="doi">10.1146/annurev.es.05.110174.001441</pub-id></element-citation></ref><ref id="CR42"><label>42.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Shannon</surname><given-names>CE</given-names></name></person-group><article-title>The mathematical theory of communication. 1963</article-title><source>MD Comput</source><year>1997</year><volume>14</volume><issue>4</issue><fpage>306</fpage><lpage>317</lpage><pub-id pub-id-type="pmid">9230594</pub-id></element-citation></ref><ref id="CR43"><label>43.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lee</surname><given-names>SC</given-names></name><name><surname>Tang</surname><given-names>MS</given-names></name><name><surname>Lim</surname><given-names>YAL</given-names></name><name><surname>Choy</surname><given-names>SH</given-names></name><name><surname>Kurtz</surname><given-names>ZD</given-names></name><name><surname>Cox</surname><given-names>LM</given-names></name><etal></etal></person-group><article-title>Helminth colonization is associated with increased diversity of the gut microbiota</article-title><source>PLoS Negl Trop Dis</source><year>2014</year><volume>8</volume><issue>5</issue><fpage>e2880</fpage><pub-id pub-id-type="doi">10.1371/journal.pntd.0002880</pub-id><pub-id pub-id-type="pmid">24851867</pub-id></element-citation></ref><ref id="CR44"><label>44.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kay</surname><given-names>GL</given-names></name><name><surname>Millard</surname><given-names>A</given-names></name><name><surname>Sergeant</surname><given-names>MJ</given-names></name><name><surname>Midzi</surname><given-names>N</given-names></name><name><surname>Gwisai</surname><given-names>R</given-names></name><name><surname>Mduluza</surname><given-names>T</given-names></name><etal></etal></person-group><article-title>Differences in the faecal microbiome in <italic>Schistosoma haematobium</italic> infected children <italic>vs</italic> uninfected children</article-title><source>PLoS Negl Trop Dis</source><year>2015</year><volume>9</volume><issue>6</issue><fpage>e0003861</fpage><pub-id pub-id-type="doi">10.1371/journal.pntd.0003861</pub-id><pub-id pub-id-type="pmid">26114287</pub-id></element-citation></ref><ref id="CR45"><label>45.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Jaoko</surname><given-names>WG</given-names></name><name><surname>Muchemi</surname><given-names>G</given-names></name><name><surname>Oguya</surname><given-names>FO</given-names></name></person-group><article-title>Praziquantel side effects during treatment of <italic>Schistosoma mansoni</italic> infected pupils in Kibwezi Kenya</article-title><source>East Afr Med J</source><year>1996</year><volume>73</volume><issue>8</issue><fpage>499</fpage><lpage>501</lpage><pub-id pub-id-type="pmid">8898462</pub-id></element-citation></ref><ref id="CR46"><label>46.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rooks</surname><given-names>MG</given-names></name><name><surname>Veiga</surname><given-names>P</given-names></name><name><surname>Wardwell-Scott</surname><given-names>LH</given-names></name><name><surname>Tickle</surname><given-names>T</given-names></name><name><surname>Segata</surname><given-names>N</given-names></name><name><surname>Michaud</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Gut microbiome composition and function in experimental colitis during active disease and treatment-induced remission</article-title><source>ISME J</source><year>2014</year><volume>8</volume><issue>7</issue><fpage>1403</fpage><lpage>1417</lpage><pub-id pub-id-type="doi">10.1038/ismej.2014.3</pub-id><pub-id pub-id-type="pmid">24500617</pub-id></element-citation></ref><ref id="CR47"><label>47.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Satchanska</surname><given-names>G</given-names></name><name><surname>Topalova</surname><given-names>Y</given-names></name><name><surname>Ivanov</surname><given-names>I</given-names></name><name><surname>Golovinsky</surname><given-names>E</given-names></name></person-group><article-title>Xenobiotic biotransformation potential of <italic>Pseudomonas rhodesiae</italic> KCM-R5 and <italic>Bacillus subtilis</italic> KCM-RG5, tolerant to heavy metals and phenol derivatives</article-title><source>Biotechnol Biotechnol Equip</source><year>2006</year><volume>20</volume><issue>1</issue><fpage>97</fpage><lpage>102</lpage><pub-id pub-id-type="doi">10.1080/13102818.2006.10817312</pub-id></element-citation></ref><ref id="CR48"><label>48.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Soleim</surname><given-names>HA</given-names></name><name><surname>Scheline</surname><given-names>RR</given-names></name></person-group><article-title>Metabolism of xenobiotics by strains of intestinal bacteria</article-title><source>Acta Pharmacol Toxicol (Copenh)</source><year>1972</year><volume>31</volume><issue>5-7</issue><fpage>471</fpage><lpage>480</lpage><pub-id pub-id-type="doi">10.1111/j.1600-0773.1972.tb03610.x</pub-id><pub-id pub-id-type="pmid">4631168</pub-id></element-citation></ref><ref id="CR49"><label>49.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Leenstra</surname><given-names>T</given-names></name><name><surname>Coutinho</surname><given-names>HM</given-names></name><name><surname>Acosta</surname><given-names>LP</given-names></name><name><surname>Langdon</surname><given-names>GC</given-names></name><name><surname>Su</surname><given-names>L</given-names></name><name><surname>Olveda</surname><given-names>RM</given-names></name><etal></etal></person-group><article-title><italic>Schistosoma japonicum</italic> reinfection after praziquantel treatment causes anemia associated with inflammation</article-title><source>Infect Immun</source><year>2006</year><volume>74</volume><issue>11</issue><fpage>6398</fpage><lpage>6407</lpage><pub-id pub-id-type="doi">10.1128/IAI.00757-06</pub-id><pub-id pub-id-type="pmid">16923790</pub-id></element-citation></ref><ref id="CR50"><label>50.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Matsumoto</surname><given-names>J</given-names></name></person-group><article-title>Adverse effects of praziquantel treatment of <italic>Schistosoma japonicum</italic> infection: involvement of host anaphylactic reactions induced by parasite antigen release</article-title><source>Int J Parasitol</source><year>2002</year><volume>32</volume><issue>4</issue><fpage>461</fpage><lpage>471</lpage><pub-id pub-id-type="doi">10.1016/S0020-7519(01)00368-X</pub-id><pub-id pub-id-type="pmid">11849642</pub-id></element-citation></ref><ref id="CR51"><label>51.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Aliyu</surname><given-names>SH</given-names></name><name><surname>Marriott</surname><given-names>RK</given-names></name><name><surname>Curran</surname><given-names>MD</given-names></name><name><surname>Parmar</surname><given-names>S</given-names></name><name><surname>Bentley</surname><given-names>N</given-names></name><name><surname>Brown</surname><given-names>NM</given-names></name><etal></etal></person-group><article-title>Real-time PCR investigation into the importance of <italic>Fusobacterium necrophorum</italic> as a cause of acute pharyngitis in general practice</article-title><source>J Med Microbiol</source><year>2004</year><volume>53</volume><issue>Pt 10</issue><fpage>1029</fpage><lpage>1035</lpage><pub-id pub-id-type="doi">10.1099/jmm.0.45648-0</pub-id><pub-id pub-id-type="pmid">15358827</pub-id></element-citation></ref><ref id="CR52"><label>52.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kostic</surname><given-names>AD</given-names></name><name><surname>Chun</surname><given-names>E</given-names></name><name><surname>Robertson</surname><given-names>L</given-names></name><name><surname>Glickman</surname><given-names>JN</given-names></name><name><surname>Gallini</surname><given-names>CA</given-names></name><name><surname>Michaud</surname><given-names>M</given-names></name><etal></etal></person-group><article-title><italic>Fusobacterium nucleatum</italic> potentiates intestinal tumorigenesis and modulates the tumor-immune microenvironment</article-title><source>Cell Host Microbe</source><year>2013</year><volume>14</volume><issue>2</issue><fpage>207</fpage><lpage>215</lpage><pub-id pub-id-type="doi">10.1016/j.chom.2013.07.007</pub-id><pub-id pub-id-type="pmid">23954159</pub-id></element-citation></ref><ref id="CR53"><label>53.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Krisanaprakornkit</surname><given-names>S</given-names></name><name><surname>Kimball</surname><given-names>JR</given-names></name><name><surname>Weinberg</surname><given-names>A</given-names></name><name><surname>Darveau</surname><given-names>RP</given-names></name><name><surname>Bainbridge</surname><given-names>BW</given-names></name><name><surname>Dale</surname><given-names>BA</given-names></name></person-group><article-title>Inducible expression of human β-defensin 2 by <italic>Fusobacterium nucleatum</italic> in oral epithelial cells: multiple signaling pathways and role of commensal bacteria in innate immunity and the epithelial barrier</article-title><source>Infect Immun</source><year>2000</year><volume>68</volume><issue>5</issue><fpage>2907</fpage><lpage>2915</lpage><pub-id pub-id-type="doi">10.1128/IAI.68.5.2907-2915.2000</pub-id><pub-id pub-id-type="pmid">10768988</pub-id></element-citation></ref></ref-list></back></article>