<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">J Transl Med</journal-id><journal-id journal-id-type="iso-abbrev">J Transl Med</journal-id><journal-title-group><journal-title>Journal of Translational Medicine</journal-title></journal-title-group><issn pub-type="epub">1479-5876</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">29378619</article-id><article-id pub-id-type="pmc">5789697</article-id><article-id pub-id-type="publisher-id">1387</article-id><article-id pub-id-type="doi">10.1186/s12967-018-1387-9</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Chromosome conformation signatures define predictive markers of inadequate response to methotrexate in early rheumatoid arthritis</article-title></title-group><contrib-group><contrib contrib-type="author" corresp="yes"><contrib-id contrib-id-type="orcid">http://orcid.org/0000-0002-2222-0296</contrib-id><name><surname>Carini</surname><given-names>Claudio</given-names></name><address><email>claudio.carini@kcl.ac.uk</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Hunter</surname><given-names>Ewan</given-names></name><address><email>ewan.hunter@oxfordbiodynamics.com</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><collab>Scottish Early Rheumatoid Arthritis Inception cohort Investigators<contrib-group><contrib contrib-type="author"><name><surname>Porter</surname><given-names>Duncan</given-names></name></contrib><contrib contrib-type="author"><name><surname>McInnes</surname><given-names>Iain</given-names></name></contrib><contrib contrib-type="author"><name><surname>Reid</surname><given-names>David</given-names></name></contrib><contrib contrib-type="author"><name><surname>Ralston</surname><given-names>Stuart H.</given-names></name></contrib><contrib contrib-type="author"><name><surname>Basu</surname><given-names>Neil</given-names></name></contrib><contrib contrib-type="author"><name><surname>Collis</surname><given-names>Leon</given-names></name></contrib><contrib contrib-type="author"><name><surname>Goodyear</surname><given-names>Carl S.</given-names></name></contrib><contrib contrib-type="author"><name><surname>Liversidge</surname><given-names>Janet</given-names></name></contrib><contrib contrib-type="author"><name><surname>Paterson</surname><given-names>Caron</given-names></name></contrib><contrib contrib-type="author"><name><surname>Hair</surname><given-names>Jane</given-names></name></contrib><contrib contrib-type="author"><name><surname>Kean</surname><given-names>Sharon</given-names></name></contrib><contrib contrib-type="author"><name><surname>Gilmour</surname><given-names>Ashley</given-names></name></contrib><contrib contrib-type="author"><name><surname>Duncan</surname><given-names>Margaret</given-names></name></contrib><contrib contrib-type="author"><name><surname>Fraser</surname><given-names>Susan</given-names></name></contrib><contrib contrib-type="author"><name><surname>Hutton</surname><given-names>Lisa</given-names></name></contrib><contrib contrib-type="author"><name><surname>Harvie</surname><given-names>John</given-names></name></contrib><contrib contrib-type="author"><name><surname>Kumar</surname><given-names>Vinod</given-names></name></contrib><contrib contrib-type="author"><name><surname>McMahon</surname><given-names>Mike</given-names></name></contrib><contrib contrib-type="author"><name><surname>Munro</surname><given-names>Robin</given-names></name></contrib><contrib contrib-type="author"><name><surname>Larkin</surname><given-names>John</given-names></name></contrib><contrib contrib-type="author"><name><surname>McKay</surname><given-names>Neil</given-names></name></contrib><contrib contrib-type="author"><name><surname>McLaren</surname><given-names>John</given-names></name></contrib><contrib contrib-type="author"><name><surname>Reid</surname><given-names>David M.</given-names></name></contrib><contrib contrib-type="author"><name><surname>Porter</surname><given-names>Duncan</given-names></name></contrib><contrib contrib-type="author"><name><surname>Richmond</surname><given-names>Ruth</given-names></name></contrib><contrib contrib-type="author"><name><surname>Roberts</surname><given-names>Gillian</given-names></name></contrib><contrib contrib-type="author"><name><surname>Saunders</surname><given-names>Sarah</given-names></name></contrib><contrib contrib-type="author"><name><surname>Wilson</surname><given-names>Hilary</given-names></name></contrib></contrib-group></collab></contrib><contrib contrib-type="author"><name><surname>Ramadass</surname><given-names>Aroul S.</given-names></name><address><email>aroul.ramadass@oxfordbiodynamics.com</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Green</surname><given-names>Jayne</given-names></name><address><email>jayne.green@oxfordbiodynamics.com</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Akoulitchev</surname><given-names>Alexandre</given-names></name><address><email>alexandre.akoulitchev@oxfordbiodynamics.com</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>McInnes</surname><given-names>Iain B.</given-names></name><address><email>Iain.McInnes@glasgow.ac.uk</email></address><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Goodyear</surname><given-names>Carl S.</given-names></name><address><email>Carl.Goodyear@glasgow.ac.uk</email></address><xref ref-type="aff" rid="Aff3">3</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0000 8800 7493</institution-id><institution-id institution-id-type="GRID">grid.410513.2</institution-id><institution>Pfizer Inc., </institution></institution-wrap>Cambridge, USA </aff><aff id="Aff2"><label>2</label>Oxford BioDynamics Plc, Oxford, UK </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2193 314X</institution-id><institution-id institution-id-type="GRID">grid.8756.c</institution-id><institution>Institute of Infection, Immunity and Inflammation, </institution><institution>University of Glasgow, </institution></institution-wrap>Glasgow, UK </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2322 6764</institution-id><institution-id institution-id-type="GRID">grid.13097.3c</institution-id><institution>Department of Asthma, Allergy &amp; Lung Biology, GSTT Campus, </institution><institution>King’s College School of Medicine, </institution></institution-wrap>London, UK </aff></contrib-group><pub-date pub-type="epub"><day>29</day><month>1</month><year>2018</year></pub-date><pub-date pub-type="pmc-release"><day>29</day><month>1</month><year>2018</year></pub-date><pub-date pub-type="collection"><year>2018</year></pub-date><volume>16</volume><elocation-id>18</elocation-id><history><date date-type="received"><day>30</day><month>11</month><year>2017</year></date><date date-type="accepted"><day>13</day><month>1</month><year>2018</year></date></history><permissions><copyright-statement>© The Author(s) 2018</copyright-statement><license license-type="OpenAccess"><license-p><bold>Open Access</bold>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</ext-link>) applies to the data made available in this article, unless otherwise stated.</license-p></license></permissions><abstract id="Abs1"><sec><title><offsets xml_i="8676" xml_f="8686" txt_i="11" txt_f="21">Background</offsets></title><p id="Par1"><offsets xml_i="8707" xml_f="9222" txt_i="22" txt_f="537">There is a pressing need in rheumatoid arthritis (RA) to identify patients who will not respond to first-line disease-modifying anti-rheumatic drugs (DMARD). We explored whether differences in genomic architecture represented by a chromosome conformation signature (CCS) in blood taken from early RA patients before methotrexate (MTX) treatment could assist in identifying non-response to DMARD and, whether there is an association between such a signature and RA specific expression quantitative trait loci (eQTL).</offsets></p></sec><sec><title><offsets xml_i="9244" xml_f="9251" txt_i="539" txt_f="546">Methods</offsets></title><p id="Par2"><offsets xml_i="9272" xml_f="9987" txt_i="547" txt_f="1262">We looked for the presence of a CCS in blood from early RA patients commencing MTX using chromosome conformation capture by EpiSwitch™. Using blood samples from MTX responders, non-responders and healthy controls, a custom designed biomarker discovery array was refined to a 5-marker CCS that could discriminate between responders and non-responders to MTX. We cross-validated the predictive power of the CCS by generating 150 randomized groups of 59 early RA patients (30 responders and 29 non-responders) before MTX treatment. The CCS was validated using a blinded, independent cohort of 19 early RA patients (9 responders and 10 non-responders). Last, the loci of the CCS markers were mapped to RA-specific eQTL.</offsets></p></sec><sec><title><offsets xml_i="10009" xml_f="10016" txt_i="1264" txt_f="1271">Results</offsets></title><p id="Par3"><offsets xml_i="10037" xml_f="10639" txt_i="1272" txt_f="1874">We identified a 5-marker CCS that could identify, at baseline, responders and non-responders to MTX. The CCS consisted of binary chromosome conformations in the genomic regions of IFNAR1, IL-21R, IL-23, CXCL13 and IL-17A. When tested on a cohort of 59 RA patients, the CCS provided a negative predictive value of 90.0% for MTX response. When tested on a blinded independent validation cohort of 19 early RA patients, the signature demonstrated a true negative response rate of 86 and a 90% sensitivity for detection of non-responders to MTX. Only conformations in responders mapped to RA-specific eQTL.</offsets></p></sec><sec><title><offsets xml_i="10661" xml_f="10672" txt_i="1876" txt_f="1887">Conclusions</offsets></title><p id="Par4"><offsets xml_i="10693" xml_f="11056" txt_i="1888" txt_f="2251">Here we demonstrate that detection of a CCS in blood in early RA is able to predict inadequate response to MTX with a high degree of accuracy. Our results provide a proof of principle that a priori stratification of response to MTX is possible, offering a mechanism to provide alternative treatments for non-responders to MTX earlier in the course of the disease.</offsets></p></sec><sec><title><offsets xml_i="11078" xml_f="11111" txt_i="2253" txt_f="2286">Electronic supplementary material</offsets></title><p><offsets xml_i="11122" xml_f="11257" txt_i="2287" txt_f="2422">The online version of this article (10.1186/s12967-018-1387-9) contains supplementary material, which is available to authorized users.</offsets></p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Early rheumatoid arthritis</kwd><kwd>Methotrexate</kwd><kwd>Rheumatoid arthritis</kwd><kwd>DMARDs (synthetic)</kwd><kwd>Precision medicine drug response biomarkers</kwd><kwd>Methotrexate (MTX)</kwd><kwd>Chromatin conformation signatures (CCS)</kwd><kwd>Expression quantitative trait loci (eQTL)</kwd></kwd-group><funding-group><award-group><funding-source><institution>Oxford BioDynamics</institution></funding-source></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2018</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title><offsets xml_i="11989" xml_f="11999" txt_i="2431" txt_f="2441">Background</offsets></title><p id="Par22"><offsets xml_i="12021" xml_f="12155" txt_i="2442" txt_f="2576">Rheumatoid arthritis (RA) is an autoimmune inflammatory disorder driven by interacting genetic, epigenetic and environmental factors [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="12187" xml_f="12188" txt_i="2576" txt_f="2577">1</offsets></xref><offsets xml_i="12195" xml_f="12196" txt_i="2577" txt_f="2578">–</offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="12228" xml_f="12229" txt_i="2578" txt_f="2579">3</offsets></xref><offsets xml_i="12236" xml_f="12624" txt_i="2579" txt_f="2967">]. The diagnosis of RA prompts early initiation of methotrexate (MTX), the first choice of disease modifying anti-rheumatic drug (DMARD) as recommended by European League against Rheumatism (EULAR) and American College of Rheumatology (ACR) ‘treat-to-target’ strategy (target being remission or low disease activity). This approach has substantially improved outcomes in the last decade [</offsets><xref ref-type="bibr" rid="CR4"><offsets xml_i="12656" xml_f="12657" txt_i="2967" txt_f="2968">4</offsets></xref><offsets xml_i="12664" xml_f="12665" txt_i="2968" txt_f="2969">–</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="12697" xml_f="12698" txt_i="2969" txt_f="2970">6</offsets></xref><offsets xml_i="12705" xml_f="12817" txt_i="2970" txt_f="3082">]. However, approximately 35–59% of patients do not achieve clinically meaningful responses after starting MTX [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="12849" xml_f="12850" txt_i="3082" txt_f="3083">7</offsets></xref><offsets xml_i="12857" xml_f="12962" txt_i="3083" txt_f="3188">]. Predicting response to MTX has been one of the main challenges in RA management for over two decades [</offsets><xref ref-type="bibr" rid="CR8"><offsets xml_i="12994" xml_f="12995" txt_i="3188" txt_f="3189">8</offsets></xref><offsets xml_i="13002" xml_f="13141" txt_i="3189" txt_f="3328">]. Delay of effective treatment has clinical implications as response to MTX is the most significant predictor of long-term outcome in RA [</offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="13173" xml_f="13174" txt_i="3328" txt_f="3329">9</offsets></xref><offsets xml_i="13181" xml_f="13182" txt_i="3329" txt_f="3330">–</offsets><xref ref-type="bibr" rid="CR11"><offsets xml_i="13215" xml_f="13217" txt_i="3330" txt_f="3332">11</offsets></xref><offsets xml_i="13224" xml_f="13331" txt_i="3332" txt_f="3439">]. The ability to identify biomarkers able to predict inadequate response has thus far proven challenging [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="13363" xml_f="13364" txt_i="3439" txt_f="3440">7</offsets></xref><offsets xml_i="13371" xml_f="13373" txt_i="3440" txt_f="3442">, </offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="13406" xml_f="13408" txt_i="3442" txt_f="3444">12</offsets></xref><offsets xml_i="13415" xml_f="13416" txt_i="3444" txt_f="3445">–</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="13449" xml_f="13451" txt_i="3445" txt_f="3447">15</offsets></xref><offsets xml_i="13458" xml_f="13546" txt_i="3447" txt_f="3535">]. Several attempts have been made to develop diagnostics able to predict MTX-response [</offsets><xref ref-type="bibr" rid="CR16"><offsets xml_i="13579" xml_f="13581" txt_i="3535" txt_f="3537">16</offsets></xref><offsets xml_i="13588" xml_f="13589" txt_i="3537" txt_f="3538">–</offsets><xref ref-type="bibr" rid="CR23"><offsets xml_i="13622" xml_f="13624" txt_i="3538" txt_f="3540">23</offsets></xref><offsets xml_i="13631" xml_f="13650" txt_i="3540" txt_f="3559">] (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="13686" xml_f="13687" txt_i="3559" txt_f="3560">1</offsets></xref><offsets xml_i="13694" xml_f="13740" txt_i="3560" txt_f="3606">: Table S1) however, most studies have failed.</offsets></p><p><offsets xml_i="13747" xml_f="13820" txt_i="3607" txt_f="3680">RA is dependent on the interaction of genetic and environmental factors [</offsets><xref ref-type="bibr" rid="CR24"><offsets xml_i="13853" xml_f="13855" txt_i="3680" txt_f="3682">24</offsets></xref><offsets xml_i="13862" xml_f="13864" txt_i="3682" txt_f="3684">, </offsets><xref ref-type="bibr" rid="CR25"><offsets xml_i="13897" xml_f="13899" txt_i="3684" txt_f="3686">25</offsets></xref><offsets xml_i="13906" xml_f="14044" txt_i="3686" txt_f="3824">]. Epigenetic markers are closely linked to transcriptional regulation and may reflect pathogenic changes associated with disease states [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="14077" xml_f="14079" txt_i="3824" txt_f="3826">26</offsets></xref><offsets xml_i="14086" xml_f="14087" txt_i="3826" txt_f="3827">–</offsets><xref ref-type="bibr" rid="CR30"><offsets xml_i="14120" xml_f="14122" txt_i="3827" txt_f="3829">30</offsets></xref><offsets xml_i="14129" xml_f="14349" txt_i="3829" txt_f="4049">]. The first evidence suggesting that epigenetic mechanisms may play a role in autoimmune diseases came from studies performed by Richardson et al. looking at the effect of DNA methyltransferase inhibitor 5-azacytidine [</offsets><xref ref-type="bibr" rid="CR31"><offsets xml_i="14382" xml_f="14384" txt_i="4049" txt_f="4051">31</offsets></xref><offsets xml_i="14391" xml_f="14455" txt_i="4051" txt_f="4115">]. Other studies have also reported aberrant methylation in RA [</offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="14488" xml_f="14490" txt_i="4115" txt_f="4117">32</offsets></xref><offsets xml_i="14497" xml_f="14498" txt_i="4117" txt_f="4118">–</offsets><xref ref-type="bibr" rid="CR38"><offsets xml_i="14531" xml_f="14533" txt_i="4118" txt_f="4120">38</offsets></xref><offsets xml_i="14540" xml_f="14622" txt_i="4120" txt_f="4202">]. Interestingly, MTX treatment also plays a role in epigenetic regulation in RA [</offsets><xref ref-type="bibr" rid="CR39"><offsets xml_i="14655" xml_f="14657" txt_i="4202" txt_f="4204">39</offsets></xref><offsets xml_i="14664" xml_f="14665" txt_i="4204" txt_f="4205">–</offsets><xref ref-type="bibr" rid="CR41"><offsets xml_i="14698" xml_f="14700" txt_i="4205" txt_f="4207">41</offsets></xref><offsets xml_i="14707" xml_f="14934" txt_i="4207" txt_f="4434">]. Gene expression in mammals is regulated by non-coding elements that can impact physiology and disease; the principle mechanism of regulation is through the architectural status at both coding and non-coding genomic regions [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="14967" xml_f="14969" txt_i="4434" txt_f="4436">26</offsets></xref><offsets xml_i="14976" xml_f="15119" txt_i="4436" txt_f="4579">]. Under external perturbations, genomic regions can alter their 3-dimensional structure as a way of functional regulation of gene expression [</offsets><xref ref-type="bibr" rid="CR42"><offsets xml_i="15152" xml_f="15154" txt_i="4579" txt_f="4581">42</offsets></xref><offsets xml_i="15161" xml_f="15323" txt_i="4581" txt_f="4743">]. These structural changes can be measured by EpiSwitch, a high-throughput molecular technique that analyzes the spatial organization of genomic loci in a cell [</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="15356" xml_f="15358" txt_i="4743" txt_f="4745">43</offsets></xref><offsets xml_i="15365" xml_f="15366" txt_i="4745" txt_f="4746">–</offsets><xref ref-type="bibr" rid="CR45"><offsets xml_i="15399" xml_f="15401" txt_i="4746" txt_f="4748">45</offsets></xref><offsets xml_i="15408" xml_f="15517" txt_i="4748" txt_f="4857">]. As multiple genomic regions contribute to phenotypic differences through changes in genomic architecture [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="15550" xml_f="15552" txt_i="4857" txt_f="4859">26</offsets></xref><offsets xml_i="15559" xml_f="15910" txt_i="4859" txt_f="5210">], this approach allows for the development of a chromosomal conformation signature (CCS) of alterations in genomic architecture between two states (disease vs. non-disease, pre-treatment vs. post-treatment). The evaluation of long range chromosome interactions has provided useful blood-based biomarkers in oncology and other non-rheumatic diseases [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="15943" xml_f="15945" txt_i="5210" txt_f="5212">44</offsets></xref><offsets xml_i="15952" xml_f="15953" txt_i="5212" txt_f="5213">–</offsets><xref ref-type="bibr" rid="CR48"><offsets xml_i="15986" xml_f="15988" txt_i="5213" txt_f="5215">48</offsets></xref><offsets xml_i="15995" xml_f="16099" txt_i="5215" txt_f="5319">]. CCS offer a stable, binary readout of cellular states and represent an emerging class of biomarkers [</offsets><xref ref-type="bibr" rid="CR49"><offsets xml_i="16132" xml_f="16134" txt_i="5319" txt_f="5321">49</offsets></xref><offsets xml_i="16141" xml_f="16449" txt_i="5321" txt_f="5629">]. Here, we used a chromosomal architecture based approach to predict the response to MTX by developing a blood-based classifier based on a CCS. We hypothesized that interrogation of genomic architectural changes in early RA patients would define a functional endotype able to guide clinical decision making.</offsets></p></sec><sec id="Sec2" sec-type="materials|methods"><title><offsets xml_i="16510" xml_f="16517" txt_i="5631" txt_f="5638">Methods</offsets></title><sec id="Sec3"><title><offsets xml_i="16547" xml_f="16565" txt_i="5639" txt_f="5657">Patient population</offsets></title><p id="Par24"><offsets xml_i="16587" xml_f="16743" txt_i="5658" txt_f="5814">The SERA study is a national prospective inception cohort of patients with new RA or undifferentiated arthritis, together with age-gender matched controls [</offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="16776" xml_f="16778" txt_i="5814" txt_f="5816">50</offsets></xref><offsets xml_i="16785" xml_f="16787" txt_i="5816" txt_f="5818">, </offsets><xref ref-type="bibr" rid="CR51"><offsets xml_i="16820" xml_f="16822" txt_i="5818" txt_f="5820">51</offsets></xref><offsets xml_i="16829" xml_f="17097" txt_i="5820" txt_f="6088">]. We used 67 SERA patients for the discovery/validation phases (plus 34 healthy controls) and an additional 19 patients from an independent cohort, who were enrolled in a blinded validation study (demographic and clinical characteristics are shown in Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="17133" xml_f="17134" txt_i="6088" txt_f="6089">1</offsets></xref><offsets xml_i="17141" xml_f="17348" txt_i="6089" txt_f="6296">: Tables S2–S4). Participants fulfilled the 2010 ACR/EULAR RA criteria at recruitment and commenced monotherapy with MTX. Importantly, patients were treated to standard of care according to SIGN Guidelines (</offsets><ext-link ext-link-type="uri" xlink:href="http://www.sign.ac.uk/pdf/sign123.pdf"><offsets xml_i="17429" xml_f="17466" txt_i="6296" txt_f="6333">http://www.sign.ac.uk/pdf/sign123.pdf</offsets></ext-link><offsets xml_i="17477" xml_f="17872" txt_i="6333" txt_f="6728">) reflecting the true to life, population wide cohort design of SERA. Patients were treated with standard therapy, which included intra-articular or intra-muscular glucocorticoids, offered as required in the absence of low dose oral glucocorticoids. Clinical data to calculate a range of composite clinical outcomes were obtained at baseline and after 6 months of MTX treatment (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="17908" xml_f="17909" txt_i="6728" txt_f="6729">1</offsets></xref><offsets xml_i="17916" xml_f="18561" txt_i="6729" txt_f="7368">: Tables S2–S4). DAS (Disease Activity Score) response was calculated on the basis of either DAS28 (ESR) or DAS28 (CRP) change to meet the following criteria for response. Responders (R) were defined as those achieving DAS28 remission (DAS28 &lt; 2.6) or a good response (DAS28 improvement of &gt; 1.2 and DAS28 ≤ 3.2) at 6 months. Non-responders (NR) were defined as patients who had no improvement in DAS28 (≤ 0.6) by 6 months. For consistency, we then calculated a CDAI (Clinical Disease Activity Index) response for each patient and used this outcome measure to ensure comparability of clinical response across all patients in the analysis [</offsets><xref ref-type="bibr" rid="CR52"><offsets xml_i="18594" xml_f="18596" txt_i="7368" txt_f="7370">52</offsets></xref><offsets xml_i="18603" xml_f="18605" txt_i="7370" txt_f="7372">, </offsets><xref ref-type="bibr" rid="CR53"><offsets xml_i="18638" xml_f="18640" txt_i="7372" txt_f="7374">53</offsets></xref><offsets xml_i="18647" xml_f="18883" txt_i="7374" txt_f="7610">]. Baseline peripheral blood samples with EDTA were collected and centrifuged to generate a buffy layer, stored at − 80 °C. Local ethics committees approved the SERA protocol and all participants gave informed consent before enrollment.</offsets></p></sec><sec id="Sec4"><title><offsets xml_i="18915" xml_f="18971" txt_i="7612" txt_f="7668">Identification of markers by EpiSwitch™ and probe design</offsets></title><p id="Par25"><offsets xml_i="18993" xml_f="19316" txt_i="7669" txt_f="7992">A pattern recognition algorithm was used to annotate the human genome for sites with the potential to form long-range chromosome conformations. The proprietary EpiSwitch pattern recognition software, based on Bayesian-modeling, provides a probabilistic score that a region is involved in long-range chromatin interactions [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="19349" xml_f="19351" txt_i="7992" txt_f="7994">44</offsets></xref><offsets xml_i="19358" xml_f="19360" txt_i="7994" txt_f="7996">, </offsets><xref ref-type="bibr" rid="CR45"><offsets xml_i="19393" xml_f="19395" txt_i="7996" txt_f="7998">45</offsets></xref><offsets xml_i="19402" xml_f="19451" txt_i="7998" txt_f="8047">]. Sequences from 123 gene loci (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="19487" xml_f="19488" txt_i="8047" txt_f="8048">1</offsets></xref><offsets xml_i="19495" xml_f="19602" txt_i="8048" txt_f="8155">: Table S5), selected based on a systematic literature review for genes that have been associated with RA [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="19635" xml_f="19637" txt_i="8155" txt_f="8157">35</offsets></xref><offsets xml_i="19644" xml_f="19646" txt_i="8157" txt_f="8159">, </offsets><xref ref-type="bibr" rid="CR54"><offsets xml_i="19679" xml_f="19681" txt_i="8159" txt_f="8161">54</offsets></xref><offsets xml_i="19688" xml_f="19689" txt_i="8161" txt_f="8162">–</offsets><xref ref-type="bibr" rid="CR57"><offsets xml_i="19722" xml_f="19724" txt_i="8162" txt_f="8164">57</offsets></xref><offsets xml_i="19731" xml_f="20381" txt_i="8164" txt_f="8814">], were processed through the pattern recognition software to generate a list of the 13,322 chromosomal interactions to be screened for association with response to MTX in RA. Sixty 60-mer oligonucleotide probes were designed to interrogate these potential interactions and uploaded as a custom array to the Agilent SureDesign website. Importantly, each probe was present in quadruplicate on the EpiSwitch™ microarray. To subsequently evaluate a potential CCS, nested PCR (EpiSwitch™ PCR) was performed using sequence-specific oligonucleotides designed using Primer3. Oligonucleotides were tested for specificity using oligonucleotide specific BLAST.</offsets></p></sec><sec id="Sec5"><title><offsets xml_i="20413" xml_f="20445" txt_i="8816" txt_f="8848">Preparation of genomic templates</offsets></title><p id="Par26"><offsets xml_i="20467" xml_f="20655" txt_i="8849" txt_f="9037">Chromatin with intact chromosome conformations from 50 µl of each blood sample was extracted using the EpiSwitch assay following the manufacturer’s instructions (Oxford BioDynamics Ltd.) [</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="20688" xml_f="20690" txt_i="9037" txt_f="9039">43</offsets></xref><offsets xml_i="20697" xml_f="20699" txt_i="9039" txt_f="9041">, </offsets><xref ref-type="bibr" rid="CR45"><offsets xml_i="20732" xml_f="20734" txt_i="9041" txt_f="9043">45</offsets></xref><offsets xml_i="20741" xml_f="20743" txt_i="9043" txt_f="9045">, </offsets><xref ref-type="bibr" rid="CR46"><offsets xml_i="20776" xml_f="20778" txt_i="9045" txt_f="9047">46</offsets></xref><offsets xml_i="20785" xml_f="20917" txt_i="9047" txt_f="9179">]. The methods used to perform an EpiSwitch microarray and EpiSwitch PCR detection are described in detail in Supplementary Methods.</offsets></p></sec><sec id="Sec6"><title><offsets xml_i="20949" xml_f="20969" txt_i="9181" txt_f="9201">Statistical analysis</offsets></title><p id="Par27"><offsets xml_i="20991" xml_f="21581" txt_i="9202" txt_f="9792">GraphPad Prism and SPSS were used for all statistical analyses of clinical data. The R statistical environment was used to analyze the CCS data. The Fisher’s exact test (for categorical variables), the t test for independent samples (for continuous normally distributed variables), and the Mann–Whitney U test (for continuous variables without normal distribution) were used to identify differences. The level of statistical significance was set at p ≥ 0.05, and all tests were 2-sided. A full description of the statistical methods used to identify the CCS can be found in Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="21617" xml_f="21618" txt_i="9792" txt_f="9793">1</offsets></xref><offsets xml_i="21625" xml_f="21646" txt_i="9793" txt_f="9814">: Additional Methods.</offsets></p></sec><sec id="Sec7"><title><offsets xml_i="21678" xml_f="21747" txt_i="9816" txt_f="9885">Mapping of CCS locations to expressed quantitative trait locus (eQTL)</offsets></title><p id="Par28"><offsets xml_i="21769" xml_f="21902" txt_i="9886" txt_f="10019">The genomic locations of the five markers (IFNAR1, IL-21R, IL-23, CXCL13 and IL-17A) in the CSS were compared with the Walsh et al. [</offsets><xref ref-type="bibr" rid="CR58"><offsets xml_i="21935" xml_f="21937" txt_i="10019" txt_f="10021">58</offsets></xref><offsets xml_i="21944" xml_f="22037" txt_i="10021" txt_f="10114">] eQTLs data using a Window function within the Bedtools suit of genomic analysis functions [</offsets><xref ref-type="bibr" rid="CR59"><offsets xml_i="22070" xml_f="22072" txt_i="10114" txt_f="10116">59</offsets></xref><offsets xml_i="22079" xml_f="22196" txt_i="10116" txt_f="10233">]. Only eQTLs that were within a genomic window of 100 bp to 2 Kbp were considered when mapping to the CCS locations.</offsets></p></sec></sec><sec id="Sec8" sec-type="results"><title><offsets xml_i="22253" xml_f="22260" txt_i="10236" txt_f="10243">Results</offsets></title><sec id="Sec9"><title><offsets xml_i="22290" xml_f="22313" txt_i="10244" txt_f="10267">Patient characteristics</offsets></title><p id="Par29"><offsets xml_i="22335" xml_f="22526" txt_i="10268" txt_f="10459">Demographic and clinical characteristics was captured for the early RA patient groups (Stages I, II and III) at the time of recruitment and at 6 months following MTX therapy (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="22562" xml_f="22563" txt_i="10459" txt_f="10460">1</offsets></xref><offsets xml_i="22570" xml_f="22770" txt_i="10460" txt_f="10660">: Table S2–S4). A total of 86 early RA patients and 34 healthy controls (HC) were included. 67 were used to develop the CCS, and an independent, blinded cohort of 19 patients used in validation (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="22802" xml_f="22803" txt_i="10660" txt_f="10661">1</offsets></xref><offsets xml_i="22810" xml_f="22871" txt_i="10661" txt_f="10722">). The biomarker discovery pipeline comprised 3 stages (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="22903" xml_f="22904" txt_i="10722" txt_f="10723">1</offsets></xref><offsets xml_i="22911" xml_f="23205" txt_i="10723" txt_f="11017">). Stage I, the discovery phase was performed using 8 early RA patients (4 MTX-NR and 4 MTX-R) and 4 HC. Stage II, conducted to test the CCS involved 59 early RA patients (30 MTX-NR and 29 MTX-R) and 30 HC. Stage III validation used an independent, blinded cohort of 19 early RA patients (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="23237" xml_f="23238" txt_i="11017" txt_f="11018">1</offsets></xref><offsets xml_i="23245" xml_f="23247" txt_i="11018" txt_f="11020">).</offsets><fig id="Fig1"><label><offsets xml_i="23269" xml_f="23275" txt_i="11020" txt_f="11026">Fig. 1</offsets></label><caption><p><offsets xml_i="23295" xml_f="23309" txt_i="11026" txt_f="11040">Study design. </offsets><bold><offsets xml_i="23315" xml_f="23316" txt_i="11040" txt_f="11041">a</offsets></bold><offsets xml_i="23323" xml_f="23526" txt_i="11041" txt_f="11244"> Samples used for biomarker discovery and validation. A subset of patient samples from the SERA Inception Cohort (86 RA patients and 34 HC) were used to discover and validate the MTX response biomarker. </offsets><bold><offsets xml_i="23532" xml_f="23533" txt_i="11244" txt_f="11245">b</offsets></bold><offsets xml_i="23540" xml_f="24198" txt_i="11245" txt_f="11903"> Workflow for discovery and validation of the epigenetic stratifying biomarker. For Stage I and Stage II biomarker discovery and testing, clinical samples from MTX-treatment naïve patients were provided after confirmation of response by SDAI. In Stage I, an initial panel of 13,322 potential biomarkers was refined to a 5-marker chromosomal conformation signature (CCS). In Stage II, the disease specific nature of the biomarker panel was confirmed by stratification against HC and further testing was performed on 59 RA patients (29 R and 30 NR) and 30 HC. Final validation of the biomarker panel was done on an independent, blinded cohort of 19 RA patients</offsets></p></caption><graphic xlink:href="12967_2018_1387_Fig1_HTML" id="MO1"></graphic></fig><offsets xml_i="24285" xml_f="24286" txt_i="11904" txt_f="11905">
</offsets></p></sec><sec id="Sec10"><title><offsets xml_i="24319" xml_f="24360" txt_i="11907" txt_f="11948">Initial definition of the CCS in early RA</offsets></title><p><offsets xml_i="24371" xml_f="24484" txt_i="11949" txt_f="12062">To identify an initial set of common epigenetic profiles in early RA patients, 123 genetic loci (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="24520" xml_f="24521" txt_i="12062" txt_f="12063">1</offsets></xref><offsets xml_i="24528" xml_f="24571" txt_i="12063" txt_f="12106">: Table S5) previously associated with RA [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="24604" xml_f="24606" txt_i="12106" txt_f="12108">35</offsets></xref><offsets xml_i="24613" xml_f="24615" txt_i="12108" txt_f="12110">, </offsets><xref ref-type="bibr" rid="CR54"><offsets xml_i="24648" xml_f="24650" txt_i="12110" txt_f="12112">54</offsets></xref><offsets xml_i="24657" xml_f="24658" txt_i="12112" txt_f="12113">–</offsets><xref ref-type="bibr" rid="CR57"><offsets xml_i="24691" xml_f="24693" txt_i="12113" txt_f="12115">57</offsets></xref><offsets xml_i="24700" xml_f="24832" txt_i="12115" txt_f="12247">] were selected and annotated with chromosome conformation interactions predicted using the EpiSwitch in silico annotation package [</offsets><xref ref-type="bibr" rid="CR44"><offsets xml_i="24865" xml_f="24867" txt_i="12247" txt_f="12249">44</offsets></xref><offsets xml_i="24874" xml_f="24875" txt_i="12249" txt_f="12250">–</offsets><xref ref-type="bibr" rid="CR46"><offsets xml_i="24908" xml_f="24910" txt_i="12250" txt_f="12252">46</offsets></xref><offsets xml_i="24917" xml_f="25082" txt_i="12252" txt_f="12417">]. The in silico prediction generated 13,322 high-confidence chromosome interaction candidates, with an average of 99 per loci (99 ± 64; mean ± SD) (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="25118" xml_f="25119" txt_i="12417" txt_f="12418">1</offsets></xref><offsets xml_i="25126" xml_f="25225" txt_i="12418" txt_f="12517">: Table S5). These candidates were used to generate a bespoke discovery CCS array (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="25261" xml_f="25262" txt_i="12517" txt_f="12518">1</offsets></xref><offsets xml_i="25269" xml_f="25352" txt_i="12518" txt_f="12601">: Table S5) to screen baseline whole blood samples from 4 MTX-R and 4 MTX-NR (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="25384" xml_f="25385" txt_i="12601" txt_f="12602">1</offsets></xref><offsets xml_i="25392" xml_f="25410" txt_i="12602" txt_f="12620">; Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="25446" xml_f="25447" txt_i="12620" txt_f="12621">1</offsets></xref><offsets xml_i="25454" xml_f="25572" txt_i="12621" txt_f="12739">: Table S2), and 4 HC. Age, gender makeup, DAS28, baseline CDAI and MTX doses were similar in R and NR patients (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="25604" xml_f="25605" txt_i="12739" txt_f="12740">2</offsets></xref><offsets xml_i="25612" xml_f="25634" txt_i="12740" txt_f="12762">a, b; Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="25670" xml_f="25671" txt_i="12762" txt_f="12763">1</offsets></xref><offsets xml_i="25678" xml_f="26058" txt_i="12763" txt_f="13143">: Table S2). We identified 922 statistically significant chromosomal interactions that differentiated R, NR and HC. Of the 922 lead interactions, 420 were associated with NR, 210 with R and 159 with HC. A stepwise hierarchical clustering approach reduced the number of significant interactions from 922 to a 30-marker profile that effectively stratified patients (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="26094" xml_f="26095" txt_i="13143" txt_f="13144">1</offsets></xref><offsets xml_i="26102" xml_f="26125" txt_i="13144" txt_f="13167">: Figure S1, Table S6).</offsets><fig id="Fig2"><label><offsets xml_i="26147" xml_f="26153" txt_i="13167" txt_f="13173">Fig. 2</offsets></label><caption><p><offsets xml_i="26173" xml_f="26241" txt_i="13173" txt_f="13241">Clinical Data for Discovery and Testing Cohorts. Discovery cohort: (</offsets><bold><offsets xml_i="26247" xml_f="26248" txt_i="13241" txt_f="13242">a</offsets></bold><offsets xml_i="26255" xml_f="26257" txt_i="13242" txt_f="13244">, </offsets><bold><offsets xml_i="26263" xml_f="26264" txt_i="13244" txt_f="13245">b</offsets></bold><offsets xml_i="26271" xml_f="26274" txt_i="13245" txt_f="13248">). </offsets><bold><offsets xml_i="26280" xml_f="26281" txt_i="13248" txt_f="13249">a</offsets></bold><offsets xml_i="26288" xml_f="26385" txt_i="13249" txt_f="13346"> Breakdown of disease severity by CDAI scores for the discovery cohort at Baseline and 6 months. </offsets><bold><offsets xml_i="26391" xml_f="26392" txt_i="13346" txt_f="13347">b</offsets></bold><offsets xml_i="26399" xml_f="26534" txt_i="13347" txt_f="13482"> Change in CDAI scores between responders (R) and non-responders (NR) at Baseline and 6 months following MTX therapy. Testing cohort: (</offsets><bold><offsets xml_i="26540" xml_f="26541" txt_i="13482" txt_f="13483">c</offsets></bold><offsets xml_i="26548" xml_f="26550" txt_i="13483" txt_f="13485">, </offsets><bold><offsets xml_i="26556" xml_f="26557" txt_i="13485" txt_f="13486">d</offsets></bold><offsets xml_i="26564" xml_f="26567" txt_i="13486" txt_f="13489">). </offsets><bold><offsets xml_i="26573" xml_f="26574" txt_i="13489" txt_f="13490">c</offsets></bold><offsets xml_i="26581" xml_f="26676" txt_i="13490" txt_f="13585"> Breakdown of disease severity by CDAI scores for the testing cohort at Baseline and 6 months. </offsets><bold><offsets xml_i="26682" xml_f="26683" txt_i="13585" txt_f="13586">d</offsets></bold><offsets xml_i="26690" xml_f="27004" txt_i="13586" txt_f="13897"> Change in CDAI scores between responders (R) and non-responders (NR) at Baseline and 6 months following MTX therapy. Significant reductions in CDAI scores were seen between R and NR at Baseline compared to R at 6 months as well as between NR and R at 6 months (****p&lt;0.01). For CDAI scores see Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="27040" xml_f="27041" txt_i="13897" txt_f="13898">1</offsets></xref><offsets xml_i="27048" xml_f="27063" txt_i="13898" txt_f="13913">: Tables S2, S3</offsets></p></caption><graphic xlink:href="12967_2018_1387_Fig2_HTML" id="MO2"></graphic></fig></p></sec><sec id="Sec11"><title><offsets xml_i="27183" xml_f="27204" txt_i="13916" txt_f="13937">Refinement of the CCS</offsets></title><p><offsets xml_i="27215" xml_f="27409" txt_i="13938" txt_f="14132">The 30 interactions identified in the initial screen were narrowed to a smaller pool using a second SERA-derived early RA patient cohort of 59 patients (30 MTX-R and 29 MTX-NR) and 30 HCs (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="27441" xml_f="27442" txt_i="14132" txt_f="14133">2</offsets></xref><offsets xml_i="27449" xml_f="27471" txt_i="14133" txt_f="14155">c, d; Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="27507" xml_f="27508" txt_i="14155" txt_f="14156">1</offsets></xref><offsets xml_i="27515" xml_f="27608" txt_i="14156" txt_f="14249">: Table S3). Employing a stepwise approach using the EpiSwitch PCR platform (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="27644" xml_f="27645" txt_i="14249" txt_f="14250">1</offsets></xref><offsets xml_i="27652" xml_f="27873" txt_i="14250" txt_f="14471">: Figure S1, blood samples from 24 RA patients (12 R and 12 NR) were analyzed for the interactions that best differentiated between R and NR. This resulted in a set of 10 best discriminatory interactions (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="27909" xml_f="27910" txt_i="14471" txt_f="14472">1</offsets></xref><offsets xml_i="27917" xml_f="28309" txt_i="14472" txt_f="14864">: Table S6). These ten interactions were used to probe blood samples from an additional set of 35 RA (18 R and 17 NR). Using logistical regression on the combined data from all patients used to refine the CCS (that is 30 R and 29 NR), we identified five conformations within the IFNAR1, IL-21R, IL-23, IL-17A, and CXCL13 loci which provided the final CCS for response to MTX (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="28345" xml_f="28346" txt_i="14864" txt_f="14865">1</offsets></xref><offsets xml_i="28353" xml_f="28476" txt_i="14865" txt_f="14988">: Figures S2–S6, Table S8). The regression coefficients and odds ratio of the logistic regression model are shown in Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="28510" xml_f="28511" txt_i="14988" txt_f="14989">1</offsets></xref><offsets xml_i="28518" xml_f="28519" txt_i="14989" txt_f="14990">.</offsets><table-wrap id="Tab1"><label><offsets xml_i="28548" xml_f="28555" txt_i="14990" txt_f="14997">Table 1</offsets></label><caption><p><offsets xml_i="28575" xml_f="28729" txt_i="14997" txt_f="15151">Coefficients of the logistic regression model for predicting efficacy of MTX monotherapy in baseline samples based on retrospective annotation at 6 months</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left"><offsets xml_i="28808" xml_f="28813" txt_i="15152" txt_f="15157">Locus</offsets></th><th align="left"><offsets xml_i="28835" xml_f="28857" txt_i="15157" txt_f="15179">Regression coefficient</offsets></th><th align="left"><offsets xml_i="28879" xml_f="28898" txt_i="15179" txt_f="15198">Odds ratio (95% CI)</offsets></th></tr></thead><tbody><tr><td align="left"><offsets xml_i="28944" xml_f="28950" txt_i="15198" txt_f="15204">IFNAR1</offsets></td><td char="." align="char"><offsets xml_i="28981" xml_f="28984" txt_i="15205" txt_f="15208">2.0</offsets></td><td char="(" align="char"><offsets xml_i="29015" xml_f="29027" txt_i="15209" txt_f="15221">7.6 (1.4–57)</offsets></td></tr><tr><td align="left"><offsets xml_i="29058" xml_f="29064" txt_i="15222" txt_f="15228">IL-17A</offsets></td><td char="." align="char"><offsets xml_i="29095" xml_f="29100" txt_i="15229" txt_f="15234">− 3.2</offsets></td><td char="(" align="char"><offsets xml_i="29131" xml_f="29148" txt_i="15235" txt_f="15252">0.04 (0.001–0.41)</offsets></td></tr><tr><td align="left"><offsets xml_i="29179" xml_f="29185" txt_i="15253" txt_f="15259">CXCL13</offsets></td><td char="." align="char"><offsets xml_i="29216" xml_f="29221" txt_i="15260" txt_f="15265">− 1.9</offsets></td><td char="(" align="char"><offsets xml_i="29252" xml_f="29268" txt_i="15266" txt_f="15282">0.15 (0.02–0.69)</offsets></td></tr><tr><td align="left"><offsets xml_i="29299" xml_f="29305" txt_i="15283" txt_f="15289">IL-21R</offsets></td><td char="." align="char"><offsets xml_i="29336" xml_f="29339" txt_i="15290" txt_f="15293">2.2</offsets></td><td char="(" align="char"><offsets xml_i="29370" xml_f="29382" txt_i="15294" txt_f="15306">9.0 (2.1–49)</offsets></td></tr><tr><td align="left"><offsets xml_i="29413" xml_f="29418" txt_i="15307" txt_f="15312">IL-23</offsets></td><td char="." align="char"><offsets xml_i="29449" xml_f="29452" txt_i="15313" txt_f="15316">1.4</offsets></td><td char="(" align="char"><offsets xml_i="29483" xml_f="29496" txt_i="15317" txt_f="15330">4.2 (0.78–33)</offsets></td></tr></tbody></table></table-wrap></p><p id="Par32"><offsets xml_i="29553" xml_f="30035" txt_i="15332" txt_f="15814">The model provided a prediction probability score for R and NR, ranging from 0.0098 to 0.99 (0 = responder, 1 = non-responder). The cut-off values were set at ≤ 0.30 for R and ≥ 0.70 for NR. The score of ≤ 0.30 had a true positive rate (sensitivity) of 89% (95% confidence interval [95% CI] 72–98%), while a score of ≥ 0.70 had a true negative response rate (specificity) of 87% (95% CI 70–96%). The number of observed and predicted patients per response category is shown in Table </offsets><xref rid="Tab2" ref-type="table"><offsets xml_i="30069" xml_f="30070" txt_i="15814" txt_f="15815">2</offsets></xref><offsets xml_i="30077" xml_f="30166" txt_i="15815" txt_f="15904">. With the CCS classifier, 53 patients (90%) were correctly classified as either R or NR.</offsets><table-wrap id="Tab2"><label><offsets xml_i="30195" xml_f="30202" txt_i="15904" txt_f="15911">Table 2</offsets></label><caption><p><offsets xml_i="30222" xml_f="30319" txt_i="15911" txt_f="16008">Observed and predicted number of R and NR to MTX monotherapy at 6 months using the CCS classifier</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" rowspan="2"><offsets xml_i="30410" xml_f="30427" txt_i="16009" txt_f="16026">Observed response</offsets></th><th align="left" colspan="3"><offsets xml_i="30461" xml_f="30479" txt_i="16026" txt_f="16044">Predicted response</offsets></th></tr><tr><th align="left"><offsets xml_i="30510" xml_f="30523" txt_i="16044" txt_f="16057">Non-responder</offsets></th><th align="left"><offsets xml_i="30545" xml_f="30554" txt_i="16057" txt_f="16066">Undefined</offsets></th><th align="left"><offsets xml_i="30576" xml_f="30585" txt_i="16066" txt_f="16075">Responder</offsets></th></tr></thead><tbody><tr><td align="left"><offsets xml_i="30631" xml_f="30644" txt_i="16075" txt_f="16088">Non-responder</offsets></td><td char="." align="char"><offsets xml_i="30675" xml_f="30677" txt_i="16089" txt_f="16091">25</offsets></td><td align="left"><offsets xml_i="30699" xml_f="30700" txt_i="16092" txt_f="16093">1</offsets></td><td char="." align="char"><offsets xml_i="30731" xml_f="30732" txt_i="16094" txt_f="16095">3</offsets></td></tr><tr><td align="left"><offsets xml_i="30763" xml_f="30772" txt_i="16096" txt_f="16105">Responder</offsets></td><td char="." align="char"><offsets xml_i="30803" xml_f="30804" txt_i="16106" txt_f="16107">3</offsets></td><td align="left"><offsets xml_i="30826" xml_f="30827" txt_i="16108" txt_f="16109">7</offsets></td><td char="." align="char"><offsets xml_i="30858" xml_f="30860" txt_i="16110" txt_f="16112">20</offsets></td></tr></tbody></table><table-wrap-foot><p><offsets xml_i="30906" xml_f="31104" txt_i="16113" txt_f="16305">Cut off levels were chosen based on the logistic model probabilities of response to MTX of (approximately) &gt; 0.70 for NR and &lt; 0.3 for R. NR and R were defined as described in Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="31140" xml_f="31141" txt_i="16305" txt_f="16306">1</offsets></xref><offsets xml_i="31148" xml_f="31168" txt_i="16306" txt_f="16326">: Additional Methods</offsets></p></table-wrap-foot></table-wrap><offsets xml_i="31203" xml_f="31204" txt_i="16327" txt_f="16328">
</offsets></p></sec><sec id="Sec12"><title><offsets xml_i="31237" xml_f="31271" txt_i="16330" txt_f="16364">Testing the performance of the CCS</offsets></title><p id="Par33"><offsets xml_i="31293" xml_f="31663" txt_i="16365" txt_f="16735">To test the accuracy and performance robustness of the CCS classifier to discriminate non-responders to MTX in early RA prior to treatment, data from the 59 early RA patient cohort used to develop the classifier was split into 150 different training and test sets by random data re-sampling using fivefold cross-validation. While this analysis was not intended as a true</offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="31695" xml_f="31696" txt_i="16735" txt_f="16736">3</offsets></xref><offsets xml_i="31703" xml_f="31849" txt_i="16736" txt_f="16882">a). To determine the predictive capability of the CCS classifier, the average model accuracy statistics were adjusted for population R/NR to MTX [</offsets><xref ref-type="bibr" rid="CR60"><offsets xml_i="31882" xml_f="31884" txt_i="16882" txt_f="16884">60</offsets></xref><offsets xml_i="31891" xml_f="32132" txt_i="16884" txt_f="17125">]. Using a 55% MTX response rate, the positive predictive value (PPV) was 88% while the negative predictive value (NPV) was 86.4%. When the response rate was adjusted to 60%, this decreased the PPV to 85.7% while increasing the NPV to 88.6%.</offsets><fig id="Fig3"><label><offsets xml_i="32154" xml_f="32160" txt_i="17125" txt_f="17131">Fig. 3</offsets></label><caption><p><offsets xml_i="32180" xml_f="32259" txt_i="17131" txt_f="17210">CCS performance on randomized subsets of testing cohort and validation cohort. </offsets><bold><offsets xml_i="32265" xml_f="32266" txt_i="17210" txt_f="17211">a</offsets></bold><offsets xml_i="32273" xml_f="32812" txt_i="17211" txt_f="17750"> Receiver operating characteristic (ROC) plots for 150 runs of the EpiSwitch™ logistic classifier. Data for the 59 patient cohort was randomised 150 times using the WEKA sample randomisation function. This reorders the data prior to splitting in developing the training set, ensuring that the same starting point for the classifier is not used and allowing multiple accuracy calculations for the same data. The average area under the curve (AUC) for the 150 classifier runs was 0.90. The plot is the average ROC from the 150 test results. </offsets><bold><offsets xml_i="32818" xml_f="32819" txt_i="17750" txt_f="17751">b</offsets></bold><offsets xml_i="32826" xml_f="33054" txt_i="17751" txt_f="17979"> Receiver operating characteristic (ROC) plots for the EpiSwitch™ logistic classifier run on the blinded validation cohort of 19 RA patients. The classifier had a sensitivity of 75.0% and specificity of 85.7% with an AUC of 0.91</offsets></p></caption><graphic xlink:href="12967_2018_1387_Fig3_HTML" id="MO3"></graphic></fig><offsets xml_i="33141" xml_f="33142" txt_i="17980" txt_f="17981">
</offsets></p></sec><sec id="Sec13"><title><offsets xml_i="33175" xml_f="33205" txt_i="17983" txt_f="18013">Independent blinded validation</offsets></title><p id="Par34"><offsets xml_i="33227" xml_f="33418" txt_i="18014" txt_f="18205">To validate the performance of the CCS, a blinded analysis was undertaken on an additional 19 early RA patients from an independent cohort selected from SERA (9 R and 10 NR) (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="33454" xml_f="33455" txt_i="18205" txt_f="18206">1</offsets></xref><offsets xml_i="33462" xml_f="33700" txt_i="18206" txt_f="18444">: Table S4, Figure S8). This validation cohort had similar clinical characteristics to those used to develop the CCS with respect to age, gender, RA severity (as measured by CDAI, DAS28-CRP, and DAS28-ESR) and MTX dosage (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="33736" xml_f="33737" txt_i="18444" txt_f="18445">1</offsets></xref><offsets xml_i="33744" xml_f="34223" txt_i="18445" txt_f="18924">: Tables S3, S4). The analysis of the validation cohort was done using a blinded analysis, where the actual response calls were not known until after the identification of R and NR using the CCS classifier. In this independent cohort, we obtained a PPV of 90% (95% CI 59–98%), and a NPV rate of 66.7% (95% CI 42–85%), sensitivity of 75.0% (95% CI 43–95%) and specificity of 85.7% (95% CI 42–97%). The area under the receiver operating characteristic curve (AUROC) was 0.91 (Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="34255" xml_f="34256" txt_i="18924" txt_f="18925">3</offsets></xref><offsets xml_i="34263" xml_f="34353" txt_i="18925" txt_f="19015">b). The number of observed and predicted patients per response category is shown in Table </offsets><xref rid="Tab3" ref-type="table"><offsets xml_i="34387" xml_f="34388" txt_i="19015" txt_f="19016">3</offsets></xref><offsets xml_i="34395" xml_f="34555" txt_i="19016" txt_f="19176">. The strong NPV statistics for the blinded samples confirms the accuracy and robust stratification power of the CCS for non-response to MTX prior to treatment.</offsets><table-wrap id="Tab3"><label><offsets xml_i="34584" xml_f="34591" txt_i="19176" txt_f="19183">Table 3</offsets></label><caption><p><offsets xml_i="34611" xml_f="34713" txt_i="19183" txt_f="19285">CCS classifier performance for predicting non-response to MTX in a blinded cohort of early RA patients</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" rowspan="2"><offsets xml_i="34804" xml_f="34821" txt_i="19286" txt_f="19303">Observed response</offsets></th><th align="left" colspan="2"><offsets xml_i="34855" xml_f="34873" txt_i="19303" txt_f="19321">Predicted response</offsets></th></tr><tr><th align="left"><offsets xml_i="34904" xml_f="34917" txt_i="19321" txt_f="19334">Non-responder</offsets></th><th align="left"><offsets xml_i="34939" xml_f="34948" txt_i="19334" txt_f="19343">Responder</offsets></th></tr></thead><tbody><tr><td align="left"><offsets xml_i="34994" xml_f="35007" txt_i="19343" txt_f="19356">Non-responder</offsets></td><td align="left"><offsets xml_i="35029" xml_f="35030" txt_i="19357" txt_f="19358">9</offsets></td><td align="left"><offsets xml_i="35052" xml_f="35053" txt_i="19359" txt_f="19360">1</offsets></td></tr><tr><td align="left"><offsets xml_i="35084" xml_f="35093" txt_i="19361" txt_f="19370">Responder</offsets></td><td align="left"><offsets xml_i="35115" xml_f="35116" txt_i="19371" txt_f="19372">3</offsets></td><td align="left"><offsets xml_i="35138" xml_f="35139" txt_i="19373" txt_f="19374">6</offsets></td></tr></tbody></table></table-wrap><offsets xml_i="35178" xml_f="35179" txt_i="19375" txt_f="19376">
</offsets></p></sec><sec id="Sec14"><title><offsets xml_i="35212" xml_f="35292" txt_i="19378" txt_f="19458">Biological and functional characterization of the genomic loci making up the CCS</offsets></title><p id="Par35"><offsets xml_i="35314" xml_f="35629" txt_i="19459" txt_f="19774">The five genomic loci (IFNAR1, IL-21R, IL-23, IL-17A, and CXCL13) that make up the CCS fall within genes that encode for proteins involved in the immune response. Recently, eQTLs that overlap with enhancer elements have been identified in RA patients with moderate to severe disease and inadequate response to MTX [</offsets><xref ref-type="bibr" rid="CR58"><offsets xml_i="35662" xml_f="35664" txt_i="19774" txt_f="19776">58</offsets></xref><offsets xml_i="35671" xml_f="35854" txt_i="19776" txt_f="19959">]. eQTLs are genomic loci that contribute to variation in expression levels of mRNAs. We mapped the genomic locations of the five loci in the CCS to these previously published eQTLs [</offsets><xref ref-type="bibr" rid="CR58"><offsets xml_i="35887" xml_f="35889" txt_i="19959" txt_f="19961">58</offsets></xref><offsets xml_i="35896" xml_f="36067" txt_i="19961" txt_f="20132">]. The two genomic sites that corresponded to the junction of each CCS locus were mapped to eQTLs that were within 100, 200, 500, 1000 or 2000 base pairs (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="36103" xml_f="36104" txt_i="20132" txt_f="20133">1</offsets></xref><offsets xml_i="36111" xml_f="36327" txt_i="20133" txt_f="20349">: Table S9). There was enrichment of eQTLs co-localized with the R-associated CCS loci (IFNAR1, IL-21R and IL-23). In contrast, no co-localization was observed in the NR-associated CCS loci (IL-17A and CXCL13) (Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="36359" xml_f="36360" txt_i="20349" txt_f="20350">4</offsets></xref><offsets xml_i="36367" xml_f="36369" txt_i="20350" txt_f="20352">).</offsets><fig id="Fig4"><label><offsets xml_i="36391" xml_f="36397" txt_i="20352" txt_f="20358">Fig. 4</offsets></label><caption><p><offsets xml_i="36417" xml_f="36615" txt_i="20358" txt_f="20556">Chromosome conformations in MTX responders map to RA eQTLs. Bedtools shot of CCS markers mapped within 200 bp of previously identified RA eQTLs. The overlapping CCS:eQTLs are highlighted in yellow. </offsets><bold><offsets xml_i="36621" xml_f="36622" txt_i="20556" txt_f="20557">a</offsets></bold><offsets xml_i="36629" xml_f="36712" txt_i="20557" txt_f="20640"> The CCS regions associated with the IFNAR1 locus on chromosome 21 map to 6 eQTLs. </offsets><bold><offsets xml_i="36718" xml_f="36719" txt_i="20640" txt_f="20641">b</offsets></bold><offsets xml_i="36726" xml_f="36810" txt_i="20641" txt_f="20725"> The CCS regions associated with the IL-21R locus on chromosome 16 map to 21 eQTLs. </offsets><bold><offsets xml_i="36816" xml_f="36817" txt_i="20725" txt_f="20726">c</offsets></bold><offsets xml_i="36824" xml_f="36906" txt_i="20726" txt_f="20808"> The CCS regions associated with the IL-23 locus on chromosome 12 map to 4 eQTLs. </offsets><bold><offsets xml_i="36912" xml_f="36913" txt_i="20808" txt_f="20809">d</offsets></bold><offsets xml_i="36920" xml_f="37011" txt_i="20809" txt_f="20900"> The CCS regions associated with the IL-17A locus on chromosome 6 do not map to any eQTLs. </offsets><bold><offsets xml_i="37017" xml_f="37018" txt_i="20900" txt_f="20901">e</offsets></bold><offsets xml_i="37025" xml_f="37114" txt_i="20901" txt_f="20990"> The CCS regions associated with the CXCL13 locus on chromosome 4 do not map to any eQTLs</offsets></p></caption><graphic xlink:href="12967_2018_1387_Fig4_HTML" id="MO4"></graphic></fig><offsets xml_i="37201" xml_f="37202" txt_i="20991" txt_f="20992">
</offsets></p></sec></sec><sec id="Sec15" sec-type="discussion"><title><offsets xml_i="37263" xml_f="37273" txt_i="20995" txt_f="21005">Discussion</offsets></title><p id="Par36"><offsets xml_i="37295" xml_f="37545" txt_i="21006" txt_f="21256">To address an unmet clinical need of predicting those RA patients who will not respond to MTX, we analyzed the CCS in whole blood taken at baseline from SERA-RA patients using a well-established method of analyzing long-range chromatin interactions [</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="37578" xml_f="37580" txt_i="21256" txt_f="21258">43</offsets></xref><offsets xml_i="37587" xml_f="37588" txt_i="21258" txt_f="21259">–</offsets><xref ref-type="bibr" rid="CR45"><offsets xml_i="37621" xml_f="37623" txt_i="21259" txt_f="21261">45</offsets></xref><offsets xml_i="37630" xml_f="38051" txt_i="21261" txt_f="21682">]. We identified a 5-marker panel consisting of chromosomal conformations in the genomic loci of IFNAR1, IL-21R, IL-23, IL-17A, and CXCL13 that could identify R and NR to MTX with 90% sensitivity in an independent blinded validation cohort. This could facilitate earlier access to more effective therapies, thus avoiding disease progression, unnecessary exposure to potentially toxic drugs and diminished quality of life.</offsets></p><p id="Par37"><offsets xml_i="38069" xml_f="38154" txt_i="21683" txt_f="21768">Historically, identifying predictive biomarkers for MTX response has been difficult [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="38186" xml_f="38187" txt_i="21768" txt_f="21769">7</offsets></xref><offsets xml_i="38194" xml_f="38196" txt_i="21769" txt_f="21771">, </offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="38229" xml_f="38231" txt_i="21771" txt_f="21773">12</offsets></xref><offsets xml_i="38238" xml_f="38239" txt_i="21773" txt_f="21774">–</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="38272" xml_f="38274" txt_i="21774" txt_f="21776">15</offsets></xref><offsets xml_i="38281" xml_f="38346" txt_i="21776" txt_f="21841">]. While clinical predictors of RA disease are well established [</offsets><xref ref-type="bibr" rid="CR61"><offsets xml_i="38379" xml_f="38381" txt_i="21841" txt_f="21843">61</offsets></xref><offsets xml_i="38388" xml_f="38390" txt_i="21843" txt_f="21845">, </offsets><xref ref-type="bibr" rid="CR62"><offsets xml_i="38423" xml_f="38425" txt_i="21845" txt_f="21847">62</offsets></xref><offsets xml_i="38432" xml_f="38514" txt_i="21847" txt_f="21929">], they do not correlate well with response to treatment at the individual level [</offsets><xref ref-type="bibr" rid="CR13"><offsets xml_i="38547" xml_f="38549" txt_i="21929" txt_f="21931">13</offsets></xref><offsets xml_i="38556" xml_f="38644" txt_i="21931" txt_f="22019">]. 35–59% of patients do not achieve clinically meaningful response after starting MTX [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="38676" xml_f="38677" txt_i="22019" txt_f="22020">7</offsets></xref><offsets xml_i="38684" xml_f="38686" txt_i="22020" txt_f="22022">, </offsets><xref ref-type="bibr" rid="CR63"><offsets xml_i="38719" xml_f="38721" txt_i="22022" txt_f="22024">63</offsets></xref><offsets xml_i="38728" xml_f="38801" txt_i="22024" txt_f="22097">]. In the SERA study, only 30% of patients responded to MTX monotherapy [</offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="38834" xml_f="38836" txt_i="22097" txt_f="22099">50</offsets></xref><offsets xml_i="38843" xml_f="39064" txt_i="22099" txt_f="22320">]. While the results presented here provide a proof-of-concept and further validation is warranted, the EpiSwitch technology has several attractive features from the standpoint of a biomarker that can be used clinically [</offsets><xref ref-type="bibr" rid="CR49"><offsets xml_i="39097" xml_f="39099" txt_i="22320" txt_f="22322">49</offsets></xref><offsets xml_i="39106" xml_f="39790" txt_i="22322" txt_f="23006">]. First, it requires a very small amount of blood sample (typically 50 μl). Second, it utilizes an established laboratory methodology and readouts (qPCR). Last, the turnaround time is short (~ 8 h). Thus, once further validated, the approach described here faces low barriers to clinical adoption. The CCS defined a signature that suggests epigenomic control over loci with a central role in the IL-17/IL-23 axis, with the two most informative long-range chromatin interactions for predicting MTX-NR coming from IL-17A and CXCL13 loci. In agreement with our finding, earlier reports have suggested that IL-23, IFNAR1 and IL-21R are predictors of positive response in other diseases [</offsets><xref ref-type="bibr" rid="CR64"><offsets xml_i="39823" xml_f="39825" txt_i="23006" txt_f="23008">64</offsets></xref><offsets xml_i="39832" xml_f="39833" txt_i="23008" txt_f="23009">–</offsets><xref ref-type="bibr" rid="CR66"><offsets xml_i="39866" xml_f="39868" txt_i="23009" txt_f="23011">66</offsets></xref><offsets xml_i="39875" xml_f="39964" txt_i="23011" txt_f="23100">], while IL-17A and CXL13 are predictors of poor outcome and increased disease severity [</offsets><xref ref-type="bibr" rid="CR67"><offsets xml_i="39997" xml_f="39999" txt_i="23100" txt_f="23102">67</offsets></xref><offsets xml_i="40006" xml_f="40007" txt_i="23102" txt_f="23103">–</offsets><xref ref-type="bibr" rid="CR71"><offsets xml_i="40040" xml_f="40042" txt_i="23103" txt_f="23105">71</offsets></xref><offsets xml_i="40049" xml_f="40215" txt_i="23105" txt_f="23271">]. Several studies support the notion that IL-23/Th17 axis drive inflammation in chronic diseases and perhaps plays a role in the response to immunomodulating drugs [</offsets><xref ref-type="bibr" rid="CR72"><offsets xml_i="40248" xml_f="40250" txt_i="23271" txt_f="23273">72</offsets></xref><offsets xml_i="40257" xml_f="40258" txt_i="23273" txt_f="23274">–</offsets><xref ref-type="bibr" rid="CR75"><offsets xml_i="40291" xml_f="40293" txt_i="23274" txt_f="23276">75</offsets></xref><offsets xml_i="40300" xml_f="40494" txt_i="23276" txt_f="23470">]. Interestingly, no loci previously associated with MTX metabolism were implicated in our study, indicating that the response to MTX was independent of how the drug is known to be metabolized [</offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="40527" xml_f="40529" txt_i="23470" txt_f="23472">20</offsets></xref><offsets xml_i="40536" xml_f="40538" txt_i="23472" txt_f="23474">, </offsets><xref ref-type="bibr" rid="CR76"><offsets xml_i="40571" xml_f="40573" txt_i="23474" txt_f="23476">76</offsets></xref><offsets xml_i="40580" xml_f="40582" txt_i="23476" txt_f="23478">].</offsets></p><p id="Par38"><offsets xml_i="40600" xml_f="40778" txt_i="23479" txt_f="23657">Our results are consistent with previous data reporting the presence of regulatory elements (i.e. eQTL) only in the context of inflammatory diseases and not in healthy controls [</offsets><xref ref-type="bibr" rid="CR77"><offsets xml_i="40811" xml_f="40813" txt_i="23657" txt_f="23659">77</offsets></xref><offsets xml_i="40820" xml_f="40822" txt_i="23659" txt_f="23661">].</offsets></p><p id="Par39"><offsets xml_i="40840" xml_f="40996" txt_i="23662" txt_f="23818">Recently, Walsh et al. reported that eQTL mapped from RA are particularly enriched in enhancer regions of disease related cell types such as T and B cells [</offsets><xref ref-type="bibr" rid="CR58"><offsets xml_i="41029" xml_f="41031" txt_i="23818" txt_f="23820">58</offsets></xref><offsets xml_i="41038" xml_f="41140" txt_i="23820" txt_f="23922">]. Previous studies have recognized that the eQTL are highly specific to different leucocyte subsets [</offsets><xref ref-type="bibr" rid="CR78"><offsets xml_i="41173" xml_f="41175" txt_i="23922" txt_f="23924">78</offsets></xref><offsets xml_i="41182" xml_f="41360" txt_i="23924" txt_f="24102">]. To explore the concordance of CCS data with existing genetic regulatory datasets, we mapped the genomic locations of the regions in the CCS against reported RA-specific eQTL [</offsets><xref ref-type="bibr" rid="CR58"><offsets xml_i="41393" xml_f="41395" txt_i="24102" txt_f="24104">58</offsets></xref><offsets xml_i="41402" xml_f="41887" txt_i="24104" txt_f="24589">]. This revealed a high level of co-localization of R-specific loci (IFNR1, IL-21R, and IL-23) to RA-specific eQTL. Notably, this level of concordance was not observed in NR loci (IL-17 and CXCL-13). The increased association of RA eQTLs with CCS regions observed in R, but not in NR, suggest dysregulation at the level of regulatory 3D genome architecture. It has been reported that eQTL are associated with expression of mRNA transcripts, with concomitant effects on protein levels [</offsets><xref ref-type="bibr" rid="CR79"><offsets xml_i="41920" xml_f="41922" txt_i="24589" txt_f="24591">79</offsets></xref><offsets xml_i="41929" xml_f="41930" txt_i="24591" txt_f="24592">–</offsets><xref ref-type="bibr" rid="CR82"><offsets xml_i="41963" xml_f="41965" txt_i="24592" txt_f="24594">82</offsets></xref><offsets xml_i="41972" xml_f="42229" txt_i="24594" txt_f="24851">]. This leaves open the possibility that the association of eQTL with CCS observed in RA-R result from phenotypic consequences due to the effects of mRNA and protein expression levels. However, the impact of eQTL on protein levels remains poorly understood.</offsets></p><p id="Par40"><offsets xml_i="42247" xml_f="42348" txt_i="24852" txt_f="24953">It is now clear that the developing immune response is influenced by genetic and epigenetic factors [</offsets><xref ref-type="bibr" rid="CR41"><offsets xml_i="42381" xml_f="42383" txt_i="24953" txt_f="24955">41</offsets></xref><offsets xml_i="42390" xml_f="42488" txt_i="24955" txt_f="25053">]. IFNAR1, IL-21R and IL-23 loci have been reported to play a key role in the pathogenesis of RA [</offsets><xref ref-type="bibr" rid="CR83"><offsets xml_i="42521" xml_f="42523" txt_i="25053" txt_f="25055">83</offsets></xref><offsets xml_i="42530" xml_f="42531" txt_i="25055" txt_f="25056">–</offsets><xref ref-type="bibr" rid="CR85"><offsets xml_i="42564" xml_f="42566" txt_i="25056" txt_f="25058">85</offsets></xref><offsets xml_i="42573" xml_f="42653" txt_i="25058" txt_f="25138">] and Th17 cells are implicated in pathogenesis especially in the pre-RA phase [</offsets><xref ref-type="bibr" rid="CR68"><offsets xml_i="42686" xml_f="42688" txt_i="25138" txt_f="25140">68</offsets></xref><offsets xml_i="42695" xml_f="42697" txt_i="25140" txt_f="25142">, </offsets><xref ref-type="bibr" rid="CR72"><offsets xml_i="42730" xml_f="42732" txt_i="25142" txt_f="25144">72</offsets></xref><offsets xml_i="42739" xml_f="42741" txt_i="25144" txt_f="25146">, </offsets><xref ref-type="bibr" rid="CR73"><offsets xml_i="42774" xml_f="42776" txt_i="25146" txt_f="25148">73</offsets></xref><offsets xml_i="42783" xml_f="43405" txt_i="25148" txt_f="25770">]. Whether the CCS differences we observed represent changes impacting the pre-RA phase or are acquired during the early phase of RA is at present unclear. We hypothesize that the different genomic architecture observed for R and NR might reflect differences in epigenetic host responses to early pathogenetic events, or particular environmental exposures. We anticipate that the association observed between CCS and eQTL in RA patients may be important to understand the heterogeneity of the response observed between individuals. Furthermore, eQTLs present in active inflammatory diseases can disappear after treatment [</offsets><xref ref-type="bibr" rid="CR77"><offsets xml_i="43438" xml_f="43440" txt_i="25770" txt_f="25772">77</offsets></xref><offsets xml_i="43447" xml_f="43790" txt_i="25772" txt_f="26115">]. It would therefore be useful to determine whether the concordance of the CCS and eQTL observed in our study in treatment naïve RA responders will be linked only to the disease or change after MTX treatment. Moreover, it would be interesting to see whether the reported eQTL are linked to the disease state and/or the level of inflammation [</offsets><xref ref-type="bibr" rid="CR68"><offsets xml_i="43823" xml_f="43825" txt_i="26115" txt_f="26117">68</offsets></xref><offsets xml_i="43832" xml_f="43985" txt_i="26117" txt_f="26270">]. Our data indicate that the mapping of the QTL can reveal an altered biological status in R and NR, however further studies are needed to confirm this.</offsets></p><p id="Par41"><offsets xml_i="44003" xml_f="44160" txt_i="26271" txt_f="26428">The samples used in this study came from the SERA study, a large, multi-institutional investigational program designed to identify predictive markers of RA [</offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="44193" xml_f="44195" txt_i="26428" txt_f="26430">50</offsets></xref><offsets xml_i="44202" xml_f="45013" txt_i="26430" txt_f="27241">]. Patients enrolled in the SERA study were carefully characterized for clinical phenotype (the majority of the patients were Caucasians, non-smokers with seropositive established disease), longitudinal follow-up of outcomes and blood samples were stored following a Standard Operating Procedure. This combination of clinical rigor and quality assurance of the inputs to the CCS are particular strengths of this study. An additional strength of the study is the approach that was used to generate the CCS. We focused on the discovery and establishment of a molecular signature using an approach informed by current biological knowledge. We evaluated a network of loci that had plausible pathophysiological relevance in RA via synovial pathogenesis studies, GWAS association and postulated MTX pharmacogenetics [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="45046" xml_f="45048" txt_i="27241" txt_f="27243">35</offsets></xref><offsets xml_i="45055" xml_f="45057" txt_i="27243" txt_f="27245">, </offsets><xref ref-type="bibr" rid="CR54"><offsets xml_i="45090" xml_f="45092" txt_i="27245" txt_f="27247">54</offsets></xref><offsets xml_i="45099" xml_f="45100" txt_i="27247" txt_f="27248">–</offsets><xref ref-type="bibr" rid="CR57"><offsets xml_i="45133" xml_f="45135" txt_i="27248" txt_f="27250">57</offsets></xref><offsets xml_i="45142" xml_f="45144" txt_i="27250" txt_f="27252">, </offsets><xref ref-type="bibr" rid="CR86"><offsets xml_i="45177" xml_f="45179" txt_i="27252" txt_f="27254">86</offsets></xref><offsets xml_i="45186" xml_f="45921" txt_i="27254" txt_f="27989">]. The step-wise selection of biomarkers used in this study, coupled with the strict separation of discovery and validation cohorts, was performed to prevent marker and model over-fitting. The robust statistical properties of the CCS classifier are another advantage of the approach presented here. Using epigenetic markers, which provide a binary readout (presence or absence) and are stable in isolated whole blood, provides high efficiency stratification. Statistical power analysis confirmed that the sample size was adequate for the development and evaluation of the signature, a critical step in biomarker development and aligned with successful development of companion diagnostics in limited size cohorts in other indications [</offsets><xref ref-type="bibr" rid="CR87"><offsets xml_i="45954" xml_f="45956" txt_i="27989" txt_f="27991">87</offsets></xref><offsets xml_i="45963" xml_f="46139" txt_i="27991" txt_f="28167">]. A further strength to our findings and applicability in the clinical setting is the ability to identify this molecular signature in the whole blood by using a drop of blood.</offsets></p><p id="Par42"><offsets xml_i="46157" xml_f="46458" txt_i="28168" txt_f="28469">Some of the caveats associated with our findings are the relatively modest sample size and the cellular heterogeneity present in the whole blood. While the sample size for the validation cohort was not as large as previous studies seeking to identify a biomarker for MTX non-response (Additional file </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="46494" xml_f="46495" txt_i="28469" txt_f="28470">1</offsets></xref><offsets xml_i="46502" xml_f="46658" txt_i="28470" txt_f="28626">: Table S1), the use of chromosome conformations as a readout, which can generate robust signatures in smaller cohorts, provide confidence in the approach [</offsets><xref ref-type="bibr" rid="CR88"><offsets xml_i="46691" xml_f="46693" txt_i="28626" txt_f="28628">88</offsets></xref><offsets xml_i="46700" xml_f="46702" txt_i="28628" txt_f="28630">, </offsets><xref ref-type="bibr" rid="CR89"><offsets xml_i="46735" xml_f="46737" txt_i="28630" txt_f="28632">89</offsets></xref><offsets xml_i="46744" xml_f="46968" txt_i="28632" txt_f="28856">]. The heterogeneity of cell populations is an issue that has implications for any analysis in whole blood. In RA, it is known that there is significant heterogeneity in cell populations present in the blood of RA patients [</offsets><xref ref-type="bibr" rid="CR90"><offsets xml_i="47001" xml_f="47003" txt_i="28856" txt_f="28858">90</offsets></xref><offsets xml_i="47010" xml_f="47563" txt_i="28858" txt_f="29411">]. While outside the scope of the current study, future studies that look at the CCS in distinct cell types in whole blood may shed greater light on the similarities and/or differences exhibited within individual populations. A final caveat of our study was limited in that it could not determine whether the observed epigenetic marks are causal or consequential (secondary to the inflammatory response). Future studies looking at larger patient sets as well as the inclusion of individuals treated with other DMARDs, including biologics, are warranted.</offsets></p></sec><sec id="Sec16" sec-type="conclusion"><title><offsets xml_i="47618" xml_f="47629" txt_i="29413" txt_f="29424">Conclusions</offsets></title><p id="Par43"><offsets xml_i="47651" xml_f="48482" txt_i="29425" txt_f="30256">The ability to identify individuals who are unlikely to respond to a specific therapeutic agent has the potential to not only reduce side-effects to treatments that will provide no benefit but also accelerate their journey through the treatment algorithm and thus increase likelihood of a positive treatment outcome. Given the high worldwide prevalence of RA, a stratifying signature for MTX response offers direct and practical benefits. The validated classifier described here, which is based on five conditional binary biomarkers detected in the blood, offers cost-effective detection by an established ISO-certified industrial platform and the practicality that this method to stratify patients has the potential to be routinely available within clinical practice. This could be a first step toward personalized medicine in RA.</offsets></p></sec></body><back><app-group><app id="App1"><sec id="Sec17"><title>Additional file</title><p>
<media position="anchor" xlink:href="12967_2018_1387_MOESM1_ESM.docx" id="MOESM1"><caption><p><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1186/s12967-018-1387-9"><bold>Additional file 1: Note.</bold></ext-link> Scottish Early Rheumatoid Arthritis inception cohort (SERA). <bold>Table S1.</bold> Previous studies on prediction of MTX response in RA. <bold>Table S2.</bold> Patient characteristics—discovery cohort. <bold>Table S3.</bold> Patient characteristics—test cohort. <bold>Table S4.</bold> Patient characteristics—blinded validation cohort. <bold>Additional Methods.</bold> Statistical analysis and details of assays used to generate the CCS. <bold>Table S5.</bold> Selected genes for EpiSwitch Array. <bold>Figure S1.</bold> Design for discovery and validation of epigenetic stratifying biomarker signature for SERA patients. <bold>Table S6.</bold> Stepwise marker selection. <bold>Table S7.</bold> 65 Selected genes from EpiSwitch Array analysis. <bold>Table S8.</bold> 12 Selected genes from EpiSwitch PCR. <bold>Figure S2.</bold> Graphical representation of the genomic co-ordinates of the CXCL13 CCS marker associated with non-response to MTX. <bold>Figure S3.</bold> Graphical representation of the genomic co-ordinates of the IL- 17A CCS marker associated with non-response to MTX. <bold>Figure S4.</bold> Graphical representation of the genomic co-ordinates of the IFNAR1 CCS marker associated with non-response to MTX. <bold>Figure S5.</bold> Graphical representation of the genomic co-ordinates of the IL- 21R CCS marker associated with non-response to MTX. <bold>Figure S6.</bold> Graphical representation of the genomic co-ordinates of the IL-23 CCS marker associated with non-response to MTX. <bold>Figure S7.</bold> Clinical characteristics of independent blinded validation cohort. <bold>Table S9.</bold> CCS markers associated with eQTLs.</p></caption></media>
</p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>GWAS</term><def><p id="Par5">genome wide association study</p></def></def-item><def-item><term>EULAR</term><def><p id="Par6">European League Against Rheumatism</p></def></def-item><def-item><term>DAS</term><def><p id="Par7">Disease Activity Index</p></def></def-item><def-item><term>ESR</term><def><p id="Par8">erythrocyte sedimentation rate</p></def></def-item><def-item><term>CRP</term><def><p id="Par9">C reactive protein</p></def></def-item><def-item><term>CDAI</term><def><p id="Par10">Clinical Disease Activity Index</p></def></def-item><def-item><term>R</term><def><p id="Par11">responders</p></def></def-item><def-item><term>NR</term><def><p id="Par12">non-responders</p></def></def-item><def-item><term>HC</term><def><p id="Par13">healthy controls</p></def></def-item><def-item><term>RA</term><def><p id="Par14">rheumatoid arthritis</p></def></def-item><def-item><term>MTX</term><def><p id="Par15">methotrexate</p></def></def-item><def-item><term>CCS</term><def><p id="Par16">chromatin conformation signatures</p></def></def-item><def-item><term>SERA</term><def><p id="Par17">Scottish Early Rheumatoid Arthritis</p></def></def-item><def-item><term>TNR</term><def><p id="Par18">true negative response</p></def></def-item><def-item><term>DMARDS</term><def><p id="Par19">disease modifying anti-rheumatic drug</p></def></def-item><def-item><term>ACR</term><def><p id="Par20">American College of Rheumathologists</p></def></def-item><def-item><term>eQTL</term><def><p id="Par21">expression quantitative trait loci</p></def></def-item></def-list></glossary><fn-group><fn><p>Iain B. McInnes and Carl S. Goodyear are senior authors</p></fn><fn><p><bold>Electronic supplementary material</bold></p><p>The online version of this article (10.1186/s12967-018-1387-9) contains supplementary material, which is available to authorized users.</p></fn></fn-group><ack><title>Authors’ contributions</title><p>CC, EH, AA, IBM and CSG conceived the study and wrote the paper. EH, IBM and CSG planned and reviewed experiments and analyzed the data. AR, HW and JG performed experiments. SERA Investigators enrolled patients and provided samples and clinical data. All authors read and approved the final manuscript.</p><sec id="FPar1"><title>Acknowledgements</title><p id="Par44">The authors would like to thank members of OBD Reference Facility Benjamin Foulkes, Chloe Bird, Emily Corfield and Matthew Salter for expedient processing of clinical samples on the EpiSwitch™ platform and Magdalena Jeznach and Willem Westra for help with preparation of the manuscript. The study employed samples from the SERA Biobank used with permission and approval of the SERA Approval Group. We gratefully acknowledge the invaluable contribution of the clinicians and operating team in SERA. We would also like to thank Prof. Raju Kucherlapati (Harvard Medical School), and Prof. Jane Mellor (Oxford Univ.), Prof. John O’Shea (National Institute of Health) and Prof. John Isaacs (New Castle Univ.) for their independent and critical review of our study.</p><p id="Par45">A list of Scottish Early Rheumatoid Arthritis (SERA) inception cohort investigators is provided in Additional file <xref rid="MOESM1" ref-type="media">1</xref>: Additional Note.</p></sec><sec id="FPar2"><title>Competing interests</title><p id="Par46">CC is a former employee at Pfizer Inc. and has no other competing financial interests. ASR, JG, EH and AA are full-time employees at Oxford Biodynamics Limited and have no other competing financial interests. IBM has received consultancy fees and grant funding from Pfizer. IBM has received honorarium from AstraZeneca, BMS, Abbvie, UCB and Roche. CSG receives consultancy fees from AstraZeneca. The remaining authors report no competing financial interests.</p></sec><sec id="FPar3"><title>Availability of data and materials</title><p id="Par47">The datasets used and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></sec><sec id="FPar4"><title>Consent for publication</title><p id="Par48">Not applicable.</p></sec><sec id="FPar6"><title>Ethics approval and consent to participate</title><p id="Par50">Local ethics committees approved the SERA protocol and all participants gave informed consent before enrollment.</p></sec><sec id="FPar7"><title>Funding</title><p id="Par51">This work was funded by Oxford BioDynamics.</p></sec><sec id="FPar8"><title>Publisher’s Note</title><p id="Par52">Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></sec></ack><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Liao</surname><given-names>KP</given-names></name><name><surname>Alfredsson</surname><given-names>L</given-names></name><name><surname>Karlson</surname><given-names>EW</given-names></name></person-group><article-title>Environmental influences on risk for rheumatoid arthritis</article-title><source>Curr Opin Rheumatol</source><year>2009</year><volume>21</volume><fpage>279</fpage><lpage>283</lpage><pub-id pub-id-type="doi">10.1097/BOR.0b013e32832a2e16</pub-id><pub-id pub-id-type="pmid">19318947</pub-id></element-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bottini</surname><given-names>N</given-names></name><name><surname>Firestein</surname><given-names>GS</given-names></name></person-group><article-title>Epigenetics in rheumatoid arthritis: a primer for rheumatologists</article-title><source>Curr Rheumatol Rep</source><year>2013</year><volume>15</volume><fpage>372</fpage><lpage>376</lpage><pub-id pub-id-type="doi">10.1007/s11926-013-0372-9</pub-id><pub-id pub-id-type="pmid">24072602</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>McInnes</surname><given-names>IB</given-names></name><name><surname>Schett</surname><given-names>G</given-names></name></person-group><article-title>The pathogenesis of rheumatoid arthritis</article-title><source>N Engl J Med</source><year>2011</year><volume>365</volume><fpage>2205</fpage><lpage>2219</lpage><pub-id pub-id-type="doi">10.1056/NEJMra1004965</pub-id><pub-id pub-id-type="pmid">22150039</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Smolen</surname><given-names>JS</given-names></name><name><surname>Aletaha</surname><given-names>D</given-names></name><name><surname>Bijlsma</surname><given-names>JWJ</given-names></name><name><surname>Breedveld</surname><given-names>FC</given-names></name><name><surname>Boumpas</surname><given-names>D</given-names></name><name><surname>Burmester</surname><given-names>G</given-names></name><etal></etal></person-group><article-title>Treating rheumatoid arthritis to target: recommendations of an international task force</article-title><source>Ann Rheum Dis</source><year>2010</year><volume>69</volume><fpage>631</fpage><lpage>637</lpage><pub-id pub-id-type="doi">10.1136/ard.2009.123919</pub-id><pub-id pub-id-type="pmid">20215140</pub-id></element-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Singh</surname><given-names>JA</given-names></name><name><surname>Saag</surname><given-names>KG</given-names></name><name><surname>Bridges</surname><given-names>SL</given-names></name><name><surname>Akl</surname><given-names>EA</given-names></name><name><surname>Bannuru</surname><given-names>RR</given-names></name><name><surname>Sullivan</surname><given-names>MC</given-names></name><etal></etal></person-group><article-title>2015 American College of rheumatology guideline for the treatment of rheumatoid arthritis</article-title><source>Arthritis Rheumatol</source><year>2016</year><volume>68</volume><fpage>1</fpage><pub-id pub-id-type="pmid">26545940</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Smolen</surname><given-names>JS</given-names></name><name><surname>Landewé</surname><given-names>R</given-names></name><name><surname>Breedveld</surname><given-names>FC</given-names></name><name><surname>Buch</surname><given-names>M</given-names></name><name><surname>Burmester</surname><given-names>G</given-names></name><name><surname>Dougados</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>EULAR recommendations for the management of rheumatoid arthritis with synthetic</article-title><source>Ann Rheum Dis</source><year>2014</year><volume>73</volume><fpage>492</fpage><lpage>509</lpage><pub-id pub-id-type="doi">10.1136/annrheumdis-2013-204573</pub-id><pub-id pub-id-type="pmid">24161836</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hider</surname><given-names>SL</given-names></name><name><surname>Silman</surname><given-names>AJ</given-names></name><name><surname>Thomson</surname><given-names>W</given-names></name><name><surname>Lunt</surname><given-names>M</given-names></name><name><surname>Bunn</surname><given-names>D</given-names></name><name><surname>Symmons</surname><given-names>DPM</given-names></name></person-group><article-title>Can clinical factors at presentation be used to predict outcome of treatment with methotrexate in patients with early inflammatory polyarthritis?</article-title><source>Ann Rheum Dis</source><year>2009</year><volume>68</volume><fpage>57</fpage><lpage>62</lpage><pub-id pub-id-type="doi">10.1136/ard.2008.088237</pub-id><pub-id pub-id-type="pmid">18292102</pub-id></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>O’Dell</surname><given-names>JR</given-names></name></person-group><article-title>Therapeutic strategies for rheumatoid arthritis</article-title><source>N Engl J Med</source><year>2004</year><volume>350</volume><fpage>2591</fpage><lpage>2602</lpage><pub-id pub-id-type="doi">10.1056/NEJMra040226</pub-id><pub-id pub-id-type="pmid">15201416</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Barrera</surname><given-names>P</given-names></name><name><surname>van der Maas</surname><given-names>A</given-names></name><name><surname>van Ede</surname><given-names>AE</given-names></name><name><surname>Kiemeney</surname><given-names>BA</given-names></name><name><surname>Laan</surname><given-names>RF</given-names></name><name><surname>van de Putte</surname><given-names>LB</given-names></name><etal></etal></person-group><article-title>Drug survival, efficacy and toxicity of monotherapy with a fully human anti-tumour necrosis factor-alpha antibody compared with methotrexate in long-standing rheumatoid arthritis</article-title><source>Rheumatology</source><year>2002</year><volume>41</volume><fpage>430</fpage><lpage>439</lpage><pub-id pub-id-type="doi">10.1093/rheumatology/41.4.430</pub-id><pub-id pub-id-type="pmid">11961174</pub-id></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tm</surname><given-names>F</given-names></name><name><surname>Lunt</surname><given-names>M</given-names></name><name><surname>Fu</surname><given-names>B</given-names></name><name><surname>Bunn</surname><given-names>D</given-names></name><name><surname>Symmons</surname><given-names>D</given-names></name></person-group><article-title>Early treatment with and time on first disease modifying anti-rheumatic drug predicts long-term function in patients with inflammatory polyarthritis</article-title><source>Ann Rheum Dis</source><year>2009</year><volume>69</volume><fpage>689</fpage><lpage>695</lpage><pub-id pub-id-type="pmid">19858538</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bakker</surname><given-names>MF</given-names></name><name><surname>Jacobs</surname><given-names>JWG</given-names></name><name><surname>Welsing</surname><given-names>PMJ</given-names></name><name><surname>Vreugdenhil</surname><given-names>SA</given-names></name><name><surname>van Booma-Frankfort</surname><given-names>C</given-names></name><name><surname>Linn-Rasker</surname><given-names>SP</given-names></name><etal></etal></person-group><article-title>Early clinical response to treatment predicts 5-year outcome in RA patients: follow-up results from the CAMERA study</article-title><source>Ann Rheum Dis</source><year>2011</year><volume>70</volume><fpage>1099</fpage><lpage>1103</lpage><pub-id pub-id-type="doi">10.1136/ard.2010.137943</pub-id><pub-id pub-id-type="pmid"></pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Drouin</surname><given-names>J</given-names></name><name><surname>Haraoui</surname><given-names>B</given-names></name></person-group><article-title>Predictors of clinical response and radiographic progression in patients with rheumatoid arthritis treated with methotrexate monotherapy</article-title><source>J Rheumatol</source><year>2010</year><volume>37</volume><fpage>1405</fpage><lpage>1410</lpage><pub-id pub-id-type="doi">10.3899/jrheum.090838</pub-id><pub-id pub-id-type="pmid">20436076</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hider</surname><given-names>SL</given-names></name><name><surname>Buckley</surname><given-names>C</given-names></name><name><surname>Silman</surname><given-names>AJ</given-names></name><name><surname>Symmons</surname><given-names>DPBI</given-names></name></person-group><article-title>Factors influencing response to disease modifying antirheumatic drugs in patients with rheumatoid arthritis</article-title><source>J Rheumatol</source><year>2005</year><volume>32</volume><fpage>11</fpage><lpage>16</lpage><pub-id pub-id-type="pmid">15700388</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Scherer</surname><given-names>HU</given-names></name><name><surname>Dörner</surname><given-names>T</given-names></name><name><surname>Burmester</surname><given-names>GR</given-names></name></person-group><article-title>Patient-tailored therapy in rheumatoid arthritis: an editorial review</article-title><source>Curr Opin Rheumatol</source><year>2010</year><volume>22</volume><fpage>237</fpage><lpage>245</lpage><pub-id pub-id-type="doi">10.1097/BOR.0b013e328337b832</pub-id><pub-id pub-id-type="pmid">20177384</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ranganathan</surname><given-names>P</given-names></name><name><surname>McLeod</surname><given-names>HL</given-names></name></person-group><article-title>Methotrexate pharmacogenetics: the first step toward individualized therapy in rheumatoid arthritis</article-title><source>Arthritis Rheum</source><year>2006</year><volume>54</volume><fpage>1366</fpage><lpage>1377</lpage><pub-id pub-id-type="doi">10.1002/art.21762</pub-id><pub-id pub-id-type="pmid">16645965</pub-id></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Seitz</surname><given-names>M</given-names></name><name><surname>Zwicker</surname><given-names>M</given-names></name><name><surname>Villiger</surname><given-names>PM</given-names></name></person-group><article-title>Pretreatment cytokine profiles of peripheral blood mononuclear cells and serum from patients with rheumatoid arthritis in different american college of rheumatology response groups to methotrexate</article-title><source>J Rheumatol</source><year>2003</year><volume>30</volume><fpage>28</fpage><lpage>35</lpage><pub-id pub-id-type="pmid">12508386</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stuhlmüller</surname><given-names>B</given-names></name><name><surname>Häupl</surname><given-names>T</given-names></name><name><surname>Hernandez</surname><given-names>MM</given-names></name><name><surname>Grützkau</surname><given-names>A</given-names></name><name><surname>Kuban</surname><given-names>R-J</given-names></name><name><surname>Tandon</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>CD11c as a transcriptional biomarker to predict response to anti-TNF monotherapy with adalimumab in patients with rheumatoid arthritis</article-title><source>Clin Pharmacol Ther</source><year>2010</year><volume>87</volume><fpage>311</fpage><lpage>321</lpage><pub-id pub-id-type="doi">10.1038/clpt.2009.244</pub-id><pub-id pub-id-type="pmid">20032971</pub-id></element-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Dhir</surname><given-names>V</given-names></name><name><surname>Sandhu</surname><given-names>A</given-names></name><name><surname>Gupta</surname><given-names>N</given-names></name><name><surname>Dhawan</surname><given-names>V</given-names></name><name><surname>Sharma</surname><given-names>S</given-names></name><name><surname>Sharma</surname><given-names>A</given-names></name></person-group><article-title>Low serum levels of myeloid progenitor inhibitory factor-1 predict good response to methotrexate in rheumatoid arthritis</article-title><source>ISRN Inflamm</source><year>2013</year><volume>2013</volume><fpage>460469</fpage><pub-id pub-id-type="doi">10.1155/2013/460469</pub-id><pub-id pub-id-type="pmid">24455421</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ally</surname><given-names>M</given-names></name><name><surname>Hodkinson</surname><given-names>B</given-names></name><name><surname>Meyer</surname><given-names>PWA</given-names></name><name><surname>Musenge</surname><given-names>E</given-names></name><name><surname>Tintinger</surname><given-names>GR</given-names></name><name><surname>Tikly</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Circulating anti-citrullinated peptide antibodies, cytokines and genotype as biomarkers of response to disease-modifying antirheumatic drug therapy in early rheumatoid arthritis</article-title><source>BMC Musculoskelet Disord</source><year>2015</year><volume>16</volume><fpage>130</fpage><pub-id pub-id-type="doi">10.1186/s12891-015-0587-1</pub-id><pub-id pub-id-type="pmid">26021985</pub-id></element-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Dervieux</surname><given-names>T</given-names></name><name><surname>Greenstein</surname><given-names>N</given-names></name><name><surname>Kremer</surname><given-names>J</given-names></name></person-group><article-title>Pharmacogenomic and metabolic biomarkers in the folate pathway and their association with methotrexate effects during dosage escalation in rheumatoid arthritis</article-title><source>Arthritis Rheum</source><year>2006</year><volume>54</volume><fpage>3095</fpage><lpage>3103</lpage><pub-id pub-id-type="doi">10.1002/art.22129</pub-id><pub-id pub-id-type="pmid">17009228</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kumagai</surname><given-names>K</given-names></name><name><surname>Hiyama</surname><given-names>K</given-names></name><name><surname>Oyama</surname><given-names>T</given-names></name><name><surname>Maeda</surname><given-names>H</given-names></name><name><surname>Kohno</surname><given-names>N</given-names></name></person-group><article-title>Polymorphisms in the thymidylate synthase and methylenetetrahydrofolate reductase genes and sensitivity to the low-dose methotrexate therapy in patients with rheumatoid arthritis</article-title><source>Int J Mol Med</source><year>2003</year><volume>11</volume><fpage>593</fpage><lpage>600</lpage><pub-id pub-id-type="pmid">12684695</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Inoue</surname><given-names>S</given-names></name><name><surname>Hashiguchi</surname><given-names>M</given-names></name><name><surname>Takagi</surname><given-names>K</given-names></name><name><surname>Kawai</surname><given-names>S</given-names></name><name><surname>Mochizuki</surname><given-names>M</given-names></name></person-group><article-title>Preliminary study to identify the predictive factors for the response to methotrexate therapy in patients with rheumatoid arthritis</article-title><source>Yakugaku Zasshi</source><year>2009</year><volume>129</volume><fpage>843</fpage><lpage>849</lpage><pub-id pub-id-type="doi">10.1248/yakushi.129.843</pub-id><pub-id pub-id-type="pmid">19571519</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hayashi</surname><given-names>H</given-names></name><name><surname>Fujimaki</surname><given-names>C</given-names></name><name><surname>Daimon</surname><given-names>T</given-names></name><name><surname>Tsuboi</surname><given-names>S</given-names></name><name><surname>Matsuyama</surname><given-names>T</given-names></name><name><surname>Itoh</surname><given-names>K</given-names></name></person-group><article-title>Genetic polymorphisms in folate pathway enzymes as a possible marker for predicting the outcome of methotrexate therapy in Japanese patients with rheumatoid arthritis</article-title><source>J Clin Pharm Ther</source><year>2009</year><volume>34</volume><fpage>355</fpage><lpage>361</lpage><pub-id pub-id-type="doi">10.1111/j.1365-2710.2009.01046.x</pub-id><pub-id pub-id-type="pmid">19827168</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Klareskog</surname><given-names>L</given-names></name><name><surname>Catrina</surname><given-names>AI</given-names></name><name><surname>Paget</surname><given-names>S</given-names></name></person-group><article-title>Rheumatoid arthritis</article-title><source>Lancet</source><year>2009</year><volume>373</volume><fpage>659</fpage><lpage>672</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(09)60008-8</pub-id><pub-id pub-id-type="pmid">19157532</pub-id></element-citation></ref><ref id="CR25"><label>25.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Scott</surname><given-names>D</given-names></name><name><surname>Wolfe</surname><given-names>F</given-names></name><name><surname>Huizinga</surname><given-names>T</given-names></name></person-group><article-title>Rheumatoid arthritis</article-title><source>Lancet</source><year>2010</year><volume>376</volume><fpage>1094</fpage><lpage>1108</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(10)60826-4</pub-id><pub-id pub-id-type="pmid">20870100</pub-id></element-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tordini</surname><given-names>F</given-names></name><name><surname>Aldinucci</surname><given-names>M</given-names></name><name><surname>Milanesi</surname><given-names>L</given-names></name><name><surname>Liò</surname><given-names>P</given-names></name><name><surname>Merelli</surname><given-names>I</given-names></name></person-group><article-title>The genome conformation as an integrator of multi-omic data: the example of damage spreading in cancer</article-title><source>Front Genet</source><year>2016</year><volume>7</volume><fpage>1</fpage><lpage>16</lpage><pub-id pub-id-type="doi">10.3389/fgene.2016.00194</pub-id><pub-id pub-id-type="pmid">26858746</pub-id></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ling</surname><given-names>JQ</given-names></name><name><surname>Hoffman</surname><given-names>AR</given-names></name></person-group><article-title>Epigenetics of long-range chromatin interactions</article-title><source>Pediatr Res</source><year>2007</year><volume>61</volume><fpage>11R</fpage><lpage>16R</lpage><pub-id pub-id-type="doi">10.1203/pdr.0b013e31804575db</pub-id><pub-id pub-id-type="pmid">17413850</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Deng</surname><given-names>W</given-names></name><name><surname>Blobel</surname><given-names>GA</given-names></name></person-group><article-title>Do chromatin loops provide epigenetic gene expression states?</article-title><source>Curr Opin Genet Dev</source><year>2010</year><volume>20</volume><fpage>548</fpage><lpage>554</lpage><pub-id pub-id-type="doi">10.1016/j.gde.2010.06.007</pub-id><pub-id pub-id-type="pmid">20598523</pub-id></element-citation></ref><ref id="CR29"><label>29.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kadauke</surname><given-names>S</given-names></name><name><surname>Blobel</surname><given-names>GA</given-names></name></person-group><article-title>Chromatin loops in gene regulation</article-title><source>Biochim Biophys Acta Gene Regul Mech</source><year>2009</year><volume>1789</volume><fpage>17</fpage><lpage>25</lpage><pub-id pub-id-type="doi">10.1016/j.bbagrm.2008.07.002</pub-id></element-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Christova</surname><given-names>R</given-names></name><name><surname>Jones</surname><given-names>T</given-names></name><name><surname>Wu</surname><given-names>P-J</given-names></name><name><surname>Bolzer</surname><given-names>A</given-names></name><name><surname>Costa-Pereira</surname><given-names>AP</given-names></name><name><surname>Watling</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>P-STAT1 mediates higher-order chromatin remodelling of the human MHC in response to IFNgamma</article-title><source>J Cell Sci</source><year>2007</year><volume>120</volume><fpage>3262</fpage><lpage>3270</lpage><pub-id pub-id-type="doi">10.1242/jcs.012328</pub-id><pub-id pub-id-type="pmid">17726060</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Richardson</surname><given-names>B</given-names></name><name><surname>Kahn</surname><given-names>L</given-names></name><name><surname>Lovett</surname><given-names>EJ</given-names></name><name><surname>Hudson</surname><given-names>J</given-names></name></person-group><article-title>Effect of an inhibitor of DNA methylation on T cells. I. 5-Azacytidine induces T4 expression on T8+ T cells</article-title><source>J Immunol</source><year>1986</year><volume>137</volume><fpage>35</fpage><lpage>39</lpage><pub-id pub-id-type="pmid">2423610</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Glossop</surname><given-names>JR</given-names></name><name><surname>Emes</surname><given-names>RD</given-names></name><name><surname>Nixon</surname><given-names>NB</given-names></name><name><surname>Haworth</surname><given-names>KE</given-names></name><name><surname>Packham</surname><given-names>JC</given-names></name><name><surname>Dawes</surname><given-names>PT</given-names></name><etal></etal></person-group><article-title>Genome-wide DNA methylation profiling in rheumatoid arthritis identifies disease-associated methylation changes that are distinct to individual T- and B-lymphocyte populations</article-title><source>Epigenetics</source><year>2014</year><volume>9</volume><fpage>1228</fpage><lpage>1237</lpage><pub-id pub-id-type="doi">10.4161/epi.29718</pub-id><pub-id pub-id-type="pmid">25147922</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kennedy</surname><given-names>A</given-names></name><name><surname>Schmidt</surname><given-names>EM</given-names></name><name><surname>Cribbs</surname><given-names>AP</given-names></name><name><surname>Penn</surname><given-names>H</given-names></name><name><surname>Amjadi</surname><given-names>P</given-names></name><name><surname>Syed</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>A novel upstream enhancer of FOXP3, sensitive to methylation-induced silencing, exhibits dysregulated methylation in rheumatoid arthritis Treg cells</article-title><source>Eur J Immunol</source><year>2014</year><volume>44</volume><fpage>2968</fpage><lpage>2978</lpage><pub-id pub-id-type="doi">10.1002/eji.201444453</pub-id><pub-id pub-id-type="pmid">25042153</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Nile</surname><given-names>CJ</given-names></name><name><surname>Read</surname><given-names>RC</given-names></name><name><surname>Akil</surname><given-names>M</given-names></name><name><surname>Duff</surname><given-names>GW</given-names></name><name><surname>Wilson</surname><given-names>AG</given-names></name></person-group><article-title>Methylation status of a single CpG site in the IL6 promoter is related to IL6 messenger RNA levels and rheumatoid arthritis</article-title><source>Arthritis Rheum</source><year>2008</year><volume>58</volume><fpage>2686</fpage><lpage>2693</lpage><pub-id pub-id-type="doi">10.1002/art.23758</pub-id><pub-id pub-id-type="pmid">18759290</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Nakano</surname><given-names>K</given-names></name><name><surname>Whitaker</surname><given-names>JW</given-names></name><name><surname>Boyle</surname><given-names>DL</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Firestein</surname><given-names>GS</given-names></name></person-group><article-title>DNA methylome signature in rheumatoid arthritis</article-title><source>Ann Rheum Dis</source><year>2013</year><volume>72</volume><fpage>110</fpage><lpage>117</lpage><pub-id pub-id-type="doi">10.1136/annrheumdis-2012-201526</pub-id><pub-id pub-id-type="pmid">22736089</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Aryee</surname><given-names>MJ</given-names></name><name><surname>Padyukov</surname><given-names>L</given-names></name><name><surname>Fallin</surname><given-names>MD</given-names></name><name><surname>Hesselberg</surname><given-names>E</given-names></name><name><surname>Runarsson</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Epigenome-wide association data implicate DNA methylation as an intermediary of genetic risk in rheumatoid arthritis</article-title><source>Nat Biotechnol</source><year>2013</year><volume>31</volume><fpage>142</fpage><lpage>147</lpage><pub-id pub-id-type="doi">10.1038/nbt.2487</pub-id><pub-id pub-id-type="pmid">23334450</pub-id></element-citation></ref><ref id="CR37"><label>37.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>de la Rica</surname><given-names>L</given-names></name><name><surname>Urquiza</surname><given-names>JM</given-names></name><name><surname>Gómez-Cabrero</surname><given-names>D</given-names></name><name><surname>Islam</surname><given-names>ABMMK</given-names></name><name><surname>López-Bigas</surname><given-names>N</given-names></name><name><surname>Tegnér</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Identification of novel markers in rheumatoid arthritis through integrated analysis of DNA methylation and microRNA expression</article-title><source>J Autoimmun</source><year>2013</year><volume>41</volume><fpage>6</fpage><lpage>16</lpage><pub-id pub-id-type="doi">10.1016/j.jaut.2012.12.005</pub-id><pub-id pub-id-type="pmid">23306098</pub-id></element-citation></ref><ref id="CR38"><label>38.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Viatte</surname><given-names>S</given-names></name><name><surname>Plant</surname><given-names>D</given-names></name><name><surname>Raychaudhuri</surname><given-names>S</given-names></name></person-group><article-title>Genetics and epigenetics of rheumatoid arthritis</article-title><source>Nat Rev Rheumatol</source><year>2013</year><volume>9</volume><fpage>141</fpage><lpage>153</lpage><pub-id pub-id-type="doi">10.1038/nrrheum.2012.237</pub-id><pub-id pub-id-type="pmid">23381558</pub-id></element-citation></ref><ref id="CR39"><label>39.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cribbs</surname><given-names>AP</given-names></name><name><surname>Kennedy</surname><given-names>A</given-names></name><name><surname>Penn</surname><given-names>H</given-names></name><name><surname>Amjadi</surname><given-names>P</given-names></name><name><surname>Green</surname><given-names>P</given-names></name><name><surname>Read</surname><given-names>JE</given-names></name><etal></etal></person-group><article-title>Methotrexate restores regulatory T cell function through demethylation of the FoxP3 upstream enhancer in patients with rheumatoid arthritis</article-title><source>Arthritis Rheumatol</source><year>2015</year><volume>67</volume><fpage>1182</fpage><lpage>1192</lpage><pub-id pub-id-type="doi">10.1002/art.39031</pub-id><pub-id pub-id-type="pmid">25604080</pub-id></element-citation></ref><ref id="CR40"><label>40.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ellis</surname><given-names>JA</given-names></name><name><surname>Munro</surname><given-names>JE</given-names></name><name><surname>Chavez</surname><given-names>RA</given-names></name><name><surname>Gordon</surname><given-names>L</given-names></name><name><surname>Joo</surname><given-names>JE</given-names></name><name><surname>Akikusa</surname><given-names>JD</given-names></name><etal></etal></person-group><article-title>Genome-scale case-control analysis of CD4+ T-cell DNA methylation in juvenile idiopathic arthritis reveals potential targets involved in disease</article-title><source>Clin Epigenetics</source><year>2012</year><volume>4</volume><fpage>20</fpage><pub-id pub-id-type="doi">10.1186/1868-7083-4-20</pub-id><pub-id pub-id-type="pmid">23148518</pub-id></element-citation></ref><ref id="CR41"><label>41.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kim</surname><given-names>Y-I</given-names></name><name><surname>Logan</surname><given-names>JW</given-names></name><name><surname>Mason</surname><given-names>JB</given-names></name><name><surname>Roubenoff</surname><given-names>R</given-names></name></person-group><article-title>DNA hypomethylation in inflammatory arthritis: reversal with methotrexate</article-title><source>J Lab Clin Med</source><year>1996</year><volume>128</volume><fpage>165</fpage><lpage>172</lpage><pub-id pub-id-type="doi">10.1016/S0022-2143(96)90008-6</pub-id><pub-id pub-id-type="pmid">8765212</pub-id></element-citation></ref><ref id="CR42"><label>42.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Boyan</surname><given-names>B</given-names></name><name><surname>Giacomo</surname><given-names>C</given-names></name></person-group><article-title>Organization and function of the 3D genome</article-title><source>Nat Rev Genet</source><year>2016</year><volume>17</volume><fpage>661</fpage><lpage>678</lpage><pub-id pub-id-type="doi">10.1038/nrg.2016.112</pub-id><pub-id pub-id-type="pmid">27739532</pub-id></element-citation></ref><ref id="CR43"><label>43.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cao</surname><given-names>F</given-names></name><name><surname>Fang</surname><given-names>Y</given-names></name><name><surname>Tan</surname><given-names>HK</given-names></name><name><surname>Goh</surname><given-names>Y</given-names></name><name><surname>Choy</surname><given-names>JYH</given-names></name><name><surname>Koh</surname><given-names>BTH</given-names></name><etal></etal></person-group><article-title>Super-enhancers and broad H3K4me3 domains form complex gene regulatory circuits involving chromatin interactions</article-title><source>Sci Rep</source><year>2017</year><volume>7</volume><fpage>2186</fpage><pub-id pub-id-type="doi">10.1038/s41598-017-02257-3</pub-id><pub-id pub-id-type="pmid">28526829</pub-id></element-citation></ref><ref id="CR44"><label>44.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bastonini</surname><given-names>E</given-names></name><name><surname>Jeznach</surname><given-names>M</given-names></name><name><surname>Field</surname><given-names>M</given-names></name><name><surname>Juszczyk</surname><given-names>K</given-names></name><name><surname>Corfield</surname><given-names>E</given-names></name><name><surname>Dezfouli</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Chromatin barcodes as biomarkers for melanoma</article-title><source>Pigment Cell Melanoma Res</source><year>2014</year><volume>27</volume><fpage>788</fpage><lpage>800</lpage><pub-id pub-id-type="doi">10.1111/pcmr.12258</pub-id><pub-id pub-id-type="pmid">24807349</pub-id></element-citation></ref><ref id="CR45"><label>45.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mukhopadhyay</surname><given-names>S</given-names></name><name><surname>Ramadass</surname><given-names>AS</given-names></name><name><surname>Akoulitchev</surname><given-names>A</given-names></name><name><surname>Gordon</surname><given-names>S</given-names></name></person-group><article-title>Formation of distinct chromatin conformation signatures epigenetically regulate macrophage activation</article-title><source>Int Immunopharmacol</source><year>2014</year><volume>18</volume><fpage>7</fpage><lpage>11</lpage><pub-id pub-id-type="doi">10.1016/j.intimp.2013.10.024</pub-id><pub-id pub-id-type="pmid">24211766</pub-id></element-citation></ref><ref id="CR46"><label>46.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Jakub</surname><given-names>JW</given-names></name><name><surname>Grotz</surname><given-names>TE</given-names></name><name><surname>Jordan</surname><given-names>P</given-names></name><name><surname>Hunter</surname><given-names>E</given-names></name><name><surname>Pittelkow</surname><given-names>M</given-names></name><name><surname>Ramadass</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>A pilot study of chromosomal aberrations and epigenetic changes in peripheral blood samples to identify patients with melanoma</article-title><source>Melanoma Res</source><year>2015</year><volume>25</volume><fpage>406</fpage><lpage>411</lpage><pub-id pub-id-type="doi">10.1097/CMR.0000000000000182</pub-id><pub-id pub-id-type="pmid">26225582</pub-id></element-citation></ref><ref id="CR47"><label>47.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Watanabe</surname><given-names>T</given-names></name><name><surname>Ishihara</surname><given-names>K</given-names></name><name><surname>Hirosue</surname><given-names>A</given-names></name><name><surname>Watanabe</surname><given-names>S</given-names></name><name><surname>Hino</surname><given-names>S</given-names></name><name><surname>Ojima</surname><given-names>H</given-names></name><etal></etal></person-group><article-title>Higher-order chromatin regulation and differential gene expression in the human tumor necrosis factor/lymphotoxin locus in hepatocellular carcinoma cells</article-title><source>Mol Cell Biol</source><year>2012</year><volume>32</volume><fpage>1529</fpage><lpage>1541</lpage><pub-id pub-id-type="doi">10.1128/MCB.06478-11</pub-id><pub-id pub-id-type="pmid">22354988</pub-id></element-citation></ref><ref id="CR48"><label>48.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Harismendy</surname><given-names>O</given-names></name><name><surname>Notani</surname><given-names>D</given-names></name><name><surname>Song</surname><given-names>X</given-names></name><name><surname>Rahim</surname><given-names>NG</given-names></name><name><surname>Tanasa</surname><given-names>B</given-names></name><name><surname>Heintzman</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>9p21 DNA variants associated with coronary artery disease impair interferon-gamma signalling response</article-title><source>Nature</source><year>2011</year><volume>470</volume><fpage>264</fpage><lpage>268</lpage><pub-id pub-id-type="doi">10.1038/nature09753</pub-id><pub-id pub-id-type="pmid">21307941</pub-id></element-citation></ref><ref id="CR49"><label>49.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Crutchley</surname><given-names>JL</given-names></name><name><surname>Wang</surname><given-names>XQD</given-names></name><name><surname>Ferraiuolo</surname><given-names>MA</given-names></name><name><surname>Dostie</surname><given-names>J</given-names></name></person-group><article-title>Chromatin conformation signatures: ideal human disease biomarkers?</article-title><source>Biomark Med</source><year>2010</year><volume>4</volume><fpage>611</fpage><lpage>629</lpage><pub-id pub-id-type="doi">10.2217/bmm.10.68</pub-id><pub-id pub-id-type="pmid">20701449</pub-id></element-citation></ref><ref id="CR50"><label>50.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Dale</surname><given-names>J</given-names></name><name><surname>Paterson</surname><given-names>C</given-names></name><name><surname>Tierney</surname><given-names>A</given-names></name><name><surname>Ralston</surname><given-names>SH</given-names></name><name><surname>Reid</surname><given-names>DM</given-names></name><name><surname>Basu</surname><given-names>N</given-names></name><etal></etal></person-group><article-title>The Scottish Early Rheumatoid Arthritis (SERA) Study: an inception cohort and biobank</article-title><source>BMC Musculoskelet Disord</source><year>2016</year><volume>17</volume><fpage>461</fpage><pub-id pub-id-type="doi">10.1186/s12891-016-1318-y</pub-id><pub-id pub-id-type="pmid">27829394</pub-id></element-citation></ref><ref id="CR51"><label>51.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kronisch</surname><given-names>C</given-names></name><name><surname>McLernon</surname><given-names>DJ</given-names></name><name><surname>Dale</surname><given-names>J</given-names></name><name><surname>Paterson</surname><given-names>C</given-names></name><name><surname>Ralston</surname><given-names>SH</given-names></name><name><surname>Reid</surname><given-names>DM</given-names></name><etal></etal></person-group><article-title>Brief report: predicting functional disability: one-year results from the scottish early rheumatoid arthritis inception cohort</article-title><source>Arthritis Rheumatol</source><year>2016</year><volume>68</volume><fpage>1596</fpage><lpage>1602</lpage><pub-id pub-id-type="doi">10.1002/art.39627</pub-id><pub-id pub-id-type="pmid">26866516</pub-id></element-citation></ref><ref id="CR52"><label>52.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Smolen</surname><given-names>JS</given-names></name><name><surname>Aletaha</surname><given-names>D</given-names></name></person-group><article-title>Scores for all seasons: SDAI and CDAI</article-title><source>Clin Exp Rheumatol</source><year>2014</year><volume>32</volume><fpage>S75</fpage><lpage>S79</lpage><pub-id pub-id-type="pmid">25268663</pub-id></element-citation></ref><ref id="CR53"><label>53.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Aletaha</surname><given-names>D</given-names></name><name><surname>Martinez-Avila</surname><given-names>J</given-names></name><name><surname>Kvien</surname><given-names>TK</given-names></name><name><surname>Smolen</surname><given-names>JS</given-names></name></person-group><article-title>Definition of treatment response in rheumatoid arthritis based on the simplified and the clinical disease activity index</article-title><source>Ann Rheum Dis</source><year>2012</year><volume>71</volume><fpage>1190</fpage><lpage>1196</lpage><pub-id pub-id-type="doi">10.1136/annrheumdis-2012-201491</pub-id><pub-id pub-id-type="pmid">22454398</pub-id></element-citation></ref><ref id="CR54"><label>54.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kooloos</surname><given-names>WM</given-names></name><name><surname>Wessels</surname><given-names>JA</given-names></name><name><surname>van der Straaten</surname><given-names>T</given-names></name><name><surname>Allaart</surname><given-names>CF</given-names></name><name><surname>Huizinga</surname><given-names>TW</given-names></name><name><surname>Guchelaar</surname><given-names>HJ</given-names></name></person-group><article-title>Functional polymorphisms and methotrexate treatment outcome in recent-onset</article-title><source>Pharmacogenomics</source><year>2010</year><volume>11</volume><fpage>163</fpage><lpage>175</lpage><pub-id pub-id-type="doi">10.2217/pgs.09.139</pub-id><pub-id pub-id-type="pmid">20136356</pub-id></element-citation></ref><ref id="CR55"><label>55.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stahl</surname><given-names>EA</given-names></name><name><surname>Raychaudhuri</surname><given-names>S</given-names></name><name><surname>Remmers</surname><given-names>EF</given-names></name><name><surname>Xie</surname><given-names>G</given-names></name><name><surname>Eyre</surname><given-names>S</given-names></name><name><surname>Thomson</surname><given-names>BP</given-names></name><etal></etal></person-group><article-title>Genome-wide association study meta-analysis identifies seven new rheumatoid arthritis risk loci</article-title><source>Nat Genet</source><year>2010</year><volume>42</volume><fpage>508</fpage><lpage>514</lpage><pub-id pub-id-type="doi">10.1038/ng.582</pub-id><pub-id pub-id-type="pmid">20453842</pub-id></element-citation></ref><ref id="CR56"><label>56.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Eyre</surname><given-names>S</given-names></name><name><surname>Bowes</surname><given-names>J</given-names></name><name><surname>Diogo</surname><given-names>D</given-names></name><name><surname>Lee</surname><given-names>A</given-names></name><name><surname>Barton</surname><given-names>A</given-names></name><name><surname>Martin</surname><given-names>P</given-names></name><etal></etal></person-group><article-title>High-density genetic mapping identifies new susceptibility loci for rheumatoid arthritis</article-title><source>Nat Genet</source><year>2012</year><volume>44</volume><fpage>1336</fpage><lpage>1340</lpage><pub-id pub-id-type="doi">10.1038/ng.2462</pub-id><pub-id pub-id-type="pmid">23143596</pub-id></element-citation></ref><ref id="CR57"><label>57.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Okada</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>D</given-names></name><name><surname>Trynka</surname><given-names>G</given-names></name><name><surname>Raj</surname><given-names>T</given-names></name><name><surname>Terao</surname><given-names>C</given-names></name><name><surname>Ikari</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>Genetics of rheumatoid arthritis contributes to biology and drug discovery</article-title><source>Nature</source><year>2014</year><volume>506</volume><fpage>376</fpage><lpage>381</lpage><pub-id pub-id-type="doi">10.1038/nature12873</pub-id><pub-id pub-id-type="pmid">24390342</pub-id></element-citation></ref><ref id="CR58"><label>58.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Walsh</surname><given-names>AM</given-names></name><name><surname>Whitaker</surname><given-names>JW</given-names></name><name><surname>Huang</surname><given-names>CC</given-names></name><name><surname>Cherkas</surname><given-names>Y</given-names></name><name><surname>Lamberth</surname><given-names>SL</given-names></name><name><surname>Brodmerkel</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>Integrative genomic deconvolution of rheumatoid arthritis GWAS loci into gene and cell type associations</article-title><source>Genome Biol</source><year>2016</year><volume>17</volume><fpage>79</fpage><pub-id pub-id-type="doi">10.1186/s13059-016-0948-6</pub-id><pub-id pub-id-type="pmid">27140173</pub-id></element-citation></ref><ref id="CR59"><label>59.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Quinlan</surname><given-names>AR</given-names></name><name><surname>Hall</surname><given-names>IM</given-names></name></person-group><article-title>BEDTools: a flexible suite of utilities for comparing genomic features</article-title><source>Bioinformatics</source><year>2010</year><volume>26</volume><fpage>841</fpage><lpage>842</lpage><pub-id pub-id-type="doi">10.1093/bioinformatics/btq033</pub-id><pub-id pub-id-type="pmid">20110278</pub-id></element-citation></ref><ref id="CR60"><label>60.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rau</surname><given-names>R</given-names></name><name><surname>Herborn</surname><given-names>G</given-names></name></person-group><article-title>Benefit and risk of methotrexate treatment in rheumatoid arthritis</article-title><source>Clin Exp Rheumatol</source><year>2004</year><volume>22</volume><fpage>S83</fpage><lpage>S94</lpage><pub-id pub-id-type="pmid">15552520</pub-id></element-citation></ref><ref id="CR61"><label>61.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Polido-Pereira</surname><given-names>J</given-names></name><name><surname>Vieira-Sousa</surname><given-names>E</given-names></name><name><surname>Fonseca</surname><given-names>JE</given-names></name></person-group><article-title>Rheumatoid arthritis: what is refractory disease and how to manage it?</article-title><source>Autoimmun Rev</source><year>2011</year><volume>10</volume><fpage>707</fpage><lpage>713</lpage><pub-id pub-id-type="doi">10.1016/j.autrev.2011.04.023</pub-id><pub-id pub-id-type="pmid">21570496</pub-id></element-citation></ref><ref id="CR62"><label>62.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kwoh</surname><given-names>CK</given-names></name><name><surname>Anderson</surname><given-names>LG</given-names></name><name><surname>Greene</surname><given-names>JM</given-names></name><name><surname>Johnson</surname><given-names>DA</given-names></name><name><surname>O’Dell</surname><given-names>JR</given-names></name><name><surname>Robbins</surname><given-names>ML</given-names></name><etal></etal></person-group><article-title>Guidelines for the management of rheumatoid arthritis: 2002 update—American College of Rheumatology subcommittee on rheumatoid arthritis guidelines</article-title><source>Arthritis Rheum</source><year>2002</year><volume>46</volume><fpage>328</fpage><lpage>346</lpage><pub-id pub-id-type="doi">10.1002/art.10506</pub-id><pub-id pub-id-type="pmid">11840435</pub-id></element-citation></ref><ref id="CR63"><label>63.</label><mixed-citation publication-type="other">Hazlewood GS, Barnabe C, Tomlinson G, Marshall D, Devoe DJA, Bombardier C. Methotrexate monotherapy and methotrexate combination therapy with traditional and biologic disease modifying anti-rheumatic drugs for rheumatoid arthritis: a network meta-analysis. Cochrane Database Syst Rev. 2016.</mixed-citation></ref><ref id="CR64"><label>64.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Morita</surname><given-names>K</given-names></name><name><surname>Tanaka</surname><given-names>K</given-names></name><name><surname>Saito</surname><given-names>S</given-names></name><name><surname>Kitamura</surname><given-names>K</given-names></name><name><surname>Morimoto</surname><given-names>M</given-names></name><name><surname>Sekihara</surname><given-names>H</given-names></name></person-group><article-title>H S. Expression of interferon receptor genes (IFNAR1 and IFNAR2 mRNA) in the liver may predict outcome after interferon therapy in patients with chronic genotype 2a or 2b hepatitis C virus infection</article-title><source>J Clin Gastroenterol</source><year>1998</year><volume>26</volume><fpage>135</fpage><lpage>140</lpage><pub-id pub-id-type="doi">10.1097/00004836-199803000-00010</pub-id><pub-id pub-id-type="pmid">9563926</pub-id></element-citation></ref><ref id="CR65"><label>65.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hsu</surname><given-names>CS</given-names></name><name><surname>Hsu</surname><given-names>SJ</given-names></name><name><surname>Liu</surname><given-names>WL</given-names></name><name><surname>Chen</surname><given-names>CL</given-names></name><name><surname>Liu</surname><given-names>CJ</given-names></name><name><surname>Chen</surname><given-names>PJ</given-names></name><etal></etal></person-group><article-title>IL-21R gene polymorphisms and serum IL-21 levels predict virological response to interferon-based therapy in Asian chronic hepatitis C patients</article-title><source>Antivir Ther</source><year>2013</year><volume>18</volume><fpage>599</fpage><lpage>606</lpage><pub-id pub-id-type="doi">10.3851/IMP2502</pub-id><pub-id pub-id-type="pmid">23296193</pub-id></element-citation></ref><ref id="CR66"><label>66.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Massirer</surname><given-names>KB</given-names></name><name><surname>Hirata</surname><given-names>MH</given-names></name><name><surname>Silva</surname><given-names>AEB</given-names></name><name><surname>Ferraz</surname><given-names>MLG</given-names></name><name><surname>Nguyen</surname><given-names>NY</given-names></name><name><surname>Hirata</surname><given-names>RDC</given-names></name></person-group><article-title>Interferon-alpha receptor 1 mRNA expression in peripheral blood mononuclear cells is associated with response to interferon-alpha therapy of patients with chronic hepatitis C</article-title><source>Braz J Med Biol Res</source><year>2004</year><volume>37</volume><fpage>643</fpage><lpage>647</lpage><pub-id pub-id-type="doi">10.1590/S0100-879X2004000500003</pub-id><pub-id pub-id-type="pmid">15107924</pub-id></element-citation></ref><ref id="CR67"><label>67.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kirkham</surname><given-names>BW</given-names></name><name><surname>Lassere</surname><given-names>MN</given-names></name><name><surname>Edmonds</surname><given-names>JP</given-names></name><name><surname>Juhasz</surname><given-names>KM</given-names></name><name><surname>Bird</surname><given-names>PA</given-names></name><name><surname>Lee</surname><given-names>CS</given-names></name><etal></etal></person-group><article-title>Synovial membrane cytokine expression is predictive of joint damage progression in rheumatoid arthritis: a two-year prospective study (the DAMAGE study cohort)</article-title><source>Arthritis Rheum</source><year>2006</year><volume>54</volume><fpage>1122</fpage><lpage>1131</lpage><pub-id pub-id-type="doi">10.1002/art.21749</pub-id><pub-id pub-id-type="pmid">16572447</pub-id></element-citation></ref><ref id="CR68"><label>68.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chen</surname><given-names>D-Y</given-names></name><name><surname>Chen</surname><given-names>Y-M</given-names></name><name><surname>Chen</surname><given-names>H-H</given-names></name><name><surname>Hsieh</surname><given-names>C-W</given-names></name><name><surname>Lin</surname><given-names>C-C</given-names></name><name><surname>Lan</surname><given-names>J-L</given-names></name></person-group><article-title>Increasing levels of circulating Th17 cells and interleukin-17 in rheumatoid arthritis patients with an inadequate response to anti-TNF-α therapy</article-title><source>Arthritis Res Ther</source><year>2011</year><volume>13</volume><fpage>R126</fpage><pub-id pub-id-type="doi">10.1186/ar3431</pub-id><pub-id pub-id-type="pmid">21801431</pub-id></element-citation></ref><ref id="CR69"><label>69.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kellner</surname><given-names>H</given-names></name></person-group><article-title>Targeting interleukin-17 in patients with active rheumatoid arthritis: rationale and clinical potential</article-title><source>Ther Adv Musculoskelet Dis</source><year>2013</year><volume>5</volume><fpage>141</fpage><lpage>152</lpage><pub-id pub-id-type="doi">10.1177/1759720X13485328</pub-id><pub-id pub-id-type="pmid">23858337</pub-id></element-citation></ref><ref id="CR70"><label>70.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Meeuwisse</surname><given-names>CM</given-names></name><name><surname>Van Der Linden</surname><given-names>MP</given-names></name><name><surname>Rullmann</surname><given-names>TA</given-names></name><name><surname>Allaart</surname><given-names>CF</given-names></name><name><surname>Nelissen</surname><given-names>R</given-names></name><name><surname>Huizinga</surname><given-names>TW</given-names></name><etal></etal></person-group><article-title>Identification of CXCL13 as a marker for rheumatoid arthritis outcome using an in silico model of the rheumatic joint</article-title><source>Arthritis Rheum</source><year>2011</year><volume>63</volume><fpage>1265</fpage><lpage>1273</lpage><pub-id pub-id-type="doi">10.1002/art.30273</pub-id><pub-id pub-id-type="pmid">21305530</pub-id></element-citation></ref><ref id="CR71"><label>71.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Dennis</surname><given-names>G</given-names><suffix>Jr</suffix></name><name><surname>Holweg</surname><given-names>CT</given-names></name><name><surname>Kummerfeld</surname><given-names>SK</given-names></name><name><surname>Choy</surname><given-names>DF</given-names></name><name><surname>Setiadi</surname><given-names>AF</given-names></name><name><surname>Hackney</surname><given-names>JA</given-names></name><etal></etal></person-group><article-title>Synovial phenotypes in rheumatoid arthritis correlate with response to biologic therapeutics</article-title><source>Arthritis Res Ther</source><year>2014</year><volume>16</volume><fpage>1</fpage><lpage>18</lpage><pub-id pub-id-type="doi">10.1186/ar4555</pub-id></element-citation></ref><ref id="CR72"><label>72.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Sherlock</surname><given-names>JP</given-names></name><name><surname>Taylor</surname><given-names>PC</given-names></name><name><surname>Buckley</surname><given-names>CD</given-names></name></person-group><article-title>The biology of IL-23 and IL-17 and their therapeutic targeting in rheumatic diseases</article-title><source>Curr Opin Rheumatol</source><year>2015</year><volume>27</volume><fpage>71</fpage><lpage>75</lpage><pub-id pub-id-type="doi">10.1097/BOR.0000000000000132</pub-id><pub-id pub-id-type="pmid">25415528</pub-id></element-citation></ref><ref id="CR73"><label>73.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Toussirot</surname><given-names>E</given-names></name></person-group><article-title>The IL23/Th17 pathway as a therapeutic target in chronic inflammatory diseases</article-title><source>Inflamm Allergy Drug Targets</source><year>2012</year><volume>11</volume><fpage>159</fpage><lpage>168</lpage><pub-id pub-id-type="doi">10.2174/187152812800392805</pub-id><pub-id pub-id-type="pmid">22280236</pub-id></element-citation></ref><ref id="CR74"><label>74.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Hu</surname><given-names>J</given-names></name><name><surname>Suo</surname><given-names>L</given-names></name><name><surname>Yang</surname><given-names>Z</given-names></name><name><surname>Xu</surname><given-names>T</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name></person-group><article-title>IL-17 and IL23 expression as a predictor of response to infliximab treatment in Crohn’s disease</article-title><source>Zhonghua nei ke za zhi</source><year>2015</year><volume>54</volume><fpage>940</fpage><lpage>944</lpage><pub-id pub-id-type="pmid">26922821</pub-id></element-citation></ref><ref id="CR75"><label>75.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Yu</surname><given-names>C</given-names></name><name><surname>Gong</surname><given-names>X</given-names></name><name><surname>Yang</surname><given-names>Q</given-names></name><name><surname>Lian</surname><given-names>J</given-names></name><name><surname>Xu</surname><given-names>K</given-names></name><name><surname>Ruan</surname><given-names>B</given-names></name><etal></etal></person-group><article-title>The serum IL-23 level predicts the response to pegylated interferon therapy in patients with chronic hepatitis B</article-title><source>Liver Int</source><year>2015</year><volume>35</volume><fpage>1549</fpage><lpage>1556</lpage><pub-id pub-id-type="doi">10.1111/liv.12701</pub-id><pub-id pub-id-type="pmid">25312687</pub-id></element-citation></ref><ref id="CR76"><label>76.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kremer</surname><given-names>JM</given-names></name><name><surname>Galivan</surname><given-names>J</given-names></name><name><surname>Streckfuss</surname><given-names>A</given-names></name><name><surname>Kamen</surname><given-names>B</given-names></name></person-group><article-title>Methotrexate metabolism analysis in blood and liver of rheumatoid arthritis patients: association with hepatic folate deficiency and formation of polyglutamates</article-title><source>Arthritis Rheum</source><year>1986</year><volume>29</volume><fpage>832</fpage><lpage>835</lpage><pub-id pub-id-type="doi">10.1002/art.1780290703</pub-id><pub-id pub-id-type="pmid">2427090</pub-id></element-citation></ref><ref id="CR77"><label>77.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Peters</surname><given-names>JE</given-names></name><name><surname>Lyons</surname><given-names>PA</given-names></name><name><surname>Lee</surname><given-names>JC</given-names></name><name><surname>Richard</surname><given-names>AC</given-names></name><name><surname>Fortune</surname><given-names>MD</given-names></name><name><surname>Newcombe</surname><given-names>PJ</given-names></name><etal></etal></person-group><article-title>Insight into genotype–phenotype associations through eQTL mapping in multiple cell types in health and immune-mediated disease</article-title><source>PLoS Genet</source><year>2016</year><volume>12</volume><fpage>e1005908</fpage><pub-id pub-id-type="doi">10.1371/journal.pgen.1005908</pub-id><pub-id pub-id-type="pmid">27015630</pub-id></element-citation></ref><ref id="CR78"><label>78.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Fairfax</surname><given-names>BP</given-names></name><name><surname>Makino</surname><given-names>S</given-names></name><name><surname>Radhakrishnan</surname><given-names>J</given-names></name><name><surname>Plant</surname><given-names>K</given-names></name><name><surname>Leslie</surname><given-names>S</given-names></name><name><surname>Dilthey</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles</article-title><source>Nat Genet</source><year>2012</year><volume>44</volume><fpage>502</fpage><lpage>510</lpage><pub-id pub-id-type="doi">10.1038/ng.2205</pub-id><pub-id pub-id-type="pmid">22446964</pub-id></element-citation></ref><ref id="CR79"><label>79.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Battle</surname><given-names>A</given-names></name><name><surname>Khan</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>SH</given-names></name><name><surname>Mitrano</surname><given-names>A</given-names></name><name><surname>Ford</surname><given-names>MJ</given-names></name><name><surname>Pritchard</surname><given-names>JK</given-names></name><etal></etal></person-group><article-title>Genomic variation. Impact of regulatory variation from RNA to protein</article-title><source>Science</source><year>2015</year><volume>347</volume><fpage>664</fpage><lpage>667</lpage><pub-id pub-id-type="doi">10.1126/science.1260793</pub-id><pub-id pub-id-type="pmid">25657249</pub-id></element-citation></ref><ref id="CR80"><label>80.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Wu</surname><given-names>L</given-names></name><name><surname>Candille</surname><given-names>SI</given-names></name><name><surname>Choi</surname><given-names>Y</given-names></name><name><surname>Xie</surname><given-names>D</given-names></name><name><surname>Jiang</surname><given-names>L</given-names></name><name><surname>Li-Pook-Than</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Variation and genetic control of protein abundance in humans</article-title><source>Nature</source><year>2013</year><volume>499</volume><fpage>79</fpage><lpage>82</lpage><pub-id pub-id-type="doi">10.1038/nature12223</pub-id><pub-id pub-id-type="pmid">23676674</pub-id></element-citation></ref><ref id="CR81"><label>81.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hause</surname><given-names>RJ</given-names></name><name><surname>Stark</surname><given-names>AL</given-names></name><name><surname>Antao</surname><given-names>NN</given-names></name><name><surname>Gorsic</surname><given-names>LK</given-names></name><name><surname>Chung</surname><given-names>SH</given-names></name><name><surname>Brown</surname><given-names>CD</given-names></name><etal></etal></person-group><article-title>Identification and validation of genetic variants that influence transcription factor and cell signaling protein levels</article-title><source>Am J Hum Genet</source><year>2014</year><volume>95</volume><fpage>194</fpage><lpage>208</lpage><pub-id pub-id-type="doi">10.1016/j.ajhg.2014.07.005</pub-id><pub-id pub-id-type="pmid">25087611</pub-id></element-citation></ref><ref id="CR82"><label>82.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Garge</surname><given-names>N</given-names></name><name><surname>Pan</surname><given-names>H</given-names></name><name><surname>Rowland</surname><given-names>MD</given-names></name><name><surname>Cargile</surname><given-names>BJ</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Cooley</surname><given-names>PC</given-names></name><etal></etal></person-group><article-title>Identification of quantitative trait loci underlying proteome variation in human lymphoblastoid cells</article-title><source>Mol Cell Proteom</source><year>2010</year><volume>9</volume><fpage>1383</fpage><lpage>1399</lpage><pub-id pub-id-type="doi">10.1074/mcp.M900378-MCP200</pub-id></element-citation></ref><ref id="CR83"><label>83.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Fragoulis</surname><given-names>GE</given-names></name><name><surname>Siebert</surname><given-names>S</given-names></name><name><surname>McInnes</surname><given-names>IB</given-names></name></person-group><article-title>Therapeutic targeting of IL-17 and IL-23 cytokines in immune-mediated diseases</article-title><source>Annu Rev Med</source><year>2016</year><volume>67</volume><fpage>337</fpage><lpage>353</lpage><pub-id pub-id-type="doi">10.1146/annurev-med-051914-021944</pub-id><pub-id pub-id-type="pmid">26565676</pub-id></element-citation></ref><ref id="CR84"><label>84.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rao</surname><given-names>DA</given-names></name><name><surname>Gurish</surname><given-names>MF</given-names></name><name><surname>Marshall</surname><given-names>JL</given-names></name><name><surname>Slowikowski</surname><given-names>K</given-names></name><name><surname>Fonseka</surname><given-names>CY</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><etal></etal></person-group><article-title>Pathologically expanded peripheral T helper cell subset drives B cells in rheumatoid arthritis</article-title><source>Nature</source><year>2017</year><volume>542</volume><fpage>110</fpage><lpage>114</lpage><pub-id pub-id-type="doi">10.1038/nature20810</pub-id><pub-id pub-id-type="pmid">28150777</pub-id></element-citation></ref><ref id="CR85"><label>85.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>de Jong</surname><given-names>T</given-names></name><name><surname>Lubbers</surname><given-names>J</given-names></name><name><surname>Turk</surname><given-names>S</given-names></name><name><surname>Vosslamber</surname><given-names>S</given-names></name><name><surname>Mantel</surname><given-names>E</given-names></name><name><surname>Bontkes</surname><given-names>H</given-names></name><etal></etal></person-group><article-title>The type I interferon signature in leukocyte subsets from peripheral blood of patients with early arthritis: a major contribution by granulocytes</article-title><source>Arthritis Res Ther</source><year>2016</year><volume>18</volume><fpage>165</fpage><pub-id pub-id-type="doi">10.1186/s13075-016-1065-3</pub-id><pub-id pub-id-type="pmid">27411379</pub-id></element-citation></ref><ref id="CR86"><label>86.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Firestein</surname><given-names>GS</given-names></name><name><surname>McInnes</surname><given-names>IB</given-names></name></person-group><article-title>Immunopathogenesis of rheumatoid arthritis</article-title><source>Immunity</source><year>2017</year><volume>46</volume><fpage>183</fpage><lpage>196</lpage><pub-id pub-id-type="doi">10.1016/j.immuni.2017.02.006</pub-id><pub-id pub-id-type="pmid">28228278</pub-id></element-citation></ref><ref id="CR87"><label>87.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ray</surname><given-names>P</given-names></name><name><surname>Le Manach</surname><given-names>Y</given-names></name><name><surname>Riou</surname><given-names>B</given-names></name><name><surname>Houle</surname><given-names>TT</given-names></name></person-group><article-title>Statistical evaluation of a biomarker</article-title><source>Anesthesiology</source><year>2010</year><volume>112</volume><fpage>1023</fpage><lpage>1040</lpage><pub-id pub-id-type="doi">10.1097/ALN.0b013e3181d47604</pub-id><pub-id pub-id-type="pmid">20234303</pub-id></element-citation></ref><ref id="CR88"><label>88.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chatterjee</surname><given-names>N</given-names></name><name><surname>Shi</surname><given-names>J</given-names></name><name><surname>García-Closas</surname><given-names>M</given-names></name></person-group><article-title>Developing and evaluating polygenic risk prediction models for stratified disease prevention</article-title><source>Nat Rev Genet</source><year>2016</year><volume>14210</volume><fpage>14205</fpage><lpage>14210</lpage></element-citation></ref><ref id="CR89"><label>89.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Stringer</surname><given-names>S</given-names></name><name><surname>Wray</surname><given-names>NR</given-names></name><name><surname>Kahn</surname><given-names>RS</given-names></name><name><surname>Derks</surname><given-names>EM</given-names></name></person-group><article-title>Underestimated effect sizes in GWAS: fundamental limitations of single SNP analysis for dichotomous phenotypes</article-title><source>PLoS ONE</source><year>2011</year><volume>6</volume><fpage>e27964</fpage><pub-id pub-id-type="doi">10.1371/journal.pone.0027964</pub-id><pub-id pub-id-type="pmid">22140493</pub-id></element-citation></ref><ref id="CR90"><label>90.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Udalova</surname><given-names>IA</given-names></name><name><surname>Mantovani</surname><given-names>A</given-names></name><name><surname>Feldmann</surname><given-names>M</given-names></name></person-group><article-title>Macrophage heterogeneity in the context of rheumatoid arthritis</article-title><source>Nat Rev Rheumatol</source><year>2016</year><volume>12</volume><fpage>472</fpage><lpage>485</lpage><pub-id pub-id-type="doi">10.1038/nrrheum.2016.91</pub-id><pub-id pub-id-type="pmid">27383913</pub-id></element-citation></ref></ref-list></back></article>