<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Sci Rep</journal-id><journal-id journal-id-type="iso-abbrev">Sci Rep</journal-id><journal-title-group><journal-title>Scientific Reports</journal-title></journal-title-group><issn pub-type="epub">2045-2322</issn><publisher><publisher-name>Nature Publishing Group UK</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">29795295</article-id><article-id pub-id-type="pmc">5966460</article-id><article-id pub-id-type="publisher-id">26313</article-id><article-id pub-id-type="doi">10.1038/s41598-018-26313-8</article-id><article-categories><subj-group subj-group-type="heading"><subject>Article</subject></subj-group></article-categories><title-group><article-title>Impact of the griffithsin anti-HIV microbicide and placebo gels on the rectal mucosal proteome and microbiome in non-human primates</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Girard</surname><given-names>Lauren</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Birse</surname><given-names>Kenzie</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Holm</surname><given-names>Johanna B.</given-names></name><xref ref-type="aff" rid="Aff3">3</xref><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Gajer</surname><given-names>Pawel</given-names></name><xref ref-type="aff" rid="Aff3">3</xref><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Humphrys</surname><given-names>Mike S.</given-names></name><xref ref-type="aff" rid="Aff3">3</xref><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Garber</surname><given-names>David</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Guenthner</surname><given-names>Patricia</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Noël-Romas</surname><given-names>Laura</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Abou</surname><given-names>Max</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>McCorrister</surname><given-names>Stuart</given-names></name><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Westmacott</surname><given-names>Garrett</given-names></name><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Wang</surname><given-names>Lin</given-names></name><xref ref-type="aff" rid="Aff7">7</xref><xref ref-type="aff" rid="Aff8">8</xref></contrib><contrib contrib-type="author"><name><surname>Rohan</surname><given-names>Lisa C.</given-names></name><xref ref-type="aff" rid="Aff7">7</xref><xref ref-type="aff" rid="Aff8">8</xref></contrib><contrib contrib-type="author"><name><surname>Matoba</surname><given-names>Nobuyuki</given-names></name><xref ref-type="aff" rid="Aff9">9</xref><xref ref-type="aff" rid="Aff10">10</xref><xref ref-type="aff" rid="Aff11">11</xref></contrib><contrib contrib-type="author"><name><surname>McNicholl</surname><given-names>Janet</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Palmer</surname><given-names>Kenneth E.</given-names></name><xref ref-type="aff" rid="Aff9">9</xref><xref ref-type="aff" rid="Aff10">10</xref><xref ref-type="aff" rid="Aff11">11</xref></contrib><contrib contrib-type="author"><contrib-id contrib-id-type="orcid">http://orcid.org/0000-0002-0851-2233</contrib-id><name><surname>Ravel</surname><given-names>Jacques</given-names></name><xref ref-type="aff" rid="Aff3">3</xref><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Burgener</surname><given-names>Adam D.</given-names></name><address><email>Adam.Burgener@umanitoba.ca</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff12">12</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 0805 4386</institution-id><institution-id institution-id-type="GRID">grid.415368.d</institution-id><institution>National HIV and Retrovirology Labs, </institution><institution>JC Wilt Infectious Diseases Research Centre, Public Health Agency of Canada, </institution></institution-wrap>Winnipeg, Canada </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 1936 9609</institution-id><institution-id institution-id-type="GRID">grid.21613.37</institution-id><institution>Department of Medical Microbiology, </institution><institution>University of Manitoba, </institution></institution-wrap>Winnipeg, Canada </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2175 4264</institution-id><institution-id institution-id-type="GRID">grid.411024.2</institution-id><institution>Institute for Genome Sciences, </institution><institution>University of Maryland School of Medicine, </institution></institution-wrap>Baltimore, USA </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2175 4264</institution-id><institution-id institution-id-type="GRID">grid.411024.2</institution-id><institution>Department of Microbiology and Immunology, </institution><institution>University of Maryland School of Medicine, </institution></institution-wrap>Baltimore, USA </aff><aff id="Aff5"><label>5</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2163 0069</institution-id><institution-id institution-id-type="GRID">grid.416738.f</institution-id><institution>Laboratory Branch, Division of HIV/AIDS Prevention, </institution><institution>National Centre for HIV/AIDS, Viral Hepatitis, Sexually Transmitted Disease and Tuberculosis Prevention, CDC, </institution></institution-wrap>Atlanta, USA </aff><aff id="Aff6"><label>6</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 0805 4386</institution-id><institution-id institution-id-type="GRID">grid.415368.d</institution-id><institution>Mass Spectrometry and Proteomics Core Facility, </institution><institution>National Microbiology Laboratory, Public Health Agency of Canada, </institution></institution-wrap>Winnipeg, Canada </aff><aff id="Aff7"><label>7</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 0387 4432</institution-id><institution-id institution-id-type="GRID">grid.460217.6</institution-id><institution>Magee Women’s Research Institute, </institution></institution-wrap>Pittsburgh, USA </aff><aff id="Aff8"><label>8</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 1936 9000</institution-id><institution-id institution-id-type="GRID">grid.21925.3d</institution-id><institution>School of Pharmacy, </institution><institution>University of Pittsburgh, </institution></institution-wrap>Pittsburgh, USA </aff><aff id="Aff9"><label>9</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2113 1622</institution-id><institution-id institution-id-type="GRID">grid.266623.5</institution-id><institution>Center for Predictive Medicine, </institution><institution>University of Louisville, </institution></institution-wrap>Louisville, USA </aff><aff id="Aff10"><label>10</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2113 1622</institution-id><institution-id institution-id-type="GRID">grid.266623.5</institution-id><institution>Department of Pharmacology and Toxicology, </institution><institution>University of Louisville, </institution></institution-wrap>Louisville, USA </aff><aff id="Aff11"><label>11</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2113 1622</institution-id><institution-id institution-id-type="GRID">grid.266623.5</institution-id><institution>James Graham Brown Cancer Centre, </institution><institution>University of Louisville, </institution></institution-wrap>Louisville, USA </aff><aff id="Aff12"><label>12</label>Unit of Infectious Diseases, Department of Medicine Solna, Centre for Molecular Medicine, Karolinska Institute, Karolinska University Hospital, Stockholm, Sweden </aff></contrib-group><pub-date pub-type="epub"><day>23</day><month>5</month><year>2018</year></pub-date><pub-date pub-type="pmc-release"><day>23</day><month>5</month><year>2018</year></pub-date><pub-date pub-type="collection"><year>2018</year></pub-date><volume>8</volume><elocation-id>8059</elocation-id><history><date date-type="received"><day>18</day><month>1</month><year>2018</year></date><date date-type="accepted"><day>10</day><month>5</month><year>2018</year></date></history><permissions><copyright-statement>© The Author(s) 2018</copyright-statement><license license-type="OpenAccess"><license-p><bold>Open Access</bold> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</ext-link>.</license-p></license></permissions><abstract id="Abs1"><p id="Par1"><offsets xml_i="10563" xml_f="10688" txt_i="11" txt_f="136">Topical microbicides are being explored as an HIV prevention method for individuals who practice receptive anal intercourse. </offsets><italic><offsets xml_i="10696" xml_f="10703" txt_i="136" txt_f="143">In vivo</offsets></italic><offsets xml_i="10712" xml_f="11752" txt_i="143" txt_f="1174"> studies of these microbicides are critical to confirm safety. Here, we evaluated the impact of a rectal microbicide containing the antiviral lectin, Griffithsin (GRFT), on the rectal mucosal proteome and microbiome. Using a randomized, crossover placebo-controlled design, six rhesus macaques received applications of hydroxyethylcellulose (HEC)- or carbopol-formulated 0.1% GRFT gels. Rectal mucosal samples were then evaluated by label-free tandem MS/MS and 16 S rRNA gene amplicon sequencing, for proteomics and microbiome analyses, respectively. Compared to placebo, GRFT gels were not associated with any significant changes to protein levels at any time point (FDR &lt; 5%), but increased abundances of two common and beneficial microbial taxa after 24 hours were observed in HEC-GRFT gel (p &lt; 2E-09). Compared to baseline, both placebo formulations were associated with alterations to proteins involved in proteolysis, activation of the immune response and inflammation after 2 hours (p &lt; 0.0001), and increases in beneficial </offsets><italic><offsets xml_i="11760" xml_f="11776" txt_i="1174" txt_f="1190">Faecalibacterium</offsets></italic><offsets xml_i="11785" xml_f="12036" txt_i="1190" txt_f="1441"> spp. after 24 hours in HEC placebo gel (p = 4.21E-15). This study supports the safety profile of 0.1% GRFT gel as an anti-HIV microbicide and demonstrates that current placebo formulations may associate with changes to rectal proteome and microbiota.</offsets></p></abstract><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2018</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1" sec-type="introduction"><title><offsets xml_i="12285" xml_f="12297" txt_i="1449" txt_f="1461">Introduction</offsets></title><p id="Par2"><offsets xml_i="12318" xml_f="12574" txt_i="1462" txt_f="1718">Receptive anal intercourse (RAI) is a large contributing factor to new HIV infections globally. Specifically, HIV transmission risk during condom-less receptive anal intercourse (CRAI) is estimated to be 138 per 10,000 exposures (95% CI 102-186 per 10,000)</offsets><sup><xref ref-type="bibr" rid="CR1"><offsets xml_i="12611" xml_f="12612" txt_i="1718" txt_f="1719">1</offsets></xref></sup><offsets xml_i="12625" xml_f="12762" txt_i="1719" txt_f="1856">. RAI is commonly practiced by men who have sex with men (MSM), who continue to be disproportionately affected by the global HIV epidemic</offsets><sup><xref ref-type="bibr" rid="CR2"><offsets xml_i="12799" xml_f="12800" txt_i="1856" txt_f="1857">2</offsets></xref><offsets xml_i="12807" xml_f="12808" txt_i="1857" txt_f="1858">,</offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="12840" xml_f="12841" txt_i="1858" txt_f="1859">3</offsets></xref></sup><offsets xml_i="12854" xml_f="13081" txt_i="1859" txt_f="2086">, accounting for 49% of new infections in Western and Central Europe and North America, 30% of new infections in Latin America, and 18% of new infections in Asia and the Pacific according to the most recent UNAIDS Global Report</offsets><sup><xref ref-type="bibr" rid="CR4"><offsets xml_i="13118" xml_f="13119" txt_i="2086" txt_f="2087">4</offsets></xref></sup><offsets xml_i="13132" xml_f="13331" txt_i="2087" txt_f="2286">. CRAI may also be contributing to HIV infection rates observed in other key populations, such as transgender women, in which HIV prevalence is 19.1% worldwide, as well as women in sub-Saharan Africa</offsets><sup><xref ref-type="bibr" rid="CR5"><offsets xml_i="13368" xml_f="13369" txt_i="2286" txt_f="2287">5</offsets></xref><offsets xml_i="13376" xml_f="13377" txt_i="2287" txt_f="2288">,</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="13409" xml_f="13410" txt_i="2288" txt_f="2289">6</offsets></xref></sup><offsets xml_i="13423" xml_f="13528" txt_i="2289" txt_f="2394">. Despite this high risk of acquisition, the level of condom use among those practicing RAI is inadequate</offsets><sup><xref ref-type="bibr" rid="CR7"><offsets xml_i="13565" xml_f="13566" txt_i="2394" txt_f="2395">7</offsets></xref><offsets xml_i="13573" xml_f="13574" txt_i="2395" txt_f="2396">–</offsets><xref ref-type="bibr" rid="CR16"><offsets xml_i="13607" xml_f="13609" txt_i="2396" txt_f="2398">16</offsets></xref></sup><offsets xml_i="13622" xml_f="13767" txt_i="2398" txt_f="2543">; demonstrating the importance of providing novel HIV prevention tools for these populations is critical for stemming the rate of new infections.</offsets></p><p id="Par3"><offsets xml_i="13784" xml_f="14089" txt_i="2544" txt_f="2849">Microbicides are topical antiviral compounds used to prevent HIV transmission at mucosal surfaces. Since people who practice RAI often use sexual lubricants, a microbicide incorporated into a gel-based lubricant may represent a practical and acceptable prevention method against rectal transmission of HIV</offsets><sup><xref ref-type="bibr" rid="CR17"><offsets xml_i="14127" xml_f="14129" txt_i="2849" txt_f="2851">17</offsets></xref></sup><offsets xml_i="14142" xml_f="14787" txt_i="2851" txt_f="3496">. An essential step in the development of new microbicide candidates is the determination of their effects on biological systems. Microbicides that elicit undesirable side effects may impact tolerance and adherence to the product. Notably, certain microbicides may elicit biological responses that lead to increased risk of HIV acquisition. The importance of pre-clinical toxicity screening has been demonstrated by the early microbicide candidate, Nonoxynol-9 (N-9). Although N-9 was presumed safe and used as a spermicide, it was shown to cause increased genital lesions, increased inflammation, toxicity and increased risk for HIV acquisition</offsets><sup><xref ref-type="bibr" rid="CR18"><offsets xml_i="14825" xml_f="14827" txt_i="3496" txt_f="3498">18</offsets></xref><offsets xml_i="14834" xml_f="14835" txt_i="3498" txt_f="3499">,</offsets><xref ref-type="bibr" rid="CR19"><offsets xml_i="14868" xml_f="14870" txt_i="3499" txt_f="3501">19</offsets></xref></sup><offsets xml_i="14883" xml_f="14991" txt_i="3501" txt_f="3609">. Recent studies have highlighted the significance of mucosal inflammation on increased HIV acquisition risk</offsets><sup><xref ref-type="bibr" rid="CR20"><offsets xml_i="15029" xml_f="15031" txt_i="3609" txt_f="3611">20</offsets></xref></sup><offsets xml_i="15044" xml_f="15138" txt_i="3611" txt_f="3705">. In addition, N-9 delivered intra-vaginally shifted the microbiota toward a state of lowered </offsets><italic><offsets xml_i="15146" xml_f="15159" txt_i="3705" txt_f="3718">Lactobacillus</offsets></italic><offsets xml_i="15168" xml_f="15297" txt_i="3718" txt_f="3847"> spp. levels and increases in strict and facultative anaerobes, which is a known risk factor for HIV acquisition and transmission</offsets><sup><xref ref-type="bibr" rid="CR21"><offsets xml_i="15335" xml_f="15337" txt_i="3847" txt_f="3849">21</offsets></xref><offsets xml_i="15344" xml_f="15345" txt_i="3849" txt_f="3850">–</offsets><xref ref-type="bibr" rid="CR23"><offsets xml_i="15378" xml_f="15380" txt_i="3850" txt_f="3852">23</offsets></xref></sup><offsets xml_i="15393" xml_f="15520" txt_i="3852" txt_f="3979">. However, there is limited information of rectal inflammation or microbiome parameters that lead to increased HIV acquisition.</offsets></p><p id="Par4"><offsets xml_i="15537" xml_f="15663" txt_i="3980" txt_f="4106">Currently, the lead rectal microbicide candidate is Tenofovir (TFV) 1% gel, with a phase II trial (MTN-017) currently underway</offsets><sup><xref ref-type="bibr" rid="CR24"><offsets xml_i="15701" xml_f="15703" txt_i="4106" txt_f="4108">24</offsets></xref></sup><offsets xml_i="15716" xml_f="16109" txt_i="4108" txt_f="4501">. Despite this progress, there are safety concerns related to the topical use of TFV. A study on the long-term use of TFV 1% gel demonstrated a broad range of effects on rectal mucosa, including suppressed anti-inflammatory mediators, increased T cell densities, mitochondrial dysfunction, altered regulatory pathways of cell differentiation and survival, and stimulation of cell proliferation</offsets><sup><xref ref-type="bibr" rid="CR25"><offsets xml_i="16147" xml_f="16149" txt_i="4501" txt_f="4503">25</offsets></xref></sup><offsets xml_i="16162" xml_f="16205" txt_i="4503" txt_f="4546">. Additionally, in a recent study by Romas </offsets><italic><offsets xml_i="16213" xml_f="16218" txt_i="4546" txt_f="4551">et al</offsets></italic><offsets xml_i="16227" xml_f="16308" txt_i="4551" txt_f="4632">., rectal TFV 1% gel use was associated with altered epidermal protein expression</offsets><sup><xref ref-type="bibr" rid="CR26"><offsets xml_i="16346" xml_f="16348" txt_i="4632" txt_f="4634">26</offsets></xref></sup><offsets xml_i="16361" xml_f="16594" txt_i="4634" txt_f="4867">. While the significance of these alterations with HIV acquisition risk and gel efficacy are currently unknown, efforts to develop safe rectal microbicide that do not impact the natural environment of the rectal mucosa are desirable.</offsets></p><p id="Par5"><offsets xml_i="16611" xml_f="16722" txt_i="4868" txt_f="4979">Another important consideration in the development of new microbicide products is the choice of gel formulation</offsets><sup><xref ref-type="bibr" rid="CR27"><offsets xml_i="16760" xml_f="16762" txt_i="4979" txt_f="4981">27</offsets></xref></sup><offsets xml_i="16775" xml_f="17405" txt_i="4981" txt_f="5611">. Gel formulation determines the characteristics of the gel product, which include gel appearance, pH, osmolality and viscosity. In turn, these features can influence the safety of and the tolerability to the product. The importance of gel formulation was demonstrated by the use of glycerin as an excipient in the RMP-02/MTN-006 and the MTN-007 clinical trials. Notably, glycerin is commonly used in gel products in order to increase solubility of active ingredients. Conversely, it has been suggested that the hyper-osmolar properties of glycerin may induce changes in the rectal compartment, leading to enhanced HIV-1 infection</offsets><sup><xref ref-type="bibr" rid="CR28"><offsets xml_i="17443" xml_f="17445" txt_i="5611" txt_f="5613">28</offsets></xref><offsets xml_i="17452" xml_f="17453" txt_i="5613" txt_f="5614">,</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="17486" xml_f="17488" txt_i="5614" txt_f="5616">29</offsets></xref></sup><offsets xml_i="17501" xml_f="17669" txt_i="5616" txt_f="5784">. In contrast to RMP-02/MTN-006, MTN-007 used a gel product containing a reduced amount of glycerin. This change in formulation was associated with fewer adverse events</offsets><sup><xref ref-type="bibr" rid="CR30"><offsets xml_i="17707" xml_f="17709" txt_i="5784" txt_f="5786">30</offsets></xref><offsets xml_i="17716" xml_f="17717" txt_i="5786" txt_f="5787">,</offsets><xref ref-type="bibr" rid="CR31"><offsets xml_i="17750" xml_f="17752" txt_i="5787" txt_f="5789">31</offsets></xref></sup><offsets xml_i="17765" xml_f="17885" txt_i="5789" txt_f="5909">, demonstrating the importance of exploring multiple gel formulations in the development of rectal microbicide products.</offsets></p><p id="Par6"><offsets xml_i="17902" xml_f="18347" txt_i="5910" txt_f="6355">PREVENT (PREvention of Viral ENTry) is a placebo-controlled crossover preclinical trial with the goal of evaluating the tolerability and safety of GRFT, a potent antiviral lectin, as a rectal microbicide to prevent HIV infection. Past studies have shown GRFT’s safety in various systems, including no cytotoxicity in several different cell lines, including cervical (End1/E6E7, Ect1/E6E7, CaSki), fibroblast (3T3) and dendritic (moDC) cell lines</offsets><sup><xref ref-type="bibr" rid="CR32"><offsets xml_i="18385" xml_f="18387" txt_i="6355" txt_f="6357">32</offsets></xref><offsets xml_i="18394" xml_f="18395" txt_i="6357" txt_f="6358">,</offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="18428" xml_f="18430" txt_i="6358" txt_f="6360">33</offsets></xref></sup><offsets xml_i="18443" xml_f="18716" txt_i="6360" txt_f="6633">. These studies also showed that GRFT was associated with little or no induction of inflammatory cytokines in both human cervical explant tissues and Peripheral Blood Mononuclear Cells (PBMCs), no induction of T-cell activation, no off-target alterations in gene expression</offsets><sup><xref ref-type="bibr" rid="CR32"><offsets xml_i="18754" xml_f="18756" txt_i="6633" txt_f="6635">32</offsets></xref><offsets xml_i="18763" xml_f="18764" txt_i="6635" txt_f="6636">–</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="18797" xml_f="18799" txt_i="6636" txt_f="6638">35</offsets></xref></sup><offsets xml_i="18812" xml_f="18891" txt_i="6638" txt_f="6717"> and demonstrated safety in rodent models and rabbit vaginal irritation studies</offsets><sup><xref ref-type="bibr" rid="CR35"><offsets xml_i="18929" xml_f="18931" txt_i="6717" txt_f="6719">35</offsets></xref><offsets xml_i="18938" xml_f="18939" txt_i="6719" txt_f="6720">,</offsets><xref ref-type="bibr" rid="CR36"><offsets xml_i="18972" xml_f="18974" txt_i="6720" txt_f="6722">36</offsets></xref></sup><offsets xml_i="18987" xml_f="18988" txt_i="6722" txt_f="6723">.</offsets></p><p id="Par7"><offsets xml_i="19005" xml_f="19329" txt_i="6724" txt_f="7048">The recent emergence of systems biology tools, such as proteomics and transcriptomics, has made available new tools to study mucosal systems. These techniques have provided insight into mechanisms of mucosal system perturbations during a wide variety of mucosal inflammatory conditions, including vaginal microbial dysbiosis</offsets><sup><xref ref-type="bibr" rid="CR37"><offsets xml_i="19367" xml_f="19369" txt_i="7048" txt_f="7050">37</offsets></xref></sup><offsets xml_i="19382" xml_f="19410" txt_i="7050" txt_f="7078">, elevated mucosal cytokines</offsets><sup><xref ref-type="bibr" rid="CR38"><offsets xml_i="19448" xml_f="19450" txt_i="7078" txt_f="7080">38</offsets></xref></sup><offsets xml_i="19463" xml_f="19499" txt_i="7080" txt_f="7116">, effects of hormonal contraceptives</offsets><sup><xref ref-type="bibr" rid="CR39"><offsets xml_i="19537" xml_f="19539" txt_i="7116" txt_f="7118">39</offsets></xref></sup><offsets xml_i="19552" xml_f="19604" txt_i="7118" txt_f="7170">, and the effects of sexual behaviours, such as CRAI</offsets><sup><xref ref-type="bibr" rid="CR40"><offsets xml_i="19642" xml_f="19644" txt_i="7170" txt_f="7172">40</offsets></xref></sup><offsets xml_i="19657" xml_f="19899" txt_i="7172" txt_f="7414">. Recently, systems biology techniques demonstrated the off-target effects of a tenofovir-based microbicide gel on rectal compartments, proving the utility of this method to screen potential toxic effects of anti-retroviral based microbicides</offsets><sup><xref ref-type="bibr" rid="CR26"><offsets xml_i="19937" xml_f="19939" txt_i="7414" txt_f="7416">26</offsets></xref></sup><offsets xml_i="19952" xml_f="20237" txt_i="7416" txt_f="7701">. In this study, we evaluated the rectal mucosal proteome, as well as the composition and structure of the rectal microbiota associated with intra-rectal application of 0.1% GRFT gel, while comparing two different gel formulations, HEC and Carbopol, in a non-human primate (NHP) model.</offsets></p></sec><sec id="Sec2" sec-type="results"><title><offsets xml_i="20288" xml_f="20295" txt_i="7703" txt_f="7710">Results</offsets></title><sec id="Sec3"><title><offsets xml_i="20325" xml_f="20337" txt_i="7711" txt_f="7723">Study design</offsets></title><p id="Par8"><offsets xml_i="20358" xml_f="20538" txt_i="7724" txt_f="7904">Rhesus macaques were maintained at the Centers for Disease Control and Prevention (CDC; Atlanta, GA, USA) in accordance with the Guide for the Care and Use of Laboratory Animals (8</offsets><sup><offsets xml_i="20543" xml_f="20545" txt_i="7904" txt_f="7906">th</offsets></sup><offsets xml_i="20551" xml_f="21010" txt_i="7906" txt_f="8365"> edition) in an AALAC-accredited facility, according to institutional standard operating procedures. All study methods and procedures were approved by the CDC Institutional Animal Care and Use Committee (IACUC, protocol 2700GARMONC). Six rhesus macaques were split into two groups and treated intra-rectally with either HEC or carbopol formulations of both 0.1% GRFT and placebo gels, followed by a formulation crossover after a two-week washout period (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="21042" xml_f="21043" txt_i="8365" txt_f="8366">1</offsets></xref><offsets xml_i="21050" xml_f="21654" txt_i="8366" txt_f="8970">). Multiple (two or four samples for for microbiota and proteomic studies, respectively) longitudinal rectal swabs were collected over a period of 4 weeks from all six macaques prior to gel initiation to establish robust baseline protein abundance measurements. Each gel application was preceded by collection of pre-gel (T0) rectal swabs and followed by collection of post-gel rectal swabs (2 hours and 24 hours post-gel application for proteomic analysis; 24 hours and 7 days for microbiota analysis). These samples were then analysed by tandem mass spectrometry and 16 S rRNA gene amplicon sequencing.</offsets><fig id="Fig1"><label><offsets xml_i="21676" xml_f="21684" txt_i="8970" txt_f="8978">Figure 1</offsets></label><caption><p><offsets xml_i="21704" xml_f="22299" txt_i="8978" txt_f="9573">An outline of the PREVENT study in rhesus macaques. Baseline samples were collected from six rhesus macaques prior to dividing the macaques into two groups (Group 1 and Group 2). Macaques were then treated once intra-rectally with placebo gels, followed by a washout period of 14 days, and subsequently treated intra-rectally with GRFT 0.1% gels. The formulation applied was dependent on group assignment. After another 14-day washout period, there was a mid-trial crossover point in which the two groups switched gel formulations. The placebo and 0.1% GRFT gel application process was repeated.</offsets></p></caption><graphic xlink:href="41598_2018_26313_Fig1_HTML" id="d29e607"></graphic></fig></p></sec><sec id="Sec4"><title><offsets xml_i="22423" xml_f="22518" txt_i="9576" txt_f="9671">Mass spectrometry analysis and proteome coverage of rectal mucosal samples from rhesus macaques</offsets></title><p id="Par9"><offsets xml_i="22539" xml_f="22853" txt_i="9672" txt_f="9986">We first evaluated rectal sample proteome variability and reproducibility by mass spectrometry. Mass spectrometry analysis confidently identified 382 host proteins across all baseline, placebo-treated and GRFT-treated samples. Total protein abundance was visualized to identify sample outliers (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="22889" xml_f="22891" txt_i="9986" txt_f="9988">S1</offsets></xref><offsets xml_i="22898" xml_f="23226" txt_i="9988" txt_f="10313">). Nine technical replicates of a reference sample (containing equal amounts of peptide from each rectal mucosal sample) were used to evaluate the reproducibility of the mass spectrometer. This comparison showed strong correlation of individual proteins between sample runs, with Pearson r values &gt; 0.985 (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="23262" xml_f="23264" txt_i="10313" txt_f="10315">S2</offsets></xref><offsets xml_i="23271" xml_f="23361" txt_i="10315" txt_f="10405">). This demonstrated high fidelity of mass spectrometry to profile rectal mucosal samples.</offsets></p><p id="Par10"><offsets xml_i="23379" xml_f="23472" txt_i="10406" txt_f="10499">The abundance of proteins detected in rectal mucosa spanned over 4 orders of magnitude (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="23504" xml_f="23506" txt_i="10499" txt_f="10501">2a</offsets></xref><offsets xml_i="23513" xml_f="23754" txt_i="10501" txt_f="10742">). According to gene ontology and pathway analysis, the top biological processes associated with these proteins included in cell growth, cell/tissue organization, coagulation, immunity, protease activity and movement of macromolecules (Fig. </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="23786" xml_f="23788" txt_i="10742" txt_f="10744">2b</offsets></xref><offsets xml_i="23795" xml_f="24128" txt_i="10744" txt_f="11077">). Functional annotation ascribed rectal proteins to several cellular compartments, including the cytoplasm (p = 3.12E-32), extracellular vesicles (p = 1.54E-187), intracellular organelles (p = 2.82E-066), adherens junctions (p = 7.81E-40), the cytoskeleton (p = 4.16E-10) and blood microparticles (p = 5.17E-40) (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="24164" xml_f="24166" txt_i="11077" txt_f="11079">S3</offsets></xref><offsets xml_i="24173" xml_f="24313" txt_i="11079" txt_f="11219">). Overall this shows that a diverse set of biological functions and pathways are observable in rectal mucosal samples by mass spectrometry.</offsets><fig id="Fig2"><label><offsets xml_i="24335" xml_f="24343" txt_i="11219" txt_f="11227">Figure 2</offsets></label><caption><p><offsets xml_i="24363" xml_f="24436" txt_i="11227" txt_f="11300">Diversity of the rectal mucosal proteome observed by mass spectrometry. (</offsets><bold><offsets xml_i="24442" xml_f="24443" txt_i="11300" txt_f="11301">a</offsets></bold><offsets xml_i="24450" xml_f="24757" txt_i="11301" txt_f="11608">) Log Rank plot of protein abundances shows rectal proteome coverage spans greater than 4 orders of magnitude. Immune proteins belonging to specific pathways are indicated in dark blue (acute inflammatory response), dark green (wound healing), pink (complement) and grey (epithelial cell differentiation); (</offsets><bold><offsets xml_i="24763" xml_f="24764" txt_i="11608" txt_f="11609">b</offsets></bold><offsets xml_i="24771" xml_f="25174" txt_i="11609" txt_f="12012">) Radial chart of the top biological processes in the rectal mucosa of rhesus macaques in our study according to DAVID Functional Annotation v6.8. The innermost bar chart represents the number of proteins involved in each process. Processes are grouped into broader categories, which include cell growth, cell/tissue organization, coagulation, immunity, protease activity and movement of macromolecules.</offsets></p></caption><graphic xlink:href="41598_2018_26313_Fig2_HTML" id="d29e644"></graphic></fig></p></sec><sec id="Sec5"><title><offsets xml_i="25298" xml_f="25382" txt_i="12015" txt_f="12099">Transient host proteome changes are associated with HEC and Carbopol placebo gel use</offsets></title><p id="Par11"><offsets xml_i="25404" xml_f="25583" txt_i="12100" txt_f="12279">Four baseline samples were taken and protein abundances from these samples were averaged to establish a baseline proteome measurement for each animal (shown in Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="25619" xml_f="25621" txt_i="12279" txt_f="12281">S4</offsets></xref><offsets xml_i="25628" xml_f="25798" txt_i="12281" txt_f="12451">). Further visualization by Principle Component Analysis showed that baseline averages were less variable than samples collected at other time points (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="25834" xml_f="25836" txt_i="12451" txt_f="12453">S5</offsets></xref><offsets xml_i="25843" xml_f="25845" txt_i="12453" txt_f="12455">).</offsets></p><p id="Par12"><offsets xml_i="25863" xml_f="25981" txt_i="12456" txt_f="12574">We first evaluated changes in the rectal mucosal proteome associated within the placebo arms in rhesus macaques (Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="26013" xml_f="26014" txt_i="12574" txt_f="12575">3</offsets></xref><offsets xml_i="26021" xml_f="26523" txt_i="12575" txt_f="13071">). Proteomic data was normalized to total proteomic signal using linear adjustments. Adjusted values were then normalized using log transformation, and normality was confirmed using PP plots. In both placebo gel formulations, significant intra-macaque differences in protein expression were observed between the averaged baseline and the 2 hours post-gel timepoint, with carbopol inducing more changes (76/382 proteins (19.9%), p &lt; 0.017) than HEC (36/382 (9.4%), p &lt; 0.0073) (two-tailed, paired </offsets><italic><offsets xml_i="26531" xml_f="26532" txt_i="13071" txt_f="13072">t</offsets></italic><offsets xml_i="26541" xml_f="26679" txt_i="13072" txt_f="13207"> test, Local FDR &lt;5%). Carbopol placebo gel was associated with a larger magnitude of change from baseline, ranging from 4.9 to 4.6 log</offsets><sub><offsets xml_i="26684" xml_f="26685" txt_i="13207" txt_f="13208">2</offsets></sub><offsets xml_i="26691" xml_f="26773" txt_i="13208" txt_f="13290"> fold-change protein abundance, compared to HEC, which ranged from −4.8 to 3.2 log</offsets><sub><offsets xml_i="26778" xml_f="26779" txt_i="13290" txt_f="13291">2</offsets></sub><offsets xml_i="26785" xml_f="26926" txt_i="13291" txt_f="13432"> fold-change protein abundance. No significant changes were observed 24-hour post-placebo gel application, as none passed a 5% FDR threshold.</offsets><fig id="Fig3"><label><offsets xml_i="26948" xml_f="26956" txt_i="13432" txt_f="13440">Figure 3</offsets></label><caption><p><offsets xml_i="26976" xml_f="27269" txt_i="13440" txt_f="13733">Changes to the rectal mucosal proteome after application of either drug (0.1% GRFT) or placebo using either hydroxyethylcellulose (HEC) or carbopol formulations. Volcano plots display the log2-fold change values of protein expression along the x-axis and −log p-value (as determined by paired </offsets><italic><offsets xml_i="27277" xml_f="27278" txt_i="13733" txt_f="13734">t</offsets></italic><offsets xml_i="27287" xml_f="27786" txt_i="13734" txt_f="14230"> tests) along the y-axis. Dashed lines indicate the cut off for p = 0.05 and the 5% false discovery rate (FDR) where applicable. Blue and red points represent proteins that were significantly over- and under-abundantly expressed in placebo-treated samples, respectively (threshold &lt; 5% FDR). Significant effects were observed 2 hours post-placebo gel application for both carbopol and HEC gels, where 19.9% and 9.4% of proteins were differentially expressed, respectively (upper middle panels of </offsets><bold><offsets xml_i="27792" xml_f="27793" txt_i="14230" txt_f="14231">a</offsets></bold><offsets xml_i="27800" xml_f="27805" txt_i="14231" txt_f="14236"> and </offsets><bold><offsets xml_i="27811" xml_f="27812" txt_i="14236" txt_f="14237">b</offsets></bold><offsets xml_i="27819" xml_f="27949" txt_i="14237" txt_f="14367">). Other than relatively higher levels of Myeloperoxidase at 24 hours post-GRFT 0.1% gel application in the carbopol formulation (</offsets><bold><offsets xml_i="27955" xml_f="27956" txt_i="14367" txt_f="14368">a</offsets></bold><offsets xml_i="27963" xml_f="28092" txt_i="14368" txt_f="14497">, bottom right), 0.1% GRFT gels did not elicit any significant changes in rectal mucosal protein expression (threshold = FDR 5%).</offsets></p></caption><graphic xlink:href="41598_2018_26313_Fig3_HTML" id="d29e690"></graphic></fig></p><p id="Par13"><offsets xml_i="28202" xml_f="28400" txt_i="14499" txt_f="14694">Hierarchical clustering of differently expressed proteins (FDR &lt; 5%) showed a distinct and transient expression signature associated with samples taken 2 hours post-placebo gel application (Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="28432" xml_f="28434" txt_i="14694" txt_f="14696">4a</offsets></xref><offsets xml_i="28441" xml_f="28559" txt_i="14696" txt_f="14814">). The DAVID gene ontology tool was used to elucidate known biological functions associated with these proteins (Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="28591" xml_f="28593" txt_i="14814" txt_f="14816">4b</offsets></xref><offsets xml_i="28600" xml_f="29806" txt_i="14816" txt_f="16022">). Biological functions that were significantly altered (Benjamini-Hochberg corrected Fisher’s Exact Tests) in both HEC and carbopol gel arms included proteolysis (carbopol, p = 1.38E-3, n = 22; HEC, p = 7.12E-4, n = 15), secretion by the cell (carbopol, p = 2.52E-5, n = 14; HEC, p = 2.93E-5, n = 10), endocytosis (carbopol, p = 3.18E-3; n = 13; HEC, p = 3.64E-3, n = 9), activation of immune response (carbopol, p = 2.99E-3, n = 12; HEC, p = 7.37E-3,n = 8), protein activation cascade (carbopol, p = 7.93E-4, n = 8; HEC, p = 6.5E-6, n = 8), protein maturation/processing (carbopol, p = 6.02E-3, n = 8; HEC, p = 1.91E-3, n = 7), the acute inflammatory response (carbopol, p = 2.94E-3, n = 7; HEC, p = 6.59E-6, n = 8) and regulation of peptidase (carbopol, p = 1.48E-3, n = 11; HEC, p = 8.62E-3, n = 7) activity. Despite a large overlap between biological pathways in the two placebo gel formulations, a few pathways were unique. Pathways unique to carbopol included actin filament organization (p = 1.86E-5, n = 13) and regulation of actin filament length (p = 1.34E-3, n = 8). Regulation of protein metabolic process (p = 1.8E-3, n = 17) and blood coagulation (p = 1.92E-3, n = 4) were unique to HEC gel.</offsets><fig id="Fig4"><label><offsets xml_i="29828" xml_f="29836" txt_i="16022" txt_f="16030">Figure 4</offsets></label><caption><p><offsets xml_i="29856" xml_f="29994" txt_i="16030" txt_f="16168">Transient placebo gel-induced effects to the rectal mucosal proteome 2 hours post application in rhesus macaques. Heatmaps display the log</offsets><sub><offsets xml_i="29999" xml_f="30000" txt_i="16168" txt_f="16169">2</offsets></sub><offsets xml_i="30006" xml_f="30188" txt_i="16169" txt_f="16351">-fold change values of protein abundance, with darker colours representing stronger changes. Non-human primate (NHP) identifiers are indicated along the top border of the heatmaps. (</offsets><bold><offsets xml_i="30194" xml_f="30195" txt_i="16351" txt_f="16352">a</offsets></bold><offsets xml_i="30202" xml_f="30550" txt_i="16352" txt_f="16700">) Protein abundance values that were altered 2 hours post-gel application compared to baseline (5% FDR cut off). Baseline, 2 hours, and 24 hours post-gel application time points are indicated in purple, green and orange, respectively. Samples treated with carbopol placebo and HEC placebo are differentiated by turquoise and yellow, respectively. (</offsets><bold><offsets xml_i="30556" xml_f="30557" txt_i="16700" txt_f="16701">b</offsets></bold><offsets xml_i="30564" xml_f="30865" txt_i="16701" txt_f="17002">) Top biological processes associated with altered proteins 2 hours post-placebo gel application, as determined by DAVID v6.8 for carbopol and HEC formulations. Biological functions are listed along the left y-axis and the number of proteins identified in each pathway is indicated along the x-axis. (</offsets><bold><offsets xml_i="30871" xml_f="30872" txt_i="17002" txt_f="17003">c</offsets></bold><offsets xml_i="30879" xml_f="31251" txt_i="17003" txt_f="17375">) Venn-diagram representing the number of altered proteins overlapping between HEC and carbopol placebo gel treatments. Heatmap displaying the 21 proteins commonly affected 2 hours post- HEC and carbopol placebo gel application. The samples clustered into two distinct branches, with all of the samples collected 2 hours post-application clustering together into branch 2.</offsets></p></caption><graphic xlink:href="41598_2018_26313_Fig4_HTML" id="d29e719"></graphic></fig></p><p id="Par14"><xref rid="Fig4" ref-type="fig"><offsets xml_i="31393" xml_f="31395" txt_i="17377" txt_f="17379">4c</offsets></xref><offsets xml_i="31402" xml_f="32171" txt_i="17379" txt_f="18148">). Cluster analysis of these proteins showed a distinct pattern, in which samples clustered according to their collection time point. Branch 1 contained 75% of the 24 hours post-gel application samples and 100% of the baseline samples. Conversely, Branch 2 contained 100% of the 2 hours post-gel application samples and 25% of the 24 hours post-gel application samples. Thus, the protein expression profile in the samples collected 24 hours post-gel application resembled that of the baseline samples. This suggests that after the response at 2 hours, protein levels returned to baseline levels after 24 hours. This distinction further supports our finding that the response seen 2 hours post-gel application was short-lived, disappearing 24 hours post-gel application.</offsets></p></sec><sec id="Sec6"><title><offsets xml_i="32203" xml_f="32272" txt_i="18150" txt_f="18219">Griffithsin gel is not associated with changes to the rectal proteome</offsets></title><p id="Par15"><offsets xml_i="32294" xml_f="32510" txt_i="18220" txt_f="18433">We next compared the effect of GRFT-containing gels on the rectal mucosal proteome. Compared to placebo, 0.1% GRFT gel did not elicit any significant changes in the rectal mucosa at any time point (FDR &lt; 5%, Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="32542" xml_f="32543" txt_i="18433" txt_f="18434">3</offsets></xref><offsets xml_i="32550" xml_f="32885" txt_i="18434" txt_f="18769">). In case placebo-associated effects overshadowed any GRFT-related proteome alterations, we compared protein expression in participants using the GRFT gel to pre-gel baseline time points; however, the GRFT-containing gels also did not show any significant effects on rectal protein expression relative to baseline (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="32921" xml_f="32923" txt_i="18769" txt_f="18771">S6</offsets></xref><offsets xml_i="32930" xml_f="33185" txt_i="18771" txt_f="19026">). In order to confirm this finding, we evaluated the expression of proteins involved in specific immune processes, including proteolysis, complement, the acute inflammatory response, epithelial cell differentiation, and wound healing (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="33221" xml_f="33223" txt_i="19026" txt_f="19028">S7</offsets></xref><offsets xml_i="33230" xml_f="33476" txt_i="19028" txt_f="19274">). Complement activation proteins in the 0.1% GRFT-HEC gel arm appeared to cluster based by time point, with the 2 hour post gel-application samples and the 24 hour post-gel application samples separating into distinct groups (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="33512" xml_f="33514" txt_i="19274" txt_f="19276">S7</offsets></xref><offsets xml_i="33521" xml_f="33965" txt_i="19276" txt_f="19720">). However, the statistical significance of this finding could not be determined due to the small sample size. In all other comparisons, there was no observable clustering between the 2 hour and 24 hour post-application time points. Notably, one protein, Myeloperoxidase (MPO), trended towards significance 24 hours post-gel application (p value = 1.73E-4, FDR = 5.23% when placebo and GRFT arms of the carbopol formulation were compared (Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="33997" xml_f="33999" txt_i="19720" txt_f="19722">3a</offsets></xref><offsets xml_i="34006" xml_f="34252" txt_i="19722" txt_f="19968">). Further analyses of the carbopol formulation show that this effect was likely driven by relatively low MPO levels 24 hours post-placebo gel treatment, skewing the comparison against the 24 hour post-0.1% GRFT gel treatment (Supplementary Fig. </offsets><xref rid="MOESM1" ref-type="media"><offsets xml_i="34288" xml_f="34290" txt_i="19968" txt_f="19970">S8</offsets></xref><offsets xml_i="34297" xml_f="34299" txt_i="19970" txt_f="19972">).</offsets></p></sec><sec id="Sec7"><title><offsets xml_i="34331" xml_f="34380" txt_i="19974" txt_f="20023">Structure of the rhesus macaque rectal microbiota</offsets></title><p id="Par16"><offsets xml_i="34402" xml_f="34500" txt_i="20024" txt_f="20122">A total of 81/84 samples were successfully processed (14 samples per animal, Supplementary Tables </offsets><xref rid="MOESM2" ref-type="media"><offsets xml_i="34536" xml_f="34538" txt_i="20122" txt_f="20124">S1</offsets></xref><offsets xml_i="34545" xml_f="34550" txt_i="20124" txt_f="20129"> and </offsets><xref rid="MOESM3" ref-type="media"><offsets xml_i="34586" xml_f="34588" txt_i="20129" txt_f="20131">S2</offsets></xref><offsets xml_i="34595" xml_f="34934" txt_i="20131" txt_f="20470">) and generated a mean number of 58,671 16 S rRNA gene sequences per sample after stringent quality control and assembly. No statistically significant changes in the rhesus macaque rectal microbiota Shannon diversity were observed when comparing placebo (HEC and carbopol) to baseline samples or the 0.1% GRFT gel to placebo samples (Fig. </offsets><xref rid="Fig5" ref-type="fig"><offsets xml_i="34966" xml_f="34967" txt_i="20470" txt_f="20471">5</offsets></xref><offsets xml_i="34974" xml_f="34976" txt_i="20471" txt_f="20473">).</offsets><fig id="Fig5"><label><offsets xml_i="34998" xml_f="35006" txt_i="20473" txt_f="20481">Figure 5</offsets></label><caption><p><offsets xml_i="35026" xml_f="35453" txt_i="20481" txt_f="20908">Effects of HEC and carbopol placebo gel or 0.1% GRFT gels on Shannon diversity of rhesus macaque rectal microbiota. Volcano plot displays the effect sizes of compared treatments or placebos on Shannon diversity values on the x-axis and the −log10 p-values on the y-axis. The gray horizontal line indicates p-value of 0.01. There were no significant effects on the Shannon diversity of rhesus macaque rectal microbiota detected.</offsets></p></caption><graphic xlink:href="41598_2018_26313_Fig5_HTML" id="d29e771"></graphic></fig></p><p id="Par17"><offsets xml_i="35563" xml_f="35824" txt_i="20910" txt_f="21171">The effects of placebo (HEC and carbopol) or GRFT-gel application on specific rectum-associated bacterial taxa were evaluated. For each comparison, estimates of the mean of log ratios of taxa relative abundances in each treatment and control were modeled (Fig. </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="35856" xml_f="35857" txt_i="21171" txt_f="21172">6</offsets></xref><offsets xml_i="35864" xml_f="35919" txt_i="21172" txt_f="21227">). A significant increase in the relative abundance of </offsets><italic><offsets xml_i="35927" xml_f="35943" txt_i="21227" txt_f="21243">Faecalibacterium</offsets></italic><offsets xml_i="35952" xml_f="36054" txt_i="21243" txt_f="21345"> was observed 24-hours post HEC placebo gel application relative to baseline samples (q-value 5.5 × 10</offsets><sup><offsets xml_i="36059" xml_f="36062" txt_i="21345" txt_f="21348">−13</offsets></sup><offsets xml_i="36068" xml_f="36076" txt_i="21348" txt_f="21356">, Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="36110" xml_f="36111" txt_i="21356" txt_f="21357">1</offsets></xref><offsets xml_i="36118" xml_f="36266" txt_i="21357" txt_f="21505">). However, this effect was not observed 7-days post HEC placebo gel application. In addition, a significant increase in the relative abundances of </offsets><italic><offsets xml_i="36274" xml_f="36293" txt_i="21505" txt_f="21524">Christensenellaceae</offsets></italic><offsets xml_i="36302" xml_f="36311" txt_i="21524" txt_f="21533"> R.7 and </offsets><italic><offsets xml_i="36319" xml_f="36334" txt_i="21533" txt_f="21548">Ruminococcaceae</offsets></italic><offsets xml_i="36343" xml_f="36512" txt_i="21548" txt_f="21717"> NK4A214 were detected 24-hours after application of GRFT-HEC gel relative to 24-hours post HEC placebo gel application (q-values 2.2E-09and 8.4E-08, respectively Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="36546" xml_f="36547" txt_i="21717" txt_f="21718">1</offsets></xref><offsets xml_i="36554" xml_f="36671" txt_i="21718" txt_f="21835">). No statistically significant changes in the rectal microbiota were observed in the other comparisons tested (Fig. </offsets><xref rid="Fig6" ref-type="fig"><offsets xml_i="36703" xml_f="36704" txt_i="21835" txt_f="21836">6</offsets></xref><offsets xml_i="36711" xml_f="36872" txt_i="21836" txt_f="21997">). Notably, carbopol placebo or GRFT-carbopol gel application had no statistically significant effects on the structure and composition of the rectal microbiota.</offsets><fig id="Fig6"><label><offsets xml_i="36894" xml_f="36902" txt_i="21997" txt_f="22005">Figure 6</offsets></label><caption><p><offsets xml_i="36922" xml_f="37596" txt_i="22005" txt_f="22679">Changes in the rectal microbiota after application of either drug (0.1% GRFT) or placebo using either HEC or carbopol formulations. Volcano plots display the log10-fold change in relative abundances of each detected bacterial taxon along the x-axis and -log10 of the q-values (as determined by Bayesian logistic-normal paired models) along the y-axis. The gray horizontal lines indicate q-value of 0.01. Green points represent taxa for which relative abundances were significantly increased for the treatment (placebo or GRFT) indicated in the right panel of each window. Significant effects were observed 24 h post HEC placebo gel application relative to baseline samples (</offsets><bold><offsets xml_i="37602" xml_f="37603" txt_i="22679" txt_f="22680">b</offsets></bold><offsets xml_i="37610" xml_f="37692" txt_i="22680" txt_f="22762">), and 24 h post-GRFT-HEC gel relative to 24 h post-HEC placebo gel applications (</offsets><bold><offsets xml_i="37698" xml_f="37699" txt_i="22762" txt_f="22763">b</offsets></bold><offsets xml_i="37706" xml_f="37788" txt_i="22763" txt_f="22845">). No significant effects were observed in any other bacterial taxa or treatments.</offsets></p></caption><graphic xlink:href="41598_2018_26313_Fig6_HTML" id="d29e813"></graphic></fig><table-wrap id="Tab1"><label><offsets xml_i="37909" xml_f="37916" txt_i="22846" txt_f="22853">Table 1</offsets></label><caption><p><offsets xml_i="37936" xml_f="38022" txt_i="22853" txt_f="22939">Taxa for which relative abundances significantly changed between indicated conditions.</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th></th><th><offsets xml_i="38097" xml_f="38098" txt_i="22940" txt_f="22941">
</offsets><bold><offsets xml_i="38104" xml_f="38115" txt_i="22941" txt_f="22952">Effect Size</offsets></bold><offsets xml_i="38122" xml_f="38123" txt_i="22952" txt_f="22953">
</offsets></th><th><offsets xml_i="38132" xml_f="38133" txt_i="22953" txt_f="22954">
</offsets><bold><offsets xml_i="38139" xml_f="38146" txt_i="22954" txt_f="22961">p-value</offsets></bold><offsets xml_i="38153" xml_f="38154" txt_i="22961" txt_f="22962">
</offsets></th><th><offsets xml_i="38163" xml_f="38164" txt_i="22962" txt_f="22963">
</offsets><bold><offsets xml_i="38170" xml_f="38177" txt_i="22963" txt_f="22970">q-value</offsets></bold><offsets xml_i="38184" xml_f="38185" txt_i="22970" txt_f="22971">
</offsets></th><th><offsets xml_i="38194" xml_f="38195" txt_i="22971" txt_f="22972">
</offsets><bold><offsets xml_i="38201" xml_f="38202" txt_i="22972" txt_f="22973">n</offsets></bold><offsets xml_i="38209" xml_f="38210" txt_i="22973" txt_f="22974">
</offsets></th><th><offsets xml_i="38219" xml_f="38220" txt_i="22974" txt_f="22975">
</offsets><bold><offsets xml_i="38226" xml_f="38255" txt_i="22975" txt_f="23004">Median proportion (Treatment)</offsets></bold><offsets xml_i="38262" xml_f="38263" txt_i="23004" txt_f="23005">
</offsets></th><th><offsets xml_i="38272" xml_f="38273" txt_i="23005" txt_f="23006">
</offsets><bold><offsets xml_i="38279" xml_f="38306" txt_i="23006" txt_f="23033">Median proportion (Control)</offsets></bold><offsets xml_i="38313" xml_f="38314" txt_i="23033" txt_f="23034">
</offsets></th><th><offsets xml_i="38323" xml_f="38324" txt_i="23034" txt_f="23035">
</offsets><bold><offsets xml_i="38330" xml_f="38347" txt_i="23035" txt_f="23052">Median proportion</offsets></bold><offsets xml_i="38354" xml_f="38355" txt_i="23052" txt_f="23053">
</offsets><break></break><offsets xml_i="38370" xml_f="38371" txt_i="23053" txt_f="23054">
</offsets><bold><offsets xml_i="38377" xml_f="38382" txt_i="23054" txt_f="23059">(7 d)</offsets></bold><offsets xml_i="38389" xml_f="38390" txt_i="23059" txt_f="23060">
</offsets></th></tr></thead><tbody><tr><td colspan="8"><offsets xml_i="38435" xml_f="38436" txt_i="23060" txt_f="23061">
</offsets><bold><offsets xml_i="38442" xml_f="38470" txt_i="23061" txt_f="23089">HEC Placebo 24 h vs Baseline</offsets></bold><offsets xml_i="38477" xml_f="38478" txt_i="23089" txt_f="23090">
</offsets></td></tr><tr><td><offsets xml_i="38496" xml_f="38497" txt_i="23091" txt_f="23092">
</offsets><italic><offsets xml_i="38505" xml_f="38521" txt_i="23092" txt_f="23108">Faecalibacterium</offsets></italic><offsets xml_i="38530" xml_f="38531" txt_i="23108" txt_f="23109">
</offsets></td><td><offsets xml_i="38540" xml_f="38545" txt_i="23110" txt_f="23115">0.352</offsets></td><td><offsets xml_i="38554" xml_f="38562" txt_i="23116" txt_f="23124">4.21E-15</offsets></td><td><offsets xml_i="38571" xml_f="38579" txt_i="23125" txt_f="23133">5.52E−13</offsets></td><td><offsets xml_i="38588" xml_f="38589" txt_i="23134" txt_f="23135">6</offsets></td><td><offsets xml_i="38598" xml_f="38603" txt_i="23136" txt_f="23141">0.028</offsets></td><td><offsets xml_i="38612" xml_f="38617" txt_i="23142" txt_f="23147">0.010</offsets></td><td><offsets xml_i="38626" xml_f="38631" txt_i="23148" txt_f="23153">0.018</offsets></td></tr><tr><td colspan="8"><offsets xml_i="38661" xml_f="38662" txt_i="23154" txt_f="23155">
</offsets><bold><offsets xml_i="38668" xml_f="38697" txt_i="23155" txt_f="23184">GRFT-HEC 24 h vs Placebo 24 h</offsets></bold><offsets xml_i="38704" xml_f="38705" txt_i="23184" txt_f="23185">
</offsets></td></tr><tr><td><italic><offsets xml_i="38731" xml_f="38750" txt_i="23186" txt_f="23205">Christensenellaceae</offsets></italic><offsets xml_i="38759" xml_f="38769" txt_i="23205" txt_f="23215"> R.7 group</offsets></td><td><offsets xml_i="38778" xml_f="38783" txt_i="23216" txt_f="23221">0.351</offsets></td><td><offsets xml_i="38792" xml_f="38800" txt_i="23222" txt_f="23230">1.98E-11</offsets></td><td><offsets xml_i="38809" xml_f="38817" txt_i="23231" txt_f="23239">2.24E-09</offsets></td><td><offsets xml_i="38826" xml_f="38827" txt_i="23240" txt_f="23241">6</offsets></td><td><offsets xml_i="38836" xml_f="38841" txt_i="23242" txt_f="23247">0.020</offsets></td><td><offsets xml_i="38850" xml_f="38855" txt_i="23248" txt_f="23253">0.008</offsets></td><td><offsets xml_i="38864" xml_f="38869" txt_i="23254" txt_f="23259">0.014</offsets></td></tr><tr><td><italic><offsets xml_i="38895" xml_f="38910" txt_i="23260" txt_f="23275">Ruminococcaceae</offsets></italic><offsets xml_i="38919" xml_f="38933" txt_i="23275" txt_f="23289"> NK4A214 group</offsets></td><td><offsets xml_i="38942" xml_f="38947" txt_i="23290" txt_f="23295">0.348</offsets></td><td><offsets xml_i="38956" xml_f="38964" txt_i="23296" txt_f="23304">1.49E-09</offsets></td><td><offsets xml_i="38973" xml_f="38981" txt_i="23305" txt_f="23313">8.42E-08</offsets></td><td><offsets xml_i="38990" xml_f="38991" txt_i="23314" txt_f="23315">6</offsets></td><td><offsets xml_i="39000" xml_f="39005" txt_i="23316" txt_f="23321">0.008</offsets></td><td><offsets xml_i="39014" xml_f="39019" txt_i="23322" txt_f="23327">0.004</offsets></td><td><offsets xml_i="39028" xml_f="39033" txt_i="23328" txt_f="23333">0.009</offsets></td></tr></tbody></table><table-wrap-foot><p><italic><offsets xml_i="39087" xml_f="39103" txt_i="23334" txt_f="23350">Faecalibacterium</offsets></italic><offsets xml_i="39112" xml_f="39275" txt_i="23350" txt_f="23513"> relative abundances were significantly increased 24 h post HEC placebo gel application (Treatment) relative to baseline samples (Control). Relative abundances of </offsets><italic><offsets xml_i="39283" xml_f="39302" txt_i="23513" txt_f="23532">Christensenellaceae</offsets></italic><offsets xml_i="39311" xml_f="39320" txt_i="23532" txt_f="23541"> R.7 and </offsets><italic><offsets xml_i="39328" xml_f="39343" txt_i="23541" txt_f="23556">Ruminococcaceae</offsets></italic><offsets xml_i="39352" xml_f="39505" txt_i="23556" txt_f="23709"> NK4A214 were significantly increased 24 h after application of GRFT-HEC gel (Treatment) relative to 24-hours post HEC placebo gel application (Control).</offsets></p></table-wrap-foot></table-wrap></p></sec></sec><sec id="Sec8" sec-type="discussion"><title><offsets xml_i="39600" xml_f="39610" txt_i="23713" txt_f="23723">Discussion</offsets></title><p id="Par18"><offsets xml_i="39632" xml_f="40050" txt_i="23724" txt_f="24142">This study was performed to evaluate the effects of 0.1% GRFT gel on the rectal mucosal proteome and microbiome when formulated in either HEC or carbopol. Importantly, this pilot study showed that 0.1% GRFT gels did not associate with any significant alterations to the rectal mucosal proteome when compared to matched placebo gel formulations. Additionally, small, but significant increases in relative abundances of </offsets><italic><offsets xml_i="40058" xml_f="40073" txt_i="24142" txt_f="24157">Ruminococcaceae</offsets></italic><offsets xml_i="40082" xml_f="40095" txt_i="24157" txt_f="24170"> NK4A214 and </offsets><italic><offsets xml_i="40103" xml_f="40122" txt_i="24170" txt_f="24189">Christensenellaceae</offsets></italic><offsets xml_i="40131" xml_f="40575" txt_i="24189" txt_f="24633"> R-7, both of which are low-abundant members of the rhesus macaque rectum-associated microbiota, were detected. This effect disappeared 7 days post gel application. In both the proteome and the microbiota, significant changes were associated with placebo gel application. This initial pilot study using a model of non-human primates suggests that placebo gels may not be completely inert with respect to effects on rectal mucosa and microbiota.</offsets></p><p id="Par19"><offsets xml_i="40593" xml_f="40739" txt_i="24634" txt_f="24780">The antiviral activity of lectins is due to their ability to inhibit viral replication through interactions with the glycans on the viral envelope</offsets><sup><xref ref-type="bibr" rid="CR41"><offsets xml_i="40777" xml_f="40779" txt_i="24780" txt_f="24782">41</offsets></xref></sup><offsets xml_i="40792" xml_f="41174" txt_i="24782" txt_f="25164">. Importantly, the oligosaccharides of the viral envelope are synthesized and attached by host enzymes, and thus, similar oligosaccharides may be found on the surface of host cells. Consequently, lectins used as antiviral therapies may have off-target effects on host cells. Specifically, lectins are known to agglutinate cells, and many lectins have demonstrated mitogenic activity</offsets><sup><xref ref-type="bibr" rid="CR42"><offsets xml_i="41212" xml_f="41214" txt_i="25164" txt_f="25166">42</offsets></xref><offsets xml_i="41221" xml_f="41222" txt_i="25166" txt_f="25167">,</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="41255" xml_f="41257" txt_i="25167" txt_f="25169">43</offsets></xref></sup><offsets xml_i="41270" xml_f="41862" txt_i="25169" txt_f="25761">. For these reasons, there is concern that lectins could have potential off-target side effects in the host. Additionally, as foreign proteins, lectins may induce acute immune responses in the host. These responses are important in the context of HIV infection, in which increased immune activation may lead to increased HIV target cell recruitment at mucosal surfaces. Previous studies of GRFT show that it has a promising safety profile, with no cytotoxicity in several different cell lines, including cervical (End1/E6E7, Ect1/E6E7, CaSki), fibroblast (3T3) and dendritic (moDC) cell lines</offsets><sup><xref ref-type="bibr" rid="CR32"><offsets xml_i="41900" xml_f="41902" txt_i="25761" txt_f="25763">32</offsets></xref><offsets xml_i="41909" xml_f="41910" txt_i="25763" txt_f="25764">,</offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="41943" xml_f="41945" txt_i="25764" txt_f="25766">33</offsets></xref></sup><offsets xml_i="41958" xml_f="42063" txt_i="25766" txt_f="25871">. These studies also showed that GRFT had no mitogenic activity in cervical cell lines, or in human PBMCs</offsets><sup><xref ref-type="bibr" rid="CR32"><offsets xml_i="42101" xml_f="42103" txt_i="25871" txt_f="25873">32</offsets></xref><offsets xml_i="42110" xml_f="42111" txt_i="25873" txt_f="25874">,</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="42144" xml_f="42146" txt_i="25874" txt_f="25876">35</offsets></xref></sup><offsets xml_i="42159" xml_f="42302" txt_i="25876" txt_f="26019">. Additionally, GRFT induced little to no changes in inflammatory cytokine profiles in cervical cell lines and in human cervical explant tissue</offsets><sup><xref ref-type="bibr" rid="CR32"><offsets xml_i="42340" xml_f="42342" txt_i="26019" txt_f="26021">32</offsets></xref><offsets xml_i="42349" xml_f="42350" txt_i="26021" txt_f="26022">,</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="42383" xml_f="42385" txt_i="26022" txt_f="26024">35</offsets></xref></sup><offsets xml_i="42398" xml_f="42400" txt_i="26024" txt_f="26026">. </offsets><italic><offsets xml_i="42408" xml_f="42415" txt_i="26026" txt_f="26033">In vivo</offsets></italic><offsets xml_i="42424" xml_f="42595" txt_i="26033" txt_f="26204"> studies showed that GRFT was non-irritating and non-inflammatory in the rabbit vaginal irritation model and that GRFT had no effect on the mucosal immune response in mice</offsets><sup><xref ref-type="bibr" rid="CR33"><offsets xml_i="42633" xml_f="42635" txt_i="26204" txt_f="26206">33</offsets></xref><offsets xml_i="42642" xml_f="42643" txt_i="26206" txt_f="26207">,</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="42676" xml_f="42678" txt_i="26207" txt_f="26209">35</offsets></xref></sup><offsets xml_i="42691" xml_f="42922" txt_i="26209" txt_f="26440">. The results from the current study support the safety of GRFT, as we found no association of GRFT gel formulations with any significant alterations to the rectal mucosal proteome when compared to matched placebo gel formulations.</offsets></p><p id="Par20"><offsets xml_i="42940" xml_f="43210" txt_i="26441" txt_f="26711">We observed that application of HEC and carbopol placebo gels associated with transient changes in the acute inflammatory response and activation of the immune system. This may be important in the context of HIV infection, as inflammation increases risk for transmission</offsets><sup><xref ref-type="bibr" rid="CR20"><offsets xml_i="43248" xml_f="43250" txt_i="26711" txt_f="26713">20</offsets></xref></sup><offsets xml_i="43263" xml_f="43412" txt_i="26713" txt_f="26862">, although the importance of these molecular pathways at mucosal surfaces to viral susceptibility is unknown. Notably, a study in a macaque model by </offsets><italic><offsets xml_i="43420" xml_f="43438" txt_i="26862" txt_f="26880">Vishwanathan et al</offsets></italic><offsets xml_i="43447" xml_f="43576" txt_i="26880" txt_f="27009">. showed that the application of a rectal lubricant induced a transient cytotoxicity, but did not increase risk of SHIV infection</offsets><sup><xref ref-type="bibr" rid="CR44"><offsets xml_i="43614" xml_f="43616" txt_i="27009" txt_f="27011">44</offsets></xref></sup><offsets xml_i="43629" xml_f="43750" txt_i="27011" txt_f="27132">. While many studies have documented the safety of vaginal and rectal placebo gels containing hydroxyethylcellulose (HEC)</offsets><sup><xref ref-type="bibr" rid="CR45"><offsets xml_i="43788" xml_f="43790" txt_i="27132" txt_f="27134">45</offsets></xref><offsets xml_i="43797" xml_f="43798" txt_i="27134" txt_f="27135">–</offsets><xref ref-type="bibr" rid="CR47"><offsets xml_i="43831" xml_f="43833" txt_i="27135" txt_f="27137">47</offsets></xref></sup><offsets xml_i="43846" xml_f="43944" txt_i="27137" txt_f="27235">, others have reported undesirable effects of HEC-containing gels on the vaginal and rectal mucosa</offsets><sup><xref ref-type="bibr" rid="CR26"><offsets xml_i="43982" xml_f="43984" txt_i="27235" txt_f="27237">26</offsets></xref><offsets xml_i="43991" xml_f="43992" txt_i="27237" txt_f="27238">,</offsets><xref ref-type="bibr" rid="CR48"><offsets xml_i="44025" xml_f="44027" txt_i="27238" txt_f="27240">48</offsets></xref><offsets xml_i="44034" xml_f="44035" txt_i="27240" txt_f="27241">–</offsets><xref ref-type="bibr" rid="CR50"><offsets xml_i="44068" xml_f="44070" txt_i="27241" txt_f="27243">50</offsets></xref></sup><offsets xml_i="44083" xml_f="44191" txt_i="27243" txt_f="27351">. There have also been many studies conducted on gels formulated with Carbopol 974 P showing various results</offsets><sup><xref ref-type="bibr" rid="CR51"><offsets xml_i="44229" xml_f="44231" txt_i="27351" txt_f="27353">51</offsets></xref><offsets xml_i="44238" xml_f="44239" txt_i="27353" txt_f="27354">–</offsets><xref ref-type="bibr" rid="CR53"><offsets xml_i="44272" xml_f="44274" txt_i="27354" txt_f="27356">53</offsets></xref></sup><offsets xml_i="44287" xml_f="44554" txt_i="27356" txt_f="27623">. It is important to acknowledge the compositional differences between the HEC- and carbopol- formulated gels used in our study and the gels used in the studies mentioned above. For this reason, it is difficult to make direct comparisons between our study and others.</offsets></p><p id="Par21"><offsets xml_i="44572" xml_f="44699" txt_i="27624" txt_f="27751">Microbiota analyses of samples collected 24 hours post-HEC placebo gels showed a ~2 fold increase in the relative abundance of </offsets><italic><offsets xml_i="44707" xml_f="44723" txt_i="27751" txt_f="27767">Faecalibacterium</offsets></italic><offsets xml_i="44732" xml_f="44809" txt_i="27767" txt_f="27844"> spp., which are anaerobic, broadly distributed members of the gut microbiome</offsets><sup><xref ref-type="bibr" rid="CR54"><offsets xml_i="44847" xml_f="44849" txt_i="27844" txt_f="27846">54</offsets></xref></sup><offsets xml_i="44862" xml_f="44864" txt_i="27846" txt_f="27848">. </offsets><italic><offsets xml_i="44872" xml_f="44888" txt_i="27848" txt_f="27864">Faecalibacterium</offsets></italic><offsets xml_i="44897" xml_f="45597" txt_i="27864" txt_f="28564"> are also considered important members of a healthy gut, recognized as butyrate producers, a small chain fatty acid shown to have anti-inflammatory properties. The small but statistically significant increase in relative abundance of this taxon suggests that HEC placebo gel does not have a negative effect on the structure of the gut microbiota, but it may not be inert. In addition, the effect was only observed 24 hours-post HEC application, and disappeared by day 7. No effect on the rhesus macaque rectal microbiota was observed after application of carbopol placebo gels or 0.1% GRFT carbopol gel at any of the time points tested. Notably, recent studies found an enrichment of genera from the </offsets><italic><offsets xml_i="45605" xml_f="45615" txt_i="28564" txt_f="28574">Prevotella</offsets></italic><offsets xml_i="45624" xml_f="45799" txt_i="28574" txt_f="28749"> enterotype, in MSM compared to non-MSM. They suggest that this difference may be associated with certain sexual preferences of MSM, including the common use of lubricant gels</offsets><sup><xref ref-type="bibr" rid="CR55"><offsets xml_i="45837" xml_f="45839" txt_i="28749" txt_f="28751">55</offsets></xref></sup><offsets xml_i="45852" xml_f="45992" txt_i="28751" txt_f="28891">. We did not observe any significant alterations in the relative abundance of Prevotella associated with gel application in rhesus macaques.</offsets></p><p id="Par22"><offsets xml_i="46010" xml_f="47168" txt_i="28892" txt_f="30050">Our study had a few limitations. First, the power of our study was restricted by the small sample size. Second, it is difficult to determine whether the inflammatory processes associated with placebo gel application are due to the chemical components of the placebo gels, or rather, due to the mechanical application procedure itself. With only a single layer of columnar epithelium, the rectal mucosal barrier is relatively fragile, and is possible that the application of any gel may cause injury and induced transient inflammatory events. Additionally, the gel products used in this study contained additional gel excipients, such as glycerine, EDTA and preservatives, which may be associated with a biological response in rectal mucosa. Unfortunately, without proper control groups, we are unable to assess the importance of these effects. Future studies should include a mechanical control (i.e. sham gel application) and positive gel controls, such as imiquimod or N-9. Finally, the samples were collected without aid of an anoscope which may have contributed to more variability in sample collection and limiting sensitivity to detect mucosal changes.</offsets></p><p id="Par23"><offsets xml_i="47186" xml_f="47899" txt_i="30051" txt_f="30764">In conclusion, this multi-platform study incorporating proteomics and microbiome techniques provided comprehensive information of the rectal mucosal environment at the systems level, providing an additional toolset for pre-clinical microbicide safety. Furthermore, the transient inflammatory alterations to the proteome observed with placebo gel use demonstrates that these gels may not be completely inert and will serve as important parameters to monitor in future safety studies. Overall, compared to placebo, 0.1% Griffithsin microbicide gel did not elicit any significant effects to the mucosal proteome and minor effects to the microbiome, suggesting tolerability and suitability as a microbicide candidate.</offsets></p></sec><sec id="Sec9" sec-type="materials|methods"><title><offsets xml_i="47960" xml_f="47967" txt_i="30766" txt_f="30773">Methods</offsets></title><sec id="Sec10"><title><offsets xml_i="47998" xml_f="48010" txt_i="30774" txt_f="30786">Gel products</offsets></title><p id="Par24"><offsets xml_i="48032" xml_f="48342" txt_i="30787" txt_f="31097">Carbopol-formulated 0.1% GRFT and placebo gels contained Carbopol 974 P, methylparaben, propylparaben, methionine, Ethylenediaminetetraacetic acid (EDTA) and purified water. HEC-formulated 0.1% GRFT and placebo gels contained hydroxyethylcellulose, methylparaben, propylparaben, methionine, and purified water.</offsets></p></sec><sec id="Sec11"><title><offsets xml_i="48375" xml_f="48397" txt_i="31099" txt_f="31121">Rectal swab collection</offsets></title><p id="Par25"><offsets xml_i="48419" xml_f="48555" txt_i="31122" txt_f="31258">Study design details are provided in the results section and an overview of the pre-clinical trial, which involved six Rhesus Macaques (</offsets><italic><offsets xml_i="48563" xml_f="48577" txt_i="31258" txt_f="31272">Macaca mulatta</offsets></italic><offsets xml_i="48586" xml_f="48616" txt_i="31272" txt_f="31302">, n = 6), is provided in Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="48648" xml_f="48649" txt_i="31302" txt_f="31303">1</offsets></xref><offsets xml_i="48656" xml_f="49716" txt_i="31303" txt_f="32363">. Each gel application was preceded by collection of pre-gel (T0) rectal swabs and followed by collection of post-gel rectal swabs (2 hours and 24 hours post-gel application for proteomic analysis; 24 hours and 7 days post-gel application for microbiota analysis). Swabs for proteomic analysis (Weck-cel swabs or Merocel sponges, Beaver Visitec Intl Ltd, Fisher Scientific #NC0240644) were pre-wet with cold PBS and inserted into the rectum. Swabs for proteomics were then removed after 5 minutes and excess feces were scraped off. These swabs were then placed in a Costar Spin-X centrifuge tube filter (Sigma Cat. #CLS8160) and eluted with PBS by centrifugation at 4 °C for 30 minutes (16000 rpm). Rectal swab eluates (RSE) were placed into new Spin-X tubes and stored at −80 °C until further proteomic processing. Swabs for microbiota analysis were collected using Copan ESwab system (Copan #480 C) by inserting and rotating the swab three times in the rectum. The swab was then stored in 1 mL of Amies transport medium and stored at −80 °C until processing.</offsets></p></sec><sec id="Sec12"><title><offsets xml_i="49749" xml_f="49826" txt_i="32365" txt_f="32442">Mass spectrometry analysis of rectal swab eluates using label-free proteomics</offsets></title><p id="Par26"><offsets xml_i="49848" xml_f="50222" txt_i="32443" txt_f="32817">Rectal swab samples (n = 96) from six macaques across the 16 time points were collected, processed and digested into peptides. Remaining cellular debris was removed by centrifugation at 23,000 g for 30 minutes. Equal volumes of RSE samples were denatured by adding 600 μl 8 M Urea Exchange Buffer and then digested using filter-aided sample preparation, described previously</offsets><sup><xref ref-type="bibr" rid="CR56"><offsets xml_i="50260" xml_f="50262" txt_i="32817" txt_f="32819">56</offsets></xref><offsets xml_i="50269" xml_f="50270" txt_i="32819" txt_f="32820">,</offsets><xref ref-type="bibr" rid="CR57"><offsets xml_i="50303" xml_f="50305" txt_i="32820" txt_f="32822">57</offsets></xref></sup><offsets xml_i="50318" xml_f="50930" txt_i="32822" txt_f="33434">. Salts and detergents were removed using reverse-phase liquid chromatography (high pH RP, Agilent 1200 series micro-flow pump, Water XBridge column) using a step function gradient eluted into a single sample. Peptide concentration was determined using a FluorProphile® Protein Quantification kit (Sigma-Aldrich, St. Louis, MO) and RSE samples (1 μg peptide per sample) were analysed using an Easy nLC nanoflow LC system (Thermo Fisher Scientific, Waltham, MA) connected in line with an Linear Trap Quadrupole (LTQ) Orbitrap Velos mass spectrometer (Thermo Fisher Scientific, Waltham, MA) as described previously</offsets><sup><xref ref-type="bibr" rid="CR57"><offsets xml_i="50968" xml_f="50970" txt_i="33434" txt_f="33436">57</offsets></xref></sup><offsets xml_i="50983" xml_f="51469" txt_i="33436" txt_f="33922">. Briefly, tryptic peptides were loaded onto a low-pH reversed-phase column (0.075 × 15 cm), eluted over a 120-min gradient (2%–30% acetonitrile, 0.1% formic acid), and nanosprayed into the LTQ Orbitrap Velos. The survey scans (MS1) were acquired in the Orbitrap at resolution of 60,000, and data-dependent acquisition was used to select the top 10 abundant peptide ions for fragmentation by collision induced dissociation. The peptide fragment ion scans (MS2) were acquired in the LTQ.</offsets></p></sec><sec id="Sec13"><title><offsets xml_i="51502" xml_f="51536" txt_i="33924" txt_f="33958">Analysis of mass spectrometry data</offsets></title><p id="Par27"><offsets xml_i="51558" xml_f="52971" txt_i="33959" txt_f="35372">Progenesis Software (v4.0; Nonlinear Dynamics, Durham, NC) was used to process raw MS spectra. One reference sample, which included equal volumes from each sample, was used to align sample spectra automatically, with manual revision for errors. The reference was run every 10 samples on the mass spectrometer, and these technical replicates were used to determine the reproducibility of the MS data. Peptides with a charge state between +2 and +7 and a retention time between 10 and 125 min were retained. Filtered spectra were annotated using Mascot software (v2.4; Matrix Science, Boston, MA) using the UniProtKB/SwissProt (2012, v3.87) database for human and bacterial proteins. Searches were performed with the following criteria: carbamidomethyl (C) fixed modifications, oxidative (M) variable modifications, a fragment ion mass tolerance of 0.5 Da, a parent ion tolerance of 10 ppm, tryptic enzyme digestion with a tolerance of one missed cleavage, and a decoy database. Search results were imported into Scaffold (v4.4.1.1; Proteome Software, Portland, OR) to filter protein identifications (80% confidence for peptide identification, 95% confidence for protein identification, and a minimum of two unique peptides identified per protein). Only proteins matched to human were included in downstream analysis. Feature detection and quantification were all performed using default settings from the software.</offsets></p></sec><sec id="Sec14"><title><offsets xml_i="53004" xml_f="53067" txt_i="35374" txt_f="35437">DNA extraction and 16 S rRNA gene amplication from rectal swabs</offsets></title><p id="Par28"><offsets xml_i="53089" xml_f="53644" txt_i="35438" txt_f="35993">The swabs were thawed on ice, and 300 µl of Amies transport medium containing vaginal secretion were processed using the MoBio Microbiome kit automated on a Hamilton Microlab STAR robotic platform after a bead-beating step on a Qiagen TissueLyser II (20 Hz for 20 min) in 96 deep well plate. Amplification of the V3-V4 regions of the 16 S rRNA gene was performed using a two step-PCR in which the sample specific barcode is added during the second PCR, to maximize target amplification. The first PCR used the short 16 S rRNA gene specific primers 319 F (</offsets><underline><offsets xml_i="53655" xml_f="53677" txt_i="35993" txt_f="36015">ACACTGACGACATGGTTCTACA</offsets></underline><offsets xml_i="53689" xml_f="53694" txt_i="36015" txt_f="36020">[0–7]</offsets><bold><offsets xml_i="53700" xml_f="53720" txt_i="36020" txt_f="36040">ACTCCTRCGGGAGGCAGCAG</offsets></bold><offsets xml_i="53727" xml_f="53740" txt_i="36040" txt_f="36053">) and 806 R (</offsets><underline><offsets xml_i="53751" xml_f="53773" txt_i="36053" txt_f="36075">TACGGTAGCAGAGACTTGGTCT</offsets></underline><offsets xml_i="53785" xml_f="53790" txt_i="36075" txt_f="36080">[0–7]</offsets><bold><offsets xml_i="53796" xml_f="53816" txt_i="36080" txt_f="36100">GGACTACHVGGGTWTCTAAT</offsets></bold><offsets xml_i="53823" xml_f="53992" txt_i="36100" txt_f="36269">) where the underlined sequence is the Illumina sequencing primer sequence and [0–7] indicate the presence of an heterogeneous pad sequence to improve sequencing quality</offsets><sup><xref ref-type="bibr" rid="CR58"><offsets xml_i="54030" xml_f="54032" txt_i="36269" txt_f="36271">58</offsets></xref></sup><offsets xml_i="54045" xml_f="54169" txt_i="36271" txt_f="36395">, for a total of 20 cycles. This first step was followed by 10 cycles with primers H1 (AATGATACGGCGACCACCGAGATCTACACNNNNNNNN</offsets><underline><offsets xml_i="54180" xml_f="54202" txt_i="36395" txt_f="36417">ACACTGACGACATGGTTCTACA</offsets></underline><offsets xml_i="54214" xml_f="54578" txt_i="36417" txt_f="36781">) and H2 (CAAGCAGAAGACGGCATACGAGATNNNNNNNNTACGGTAGCAGAGACTTGGTCT) where NNNNNN indicates a sample specific barcode sequence and the underlined sequence corresponds to the Illumina sequencing primer for priming to the first step amplicon. This second step extends the amplicon with the Illumina required adaptor sequences and the sample specific dual barcode system</offsets><sup><xref ref-type="bibr" rid="CR58"><offsets xml_i="54616" xml_f="54618" txt_i="36781" txt_f="36783">58</offsets></xref></sup><offsets xml_i="54631" xml_f="55145" txt_i="36783" txt_f="37297">. Amplicons were visualized on a 2% agarose gel, quantified, pooled in equimolar concentration and purified prior to loading on an Illumina HiSeq 2500 (San Diego, CA, USA) modified to generate 300 bp paired-end reads. Four extraction and four PCR negative controls were processed in parallel. Additionally, 14 positive controls composed of a mixture of fecal and vaginal biological specimen were processed and sequenced in parallel to the study samples as per the laboratory standard protocol (Supplementary Table </offsets><xref rid="MOESM4" ref-type="media"><offsets xml_i="55181" xml_f="55183" txt_i="37297" txt_f="37299">S3</offsets></xref><offsets xml_i="55190" xml_f="55192" txt_i="37299" txt_f="37301">).</offsets></p></sec><sec id="Sec15"><title><offsets xml_i="55225" xml_f="55271" txt_i="37303" txt_f="37349">Bioinformatics methods for microbiota analysis</offsets></title><sec id="Sec16"><title><offsets xml_i="55302" xml_f="55318" txt_i="37350" txt_f="37366">Quality controls</offsets></title><p id="Par29"><offsets xml_i="55340" xml_f="55511" txt_i="37367" txt_f="37538">The sequences were demultiplexed using the dual-barcode strategy, the mapping file generated on the robotic platform and split_libraries_fastq.py, a QIIME-dependent script</offsets><sup><xref ref-type="bibr" rid="CR59"><offsets xml_i="55549" xml_f="55551" txt_i="37538" txt_f="37540">59</offsets></xref></sup><offsets xml_i="55564" xml_f="55646" txt_i="37540" txt_f="37622">. The resulting forward and reverse fastq files were split by sample using seqtk (</offsets><ext-link ext-link-type="uri" xlink:href="https://github.com/lh3/seqtk"><offsets xml_i="55718" xml_f="55746" txt_i="37622" txt_f="37650">https://github.com/lh3/seqtk</offsets></ext-link><offsets xml_i="55757" xml_f="55831" txt_i="37650" txt_f="37724">), and primer sequences were removed using TagCleaner (0.16) (Schmieder R </offsets><italic><offsets xml_i="55839" xml_f="55844" txt_i="37724" txt_f="37729">et al</offsets></italic><offsets xml_i="55853" xml_f="55963" txt_i="37729" txt_f="37839">. 2017 BMC Bioinformatics). Further processing followed the DADA2 Workflow for Big Data and dada2 (v. 1.5.2) (</offsets><ext-link ext-link-type="uri" xlink:href="https://benjjneb.github.io/dada2/bigdata.html,60"><offsets xml_i="56055" xml_f="56103" txt_i="37839" txt_f="37887">https://benjjneb.github.io/dada2/bigdata.html,60</offsets></ext-link><offsets xml_i="56114" xml_f="56607" txt_i="37887" txt_f="38380">). Forward and reverse reads were each trimmed using lengths of 255 and 225 bp, respectively, and filtered to contain no ambiguous bases, minimum quality score of 2, and required to contain less than two expected errors based on their quality score. The relationship between quality scores and error rates were estimated for both sequencing runs to reduce batch effects arising from run-to-run variability. Reads were assembled and chimeras for the combined runs removed as per dada2 protocol.</offsets></p><p id="Par30"><offsets xml_i="56625" xml_f="56811" txt_i="38381" txt_f="38567">Taxonomy was assigned to each amplicon sequence variant (ASV) generated by dada2 using the SILVA v128 database and the RDP naïve Bayesian classifier as implemented in the dada2 R package</offsets><sup><xref ref-type="bibr" rid="CR60"><offsets xml_i="56849" xml_f="56851" txt_i="38567" txt_f="38569">60</offsets></xref><offsets xml_i="56858" xml_f="56859" txt_i="38569" txt_f="38570">,</offsets><xref ref-type="bibr" rid="CR61"><offsets xml_i="56892" xml_f="56894" txt_i="38570" txt_f="38572">61</offsets></xref></sup><offsets xml_i="56907" xml_f="57130" txt_i="38572" txt_f="38795">. Read counts for ASVs assigned the same taxonomy were summed for each sample. A table including total sequence count for each taxon and for each sample was generated and used for statistical analyses (Supplementary Tables </offsets><xref rid="MOESM3" ref-type="media"><offsets xml_i="57166" xml_f="57168" txt_i="38795" txt_f="38797">S2</offsets></xref><offsets xml_i="57175" xml_f="57180" txt_i="38797" txt_f="38802"> and </offsets><xref rid="MOESM4" ref-type="media"><offsets xml_i="57216" xml_f="57218" txt_i="38802" txt_f="38804">S3</offsets></xref><offsets xml_i="57225" xml_f="57227" txt_i="38804" txt_f="38806">).</offsets></p></sec></sec><sec id="Sec17"><title><offsets xml_i="57266" xml_f="57286" txt_i="38809" txt_f="38829">Statistical analysis</offsets></title><sec id="Sec18"><title><offsets xml_i="57317" xml_f="57327" txt_i="38830" txt_f="38840">Proteomics</offsets></title><p id="Par31"><offsets xml_i="57349" xml_f="58605" txt_i="38841" txt_f="40097">Progenesis Software was used to normalize protein abundances in each sample to total ion current. Technical variability was assessed using technical replicates of a reference sample (consisting of equal amounts of peptide from each sample) run once every 10 samples on the mass spectrometer. Proteins with a coefficient of variance below 25% between these mixes were retained and those greater than 25% were removed from the downstream analysis. All samples were further normalized to the median protein abundance using linear regression. Outlier samples were identified as having a normalized protein abundance that was greater than 1.5 times the interquartile range of the median normalized abundance of proteins identified across all samples. A total of 3/96 samples were identified as outliers, but were kept in downstream analysis due to the longitudinal nature of the study. Samples and protein identifications were subject to differential expression analysis. For each macaque, protein levels from all baseline time points (n = 4) were averaged. Protein levels from samples collected at T0, 2 h post gel and 24 h post-gel application were then log2 transformed, normalized to their respective baseline average, and analysed using two-tailed, paired </offsets><italic><offsets xml_i="58613" xml_f="58614" txt_i="40097" txt_f="40098">t</offsets></italic><offsets xml_i="58623" xml_f="59780" txt_i="40098" txt_f="41255"> tests to determine changes in protein expression throughout the trial. Q-q plots were generated and assessed to ensure the assumptions of normality applied to our data set and that the use of parametric statistics on our small sample size was acceptable. Notably, protein abundances were not normalized to their respective baseline average in cluster analyses that included baseline values. This was done in order to avoid skewing the clustering algorithm where Pearson’s correlation distance metric and complete linkage were applied. G*Power 3.1 was used to calculate the power of this study. Our study was powered to confidently detect protein changes greater than |2.3 Log2 FC| to retain an experimental power of 80%. Proteins determined to be significantly differentially expressed were those that passed a local false discovery rate (FDR) of 5% using the Benjimani-Hochberg method to adjust for a total of 382 protein comparisons. Functional annotation analysis and pathway analysis of these proteins were performed using DAVID (Database for Annotation, Visualization, and Integrated Discovery, v6.8) and Ingenuity Pathway Analysis (IPA, QIAGEN Inc., </offsets><ext-link ext-link-type="uri" xlink:href="https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis"><offsets xml_i="59896" xml_f="59968" txt_i="41255" txt_f="41327">https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis</offsets></ext-link><offsets xml_i="59979" xml_f="60068" txt_i="41327" txt_f="41416">); statistical enrichment values were calculated according to default software parameters</offsets><sup><xref ref-type="bibr" rid="CR62"><offsets xml_i="60106" xml_f="60108" txt_i="41416" txt_f="41418">62</offsets></xref><offsets xml_i="60115" xml_f="60116" txt_i="41418" txt_f="41419">,</offsets><xref ref-type="bibr" rid="CR63"><offsets xml_i="60149" xml_f="60151" txt_i="41419" txt_f="41421">63</offsets></xref></sup><offsets xml_i="60164" xml_f="60408" txt_i="41421" txt_f="41665">. Right-tailed Fisher’s Exact Tests (Benjamini-Hochberg corrected) were used to calculate the probability that the association between each protein in our experimental dataset compared to the manually curated and annotated datasets were random.</offsets></p></sec><sec id="Sec19"><title><offsets xml_i="60441" xml_f="60451" txt_i="41667" txt_f="41677">Microbiota</offsets></title><p id="Par32"><offsets xml_i="60473" xml_f="60540" txt_i="41678" txt_f="41745">Taxa were filtered before analysis if observed at frequencies of 10</offsets><sup><offsets xml_i="60545" xml_f="60547" txt_i="41745" txt_f="41747">−5</offsets></sup><offsets xml_i="60553" xml_f="61060" txt_i="41747" txt_f="42254"> study-wide or if observed in fewer than 25% of samples study-wide or, within each comparison, present in at least 4 subjects. To test for the effect of treatment on community diversity, the mean value of Shannon diversity was compared using a Bayesian paired normal-Laplace model fitted to the logit transformed Shannon diversity data. The logit transformation was applied so that the distribution of transformed values was normally distributed. The paired normal-Laplace model had the following structure:</offsets><disp-formula id="Equa"><alternatives><tex-math id="M1"><offsets xml_i="61116" xml_f="61563" txt_i="42254" txt_f="42701">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$${\rm{yT}}\_{\rm{i}} \sim \mathrm{norm}(\mathrm{muT}\_{\rm{i}},\,\mathrm{sigmaT})\mathrm{yC}\_{\rm{i}} \sim \mathrm{norm}(\mathrm{muC}\_{\rm{i}},\,\mathrm{sigmaC})$$\end{document}</offsets></tex-math><mml:math id="M2" display="block"><mml:mi mathvariant="normal"><offsets xml_i="61637" xml_f="61639" txt_i="42701" txt_f="42703">yT</offsets></mml:mi><mml:mo><offsets xml_i="61656" xml_f="61657" txt_i="42703" txt_f="42704">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="61695" xml_f="61696" txt_i="42704" txt_f="42705">i</offsets></mml:mi><mml:mo><offsets xml_i="61713" xml_f="61714" txt_i="42705" txt_f="42706">~</offsets></mml:mo><mml:mi><offsets xml_i="61731" xml_f="61735" txt_i="42706" txt_f="42710">norm</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="61769" xml_f="61770" txt_i="42710" txt_f="42711">(</offsets></mml:mo><mml:mi><offsets xml_i="61787" xml_f="61790" txt_i="42711" txt_f="42714">muT</offsets></mml:mi><mml:mo><offsets xml_i="61807" xml_f="61808" txt_i="42714" txt_f="42715">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="61846" xml_f="61847" txt_i="42715" txt_f="42716">i</offsets></mml:mi><mml:mo><offsets xml_i="61864" xml_f="61865" txt_i="42716" txt_f="42717">,</offsets></mml:mo><mml:mspace width=".25em"></mml:mspace><mml:mi><offsets xml_i="61921" xml_f="61927" txt_i="42717" txt_f="42723">sigmaT</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="61961" xml_f="61962" txt_i="42723" txt_f="42724">)</offsets></mml:mo><mml:mi><offsets xml_i="61979" xml_f="61981" txt_i="42724" txt_f="42726">yC</offsets></mml:mi><mml:mo><offsets xml_i="61998" xml_f="61999" txt_i="42726" txt_f="42727">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="62037" xml_f="62038" txt_i="42727" txt_f="42728">i</offsets></mml:mi><mml:mo><offsets xml_i="62055" xml_f="62056" txt_i="42728" txt_f="42729">~</offsets></mml:mo><mml:mi><offsets xml_i="62073" xml_f="62077" txt_i="42729" txt_f="42733">norm</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="62111" xml_f="62112" txt_i="42733" txt_f="42734">(</offsets></mml:mo><mml:mi><offsets xml_i="62129" xml_f="62132" txt_i="42734" txt_f="42737">muC</offsets></mml:mi><mml:mo><offsets xml_i="62149" xml_f="62150" txt_i="42737" txt_f="42738">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="62188" xml_f="62189" txt_i="42738" txt_f="42739">i</offsets></mml:mi><mml:mo><offsets xml_i="62206" xml_f="62207" txt_i="42739" txt_f="42740">,</offsets></mml:mo><mml:mspace width=".25em"></mml:mspace><mml:mi><offsets xml_i="62263" xml_f="62269" txt_i="42740" txt_f="42746">sigmaC</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="62303" xml_f="62304" txt_i="42746" txt_f="42747">)</offsets></mml:mo></mml:math><graphic xlink:href="41598_2018_26313_Article_Equa.gif" position="anchor"></graphic></alternatives></disp-formula><offsets xml_i="62438" xml_f="62947" txt_i="42747" txt_f="43256">where yT_i and yC_i are the transformed Shannon diversity values of the i-th treatment and control samples, respectively. Under this model it is assumed that the difference in the mean Shannon diversity values of treatments and controls (Delta.mu_i = muT_i − muC_i) were sampled from a Laplace distribution (double exponential); Delta.mu_i ~ Laplace(mu, sigma). For each condition, 3 chains with 10,000 iterations and thinning of 10 were run. Convergence was tested using potential scale reduction statistics.</offsets></p><p id="Par33"><disp-formula id="Equb"><alternatives><tex-math id="M3"><offsets xml_i="63021" xml_f="63386" txt_i="43257" txt_f="43622">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$${\rm{yT}}\_{\rm{i}} \sim \mathrm{bin}(\mathrm{nReadsT}\_{\rm{i}},\,{\rm{pT}}\_i)$$\end{document}</offsets></tex-math><mml:math id="M4" display="block"><mml:mi mathvariant="normal"><offsets xml_i="63460" xml_f="63462" txt_i="43622" txt_f="43624">yT</offsets></mml:mi><mml:mo><offsets xml_i="63479" xml_f="63480" txt_i="43624" txt_f="43625">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="63518" xml_f="63519" txt_i="43625" txt_f="43626">i</offsets></mml:mi><mml:mo><offsets xml_i="63536" xml_f="63537" txt_i="43626" txt_f="43627">~</offsets></mml:mo><mml:mi><offsets xml_i="63554" xml_f="63557" txt_i="43627" txt_f="43630">bin</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="63591" xml_f="63592" txt_i="43630" txt_f="43631">(</offsets></mml:mo><mml:mi><offsets xml_i="63609" xml_f="63616" txt_i="43631" txt_f="43638">nReadsT</offsets></mml:mi><mml:mo><offsets xml_i="63633" xml_f="63634" txt_i="43638" txt_f="43639">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="63672" xml_f="63673" txt_i="43639" txt_f="43640">i</offsets></mml:mi><mml:mo><offsets xml_i="63690" xml_f="63691" txt_i="43640" txt_f="43641">,</offsets></mml:mo><mml:mspace width=".25em"></mml:mspace><mml:mi mathvariant="normal"><offsets xml_i="63768" xml_f="63770" txt_i="43641" txt_f="43643">pT</offsets></mml:mi><mml:mo><offsets xml_i="63787" xml_f="63788" txt_i="43643" txt_f="43644">_</offsets></mml:mo><mml:mi><offsets xml_i="63805" xml_f="63806" txt_i="43644" txt_f="43645">i</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="63840" xml_f="63841" txt_i="43645" txt_f="43646">)</offsets></mml:mo></mml:math><graphic xlink:href="41598_2018_26313_Article_Equb.gif" position="anchor"></graphic></alternatives></disp-formula><disp-formula id="Equc"><alternatives><tex-math id="M5"><offsets xml_i="64031" xml_f="64399" txt_i="43646" txt_f="44014">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$${\rm{yC}}\_{\rm{i}} \sim {\rm{bin}}({\rm{nReadsC}}\_{\rm{i}},\,{\rm{pC}}\_{\rm{i}})$$\end{document}</offsets></tex-math><mml:math id="M6" display="block"><mml:mi mathvariant="normal"><offsets xml_i="64473" xml_f="64475" txt_i="44014" txt_f="44016">yC</offsets></mml:mi><mml:mo><offsets xml_i="64492" xml_f="64493" txt_i="44016" txt_f="44017">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="64531" xml_f="64532" txt_i="44017" txt_f="44018">i</offsets></mml:mi><mml:mo><offsets xml_i="64549" xml_f="64550" txt_i="44018" txt_f="44019">~</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="64588" xml_f="64591" txt_i="44019" txt_f="44022">bin</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="64625" xml_f="64626" txt_i="44022" txt_f="44023">(</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="64664" xml_f="64671" txt_i="44023" txt_f="44030">nReadsC</offsets></mml:mi><mml:mo><offsets xml_i="64688" xml_f="64689" txt_i="44030" txt_f="44031">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="64727" xml_f="64728" txt_i="44031" txt_f="44032">i</offsets></mml:mi><mml:mo><offsets xml_i="64745" xml_f="64746" txt_i="44032" txt_f="44033">,</offsets></mml:mo><mml:mspace width=".25em"></mml:mspace><mml:mi mathvariant="normal"><offsets xml_i="64823" xml_f="64825" txt_i="44033" txt_f="44035">pC</offsets></mml:mi><mml:mo><offsets xml_i="64842" xml_f="64843" txt_i="44035" txt_f="44036">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="64881" xml_f="64882" txt_i="44036" txt_f="44037">i</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="64916" xml_f="64917" txt_i="44037" txt_f="44038">)</offsets></mml:mo></mml:math><graphic xlink:href="41598_2018_26313_Article_Equc.gif" position="anchor"></graphic></alternatives></disp-formula><offsets xml_i="65051" xml_f="65769" txt_i="44038" txt_f="44756">where yT_i and nReadsT_i are the i-th treatment sample’s sequence count for a given taxon and the sample’s total sequence count, respectively. yC_i and nReadsC_i are the i-th control sample’s sequence count for a given taxon and the sample’s total sequence count, respectively. pT_i and pC_i are the relative abundances of the taxon in the corresponding samples. To account for measurement uncertainty of low abundant taxa, the positive control dataset was used to build a model of variance of log10 relative abundances of each taxon as a function of its median log10 relative abundances to model pT and pC variables as samples from some true (but unknown) relative abundance tpT and tpC, respectively, and as follows:</offsets><disp-formula id="Equd"><alternatives><tex-math id="M7"><offsets xml_i="65825" xml_f="66230" txt_i="44756" txt_f="45161">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$\mathrm{log}\,10({\rm{pT}}\_{\rm{i}}) \sim {\rm{norm}}(\mathrm{log}\,10({\rm{tpT}}\_{\rm{i}}),\,{\rm{sigmaT}}\_{\rm{i}})$$\end{document}</offsets></tex-math><mml:math id="M8" display="block"><mml:mi><offsets xml_i="66283" xml_f="66286" txt_i="45161" txt_f="45164">log</offsets></mml:mi><mml:mspace width=".10em"></mml:mspace><mml:mn><offsets xml_i="66342" xml_f="66344" txt_i="45164" txt_f="45166">10</offsets></mml:mn><mml:mo stretchy="false"><offsets xml_i="66378" xml_f="66379" txt_i="45166" txt_f="45167">(</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="66417" xml_f="66419" txt_i="45167" txt_f="45169">pT</offsets></mml:mi><mml:mo><offsets xml_i="66436" xml_f="66437" txt_i="45169" txt_f="45170">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="66475" xml_f="66476" txt_i="45170" txt_f="45171">i</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="66510" xml_f="66511" txt_i="45171" txt_f="45172">)</offsets></mml:mo><mml:mo><offsets xml_i="66528" xml_f="66529" txt_i="45172" txt_f="45173">~</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="66567" xml_f="66571" txt_i="45173" txt_f="45177">norm</offsets></mml:mi><mml:mrow><mml:mo stretchy="true"><offsets xml_i="66614" xml_f="66615" txt_i="45177" txt_f="45178">(</offsets></mml:mo><mml:mrow><mml:mi><offsets xml_i="66642" xml_f="66645" txt_i="45178" txt_f="45181">log</offsets></mml:mi><mml:mspace width=".10em"></mml:mspace><mml:mn><offsets xml_i="66701" xml_f="66703" txt_i="45181" txt_f="45183">10</offsets></mml:mn><mml:mo stretchy="false"><offsets xml_i="66737" xml_f="66738" txt_i="45183" txt_f="45184">(</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="66776" xml_f="66779" txt_i="45184" txt_f="45187">tpT</offsets></mml:mi><mml:mo><offsets xml_i="66796" xml_f="66797" txt_i="45187" txt_f="45188">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="66835" xml_f="66836" txt_i="45188" txt_f="45189">i</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="66870" xml_f="66871" txt_i="45189" txt_f="45190">)</offsets></mml:mo><mml:mo><offsets xml_i="66888" xml_f="66889" txt_i="45190" txt_f="45191">,</offsets></mml:mo><mml:mspace width=".25em"></mml:mspace><mml:mi mathvariant="normal"><offsets xml_i="66966" xml_f="66972" txt_i="45191" txt_f="45197">sigmaT</offsets></mml:mi><mml:mo><offsets xml_i="66989" xml_f="66990" txt_i="45197" txt_f="45198">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="67028" xml_f="67029" txt_i="45198" txt_f="45199">i</offsets></mml:mi></mml:mrow><mml:mo stretchy="true"><offsets xml_i="67073" xml_f="67074" txt_i="45199" txt_f="45200">)</offsets></mml:mo></mml:mrow></mml:math><graphic xlink:href="41598_2018_26313_Article_Equd.gif" position="anchor"></graphic></alternatives></disp-formula><disp-formula id="Eque"><alternatives><tex-math id="M9"><offsets xml_i="67275" xml_f="67680" txt_i="45200" txt_f="45605">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$\mathrm{log}\,10({\rm{pC}}\_{\rm{i}}) \sim {\rm{norm}}(\mathrm{log}\,10({\rm{tpC}}\_{\rm{i}}),\,{\rm{sigmaC}}\_{\rm{i}})$$\end{document}</offsets></tex-math><mml:math id="M10" display="block"><mml:mi><offsets xml_i="67734" xml_f="67737" txt_i="45605" txt_f="45608">log</offsets></mml:mi><mml:mspace width=".10em"></mml:mspace><mml:mn><offsets xml_i="67793" xml_f="67795" txt_i="45608" txt_f="45610">10</offsets></mml:mn><mml:mo stretchy="false"><offsets xml_i="67829" xml_f="67830" txt_i="45610" txt_f="45611">(</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="67868" xml_f="67870" txt_i="45611" txt_f="45613">pC</offsets></mml:mi><mml:mo><offsets xml_i="67887" xml_f="67888" txt_i="45613" txt_f="45614">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="67926" xml_f="67927" txt_i="45614" txt_f="45615">i</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="67961" xml_f="67962" txt_i="45615" txt_f="45616">)</offsets></mml:mo><mml:mo><offsets xml_i="67979" xml_f="67980" txt_i="45616" txt_f="45617">~</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="68018" xml_f="68022" txt_i="45617" txt_f="45621">norm</offsets></mml:mi><mml:mrow><mml:mo stretchy="true"><offsets xml_i="68065" xml_f="68066" txt_i="45621" txt_f="45622">(</offsets></mml:mo><mml:mrow><mml:mi><offsets xml_i="68093" xml_f="68096" txt_i="45622" txt_f="45625">log</offsets></mml:mi><mml:mspace width=".10em"></mml:mspace><mml:mn><offsets xml_i="68152" xml_f="68154" txt_i="45625" txt_f="45627">10</offsets></mml:mn><mml:mo stretchy="false"><offsets xml_i="68188" xml_f="68189" txt_i="45627" txt_f="45628">(</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="68227" xml_f="68230" txt_i="45628" txt_f="45631">tpC</offsets></mml:mi><mml:mo><offsets xml_i="68247" xml_f="68248" txt_i="45631" txt_f="45632">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="68286" xml_f="68287" txt_i="45632" txt_f="45633">i</offsets></mml:mi><mml:mo stretchy="false"><offsets xml_i="68321" xml_f="68322" txt_i="45633" txt_f="45634">)</offsets></mml:mo><mml:mo><offsets xml_i="68339" xml_f="68340" txt_i="45634" txt_f="45635">,</offsets></mml:mo><mml:mspace width=".25em"></mml:mspace><mml:mi mathvariant="normal"><offsets xml_i="68417" xml_f="68423" txt_i="45635" txt_f="45641">sigmaC</offsets></mml:mi><mml:mo><offsets xml_i="68440" xml_f="68441" txt_i="45641" txt_f="45642">_</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="68479" xml_f="68480" txt_i="45642" txt_f="45643">i</offsets></mml:mi></mml:mrow><mml:mo stretchy="true"><offsets xml_i="68524" xml_f="68525" txt_i="45643" txt_f="45644">)</offsets></mml:mo></mml:mrow></mml:math><graphic xlink:href="41598_2018_26313_Article_Eque.gif" position="anchor"></graphic></alternatives></disp-formula><offsets xml_i="68670" xml_f="68888" txt_i="45644" txt_f="45862">where sigmaT_i and sigmaC_i are the standard deviations of the log10 relative abundances corresponding to log10(pT_i) and log10(pC_i) values, respectively. This model estimates the mean of log ratios log10(tpT_i/tpC_i)</offsets><sup><xref ref-type="bibr" rid="CR64"><offsets xml_i="68926" xml_f="68928" txt_i="45862" txt_f="45864">64</offsets></xref></sup><offsets xml_i="68941" xml_f="69105" txt_i="45864" txt_f="46028">. For each taxon and each condition, 3 chains with 10,000 iterations and thinning of 10 were run. Convergence was tested using potential scale reduction statistics.</offsets></p><p id="Par34"><offsets xml_i="69123" xml_f="69183" txt_i="46029" txt_f="46089">All models were implemented using rstan (v 2.16.2) R package</offsets><sup><xref ref-type="bibr" rid="CR65"><offsets xml_i="69221" xml_f="69223" txt_i="46089" txt_f="46091">65</offsets></xref></sup><offsets xml_i="69236" xml_f="69391" txt_i="46091" txt_f="46246">.To account for multiple testing, p-values were adjusted using false discovery rates and statistical significance level was set at q-value = 0.01 (1% FDR).</offsets></p></sec></sec><sec id="Sec20"><title><offsets xml_i="69430" xml_f="69447" txt_i="46249" txt_f="46266">Data Availability</offsets></title><p id="Par35"><offsets xml_i="69469" xml_f="69575" txt_i="46267" txt_f="46373">The data produced and analysed during this study are available from the corresponding author upon request.</offsets></p></sec></sec><sec sec-type="supplementary-material"><title><offsets xml_i="69637" xml_f="69670" txt_i="46376" txt_f="46409">Electronic supplementary material</offsets></title><sec id="Sec21"><p><offsets xml_i="69697" xml_f="69698" txt_i="46410" txt_f="46411">
</offsets><supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="41598_2018_26313_MOESM1_ESM.docx"><caption><p><offsets xml_i="69825" xml_f="69845" txt_i="46411" txt_f="46431">Supplemental Figures</offsets></p></caption></media></supplementary-material><offsets xml_i="69892" xml_f="69893" txt_i="46432" txt_f="46433">
</offsets><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="41598_2018_26313_MOESM2_ESM.xlsx"><caption><p><offsets xml_i="70020" xml_f="70036" txt_i="46433" txt_f="46449">Dataset Table S1</offsets></p></caption></media></supplementary-material><offsets xml_i="70083" xml_f="70084" txt_i="46450" txt_f="46451">
</offsets><supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="41598_2018_26313_MOESM3_ESM.xlsx"><caption><p><offsets xml_i="70211" xml_f="70227" txt_i="46451" txt_f="46467">Dataset Table S2</offsets></p></caption></media></supplementary-material><offsets xml_i="70274" xml_f="70275" txt_i="46468" txt_f="46469">
</offsets><supplementary-material content-type="local-data" id="MOESM4"><media xlink:href="41598_2018_26313_MOESM4_ESM.xlsx"><caption><p><offsets xml_i="70402" xml_f="70418" txt_i="46469" txt_f="46485">Dataset Table S3</offsets></p></caption></media></supplementary-material><offsets xml_i="70465" xml_f="70466" txt_i="46486" txt_f="46487">
</offsets></p></sec></sec></body><back><fn-group><fn><p><bold>Electronic supplementary material</bold></p><p><bold>Supplementary information</bold> accompanies this paper at 10.1038/s41598-018-26313-8.</p></fn><fn><p><bold>Publisher's note:</bold> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></fn></fn-group><ack><title>Acknowledgements</title><p>This project was supported by a grant from the National Institutes of Health (grant #: 1U19AI113182–01). The findings and conclusions in this paper are those of the authors and do not necessarily represent the views of their sponsoring institutions.</p></ack><notes notes-type="author-contribution"><title>Author Contributions</title><p>L.G. and K.B. performed proteomic analysis, interpreted data, and L.G. wrote first draft of manuscript. L.R. assisted in proteomic data analysis. P.G., J.B.H. and M.S.H. performed microbiome analysis. S.M., G.W., and M.A. performed liquid chromatography and mass spectrometry analysis. L.W. and L.C.R. provided gels used for the study. D.G., P.G. and J.M. performed non-human primate experiments. K.E.P. and N.M. provided overall study conception and manuscript editing. A.D.B. and J.R. helped with study conception and proteomic/microbiome data analysis. L.G., K.B., A.D.B., J.B.H. and J.R. contributed to writing the manuscript.</p></notes><notes notes-type="COI-statement"><sec id="FPar1"><title>Competing Interests</title><p id="Par37">The authors declare no competing interests.</p></sec></notes><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Patel</surname><given-names>P</given-names></name><etal></etal></person-group><article-title>Estimating per-act HIV transmission risk: a systematic review</article-title><source>AIDS</source><year>2014</year><volume>28</volume><fpage>1</fpage><lpage>11</lpage><pub-id pub-id-type="doi">10.1097/QAD.0000000000000298</pub-id><pub-id pub-id-type="pmid">23939240</pub-id></element-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Beyrer</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>Global epidemiology of HIV infection in men who have sex with men</article-title><source>Lancet</source><year>2012</year><volume>380</volume><fpage>367</fpage><lpage>377</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(12)60821-6</pub-id><pub-id pub-id-type="pmid">22819660</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Beyrer</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>The increase in global HIV epidemics in MSM</article-title><source>AIDS</source><year>2013</year><volume>27</volume><fpage>2665</fpage><lpage>2678</lpage><pub-id pub-id-type="doi">10.1097/01.aids.0000432449.30239.fe</pub-id><pub-id pub-id-type="pmid">23842129</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><mixed-citation publication-type="other">UN Joint Programme on HIV/AIDS (UNAIDS). <italic>Global AIDS update 2016</italic>. 1–13 (2016).</mixed-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Baral</surname><given-names>SD</given-names></name><etal></etal></person-group><article-title>Worldwide burden of HIV in transgender women: a systematic review and meta-analysis</article-title><source>Lancet Infect Dis</source><year>2012</year><volume>13</volume><fpage>214</fpage><lpage>222</lpage><pub-id pub-id-type="doi">10.1016/S1473-3099(12)70315-8</pub-id><pub-id pub-id-type="pmid">23260128</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mannava</surname><given-names>P</given-names></name><name><surname>Geibel</surname><given-names>S</given-names></name><name><surname>King’ola</surname><given-names>N</given-names></name><name><surname>Temmerman</surname><given-names>M</given-names></name><name><surname>Luchters</surname><given-names>S</given-names></name></person-group><article-title>Male Sex Workers Who Sell Sex to Men Also Engage in Anal Intercourse with Women: Evidence from Mombasa, Kenya</article-title><source>PLoS ONE</source><year>2013</year><volume>8</volume><fpage>e52547</fpage><pub-id pub-id-type="doi">10.1371/journal.pone.0052547</pub-id><pub-id pub-id-type="pmid">23300978</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hess</surname><given-names>KL</given-names></name><name><surname>Crepaz</surname><given-names>N</given-names></name><name><surname>Rose</surname><given-names>C</given-names></name><name><surname>Purcell</surname><given-names>D</given-names></name><name><surname>Paz-Bailey</surname><given-names>G</given-names></name></person-group><article-title>Trends in Sexual Behavior Among Men Who have Sex with Men (MSM) in High-Income Countries, 1990–2013: A Systematic Review</article-title><source>AIDS and Behavior</source><year>2017</year><volume>21</volume><fpage>2811</fpage><lpage>2834</lpage><pub-id pub-id-type="doi">10.1007/s10461-017-1799-1</pub-id><pub-id pub-id-type="pmid">28555317</pub-id></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Semple</surname><given-names>SJ</given-names></name><etal></etal></person-group><article-title>Correlates of condomless anal sex among men who have sex with men (MSM) in Tijuana, Mexico: The role of public sex venues</article-title><source>PLoS ONE</source><year>2017</year><volume>12</volume><fpage>e0186814</fpage><lpage>16</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0186814</pub-id><pub-id pub-id-type="pmid">29065132</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lane</surname><given-names>T</given-names></name><name><surname>Shade</surname><given-names>SB</given-names></name><name><surname>McIntyre</surname><given-names>J</given-names></name><name><surname>Morin</surname><given-names>SF</given-names></name></person-group><article-title>Alcohol and Sexual Risk Behavior Among Men Who Have Sex with Men in South African Township Communities</article-title><source>AIDS and Behavior</source><year>2008</year><volume>12</volume><fpage>78</fpage><lpage>85</lpage><pub-id pub-id-type="doi">10.1007/s10461-008-9389-x</pub-id><pub-id pub-id-type="pmid">17295075</pub-id></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Yi</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Factors Associated with Inconsistent Condom Use among Men Who Have Sex with Men in Cambodia</article-title><source>PLoS ONE</source><year>2015</year><volume>10</volume><fpage>e0136114</fpage><lpage>15</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0136114</pub-id><pub-id pub-id-type="pmid">26287731</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cai</surname><given-names>Y</given-names></name><etal></etal></person-group><article-title>Prevalence and associated factors of condomless receptive anal intercourse with male clients among transgender women sex workers in Shenyang, China</article-title><source>Journal of the International AIDS Society</source><year>2016</year><volume>19</volume><fpage>210</fpage><lpage>9</lpage><pub-id pub-id-type="doi">10.7448/IAS.19.3.20800</pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Herbst</surname><given-names>JH</given-names></name><etal></etal></person-group><article-title>Estimating HIV Prevalence and Risk Behaviors of Transgender Persons in the United States: A Systematic Review</article-title><source>AIDS and Behavior</source><year>2007</year><volume>12</volume><fpage>1</fpage><lpage>17</lpage><pub-id pub-id-type="doi">10.1007/s10461-007-9299-3</pub-id><pub-id pub-id-type="pmid">17694429</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Reisner</surname><given-names>SL</given-names></name><etal></etal></person-group><article-title>Laboratory-confirmed HIV and sexually transmitted infection seropositivity and risk behavior among sexually active transgender patients at an adolescent and young adult urban community health center</article-title><source>AIDS Care</source><year>2015</year><volume>27</volume><fpage>1031</fpage><lpage>1036</lpage><pub-id pub-id-type="doi">10.1080/09540121.2015.1020750</pub-id><pub-id pub-id-type="pmid">25790139</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>van Loggerenberg</surname><given-names>F</given-names></name><etal></etal></person-group><article-title>HIV Prevention in High-Risk Women in South Africa: Condom Use and the Need for Change</article-title><source>PLoS ONE</source><year>2012</year><volume>7</volume><fpage>e30669</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0030669</pub-id><pub-id pub-id-type="pmid">22363467</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Chopra</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Estimating HIV Prevalence and Risk Behaviors Among High-Risk Heterosexual Men With Multiple Sex Partners:</article-title><source>Use of Respondent-Driven Sampling.</source><year>2009</year><volume>51</volume><fpage>72</fpage><lpage>77</lpage></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lurie</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Sexual Behavior and Reproductive Health Among HIV-Infected Patients in Urban and Rural South Africa</article-title><source>JAIDS</source><year>2008</year><volume>47</volume><fpage>484</fpage><lpage>493</lpage><pub-id pub-id-type="pmid">18209685</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Carballo-Diéguez</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Frequent Use of Lubricants for Anal Sex Among Men Who Have Sex With Men: The HIV Prevention Potential of a Microbicidal Gel</article-title><source>Am J Public Health</source><year>2000</year><volume>90</volume><fpage>1117</fpage><lpage>1121</lpage><pub-id pub-id-type="doi">10.2105/AJPH.90.7.1117</pub-id><pub-id pub-id-type="pmid">10897191</pub-id></element-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Van Damme</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Effectiveness of COL-1492, a nonoxynol-9 vaginal gel, on HIV-1 transmission in female sex workers: a randomised controlled trial</article-title><source>Lancet</source><year>2002</year><volume>360</volume><fpage>971</fpage><lpage>977</lpage><pub-id pub-id-type="doi">10.1016/S0140-6736(02)11079-8</pub-id><pub-id pub-id-type="pmid">12383665</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tabet</surname><given-names>SR</given-names></name><etal></etal></person-group><article-title>Safety and Toxicity of Nonoxynol-9 Gel As a Rectal Microbicide</article-title><source>Sex Transm Dis</source><year>1999</year><volume>26</volume><fpage>564</fpage><lpage>571</lpage><pub-id pub-id-type="doi">10.1097/00007435-199911000-00005</pub-id><pub-id pub-id-type="pmid">10560720</pub-id></element-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Masson</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Genital Inflammation and the Risk of HIV Acquisition in Women</article-title><source>Clin Infect Dis.</source><year>2015</year><volume>61</volume><fpage>260</fpage><lpage>269</lpage><pub-id pub-id-type="doi">10.1093/cid/civ298</pub-id><pub-id pub-id-type="pmid">25900168</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ravel</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Twice-daily application of HIV microbicides alter the vaginal microbiota</article-title><source>mBio</source><year>2012</year><volume>3</volume><fpage>e00370</fpage><lpage>12</lpage><pub-id pub-id-type="doi">10.1128/mBio.00370-12</pub-id><pub-id pub-id-type="pmid">23249810</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Taha</surname><given-names>TE</given-names></name><etal></etal></person-group><article-title>Bacterial vaginosis and disturbances of vaginal flora: association with increased acquisition of HIV</article-title><source>AIDS</source><year>1998</year><volume>12</volume><fpage>1</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1097/00002030-199801000-00001</pub-id><pub-id pub-id-type="pmid">9456249</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cohen</surname><given-names>CR</given-names></name><etal></etal></person-group><article-title>Bacterial Vaginosis Associated with Increased Risk of Female-to-Male HIV-1 Transmission: A Prospective Cohort Analysis among African Couples</article-title><source>PLoS Med</source><year>2012</year><volume>9</volume><fpage>e1001251</fpage><lpage>9</lpage><pub-id pub-id-type="doi">10.1371/journal.pmed.1001251</pub-id><pub-id pub-id-type="pmid">22745608</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>McGowan</surname><given-names>I</given-names></name></person-group><article-title>The development of rectal microbicides for HIV prevention</article-title><source>Expert Opinion on Drug Delivery</source><year>2013</year><volume>11</volume><fpage>69</fpage><lpage>82</lpage><pub-id pub-id-type="doi">10.1517/17425247.2013.860132</pub-id><pub-id pub-id-type="pmid">24266648</pub-id></element-citation></ref><ref id="CR25"><label>25.</label><mixed-citation publication-type="other">Hladik, F. <italic>et al</italic>. Mucosal effects of tenofovir 1% gel. <italic>eLife</italic> 1–21, 10.7554/eLife.04525.001 (2015).</mixed-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Romas</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Rectal 1% Tenofovir Gel Use Associates with Altered Epidermal Protein Expression</article-title><source>AIDS Res Hum Retroviruses</source><year>2016</year><volume>32</volume><fpage>1005</fpage><lpage>1015</lpage><pub-id pub-id-type="doi">10.1089/aid.2015.0381</pub-id><pub-id pub-id-type="pmid">27316778</pub-id></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Rohan</surname><given-names>LC</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>L</given-names></name></person-group><article-title>Rectal pre-exposure prophylaxis (PrEP)</article-title><source>Antiviral Research</source><year>2013</year><volume>100</volume><fpage>S17</fpage><lpage>S24</lpage><pub-id pub-id-type="doi">10.1016/j.antiviral.2013.09.023</pub-id><pub-id pub-id-type="pmid">24188705</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Begay</surname><given-names>O</given-names></name><etal></etal></person-group><article-title>Identification of Personal Lubricants That Can Cause Rectal Epithelial Cell Damage and Enhance HIV Type 1 Replication <italic>in Vitro</italic></article-title><source>AIDS Res Hum Retroviruses</source><year>2011</year><volume>27</volume><fpage>1019</fpage><lpage>1024</lpage><pub-id pub-id-type="doi">10.1089/aid.2010.0252</pub-id><pub-id pub-id-type="pmid">21309617</pub-id></element-citation></ref><ref id="CR29"><label>29.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Fuchs</surname><given-names>EJ</given-names></name><etal></etal></person-group><article-title>Hyperosmolar Sexual Lubricant Causes Epithelial Damage in the Distal Colon: Potential Implication for HIV Transmission</article-title><source>J Infect Dis</source><year>2007</year><volume>195</volume><fpage>703</fpage><lpage>710</lpage><pub-id pub-id-type="doi">10.1086/511279</pub-id><pub-id pub-id-type="pmid">17262713</pub-id></element-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Anton</surname><given-names>PA</given-names></name><etal></etal></person-group><article-title>RMP-02/MTN-006: A Phase 1 Rectal Safety, Acceptability, Pharmacokinetic, and Pharmacodynamic Study of Tenofovir 1% Gel Compared with Oral Tenofovir Disoproxil Fumarate</article-title><source>AIDS Res Hum Retroviruses</source><year>2012</year><volume>28</volume><fpage>1412</fpage><lpage>1421</lpage><pub-id pub-id-type="doi">10.1089/aid.2012.0262</pub-id><pub-id pub-id-type="pmid">22943559</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>McGowan</surname><given-names>I</given-names></name><etal></etal></person-group><article-title>A Phase 1 Randomized, Double Blind, Placebo Controlled Rectal Safety and Acceptability Study of Tenofovir 1% Gel (MTN-007)</article-title><source>PLoS ONE</source><year>2013</year><volume>8</volume><fpage>e60147</fpage><lpage>9</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0060147</pub-id><pub-id pub-id-type="pmid">23573238</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kouokam</surname><given-names>JC</given-names></name><etal></etal></person-group><article-title>Investigation of Griffithsin’s Interactions with Human Cells Confirms Its Outstanding Safety and Efficacy Profile as a Microbicide Candidate</article-title><source>PLoS ONE</source><year>2011</year><volume>6</volume><fpage>e22635</fpage><pub-id pub-id-type="doi">10.1371/journal.pone.0022635</pub-id><pub-id pub-id-type="pmid">21829638</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Nixon</surname><given-names>B</given-names></name><etal></etal></person-group><article-title>Griffithsin Protects Mice from Genital Herpes by Preventing Cell-to-Cell Spread</article-title><source>J Virol</source><year>2013</year><volume>87</volume><fpage>6257</fpage><lpage>6269</lpage><pub-id pub-id-type="doi">10.1128/JVI.00012-13</pub-id><pub-id pub-id-type="pmid">23536670</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mori</surname><given-names>T</given-names></name><etal></etal></person-group><article-title>Isolation and Characterization of Griffithsin, a Novel HIV-inactivating Protein, from the Red Alga Griffithsia sp</article-title><source>Journal of Biological Chemistry</source><year>2005</year><volume>280</volume><fpage>9345</fpage><lpage>9353</lpage><pub-id pub-id-type="doi">10.1074/jbc.M411122200</pub-id><pub-id pub-id-type="pmid">15613479</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>O’Keefe</surname><given-names>BR</given-names></name><etal></etal></person-group><article-title>Scaleable manufacture of HIV-1 entry inhibitor griffithsin and validation of its safety and efficacy as a topical microbicide component</article-title><source>Proc Natl Acad Sci</source><year>2009</year><volume>106</volume><fpage>6099</fpage><lpage>6104</lpage><pub-id pub-id-type="doi">10.1073/pnas.0901506106</pub-id><pub-id pub-id-type="pmid">19332801</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Kouokam</surname><given-names>J</given-names></name><name><surname>Lasnik</surname><given-names>A</given-names></name><name><surname>Palmer</surname><given-names>K</given-names></name></person-group><article-title>Studies in a Murine Model Confirm the Safety of Griffithsin and Advocate Its Further Development as a Microbicide Targeting HIV-1 and Other Enveloped Viruses</article-title><source>Viruses</source><year>2016</year><volume>8</volume><fpage>311</fpage><pub-id pub-id-type="doi">10.3390/v8110311</pub-id></element-citation></ref><ref id="CR37"><label>37.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Zevin</surname><given-names>AS</given-names></name><etal></etal></person-group><article-title>Microbiome Composition and Function Drives Wound-Healing Impairment in the Female Genital Tract</article-title><source>PLoS Pathog</source><year>2016</year><volume>12</volume><fpage>e1005889</fpage><lpage>20</lpage><pub-id pub-id-type="doi">10.1371/journal.ppat.1005889</pub-id><pub-id pub-id-type="pmid">27656899</pub-id></element-citation></ref><ref id="CR38"><label>38.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Arnold</surname><given-names>KB</given-names></name><etal></etal></person-group><article-title>Increased levels of inflammatory cytokines in the female reproductive tract are associated with altered expression of proteases, mucosal barrier proteins, and an influx of HIV-susceptible target cells</article-title><source>Mucosal Immunol</source><year>2016</year><volume>9</volume><fpage>194</fpage><lpage>205</lpage><pub-id pub-id-type="doi">10.1038/mi.2015.51</pub-id><pub-id pub-id-type="pmid">26104913</pub-id></element-citation></ref><ref id="CR39"><label>39.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Birse</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>Molecular Signatures of Immune Activation and Epithelial Barrier Remodeling Are Enhanced during the Luteal Phase of the Menstrual Cycle: Implications for HIV Susceptibility</article-title><source>J Virol</source><year>2015</year><volume>89</volume><fpage>8793</fpage><lpage>8805</lpage><pub-id pub-id-type="doi">10.1128/JVI.00756-15</pub-id><pub-id pub-id-type="pmid">26085144</pub-id></element-citation></ref><ref id="CR40"><label>40.</label><mixed-citation publication-type="other">Kelley, C. F. <italic>et al</italic>. The rectal mucosa and condomless receptive anal intercourse in HIV-negative MSM: implications for HIV transmission and prevention. <italic>Mucosal Immunol</italic> 1–12, 10.1038/mi.2016.97 (2016).</mixed-citation></ref><ref id="CR41"><label>41.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mitchell</surname><given-names>CA</given-names></name><name><surname>Ramessar</surname><given-names>K</given-names></name><name><surname>O’Keefe</surname><given-names>BR</given-names></name></person-group><article-title>Antiviral lectins: Selective inhibitors of viral entry</article-title><source>Antiviral Research</source><year>2017</year><volume>142</volume><fpage>37</fpage><lpage>54</lpage><pub-id pub-id-type="doi">10.1016/j.antiviral.2017.03.007</pub-id><pub-id pub-id-type="pmid">28322922</pub-id></element-citation></ref><ref id="CR42"><label>42.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Lis</surname><given-names>H</given-names></name><name><surname>Sharon</surname><given-names>N</given-names></name></person-group><article-title>Lectins: Cell-Agglutinating and Sugar-Specific Proteins</article-title><source>Science</source><year>1972</year><volume>177</volume><fpage>949</fpage><lpage>959</lpage><pub-id pub-id-type="doi">10.1126/science.177.4053.949</pub-id><pub-id pub-id-type="pmid">5055944</pub-id></element-citation></ref><ref id="CR43"><label>43.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Ashraf</surname><given-names>MT</given-names></name><name><surname>Khan</surname><given-names>RH</given-names></name></person-group><article-title>Mitogenic Lectins</article-title><source>Med Sci Monit</source><year>2003</year><volume>9</volume><fpage>RA265</fpage><lpage>269</lpage><pub-id pub-id-type="pmid">14586291</pub-id></element-citation></ref><ref id="CR44"><label>44.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Vishwanathan</surname><given-names>SA</given-names></name><etal></etal></person-group><article-title>Rectal Application of a Highly Osmolar Personal Lubricant in a Macaque Model Induces Acute Cytotoxicity but Does Not Increase Risk of SHIV Infection</article-title><source>PLoS ONE</source><year>2015</year><volume>10</volume><fpage>e0120021</fpage><lpage>17</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0120021</pub-id><pub-id pub-id-type="pmid">25853710</pub-id></element-citation></ref><ref id="CR45"><label>45.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Anton</surname><given-names>PA</given-names></name><etal></etal></person-group><article-title>First Phase 1 Double-Blind, Placebo-Controlled, Randomized Rectal Microbicide Trial Using UC781 Gel with a Novel Index of <italic>Ex Vivo</italic> Efficacy</article-title><source>PLoS ONE</source><year>2011</year><volume>6</volume><fpage>e23243</fpage><lpage>15</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0023243</pub-id><pub-id pub-id-type="pmid">21969851</pub-id></element-citation></ref><ref id="CR46"><label>46.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Richardson</surname><given-names>BA</given-names></name><etal></etal></person-group><article-title>Appropriateness of Hydroxyethylcellulose Gel as a Placebo Control in Vaginal MicrobicideTrials</article-title><source>JAIDS</source><year>2013</year><volume>63</volume><fpage>120</fpage><lpage>125</lpage><pub-id pub-id-type="pmid">23334506</pub-id></element-citation></ref><ref id="CR47"><label>47.</label><mixed-citation publication-type="other">Tien, D. <italic>et al</italic>. <italic>In vitro</italic> and <italic>In vivo</italic> Characterization of a Potential Universal Placebo Designed for Use in Vaginal Microbicide Clinical Trials. 1–9 (2005).</mixed-citation></ref><ref id="CR48"><label>48.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Nixon</surname><given-names>B</given-names></name><etal></etal></person-group><article-title>Vaginally Delivered Tenofovir Disoproxil Fumarate Provides Greater Protection than Tenofovir against Genital Herpes in a Murine Model of Efficacy and Safety</article-title><source>Antimicrob Agents Chemother</source><year>2014</year><volume>58</volume><fpage>1153</fpage><lpage>1160</lpage><pub-id pub-id-type="doi">10.1128/AAC.01818-13</pub-id><pub-id pub-id-type="pmid">24323471</pub-id></element-citation></ref><ref id="CR49"><label>49.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Smith-McCune</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>Unexpected Inflammatory Effects of Intravaginal Gels (Universal Placebo Gel and Nonoxynol-9) on the Upper Female Reproductive Tract: A Randomized Crossover Study</article-title><source>PLoS ONE</source><year>2015</year><volume>10</volume><fpage>e0129769</fpage><pub-id pub-id-type="doi">10.1371/journal.pone.0129769</pub-id><pub-id pub-id-type="pmid">26177352</pub-id></element-citation></ref><ref id="CR50"><label>50.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Schwartz</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>A Randomized Six-Day Safety Study of an Antiretroviral Microbicide Candidate UC781, a Non-Nucleoside Reverse Transcriptase Inhibitor</article-title><source>Sex Transm Dis</source><year>2008</year><volume>35</volume><fpage>414</fpage><lpage>419</lpage><pub-id pub-id-type="doi">10.1097/OLQ.0b013e318162c4d8</pub-id><pub-id pub-id-type="pmid">18362865</pub-id></element-citation></ref><ref id="CR51"><label>51.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tabet</surname><given-names>SR</given-names></name><etal></etal></person-group><article-title>Safety and Acceptability of Penile Application of 2 Candidate Topical Microbicides: BufferGel and PRO 2000 Gel</article-title><source>JAIDS</source><year>2003</year><volume>33</volume><fpage>476</fpage><lpage>483</lpage><pub-id pub-id-type="pmid">12869836</pub-id></element-citation></ref><ref id="CR52"><label>52.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mayer</surname><given-names>KH</given-names></name><etal></etal></person-group><article-title>Safety and Tolerability of BufferGel, a Novel Vaginal Microbicide, in Women in the United States</article-title><source>Clin Infect Dis.</source><year>2001</year><volume>32</volume><fpage>476</fpage><lpage>482</lpage><pub-id pub-id-type="doi">10.1086/318496</pub-id><pub-id pub-id-type="pmid">11170957</pub-id></element-citation></ref><ref id="CR53"><label>53.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Patton</surname><given-names>DL</given-names></name><etal></etal></person-group><article-title>Safety and Efficacy Evaluations for Vaginal and Rectal Use of BufferGel in the Macaque Model</article-title><source>Sex Transm Dis</source><year>2004</year><volume>31</volume><fpage>290</fpage><lpage>296</lpage><pub-id pub-id-type="doi">10.1097/01.OLQ.0000124614.91448.D4</pub-id><pub-id pub-id-type="pmid">15107631</pub-id></element-citation></ref><ref id="CR54"><label>54.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Miquel</surname><given-names>S</given-names></name><etal></etal></person-group><article-title>Ecology and metabolism of the beneficial intestinal commensal bacterium Faecalibacterium prausnitzii</article-title><source>Gut Microbes</source><year>2014</year><volume>5</volume><fpage>146</fpage><lpage>151</lpage><pub-id pub-id-type="doi">10.4161/gmic.27651</pub-id><pub-id pub-id-type="pmid">24637606</pub-id></element-citation></ref><ref id="CR55"><label>55.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Noguera-Julian</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Gut Microbiota Linked to Sexual Preference and HIVInfection</article-title><source>EBIOM</source><year>2016</year><volume>5</volume><fpage>135</fpage><lpage>146</lpage><pub-id pub-id-type="doi">10.1016/j.ebiom.2016.01.032</pub-id></element-citation></ref><ref id="CR56"><label>56.</label><mixed-citation publication-type="other">Miquel, S. <italic>et al</italic>. Anti-nociceptive effect of Faecalibacterium prausnitzii in non-inflammatory IBS-like models. <italic>Scientific Reports</italic> 1–8, 10.1038/srep19399 (2016).</mixed-citation></ref><ref id="CR57"><label>57.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Romas</surname><given-names>LM</given-names></name><etal></etal></person-group><article-title>A Comparative Proteomic Analysis of the Soluble Immune Factor Environment of Rectal and Oral Mucosa</article-title><source>PLoS ONE</source><year>2014</year><volume>9</volume><fpage>e100820</fpage><lpage>10</lpage><pub-id pub-id-type="doi">10.1371/journal.pone.0100820</pub-id><pub-id pub-id-type="pmid">24978053</pub-id></element-citation></ref><ref id="CR58"><label>58.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Fadrosh</surname><given-names>DW</given-names></name><etal></etal></person-group><article-title>An improved dual-indexing approach for multiplexed 16S rRNA gene sequencing on the Illumina MiSeq platform</article-title><source>Microbiome</source><year>2014</year><volume>2</volume><fpage>1</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1186/2049-2618-2-6</pub-id><pub-id pub-id-type="pmid">24468033</pub-id></element-citation></ref><ref id="CR59"><label>59.</label><mixed-citation publication-type="other">Kuczynski, J. <italic>et al</italic>. In <italic>Current protocols in microbiology</italic><bold>474</bold>, Unit 1E.5. (John Wiley &amp; Sons, Inc., 2012).</mixed-citation></ref><ref id="CR60"><label>60.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Callahan</surname><given-names>BJ</given-names></name><etal></etal></person-group><article-title>DADA2: High-resolution sample inference from Illumina amplicon data</article-title><source>Nat Methods</source><year>2016</year><volume>13</volume><fpage>581</fpage><lpage>583</lpage><pub-id pub-id-type="doi">10.1038/nmeth.3869</pub-id><pub-id pub-id-type="pmid">27214047</pub-id></element-citation></ref><ref id="CR61"><label>61.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Quast</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>The SILVA ribosomal RNA gene database project: improved data processing and web-based tools</article-title><source>Nucleic Acids Res</source><year>2012</year><volume>41</volume><fpage>D590</fpage><lpage>D596</lpage><pub-id pub-id-type="doi">10.1093/nar/gks1219</pub-id><pub-id pub-id-type="pmid">23193283</pub-id></element-citation></ref><ref id="CR62"><label>62.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Huang</surname><given-names>DW</given-names></name><name><surname>Sherman</surname><given-names>BT</given-names></name><name><surname>Lempicki</surname><given-names>RA</given-names></name></person-group><article-title>Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</article-title><source>Nucleic Acids Res</source><year>2009</year><volume>37</volume><fpage>1</fpage><lpage>13</lpage><pub-id pub-id-type="doi">10.1093/nar/gkn923</pub-id><pub-id pub-id-type="pmid">19033363</pub-id></element-citation></ref><ref id="CR63"><label>63.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Huang</surname><given-names>DW</given-names></name><name><surname>Sherman</surname><given-names>BT</given-names></name><name><surname>Lempicki</surname><given-names>RA</given-names></name></person-group><article-title>Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</article-title><source>Nat Protoc</source><year>2009</year><volume>4</volume><fpage>44</fpage><lpage>57</lpage><pub-id pub-id-type="doi">10.1038/nprot.2008.211</pub-id><pub-id pub-id-type="pmid">19131956</pub-id></element-citation></ref><ref id="CR64"><label>64.</label><mixed-citation publication-type="other">Congdon, P. D. <italic>Applied Baysian Hierarchial Methods</italic>. (CRC Press, 2010).</mixed-citation></ref><ref id="CR65"><label>65.</label><mixed-citation publication-type="other">Team SD. RStan: the R interface to Stan. R package version 2.14.1. (2016). Available at: httpmc-stan.org. (Accessed: 7 October 2017).</mixed-citation></ref></ref-list></back></article>