<article xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">Oncotarget</journal-id><journal-id journal-id-type="iso-abbrev">Oncotarget</journal-id><journal-id journal-id-type="publisher-id">Oncotarget</journal-id><journal-id journal-id-type="publisher-id">ImpactJ</journal-id><journal-title-group><journal-title>Oncotarget</journal-title></journal-title-group><issn pub-type="epub">1949-2553</issn><publisher><publisher-name>Impact Journals LLC</publisher-name></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">29212194</article-id><article-id pub-id-type="pmc">5706840</article-id><article-id pub-id-type="publisher-id">21291</article-id><article-id pub-id-type="doi">10.18632/oncotarget.21291</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Paper</subject></subj-group></article-categories><title-group><article-title>Association between miR-31-3p expression and cetuximab efficacy in patients with KRAS wild-type metastatic colorectal cancer: a post-hoc analysis of the New EPOC trial</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Pugh</surname><given-names>Siân</given-names></name><xref ref-type="aff" rid="A1"><sup>1</sup></xref><xref ref-type="author-notes" rid="FN1">*</xref></contrib><contrib contrib-type="author"><name><surname>Thiébaut</surname><given-names>Raphaële</given-names></name><xref ref-type="aff" rid="A2"><sup>2</sup></xref><xref ref-type="author-notes" rid="FN1">*</xref></contrib><contrib contrib-type="author"><name><surname>Bridgewater</surname><given-names>John</given-names></name><xref ref-type="aff" rid="A3"><sup>3</sup></xref><xref ref-type="author-notes" rid="FN1">*</xref></contrib><contrib contrib-type="author"><name><surname>Grisoni</surname><given-names>Marie-Lise</given-names></name><xref ref-type="aff" rid="A2"><sup>2</sup></xref></contrib><contrib contrib-type="author"><name><surname>Moutasim</surname><given-names>Karwan</given-names></name><xref ref-type="aff" rid="A4"><sup>4</sup></xref></contrib><contrib contrib-type="author"><name><surname>Rousseau</surname><given-names>Francis</given-names></name><xref ref-type="aff" rid="A2"><sup>2</sup></xref></contrib><contrib contrib-type="author"><name><surname>Thomas</surname><given-names>Gareth J.</given-names></name><xref ref-type="aff" rid="A4"><sup>4</sup></xref></contrib><contrib contrib-type="author"><name><surname>Griffiths</surname><given-names>Gareth</given-names></name><xref ref-type="aff" rid="A5"><sup>5</sup></xref></contrib><contrib contrib-type="author"><name><surname>Liebaert</surname><given-names>François</given-names></name><xref ref-type="aff" rid="A2"><sup>2</sup></xref></contrib><contrib contrib-type="author"><name><surname>Primrose</surname><given-names>John</given-names></name><xref ref-type="aff" rid="A1"><sup>1</sup></xref><xref ref-type="author-notes" rid="FN1">*</xref></contrib><contrib contrib-type="author"><name><surname>Laurent-Puig</surname><given-names>Pierre</given-names></name><xref ref-type="aff" rid="A6"><sup>6</sup></xref><xref ref-type="aff" rid="A7"><sup>7</sup></xref><xref ref-type="author-notes" rid="FN1">*</xref></contrib><aff id="A1"><sup>1</sup> University Surgery, Cancer Sciences, University of Southampton, Southampton, United Kingdom</aff><aff id="A2"><sup>2</sup> IntegraGen SA, Evry, France</aff><aff id="A3"><sup>3</sup> UCL Cancer Institute, London, United Kingdom</aff><aff id="A4"><sup>4</sup> Cancer Sciences Division, University of Southampton, Southampton, United Kingdom</aff><aff id="A5"><sup>5</sup> Southampton Clinical Trials Unit, University of Southampton, Southampton, United Kingdom</aff><aff id="A6"><sup>6</sup> UMR-S1147, University Paris Descartes, Paris, France</aff><aff id="A7"><sup>7</sup> Assistance Publique-Hôpitaux de Paris, Hôpital Georges Pompidou, Paris, France</aff></contrib-group><author-notes><corresp id="cor1"><bold><italic>Correspondence to:</italic></bold><italic>Raphaële Thiébaut,</italic><email>raphaele.thiebaut@integragen.com</email></corresp><fn fn-type="equal" id="FN1"><label>*</label><p>These authors have contributed equally to this work</p></fn></author-notes><pub-date pub-type="collection"><day>7</day><month>11</month><year>2017</year></pub-date><pub-date pub-type="epub"><day>27</day><month>9</month><year>2017</year></pub-date><volume>8</volume><issue>55</issue><fpage>93856</fpage><lpage>93866</lpage><history><date date-type="received"><day>22</day><month>3</month><year>2017</year></date><date date-type="accepted"><day>27</day><month>8</month><year>2017</year></date></history><permissions><copyright-statement>Copyright: © 2017 Pugh et al.</copyright-statement><copyright-year>2017</copyright-year><license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/3.0/"><license-p>This is an open-access article distributed under the terms of the <ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution License</ext-link> 3.0 (CC BY 3.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</license-p></license></permissions><abstract><sec><title><offsets xml_i="5367" xml_f="5377" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="5388" xml_f="5674" txt_i="22" txt_f="308">High miR-31-3p expression is associated with inferior outcomes in KRAS wild-type (WT) advanced colorectal cancer patients treated with anti-EGFR therapy. This study evaluated miR-31-3p expression in patients with operable colorectal liver metastases (LM) enrolled in the New EPOC study.</offsets></p></sec><sec><title><offsets xml_i="5696" xml_f="5703" txt_i="310" txt_f="317">Methods</offsets></title><p><offsets xml_i="5714" xml_f="6067" txt_i="318" txt_f="671">MiR-31-3p expression was measured in primary tumors (PT) from 149 KRAS WT patients including 71 receiving chemotherapy alone (CT) and 78 receiving chemotherapy plus cetuximab (CTX). Each treatment arm was split into tertiles based on miR-31-3p expression levels. MiR-31-3p expression was also measured in LM from 94 patients with tumor tissue available.</offsets></p></sec><sec><title><offsets xml_i="6089" xml_f="6096" txt_i="673" txt_f="680">Results</offsets></title><p><offsets xml_i="6107" xml_f="6536" txt_i="681" txt_f="1110">The median progression-free survival for the combined populations with mid or high miR-31-3p expression was shorter in the CTX versus the CT arm (26.7 months versus 12.3 months, HR=2.28 95%CI 1.27; 4.09 p=0.006). Low miR-31-3p expressers had similar outcomes irrespective of treatment (HR=1.06 95%CI 0.43; 2.61 p=0.9). MiR-31-3p expression was correlated between paired PT and LM samples in the CT group but not in the CTX group.</offsets></p></sec><sec><title><offsets xml_i="6558" xml_f="6569" txt_i="1112" txt_f="1123">Conclusions</offsets></title><p><offsets xml_i="6580" xml_f="6807" txt_i="1124" txt_f="1351">Patients with low miR-31-3p expression in the New EPOC study were not harmed by the addition of cetuximab. This supports miR-31-3p as a promising predictive biomarker for anti-EGFR therapy in KRAS WT advanced colorectal cancer.</offsets></p></sec></abstract><kwd-group><kwd>metastatic colorectal cancer</kwd><kwd>biomarker</kwd><kwd>miR-31-3p</kwd><kwd>anti-EGFR</kwd><kwd>cetuximab</kwd></kwd-group></article-meta></front><body><sec id="s1"><title><offsets xml_i="7019" xml_f="7031" txt_i="1360" txt_f="1372">INTRODUCTION</offsets></title><p><offsets xml_i="7042" xml_f="7496" txt_i="1373" txt_f="1827">There have been significant improvements in the management of colorectal cancer over the last decade involving increasingly sophisticated combinations of chemotherapy, targeted agents, and surgery. This has included the use of anti-epidermal growth factor receptor (EGFR) antibodies such as cetuximab. Following several studies demonstrating the benefit of cetuximab as a single agent or in combination with chemotherapy for advanced colorectal cancer, [</offsets><xref rid="R1" ref-type="bibr"><offsets xml_i="7527" xml_f="7528" txt_i="1827" txt_f="1828">1</offsets></xref><offsets xml_i="7535" xml_f="7536" txt_i="1828" txt_f="1829">-</offsets><xref rid="R4" ref-type="bibr"><offsets xml_i="7567" xml_f="7568" txt_i="1829" txt_f="1830">4</offsets></xref><offsets xml_i="7575" xml_f="7707" txt_i="1830" txt_f="1962">] its role was evaluated as an adjunct to chemotherapy and surgery for operable colorectal liver metastases in the New EPOC study. [</offsets><xref rid="R5" ref-type="bibr"><offsets xml_i="7738" xml_f="7739" txt_i="1962" txt_f="1963">5</offsets></xref><offsets xml_i="7746" xml_f="8101" txt_i="1963" txt_f="2318">] Unexpectedly, this resulted in a shorter progression-free survival for patients treated with cetuximab in contrast to previous data associated with its use in inoperable disease. This effect persisted despite restriction of patients to an all RAS subgroup, suggesting that additional biomarkers linked with treatment efficacy have yet to be identified [</offsets><xref rid="R6" ref-type="bibr"><offsets xml_i="8132" xml_f="8133" txt_i="2318" txt_f="2319">6</offsets></xref><offsets xml_i="8140" xml_f="8142" txt_i="2319" txt_f="2321">].</offsets></p><p><offsets xml_i="8149" xml_f="8557" txt_i="2322" txt_f="2730">MicroRNAs (miRNA) are small non-coding RNA molecules that play a key role in the regulation of intracellular processes through post-transcriptional regulation of gene expression. The stable presence of miRNAs in formalin-fixed paraffin-embedded (FFPE) tissues and their involvement in multiple physiological pathways and pathologies suggests an opportunity to identify new biomarkers for multiple diseases. [</offsets><xref rid="R7" ref-type="bibr"><offsets xml_i="8588" xml_f="8589" txt_i="2730" txt_f="2731">7</offsets></xref><offsets xml_i="8596" xml_f="8597" txt_i="2731" txt_f="2732">-</offsets><xref rid="R11" ref-type="bibr"><offsets xml_i="8629" xml_f="8631" txt_i="2732" txt_f="2734">11</offsets></xref><offsets xml_i="8638" xml_f="8775" txt_i="2734" txt_f="2871">] MicroRNAs controlling expression of oncogenes and tumor suppressor genes have been shown to be frequently deregulated in cancer cells [</offsets><xref rid="R7" ref-type="bibr"><offsets xml_i="8806" xml_f="8807" txt_i="2871" txt_f="2872">7</offsets></xref><offsets xml_i="8814" xml_f="8815" txt_i="2872" txt_f="2873">-</offsets><xref rid="R9" ref-type="bibr"><offsets xml_i="8846" xml_f="8847" txt_i="2873" txt_f="2874">9</offsets></xref><offsets xml_i="8854" xml_f="8921" txt_i="2874" txt_f="2941">] suggesting potential biomarker candidates for precision oncology.</offsets></p><p><offsets xml_i="8928" xml_f="9059" txt_i="2942" txt_f="3073">A miRNA which has both oncogenic and tumor suppressive functional roles frequently deregulated in a variety of cancers is mir-31. [</offsets><xref rid="R12" ref-type="bibr"><offsets xml_i="9091" xml_f="9093" txt_i="3073" txt_f="3075">12</offsets></xref><offsets xml_i="9100" xml_f="9215" txt_i="3075" txt_f="3190">] In colorectal cancer, mir-31 is typically overexpressed with high expression correlating with advanced disease. [</offsets><xref rid="R13" ref-type="bibr"><offsets xml_i="9247" xml_f="9249" txt_i="3190" txt_f="3192">13</offsets></xref><offsets xml_i="9256" xml_f="9257" txt_i="3192" txt_f="3193">-</offsets><xref rid="R15" ref-type="bibr"><offsets xml_i="9289" xml_f="9291" txt_i="3193" txt_f="3195">15</offsets></xref><offsets xml_i="9298" xml_f="9445" txt_i="3195" txt_f="3342">] Functional studies have shown mir-31 has pleiotropic activity, promoting colorectal cancer progression through enhanced migration and invasion. [</offsets><xref rid="R16" ref-type="bibr"><offsets xml_i="9477" xml_f="9479" txt_i="3342" txt_f="3344">16</offsets></xref><offsets xml_i="9486" xml_f="9488" txt_i="3344" txt_f="3346">, </offsets><xref rid="R17" ref-type="bibr"><offsets xml_i="9520" xml_f="9522" txt_i="3346" txt_f="3348">17</offsets></xref><offsets xml_i="9529" xml_f="9780" txt_i="3348" txt_f="3599">] While the most frequently identified mature sequence of mir-31 identified in the literature is miR-31-5p, the other mature sequence of mir-31, miR-31-3p, has been reported to be associated with outcomes for patients treated with anti-EGFR therapy. [</offsets><xref rid="R18" ref-type="bibr"><offsets xml_i="9812" xml_f="9814" txt_i="3599" txt_f="3601">18</offsets></xref><offsets xml_i="9821" xml_f="9822" txt_i="3601" txt_f="3602">-</offsets><xref rid="R20" ref-type="bibr"><offsets xml_i="9854" xml_f="9856" txt_i="3602" txt_f="3604">20</offsets></xref><offsets xml_i="9863" xml_f="10130" txt_i="3604" txt_f="3871">] As a follow-up to these findings, the present study sought to evaluate whether expression of miR-31-3p could predict outcomes in the setting of operable colorectal liver metastasis in patients treated with chemotherapy plus cetuximab enrolled in the New EPOC study.</offsets></p></sec><sec id="s2"><title><offsets xml_i="10160" xml_f="10167" txt_i="3873" txt_f="3880">RESULTS</offsets></title><p><offsets xml_i="10178" xml_f="10344" txt_i="3881" txt_f="4047">The modified ITT population (mITT) was composed of 149 patients, 78 patients in a chemotherapy plus cetuximab arm and 71 patients in a chemotherapy alone arm (Figure </offsets><xref ref-type="fig" rid="F1"><offsets xml_i="10374" xml_f="10375" txt_i="4047" txt_f="4048">1</offsets></xref><offsets xml_i="10382" xml_f="10671" txt_i="4048" txt_f="4337">). One hundred and thirteen of these patients were confirmed all RAS WT. Thirty-six patients in the mITT population had incomplete RAS mutational status by NGS but were known to be KRAS WT (codons 12, 13, and 61) based on pyrosequencing results obtained during the original New EPOC study.</offsets></p><fig id="F1" orientation="portrait" position="float"><label><offsets xml_i="10735" xml_f="10743" txt_i="4338" txt_f="4346">Figure 1</offsets></label><caption><title><offsets xml_i="10767" xml_f="10810" txt_i="4346" txt_f="4389">Flow diagram for patients included in study</offsets></title></caption><graphic xlink:href="oncotarget-08-93856-g001"></graphic></fig><sec id="s2_1"><title><offsets xml_i="10913" xml_f="10995" txt_i="4390" txt_f="4472">Patient baseline characteristics in mITT and association with miR-31-3p expression</offsets></title><p><offsets xml_i="11006" xml_f="11104" txt_i="4473" txt_f="4571">In the mITT population, baseline characteristics were well balanced between treatment arms (Table </offsets><xref ref-type="table" rid="T1"><offsets xml_i="11136" xml_f="11137" txt_i="4571" txt_f="4572">1</offsets></xref><offsets xml_i="11144" xml_f="11195" txt_i="4572" txt_f="4623">) and between miR-31-3p expression tertiles (Table </offsets><xref ref-type="table" rid="T2"><offsets xml_i="11227" xml_f="11228" txt_i="4623" txt_f="4624">2</offsets></xref><offsets xml_i="11235" xml_f="11411" txt_i="4624" txt_f="4800">) with the exception of BRAF mutational status, which was more frequent in the high tertile group (6/7 BRAF mutations were in patients with high miR-31-3p expression, p=0.017).</offsets></p><table-wrap id="T1" orientation="portrait" position="float"><label><offsets xml_i="11482" xml_f="11489" txt_i="4801" txt_f="4808">Table 1</offsets></label><caption><title><offsets xml_i="11513" xml_f="11583" txt_i="4808" txt_f="4878">Baseline characteristics in the mITT population for each treatment arm</offsets></title></caption><table frame="box" rules="all"><thead><tr><th align="left" valign="middle" rowspan="1" colspan="1"></th><th align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="11764" xml_f="11776" txt_i="4879" txt_f="4891">Chemotherapy</offsets><break></break><offsets xml_i="11791" xml_f="11797" txt_i="4891" txt_f="4897">n = 71</offsets></th><th align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="11861" xml_f="11888" txt_i="4897" txt_f="4924">Chemotherapy plus cetuximab</offsets><break></break><offsets xml_i="11903" xml_f="11909" txt_i="4924" txt_f="4930">n = 78</offsets></th></tr></thead><tbody><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="12001" xml_f="12004" txt_i="4930" txt_f="4933">Sex</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12206" xml_f="12208" txt_i="4936" txt_f="4938"> F</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12272" xml_f="12282" txt_i="4939" txt_f="4949">25 (35.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12346" xml_f="12356" txt_i="4950" txt_f="4960">22 (28.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12427" xml_f="12429" txt_i="4961" txt_f="4963"> M</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12493" xml_f="12503" txt_i="4964" txt_f="4974">46 (64.8%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12567" xml_f="12577" txt_i="4975" txt_f="4985">56 (71.8%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="12654" xml_f="12657" txt_i="4986" txt_f="4989">Age</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12859" xml_f="12866" txt_i="4992" txt_f="4999"> [0-60]</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="12930" xml_f="12940" txt_i="5000" txt_f="5010">20 (28.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13004" xml_f="13014" txt_i="5011" txt_f="5021">19 (24.4%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13085" xml_f="13093" txt_i="5022" txt_f="5030"> [60-70]</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13157" xml_f="13167" txt_i="5031" txt_f="5041">34 (47.9%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13231" xml_f="13239" txt_i="5042" txt_f="5050">39 (50%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13310" xml_f="13316" txt_i="5051" txt_f="5057"> [70+]</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13380" xml_f="13390" txt_i="5058" txt_f="5068">17 (23.9%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13454" xml_f="13464" txt_i="5069" txt_f="5079">20 (25.6%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="13541" xml_f="13561" txt_i="5080" txt_f="5100">Chemotherapy regimen</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13763" xml_f="13769" txt_i="5103" txt_f="5109"> CapOx</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13833" xml_f="13843" txt_i="5110" txt_f="5120">15 (21.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13907" xml_f="13917" txt_i="5121" txt_f="5131">18 (23.4%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="13988" xml_f="13996" txt_i="5132" txt_f="5140"> FOLFIRI</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14060" xml_f="14068" txt_i="5141" txt_f="5149">6 (8.6%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14132" xml_f="14142" txt_i="5150" txt_f="5160">12 (15.6%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14213" xml_f="14220" txt_i="5161" txt_f="5168"> FOLFOX</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14284" xml_f="14292" txt_i="5169" txt_f="5177">49 (70%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14356" xml_f="14364" txt_i="5178" txt_f="5186">47 (61%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="14441" xml_f="14452" txt_i="5187" txt_f="5198">BRAF status</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14654" xml_f="14665" txt_i="5201" txt_f="5212"> WT/Unknown</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14729" xml_f="14739" txt_i="5213" txt_f="5223">68 (95.8%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14803" xml_f="14813" txt_i="5224" txt_f="5234">73 (94.8%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14884" xml_f="14892" txt_i="5235" txt_f="5243"> Mutated</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="14956" xml_f="14964" txt_i="5244" txt_f="5252">3 (4.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15028" xml_f="15036" txt_i="5253" txt_f="5261">4 (5.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="15113" xml_f="15135" txt_i="5262" txt_f="5284">Primary tumor location</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15337" xml_f="15348" txt_i="5287" txt_f="5298"> Left sided</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15412" xml_f="15422" txt_i="5299" txt_f="5309">59 (83.1%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15486" xml_f="15496" txt_i="5310" txt_f="5320">59 (75.6%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15567" xml_f="15579" txt_i="5321" txt_f="5333"> Right sided</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15643" xml_f="15653" txt_i="5334" txt_f="5344">12 (16.9%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="15717" xml_f="15727" txt_i="5345" txt_f="5355">19 (24.4%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="15804" xml_f="15830" txt_i="5356" txt_f="5382">Presentation of metastases</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16032" xml_f="16048" txt_i="5385" txt_f="5401"> Non-synchronous</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16112" xml_f="16122" txt_i="5402" txt_f="5412">33 (46.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16186" xml_f="16196" txt_i="5413" txt_f="5423">24 (30.8%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16267" xml_f="16279" txt_i="5424" txt_f="5436"> Synchronous</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16343" xml_f="16353" txt_i="5437" txt_f="5447">38 (53.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16417" xml_f="16427" txt_i="5448" txt_f="5458">54 (69.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="16504" xml_f="16530" txt_i="5459" txt_f="5485">Number of liver metastases</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16732" xml_f="16739" txt_i="5488" txt_f="5492"> &lt; 4</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16803" xml_f="16813" txt_i="5493" txt_f="5503">57 (81.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16877" xml_f="16887" txt_i="5504" txt_f="5514">58 (74.4%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="16958" xml_f="16962" txt_i="5515" txt_f="5519"> ≥ 4</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17026" xml_f="17036" txt_i="5520" txt_f="5530">13 (18.6%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17100" xml_f="17110" txt_i="5531" txt_f="5541">20 (25.6%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="17187" xml_f="17210" txt_i="5542" txt_f="5565">ECOG performance status</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17412" xml_f="17414" txt_i="5568" txt_f="5570"> 0</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17478" xml_f="17488" txt_i="5571" txt_f="5581">51 (71.8%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17552" xml_f="17562" txt_i="5582" txt_f="5592">54 (69.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17633" xml_f="17635" txt_i="5593" txt_f="5595"> 1</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17699" xml_f="17709" txt_i="5596" txt_f="5606">20 (28.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17773" xml_f="17783" txt_i="5607" txt_f="5617">22 (28.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17854" xml_f="17856" txt_i="5618" txt_f="5620"> 2</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17920" xml_f="17926" txt_i="5621" txt_f="5627">0 (0%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="17990" xml_f="17998" txt_i="5628" txt_f="5636">2 (2.6%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="18075" xml_f="18087" txt_i="5637" txt_f="5649">miR tertiles</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18289" xml_f="18293" txt_i="5652" txt_f="5656"> Low</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18357" xml_f="18367" txt_i="5657" txt_f="5667">23 (32.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18431" xml_f="18441" txt_i="5668" txt_f="5678">27 (34.6%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18512" xml_f="18525" txt_i="5679" txt_f="5692"> Intermediate</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18589" xml_f="18599" txt_i="5693" txt_f="5703">20 (28.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18663" xml_f="18673" txt_i="5704" txt_f="5714">29 (37.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18744" xml_f="18749" txt_i="5715" txt_f="5720"> High</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18813" xml_f="18823" txt_i="5721" txt_f="5731">28 (39.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="18887" xml_f="18897" txt_i="5732" txt_f="5742">22 (28.2%)</offsets></td></tr></tbody></table></table-wrap><table-wrap id="T2" orientation="portrait" position="float"><label><offsets xml_i="19003" xml_f="19010" txt_i="5743" txt_f="5750">Table 2</offsets></label><caption><title><offsets xml_i="19034" xml_f="19108" txt_i="5750" txt_f="5824">Baseline characteristics in the mITT population for each miR-31-3p tertile</offsets></title></caption><table frame="box" rules="all"><thead><tr><th align="left" valign="middle" rowspan="1" colspan="1"></th><th align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="19289" xml_f="19299" txt_i="5825" txt_f="5835">Low n = 50</offsets></th><th align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="19363" xml_f="19382" txt_i="5835" txt_f="5854">Intermediate n = 49</offsets></th><th align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="19446" xml_f="19457" txt_i="5854" txt_f="5865">High n = 50</offsets></th><th align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="19521" xml_f="19528" txt_i="5865" txt_f="5872">p-value</offsets></th></tr></thead><tbody><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="19620" xml_f="19623" txt_i="5872" txt_f="5875">Sex</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="19949" xml_f="19951" txt_i="5880" txt_f="5882"> F</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20015" xml_f="20023" txt_i="5883" txt_f="5891">13 (26%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20087" xml_f="20097" txt_i="5892" txt_f="5902">19 (38.8%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20161" xml_f="20169" txt_i="5903" txt_f="5911">15 (30%)</offsets></td><td rowspan="2" align="center" valign="middle" colspan="1"><offsets xml_i="20233" xml_f="20238" txt_i="5912" txt_f="5917">0.401</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20309" xml_f="20311" txt_i="5918" txt_f="5920"> M</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20375" xml_f="20383" txt_i="5921" txt_f="5929">37 (74%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20447" xml_f="20457" txt_i="5930" txt_f="5940">30 (61.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20521" xml_f="20529" txt_i="5941" txt_f="5949">35 (70%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="20606" xml_f="20609" txt_i="5950" txt_f="5953">Age</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="20935" xml_f="20942" txt_i="5958" txt_f="5965"> [0-60]</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21006" xml_f="21014" txt_i="5966" txt_f="5974">11 (22%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21078" xml_f="21088" txt_i="5975" txt_f="5985">12 (24.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21152" xml_f="21160" txt_i="5986" txt_f="5994">16 (32%)</offsets></td><td rowspan="3" align="center" valign="middle" colspan="1"><offsets xml_i="21224" xml_f="21229" txt_i="5995" txt_f="6000">0.815</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21300" xml_f="21308" txt_i="6001" txt_f="6009"> [60-70]</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21372" xml_f="21380" txt_i="6010" txt_f="6018">27 (54%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21444" xml_f="21452" txt_i="6019" txt_f="6027">24 (49%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21516" xml_f="21524" txt_i="6028" txt_f="6036">22 (44%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21595" xml_f="21601" txt_i="6037" txt_f="6043"> [70+]</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21665" xml_f="21673" txt_i="6044" txt_f="6052">12 (24%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21737" xml_f="21747" txt_i="6053" txt_f="6063">13 (26.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="21811" xml_f="21819" txt_i="6064" txt_f="6072">12 (24%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="21896" xml_f="21916" txt_i="6073" txt_f="6093">Chemotherapy regimen</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22242" xml_f="22248" txt_i="6098" txt_f="6104"> CapOx</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22312" xml_f="22322" txt_i="6105" txt_f="6115">10 (20.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22386" xml_f="22396" txt_i="6116" txt_f="6126">10 (20.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22460" xml_f="22470" txt_i="6127" txt_f="6137">13 (26.5%)</offsets></td><td rowspan="3" align="center" valign="middle" colspan="1"><offsets xml_i="22534" xml_f="22539" txt_i="6138" txt_f="6143">0.439</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22610" xml_f="22618" txt_i="6144" txt_f="6152"> FOLFIRI</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22682" xml_f="22690" txt_i="6153" txt_f="6161">3 (6.1%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22754" xml_f="22763" txt_i="6162" txt_f="6171">8 (16.3%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22827" xml_f="22836" txt_i="6172" txt_f="6181">7 (14.3%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22907" xml_f="22914" txt_i="6182" txt_f="6189"> FOLFOX</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="22978" xml_f="22988" txt_i="6190" txt_f="6200">36 (73.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23052" xml_f="23062" txt_i="6201" txt_f="6211">31 (63.3%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23126" xml_f="23136" txt_i="6212" txt_f="6222">29 (59.2%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="23213" xml_f="23224" txt_i="6223" txt_f="6234">BRAF status</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23550" xml_f="23561" txt_i="6239" txt_f="6250"> WT/Unknown</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23625" xml_f="23633" txt_i="6251" txt_f="6259">49 (98%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23697" xml_f="23706" txt_i="6260" txt_f="6269">48 (100%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23770" xml_f="23778" txt_i="6270" txt_f="6278">44 (88%)</offsets></td><td rowspan="2" align="center" valign="middle" colspan="1"><offsets xml_i="23842" xml_f="23847" txt_i="6279" txt_f="6284">0.017</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23918" xml_f="23926" txt_i="6285" txt_f="6293"> Mutated</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="23990" xml_f="23996" txt_i="6294" txt_f="6300">1 (2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24060" xml_f="24066" txt_i="6301" txt_f="6307">0 (0%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24130" xml_f="24137" txt_i="6308" txt_f="6315">6 (12%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="24214" xml_f="24236" txt_i="6316" txt_f="6338">Primary tumor location</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24562" xml_f="24574" txt_i="6343" txt_f="6355"> Left+rectum</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24638" xml_f="24646" txt_i="6356" txt_f="6364">43 (86%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24710" xml_f="24720" txt_i="6365" txt_f="6375">39 (79.6%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24784" xml_f="24792" txt_i="6376" txt_f="6384">36 (72%)</offsets></td><td rowspan="2" align="center" valign="middle" colspan="1"><offsets xml_i="24856" xml_f="24861" txt_i="6385" txt_f="6390">0.229</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="24932" xml_f="24938" txt_i="6391" txt_f="6397"> Right</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25002" xml_f="25009" txt_i="6398" txt_f="6405">7 (14%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25073" xml_f="25083" txt_i="6406" txt_f="6416">10 (20.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25147" xml_f="25155" txt_i="6417" txt_f="6425">14 (28%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="25232" xml_f="25258" txt_i="6426" txt_f="6452">Presentation of metastases</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25584" xml_f="25600" txt_i="6457" txt_f="6473"> Non-synchronous</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25664" xml_f="25672" txt_i="6474" txt_f="6482">18 (36%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25736" xml_f="25746" txt_i="6483" txt_f="6493">19 (38.8%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25810" xml_f="25818" txt_i="6494" txt_f="6502">20 (40%)</offsets></td><td rowspan="2" align="center" valign="middle" colspan="1"><offsets xml_i="25882" xml_f="25887" txt_i="6503" txt_f="6508">0.933</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="25958" xml_f="25970" txt_i="6509" txt_f="6521"> Synchronous</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26034" xml_f="26042" txt_i="6522" txt_f="6530">32 (64%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26106" xml_f="26116" txt_i="6531" txt_f="6541">30 (61.2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26180" xml_f="26188" txt_i="6542" txt_f="6550">30 (60%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="26265" xml_f="26291" txt_i="6551" txt_f="6577">Number of liver metastases</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26617" xml_f="26624" txt_i="6582" txt_f="6586"> &lt; 4</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26688" xml_f="26698" txt_i="6587" txt_f="6597">37 (75.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26762" xml_f="26772" txt_i="6598" txt_f="6608">38 (77.6%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26836" xml_f="26844" txt_i="6609" txt_f="6617">40 (80%)</offsets></td><td rowspan="2" align="center" valign="middle" colspan="1"><offsets xml_i="26908" xml_f="26913" txt_i="6618" txt_f="6623">0.855</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="26984" xml_f="26988" txt_i="6624" txt_f="6628"> ≥ 4</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27052" xml_f="27062" txt_i="6629" txt_f="6639">12 (24.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27126" xml_f="27136" txt_i="6640" txt_f="6650">11 (22.5%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27200" xml_f="27208" txt_i="6651" txt_f="6659">10 (20%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><bold><offsets xml_i="27285" xml_f="27308" txt_i="6660" txt_f="6683">ECOG performance status</offsets></bold></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td><td align="left" valign="middle" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27634" xml_f="27636" txt_i="6688" txt_f="6690"> 0</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27700" xml_f="27708" txt_i="6691" txt_f="6699">34 (68%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27772" xml_f="27782" txt_i="6700" txt_f="6710">34 (69.4%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27846" xml_f="27854" txt_i="6711" txt_f="6719">37 (74%)</offsets></td><td rowspan="3" align="center" valign="middle" colspan="1"><offsets xml_i="27918" xml_f="27923" txt_i="6720" txt_f="6725">0.850</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="27994" xml_f="27996" txt_i="6726" txt_f="6728"> 1</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28060" xml_f="28068" txt_i="6729" txt_f="6737">15 (30%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28132" xml_f="28142" txt_i="6738" txt_f="6748">15 (30.6%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28206" xml_f="28214" txt_i="6749" txt_f="6757">12 (24%)</offsets></td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28285" xml_f="28287" txt_i="6758" txt_f="6760"> 2</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28351" xml_f="28357" txt_i="6761" txt_f="6767">1 (2%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28421" xml_f="28427" txt_i="6768" txt_f="6774">0 (0%)</offsets></td><td align="center" valign="middle" rowspan="1" colspan="1"><offsets xml_i="28491" xml_f="28497" txt_i="6775" txt_f="6781">1 (2%)</offsets></td></tr></tbody></table></table-wrap></sec><sec id="s2_2"><title><offsets xml_i="28564" xml_f="28608" txt_i="6783" txt_f="6827">Patient clinical outcomes in mITT population</offsets></title><p><offsets xml_i="28619" xml_f="29096" txt_i="6828" txt_f="7305">Despite a high percentage of censored patients (52% for progression-free survival and 82% for overall survival) progression-free survival was significantly shorter in the chemotherapy plus cetuximab arm versus the chemotherapy alone arm with a median of 14.5 months versus 24.2 months (HR=1.8 [1.1 ; 2.9], p=0.02). The same was observed for overall survival (HR=2.2 [1.0 ; 4.9], p=0.047). The above aligns with previously published results from the New EPOC interim analysis. [</offsets><xref rid="R20" ref-type="bibr"><offsets xml_i="29128" xml_f="29130" txt_i="7305" txt_f="7307">20</offsets></xref><offsets xml_i="29137" xml_f="29138" txt_i="7307" txt_f="7308">]</offsets></p></sec><sec id="s2_3"><title><offsets xml_i="29170" xml_f="29255" txt_i="7310" txt_f="7395">miR-31-3p as a prognostic marker in patients treated with chemotherapy plus cetuximab</offsets></title><p><offsets xml_i="29266" xml_f="29404" txt_i="7396" txt_f="7534">Progression-free survival was not significantly different between the miR-31-3p expression subgroups in the whole mITT population (Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="29434" xml_f="29436" txt_i="7534" txt_f="7536">2A</offsets></xref><offsets xml_i="29443" xml_f="29667" txt_i="7536" txt_f="7760">) with a median progression free survival of 18.0 months in the low expression subgroup versus 15.9 months in the combined mid plus high expression subgroup (HR=1.4 [0.8 ; 2.3] p=0.25). In the chemotherapy alone arm (Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="29697" xml_f="29699" txt_i="7760" txt_f="7762">2B</offsets></xref><offsets xml_i="29706" xml_f="29909" txt_i="7762" txt_f="7965">), progression-free survival was not statistically different between the miR-31-3p expression subgroups (HR=0.9 [0.4 ; 2.0], p=0.79). In contrast, patients in the chemotherapy plus cetuximab arm (Figure </offsets><xref ref-type="fig" rid="F2"><offsets xml_i="29939" xml_f="29941" txt_i="7965" txt_f="7967">2C</offsets></xref><offsets xml_i="29948" xml_f="30442" txt_i="7967" txt_f="8461">) with low miR-31-3p expression had a longer progression-free survival compared to patients in the combined mid plus high expression subgroup (median of 20.3 months versus 12.3 months, HR=2.0 [1.0 ; 4.2], p=0.049). Overall survival was not significantly different between the miR-31-3p expression groups in the whole mITT population (HR=0.9 [0.4 ; 2.1], p=0.86), in the chemotherapy alone arm (HR=0.7 [0.2 ; 2.9], p=0.62), or in the chemotherapy plus cetuximab arm (HR=1.1 [0.4 ; 3.0], p=0.85).</offsets></p><fig id="F2" orientation="portrait" position="float"><label><offsets xml_i="30506" xml_f="30514" txt_i="8462" txt_f="8470">Figure 2</offsets></label><caption><title><offsets xml_i="30538" xml_f="30639" txt_i="8470" txt_f="8571">Kaplan-Meier curves of progression-free survival (PFS) according to miR-31-3p expression level groups</offsets></title><p><bold><offsets xml_i="30656" xml_f="30659" txt_i="8572" txt_f="8575">(A)</offsets></bold><offsets xml_i="30666" xml_f="30703" txt_i="8575" txt_f="8612"> PFS for the entire miTT population; </offsets><bold><offsets xml_i="30709" xml_f="30712" txt_i="8612" txt_f="8615">(B)</offsets></bold><offsets xml_i="30719" xml_f="30765" txt_i="8615" txt_f="8661"> PFS for the chemotherapy alone treatment arm </offsets><bold><offsets xml_i="30771" xml_f="30774" txt_i="8661" txt_f="8664">(C)</offsets></bold><offsets xml_i="30781" xml_f="30836" txt_i="8664" txt_f="8719"> PFS for the chemotherapy plus cetuximab treatment arm.</offsets></p></caption><graphic xlink:href="oncotarget-08-93856-g002"></graphic></fig><p><offsets xml_i="30916" xml_f="31757" txt_i="8720" txt_f="9561">Objective response rates did not significantly vary according to miR-31-3p expression in the whole mITT population with 34 patients (68%) achieving complete or partial response out of the 50 patients with low miR-31-3p expression and 62 patients (63%) out of the 99 in the mid plus high miR-31-3p expression subgroup, (p=0.59). There was also no difference in objective response rate when analyzed by treatment arm. For the chemotherapy alone arm, 15 of the 23 patients (65%) with low miR-31-3p expression and 27 of the 48 patients (56%) in the mid plus high miR-31-3p expression subgroup achieved complete or partial responses (p=0.61). For the chemotherapy plus cetuximab arm, 19 of the 27 patients (70%) with low miR-31-3p expression and 35 of the 51 (69%) with mid or high miR-31-3p expression had a complete or partial response (p=1.0).</offsets></p></sec><sec id="s2_4"><title><offsets xml_i="31789" xml_f="31862" txt_i="9563" txt_f="9636">miR-31-3p as predictive marker of cetuximab efficacy on survival outcomes</offsets></title><p><offsets xml_i="31873" xml_f="32313" txt_i="9637" txt_f="10077">Patients in the mid plus high miR-31-3p expression group treated with cetuximab had a shorter progression free survival compared to patients treated with chemotherapy alone (median progression free survival 12.3 versus 26.7 months respectively; HR=2.3 [1.3 ; 4.1], p=0.005). This difference was not observed in low expressers (median progression free survival 20.3 months versus 18.9 months respectively; HR=1.1 [0.4 ; 2.6] p=0.91) (Figure </offsets><xref ref-type="fig" rid="F3"><offsets xml_i="32343" xml_f="32344" txt_i="10077" txt_f="10078">3</offsets></xref><xref ref-type="fig" rid="F4"><offsets xml_i="32381" xml_f="32382" txt_i="10078" txt_f="10079">4</offsets></xref><offsets xml_i="32389" xml_f="32479" txt_i="10079" txt_f="10169">). The treatment-miR-31-3p expression group interaction test was not significant (p=0.16).</offsets></p><fig id="F3" orientation="portrait" position="float"><label><offsets xml_i="32543" xml_f="32551" txt_i="10170" txt_f="10178">Figure 3</offsets></label><caption><title><offsets xml_i="32575" xml_f="32646" txt_i="10178" txt_f="10249">Kaplan-Meier curves of progression-free survival (PFS) by treatment arm</offsets></title><p><bold><offsets xml_i="32663" xml_f="32666" txt_i="10250" txt_f="10253">(A)</offsets></bold><offsets xml_i="32673" xml_f="32721" txt_i="10253" txt_f="10301"> OS for the mid plus high miR-31-3p expressers; </offsets><bold><offsets xml_i="32727" xml_f="32730" txt_i="10301" txt_f="10304">(B)</offsets></bold><offsets xml_i="32737" xml_f="32770" txt_i="10304" txt_f="10337"> OS for low miR-31-3p expressers.</offsets></p></caption><graphic xlink:href="oncotarget-08-93856-g003"></graphic></fig><fig id="F4" orientation="portrait" position="float"><label><offsets xml_i="32907" xml_f="32915" txt_i="10338" txt_f="10346">Figure 4</offsets></label><caption><title><offsets xml_i="32939" xml_f="33000" txt_i="10346" txt_f="10407">Kaplan-Meier curves of overall survival (OS) by treatment arm</offsets></title><p><bold><offsets xml_i="33017" xml_f="33020" txt_i="10408" txt_f="10411">(A)</offsets></bold><offsets xml_i="33027" xml_f="33075" txt_i="10411" txt_f="10459"> OS for the mid plus high miR-31-3p expressers; </offsets><bold><offsets xml_i="33081" xml_f="33084" txt_i="10459" txt_f="10462">(B)</offsets></bold><offsets xml_i="33091" xml_f="33124" txt_i="10462" txt_f="10495"> OS for low miR-31-3p expressers.</offsets></p></caption><graphic xlink:href="oncotarget-08-93856-g004"></graphic></fig><p><offsets xml_i="33204" xml_f="33746" txt_i="10496" txt_f="11038">Objective response rates were not significantly different according to treatment arm. For the mid plus high miR-31-3p expression subgroup, 35 of the 51 patients (68%) in the chemotherapy plus cetuximab arm, and 27 of the 48 patients (56%) in the chemotherapy alone arm achieved either a partial or complete response; p=0.22. For the low expressers subgroup, 19 of the 27 patients (70%) in the chemotherapy plus cetuximab arm and 15 of the 23 patients (65%) in the chemotherapy alone arm achieved either a partial or complete response; p=0.77.</offsets></p></sec><sec id="s2_5"><title><offsets xml_i="33778" xml_f="33799" txt_i="11040" txt_f="11061">Multivariate analyses</offsets></title><p><offsets xml_i="33810" xml_f="34195" txt_i="11062" txt_f="11447">Consistent with the univariate analysis results, multivariate analyses adjusted for confounding factors showed that miR-31-3p expression group was not a significant predictive factor for progression free survival in the overall population or in the chemotherapy alone arm, but remained significant in the cetuximab treated population (low versus mid-high, HR = 2.1 [1.0; 4.4], p=0.05).</offsets></p></sec><sec id="s2_6"><title><offsets xml_i="34227" xml_f="34299" txt_i="11449" txt_f="11521">Comparison of miR-31-3p expression between metastases and primary tumors</offsets></title><p><offsets xml_i="34310" xml_f="35238" txt_i="11522" txt_f="12447">An analysis of miR-31-3p expression in liver metastases collected post neoadjuvant treatment versus primary tumors collected at baseline was performed using 94 paired samples of primary tumors and liver metastases. MiR-31-3p expression was significantly lower in metastases compared to primary tumors with a mean (std) of log-fold of -0.66 (1.18) for primary tumors versus -1.79 (1.37) for liver metastases, p&lt;0.001. Expression levels did not significantly differ between treatment arms either in primary tumors (-0.81 [1.11] for chemotherapy plus cetuximab versus -0.87 [1.18] for chemotherapy alone, p=0.45) or in liver metastases (-1.68 [1.28] for chemotherapy plus cetuximab versus -1.96 [1.43] for chemotherapy alone, p=0.76). The absolute change of expression between treatment arms was not significantly different (0.87 [1.32] for chemotherapy plus cetuximab versus 1.08 [1.62] for chemotherapy alone, p=0.36) (Figure </offsets><xref ref-type="fig" rid="F5"><offsets xml_i="35268" xml_f="35269" txt_i="12447" txt_f="12448">5</offsets></xref><offsets xml_i="35276" xml_f="35278" txt_i="12448" txt_f="12450">).</offsets></p><fig id="F5" orientation="portrait" position="float"><label><offsets xml_i="35342" xml_f="35350" txt_i="12451" txt_f="12459">Figure 5</offsets></label><caption><title><offsets xml_i="35374" xml_f="35486" txt_i="12459" txt_f="12571">Distribution of miR-31-3p expression (logarithmic scale) by treatment arm, in primary tumor and liver metastases</offsets></title><p><offsets xml_i="35497" xml_f="35586" txt_i="12572" txt_f="12661">(CT = chemotherapy alone treatment arm; CTX = chemotherapy + chemotherapy treatment arm).</offsets></p></caption><graphic xlink:href="oncotarget-08-93856-g005"></graphic></fig><p><offsets xml_i="35666" xml_f="36066" txt_i="12662" txt_f="13062">In the 47 paired samples from the chemotherapy alone arm, miR-31-3p expression levels in primary tumors and corresponding metastases were highly correlated with a correlation coefficient of 0.42; p=0.0031. In contrast, for the 47 paired samples from the chemotherapy plus cetuximab arm, a lower, non-statistically significant correlation was observed (correlation coefficient = 0.22, p=0.14) (Figure </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="36096" xml_f="36098" txt_i="13062" txt_f="13064">6A</offsets></xref><offsets xml_i="36105" xml_f="36110" txt_i="13064" txt_f="13069"> and </offsets><xref ref-type="fig" rid="F6"><offsets xml_i="36140" xml_f="36142" txt_i="13069" txt_f="13071">6B</offsets></xref><offsets xml_i="36149" xml_f="36151" txt_i="13071" txt_f="13073">).</offsets></p><fig id="F6" orientation="portrait" position="float"><label><offsets xml_i="36215" xml_f="36223" txt_i="13074" txt_f="13082">Figure 6</offsets></label><caption><p><offsets xml_i="36243" xml_f="36348" txt_i="13082" txt_f="13187">Correlation of miR-31-3p expression (log-transformed) between paired primary tumors and liver metastases </offsets><bold><offsets xml_i="36354" xml_f="36357" txt_i="13187" txt_f="13190">(A)</offsets></bold><offsets xml_i="36364" xml_f="36399" txt_i="13190" txt_f="13225"> chemotherapy alone treatment arm; </offsets><bold><offsets xml_i="36405" xml_f="36408" txt_i="13225" txt_f="13228">(B)</offsets></bold><offsets xml_i="36415" xml_f="36455" txt_i="13228" txt_f="13268"> chemotherapy + cetuximab treatment arm.</offsets></p></caption><graphic xlink:href="oncotarget-08-93856-g006"></graphic></fig></sec></sec><sec id="s3"><title><offsets xml_i="36564" xml_f="36574" txt_i="13271" txt_f="13281">DISCUSSION</offsets></title><p><offsets xml_i="36585" xml_f="36757" txt_i="13282" txt_f="13454">The New EPOC study demonstrated a shorter progression-free survival with the addition of cetuximab to chemotherapy for patients with operable colorectal liver metastases. [</offsets><xref rid="R5" ref-type="bibr"><offsets xml_i="36788" xml_f="36789" txt_i="13454" txt_f="13455">5</offsets></xref><offsets xml_i="36796" xml_f="37283" txt_i="13455" txt_f="13942">] While it is a reminder that the consequences of combining treatment strategies can be unpredictable, it does afford a unique opportunity to increase our understanding of tumor responses to treatment with anti-EGFR therapy. Not only can putative predictive biomarkers be assessed in the “pre-treatment” primary tumor tissue, but the additional availability of “post-treatment” resected liver metastases enables paired analyses to aid a mechanistic dissection of the effect of cetuximab.</offsets></p><p><offsets xml_i="37290" xml_f="38003" txt_i="13943" txt_f="14656">In our study we observed that in the subgroup of patients treated with cetuximab, patients in the combined mid plus high miR-31-3p expression group had a shorter progression free survival in comparison to patients treated with low miR-31-3p expression. By contrast in the chemotherapy alone arm, progression free survival was similar regardless of miR-31-3p expression level. We also observed that patients in the mid plus high miR-31-3p expression group had a shorter progression free survival and shorter overall survival when treated with cetuximab compared to patients treated with chemotherapy alone, whereas patients with low miR-31-3p expression had similar clinical outcomes for both study treatment arms.</offsets></p><p><offsets xml_i="38010" xml_f="38368" txt_i="14657" txt_f="15015">Although we could not demonstrate the predictivity of miR-31-3p (homogeneity test p-value p=0.16), this is likely due to the low number of patients and lack of maturity of the data (52% censored patients for progression free survival and 82% for overall survival). The power of this study to demonstrate predictivity, computed a posteriori, was only 65.3%. [</offsets><xref rid="R21" ref-type="bibr"><offsets xml_i="38400" xml_f="38402" txt_i="15015" txt_f="15017">21</offsets></xref><offsets xml_i="38409" xml_f="38599" txt_i="15017" txt_f="15207">] Similarly, the absence of demonstration of a prognostic effect of miR-31-3p on overall survival in the cetuximab-treated population is likely to be related to the limited number of events.</offsets></p><p><offsets xml_i="38606" xml_f="39056" txt_i="15208" txt_f="15658">We failed to observe a clear association between miR-31-3p expression and response to cetuximab. This could be due to several reasons including the high objective response rate observed both in the chemotherapy alone (42/71 59%) and cetuximab (54/78 69%) arms suggesting that cetuximab impact on the tumor response was relatively marginal, limiting the capacity to demonstrate an interaction between miR-31-3p expression and response in this setting.</offsets></p><p><offsets xml_i="39063" xml_f="39820" txt_i="15659" txt_f="16416">While analysis of miR-31-3p expression and its association to clinical outcomes did not enable the ability to identify a patient subgroup that benefited from the addition of cetuximab to chemotherapy, this needs to be interpreted in the context of the results of the New EPOC study in which the addition of cetuximab to chemotherapy produced a detrimental effect. However, our results show that the poorer outcomes associated with the addition of cetuximab to chemotherapy were limited to patients who had middle to high miR-31-3p expression and that this detrimental effect was not observed in patients with low miR-31-3p expression levels. As such, the predictive value of miR-31-3p expression is consistent with the primary outcome of the New EPOC study.</offsets></p><p><offsets xml_i="39827" xml_f="40018" txt_i="16417" txt_f="16608">The potential interest of miR-31-3p expression as a predictive biomarker of the response to anti-EGFR therapy in metastatic colorectal cancer has been previously reported in several papers. [</offsets><xref rid="R19" ref-type="bibr"><offsets xml_i="40050" xml_f="40052" txt_i="16608" txt_f="16610">19</offsets></xref><offsets xml_i="40059" xml_f="40061" txt_i="16610" txt_f="16612">, </offsets><xref rid="R20" ref-type="bibr"><offsets xml_i="40093" xml_f="40095" txt_i="16612" txt_f="16614">20</offsets></xref><offsets xml_i="40102" xml_f="40295" txt_i="16614" txt_f="16807">] Preliminary results of studies assessing the association with miR-31-3p expression and therapeutic benefit from anti-EGFR therapy have also been reported for the PICCOLO and FIRE-3 studies. [</offsets><xref rid="R22" ref-type="bibr"><offsets xml_i="40327" xml_f="40329" txt_i="16807" txt_f="16809">22</offsets></xref><offsets xml_i="40336" xml_f="40338" txt_i="16809" txt_f="16811">, </offsets><xref rid="R23" ref-type="bibr"><offsets xml_i="40370" xml_f="40372" txt_i="16811" txt_f="16813">23</offsets></xref><offsets xml_i="40379" xml_f="40603" txt_i="16813" txt_f="17037">] The present analysis was performed prior to the determination of the optimal cut-off value for miR-31-3p that has recently been reported based on the analysis of tumors samples from patients enrolled in the FIRE-3 trial. [</offsets><xref rid="R23" ref-type="bibr"><offsets xml_i="40635" xml_f="40637" txt_i="17037" txt_f="17039">23</offsets></xref><offsets xml_i="40644" xml_f="41271" txt_i="17039" txt_f="17666">] As a result, the present study in which the patient population was separated into tertiles represents an initial exploration of potential cut-off points for miR-31-3p expression as it relates to differential outcomes. Although the specific context and results of the study as well as the lack of maturity of the data, prevent the generalization of the conclusions to other clinical settings, we believe the present study contributes to the accumulation of data supporting the clinical and biological rationale of miR-31-3p expression as a predictive biomarker of response to anti-EGFR therapy in metastatic colorectal cancer.</offsets></p><p><offsets xml_i="41278" xml_f="41811" txt_i="17667" txt_f="18200">The loss of correlation of miR-31-3p expression level for the cetuximab group between primary tumors, which were collected prior to any treatment, and liver metastasis, which were collected after treatment, suggests that treatment with cetuximab may alter the expression of this microRNA. This finding suggests the regulation of miR-31-3p expression as a part of the EGFR pathway and is consistent with a previous report where maturation of mir31 in the miR-31-5p form was found to be regulated via EGFR in a tumor hypoxia context. [</offsets><xref rid="R24" ref-type="bibr"><offsets xml_i="41843" xml_f="41845" txt_i="18200" txt_f="18202">24</offsets></xref><offsets xml_i="41852" xml_f="41992" txt_i="18202" txt_f="18342">] It is also important to note that the selection of miR-31-3p for the present study was done following a broad screening of 1,145 miRNAs, [</offsets><xref rid="R19" ref-type="bibr"><offsets xml_i="42024" xml_f="42026" txt_i="18342" txt_f="18344">19</offsets></xref><offsets xml_i="42033" xml_f="42183" txt_i="18344" txt_f="18494">] and as a result of previous findings that pre-mir-31 leads to maturation of the highly correlated mature forms of mir-31, miR-31-3p and miR-31-5p. [</offsets><xref rid="R20" ref-type="bibr"><offsets xml_i="42215" xml_f="42217" txt_i="18494" txt_f="18496">20</offsets></xref><offsets xml_i="42224" xml_f="42542" txt_i="18496" txt_f="18814">] Despite this high correlation between the two mature forms of mir-31, prediction of response to anti-EGFR therapy differs between miR-31-3p and miR-31-5p, suggesting the existence of fine regulation mechanisms involving specifically miR-31-3p or genes targeted by miR-31-3p, beyond regulation of mir-31 expression. [</offsets><xref rid="R18" ref-type="bibr"><offsets xml_i="42574" xml_f="42576" txt_i="18814" txt_f="18816">18</offsets></xref><offsets xml_i="42583" xml_f="42584" txt_i="18816" txt_f="18817">]</offsets></p><p><offsets xml_i="42591" xml_f="43227" txt_i="18818" txt_f="19454">In summary, analysis of miR-31-3p expression enabled the identification of a subgroup of patients with operable colorectal liver metastases who have a poorer outcome when treated with chemotherapy plus cetuximab. While the mechanism of this differential effect relative to miR-31-3p expression requires investigation, these data support a biological explanation for the detrimental effect observed with the use of cetuximab in patients in the New EPOC study, rather than, for example, inadequate surgery being responsible. Further work is ongoing to validate the use of this promising predictive biomarker in advanced colorectal cancer.</offsets></p></sec><sec id="s4"><title><offsets xml_i="43257" xml_f="43278" txt_i="19456" txt_f="19477">MATERIALS AND METHODS</offsets></title><sec id="s4_1"><title><offsets xml_i="43308" xml_f="43328" txt_i="19478" txt_f="19498">Patients and samples</offsets></title><p><offsets xml_i="43339" xml_f="43388" txt_i="19499" txt_f="19548">The New EPOC trial has previously been reported [</offsets><xref rid="R5" ref-type="bibr"><offsets xml_i="43419" xml_f="43420" txt_i="19548" txt_f="19549">5</offsets></xref><offsets xml_i="43427" xml_f="43513" txt_i="19549" txt_f="19635">] and the full protocol can be found on the Southampton Clinical Trials Unit website (</offsets><ext-link ext-link-type="uri" xlink:href="http://www.ctu.soton.ac.uk"><offsets xml_i="43583" xml_f="43602" txt_i="19635" txt_f="19654">www.ctu.soton.ac.uk</offsets></ext-link><offsets xml_i="43613" xml_f="44162" txt_i="19654" txt_f="20200">). Patients with resectable or suboptimally resectable liver metastases from colorectal cancer were randomly allocated to two treatment arms; chemotherapy (oxaliplatin/irinotecan plus fluorouracil) with or without cetuximab given before and after liver resection. The study was closed by the Trial Steering Committee on advice from the Data Monitoring and Ethics Committee (DMEC) in November 2012 when the protocol defined futility criteria were met (the lower limit of the 95% Confidence Interval (CI) for progression free survival HR was &gt;1). [</offsets><xref rid="R25" ref-type="bibr"><offsets xml_i="44194" xml_f="44196" txt_i="20200" txt_f="20202">25</offsets></xref><offsets xml_i="44203" xml_f="44204" txt_i="20202" txt_f="20203">]</offsets></p><p><offsets xml_i="44211" xml_f="44621" txt_i="20204" txt_f="20614">Primary tumor and liver metastasis samples collected from patients enrolled in the New EPOC trial were analyzed for the present study. As per the New EPOC study protocol, primary tumor samples were collected at baseline and samples for liver metastases were collected following neoadjuvant treatment and liver resection. Primary tumors from 167 of the 236 patients in the original primary analysis population [</offsets><xref rid="R5" ref-type="bibr"><offsets xml_i="44652" xml_f="44653" txt_i="20614" txt_f="20615">5</offsets></xref><offsets xml_i="44660" xml_f="45320" txt_i="20615" txt_f="21275">] were available for extended RAS mutational testing and miR-31-3p expression in the current analysis. The modified intention to treat (mITT) population was defined as those patients included in the original New EPOC primary analysis population (e.g. those who were KRAS wild-type for codons 12, 13 and 61 at trial entry) in whom further testing did not demonstrate any RAS mutation. This resulted in a mITT population comprised of 149 patients. Liver metastasis samples were available for 147 patients enrolled in the original New EPOC study with paired samples of both primary tumor and metastasis available for a total of 94 included in the mITT population.</offsets></p></sec><sec id="s4_2"><title><offsets xml_i="45352" xml_f="45371" txt_i="21277" txt_f="21296">Mutational analyses</offsets></title><p><offsets xml_i="45382" xml_f="47181" txt_i="21297" txt_f="23096">Primary tumors and metastases analyzed for the present study were sequenced for KRAS codons 12, 13, 61, 117 and 146, NRAS codons 12, 13, 61, 117 146, and BRAF codon 600 using Sequencing By Synthesis (SBS) on a MiSeq platform (Illumina, San Diego, California). Mutations were identified with respect to two known non-mutated controls for all mutations considered and the three positions of the codon were analyzed when needed. A 99% confidence interval for the minor allele frequency was calculated, taking into account the total number of bases read for a nucleotide in a particular codon and the number of “new” bases read for the same nucleotide of the same codon. Mutations were considered actual results when the same base was identified regarding the 2 controls. A frequency below 5% was considered as non-mutated. Samples which had indeterminate results were resequenced using Ion Torrent™ Technology (Thermo Fisher Scientific, Carlsbad, CA). For this analysis libraries were prepared according the manufacturer’s instruction (Ion AmpliSeq™ DNA and RNA Library Preparation Rev.B, Ion Torrent, Thermo Fisher Scientific, Carlsbad, CA). Samples were amplified with the Ion AmpliSeq™ Cancer HotSpot Panel v2 and the concentration of each library was determined with Experion™ Automated Electrophoresis system (Bio-Rad Laboratories Inc., Hercules, CA). Emulsion PCR was done according to the manufacturer’s user guide (Ion PGM™ Template OT2 200 Kit Revision A.0). Sequencing was performed using two Ion 318TM Chip V2 on an Ion PGM System™. Data analysis, including alignment to the hg19 human reference genome and variant calling, was done using the Torrent Suite Software v4.0. Alignments were visually verified and annotated with the ALAMUT software v2.2 (Interactive Biosoftware, Rouen, France).</offsets></p></sec><sec id="s4_3"><title><offsets xml_i="47213" xml_f="47242" txt_i="23098" txt_f="23127">MiR-31-3p expression analyses</offsets></title><p><offsets xml_i="47253" xml_f="47929" txt_i="23128" txt_f="23804">A pathologist reviewed all samples and the tumor area was marked for subsequent macrodissection. Only samples with more than 20% tumor cell content were selected for nucleic acid extraction. For each tumor sample, 5 FFPE slides of 5μm thickness were scratched in the tumor area and total RNA was extracted using the FFPE miRNeasy extraction kit (Qiagen, Hilden, Germany) according to the manufacturer’s instructions. Specific quantification of expression level of miRNA hsa-miR-31-3p was performed on retrotranscribed RNA using specific TaqMan pre-designed assays on ABI7900HT Real-Time PCR System. Expression levels were normalized to a reference miRNA using the ΔΔCt method.</offsets></p></sec><sec id="s4_4"><title><offsets xml_i="47961" xml_f="47981" txt_i="23806" txt_f="23826">Statistical analysis</offsets></title><p><offsets xml_i="47992" xml_f="48533" txt_i="23827" txt_f="24368">The primary endpoint was progression free survival defined as the time from randomization to recurrence, disease progression, or death, whichever occurred first. Secondary endpoints included overall survival (time from randomization to death; patients still alive at the date of last follow-up were censored) and preoperative response (using Response Evaluation Criteria in Solid Tumors (RECIST) version 1.0). All analyses reported here include data up to November 1, 2012 to match the time frame presented in the interim report to the DMEC.</offsets></p><p><offsets xml_i="48540" xml_f="49491" txt_i="24369" txt_f="25320">Analyses were performed for the modified Intent-to-Treat (mITT) population which consisted of 149 KRAS WT patients whose RAS status was either WT or indeterminate. Patients whose tumors were confirmed to be KRAS or RAS mutated were excluded from the analysis. In line with the study’s pre-specified statistical analysis plan, patients were divided into three subgroups defined as either low, medium (mid) or high miR-31-3p expression levels based on tertiles of miR-31-3p expression distribution. Progression free survival and overall survival comparisons according to miR-31-3p expression tertile and between each treatment arm were performed using Kaplan-Meier (KM) methodology and two-sided log-rank test. Objective Response Rate according to tertiles and across treatment arms were compared using Fisher exact test, considering non-evaluable patients as non-responders. Because similar results were observed in the mid and high tertile subgroups (</offsets><xref ref-type="supplementary-material" rid="SD1"><offsets xml_i="49541" xml_f="49563" txt_i="25320" txt_f="25342">Supplementary Figure 1</offsets></xref><offsets xml_i="49570" xml_f="50774" txt_i="25342" txt_f="26546">), these subgroups were merged into a single “high” miR-31-3p expression subgroup and subsequent analyses compared the “low” miR-31-3p expression subgroup (low tertile) with the “high” subgroup (mid + high tertiles). Impact of covariates on progression free survival or overall survival (i.e. age, gender, ECOG status, tumor location, BRAF mutational status, chemotherapy backbone, number of liver metastases) was first analyzed in separate univariate Cox models and then in a multivariate stepwise Cox model. Potential predictors with univariate p-value below 0.20 were selected to enter in the model and were kept in the multivariate model if their multivariate p-value was below 0.10. MiR-31-3p predictivity was explored in this multivariate model using an interaction term of treatment with miR-31-3p subgroup. Expression levels of miR-31-3p in LM vs PT were compared using the log of miR-31-3p expression fold and a paired Student t test. A Pearson test was performed to evaluate correlation of miR-31-3p expression in primary tumors and their paired liver metastases. Two-way analysis of variance was used to test paired primary tumor-liver metastases and treatment effects on miR-31-3p expression.</offsets></p></sec></sec><sec sec-type="supplementary-material" id="s5"><title><offsets xml_i="50844" xml_f="50874" txt_i="26549" txt_f="26579">SUPPLEMENTARY MATERIALS FIGURE</offsets></title><supplementary-material content-type="local-data" id="SD1"><media mimetype="application" mime-subtype="pdf" xlink:href="oncotarget-08-93856-s001.pdf" orientation="portrait" id="d35e1073" position="anchor"></media></supplementary-material></sec></body><back><fn-group><fn fn-type="COI-statement"><p><bold>CONFLICTS OF INTEREST</bold></p><p>IntegraGen SA performed all of the experimental and statistical analyses for the data reported. RT, MLG, FR, FL are employees of IntegraGen S.A. PLP received honoraria from IntegraGen; Merck-Serono; Amgen; Boerhinger-Ingelheim; Roche; Lilly; Sanofi. JB received honoraria from Merck-Serono; Amgen; Servier.</p></fn><fn fn-type="other"><p><bold>Assurances</bold></p><p>The New EPOC trial was approved by the South West Research Ethics Committee and MHRA in the UK and run in accordance to the EU Directive at the Southampton Clinical Trials Unit.</p><p>The New EPOC Clinical Data was provided by the Southampton CTU Statisticians which has been previously published elsewhere.</p></fn><fn fn-type="supported-by"><p><bold>FINANCIAL SUPPORT</bold></p><p>The current study was conducted by IntegraGen SA, Evry, France. The New EPOC trial was funded and supported by Cancer Research UK (C317/A7275) and was supported by a medical grant from Merck KGaA, Darmstadt, Germany. Merck KGaA reviewed the manuscript for medical accuracy only before journal submission. The authors are fully responsible for the content of this manuscript, and the views and opinions described in the publication reflect solely those of the authors.</p></fn></fn-group><ref-list><title>REFERENCES</title><ref id="R1"><label>1</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Tabernero</surname><given-names>J</given-names></name><name name-style="western"><surname>Van Cutsem</surname><given-names>E</given-names></name><name name-style="western"><surname>Diaz-Rubio</surname><given-names>E</given-names></name><name name-style="western"><surname>Cervantes</surname><given-names>A</given-names></name><name name-style="western"><surname>Humblet</surname><given-names>Y</given-names></name><name name-style="western"><surname>André</surname><given-names>T</given-names></name><name name-style="western"><surname>Van Laethem</surname><given-names>JL</given-names></name><name name-style="western"><surname>Soulié</surname><given-names>P</given-names></name><name name-style="western"><surname>Casado</surname><given-names>E</given-names></name><name name-style="western"><surname>Verslype</surname><given-names>C</given-names></name><name name-style="western"><surname>Valera</surname><given-names>JS</given-names></name><name name-style="western"><surname>Tortora</surname><given-names>G</given-names></name><name name-style="western"><surname>Ciardiello</surname><given-names>F</given-names></name><etal></etal></person-group><article-title>Phase II trial of cetuximab in combination with fluorouracil, leucovorin, and oxaliplatin in the first-line treatment of metastatic colorectal cancer</article-title><source>J Clin Oncol</source><year>2007</year><volume>25</volume><fpage>5225</fpage><lpage>32</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1200/JCO.2007.13.2183">https://doi.org/10.1200/JCO.2007.13.2183</ext-link></comment><pub-id pub-id-type="pmid">18024868</pub-id></element-citation></ref><ref id="R2"><label>2</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Cunningham</surname><given-names>D</given-names></name><name name-style="western"><surname>Humblet</surname><given-names>Y</given-names></name><name name-style="western"><surname>Siena</surname><given-names>S</given-names></name><name name-style="western"><surname>Khayat</surname><given-names>D</given-names></name><name name-style="western"><surname>Bleiberg</surname><given-names>H</given-names></name><name name-style="western"><surname>Santoro</surname><given-names>A</given-names></name><name name-style="western"><surname>Bets</surname><given-names>D</given-names></name><name name-style="western"><surname>Mueser</surname><given-names>M</given-names></name><name name-style="western"><surname>Harstrick</surname><given-names>A</given-names></name><name name-style="western"><surname>Verslype</surname><given-names>C</given-names></name><name name-style="western"><surname>Chau</surname><given-names>I</given-names></name><name name-style="western"><surname>Van Cutsem</surname><given-names>E</given-names></name></person-group><article-title>Cetuximab monotherapy and cetuximab plus irinotecan in irinotecan-refractory metastatic colorectal cancer</article-title><source>N Engl J Med</source><year>2004</year><volume>351</volume><fpage>337</fpage><lpage>45</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1056/NEJMoa033025">https://doi.org/10.1056/NEJMoa033025</ext-link></comment><pub-id pub-id-type="pmid">15269313</pub-id></element-citation></ref><ref id="R3"><label>3</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Bokemeyer</surname><given-names>C</given-names></name><name name-style="western"><surname>Bondarenko</surname><given-names>I</given-names></name><name name-style="western"><surname>Makhson</surname><given-names>A</given-names></name><name name-style="western"><surname>Hartmann</surname><given-names>JT</given-names></name><name name-style="western"><surname>Aparicio</surname><given-names>J</given-names></name><name name-style="western"><surname>de Braud</surname><given-names>F</given-names></name><name name-style="western"><surname>Donea</surname><given-names>S</given-names></name><name name-style="western"><surname>Ludwig</surname><given-names>H</given-names></name><name name-style="western"><surname>Schuch</surname><given-names>G</given-names></name><name name-style="western"><surname>Stroh</surname><given-names>C</given-names></name><name name-style="western"><surname>Loos</surname><given-names>AH</given-names></name><name name-style="western"><surname>Zubel</surname><given-names>A</given-names></name><name name-style="western"><surname>Koralewski</surname><given-names>P</given-names></name></person-group><article-title>Fluorouracil, leucovorin, and oxaliplatin with and without cetuximab in the first-line treatment of metastatic colorectal cancer</article-title><source>J Clin Oncol</source><year>2009</year><volume>27</volume><fpage>663</fpage><lpage>71</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1200/JCO.2008.20.8397">https://doi.org/10.1200/JCO.2008.20.8397</ext-link></comment><pub-id pub-id-type="pmid">19114683</pub-id></element-citation></ref><ref id="R4"><label>4</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Van Cutsem</surname><given-names>E</given-names></name><name name-style="western"><surname>Kohne</surname><given-names>CH</given-names></name><name name-style="western"><surname>Hitre</surname><given-names>E</given-names></name><name name-style="western"><surname>Zaluski</surname><given-names>J</given-names></name><name name-style="western"><surname>Chang Chien</surname><given-names>CR</given-names></name><name name-style="western"><surname>Makhson</surname><given-names>A</given-names></name><name name-style="western"><surname>D'Haens</surname><given-names>G</given-names></name><name name-style="western"><surname>Pintér</surname><given-names>T</given-names></name><name name-style="western"><surname>Lim</surname><given-names>R</given-names></name><name name-style="western"><surname>Bodoky</surname><given-names>G</given-names></name><name name-style="western"><surname>Roh</surname><given-names>JK</given-names></name><name name-style="western"><surname>Folprecht</surname><given-names>G</given-names></name><name name-style="western"><surname>Ruff</surname><given-names>P</given-names></name></person-group><article-title>Cetuximab and chemotherapy as initial treatment for metastatic colorectal cancer</article-title><source>N Engl J Med</source><year>2009</year><volume>360</volume><fpage>1408</fpage><lpage>17</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1056/NEJMoa0805019">https://doi.org/10.1056/NEJMoa0805019</ext-link></comment><pub-id pub-id-type="pmid">19339720</pub-id></element-citation></ref><ref id="R5"><label>5</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Primrose</surname><given-names>J</given-names></name><name name-style="western"><surname>Falk</surname><given-names>S</given-names></name><name name-style="western"><surname>Finch-Jones</surname><given-names>M</given-names></name><name name-style="western"><surname>Valle</surname><given-names>J</given-names></name><name name-style="western"><surname>O'Reilly</surname><given-names>D</given-names></name><name name-style="western"><surname>Siriwardena</surname><given-names>A</given-names></name><name name-style="western"><surname>Hornbuckle</surname><given-names>J</given-names></name><name name-style="western"><surname>Peterson</surname><given-names>M</given-names></name><name name-style="western"><surname>Rees</surname><given-names>M</given-names></name><name name-style="western"><surname>Iveson</surname><given-names>T</given-names></name><name name-style="western"><surname>Hickish</surname><given-names>T</given-names></name><name name-style="western"><surname>Butler</surname><given-names>R</given-names></name><name name-style="western"><surname>Stanton</surname><given-names>L</given-names></name><etal></etal></person-group><article-title>Systemic chemotherapy with or without cetuximab in patients with resectable colorectal liver metastasis: the New EPOC randomised controlled trial</article-title><source>Lancet Oncol</source><year>2014</year><volume>15</volume><fpage>601</fpage><lpage>11</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1016/S1470-2045(14)70105-6">https://doi.org/10.1016/S1470-2045(14)70105-6</ext-link></comment><pub-id pub-id-type="pmid">24717919</pub-id></element-citation></ref><ref id="R6"><label>6</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Bridgewater</surname><given-names>JA</given-names></name><name name-style="western"><surname>Pugh</surname><given-names>SA</given-names></name><name name-style="western"><surname>Moutasim</surname><given-names>K</given-names></name><name name-style="western"><surname>Thomas</surname><given-names>G</given-names></name><name name-style="western"><surname>Thiébaut</surname><given-names>R</given-names></name><name name-style="western"><surname>Liebaert</surname><given-names>F</given-names></name><name name-style="western"><surname>Falk</surname><given-names>S</given-names></name><name name-style="western"><surname>Finch-Jones</surname><given-names>M</given-names></name><name name-style="western"><surname>Valle</surname><given-names>JW</given-names></name><name name-style="western"><surname>O'Reilly</surname><given-names>D</given-names></name><name name-style="western"><surname>Siriwardena</surname><given-names>AK</given-names></name><name name-style="western"><surname>Hornbuckle</surname><given-names>J</given-names></name><name name-style="western"><surname>Rees</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Analysis of progression-free survival in the new EPOC study in an “all wild-type” population</article-title><source>J Clin Oncol</source><year>2014</year><volume>32</volume><fpage>5s (abstr 3566)</fpage></element-citation></ref><ref id="R7"><label>7</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Soriano</surname><given-names>A</given-names></name><name name-style="western"><surname>Jubierre</surname><given-names>L</given-names></name><name name-style="western"><surname>Almazan-Moga</surname><given-names>A</given-names></name><name name-style="western"><surname>Molist</surname><given-names>C</given-names></name><name name-style="western"><surname>Roma</surname><given-names>J</given-names></name><name name-style="western"><surname>de Toledo</surname><given-names>JS</given-names></name><name name-style="western"><surname>Gallego</surname><given-names>S</given-names></name><name name-style="western"><surname>Segura</surname><given-names>MF</given-names></name></person-group><article-title>microRNAs as pharmacological targets in cancer</article-title><source>Pharmacol Res</source><year>2013</year><volume>75</volume><fpage>3</fpage><lpage>14</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1016/j.phrs.2013.03.006">https://doi.org/10.1016/j.phrs.2013.03.006</ext-link></comment><pub-id pub-id-type="pmid">23537752</pub-id></element-citation></ref><ref id="R8"><label>8</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Adams</surname><given-names>BD</given-names></name><name name-style="western"><surname>Kasinski</surname><given-names>AL</given-names></name><name name-style="western"><surname>Slack</surname><given-names>FJ</given-names></name></person-group><article-title>Aberrant regulation and function of microRNAs in cancer</article-title><source>Curr Biol</source><year>2014</year><volume>24</volume><fpage>R762</fpage><lpage>76</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1016/j.cub.2014.06.043"></ext-link></comment><pub-id pub-id-type="pmid">25137592</pub-id></element-citation></ref><ref id="R9"><label>9</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Iuliano</surname><given-names>R</given-names></name><name name-style="western"><surname>Vismara</surname><given-names>MF</given-names></name><name name-style="western"><surname>Dattilo</surname><given-names>V</given-names></name><name name-style="western"><surname>Trapasso</surname><given-names>F</given-names></name><name name-style="western"><surname>Baudi</surname><given-names>F</given-names></name><name name-style="western"><surname>Perrotti</surname><given-names>N</given-names></name></person-group><article-title>The role of microRNAs in cancer susceptibility</article-title><source>Biomed Res Int</source><year>2013</year><fpage>591931</fpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1155/2013/591931">https://doi.org/10.1155/2013/591931</ext-link></comment><pub-id pub-id-type="pmid">23586049</pub-id></element-citation></ref><ref id="R10"><label>10</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Li</surname><given-names>JY</given-names></name><name name-style="western"><surname>Yong</surname><given-names>TY</given-names></name><name name-style="western"><surname>Michael</surname><given-names>MZ</given-names></name><name name-style="western"><surname>Gleadle</surname><given-names>JM</given-names></name></person-group><article-title>Review: the role of microRNAs in kidney disease</article-title><source>Nephrology (Carlton)</source><year>2010</year><volume>15</volume><fpage>599</fpage><lpage>608</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1111/j.1440-1797.2010.01363.x">https://doi.org/10.1111/j.1440-1797.2010.01363.x</ext-link></comment><pub-id pub-id-type="pmid">20883280</pub-id></element-citation></ref><ref id="R11"><label>11</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Sampson</surname><given-names>VB</given-names></name><name name-style="western"><surname>Yoo</surname><given-names>S</given-names></name><name name-style="western"><surname>Kumar</surname><given-names>A</given-names></name><name name-style="western"><surname>Vetter</surname><given-names>NS</given-names></name><name name-style="western"><surname>Kolb</surname><given-names>EA</given-names></name></person-group><article-title>MicroRNAs and potential targets in osteosarcoma: review</article-title><source>Front Pediatr</source><year>2015</year><volume>3</volume><fpage>69</fpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.3389/fped.2015.00069">https://doi.org/10.3389/fped.2015.00069</ext-link></comment><pub-id pub-id-type="pmid">26380245</pub-id></element-citation></ref><ref id="R12"><label>12</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Laurila</surname><given-names>EM</given-names></name><name name-style="western"><surname>Kallioniemi</surname><given-names>A</given-names></name></person-group><article-title>The diverse role of miR-31 in regulating cancer associated phenotypes</article-title><source>Genes Chromosomes Cancer</source><year>2013</year><volume>52</volume><fpage>1103</fpage><lpage>13</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1002/gcc.22107">https://doi.org/10.1002/gcc.22107</ext-link></comment><pub-id pub-id-type="pmid">23999990</pub-id></element-citation></ref><ref id="R13"><label>13</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Schee</surname><given-names>K</given-names></name><name name-style="western"><surname>Boye</surname><given-names>K</given-names></name><name name-style="western"><surname>Abrahamsen</surname><given-names>TW</given-names></name><name name-style="western"><surname>Fodstad</surname><given-names>O</given-names></name><name name-style="western"><surname>Flatmark</surname><given-names>K</given-names></name></person-group><article-title>Clinical relevance of microRNA miR-21, miR-31, miR-92a, miR-101, miR-106a and miR-145 in colorectal cancer</article-title><source>BMC Cancer</source><year>2012</year><volume>12</volume><fpage>505</fpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1186/1471-2407-12-505">https://doi.org/10.1186/1471-2407-12-505</ext-link></comment><pub-id pub-id-type="pmid">23121918</pub-id></element-citation></ref><ref id="R14"><label>14</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Wang</surname><given-names>CJ</given-names></name><name name-style="western"><surname>Zhou</surname><given-names>ZG</given-names></name><name name-style="western"><surname>Wang</surname><given-names>L</given-names></name><name name-style="western"><surname>Yang</surname><given-names>L</given-names></name><name name-style="western"><surname>Zhou</surname><given-names>B</given-names></name><name name-style="western"><surname>Gu</surname><given-names>J</given-names></name><name name-style="western"><surname>Chen</surname><given-names>HY</given-names></name><name name-style="western"><surname>Sun</surname><given-names>XF</given-names></name></person-group><article-title>Clinicopathological significance of microRNA-31, -143 and -145 expression in colorectal cancer</article-title><source>Dis Markers</source><year>2009</year><volume>26</volume><fpage>27</fpage><lpage>34</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.3233/DMA-2009-0601">https://doi.org/10.3233/DMA-2009-0601</ext-link></comment><pub-id pub-id-type="pmid">19242066</pub-id></element-citation></ref><ref id="R15"><label>15</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Bandres</surname><given-names>E</given-names></name><name name-style="western"><surname>Cubedo</surname><given-names>E</given-names></name><name name-style="western"><surname>Agirre</surname><given-names>X</given-names></name><name name-style="western"><surname>Malumbres</surname><given-names>R</given-names></name><name name-style="western"><surname>Zárate</surname><given-names>R</given-names></name><name name-style="western"><surname>Ramirez</surname><given-names>N</given-names></name><name name-style="western"><surname>Abajo</surname><given-names>A</given-names></name><name name-style="western"><surname>Navarro</surname><given-names>A</given-names></name><name name-style="western"><surname>Moreno</surname><given-names>I</given-names></name><name name-style="western"><surname>Monzó</surname><given-names>M</given-names></name><name name-style="western"><surname>García-Foncillas</surname><given-names>J</given-names></name></person-group><article-title>Identification by Real-time PCR of 13 mature microRNAs differentially expressed in colorectal cancer and non-tumoral tissues</article-title><source>Mol Cancer</source><year>2006</year><volume>5</volume><fpage>29</fpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1186/1476-4598-5-29">https://doi.org/10.1186/1476-4598-5-29</ext-link></comment><pub-id pub-id-type="pmid">16854228</pub-id></element-citation></ref><ref id="R16"><label>16</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Wang</surname><given-names>CJ</given-names></name><name name-style="western"><surname>Stratmann</surname><given-names>J</given-names></name><name name-style="western"><surname>Zhou</surname><given-names>ZG</given-names></name><name name-style="western"><surname>Sun</surname><given-names>XF</given-names></name></person-group><article-title>Suppression of microRNA-31 increases sensitivity to 5-FU at an early stage, and affects cell migration and invasion in HCT-116 colon cancer cells</article-title><source>BMC Cancer</source><year>2010</year><volume>10</volume><fpage>616</fpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1186/1471-2407-10-616">https://doi.org/10.1186/1471-2407-10-616</ext-link></comment><pub-id pub-id-type="pmid">21062447</pub-id></element-citation></ref><ref id="R17"><label>17</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Liu</surname><given-names>CJ</given-names></name><name name-style="western"><surname>Tsai</surname><given-names>MM</given-names></name><name name-style="western"><surname>Hung</surname><given-names>PS</given-names></name><name name-style="western"><surname>Kao</surname><given-names>SY</given-names></name><name name-style="western"><surname>Liu</surname><given-names>TY</given-names></name><name name-style="western"><surname>Wu</surname><given-names>KJ</given-names></name><name name-style="western"><surname>Chiou</surname><given-names>SH</given-names></name><name name-style="western"><surname>Lin</surname><given-names>SC</given-names></name><name name-style="western"><surname>Chang</surname><given-names>KW</given-names></name></person-group><article-title>miR-31 ablates expression of the HIF regulatory factor FIH to activate the HIF pathway in head and neck carcinoma</article-title><source>Cancer Res</source><year>2010</year><volume>70</volume><fpage>1635</fpage><lpage>44</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1158/0008-5472.CAN-09-2291">https://doi.org/10.1158/0008-5472.CAN-09-2291</ext-link></comment><pub-id pub-id-type="pmid">20145132</pub-id></element-citation></ref><ref id="R18"><label>18</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Mosakhani</surname><given-names>N</given-names></name><name name-style="western"><surname>Lahti</surname><given-names>L</given-names></name><name name-style="western"><surname>Borze</surname><given-names>I</given-names></name><name name-style="western"><surname>Karjalainen-Lindsberg</surname><given-names>ML</given-names></name><name name-style="western"><surname>Sundström</surname><given-names>J</given-names></name><name name-style="western"><surname>Ristamäki</surname><given-names>R</given-names></name><name name-style="western"><surname>Osterlund</surname><given-names>P</given-names></name><name name-style="western"><surname>Knuutila</surname><given-names>S</given-names></name><name name-style="western"><surname>Sarhadi</surname><given-names>VK</given-names></name></person-group><article-title>MicroRNA profiling predicts survival in anti-EGFR treated chemorefractory metastatic colorectal cancer patients with wild-type KRAS and BRAF</article-title><source>Cancer Genet</source><year>2012</year><volume>205</volume><fpage>545</fpage><lpage>51</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1016/j.cancergen.2012.08.003">https://doi.org/10.1016/j.cancergen.2012.08.003</ext-link></comment><pub-id pub-id-type="pmid">23098991</pub-id></element-citation></ref><ref id="R19"><label>19</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Manceau</surname><given-names>G</given-names></name><name name-style="western"><surname>Imbeaud</surname><given-names>S</given-names></name><name name-style="western"><surname>Thiébaut</surname><given-names>R</given-names></name><name name-style="western"><surname>Liébaert</surname><given-names>F</given-names></name><name name-style="western"><surname>Fontaine</surname><given-names>K</given-names></name><name name-style="western"><surname>Rousseau</surname><given-names>F</given-names></name><name name-style="western"><surname>Génin</surname><given-names>B</given-names></name><name name-style="western"><surname>Le Corre</surname><given-names>D</given-names></name><name name-style="western"><surname>Didelot</surname><given-names>A</given-names></name><name name-style="western"><surname>Vincent</surname><given-names>M</given-names></name><name name-style="western"><surname>Bachet</surname><given-names>JB</given-names></name><name name-style="western"><surname>Chibaudel</surname><given-names>B</given-names></name><name name-style="western"><surname>Bouché</surname><given-names>O</given-names></name><etal></etal></person-group><article-title>Hsa-miR-31-3p expression is linked to progression-free survival in patients with KRAS wild-type metastatic colorectal cancer treated with anti-EGFR therapy</article-title><source>Clin Cancer Res</source><year>2014</year><volume>20</volume><fpage>3338</fpage><lpage>47</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1158/1078-0432.CCR-13-2750">https://doi.org/10.1158/1078-0432.CCR-13-2750</ext-link></comment><pub-id pub-id-type="pmid">24771647</pub-id></element-citation></ref><ref id="R20"><label>20</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Mlcochova</surname><given-names>J</given-names></name><name name-style="western"><surname>Faltejskova-Vychytilova</surname><given-names>P</given-names></name><name name-style="western"><surname>Ferracin</surname><given-names>M</given-names></name><name name-style="western"><surname>Zagatti</surname><given-names>B</given-names></name><name name-style="western"><surname>Radova</surname><given-names>L</given-names></name><name name-style="western"><surname>Svoboda</surname><given-names>M</given-names></name><name name-style="western"><surname>Nemecek</surname><given-names>R</given-names></name><name name-style="western"><surname>John</surname><given-names>S</given-names></name><name name-style="western"><surname>Kiss</surname><given-names>I</given-names></name><name name-style="western"><surname>Vyzula</surname><given-names>R</given-names></name><name name-style="western"><surname>Negrini</surname><given-names>M</given-names></name><name name-style="western"><surname>Slaby</surname><given-names>O</given-names></name></person-group><article-title>MicroRNA expression profiling identifies miR-31-5p/3p as associated with time to progression in wild-type RAS metastatic colorectal cancer treated with cetuximab</article-title><source>Oncotarget</source><year>2015</year><volume>6</volume><fpage>38695</fpage><lpage>704</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.18632/oncotarget.5735">https://doi.org/10.18632/oncotarget.5735</ext-link></comment><pub-id pub-id-type="pmid">26497852</pub-id></element-citation></ref><ref id="R21"><label>21</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Peterson</surname><given-names>B</given-names></name><name name-style="western"><surname>George</surname><given-names>SL</given-names></name></person-group><article-title>Sample size requirements and length of study for testing interaction in a 2 x k factorial design when time-to-failure is the outcome [corrected]</article-title><source>Control Clin Trials</source><year>1993</year><volume>14</volume><fpage>511</fpage><lpage>22</lpage><pub-id pub-id-type="pmid">8119066</pub-id></element-citation></ref><ref id="R22"><label>22</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Laurent-Puig</surname><given-names>P</given-names></name><name name-style="western"><surname>Paget-Bailly</surname><given-names>S</given-names></name><name name-style="western"><surname>Vernerey</surname><given-names>D</given-names></name><name name-style="western"><surname>Vazart</surname><given-names>C</given-names></name><name name-style="western"><surname>Decaulne</surname><given-names>V</given-names></name><name name-style="western"><surname>Fontaine</surname><given-names>K</given-names></name><name name-style="western"><surname>Rousseau</surname><given-names>F</given-names></name><name name-style="western"><surname>Elliott</surname><given-names>F</given-names></name><name name-style="western"><surname>Quirke</surname><given-names>P</given-names></name><name name-style="western"><surname>Richman</surname><given-names>S</given-names></name><name name-style="western"><surname>Seligman</surname><given-names>J</given-names></name><name name-style="western"><surname>Seymour</surname><given-names>MT</given-names></name><name name-style="western"><surname>Bonnetain</surname><given-names>F</given-names></name><etal></etal></person-group><article-title>Evaluation of miR 31 3p as a biomarker of prognosis and panitumumab benefit in RAS-wt advanced colorectal cancer (aCRC): analysis of patients (pts) from the PICCOLO trial</article-title><source>J Clin Oncol</source><year>2015</year><volume>33</volume><fpage>3547</fpage></element-citation></ref><ref id="R23"><label>23</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Laurent-Puig</surname><given-names>P</given-names></name><name name-style="western"><surname>Grisoni</surname><given-names>ML</given-names></name><name name-style="western"><surname>Heinemann</surname><given-names>V</given-names></name><name name-style="western"><surname>Bonnetain</surname><given-names>F</given-names></name><name name-style="western"><surname>Fontaine</surname><given-names>K</given-names></name><name name-style="western"><surname>Vazar</surname><given-names>C</given-names></name><name name-style="western"><surname>Decaulne</surname><given-names>V</given-names></name><name name-style="western"><surname>Rousseau</surname><given-names>F</given-names></name><name name-style="western"><surname>Courtieu</surname><given-names>B</given-names></name><name name-style="western"><surname>Liebaert</surname><given-names>F</given-names></name><name name-style="western"><surname>Jung</surname><given-names>A</given-names></name><name name-style="western"><surname>Neureiter</surname><given-names>D</given-names></name><name name-style="western"><surname>Thiébaut</surname><given-names>R</given-names></name><etal></etal></person-group><article-title>miR-31-3p as a predictive biomarker of cetuximab efficacy effect in metastatic colorectal cancer (mCRC) patients enrolled in FIRE-3 study</article-title><source>J Clin Oncol</source><year>2016</year><volume>34</volume><fpage>3216</fpage></element-citation></ref><ref id="R24"><label>24</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Shen</surname><given-names>J</given-names></name><name name-style="western"><surname>Xia</surname><given-names>W</given-names></name><name name-style="western"><surname>Khotskaya</surname><given-names>YB</given-names></name><name name-style="western"><surname>Huo</surname><given-names>L</given-names></name><name name-style="western"><surname>Nakanishi</surname><given-names>K</given-names></name><name name-style="western"><surname>Lim</surname><given-names>SO</given-names></name><name name-style="western"><surname>Du</surname><given-names>Y</given-names></name><name name-style="western"><surname>Wang</surname><given-names>Y</given-names></name><name name-style="western"><surname>Chang</surname><given-names>WC</given-names></name><name name-style="western"><surname>Chen</surname><given-names>CH</given-names></name><name name-style="western"><surname>Hsu</surname><given-names>JL</given-names></name><name name-style="western"><surname>Wu</surname><given-names>Y</given-names></name><name name-style="western"><surname>Lam</surname><given-names>YC</given-names></name><etal></etal></person-group><article-title>EGFR modulates microRNA maturation in response to hypoxia through phosphorylation of AGO2</article-title><source>Nature</source><year>2013</year><volume>497</volume><fpage>383</fpage><lpage>7</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1038/nature12080">https://doi.org/10.1038/nature12080</ext-link></comment><pub-id pub-id-type="pmid">23636329</pub-id></element-citation></ref><ref id="R25"><label>25</label><element-citation publication-type="journal"><person-group person-group-type="author"><name name-style="western"><surname>Freidlin</surname><given-names>B</given-names></name><name name-style="western"><surname>Korn</surname><given-names>EL</given-names></name><name name-style="western"><surname>Gray</surname><given-names>R</given-names></name></person-group><article-title>A general inefficacy interim monitoring rule for randomized clinical trials</article-title><source>Clin Trials</source><year>2010</year><volume>7</volume><fpage>197</fpage><lpage>208</lpage><comment><ext-link ext-link-type="uri" xlink:href="https://doi.org/10.1177/1740774510369019">https://doi.org/10.1177/1740774510369019</ext-link></comment><pub-id pub-id-type="pmid">20423925</pub-id></element-citation></ref></ref-list></back></article>