<article xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article"><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Cancer</journal-id><journal-id journal-id-type="iso-abbrev">BMC Cancer</journal-id><journal-title-group><journal-title>BMC Cancer</journal-title></journal-title-group><issn pub-type="epub">1471-2407</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta><article-meta><article-id pub-id-type="pmid">28193205</article-id><article-id pub-id-type="pmc">5307758</article-id><article-id pub-id-type="publisher-id">3111</article-id><article-id pub-id-type="doi">10.1186/s12885-017-3111-1</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research Article</subject></subj-group></article-categories><title-group><article-title>Comparison of immunohistochemistry with PCR for assessment of ER, PR, and Ki-67 and prediction of pathological complete response in breast cancer</article-title></title-group><contrib-group><contrib contrib-type="author" corresp="yes"><name><surname>Sinn</surname><given-names>Hans-Peter</given-names></name><address><phone>+49 6221 56-7931</phone><email>peter.sinn@med.uni-heidelberg.de</email></address><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Schneeweiss</surname><given-names>Andreas</given-names></name><address><email>Andreas.Schneeweiss@med.uni-heidelberg.de</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Keller</surname><given-names>Marius</given-names></name><address><email>Marius.Keller@med.uni-heidelberg.de</email></address><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Schlombs</surname><given-names>Kornelia</given-names></name><address><email>Kornelia.Schlombs@biontechdiagnostics.de</email></address><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Laible</surname><given-names>Mark</given-names></name><address><email>Mark.Laible@biontechdiagnostics.de</email></address><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Seitz</surname><given-names>Julia</given-names></name><address><email>Julia.seitz@med.uni-heidelberg.de</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Lakis</surname><given-names>Sotirios</given-names></name><address><email>sotiris.lakis@stratifyer.de</email></address><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Veltrup</surname><given-names>Elke</given-names></name><address><email>elke.veltrup@stratifyer.de</email></address><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Altevogt</surname><given-names>Peter</given-names></name><address><email>p.altevogt@dkfz-heidelberg.de</email></address><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Eidt</surname><given-names>Sebastian</given-names></name><address><email>sebastian.eidt@gmx.de</email></address><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Wirtz</surname><given-names>Ralph M.</given-names></name><address><email>ralph.wirtz@stratifyer.de</email></address><xref ref-type="aff" rid="Aff4">4</xref><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Marmé</surname><given-names>Frederik</given-names></name><address><email>frederik.marme@med.uni-heidelberg.de</email></address><xref ref-type="aff" rid="Aff2">2</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 2190 4373</institution-id><institution-id institution-id-type="GRID">grid.7700.0</institution-id><institution>Institute of Pathology, </institution><institution>University of Heidelberg, </institution></institution-wrap>Im Neuenheimer Feld 220-221, 69120 Heidelberg, Germany </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0001 0328 4908</institution-id><institution-id institution-id-type="GRID">grid.5253.1</institution-id><institution>National Center for Tumor Diseases, </institution><institution>University-Hospital Heidelberg, </institution></institution-wrap>Im Neuenheimer Feld 460, 69120 Heidelberg, Germany </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ISNI">0000 0004 0492 0584</institution-id><institution-id institution-id-type="GRID">grid.7497.d</institution-id><institution></institution><institution>German Cancer Research Center, </institution></institution-wrap>Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </aff><aff id="Aff4"><label>4</label>STRATIFYER Molecular Pathology GmbH, Werthmannstr. 1c, 50935 Köln, Germany </aff><aff id="Aff5"><label>5</label>Department of Pathology, St. Elisabeth-Krankenhaus, Werthmannstr. 1c, 50935 Köln, Germany </aff><aff id="Aff6"><label>6</label>BioNTech Diagnostics GmbH, 55131 Mainz, Germany </aff></contrib-group><pub-date pub-type="epub"><day>13</day><month>2</month><year>2017</year></pub-date><pub-date pub-type="pmc-release"><day>13</day><month>2</month><year>2017</year></pub-date><pub-date pub-type="collection"><year>2017</year></pub-date><volume>17</volume><elocation-id>124</elocation-id><history><date date-type="received"><day>8</day><month>4</month><year>2016</year></date><date date-type="accepted"><day>4</day><month>2</month><year>2017</year></date></history><permissions><copyright-statement>© The Author(s). 2017</copyright-statement><license license-type="OpenAccess"><license-p>
<bold>Open Access</bold>This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">http://creativecommons.org/licenses/by/4.0/</ext-link>), which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The Creative Commons Public Domain Dedication waiver (<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">http://creativecommons.org/publicdomain/zero/1.0/</ext-link>) applies to the data made available in this article, unless otherwise stated.</license-p></license></permissions><abstract id="Abs1"><sec><title><offsets xml_i="6746" xml_f="6756" txt_i="11" txt_f="21">Background</offsets></title><p><offsets xml_i="6767" xml_f="7183" txt_i="22" txt_f="438">Proliferation may predict response to neoadjuvant therapy of breast cancer and is commonly assessed by manual scoring of slides stained by immunohistochemistry (IHC) for Ki-67 similar to ER and PgR. This method carries significant intra- and inter-observer variability. Automatic scoring of Ki-67 with digital image analysis (qIHC) or assessment of MKI67 gene expression with RT-qPCR may improve diagnostic accuracy.</offsets></p></sec><sec><title><offsets xml_i="7205" xml_f="7212" txt_i="440" txt_f="447">Methods</offsets></title><p><offsets xml_i="7223" xml_f="7745" txt_i="448" txt_f="970">Ki-67 IHC visual assessment was compared to the IHC nuclear tool (AperioTM) on core biopsies from a randomized neoadjuvant clinical trial. Expression of ESR1, PGR and MKI67 by RT-qPCR was performed on RNA extracted from the same formalin-fixed paraffin-embedded tissue. Concordance between the three methods (vIHC, qIHC and RT-qPCR) was assessed for all 3 markers. The potential of Ki-67 IHC and RT-qPCR to predict pathological complete response (pCR) was evaluated using ROC analysis and non-parametric Mann-Whitney Test.</offsets></p></sec><sec><title><offsets xml_i="7767" xml_f="7774" txt_i="972" txt_f="979">Results</offsets></title><p><offsets xml_i="7785" xml_f="7871" txt_i="980" txt_f="1066">Correlation between methods (qIHC versus RT-qPCR) was high for ER and PgR (spearman´s </offsets><italic><offsets xml_i="7879" xml_f="7880" txt_i="1066" txt_f="1067">r</offsets></italic><offsets xml_i="7889" xml_f="7898" txt_i="1067" txt_f="1076"> = 0.82, </offsets><italic><offsets xml_i="7906" xml_f="7907" txt_i="1076" txt_f="1077">p</offsets></italic><offsets xml_i="7916" xml_f="7933" txt_i="1077" txt_f="1091"> &lt; 0.0001 and </offsets><italic><offsets xml_i="7941" xml_f="7942" txt_i="1091" txt_f="1092">r</offsets></italic><offsets xml_i="7951" xml_f="7960" txt_i="1092" txt_f="1101"> = 0.86, </offsets><italic><offsets xml_i="7968" xml_f="7969" txt_i="1101" txt_f="1102">p</offsets></italic><offsets xml_i="7978" xml_f="8440" txt_i="1102" txt_f="1561"> &lt; 0.0001, respectively) resulting in high levels of concordance using predefined cut-offs. When comparing qIHC of ER and PgR with RT-qPCR of ESR1 and PGR the overall agreement was 96.6 and 91.4%, respectively, while overall agreement of visual IHC with RT-qPCR was slightly lower for ER/ESR1 and PR/PGR (91.2 and 92.9%, respectively). In contrast, only a moderate correlation was observed between qIHC and RT-qPCR continuous data for Ki-67/MKI67 (Spearman’s </offsets><italic><offsets xml_i="8448" xml_f="8449" txt_i="1561" txt_f="1562">r</offsets></italic><offsets xml_i="8458" xml_f="8467" txt_i="1562" txt_f="1571"> = 0.50, </offsets><italic><offsets xml_i="8475" xml_f="8476" txt_i="1571" txt_f="1572">p</offsets></italic><offsets xml_i="8485" xml_f="8925" txt_i="1572" txt_f="2012"> = 0.0001). Up to now no predictive cut-off for Ki-67 assessment by IHC has been established to predict response to neoadjuvant chemotherapy. Setting the desired sensitivity at 100%, specificity for the prediction of pCR (ypT0ypN0) was significantly higher for mRNA than for protein (68.9% vs. 22.2%). Moreover, the proliferation levels in patients achieving a pCR versus not differed significantly using MKI67 RNA expression (Mann-Whitney </offsets><italic><offsets xml_i="8933" xml_f="8934" txt_i="2012" txt_f="2013">p</offsets></italic><offsets xml_i="8943" xml_f="8995" txt_i="2013" txt_f="2065"> = 0.002), but not with qIHC of Ki-67 (Mann-Whitney </offsets><italic><offsets xml_i="9003" xml_f="9004" txt_i="2065" txt_f="2066">p</offsets></italic><offsets xml_i="9013" xml_f="9041" txt_i="2066" txt_f="2094"> = 0.097) or vIHC of Ki-67 (</offsets><italic><offsets xml_i="9049" xml_f="9050" txt_i="2094" txt_f="2095">p</offsets></italic><offsets xml_i="9059" xml_f="9069" txt_i="2095" txt_f="2105"> = 0.131).</offsets></p></sec><sec><title><offsets xml_i="9091" xml_f="9101" txt_i="2107" txt_f="2117">Conclusion</offsets></title><p><offsets xml_i="9112" xml_f="9595" txt_i="2118" txt_f="2601">Digital image analysis can successfully be implemented for assessing ER, PR and Ki-67. IHC for ER and PR reveals high concordance with RT-qPCR. However, RT-qPCR displays a broader dynamic range and higher sensitivity than IHC. Moreover, correlation between Ki-67 qIHC and RT-qPCR is only moderate and RT-qPCR with MammaTyper® outperforms qIHC in predicting pCR. Both methods yield improvements to error-prone manual scoring of Ki-67. However, RT-qPCR was significantly more specific.</offsets></p></sec><sec><title><offsets xml_i="9617" xml_f="9650" txt_i="2603" txt_f="2636">Electronic supplementary material</offsets></title><p><offsets xml_i="9661" xml_f="9800" txt_i="2637" txt_f="2776">The online version of this article (doi:10.1186/s12885-017-3111-1) contains supplementary material, which is available to authorized users.</offsets></p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Image analysis</kwd><kwd>Breast cancer</kwd><kwd>Ki67</kwd><kwd>mRNA</kwd><kwd>RT-qPCR</kwd><kwd>Prediction</kwd><kwd>Pathologic complete response</kwd><kwd>neoadjuvant</kwd><kwd>Immunohistochemistry (IHC)</kwd><kwd>MammaTyper®</kwd></kwd-group><funding-group><award-group><funding-source><institution>Dietmar Hopp-Stiftung</institution></funding-source><award-id>23011195</award-id></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>© The Author(s) 2017</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1" sec-type="introduction"><title><offsets xml_i="10521" xml_f="10531" txt_i="2785" txt_f="2795">Background</offsets></title><p><offsets xml_i="10542" xml_f="10799" txt_i="2796" txt_f="3053">The proliferative activity of individual cells is a hallmark of tumor biological aggressiveness and a key determinant of sensitivity to (neo)adjuvant chemotherapy, thus being among the principal factors guiding clinical management in primary breast cancer [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="10831" xml_f="10832" txt_i="3053" txt_f="3054">1</offsets></xref><offsets xml_i="10839" xml_f="10841" txt_i="3054" txt_f="3056">, </offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="10873" xml_f="10874" txt_i="3056" txt_f="3057">2</offsets></xref><offsets xml_i="10881" xml_f="11005" txt_i="3057" txt_f="3181">]. The most widely used method to assess proliferation as well as hormone receptor expression is immunohistochemistry (IHC).</offsets></p><p><offsets xml_i="11012" xml_f="11230" txt_i="3182" txt_f="3400">Nuclear staining of the nuclear antigen Ki-67 is most widely used as a surrogate for proliferative activity. Ki-67 is present in the cell nucleus throughout all stages of the cell-cycle excluding the resting phase G0 [</offsets><xref ref-type="bibr" rid="CR3"><offsets xml_i="11262" xml_f="11263" txt_i="3400" txt_f="3401">3</offsets></xref><offsets xml_i="11270" xml_f="11499" txt_i="3401" txt_f="3630">]. The recently proposed St Gallen recommendations for the identification of the intrinsic subtypes using surrogate pathologic-based definitions have underlined the value of Ki-67 as a clinical tool in routine clinical practice [</offsets><xref ref-type="bibr" rid="CR2"><offsets xml_i="11531" xml_f="11532" txt_i="3630" txt_f="3631">2</offsets></xref><offsets xml_i="11539" xml_f="11699" txt_i="3631" txt_f="3791">]. Ki-67 is recommended as a valuable factor to distinguish between Luminal A- and B-like tumors, a fundamental distinction in clinical decision-making today. [</offsets><xref ref-type="bibr" rid="CR4"><offsets xml_i="11731" xml_f="11732" txt_i="3791" txt_f="3792">4</offsets></xref><offsets xml_i="11739" xml_f="11740" txt_i="3792" txt_f="3793">–</offsets><xref ref-type="bibr" rid="CR6"><offsets xml_i="11772" xml_f="11773" txt_i="3793" txt_f="3794">6</offsets></xref><offsets xml_i="11780" xml_f="11782" txt_i="3794" txt_f="3796">].</offsets></p><p><offsets xml_i="11789" xml_f="12089" txt_i="3797" txt_f="4097">Despite its widespread use, driven by the premise of solving delicate therapeutic dilemmas combined with several advantages such as universal accessibility, easy application and low cost, the assessment of Ki-67, ER and PR is affected by technical and observer-based variabilities of the IHC method [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="12121" xml_f="12122" txt_i="4097" txt_f="4098">7</offsets></xref><offsets xml_i="12129" xml_f="12131" txt_i="4098" txt_f="4100">, </offsets><xref ref-type="bibr" rid="CR8"><offsets xml_i="12163" xml_f="12164" txt_i="4100" txt_f="4101">8</offsets></xref><offsets xml_i="12171" xml_f="12507" txt_i="4101" txt_f="4437">]. This can be illustrated by observations, that tumors with as little as 1% positive nuclei still respond to anti-hormonal treatment, which indicates that tumor cells lacking nuclear ER staining within in these tumors do have some extend of ER expression rendering them sensitive to estrogen deprivation or estrogen receptor blockade [</offsets><xref ref-type="bibr" rid="CR9"><offsets xml_i="12539" xml_f="12540" txt_i="4437" txt_f="4438">9</offsets></xref><offsets xml_i="12547" xml_f="12683" txt_i="4438" txt_f="4574">]. While the clinical role particularly of ER testing by IHC is well established, the clinical utility of Ki-67 is still controversial [</offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="12716" xml_f="12718" txt_i="4574" txt_f="4576">10</offsets></xref><offsets xml_i="12725" xml_f="12846" txt_i="4576" txt_f="4697">]. The reason lies in a series of analytical and preanalytical factors, but also in staining interpretation and scoring [</offsets><xref ref-type="bibr" rid="CR11"><offsets xml_i="12879" xml_f="12881" txt_i="4697" txt_f="4699">11</offsets></xref><offsets xml_i="12888" xml_f="13098" txt_i="4699" txt_f="4909">]. Importantly, attempts to reduce the high discordance rates either by means of formal counting quantification methods (as opposed to simple eyeballing) or by training of individuals have not been successful [</offsets><xref ref-type="bibr" rid="CR12"><offsets xml_i="13131" xml_f="13133" txt_i="4909" txt_f="4911">12</offsets></xref><offsets xml_i="13140" xml_f="13142" txt_i="4911" txt_f="4913">].</offsets></p><p><offsets xml_i="13149" xml_f="13374" txt_i="4914" txt_f="5139">Despite methodological concerns, overall a strong correlation of Ki-67 with breast cancer outcome is sufficiently supported, particularly by data originating from randomized clinical trials with central review of biomarkers [</offsets><xref ref-type="bibr" rid="CR10"><offsets xml_i="13407" xml_f="13409" txt_i="5139" txt_f="5141">10</offsets></xref><offsets xml_i="13416" xml_f="13585" txt_i="5141" txt_f="5310">]. This has also been shown in the neoadjuvant setting, where higher Ki-67 values are consistently associated with higher rates of pathological complete response (pCR) [</offsets><xref ref-type="bibr" rid="CR13"><offsets xml_i="13618" xml_f="13620" txt_i="5310" txt_f="5312">13</offsets></xref><offsets xml_i="13627" xml_f="13821" txt_i="5312" txt_f="5506">], a finding which reflects the fundamental link between tumor replication fraction and activity of cytotoxic agents. Still it remains difficult to identify a reasonable cut-off to predict pCR [</offsets><xref ref-type="bibr" rid="CR14"><offsets xml_i="13854" xml_f="13856" txt_i="5506" txt_f="5508">14</offsets></xref><offsets xml_i="13863" xml_f="13865" txt_i="5508" txt_f="5510">].</offsets></p><p><offsets xml_i="13872" xml_f="14364" txt_i="5511" txt_f="6003">Two techniques which could circumvent the inter- or intra-observer variability of Ki-67 manual microscopic assessment are automated image analysis and reverse transcription quantitative real-time PCR (RT-qPCR). A trained human eye may achieve an excellent understanding of images and patterns, but is less accurate when it comes to quantification. Computer-based vision methods could represent a solution to this problem by offering standardized image processing and reliable quantification [</offsets><xref ref-type="bibr" rid="CR15"><offsets xml_i="14397" xml_f="14399" txt_i="6003" txt_f="6005">15</offsets></xref><offsets xml_i="14406" xml_f="14728" txt_i="6005" txt_f="6327">]. However with regard to the assessment of tumor proliferation, the areas of interest and staining intensities have to be defined, and measured reproducibly. Also, it is still under debate how to deal with areas of increased proliferative activity (hot spots), and if low intensity staining should be taken into account [</offsets><xref ref-type="bibr" rid="CR16"><offsets xml_i="14761" xml_f="14763" txt_i="6327" txt_f="6329">16</offsets></xref><offsets xml_i="14770" xml_f="14952" txt_i="6329" txt_f="6511">]. Therefore, automated estimations of Ki-67 are highly correlated with manual assessments, but it is not yet certain whether or not they can improve prediction and prognostication [</offsets><xref ref-type="bibr" rid="CR17"><offsets xml_i="14985" xml_f="14987" txt_i="6511" txt_f="6513">17</offsets></xref><offsets xml_i="14994" xml_f="14995" txt_i="6513" txt_f="6514">–</offsets><xref ref-type="bibr" rid="CR19"><offsets xml_i="15028" xml_f="15030" txt_i="6514" txt_f="6516">19</offsets></xref><offsets xml_i="15037" xml_f="15039" txt_i="6516" txt_f="6518">].</offsets></p><p><offsets xml_i="15046" xml_f="15387" txt_i="6519" txt_f="6860">RT-qPCR has a series of widely acknowledged methodological advantages over IHC, which appear particularly beneficial in the context of reducing the bias of routine Ki67 assessment; it is quantitative by nature with much wider dynamic range, it does not require an experienced eye, and results are not affected by subjective interpretations [</offsets><xref ref-type="bibr" rid="CR20"><offsets xml_i="15420" xml_f="15422" txt_i="6860" txt_f="6862">20</offsets></xref><offsets xml_i="15429" xml_f="15764" txt_i="6862" txt_f="7197">]. Moreover, access to standardized protocols and automation ensures accurate performance and fast turn-around. In recent years highly specific and sensitive techniques have been developed, which allow for fast and efficient extraction of high-quality nucleic acids from FFPE overcoming the challenges posed by fixation and embedding [</offsets><xref ref-type="bibr" rid="CR21"><offsets xml_i="15797" xml_f="15799" txt_i="7197" txt_f="7199">21</offsets></xref><offsets xml_i="15806" xml_f="15808" txt_i="7199" txt_f="7201">].</offsets></p><p><offsets xml_i="15815" xml_f="16348" txt_i="7202" txt_f="7735">Validation of the various available methods for the assessment of Ki-67 requires comparative testing preferably in a specifically defined clinical context. In the present study we used the neoadjuvant setting of a phase II trial randomizing patients receiving anthracyclin/taxane based standard treatment between pemetrexed and cyclophospamide in order to directly compare the assessment of Ki-67 with automatic, quantitative read-out of IHC (qIHC) and the determination of tumor MKI67 mRNA with RT-qPCR on FFPE tissue extracted RNA.</offsets></p></sec><sec id="Sec2" sec-type="materials|methods"><title><offsets xml_i="16409" xml_f="16416" txt_i="7737" txt_f="7744">Methods</offsets></title><sec id="Sec3"><title><offsets xml_i="16446" xml_f="16462" txt_i="7745" txt_f="7761">Study population</offsets></title><p><offsets xml_i="16473" xml_f="16684" txt_i="7762" txt_f="7973">Core needle biopsies from 101 out of 105 patients (96,2%) with primary invasive breast cancer, that had been enrolled in the H3E-MC-S080 (NCT00149214, Sponsor: Eli Lilly and Company) neoadjuvant phase II study [</offsets><xref ref-type="bibr" rid="CR1"><offsets xml_i="16716" xml_f="16717" txt_i="7973" txt_f="7974">1</offsets></xref><offsets xml_i="16724" xml_f="17510" txt_i="7974" txt_f="8760">], were obtained. All patients had been diagnosed with operable (T2-T4/N0-2/M0) breast cancer at a single institution (National Center for Tumor Diseases, University-Hospital, Heidelberg) had been randomized to receive sequential anthracycline/taxane-based regimens containing either pemetrexed or cyclophosphamide in combination with epirubicin. A written informed consent for the research use of patient biological material was granted at the time of enrolment. The study was approved by the local ethics committee. Complete molecular data (including RT-qPCR data) and clinical follow-up information were available in 83 out of 105 (79%) patients (statistics data set #1). Ki-67, ER, and PR IHC slides were available in 54 (51%) patients for quantitative IHC (statistics data set #2).</offsets></p></sec><sec id="Sec4"><title><offsets xml_i="17542" xml_f="17564" txt_i="8762" txt_f="8784">Isolation of tumor RNA</offsets></title><p><offsets xml_i="17575" xml_f="18338" txt_i="8785" txt_f="9548">For RNA extraction from FFPE tissue, a single 10 μm curl was processed according to a commercially available bead-based extraction method (RNXtract® kit; BioNTech Diagnostics GmbH, Mainz, Germany). In brief, a lysis buffer was used to liquefy FFPE tissue slices while melting of paraffin was carried out in a thermo-mixer. Tissue lysis was accomplished with a proteinase K solution. Thereafter, lysates were admixed with germanium-coated magnetic particles in the presence of special buffers, which promote the binding of nucleic acids. Purification was carried out by means of consecutive cycles of mixing, magnetization, centrifugation and removal of contaminants. RNA was eluated with 100 μl elution buffer and RNA eluates were then stored at −80 °C until use.</offsets></p></sec><sec id="Sec5"><title><offsets xml_i="18370" xml_f="18396" txt_i="9550" txt_f="9576">Gene expression by RT-qPCR</offsets></title><p><offsets xml_i="18407" xml_f="19078" txt_i="9577" txt_f="10248">The MammaTyper® is a molecular in vitro diagnostic tool for the assessment of the gene expression levels of the four cancer biomarkers that are required for the clinical management of breast cancer patients in daily routine clinical practice. Instead of using IHC to assess protein expression of HER2, ERα, PR, and Ki-67, with MammaTyper®, it is possible to measure the mRNA transcripts of the corresponding genes (ERBB2, ESR1, PGR, and MKI67), doing so by using routine FFPE material and by achieving accurate, reproducible and objective results. The gene expression data may be then integrated so as to assign individual samples to a molecular subtype of breast cancer.</offsets></p><p><offsets xml_i="19085" xml_f="19624" txt_i="10249" txt_f="10788">The mRNA expression levels of ERBB2, ESR1, PGR, and MKI67 as well as of two reference genes (REF), namely B2M and CALM2, were determined by RT-qPCR, which involves reverse transcription of RNA and subsequent amplification of cDNA executed successively as a 1-step reaction. In MammaTyper®, the 6 assays (assay = primer pair and probe specific for the respective target sequence) are duplexed into three assay mixes, each using a pair of hydrolysis probes labelled with different fluorophores for separate detection of the duplexed assays [</offsets><xref ref-type="bibr" rid="CR22"><offsets xml_i="19657" xml_f="19659" txt_i="10788" txt_f="10790">22</offsets></xref><offsets xml_i="19666" xml_f="19668" txt_i="10790" txt_f="10792">].</offsets></p><p><offsets xml_i="19675" xml_f="20061" txt_i="10793" txt_f="11179">Each patient sample or control was analyzed with each assay mix in triplicates. The experiments were run on a Versant kPCR Molecular System (Siemens Healthcare, Erlangen, Germany) according to the following protocol: 5 min at 50 ° C, 20 sec at 95 ° C followed by 40 cycles of 15 sec at 95 ° C and 60 s at 60 ° C and according to MammaTyper® instructions for use 140603-90020-EU Rev 2.0.</offsets></p><p><offsets xml_i="20068" xml_f="20818" txt_i="11180" txt_f="11930">Forty amplification cycles were applied and the cycle quantification threshold (Cq) values of MKI67 and the two REF genes for each sample (S) were estimated as the median of the triplicate measurements. These were then normalized against the mean expression of the REF genes and set off against a calibrator (PC), to correct for inter-run variations (ΔΔCq method) (Livak et al. 2001). The final values were generated by subtracting ΔΔCq from the total number of cycles to ensure that normalized gene expression obtained by the test is proportional to the corresponding mRNA expression levels, a method that facilitates interpretation of data and clinicopathological correlations. The various calculation steps are summarized in the following formula:</offsets><disp-formula id="Equa"><alternatives><tex-math id="M1"><offsets xml_i="20874" xml_f="21524" txt_i="11930" txt_f="12580">\documentclass[12pt]{minimal}
				\usepackage{amsmath}
				\usepackage{wasysym} 
				\usepackage{amsfonts} 
				\usepackage{amssymb} 
				\usepackage{amsbsy}
				\usepackage{mathrsfs}
				\usepackage{upgreek}
				\setlength{\oddsidemargin}{-69pt}
				\begin{document}$$ 40\hbox{-} \Delta \Delta \mathrm{Cq}\left(\mathrm{MKI}67\right)\mathrm{S} = 40\hbox{-} \left(\left(\mathrm{Cq}\left[\mathrm{MKI}67\right]\mathrm{S}\ \hbox{--}\ \mathrm{meanCq}\left[\mathrm{REF}\right]\mathrm{S}\right)\ \hbox{--}\ \left(\mathrm{Cq}\left[\mathrm{MKI}67\right]\mathrm{pc}\ \hbox{--}\ \mathrm{meanCq}\left[\mathrm{REF}\right]\mathrm{pc}\right)\right) $$\end{document}</offsets></tex-math><mml:math id="M2"><mml:mn><offsets xml_i="21561" xml_f="21563" txt_i="12580" txt_f="12582">40</offsets></mml:mn><mml:mo><offsets xml_i="21580" xml_f="21581" txt_i="12582" txt_f="12583">‐</offsets></mml:mo><mml:mi mathvariant="normal"><offsets xml_i="21619" xml_f="21623" txt_i="12583" txt_f="12587">ΔΔCq</offsets></mml:mi><mml:mfenced close=")" open="("><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="21703" xml_f="21704" txt_i="12587" txt_f="12588">M</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="21742" xml_f="21743" txt_i="12588" txt_f="12589">K</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="21781" xml_f="21782" txt_i="12589" txt_f="12590">I</offsets></mml:mi><mml:mn><offsets xml_i="21799" xml_f="21801" txt_i="12590" txt_f="12592">67</offsets></mml:mn></mml:mrow></mml:mfenced><mml:mi mathvariant="normal"><offsets xml_i="21864" xml_f="21865" txt_i="12592" txt_f="12593">S</offsets></mml:mi><mml:mspace width="0.25em"></mml:mspace><mml:mo><offsets xml_i="21922" xml_f="21923" txt_i="12593" txt_f="12594">=</offsets></mml:mo><mml:mspace width="0.25em"></mml:mspace><mml:mn><offsets xml_i="21980" xml_f="21982" txt_i="12594" txt_f="12596">40</offsets></mml:mn><mml:mo><offsets xml_i="21999" xml_f="22000" txt_i="12596" txt_f="12597">‐</offsets></mml:mo><mml:mfenced close=")" open="("><mml:mrow><mml:mfenced close=")" open="("><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="22122" xml_f="22123" txt_i="12597" txt_f="12598">C</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="22161" xml_f="22162" txt_i="12598" txt_f="12599">q</offsets></mml:mi><mml:mfenced close="]" open="["><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="22242" xml_f="22243" txt_i="12599" txt_f="12600">M</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="22281" xml_f="22282" txt_i="12600" txt_f="12601">K</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="22320" xml_f="22321" txt_i="12601" txt_f="12602">I</offsets></mml:mi><mml:mn><offsets xml_i="22338" xml_f="22340" txt_i="12602" txt_f="12604">67</offsets></mml:mn></mml:mrow></mml:mfenced><mml:mi mathvariant="normal"><offsets xml_i="22403" xml_f="22404" txt_i="12604" txt_f="12605">S</offsets></mml:mi><mml:mspace width="0.25em"></mml:mspace><mml:mo><offsets xml_i="22461" xml_f="22462" txt_i="12605" txt_f="12606">–</offsets></mml:mo><mml:mspace width="0.25em"></mml:mspace><mml:mi mathvariant="normal"><offsets xml_i="22540" xml_f="22546" txt_i="12606" txt_f="12612">meanCq</offsets></mml:mi><mml:mfenced close="]" open="["><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="22626" xml_f="22627" txt_i="12612" txt_f="12613">R</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="22665" xml_f="22666" txt_i="12613" txt_f="12614">E</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="22704" xml_f="22705" txt_i="12614" txt_f="12615">F</offsets></mml:mi></mml:mrow></mml:mfenced><mml:mi mathvariant="normal"><offsets xml_i="22768" xml_f="22769" txt_i="12615" txt_f="12616">S</offsets></mml:mi></mml:mrow></mml:mfenced><mml:mspace width="0.25em"></mml:mspace><mml:mo><offsets xml_i="22851" xml_f="22852" txt_i="12616" txt_f="12617">–</offsets></mml:mo><mml:mspace width="0.25em"></mml:mspace><mml:mfenced close=")" open="("><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="22972" xml_f="22973" txt_i="12617" txt_f="12618">C</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23011" xml_f="23012" txt_i="12618" txt_f="12619">q</offsets></mml:mi><mml:mfenced close="]" open="["><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="23092" xml_f="23093" txt_i="12619" txt_f="12620">M</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23131" xml_f="23132" txt_i="12620" txt_f="12621">K</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23170" xml_f="23171" txt_i="12621" txt_f="12622">I</offsets></mml:mi><mml:mn><offsets xml_i="23188" xml_f="23190" txt_i="12622" txt_f="12624">67</offsets></mml:mn></mml:mrow></mml:mfenced><mml:mi mathvariant="normal"><offsets xml_i="23253" xml_f="23254" txt_i="12624" txt_f="12625">p</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23292" xml_f="23293" txt_i="12625" txt_f="12626">c</offsets></mml:mi><mml:mspace width="0.25em"></mml:mspace><mml:mo><offsets xml_i="23350" xml_f="23351" txt_i="12626" txt_f="12627">–</offsets></mml:mo><mml:mspace width="0.25em"></mml:mspace><mml:mi mathvariant="normal"><offsets xml_i="23429" xml_f="23435" txt_i="12627" txt_f="12633">meanCq</offsets></mml:mi><mml:mfenced close="]" open="["><mml:mrow><mml:mi mathvariant="normal"><offsets xml_i="23515" xml_f="23516" txt_i="12633" txt_f="12634">R</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23554" xml_f="23555" txt_i="12634" txt_f="12635">E</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23593" xml_f="23594" txt_i="12635" txt_f="12636">F</offsets></mml:mi></mml:mrow></mml:mfenced><mml:mi mathvariant="normal"><offsets xml_i="23657" xml_f="23658" txt_i="12636" txt_f="12637">p</offsets></mml:mi><mml:mi mathvariant="normal"><offsets xml_i="23696" xml_f="23697" txt_i="12637" txt_f="12638">c</offsets></mml:mi></mml:mrow></mml:mfenced></mml:mrow></mml:mfenced></mml:math><graphic xlink:href="12885_2017_3111_Article_Equa.gif" position="anchor"></graphic></alternatives></disp-formula><offsets xml_i="23880" xml_f="23881" txt_i="12638" txt_f="12639">
</offsets></p><p><offsets xml_i="23888" xml_f="24077" txt_i="12640" txt_f="12829">In 18 patients the MammaTyper® assay failed, because the required amount of RNA was not sufficient for analysis according to pre-specified criteria as described in the instructions for use.</offsets></p></sec><sec id="Sec6"><title><offsets xml_i="24109" xml_f="24143" txt_i="12831" txt_f="12865">Pathology and Immunohistochemistry</offsets></title><p><offsets xml_i="24154" xml_f="24528" txt_i="12866" txt_f="13240">Tumor grading, tumor typing and immunohistochemistry (ER, PR, Ki-67) was performed on the pretreatment core biopsies on all patients. Pathological complete response (pCR) was determined on tumor resection specimens after completion of neoadjuvant chemotherapy, and was defined as no evidence of residual invasive and ductal disease in the breast and lymph nodes (ypT0,ypN0).</offsets></p><p><offsets xml_i="24535" xml_f="24709" txt_i="13241" txt_f="13415">Immunohistochemistry was performed according to previously standardized protocols on an automated IHC platform (Dako Techmate 500) with citrate buffer for antigen retrieval [</offsets><xref ref-type="bibr" rid="CR23"><offsets xml_i="24742" xml_f="24744" txt_i="13415" txt_f="13417">23</offsets></xref><offsets xml_i="24751" xml_f="24817" txt_i="13417" txt_f="13483">] and observing the ASCO/CAP guidelines for immunohistochemistry [</offsets><xref ref-type="bibr" rid="CR7"><offsets xml_i="24849" xml_f="24850" txt_i="13483" txt_f="13484">7</offsets></xref><offsets xml_i="24857" xml_f="25426" txt_i="13484" txt_f="14053">]. The following primary antibodies and corresponding dilutions were used (DakoCytomation, Glostrup, Denmark): ER (clone 1D5, 1:100), PR (clone PgR636, 1:100) and Ki-67 (MIB-1, 1:200). Slides were assessed by quantitative image analysis (qIHC) using the Aperio Image Analysis toolbox (Leica Biosystems, Nussloch, Germany). Staining intensity and percentage of positive nuclei were recorded after manually segmenting tumor from adjacent stroma. Tumors with ER/PR Remmele scores greater than 3 or positive nuclei greater than 1% were considered hormone receptor positive.</offsets></p></sec><sec id="Sec7"><title><offsets xml_i="25458" xml_f="25477" txt_i="14055" txt_f="14074">Statistical methods</offsets></title><p><offsets xml_i="25488" xml_f="26384" txt_i="14075" txt_f="14971">The Spearman correlation coefficient r was used as a measure of the strength and direction of the linear relationship between variables. 2×2 contingency tables were used to calculate positive percent agreement (PPA) and negative percent agreement (NPA) as a measure of agreement: PPA = 100% x a/(a + c), NPA = 100% x d/(b + d). Receiver Operating Characteristics (ROC) analysis was performed to determine the optimal cut-off for MammaTyper® gene and qIHC protein measurements with pCR as the endpoint. ROC analysis instead of comparing odds ratios to take into account the ratios of clinically relevant false positive and false negative determinations and to identify cut points for each method at clinically relevant prerequisites (i.e., detect all responding tumours). ROC analysis has been used to objectively address each method providing different result codings in a non-parametric manner [</offsets><xref ref-type="bibr" rid="CR24"><offsets xml_i="26417" xml_f="26419" txt_i="14971" txt_f="14973">24</offsets></xref><offsets xml_i="26426" xml_f="26554" txt_i="14973" txt_f="15101">]. On the other hand ROC analysis bears the risk of misinterpreting clinical validity when analyzing heterogeneous populations [</offsets><xref ref-type="bibr" rid="CR25"><offsets xml_i="26587" xml_f="26589" txt_i="15101" txt_f="15103">25</offsets></xref><offsets xml_i="26596" xml_f="27592" txt_i="15103" txt_f="16099">]. However, we have analyzed the response to neoadjuvant chemotherapy within controlled, randomized phase II trial which has defined inclusion and exclusion criteria to have the most comparable basic risk situation. As optimal cut-off for the identification of complete response by the methodologies the point of highest sensitivity still retaining 100% specificity was chosen. The p value reported for evaluating the ROC curve tests the null hypothesis that the area under the curve really equals 0.50 as provided by the statistical program used (GraphPad Prism). The non-parametric Mann-Whitney test was used to confirm the statistical significance when comparing responding versus non-responding tumors and box plots were used to illustrate each case of responding and non-responding tumor above and below the cut-off value. Statistical analyses were performed with JMP SAS (SAS Institute, Cary, NC, USA) and Graph Pad Prism software (Version 5.04; Graph Pad Software Inc., La Jolla, CA, USA).</offsets></p></sec></sec><sec id="Sec8" sec-type="results"><title><offsets xml_i="27649" xml_f="27656" txt_i="16102" txt_f="16109">Results</offsets></title><sec id="Sec9"><title><offsets xml_i="27686" xml_f="27704" txt_i="16110" txt_f="16128">Patient population</offsets></title><p><offsets xml_i="27715" xml_f="27922" txt_i="16129" txt_f="16336">Biopsy tissue was available from 101 out of 105 patients. Gene expression analysis by MammaTyper® was successful in 83 biopsy specimens with full clinical data out of a total of 105 trial participants (Fig. </offsets><xref rid="Fig1" ref-type="fig"><offsets xml_i="27954" xml_f="27955" txt_i="16336" txt_f="16337">1</offsets></xref><offsets xml_i="27962" xml_f="28170" txt_i="16337" txt_f="16545">). 12 patients out of this group had achieved complete pathological remission (pCR). Basic clinicopathological characteristics of statistics data set #2 that includes quantitative IHC data is listed in Table </offsets><xref rid="Tab1" ref-type="table"><offsets xml_i="28204" xml_f="28205" txt_i="16545" txt_f="16546">1</offsets></xref><offsets xml_i="28212" xml_f="28213" txt_i="16546" txt_f="16547">.</offsets><fig id="Fig1"><label><offsets xml_i="28235" xml_f="28241" txt_i="16547" txt_f="16553">Fig. 1</offsets></label><caption><p><offsets xml_i="28261" xml_f="28295" txt_i="16553" txt_f="16587">Remark diagram of sample selection</offsets></p></caption><graphic xlink:href="12885_2017_3111_Fig1_HTML" id="MO1"></graphic></fig><offsets xml_i="28382" xml_f="28383" txt_i="16588" txt_f="16589">
</offsets><table-wrap id="Tab1"><label><offsets xml_i="28412" xml_f="28419" txt_i="16589" txt_f="16596">Table 1</offsets></label><caption><p><offsets xml_i="28439" xml_f="28466" txt_i="16596" txt_f="16623">Basic tumor characteristics</offsets></p></caption><table frame="hsides" rules="groups"><thead><tr><th></th><th><offsets xml_i="28541" xml_f="28547" txt_i="16624" txt_f="16630">no-pCR</offsets></th><th></th><th><offsets xml_i="28565" xml_f="28568" txt_i="16630" txt_f="16633">pCR</offsets></th><th></th><th></th></tr></thead><tbody><tr><td><offsets xml_i="28619" xml_f="28636" txt_i="16633" txt_f="16650">Histological Type</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="28681" xml_f="28682" txt_i="16655" txt_f="16656">
</offsets><italic><offsets xml_i="28690" xml_f="28691" txt_i="16656" txt_f="16657">p</offsets></italic><offsets xml_i="28700" xml_f="28706" txt_i="16657" txt_f="16663">= 0.39</offsets></td></tr><tr><td><offsets xml_i="28724" xml_f="28744" txt_i="16664" txt_f="16684"> Invasive ductal NOS</offsets></td><td><offsets xml_i="28753" xml_f="28755" txt_i="16685" txt_f="16687">37</offsets></td><td><offsets xml_i="28764" xml_f="28769" txt_i="16688" txt_f="16693">82.2%</offsets></td><td><offsets xml_i="28778" xml_f="28779" txt_i="16694" txt_f="16695">9</offsets></td><td><offsets xml_i="28788" xml_f="28794" txt_i="16696" txt_f="16702">100.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="28821" xml_f="28838" txt_i="16704" txt_f="16721"> Invasive lobular</offsets></td><td><offsets xml_i="28847" xml_f="28848" txt_i="16722" txt_f="16723">7</offsets></td><td><offsets xml_i="28857" xml_f="28862" txt_i="16724" txt_f="16729">15.6%</offsets></td><td><offsets xml_i="28871" xml_f="28872" txt_i="16730" txt_f="16731">0</offsets></td><td><offsets xml_i="28881" xml_f="28885" txt_i="16732" txt_f="16736">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="28912" xml_f="28918" txt_i="16738" txt_f="16744"> Other</offsets></td><td><offsets xml_i="28927" xml_f="28928" txt_i="16745" txt_f="16746">1</offsets></td><td><offsets xml_i="28937" xml_f="28941" txt_i="16747" txt_f="16751">2.2%</offsets></td><td><offsets xml_i="28950" xml_f="28951" txt_i="16752" txt_f="16753">0</offsets></td><td><offsets xml_i="28960" xml_f="28964" txt_i="16754" txt_f="16758">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="28991" xml_f="29009" txt_i="16760" txt_f="16778">Histological Grade</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="29054" xml_f="29055" txt_i="16783" txt_f="16784">
</offsets><italic><offsets xml_i="29063" xml_f="29064" txt_i="16784" txt_f="16785">p</offsets></italic><offsets xml_i="29073" xml_f="29079" txt_i="16785" txt_f="16791">= 1.00</offsets></td></tr><tr><td><offsets xml_i="29097" xml_f="29105" txt_i="16792" txt_f="16800"> Grade 2</offsets></td><td><offsets xml_i="29114" xml_f="29116" txt_i="16801" txt_f="16803">15</offsets></td><td><offsets xml_i="29125" xml_f="29130" txt_i="16804" txt_f="16809">33.3%</offsets></td><td><offsets xml_i="29139" xml_f="29140" txt_i="16810" txt_f="16811">3</offsets></td><td><offsets xml_i="29149" xml_f="29154" txt_i="16812" txt_f="16817">33.3%</offsets></td><td></td></tr><tr><td><offsets xml_i="29181" xml_f="29189" txt_i="16819" txt_f="16827"> Grade 3</offsets></td><td><offsets xml_i="29198" xml_f="29200" txt_i="16828" txt_f="16830">30</offsets></td><td><offsets xml_i="29209" xml_f="29214" txt_i="16831" txt_f="16836">66.7%</offsets></td><td><offsets xml_i="29223" xml_f="29224" txt_i="16837" txt_f="16838">6</offsets></td><td><offsets xml_i="29233" xml_f="29238" txt_i="16839" txt_f="16844">66.7%</offsets></td><td></td></tr><tr><td><offsets xml_i="29265" xml_f="29277" txt_i="16846" txt_f="16858">ypT Category</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="29322" xml_f="29323" txt_i="16863" txt_f="16864">
</offsets><italic><offsets xml_i="29331" xml_f="29332" txt_i="16864" txt_f="16865">p</offsets></italic><offsets xml_i="29341" xml_f="29347" txt_i="16865" txt_f="16871">= 0.81</offsets></td></tr><tr><td><offsets xml_i="29365" xml_f="29370" txt_i="16872" txt_f="16877"> ypT0</offsets></td><td><offsets xml_i="29379" xml_f="29380" txt_i="16878" txt_f="16879">0</offsets></td><td><offsets xml_i="29389" xml_f="29393" txt_i="16880" txt_f="16884">0.0%</offsets></td><td><offsets xml_i="29402" xml_f="29403" txt_i="16885" txt_f="16886">9</offsets></td><td><offsets xml_i="29412" xml_f="29418" txt_i="16887" txt_f="16893">100.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="29445" xml_f="29450" txt_i="16895" txt_f="16900"> ypT1</offsets></td><td><offsets xml_i="29459" xml_f="29461" txt_i="16901" txt_f="16903">27</offsets></td><td><offsets xml_i="29470" xml_f="29475" txt_i="16904" txt_f="16909">50.0%</offsets></td><td><offsets xml_i="29484" xml_f="29485" txt_i="16910" txt_f="16911">0</offsets></td><td><offsets xml_i="29494" xml_f="29498" txt_i="16912" txt_f="16916">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="29525" xml_f="29530" txt_i="16918" txt_f="16923"> ypT2</offsets></td><td><offsets xml_i="29539" xml_f="29540" txt_i="16924" txt_f="16925">9</offsets></td><td><offsets xml_i="29549" xml_f="29554" txt_i="16926" txt_f="16931">20.0%</offsets></td><td><offsets xml_i="29563" xml_f="29564" txt_i="16932" txt_f="16933">0</offsets></td><td><offsets xml_i="29573" xml_f="29577" txt_i="16934" txt_f="16938">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="29604" xml_f="29609" txt_i="16940" txt_f="16945"> ypT3</offsets></td><td><offsets xml_i="29618" xml_f="29619" txt_i="16946" txt_f="16947">8</offsets></td><td><offsets xml_i="29628" xml_f="29633" txt_i="16948" txt_f="16953">17.8%</offsets></td><td><offsets xml_i="29642" xml_f="29643" txt_i="16954" txt_f="16955">0</offsets></td><td><offsets xml_i="29652" xml_f="29656" txt_i="16956" txt_f="16960">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="29683" xml_f="29688" txt_i="16962" txt_f="16967"> ypT4</offsets></td><td><offsets xml_i="29697" xml_f="29698" txt_i="16968" txt_f="16969">1</offsets></td><td><offsets xml_i="29707" xml_f="29711" txt_i="16970" txt_f="16974">2.0%</offsets></td><td><offsets xml_i="29720" xml_f="29721" txt_i="16975" txt_f="16976">0</offsets></td><td><offsets xml_i="29730" xml_f="29734" txt_i="16977" txt_f="16981">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="29761" xml_f="29773" txt_i="16983" txt_f="16995">ypN Category</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="29818" xml_f="29819" txt_i="17000" txt_f="17001">
</offsets><italic><offsets xml_i="29827" xml_f="29828" txt_i="17001" txt_f="17002">p</offsets></italic><offsets xml_i="29837" xml_f="29843" txt_i="17002" txt_f="17008">= 0.81</offsets></td></tr><tr><td><offsets xml_i="29861" xml_f="29866" txt_i="17009" txt_f="17014"> ypN0</offsets></td><td><offsets xml_i="29875" xml_f="29877" txt_i="17015" txt_f="17017">27</offsets></td><td><offsets xml_i="29886" xml_f="29891" txt_i="17018" txt_f="17023">50.0%</offsets></td><td><offsets xml_i="29900" xml_f="29901" txt_i="17024" txt_f="17025">9</offsets></td><td><offsets xml_i="29910" xml_f="29916" txt_i="17026" txt_f="17032">100.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="29943" xml_f="29948" txt_i="17034" txt_f="17039"> ypN+</offsets></td><td><offsets xml_i="29957" xml_f="29959" txt_i="17040" txt_f="17042">25</offsets></td><td><offsets xml_i="29968" xml_f="29973" txt_i="17043" txt_f="17048">46.3%</offsets></td><td><offsets xml_i="29982" xml_f="29983" txt_i="17049" txt_f="17050">0</offsets></td><td><offsets xml_i="29992" xml_f="29996" txt_i="17051" txt_f="17055">0.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="30023" xml_f="30028" txt_i="17057" txt_f="17062"> ypNX</offsets></td><td><offsets xml_i="30037" xml_f="30038" txt_i="17063" txt_f="17064">2</offsets></td><td><offsets xml_i="30047" xml_f="30051" txt_i="17065" txt_f="17069">3.7%</offsets></td><td></td><td></td><td></td></tr><tr><td><offsets xml_i="30096" xml_f="30107" txt_i="17073" txt_f="17084">HER2 status</offsets></td><td></td><td></td><td></td><td></td><td><offsets xml_i="30152" xml_f="30153" txt_i="17089" txt_f="17090">
</offsets><italic><offsets xml_i="30161" xml_f="30162" txt_i="17090" txt_f="17091">p</offsets></italic><offsets xml_i="30171" xml_f="30177" txt_i="17091" txt_f="17097">= 0.68</offsets></td></tr><tr><td><offsets xml_i="30195" xml_f="30199" txt_i="17098" txt_f="17102"> Neg</offsets></td><td><offsets xml_i="30208" xml_f="30210" txt_i="17103" txt_f="17105">36</offsets></td><td><offsets xml_i="30219" xml_f="30224" txt_i="17106" txt_f="17111">66.7%</offsets></td><td><offsets xml_i="30233" xml_f="30234" txt_i="17112" txt_f="17113">7</offsets></td><td><offsets xml_i="30243" xml_f="30248" txt_i="17114" txt_f="17119">13.0%</offsets></td><td></td></tr><tr><td><offsets xml_i="30275" xml_f="30279" txt_i="17121" txt_f="17125"> Pos</offsets></td><td><offsets xml_i="30288" xml_f="30289" txt_i="17126" txt_f="17127">7</offsets></td><td><offsets xml_i="30298" xml_f="30303" txt_i="17128" txt_f="17133">13.0%</offsets></td><td><offsets xml_i="30312" xml_f="30313" txt_i="17134" txt_f="17135">2</offsets></td><td><offsets xml_i="30322" xml_f="30326" txt_i="17136" txt_f="17140">3.7%</offsets></td><td></td></tr></tbody></table></table-wrap><offsets xml_i="30374" xml_f="30375" txt_i="17142" txt_f="17143">
</offsets></p></sec><sec id="Sec10"><title><offsets xml_i="30408" xml_f="30475" txt_i="17145" txt_f="17212">Comparison of IHC with RT-qPCR based assessment of ER, PR and Ki-67</offsets></title><p><offsets xml_i="30486" xml_f="30670" txt_i="17213" txt_f="17397">Comparing qIHC of ER with RT-qPCR of ESR1 demonstrated a good overall agreement of 96,6% (PPA 100%; NPA 92.3%) as well as a good correlation looking at the continuous data (spearman’s </offsets><italic><offsets xml_i="30678" xml_f="30679" txt_i="17397" txt_f="17398">r</offsets></italic><offsets xml_i="30688" xml_f="30697" txt_i="17398" txt_f="17407"> = 0.82, </offsets><italic><offsets xml_i="30705" xml_f="30706" txt_i="17407" txt_f="17408">p</offsets></italic><offsets xml_i="30715" xml_f="30794" txt_i="17408" txt_f="17484"> &lt; 0.0001). Correlation between vIHC of ER and RT-qPCR for ESR1 (spearman’s </offsets><italic><offsets xml_i="30802" xml_f="30803" txt_i="17484" txt_f="17485">r</offsets></italic><offsets xml_i="30812" xml_f="30821" txt_i="17485" txt_f="17494"> = 0.85, </offsets><italic><offsets xml_i="30829" xml_f="30830" txt_i="17494" txt_f="17495">p</offsets></italic><offsets xml_i="30839" xml_f="30894" txt_i="17495" txt_f="17547"> &lt; 0.0001) and between vIHC and qIHC ER (spearman’s </offsets><italic><offsets xml_i="30902" xml_f="30903" txt_i="17547" txt_f="17548">r</offsets></italic><offsets xml_i="30912" xml_f="30921" txt_i="17548" txt_f="17557"> = 0.88, </offsets><italic><offsets xml_i="30929" xml_f="30930" txt_i="17557" txt_f="17558">p</offsets></italic><offsets xml_i="30939" xml_f="31142" txt_i="17558" txt_f="17758"> &lt; 0.0001) was high, too. Overall agreement for PR protein and PGR mRNA expression was 91.4% (PPA 83.3%; NPA 100%) comparing qIHC and RT-qPCR and there was a high correlation for the continuous data (</offsets><italic><offsets xml_i="31150" xml_f="31151" txt_i="17758" txt_f="17759">r</offsets></italic><offsets xml_i="31160" xml_f="31169" txt_i="17759" txt_f="17768"> = 0.86, </offsets><italic><offsets xml_i="31177" xml_f="31178" txt_i="17768" txt_f="17769">p</offsets></italic><offsets xml_i="31187" xml_f="31285" txt_i="17769" txt_f="17864"> &lt; 0.0001). Correlation between vIHC and RT-qPCR and between vIHC and qIHC was very high, too (</offsets><italic><offsets xml_i="31293" xml_f="31294" txt_i="17864" txt_f="17865">r</offsets></italic><offsets xml_i="31303" xml_f="31312" txt_i="17865" txt_f="17874"> = 0.88, </offsets><italic><offsets xml_i="31320" xml_f="31321" txt_i="17874" txt_f="17875">p</offsets></italic><offsets xml_i="31330" xml_f="31347" txt_i="17875" txt_f="17889"> &lt; 0.0001 and </offsets><italic><offsets xml_i="31355" xml_f="31356" txt_i="17889" txt_f="17890">r</offsets></italic><offsets xml_i="31365" xml_f="31374" txt_i="17890" txt_f="17899"> = 0.90, </offsets><italic><offsets xml_i="31382" xml_f="31383" txt_i="17899" txt_f="17900">p</offsets></italic><offsets xml_i="31392" xml_f="31864" txt_i="17900" txt_f="18369"> &lt; 0.0001, respectively). Concordance when comparing visual IHC protein with RT-qPCR RNA expression was good for ESR1 as well as for PGR, although slightly lower compared to the agreement between qIHC and RT-qPCR (OPA 91.2%; PPA 90.9%; NPA 91.7%) and for PGR (OPA 92.9%; PPA 88.0%; NPA 96.9%, respectively). For both, ESR1 and PGR, only 4 cases were discordant, with 3 cases each positive by vIHC and negative by RT-qPCR, while 1 case was negative by vIHC and positive </offsets><xref rid="Fig2" ref-type="fig"><offsets xml_i="31896" xml_f="31897" txt_i="18369" txt_f="18370">2</offsets></xref><offsets xml_i="31904" xml_f="32519" txt_i="18370" txt_f="18985">). However, several of these discrepancies could be resolved by using quantitative IHC, as these cases were also discrepant when comparing vIHC with qIHC. Moreover, qIHC could delineate quantitative differences of hormone receptor expression at the highest Remmele Score value of 12, where vIHC could not resolve expression differences. In addition, at the lower range of expression levels RT-qPCR based assessment could still determine substantial differences of mRNA levels while the IHC based assessment could not detect any protein expression. The inter-gene spearman correlation was moderate for ESR1 and PGR (</offsets><italic><offsets xml_i="32527" xml_f="32528" txt_i="18985" txt_f="18986">r</offsets></italic><offsets xml_i="32537" xml_f="32546" txt_i="18986" txt_f="18995"> = 0.59, </offsets><italic><offsets xml_i="32554" xml_f="32555" txt_i="18995" txt_f="18996">p</offsets></italic><offsets xml_i="32564" xml_f="32630" txt_i="18996" txt_f="19059"> &lt; 0.0001), while Ki-67 correlated negatively with PGR (−0.37, </offsets><italic><offsets xml_i="32638" xml_f="32639" txt_i="19059" txt_f="19060">p</offsets></italic><offsets xml_i="32648" xml_f="32786" txt_i="19060" txt_f="19198"> = 0.007). While the correlation for ESR1 and PGR protein and RNA expression was high when comparing IHC with RT-qPCR results (spearman’s </offsets><italic><offsets xml_i="32794" xml_f="32795" txt_i="19198" txt_f="19199">r</offsets></italic><offsets xml_i="32804" xml_f="32813" txt_i="19199" txt_f="19208"> = 0.82, </offsets><italic><offsets xml_i="32821" xml_f="32822" txt_i="19208" txt_f="19209">p</offsets></italic><offsets xml_i="32831" xml_f="32848" txt_i="19209" txt_f="19223"> &lt; 0.0001 and </offsets><italic><offsets xml_i="32856" xml_f="32857" txt_i="19223" txt_f="19224">r</offsets></italic><offsets xml_i="32866" xml_f="32875" txt_i="19224" txt_f="19233"> = 0.86, </offsets><italic><offsets xml_i="32883" xml_f="32884" txt_i="19233" txt_f="19234">p</offsets></italic><offsets xml_i="32893" xml_f="33009" txt_i="19234" txt_f="19347"> &lt; 0.0001, respectively), the correlation between MKI67 protein and RNA expression was only moderate (spearman’s </offsets><italic><offsets xml_i="33017" xml_f="33018" txt_i="19347" txt_f="19348">r</offsets></italic><offsets xml_i="33027" xml_f="33048" txt_i="19348" txt_f="19369"> = 0.56 for vIHC and </offsets><italic><offsets xml_i="33056" xml_f="33057" txt_i="19369" txt_f="19370">r</offsets></italic><offsets xml_i="33066" xml_f="33172" txt_i="19370" txt_f="19476"> = 0.47 for qIHC). In contrast the correlation between both methods for Ki-67 assessment by IHC was high (</offsets><italic><offsets xml_i="33180" xml_f="33181" txt_i="19476" txt_f="19477">r</offsets></italic><offsets xml_i="33190" xml_f="33199" txt_i="19477" txt_f="19486"> = 0.80, </offsets><italic><offsets xml_i="33207" xml_f="33208" txt_i="19486" txt_f="19487">p</offsets></italic><offsets xml_i="33217" xml_f="33620" txt_i="19487" txt_f="19887"> &lt; 0.0001). The median value of Ki-67 proliferation index by image analysis IHC (qIHC) was 23.4%, by conventional visual IHC (vIHC) 35.0%, and 37.01 for RT-qPCR, clearly reflecting the inclusion criteria of the S080 trial which targeted clinically higher-risk patients. Scatter plot analysis displays the positive correlation between RT-qPCR and visual as well as quantitative IHC assessment in Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="33652" xml_f="33653" txt_i="19887" txt_f="19888">4</offsets></xref><offsets xml_i="33660" xml_f="33661" txt_i="19888" txt_f="19889">.</offsets><fig id="Fig2"><label><offsets xml_i="33683" xml_f="33689" txt_i="19889" txt_f="19895">Fig. 2</offsets></label><caption><p><offsets xml_i="33709" xml_f="33736" txt_i="19895" txt_f="19922">Correlation of RT-qPCR for </offsets><italic><offsets xml_i="33744" xml_f="33748" txt_i="19922" txt_f="19926">ESR1</offsets></italic><offsets xml_i="33757" xml_f="33759" txt_i="19926" txt_f="19928">, </offsets><italic><offsets xml_i="33767" xml_f="33770" txt_i="19928" txt_f="19931">PGR</offsets></italic><offsets xml_i="33779" xml_f="33784" txt_i="19931" txt_f="19936"> and </offsets><italic><offsets xml_i="33792" xml_f="33797" txt_i="19936" txt_f="19941">MKI67</offsets></italic><offsets xml_i="33806" xml_f="33848" txt_i="19941" txt_f="19983"> with quantitative IHC by image analysis (</offsets><bold><offsets xml_i="33854" xml_f="33855" txt_i="19983" txt_f="19984">a</offsets></bold><offsets xml_i="33862" xml_f="33864" txt_i="19984" txt_f="19986">, </offsets><bold><offsets xml_i="33870" xml_f="33871" txt_i="19986" txt_f="19987">c</offsets></bold><offsets xml_i="33878" xml_f="33880" txt_i="19987" txt_f="19989">, </offsets><bold><offsets xml_i="33886" xml_f="33887" txt_i="19989" txt_f="19990">e</offsets></bold><offsets xml_i="33894" xml_f="33923" txt_i="19990" txt_f="20019">) and visual IHC assessment (</offsets><bold><offsets xml_i="33929" xml_f="33930" txt_i="20019" txt_f="20020">b</offsets></bold><offsets xml_i="33937" xml_f="33939" txt_i="20020" txt_f="20022">, </offsets><bold><offsets xml_i="33945" xml_f="33946" txt_i="20022" txt_f="20023">d</offsets></bold><offsets xml_i="33953" xml_f="33955" txt_i="20023" txt_f="20025">, </offsets><bold><offsets xml_i="33961" xml_f="33962" txt_i="20025" txt_f="20026">f</offsets></bold><offsets xml_i="33969" xml_f="33970" txt_i="20026" txt_f="20027">)</offsets></p></caption><graphic xlink:href="12885_2017_3111_Fig2_HTML" id="MO2"></graphic></fig><offsets xml_i="34057" xml_f="34058" txt_i="20028" txt_f="20029">
</offsets></p></sec><sec id="Sec11"><title><offsets xml_i="34091" xml_f="34108" txt_i="20031" txt_f="20048">Prediction of pCR</offsets></title><p><offsets xml_i="34119" xml_f="34312" txt_i="20049" txt_f="20242">To compare the clinical utility we performed a ROC analysis to determine the optimal cutoff for predicting the pCR. The results of the ROC analysis are presented in the graphical plots of Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="34344" xml_f="34345" txt_i="20242" txt_f="20243">3</offsets></xref><offsets xml_i="34352" xml_f="34532" txt_i="20243" txt_f="20423">. With RT-qPCR, 100% of responders could be detected with a specificity of 68.9% at a 40-ddCT level of 37.31 which almost reflected the median mRNA expression in this cohort (Fig. </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="34564" xml_f="34566" txt_i="20423" txt_f="20425">3a</offsets></xref><offsets xml_i="34573" xml_f="34575" txt_i="20425" txt_f="20427">, </offsets><xref rid="Fig3" ref-type="fig"><offsets xml_i="34607" xml_f="34608" txt_i="20427" txt_f="20428">b</offsets></xref><offsets xml_i="34615" xml_f="34676" txt_i="20428" txt_f="20489">). Conversely, no responder was below RT-qPCR of 37.31 (Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="34708" xml_f="34710" txt_i="20489" txt_f="20491">4a</offsets></xref><offsets xml_i="34717" xml_f="34946" txt_i="20491" txt_f="20720">). For IHC assessment, it was difficult to determine a reliable cutoff reaching high sensitivity and specificity. With RT-qPCR the area under the curve was 0.78 for the overall cohort and 0.80 for the IHC cohort (Statistics #2) (</offsets><italic><offsets xml_i="34954" xml_f="34955" txt_i="20720" txt_f="20721">p</offsets></italic><offsets xml_i="34964" xml_f="34977" txt_i="20721" txt_f="20734"> = 0.002 and </offsets><italic><offsets xml_i="34985" xml_f="34986" txt_i="20734" txt_f="20735">p</offsets></italic><offsets xml_i="34995" xml_f="35056" txt_i="20735" txt_f="20796"> = 0.004). For both IHC methods, the ROC was not significant.</offsets><fig id="Fig3"><label><offsets xml_i="35078" xml_f="35084" txt_i="20796" txt_f="20802">Fig. 3</offsets></label><caption><p><offsets xml_i="35104" xml_f="35181" txt_i="20802" txt_f="20879">ROC analysis for prediction of pathological complete response by quantifying </offsets><italic><offsets xml_i="35189" xml_f="35211" txt_i="20879" txt_f="20901">MKI67/Ki-67 expression</offsets></italic><offsets xml_i="35220" xml_f="35233" txt_i="20901" txt_f="20914"> by RT-qPCR (</offsets><bold><offsets xml_i="35239" xml_f="35240" txt_i="20914" txt_f="20915">a</offsets></bold><offsets xml_i="35247" xml_f="35249" txt_i="20915" txt_f="20917">, </offsets><bold><offsets xml_i="35255" xml_f="35256" txt_i="20917" txt_f="20918">b</offsets></bold><offsets xml_i="35263" xml_f="35274" txt_i="20918" txt_f="20929">) and IHC (</offsets><bold><offsets xml_i="35280" xml_f="35281" txt_i="20929" txt_f="20930">c</offsets></bold><offsets xml_i="35288" xml_f="35290" txt_i="20930" txt_f="20932">, </offsets><bold><offsets xml_i="35296" xml_f="35297" txt_i="20932" txt_f="20933">d</offsets></bold><offsets xml_i="35304" xml_f="35413" txt_i="20933" txt_f="21042">) showing overall increased ability of mRNA assessment to correctly identify responders versus non-responders</offsets></p></caption><graphic xlink:href="12885_2017_3111_Fig3_HTML" id="MO3"></graphic></fig><offsets xml_i="35500" xml_f="35501" txt_i="21043" txt_f="21044">
</offsets><fig id="Fig4"><label><offsets xml_i="35523" xml_f="35529" txt_i="21044" txt_f="21050">Fig. 4</offsets></label><caption><p><offsets xml_i="35549" xml_f="35610" txt_i="21050" txt_f="21111">Scatter plots illustrating the distribution of RT-qPCR mRNA (</offsets><italic><offsets xml_i="35618" xml_f="35629" txt_i="21111" txt_f="21122">upper panel</offsets></italic><offsets xml_i="35638" xml_f="35650" txt_i="21122" txt_f="21134">) and qIHC (</offsets><italic><offsets xml_i="35658" xml_f="35669" txt_i="21134" txt_f="21145">lower panel</offsets></italic><offsets xml_i="35678" xml_f="35690" txt_i="21145" txt_f="21157">) and vIHC (</offsets><italic><offsets xml_i="35698" xml_f="35709" txt_i="21157" txt_f="21168">right panel</offsets></italic><offsets xml_i="35718" xml_f="35782" txt_i="21168" txt_f="21232">) protein measurements in relation to the groups of responders (</offsets><italic><offsets xml_i="35790" xml_f="35800" txt_i="21232" txt_f="21242">green dots</offsets></italic><offsets xml_i="35809" xml_f="35831" txt_i="21242" txt_f="21264">) and non-responders (</offsets><italic><offsets xml_i="35839" xml_f="35848" txt_i="21264" txt_f="21273">blue dots</offsets></italic><offsets xml_i="35857" xml_f="35912" txt_i="21273" txt_f="21328">). Differences were tested with the Mann-Whitney test (</offsets><bold><offsets xml_i="35918" xml_f="35919" txt_i="21328" txt_f="21329">a</offsets></bold><offsets xml_i="35926" xml_f="35941" txt_i="21329" txt_f="21344"> = data set 1, </offsets><italic><offsets xml_i="35949" xml_f="35950" txt_i="21344" txt_f="21345">n</offsets></italic><offsets xml_i="35959" xml_f="35966" txt_i="21345" txt_f="21352"> = 83, </offsets><bold><offsets xml_i="35972" xml_f="35973" txt_i="21352" txt_f="21353">b</offsets></bold><offsets xml_i="35980" xml_f="35982" txt_i="21353" txt_f="21355">, </offsets><bold><offsets xml_i="35988" xml_f="35989" txt_i="21355" txt_f="21356">c</offsets></bold><offsets xml_i="35996" xml_f="35998" txt_i="21356" txt_f="21358">, </offsets><bold><offsets xml_i="36004" xml_f="36005" txt_i="21358" txt_f="21359">d</offsets></bold><offsets xml_i="36012" xml_f="36032" txt_i="21359" txt_f="21379"> = common data set, </offsets><italic><offsets xml_i="36040" xml_f="36041" txt_i="21379" txt_f="21380">n</offsets></italic><offsets xml_i="36050" xml_f="36056" txt_i="21380" txt_f="21386"> = 54)</offsets></p></caption><graphic xlink:href="12885_2017_3111_Fig4_HTML" id="MO4"></graphic></fig><offsets xml_i="36143" xml_f="36144" txt_i="21387" txt_f="21388">
</offsets></p><p><offsets xml_i="36151" xml_f="36591" txt_i="21389" txt_f="21829">Since maximum sensitivity was a pre-requisite, the two methods were compared with respect to specificity, which was found to be substantially higher for MammaTyper® (68.9%) compared to qIHC (22.2%). Using the cut-offs indicated by ROC analysis (37.31 for RT-qPCR, 13.2% for qIHC and 3.5% for vIHC), tumors were characterized as bearing either high or low MKI67 RNA or Ki-67 protein expression, respectively. However, as illustrated in Fig. </offsets><xref rid="Fig4" ref-type="fig"><offsets xml_i="36623" xml_f="36624" txt_i="21829" txt_f="21830">4</offsets></xref><offsets xml_i="36631" xml_f="36816" txt_i="21830" txt_f="22015">, statistically significant differences between groups were found for the RT-qPCR but not for IHC methods when patients were stratified according to proliferation and pCR (Mann–Whitney </offsets><italic><offsets xml_i="36824" xml_f="36825" txt_i="22015" txt_f="22016">p</offsets></italic><offsets xml_i="36834" xml_f="36847" txt_i="22016" txt_f="22029"> = 0.003 and </offsets><italic><offsets xml_i="36855" xml_f="36856" txt_i="22029" txt_f="22030">p</offsets></italic><offsets xml_i="36865" xml_f="36890" txt_i="22030" txt_f="22055"> = 0,005 for RT-qPCR and </offsets><italic><offsets xml_i="36898" xml_f="36899" txt_i="22055" txt_f="22056">p</offsets></italic><offsets xml_i="36908" xml_f="36930" txt_i="22056" txt_f="22078"> = 0.099 for qIHC and </offsets><italic><offsets xml_i="36938" xml_f="36939" txt_i="22078" txt_f="22079">p</offsets></italic><offsets xml_i="36948" xml_f="37058" txt_i="22079" txt_f="22189"> = 0.133 for vIHC). Using the cut-offs obtained by ROC analysis, pCR was observed in 9 of 24 patients (37.5%, </offsets><italic><offsets xml_i="37066" xml_f="37067" txt_i="22189" txt_f="22190">p</offsets></italic><offsets xml_i="37076" xml_f="37230" txt_i="22190" txt_f="22341"> &lt; 0.001) with high MKI67 RNA expression but in no patient with low RNA expression. Accordingly for qIHC, pCR was observed in 9 of 44 patients (20.5%, </offsets><italic><offsets xml_i="37238" xml_f="37239" txt_i="22341" txt_f="22342">p</offsets></italic><offsets xml_i="37248" xml_f="37529" txt_i="22342" txt_f="22623"> = 0.27) with high Ki-67 labelling index and similarly it was entirely lacking in patients with low proliferation. For RT-qPCR the ROC analysis was also highly significant when only luminal tumors had been assessed, though the sample size was small in this subset (data not shown).</offsets></p></sec></sec><sec id="Sec12" sec-type="discussion"><title><offsets xml_i="37590" xml_f="37600" txt_i="22626" txt_f="22636">Discussion</offsets></title><p><offsets xml_i="37611" xml_f="38273" txt_i="22637" txt_f="23299">In this study we have validated clinical performance of hormone receptor gene expression by RT-qPCR by comparing predefined cut-offs in a blinded fashion with the current standard of IHC. Furthermore, we have investigated the diagnostic performance of two methods for assessing MKI67 gene expression, namely IHC with computerized quantification of protein and RT-qPCR RNA quantification with the MammaTyper® IVD kit in the setting of pCR prediction. When continuous data were dichotomized to reflect high- and low-MKI67 categories with cut-offs obtained by ROC curve analysis after considering 100% sensitivity, RT-qPCR was significantly more specific than qIHC.</offsets></p><p><offsets xml_i="38280" xml_f="39132" txt_i="23300" txt_f="24152">To the best of our knowledge this is the first direct comparison of this kind in the context of a clinical trial. For the mRNA estimation we used the MammaTyper®, a novel in vitro diagnostic test for breast cancer molecular subtyping. To prove the clinical utility of mRNA based assessment, we compared RT-qPCR with conventional visual assessment as well as digital image analysis based determination at a reference pathology lab in the context of a clinical trial. Moreover, the methods were examined with respect to their ability to predict pCR according to Ki-67 protein or MKI67 mRNA expression levels measured on pretreatment core biopsies. Our results indicate, that, when using RT-qPCR valid cut-offs for mRNA expression, which reliably distinguish between non-responding and responding tumors as determined by pCR (ypT0 ypN0) can be identified.</offsets></p><p><offsets xml_i="39139" xml_f="39298" txt_i="24153" txt_f="24312">Pathological complete response has gained wide acceptance as one of the strongest predictors of prolonged survival in the setting of neoadjuvant chemotherapy [</offsets><xref ref-type="bibr" rid="CR26"><offsets xml_i="39331" xml_f="39333" txt_i="24312" txt_f="24314">26</offsets></xref><offsets xml_i="39340" xml_f="39342" txt_i="24314" txt_f="24316">, </offsets><xref ref-type="bibr" rid="CR27"><offsets xml_i="39375" xml_f="39377" txt_i="24316" txt_f="24318">27</offsets></xref><offsets xml_i="39384" xml_f="39602" txt_i="24318" txt_f="24536">]. Therefore, laboratory assays that can efficiently predict a patient’s response to a given preoperative chemotherapeutic combination may serve as tools for individualizing treatment and improving long-term outcomes [</offsets><xref ref-type="bibr" rid="CR28"><offsets xml_i="39635" xml_f="39637" txt_i="24536" txt_f="24538">28</offsets></xref><offsets xml_i="39644" xml_f="39886" txt_i="24538" txt_f="24780">]. As with adjuvant chemotherapy, neoadjuvant regimens also suffer from the fact that substantial therapeutic benefit is restricted only to a fraction of those treated, whereas all patients will experience adverse events because of toxicity [</offsets><xref ref-type="bibr" rid="CR29"><offsets xml_i="39919" xml_f="39921" txt_i="24780" txt_f="24782">29</offsets></xref><offsets xml_i="39928" xml_f="39930" txt_i="24782" txt_f="24784">].</offsets></p><p><offsets xml_i="39937" xml_f="40177" txt_i="24785" txt_f="25025">In several neoadjuvant studies Ki-67 protein expression has been investigated in pre-operative biopsies in relation to the response to treatment and in most cases a high Ki-67 proliferation rate was predictive of higher probability of pCR [</offsets><xref ref-type="bibr" rid="CR13"><offsets xml_i="40210" xml_f="40212" txt_i="25025" txt_f="25027">13</offsets></xref><offsets xml_i="40219" xml_f="40443" txt_i="25027" txt_f="25251">]. Fasching et al. analyzed Ki-67 by conventional IHC in core biopsies from 552 patients from a single German institution and showed that a pre-defined 13% cut-off could predict pCR with 94% sensitivity and 36% specificity [</offsets><xref ref-type="bibr" rid="CR30"><offsets xml_i="40476" xml_f="40478" txt_i="25251" txt_f="25253">30</offsets></xref><offsets xml_i="40485" xml_f="40827" txt_i="25253" txt_f="25595">]. Interestingly, our ROC analysis for qIHC requiring 100% sensitivity with the least possible loss on specificity led to an identical cut-off (13.2%) for Ki-67. However, this finding requires careful interpretation, due to differences characterizing the clinical settings between the two neoadjuvant studies and the original work by Cheang [</offsets><xref ref-type="bibr" rid="CR31"><offsets xml_i="40860" xml_f="40862" txt_i="25595" txt_f="25597">31</offsets></xref><offsets xml_i="40869" xml_f="41270" txt_i="25597" txt_f="25998">]. In the latter case, the Ki-67 cut-off was fine-tuned against gene expression profiling in order to distinguish Luminal A from Luminal B tumors in a population containing both high- and low-risk breast cancers, whereas in the neoadjuvant setting the same cut-off was intended to identify the majority of, mainly high-risk, patients that would most likely benefit from preoperative cytotoxic therapy.</offsets></p><p><offsets xml_i="41277" xml_f="41588" txt_i="25999" txt_f="26310">Alike what has been repeatedly shown in the adjuvant setting, it appears that the molecular architecture of tumors as defined by the expression of hormone receptors and HER2/neu may act as a modifier of the association between Ki-67 and response to neoadjuvant treatment and between pCR and long-term outcomes [</offsets><xref ref-type="bibr" rid="CR14"><offsets xml_i="41621" xml_f="41623" txt_i="26310" txt_f="26312">14</offsets></xref><offsets xml_i="41630" xml_f="41632" txt_i="26312" txt_f="26314">, </offsets><xref ref-type="bibr" rid="CR32"><offsets xml_i="41665" xml_f="41667" txt_i="26314" txt_f="26316">32</offsets></xref><offsets xml_i="41674" xml_f="41899" txt_i="26316" txt_f="26541">]. While 101 tumors were available for analysis, the inclusion of 83 or 54 tumors in this study was not based on a statistical rational but was dictated by the availability of tumor tissue with complete RT-qPCR and qIHC data.</offsets></p><p><offsets xml_i="41906" xml_f="42803" txt_i="26542" txt_f="27435">A novel aspect of the present work is the comparison between protein-based and mRNA-based methods for the assessment of tumor proliferation. Our findings highlight the feasibility of using RT-qPCR for the routine assessment of ESR1, PGR &amp; MKI67 in order to assist the selection of breast cancer patients for neoadjuvant treatment. Even though both RT-qPCR and qIHC of MKI67/Ki-67 could be calibrated to maximize negative predictive value, only with the former this was achieved whilst ensuring sufficient specificity, which if validated would signify that MammaTyper® could help a considerable number of patients safely forego unnecessary treatment. These data collectively indicate that MammaTyper®MKI67 RNA was overall more representative of the true proliferation state of the tumor than was computer assisted Ki-67 protein estimation, a finding that is worth validating in larger datasets.</offsets></p><p><offsets xml_i="42810" xml_f="42923" txt_i="27436" txt_f="27549">Significant correlations between conventional Ki-67 visual assessment and RT-qPCR have been previously reported [</offsets><xref ref-type="bibr" rid="CR33"><offsets xml_i="42956" xml_f="42958" txt_i="27549" txt_f="27551">33</offsets></xref><offsets xml_i="42965" xml_f="42967" txt_i="27551" txt_f="27553">, </offsets><xref ref-type="bibr" rid="CR34"><offsets xml_i="43000" xml_f="43002" txt_i="27553" txt_f="27555">34</offsets></xref><offsets xml_i="43009" xml_f="43208" txt_i="27555" txt_f="27754">], indicating a strong biological link between mRNA and protein expression despite methodological variations, as is further indicated by comparable prognostic hazard ratios obtained by both methods [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="43241" xml_f="43243" txt_i="27754" txt_f="27756">35</offsets></xref><offsets xml_i="43250" xml_f="44051" txt_i="27756" txt_f="28557">]. To the best of our knowledge however, our study is the first to compare image analysis with RT-qPCR for the assessment of tumor proliferation with the additional advantage of using material from a randomized clinical trial. The correlation between mRNA and protein was significant but moderate, a finding which may reflect post translational modifications or may be related to the increased dynamic range of RT-qPCR as compared to IHC. Another possible explanation might be that mRNA levels are a reflection of the average gene expression in the entire FFPE slice, whereas IHC may be biased in favor of selected “representative” tumor areas. Even in the case of image analysis systems, inspection of digitalized images and manual identification of tumor areas is necessary before automatic scoring.</offsets></p><p><offsets xml_i="44058" xml_f="44457" txt_i="28558" txt_f="28957">Computerized methods have been recommended as a solution to the problem of subjectivity in the visual assessment and scoring of IHC-stained slides. Not surprisingly, Ki-67 scores from image analysis systems are generally in close agreement with those of manual methods because manual scoring for research purposes is customarily performed by a pathologist with longstanding experience in the field [</offsets><xref ref-type="bibr" rid="CR17"><offsets xml_i="44490" xml_f="44492" txt_i="28957" txt_f="28959">17</offsets></xref><offsets xml_i="44499" xml_f="44501" txt_i="28959" txt_f="28961">, </offsets><xref ref-type="bibr" rid="CR36"><offsets xml_i="44534" xml_f="44536" txt_i="28961" txt_f="28963">36</offsets></xref><offsets xml_i="44543" xml_f="44790" txt_i="28963" txt_f="29210">]. It is worth mentioning, however, that in a routine decentralized setting, digital processing and scoring of slides would probably outperform manual assessment which is prone to considerable subjectivity often not improved upon standardization [</offsets><xref ref-type="bibr" rid="CR37"><offsets xml_i="44823" xml_f="44825" txt_i="29210" txt_f="29212">37</offsets></xref><offsets xml_i="44832" xml_f="45193" txt_i="29212" txt_f="29573">]. Digital analysis yields more reproducible results with regard to staining intensity, by facilitation the definition of low grade staining intensities. Definitive conclusions would require comparisons between all three methods (central versus local versus automatic) performed preferably in the prospective retrospective setting of a large multi-center trial.</offsets></p><p><offsets xml_i="45200" xml_f="45314" txt_i="29574" txt_f="29688">Multi-gene molecular signatures have also been tested as a way for predicting pCR in patients with breast cancer [</offsets><xref ref-type="bibr" rid="CR38"><offsets xml_i="45347" xml_f="45349" txt_i="29688" txt_f="29690">38</offsets></xref><offsets xml_i="45356" xml_f="45357" txt_i="29690" txt_f="29691">–</offsets><xref ref-type="bibr" rid="CR40"><offsets xml_i="45390" xml_f="45392" txt_i="29691" txt_f="29693">40</offsets></xref><offsets xml_i="45399" xml_f="45909" txt_i="29693" txt_f="30203">]. However, generalized use of these commercialized assays is limited by their increased cost and the requirement to run in centralized platforms or both. Interestingly, proliferation genes, including MKI67, are often heavily weighted in multi-gene scores which serve as estimators of a patients’ risk of developing recurrences. This is perhaps one of the reasons why multi-gene tests do not always prove to be convincingly superior to conventional or less sophisticated methods for tumor risk stratification [</offsets><xref ref-type="bibr" rid="CR35"><offsets xml_i="45942" xml_f="45944" txt_i="30203" txt_f="30205">35</offsets></xref><offsets xml_i="45951" xml_f="45953" txt_i="30205" txt_f="30207">, </offsets><xref ref-type="bibr" rid="CR41"><offsets xml_i="45986" xml_f="45988" txt_i="30207" txt_f="30209">41</offsets></xref><offsets xml_i="45995" xml_f="45997" txt_i="30209" txt_f="30211">, </offsets><xref ref-type="bibr" rid="CR42"><offsets xml_i="46030" xml_f="46032" txt_i="30211" txt_f="30213">42</offsets></xref><offsets xml_i="46039" xml_f="46101" txt_i="30213" txt_f="30275">], leading some authors to question their cost-effectiveness [</offsets><xref ref-type="bibr" rid="CR43"><offsets xml_i="46134" xml_f="46136" txt_i="30275" txt_f="30277">43</offsets></xref><offsets xml_i="46143" xml_f="46787" txt_i="30277" txt_f="30921">]. Moreover, for several commercially available tests, neither doctors nor consumers can gain access to the continuous expression data of individual proliferation markers that make up the final risk scores. This restriction overall minimizes the possibility of potentially interesting comparisons between proliferation motifs or scores and single proliferation markers based on RT-qPCR or IHC. Strikingly, our ROC curve analysis of MKI67 40-ΔΔCq values for the prediction of pCR displayed performance characteristics that are comparable with those of a 50-gene predictor of tumor recurrence risk developed by supervised training of Cox models [</offsets><xref ref-type="bibr" rid="CR39"><offsets xml_i="46820" xml_f="46822" txt_i="30921" txt_f="30923">39</offsets></xref><offsets xml_i="46829" xml_f="47067" txt_i="30923" txt_f="31161">]. Along these lines, single-gene MKI67 RT-qPCR may be worth considering as a golden means for assessing tumor proliferation due to its unique ability to combine technical advancements and diagnostic accuracy with more affordable pricing.</offsets></p></sec><sec id="Sec13" sec-type="conclusion"><title><offsets xml_i="47122" xml_f="47133" txt_i="31163" txt_f="31174">Conclusions</offsets></title><p><offsets xml_i="47144" xml_f="47784" txt_i="31175" txt_f="31815">Image analysis-assisted scoring of ER, PR and Ki-67 IHC and quantification of ESR1, PGR and MIKI67 RNA expression with RT-qPCR both represent promising alternatives to conventional visual estimation and may assist in improving reproducibility and accuracy in the field. However, RT-qPCR assessment of tumor proliferation was overall more accurate than quantitative IHC. This is the first study to compare tumor MKI67 gene expression by RNA and protein assessment in a prospective retrospective neoadjuvant setting. Due to the relatively small sample size, these data should be considered preliminary and worth validating in larger datasets.</offsets></p></sec></body><back><app-group><app id="App1"><sec id="Sec14"><title>Additional file</title><p>
<media position="anchor" xlink:href="12885_2017_3111_MOESM1_ESM.xlsx" id="MOESM1"><label>Additional file 1:</label><caption><p>Raw data; qPCR data, quantitative and visual IHC values, pathologic complete response yes/no. (XLSX 42 kb)</p></caption></media>
</p></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>CISH</term><def><p>Chromogenic in situ hybridization</p></def></def-item><def-item><term>Cq</term><def><p>Quantification cycle</p></def></def-item><def-item><term>DDFS</term><def><p>Distant disease free survival</p></def></def-item><def-item><term>ESR1/ER</term><def><p>Oestrogen receptor alpha; ERBB2/HER2</p></def></def-item><def-item><term>FEC</term><def><p>Fluorouracil &amp; epirubicin, cyclophosphamide chemotherapy</p></def></def-item><def-item><term>FFPE</term><def><p>Formalin fixed paraffin embedded</p></def></def-item><def-item><term>GOI</term><def><p>Gene of interest</p></def></def-item><def-item><term>HR</term><def><p>Hazard ratio</p></def></def-item><def-item><term>IHC</term><def><p>Immunohistochemistry</p></def></def-item><def-item><term>MKI67/Ki67</term><def><p>marker of proliferation Ki-67</p></def></def-item><def-item><term>mRNA</term><def><p>Messenger ribonucleic acid</p></def></def-item><def-item><term>NPA</term><def><p>Negative percentage agreement</p></def></def-item><def-item><term>OPA</term><def><p>Overall percentage agreement</p></def></def-item><def-item><term>OS</term><def><p>Overall survival</p></def></def-item><def-item><term>PGR/PgR</term><def><p>Progesterone receptor</p></def></def-item><def-item><term>PPA</term><def><p>Positive percentage agreement</p></def></def-item><def-item><term>REF</term><def><p>Reference gene</p></def></def-item><def-item><term>RT-qPCR</term><def><p>Reverse transcription quantitative real time polymerase chain reaction</p></def></def-item></def-list></glossary><ack><title>Acknowledgements</title><p>We thank Susanne Scharff, Silke Claas and Torsten Acht for excellent technical support in developing molecular subtyping technologies, and Drs. Thomas Keller and Stefan Weber for performing statistical analyses.</p><p>We acknowledge the financial support of the Deutsche Forschungsgemeinschaft and Ruprecht-Karls-Universität Heidelberg within the funding programme Open Access Publishing.</p><sec id="FPar1"><title>Funding</title><p>Dietmar Hopp-Stiftung,</p><p>Award Number: 23011195,</p><p>Grant Recipient: Peter Sinn MD PhD</p></sec><sec id="FPar2"><title>Availability of data and materials</title><p>The dataset has been made available in the Excel file format as Additional file <xref rid="MOESM1" ref-type="media">1</xref>.</p></sec><sec id="FPar3"><title>Authors’ contributions</title><p>HPS, KS, ML, SL, RMW and FM were involved in the conception of the study. EV performed the RT-qPCR assays; HPS, AS, MK, JS and FM provided study data and materials; HPS, SL, PA and RMW performed the statistical analysis and wrote the statistical plan; HPS, AS, KS, ML, SL, SE, RMW and FM interpreted the data; HPS, SL, RMW and FM drafted the manuscript; all authors read and approved the final manuscript.</p></sec><sec id="FPar4"><title>Competing interests</title><p>RMW and SE are founders of STRATIFYER Molecular Pathology GmbH. SL, EV and RMW are employees of STRATIFYER Molecular Pathology GmbH. KS and ML are employees of BioNTech Diagnostics GmbH. All other authors declare that they have no competing interests.</p></sec><sec id="FPar5"><title>Consent for publication</title><p>Not applicable.</p></sec><sec id="FPar6"><title>Ethics approval and consent to participate</title><p>All patients had been enrolled in the H3E-MC-S080 (NCT00149214) neoadjuvant phase II study [<xref ref-type="bibr" rid="CR1">1</xref>] at a single institution (National Center for Tumor Diseases, University-Hospital, Heidelberg). A written informed consent for the research use of patient biological material was granted at the time of enrolment. The research described herein is completely independent from the sponsor of the original study (Eli Lilly and Company). The study was approved by the local ethics committee of the Heidelberg University Hospital.</p></sec></ack><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Senkus</surname><given-names>E</given-names></name><name><surname>Kyriakides</surname><given-names>S</given-names></name><name><surname>Ohno</surname><given-names>S</given-names></name><name><surname>Penault-Llorca</surname><given-names>F</given-names></name><name><surname>Poortmans</surname><given-names>P</given-names></name><name><surname>Rutgers</surname><given-names>E</given-names></name><name><surname>Zackrisson</surname><given-names>S</given-names></name><name><surname>Cardoso</surname><given-names>F</given-names></name><name><surname>Committee</surname><given-names>EG</given-names></name></person-group><article-title>Primary breast cancer: ESMO clinical practice guidelines for diagnosis, treatment and follow-up</article-title><source>Ann Oncol</source><year>2015</year><volume>26</volume><issue>Suppl 5</issue><fpage>v8</fpage><lpage>30</lpage><pub-id pub-id-type="doi">10.1093/annonc/mdv298</pub-id><pub-id pub-id-type="pmid">26314782</pub-id></element-citation></ref><ref id="CR2"><label>2.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Coates</surname><given-names>AS</given-names></name><name><surname>Winer</surname><given-names>EP</given-names></name><name><surname>Goldhirsch</surname><given-names>A</given-names></name><name><surname>Gelber</surname><given-names>RD</given-names></name><name><surname>Gnant</surname><given-names>M</given-names></name><name><surname>Piccart-Gebhart</surname><given-names>M</given-names></name><name><surname>Thurlimann</surname><given-names>B</given-names></name><name><surname>Senn</surname><given-names>HJ</given-names></name><name><surname>Panel</surname><given-names>M</given-names></name></person-group><article-title>Tailoring therapies--improving the management of early breast cancer: St Gallen International Expert Consensus on the Primary Therapy of Early Breast Cancer 2015</article-title><source>Ann Oncol</source><year>2015</year><volume>26</volume><issue>8</issue><fpage>1533</fpage><lpage>46</lpage><pub-id pub-id-type="doi">10.1093/annonc/mdv221</pub-id><pub-id pub-id-type="pmid">25939896</pub-id></element-citation></ref><ref id="CR3"><label>3.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Gerdes</surname><given-names>J</given-names></name><name><surname>Lemke</surname><given-names>H</given-names></name><name><surname>Baisch</surname><given-names>H</given-names></name><name><surname>Wacker</surname><given-names>HH</given-names></name><name><surname>Schwab</surname><given-names>U</given-names></name><name><surname>Stein</surname><given-names>H</given-names></name></person-group><article-title>Cell cycle analysis of a cell proliferation-associated human nuclear antigen defined by the monoclonal antibody Ki-67</article-title><source>J Immunol</source><year>1984</year><volume>133</volume><issue>4</issue><fpage>1710</fpage><lpage>5</lpage><pub-id pub-id-type="pmid">6206131</pub-id></element-citation></ref><ref id="CR4"><label>4.</label><mixed-citation publication-type="other">Braun L, Mietzsch F, Seibold P, Schneeweiss A, Schirmacher P, Chang-Claude J, Sinn P, Aulmann S. Intrinsic breast cancer subtypes defined by estrogen receptor signalling-prognostic relevance of progesterone receptor loss. Mod Pathol. 2013;26(9):1161–71.</mixed-citation></ref><ref id="CR5"><label>5.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Maisonneuve</surname><given-names>P</given-names></name><name><surname>Disalvatore</surname><given-names>D</given-names></name><name><surname>Rotmensz</surname><given-names>N</given-names></name><name><surname>Curigliano</surname><given-names>G</given-names></name><name><surname>Colleoni</surname><given-names>M</given-names></name><name><surname>Dellapasqua</surname><given-names>S</given-names></name><name><surname>Pruneri</surname><given-names>G</given-names></name><name><surname>Mastropasqua</surname><given-names>MG</given-names></name><name><surname>Luini</surname><given-names>A</given-names></name><name><surname>Bassi</surname><given-names>F</given-names></name><etal></etal></person-group><article-title>Proposed new clinicopathological surrogate definitions of luminal A and luminal B (HER2-negative) intrinsic breast cancer subtypes</article-title><source>Breast Cancer Res</source><year>2014</year><volume>16</volume><issue>3</issue><fpage>R65</fpage><pub-id pub-id-type="doi">10.1186/bcr3679</pub-id><pub-id pub-id-type="pmid">24951027</pub-id></element-citation></ref><ref id="CR6"><label>6.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Peppercorn</surname><given-names>J</given-names></name><name><surname>Perou</surname><given-names>CM</given-names></name><name><surname>Carey</surname><given-names>LA</given-names></name></person-group><article-title>Molecular subtypes in breast cancer evaluation and management: divide and conquer</article-title><source>Cancer Invest</source><year>2008</year><volume>26</volume><issue>1</issue><fpage>1</fpage><lpage>10</lpage><pub-id pub-id-type="doi">10.1080/07357900701784238</pub-id><pub-id pub-id-type="pmid">18181038</pub-id></element-citation></ref><ref id="CR7"><label>7.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Hammond</surname><given-names>ME</given-names></name><name><surname>Hayes</surname><given-names>DF</given-names></name><name><surname>Dowsett</surname><given-names>M</given-names></name><name><surname>Allred</surname><given-names>DC</given-names></name><name><surname>Hagerty</surname><given-names>KL</given-names></name><name><surname>Badve</surname><given-names>S</given-names></name><name><surname>Fitzgibbons</surname><given-names>PL</given-names></name><name><surname>Francis</surname><given-names>G</given-names></name><name><surname>Goldstein</surname><given-names>NS</given-names></name><name><surname>Hayes</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>American Society of Clinical Oncology/College of American Pathologists guideline recommendations for immunohistochemical testing of estrogen and progesterone receptors in breast cancer</article-title><source>Arch Pathol Lab Med</source><year>2010</year><volume>134</volume><issue>6</issue><fpage>907</fpage><lpage>22</lpage><pub-id pub-id-type="pmid">20524868</pub-id></element-citation></ref><ref id="CR8"><label>8.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>de Gramont</surname><given-names>A</given-names></name><name><surname>Watson</surname><given-names>S</given-names></name><name><surname>Ellis</surname><given-names>LM</given-names></name><name><surname>Rodon</surname><given-names>J</given-names></name><name><surname>Tabernero</surname><given-names>J</given-names></name><name><surname>de Gramont</surname><given-names>A</given-names></name><name><surname>Hamilton</surname><given-names>SR</given-names></name></person-group><article-title>Pragmatic issues in biomarker evaluation for targeted therapies in cancer</article-title><source>Nat Rev Clin Oncol</source><year>2015</year><volume>12</volume><issue>4</issue><fpage>197</fpage><lpage>212</lpage><pub-id pub-id-type="doi">10.1038/nrclinonc.2014.202</pub-id><pub-id pub-id-type="pmid">25421275</pub-id></element-citation></ref><ref id="CR9"><label>9.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Harvey</surname><given-names>J</given-names></name><name><surname>Clark</surname><given-names>G</given-names></name><name><surname>Osborne</surname><given-names>C</given-names></name><name><surname>Allred</surname><given-names>D</given-names></name></person-group><article-title>Estrogen receptor status by immunohistochemistry is superior to the ligand-binding assay for predicting response to adjuvant endocrine therapy in breast cancer</article-title><source>J Clin Oncol</source><year>1999</year><volume>17</volume><issue>5</issue><fpage>1474</fpage><lpage>81</lpage><pub-id pub-id-type="pmid">10334533</pub-id></element-citation></ref><ref id="CR10"><label>10.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Luporsi</surname><given-names>E</given-names></name><name><surname>Andre</surname><given-names>F</given-names></name><name><surname>Spyratos</surname><given-names>F</given-names></name><name><surname>Martin</surname><given-names>PM</given-names></name><name><surname>Jacquemier</surname><given-names>J</given-names></name><name><surname>Penault-Llorca</surname><given-names>F</given-names></name><name><surname>Tubiana-Mathieu</surname><given-names>N</given-names></name><name><surname>Sigal-Zafrani</surname><given-names>B</given-names></name><name><surname>Arnould</surname><given-names>L</given-names></name><name><surname>Gompel</surname><given-names>A</given-names></name><etal></etal></person-group><article-title>Ki-67: level of evidence and methodological considerations for its role in the clinical management of breast cancer: analytical and critical review</article-title><source>Breast Cancer Res Treat</source><year>2012</year><volume>132</volume><issue>3</issue><fpage>895</fpage><lpage>915</lpage><pub-id pub-id-type="doi">10.1007/s10549-011-1837-z</pub-id><pub-id pub-id-type="pmid">22048814</pub-id></element-citation></ref><ref id="CR11"><label>11.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Polley</surname><given-names>MY</given-names></name><name><surname>Leung</surname><given-names>SC</given-names></name><name><surname>McShane</surname><given-names>LM</given-names></name><name><surname>Gao</surname><given-names>D</given-names></name><name><surname>Hugh</surname><given-names>JC</given-names></name><name><surname>Mastropasqua</surname><given-names>MG</given-names></name><name><surname>Viale</surname><given-names>G</given-names></name><name><surname>Zabaglo</surname><given-names>LA</given-names></name><name><surname>Penault-Llorca</surname><given-names>F</given-names></name><name><surname>Bartlett</surname><given-names>JM</given-names></name><etal></etal></person-group><article-title>An international Ki67 reproducibility study</article-title><source>J Natl Cancer Inst</source><year>2013</year><volume>105</volume><issue>24</issue><fpage>1897</fpage><lpage>906</lpage><pub-id pub-id-type="doi">10.1093/jnci/djt306</pub-id><pub-id pub-id-type="pmid">24203987</pub-id></element-citation></ref><ref id="CR12"><label>12.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Varga</surname><given-names>Z</given-names></name><name><surname>Diebold</surname><given-names>J</given-names></name><name><surname>Dommann-Scherrer</surname><given-names>C</given-names></name><name><surname>Frick</surname><given-names>H</given-names></name><name><surname>Kaup</surname><given-names>D</given-names></name><name><surname>Noske</surname><given-names>A</given-names></name><name><surname>Obermann</surname><given-names>E</given-names></name><name><surname>Ohlschlegel</surname><given-names>C</given-names></name><name><surname>Padberg</surname><given-names>B</given-names></name><name><surname>Rakozy</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>How reliable is Ki-67 immunohistochemistry in grade 2 breast carcinomas? A QA study of the Swiss working group of breast- and gynecopathologists</article-title><source>PLoS ONE</source><year>2012</year><volume>7</volume><issue>5</issue><fpage>e37379</fpage><pub-id pub-id-type="doi">10.1371/journal.pone.0037379</pub-id><pub-id pub-id-type="pmid">22662150</pub-id></element-citation></ref><ref id="CR13"><label>13.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Yerushalmi</surname><given-names>R</given-names></name><name><surname>Woods</surname><given-names>R</given-names></name><name><surname>Ravdin</surname><given-names>PM</given-names></name><name><surname>Hayes</surname><given-names>MM</given-names></name><name><surname>Gelmon</surname><given-names>KA</given-names></name></person-group><article-title>Ki67 in breast cancer: prognostic and predictive potential</article-title><source>Lancet Oncol</source><year>2010</year><volume>11</volume><issue>2</issue><fpage>174</fpage><lpage>83</lpage><pub-id pub-id-type="doi">10.1016/S1470-2045(09)70262-1</pub-id><pub-id pub-id-type="pmid">20152769</pub-id></element-citation></ref><ref id="CR14"><label>14.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Denkert</surname><given-names>C</given-names></name><name><surname>Loibl</surname><given-names>S</given-names></name><name><surname>Muller</surname><given-names>BM</given-names></name><name><surname>Eidtmann</surname><given-names>H</given-names></name><name><surname>Schmitt</surname><given-names>WD</given-names></name><name><surname>Eiermann</surname><given-names>W</given-names></name><name><surname>Gerber</surname><given-names>B</given-names></name><name><surname>Tesch</surname><given-names>H</given-names></name><name><surname>Hilfrich</surname><given-names>J</given-names></name><name><surname>Huober</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Ki67 levels as predictive and prognostic parameters in pretherapeutic breast cancer core biopsies: a translational investigation in the neoadjuvant GeparTrio trial</article-title><source>Ann Oncol</source><year>2013</year><volume>24</volume><issue>11</issue><fpage>2786</fpage><lpage>93</lpage><pub-id pub-id-type="doi">10.1093/annonc/mdt350</pub-id><pub-id pub-id-type="pmid">23970015</pub-id></element-citation></ref><ref id="CR15"><label>15.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Di Cataldo</surname><given-names>S</given-names></name><name><surname>Ficarra</surname><given-names>E</given-names></name><name><surname>Macii</surname><given-names>E</given-names></name></person-group><article-title>Computer-aided techniques for chromogenic immunohistochemistry: status and directions</article-title><source>Comput Biol Med</source><year>2012</year><volume>42</volume><issue>10</issue><fpage>1012</fpage><lpage>25</lpage><pub-id pub-id-type="doi">10.1016/j.compbiomed.2012.08.004</pub-id><pub-id pub-id-type="pmid">22980752</pub-id></element-citation></ref><ref id="CR16"><label>16.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Voros</surname><given-names>A</given-names></name><name><surname>Csorgo</surname><given-names>E</given-names></name><name><surname>Nyari</surname><given-names>T</given-names></name><name><surname>Cserni</surname><given-names>G</given-names></name></person-group><article-title>An intra- and interobserver reproducibility analysis of the Ki-67 proliferation marker assessment on core biopsies of breast cancer patients and its potential clinical implications</article-title><source>Pathobiology</source><year>2013</year><volume>80</volume><issue>3</issue><fpage>111</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1159/000343795</pub-id><pub-id pub-id-type="pmid">23258384</pub-id></element-citation></ref><ref id="CR17"><label>17.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Konsti</surname><given-names>J</given-names></name><name><surname>Lundin</surname><given-names>M</given-names></name><name><surname>Joensuu</surname><given-names>H</given-names></name><name><surname>Lehtimaki</surname><given-names>T</given-names></name><name><surname>Sihto</surname><given-names>H</given-names></name><name><surname>Holli</surname><given-names>K</given-names></name><name><surname>Turpeenniemi-Hujanen</surname><given-names>T</given-names></name><name><surname>Kataja</surname><given-names>V</given-names></name><name><surname>Sailas</surname><given-names>L</given-names></name><name><surname>Isola</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Development and evaluation of a virtual microscopy application for automated assessment of Ki-67 expression in breast cancer</article-title><source>BMC Clin Pathol</source><year>2011</year><volume>11</volume><fpage>3</fpage><pub-id pub-id-type="doi">10.1186/1472-6890-11-3</pub-id><pub-id pub-id-type="pmid">21262004</pub-id></element-citation></ref><ref id="CR18"><label>18.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cass</surname><given-names>JD</given-names></name><name><surname>Varma</surname><given-names>S</given-names></name><name><surname>Day</surname><given-names>AG</given-names></name><name><surname>Sangrar</surname><given-names>W</given-names></name><name><surname>Rajput</surname><given-names>AB</given-names></name><name><surname>Raptis</surname><given-names>LH</given-names></name><name><surname>Squire</surname><given-names>J</given-names></name><name><surname>Madarnas</surname><given-names>Y</given-names></name><name><surname>Sengupta</surname><given-names>SK</given-names></name><name><surname>Elliott</surname><given-names>BE</given-names></name></person-group><article-title>Automated quantitative analysis of p53, Cyclin D1, Ki67 and pERK expression in breast carcinoma does not differ from expert pathologist scoring and correlates with clinico-pathological characteristics</article-title><source>Cancers (Basel)</source><year>2012</year><volume>4</volume><issue>3</issue><fpage>725</fpage><lpage>42</lpage><pub-id pub-id-type="doi">10.3390/cancers4030725</pub-id><pub-id pub-id-type="pmid">24213463</pub-id></element-citation></ref><ref id="CR19"><label>19.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Abubakar</surname><given-names>M</given-names></name><name><surname>Howat</surname><given-names>WJ</given-names></name><name><surname>Daley</surname><given-names>F</given-names></name><name><surname>Zabaglo</surname><given-names>L</given-names></name><name><surname>McDuffus</surname><given-names>LA</given-names></name><name><surname>Blows</surname><given-names>F</given-names></name><name><surname>Coulson</surname><given-names>P</given-names></name><name><surname>Raza Ali</surname><given-names>H</given-names></name><name><surname>Benitez</surname><given-names>J</given-names></name><name><surname>Milne</surname><given-names>R</given-names></name><etal></etal></person-group><article-title>High-throughput automated scoring of Ki67 in breast cancer tissue microarrays from the Breast Cancer Association Consortium</article-title><source>J Pathol Clin Res</source><year>2016</year><volume>2</volume><issue>3</issue><fpage>138</fpage><lpage>53</lpage><pub-id pub-id-type="doi">10.1002/cjp2.42</pub-id><pub-id pub-id-type="pmid">27499923</pub-id></element-citation></ref><ref id="CR20"><label>20.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Noske</surname><given-names>A</given-names></name><name><surname>Loibl</surname><given-names>S</given-names></name><name><surname>Darb-Esfahani</surname><given-names>S</given-names></name><name><surname>Roller</surname><given-names>M</given-names></name><name><surname>Kronenwett</surname><given-names>R</given-names></name><name><surname>Muller</surname><given-names>BM</given-names></name><name><surname>Steffen</surname><given-names>J</given-names></name><name><surname>von Toerne</surname><given-names>C</given-names></name><name><surname>Wirtz</surname><given-names>R</given-names></name><name><surname>Baumann</surname><given-names>I</given-names></name><etal></etal></person-group><article-title>Comparison of different approaches for assessment of HER2 expression on protein and mRNA level: prediction of chemotherapy response in the neoadjuvant GeparTrio trial (NCT00544765)</article-title><source>Breast Cancer Res Treat</source><year>2011</year><volume>126</volume><issue>1</issue><fpage>109</fpage><lpage>17</lpage><pub-id pub-id-type="doi">10.1007/s10549-010-1316-y</pub-id><pub-id pub-id-type="pmid">21190079</pub-id></element-citation></ref><ref id="CR21"><label>21.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Bohmann</surname><given-names>K</given-names></name><name><surname>Hennig</surname><given-names>G</given-names></name><name><surname>Rogel</surname><given-names>U</given-names></name><name><surname>Poremba</surname><given-names>C</given-names></name><name><surname>Mueller</surname><given-names>BM</given-names></name><name><surname>Fritz</surname><given-names>P</given-names></name><name><surname>Stoerkel</surname><given-names>S</given-names></name><name><surname>Schaefer</surname><given-names>KL</given-names></name></person-group><article-title>RNA extraction from archival formalin-fixed paraffin-embedded tissue: a comparison of manual, semiautomated, and fully automated purification methods</article-title><source>Clin Chem</source><year>2009</year><volume>55</volume><issue>9</issue><fpage>1719</fpage><lpage>27</lpage><pub-id pub-id-type="doi">10.1373/clinchem.2008.122572</pub-id><pub-id pub-id-type="pmid">19617290</pub-id></element-citation></ref><ref id="CR22"><label>22.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Laible</surname><given-names>M</given-names></name><name><surname>Schlombs</surname><given-names>K</given-names></name><name><surname>Kaiser</surname><given-names>K</given-names></name><name><surname>Veltrup</surname><given-names>E</given-names></name><name><surname>Herlein</surname><given-names>S</given-names></name><name><surname>Lakis</surname><given-names>S</given-names></name><name><surname>Stohr</surname><given-names>R</given-names></name><name><surname>Eidt</surname><given-names>S</given-names></name><name><surname>Hartmann</surname><given-names>A</given-names></name><name><surname>Wirtz</surname><given-names>RM</given-names></name><etal></etal></person-group><article-title>Technical validation of an RT-qPCR in vitro diagnostic test system for the determination of breast cancer molecular subtypes by quantification of ERBB2, ESR1, PGR and MKI67 mRNA levels from formalin-fixed paraffin-embedded breast tumor specimens</article-title><source>BMC Cancer</source><year>2016</year><volume>16</volume><fpage>398</fpage><pub-id pub-id-type="doi">10.1186/s12885-016-2476-x</pub-id><pub-id pub-id-type="pmid">27389414</pub-id></element-citation></ref><ref id="CR23"><label>23.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Elsawaf</surname><given-names>Z</given-names></name><name><surname>Sinn</surname><given-names>HP</given-names></name><name><surname>Rom</surname><given-names>J</given-names></name><name><surname>Bermejo</surname><given-names>JL</given-names></name><name><surname>Schneeweiss</surname><given-names>A</given-names></name><name><surname>Aulmann</surname><given-names>S</given-names></name></person-group><article-title>Biological subtypes of triple-negative breast cancer are associated with distinct morphological changes and clinical behaviour</article-title><source>Breast</source><year>2013</year><volume>22</volume><issue>5</issue><fpage>986</fpage><lpage>92</lpage><pub-id pub-id-type="doi">10.1016/j.breast.2013.05.012</pub-id><pub-id pub-id-type="pmid">23806603</pub-id></element-citation></ref><ref id="CR24"><label>24.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Pepe</surname><given-names>MS</given-names></name><name><surname>Janes</surname><given-names>H</given-names></name><name><surname>Longton</surname><given-names>G</given-names></name><name><surname>Leisenring</surname><given-names>W</given-names></name><name><surname>Newcomb</surname><given-names>P</given-names></name></person-group><article-title>Limitations of the odds ratio in gauging the performance of a diagnostic, prognostic, or screening marker</article-title><source>Am J Epidemiol</source><year>2004</year><volume>159</volume><issue>9</issue><fpage>882</fpage><lpage>90</lpage><pub-id pub-id-type="doi">10.1093/aje/kwh101</pub-id><pub-id pub-id-type="pmid">15105181</pub-id></element-citation></ref><ref id="CR25"><label>25.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cook</surname><given-names>NR</given-names></name></person-group><article-title>Use and misuse of the receiver operating characteristic curve in risk prediction</article-title><source>Circulation</source><year>2007</year><volume>115</volume><issue>7</issue><fpage>928</fpage><lpage>35</lpage><pub-id pub-id-type="doi">10.1161/CIRCULATIONAHA.106.672402</pub-id><pub-id pub-id-type="pmid">17309939</pub-id></element-citation></ref><ref id="CR26"><label>26.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Guarneri</surname><given-names>V</given-names></name><name><surname>Broglio</surname><given-names>K</given-names></name><name><surname>Kau</surname><given-names>SW</given-names></name><name><surname>Cristofanilli</surname><given-names>M</given-names></name><name><surname>Buzdar</surname><given-names>AU</given-names></name><name><surname>Valero</surname><given-names>V</given-names></name><name><surname>Buchholz</surname><given-names>T</given-names></name><name><surname>Meric</surname><given-names>F</given-names></name><name><surname>Middleton</surname><given-names>L</given-names></name><name><surname>Hortobagyi</surname><given-names>GN</given-names></name><etal></etal></person-group><article-title>Prognostic value of pathologic complete response after primary chemotherapy in relation to hormone receptor status and other factors</article-title><source>J Clin Oncol</source><year>2006</year><volume>24</volume><issue>7</issue><fpage>1037</fpage><lpage>44</lpage><pub-id pub-id-type="doi">10.1200/JCO.2005.02.6914</pub-id><pub-id pub-id-type="pmid">16505422</pub-id></element-citation></ref><ref id="CR27"><label>27.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cortazar</surname><given-names>P</given-names></name><name><surname>Geyer</surname><given-names>CE</given-names><suffix>Jr</suffix></name></person-group><article-title>Pathological complete response in neoadjuvant treatment of breast cancer</article-title><source>Ann Surg Oncol</source><year>2015</year><volume>22</volume><issue>5</issue><fpage>1441</fpage><lpage>6</lpage><pub-id pub-id-type="doi">10.1245/s10434-015-4404-8</pub-id><pub-id pub-id-type="pmid">25727556</pub-id></element-citation></ref><ref id="CR28"><label>28.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Berruti</surname><given-names>A</given-names></name><name><surname>Generali</surname><given-names>D</given-names></name><name><surname>Kaufmann</surname><given-names>M</given-names></name><name><surname>Puztai</surname><given-names>L</given-names></name><name><surname>Curigliano</surname><given-names>G</given-names></name><name><surname>Aglietta</surname><given-names>M</given-names></name><name><surname>Gianni</surname><given-names>L</given-names></name><name><surname>Miller</surname><given-names>WR</given-names></name><name><surname>Untch</surname><given-names>M</given-names></name><name><surname>Sotiriou</surname><given-names>C</given-names></name><etal></etal></person-group><article-title>International expert consensus on primary systemic therapy in the management of early breast cancer: highlights of the fourth symposium on primary systemic therapy in the management of operable breast cancer, Cremona, Italy (2010)</article-title><source>J Natl Cancer Inst Monogr</source><year>2011</year><volume>2011</volume><issue>43</issue><fpage>147</fpage><lpage>51</lpage><pub-id pub-id-type="doi">10.1093/jncimonographs/lgr037</pub-id><pub-id pub-id-type="pmid">22043063</pub-id></element-citation></ref><ref id="CR29"><label>29.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mauri</surname><given-names>D</given-names></name><name><surname>Pavlidis</surname><given-names>N</given-names></name><name><surname>Ioannidis</surname><given-names>JP</given-names></name></person-group><article-title>Neoadjuvant versus adjuvant systemic treatment in breast cancer: a meta-analysis</article-title><source>J Natl Cancer Inst</source><year>2005</year><volume>97</volume><issue>3</issue><fpage>188</fpage><lpage>94</lpage><pub-id pub-id-type="doi">10.1093/jnci/dji021</pub-id><pub-id pub-id-type="pmid">15687361</pub-id></element-citation></ref><ref id="CR30"><label>30.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Fasching</surname><given-names>PA</given-names></name><name><surname>Heusinger</surname><given-names>K</given-names></name><name><surname>Haeberle</surname><given-names>L</given-names></name><name><surname>Niklos</surname><given-names>M</given-names></name><name><surname>Hein</surname><given-names>A</given-names></name><name><surname>Bayer</surname><given-names>CM</given-names></name><name><surname>Rauh</surname><given-names>C</given-names></name><name><surname>Schulz-Wendtland</surname><given-names>R</given-names></name><name><surname>Bani</surname><given-names>MR</given-names></name><name><surname>Schrauder</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Ki67, chemotherapy response, and prognosis in breast cancer patients receiving neoadjuvant treatment</article-title><source>BMC Cancer</source><year>2011</year><volume>11</volume><fpage>486</fpage><pub-id pub-id-type="doi">10.1186/1471-2407-11-486</pub-id><pub-id pub-id-type="pmid">22081974</pub-id></element-citation></ref><ref id="CR31"><label>31.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cheang</surname><given-names>MC</given-names></name><name><surname>Chia</surname><given-names>SK</given-names></name><name><surname>Voduc</surname><given-names>D</given-names></name><name><surname>Gao</surname><given-names>D</given-names></name><name><surname>Leung</surname><given-names>S</given-names></name><name><surname>Snider</surname><given-names>J</given-names></name><name><surname>Watson</surname><given-names>M</given-names></name><name><surname>Davies</surname><given-names>S</given-names></name><name><surname>Bernard</surname><given-names>PS</given-names></name><name><surname>Parker</surname><given-names>JS</given-names></name><etal></etal></person-group><article-title>Ki67 index, HER2 status, and prognosis of patients with luminal B breast cancer</article-title><source>J Natl Cancer Inst</source><year>2009</year><volume>101</volume><issue>10</issue><fpage>736</fpage><lpage>50</lpage><pub-id pub-id-type="doi">10.1093/jnci/djp082</pub-id><pub-id pub-id-type="pmid">19436038</pub-id></element-citation></ref><ref id="CR32"><label>32.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>von Minckwitz</surname><given-names>G</given-names></name><name><surname>Untch</surname><given-names>M</given-names></name><name><surname>Blohmer</surname><given-names>JU</given-names></name><name><surname>Costa</surname><given-names>SD</given-names></name><name><surname>Eidtmann</surname><given-names>H</given-names></name><name><surname>Fasching</surname><given-names>PA</given-names></name><name><surname>Gerber</surname><given-names>B</given-names></name><name><surname>Eiermann</surname><given-names>W</given-names></name><name><surname>Hilfrich</surname><given-names>J</given-names></name><name><surname>Huober</surname><given-names>J</given-names></name><etal></etal></person-group><article-title>Definition and impact of pathologic complete response on prognosis after neoadjuvant chemotherapy in various intrinsic breast cancer subtypes</article-title><source>J Clin Oncol</source><year>2012</year><volume>30</volume><issue>15</issue><fpage>1796</fpage><lpage>804</lpage><pub-id pub-id-type="doi">10.1200/JCO.2011.38.8595</pub-id><pub-id pub-id-type="pmid">22508812</pub-id></element-citation></ref><ref id="CR33"><label>33.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Potemski</surname><given-names>P</given-names></name><name><surname>Pluciennik</surname><given-names>E</given-names></name><name><surname>Bednarek</surname><given-names>AK</given-names></name><name><surname>Kusinska</surname><given-names>R</given-names></name><name><surname>Kubiak</surname><given-names>R</given-names></name><name><surname>Jesionek-Kupnicka</surname><given-names>D</given-names></name><name><surname>Watala</surname><given-names>C</given-names></name><name><surname>Kordek</surname><given-names>R</given-names></name></person-group><article-title>Ki-67 expression in operable breast cancer: a comparative study of immunostaining and a real-time RT-PCR assay</article-title><source>Pathol Res Pract</source><year>2006</year><volume>202</volume><issue>7</issue><fpage>491</fpage><lpage>5</lpage><pub-id pub-id-type="doi">10.1016/j.prp.2006.02.005</pub-id><pub-id pub-id-type="pmid">16678980</pub-id></element-citation></ref><ref id="CR34"><label>34.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Yamamoto</surname><given-names>S</given-names></name><name><surname>Ibusuki</surname><given-names>M</given-names></name><name><surname>Yamamoto</surname><given-names>Y</given-names></name><name><surname>Fu</surname><given-names>P</given-names></name><name><surname>Fujiwara</surname><given-names>S</given-names></name><name><surname>Murakami</surname><given-names>K</given-names></name><name><surname>Iwase</surname><given-names>H</given-names></name></person-group><article-title>Clinical relevance of Ki67 gene expression analysis using formalin-fixed paraffin-embedded breast cancer specimens</article-title><source>Breast Cancer</source><year>2013</year><volume>20</volume><issue>3</issue><fpage>262</fpage><lpage>70</lpage><pub-id pub-id-type="doi">10.1007/s12282-012-0332-7</pub-id><pub-id pub-id-type="pmid">22362219</pub-id></element-citation></ref><ref id="CR35"><label>35.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Niikura</surname><given-names>N</given-names></name><name><surname>Iwamoto</surname><given-names>T</given-names></name><name><surname>Masuda</surname><given-names>S</given-names></name><name><surname>Kumaki</surname><given-names>N</given-names></name><name><surname>Xiaoyan</surname><given-names>T</given-names></name><name><surname>Shirane</surname><given-names>M</given-names></name><name><surname>Mori</surname><given-names>K</given-names></name><name><surname>Tsuda</surname><given-names>B</given-names></name><name><surname>Okamura</surname><given-names>T</given-names></name><name><surname>Saito</surname><given-names>Y</given-names></name><etal></etal></person-group><article-title>Immunohistochemical Ki67 labeling index has similar proliferation predictive power to various gene signatures in breast cancer</article-title><source>Cancer Sci</source><year>2012</year><volume>103</volume><issue>8</issue><fpage>1508</fpage><lpage>12</lpage><pub-id pub-id-type="doi">10.1111/j.1349-7006.2012.02319.x</pub-id><pub-id pub-id-type="pmid">22537114</pub-id></element-citation></ref><ref id="CR36"><label>36.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Klauschen</surname><given-names>F</given-names></name><name><surname>Wienert</surname><given-names>S</given-names></name><name><surname>Schmitt</surname><given-names>WD</given-names></name><name><surname>Loibl</surname><given-names>S</given-names></name><name><surname>Gerber</surname><given-names>B</given-names></name><name><surname>Blohmer</surname><given-names>JU</given-names></name><name><surname>Huober</surname><given-names>J</given-names></name><name><surname>Rudiger</surname><given-names>T</given-names></name><name><surname>Erbstosser</surname><given-names>E</given-names></name><name><surname>Mehta</surname><given-names>K</given-names></name><etal></etal></person-group><article-title>Standardized Ki67 diagnostics using automated scoring--clinical validation in the GeparTrio breast cancer study</article-title><source>Clin Cancer Res</source><year>2015</year><volume>21</volume><issue>16</issue><fpage>3651</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1158/1078-0432.CCR-14-1283</pub-id><pub-id pub-id-type="pmid">25501130</pub-id></element-citation></ref><ref id="CR37"><label>37.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Mohammed</surname><given-names>ZM</given-names></name><name><surname>McMillan</surname><given-names>DC</given-names></name><name><surname>Elsberger</surname><given-names>B</given-names></name><name><surname>Going</surname><given-names>JJ</given-names></name><name><surname>Orange</surname><given-names>C</given-names></name><name><surname>Mallon</surname><given-names>E</given-names></name><name><surname>Doughty</surname><given-names>JC</given-names></name><name><surname>Edwards</surname><given-names>J</given-names></name></person-group><article-title>Comparison of visual and automated assessment of Ki-67 proliferative activity and their impact on outcome in primary operable invasive ductal breast cancer</article-title><source>Br J Cancer</source><year>2012</year><volume>106</volume><issue>2</issue><fpage>383</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1038/bjc.2011.569</pub-id><pub-id pub-id-type="pmid">22251968</pub-id></element-citation></ref><ref id="CR38"><label>38.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Gianni</surname><given-names>L</given-names></name><name><surname>Zambetti</surname><given-names>M</given-names></name><name><surname>Clark</surname><given-names>K</given-names></name><name><surname>Baker</surname><given-names>J</given-names></name><name><surname>Cronin</surname><given-names>M</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Mariani</surname><given-names>G</given-names></name><name><surname>Rodriguez</surname><given-names>J</given-names></name><name><surname>Carcangiu</surname><given-names>M</given-names></name><name><surname>Watson</surname><given-names>D</given-names></name><etal></etal></person-group><article-title>Gene expression profiles in paraffin-embedded core biopsy tissue predict response to chemotherapy in women with locally advanced breast cancer</article-title><source>J Clin Oncol</source><year>2005</year><volume>23</volume><issue>29</issue><fpage>7265</fpage><lpage>77</lpage><pub-id pub-id-type="doi">10.1200/JCO.2005.02.0818</pub-id><pub-id pub-id-type="pmid">16145055</pub-id></element-citation></ref><ref id="CR39"><label>39.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Parker</surname><given-names>JS</given-names></name><name><surname>Mullins</surname><given-names>M</given-names></name><name><surname>Cheang</surname><given-names>MC</given-names></name><name><surname>Leung</surname><given-names>S</given-names></name><name><surname>Voduc</surname><given-names>D</given-names></name><name><surname>Vickery</surname><given-names>T</given-names></name><name><surname>Davies</surname><given-names>S</given-names></name><name><surname>Fauron</surname><given-names>C</given-names></name><name><surname>He</surname><given-names>X</given-names></name><name><surname>Hu</surname><given-names>Z</given-names></name><etal></etal></person-group><article-title>Supervised risk predictor of breast cancer based on intrinsic subtypes</article-title><source>J Clin Oncol</source><year>2009</year><volume>27</volume><issue>8</issue><fpage>1160</fpage><lpage>7</lpage><pub-id pub-id-type="doi">10.1200/JCO.2008.18.1370</pub-id><pub-id pub-id-type="pmid">19204204</pub-id></element-citation></ref><ref id="CR40"><label>40.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Straver</surname><given-names>ME</given-names></name><name><surname>Glas</surname><given-names>AM</given-names></name><name><surname>Hannemann</surname><given-names>J</given-names></name><name><surname>Wesseling</surname><given-names>J</given-names></name><name><surname>van de Vijver</surname><given-names>MJ</given-names></name><name><surname>Rutgers</surname><given-names>EJ</given-names></name><name><surname>Vrancken Peeters</surname><given-names>MJ</given-names></name><name><surname>van Tinteren</surname><given-names>H</given-names></name><name><surname>Van't Veer</surname><given-names>LJ</given-names></name><name><surname>Rodenhuis</surname><given-names>S</given-names></name></person-group><article-title>The 70-gene signature as a response predictor for neoadjuvant chemotherapy in breast cancer</article-title><source>Breast Cancer Res Treat</source><year>2010</year><volume>119</volume><issue>3</issue><fpage>551</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1007/s10549-009-0333-1</pub-id><pub-id pub-id-type="pmid">19214742</pub-id></element-citation></ref><ref id="CR41"><label>41.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Cuzick</surname><given-names>J</given-names></name><name><surname>Dowsett</surname><given-names>M</given-names></name><name><surname>Pineda</surname><given-names>S</given-names></name><name><surname>Wale</surname><given-names>C</given-names></name><name><surname>Salter</surname><given-names>J</given-names></name><name><surname>Quinn</surname><given-names>E</given-names></name><name><surname>Zabaglo</surname><given-names>L</given-names></name><name><surname>Mallon</surname><given-names>E</given-names></name><name><surname>Green</surname><given-names>AR</given-names></name><name><surname>Ellis</surname><given-names>IO</given-names></name><etal></etal></person-group><article-title>Prognostic value of a combined estrogen receptor, progesterone receptor, Ki-67, and human epidermal growth factor receptor 2 immunohistochemical score and comparison with the Genomic Health recurrence score in early breast cancer</article-title><source>J Clin Oncol</source><year>2011</year><volume>29</volume><issue>32</issue><fpage>4273</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1200/JCO.2010.31.2835</pub-id><pub-id pub-id-type="pmid">21990413</pub-id></element-citation></ref><ref id="CR42"><label>42.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Tobin</surname><given-names>NP</given-names></name><name><surname>Harrell</surname><given-names>JC</given-names></name><name><surname>Lovrot</surname><given-names>J</given-names></name><name><surname>Egyhazi Brage</surname><given-names>S</given-names></name><name><surname>Frostvik Stolt</surname><given-names>M</given-names></name><name><surname>Carlsson</surname><given-names>L</given-names></name><name><surname>Einbeigi</surname><given-names>Z</given-names></name><name><surname>Linderholm</surname><given-names>B</given-names></name><name><surname>Loman</surname><given-names>N</given-names></name><name><surname>Malmberg</surname><given-names>M</given-names></name><etal></etal></person-group><article-title>Molecular subtype and tumor characteristics of breast cancer metastases as assessed by gene expression significantly influence patient post-relapse survival</article-title><source>Ann Oncol</source><year>2015</year><volume>26</volume><issue>1</issue><fpage>81</fpage><lpage>8</lpage><pub-id pub-id-type="doi">10.1093/annonc/mdu498</pub-id><pub-id pub-id-type="pmid">25361981</pub-id></element-citation></ref><ref id="CR43"><label>43.</label><element-citation publication-type="journal"><person-group person-group-type="author"><name><surname>Williams</surname><given-names>DJ</given-names></name><name><surname>Cohen</surname><given-names>C</given-names></name><name><surname>Darrow</surname><given-names>M</given-names></name><name><surname>Page</surname><given-names>AJ</given-names></name><name><surname>Chastain</surname><given-names>B</given-names></name><name><surname>Adams</surname><given-names>AL</given-names></name></person-group><article-title>Proliferation (Ki-67 and phosphohistone H3) and oncotype DX recurrence score in estrogen receptor-positive breast cancer</article-title><source>Appl Immunohistochem Mol Morphol</source><year>2011</year><volume>19</volume><issue>5</issue><fpage>431</fpage><lpage>6</lpage><pub-id pub-id-type="doi">10.1097/PAI.0b013e318206d23d</pub-id><pub-id pub-id-type="pmid">21297447</pub-id></element-citation></ref></ref-list></back></article>